-- MySQL dump 10.13  Distrib 5.7.27, for Linux (x86_64)
--
-- Host: localhost    Database: website
-- ------------------------------------------------------
-- Server version	5.7.27-0ubuntu0.18.04.1

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `rlp_action_log_config`
--

DROP TABLE IF EXISTS `rlp_action_log_config`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_action_log_config` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `type_title` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `type_alias` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `id_holder` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `title_holder` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `table_name` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `text_prefix` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=20 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_action_log_config`
--

LOCK TABLES `rlp_action_log_config` WRITE;
/*!40000 ALTER TABLE `rlp_action_log_config` DISABLE KEYS */;
INSERT INTO `rlp_action_log_config` VALUES (1,'article','com_content.article','id','title','#__content','PLG_ACTIONLOG_JOOMLA'),(2,'article','com_content.form','id','title','#__content','PLG_ACTIONLOG_JOOMLA'),(3,'banner','com_banners.banner','id','name','#__banners','PLG_ACTIONLOG_JOOMLA'),(4,'user_note','com_users.note','id','subject','#__user_notes','PLG_ACTIONLOG_JOOMLA'),(5,'media','com_media.file','','name','','PLG_ACTIONLOG_JOOMLA'),(6,'category','com_categories.category','id','title','#__categories','PLG_ACTIONLOG_JOOMLA'),(7,'menu','com_menus.menu','id','title','#__menu_types','PLG_ACTIONLOG_JOOMLA'),(8,'menu_item','com_menus.item','id','title','#__menu','PLG_ACTIONLOG_JOOMLA'),(9,'newsfeed','com_newsfeeds.newsfeed','id','name','#__newsfeeds','PLG_ACTIONLOG_JOOMLA'),(10,'link','com_redirect.link','id','old_url','#__redirect_links','PLG_ACTIONLOG_JOOMLA'),(11,'tag','com_tags.tag','id','title','#__tags','PLG_ACTIONLOG_JOOMLA'),(12,'style','com_templates.style','id','title','#__template_styles','PLG_ACTIONLOG_JOOMLA'),(13,'plugin','com_plugins.plugin','extension_id','name','#__extensions','PLG_ACTIONLOG_JOOMLA'),(14,'component_config','com_config.component','extension_id','name','','PLG_ACTIONLOG_JOOMLA'),(15,'contact','com_contact.contact','id','name','#__contact_details','PLG_ACTIONLOG_JOOMLA'),(16,'module','com_modules.module','id','title','#__modules','PLG_ACTIONLOG_JOOMLA'),(17,'access_level','com_users.level','id','title','#__viewlevels','PLG_ACTIONLOG_JOOMLA'),(18,'banner_client','com_banners.client','id','name','#__banner_clients','PLG_ACTIONLOG_JOOMLA'),(19,'application_config','com_config.application','','name','','PLG_ACTIONLOG_JOOMLA');
/*!40000 ALTER TABLE `rlp_action_log_config` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_action_logs`
--

DROP TABLE IF EXISTS `rlp_action_logs`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_action_logs` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `message_language_key` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `message` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `log_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `extension` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `user_id` int(11) NOT NULL DEFAULT '0',
  `item_id` int(11) NOT NULL DEFAULT '0',
  `ip_address` varchar(40) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '0.0.0.0',
  PRIMARY KEY (`id`),
  KEY `idx_user_id` (`user_id`),
  KEY `idx_user_id_logdate` (`user_id`,`log_date`),
  KEY `idx_user_id_extension` (`user_id`,`extension`),
  KEY `idx_extension_item_id` (`extension`,`item_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_action_logs`
--

LOCK TABLES `rlp_action_logs` WRITE;
/*!40000 ALTER TABLE `rlp_action_logs` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_action_logs` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_action_logs_extensions`
--

DROP TABLE IF EXISTS `rlp_action_logs_extensions`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_action_logs_extensions` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `extension` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=19 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_action_logs_extensions`
--

LOCK TABLES `rlp_action_logs_extensions` WRITE;
/*!40000 ALTER TABLE `rlp_action_logs_extensions` DISABLE KEYS */;
INSERT INTO `rlp_action_logs_extensions` VALUES (1,'com_banners'),(2,'com_cache'),(3,'com_categories'),(4,'com_config'),(5,'com_contact'),(6,'com_content'),(7,'com_installer'),(8,'com_media'),(9,'com_menus'),(10,'com_messages'),(11,'com_modules'),(12,'com_newsfeeds'),(13,'com_plugins'),(14,'com_redirect'),(15,'com_tags'),(16,'com_templates'),(17,'com_users'),(18,'com_checkin');
/*!40000 ALTER TABLE `rlp_action_logs_extensions` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_action_logs_users`
--

DROP TABLE IF EXISTS `rlp_action_logs_users`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_action_logs_users` (
  `user_id` int(11) unsigned NOT NULL,
  `notify` tinyint(1) unsigned NOT NULL,
  `extensions` text COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`user_id`),
  KEY `idx_notify` (`notify`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_action_logs_users`
--

LOCK TABLES `rlp_action_logs_users` WRITE;
/*!40000 ALTER TABLE `rlp_action_logs_users` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_action_logs_users` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_assets`
--

DROP TABLE IF EXISTS `rlp_assets`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_assets` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Primary Key',
  `parent_id` int(11) NOT NULL DEFAULT '0' COMMENT 'Nested set parent.',
  `lft` int(11) NOT NULL DEFAULT '0' COMMENT 'Nested set lft.',
  `rgt` int(11) NOT NULL DEFAULT '0' COMMENT 'Nested set rgt.',
  `level` int(10) unsigned NOT NULL COMMENT 'The cached level in the nested tree.',
  `name` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The unique name for the asset.\n',
  `title` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The descriptive title for the asset.',
  `rules` varchar(5120) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'JSON encoded access control.',
  PRIMARY KEY (`id`),
  UNIQUE KEY `idx_asset_name` (`name`),
  KEY `idx_lft_rgt` (`lft`,`rgt`),
  KEY `idx_parent_id` (`parent_id`)
) ENGINE=InnoDB AUTO_INCREMENT=790 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_assets`
--

LOCK TABLES `rlp_assets` WRITE;
/*!40000 ALTER TABLE `rlp_assets` DISABLE KEYS */;
INSERT INTO `rlp_assets` VALUES (1,0,0,473,0,'root.1','Root Asset','{\"core.login.site\":{\"6\":1,\"2\":1},\"core.login.admin\":{\"6\":1,\"10\":0},\"core.login.offline\":{\"6\":1,\"10\":0},\"core.admin\":{\"8\":1},\"core.manage\":{\"7\":1},\"core.create\":{\"6\":1,\"3\":1,\"10\":1},\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1},\"core.edit.own\":{\"6\":1,\"3\":1}}'),(2,1,1,2,1,'com_admin','com_admin','{}'),(3,1,3,6,1,'com_banners','com_banners','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1},\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(4,1,7,8,1,'com_cache','com_cache','{\"core.admin\":{\"7\":1},\"core.manage\":{\"7\":1}}'),(5,1,9,10,1,'com_checkin','com_checkin','{\"core.admin\":{\"7\":1},\"core.manage\":{\"7\":1}}'),(6,1,11,12,1,'com_config','com_config','{}'),(7,1,13,16,1,'com_contact','com_contact','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1},\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"core.edit.own\":[]}'),(8,1,17,252,1,'com_content','com_content','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(9,1,253,254,1,'com_cpanel','com_cpanel','{}'),(10,1,255,256,1,'com_installer','com_installer','{\"core.admin\":[],\"core.manage\":{\"7\":0},\"core.delete\":{\"7\":0},\"core.edit.state\":{\"7\":0}}'),(11,1,257,258,1,'com_languages','com_languages','{\"core.admin\":{\"7\":1},\"core.manage\":[],\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(12,1,259,260,1,'com_login','com_login','{}'),(13,1,261,262,1,'com_mailto','com_mailto','{}'),(14,1,263,264,1,'com_massmail','com_massmail','{}'),(15,1,265,266,1,'com_media','com_media','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":{\"5\":1}}'),(16,1,267,274,1,'com_menus','com_menus','{\"core.admin\":{\"7\":1},\"core.manage\":[],\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(17,1,275,276,1,'com_messages','com_messages','{\"core.admin\":{\"7\":1},\"core.manage\":{\"7\":1}}'),(18,1,277,398,1,'com_modules','com_modules','{\"core.admin\":{\"7\":1},\"core.manage\":[],\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(19,1,399,402,1,'com_newsfeeds','com_newsfeeds','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1},\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"core.edit.own\":[]}'),(20,1,403,404,1,'com_plugins','com_plugins','{\"core.admin\":{\"7\":1},\"core.manage\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(21,1,405,406,1,'com_redirect','com_redirect','{\"core.admin\":{\"7\":1},\"core.manage\":[]}'),(22,1,407,408,1,'com_search','com_search','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1}}'),(23,1,409,410,1,'com_templates','com_templates','{\"core.admin\":{\"7\":1},\"core.manage\":[],\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(24,1,411,414,1,'com_users','com_users','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":[],\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(26,1,415,416,1,'com_wrapper','com_wrapper','{}'),(27,8,18,31,2,'com_content.category.2','Uncategorised','{\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"core.edit.own\":[]}'),(28,3,4,5,2,'com_banners.category.3','Uncategorised','{\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(29,7,14,15,2,'com_contact.category.4','Uncategorised','{\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"core.edit.own\":[]}'),(30,19,400,401,2,'com_newsfeeds.category.5','Uncategorised','{\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"core.edit.own\":[]}'),(32,24,412,413,1,'com_users.category.7','Uncategorised','{\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(33,1,417,418,1,'com_finder','com_finder','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1},\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(34,1,419,420,1,'com_joomlaupdate','com_joomlaupdate','{\"core.admin\":[],\"core.manage\":[],\"core.delete\":[],\"core.edit.state\":[]}'),(35,1,421,422,1,'com_tags','com_tags','{\"core.admin\":[],\"core.manage\":[],\"core.manage\":[],\"core.delete\":[],\"core.edit.state\":[]}'),(36,1,423,424,1,'com_contenthistory','com_contenthistory','{}'),(37,1,425,426,1,'com_ajax','com_ajax','{}'),(38,1,427,428,1,'com_postinstall','com_postinstall','{}'),(39,18,278,279,2,'com_modules.module.1','Main Menu','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(40,18,280,281,2,'com_modules.module.2','Login','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(41,18,282,283,2,'com_modules.module.3','Popular Articles','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(42,18,284,285,2,'com_modules.module.4','Recently Added Articles','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(43,18,286,287,2,'com_modules.module.8','Toolbar','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(44,18,288,289,2,'com_modules.module.9','Quick Icons','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(45,18,290,291,2,'com_modules.module.10','Logged-in Users','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(46,18,292,293,2,'com_modules.module.12','Admin Menu','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(47,18,294,295,2,'com_modules.module.13','Admin Submenu','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(48,18,296,297,2,'com_modules.module.14','User Status','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(49,18,298,299,2,'com_modules.module.15','Title','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(51,18,300,301,2,'com_modules.module.17','Breadcrumb','{}'),(52,18,302,303,2,'com_modules.module.79','Multilanguage status','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(53,18,304,305,2,'com_modules.module.86','Joomla Version','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(55,18,306,307,2,'com_modules.module.88','Site Information','{\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1}}'),(56,18,308,309,2,'com_modules.module.89','Release News','{\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1}}'),(63,8,32,33,2,'com_content.category.8','Home','{\"core.create\":{\"6\":1,\"3\":1},\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1},\"core.edit.own\":{\"6\":1,\"3\":1}}'),(69,8,34,49,2,'com_content.category.14','Docs','{\"core.create\":{\"6\":1,\"3\":1},\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1},\"core.edit.own\":{\"6\":1,\"3\":1}}'),(296,18,310,310,2,'com_modules.module.300','Image Center','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(483,18,312,313,2,'com_modules.module.475','FavGlyph - Home Page','{}'),(487,18,314,315,2,'com_modules.module.479','FavSocial - Reactome Social Medias','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(505,18,316,317,2,'com_modules.module.484','Use Reactome Graph Database in your Project','{\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1}}'),(518,18,318,319,2,'com_modules.module.496','Tweets','{}'),(521,18,320,321,2,'com_modules.module.499','Find Reactions, Proteins and Pathways','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(522,18,322,323,2,'com_modules.module.500','Notice','{\"core.edit\":{\"10\":0}}'),(523,1,429,430,1,'com_jce','COM_JCE','{}'),(524,8,50,151,2,'com_content.category.25','About','{}'),(525,524,51,52,3,'com_content.article.11','What is Reactome ?','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(526,524,53,138,3,'com_content.category.26','News','{}'),(528,524,139,140,3,'com_content.article.13','Reactome Team','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(529,524,141,142,3,'com_content.article.14','Scientific Advisory Board','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(530,524,147,148,3,'com_content.article.15','Other Reactomes','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(531,524,143,144,3,'com_content.article.16','License Agreement','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(532,524,145,146,3,'com_content.article.17','Disclaimer','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(537,637,231,232,3,'com_content.article.18','Downloads','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(539,18,324,325,2,'com_modules.module.505','Why Reactome','{\"module.edit.frontend\":{\"11\":1}}'),(544,18,326,327,2,'com_modules.module.508','Latest News','{}'),(548,1,431,432,1,'com_fields','com_fields','{}'),(549,1,433,434,1,'com_associations','com_associations','{}'),(552,16,268,269,2,'com_menus.menu.1','Main Menu','{}'),(555,8,152,171,2,'com_content.category.27','Community','{}'),(556,555,153,154,3,'com_content.article.22','Outreach','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(557,555,155,156,3,'com_content.article.23','Events','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(558,555,157,158,3,'com_content.article.24','Publications','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(559,555,159,160,3,'com_content.article.25','Training Materials','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(560,555,161,162,3,'com_content.article.26','Partners','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(561,555,163,164,3,'com_content.article.27','Resources Guide','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(562,69,35,36,3,'com_content.article.28','Linking to us','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(563,69,37,38,3,'com_content.article.29','Referencing our Publications','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(566,1,435,436,1,'com_easyjoomlabackup','COM_EASYJOOMLABACKUP','{}'),(568,16,270,271,2,'com_menus.menu.2','User Menu','{}'),(571,18,328,329,2,'com_modules.module.518','Maintop Blank Space','{}'),(572,18,330,331,2,'com_modules.module.519','Find Reactions, Proteins and Pathways - Others','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(573,8,172,195,2,'com_content.category.28','User Guide','{}'),(586,18,332,333,2,'com_modules.module.523','User Guide Anchor Menu','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(587,573,173,174,3,'com_content.article.39','User Guide','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(591,18,334,335,2,'com_modules.module.524','Articles Sidebar Menu','{}'),(592,18,336,337,2,'com_modules.module.525','Icon Library','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(594,27,19,20,3,'com_content.article.43','404','{\"core.admin\":{\"7\":1},\"core.options\":{},\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":{},\"core.edit.own\":{},\"core.edit\":{\"4\":0},\"core.edit.state\":{\"5\":0}}'),(595,18,338,339,2,'com_modules.module.526','[TEMPLATE] data-content / swagger placeholder','{}'),(596,18,340,341,2,'com_modules.module.527','[TEMPLATE] search placeholder','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(599,16,272,273,2,'com_menus.menu.4','Hidden Menu','{}'),(603,18,342,343,2,'com_modules.module.533','Footer','{}'),(604,573,175,176,3,'com_content.article.44','ReactomeFIVIz','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(605,606,197,198,3,'com_content.article.45','Graph Database','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(606,8,196,221,2,'com_content.category.29','Developers Guide','{}'),(612,613,223,224,3,'com_content.article.46','Teleconferences','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(613,8,222,227,2,'com_content.category.30','Reactome Staff','{}'),(615,18,344,345,2,'com_modules.module.540','Developers Zone - Row 1','{}'),(619,606,199,200,3,'com_content.article.47','Analysis Service','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(620,606,201,202,3,'com_content.article.48','Content Service','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(621,606,203,204,3,'com_content.article.49','Pathways Overview','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(622,606,205,206,3,'com_content.article.50','Pathway Diagrams','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(625,69,39,40,3,'com_content.article.53','Data Model','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(626,8,228,229,2,'com_content.category.31','Release','{}'),(627,18,346,347,2,'com_modules.module.544','Documentation','{}'),(628,606,207,208,3,'com_content.article.54','Tutorial: Extracting participating molecules using the Graph Database','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(629,1,437,438,1,'#__ucm_content.1','#__ucm_content.1','{}'),(630,69,41,42,3,'com_content.article.55','Icons Info ','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(631,606,209,210,3,'com_content.article.56','Pathways Overview JS','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(632,606,211,212,3,'com_content.article.57','Pathways Overview GWT','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(633,606,213,214,3,'com_content.article.58','Diagram JS','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(634,606,215,216,3,'com_content.article.59','Diagram GWT','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(637,8,230,237,2,'com_content.category.32','Download','{}'),(638,702,239,240,3,'com_content.article.60','Identifiers Mapping','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(639,27,21,22,3,'com_content.article.61','Pathway Information','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(640,702,249,250,3,'com_content.article.62','Tab-Delimiter Format','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(641,69,47,48,3,'com_content.article.63','Curator Tool (version 3.3, build 98, August 6, 2019)','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(647,18,348,349,2,'com_modules.module.551','Partners','{}'),(651,18,350,351,2,'com_modules.module.555','Do you need help ?','{}'),(652,18,352,353,2,'com_modules.module.556','API and Data access','{}'),(655,18,354,355,2,'com_modules.module.559','Pathway Diagram - Dev Guide, 4 buttons','{}'),(656,18,356,357,2,'com_modules.module.560','Pathway Overview - Dev Guide, 4 buttons','{}'),(657,18,358,359,2,'com_modules.module.561','Pathway Diagram - Dev Guide - Get Started, 2 buttons ','{}'),(659,18,360,361,2,'com_modules.module.563','Pathway Diagram - Dev Guide - API, 2 buttons','{}'),(660,18,362,363,2,'com_modules.module.564','Pathway Overview - Dev Guide - Get Started, 2 buttons','{}'),(661,18,364,365,2,'com_modules.module.565','Pathway Overview - Dev Guide - API, 2 buttons','{}'),(662,18,366,367,2,'com_modules.module.566','Pathway Overview GWT - Dev Guide, 4 buttons','{}'),(663,18,368,369,2,'com_modules.module.567','Pathway Diagram GWT - Dev Guide, 4 buttons','{}'),(664,18,370,371,2,'com_modules.module.568','Content Service - Dev Guide, 4 buttons','{}'),(665,18,372,373,2,'com_modules.module.569','Analysis Service - Dev Guide, 4 buttons','{}'),(666,18,374,375,2,'com_modules.module.570','Graph Database - Dev Guide, 4 buttons','{}'),(667,18,376,377,2,'com_modules.module.571','Developers Zone - Row 2','{}'),(668,27,23,24,3,'com_content.article.64','Developer\'s Zone','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(673,18,378,379,2,'com_modules.module.573','Icon Lib Menu','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(674,18,380,381,2,'com_modules.module.574','Version 70 released on September 9, 2019','{\"module.edit.frontend\":{\"11\":1}}'),(675,27,25,26,3,'com_content.article.66','Statistics','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(676,27,27,28,3,'com_content.article.67','ORCID Integration Project','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(681,18,382,383,2,'com_modules.module.579','Footer (For Small Devices)','{}'),(682,69,43,44,3,'com_content.article.68','Computationally Inferred Events','{}'),(683,555,165,166,3,'com_content.article.69','Papers citing us','{}'),(684,69,45,46,3,'com_content.article.70','External Identifiers','{}'),(685,526,54,55,4,'com_content.article.71','Version 57 Released','{}'),(686,526,72,73,4,'com_content.article.72','Version 58 Released','{}'),(687,526,56,57,4,'com_content.article.73','Reactome celebrates release of 10,000th annotated protein','{}'),(688,526,58,59,4,'com_content.article.74','Version 59 Released','{}'),(689,526,60,61,4,'com_content.article.75','New Reactome Publication','{}'),(690,526,62,63,4,'com_content.article.76','New Reactome Paper published','{}'),(691,526,64,65,4,'com_content.article.77','Version 60 Released','{}'),(692,526,66,67,4,'com_content.article.78','Version 61 Released','{}'),(693,526,68,69,4,'com_content.article.79','New Protein-protein Interaction files','{}'),(694,526,70,71,4,'com_content.article.80','New SBML Level 3 Version 1 export is now available','{}'),(695,573,177,178,3,'com_content.article.81','The Pathway Browser','{}'),(696,573,179,180,3,'com_content.article.82','Details Panel','{}'),(697,573,181,182,3,'com_content.article.83','Searching Reactome','{}'),(698,573,183,184,3,'com_content.article.84','Analysis Tools','{}'),(699,573,185,186,3,'com_content.article.85','Diseases','{}'),(700,18,384,385,2,'com_modules.module.580','User Guide Glyphs','{\"module.edit.frontend\":{\"5\":1}}'),(701,702,241,242,3,'com_content.article.86','Physical Entity Identifiers Mapping','{}'),(702,8,238,251,2,'com_content.category.33','PopUps and Modals','{}'),(703,702,243,244,3,'com_content.article.87','Reaction PMIDs','{}'),(704,702,245,246,3,'com_content.article.88','Reactome to OMIM','{}'),(705,702,247,248,3,'com_content.article.89','Protein Role Reaction','{}'),(709,526,74,75,4,'com_content.article.90','Version 62 Released','{}'),(710,1,439,440,1,'#__ucm_content.2','#__ucm_content.2','{}'),(712,18,386,387,2,'com_modules.module.581','Warning','{\"core.edit\":{\"10\":0},\"core.edit.state\":{\"11\":1},\"module.edit.frontend\":{\"11\":1}}'),(713,18,388,389,2,'com_modules.module.582','Why Reactome','{}'),(714,524,149,150,3,'com_content.article.91','Introducing our Logo','{}'),(715,18,390,391,2,'com_modules.module.583','[TEMPLATE] CGI placeholder','{}'),(718,526,76,77,4,'com_content.article.93','New responsive website with a fresh look','{}'),(719,526,78,79,4,'com_content.article.94','Proudly introducing our new logo','{}'),(720,18,392,393,2,'com_modules.module.584','Synchronization Tool','{}'),(721,526,80,81,4,'com_content.article.95','New Reactome Publication published in 2018 NAR Database Issue','{}'),(722,606,217,218,3,'com_content.article.96','Diagram exporter','{}'),(723,1,441,442,1,'#__ucm_content.3','#__ucm_content.3','{}'),(724,1,443,444,1,'#__ucm_content.4','#__ucm_content.4','{}'),(725,1,445,446,1,'#__ucm_content.5','#__ucm_content.5','{}'),(726,1,447,448,1,'#__ucm_content.6','#__ucm_content.6','{}'),(727,1,449,450,1,'#__ucm_content.7','#__ucm_content.7','{}'),(728,526,82,83,4,'com_content.article.97','Updated License Agreement','{}'),(729,526,84,85,4,'com_content.article.98','Version 63 Released','{}'),(731,1,451,452,1,'#__ucm_content.8','#__ucm_content.8','{}'),(732,613,225,226,3,'com_content.article.99','Help','{}'),(733,606,219,220,3,'com_content.article.100','Pathway Diagrams Specifications','{}'),(736,526,86,87,4,'com_content.article.102','Version 64 Released','{}'),(737,1,453,454,1,'#__ucm_content.9','#__ucm_content.9','{}'),(738,18,394,395,2,'com_modules.module.585','Elixir Banner','{}'),(739,526,88,89,4,'com_content.article.103','Version 65 Released','{}'),(740,1,455,456,1,'#__ucm_content.10','#__ucm_content.10','{}'),(741,573,187,188,3,'com_content.article.104','Searching Reactome - New Search','{}'),(742,526,90,91,4,'com_content.article.105','Our data now available through Google Dataset Search','{}'),(743,526,92,93,4,'com_content.article.106','Version 66 Released','{}'),(744,1,457,458,1,'#__ucm_content.11','#__ucm_content.11','{}'),(745,526,94,95,4,'com_content.article.107','New tool to generate analysis PDF reports','{}'),(747,526,96,97,4,'com_content.article.109','New advanced search available in our PathwayBrowser','{}'),(748,526,98,99,4,'com_content.article.110','SBGN files revamp','{}'),(749,526,100,101,4,'com_content.article.111','International Open Access Week: Analysis Tools','{}'),(750,526,102,103,4,'com_content.article.112','International Open Access Week: Network Visualization & Analysis Tool','{}'),(751,526,104,105,4,'com_content.article.113','International Open Access Week: Interactive illustrations and icon library','{}'),(752,526,106,107,4,'com_content.article.114','International Open Access Week: GraphDB and Content Service','{}'),(753,526,108,109,4,'com_content.article.115','Performing GSEA analysis with ReactomeFIViz app','{}'),(754,526,110,111,4,'com_content.article.116','Version 67 Released','{}'),(755,637,233,234,3,'com_content.article.117','Icon Mapping','{}'),(756,637,235,236,3,'com_content.article.118','Icons Mapping Download','{}'),(757,1,459,460,1,'#__ucm_content.12','#__ucm_content.12','{}'),(758,1,461,462,1,'com_privacy','com_privacy','{}'),(759,1,463,464,1,'com_actionlogs','com_actionlogs','{}'),(760,526,112,113,4,'com_content.article.119','New reactions layout with nested compartments','{}'),(761,526,114,115,4,'com_content.article.120','New tool to classify pathways into tissue types based on protein expression','{}'),(762,526,116,117,4,'com_content.article.121','Download our biological pathway knowledge in a PDF book','{}'),(765,526,118,119,4,'com_content.article.124','Deprecated support for Reactome RESTful API','{}'),(766,526,120,121,4,'com_content.article.125','Version 68 Released','{}'),(767,1,465,466,1,'#__ucm_content.13','#__ucm_content.13','{}'),(768,526,122,123,4,'com_content.article.126','PANTHER resource now used to identify orthologs of curated human proteins','{}'),(769,526,124,125,4,'com_content.article.127','Come to our release 68 webinar to learn more about Reactome','{}'),(770,1,467,468,1,'#__ucm_content.14','#__ucm_content.14','{}'),(772,18,396,397,2,'com_modules.module.586','EHLD / Icon Library (Specs & Guidelines)','{}'),(773,27,29,30,3,'com_content.article.129','EHLD Specs & Guidelines','{}'),(774,573,191,192,3,'com_content.article.130','Icons Library Guidelines','{}'),(777,526,126,127,4,'com_content.article.133','ReactomeFI Network v2018 and FIViz App v2.7.0 Released','{}'),(778,573,189,190,3,'com_content.article.134','Documentation','{}'),(780,526,128,129,4,'com_content.article.136','Season of Docs','{}'),(781,526,130,131,4,'com_content.article.137','Version 69 Released','{}'),(782,1,469,470,1,'#__ucm_content.15','#__ucm_content.15','{}'),(783,526,132,133,4,'com_content.article.138','ORCID: Claim your works','{}'),(784,573,193,194,3,'com_content.article.139','Tutorial: How to claim your works','{}'),(785,555,167,168,3,'com_content.article.140','External Contribution','{}'),(786,526,134,135,4,'com_content.article.141','ReacFoam: Genome-wide pathway overview based on Voronoi tessellation','{}'),(787,526,136,137,4,'com_content.article.142','Version 70 Released','{}'),(788,1,471,472,1,'#__ucm_content.16','#__ucm_content.16','{}'),(789,555,169,170,3,'com_content.article.143','FAQ for prospective Reactome reviewers and authors','{}');
/*!40000 ALTER TABLE `rlp_assets` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_associations`
--

DROP TABLE IF EXISTS `rlp_associations`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_associations` (
  `id` int(11) NOT NULL COMMENT 'A reference to the associated item.',
  `context` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The context of the associated item.',
  `key` char(32) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The key for the association computed from an md5 on associated ids.',
  PRIMARY KEY (`context`,`id`),
  KEY `idx_key` (`key`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_associations`
--

LOCK TABLES `rlp_associations` WRITE;
/*!40000 ALTER TABLE `rlp_associations` DISABLE KEYS */;
INSERT INTO `rlp_associations` VALUES (208,'com_menus.item','d8de03296c20dc0e00a095d1bdf9d908'),(209,'com_menus.item','d8de03296c20dc0e00a095d1bdf9d908'),(214,'com_menus.item','d8de03296c20dc0e00a095d1bdf9d908');
/*!40000 ALTER TABLE `rlp_associations` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_banner_clients`
--

DROP TABLE IF EXISTS `rlp_banner_clients`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_banner_clients` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `contact` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `email` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `extrainfo` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `state` tinyint(3) NOT NULL DEFAULT '0',
  `checked_out` int(10) unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `metakey` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `own_prefix` tinyint(4) NOT NULL DEFAULT '0',
  `metakey_prefix` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `purchase_type` tinyint(4) NOT NULL DEFAULT '-1',
  `track_clicks` tinyint(4) NOT NULL DEFAULT '-1',
  `track_impressions` tinyint(4) NOT NULL DEFAULT '-1',
  PRIMARY KEY (`id`),
  KEY `idx_own_prefix` (`own_prefix`),
  KEY `idx_metakey_prefix` (`metakey_prefix`(100))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_banner_clients`
--

LOCK TABLES `rlp_banner_clients` WRITE;
/*!40000 ALTER TABLE `rlp_banner_clients` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_banner_clients` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_banner_tracks`
--

DROP TABLE IF EXISTS `rlp_banner_tracks`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_banner_tracks` (
  `track_date` datetime NOT NULL,
  `track_type` int(10) unsigned NOT NULL,
  `banner_id` int(10) unsigned NOT NULL,
  `count` int(10) unsigned NOT NULL DEFAULT '0',
  PRIMARY KEY (`track_date`,`track_type`,`banner_id`),
  KEY `idx_track_date` (`track_date`),
  KEY `idx_track_type` (`track_type`),
  KEY `idx_banner_id` (`banner_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_banner_tracks`
--

LOCK TABLES `rlp_banner_tracks` WRITE;
/*!40000 ALTER TABLE `rlp_banner_tracks` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_banner_tracks` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_banners`
--

DROP TABLE IF EXISTS `rlp_banners`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_banners` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `cid` int(11) NOT NULL DEFAULT '0',
  `type` int(11) NOT NULL DEFAULT '0',
  `name` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `imptotal` int(11) NOT NULL DEFAULT '0',
  `impmade` int(11) NOT NULL DEFAULT '0',
  `clicks` int(11) NOT NULL DEFAULT '0',
  `clickurl` varchar(200) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `state` tinyint(3) NOT NULL DEFAULT '0',
  `catid` int(10) unsigned NOT NULL DEFAULT '0',
  `description` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `custombannercode` varchar(2048) COLLATE utf8mb4_unicode_ci NOT NULL,
  `sticky` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `ordering` int(11) NOT NULL DEFAULT '0',
  `metakey` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `params` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `own_prefix` tinyint(1) NOT NULL DEFAULT '0',
  `metakey_prefix` varchar(400) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `purchase_type` tinyint(4) NOT NULL DEFAULT '-1',
  `track_clicks` tinyint(4) NOT NULL DEFAULT '-1',
  `track_impressions` tinyint(4) NOT NULL DEFAULT '-1',
  `checked_out` int(10) unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `reset` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `language` char(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `created_by` int(10) unsigned NOT NULL DEFAULT '0',
  `created_by_alias` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int(10) unsigned NOT NULL DEFAULT '0',
  `version` int(10) unsigned NOT NULL DEFAULT '1',
  PRIMARY KEY (`id`),
  KEY `idx_state` (`state`),
  KEY `idx_own_prefix` (`own_prefix`),
  KEY `idx_banner_catid` (`catid`),
  KEY `idx_language` (`language`),
  KEY `idx_metakey_prefix` (`metakey_prefix`(100))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_banners`
--

LOCK TABLES `rlp_banners` WRITE;
/*!40000 ALTER TABLE `rlp_banners` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_banners` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_categories`
--

DROP TABLE IF EXISTS `rlp_categories`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_categories` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `asset_id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'FK to the #__assets table.',
  `parent_id` int(10) unsigned NOT NULL DEFAULT '0',
  `lft` int(11) NOT NULL DEFAULT '0',
  `rgt` int(11) NOT NULL DEFAULT '0',
  `level` int(10) unsigned NOT NULL DEFAULT '0',
  `path` varchar(400) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `extension` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `title` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `note` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `description` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `published` tinyint(1) NOT NULL DEFAULT '0',
  `checked_out` int(11) unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `access` int(10) unsigned NOT NULL DEFAULT '0',
  `params` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadesc` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'The meta description for the page.',
  `metakey` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'The meta keywords for the page.',
  `metadata` varchar(2048) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'JSON encoded metadata properties.',
  `created_user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `created_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `modified_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `hits` int(10) unsigned NOT NULL DEFAULT '0',
  `language` char(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `version` int(10) unsigned NOT NULL DEFAULT '1',
  PRIMARY KEY (`id`),
  KEY `cat_idx` (`extension`,`published`,`access`),
  KEY `idx_access` (`access`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_left_right` (`lft`,`rgt`),
  KEY `idx_language` (`language`),
  KEY `idx_path` (`path`(100)),
  KEY `idx_alias` (`alias`(100))
) ENGINE=InnoDB AUTO_INCREMENT=34 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_categories`
--

LOCK TABLES `rlp_categories` WRITE;
/*!40000 ALTER TABLE `rlp_categories` DISABLE KEYS */;
INSERT INTO `rlp_categories` VALUES (1,0,0,0,33,0,'','system','ROOT','root','','',1,0,'0000-00-00 00:00:00',1,'{}','','','{}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(2,27,1,1,2,1,'uncategorised','com_content','Uncategorised','uncategorised','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(3,28,1,19,20,1,'uncategorised','com_banners','Uncategorised','uncategorised','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(4,29,1,21,22,1,'uncategorised','com_contact','Uncategorised','uncategorised','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(5,30,1,23,24,1,'uncategorised','com_newsfeeds','Uncategorised','uncategorised','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(7,32,1,25,26,1,'uncategorised','com_users','Uncategorised','uncategorised','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(8,63,1,3,4,1,'home','com_content','Home','home','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2015-04-12 12:45:31',0,'2015-04-12 12:45:31',0,'*',1),(14,69,1,11,12,1,'docs','com_content','Docs','docs','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2015-04-12 13:36:52',851,'2015-06-05 11:15:55',0,'*',1),(25,524,1,5,8,1,'about','com_content','About','about','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-04-20 15:00:05',0,'2017-04-20 15:00:05',0,'*',1),(26,526,25,6,7,2,'about/news','com_content','News','news','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-04-20 15:08:31',0,'2017-04-20 15:08:31',0,'*',1),(27,555,1,9,10,1,'community','com_content','Community','community','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-04-27 15:43:12',0,'2017-04-27 15:43:12',0,'*',1),(28,573,1,13,14,1,'user-guide','com_content','User Guide','user-guide','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-05-08 14:42:40',0,'2017-05-08 14:42:40',0,'*',1),(29,606,1,15,16,1,'developers-guide','com_content','Developers Guide','developers-guide','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-05-26 08:41:01',0,'2017-05-26 08:41:01',0,'*',1),(30,613,1,17,18,1,'reactome-staff','com_content','Reactome Staff','reactome-staff','','',1,0,'0000-00-00 00:00:00',2,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-05-30 09:59:41',0,'2017-05-30 09:59:41',0,'*',1),(31,626,1,27,28,1,'release','com_content','Release','release','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-06-02 12:44:42',0,'2017-06-02 12:44:42',0,'*',1),(32,637,1,29,30,1,'download','com_content','Download','download','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-06-20 11:18:43',0,'2017-06-20 11:18:43',0,'*',1),(33,702,1,31,32,1,'popups-and-modals','com_content','PopUps and Modals','popups-and-modals','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"_:blog\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-09-25 13:53:42',0,'2017-09-25 13:53:42',0,'*',1);
/*!40000 ALTER TABLE `rlp_categories` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_contact_details`
--

DROP TABLE IF EXISTS `rlp_contact_details`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_contact_details` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
  `con_position` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `address` mediumtext COLLATE utf8mb4_unicode_ci,
  `suburb` varchar(100) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `state` varchar(100) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `country` varchar(100) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `postcode` varchar(100) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `telephone` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `fax` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `misc` longtext COLLATE utf8mb4_unicode_ci,
  `image` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `email_to` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `default_con` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `published` tinyint(1) NOT NULL DEFAULT '0',
  `checked_out` int(10) unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `ordering` int(11) NOT NULL DEFAULT '0',
  `params` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `user_id` int(11) NOT NULL DEFAULT '0',
  `catid` int(11) NOT NULL DEFAULT '0',
  `access` int(10) unsigned NOT NULL DEFAULT '0',
  `mobile` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `webpage` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `sortname1` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `sortname2` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `sortname3` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `language` varchar(7) COLLATE utf8mb4_unicode_ci NOT NULL,
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by` int(10) unsigned NOT NULL DEFAULT '0',
  `created_by_alias` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int(10) unsigned NOT NULL DEFAULT '0',
  `metakey` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadesc` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadata` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `featured` tinyint(3) unsigned NOT NULL DEFAULT '0' COMMENT 'Set if article is featured.',
  `xreference` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'A reference to enable linkages to external data sets.',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `version` int(10) unsigned NOT NULL DEFAULT '1',
  `hits` int(10) unsigned NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`),
  KEY `idx_access` (`access`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_state` (`published`),
  KEY `idx_catid` (`catid`),
  KEY `idx_createdby` (`created_by`),
  KEY `idx_featured_catid` (`featured`,`catid`),
  KEY `idx_language` (`language`),
  KEY `idx_xreference` (`xreference`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_contact_details`
--

LOCK TABLES `rlp_contact_details` WRITE;
/*!40000 ALTER TABLE `rlp_contact_details` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_contact_details` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_content`
--

DROP TABLE IF EXISTS `rlp_content`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_content` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `asset_id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'FK to the #__assets table.',
  `title` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `introtext` longtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `fulltext` longtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `state` tinyint(3) NOT NULL DEFAULT '0',
  `catid` int(10) unsigned NOT NULL DEFAULT '0',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by` int(10) unsigned NOT NULL DEFAULT '0',
  `created_by_alias` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int(10) unsigned NOT NULL DEFAULT '0',
  `checked_out` int(10) unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `images` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `urls` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `attribs` varchar(5120) COLLATE utf8mb4_unicode_ci NOT NULL,
  `version` int(10) unsigned NOT NULL DEFAULT '1',
  `ordering` int(11) NOT NULL DEFAULT '0',
  `metakey` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadesc` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `access` int(10) unsigned NOT NULL DEFAULT '0',
  `hits` int(10) unsigned NOT NULL DEFAULT '0',
  `metadata` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `featured` tinyint(3) unsigned NOT NULL DEFAULT '0' COMMENT 'Set if article is featured.',
  `language` char(7) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The language code for the article.',
  `xreference` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `note` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  PRIMARY KEY (`id`),
  KEY `idx_access` (`access`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_state` (`state`),
  KEY `idx_catid` (`catid`),
  KEY `idx_createdby` (`created_by`),
  KEY `idx_featured_catid` (`featured`,`catid`),
  KEY `idx_language` (`language`),
  KEY `idx_xreference` (`xreference`),
  KEY `idx_alias` (`alias`(191))
) ENGINE=InnoDB AUTO_INCREMENT=144 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_content`
--

LOCK TABLES `rlp_content` WRITE;
/*!40000 ALTER TABLE `rlp_content` DISABLE KEYS */;
INSERT INTO `rlp_content` VALUES (11,525,'What is Reactome ?','about-reactome','<p><b>Mission Statement</b></p>\r\n<p><span>REACTOME is an open-source, open access, manually curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic and clinical research, genome analysis, modeling, systems biology and education. Founded in 2003, the Reactome project is led by Lincoln Stein of&nbsp;</span><a href=\"http://oicr.on.ca/\"><span>OICR</span></a><span>, Peter DEustachio of&nbsp;</span><a href=\"http://nyulangone.org/\"><span>NYULMC</span></a><span>, Henning Hermjakob of&nbsp;</span><a href=\"http://www.ebi.ac.uk/\"><span>EMBL-EBI</span></a><span>, and Guanming Wu of&nbsp;</span><a href=\"http://www.ohsu.edu/\"><span>OHSU</span></a><span>.</span></p>\r\n<p><b>The Reactome Project</b></p>\r\n<p><span>Biological information has become so abundant and complex in recent years that it is difficult, if not impossible, even for expert individuals to manage in traditional publication formats and with existing knowledge management tools. It is an ongoing challenge for researchers to keep up-to-date on research developments in their fields, and identify relevant research to support their own studies without devoting too much time collecting unconnected information. The Reactome group has recognized this challenge and is developing a set of novel online resources that use features of the electronic media to organize biological pathway &nbsp;information in ways that provide for more efficient access and that allow new forms of analysis that were not possible with information stored in the traditional printed literature.</span></p>\r\n<p><span>The cornerstone of Reactome is a freely available, open source relational database of signaling and metabolic molecules and their relations organized into biological pathways and processes. The core unit of the Reactome data model is the reaction. Entities (nucleic acids, proteins, complexes, vaccines, anti-cancer therapeutics and small molecules) participating in reactions form a network of biological interactions and are grouped into pathways. Examples of biological pathways in Reactome include&nbsp;</span><a href=\"PathwayBrowser/#/R-HSA-1430728\"><span>classical intermediary metabolism</span></a><span>,&nbsp;</span><a href=\"PathwayBrowser/#/R-HSA-162582\"><span>signaling</span></a><span>,&nbsp;</span><a href=\"PathwayBrowser/#/R-HSA-74160\"><span>transcriptional regulation</span></a><span>,&nbsp;</span><a href=\"PathwayBrowser/#/R-HSA-109581&amp;PATH=R-HSA-5357801\"><span>apoptosis</span></a><span>&nbsp;and&nbsp;</span><a href=\"PathwayBrowser/#/R-HSA-1643685\"><span>disease</span></a><span>. The Reactome curation process for a pathway is similar to the editing of a scientific review. An external domain expert provides his or her expertise, a curator formalizes it into the database structure, and an external domain expert reviews the representation. A system of evidence tracking ensures that all assertions are backed up by the primary literature.</span></p>\r\n<p><span>The Reactome website is designed to literally give the user a graphical map of known biological processes and pathways that is also an interface which the user can click through to authoritative detailed information on components and their relations. The Reactome database and website enable scientists, researchers, students, and educators to find, organize, and utilize biological information to support&nbsp;</span><a href=\"PathwayBrowser/\"><span>data visualization</span></a><span>,&nbsp;</span><a href=\"dev/\"><span>integration</span></a><span>&nbsp;and&nbsp;</span><a href=\"PathwayBrowser/#TOOL=AT\"><span>analysis</span></a><span>. Reactome pathway, reaction and molecules pages extensively cross-reference to over 100 different online bioinformatics resources, including NCBI Gene, Ensembl and UniProt databases, the UCSC Genome Browser, ChEBI small molecule databases, and the PubMed literature database.</span></p>\r\n<p><span>Reactome is used by clinicians, geneticists, genomics researchers, and molecular biologists to interpret the results of high-throughput experimental studies, by bioinformaticians seeking to develop novel algorithms for mining knowledge from genomic studies, and by systems biologists building predictive models of normal and disease variant pathways.</span></p>\r\n<p><span>All data and software are freely available for&nbsp;</span><a href=\"download-data\"><span>download</span></a><span>. Interaction, reaction and pathway data are provided as downloadable flat,&nbsp;</span><a href=\"http://neo4j.com/\"><span>Neo4j GraphDB</span></a><span>,&nbsp;</span><a href=\"http://www.mysql.com/\"><span>MySQL</span></a><span>,&nbsp;</span><a href=\"http://www.biopax.org/\"><span>BioPAX</span></a><span>,&nbsp;</span><a href=\"http://sbml.org/Main_Page\"><span>SBML&nbsp;</span></a><span>and&nbsp;</span><a href=\"https://github.com/HUPO-PSI/miTab/blob/master/PSI-MITAB27Format.md\"><span>PSI-MITAB</span></a><span>&nbsp;files and are also accessible through our Web Services APIs. Software and instructions for local installation of the Reactome database, website, and data entry tools are also available to support independent pathway curation.</span></p>\r\n<p><span>If you have any feedback or questions, please contact us at the Reactome&nbsp;</span><a href=\"mailto:help@reactome.org\"><span>HelpDesk</span></a><span>.</span></p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 15:01:50','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,7,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(13,528,'Reactome Team','team','<p>The Reactome group consists of an international multidisciplinary team from OICR, OHSU, EMBL-EBI and NYULMC, with expertise in pathway curation and annotation, software development, and training and outreach, dedicated to providing the research community with openly accessible biological pathway knowledge.</p>\r\n<p>&nbsp;</p>\r\n<p><b>Principal Investigators</b></p>\r\n<p><a href=\"https://oicr.on.ca/investigators/lincoln-stein/\" target=\"_blank\" rel=\"noopener\">Lincoln Stein</a> (<a href=\"http://oicr.on.ca/\" target=\"_blank\" rel=\"noopener\">OICR</a>)</p>\r\n<p><a href=\"https://med.nyu.edu/faculty/peter-g-d-eustachio\" target=\"_blank\" rel=\"noopener\">Peter D\'Eustachio</a> (<a href=\"http://nyulangone.org/\" target=\"_blank\" rel=\"noopener\">NYULMC</a>)</p>\r\n<p><a href=\"https://www.ebi.ac.uk/about/people/henning-hermjakob\" target=\"_blank\" rel=\"noopener\">Henning Hermjakob</a> (<a href=\"http://www.ebi.ac.uk/\" target=\"_blank\" rel=\"noopener\">EMBL-EBI</a>)</p>\r\n<p><a href=\"http://www.ohsu.edu/xd/education/schools/school-of-medicine/academic-programs/graduate-studies/faculty/grad-studies-faculty.cfm?facultyid=818\" target=\"_blank\" rel=\"noopener\">Guanming Wu</a> (<a href=\"http://www.ohsu.edu/\" target=\"_blank\" rel=\"noopener\">OHSU</a>)</p>\r\n<p>&nbsp;</p>\r\n<p><b>Ontario Institute for Cancer Research</b></p>\r\n<p>Lincoln Stein</p>\r\n<p>Justin Cook</p>\r\n<p>Marc Gillespie</p>\r\n<p>Robin Haw</p>\r\n<p>Bijay Jassal</p>\r\n<p>Bruce May</p>\r\n<p>Marija Orlic-Milacic</p>\r\n<p>Karen Rothfels</p>\r\n<p>Solomon Shorser</p>\r\n<p>Joel Weiser</p>\r\n<p>&nbsp;</p>\r\n<p><b>New York University Langone Medical Center</b></p>\r\n<p>Peter D\'Eustachio</p>\r\n<p>Lisa Matthews</p>\r\n<p>Veronica Shamovsky</p>\r\n<p>&nbsp;&nbsp;</p>\r\n<p><b>European Bioinformatics Institute</b></p>\r\n<p>Henning Hermjakob</p>\r\n<p>Ewan Birney</p>\r\n<p>Antonio Fabregat</p>\r\n<p>Pascual Lorente</p>\r\n<p>Cristoffer Sevilla</p>\r\n<p>Konstantinos Sidiropoulos</p>\r\n<p>Guilherme Viteri</p>\r\n<p>&nbsp;</p>\r\n<p><b>Oregon Health &amp; Science University</b></p>\r\n<p>Guanming Wu</p>\r\n<p>Fred Loney</p>\r\n<p>&nbsp;</p>\r\n<p><b>Alumni</b></p>\r\n<p>Bernard de Bono</p>\r\n<p>Michael Caudy</p>\r\n<p>David Croft</p>\r\n<p>Corina Duenas</p>\r\n<p>Phani Garapati</p>\r\n<p>Gopal Gopinath</p>\r\n<p>Kerstin Hausmann</p>\r\n<p>Jill Hemish</p>\r\n<p>Geeta Joshi-Tope</p>\r\n<p>Steve Jupe</p>\r\n<p>Sarah Keating</p>\r\n<p>Maximilian Koch</p>\r\n<p>Florian Korninger</p>\r\n<p>Shahana Mahajan</p>\r\n<p>Sheldon McKay</p>\r\n<p>Nelson Ndegwa</p>\r\n<p>Elizabeth Nickerson</p>\r\n<p>Gavin O\'Kelly</p>\r\n<p>Carl Schmidt</p>\r\n<p>Esther Schmidt</p>\r\n<p>Heeyeon Song</p>\r\n<p>Imre Vastrik</p>\r\n<p>Marissa Webber</p>\r\n<p>Mark Williams</p>\r\n<p>Florent Yvon</p>\r\n<p>Christina Yung</p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 15:59:20','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(14,529,'Scientific Advisory Board','scientific-advisory-board','<p><span>Our Scientific Advisory Board members are internationally recognized researchers. The SAB meets annually to: i) discuss scientific agenda, ii) explore ways to expand our research efforts, and iii) critically review of our database, curation practices and software development programs.</span></p>\r\n<ul style=\"list-style-type: square;\">\r\n<li><span>Gary Bader</span> <a href=\"http://baderlab.org/GaryBader\"><span>University of Toronto</span></a></li>\r\n<li><span>Fiona Brinkman</span> <a href=\"http://www.brinkman.mbb.sfu.ca/fbrinkman/\"><span>Simon Fraser University</span></a></li>\r\n<li><span>Melissa Haendel</span> <span><a href=\"https://www.ohsu.edu/people/melissa-haendel/AFE044BDE8046E5D6FBDA51F448BDE2A\">Oregon Health and Science University</a></span></li>\r\n<li><span>Emma Lundberg&nbsp;<a href=\"https://www.scilifelab.se/researchers/emma-lundberg/\">KTH Royal Institute of Technology</a></span></li>\r\n<li><span>Trevor Pugh <a href=\"http://medbio.utoronto.ca/faculty/pugh.html\">University of Toronto</a></span></li>\r\n<li><span>Ben Raphael <a href=\"http://compbio.cs.brown.edu/\">Princeton University</a></span></li>\r\n<li>Shannon McWeeney <a href=\"https://www.ohsu.edu/people/shannon-mcweeney/0BA87D0039DE4BDFA0F88F4FB84616A2\">Oregon Health and Science University</a></li>\r\n<li><span>Josh Stuart</span> <span><a href=\"https://jstuart.soe.ucsc.edu/\">University of California Santa Cruz</a></span></li>\r\n</ul>\r\n<p><br /><br /><br /></p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 11:59:59','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":\"\",\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":\"\",\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":\"\",\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(15,530,'Other Reactomes','other-reactomes','<p>The following Reactomes are known to us. Clicking on the name of any one of these will take you either to the home page or a description of planned work.</p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 16:00:24','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(16,531,'License Agreement','license-agreement','<p>Reactome is an open source and open access resource, available to anyone. Usage of Reactome material is covered by two Creative Commons licenses:</p>\r\n<p>The terms of the <a href=\"https://creativecommons.org/publicdomain/zero/1.0/\">Creative Commons&nbsp;Public Domain (CC0)&nbsp;License<strong>&nbsp;</strong></a>apply to all Reactome annotation files, e.g. identifier mapping data, specialized data files, and interaction data derived from Reactome.</p>\r\n<p>The terms of the <a href=\"https://creativecommons.org/licenses/by/4.0/\">Creative Commons&nbsp;Attribution 4.0 International (CC BY 4.0)&nbsp;License</a><strong>&nbsp;</strong>apply to all&nbsp;software and code, e.g. relating to the functionality of the <a href=\"https://reactome.org\">reactome.org</a>,&nbsp;derived websites and webservices, the Curator Tool, the Functional Interaction application, SQL and Graph Database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials.</p>\r\n<p style=\"text-align: justify;\">Finally,&nbsp;<span>KEGG gene and pathway annotations, used to construct the Reactome Functional Interaction (FI) Network, are provided under license to OICR by Pathway Solutions, Inc. The Reactome FI Network data set is intended for use only by the original recipient. The recipient may not distribute the Reactome Functional interaction data files to other users without a license from Pathway Solutions, Inc.</span></p>\r\n<p>For information about how to properly credit data use, please review the <a href=\"https://creativecommons.org/faq/\">Creative Commons FAQ</a> or contact the <a href=\"mailto:help@reactome.org\">Reactome Helpdesk.</a></p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 16:01:10','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(17,532,'Disclaimer','disclaimer','<p style=\"text-align: justify;\"><b>DISCLAIMER OF LIABILITY AND OF WARRANTIES </b></p>\r\n<p style=\"text-align: justify;\"><span >In no event shall Ontario Institute for Cancer Research (OICR), New York University (NYU), Oregon Health &amp; Science University (OHSU), and/or EMBL-European Bioinformatics Institute (EBI) be liable for any use of the Reactome website, its contents or information derived thereof. The content of and information contained in the Reactome website is the opinion of the contributor and/or the author of such content and/or information and is not supplied as medical information. OICR, NYU, OHSU, and/or EBI accepts no responsibility or liability for any loss, cost, claim or expense arising from any reliance on such content or information.</span></p>\r\n<p style=\"text-align: justify;\"><span >BY USING THIS SITE YOU AGREE TO ASSUME ALL RISKS ASSOCIATED WITH YOUR USE OR TRANSFER OF ANY AND ALL INFORMATION CONTAINED ON THIS SITE AND TO HOLD OICR, NYU, OHSU, AND/OR EBI HARMLESS FROM ANY CLAIMS RELATING TO CONTENT OR INFORMATION IN EXCHANGE FOR YOUR USE OF THE SITE. THE REACTOME SITE IS PROVIDED AS IS WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, OR NON-INFRINGEMENT. OICR, NYU, OHSU, and/or EBI MAKES NO WARRANTIES ABOUT THE ACCURACY, RELIABILITY, COMPLETENESS, OR TIMELINESS OF THE MATERIAL, SERVICES, SOFTWARE, TEXT, GRAPHICS, AND LINKS. DESCRIPTION OF, OR REFERENCES TO, PRODUCTS OR PUBLICATIONS DOES NOT IMPLY ENDORSEMENT OF THAT PRODUCT OR PUBLICATION.</span></p>\r\n<p style=\"text-align: justify;\"><span >OICR, NYU, OHSU, and/or EBI DOES NOT WARRANT THAT THE SITE WILL OPERATE ERROR-FREE OR THAT THIS SITE AND ITS SERVER ARE FREE OF COMPUTER VIRUSES OR OTHER HARMFUL MECHANISMS. IF YOUR USE OR TRANSFER OF THE SITE OR THE MATERIALS RESULTS IN THE NEED FOR SERVICING OR REPLACING EQUIPMENT OR DATA, OICR, NYU, OHSU, AND/OR EBI IS NOT RESPONSIBLE FOR THOSE COSTS.</span></p>\r\n<p style=\"text-align: justify;\"><b>INDEMNITY</b></p>\r\n<p style=\"text-align: justify;\"><span >You agree to defend, indemnify, and hold harmless OICR, NYU, OHSU, and/or EBI, its officers, directors, employees and agents, from and against any and all claims, actions or demands, (including without limitation all legal and accounting fees) which may arise due to your use or transfer of the Reactome web site, its contents, or information thereof.</span></p>\r\n<p style=\"text-align: justify;\"><b>PRIVACY</b></p>\r\n<p style=\"text-align: justify;\"><span >Reactome logs the IP addresses of your web browser and each database query. This is done in order to track usage statistics and to identify operational problems. This information is not used to identify individuals or organizations, and is never shared with third parties. Cookies are used by the search pages in order to remember your search settings. Some cookies persist after you exit the browser, but they are never used for either identification or tracking purposes.</span></p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 16:02:46','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(18,537,'Downloads','downloads','<p><span data-mce-json=\"{}\"><!--ul,ol {\r\n  margin-left: 25px;\r\n  padding-left: 0 !important;\r\n}-->&nbsp;</span></p>\r\n<div id=\"downloads-page\" class=\"moduletable-sfx4\">\r\n<p>Reactome provides <a href=\"http://en.wikipedia.org/wiki/Open_source\">open-source</a> and <a href=\"http://en.wikipedia.org/wiki/Open_data\">open-data</a>. We have continuously supported the major open-data standards, including <a href=\"http://www.biopax.org/\">BioPAX</a>, <a href=\"http://www.psidev.info/groups/molecular-interactions\">PSI-MITAB</a>, <a href=\"http://sbml.org/\">SBML</a> and <a href=\"https://sbgn.github.io/\">SBGN</a> export formats. The Reactome data and source code continue to be publicly accessible under the terms of Creative Commons Licenses. Please refer to our&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=16:license-agreement&amp;catid=25\">License Agreement</a>.</p>\r\n<h3>Graph Database</h3>\r\n<p>For more information on the installation of the Reactome Graph Database, please refer to the&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=45#GetStarted\">Get Started</a>&nbsp;section of the&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=45\">Graph Database</a>&nbsp;documentation in the&nbsp;<a href=\"dev\">Developer Guide</a>.</p>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/reactome.graphdb.tgz\">Reactome Graph Database</a></li>\r\n</ul>\r\n<h3>Services and GitHub Repository</h3>\r\n<ul>\r\n<li style=\"list-style-type: circle;\"><strong>Content Service</strong>: Our&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=48\">Content Service</a> provides outside users with direct access to pathway data in Reactome.</li>\r\n<li style=\"list-style-type: circle;\"><strong>Analysis Service</strong>: The Reactome Pathway <a style=\"text-align: justify; font-size: 14px;\" href=\"index.php?option=com_content&amp;view=article&amp;id=47#Resources\">Analysis Tools</a><span style=\"text-align: justify; color: #333333; font-size: 14px;\"> are also available for integration into third party websites.</span></li>\r\n<li style=\"list-style-type: circle;\"><a href=\"https://github.com/reactome\">GitHub Repository</a></li>\r\n</ul>\r\n<h3>Identifier mapping files</h3>\r\n<p>Mapping files link the source database identifier to the lowest level pathway diagram or subset of the pathway, all levels of the pathway hierarchy or database identifier to all reactions. {modal url=\"index.php?option=com_content&amp;view=article&amp;id=60\"}Read more{/modal}&nbsp;about the format.</p>\r\n<ul>\r\n<li><i class=\"fa fa-folder\"></i>Lowest level pathway diagram / Subset of the pathway\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2Reactome.txt\">UniProt to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2Reactome.txt\">ChEBI to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2Reactome.txt\">ENSEMBL to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2Reactome.txt\">miRBase to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2Reactome.txt\">NCBI to pathways</a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<ul>\r\n<li><i class=\"fa fa-folder\"></i>All levels of the pathway hierarchy\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2Reactome_All_Levels.txt\">UniProt to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2Reactome_All_Levels.txt\">ChEBI to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2Reactome_All_Levels.txt\">ENSEMBL to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2Reactome_All_Levels.txt\">miRBase to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2Reactome_All_Levels.txt\">NCBI to All pathways</a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<ul>\r\n<li><i class=\"fa fa-folder\"></i>Database identifier to all reactions\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2ReactomeReactions.txt\">UniProt to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2ReactomeReactions.txt\">ChEBI to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2ReactomeReactions.txt\">ENSEMBL to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2ReactomeReactions.txt\">miRBase to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2ReactomeReactions.txt\">NCBI to All reactions</a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<h3>Physical Entity (PE) Identifier mapping files</h3>\r\n<p>Mapping files link the source database identifier to the lowest level pathway diagram or subset of the pathway. {modal url=\"index.php?option=com_content&amp;view=article&amp;id=86\"}Read more{/modal}&nbsp;about the format.</p>\r\n<ul>\r\n<li><i class=\"fa fa-folder\"></i>Lowest level pathway diagram / Subset of the pathway\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2Reactome_PE_Pathway.txt\">UniProt to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2Reactome_PE_Pathway.txt\">ChEBI to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2Reactome_PE_Pathway.txt\">ENSEMBL to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2Reactome_PE_Pathway.txt\">miRBase to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2Reactome_PE_Pathway.txt\">NCBI to pathways</a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<ul>\r\n<li><i class=\"fa fa-folder\"></i>All levels of the pathway hierarchy\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2Reactome_PE_All_Levels.txt\">UniProt to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2Reactome_PE_All_Levels.txt\">ChEBI to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2Reactome_PE_All_Levels.txt\">ENSEMBL to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2Reactome_PE_All_Levels.txt\">miRBase to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2Reactome_PE_All_Levels.txt\">NCBI to All pathways</a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<ul>\r\n<li><i class=\"fa fa-folder\"></i>Database identifier to all reactions\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2Reactome_PE_Reactions.txt\">UniProt to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2Reactome_PE_Reactions.txt\">ChEBI to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2Reactome_PE_Reactions.txt\">ENSEMBL to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2Reactome_PE_Reactions.txt\">miRBase to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2Reactome_PE_Reactions.txt\">NCBI to All reactions</a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<h3>Other Mappings</h3>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/models2pathways.tsv\">BioModels to Pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ReactionPMIDS.txt\">Reaction PMIDs</a> - {modal url=\"index.php?option=com_content&amp;view=article&amp;id=87\"}Read more{/modal} about the format</li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Reactome2OMIM.txt\">Reactome to OMIM</a> - {modal url=\"index.php?option=com_content&amp;view=article&amp;id=88\"}Read more{/modal} about the format</li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ProteinRoleReaction.txt\">Protein Role Reaction</a> - {modal url=\"index.php?option=com_content&amp;view=article&amp;id=89\"}Read more{/modal} about the format</li>\r\n</ul>\r\n<h3>Pathways</h3>\r\n<ul>\r\n<li><i class=\"fa fa-folder\"></i>Pathway Information. {modal url=\"index.php?option=com_content&amp;view=article&amp;id=61\"}Read more{/modal} about the format.\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ReactomePathways.txt\">Complete List of Pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ReactomePathwaysRelation.txt\">Pathways hierarchy relationship</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Complex_2_Pathway_human.txt\">Complexes to Pathways (Human)</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ewas2Pathway_human.txt\">EWAS to Pathways (Human)</a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<ul>\r\n<li><i class=\"fa fa-folder\"></i>Pathway Diagrams and Illustrations\r\n<ul>\r\n<li><i class=\"fa fa-file-zip-o\"></i>Human pathway diagrams and illustrations: <a href=\"/download/current/diagrams.svg.tgz\">SVG</a> or <a href=\"/download/current/diagrams.png.tgz\">PNG</a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<h3>Icons</h3>\r\n<p class=\"p1\"><span class=\"s1\">The <a href=\"index.php?option=com_content&amp;view=article&amp;id=118\" hreflang=\"en\">icon mapping files</a>&nbsp;link the source database identifier to elements of the icon library. This folder contains a number of different mapping files to reflect the different source databases.&nbsp;{modal url=\"index.php?option=com_content&amp;view=article&amp;id=117\"}Read more{/modal} about the format.</span></p>\r\n<h3>Complexes</h3>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ComplexParticipantsPubMedIdentifiers_human.txt\">Human complexes with their participating&nbsp;protein molecules&nbsp;</a></li>\r\n</ul>\r\n<h3>Reactome External Identifiers</h3>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/reactome_stable_ids.txt\">Reactome Stable Identifiers</a>&nbsp;</li>\r\n</ul>\r\n<h3>Protein-Protein Interactions derived from Reactome pathways</h3>\r\n<p>Interactions are inferred between all protein components of a complex. To limit the creation of false-positive inferences, only complexes with four or fewer protein components are considered. <a href=\"https://github.com/reactome/interaction-exporter/wiki/Methods\" target=\"_blank\" rel=\"noindex,nofollow noopener\">More information</a></p>\r\n<ul>\r\n<li><i class=\"fa fa-folder\"></i>{modal url=\"https://psicquic.github.io/MITAB27Format.html\"}PSI-MITAB format{/modal} interaction files\r\n<ul>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/interactors/reactome.all_species.interactions.psi-mitab.txt\">All interactions, including non-human species</a></li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a title=\"Human protein-protein interactions\" href=\"/download/current/interactors/reactome.homo_sapiens.interactions.psi-mitab.txt\">Human protein-protein interactions</a></li>\r\n<li></li>\r\n</ul>\r\n</li>\r\n<li><i class=\"fa fa-folder\"></i>{modal url=\"index.php?option=com_content&amp;view=article&amp;id=62\"}TAB-delimited format{/modal} interaction files\r\n<ul>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/interactors/reactome.all_species.interactions.tab-delimited.txt\">All interactions, including non-human species</a></li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/interactors/reactome.homo_sapiens.interactions.tab-delimited.txt\">Human protein-protein interactions</a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<h3>Specialized data formats</h3>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/gene_association.reactome.gz\">Gene association file</a> (<a href=\"http://geneontology.org/\">GO annotation</a> in Reactome)</li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/homo_sapiens.3.1.sbml.tgz\">Human reactions</a> in <a href=\"http://sbml.org/Main_Page\">SBML</a>&nbsp;&nbsp;(level 3, version 1) format</li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/all_species.3.1.sbml.tgz\">All Species reactions</a>&nbsp;in <a href=\"http://sbml.org/Main_Page\">SBML</a>&nbsp;&nbsp;(level 3, version 1) format</li>\r\n<li><i class=\"fa fa-file-zip-o\"></i>Events in the <a href=\"/download/current/biopax2.zip\">BioPAX level 2 </a>format.</li>\r\n<li><i class=\"fa fa-file-zip-o\"></i>Events in the <a href=\"/download/current/biopax.zip\">BioPAX level 3 </a>format.</li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/homo_sapiens.sbgn.tar.gz\">Human pathway diagrams</a> in the&nbsp;<a href=\"https://sbgn.github.io/\">SBGN</a> format.</li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/ReactomePathways.gmt.zip\">Reactome Pathways Gene Set</a>.</li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/protege_files.tar.gz\">Protg 2.0 ontology files</a> for Reactome data model.</li>\r\n</ul>\r\n<h3>Functional interactions (FIs) derived from Reactome, and other pathway and interaction databases</h3>\r\n<p>For details about FIs, please see&nbsp;<a href=\"http://genomebiology.com/content/11/5/R53\">Wu G, Feng X, Stein L</a>. To use these FI networks, please try our&nbsp;<a title=\"ReactomeFIViz Cytoscape app\" href=\"index.php?option=com_content&amp;view=article&amp;id=44\">ReactomeFIViz Cytoscape app</a>.</p>\r\n<ul>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"http://cpws.reactome.org/caBigR3WebApp2018/FIsInGene_122718_with_annotations.txt.zip\" target=\"_blank\" rel=\"noopener noreferrer\">Version 2018</a></li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"http://cpws.reactome.org/caBigR3WebApp2017/FIsInGene_071718_with_annotations.txt.zip\" target=\"_blank\" rel=\"noopener noreferrer\">Version 2017</a></li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"http://cpws.reactome.org/caBigR3WebApp2016/FIsInGene_022717_with_annotations.txt.zip\" target=\"_blank\" rel=\"noopener noreferrer\">Version 2016</a></li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"http://cpws.reactome.org/caBigR3WebApp2015/FIsInGene_031516_with_annotations.txt.zip\" target=\"_blank\" rel=\"noopener noreferrer\">Version 2015</a></li>\r\n</ul>\r\n<h3>MySQL dumps of Reactome databases</h3>\r\n<ul>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/databases/gk_current.sql.gz\">Main database</a></li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/databases/gk_stable_ids.sql.gz\">Stable identifiers database</a></li>\r\n</ul>\r\n<h3>Curation Software</h3>\r\n<p>The Curator tool is meant for use by curators to annotate biological pathways based on the Reactome schema. Please note that this tool will only work with a database using our internal database format; the database available from this web page will not work with them.</p>\r\n<ul>\r\n<li><i class=\"fa fa-book\"></i><a href=\"index.php?option=com_content&amp;view=article&amp;id=63\">Curator Tool (version 3.3, build 98, August 6, 2019)</a></li>\r\n</ul>\r\n<h3>Our textbook of biological pathways and processes</h3>\r\n<ul>\r\n<li><i class=\"fa fa-file-pdf-o\"></i><a href=\"/download/current/TheReactomeBook.pdf.tgz\">PDF format</a></li>\r\n</ul>\r\n</div>','',1,32,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-21 14:47:36','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(22,556,'Outreach','outreach','<p style=\"text-align: justify;\"><span >Reactomes continued success will be determined by the extent to which we reach out to current and new potential users. We will reach out to bioinformaticians, traditional molecular and cellular biologists, computational biologists, software developers, geneticists, clinicians, educators and students to ensure that all potential uses are aware of the resource and take full advantage of the services we offer. Therefore, we offer a series of outreach and training programs which will: </span></p>\r\n<ol style=\"list-style-type: lower-roman; text-align: justify;\">\r\n<li><span >foster the training of the next generation of scientists</span></li>\r\n<li><span >collaborate with data providers</span></li>\r\n<li><span >actively reach out to present and new users and collaborators. </span></li>\r\n</ol>\r\n<p style=\"text-align: justify;\"><span >Simultaneously, we encourage the reuse of the Reactome data model and software by supporting its adoption by outside groups, in particular by the cancer, clinical and disease research communities.</span></p>\r\n<p style=\"text-align: justify;\"><span >Outreach takes on a variety of forms, such as public talks, lectures, visiting universities and colleges and supporting traditional science events (meetings, conferences, and workshops). Education is a major focus of Reactome community outreach and aims to inform biologists, clinicians, and bioinformaticians about what Reactome can do. </span></p>\r\n<p style=\"text-align: justify;\"><span >Reactome training teaches new and current users how to use Reactome resources to help meet the efforts of their organizations, groups, or specific audience.</span></p>\r\n<p style=\"text-align: justify;\"><b>Training and Community Outreach</b><span >. If you have a particular outreach question or request, you can contact our </span><a href=\"mailto:help@reactome.org\"><span >HelpDesk</span></a><span >, or post your question to other research colleagues on our Reactome Forum. All our training and outreach materials are available under a </span><a href=\"license\"><span >Creative Commons Attribution 4.0 Unported License</span></a><span >. We have just two simple requests, please attribute Reactome, and let us know if you use our presentations, posters or training resources.</span></p>\r\n<p style=\"text-align: justify;\"><b>Following Reactome.</b><span > In addition to our established </span><a href=\"https://lists.reactome.org/mailman/listinfo/reactome-announce\"><span >Mailing List</span></a><span > and </span><a href=\"about/news\"><span >News feature</span></a><span >, Reactome has embraced the social networking scene. You can follow Reactome on </span><a href=\"http://www.facebook.com/group.php?gid=244908260192\"><span >Facebook</span></a><span >, </span><a href=\"http://www.linkedin.com/groups?gid=2118372\"><span >LinkedIn</span></a><span > and </span><a href=\"http://twitter.com/reactome\"><span >Twitter</span></a><span >.</span></p>\r\n<p style=\"text-align: justify;\"><b>Getting Involved. </b><span >Reactome is an open-source project that promotes community participation. We are eager for contributions from the community and would welcome inquiries from researchers interested in contributing information on pathways in their area of expertise. </span></p>\r\n<p style=\"text-align: justify;\"><a href=\"mailto:help@reactome.org\"><span >Contact us</span></a><span > with your ideas.</span></p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 15:45:28','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,7,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(23,557,'Events','events','<p><strong>Here is a list of all the Reactome talks, poster presentations, training events and workshops</strong></p>\r\n<p>{tab 2019}</p>\r\n<p>Bioinformatics Resources for Protein Biology, EMBL-EBI, Cambridge, UK, 28 February</p>\r\n<p>Society of Toxicology, Baltimore, MD, USA,&nbsp;12 March</p>\r\n<p>VIZBI, EMBO, Heidelberg, Germany, 13-15 March</p>\r\n<p>Bioinformatics Resources for Protein Biology, EMBL-EBI, Cambridge, UK, 28 March</p>\r\n<p>GO Consortium Meeting, Cambridge, UK, 11-12 April</p>\r\n<p>Biocuration 2019, Cambridge, UK,&nbsp;8 April</p>\r\n<p>Bioinformatics resources for protein biology, Cambridge University, Cambridge, UK,&nbsp;1 May</p>\r\n<p>Biophysical Society Meeting, Baltimore, USA,&nbsp;3 May</p>\r\n<p>ProteoNet Milan, ProteoNet Milan, Milan, Italy, 4 May</p>\r\n<p>{tab 2018}</p>\r\n<p>EMBL-EBI Industry Programme Workshop, Pfizer, La Jolla, CA, USA, 7-8 February</p>\r\n<p>CBW Bioinformatics for Cancer Genomics, CSHL, NY 12-17 March</p>\r\n<p>Interactions and Pathways, EMBL-EBI, Cambridge, UK, 13&nbsp;March</p>\r\n<p>Biocuration 2018, Fudan University, Shanghai, China, 04-08 April</p>\r\n<p>GO Consortium Meeting, NYUMC, New York, USA, 12-14 May 2018</p>\r\n<p>International Proteogenome Workshop, University of Kyoto, Kyoto, Japan, 13 May</p>\r\n<p>University of Chongqing, Chongqing, China,&nbsp;13 May</p>\r\n<p>Network and Pathways, EMBL-EBI, Cambridge, UK, 17&nbsp;May</p>\r\n<p>Disease Maps Community Meeting, Institut Curie, Paris, France,&nbsp;21 June</p>\r\n<p>CBW Pathway and Network Analysis of Omics Data, Toronto, ON, Canada, 25-27 June</p>\r\n<p>ISMB 2018, N/A, Chicago, IL, USA, 6-10 July</p>\r\n<p>British Society for Proteome Research, Bradford, UK,&nbsp;10 July</p>\r\n<p>Proteomics Bioinformatics, EMBL-EBI, Cambridge, UK, 20 July</p>\r\n<p>Universitat de Valncia, Valencia, Spain,&nbsp;27 July</p>\r\n<p>GlyGen consortium, USA,&nbsp;September 4</p>\r\n<p>Italian Proteomics Association 2018, Como, Italy, 5-7 September</p>\r\n<p>ECCB 2018, Athens, Greece, 8-12 September</p>\r\n<p>EMBL-EBI Industry Programme Workshop, EMBL-EBI, Cambridge, UK, 17 September</p>\r\n<p>Computational systems biology of cancer, Institut Curie, Paris, France, 24 September</p>\r\n<p>HUPO 2018, Orlando, USA, 30 September - 3 October</p>\r\n<p>Bioinformatics &amp; Functional Genomics in Zebrafish, EMBL-EBI, Cambridge, UK, 17 September</p>\r\n<p>CABANA, EMBL-EBI, Cambridge, UK, 6 November</p>\r\n<p>WikiPathways Summit 2018, UCSF, San Francisco, USA, 8-10&nbsp;November</p>\r\n<p>Chinese Human Proteome Organisation Conference, Guangzhou, China, 16 November</p>\r\n<p>GO Consortium Meeting, Montreal, Canada, 17-19&nbsp;November 2018</p>\r\n<p>Train Malta, EMBL-EBI, Cambridge, UK, 19 November</p>\r\n<p>Boehringer Ingelheim, Ridge Field, CT, USA, 31 November 2018</p>\r\n<p>GO Ontology Editors, Geneva, Switzerland, 10-13 December</p>\r\n<p>{tab 2017}</p>\r\n<p><span>Pathway and Network Analysis of -omics Data (2017), OICR, Toronto, Canada, 26-28 June</span></p>\r\n<p><span>Neo4j Life &amp; Health Sciences Day, Berlin, Germany, 21 June</span></p>\r\n<p><span>Bioinformatics for Cancer Genomics (2017), OICR, Toronto, Canada, 29 May  2 June</span></p>\r\n<p><span>3rd BiVi Annual Meeting, Edinburgh Napier University, UK, 20-21 April</span></p>\r\n<p><span>Biocuration 2017, Stanford University, US, 26-29 March</span></p>\r\n<p><span>High-Throughput Biology: From Sequence to Networks, CSHL, Cold Spring Harbor, US, 20-26 March</span></p>\r\n<p><span>EBI Training Workshop, Hinxton, UK, 9 March</span></p>\r\n<p><span>Interactions and Pathways  Reactome Workshop, Cambridge University, UK, 2 February</span></p>\r\n<p>{tab 2016}</p>\r\n<p><span>Proteomics bioinformatics, Hinxton, UK, 5 December</span></p>\r\n<p><span>Data sharing Best Practice workshop, EMBL-ABR, Australia, 24-28 October</span></p>\r\n<p><span>ECCB 2016, Amsterdam, Netherlands, 5-6 September</span></p>\r\n<p><span>BioNetVisA Workshop, Amsterdam, Netherlands, 4 September</span></p>\r\n<p><span>Integrating Interactions and Pathways, Hinxton, UK, 22 July</span></p>\r\n<p><span>PI seminar series, National Center for Protein Sciences Beijing, China, 21 July</span></p>\r\n<p><span>Reactome webinar (live), Hinxton, UK, 22 June</span></p>\r\n<p><span>SAC seminar series: Harvesting and analysing proteomics big data, Hinxton, UK, 15 June</span></p>\r\n<p><span>Pathway and Network Analysis of -omics Data, OICR, Toronto, 13-15 June</span></p>\r\n<p><span>Bioinformatics for Cancer Genomics, OICR, Toronto, 30 May  2 June</span></p>\r\n<p><span>CNHUPO, Xiamen University, China, 23 May</span></p>\r\n<p><span>Reactome, IMEx, ProteomeXchange, Tsinghua University, China, 17 May</span></p>\r\n<p><span>GLBIO/CCBC 2016, University of Toronto, Canada, 16-19 May</span></p>\r\n<p><span>Reactome web services and widgets for third-party integration, UV, Spain, 12 May</span></p>\r\n<p><span>Networks and Pathways 2016, Hinxton, 12 May</span></p>\r\n<p><span>Data integration, analysis and visualisation. Experiences from a biological pathway database, UDC, Spain, 25 April</span></p>\r\n<p><span>251st American Chemical Society National Meeting Exposition, San Diego, 13-16 March</span></p>\r\n<p><span>SME Industry Forum, Hinxton, UK, 7 March</span></p>\r\n<p><span>Bioinformatics Resources for Protein Biology, Hinxton, UK, 2 February</span></p>\r\n<p><span>Pathways Overview Widget, Hinxton, 30 January</span></p>\r\n<p><span>Diagram Widget Training Session, Hinxton, 30 January</span></p>\r\n<p>{tab 2015}</p>\r\n<p><span>WGC Advanced Course: Proteomics Bioinformatics, Hinxton, UK, 11 December</span></p>\r\n<p><span>MiMS / NDPIA Bioinformatics Workshop  Umea, Hinxton, UK, 10-14 November</span></p>\r\n<p><span>Networks &amp; Pathways, Hinxton, UK, 5 November</span></p>\r\n<p><span>Interactions &amp; Pathways  Reactome, Cambridge, UK, 3 November</span></p>\r\n<p><span>Pathway and Network Analysis Workshop, Liverpool, UK, 1 November</span></p>\r\n<p><span>EBI Open Day, Hinxton, UK, 29 October</span></p>\r\n<p><span>EBI VI Argentinian Conference of Bioinformatics and Computational Biology (CAB2C), Baha Blanca, Argentina, 13 October</span></p>\r\n<p><span>EBI Bioinformatics Roadshow, Campinas, Brazil, 5 August</span></p>\r\n<p><span>EBI Bioinformatics Roadshow, Sao Paulo, Brazil, 31 July</span></p>\r\n<p><span>Reactome Webinar, Toronto, Canada, July 24</span></p>\r\n<p><span>Reactome SOT Webinar, New York, USA, 15 June</span></p>\r\n<p><span>Pathway and Network Analysis of omics Data, Toronto, Canada, 1-3 June</span></p>\r\n<p><span>Royal Vet College, London, UK, 28 May</span></p>\r\n<p><span>Bioinformatics for Cancer Genomics, Toronto, Canada, 25-29 May</span></p>\r\n<p><span>Reactome Exploring and Analysing Biological Pathways, Hinxton, UK, 12 May</span></p>\r\n<p><span>Reactome: Networks and Pathways, Toronto, Canada, 6 May</span></p>\r\n<p><span>Reactome Webinar, Hinxton, UK, 6 May</span></p>\r\n<p><span>High-Throughput Biology: From Sequence to Networks, Cold Spring Harbor Laboratory and the New York Genome Center, New York, USA, 27 April  3 May</span></p>\r\n<p><span>EBI Industry Workshop, Boston, USA, April 1-2, 2015</span></p>\r\n<p><span>Reactome Webinar, Toronto, Canada, March 26, 2015</span></p>\r\n<p><span>Introduction to EBI, Cambridge, UK, 16 February</span></p>\r\n<p>{tab 2014}</p>\r\n<p><span>ACSB 2014, Philadelphia, USA, 6-10 December</span></p>\r\n<p><span>Wellcome Trust Bioinformatics Course, Hinxton, UK, 14 November</span></p>\r\n<p><span>Cambridge University Computer Training, University of Cambridge, UK, 22 October</span></p>\r\n<p><span>EBI Open Day, Hinxton, UK, 16 October</span></p>\r\n<p><span>Argentinian Bioinformatics Workshop, University of San Martin, Buenos Aires, Argentina, 1 October</span></p>\r\n<p><span>ECCB 2014: The 13th European Conference on Computational Biology, Strasbourg, France, 7-10 September</span></p>\r\n<p><span>BioNetVisA 2014, Strasbourg, France, 6 September</span></p>\r\n<p><span>Proteomics Data, Functional Analysis and Dissemination, Strasbourg, France, 6 September</span></p>\r\n<p><span>Aveiro Roadshow, University of Aveiro, Portugal, 23 July</span></p>\r\n<p><span>ProteomeXchange/Reactome, NIDDK, USA, 19 June</span></p>\r\n<p><span>Functional Genomics and Systems Biology, WTSI, Hinxton, UK, 12 June</span></p>\r\n<p><span>Networks &amp; Pathways, Hinxton, UK, 12 June</span></p>\r\n<p><span>ProteomeXchange/Reactome, NHLBI, USA, 12 June</span></p>\r\n<p><span>IOCB AS CR, Prague, Czech Republic, 4 June</span></p>\r\n<p><span>Networks and Pathways Course, University of Lisbon, Portugal, 23 May</span></p>\r\n<p><span>ORCID Outreach Meeting, University of Illinois, Chicago, USA, 21-22 May</span></p>\r\n<p><span>PAG Asia, Singapore, Republic of Singapore, 21 May</span></p>\r\n<p><span>Leiden Students Day, Hinxton, UK, 28 April</span></p>\r\n<p><span>diXa: Functional Interpretation of Toxicogenomic Data, Hinxton, UK, 10 April</span></p>\r\n<p><span>Biocuration 2014, Toronto University, Canada, 9 April</span></p>\r\n<p><span>SME forum, Hinxton, UK, 6 March</span></p>\r\n<p><span>Reactome Course, Cambridge University Dept. of Genetics, UK, 24 January</span></p>\r\n<p><span>Plant &amp; Animal Genome Conference, San Diego, USA, 12 January</span></p>\r\n<p>{tab 2013}</p>\r\n<p><span>Canadian Cancer Research Conference, CCRC, 3-6 November, Toronto, Canada</span></p>\r\n<p><span>GOC Meeting, Bar Harbor, Maine, USA, October 4-6, 2013</span></p>\r\n<p><span>ISSX 10th International Meeting, ISSX, Toronto, Canada, 29 October-3 November</span></p>\r\n<p><span>EMBO Meeting, EMBO, Amsterdam, The Netherlands, 21-23 October</span></p>\r\n<p><span>Bioinformatics for Immunologists, University of Cambridge, Cambridge, UK, 5 September</span></p>\r\n<p><span>SciKnowMine Release Workshop  Bridging BioNLP and Biocuration, USC Viterbi ISI, Los Angeles, USA, 19 August</span></p>\r\n<p><span>ISMB/ISMB  The Art and Science Exhibit, ISMB, Berlin, Germany, 21-23 July</span></p>\r\n<p><span>NetBioSIG, Berlin, Germany, 19 July</span></p>\r\n<p><span>Networks &amp; Pathways, EBI, Cambridge, UK, 11 July</span></p>\r\n<p><span>Gordon Conference Elastin &amp; Elastic Fibers, University of New England, Biddeford, USA, 22 July</span></p>\r\n<p><span>Summer School of Bioinformatics, EBI, Cambridge, UK, 12 June</span></p>\r\n<p><span>UCL, London, UCL, London, UK, 30 May</span></p>\r\n<p><span>University of Rome, Tor Vergata (Rome 2), Rome, Italy, 7 May</span></p>\r\n<p><span>University of Singapore, Singapore, 20 March</span></p>\r\n<p><span>CAGE-KID Cancer Genomics Workshop, CAGE-KID/EBI, 22 March, UK</span></p>\r\n<p><span>SBGN Workshop, University of Edinburgh, Edinburgh, UK, 29 April-2 May</span></p>\r\n<p><span>GenomeSpace Collaboration, BROAD Institute, Cambridge, USA, 26 March</span></p>\r\n<p><span>CAGE-KID Cancer Genomics Workshop, EBI, Cambridge, UK, 22 March</span></p>\r\n<p><span>EBI Open day, EBI, Cambridge, UK, 14 March</span></p>\r\n<p><span>International Plant &amp; Animal Genome XXI, San Diego, USA, 13 January</span></p>\r\n<p><span>Nimbios Workshop, University of Tennesse, Tennesse, USA, 8-10 January</span></p>\r\n<p>{tab 2012}</p>\r\n<p><span>EBI Open Day, EBI, Cambridge, UK, 20 December</span></p>\r\n<p><span>HPH 2012, Center for Genetic Engineering and Biotechnology, Havana, Cuba, 9 December</span></p>\r\n<p><span>TCAG Analysis Group, TCAG, Washington, DC, USA, 27 November</span></p>\r\n<p><span>Toronto Bioinformatics User Group, University of Toronto, Toronto, Canada, 31 October</span></p>\r\n<p><span>Cambridge University, Cambridge University, Cambridge, UK, 24 October</span></p>\r\n<p><span>Randall Lad, KCL, Randall Lad, Kings College London, London, UK, 23 October</span></p>\r\n<p><span>Google Summer of Code Open Software Meeting, Google, Mountain View, USA, 21 October</span></p>\r\n<p><span>The Jackson Laboratory, Maine, The Jackson Laboratory, Maine, Bar Harbor, USA, 19 October</span></p>\r\n<p><span>Norwestern University Chicago, Norwestern University, Chicago, USA, 17 October</span></p>\r\n<p><span>Workshop on the Concept and Tools for Pathways of Toxicity (PoT), John Hopkins University, Baltimore, USA, 10 October</span></p>\r\n<p><span>Programmatic Access to EBI Proteomics Resources, EBI, Hinxton, UK, 4 October</span></p>\r\n<p><span>University of Leicester/B/BASH, University of Leicester, Leicester, UK, 12 September</span></p>\r\n<p><span>Animal Biotechnology Workshop, EBI, Hinxton, UK, 13 September</span></p>\r\n<p><span>COMBINE 2012, University of Toronto, Toronto, Canada, 18 August</span></p>\r\n<p><span>University of Toronto, Toronto, Canada, 25 July</span></p>\r\n<p><span>Division of Translational Research, National Cancer Center Hospital East, Kashiwanoha, Tokyo University, Kashiwanoha, Japan, 13 July</span></p>\r\n<p><span>Department of Medical Genome Sciences, Kashiwanoha campus, Tokyo University, Kashiwanoha, Japan, 12 July</span></p>\r\n<p><span>Human Genome Center, Institute of Medical Science, Shirokanedai campus, Tokyo University, Tokyo, Japan, 11 July</span></p>\r\n<p><span>Current and Future in Pathway Research International Workshop, Korea Institute of Science and Technology (KISTI), Daejeon, South Korea, 6 July</span></p>\r\n<p><span>Erasmus MC, Rotterdam, Erasmus MC, Rotterdam, The Netherlands, 13 June</span></p>\r\n<p><span>OICR, Toronto, Canada, 12 June</span></p>\r\n<p><span>Ontologies for Immunity and Infectious Disease, University at Buffalo, Buffalo, USA, 11-13 June</span></p>\r\n<p><span>HARMONY Hackathon, Department of Bioinformatics  BiGCaT, Maastricht, The Netherlands, 21 May</span></p>\r\n<p><span>Networks and Pathways Bioinformatics for Biologists, EBI, Hinxton, UK, 18 May</span></p>\r\n<p><span>6th Biocuration Conference, University of Georgetown, Washington, USA, 2-4 April</span></p>\r\n<p><span>Perspectives in Clinical Proteomics Training Workshop, EBI, Hinxton, UK, 17 March</span></p>\r\n<p><span>Fobiotech Meeting, Fobiotech, Torino, Italy, 5 March</span></p>\r\n<p><span>GO Consortium Annotation Meeting, Stanford University, Palo Alto, USA, 26-28 February</span></p>\r\n<p><span>Reactome Outreach, University of Toronto, Toronto, Canada, 17 February</span></p>\r\n<p>{tab 2011}</p>\r\n<p><span>Canadian Cancer Research Conference, Toronto, Canada, 27-30 November</span></p>\r\n<p><span>X-Meeting 2011, Florianopolis, Brazil, 12-15 October</span></p>\r\n<p><span>PSIMEx Workshop: Interactions and Pathways, Lausanne, Switzerland, 6-7 October</span></p>\r\n<p><span>AlzForum / PRO workshop, Buffalo, USA, 4-5 October</span></p>\r\n<p><span>COMBINE 2011, Heidelberg, Germany, 3-7 September</span></p>\r\n<p><span>OICR Reactome Workshop, Ottawa Hospital Research Institute, University of Ottawa, Ottawa, Canada, 4 August</span></p>\r\n<p><span>OICR Reactome Workshop, University of Kingston, Kingston, Canada, 3 August.</span></p>\r\n<p><span>Network Biology SIG, ISMB/ECCB 2011, Vienna, Austria, 15 July.</span></p>\r\n<p><span>International Institute of Molecular and Cell Biology, Warsaw, Poland, 15 July</span></p>\r\n<p><span>9th Poznan Summer School of Bioinformatics, Poznan, Poland, 11-15 July</span></p>\r\n<p><span>OICR-OGI Pathway Workshop, Hospital for Sick Children, Toronto, Canada, 2-3 June</span></p>\r\n<p><span>EBI Roadshow, University of Dsseldorf &amp; Center for Neuronal Regenaration, Dsseldorf, Germany, 15-17 March.</span></p>\r\n<p><span>EBI Roadshow, University of the Western Cape, Cape Town, South Africa, 7-8 March</span></p>\r\n<p><span>EBI Roadshow, International Livestock Research Institute, Nairobi, Kenya, 2-3 March</span></p>\r\n<p><span>EBI Roadshow, Centro de Edafologa y Biologa Aplicada del Segura (CEBAS)-CSIC, Murcia, Spain, 1-3 February</span></p>\r\n<p><span>EBI Workshop, City of Hope, Duarte, California, United States, 20-21 January</span></p>\r\n<p>{tab 2010}</p>\r\n<p><span>Joint EBI-Wellcome Trust Proteomics Workshop, Hinxton, UK, 13-17 December</span></p>\r\n<p><span>Introduction to Bioinformatics at the EBI, Dept. of Genetics Cambridge University, Cambridge, UK, 30 November</span></p>\r\n<p><span>EBI Roadshow, Murdoch University, Perth, Australia, 15-18 November</span></p>\r\n<p><span>EBI Roadshow, MRC National Institute for Medical Research, London, UK, 15-19 November</span></p>\r\n<p><span>EBI Roadshow, Vilnius Institute of Biotechnology, Vilnius, Liuania, 4-5 November</span></p>\r\n<p><span>Predoc Training Course, EMBL Heidelberg, Heidelberg, Germany, 2 November</span></p>\r\n<p><span>EMBL-EBI Open Day, Hinxton, UK, 2 November</span></p>\r\n<p><span>EBI Roadshow, Dokuz Eyll University, Izmir, Turkey, 12-14 October</span></p>\r\n<p><span>COMBINE 2010, University of Edinburgh, Edinburgh, UK, 6-9 October</span></p>\r\n<p><span>EBI Roadshow, Institute of Molecular Biology, Bratislava, Slovakia, 6-7 October</span></p>\r\n<p><span>EBI Roadshow, University of Rostock, Rostock, Germany, 24-25 September</span></p>\r\n<p><span>Protein databases and tools from the EBI, Dept. of Genetics Cambridge University, Cambridge, UK, 20-21 September</span></p>\r\n<p><span>OICR Scientific Information Resource Interoperability Symposium (OSIRIS), Toronto, Canada, 15 October</span></p>\r\n<p><span>7th International Conference Data Integration in e Life Sciences conference, Goenburg, Sweden, 25-27 August</span></p>\r\n<p><span>12th UK Meeting on Platelets/12th Erfurt Conference on Platelets, Nottingham, UK, 15-16 July</span></p>\r\n<p><span>BSPR/EBI Educational Workshop  Quantitative Proteomics, Hinxton, UK, 15-16 July</span></p>\r\n<p><span>18 Annual International Conference on Intelligent Systems for Molecular Biology (ISMB 2010), Boston, USA, July 11  13</span></p>\r\n<p><span>EBI Roadshow, Teleon Institute of Genetics and Medicine, Naples, Italy, 1-2 July</span></p>\r\n<p><span>35 FEBS Congress, Gotenborg Convention Centre, Goenburg, Sweden, 26 June  1 July</span></p>\r\n<p><span>Joint EBI-Wellcome Trust Summer School in Bioinformatics, Hinxton, UK, 14  18 June</span></p>\r\n<p><span>EBI Roadshow, Charles University, Prague, Czech Republic, 2 June</span></p>\r\n<p><span>5th Canadian Conference on Ovarian Cancer Research, Toronto, Canada, 15  18 May</span></p>\r\n<p><span>ELLS LearningLab, EMBL Heidelberg, Heidelberg, Germany, 10-12 May</span></p>\r\n<p><span>97th AAI Annual Meeting Immunology 2010, Baltimore, USA, 7  11 May</span></p>\r\n<p><span>EBI Roadshow, North East USA, Chicago, USA, 26-27 April</span></p>\r\n<p><span>LipidomicNet Bioinformatics Workshop, Cambridge, UK, 26-27 April</span></p>\r\n<p><span>3nd Annual Protein Ontology Meeting, Newark, USA, April 26  28</span></p>\r\n<p><span>EBI Roadshow, University of Tor Vergata, Rome, Italy, April 15-16</span></p>\r\n<p><span>EBI Roadshow, Institut fr Molekulare Enzymtechnologie (IMET), Aachen, Germany, 13-14 April</span></p>\r\n<p><span>EBI Roadshow, Universit degli Studi di Firenze, Florence, Italy, 7-9 April</span></p>\r\n<p><span>Gene Ontology Consortium Meeting, Palo Alto, USA, 30  31 March</span></p>\r\n<p><span>Plant Bioinformatics, Hinxton, UK, 29-31 March</span></p>\r\n<p><span>International Symposium for Integrative Bioinformatics, Cambridge, UK, 22-24 March</span></p>\r\n<p><span>Perspectives in Clinical Proteomics Training Workshop, Hinxton, UK, 18-19 March</span></p>\r\n<p><span>Keystone Symposia: Biomolecular Interaction Networks: Function and Disease, Quebec City, Canada, &nbsp;7  12 March</span></p>\r\n<p><span>49th Annual Meeting of the Society of Toxicology, Salt Lake City, USA, 7  11 March</span></p>\r\n<p><span>EBI Open Day, Hinxton, UK, 4 March</span></p>\r\n<p><span>Therapeutic Applications of Computational Biology and Chemistry: TACBAC 2010, Hinxton, UK, 1-3 March</span></p>\r\n<p><span>OICR Annual Scientific Meeting 2010, Nottawasaga, Canada, 28 February  2 March</span></p>\r\n<p><span>EBI Roadshow, Jozef Stefan Institute and University of Ljubljana, Llubljana, Slovenia, 16-18 February</span></p>\r\n<p><span>EBI Roadshow, Gulbenkian Institute, Lisbon, Portugal, 9-11 February</span></p>\r\n<p><span>Bloodomics 2 Workshop &amp; Meeting Madingley Hall, Cambridge, UK, 5 February</span></p>\r\n<p><span>ENFIN Fif Annual General Meeting, Berlin, Germany, 3-5 February</span></p>\r\n<p><span>EBI Roadshow, University of Aberdeen, Aberdeen, UK, 3-4 February</span></p>\r\n<p><span>CIP-MCMM Bioinformatics Workshop, Toronto, Canada, 21 January</span></p>\r\n<p><span>Plant &amp; Animal Genomes XVIII Conference, San Diego, USA, 9  13 January</span></p>\r\n<p>{tab 2009}</p>\r\n<p><span>OGI-OICR Reactome Webinar, Toronto, Canada, 17 December</span></p>\r\n<p><span>NIH 7th Symposium on the Functional Genomics of Critical Illness &amp; Injury, Bethesda, USA, 7 December</span></p>\r\n<p><span>Functional Genomics &amp; Systems Biology Workshop, Hinxton, UK, 30 November  2 December</span></p>\r\n<p><span>Rb International Meeting, Toronto, Canada, 19  21 November</span></p>\r\n<p><span>BioPAX Hackathon and Workshop, New York, USA, 11  13 November</span></p>\r\n<p><span>Bloodomics 2 Student Workshop &amp; Meeting, Cambridge, UK, 11  12 November</span></p>\r\n<p><span>Joint EBI-Wellcome Trust Proteomics Workshop, Hinxton, UK, 9  13 November</span></p>\r\n<p><span>CSHL Genome Informatics Meeting, Cold Spring Harbor, USA, 27  30 October</span></p>\r\n<p><span>Oeiras  PaProt II Workshop, Lisbon, Portugal, 13  15 October</span></p>\r\n<p><span>Enfin EMBRACE Workshop, Helsinki, Finland, 4  6 October</span></p>\r\n<p><span>Gene Ontology Consortium Meeting, Cambridge UK, 23  25 September</span></p>\r\n<p><span>5th Metabolomics Meeting, Edmonton, Canada, 30 August  2 September</span></p>\r\n<p><span>Cancer Proteomics 2009, Dublin, Ireland, 8  11 June</span></p>\r\n<p><span>TCGA Ovarian Analysis Workshop, Berkeley, USA, 21  22 May</span></p>\r\n<p><span>3rd International Biocuration Conference, Berlin, Germany, 16  19 April</span></p>\r\n<p><span>Gene Ontology Consortium Meeting, Eugene, USA, 30  31 March</span></p>\r\n<p><span>Semantic Enrichment of the Scientific Literature 2009 (SESL 2009), 30  31 Hinxton, UK, March</span></p>\r\n<p><span>OICR Annual Scientific Meeting, Nottawasaga, Canada, 22  24 February</span></p>\r\n<p>{/tabs}</p>\r\n<p>&nbsp;</p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 15:45:48','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(24,558,'Publications','publications','<div class=\"article-publications\">\r\n<p><strong>2019</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Blucher AS, McWeeney SK, Stein L, Wu G. Visualization of drug target interactions in the contexts of pathways and networks with ReactomeFIViz. <em>F1000Res</em>. 2019 Jun 20.&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/31372215\">PubMed.</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2018</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Jupe S, Ray K, Roca CD, Varusai T, Shamovsky V, Stein L, D\'Eustachio P, Hermjakob H. Interleukins and their signaling pathways in the Reactome biological pathway database.<em>J Allergy Clin Immunol</em>. Apr 14. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29378288\">PubMed</a>.</li>\r\n<li>Fabregat A, Sidiropoulos K, Viteri G, Marin-Garcia P, Ping P, Stein L, D\'Eustachio P, Hermjakob H. Reactome diagram viewer: data structures and strategies to boost performance.<em> Bioinformatics.</em>Apr 1. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29186351\">PubMed.</a></li>\r\n<li>Fabregat A, Korninger F, Viteri G, Sidiropoulos K, Marin-Garcia P, Ping P, Wu G, Stein L, D\'Eustachio P, Hermjakob H. Reactome graph database: Efficient access to complex pathway data. <em>PLoS Comput Biol.</em>Jan 29. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29377902\">PubMed.</a></li>\r\n<li>Fabregat A, Jupe S, Matthews L, Sidiropoulos K, Gillespie M, Garapati P, Haw R, Jassal B, Korninger F, May B, Milacic M, Duenas Roca C, Rothfels K, Sevilla C, Shamovsky V, Shorser S, Varusai T, Viteri G, Weiser J, Wu G, Stein L, Hermjakob H, DEustachio P. The Reactome Pathway Knowledgebase.<em>Nucleic Acids Research.</em> Jan 4. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29145629\">PubMed.</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2017</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Sidiropoulos K, Viteri G, Sevilla C, Jupe S, Webber M, Orlic-Milacic M, Jassal B, May B, Shamovsky V, Duenas C, Rothfels K, Matthews L, Song H, Stein L, Haw R, DEustachio P, Ping P, Hermjakob H, Fabregat A. Reactome enhanced pathway visualization. <em>Oxford Bioinformatics.</em> July 6. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29077811\">PubMed</a>.</li>\r\n<li>Fabregat A, Sidiropoulos K, Viteri G, Forner O, Marin-Garcia P, Arnau V, DEustachio P, Stein L, Hermjakob H. Reactome pathway analysis: a high-performance in-memory approach.<em>BMC Bioinformatics.</em>Mar 2.&nbsp;<a href=\"http://ncbi.nlm.nih.gov/pubmed/28249561\">PubMed</a>.</li>\r\n<li>Wu G, Haw R. Functional Interaction Network Construction and Analysis for Disease Discovery.<em>Methods Mol Biol.&nbsp;</em><a href=\"http://ncbi.nlm.nih.gov/pubmed/28150241\">PubMed</a>.</li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2016</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Hill DP, DEustachio P, Berardini TZ, Mungall CJ, Renedo N, Blake JA. Modeling biochemical pathways in the gene ontology.<em>Database (Oxford)</em>. 2016 Sep 1.&nbsp;<a href=\"http://ncbi.nlm.nih.gov/pubmed/27589964\">PubMed</a>.</li>\r\n<li>Bohler A, Wu G, Kutmon M, Pradhana LA, Coort SL, Hanspers K, Haw R, Pico AR, Evelo CT. Reactome from a WikiPathways Perspective. PLoS Comput Biol. May.&nbsp;<a href=\"http://ncbi.nlm.nih.gov/pubmed/27203685\">PubMed</a>.</li>\r\n<li>Fabregat A, Sidiropoulos K, Garapati P, Gillespie M, Hausmann K, Haw R, Jassal B, Jupe S, Korninger F, McKay S, Matthews L, May B, Milacic M, Rothfels K, Shamovsky V, Webber M, Weiser J, Williams M, Wu G, Stein L, Hermjakob H, DEustachio P. The Reactome pathway Knowledgebase.<em>Nucleic Acids Res</em>. Dec 9.&nbsp;<a href=\"http://ncbi.nlm.nih.gov/pubmed/26656494\">PubMed</a>.</li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2015</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li class=\"title\">McKay SJ, Weiser J. Installing a Local Copy of the Reactome Web Site and Knowledgebase. Curr Protoc<em>Bioinformatics</em>. 50:9.10.1-9.10.10.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/26087747\">PubMed</a>.</li>\r\n<li class=\"title\">Jupe S, Fabregat A, and Hermjakob H.Expression data analysis withreactome.<em><span class=\"jrnl\" title=\"Current protocols in bioinformatics / editoral board, Andreas D. Baxevanis ... [et al.]\">Curr Protoc Bioinformatics</span></em>. Chapter 8 Unit8.20.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/25754994\">PubMed</a>.</li>\r\n<li class=\"title\">Porras P, Duesbury M, Fabregat A, Ueffing M, Orchard S, Gloeckner CJ, and Hermjakob H.A visual review of the interactome of LRRK2: Using deep-curated molecular interactions data to represent biology.<em>Proteomics.&nbsp;</em><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/25648416\">PubMed</a>.</li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2014</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li><span class=\"article-title-and-info in-popup\">Wu G, Dawson E, Duong A, Haw R and Stein L.ReactomeFIViz: the Reactome FI Cytoscape app for pathway and network-based data analysis.3:146</span><em>F1000Research</em>.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/25309732\">PubMed</a>.</li>\r\n<li>Jupe S, Jassal B, Williams M, and Wu G. A controlled vocabulary for pathway entities and events.<em>Database (Oxford)</em><em>Jun.&nbsp;</em><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/24951798\">PubMed</a>.</li>\r\n<li>Backman S, Kollara A, Haw R, Stein L, Brown TJ. Glucocorticoid-Induced Reversal of Interleukin-1-Stimulated Inflammatory Gene Expression in Human Oviductal Cells. PLoS ONE 9(5).&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/24848801\">PubMed</a>.</li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2013</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Croft D, Fabregat Mundo A, Haw R, Milacic, M, Weiser J, Wu G, Caudy M,Garapati P, Gillespie M, Kamdar MR,Jassal B, Jupe S, Matthews L,May B, Palatnik S, Rothfels K, Shamovsky V, Song H, Williams M, Birney E, Hermjakob H, Stein L and DEustachio P.The Reactome pathway knowledgebase.<em>Nucleic Acids Res.&nbsp;</em><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/24243840\">PubMed</a></li>\r\n<li>Monaco MK, Stein J, Naithani S, Wei S, Dharmawardhana P, Kumari S, Amarasinghe V, Youens-Clark K, Thomason J, Preece J, Pasternak S, Olson A, Jiao Y, Lu Z, Bolser D, Kerhornou A, Staines D, Walts B, Wu G, DEustachio P, Haw R, Croft D, Kersey PJ, Stein L, Jaiswal P, Ware D. Gramene 2013: comparative plant genomics resources.<em>Nucleic Acids Res</em>.<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/24217918\">PubMed</a>Croft, D. Building models using reactome pathways as templates.<em>Methods Mol Biol</em>. 1021:273-83.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/23715990\">PubMed</a></li>\r\n<li>DEustachio, P. Pathway databases: making chemical and biological sense of the genomic data flood.<em>Chem Biol</em>. 20(5):629-35.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/23706629\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2012</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Milacic M, Haw R, Rothfels K, Wu G, Croft D, Hermjakob H, DEustachio P and Stein L. Annotating Cancer Variants and Anti-Cancer Therapeutics in Reactome.<em>Cancers.</em>4 (4), 1180-1211.&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/24213504\">PubMed.</a></li>\r\n<li>Wu G and Stein L. A network module-based method for identifying cancer prognostic signatures.<em>Genome Biol</em>. 13(12):R112.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/23228031\">PubMed</a></li>\r\n<li>Kim Y, Ignatchenko V, Yao CQ, Kalatskaya I, Nyalwidhe JO, Lance RS, Gramolini AO, Troyer DA, Stein LD, Boutros PC, Medin JA, Semmes OJ, Drake RR, Kislinger T. Identification of differentially expressed proteins in direct expressed prostatic secretions of men with organ-confined versus extracapsular prostate cancer.<em>Mol Cell Proteomics</em>. 11(12):1870-84.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22986220\">PubMed</a></li>\r\n<li>Haw R and Stein L. Using the reactome database.<em>Curr Protoc Bioinformatics</em>. Chapter 8:Unit 8.7.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22700314\">PubMed</a></li>\r\n<li>Reveiller M, Ghatak S, Toia L, Kalatskaya I, Stein L, DSouza M, Zhou Z, Bandla S, Gooding WE, Godfrey TE, Peters JH. Bile exposure inhibits expression of squamous differentiation genes in human esophageal epithelial cells.<em>Ann Surg</em>. 255(6):1113-20.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22498892\">PubMed</a></li>\r\n<li>Sprowl JA, Reed K, Armstrong SR, Lanner C, Guo B, Kalatskaya I, Stein L, Hembruff SL, Tam A, Parissenti AM. Alterations in tumor necrosis factor signaling pathways are associated with cytotoxicity and resistance to taxanes: a study in isogenic resistant tumor cells.<em>Breast Cancer Res</em>. 14(1):R2.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22225778\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2011</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Gieger C, Radhakrishnan A, Cvejic A, Tang W, Porcu E, Pistis G, Serbanovic-Canic J, Elling U,Goodall AH,.. Soranzo N. New gene functions in megakaryopoiesis and platelet formation.<em>Nature</em>. 480(7376):201-8.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22139419\">PubMed</a></li>\r\n<li>Ndegwa N, Ct RG, Ovelleiro D, DEustachio P, Hermjakob H, Vizcano JA, Croft D. Critical amino acid residues in proteins: a BioMart integration of Reactome protein annotations with PRIDE mass spectrometry data and COSMIC somatic mutations.<em>Database (Oxford)</em>. bar047.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22025670\">PubMed</a></li>\r\n<li>Haw RA, Croft D, Yung CK, Ndegwa N, DEustachio P, Hermjakob H, Stein LD. The Reactome BioMart.<em>Database (Oxford)</em>. bar047.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22012987\">PubMed</a></li>\r\n<li>Bult CJ, Drabkin HJ, Evsikov A, Natale D, Arighi C, Roberts N, Ruttenberg A, DEustachio P, Smith B, Blake JA, Wu C.<em>The representation of protein complexes in the Protein Ontology (PRO)</em>. BMC Bioinformatics. 12:371.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21929785\">PubMed</a></li>\r\n<li>Haw R, Hermjakob H, DEustachio P, Stein L. Reactome pathway analysis to enrich biological discovery in proteomics data sets.<em>Proteomics</em>. 11(18):3598-613.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21751369\">PubMed</a></li>\r\n<li>Ismail A, Bandla S, Reveiller M, Toia L, Zhou Z, Gooding WE, Kalatskaya I, Stein L, DSouza M, Litle VR, Peters JH, Pennathur A, Luketich JD, Godfrey TE. Early G cyclin-dependent kinases as prognostic markers and potential therapeutic targets in esophageal adenocarcinoma.<em>Clin Cancer Res</em>. 17(13):4513-22.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21593195\">PubMed</a></li>\r\n<li>Sawey ET, Chanrion M, Cai C, Wu G, Zhang J, Zender L, Zhao A, Busuttil RW, Yee H, Stein L, French DM, Finn RS, Lowe SW, and Powers S. Identification of a therapeutic strategy targeting amplified FGF19 in liver cancer by oncogenomic screening.<em>Cancer Cell.</em>19(3): 347-58.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21397858\">PubMed</a></li>\r\n<li>Jassal B. Pathway annotation and analysis with Reactome: The solute carrier class of membrane transporters.<em>Human Genomics</em>. 5(4):310-5.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21712192\">PubMed</a></li>\r\n<li>DEustachio P. Reactome knowledgebase of human biological pathways and processes.<em>Methods Mol Biol.</em>694:49-61.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21082427\">PubMed</a></li>\r\n<li>Gillespie M, Shamovsky V, DEustachio P. Human and chicken TLR pathways: manual curation and computer-based orthology analysis.<em>Mammalian Genome.</em>22(1-2):130-8.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21052677\">PubMed</a></li>\r\n<li>Croft D, OKelly G, Wu G, Haw R, Gillespie M, Matthews L, Caudy M, Garapati P, Gopinath G, Jassal B, Jupe S, Kalatskaya I, Mahajan S, May B, Ndegwa N, Schmidt E, Shamovsky V, Yung C, Birney E, Hermjakob H, DEustachio P, Stein L. Reactome: a database of reactions, pathways and biological processes.<em>Nucleic Acids Res. (Database issue).</em>D691-7.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21067998\">PubMed</a></li>\r\n<li>Dallolio GM, Jassal B, Montanucci L, Gagneux P, Bertranpetit J, Laayouni H. The annotation of the Asparagine N-linked Glycosylation pathway in the Reactome Database.<em>Glycobiology</em>. 21(11):1395-400.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21199820\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2010</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Voight BF, Scott LJ, Steinthorsdottir V, Morris AP, Dina C, Welch RP, Zeggini E, Huth C, Aulchenko YS, Thorleifsson G, McCulloch LJ, Ferreira T, Grallert H, Amin N, Wu G, . McCarthy MI. Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis.<em>Nat Genet.</em>42(7): 579-89.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/20581827\">PubMed</a></li>\r\n<li>Jassal B, Jupe S, Caudy M, Birney E, Stein L, Hermjakob H, DEustachio P. The systematic annotation of the three main GPCR families in Reactome.<em>Database (Oxford)</em>.baq018.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/20671204\">PubMed</a></li>\r\n<li>Antonov AV, Schmidt EE, Dietmann S, Krestyaninova M, Hermjakob H. R spider: a network-based analysis of gene lists by combining signaling and metabolic pathways from Reactome and KEGG databases.<em>Nucleic Acids Res. (Web Server issue)</em>38:W78-83.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/20519200\">PubMed</a></li>\r\n<li>Wu G, Feng X, Stein L. A human functional protein interaction network and its application to cancer data analysis.<em>Genome Biol.</em>11(5):R53.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/20482850\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2009</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Matthews L, Gopinath G, Gillespie M, Caudy M, Croft D, de Bono B, Garapati P, Hemish J, Hermjakob H, Jassal B, Kanapin A, Lewis S, Mahajan S, May B, Schmidt E, Vastrik I, Wu G, Birney E, Stein L, DEustachio P. Reactome knowledgebase of human biological pathways and processes.<em>Nucleic Acids Res.</em>37:D619-22.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/18981052\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2007</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Vastrik I, DEustachio P, Schmidt E, Gopinath G, Croft D, de Bono B, Gillespie M, Jassal B, Lewis S, Matthews L, Wu G, Birney E, Stein L.Reactome: a knowledge base of biologic pathways and processes.<em>Genome Biol</em>. 8:R39.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/17367534\">PubMed</a></li>\r\n<li>Matthews L, DEustachio P, Gillespie M, Croft D, de Bono B, Gopinath G, Jassal B, Lewis S, Schmidt E, Vastrik I, Wu G, Birney E, Stein L. An Introduction to the Reactome Knowledgebase of Human Biological Pathways and Processes.<em>Bioinformatics Primer, NCI/Nature Pathway Interaction Database.&nbsp;</em><a href=\"http://pid.nci.nih.gov/PID/2007/071211/full/pid.2007.3.shtml\">doi:10.1038/pid.2007.3.</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2005</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Joshi-Tope G, Gillespie M, Vastrik I, DEustachio P, Schmidt E, de Bono B, Jassal B, Gopinath GR, Wu GR, Matthews L, Lewis S, Birney E, Stein L. Reactome: a knowledgebase of biological pathways.<em>Nucleic Acids Res.</em>1:D428-32.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/15608231\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2004</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Stein LD. Using the Reactome database.<em>Curr Protoc Bioinformatics.</em>Chapter 8:Unit 8.7.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/18428737\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2003</strong></p>\r\n<ul>\r\n<ul>\r\n<li>Joshi-Tope G, Vastrik I, Gopinath GR, Matthews L, Schmidt E, Gillespie M, DEustachio P, Jassal B, Lewis S, Wu G, Birney E, Stein L. The Genome Knowledgebase: a resource for biologists and bioinformaticists.<em>Cold Spring Harb Symp Quant Biol.</em>68:237-43.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/15338623\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</div>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 15:46:05','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(25,559,'Training Materials','training','<p></p>\r\n<p style=\"text-align: justify;\"><b>Users Guide. </b><span>One of our most valuable user documents is the </span><a href=\"user/guide\"><span>Reactome Users Guide</span></a><span>. Please look through the guide and </span><span>contact us</span><span> if our guide needs something new.</span></p>\r\n<p style=\"text-align: justify;\"><b>Curators Guide</b><span>. The document contains an overview of the curatorial process as well as a step by step guide for annotating Reactome pathways. </span></p>\r\n<p style=\"text-align: justify;\"><b>Reactome Tutorial Material. </b><span>All our training and outreach materials are available under a </span><a href=\"license\"><span>Creative Commons Attribution 4.0 Unported License</span></a><span>. We have just two simple requests, please attribute Reactome, and let us know if you use our presentations, posters or training resources. Here are some of the material that we make available to download:</span></p>\r\n<ul style=\"padding-left: 30px;\">\r\n<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><a href=\"docs/training/Reactome_Website.pdf\"><span>Tutorial workshop</span></a><span> slides. These slides introduce new users to Reactome.</span></li>\r\n<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><a href=\"docs/training/Reactome_Tutorial_May_2017.docx\"><span>Tutorial handout</span></a><span>. This document provides additional information about the Reactome website and suite of data analysis tools, including exercises to allow you to check your understanding. Answers are provided and were correct when written, but please note that answers may change over time as more pathways are added to Reactome.</span></li>\r\n<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><span><a href=\"docs/training/Pathways_&amp;_Networks_Overview.pdf\">Pathways and Networks Overview</a>. These slides provide an overview of pathway and network resources for data visualisation, analysis, and integration.</span></li>\r\n<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><span><a href=\"docs/training/ReactomeFIVizapp.pdf\">Reactome FI Viz app</a>. These slides provide an introduction to the Reactome Functional Interaction Network and Cytoscape app. These Reactome </span><a href=\"docs/training/ReactomeFIVizapp_Lab_Questions.pdf\"><span>Questions</span></a><span> and </span><a href=\"docs/training/ReactomeFIVizapp_Lab_Answers.pdf\"><span>Answers</span></a><span> provide a series of questions and answers to demonstrate the functionality of the Reactome FI Viz app. </span></li>\r\n<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><span>Online Training courses. Available via the EBI Train Online system - </span><a href=\"https://www.ebi.ac.uk/training/online/course/reactome-quick-tour\"><span>Reactome:Quick Tour</span></a><span>, </span><a href=\"https://www.ebi.ac.uk/training/online/course/reactome-exploring-biological-pathways\"><span>Reactome: Exploring Biological Pathways</span></a><span>, </span><span><a href=\"https://www.ebi.ac.uk/training/online/course/reactome-tools-analysis-biological-pathways\">Reactome: Tools for Analysis of Biological Pathways</a></span></li>\r\n<li></li>\r\n</ul>\r\n<h3>Reactome Pathway Browser: A short introdution</h3>\r\n<p>{source}&lt;style&gt;.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }&lt;/style&gt;&lt;div class=\'embed-container\'&gt;&lt;iframe src=\'https://www.youtube.com/embed/-skixrvI4nU\' frameborder=\'0\' allowfullscreen&gt;&lt;/iframe&gt;&lt;/div&gt;{/source}</p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 15:46:13','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(26,560,'Partners','partners','<style type=\"text/css\">#fav-maincontent {\r\n    margin-bottom: 5px;\r\n}\r\np {\r\n    margin: 0 0 0 0;\r\n}</style>\r\n<p>The following table contains the known resources that have integrated Reactome by using the&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=47&amp;catid=29\">Analysis Service</a>, the <a href=\"index.php?option=com_content&amp;view=article&amp;id=45&amp;catid=29\">Graph Database</a>&nbsp;or any of our widgets (<a href=\"index.php?option=com_content&amp;view=article&amp;id=50&amp;catid=29\">diagram</a>&nbsp;or <a href=\"index.php?option=com_content&amp;view=article&amp;id=49&amp;catid=29\">pathways overview</a>). If your resource is using our services, widgets or content and it is not in the list,&nbsp;please <a href=\"mailto:help@reactome.org?subject=Including%20a%20Reactome%20Partner\">let us know</a>&nbsp;and we will add it.</p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 16:46:52','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":\"\",\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":\"\",\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":\"\",\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(27,561,'Resources Guide','resources-guide','<style type=\"text/css\">@media only screen and (max-width: 767px) {div[id^=\"r-responsive-table\"] table.reactome.resource-guide td:nth-of-type(1):before { content: \"Name\"; }\r\ndiv[id^=\"r-responsive-table\"] table.reactome.resource-guide td:nth-of-type(2):before { content: \"Description\"; }}</style>\r\n<p><span >We actively seek collaborations with other data resources and users to improve data integration, share efforts, and make our data maximally useful to biologists and bioinformaticians. Here are a list of software tools, websites, databases and research projects that: i) support the use of Reactome data, ii) have integrated Reactome data, or iii) provide data linkages to the Reactome website. </span></p>\r\n<p><span >If we are missing your website, database, software tool and project, please contact our </span><a href=\"mailto: &lt;a href=\"></a><a href=\"mailto:help@reactome.org\"><span >Help Desk</span></a><span >.</span></p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome resource-guide\">\r\n<thead>\r\n<tr><th scope=\"col\">\r\n<p><b>Name</b></p>\r\n</th><th scope=\"col\">\r\n<p><b>Description</b></p>\r\n</th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p><span >ABACUS</span></p>\r\n</td>\r\n<td>\r\n<p><span >ABACUS is an Algorithm based on a BivAriate CUmulative Statistic to identify single nucleotide polymorphisms (SNPs) significantly associated with a disease within predefined sets of SNPs such as pathways or genomic regions</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >AbsIDconvert</span></p>\r\n</td>\r\n<td>\r\n<p><span >AbsIDconvert is an absolute approach for converting genetic identifiers at different granularities</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ADaCGH</span></p>\r\n</td>\r\n<td>\r\n<p><span >ADaCGH is a parallelized web-based application and R package for the analysis of aCGH data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >AgBase</span></p>\r\n</td>\r\n<td>\r\n<p><span >AgBase is a functional genomics resource for agriculture</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Algal Functional Annotation Tool</span></p>\r\n</td>\r\n<td>\r\n<p><span >Algal Functional Annotation Tool is a web-based analysis suite to functionally interpret large gene lists using integrated annotation and expression data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >AltAnalyze</span></p>\r\n</td>\r\n<td>\r\n<p><span >AltAnalyze is a platform for analyzing and visualizing exon expression data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >AMMO_Prot</span></p>\r\n</td>\r\n<td>\r\n<p><span >AMMO-Prot is an amine system project 3D-model finder</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Annotate-it</span></p>\r\n</td>\r\n<td>\r\n<p><span >Annotate-it is a Swiss-knife approach to annotation, analys is and interpretation of single nucleotide variation in human disease</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Arabidopsis Reactome</span></p>\r\n</td>\r\n<td>\r\n<p><span >Arabidopsis Reactome is a curated knowledgebase of plant biological pathways that contains many plant pathways curated by the John Innes Centre team, plus plant pathways imported from AraCyc and KEGG</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ARACNe</span></p>\r\n</td>\r\n<td>\r\n<p><span >ARACne is an Algorithm for the Reconstruction of Gene Regulatory Networks in a Mammalian Cellular Context</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Asterias</span></p>\r\n</td>\r\n<td>\r\n<p><span >Asterias includes applications for the analysis of genomic and proteomic data that cover from data normalization to development of prediction models for survival data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >atBioNet</span></p>\r\n</td>\r\n<td>\r\n<p><span >atBioNet is an integrated network analysis tool for genomics and biomarker discovery</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Atlas of Genetics and Cytogenetics in Oncology and Haematology</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Atlas of Genetics and Cytogenetics in Oncology and Haematology is a peer reviewed on-line journal and database in free access on internet devoted to genes, cytogenetics, and clinical entities in cancer, and cancer-prone diseases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >AutismKB</span></p>\r\n</td>\r\n<td>\r\n<p><span >An evidence-based knowledgebase of autism genetics</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Babelomics</span></p>\r\n</td>\r\n<td>\r\n<p><span >Babelomics is a complete suite of web tools for the analysis, integration and interpretation of different types of genomic data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Baysian Pathway Analysis</span></p>\r\n</td>\r\n<td>\r\n<p><span >A computational method to identify significant pathways from microarray data with samples from test and control classes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BcCluster</span></p>\r\n</td>\r\n<td>\r\n<p><span >BcCluster is a Bladder Cancer Database at the Molecular Level</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BCML</span></p>\r\n</td>\r\n<td>\r\n<p><span >The biological Connection Markup Language is a SBGN-compliant format for visualization, filtering and analysis of biological pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BIANA</span></p>\r\n</td>\r\n<td>\r\n<p><span >Biological interactions and network analysis (BIANCA) is a software framework for compiling biological interactions and analyzing networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BindingDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >A public, web-accessible database of measured binding affinities, focusing chiefly on the interactions of protein considered to be drug-targets with small, drug-like molecules</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BiNoM</span></p>\r\n</td>\r\n<td>\r\n<p><span >Biological Network Manager is a Cytoscape plug-in, developed to facilitate the manipulation of biological networks represented in standard systems biology formats (SBML, SBGN, BioPAX)</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Bio3graph</span></p>\r\n</td>\r\n<td>\r\n<p><span >Automated extraction of biological relations from the literature, resulting in a set of (component1, reaction, component2) triplets and composing a graph structure which can be visualised, compared to the manually constructed topology and examined by the experts</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioBin</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioBin is a bioinformatics tool for automating the binning of rare variants using publicly available biological knowledge</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioDB is an ontology-enhanced information system for heterogeneous biological information</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioDWH</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioDWH is a Data Warehouse Kit for Life Science Data Integration</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Biofilter</span></p>\r\n</td>\r\n<td>\r\n<p><span >Biofilter is a knowledge-integration system for the multi-locus analysis of genome-wide association studies</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >biofilter 2.0</span></p>\r\n</td>\r\n<td>\r\n<p><span >A single interface for accessing multiple publicly available human genetic data sources that have been compiled in the supporting database of Library of Knowledge Integration (LOKI)</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioGPS</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioGPS is a free extensible and customizable gene annotation portal, a complete resource for learning about gene and protein function</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioGRID</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Biological General Repository for Interaction Datasets (BioGRID) database was developed to house and distribute collections of protein and genetic interactions from major model organism species</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioHealthBase BRC</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioHealthBase Bioinformatics Resource Center (BRC) is a public bioinformatics database and analysis resource for the study of specific biodefense and public health pathogens-Influenza virus, Francisella tularensis, Mycobacterium tuberculosis, Microsporidia species and ricin toxin</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioJS</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioJS is an open source standard for biological visualisation</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BiologicalNetworks</span></p>\r\n</td>\r\n<td>\r\n<p><span >Data integration and network based research environment with tools for inference and analysis of gene regulatory modules</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioMart</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioMart enables scientists to perform advanced querying of biological data sources through a single web interface</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioMoby</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioMOBY is an Open Source research project which aims to generate an architecture for the discovery and distribution of biological data through web services</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioModels</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioModels Database is a data resource that allows biologists to store, search and retrieve published mathematical models of biological interests</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioMyn</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioMyn is an integrated human gene and protein knowledgebase</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioPAX</span></p>\r\n</td>\r\n<td>\r\n<p><span >Biological Pathway Exchange (BioPAX) is a RDF/OWL-based standard for the exchange of biological pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioPAX Cytoscape PlugIn</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioPAX Cytoscape PlugIn enables Cytoscape users to load and visually render BioPAX files from the web or a local file system; distributed with Reactome sample files</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioPP</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioPP is a tool for web-publication of biological networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioProfiling.de</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioProfiling de provides a common interface for a collection of recently developed analytical tools for genomics, proteomics and metabolimics data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioSemantic</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioSemantic is a framework that was designed to speed integration of relational biological databases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Biovista</span></p>\r\n</td>\r\n<td>\r\n<p><span >Systematic drug repositioning capability offers decision makers a reliable tool for pipeline optimization, asset protection and life cycle management</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BioWisdom SRS</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioWisdom Sequence Retrival System (SRS) is a proven and scalable data integration platform delivering genomic data to thousands of users</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BirdsEyeView</span></p>\r\n</td>\r\n<td>\r\n<p><span >Visualizing experimental transcriptomic data using different views that users can switch among and compare</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BirdsEyeView</span></p>\r\n</td>\r\n<td>\r\n<p><span >BirdsEyeView (BEV) is a graphical overviews of experimental data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BisoGenet</span></p>\r\n</td>\r\n<td>\r\n<p><span >BisoGenet is a Cytsocape app for gene network building, visualization and analysis</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BNP</span></p>\r\n</td>\r\n<td>\r\n<p><span >Bayesian Network Prior (BNP) depicts the relation between various evidence types that contribute to the event gene interaction and is used to calculate the probability of a candidate graph (G) in the structure learning process</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >BowTieBuilder</span></p>\r\n</td>\r\n<td>\r\n<p><span >BowTieBuilder is aModeling signal transduction pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >C2Cards - Human</span></p>\r\n</td>\r\n<td>\r\n<p><span >Consensus and Conflict Cards (C2Cards) provides concise overviews of what pathway databases do or do not agree on</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >caBIG</span></p>\r\n</td>\r\n<td>\r\n<p><span >cancer Biomedical Informatics Grid caBIG is an information network enabling all constituencies in the cancer community _ researchers, physicians, and patients _ to share data and knowledge</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CAERUS</span></p>\r\n</td>\r\n<td>\r\n<p><span >CAERUS predicts CAncER oUtcomeS Using Relationship between Protein Structural Information, Protein Networks, Gene Expression Data, and Mutation Data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CancerResource</span></p>\r\n</td>\r\n<td>\r\n<p><span >CancerResource is a comprehensive database of cancer-relevant proteins and compound interactions supported by experimental knowledge</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CANGES</span></p>\r\n</td>\r\n<td>\r\n<p><span >CANGES provides a means to rapidly identify networks for any set of genes and generate novel hypotheses</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >canSAR</span></p>\r\n</td>\r\n<td>\r\n<p><span >canSAR is an integrated cancer public translational research and drug discovery resource</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CardioNet</span></p>\r\n</td>\r\n<td>\r\n<p><span >A human metabolic network suited for the study of cardiomyocyte metabolism</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CardioNet</span></p>\r\n</td>\r\n<td>\r\n<p><span >CardioNet is a human metabolic network suited for the study of cardiomyocyte metabolism</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CardioVINEdb</span></p>\r\n</td>\r\n<td>\r\n<p><span >CardioVINEdb is a data warehouse approach for integration of life science data in cardiovascular diseases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CCDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >Cell Cycle Database is a collection of genes and proteins involved in human and yeast cell cycle</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CEGP</span></p>\r\n</td>\r\n<td>\r\n<p><span >Cancer Gene Encyclopedia will allow scientists to identify new cancer targets</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Cell Cycle Ontology</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Cell Cycle Ontology is an application ontology for the representation and integrated analysis of the cell cycle process</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Cell Illustrator</span></p>\r\n</td>\r\n<td>\r\n<p><span >Cell Illustrator is an environment for simulating and representing biological systems</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CellBase</span></p>\r\n</td>\r\n<td>\r\n<p><span >CellBase is a comprehensive collection of RESTful web services for retrieving relevant biological information from heterogeneous sources</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CellCircuits</span></p>\r\n</td>\r\n<td>\r\n<p><span >CellCircuits is a database of protein network models</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CellDesigner</span></p>\r\n</td>\r\n<td>\r\n<p><span >CellDesigner is a structured diagram editor for drawing gene-regulatory and biochemical networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CellFrame</span></p>\r\n</td>\r\n<td>\r\n<p><span >CellFrame is a data structure for abstraction of cell biology experiments and construction of perturbation networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Characterization Tool</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Characterization Tool is a novel knowledge-based database and web application that allows the storage and analysis of various kinds of data associated with cell, cell line and tissue characterization across different species</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ChEBI</span></p>\r\n</td>\r\n<td>\r\n<p><span >Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on \'small\' chemical compounds</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Chem2Bio2RDF</span></p>\r\n</td>\r\n<td>\r\n<p><span >Chem2Bio2RDF is a semantic framework for linking and data mining chemogenomic and systems chemical biology data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ChemProt</span></p>\r\n</td>\r\n<td>\r\n<p><span >ChemPort is a disease chemical biology database, which is based on a compilation of multiple chemical_protein annotation resources, as well as disease-associated protein_ protein interactions</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ChiBE</span></p>\r\n</td>\r\n<td>\r\n<p><span >Chisio BioPAX Editor (ChiBE) is a free editing and visualization tool for pathway models represented by the BioPAX format</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Chipster</span></p>\r\n</td>\r\n<td>\r\n<p><span >Chipster is a user-friendly analysis software for microarray and other high-throughput data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CIDeR</span></p>\r\n</td>\r\n<td>\r\n<p><span >CIDeR is aMultifactorial interaction networks in human diseases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CIDMS</span></p>\r\n</td>\r\n<td>\r\n<p><span >Cardiac Integrated Database Management System (CIDMS) integrates the heterogeneous -omic data (available from multitude of sources and in various formats such as publication and associated supplementary files, databases and web pages) that are generated at different biological levels</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Cluepedia</span></p>\r\n</td>\r\n<td>\r\n<p><span >CluePedia provides insights into pathways by integrating experimental and in silico information</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CMS</span></p>\r\n</td>\r\n<td>\r\n<p><span >A Web-Based System for Visualization and Analysis of Genome-Wide Methylation Data of Human Cancers</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >COFECO</span></p>\r\n</td>\r\n<td>\r\n<p><span >COFECO is a web-based tool for a composite annotation of protein complexes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ConsensusPathDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >ConsensusPathDB is a database system for the integration of human functional interactions</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >COPASI</span></p>\r\n</td>\r\n<td>\r\n<p><span >COPASI is a software application for simulation and analysis of biochemical networks and their dynamics</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >cPath</span></p>\r\n</td>\r\n<td>\r\n<p><span >cPath is an open source pathway database and software suite designed for systems biology research</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CPNM</span></p>\r\n</td>\r\n<td>\r\n<p><span >Context-specific Protein Network Miner derives context-specific PPI networks in real-time from the PubMed database based on a set of user-input keywords and enhanced PubMed query system</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CSAX</span></p>\r\n</td>\r\n<td>\r\n<p><span >Characterizing Systematic Anomalies in eXpression Data (CSAX) aids in both identifying anomalies and explaining the underlying biology in expression datasets</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CTD</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Comparative Toxicogenomics Database (CTD) is a curated database that promotes understanding about the effects of environmental chemicals on human health</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CTSA Pharmaceutical Assests Portal</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Pharmaceutical Assets Portal aims to facilitate industry-academic collaborations for discovery of new indications for compounds no longer being developed by pharmaceutical companies, through eliminating barriers to access such compounds</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >CyTargetLinker</span></p>\r\n</td>\r\n<td>\r\n<p><span >A Cytoscape App to Integrate Regulatory Interactions in Network Analysis</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Cytoscape</span></p>\r\n</td>\r\n<td>\r\n<p><span >Cytoscape is a free software package for visualizing, modeling and analyzing molecular and genetic interaction networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DAPD</span></p>\r\n</td>\r\n<td>\r\n<p><span >DAPD is a knowledgebase for Diabetes Associated Proteins</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DAPPER</span></p>\r\n</td>\r\n<td>\r\n<p><span >DAPPER is a data-mining resource for protein-protein interactions</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DAPPLE</span></p>\r\n</td>\r\n<td>\r\n<p><span >Disease Association Protein-Protein Link Evaluator (DAPPLE) looks for significant physical connectivity among proteins encoded for by genes in loci associated to disease according to protein-protein interactions reported in the literature</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DAVID</span></p>\r\n</td>\r\n<td>\r\n<p><span >Database for Annotation, Visualization and Integrated Discovery (DAVID) provides a comprehensive set of functional annotation tools for investigators to understand biological meaning behind large list of genes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DawnRank</span></p>\r\n</td>\r\n<td>\r\n<p><span >DawnRank is a discover tool for personalized driver genes in cancer</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >dbOGAP</span></p>\r\n</td>\r\n<td>\r\n<p><span >dbOGAP is a Database of O-GlcNAcylated Proteins and Sites</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DC-ATLAS</span></p>\r\n</td>\r\n<td>\r\n<p><span >DC-ATLAS is a systems biology resource to dissect receptor specific signal transduction in dendritic cells</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DDEC</span></p>\r\n</td>\r\n<td>\r\n<p><span >Dragon Database of genes implicated in Esophageal Cancer (DDEC) is an integrated knowledge database aimed at representing a gateway to esophageal cancer related data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DDESC</span></p>\r\n</td>\r\n<td>\r\n<p><span >DDESC is Dragon database for exploration of sodium channels in humans,</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DDPC</span></p>\r\n</td>\r\n<td>\r\n<p><span >Dragon Database of Genes associated with Prostate Cancer (DDPC) is an integrated knowledge database aimed at representing a gateway to prostate cancer related data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DECODE</span></p>\r\n</td>\r\n<td>\r\n<p><span >DECODE is an integrated differential co-expression and differential expression analysis of gene expression data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DeDaL</span></p>\r\n</td>\r\n<td>\r\n<p><span >DeDaL is a Cytoscape 3 app for producing and morphing data-driven and structure-driven network layouts</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DEEP</span></p>\r\n</td>\r\n<td>\r\n<p><span >DEEP is a tool for differential expression effector prediction</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DESHCV</span></p>\r\n</td>\r\n<td>\r\n<p><span >Dragon Exploratory System on Hepatitis C Virus (DESHCV) is a biomedical text-mining and relationship exploring knowledgebase is the first reported comprehensive Hepatitis C Virus (HCV)</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DESSCD</span></p>\r\n</td>\r\n<td>\r\n<p><span >Dragon Exploration System for Sickle Cell Disease permits the exploration of text-mined and data-mined information about SCD</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DESTAF</span></p>\r\n</td>\r\n<td>\r\n<p><span >Dragon Exploration System for Toxicants and Fertility (DESTAF) is a specialized, public resource specifically aimed at researchers investigating reproductive toxicity</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DICS</span></p>\r\n</td>\r\n<td>\r\n<p><span >The DICS database is a dynamic web repository of computationally predicted functional modules from the human protein-protein interaction network</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Dintor</span></p>\r\n</td>\r\n<td>\r\n<p><span >Dintor is a data integration framework that provides a set of over 30 tools to assist researchers in the exploration of genomics and proteomics datasets</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Discovery-2</span></p>\r\n</td>\r\n<td>\r\n<p><span >An interactive resource for the rational selection and comparison of putative drug target proteins in malaria Phelelani T</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DisGeneNet</span></p>\r\n</td>\r\n<td>\r\n<p><span >DisGeNET is a discovery platform integrating information on gene-disease associations (GDAs) from several public data sources and the literature</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DNAtraffic</span></p>\r\n</td>\r\n<td>\r\n<p><span >DNAtraffic database is dedicated to be an unique comprehensive and richly annotated database of genome dynamics during the cell life</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DR-GAS</span></p>\r\n</td>\r\n<td>\r\n<p><span >DR-GAS is a database of functional genetic variants and their phosphorylation states in human DNA repair systems</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DriverDB:</span></p>\r\n</td>\r\n<td>\r\n<p><span >DriverDB is an exome sequencing database for cancer driver gene identification</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DriverNet</span></p>\r\n</td>\r\n<td>\r\n<p><span >DriverNet is auncovering the impact of somatic driver mutations on transcriptional networks in cancer</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DroID</span></p>\r\n</td>\r\n<td>\r\n<p><span >Drosophila Interactions Database is a comprehensive resource for annotated gene and protein interactions</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DrugViz</span></p>\r\n</td>\r\n<td>\r\n<p><span >DrugViz is a Cytoscape plugin that is designed to visualize and analyze small molecules within the framework of the interactome</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >DRYGIN</span></p>\r\n</td>\r\n<td>\r\n<p><span >Data Repository of Yeast Genetic Interactions (DRYGIN) is a web database system that provides a central platform for yeast genetic network analysis and visualization</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ECMDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >ECMDB is The E coli Metabolome Database</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >EDdb</span></p>\r\n</td>\r\n<td>\r\n<p><span >Eddb is a web resource for eating disorder and its application to identify an extended adipocytokine signaling pathway related to eating disorder</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >EndoNet</span></p>\r\n</td>\r\n<td>\r\n<p><span >EndoNet is an information resource about the intercellular signaling network</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ENFIN</span></p>\r\n</td>\r\n<td>\r\n<p><span >ENFIN is a network to enhance integrative systems biology</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Enrichment map</span></p>\r\n</td>\r\n<td>\r\n<p><span >Enrichment map is a network-based method for gene-set enrichment visualization and interpretation</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Enrichr</span></p>\r\n</td>\r\n<td>\r\n<p><span >Enrichr is an Interactive and collaborative HTML5 gene list enrichment analysis tool</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Enzyme Portal</span></p>\r\n</td>\r\n<td>\r\n<p><span >Mines and displays data about proteins with enzymatic activity from public repositories via a single search, and includes biochemical reactions, biological pathways, small molecule chemistry, disease information, 3D protein structures and relevant scientific literature</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Eu.Gene Analyzer</span></p>\r\n</td>\r\n<td>\r\n<p><span >Eu Gene Analyzer is a tool for integrating gene expression data with pathway databases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Expander</span></p>\r\n</td>\r\n<td>\r\n<p><span >EXPression ANalyzer and DisplayER) is an integrated software platform for the analysis of gene expression data, which is freely available for academic use</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ExPASy Proteomics Server</span></p>\r\n</td>\r\n<td>\r\n<p><span >Expert Protein Analysis System (ExPASy) proteomics server of the Swiss Institute of Bioinformatics (SIB) is dedicated to the analysis of protein sequences and structures as well as 2-D PAGE</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >FACT</span></p>\r\n</td>\r\n<td>\r\n<p><span >FACT is a framework for the functional interpretation of high-throughput experiments</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >FLIGHT</span></p>\r\n</td>\r\n<td>\r\n<p><span >FLIGHT an online resource compiling data from over 100 Drosophila in vivo and in vitro RNAi screens</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >FluMap</span></p>\r\n</td>\r\n<td>\r\n<p><span >Comprehensive pathway map that can serve as a graphically presented knowledgebase and as a platform to analyze functional interactions between IAV and host factors</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >FluxMap</span></p>\r\n</td>\r\n<td>\r\n<p><span >A VANTED add-on for the visual exploration of flux distributions in biological networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >FlyMine</span></p>\r\n</td>\r\n<td>\r\n<p><span >FlyMine is an integrated database for Drosophila and Anopheles genomics</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >FlyReactome</span></p>\r\n</td>\r\n<td>\r\n<p><span >FlyReactome is a curated knowledgebase of Drosophila melanogaster pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >fuga</span></p>\r\n</td>\r\n<td>\r\n<p><span >Functional Genomics Assistant is an extensible and portable MATLAB toolbox for the inference of biological relationships, graph topology analysis, random network simulation, network clustering, and functional enrichment statistics</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >FunGenES</span></p>\r\n</td>\r\n<td>\r\n<p><span >The FunGenES database is a genomics resource for mouse embryonic stem cell differentiation</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >g:Profiler</span></p>\r\n</td>\r\n<td>\r\n<p><span >g:Profiler is a web-based toolset for functional profiling of gene lists from large-scale experiments</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >G2SBC</span></p>\r\n</td>\r\n<td>\r\n<p><span >Genes-to-Systems Breast Cancer (G2SBC) Database is a bioinformatics resource that collects and integrates data about genes, transcripts and proteins which have been reported in literature to be altered in breast cancer cells</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Gaggle</span></p>\r\n</td>\r\n<td>\r\n<p><span >Gaggle is a framework for exchanging data between independently developed software tools and databases to enable interactive exploration of systems biology data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Gallus Genome GBrowse</span></p>\r\n</td>\r\n<td>\r\n<p><span >Gallus Genome GBrowse provides online access to genomic and other information about the chicken</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GallusReactome</span></p>\r\n</td>\r\n<td>\r\n<p><span >GallusReactome is a curated knowledgebase of chicken pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GANPA</span></p>\r\n</td>\r\n<td>\r\n<p><span >A novel gene-weighed pathway analysis approach, as implemented in an R package called Gene Associaqtion Network-based Pathway Analysis\"</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GAP</span></p>\r\n</td>\r\n<td>\r\n<p><span >Gene functional Association Predictor is an integrative method for predicting and characterizing gene functional associations</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GATE</span></p>\r\n</td>\r\n<td>\r\n<p><span >Grid Analysis of Time-series Expression (GATE) is an integrated computational software platform for the analys is and visualization of high-dimensional bio-molecular time-series</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GBMBase</span></p>\r\n</td>\r\n<td>\r\n<p><span >GBMBase focuses on data used in Glioblastoma multiforme (GBM) research</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GCMAP</span></p>\r\n</td>\r\n<td>\r\n<p><span >GCMAP is a user-friendly connectivity mapping with R</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GEDI</span></p>\r\n</td>\r\n<td>\r\n<p><span >Genetic Diseases Gene Discovery (GEDI) provides integrated bioinformatics platforms that will support systems analysis of Human multigene disorders</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Gene Ontology</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Gene Ontology project is a major bioinformatics initiative with the aim of standardizing the representation of gene and gene product attributes across species and databases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Gene3D</span></p>\r\n</td>\r\n<td>\r\n<p><span >Gene3D is a domain-based resource for comparative genomics, functional annotation and protein network analysis</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GeneAnswers</span></p>\r\n</td>\r\n<td>\r\n<p><span >Using the Bioconductor GeneAnswers Package to Interpret Gene Lists</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GeneBrowser 2</span></p>\r\n</td>\r\n<td>\r\n<p><span >GeneBrowser 2 is an application to explore and identify common biological traits in a set of genes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GeneCards</span></p>\r\n</td>\r\n<td>\r\n<p><span >Extracts and integrates gene-related data, including genomic, transcriptomic, proteomic, genetic, clinical, and functional information</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GeneMANIA</span></p>\r\n</td>\r\n<td>\r\n<p><span >Helps you predict the function of your favourite genes and gene sets</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GeneSigDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >GeneSigDB is a manually curated database and resource for analysis of gene expression signatures</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GeneSpring</span></p>\r\n</td>\r\n<td>\r\n<p><span >GeneSpring GX provides powerful, accessible statistical tools for fast visualization and analysis of expression and genomic structural variation data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GeneSrF</span></p>\r\n</td>\r\n<td>\r\n<p><span >GeneSrF is a web-based tool for gene selection and classification using random forest</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GeneTrail 2</span></p>\r\n</td>\r\n<td>\r\n<p><span >GeneTrail 2 is a web-interface providing access to different tools for the statistical analysis of molecular signatures</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GenMAPP</span></p>\r\n</td>\r\n<td>\r\n<p><span >Gene Map Annotator and Pathway Profiler (GenMAPP) is a free, open-source bioinformatics software tool designed to visualize and analyze genomic data in the context of pathways (metabolic, signaling), connecting gene-level datasets to biological processes and disease</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GenomeSpace</span></p>\r\n</td>\r\n<td>\r\n<p><span >A cloud-based interoperability framework to support integrative genomics analysis through an easy-to-use Web interface</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GenWay Biotech, Inc.</span></p>\r\n</td>\r\n<td>\r\n<p><span >GenWay Biotech, Inc is a protein, antibody and ELISA solutions provider</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >gespeR</span></p>\r\n</td>\r\n<td>\r\n<p><span >gespeR is a statistical model for deconvoluting off-target-confounded RNA interference screens</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GEYSIR</span></p>\r\n</td>\r\n<td>\r\n<p><span >Genomic Explorer Y Survey of Immune Response (GEYSIR) is a web application designed to correlate case-control association data with genetic and physical maps of the human genome, including key human genes associated with host immune response</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GingerSNP</span></p>\r\n</td>\r\n<td>\r\n<p><span >GingerSNP tools help you explore standard human SNP arrays by interaction, pathway, and function</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GLAMM</span></p>\r\n</td>\r\n<td>\r\n<p><span >GLAMM is a Genome-Linked Application for Metabolic Maps</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GOAL</span></p>\r\n</td>\r\n<td>\r\n<p><span >GOAL is a software tool for assessing biological significance of genes groups</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GOMiner</span></p>\r\n</td>\r\n<td>\r\n<p><span >GoMiner is a tool for biological interpretation of \'omic\' data _ including data from gene expression microarrays</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GPEC</span></p>\r\n</td>\r\n<td>\r\n<p><span >GPEC is a Cytoscape plug-in for random walk-based gene prioritization and biomedical evidence collection</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >graphite</span></p>\r\n</td>\r\n<td>\r\n<p><span >A Bioconductor package to convert pathway topology to gene network</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GraphWeb</span></p>\r\n</td>\r\n<td>\r\n<p><span >GraphWeb is a public web server for graph-based analysis of biological networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GSEA</span></p>\r\n</td>\r\n<td>\r\n<p><span >Gene Set Enrichment Analysis (GSEA) is a computational method that determines whether an a priori defined set of genes shows statistically significant, concordant differences between two biological states</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >GuavaH</span></p>\r\n</td>\r\n<td>\r\n<p><span >GuavaH is a compendium of host genomic data in HIV biology and disease</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HAPMAP</span></p>\r\n</td>\r\n<td>\r\n<p><span >The International HapMap Project is a partnership of scientists and funding agencies from Canada, China, Japan, Nigeria, the United Kingdom and the United States to develop a public resource that will help researchers find genes associated with human disease and response to pharmaceuticals</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HCVpro</span></p>\r\n</td>\r\n<td>\r\n<p><span >HCVpro is a Hepatitis C virus protein interaction database</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HDAPD</span></p>\r\n</td>\r\n<td>\r\n<p><span >HDAPD is a web tool for searching the disease-associated protein structures</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HEFalMp</span></p>\r\n</td>\r\n<td>\r\n<p><span >HEFalMp (Human Experimental/FunctionAL MaPper) is a tool that allows interactive exploration of functional maps (or functional relationship networks) predicted from integration of many genome-scale experimental results</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HepatoNet1</span></p>\r\n</td>\r\n<td>\r\n<p><span >HepatoNet1 is a comprehensive metabolic reconstruction of the human hepatocyte for the analysis of liver physiology</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HGNC</span></p>\r\n</td>\r\n<td>\r\n<p><span >The HGNC database in 2008 is a resource for the human genome</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >hiPathDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >hiPathDB is a human-integrated pathway database with facile visualization</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >hLGDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >hLGDB is a database of human lysosomal genes and their regulation</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HMDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Human Metabolome Database (HMDB) is a freely available electronic database containing detailed information about small molecule metabolites found in the human body</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HMPAS</span></p>\r\n</td>\r\n<td>\r\n<p><span >Human Membrane Protein Analysis System provides comprehensive information about human membrane proteins including specific features of certain membrane protein groups</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HMPAS</span></p>\r\n</td>\r\n<td>\r\n<p><span >HMPAS is a Human Membrane Protein Analysis System</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HMTD</span></p>\r\n</td>\r\n<td>\r\n<p><span >Human Membrane Transporter Database</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HOCCLUS2</span></p>\r\n</td>\r\n<td>\r\n<p><span >bicluster miRNAs and target messenger RNAs (mRNAs) on the basis of their experimentally-verified and/or predicted interactions</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HomoMINT</span></p>\r\n</td>\r\n<td>\r\n<p><span >HomoMINT is an inferred human network based on orthology mapping of protein interactions discovered in model organisms</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HPD</span></p>\r\n</td>\r\n<td>\r\n<p><span >Human Pathway Database (HPD) is an integrated human pathway database providing comprehensive and combined view connecting human proteins, genes, RNAs, enzymes, signaling, metabolic reactions and gene regulatory events</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HPIDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >HPIDB is a unified resource for host-pathogen interactions</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Hubba</span></p>\r\n</td>\r\n<td>\r\n<p><span >Hubba is ahub objects analyzer--a framework of interactome hubs identification for network biology</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >HuPho</span></p>\r\n</td>\r\n<td>\r\n<p><span >HuPho is a human phosphatase portal</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >IBDsite</span></p>\r\n</td>\r\n<td>\r\n<p><span >A disease-oriented platform, which collects data related to biomolecular mechanisms involved in the Inflammatory Bowel Disease onset</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >IDClight</span></p>\r\n</td>\r\n<td>\r\n<p><span >IDClight converts single IDs through an URL</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Idconverter</span></p>\r\n</td>\r\n<td>\r\n<p><span >Idconvertor map IDs to other known public DBs</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ImmPort</span></p>\r\n</td>\r\n<td>\r\n<p><span >Immunology Database and Analysis Portal (ImmPort), is a one stop shop to access reference and experiment data for immunologists</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Influenza Research Database</span></p>\r\n</td>\r\n<td>\r\n<p><span >Influenza Research Database is a resource for the influenza virus research community that will facilitate an understanding of the influenza virus and how it interacts with the host organism</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >InnateDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >InnateDB is a publicly available database of the genes, proteins, experimentally-verified interactions and signaling pathways involved in the innate immune response of humans and mice to microbial infection</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >INOH</span></p>\r\n</td>\r\n<td>\r\n<p><span >Integrating Network Objects with Hierarchies (INOH) is a pathway database of model organisms including human, mouse, rat and others</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >INRICH</span></p>\r\n</td>\r\n<td>\r\n<p><span >NRICH is an Interval-based Enrichment Analysis Tool for Genome Wide Association Studies</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >IntAct</span></p>\r\n</td>\r\n<td>\r\n<p><span >IntAct is an open source resource for molecular interaction data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >IntegromeDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >IntegromeDB is an integrated system and biological search engine</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Interactome3D</span></p>\r\n</td>\r\n<td>\r\n<p><span >A resource for the structural annotation and modeling of protein-protein interactions</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Intermine</span></p>\r\n</td>\r\n<td>\r\n<p><span >An open source data warehouse built specifically for the integration and analysis of complex biological data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >IPAD</span></p>\r\n</td>\r\n<td>\r\n<p><span >Integrated Pathway Analysis Database for Systematic Enrichment Analysis for defining inter-association between pathway, disease, drug and organ specificity</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >iPAS</span></p>\r\n</td>\r\n<td>\r\n<p><span >iPAS is series of integrative steps, which extend existing pathway analysis techniques It consists of four parts is a data processing, gene-level statistics, individualized pathway aberrance score (iPAS) and a significance test</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >IPAVS</span></p>\r\n</td>\r\n<td>\r\n<p><span >IPAVS is an Integrated Pathway Resources, Analysis and Visualization System</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ISAAC</span></p>\r\n</td>\r\n<td>\r\n<p><span >InterSpecies Analysing Application using Containers is a web based tool, which analyses sets of genes, transcripts and proteins under different biological viewpoints and to interactively modify these sets at any point of the analysis</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >KaBOB</span></p>\r\n</td>\r\n<td>\r\n<p><span >KaBOB is an ontology-based semantic integration of biomedical databases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >KDBI</span></p>\r\n</td>\r\n<td>\r\n<p><span >Update of KDBI is a Kinetic data of bio-molecular interaction database</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >KeyPathwayMiner</span></p>\r\n</td>\r\n<td>\r\n<p><span >KeyPathwayMiner is a condition-specific pathway analysis by combining multiple omics studies and networks with Cytoscape</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >KOBAS 2.0</span></p>\r\n</td>\r\n<td>\r\n<p><span >KOBAS is an update of KOBAS (KEGG Orthology Based Annotation System) Its purpose is to identify statistically enriched related pathways and diseases for a set of genes or proteins, using pathway and disease knowledge from multiple commonly used databases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >LDGIdb</span></p>\r\n</td>\r\n<td>\r\n<p><span >LDGIdb is a database of gene interactions inferred from long-range strong linkage disequilibrium between pairs of SNPs</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Linked Life Data</span></p>\r\n</td>\r\n<td>\r\n<p><span >Linked Life Data is a semantic integration platform for the biomedical domain</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >LipidHome</span></p>\r\n</td>\r\n<td>\r\n<p><span >A Database of Theoretical Lipids Optimized for High Throughput Mass Spectrometry Lipidomics</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Lists2Networks</span></p>\r\n</td>\r\n<td>\r\n<p><span >Lists2Networks provides integrated analysis of gene/protein lists</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >LiverAtlas</span></p>\r\n</td>\r\n<td>\r\n<p><span >LiverAtlas is a unique integrated knowledge database for systems-level research of liver and hepatic disease</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >LOKI</span></p>\r\n</td>\r\n<td>\r\n<p><span >Library of Knowledge Integration (LOKI), which contains diverse prior knowledge from multiple collections of biological data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MAGeCK</span></p>\r\n</td>\r\n<td>\r\n<p><span >MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Manteia</span></p>\r\n</td>\r\n<td>\r\n<p><span >Manteia is a predictive data mining system for vertebrate genes and its applications to human genetic diseases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >mAPKL</span></p>\r\n</td>\r\n<td>\r\n<p><span >mAPKL is aR/Bioconductor package for detecting gene exemplars and revealing their characteristics</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MassVis</span></p>\r\n</td>\r\n<td>\r\n<p><span >MassVis is a Visual analysis of protein complexes using mass spectrometry</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MeSH</span></p>\r\n</td>\r\n<td>\r\n<p><span >MeSH ORA framework is aR/Bioconductor packages to support MeSH over-representation analysis</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Meta-All</span></p>\r\n</td>\r\n<td>\r\n<p><span >Meta-All is a system for managing metabolic pathway information</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MetaboLights</span></p>\r\n</td>\r\n<td>\r\n<p><span >The MetaboLights repository is a Curation challenges in metabolomics</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MetaCrop</span></p>\r\n</td>\r\n<td>\r\n<p><span >MetaCrop 20 is a managing and exploring information about crop plant metabolism</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >METACYC</span></p>\r\n</td>\r\n<td>\r\n<p><span >MetaCyc is a database of nonredundant, experimentally elucidated metabolic pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MetaMapp</span></p>\r\n</td>\r\n<td>\r\n<p><span >Mapping and visualizing metabolomic data by integrating information from biochemical pathways and chemical and mass spectral similarity</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >METANNOGEN</span></p>\r\n</td>\r\n<td>\r\n<p><span >METANNOGEN is a flexible tool to manage information for the design of metabolic networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MetaReg</span></p>\r\n</td>\r\n<td>\r\n<p><span >MetaReg is a platform for modeling, analysis and visualization of biological systems using large-scale experimental data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >metaTIGER</span></p>\r\n</td>\r\n<td>\r\n<p><span >metaTIGER is a collection of metabolic profiles and phylogenomic information on a taxonomically diverse range of eukaryotes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MetNet Online</span></p>\r\n</td>\r\n<td>\r\n<p><span >MetNet Online is a novel integrated resource for plant systems biology</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MetNetAPI</span></p>\r\n</td>\r\n<td>\r\n<p><span >MetNetAPI is a flexible method to access and manipulate biological network data from MetNet</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MetRxn</span></p>\r\n</td>\r\n<td>\r\n<p><span >MetRxn is a knowledgebase of metabolites and reactions spanning metabolic models and databases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MGV</span></p>\r\n</td>\r\n<td>\r\n<p><span >MGV is a generic graph viewer for comparative omics data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MiMI</span></p>\r\n</td>\r\n<td>\r\n<p><span >Michigan Molecular Interactions (MiMI) provides access to the knowledge and data merged and integrated from numerous protein interactions databases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MIMO</span></p>\r\n</td>\r\n<td>\r\n<p><span >MIMO is an efficient tool for molecular interaction maps overlap</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Mimoza</span></p>\r\n</td>\r\n<td>\r\n<p><span >Mimoza is a web-based semantic zooming and navigation in metabolic networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MINT</span></p>\r\n</td>\r\n<td>\r\n<p><span >MINT is a public repository for molecular interaction data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >mirDIP</span></p>\r\n</td>\r\n<td>\r\n<p><span >microRNA Data Integration Portal (mirDIP) integrates twelve microRNA prediction datasets from six microRNA prediction databases, allowing users to customize their microRNA target searches</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MIRIAM</span></p>\r\n</td>\r\n<td>\r\n<p><span >MIRIAM is an effort to standardise the Minimal Information Required In the Annotation of Models, so that different groups can collaborate on annotating and curating computational models in biology</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >miRror-Suite</span></p>\r\n</td>\r\n<td>\r\n<p><span >miRror-Suite platform provides a concise, plausible explanation for the regulation of miRNAs in such complex settings</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >miRSystem</span></p>\r\n</td>\r\n<td>\r\n<p><span >miRSystem is an Integrated System for Characterizing Enriched Functions and Pathways of MicroRNA Targets</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MK4MDD</span></p>\r\n</td>\r\n<td>\r\n<p><span >MK4MDD is a Multi-Level Knowledge Base and Analysis Platform for Major Depressive Disorder</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MMMDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >MMMDB is a Mouse Multiple Tissue Metabolome Database</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >modMine</span></p>\r\n</td>\r\n<td>\r\n<p><span >modMine is a flexible access to modENCODE data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MODOMICS</span></p>\r\n</td>\r\n<td>\r\n<p><span >MODOMICS is a database of RNA modification pathways - 2013 update</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MoKCa</span></p>\r\n</td>\r\n<td>\r\n<p><span >The MoKCa (Mutations of Kinases in Cancer) database has been developed to structurally and functionally annotate, and where possible predict, the phenotypic consequences of mutations in protein kinases implicated in cancer</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MoonDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >MoonDB is a database containing information on all the candidates identified in the analys is and a set of manually curated human moonlighting proteins</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MOPED</span></p>\r\n</td>\r\n<td>\r\n<p><span >Model Organism Protein Expression Database enables discoveries through consistently processed proteomics data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MSigDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Molecular Signatures Database (MSigDB) is a collection of annotated gene sets for use with GSEA software</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MuSiC</span></p>\r\n</td>\r\n<td>\r\n<p><span >MuSiC is an Identifying mutational significance in cancer genomes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >MyBiosource</span></p>\r\n</td>\r\n<td>\r\n<p><span >MyBiosource is a provider of antibody, protein, peptide and ELISA reagents</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NaviCell</span></p>\r\n</td>\r\n<td>\r\n<p><span >NaviCell is a web-based environment for navigation, curation and maintenance of large molecular interaction maps</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NAViGaTOR</span></p>\r\n</td>\r\n<td>\r\n<p><span >A powerful graphing application for the 2D and 3D visualization of biological networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NCBI BioSystems</span></p>\r\n</td>\r\n<td>\r\n<p><span >The NCBI Biosystems database centralizes and cross-links existing biological systems databases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NCBI Gene</span></p>\r\n</td>\r\n<td>\r\n<p><span >Entrez Gene is a searchable database of genes, from RefSeq genomes, and defined by sequence and/or located in the NCBI Map Viewer</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NCBO Bioportal</span></p>\r\n</td>\r\n<td>\r\n<p><span >BioPortal is a Web-based application for accessing and sharing biomedical ontologies</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NCIS</span></p>\r\n</td>\r\n<td>\r\n<p><span >NCIS is a Network-assisted Co-clustering for the Identification of cancer Subtypes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ncRNA</span></p>\r\n</td>\r\n<td>\r\n<p><span >ncRNA is a portal site for bioinformatics tools and databases specialized for functional RNAs</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NeMo</span></p>\r\n</td>\r\n<td>\r\n<p><span >NeMo is a Network Module identification in Cytoscape</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NetBox</span></p>\r\n</td>\r\n<td>\r\n<p><span >NetBox is a Java-based software tool for performing network analysis on human interaction networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NetHiKe</span></p>\r\n</td>\r\n<td>\r\n<p><span >A simple knowledge-based mining method for exploring hidden key molecules in a human biomolecular network</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NetPath</span></p>\r\n</td>\r\n<td>\r\n<p><span >NetPath is a public resource of curated signal transduction pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NetSlim</span></p>\r\n</td>\r\n<td>\r\n<p><span >NetSlim is a High-confidence curated signaling maps</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NetWalker</span></p>\r\n</td>\r\n<td>\r\n<p><span >NetWalker is a contextual network analysis tool for functional genomics</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Network</span></p>\r\n</td>\r\n<td>\r\n<p><span >Network Screening is a New Method to Identify Active Networks from an Ensemble of Known Networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Network2Canvas</span></p>\r\n</td>\r\n<td>\r\n<p><span >Network2Canvas is a Network Visualization on a Canvas with Enrichment Analysis</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NetWORKIN</span></p>\r\n</td>\r\n<td>\r\n<p><span >NetWORKIN is a method for predicting in vivo kinase-substrate relationships, that augments consensus motifs with context for kinases and phosphoproteins</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NetworkLego</span></p>\r\n</td>\r\n<td>\r\n<p><span >NetworkLego is a general computational framework for detecting cellular networks that are activated in a particular cell state or in response to a perturbation and for determining core cellular networks that are activated in response to multiple perturbations</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NetworkPrioritizer</span></p>\r\n</td>\r\n<td>\r\n<p><span >A versatile tool for network-based prioritization of candidate disease genes or other molecules</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NeuroDNet</span></p>\r\n</td>\r\n<td>\r\n<p><span >An open source platform for constructing and analyzing neurodegenerative disease networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >neXtProt</span></p>\r\n</td>\r\n<td>\r\n<p><span >neXtProt is a human-centric discovery platform offering its users a seamless integration of and navigation through proteins-related data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >NGP</span></p>\r\n</td>\r\n<td>\r\n<p><span >Networked Gene Prioritizer integrates gene expression and protein-protein interaction network to prioritize cancer-associated genes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >OncoRep</span></p>\r\n</td>\r\n<td>\r\n<p><span >OncoRep is an n-of-1 reporting tool to support genome-guided treatment for breast cancer patients using RNA-sequencing</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Onto-Tools</span></p>\r\n</td>\r\n<td>\r\n<p><span >Onto-Tools is a New additions and improvements in 2006</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >OntoSlug</span></p>\r\n</td>\r\n<td>\r\n<p><span >OntoSlug is a dynamic visual front-end program for ontologies</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >OPHID</span></p>\r\n</td>\r\n<td>\r\n<p><span >Integrated known, experimental and predicted PPIs for five model organisms and human in the I2D database</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ORCID</span></p>\r\n</td>\r\n<td>\r\n<p><span >ORCID provides a persistent digital identifier that distinguishes you from every other researcher</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >OREMPdb</span></p>\r\n</td>\r\n<td>\r\n<p><span >Ontology Reasoning Engine for Molecular Pathways is a semantic dictionary of computational pathway models</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >p53 Knowledgebase</span></p>\r\n</td>\r\n<td>\r\n<p><span >The p53 Knowledgebase is an integrated information resource for p53 research</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PAGED</span></p>\r\n</td>\r\n<td>\r\n<p><span >PAGED is a pathway and gene-set enrichment database to enable molecular phenotype discoveries</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PAGER</span></p>\r\n</td>\r\n<td>\r\n<p><span >PAGER is a new database framework to perform integrative gene-set, network, and pathway analysis (GNPA)</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PaLS</span></p>\r\n</td>\r\n<td>\r\n<p><span >PaLS is a tool that filters, from a list or a set of lists of gene/clone/protein identifiers, those that meet certain criteria related to their references to articles in PubMed, Gene Ontology terms and KEGG and Reactome pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Pancreatic Expression database</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Pancreatic Expression database is the main repository for pancreatic-derived -omics data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Pandora</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pandora is a novel integrated approach that uses network topology to predict biological pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PARADIGM-SHIFT</span></p>\r\n</td>\r\n<td>\r\n<p><span >PARADIGM-SHIFT predicts the function of mutations in multiple cancers using pathway impact analysis</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Path2Models</span></p>\r\n</td>\r\n<td>\r\n<p><span >Path2Models is a Large-scale generation of computational models from biochemical pathway maps</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PathAgent</span></p>\r\n</td>\r\n<td>\r\n<p><span >PathAgent is a Multi-agent System for Updated Pathway Information Integration</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PathCase-SB</span></p>\r\n</td>\r\n<td>\r\n<p><span >PathCase-SB is integrating data sources and providing tools for systems biology research</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PathJam</span></p>\r\n</td>\r\n<td>\r\n<p><span >PathJam is a public tool which provides an intuitive and user-friendly framework for biological pathway analysis of human gene lists</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PATHLOGIC-S</span></p>\r\n</td>\r\n<td>\r\n<p><span >A Scalable Boolean Framework for Modelling Cellular Signalling</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PathNER</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pathway Named Entity Recognition is a tool for systematic identification of biological pathway mentions in the literature</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Pathprinting</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pathprinting is an integrative approach to understand the functional basis of disease</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PathRings</span></p>\r\n</td>\r\n<td>\r\n<p><span >PathRings is a web-based tool for exploration of ortholog and expression data in biological pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PathScan</span></p>\r\n</td>\r\n<td>\r\n<p><span >PathScan is Tool for Discerning Mutational Significance in Groups of Putative Cancer Genes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PathVar</span></p>\r\n</td>\r\n<td>\r\n<p><span >PathVar is analysis of gene and protein expression variance in cellular pathways using microarray data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Pathview</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pathview is an R/Bioconductor package for pathway-based data integration and visualization</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PathVisioRPC</span></p>\r\n</td>\r\n<td>\r\n<p><span >Automatically visualise and analyse data on pathways using PathVisioRPC from any programming environment</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Pathway Projector</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pathway projector is a Web-based zoomable pathway browser using KEGG Atlas and Google Maps API</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Pathway Commons</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pathway Commons provides researchers with convenient access to a comprehensive collection of pathways from multiple organisms</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Pathway Interaction Database</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pathway Interaction Database (PID) is a freely available collection of curated and peer-reviewed pathways composed of human molecular signaling and regulatory events and key cellular processes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Pathway Knowledge Base</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pathway Knowledge Base is a public repository for searching biological pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Pathway-GPS</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pathway-GPS and SIGORA identifies relevant pathways based on the over-representation of their gene-pair signatures</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PathwayCaseMAW</span></p>\r\n</td>\r\n<td>\r\n<p><span >PathwayCaseMAW is an online system for metabolic network analysis</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PathwayLinker</span></p>\r\n</td>\r\n<td>\r\n<p><span >A system of integrating protein-protein interaction and signaling pathway data from several sources with statistical significance tests and clear visualization</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PATIKAmad</span></p>\r\n</td>\r\n<td>\r\n<p><span >PATIKAmad puts microarray data into pathway context</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PATIKAweb</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pathway Analysis Tool for Integration and Knowledge Acquisition</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PaxTools</span></p>\r\n</td>\r\n<td>\r\n<p><span >An open-source, Java library that contains facilities and algorithms for common but difficult to implement tasks such as reading, writing, searching, merging, comparing and transforming pathway information</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Payao</span></p>\r\n</td>\r\n<td>\r\n<p><span >Payao is a community platform for SBML pathway model curation</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PBSK browser</span></p>\r\n</td>\r\n<td>\r\n<p><span >PBSK browser allows users to navigate biological pathways of PSI-MI, BioPAX, SBML and KGML formats</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >An Information Portal to Biological Macromolecular Structures</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PDBe</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Protein Data Bank in Europe (PDBe) is bringing structure to biology</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PDDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >Prion Disease Database (PDDB) focuses on data used in Prion Disease (PD) research This site provides tools which allow scientists to analyze, store, and share this PD data with other scientists</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PEANuT</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pathway enrichment analysis tool provides an up-front method to identify underlying biological principles by calculating enriched pathways in focus networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PENTACON</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Personalized NSAID Therapeutics Consortium (PENTACON) is a group of scientists with a wide range of expertise from multiple institutions whose collective aim is to develop a paradigm for the personalization of drug treatment</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PharmGKB</span></p>\r\n</td>\r\n<td>\r\n<p><span >Integrated data resource about how variation in human genetics leads to variation in response to drugs</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PhosphoELM</span></p>\r\n</td>\r\n<td>\r\n<p><span >PhosphoELM is a database of phosphorylation sites - Update 2008</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PIG</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pathogen Interaction Gateway is a database of host-pathogen PPI (HP-PPI) data from a number of public resources</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PIR</span></p>\r\n</td>\r\n<td>\r\n<p><span >Protein Information Resource (PIR) is an integrated public bioinformatics resource to support genomic, proteomic and systems biology research and scientific studies</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PlantReactome</span></p>\r\n</td>\r\n<td>\r\n<p><span >A freely accessible plant pathway database which hosts plant metabolic and regulatory pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PLAZA</span></p>\r\n</td>\r\n<td>\r\n<p><span >A Comparative Genomics Resource to Study Gene and Genome Evolution in Plants</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PolyDoms</span></p>\r\n</td>\r\n<td>\r\n<p><span >PolyDoms is a whole genome database for the identification of non-synonymous coding SNPs with the potential to impact disease</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Pomelo II</span></p>\r\n</td>\r\n<td>\r\n<p><span >Pomelo II is an open-source, web-based, freely available tool for the analysis of gene (and protein) expression and tissue array data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PosMed</span></p>\r\n</td>\r\n<td>\r\n<p><span >Positional MEDLINE (PosMed) prioritizes candidate genes for positional cloning</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PpiTrim</span></p>\r\n</td>\r\n<td>\r\n<p><span >PpiTrim is for constructing non-redundant and up-to-date interactomes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Predicted Arabidopsis Interactome Resource</span></p>\r\n</td>\r\n<td>\r\n<p><span >A database of potential plant interactions, detailed annotations and more specific per interaction confidence measurement</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PRGdb</span></p>\r\n</td>\r\n<td>\r\n<p><span >PRGdb is a community-based database model for the analysis of R-genes in plants</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PRIDE</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Proteomics Identifications database at EBI is a mass spectrometry-derived proteomics data repository</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Prioritizer</span></p>\r\n</td>\r\n<td>\r\n<p><span >A prioritization method for positional candidate genes, by assuming that the majority of causative genes are functionally closely related</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PRO</span></p>\r\n</td>\r\n<td>\r\n<p><span >Protein Ontology (PRO) is a formal representation of protein objects, providing both descriptions of these objects and the relationships between them</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PROGgeneV2</span></p>\r\n</td>\r\n<td>\r\n<p><span >PROGgeneV2 is a Pan Cancer Prognostics Database</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ProKinO</span></p>\r\n</td>\r\n<td>\r\n<p><span >ProKinO is a framework for protein kinase ontology</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Proteomic</span></p>\r\n</td>\r\n<td>\r\n<p><span >Proteomic atlas of the human olfactory bulb</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PSI-MI</span></p>\r\n</td>\r\n<td>\r\n<p><span >The HUPO Proteomics Standards Initiative Molecular Interaction (PSI-MI) workgroup defines community standards for data representation in proteomics to facilitate data comparison, exchange and verification</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PSIQUIC</span></p>\r\n</td>\r\n<td>\r\n<p><span >PSIQUIC a web service to access the PSI-MI interaction data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >pSTIING</span></p>\r\n</td>\r\n<td>\r\n<p><span >pSTIING is a \'systems\' approach towards integrating signalling pathways, interaction and transcriptional regulatory networks in inflammation and cancer</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PUPPI</span></p>\r\n</td>\r\n<td>\r\n<p><span >PUPPI is a pathway analysis method using protein-protein interaction network for case-control data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >PyBioS</span></p>\r\n</td>\r\n<td>\r\n<p><span >PyBioS is a system for the modeling and simulation of cellular processes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Quartz-Seq</span></p>\r\n</td>\r\n<td>\r\n<p><span >Quartz-Seq is a highly reproducible and sensitive single-cell RNA sequencing method, reveals non-genetic gene-expression heterogeneity</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >RDF2Graph</span></p>\r\n</td>\r\n<td>\r\n<p><span >RDF2Graph a tool to recover, understand and validate the ontology of an RDF resource</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >RDFScape</span></p>\r\n</td>\r\n<td>\r\n<p><span >RDFScape is a plugin that has been developed to to query, visualize and reason on ontologies represented in OWL or RDF within Cytoscape</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Reactome.db</span></p>\r\n</td>\r\n<td>\r\n<p><span >A set of annotation maps for reactome assembled using data from Reactome</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ReactomePA</span></p>\r\n</td>\r\n<td>\r\n<p><span >ReactomePA is a R package for Reactome Pathway Analysis</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ReConn</span></p>\r\n</td>\r\n<td>\r\n<p><span >ReConn is a Java plugin that connects Cytoscape to the Reactome database</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >RefNetBuilder</span></p>\r\n</td>\r\n<td>\r\n<p><span >RefNetBuilder is a platform for construction of integrated reference gene regulatory networks from expressed sequence tags</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Regulatory Genomics</span></p>\r\n</td>\r\n<td>\r\n<p><span >A Database of Annotated Promoters of Genes Associated with Common Respiratory and Related Diseases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ReMatch</span></p>\r\n</td>\r\n<td>\r\n<p><span >ReMatch is a web-based tool to construct, store and share stoichiometric metabolic models with carbon maps for metabolic flux analysis</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >REPAIRtoire</span></p>\r\n</td>\r\n<td>\r\n<p><span >REPAIRtoire is a database of DNA repair pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >RGD</span></p>\r\n</td>\r\n<td>\r\n<p><span >Rat Genome Database (RGD) is a collection of genetic and genomic information on the rat</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Rhea</span></p>\r\n</td>\r\n<td>\r\n<p><span >Rhea is a manually curated resource of biochemical reactions</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >RhesusBase</span></p>\r\n</td>\r\n<td>\r\n<p><span >RhesusBase is a knowledgebase for the monkey research community</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >RMOD</span></p>\r\n</td>\r\n<td>\r\n<p><span >A Tool for Regulatory Motif Detection in Signaling Network</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >RNApathwaysDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >RNApathwaysDB is a database of RNA maturation and decay pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >ROCK</span></p>\r\n</td>\r\n<td>\r\n<p><span >ROCK is a breast cancer functional genomics resource</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Rspider</span></p>\r\n</td>\r\n<td>\r\n<p><span >R spider is a web-based tool for the analysis of a gene list using the systematic knowledge of core pathways and reactions in human biology accumulated in the REACTOME and KEGG databases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >RxnFinder</span></p>\r\n</td>\r\n<td>\r\n<p><span >RxnFinder is a Biochemical reaction search engines using molecular structures, molecular fragments and reaction similarity</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SABIO-RK</span></p>\r\n</td>\r\n<td>\r\n<p><span >System for the Analysis of Biochemical Pathways - Reaction Kinetics is a web-based application based on the SABIO relational database that contains information about biochemical reactions, their kinetic equations with their parameters, and the experimental conditions under which these parameters were measured</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SAGA</span></p>\r\n</td>\r\n<td>\r\n<p><span >SAGA is a subgraph matching tool for biological graphs</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SASD</span></p>\r\n</td>\r\n<td>\r\n<p><span >Synthetic Alternative Splicing Database for identifying novel isoform from proteomics</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SASD</span></p>\r\n</td>\r\n<td>\r\n<p><span >SASD is the Synthetic Alternative Splicing Database for identifying novel isoform from proteomics</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SB-KOM</span></p>\r\n</td>\r\n<td>\r\n<p><span >SB-KOM provides integration of pathway information using BioPax</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SBGN</span></p>\r\n</td>\r\n<td>\r\n<p><span >Systems Biology Graphical Notation (SBGN) project is an effort to standardize the graphical notation used in maps of biological processes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SBKD</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Structural Biology Knowledgebase is a portal to protein structures, sequences, functions, and methods</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SBML</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Systems Biology Markup Language (SBML) is a computer-readable format for representing models of biological processes</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SBML-Bridge</span></p>\r\n</td>\r\n<td>\r\n<p><span >SBML-Bridge is an XML Pathway Converting Tool</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SciMiner</span></p>\r\n</td>\r\n<td>\r\n<p><span >SciMiner is a dictionary- and rule-based biomedical literature mining and functional enrichment analysis tool</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SIDEKICK</span></p>\r\n</td>\r\n<td>\r\n<p><span >SIDEKICK is a genomic data driven analys is and decision-making framework</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SigCS</span></p>\r\n</td>\r\n<td>\r\n<p><span >An integrated genetic information resource for human cerebral stroke</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SignaLink</span></p>\r\n</td>\r\n<td>\r\n<p><span >SignaLink allows system-level analyses using the network of the major intracellular signaling pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Signalogs</span></p>\r\n</td>\r\n<td>\r\n<p><span >Signalogs is an Orthology-based identification of novel signaling pathway components in three metazoans</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SignS</span></p>\r\n</td>\r\n<td>\r\n<p><span >SignS is a web tool for gene selection and signature finding in problems where the dependent variable is patient survival or, more generally, a right-censored variable</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SimBiology 3.1</span></p>\r\n</td>\r\n<td>\r\n<p><span >SimBiology provides graphical and programmatic tools for computational systems biology and pharmacokinetics</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SIPPER</span></p>\r\n</td>\r\n<td>\r\n<p><span >SIPPER is a flexible method to integrate heterogeneous data into a metabolic network</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SMART</span></p>\r\n</td>\r\n<td>\r\n<p><span >SMART displays domains in the context of genomes and networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SMPDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Small Molecule Pathway Database (SMPDB) is an interactive, visual database containing more than 350 small-molecule pathways found in humans</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SNPranker</span></p>\r\n</td>\r\n<td>\r\n<p><span >SNPranker is a gene-centric data mining tool for diseases associated SNP prioritization in GWAS</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SOURCE</span></p>\r\n</td>\r\n<td>\r\n<p><span >SOURCE is a unification tool which dynamically collects and compiles data from many scientific databases, and thereby attempts to encapsulate the genetics and molecular biology of genes from the genomes of Homo sapiens, Mus musculus, Rattus norvegicus into easy to navigate GeneReports</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SPARQLGraph</span></p>\r\n</td>\r\n<td>\r\n<p><span >SPARQLGraph is a web-based platform for graphically querying biological Semantic Web databases</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SPIKE</span></p>\r\n</td>\r\n<td>\r\n<p><span >Signaling Pathway Integrated Knowledge Engine is an interactive software environment that graphically displays biological signaling networks, allows dynamic layout and navigation through these networks, and enables the superposition of DNA microarray and other functional genomics data on interaction maps</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SPIRE</span></p>\r\n</td>\r\n<td>\r\n<p><span >SPIRE is a Systematic protein investigative research environment</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >StarBase</span></p>\r\n</td>\r\n<td>\r\n<p><span >StarBase will decode miRNA-ceRNA, miRNA-ncRNA and protein-RNA interaction networks from large-scale CLIP-Seq data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >STITCH</span></p>\r\n</td>\r\n<td>\r\n<p><span >STITCH is an interaction network database for small molecules and proteins</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >STRING</span></p>\r\n</td>\r\n<td>\r\n<p><span >STRING is global view on proteins and their functional interactions in 630 organisms</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SuperTarget</span></p>\r\n</td>\r\n<td>\r\n<p><span >SuperTarget is an on drug-target interactions resource</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SyBiL</span></p>\r\n</td>\r\n<td>\r\n<p><span >Systems Biology Linker is designed for visualizing pathway data files in BioPAX format, such as imported from Reactome and converting BioPAX file into SBML file</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >SZGR</span></p>\r\n</td>\r\n<td>\r\n<p><span >SZGR is a comprehensive schizophrenia gene resource</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >T1DBase</span></p>\r\n</td>\r\n<td>\r\n<p><span >T1DBase focuses on two research areas in type 1 diabetes (T1D) and the genetics of T1D susceptibility and beta cell biology</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >T2D-Db</span></p>\r\n</td>\r\n<td>\r\n<p><span >T2D-Db is a database of all molecular factors reported to be involved in the pathogenesis of Type 2 diabetes in human, mouse and rat</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >T2D@ZJU</span></p>\r\n</td>\r\n<td>\r\n<p><span >T2D@ZJU is a knowledgebase integrating heterogeneous connections associated with type 2 diabetes mellitus</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >T2DM-GeneMiner</span></p>\r\n</td>\r\n<td>\r\n<p><span >T2DM-GeneMiner is a web resource for analysing genes in relation to type 2 diabetes mellitus</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Taverna</span></p>\r\n</td>\r\n<td>\r\n<p><span >Taverna Workbench is an open source tool for designing and executing workflows created by the myGrid project</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >TcoF-DB</span></p>\r\n</td>\r\n<td>\r\n<p><span >TcoF-DB is provide a comprehensive starting point for investigations into transcriptional regulation that include PPIs between Transcription Factors, Transcription Co-factors and other proteins</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >TEAK</span></p>\r\n</td>\r\n<td>\r\n<p><span >TEAK is a Topology Enrichment Analysis frameworK for detecting activated biological subpathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >The Cell Collective</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Cell Collective is an open and collaborative approach to systems biology</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >The Cell Cycle Ontology</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Cell Cycle Ontology is an application ontology that automatically captures and integrates detailed knowledge on the cell cycle process</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >TIM</span></p>\r\n</td>\r\n<td>\r\n<p><span >TIM (Tools to Input Models) is a tool which integrates biological objects and information about biological process modelling</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Toppgene</span></p>\r\n</td>\r\n<td>\r\n<p><span >Toppgene Suite is an one-stop portal for gene list enrichment analysis and candidate gene prioritization based on functional annotations and protein interactions network</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >TranscriptomeBrowser</span></p>\r\n</td>\r\n<td>\r\n<p><span >TranscriptomeBrowser is introducing a new compendium of molecular interactions and a new visualization tool for the study of gene regulatory networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >TransDomain</span></p>\r\n</td>\r\n<td>\r\n<p><span >TransDomain is a transitive domain-based method in protein-protein interaction prediction</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >TRANSPATH</span></p>\r\n</td>\r\n<td>\r\n<p><span >TRANSPATH is an information resource for storing and visualizing signaling pathways and their pathological aberrations</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >tYNA platform</span></p>\r\n</td>\r\n<td>\r\n<p><span >The tYNA platform for comparative interactomics is a web tool for managing, comparing and mining multiple networks</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >UCD 2D-PAGE Database</span></p>\r\n</td>\r\n<td>\r\n<p><span >UCD 2D-PAGE Datavase is a Two-dimensional polyacrylamide gel electrophoresis database</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >UCSC Genome Browser</span></p>\r\n</td>\r\n<td>\r\n<p><span >UCSC Genome Browser is an up-to-date source for genome sequence data from a variety of vertebrate and invertebrate species and major model organisms, integrated with a large collection of aligned annotations</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >UCSD-Nature Signaling Gateway</span></p>\r\n</td>\r\n<td>\r\n<p><span >The UCSD-Nature Signaling Gateway is a comprehensive and up-to-the-minute resource for anyone interested in signal transduction</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >UniHI</span></p>\r\n</td>\r\n<td>\r\n<p><span >The Unified Human Interactome database provides researchers with a comprehensive platform to query and access human protein-protein interaction (PPI) data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >UniProtKB</span></p>\r\n</td>\r\n<td>\r\n<p><span >The UniProt Knowledgebase is the central hub for the collection of functional information on proteins, with accurate, consistent and rich annotation</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >VANTED</span></p>\r\n</td>\r\n<td>\r\n<p><span >Visualization and Analysis of Networks containing Experimental Data makes it possible to load and edit graphs, which may represent biological pathways or functional hierarchies</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >varSelRF</span></p>\r\n</td>\r\n<td>\r\n<p><span >varSelRF is a R package for gene selection and classification using random forest</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >VeryGene</span></p>\r\n</td>\r\n<td>\r\n<p><span >VeryGene serves as a tissue-specific genes knowledgebase and a discovery tool to generate testable hypotheses for basic and clinical research</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >viPEr</span></p>\r\n</td>\r\n<td>\r\n<p><span >Virtual pathway explorer is a cytoscape app, which integrates _omics data with interactome data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >VirHostNet</span></p>\r\n</td>\r\n<td>\r\n<p><span >VirHostNet (Virus-Host Network) is a knowledgebase system dedicated to the curation, the integration, the management and the analysis of virus-host molecular (mainly protein-protein) interaction networks as well as their functional annotation (molecular functions, cellular pathways, protein domains)</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >visANT</span></p>\r\n</td>\r\n<td>\r\n<p><span >visANT is an integrative visual analysis tool for biologucal networjs and pathways</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >VisHiC</span></p>\r\n</td>\r\n<td>\r\n<p><span >VisHiC is a public web server for clustering and interpreting gene expression data</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Voronto</span></p>\r\n</td>\r\n<td>\r\n<p><span >Voronto is a Mapper for expression data to ontologies</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >WAVe</span></p>\r\n</td>\r\n<td>\r\n<p><span >WAVe is a web analysis of the variome</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Web Apollo</span></p>\r\n</td>\r\n<td>\r\n<p><span >Web Apollo is a web-based genomic annotation editing platform</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >WhichGenes</span></p>\r\n</td>\r\n<td>\r\n<p><span >WhichGenes is a web-based interactive gene set building tool offering a very simple interface to extract always-updated gene lists from multiple databases and unstructured biological data sources</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >Wiki-Pi</span></p>\r\n</td>\r\n<td>\r\n<p><span >A Web-Server of Annotated Human Protein-Protein Interactions to Aid in Discovery of Protein Function</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >WikiCell</span></p>\r\n</td>\r\n<td>\r\n<p><span >WikiCell is a unified resource platform for human transcriptomics research</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >WikiPathways</span></p>\r\n</td>\r\n<td>\r\n<p><span >WikiPathways is an open, public platform dedicated to the curation of biological pathways by and for the scientific community</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >WIM</span></p>\r\n</td>\r\n<td>\r\n<p><span >WIM (Web Interfac for Modelling) is a tool which integrates biological objects and information about biological process modelling</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >WormBase</span></p>\r\n</td>\r\n<td>\r\n<p><span >WormBase is a central data repository for nematode biology</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >YMDB</span></p>\r\n</td>\r\n<td>\r\n<p><span >YMDB is a yeast metabolome database</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><span >yOWL</span></p>\r\n</td>\r\n<td>\r\n<p><span >yOWL represent knowledge found in the Saccharomyces Genomes Database using OWL (Web Ontology Language).</span></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 15:48:13','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(28,562,'Linking to us','linking-to-us','<p><span>Reciprocal linking between related bioinformatics resources not only encourages mutual traffic but ensures that Reactome users can quickly linkout to the source reference material as they navigate through our website. Furthermore, many Reactome visitors are referred to this web site by many bioinformatics resources that integrate Reactome data. These resources include UniProt, Genecards, OMIM, NCBI Biosystems, MSigDB, HGNC, NextProt, BioGPS, Pathway Commons, and Wikipathways.</span></p>\r\n<p><span>Our goal with distributing different sets of files for each identifier type (i.e. protein (</span><a href=\"https://www.uniprot.org\"><span>UniProt</span></a><span>), gene (Ensembl and NCBI), small molecule (ChEBI), and microRNA (miRBase)) is to link the source database identifier to: </span></p>\r\n<ol style=\"list-style-type: lower-alpha;\">\r\n<li><span>the lowest level pathway diagram or subset of the pathway,</span></li>\r\n<li><span>all level pathway diagrams, and all reaction events.</span></li>\r\n</ol>\r\n<p><span>The identifier mapping files can be obtained from our </span><a href=\"download-data\"><span>Downloads</span></a><span> section.</span></p>\r\n<p><span>An alternative strategy to link to Reactome can be achieved by creating URLs containing the name of and an identifier from an external database in the following format: </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=identifier\"><i><span>https://reactome.org/content/query?cluster=true&amp;q=identifier</span></i></a><i><span>.</span></i></p>\r\n<p><span>Click </span><a href=\"index.php?option=com_content&amp;view=article&amp;id=70\"><span>here</span></a><span> to see the external databases and list of identifiers available in the current release of Reactome</span></p>\r\n<p><span>Below are few concrete examples:</span></p>\r\n<ul style=\"list-style-type: circle;\">\r\n<li><a href=\"https://www.uniprot.org/\"><span>UniProt</span></a><span> accession numbers (the AC line, preferred) and identifiers (the ID line), e.g. UNIPROT:P30304, </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=P30304\"><span>https://reactome.org/content/query?cluster=true&amp;q=P30304</span></a></li>\r\n<li><a href=\"https://www.ebi.ac.uk/chebi/\"><span>ChEBI</span></a><span> identifiers, e.g. CHEBI:15422, </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=15422\"><span>https://reactome.org/content/query?cluster=true&amp;q=15422</span></a></li>\r\n<li><a href=\"http://www.genome.jp/kegg/compound/\"><span>COMPOUND</span></a><span> identifiers, e.g. COMPOUND:C00002, </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=C00002\"><span>https://reactome.org/content/query?cluster=true&amp;q=C00002</span></a></li>\r\n<li><a href=\"http://www.geneontology.org/\"><span>Gene Ontology</span></a><span> (GO) accession numbers, e.g. GO:0030060, </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=0030060\"><span>https://reactome.org/content/query?cluster=true&amp;q=0030060</span></a></li>\r\n<li><a href=\"http://www.chem.qmul.ac.uk/iubmb/enzyme/\"><span>Enzyme Classification</span></a><span> (EC) numbers, e.g. EC:1.1.1.37, </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=1.1.1.37\"><span>https://reactome.org/content/query?cluster=true&amp;q=1.1.1.37</span></a></li>\r\n<li><a href=\"https://www.ncbi.nlm.nih.gov/gene\"><span>NCBI Gene</span></a><span>, e.g. NCBI Gene:4171, </span><span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=4171\">https://reactome.org/content/query?cluster=true&amp;q=4171</a></span></li>\r\n<li><span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=4171\"></a></span><a href=\"//reactome.org\"><span>Reactome</span></a><span>, e.g. R-HSA-163809, </span><span><a href=\"https://reactome.org/content/detail/R-HSA-163809\">https://reactome.org/content/detail/R-HSA-163809</a><a href=\"https://reactome.org/content/detail/R-HSA-163809\"><br /></a></span></li>\r\n</ul>\r\n<p>&nbsp;</p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 16:18:42','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(29,563,'Referencing our Publications','referencing-our-publications','<p style=\"text-align: justify;\"><strong>To cite your use of Reactome in your work,</strong>&nbsp;please cite the following publication:</p>\r\n<ul style=\"text-align: justify;\">\r\n<ul>\r\n<li>Fabregat&nbsp;et al. 2018 <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/29145629\">PMID: 29145629</a></li>\r\n<li></li>\r\n</ul>\r\n</ul>\r\n<p style=\"text-align: justify;\"><strong>To cite your use of one of&nbsp;our more specialized resources or tools</strong>,&nbsp;please cite the appropriate publication:</p>\r\n<ul style=\"text-align: justify;\">\r\n<ul>\r\n<li></li>\r\n<li>Diagram Viewer: Fabregat et al. 2018 <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29186351\">PMID:&nbsp;29186351</a></li>\r\n<li>Graph database: Fabregat et al. 2018&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29377902\">PMID:&nbsp;29377902</a></li>\r\n<li>Icon Library and Interactive textbook-like illustrations:&nbsp;Sidiropoulos et al. 2017 <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29077811\">PMID: 29077811</a></li>\r\n<li>Pathway Analysis: Fabregat et al. 2017&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/28249561\">PMID:&nbsp;28249561</a></li>\r\n<li>ReactomeFIViz app and Reactome FI Network: Wu and Haw 2017 <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/28150241\">PMID:&nbsp;28150241</a></li>\r\n<li></li>\r\n</ul>\r\n</ul>\r\n<p style=\"text-align: justify;\"><strong>To cite a pathway:</strong>&nbsp;Please use the appropriate DOI from the&nbsp;<a href=\"/cgi-bin/doi_toc?DB=current\" rel=\"nofollow\">Table of Contents</a>. You can read more about the way DOIs are assigned and used by following&nbsp;<a href=\"http://wiki.reactome.org/index.php/DOI\" target=\"_blank\" rel=\"nofollow noopener noreferrer\">this link</a>. You can add a DOI to the end of your citation following the appropriate style. Generally, these citations follow this format: Author, A. (year). Title of pathway\". Reactome, release#, URL with StableID: R-HSA-xxxxxx.x or doi:xxxxxx&nbsp;(date of access).</p>\r\n<p style=\"text-align: justify;\">Please find other citing styles at&nbsp;APA Style:&nbsp;<a href=\"https://owl.purdue.edu/owl/research_and_citation/apa_style/apa_formatting_and_style_guide/reference_list_electronic_sources.html\">Purdue U Online Writing Lab</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>To cite our files available for download:</strong>&nbsp;\"Name of file\",&nbsp;Reactome,&nbsp;release#,&nbsp;<a href=\"https://reactome.org/download-data/\">https://reactome.org/download-data/</a>&nbsp;(date of access).</p>\r\n<p style=\"text-align: justify;\"><strong>When citing information obtained in a search:</strong>&nbsp;it should be remembered that while we strive to contain the most current and accurate data, Reactome should not be used in citations where other primary sources of information are available.</p>\r\n<p style=\"text-align: justify;\"><strong>To reference an image:</strong>&nbsp;Please provide a stable&nbsp;identifier or&nbsp;a DOI of the pathway that is associated with the image. For example, Image for Title of pathway. Reactome, release#, URL with StableID: R-HSA-xxxxxx.x or doi:xxxxxx&nbsp;(date of access).</p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 16:25:22','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(39,587,'User Guide','user-guide','<h3>What is Reactome?</h3>\r\n<p><span>Reactome is a curated database of pathways and reactions in human biology. Reactions can be considered as pathway steps. Reactome defines a reaction as any event in biology that changes the state of a biological molecule. Binding, activation, translocation, degradation and classical biochemical events involving a catalyst are all reactions. Information in the database is authored by expert biologists, entered and maintained by Reactomes team of Curators and Editorial staff. Reactome content frequently cross-references other resources e.g. </span><a href=\"http://www.ncbi.nlm.nih.gov\" target=\"_blank\" rel=\"noopener noreferrer\"><span>NCBI</span></a><span>, </span><a href=\"http://www.ensembl.org\" target=\"_blank\" rel=\"noopener noreferrer\"><span>Ensembl</span></a><span>, </span><a href=\"http://www.uniprot.org/\" target=\"_blank\" rel=\"noopener noreferrer\"><span>UniProt</span></a><span>, KEGG (</span><a href=\"http://www.genome.jp/kegg/genes.html\" target=\"_blank\" rel=\"noopener noreferrer\"><span>Gene </span></a><span>and </span><a href=\"http://www.genome.jp/dbget-bin/www_bfind?compound\" target=\"_blank\" rel=\"noopener noreferrer\"><span>Compound</span></a><span>), </span><a href=\"http://www.ebi.ac.uk/chebi/init.do\"><span>ChEBI</span></a><span>, </span><a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?CMD=&amp;DB=PubMed\"><span>PubMed</span></a><span> and </span><a href=\"http://www.geneontology.org\" target=\"_blank\" rel=\"noopener noreferrer\"><span>GO</span></a><span>. </span><a href=\"documentation/inferred-events\"><span>Inferred orthologous reactions</span></a><span> Inferred orthologous reactions are available for 17 non-human species including mouse, rat, chicken, puffer fish, worm, fly, yeast, rice, and Arabidopsis</span></p>\r\n<h3>What is this Tutorial For?</h3>\r\n<p><span>This tutorial introduces features of the Reactome website using a combination of short explanations and exercises. You will learn how to search, interpret the views, use the tools and if necessary find documentation or <a href=\"mailto:help@reactome.org\">contact us</a> for help.</span></p>\r\n<p>{loadposition user-guide-1}</p>\r\n','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-08 19:51:22','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"0\",\"link_titles\":\"0\",\"show_tags\":\"\",\"show_intro\":\"0\",\"info_block_position\":\"\",\"info_block_show_title\":\"0\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,7,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(43,594,'404','404','<h3 class=\"fav404\">404</h3>\r\n<p class=\"fav404\">Oops! The page you requested was not found!</p>\r\n<p class=\"fav404\"><a href=\"/\" class=\"btn\">Back to Home Page</a></p>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-17 12:15:17','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"0\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(44,604,'ReactomeFIVIz','reactome-fiviz','<div id=\"toc\" class=\"toc\">\r\n	<div id=\"toctitle\">\r\n		<h2>Contents</h2>\r\n	</div>\r\n	<ul>\r\n		<li class=\"toclevel-1 tocsection-1\"><a href=\"#Overview\"><span\r\n				class=\"tocnumber\">1.</span> <span class=\"toctext\">Overview</span></a></li>\r\n		<li class=\"toclevel-1 tocsection-2\"><a\r\n			href=\"#Download_and_Launch_ReactomeFIViz\"><span class=\"tocnumber\">2.</span>\r\n				<span class=\"toctext\">Download and Launch ReactomeFIViz</span></a></li>\r\n		<li class=\"toclevel-1 tocsection-3\"><a\r\n			href=\"#Use_Reactome_Pathways\"><span class=\"tocnumber\">3.</span> <span\r\n				class=\"toctext\">Use Reactome Pathways</span></a>\r\n			<ul>\r\n				<li class=\"toclevel-2 tocsection-4\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;&nbsp;<a href=\"#Explore_Reactome_Pathways\"><span\r\n						class=\"tocnumber\">3.1.</span> <span class=\"toctext\">Explore\r\n							Reactome Pathways</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-5\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;&nbsp;<a\r\n					href=\"#Display_Reactome_Pathways_in_the_FI_Network_View\"><span\r\n						class=\"tocnumber\">3.2.</span> <span class=\"toctext\">Display\r\n							Reactome Pathways in the FI Network View</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-6\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;&nbsp;<a href=\"#Pathway_Enrichment_Analysis\"><span\r\n						class=\"tocnumber\">3.3.</span> <span class=\"toctext\">Pathway\r\n							Enrichment Analysis</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-7\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;&nbsp;<a\r\n					href=\"#Probabilistic_Graphical_Model_based_Pathway_Analysis\"><span\r\n						class=\"tocnumber\">3.4.</span> <span class=\"toctext\">Probabilistic\r\n							Graphical Model based Pathway Analysis</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-8\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;&nbsp;<a href=\"#Boolean_Network_based_Pathway_Analysis\"><span\r\n						class=\"tocnumber\">3.5.</span> <span class=\"toctext\">Boolean\r\n							Network based Pathway Analysis</span></a>\r\n				</li>\r\n			</ul></li>\r\n		<li class=\"toclevel-1 tocsection-9\"><a\r\n			href=\"#Use_the_Reactome_Functional_Interaction_.28FI.29_Network\"><span\r\n				class=\"tocnumber\">4.</span> <span class=\"toctext\">Use the\r\n					Reactome Functional Interaction (FI) Network</span></a>\r\n			<ul>\r\n				<li class=\"toclevel-2 tocsection-10\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#Gene_Set.2FMutation_Analysis\"><span\r\n						class=\"tocnumber\">4.1.</span> <span class=\"toctext\">Gene\r\n							Set/Mutation Analysis</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-11\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#PGM_Impact_Analysis\"><span class=\"tocnumber\">4.2.</span>\r\n						<span class=\"toctext\">PGM Impact Analysis</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-12\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#HotNet_Mutation_Analysis\"><span\r\n						class=\"tocnumber\">4.3.</span> <span class=\"toctext\">HotNet\r\n							Mutation Analysis</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-13\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#Microarray_Data_Analysis\"><span\r\n						class=\"tocnumber\">4.4.</span> <span class=\"toctext\">Microarray\r\n							Data Analysis</span></a>\r\n				</li>\r\n			</ul></li>\r\n		<li class=\"toclevel-1 tocsection-14\"><a\r\n			href=\"#Visualize_Cancer_Drugs_in_Contexts_of_Reactome_Pathways_and_FI_Network\"><span\r\n				class=\"tocnumber\">5.</span> <span class=\"toctext\">Visualize\r\n					Drugs in the Contexts of Reactome Pathways and FI Network</span></a>\r\n			<ul>\r\n				<li class=\"toclevel-2 tocsection-15\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#Visualize_Cancer_Drugs_in_Reactome_Pathways\"><span\r\n						class=\"tocnumber\">5.1.</span> <span class=\"toctext\">Visualize\r\n							Cancer Drugs in Reactome Pathways</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-16\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#Visualize_Cancer_Drugs_in_the_FI_Network\"><span\r\n						class=\"tocnumber\">5.2.</span> <span class=\"toctext\">Visualize\r\n							Cancer Drugs in the FI Network</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-16\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#Visualize_DrugCentral_Drugs\"><span\r\n						class=\"tocnumber\">5.3.</span> <span class=\"toctext\">Visualize\r\n							Drug Central Drugs</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-17\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a\r\n					href=\"#Simulate_Impact_of_Cancer_Drugs_on_Pathway_Activities\"><span\r\n						class=\"tocnumber\">5.4.</span> <span class=\"toctext\">Simulate\r\n							Impact Drugs on Pathway Activities</span></a>\r\n				</li>\r\n			</ul></li>\r\n		<li class=\"toclevel-1 tocsection-18\"><a\r\n			href=\"#Other_Features_Related_to_the_FI_Network\"><span\r\n				class=\"tocnumber\">6.</span> <span class=\"toctext\">Other\r\n					Features Related to the FI Network</span></a>\r\n			<ul>\r\n				<li class=\"toclevel-2 tocsection-19\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#Query_FI_Source\"><span class=\"tocnumber\">6.1.</span>\r\n						<span class=\"toctext\">Query FI Source</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-20\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#Fetch_FIs_for_Node\"><span class=\"tocnumber\">6.2.</span>\r\n						<span class=\"toctext\">Fetch FIs for Node</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-21\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#Show_Pathway_Diagram\"><span class=\"tocnumber\">6.3.</span>\r\n						<span class=\"toctext\">Show Pathway Diagram</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-22\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#Load_Cancer_Gene_Index_Annotations\"><span\r\n						class=\"tocnumber\">6.4.</span> <span class=\"toctext\">Load\r\n							Cancer Gene Index Annotations</span></a>\r\n				</li>\r\n				<li class=\"toclevel-2 tocsection-23\">&nbsp; &nbsp; &nbsp;\r\n					&nbsp;<a href=\"#Survival_Analysis\"><span class=\"tocnumber\">6.5.</span>\r\n						<span class=\"toctext\">Survival Analysis</span></a>\r\n				</li>\r\n			</ul></li>\r\n	</ul>\r\n</div>\r\n<h3>\r\n	<span id=\"Overview\">Overview</span>\r\n</h3>\r\n<p>\r\n	The <a href=\"http://f1000research.com/articles/3-146/v2\" rel=\"nofollow\">ReactomeFIViz</a>\r\n	app is designed to find pathways and network patterns related to cancer\r\n	and other types of diseases. This app accesses the <a\r\n		href=\"https://reactome.org/\" rel=\"nofollow\">Reactome</a> pathways\r\n	stored in the database, help you to do pathway enrichment analysis for\r\n	a set of genes, visualize hit pathways using manually laid-out pathway\r\n	diagrams directly in Cytoscape, and investigate functional\r\n	relationships among genes in hit pathways. The app can also access the\r\n	Reactome Functional Interaction (FI) network, a highly reliable,\r\n	manually curated pathway-based protein functional interaction network\r\n	covering over 60% of human proteins, and allows you to construct a FI\r\n	sub-network based on a set of genes, query the FI data source for the\r\n	underlying evidence for the interaction, build and analyze network\r\n	modules of highly-interacting groups of genes, perform functional\r\n	enrichment analysis to annotate the modules, expand the network by\r\n	finding genes related to the experimental data set, display pathway\r\n	diagrams, and overlay with a variety of information sources such as\r\n	cancer gene index annotations. Recently we have also added features to\r\n	help users visualize FDA-approved cancer drugs in the contexts of the\r\n	FI network and Reactome pathways, and use Boolean network models\r\n	directly built from Reactome pathways to investigate potential\r\n	functional impacts of displayed cancer drugs.\r\n</p>\r\n<p>\r\n	For an example how we use Reactome FIs for cancer data analysis, please\r\n	see our publication: <a\r\n		href=\"http://genomebiology.com/content/11/5/R53\" rel=\"nofollow\">A\r\n		human functional protein interaction network and its application to\r\n		cancer data analysis</a>.\r\n</p>\r\n<h3>\r\n	<span id=\"Download_and_Launch_ReactomeFIViz\">Download and Launch\r\n		ReactomeFIViz</span>\r\n</h3>\r\n<p>\r\n	ReactomeFIViz app 6 needs Cytoscape 3.7.0 or above. If you have not\r\n	installed Cytoscape 3.7.0 or above, please download it from Cytoscape\'s\r\n	web site: <a href=\"http://www.cytoscape.org/\" rel=\"nofollow\">http://www.cytoscape.org</a>.\r\n	After launching Cytoscape, use menu \"Apps/App Manager\" to open the \"App\r\n	Manager\" dialog, and search for \"ReactomeFI\". You should see the\r\n	ReactomeFIViz app listed in the middle panel (See the Figure below. You\r\n	may see a different version number. <b>Note: The listed name of\r\n		this app is \"ReactomeFIPlugIn\"</b>, which is the original name of the\r\n	app.). Choose the app, and then click the \"Install\" button at the\r\n	bottom of the dialog. Follow the procedures to finish the installation.\r\n</p>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img\r\n				src=\"images/userguide/fiviz/InstallReactomeFIVizFromAppstore.png\"\r\n				alt=\"\" width=\"663\" height=\"540\" />\r\n			<div>Install ReactomeFIViz app From App Store</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<h3>\r\n	<span id=\"Use_Reactome_Pathways\">Use Reactome Pathways</span>\r\n</h3>\r\n<p>Using the pathway visualization and analysis features, you can\r\n	load pathways in the Reactome database into Cytoscape, visualize\r\n	Reactome pathways in either the native pathway diagram view or the FI\r\n	network view, do pathway enrichment analysis for a set of genes, and\r\n	check genes from your list in hit pathways.</p>\r\n<h4>\r\n	<span id=\"Explore_Reactome_Pathways\">Explore Reactome Pathways</span>\r\n</h4>\r\n<ol>\r\n	<li>Load Reactome pathways: Use menu \"Apps/Reactome FI/Reactome\r\n		Pathways\" to load pathways into Cytoscape. The loaded pathways are\r\n		organized in a hierarchical way as in the Reactome web application (<a\r\n		href=\"/PathwayBrowser\" rel=\"nofollow\">https://reactome.org/PathwayBrowser/</a>),\r\n		and listed in the left side \"Control Panel\" in the tab called\r\n		\"Reactome\".\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ReactomePathways_4.png\" alt=\"\"\r\n						width=\"429\" height=\"602\" />\r\n					<div>Reactome Pathways</div>\r\n				</div>\r\n			</div>\r\n		</div>\r\n	</li>\r\n	<li>View pathways in Reactome: After selecting a pathway in the\r\n		pathway hierarchy, you can choose \"View Reactome Source\" from the\r\n		popup menu (right click in Windows or Control-click in Macs to get the\r\n		popup menu)\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<img src=\"images/userguide/fiviz/PathwayPopup_4.png\" alt=\"\"\r\n					width=\"340\" height=\"248\" />\r\n				<div>Pathway Popup Menu</div>\r\n			</div>\r\n		</div> to view its detailed annotation in Reactome. Or you can choose \"View\r\n		in Reactome\" to view the detailed information in the Reactome web\r\n		application. <br />\r\n	<b>Note</b>: The ancestor pathways (container pathways) for a selected\r\n		pathway are displayed in the middle panel, \"Selected Event Branch\", in\r\n		the Reactome tab. You can click an ancestor pathway in this middle\r\n		panel to view the clicked pathway\'s location in the original pathway\r\n		hierarchical tree. However, the ancestor pathway will not be selected\r\n		in the original tree. This is a designed behavior to keep the\r\n		selection in the original tree.\r\n	</li>\r\n	<li>Search pathways: Choose \"Search\" in the popup menu to bring up\r\n		the search dialog. The found pathway(s) will be highlighted in blue in\r\n		the pathway tree. <br />\r\n	<b>Note</b>: Search will be against all loaded pathways, not limited to\r\n		the selected pathway and its contained sub-pathways.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/PathwaySearch_4.png\" alt=\"\"\r\n						width=\"369\" height=\"199\" />\r\n					<div>Search Pathways</div>\r\n				</div>\r\n			</div>\r\n		</div>\r\n	</li>\r\n	<li>Open pathway diagram: Pathways in Reactome are organized in a\r\n		hierarchical way. Not all pathways have their own pathway diagrams. A\r\n		smaller pathway (called sub-pathway) may be drawn in a bigger pathway,\r\n		which has its own pathway diagram. Most of top-level pathways (called\r\n		modules or super pathways) are used to organize related pathways (e.g.\r\n		Disease, Signaling Transduction), and therefore contain only rectangle\r\n		boxes representing canonical pathways.\r\n		<ol>\r\n			<li><b>Show Diagram</b>: If a selected pathway has its own\r\n				pathway diagram, you can choose \"Show Diagram\" in the popup menu to\r\n				open its pathway diagram into the central Cytoscape desktop.</li>\r\n			<li><b>View in Diagram</b>: If a selected pathway is laid-out as\r\n				a sub-pathway in a bigger one, you can choose \"View in Diagram\" in\r\n				the popup menu to view its drawing in its container pathway.\r\n				Reactions contained by the selected pathway will be highlighted in\r\n				blue after the diagram is opened. For example, see pathway \"G1/S DNA\r\n				Damage Checkpoints\" opened in pathway \"Cell Cycle Checkpoints\"\r\n				below:<br />\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img src=\"images/userguide/fiviz/PathwayDiagram_4.png\" alt=\"\"\r\n								width=\"800\" height=\"540\" />\r\n							<div>Pathway Diagram</div>\r\n						</div>\r\n					</div>\r\n				</div></li>\r\n		</ol>\r\n	</li>\r\n	<li>Search diagram: Objects displayed in a pathway diagram can be\r\n		searched using \"Search Diagram\" from the popup menu (Right click in\r\n		Windows or Control click in Macs without selecting any object in the\r\n		pathway diagram to get the popup menu). The found objects will be\r\n		selected and highlighted in blue. <br />\r\n	<b>Note</b>: Reactions will not be searched in the diagram. Use the\r\n		search feature in the pathway tree to search for reactions.\r\n	</li>\r\n	<li>Export diagram: Displayed diagram can be exported as a PDF,\r\n		JPG or PNG file. Use \"Export Diagram\" from the popup menu to export\r\n		the displayed diagram.</li>\r\n	<li>View Reactome Source or View in Reactome: Select an object,\r\n		and then right-click (or control click) to get the popup menu. Choose\r\n		\"View Reactome Source\" to view the detailed annotation for the\r\n		selected object in a table (See Figure below for an example). Or\r\n		choose \"View in Reactome\" to view the selected object in the Reactome\r\n		web application.<br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ReactomeInstanceView.png\" alt=\"\"\r\n						width=\"497\" height=\"393\" />\r\n					<div>Reactome Instance View</div>\r\n				</div>\r\n			</div>\r\n		</div>\r\n	</li>\r\n	<li>List Genes: Genes contained by a complex or protein set, or a\r\n		gene related by a displayed protein can be viewed by using a menu item\r\n		\"List Genes\" after selecting an object. For example, the following\r\n		dialog shows genes contained by complex hBUBR1:hBUB3:MAD2*:CDC20.\r\n		Clicking a gene symbol will bring you to the web page for that gene in\r\n		the GeneCard web site.<br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ListGenes_4.png\" alt=\"\"\r\n						width=\"448\" height=\"406\" />\r\n					<div>List Genes</div>\r\n				</div>\r\n			</div>\r\n		</div>\r\n	</li>\r\n</ol>\r\n<h4>\r\n	<span id=\"Display_Reactome_Pathways_in_the_FI_Network_View\">Display\r\n		Reactome Pathways in the FI Network View</span>\r\n</h4>\r\n<ol>\r\n	<li>Display pathway in the FI network view: A Reactome pathway can\r\n		be converted into a functional interaction network using the method we\r\n		have established (see <a href=\"http://genomebiology.com/2010/11/5/r53\"\r\n		rel=\"nofollow\">A human functional protein interaction network and\r\n			its application to cancer dat analysis</a>). Use \"Convert to FI Network\"\r\n		in the popup menu brought up by right-clicking (Windows) or\r\n		control-clicking (Macs) an empty area without any selection in the\r\n		pathway diagram panel. The original pathway diagram will be moved to\r\n		the bottom-left corner, and a new FI network will be generated based\r\n		on the original pathway diagram, which will be displayed in a new\r\n		network panel. <br />\r\n	<b>Note</b>: sub-pathways contained by the displayed pathway will be\r\n		extracted into the FI network too.<br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/PathwayInFINetworkView_4.png\"\r\n						alt=\"\" width=\"800\" height=\"486\" />\r\n					<div>Pathway in the FI Network View</div>\r\n				</div>\r\n			</div>\r\n		</div>\r\n	</li>\r\n	<li>Explore objects in the pathway and network views: Object\r\n		selection in three views has been synchronized. Objects that can be\r\n		selected include: events in the pathway tree view, objects in the\r\n		pathway view at the bottom-left corner, and genes and FIs in the\r\n		network view. You can select an object in one of three views, and\r\n		corresponding objects in other two views should be selected too. Also\r\n		you should use features implemented in popup menus in each individual\r\n		view to explore objects as in a single view.</li>\r\n</ol>\r\n<p>\r\n	<br />\r\n	<b>Note</b>: Using Cytoscape\'s built-in \"Saving Session\" feature can\r\n	save the converted FI networks from pathways. However, displayed\r\n	pathways cannot be saved into a session file for the time being. We\r\n	will implement this function in a future release.\r\n</p>\r\n<h4>\r\n	<span id=\"Pathway_Enrichment_Analysis\">Pathway Enrichment\r\n		Analysis</span>\r\n</h4>\r\n<ol>\r\n	<li><b>Pathway enrichment analysis</b>: A list of genes can be\r\n		used to check if any of Reactome pathways have been enriched. To do\r\n		this, use the popup menu item, \"Analyze Pathway Enrichment\" (below\r\n		left figure), to get the dialog for choosing a gene set file (below\r\n		right figure). You can use a gene set file in one of three file\r\n		formats: one gene per line, all genes in the same line and delimited\r\n		by commas, or all genes in the same line and delimited by tabs. You\r\n		can also manually input genes by clicking the \"Click to Enter\" button\r\n		(one line for one gene).<br />\r\n	<b>Note</b>: Dependent on the size of your gene list, it may take over\r\n		1 minute for running the pathway enrichment analysis. Pathways used in\r\n		this feature are different from Reactome pathways for annotating a FI\r\n		network or network modules. Here all over 2,000 pathways are used. For\r\n		annotation, only a subset of Reactome pathways, which have been\r\n		pre-selected for a certain size, are used.<br />\r\n	<img src=\"images/userguide/fiviz/AnalyzeGeneEnrichment_4.png\"\r\n		alt=\"Analyze Pathway Enrichment\" height=\"233\" /><img\r\n		src=\"images/userguide/fiviz/DialogForAnalyzeGeneEnrichment_4.png\"\r\n		alt=\"Dialog for Analyzing Pathway Enrichment\" width=\"400\" height=\"233\" /></li>\r\n	<li><b>View enrichment analysis results</b>: Pathway enrichment\r\n		results are displayed as a table labeled as \"Reactome Pathway\r\n		Enrichment\" in the \"Table Panel\" at the bottom of the main Cytoscape\r\n		window. You can use \"Views in Diagram\" to view hit pathways in the\r\n		pathway diagram view, and use \"Export Annotations\" to save the results\r\n		in the table. Pathways in the Reactome pathway tree are highlighted in\r\n		different colors based on their FDR values. Objects containing genes\r\n		from your gene list are highlighted in a purple background with a\r\n		white font in the pathway diagram view. Hit genes are displayed in a\r\n		thick purple border in the FI network view for a hit pathway. <br />\r\n	<b>Note</b>: Hit genes are displayed with same colors in the \"Gene\r\n		List\" dialog from the \"List Genes\" feature.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<img src=\"images/userguide/fiviz/PathwayEnrichmentResults_4.png\"\r\n					alt=\"\" width=\"800\" height=\"460\" />\r\n				<div>Pathway Enrichment Results</div>\r\n			</div>\r\n		</div></li>\r\n	<li><b>Perform GSEA analysis</b>: Gene Set Enrichment Analysis (<a\r\n		href=\"http://software.broadinstitute.org/gsea/index.jsp\">GSEA</a>) is\r\n		a rank-based pathway enrichment analysis approach, widely used in\r\n		pathway-based data analysis. ReactomeFIViz provides support to perform\r\n		GSEA analysis for Reactome pathways using a gene score file. Gene\r\n		score may be t-score from differential gene expression analysis or\r\n		other type of scores that can be ranked. To perform the GSEA pathway\r\n		enrichment analysis, you need to provide a tab-delimited text file\r\n		containing two columns: the first for gene symbols (human only) and\r\n		the second for gene scores. The first row is reserved for the column\r\n		headers, and will not be imported for analysis. To perform GSEA\r\n		analysis, use popup menu \"Perform GSEA Analysis\" in the pathway tree\r\n		to bring up the GSEA configuration dialog, where you can enter the\r\n		gene score file and choose the minimum and maximum size of pathways\r\n		along with the permutation number.\r\n		<div class=\"text-center\">\r\n			<img src=\"images/userguide/fiviz/PerformGSEAAnalysis.png\"\r\n				alt=\"Perform GSEA Analysis\" width=\"400\" />\r\n			<div>Perform GSEA Analysis</div>\r\n			<img src=\"images/userguide/fiviz/ConfigureGSEAAnalysis.png\"\r\n				alt=\"Congigure GSEA Analysis\" width=\"400\" />\r\n			<div>Configure GSEA Analysis</div>\r\n		</div>\r\n		<div>\r\n			<p>\r\n				The GSEA analysis results are displayed in the table labeled as\r\n				\"Reactome GSEA Analysis\" in Cytoscape Table Panel. Pathways subject\r\n				to GSEA analysis in the pathway tree are highlighted based on FDR\r\n				values as in the gene set-based pathway enrichment analysis (See\r\n				above). For details about the meanings of columns shown in the\r\n				results table, please consult the original GSEA document: <a\r\n					href=\"http://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Main_Page\">GSEA\r\n					Document</a>.\r\n			</p>\r\n		</div>\r\n		<div class=\"text-center\">\r\n			<img src=\"images/userguide/fiviz/GSEAAnalysisResults.png\"\r\n				alt=\"GSEA Analysis Results\" width=\"700\" />\r\n			<div>GSEA Analysis Results</div>\r\n		</div></li>\r\n	<li><b>Overlay Gene Scores onto Pathways</b>: For significant\r\n		pathways produced from the GSEA analysis, you can overlay gene scores\r\n		to investigate locations of products of genes having significant high\r\n		or low scores, therefore to understand potential pathway activity\r\n		impact caused by these extreme scores. To do this, use popup menu\r\n		\"Overlay Gene Scores\" in pathway diagram view to choose the gene score\r\n		file in the configuration dialog. After the file loading, entities in\r\n		pathway diagrams will be highlighted based on scores. You may choose\r\n		one or more genes in the right gene scores Table View to visualize\r\n		related entities in the pathway diagram.\r\n		<div class=\"text-center\">\r\n			<img src=\"images/userguide/fiviz/OverlayGeneScores.png\"\r\n				alt=\"Overlay Gene Scores\" width=\"375\" />\r\n			<div>Overlay Gene Scores</div>\r\n			<img src=\"images/userguide/fiviz/GeneScoreOverlayResults.png\"\r\n				alt=\"Gene Score Overlay Results\" width=\"900\" />\r\n			<div>Gene Score Overlay Results</div>\r\n		</div>\r\n		<div>\r\n			<p>\r\n				<b>Note</b>: If an entitiy (e.g. a complex or an EntitySet) is\r\n				composed of more than one gene, the score for the entity is the mean\r\n				of all genes annotated for that entity. To remove overlaid gene\r\n				scores in the pathway diagram, use popup menu \"Remove Gene Scores\".\r\n				To view the distribution of scores for genes annotated in the\r\n				displayed pathway diagram, choose \"Plot View\" in the \"Gene Scores\"\r\n				tab in Cytoscape Results Panel (see below).\r\n			</p>\r\n			<div class=\"text-center\">\r\n				<img src=\"images/userguide/fiviz/GeneScoreDistribution.png\"\r\n					alt=\"Gene Score Distribution\" width=\"350\" />\r\n				<div>Gene Score Distribution</div>\r\n			</div>\r\n		</div></li>\r\n</ol>\r\n<h4>\r\n	<span id=\"Probabilistic_Graphical_Model_based_Pathway_Analysis\">Probabilistic\r\n		Graphical Model based Pathway Analysis</span>\r\n</h4>\r\n<p>\r\n	We adapted the PARADIGM approach for Reactome pathways by converting\r\n	reactions drawn in pathway diagrams into factors in factor graphs, a\r\n	type of probabilistic graphical models (PGMs). For details about the\r\n	PARADIGM approach, see: <a\r\n		href=\"http://wiki.reactome.org/index.php/Reactome_FI_Cytoscape_Plugin#Probabilistic_Graphical_Model_based_Pathway_Analysis\"\r\n		rel=\"nofollow\">Inference of patient-specific pathway activities\r\n		from multi-dimensional cancer genomics data using PARADIGM</a>. For\r\n	introduction to factor graphs, see this wikipedia entry: <a\r\n		href=\"http://en.wikipedia.org/wiki/Factor_graph\" rel=\"nofollow\">Factor\r\n		Graph</a>. For test purposes, you can download two sample data files for\r\n	100 TCGA ovarian cancer patients: <a title=\"ov.CNV.100.txt.zip\"\r\n		href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/ImpactAnalysis/ov.CNV.100.txt.zip\">CNVs</a>\r\n	and <a title=\"ov.mRNA.100.txt.zip\"\r\n		href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/ImpactAnalysis/ov.mRNA.100.txt.zip\">mRNA\r\n		gene expression</a>. The original TCGA OV files were downloaded from <a\r\n		href=\"https://confluence.broadinstitute.org/display/GDAC/Home;jsessionid=2263803E3BFAB3B5402E7489188E2549\"\r\n		rel=\"nofollow\">the Broad Institute Firehose</a> web site.\r\n</p>\r\n<ol>\r\n	<li><b>Run graphical model analysis in batch:</b> This feature is\r\n		used to perform a batch graphical model analysis for all Reactome\r\n		pathways having manual layout diagrams.\r\n		<ol>\r\n			<li><b>Start the analysis:</b> Choose the popup menu, \"Run\r\n				Graphical Model Analysis\", in the pathway hierarchical tree. After\r\n				choosing this menu, you will be asked to choose data files and\r\n				provide parameters for inference algorithms in the following two\r\n				tabs in the \"Run Graphical Model Analysis\" dialog:<br />\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img src=\"images/userguide/fiviz/LoadData.png\" alt=\"\" width=\"500\"\r\n								height=\"509\" />\r\n							<div>LoadData</div>\r\n						</div>\r\n					</div>\r\n				</div>\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img src=\"images/userguide/fiviz/SetUpAlgorithms.png\" alt=\"\"\r\n								width=\"500\" height=\"504\" />\r\n							<div>SetUpAlgorithms</div>\r\n						</div>\r\n					</div>\r\n				</div> <br />\r\n			<b>Notes</b>: <br />1). If you choose \"Use empirical distribution\"\r\n				in the data loading dialog, your loaded data will be used directly\r\n				to construct factor functions without discretizing. At present, we\r\n				recommend to use \"Choose threshold values for discretizing\". <br />2).\r\n				It is recommended to use the default parameters for inference\r\n				algorithms for a batch analysis for quick performance. You can try\r\n				different parameters for some specific pathways after you find\r\n				interesting pathways from the batch analysis. <i><b>If you\r\n						want to perform two-case study (e.g. case-control, drug\r\n						sensitive/insensitive, etc), check the checkbox, \"Used for pathway\r\n						analysis for samples with two cases\", and provide a sample\r\n						information file as required. For two-case analysis, a random data\r\n						set will not be generated. Results will be presented by comparing\r\n						two types of samples in your uploaded data files.</b></i></li>\r\n			<li><b>Run the analysis:</b> Click the \"OK\" button to start the\r\n				batch analysis. Depending on your sample size, it may take hours to\r\n				finish the whole analysis.</li>\r\n			<li><b>Finish the analysis:</b> After the batch analysis done,\r\n				you may see the following list if some of pathways cannot be\r\n				analyzed because the inference algorithm cannot converge. Please\r\n				make sure the following list is small (probably less than 10\r\n				pathways) so that you can get enough results.\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img src=\"images/userguide/fiviz/FailedPathwaysList.png\" alt=\"\"\r\n								width=\"523\" height=\"289\" />\r\n							<div>FailedPathwaysList</div>\r\n						</div>\r\n					</div>\r\n				</div></li>\r\n			<li><b>View the results:</b> The results from the batch analysis\r\n				are displayed at the bottom table panel of the Cytoscape desktop as\r\n				the following:\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img src=\"images/userguide/fiviz/BatchResults.png\" alt=\"\"\r\n								width=\"800\" height=\"280\" />\r\n							<div>BatchResults</div>\r\n						</div>\r\n					</div>\r\n				</div> <b>Note</b>: There are 7 columns in this table: ReactomePathway for\r\n				pathway names analyzed by the App; AverageUpIPA shows how much a\r\n				pathway is up-perturbated by comparing to a random background (IPA:\r\n				integrated pathway activity. See the above PARADIGM paper for\r\n				details); AverageDownIPA shows how much a pathway is\r\n				down-perturbated; CombinedPValue is a p-value indicating how\r\n				significant this pathway is perturbed based on pathway outputs and\r\n				the Fisher\'s method; MinimumPValue is the minimum p-value for\r\n				pathway outputs; the last two columns are FDRs for two p-values\r\n				based on the BenjaminiHochberg method. AverageUpIPA or\r\n				AverageDownIPA may be NaN, which indicates there is no detected up\r\n				or down perturbation based on this analysis. The FDR filtering works\r\n				based on the FDR values displayed in the last two columns with \"OR\"\r\n				operation.</li>\r\n			<li><b>Save the results:</b> To keep the results, use the popup\r\n				menu in the table, \"Export Annotations\", to save the results into an\r\n				external text file. The saved results can be loaded later on by\r\n				using popup menu, \"Load Graphical Model Results\", in the pathway\r\n				tree.</li>\r\n		</ol></li>\r\n	<li><b>Run graphical model analysis for a single pathway:</b> This\r\n		feature is used to perform a graphical model analysis for a pathway\r\n		displayed in the Cytoscape desktop.\r\n		<ol>\r\n			<li><b>Open a pathway:</b> As before, you can choose a pathway\r\n				in the pathway tree, and open its diagram in the Cytoscape desktop.\r\n				Or you can choose an interesting pathway from the batch analysis\r\n				results table by choosing popup menu, \"View in Diagram\".</li>\r\n			<li><b>Start the analysis:</b> Choose popup menu, \"Run Graphical\r\n				Model Analysis\", from the popup menu list in the pathway diagram\r\n				window. You will be asked to provide data files and set up inference\r\n				algorithms as in the batch analysis. After clicking the \"OK\" button,\r\n				you will be asked to provide a list of escape names for entities in\r\n				the pathway that will not be considered in the graphical model (e.g.\r\n				ATP, ADP, etc) in the following dialog:\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img src=\"images/userguide/fiviz/EscapeNamesDialog.png\" alt=\"\"\r\n								width=\"401\" height=\"277\" />\r\n							<div>EscapeNameDialog</div>\r\n						</div>\r\n					</div>\r\n				</div> <br />\r\n			<b>Note</b>: If you have loaded data files, you may choose to use the\r\n				loaded data files without displaying the data loading dialog.</li>\r\n			<li><b>View the results:</b> After the analysis is done, three\r\n				tabs are displayed in the table pane of Cytoscape: IPA Pathway\r\n				Analysis, IPA Sample Analysis and IPA Node Values. IPA Pathway\r\n				analysis displays inference results for entities in the pathway by\r\n				comparing samples in your data files and in a random data set\r\n				generated dynamically by the App based on your data files. IPA\r\n				Sample Analysis shows results for each individual samples as up or\r\n				down perturbation. You may choose to show/hide p-values and FDR\r\n				values for samples in the table. IPA Node Values show inference\r\n				results for selected entities in the pathway diagram for each\r\n				sample. Entities in the pathway diagram are highlighted based on\r\n				values in the MeanDiff column in the IPA Pathway Analysis tab.\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img src=\"images/userguide/fiviz/CellCycleCheckPointsResults.png\"\r\n								alt=\"\" width=\"800\" height=\"452\" />\r\n							<div>CellCycleCheckPointsResults</div>\r\n						</div>\r\n					</div>\r\n				</div> <br />\r\n			<b>Note</b>: You may change the color spectrum mapping for pathway\r\n				diagram highlighting by double-clicking the color spectrum bar at\r\n				the bottom of pathway diagram window to get the dialog for setting\r\n				min/max values. You can save the analysis results for a pathway by\r\n				using popup menu, \"Save Analysis Results\", and load the results back\r\n				later on by \"Open Analysis Results\".</li>\r\n			<li><b>Analyze gene level results:</b> The up or down\r\n				perturbation results are inferred based on genomic data files for\r\n				individual genes. The App provides features to analyze observation\r\n				results and inference results for individual genes. You can view\r\n				gene-level observation and inference results for the whole pathway\r\n				by using popup menus, \"Show Gene Level Analysis Results\" and \"Show\r\n				Observations\". You can also view these results for genes contained\r\n				by an entity displayed in the pathway diagram after selecting that\r\n				entity and then using these two popup menus. The following two\r\n				dialogs show gene level observations and inference results for genes\r\n				whose products are contained by complex \"hBUBR1:hBUB3:MAD2*:CDC20\r\n				complex [cytosol]\" in the cell cycle checkpoints pathway:\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img src=\"images/userguide/fiviz/ObservationsForEntity.png\"\r\n								alt=\"\" width=\"799\" height=\"474\" />\r\n							<div>ObservationsForEntity</div>\r\n						</div>\r\n					</div>\r\n				</div>\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img src=\"images/userguide/fiviz/GeneLevelResultsForEntity.png\"\r\n								alt=\"\" width=\"799\" height=\"479\" />\r\n							<div>GeneLevelResultsForEntity</div>\r\n						</div>\r\n					</div>\r\n				</div></li>\r\n			<li><b>Analyze and visualize results for individual samples:</b>\r\n				The inference results and loaded observation data are displayed in\r\n				the right \"Results Panel\" (see below). By checking \"Highlight\r\n				pathway for sample\", entities in the displayed pathway diagram will\r\n				be highlighted based on inferred IPA values for the selected sample\r\n				displayed in the \"Choose sample\" box. You can also enable animation\r\n				by clicking the play button. There are two tabs in the \"Results\r\n				Panel\": \"Inference\" for showing inferred IPA values, and\r\n				\"Observation\" tab for loaded observed data related to entities in\r\n				the pathway (Note: if you choose \"discretizing\", the displayed\r\n				observation values are discretized: 0 for lower than normal, 1 for\r\n				normal, and 2 for higher normal). Objects in three views (pathway\r\n				diagram, inference table, and observation table) are synchronized\r\n				for selection.\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img src=\"images/userguide/fiviz/PGMSampleViewWithInference.png\"\r\n								alt=\"\" width=\"799\" height=\"321\" />\r\n							<div>PGM Sample View: Inference</div>\r\n						</div>\r\n					</div>\r\n				</div>\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img\r\n								src=\"images/userguide/fiviz/PGMSampleViewWithObservation.png\"\r\n								alt=\"\" width=\"799\" height=\"320\" />\r\n							<div>PGM Sample View: Observation</div>\r\n						</div>\r\n					</div>\r\n				</div></li>\r\n			<li><b>Compare analysis results for two samples:</b> You can\r\n				compare observation data and inference results for two samples. To\r\n				do this, choose two samples in the \"IPA Sample Analysis\" tab in the\r\n				bottom results pane, and use popup menu \"Compare Samples\" to bring\r\n				out another tab called \"Sample Comparison\". You can view comparing\r\n				results for inference and observation data.\r\n				<div class=\"text-center\">\r\n					<div>\r\n						<div>\r\n							<img src=\"images/userguide/fiviz/TwoSampleComparison.png\" alt=\"\"\r\n								width=\"799\" height=\"649\" />\r\n							<div>Two Sample Comparison</div>\r\n						</div>\r\n					</div>\r\n				</div></li>\r\n		</ol></li>\r\n</ol>\r\n<h4>\r\n	<span id=\"Boolean_Network_based_Pathway_Analysis\">Boolean\r\n		Network based Pathway Analysis</span>\r\n</h4>\r\n<p>\r\n	We have developed an approach (Manuscript in preparation) to convert\r\n	biochemical reactions-based Reactome pathways into Boolean networks and\r\n	then perform pathway simulation based on the constrained fuzzy logic\r\n	method according to <a\r\n		href=\"http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1001099\"\r\n		rel=\"nofollow\">Training Signaling Pathway Maps to Biochemical Data\r\n		with Constrained Fuzzy Logic: Quantitative Analysis of Liver Cell\r\n		Responses to Inflammatory Stimuli</a> and <a\r\n		href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3292705/\"\r\n		rel=\"nofollow\">Querying quantitative logic models (Q2LM) to study\r\n		intracellular signaling networks and cell-cytokine interactions</a>. Based\r\n	on this approach, the user can perform pathway simulation inside\r\n	Cytoscape using rich Reactome pathways based on fuzzy logic built upon\r\n	Boolean networks.\r\n</p>\r\n<ol>\r\n	<li><b>Set up and run logic model simulation</b>: Choose popup\r\n		menu \"Run Logic Model Analysis\" in the Pathway Diagram View to get the\r\n		New Simulation dialog. Enter a name for the simulation and the default\r\n		value, which usually should be 1.0 to enable that the simulation can\r\n		proceed, and then choose either PROD or MIN for the AND gate mode (the\r\n		default choice PROD usually should be fine). <br /> <b>Note</b>: You\r\n		may also choose an Transfer Function and adjust parameters for Hill\r\n		function. However, for simplicity, it is suggested to use \"Identity\r\n		Function\" first. For how to apply drugs for logic model simulation,\r\n		see below. <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<img src=\"images/userguide/fiviz/RunBooleanNetworkAnalysis.png\" width=\"288\"\r\n					alt=\"\" />\r\n				<div>Run Boolean Network Analysis</div>\r\n			</div>\r\n		</div>\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/NewBNSimulation.png\" alt=\"\"\r\n						width=\"400\" />\r\n					<div>New BN Simulation</div>\r\n				</div>\r\n			</div>\r\n		</div> <br /> After clicking the OK button in the New Simulation dialog,\r\n		the default initial configuration will be displayed in the Results\r\n		Panel. You may change the variable Type and Modification\r\n		in the set up table by clicking the cell for the selected variable. \r\n		To run the simulation, click the Simulate button in the\r\n		Results Panel. <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/SetupBNSimulation.png\" alt=\"\"\r\n						width=\"400\" height=\"359\" />\r\n					<div>Set up Boolean Network Analysis</div>\r\n				</div>\r\n			</div>\r\n		</div>\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<img src=\"images/userguide/fiviz/ChooseBNType.png\" alt=\"\"\r\n					width=\"400\" height=\"72\" />\r\n				<div>Choose BN Variable Type</div>\r\n			</div>\r\n		</div>\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<img src=\"images/userguide/fiviz/ChooseBNModification.png\" alt=\"\"\r\n					width=\"400\" height=\"103\" />\r\n				<div>Choose BN Modification Type</div>\r\n			</div>\r\n		</div></li>\r\n	<li><b>Visualize the simulation results</b>: After the simulation\r\n		is done, entities in the pathway diagram will be highlighted based on\r\n		simulated values, which should be between 0 and 1. You may choose one\r\n		or more entities to visualize their temporal behaviors inside the\r\n		Table Panel at the bottom of Cytoscape. Attractors computed from the\r\n		simulation are also listed in the right columns in the original set up\r\n		table inside the Results Panel. <b>Note</b>: After simulation, you\r\n		will not be able to modify any initial configuration.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/BNSimulationResults_Default.png\"\r\n						alt=\"\" width=\"800\" height=\"532\" />\r\n					<div>Boolean Network Simulation Results</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n	<b>Note</b>: To avoid clutter, if too many time steps have been\r\n		generated for a logic model simulation, only the last 20 time\r\n		steps are displayed in the table. However, the plot shows all time\r\n		steps. You may choose different columns to display in the table by use\r\n		popup menu \"Configure Columns\" after selecting any variables in the\r\n		table.</li>\r\n	<li><b>Perform pathway simulation via modification</b>: Simulation\r\n		with Boolean network can help users uncover the impact of modification\r\n		of entity activities (e.g. inhibition or activation caused by somatic\r\n		mutation) on the pathway behaviors. For example, in PIP3 activates AKT\r\n		signaling, Complex AKT:PIP3 forms a complex with EntitySet THEM4/TRIB3\r\n		to form another complex, which inhibits the activation of AKT (For\r\n		details see <a href=\"/PathwayBrowser/#/R-HSA-1257604\" rel=\"nofollow\">Reactome\r\n			PIP3 activates AKT signaling</a>). To perform simulation with\r\n		modification, choose modification type in the simulation set up table\r\n		and assign the strength to the modification. <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/SetBNInhibition.png\" alt=\"\"\r\n						width=\"750\" height=\"253\" />\r\n					<div>Choose Inhibition for Boolean Network Simulation</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />Clicking the Simulate button invokes the constrained fuzzy logic model\r\n		simulation with this configured inhibition. You can compare the\r\n		simulation results between the two configurations by selecting an\r\n		entity and then toggling the simulation result tables at the bottom\r\n		of Cytoscape. <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ActiveAKT_Inhibition.png\" alt=\"\"\r\n						width=\"800\" height=\"136\" />\r\n					<div>Active AKT in Inihibition</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ActiveAKT_Default.png\" alt=\"\"\r\n						width=\"800\" height=\"136\" />\r\n					<div>Active AKT in Default</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />You can also use the \"Compare\" button in the Results Panel to\r\n		get the comparison dialog and then choose two simulations for\r\n		comparison. The comparison results are displayed in a new table listed\r\n		at the bottom Table Panel. <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/CompareTwoBNResults.png\" alt=\"\"\r\n						width=\"450\" height=\"210\" />\r\n					<div>Comparison Dialog</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/AktiveAKT_Comparison.png\" alt=\"\"\r\n						width=\"850\" />\r\n					<div>Active AKT in Comparison</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n	<b>Note:</b> The RelativeDifference in the comparison result table is\r\n		calculated based on relative change for each fuzzy logic variable,\r\n		calculated as (valueInSim2 - valueInSim1) / (valueInSim2 + valueInSim1). \r\n		The time course may be interpolated based on the attractor until this relative\r\n		difference converges.</li>\r\n	<li>To remove all displayed constrained fuzzy logic simulation results,\r\n		use popup menu, \"Remove Analysis Results\" under \"Run Logic Model\r\n		Analysis\". The pathway diagram should be reset to the original colors,\r\n		and all tables related to logic model simulations will be\r\n		deleted.</li>\r\n</ol>\r\n<h3>\r\n	<span id=\"Use_the_Reactome_Functional_Interaction_.28FI.29_Network\">Use\r\n		the Reactome Functional Interaction (FI) Network</span>\r\n</h3>\r\n<p>\r\n	After the ReactomeFIViz app installed, you should see a menu item\r\n	called \"Reactome FI\" under the Apps menu. Clicking this menu, you will\r\n	see 6 sub-menus: <a\r\n		href=\"http://wiki.reactome.org/index.php/ReactomeFIViz#Gene_Set.2FMutation_Analysis\">Gene\r\n		Set/Mutation Analysis</a>, <a\r\n		href=\"http://wiki.reactome.org/index.php/ReactomeFIViz#PGM_Impact_Analysis\">PGM\r\n		Impact Analysis</a>, <a\r\n		href=\"http://wiki.reactome.org/index.php/ReactomeFIViz#HotNet_Mutation_Analysis\">HotNet\r\n		Mutation Analysis</a>, <a\r\n		href=\"http://wiki.reactome.org/index.php/ReactomeFIViz#Microarray_Data_Analysis\">Microarray\r\n		Data Analysis</a>, <a\r\n		href=\"http://wiki.reactome.org/index.php/ReactomeFIViz#Use_Reactome_Pathways\">Reactome\r\n		Pathways</a> and User Guide. Gene set/mutation analysis is for doing FI\r\n	network-based data analysis for a set of genes or a mutation data file,\r\n	PGM Impact analysis for performing functional impact analysis based on\r\n	a probabilistic graphical model for the Reactome FI network using\r\n	multiple omics data types, HotNet mutation analysis for the HotNet\r\n	algorithm to search for network modules (see <a\r\n		href=\"http://compbio.cs.brown.edu/projects/hotnet/\" rel=\"nofollow\">http://compbio.cs.brown.edu/projects/hotnet/</a>),\r\n	microarray data analysis for doing MCL (Markov Graph Clustering, <a\r\n		href=\"http://micans.org/mcl/\" rel=\"nofollow\">http://micans.org/mcl/</a>)\r\n	based FI network clustering analysis by converting a non-weighted FI\r\n	network to weighted network using correlations among genes in the\r\n	network, Reactome pathways for loading pathways from the Reactome\r\n	database, visualizing Reactome pathways directly in Cytoscape in a\r\n	their native way, and doing pathway enrichment analysis, and user guide\r\n	brings you to this user guide.\r\n</p>\r\n<div class=\"text-center\">\r\n	<div class=\"floatnone\">\r\n		<img src=\"images/userguide/fiviz/ReactomeFIMenu_4.png\"\r\n			alt=\"ReactomeFIViz app Menu\" width=\"441\" height=\"176\" />\r\n	</div>\r\n</div>\r\n<h4>\r\n	<span id=\"Gene_Set.2FMutation_Analysis\">Gene Set/Mutation\r\n		Analysis</span>\r\n</h4>\r\n<ol>\r\n	<li>You can enter a list of genes directly into ReactomeFIViz by\r\n		clicking the \"Enter\" button, or load it from a local file. Currently\r\n		ReactomeFIViz supports three file formats for gene set/mutation\r\n		analysis:\r\n		<ol>\r\n			<li><b>Simple gene set</b>: one line per gene. For example, <a\r\n				href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/GeneList/GWASFuzzyGenes.txt\"\r\n				rel=\"nofollow\">GWASFuzzyGenes.txt</a>, a list of T2D GWAS genes.</li>\r\n			<li><b>Gene/sample number pair</b>. For example, <a\r\n				href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/GeneList/GeneSampleNumber.txt\"\r\n				rel=\"nofollow\">GeneSampleNumber.txt</a>, which contains two required\r\n				columns, gene and number of samples having gene mutated, and an\r\n				optional third column listing sample names (delimited by \";\").</li>\r\n			<li><b>NCI MAF (mutation annotation file)</b>. For example, <a\r\n				href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/GeneList/GlioblastomaMutationTable.txt\"\r\n				rel=\"nofollow\">GlioblastomaMutationTable.txt</a>, the mutation file\r\n				from the TCGA GBM project.</li>\r\n		</ol>\r\n	</li>\r\n	<li>Choose a FI network version from listed three versions. <br />\r\n	<b>Note</b>: you may get different results using different FI network\r\n		versions because a later version may contain more proteins/genes and\r\n		more FIs. But based on our experience, a significant FI network module\r\n		is usually stable across multiple versions.\r\n	</li>\r\n	<li>Enter genes directly by clicking the \"Enter\" button or choose\r\n		a file containing genes you want to use to construct a functional\r\n		interaction network. To choose a file, select an appropriate file\r\n		format and parameters to load genes and construct FI network in the\r\n		dialog. Click the \"OK\" button to start the FI network building\r\n		process.\r\n		<div class=\"text-center\">\r\n			<div class=\"floatnone\">\r\n				<img src=\"images/userguide/fiviz/GeneSetAnalysis_4.png\"\r\n					alt=\"Gene Set/Mutation Analysis\" width=\"496\" />\r\n			</div>\r\n		</div>\r\n	</li>\r\n	<li>The constructed FI network will be displayed in the network\r\n		view panel. A FI specific visual style will be created automatically\r\n		for the FI network.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/FISubNetwork.png\" alt=\"\"\r\n						width=\"470\" height=\"375\" />\r\n					<div>Reactome FI Sub-Network</div>\r\n				</div>\r\n			</div>\r\n		</div>\r\n	</li>\r\n	<li>The main features of Reactome FI plug-in should be invoked\r\n		from a popup menu, which can be displayed by right clicking an empty\r\n		space in the network view panel.\r\n		<div class=\"text-center\">\r\n			<div class=\"floatnone\">\r\n				<img src=\"images/userguide/fiviz/PopupMenu_4.png\"\r\n					alt=\"Popup Menu for Network\" width=\"526\" height=\"239\" />\r\n			</div>\r\n		</div>\r\n	</li>\r\n</ol>\r\n<ul>\r\n	<li><b>Fetch FI annotations</b>: query detailed information on\r\n		selected FIs. Three FI related edge attribues will be created: FI\r\n		Annotation, FI Direction, and FI Score. Edges will be displayed based\r\n		on FI direction attribute values. In the following screenshot, \"-&gt;\"\r\n		for activating/catalyzing, \"-|\" for inhibition, \"-\" for FIs extracted\r\n		from complexes or inputs, and \"---\" for predicted FIs. See the\r\n		\"VizMapper\" tab, Edge Source Arrow Shape and Edge Target Arrow Shape\r\n		values for details.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/FIAnnotations.png\" alt=\"\"\r\n						width=\"515\" height=\"508\" />\r\n					<div>FI Annotations</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n	<b>Note</b>: Here is a short explanation about displayed columns:\r\n		GeneSet for pathways collected in the Reactome FI network hit by the\r\n		query gene list; RatioOfProteinInGeneSet for ratios of numbers of\r\n		genes contained in pathways to total genes in the Reactome FI network;\r\n		NumberOfProteinInGeneSet for numbers of genes in pathways;\r\n		ProteinFromNetwork for numbers of hit genes from the query gene list;\r\n		P-value for pvalues calculated based on binomial test; FDR for FDRs\r\n		calculated based on p-values using Benjamini-Hocherberg method; Nodes\r\n		for hit genes in pathways.</li>\r\n	<li><b>Analyze network functions</b>: pathway or GO term\r\n		ennrichment analysis for the displayed network. You can choose to\r\n		filter enrichment results by a FDR cutoff value. Also you can choose\r\n		to display nodes in the network panel for a selected row or rows by\r\n		checking \"Hide nodes in not selected rows\". The letter in parentheses\r\n		after each pathway gene set name corresponds to the source of the\r\n		pathway annotations: C - CellMap, R  Reactome, K  KEGG, N  NCI PID,\r\n		P - Panther, and B  BioCarta. The following screenshot shows results\r\n		from a pathway enrichment analysis.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/PathwayAnnotations.png\" alt=\"\"\r\n						width=\"799\" height=\"251\" />\r\n					<div>Pathways in FI Sub-Network</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n	<i>Tip: To analyze pathway or GO term enrichment on a set of genes\r\n			that are not linked together, select the \"Show genes not linked to\r\n			others\" option in the \"Set Parameters for FI Network\" dialog.</i></li>\r\n	<li><b>Cluster FI network</b>: run a network clustering algorithm\r\n		(spectral partition based network clustering by <a\r\n		href=\"http://www.ncbi.nlm.nih.gov/pubmed/16723398\" rel=\"nofollow\">Newman\r\n			2006</a>) on the displayed FI network. Nodes in different network modules\r\n		will be shown in different colors (different colors used only for\r\n		first 15 modules based on sizes).\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/NetworkModules.png\" alt=\"\"\r\n						width=\"679\" height=\"421\" />\r\n					<div>Network Modules</div>\r\n				</div>\r\n			</div>\r\n		</div></li>\r\n	<li><b>Analyze module functions</b>: pathway or GO term enrichment\r\n		analysis for each individual network modules. You can select a size\r\n		cutoff to filter out network modules that are too small, choose a FDR\r\n		cutoff to view enriched pathways or GO terms under a certain FDR\r\n		value, and view nodes in a selected row or rows only in the network\r\n		diagram.</li>\r\n	<li><b>Analyze functions for a set of selected genes</b>: select a\r\n		set of nodes displayed in the network view and then choose the popup\r\n		menu, Analyze Nodes Functions, to perform pathway or GO term\r\n		enrichment analysis. The results will be displayed in a dialog.</li>\r\n	<li><b>Load Cancer Gene Index</b>: load cancer gene index\r\n		annotations. For details, see section <a\r\n		href=\"#Load_Cancer_Gene_Index_Annotations\">Load Cancer Gene Index</a>.</li>\r\n</ul>\r\n<h4>\r\n	<span id=\"PGM_Impact_Analysis\">PGM Impact Analysis</span>\r\n</h4>\r\n<ol>\r\n	<li>We have developed a probabilistic graphical model (PGM)-based\r\n		functional impact analysis using the Reactome FI network by\r\n		integrating multiple omics data types together. The current version of\r\n		ReactomeFIViz supports four omics data types: CNV, mRNA expression,\r\n		DNA methylation, and somatic mutation. PGMs used for this analysis are\r\n		based on <a href=\"https://en.wikipedia.org/wiki/Markov_random_field\"\r\n		rel=\"nofollow\">Markov random field (MRF)</a>. Currently we support two\r\n		types of MRFs: <a href=\"https://class.coursera.org/pgm/lecture/31\"\r\n		rel=\"nofollow\">Pairwise MRF</a> and <a\r\n		href=\"http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1001353\"\r\n		rel=\"nofollow\">Nearest neighbor Gibbs MRF</a>. You can choose one of\r\n		these two models. We recommend pair-wise MRF for its simplicity.\r\n	</li>\r\n	<li>To perform PGM-based functional impact analysis, choose menu,\r\n		Apps/Rectome FI/PGM Impact Analysis/Analyze. You can enter your omics\r\n		data by using the PGM configuration dialog.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ReactomeFIPGMInput.png\" alt=\"\"\r\n						width=\"786\" />\r\n					<div>FI-PGM Configuration Dialog</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n	<b>Notes</b>:<br />1). ReactomeFIViz supports continuous observation\r\n		variables without discretizing by choosing \"Use empirical\r\n		distribution\" in the configuration. For somatic mutation, currently it\r\n		supports NCI MAF file format only and requires a specific column named\r\n		\"MA_FI.score\" for mutation function impact score collected from <a\r\n		href=\"http://mutationassessor.org/\" rel=\"nofollow\">Mutation\r\n			Assessor</a> or from some other sources. <br />2). The default MRF model\r\n		used is PairwiseMRF. You can choose the default setting for the first\r\n		test. <br />3). Several parameters are needed for MRF models. We have\r\n		tuned these parameters based on a small toy model. In the current\r\n		version of ReactomeFIViz, these parameters cannot be changed.\r\n	</li>\r\n	<li>It may take several hours to finish the whole analysis. The\r\n		actual running time will be dependent on the size of your data. The\r\n		progress of the job running is displayed in the following progress\r\n		pane. You can cancel the running at any time.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ProgressPaneOfFIPGM.png\" alt=\"\"\r\n						width=\"296\" height=\"197\" />\r\n					<div>Progress of FI PGM Running</div>\r\n				</div>\r\n			</div>\r\n		</div>\r\n	</li>\r\n	<li>After the analysis is finished, you should see the result\r\n		dialog similar to the following screenshot. You can use filtering\r\n		features to filter to a list of genes that you want to use to\r\n		construct a FI subnetwork for further analysis.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/FIPGMResultDialog.png\" alt=\"\"\r\n						width=\"799\" height=\"590\" />\r\n					<div>FI-PGM Result Dialog</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n	<b>Note</b>: If you select one or more genes in the result table, only\r\n		these selected genes will be used to construct a FI sub-network. You\r\n		can save the full analysis results by clicking the \"Save\" button\r\n		(Results for all genes, not just displayed ones, will be saved). We\r\n		strongly recommend to save your results first before clicking the \"OK\"\r\n		button so that you can visit your results back.\r\n	</li>\r\n	<li>After choosing the \"OK\" button, a FI subnetwork will be\r\n		constructed and displayed in a network view. The sizes of displayed\r\n		nodes are proportional to impact scores inferred from the FI-PGM\r\n		model. To view impact scores and loaded observation data, you can\r\n		choose a sample in the Sample List tab in the Results Panel. You can\r\n		also enable sample-based network visualization of the network by\r\n		checking \"Highlight network for sample\" in the Sample List tab. If you\r\n		want to review the original results used to construct the FI\r\n		subnetwork, click \"Show All Results\" in the \"Impact Gene Values\" tab\r\n		in \"Table Panel\".\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/FIPGMResultNetwork.png\" alt=\"\"\r\n						width=\"799\" height=\"670\" />\r\n					<div>FI-PGM Result Network</div>\r\n				</div>\r\n			</div>\r\n		</div>\r\n	</li>\r\n</ol>\r\n<h4>\r\n	<span id=\"HotNet_Mutation_Analysis\">HotNet Mutation Analysis</span>\r\n</h4>\r\n<p>\r\n	Reactome FI Cytoscape plug-in implements the algorithm developed by\r\n	Raphael\'s group at Brown University, called \"HotNet\", for doing cancer\r\n	mutation data analysis. For details about this algorithm, please see <a\r\n		href=\"http://www.ncbi.nlm.nih.gov/pubmed/21385051\" rel=\"nofollow\">Algorithms\r\n		for detecting significantly mutated pathways in cancer</a>, and <a\r\n		href=\"http://www.ncbi.nlm.nih.gov/pubmed/22174262\" rel=\"nofollow\">Discovery\r\n		of mutated subnetworks associated with clinical data in cancer</a>.\r\n</p>\r\n<ol>\r\n	<li><b>Select a mutation data file and run HotNet algorithm</b>:\r\n		After selecting sub-menu \"HotNet Mutation Analysis\" from menu\r\n		Plugins/Reactome FIs, you would see the following dialog. Choose a\r\n		version of FI Network, a mutation file from your local file system,\r\n		and set parameters required by the HotNet algorithm. Currently the\r\n		plug-in supports the NCI MAF mutation file only. We are going to\r\n		support more file formats in the future. If you are not sure what\r\n		delta value should be used, you may choose \"Auto\" in the dialog.\r\n		However, using \"Auto\" takes much longer time to run the algorithm.\r\n		Random permutation is used to calculate p-values and FDR values. The\r\n		largest number of permutation is 1000. For details about permutation,\r\n		please see the above two papers. After entering all required\r\n		parameters, click the \"OK\" button to start HotNet analysis. It may\r\n		take several minutes. If you choose \"Auto\" for delta, it takes even\r\n		longer time. For a test run, you may use the TCGA GBM mutation file, <a\r\n		href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/GeneList/GlioblastomaMutationTable.txt\"\r\n		rel=\"nofollow\">GlioblastomaMutationTable.txt</a>, and choose the 2012\r\n		version of FI Network with delta 1.0e-4.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/HotNetMutation_4.png\" alt=\"\"\r\n						width=\"497\"/>\r\n					<div>Set Parameters for HotNet Mutation Analysis</div>\r\n				</div>\r\n			</div>\r\n		</div></li>\r\n	<li><b>Select network modules and build a FI sub-network</b>: The\r\n		generated FI network modules from the HotNet analysis are listed in\r\n		the HotNet result dialog (see below). In the dialog, you can choose a\r\n		size cutoff, or a FDR cutoff. The displayed selected network modules\r\n		will be used to build a FI sub-network after you click the \"OK\"\r\n		button. In the dialog, you can also see the chosen delta value and the\r\n		number of permutations. You may try different delta values for better\r\n		results.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/HotNetResults.png\" alt=\"\"\r\n						width=\"711\" height=\"558\" />\r\n					<div>HotNet Mutation Analysis Results</div>\r\n				</div>\r\n			</div>\r\n		</div></li>\r\n</ol>\r\n<h4>\r\n	<span id=\"Microarray_Data_Analysis\">Microarray Data Analysis</span>\r\n</h4>\r\n<p>\r\n	The ReactomeFIViz app can load gene expression data file, calculate\r\n	correlations among genes involved in the same FIs, use the calculated\r\n	correlations as weights for edges (i.e. FIs) in the whole FI network,\r\n	apply MCL graph clustering algorithm to the weighted FI network, and\r\n	generate a sub-network for a list of selected network modules based on\r\n	module size and average correlation. The generated FI sub-network will\r\n	be displayed in the network panel, and can be used for analysis as in\r\n	Gene Set/Mutation Analysis. For details about this method, please see\r\n	our publication: <a href=\"http://genomebiology.com/2012/13/12/R112\"\r\n		rel=\"nofollow\">A network module-based method for identifying\r\n		cancer prognostic signatures</a>.\r\n</p>\r\n<p>\r\n	An array data file should be a tab-delimited text file with table\r\n	headers. The first column should be gene names. All other columns\r\n	should be expression values in different samples. <b>The data set\r\n		in the file should be pre-normalized.</b> For example, see this gene\r\n	expression file for breast cancer: <a\r\n		href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/GeneExpression/NejmLogRatioNormGlobalZScore_070111.txt.zip\"\r\n		rel=\"nofollow\">NejmLogRatioNormGlobalZScore_070111.txt.zip</a>. This\r\n	data set was download from <a\r\n		href=\"http://www.nejm.org/doi/full/10.1056/NEJMoa021967\"\r\n		rel=\"nofollow\">van de Vijver et al in 2002</a>, and has been\r\n	normalized.\r\n</p>\r\n<ol>\r\n	<li><b>Select a microarray data file and run MCL network\r\n			clustering</b>: After selecting sub-menu \"Microarray Data Analysis\" from\r\n		menu Plugins/Reactome FIs, you should see the following dialog. Choose\r\n		a microarray data file, check if you want to use absolute values as\r\n		weights for edges, and input an inflation parameter (-I) for the MCL\r\n		clustering algorithm. The smaller the inflation parameter is, the\r\n		bigger the average size of generated network modules. Based on our own\r\n		experience, we use 5.0 for the inflation parameter, the highest\r\n		recommended value, and choose the absolute value for edge weights. For\r\n		more details on how to choose the inflation parameter, please see <a\r\n		href=\"http://micans.org/mcl/\" rel=\"nofollow\">http://micans.org/mcl/</a>.\r\n		After you have set these parameters, click the OK button to load the\r\n		data file, calculate correlations, and apply the MCL clustering\r\n		algorithm.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/MicroarrayAnalysis_4.png\" alt=\"\"\r\n						width=\"498\" />\r\n					<div>Set Parameters for Microarray Data Analysis</div>\r\n				</div>\r\n			</div>\r\n		</div></li>\r\n	<li><b>Select network modules and build a FI sub-network</b>: The\r\n		generated network modules are listed in the MCL clustering results\r\n		dialog (see below). Only modules having more than 2 genes can be\r\n		listed, and used in the FI sub-network building. You can choose a\r\n		module size or an average correlation value (absolute value if\r\n		absolute has been checked before) to filter out modules that may not\r\n		be significant (Note: after set these cutoff values, please press the\r\n		\"Enter\" key to commit your changes.). In our analysis, we choose\r\n		modules having 7 or more genes with average correlation values no less\r\n		than 0.25. These values have been used as default in the dialog. In\r\n		the dialog, you can see how many modules and genes will be chosen for\r\n		building FI sub-network under your selected filter values. Click the\r\n		OK button to start the sub-network building. The built sub-network\r\n		will be displayed, and can be analyzed as with sub-networks generated\r\n		from the gene set/mutation analysis.\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/MCLClusteringResultsDialog.png\"\r\n						alt=\"\" width=\"705\" height=\"508\" />\r\n					<div>Choose MCL Modules</div>\r\n				</div>\r\n			</div>\r\n		</div></li>\r\n</ol>\r\n\r\n<h3>\r\n	<span\r\n		id=\"Visualize_Cancer_Drugs_in_Contexts_of_Reactome_Pathways_and_FI_Network\">Visualize\r\n		Drugs in the Contexts of Reactome Pathways and FI Network</span>\r\n</h3>\r\n<p>\r\n	ReactomeFIViz provide a suite of features to assist users to visualize\r\n	drugs in the contexts of Reactome pathways and networks. The drug data\r\n	sources include two: Cancer Targetome (<a\r\n		href=\"https://www.ncbi.nlm.nih.gov/pubmed/28964549\">Blucher et al\r\n		2017</a>), which&nbsp;collected all FDA-approved cancer drugs (prior to\r\n	2018) and their target interactions from four sources, including\r\n	DrugBank, Therapeutic Targets Database, IUPHAR, and BindingDB; <a\r\n		href=\"http://drugcentral.org\">DrugCentral</a>, a comprehensive drug\r\n	database supported by <a href=\"https://ncats.nih.gov/idg\">NIH IDG\r\n		program</a>.&nbsp;\r\n</p>\r\n<h4>\r\n	<span id=\"Visualize_Cancer_Drugs_in_Reactome_Pathways\">Visualize\r\n		Drugs in Reactome Pathways</span>\r\n</h4>\r\n<ol>\r\n	<li><b>List all FDA approved cancer drugs</b>: Use popup menu\r\n		\"View Cancer Drugs\" in the Reactome pathway tree to get the list of\r\n		all FDA approved cancer drugs collected in Cancer Targetome in a table\r\n		dialog. <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ViewCancerDrugs.png\"\r\n						alt=\"ViewCancerDrugs\" width=\"400\" height=\"444\" />\r\n					<div>View Cancer Drugs</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />A screenshot of this drug list is shown below:<br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/CancerDrugsTable.png\" alt=\"\"\r\n						width=\"800\" height=\"417\" />\r\n					<div>List of Cancer Drugs</div>\r\n				</div>\r\n			</div>\r\n		</div></li>\r\n	<li><b>View drug/target interactions</b>: Select a drug in the\r\n		drug table (See Figure above) to perform Google search, or view its\r\n		targets by clicking one of buttons in the dialog. Two views are\r\n		provided for drug/target interactions. Drug targets table view (below)\r\n		shows targets for the selected drug with collected binding affinities,\r\n		which are divided into different categories (e.g.): KD, IC50, Ki, and\r\n		EC50. The displayed targets are pre-filtered using a set of default\r\n		filters. You may change the default filters in the Drug/Target\r\n		Interaction Filter dialog by clicking the \"Filter\" button.<br />\r\n		<div class=\"text-center\">\r\n			<img src=\"images/userguide/fiviz/DrugTargetsView.png\"\r\n				alt=\"DrugTargetsTableView\" width=\"600\" height=\"300\" />\r\n			<div>Drug Targets Table View</div>\r\n		</div>\r\n		<div></div>\r\n		<div class=\"text-center\">\r\n			<img src=\"images/userguide/fiviz/DrugTargetInteractionFilter.png\"\r\n				alt=\"\" width=\"350\" height=\"316\" />\r\n			<div>Drug Target Interaction Filter</div>\r\n			<div></div>\r\n		</div>\r\n		<div>\r\n			In the Drug targets table view, you can choose an interaction and\r\n			then click the \"View Details\" button to view the detailed information\r\n			for the selected interaction. In the details view, you can browse the\r\n			original pubmed references for experimental evidence, Gene Card entry\r\n			for target, or Google drug by clicking the hyper links.\r\n			<div class=\"text-center\">\r\n				<img src=\"images/userguide/fiviz/CancerDrugDetailsView.png\" alt=\"\"\r\n					width=\"450\" height=\"377\" />\r\n				<div>Cancer Drug Interaction Details View</div>\r\n			</div>\r\n		</div>\r\n		<div></div>\r\n		<div>\r\n			You can also select multiple rows in the table to perform pathway\r\n			enrichment analysis by clicking \"Overlay Targets to Pathways\". See <a\r\n				href=\"http://wiki.reactome.org/index.php/ReactomeFIViz#Pathway_Enrichment_Analysis\">Pathway\r\n				Enrichment Analysis</a> for details about pathway enrichment analysis\r\n			results.\r\n		</div>\r\n		<div></div>\r\n		<div>Drug targets plot view (below) provides a stagged plot of\r\n			bar charts for evidence-supported interactions between the selected\r\n			drug and all its targets, suggesting primary target(s) and secondary\r\n			targets in a single graphical view.</div>\r\n		<div class=\"text-center\">\r\n			<img src=\"images/userguide/fiviz/DrugTargetsPlotView.png\"\r\n				alt=\"DrugTargetsPlotView\" width=\"600\" />\r\n			<div>Drug Targets Plot View</div>\r\n		</div>\r\n		<div></div></li>\r\n	<li>\r\n		<div>\r\n			<b>View cancer drugs in pathway diagrams</b>: Cancer drugs can be\r\n			overlaid directly onto displayed pathway diagrams using the popup\r\n			menu, \"Fetch Cancer Drugs\", in the <a\r\n				href=\"http://wiki.reactome.org/index.php/ReactomeFIViz#Explore_Reactome_Pathways\">Pathway\r\n				Diagram View</a>. The fetched cancer drugs are displayed in the pathway\r\n			diagram, linked to entities containing targets for the displayed\r\n			drugs.\r\n		</div>\r\n		<div class=\"text-center\">\r\n			<img src=\"images/userguide/fiviz/FetchCancerDrugs.png\" alt=\"\"\r\n				width=\"300\" />\r\n			<div>Fetch Cancer Drugs</div>\r\n		</div>\r\n		<div></div>\r\n		<div class=\"text-center\">\r\n			<img src=\"images/userguide/fiviz/DrugsInPathway.png\" alt=\"\"\r\n				width=\"450\" />\r\n			<div>Drugs in Pathway Diagram</div>\r\n		</div>\r\n		<div></div>\r\n		<div>\r\n			<b>Note:</b> Use Popup menu \"Filter Drugs\" to adjust filters to\r\n			select drugs and interactions in the pathway diagram view; Use\r\n			\"Remove Overlaid Interactions\" to remove displayed drugs; You may\r\n			choose a displayed entity in the pathway diagram and then use popup\r\n			menu \"Fetch Cancer Drugs\" to display drugs targeted to the selected\r\n			entity only.&nbsp;\r\n		</div> <br />\r\n		<div>To view the detailed information about the interaction\r\n			between the displayed drug and its targeted entity, choose the link\r\n			and use pupup menu \"Show Details\" to bring up the drug targets\r\n			view.&nbsp;</div>\r\n		<div class=\"text-center\">\r\n			<img src=\"images/userguide/fiviz/ViewDrugDetailsInPathway.png\" alt=\"\"\r\n				width=\"400\" height=\"169\" />\r\n			<div>View Details</div>\r\n		</div>\r\n	</li>\r\n	<li>\r\n		<div>\r\n			<b>Perform systems pathway impact analysis</b>: In the cancer drug\r\n			list table (see above), you may perform pathway impact analysis for\r\n			all Reactome pathways having entity level view drawn by choosing the\r\n			\"Run Pathway Impact Analysis\" button. After the analysis is done, the\r\n			impact results are displayed in the table labled with the selected\r\n			drug in the Cytoscape Table Panel. See below for an example of the\r\n			results for Imatinib:\r\n		</div>\r\n		<div class=\"text-center\">\r\n			<img src=\"images/userguide/fiviz/PathwayImpactAnalysisTable.png\"\r\n				alt=\"PathwayImpactAnalysisTable\" width=\"700\" />\r\n			<div>Pathway Impact Analysis Results</div>\r\n		</div>\r\n		<div>\r\n			<b>Note</b>: You may open the pathway diagram using popup menu \"View\r\n			in Diagram\" and export the table into a text file using menu \"Export\r\n			Table\" in the results table.\r\n		</div>\r\n	</li>\r\n</ol>\r\n<h4>\r\n	<span id=\"Visualize_Cancer_Drugs_in_the_FI_Network\">Visualize\r\n		Cancer Drugs in the FI Network</span>\r\n</h4>\r\n<p>\r\n	The Reactome FI network provides a network-based view among\r\n	proteins/genes, where each gene/protein is displayed only once.\r\n	Visualizing cancer drugs in a FI network context shows a simplified\r\n	relationship between cancer drugs and their targets. In the <a\r\n		href=\"http://wiki.reactome.org/index.php/ReactomeFIViz#Gene_Set.2FMutation_Analysis\">FI\r\n		Network View</a>, use popup menu, \"Reactome FI/Overlay Cancer Drugs/Fetch\r\n	Cancer Drugs\", to load cancer drugs for proteins/genes displayed in the\r\n	FI network. The loaded drugs and interactions between drugs and\r\n	proteins/genes are rendered in green diamonds and blue edges,\r\n	respectively.\r\n</p>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<img src=\"images/userguide/fiviz/FetchCancerDrugsInFINetwork.png\"\r\n			alt=\"\" width=\"400\" height=\"247\" />\r\n		<div>Fetch Cancer Drugs in FI Network</div>\r\n	</div>\r\n</div>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img src=\"images/userguide/fiviz/ShowDrugInteractionDetailsInFI.png\"\r\n				alt=\"\" width=\"400\" height=\"265\" />\r\n			<div>Show Drug Target In FI Network</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<p>\r\n	<br />\r\n	<b>Note</b>: To remove overlaid drugs in the FI network view, use popup\r\n	menu, \"Reactome FI/Overlay Cancer Drugs/Remove Drugs\"; As in the\r\n	Pathway diagram view, you can also apply filters by using \"Filte Drugs\"\r\n	popup menu; To view the details about an interaction between a drug and\r\n	a target, select the edge, and then use popup menu \"Reactome FI/Show\r\n	Drug/Target Interaction Details\". (You may need to zoom into the\r\n	selected edge.)\r\n</p>\r\n<h4>\r\n	<span id=\"Visualize_DrugCentral_Drugs\">Visualize DrugCentral\r\n		Drugs</span>\r\n</h4>\r\n<p>Visualize drugs collected in DrugCentral is simlar with cancer\r\n	drugs collected in Cancer Targetome. You can use popup menu \"View\r\n	DrugCentral Drugs\" in the pathway tree to list all drugs collected in\r\n	the DrugCentral database, \"Fetch DrugCentral Drugs\" popup menu in the\r\n	pathway diagram view to overlay drugs onto pathways, or\r\n	\"ReactomeFI/Overlay Drugs/Fetch DrugCentral Drugs\" to overlay drugs\r\n	onto the FI network in the network view.</p>\r\n<h4>\r\n	<span id=\"Simulate_Impact_of_Cancer_Drugs_on_Pathway_Activities\">Simulate\r\n		Impact of Drugs on Pathway Activities</span>\r\n</h4>\r\n<p>\r\n	Overlaying cancer drugs onto the contexts of Reactome pathways and its\r\n	FI network helps users to understand the potential impact of applying\r\n	drugs on the pathway activities and network behavior. However, the\r\n	actual perturbation of drugs on pathways may be much more complicated.\r\n	Performing pathway simulation may help users to understand the actual\r\n	impact. ReactomeFIViz implements features to assist users to perform\r\n	Boolean network-based drug simulation. Before you do drug simulation,\r\n	please read section <a\r\n		href=\"#Boolean_Network_based_Pathway_Analysis\">Boolean\r\n		Network based Pathway Analysis</a> first. In this section, we will use\r\n	pathway <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-5693567\"\r\n		rel=\"nofollow\">HDR through Homologous Recombination (HR) or Single\r\n		Strand Annealing (SSA)</a> and drug Imatinib as an example (Note: To get\r\n	the following screenshot, open diagram for this pathway, and then fetch\r\n	cancer drugs and filter drugs to Imatinib using the name filter).\r\n</p>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img src=\"images/userguide/fiviz/ImatinibInHDRThroughHROrSSA.png\"\r\n				alt=\"\" width=\"600\" height=\"276\" />\r\n			<div>Imatinib in Pathway</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<ol>\r\n	<li><b>Set up new simulation for drug</b>: In the pathway diagram\r\n		view, use popup menu, \"Run Logic Model Analysis\". In the New\r\n		Simulation dialog, enter a name (e.g. Imtatinib) and default value for\r\n		the simulation, and then choose the mode for the AND gate as in the\r\n		regular logic model simulation. To perform drug simulation, choose\r\n		the Drug Application tab and a drug data source, and then click the \"...\" \r\n		button to bring up the Drug Selection dialog. <b><br />Note</b>:\r\n		You will need to adjust the drug filters to show all targets for\r\n		Imatinib as displayed in the following screenshot. Based on the\r\n		collected annotations for interactions between cancer drugs and their\r\n		targets, default modification types will be selected (as expected,\r\n		most of them are inhibition). Strengths of modifications are\r\n		pre-configured based on affinities collected in our aggregated\r\n		drug/target database. <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<img src=\"images/userguide/fiviz/ImatinibNewSimulation_1.png\" alt=\"\"\r\n					width=\"400\" />\r\n				<div>Imatinib BN Simulation I</div>\r\n			</div>\r\n		</div>\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ImatinibNewSimulation_2.png\"\r\n						alt=\"\" width=\"400\" />\r\n					<div>Imatinib BN Simulation II</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/SelectDrugsForBN.png\" alt=\"\"\r\n						width=\"750\" height=\"291\" />\r\n					<div>Select Drugs for BN Simulation</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n	<b>Note:</b> Since no affinities can be found for interactions between\r\n		Imatinib and CHEK1 or CDK2, CHEK1 and CDK2 are not listed in the New\r\n		Simulation dialog. Checking \"Filter members in sets to drug targets\"\r\n		will select members in EntitySet instances that are targeted by drugs\r\n		to force showing of potential pathway impact caused by drugs.</li>\r\n	<li><b>Perform simulation with drug</b>: The configuration for\r\n		drug in the Drug Selection dialog will be copied into the Boolean\r\n		Network configuration table displayed in the Results Panel. Click the\r\n		\"Simulate\" button to perform simulation. After the simulation is done,\r\n		Entities in the pathway diagrams are highlighted in different colors\r\n		based on values in the attractor with detailed temporal values\r\n		displayed in the table at the bottom Table Panel. <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ImatinibBNSetup.png\" alt=\"\"\r\n						width=\"400\" height=\"339\" />\r\n					<div>Imatinib BN Setup</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ImatinibBNResults.png\" alt=\"\"\r\n						width=\"850\" height=\"569\" />\r\n					<div>Imatinib BN Results</div>\r\n				</div>\r\n			</div>\r\n		</div></li>\r\n	<li><b>Investigate the drug impact on pathway activities</b>: To\r\n		see the impact of a drug on the pathway activities, perform another\r\n		Boolean network simulation without applying cancer drugs (here as\r\n		Default) (For details, see <a\r\n		href=\"#Boolean_Network_based_Pathway_Analysis\">Boolean\r\n			Network based Pathway Analysis</a>). The screenshot for the logic model\r\n		simulation results with the default initial configuration for\r\n		pathway \"HDR through Homologous Recombination (HR) or Single Strand\r\n		Annealing (SSA)\" is displayed below: <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img\r\n						src=\"images/userguide/fiviz/DefaultHDRThroughHROrSSABNResults.png\"\r\n						alt=\"\" width=\"850\" height=\"567\" />\r\n					<div>HDR Through HR or SSA Default Results</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />To see the drug impact to the activity of an entity displayed\r\n		in the pathway, choose that entity and check its temporal behavior in\r\n		both BN:Default and BN:Imatinib tables. For example, below a complex\r\n		(see above screenshot) related to ABL1 is selected (Up for imatinib\r\n		applied and down for default without drug): <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/ImatinibOneVariable.png\" alt=\"\"\r\n						width=\"800\" height=\"165\" />\r\n					<div>Imatinib One Variable</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/DefaultBNOneVariable.png\" alt=\"\"\r\n						width=\"800\" height=\"171\" />\r\n					<div>Default One Variable</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />You may also use the \"Compare\" button to check the detailed\r\n		difference in the computed attractors from two simulations. <br />\r\n		<div class=\"text-center\">\r\n			<div>\r\n				<div>\r\n					<img src=\"images/userguide/fiviz/PS456ABL1Comparison.png\" alt=\"\"\r\n						width=\"800\" height=\"203\" />\r\n					<div>p-S456-ABL1 Values</div>\r\n				</div>\r\n			</div>\r\n		</div> <br />\r\n	<b>Note</b>: From the above comparison, we can see that application of\r\n		imatinib will significantly impact the formation of the complex in\r\n		HDR, an effect of imanitib on DNA repair pathway has been reported by\r\n		others (e.g. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/15840387\"\r\n		rel=\"nofollow\">Imatinib (STI571) induces DNA damage in\r\n			BCR/ABL-expressing leukemic cells but not in normal lymphocytes</a>).\r\n		You may see a little bit different simulation results because of update\r\n		in pathway annotations in new versions of ReactomeFIViz.</li>\r\n</ol>\r\n<h3>\r\n	<span id=\"Other_Features_Related_to_the_FI_Network\">Other\r\n		Features Related to the FI Network</span>\r\n</h3>\r\n<h4>\r\n	<span id=\"Query_FI_Source\">Query FI Source</span>\r\n</h4>\r\n<p>Select an edge and right click it to get the popup menu for edge.\r\n	Select a menu called \"Reactome FI/Query FI Source\". If a FI is\r\n	extracted from curated pathways or reactions, a dialog for the original\r\n	data source(s) will be displayed. Double click a row in the displayed\r\n	table to show a detailed web page for the source of the FI. If the\r\n	selected FI is a predicted one, the evidence for this FI should be\r\n	displayed.</p>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<img src=\"images/userguide/fiviz/QueryFISource.png\" alt=\"\" width=\"400\"\r\n			height=\"150\" />\r\n		<div>Query FI Source</div>\r\n	</div>\r\n</div>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img src=\"images/userguide/fiviz/ShowFISource.png\" alt=\"\" width=\"350\"\r\n				height=\"177\" />\r\n			<div>ReactomeFIViz app Menu</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<p>&nbsp;</p>\r\n<h4>\r\n	<span id=\"Fetch_FIs_for_Node\">Fetch FIs for Node</span>\r\n</h4>\r\n<p>All FIs for a node can be queried. Select a node in the network\r\n	panel, and right click it to get the popup menu for node. Select a menu\r\n	called \"Reactome FI/Fetch FIs\". FI partners for the selected node will\r\n	be displayed in two sections: partners have been displayed in the\r\n	network and partners not displayed in the network. You can select\r\n	partners from the second sections to expand the displayed network.</p>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<img src=\"images/userguide/fiviz/FetchFIs.png\" alt=\"\" width=\"350\"\r\n			height=\"238\" />\r\n		<div>Query Node FIs</div>\r\n	</div>\r\n</div>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img src=\"images/userguide/fiviz/ShowNodeFIs.png\" alt=\"\" width=\"398\"\r\n				height=\"276\" />\r\n			<div>Show Node FIs</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<p>&nbsp;</p>\r\n<h4>\r\n	<span id=\"Show_Pathway_Diagram\">Show Pathway Diagram</span>\r\n</h4>\r\n<p>Pathway diagrams can be shown for pathway hits. Select a pathway\r\n	in the \"Pathways in Network\" or \"Pathways in Modules\" tab, and right\r\n	click to get the popup menu for pathway. Select \"Show Pathway Diagram\"\r\n	from the popup menu</p>\r\n<div>\r\n	<div>\r\n		<img src=\"images/userguide/fiviz/ShowPathwayDiagram.png\" alt=\"\"\r\n			width=\"340\" height=\"102\" />\r\n		<div>Show Pathway Diagram</div>\r\n	</div>\r\n</div>\r\n<p>. If pathways are imported from KEGG, KEGG pathway diagram pages\r\n	will be shown in a browser with node genes listed in the \"Nodes\" column\r\n	highlighted in red (for text and borders in pathway diagrams). If\r\n	pathways are from Reactome or other non-KEGG databases, pathway\r\n	diagrams should be shown in a separated window. If pathways are curated\r\n	by the Reactome project, human laid-out diagrams should be displayed if\r\n	any. Otherwise, auto-laid-out diagrams should be displayed. Genes or\r\n	proteins from the displayed network should be highlighted in blue.\r\n	Detailed annotations for nodes and reactions displayed in the diagram\r\n	window can be viewed by using a popup menu called \"View Instance\".\r\n	Diagrams displayed can be zoomed in/out using the zoom slider at the\r\n	bottom of the window. The diagram can be panned by the overview window\r\n	at the top-right corner.</p>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img src=\"images/userguide/fiviz/KEGGDiagram.png\" alt=\"\" width=\"610\"\r\n				height=\"377\" />\r\n			<div>KEGG Focal Adhesion</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img src=\"images/userguide/fiviz/ReactomeDiagram.png\" alt=\"\"\r\n				width=\"610\" height=\"422\" />\r\n			<div>Reactome Signaling by PDGF</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<p>&nbsp;</p>\r\n<h4>\r\n	<span id=\"Load_Cancer_Gene_Index_Annotations\">Load Cancer Gene\r\n		Index Annotations</span>\r\n</h4>\r\n<p>\r\n	Reactome FI plug-in can load NCI cancer <a\r\n		href=\"https://wiki.nci.nih.gov/display/cageneindex/Cancer+Gene+Index+End+User+Documentation\"\r\n		rel=\"nofollow\">gene index annotations</a> for genes/proteins displayed\r\n	in the network. There are two ways to show these annotations: use a\r\n	popup menu called \"Load Cancer Gene Index\" when no object is selected\r\n	(left figure), and use another popup menu \"Fetch Cancer Gene Index\" for\r\n	a selected node (right figure).\r\n</p>\r\n<p>&nbsp;</p>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<img src=\"images/userguide/fiviz/LoadCGI.png\" alt=\"\" width=\"200\"\r\n			height=\"126\" />\r\n		<div>Load Gene Index</div>\r\n	</div>\r\n</div>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img src=\"images/userguide/fiviz/LoadNodeCGI.png\" alt=\"\" width=\"400\"\r\n				height=\"227\" />\r\n			<div>Load Node Cancer Gene Index</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<p>\r\n	<br />By using the first method, the user can load the tree of NCI\r\n	disease terms and display the tree in the left panel. The user can\r\n	select disease term in the tree, all genes or proteins have been\r\n	annotated for the selected disease and its sub-terms will be selected.\r\n</p>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img src=\"images/userguide/fiviz/CGIOverlay.png\" alt=\"\" width=\"700\"\r\n				height=\"527\" />\r\n			<div>Cancer Gene Index Overlay</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>By using the second method, the user can view detailed\r\n	annotations for the selected gene or protein. The user can sort these\r\n	annotations based on PubMedID, Cancer type, and annotation status, and\r\n	also filter annotations based on several criteria.</p>\r\n<p>&nbsp;</p>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img src=\"images/userguide/fiviz/CGIAnnotationsForNode.png\" alt=\"\"\r\n				width=\"700\" height=\"541\" />\r\n			<div>Cancer Gene Index Annotations for Node</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<p>&nbsp;</p>\r\n<h4>\r\n	<span id=\"Survival_Analysis\">Survival Analysis</span>\r\n</h4>\r\n<p>\r\n	Survival analysis is based on a server-side R script to do either coxph\r\n	or Kaplan-Meier survival analysis. To do survival analysis, a\r\n	tab-delimited text file containing at least three columns should be\r\n	provided. The names of three columns should be: Samples, OSDURATION,\r\n	and OSEVENT. For example, see this survival information file downloaded\r\n	from <a href=\"http://www.nejm.org/doi/full/10.1056/NEJMoa021967\"\r\n		rel=\"nofollow\">van de Vijver et al in 2002</a>: <a\r\n		href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/GeneExpression/Nejm_Clin_Simple.txt\"\r\n		rel=\"nofollow\">Nejm_Clin_Simple.txt</a>, which has been simplified for\r\n	our analysis purpose. To do survival analysis, use the popup menu\r\n	\"Analyze Module Functions/Survival Analysis...\" (see below)\r\n</p>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img src=\"images/userguide/fiviz/SurvivalAnalysisMenu.png\" alt=\"\"\r\n				width=\"464\" height=\"205\" />\r\n			<div>Survival Analysis Menu</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>In the survival analysis dialog (below), double click the text\r\n	field to select a file containing survival information for samples used\r\n	to build the displayed FI sub-network (Note: you cannot do survival\r\n	analysis if you use a gene set file only to construct the displayed FI\r\n	subnetweork). You can choose either coxph or Kaplan-Meier model to do\r\n	survival analysis. If you choose the Kaplan-Meier model, you have to\r\n	select a module for analysis. In the Kaplan-Meier analysis, all samples\r\n	will be divided into two groups: samples having no mutated genes in the\r\n	selected module (group 1) and samples having mutated genes in module\r\n	(group 2). It is recommended to run the coxph module first without\r\n	selecting any module in order to see which module is most significantly\r\n	related to survival times. After that, you can focus on some specific\r\n	modules for survival analysis.</p>\r\n<p>&nbsp;</p>\r\n<div class=\"text-center\">\r\n	<div>\r\n		<div>\r\n			<img src=\"images/userguide/fiviz/SurvivalAnalysisDialog.png\" alt=\"\"\r\n				width=\"605\" height=\"332\" />\r\n			<div>Survival Analysis Dialog</div>\r\n		</div>\r\n	</div>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>The results from survival analysis will be displayed in the right\r\n	Results Panel with a tab labeled \"Survival Analysis\" (below left). You\r\n	can do multiple survival analyses. All results returned from the\r\n	server-side R script will be displayed in this panel with labels based\r\n	on your parameter selections in the survival analysis dialog. The last\r\n	result will be selected as default. At most three sections are\r\n	displayed in the result panel for each analysis: Output, Error, and\r\n	Plot. If no warning or error returned from an analysis, the error\r\n	section may not be shown. Rows for modules having p-values less than\r\n	0.05 from coxph (all modules) analysis are displayed in blue with text\r\n	underlined. You can click these modules to do a quick single-module\r\n	based survival analysis without going through the above steps. Single\r\n	module-based Kaplan-Meier analysis will show a plot file. You can click\r\n	the file to view the actual plot (below right). You may need to save\r\n	the plot file for your future use.</p>\r\n<div class=\"text-center\">\r\n	<p>\r\n		<img src=\"images/userguide/fiviz/SurvivalAnalysisResult.png\"\r\n			alt=\"Survival Analysis Results\" width=\"418\" height=\"812\" />\r\n	</p>\r\n	<p>&nbsp;</p>\r\n	<p>\r\n		<img src=\"images/userguide/fiviz/KaplanMeyerPlot.png\"\r\n			alt=\"Kaplan-Meier Survival Plot\" width=\"500\" height=\"500\" />\r\n	</p>\r\n</div>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-25 16:02:02','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,8,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(45,605,'Graph Database','graph-database','<h4>The Reactome Graph Database models the Reactome knowledgebase as an interconnected graph database.</h4>\r\n<p>{loadposition dev-guide-gd-1}</p>\r\n<div style=\"text-align: justify;\">\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p>At the cellular level, life is a network of molecular reactions. In Reactome, these processes are systematically described in molecular detail to generate an ordered network of molecular transformations&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/26656494\">(Fabregat et al. 2015)</a>. This amounts to millions of interconnected terms naturally forming a&nbsp;<a href=\"http://en.wikipedia.org/wiki/Graph_(abstract_data_type)\">graph</a>&nbsp;of biological knowledge. The Reactome Graph provides an intuitive way for data retrieval as well as interpretation and analysis of pathway knowledge.</p>\r\n<p>Retrieving, and especially analysing such complex data becomes tedious when using relational databases. Queries across the pathway knowledgebase are composed by a number of expensive join operations resulting in poor performance and a hard-to-maintain project. Due to the schema-based approach, relational databases are limited in how information is stored and thus are difficult to scale for new requirements. In order to overcome these problems the Reactome database is imported in&nbsp;<a href=\"http://neo4j.com/\" target=\"_blank\" rel=\"noopener\">Neo4j</a>, creating one large interconnected graph. Graph database technology is an effective tool for modelling highly connected data.</p>\r\n<p>Storing Reactome data in this form has many benefits. No denormalisation is required so data can be stored in its natural form. Nodes in the vicinity of a starting point can quickly be traversed giving the user the possibility to not only retrieve data but also perform fast analysis of these neighbour networks. Thus, knowledge that previously was unavailable due to the limitations of relational data storage can now be retrieved.</p>\r\n<p>To easily access and benefit from the graph database, we have developed the GraphCore; an open source library implemented in Java. This project uses&nbsp;<a href=\"http://projects.spring.io/spring-data-neo4j/\" target=\"_blank\" rel=\"noopener\">Spring Data Neo4j</a>, which provides an automatic object graph mapping on top of Neo4j and tightly integrates with other parts of the spring framework used across the project.</p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p class=\"text-center\"><a href=\"/download/current/reactome.graphdb.tgz\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"favth-btn\"><i class=\"fa fa-download\"></i>Download</a></p>\r\n<p>Our Graph Database is available in our&nbsp;<a href=\"download-data\">download data section</a>. It is possible to use it in your local environment by following these steps:</p>\r\n<ol>\r\n<li>Download and install the <strong>latest</strong>&nbsp;<a href=\"https://neo4j.com/download/other-releases/\" target=\"_blank\" rel=\"noopener\">Neo4j</a>.</li>\r\n<li>Download the&nbsp;<a href=\"download/current/reactome.graphdb.tgz\">Graph Database</a>&nbsp;for the latest data release.</li>\r\n<li>Install the Graph Database. Please follow the instructions in the&nbsp;<a href=\"http://neo4j.com/docs/operations-manual/current/\">Neo4j operations tutorial</a>, specifically the sections <a href=\"http://neo4j.com/docs/operations-manual/current/#file-locations\">file locations</a> and <a href=\"http://neo4j.com/docs/operations-manual/current/#backup-restoring\">restoring a backup</a>.</li>\r\n<li>If the standard procedure has been followed, the graph database should be accessible via the Neo4j browser at your&nbsp;<a href=\"http://localhost:7474/\">localhost</a>.</li>\r\n</ol>\r\n<p>Great! Now you have your own copy of the current version of the Reactome data content in your instance of Neo4j, so lets see how you can take advantage of it either with&nbsp;<a href=\"/dev/graph-database/#QueryReactomeGraph\">direct queries to the graph database</a>&nbsp;or using our&nbsp;<a href=\"/dev/graph-database/#GraphCoreLib\">GraphCore java library</a>.</p>\r\n<h4><a href=\"#query-reactome-graph\" id=\"query-reactome-graph\">Directly querying to the Reactome Graph Database</a></h4>\r\n<p>The Neo4j browser offers a nice interface to submit your own queries to the graph database. We recommend using this platform for the first interaction with the Reactome Graph database to see how easy is to use the&nbsp;<a href=\"http://neo4j.com/developer/cypher/\">Cypher query language</a>.</p>\r\n<p>Please refer to our&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=54\">extracting pathway participating molecules</a>&nbsp;tutorial to introduce yourself to using Cypher to query the Reactome Graph Database.</p>\r\n<h4><a href=\"#graph-core\" id=\"graph-core\">Using the Reactome GraphCore java library</a></h4>\r\nThe Reactome GraphCore java library will soon be available! Follow us on <a href=\"http://twitter.com/reactome\">Twitter</a>&nbsp;to get the latest news about it!\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>API</a></h3>\r\nThe API for the Reactome GraphCore java library will soon be available! Follow us on Twitter to get the latest news about it!\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<p><a href=\"index.php?option=com_content&amp;view=article&amp;id=54&amp;catid=29\">Tutorial: Extracting participating molecules using the Graph Database</a>.</p>\r\n<p>To learn more about our graph database, have a look at our relevant publication entitled&nbsp;<a href=\"https://doi.org/10.1371/journal.pcbi.1005968\" target=\"_blank\" rel=\"noopener\">Reactome graph database: Efficient access to complex pathway data</a>.</p>\r\n<p>The API for the Reactome GraphCore Java library is available on our <a href=\"https://github.com/reactome/graph-core\"><i class=\"fa fa-github\" style=\"padding-right: 1px;\"></i>GitHub</a> repository.</p>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-25 16:27:19','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,12,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(46,612,'Teleconferences','teleconferences','','',0,30,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-29 20:01:45','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',2,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(47,619,'Analysis Service','analysis-service','<h4>Explore our tools and web services and learn how to include them in your applications</h4>\r\n<p>{loadposition dev-guide-as-1}</p>\r\n<div>The analysis tool suite contains an overrepresentation analysis, an expression data analysis and a species comparison tool. The tool suite is available via a&nbsp;<a title=\"Web Service\" href=\"https://reactome.org/AnalysisService/\">Web Service</a>&nbsp;so all the available analysis tools can be easily integrated into third party software.</div>\r\n<div style=\"text-align: justify;\">\r\n<h3><a id=\"MoreInfo\" href=\"#MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p>The Analysis Web-Service is token based, so for every analysis request a TOKEN is associated to the result. From this moment on, the results can be accessed via the API token method in order to retrieve more detailed information. Taking advantage of the token produced during every analysis, it is possible to link back to the PathwayBrowser and browse the results overlaid on either the pathways overview or a selected pathway.</p>\r\n<p>As mentioned afore, Reactome provides an overrepresentation analysis, an expression data analysis and a species comparison tool. For overrepresentation and expression analysis, the service can automatically detect which one to perform depending on the format of the submitted data (see&nbsp;<a href=\"#AnalysisFormat\">Table 1</a>). If the data contains expression values in a tab-separated-values (TSV) format, then the expression analysis will be executed, otherwise the overrepresentation analysis is the executed one.</p>\r\n<div class=\"text-center\"><a id=\"AnalysisFormat\"></a>Table 1: Analysis input data formats<br /><img src=\"images/devguide/analysis/analysis_format.png\" alt=\"analysis format\" width=\"1252\" /></div>\r\n<p>Data must be submitted in a format that includes a first row of column headers. The header for column 1 must start with the # symbol. Column 1 must contain protein, compound or other suitable identifiers, such as OMIM IDs. Identifiers can only be placed in column 1. A one-column file is sufficient for pathway over-representation analysis. For expression analysis, columns 2 and onwards must contain numeric values, with no alphabetical characters. Decimals must use the full stop (period) symbol, not commas. Columns must be tab-delimited (this format is an option when saving from most spreadsheet programs such as Excel).</p>\r\n<p>The first line must start with # to indicate the name of the sample (first column) and the names of each expression column when expression data is submitted. Please note that while this line is optional for the overrepresentation format, it is mandatory for the expression data format, because the name of each column needs to be provided. It is recommended though to have this first line defined in the submitted data, so later on it is easier to identify what kind of experiment the data is related to.</p>\r\n<p>Every other line starts with the identifier of the gene/protein/chemical. For expression analysis, the rest of the columns should contain numbers corresponding to the expression values.</p>\r\n<h4><a id=\"DataSubmission\" href=\"#DataSubmission\">Data Submission</a></h4>\r\n<p>The first use case would be when the data to be analysed is submitted using the method:</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/AnalysisService/identifiers/</pre>\r\n</div>\r\n<p>In this case the data must be sent via POST (please refer to the API documentation for more details) so the analysis will be performed against the Reactome database content.</p>\r\n<p>A second use case is when a text flat file containing the data to be analysed (in the specified format mentioned afore) is sent. In this case the method to be used is:</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/AnalysisService/identifiers/form/</pre>\r\n</div>\r\n<p>A third use case would be when the data to be analysed is accessible through the Internet. In this case, there is a specific method that indicates to the analysis service where to get the data:</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/AnalysisService/identifiers/url/</pre>\r\n</div>\r\n<p>Please note all these methods have an option to project the identifiers to human and only show the result in this species (by adding the projection/ suffix to any of them). Please refer to the API documentation for more details.</p>\r\n<h4><a id=\"analysis-result-handling\" href=\"#analysis-result-handling\">Analysis result handling</a></h4>\r\n<p>Once the analysis is finished, the result is sent back to the user in json format (please refer to the API documentation for more details). One of the fields in the analysis summary object is called token. The particular analysis result is associated to this token and it can be retrieved later on using the following method without having to send the same data again:</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/AnalysisService/token/</pre>\r\n</div>\r\n<p>Reactome ensures that the token will be available for the 7 next days after the analysis. After this period it goes into an LRU queue, so may be available for longer but this cannot be ensured as it depends on the frequency of its usage.</p>\r\n<p>The pathways field in the analysis result contains a list of the most significant pathways with the corresponding statistics results. By default, these pathways are sorted from the most to the least significant, so the first item would be the most statistically significant. However the user is advised to take into account the statistical values (pValue and FDR) and also the results coverage in the pathway, indicated by entities and reactions found.</p>\r\n<h4><a id=\"linking-back\" href=\"#linking-back\">Linking back to Reactome Pathway Browser</a></h4>\r\n<p>If linking back to Reactome to visualise the results is required, there are two methods. Providing the token is enough to link back to the PathwayBrowser and get the Fireworks results overlay view. To do so please replace {TOKEN} in the following URL by the one provided in the result.</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/PathwayBrowser/#DTAB=AN&amp;ANALYSIS=<b>{TOKEN}</b></pre>\r\n</div>\r\n<p>To build a link back pointing to a specific pathway to overlay the result on its diagram, please replace&nbsp;<b>{ST_ID}</b>&nbsp;by the pathway stable identifier (provided in the analysis result for each pathway) and&nbsp;<b>{TOKEN}</b>&nbsp;by the one provided in the result.</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/PathwayBrowser/#<b>{ST_ID}</b>&amp;DTAB=AN&amp;ANALYSIS=<b>{TOKEN}</b></pre>\r\n</div>\r\n<p>&nbsp;</p>\r\n<h3><a id=\"GetStarted\" href=\"#GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>OK! Youve reached this far :) so lets see how to start using the analysis service.</p>\r\n<p>There are two ways to submit your sample of identifiers for analysis. The first method is to POST all the identifiers (or the file containing them). The second involves letting the analysis service know where the sample is by providing its URL. Many types of identifiers that can be submitted, including UniProt, chEBI, Ensembl, miRBase, GenBank/EMBL/DDBJ, RefPep, RefSeq, EntrezGene, OMIM, InterPro, Affymetrix, Agilent, Compound, Illumina, etc. Please get in touch with our&nbsp;<a href=\"mailto:help@reactome.org?subject=Sample%20Identifiers%20are%20not%20supported\">help desk</a>&nbsp;if your sample identifiers are not supported.</p>\r\n<p>In the following examples we use&nbsp;<a href=\"http://curl.haxx.se/docs/manual.html\">curl</a>&nbsp;to query the analysis service from the command line. They show how to send some gene names (PIK3C2A, PTEN and UNC5B) to be analysed and also how to provide the location of your data via the&nbsp;<b>url</b>&nbsp;interface.</p>\r\n<p>The simplest approach is to send your gene names via POST to the&nbsp;<b><a href=\"https://reactome.org/AnalysisService/identifiers/projection\">https://reactome.org/AnalysisService/identifiers/projection</a>&nbsp;</b>method.</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>curl -H \"Content-Type: text/plain\" -d \"$(printf \'<b>#Genes</b>\\n<b>PIK3C2A</b>\\n<b>PTEN</b>\\n<b>UNC5B</b>\')\" -X POST --url https://reactome.org/AnalysisService/identifiers/projection/\\?pageSize\\=1\\&amp;page\\=1</pre>\r\n</div>\r\n</div>\r\n<p>For long lists, the previous example might be impractical. A more convenient way is to POST the content of a file containing the sample to be analysed. Lets assume you have a file called&nbsp;<b>genes.txt</b>&nbsp;that contains the following set of genes to be analysed:</p>\r\n<div class=\"code\">\r\n<pre>#Genes<br />PIK3C2A<br />PTEN<br />UNC5B</pre>\r\n</div>\r\n<p>The command changes to indicate that the data is now taken from the file (but please note that the method of the analysis service remains the same):</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>curl -H \"Content-Type: text/plain\" --data-binary @genes.txt -X POST --url https://reactome.org/AnalysisService/identifiers/projection/\\?pageSize\\=1\\&amp;page\\=1</pre>\r\n</div>\r\n</div>\r\n<p>Both of the previous commands will produce the result shown in&nbsp;<a href=\"#AnalysisResult\">Figure 1</a></p>\r\n<div><a id=\"AnalysisResult\"></a>Figure 1: Analysis result example</div>\r\n<p><img src=\"images/devguide/analysis/analysis_result_example_01.png\" alt=\"analysis result example 01\" /></p>\r\n<p>Lets focus on the retrieved&nbsp;<b>pathways</b>&nbsp;list. It only contains one pathway because in the command we have specified&nbsp;<b>\\?pageSize\\=1\\&amp;page\\=1</b>&nbsp;and that forces the analysis to provide&nbsp;<b>ONLY</b>&nbsp;the most significant pathway for the submitted data. If you want the first 10 results, the way of doing it would be&nbsp;<b>\\?pageSize\\=10\\&amp;page\\=1</b>. To see the results from 11 to 20 use&nbsp;<b>\\?pageSize\\=10\\&amp;page\\=2</b>. The reason for this&nbsp;<b>paging</b>&nbsp;mechanism is to avoid overloading the client with the full set of results. Here we are querying from the command line and probably the memory usage isnt an issue, but please consider web-clients.&nbsp;<b>Important note:</b>&nbsp;If&nbsp;<b>pageSize</b>&nbsp;and&nbsp;<b>page</b>&nbsp;are not specified, then the whole set of results is retrieved to the client.</p>\r\n<p>Other important fields in the result are&nbsp;<b>pathwaysFound</b>,&nbsp;<b>identifiersNotFound</b>,&nbsp;<b>summary.token</b>&nbsp;and&nbsp;<b>summary.type</b>. The first two are self-descriptive, so lets focus on the&nbsp;<b>summary</b>. As already mentioned, the&nbsp;<b>summary.token</b>&nbsp;can be used to retrieve the results of a previously performed analysis without the need to submit the sample again. For example, the results of the previous analysis can be easily accessed by simply calling the&nbsp;<b>https://reactome.org/AnalysisService/token</b>&nbsp;method and providing the token.</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>curl https://reactome.org/AnalysisService/token/<b>MjAxNTEwMjAwNjU0MDBfMzMw</b>\\?pageSize\\=1\\&amp;page\\=1</pre>\r\n</div>\r\n</div>\r\n<p>As illustrated in&nbsp;<a href=\"#AnalysisTokenMethods\">Figure 2</a>, the analysis service provides a thorough collection of token-based methods that can be used to access the results of a previously performed analysis.</p>\r\n<div class=\"text-center\"><a id=\"AnalysisTokenMethods\"></a>Figure 2: All the token-based methods provided by the analysis service</div>\r\n<p class=\"text-center\"><img src=\"images/devguide/analysis/token_methods.png\" alt=\"token methods\" width=\"972\" /></p>\r\n<p>The&nbsp;<b>summary.type</b>&nbsp;provides information about the type of the analysis performed and it can be OVERREPRESENTATION, EXPRESSION or SPECIES_COMPARISON.</p>\r\n<p>Every item in the&nbsp;<b>pathways</b>&nbsp;array, contains information about a specific pathway, i.e its name, stable identifier, the species it belongs to, the number of entities matching the submitted sample etc.&nbsp;<b>pValue</b>&nbsp;and&nbsp;<b>fdr</b>&nbsp;show the significance of the pathway as the result of the analysis.</p>\r\n<h4>Pointing to your resource as data provider for the analysis</h4>\r\n<p>Lets suppose that you have some data to be analysed on your server and you want to point Reactome analysis to retrieve it directly. We use&nbsp;<a href=\"https://www.ebi.ac.uk/pride\">PRIDE</a>&nbsp;data for our examples. More specifically we analyse PRIDE data stored in&nbsp;<a href=\"https://www.ebi.ac.uk/pride/ws/archive/protein/list/assay/27929.acc\">https://www.ebi.ac.uk/pride/ws/archive/protein/list/assay/27929.acc</a>. This can be done by using the&nbsp;<b>/identifiers/url</b>&nbsp;methods:</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>curl -H \"Content-Type: text/plain\" -d \"https://www.ebi.ac.uk/pride/ws/archive/protein/list/assay/27929.acc\" -X POST --url https://reactome.org/AnalysisService/identifiers/url/projection/\\?pageSize\\=1\\&amp;page\\=1</pre>\r\n</div>\r\n</div>\r\n<p>Just&nbsp;<b>POST</b>ing the&nbsp;<b>URL</b>, where the data is, to the&nbsp;<b>https://reactome.org/AnalysisService/identifiers/url/projection/</b>&nbsp;method is enough to perform the analysis. The rest of the parameters work exactly as explained above. It is also important to take into account that&nbsp;<b>URL</b>s sent to this method can either be&nbsp;<b>HTTP</b>&nbsp;or&nbsp;<b>HTTPS</b>, if your service uses&nbsp;<a href=\"http://en.wikipedia.org/wiki/HTTPS\">secure HTTP</a>, we can deal with it ;).</p>\r\n<h4><a id=\"how-do-i-query\" href=\"#how-do-i-query\">I have a JavaScript client. How do I query your service?</a></h4>\r\n<p>First we will create a simple HTML page with one button and a place holder to show a table with the results of the query. Please note we have included the&nbsp;<a href=\"http://jquery.com/\" target=\"_blank\" rel=\"nofollow noopener noreferrer\">jQuery library</a>.</p>\r\n<a id=\"HTMLExample01\"></a>\r\n<div class=\"text-center\">HTML: Base example</div>\r\n<div class=\"code\">\r\n<div>\r\n<pre>&lt;html&gt;<br />  &lt;head&gt;<br />	&lt;title&gt;Connection to the Reactome Analysis Service&lt;/title&gt;<br />	//We are using jQuery for this example<br />	&lt;script src=\"//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js\"&gt;&lt;/script&gt;<br />	&lt;script type=\"text/javascript\"&gt;\r\n		//Here is where we are going to include the magic\r\n	&lt;/script&gt;\r\n  &lt;/head&gt;\r\n  &lt;body&gt;\r\n	&lt;h1&gt;Connection to the Reactome Analysis Service&lt;/h1&gt;\r\n	&lt;p&gt;Please click the button to execute the analysis:&lt;/p&gt;\r\n	&lt;button id=\"analyseURL\"&gt;Analyse data&lt;/button&gt;\r\n	&lt;br&gt; &lt;br&gt;\r\n	&lt;div id=\"resultsHolder\"&gt;&lt;/div&gt;\r\n	&lt;div id=\"errorHolder\" style=\"color:red\"&gt;&lt;/div&gt;\r\n  &lt;/body&gt;\r\n&lt;/html&gt;<br />\r\n</pre>\r\n</div>\r\n</div>\r\n<p>As you can see in the&nbsp;<a href=\"#HTMLExample01Figure\">Figure 3</a>, the resulting page is quite a simple one, but enough for our&nbsp;<b>get started</b>&nbsp;purposes.</p>\r\n<a id=\"HTMLExample01Figure\"></a>\r\n<div class=\"text-center\">Figure 3: Example HTML</div>\r\n<p class=\"text-center\"><img style=\"border: solid 1px grey;\" src=\"images/devguide/analysis/example_01.png\" alt=\"example 01\" width=\"923\" /></p>\r\n<p>The next step is to write the code that connects to the analysis service and presents the data back to the client. Please note that we will use the&nbsp;<b>/identifiers/url/projection</b>&nbsp;method in order to request the analysis of&nbsp;<a href=\"https://www.ebi.ac.uk/pride/ws/archive/protein/list/assay/27929.acc\">data</a>&nbsp;stored in the PRIDE repository.</p>\r\n<a id=\"HTMLExample02\"></a>\r\n<div class=\"text-center\">HTML: Example with JavaScript code included</div>\r\n<div class=\"code\">\r\n<div>\r\n<pre>&lt;html&gt;\r\n  &lt;head&gt;\r\n	&lt;title&gt;Connection to the Reactome Analysis Service&lt;/title&gt;\r\n	&lt;script src=\"//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js\"&gt;&lt;/script&gt;\r\n	&lt;script type=\"text/javascript\"&gt;\r\n		$( document ).ready(function() {\r\n			$(\"#analyseURL\").click(function() {\r\n				// The URL of the analysis method\r\n				var reactomeURI = \'https://reactome.org/AnalysisService/identifiers/url/projection?pageSize=20&amp;page=1\';\r\n				// The URL of our sample to be submitted for analysis\r\n				var dataUrl = \"http://www.ebi.ac.uk/pride/ws/archive/protein/list/assay/27929.acc\";\r\n				// Perform the ajax call to the analysis service\r\n				jQuery.ajax({\r\n					type: \"POST\",\r\n					url: reactomeURI,\r\n					data: dataUrl,\r\n					headers: {\r\n						\"Content-Type\": \"text/plain\"\r\n					},\r\n					dataType: \"json\"\r\n				}).success(function (data, textStatus) {\r\n					if (textStatus === \'success\') {\r\n						var table = \"&lt;table width=\'100%\'&gt;&lt;tr&gt;&lt;td&gt;Name&lt;/td&gt;&lt;td&gt;pValue&lt;/td&gt;&lt;td&gt;FDR&lt;/td&gt;&lt;/tr&gt;\";\r\n						$.each(data.pathways, function(num, pathway){\r\n							table += \"&lt;tr&gt;&lt;td&gt;&lt;a href=\'https://reactome.org/PathwayBrowser/#/\"+  pathway.stId +\"&amp;DTAB=AN&amp;ANALYSIS=\" + data.summary.token +\"\'&gt;\" + pathway.name + \"&lt;/a&gt;&lt;/td&gt;&lt;td&gt;\" + pathway.entities.pValue + \"&lt;/td&gt;&lt;td&gt;\" + pathway.entities.fdr + \"&lt;/td&gt;\";\r\n						});\r\n						table += \"&lt;/table&gt;\";\r\n						$(\"#resultsHolder\").append(table);\r\n					} else {\r\n						$(\'#errorHolder\').text(\'An error occurred\');\r\n					}\r\n				}).error(function (data, status, headers, config) {\r\n					$(\'#errorHolder\').text(status + \": \" + headers);\r\n				});\r\n			});\r\n		});\r\n	&lt;/script&gt;\r\n  &lt;/head&gt;\r\n  &lt;body&gt;\r\n	&lt;h1&gt;Connection to the Reactome Analysis Service&lt;/h1&gt;\r\n	&lt;p&gt;Please click the button to execute the analysis:&lt;/p&gt;\r\n	&lt;button id=\"analyseURL\"&gt;Analyse data&lt;/button&gt;\r\n	&lt;br&gt; &lt;br&gt;\r\n	&lt;div id=\"resultsHolder\"&gt;&lt;/div&gt;\r\n	&lt;div id=\"errorHolder\" style=\"color:red\"&gt;&lt;/div&gt;\r\n  &lt;/body&gt;\r\n&lt;/html&gt;<br />\r\n</pre>\r\n</div>\r\n</div>\r\n<p>Quickly analysing the main parts of the&nbsp;<a href=\"#HTMLExample02\">JavaScript code</a>&nbsp;body, we see two main methods:&nbsp;<a href=\"http://learn.jquery.com/using-jquery-core/document-ready/\"><b>$( document ).ready</b></a>&nbsp;and&nbsp;<a href=\"http://api.jquery.com/jquery.ajax/\"><b>jQuery.ajax</b></a>. The first one ensures the content of the associated callback function will be executed when all the HTML is fully loaded in the&nbsp;<a href=\"http://www.w3.org/TR/WD-DOM/introduction.html\">DOM</a>. The second is responsible for performing the analysis request against Reactomes analysis service.</p>\r\n<p>Focusing on the&nbsp;<b>.success</b>&nbsp;function in the ajax query, we see how the results for a success connection with a success response are handled (please note that a success connection differs from a success response, since there can be different reasons why a success connection can end up retrieving an error, i.e. data content is not in the right format). The results can be accessed within the&nbsp;<b>data</b>&nbsp;object passed to the&nbsp;<b>.success(function(data, textStatus){})</b>&nbsp;method following the format shown in&nbsp;<a href=\"#AnalysisResult\">Figure 1</a>.</p>\r\n<p>In the example, a table is created with the retrieved set of pathways (<a href=\"#HTMLExample02Figure\">Figure 4</a>). Each row of the table contains the pathway name, the pValue and the FDR. Its also important to see how weve built the hyperlink from the name to the view of the pathway (with the analysis result overlay) directly to the&nbsp;<a href=\"/PathwayBrowser/\">Pathway Browser</a>. Please note that it is also possible to link to the Pathway Browser in order to show the general result overlaid on the pathways overview. The link for this is:</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>https://reactome.org/PathwayBrowser/#/DTAB=AN&amp;ANALYSIS=<b>{TOKEN}</b></pre>\r\n</div>\r\n</div>\r\n<a id=\"HTMLExample02Figure\"></a>\r\n<div class=\"text-center\">Figure 4: Example HTML with results already loaded</div>\r\n<p class=\"text-center\"><img style=\"border-right-color: grey; border-bottom-color: grey;\" src=\"images/devguide/analysis/example_02.png\" alt=\"example 02\" width=\"928\" /></p>\r\n<p>But this example doesnt look pretty! Yes, we know but we are explaining how to access Reactome data. Please have a look to the&nbsp;<a href=\"http://www.w3schools.com/css/\">CSS</a>&nbsp;documentation and start learning how to apply the styles to make the look fit with your expectations.</p>\r\n<p>&nbsp;</p>\r\n<h3><a id=\"Resources\" href=\"#Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<p><a href=\"/AnalysisService/\">API Documentation</a></p>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-31 15:55:45','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"1\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,11,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(48,620,'Content Service','content-service','<h4>Explore our tools and web services and learn how to include them in your applications</h4>\r\n<p>{loadposition dev-guide-cs-1}</p>\r\n<div style=\"text-align: justify;\">\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p>The Content Service constitutes an easy&nbsp;<a href=\"/ContentService/#/\" target=\"_blank\" rel=\"noopener noreferrer\">API</a>&nbsp;that provides access to the Reactome knowledgebase. Information in Reactome is authored by expert biologist researchers, maintained by Reactome editorial staff, and extensively cross-referenced to other resources e.g. NCBI, Ensembl, UniProt, UCSC Genome Browser, HapMap, KEGG (Gene and Compound), ChEBI, PubMed and GO. It also incorporates inferred orthologous reactions for over 20 non-human species including mouse, rat, chicken, puffer fish, worm, fly, yeast, rice, Arabidopsis and E.coli. Additionally, the Content Service provides access to molecular interactions integrated from PSICQUIC.</p>\r\n<p>The Content Service is based on the Representational State Transfer (REST) protocol. This eliminates the need for complex clients and renders the service simpler, more lightweight, more flexible, and, thus, easier to integrate into third party software compared to its SOAP/WSDL counterparts.</p>\r\n<p>The&nbsp;<a href=\"/ContentService/\" target=\"_blank\" rel=\"noopener noreferrer\">API</a>&nbsp;includes a set of methods classified in groups according to their functionality. For instance, expanding the&nbsp;<a href=\"/ContentService/#/pathways\" target=\"_blank\" rel=\"noopener noreferrer\">pathways group</a>&nbsp;reveals a set of methods that provide specific information about pathways such as the contained Events or the participating PhysicalEntities. All methods are specified as either GET or POST requests. GET and POST are not interchangeable.&nbsp;<a href=\"/ContentService/#!/pathways/getParticipatingPhysicalEntitiesUsingGET\" target=\"_blank\" rel=\"noopener noreferrer\">Expanding a particular method</a>&nbsp;provides more information about its input and output parameters as well as a simple test case for users to try out. In addition, users can also use any command line client such as&nbsp;<a href=\"http://www.gnu.org/software/wget/\" target=\"_blank\" rel=\"noopener noreferrer\">wget</a>&nbsp;or&nbsp;<a href=\"http://curl.haxx.se/docs/manual.html\" target=\"_blank\" rel=\"noopener noreferrer\">curl</a>&nbsp;to access the methods.</p>\r\n<p>It is worth mentioning that to use this API efficiently, the user needs to understand the Reactome&nbsp;<a href=\"/documentation/data-model\" target=\"_blank\" rel=\"noopener noreferrer\">data model</a>&nbsp;and&nbsp;<a href=\"/content/schema/\" target=\"_blank\" rel=\"noopener noreferrer\">schema</a>.</p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>Lets start with one of the most basic examples; retrieving the version of the database. It can be addressed by querying the /data/database/version method:</p>\r\n<div class=\"code\">\r\n<pre>curl -X GET --header \'Accept: text/plain\' \'https://reactome.org/ContentService/data/database/version\'</pre>\r\n</div>\r\n<p>The response will be a text/plain file containing just the number of the release.</p>\r\n<p>To retrieve the information for the reaction&nbsp;<b>Mad1 binds kinetochore</b>&nbsp;which identifier is&nbsp;<b>R-HSA-141409</b>, the query would be as follows:</p>\r\n<div class=\"code\">\r\n<pre>curl -X GET --header \'Accept: application/json\' \'https://reactome.org/ContentService/data/query/R-HSA-141409\'</pre>\r\n</div>\r\n<p>And the response comes in application/json format:</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>{\r\n  dbId: 141409,\r\n  displayName: \"Mad1 binds kinetochore\",\r\n  stId: \"R-HSA-141409\",\r\n  created: {\r\n    dbId: 143430,\r\n    displayName: \"Yen, T, 2004-05-05 00:00:00\",\r\n    dateTime: \"2004-05-05 05:00:00.0\",\r\n    schemaClass: \"InstanceEdit\"\r\n  },\r\n  modified: {\r\n    dbId: 1591212,\r\n    displayName: \"Matthews, L, 2011-09-08\",\r\n    dateTime: \"2011-09-08 21:45:40.0\",\r\n    schemaClass: \"InstanceEdit\"\r\n  },\r\n  isInDisease: false,\r\n  isInferred: false,\r\n  name: [\r\n    \"Mad1 binds kinetochore\"\r\n  ],\r\n  speciesName: \"Homo sapiens\",\r\n  authored: [\r\n    143430\r\n  ],\r\n  compartment: [\r\n    {\r\n      dbId: 70101,\r\n      displayName: \"cytosol\",\r\n      accession: \"0005829\",\r\n      databaseName: \"GO\",\r\n      definition: \"The part of the cytoplasm that does not contain organelles but which does contain other particulate matter, such as protein complexes.\",\r\n      name: \"cytosol\",\r\n      url: \"http://www.ebi.ac.uk/ego/QuickGO?mode=display&amp;entry=GO:0005829\",\r\n      schemaClass: \"EntityCompartment\"\r\n    }\r\n  ],\r\n  literatureReference: [\r\n    {\r\n      dbId: 143441,\r\n      displayName: \"Mitotic checkpoint proteins HsMAD1 and HsMAD2 are associated with nuclear pore complexes in interphase\",\r\n      title: \"Mitotic checkpoint proteins HsMAD1 and HsMAD2 are associated with nuclear pore complexes in interphase\",\r\n      journal: \"J Cell Sci\",\r\n      pages: \"953-63\",\r\n      pubMedIdentifier: 11181178,\r\n      volume: 114,\r\n      year: 2001,\r\n      url: \"http://www.ncbi.nlm.nih.gov/pubmed/11181178\",\r\n      schemaClass: \"LiteratureReference\"\r\n    }\r\n  ],\r\n  species: [\r\n    {\r\n      dbId: 48887,\r\n      displayName: \"Homo sapiens\",\r\n      name: [\r\n        \"Homo sapiens\",\r\n        \"H. sapiens\",\r\n        \"Hs\",\r\n        \"human\",\r\n        \"man\"\r\n      ],\r\n      taxId: \"9606\",\r\n      schemaClass: \"Species\"\r\n    }\r\n  ],\r\n  summation: [\r\n    {\r\n      dbId: 143355,\r\n      displayName: \"\",\r\n      text: \"The association of Mad1 with the kinetochore is the first step in the process of Mad2 mediated amplification of the signal from defective kinetochores.\",\r\n      schemaClass: \"Summation\"\r\n    }\r\n  ],\r\n  input: [\r\n    {\r\n      dbId: 141433,\r\n      displayName: \"MAD1L1 [cytosol]\",\r\n      stId: \"R-HSA-141433\",\r\n      name: [\r\n        \"MAD1L1\",\r\n        \"HsMad1\"\r\n      ],\r\n      speciesName: \"Homo sapiens\",\r\n      consumedByEvent: [\r\n        141409\r\n      ],\r\n      endCoordinate: 718,\r\n      referenceType: \"ReferenceGeneProduct\",\r\n      startCoordinate: 1,\r\n      schemaClass: \"EntityWithAccessionedSequence\"\r\n    },\r\n    {\r\n      dbId: 141398,\r\n      displayName: \"Kinetochore Complex [cytosol]\",\r\n      stId: \"R-HSA-141398\",\r\n      name: [\r\n        \"Kinetochore Complex\"\r\n      ],\r\n      speciesName: \"Homo sapiens\",\r\n      consumedByEvent: [\r\n        141409\r\n      ],\r\n      schemaClass: \"GenomeEncodedEntity\"\r\n    }\r\n  ],\r\n  output: [\r\n    {\r\n      dbId: 141441,\r\n      displayName: \"Mad1:kinetochore complex [cytosol]\",\r\n      stId: \"R-HSA-141441\",\r\n      name: [\r\n        \"Mad1:kinetochore complex\"\r\n      ],\r\n      speciesName: \"Homo sapiens\",\r\n      producedByEvent: [\r\n        141409\r\n      ],\r\n      hasComponent: [\r\n        \r\n      ],\r\n      schemaClass: \"Complex\"\r\n    }\r\n  ],\r\n  schemaClass: \"Reaction\"\r\n}</pre>\r\n</div>\r\n</div>\r\n<p>In the previous result, there are several json field keys such as dbId, displayName, stId, name, compartment, literatureReference and so on. In case we are only interested in a particular field, e.g compartment, the query would be as follows:</p>\r\n<div class=\"code\">\r\n<pre>curl -X GET --header \'Accept: text/plain\' \'https://reactome.org/ContentService/data/query/R-HSA-141409/compartment\'</pre>\r\n</div>\r\n<p>In this case, because&nbsp;<b>compartment</b>&nbsp;is an object, the result will be a TSV file where the first column is the identifier of the object, the second column is the displayName of the object and the third column is the schemaClass of the object:</p>\r\n<div class=\"code\">\r\n<pre>70101	cytosol	EntityCompartment</pre>\r\n</div>\r\n<p>In case the queried attribute is a&nbsp;<b>primitive</b>&nbsp;type:</p>\r\n<div class=\"code\">\r\n<pre>curl -X GET --header \'Accept: text/plain\' \'https://reactome.org/ContentService/data/query/R-HSA-141409/displayName\'</pre>\r\n</div>\r\n<p>Then the returned value is the content of it in a text/plain response:</p>\r\n<div class=\"code\">\r\n<pre>Mad1 binds kinetochore</pre>\r\n</div>\r\n<p>As the last example in the section, lets check out how to retrieve the participanting molecules of the reaction used in the previous examples. To do so, we call the&nbsp;<a href=\"/ContentService/#!/events/getParticipatingPhysicalEntitiesUsingGET\">getParticipatingPhysicalEntities</a>&nbsp;method in the API:</p>\r\n<div class=\"code\">\r\n<pre>curl -X GET --header \'Accept: application/json\' \'https://www.reactome.org/ContentService/data/event/R-HSA-141409/participatingPhysicalEntities\'</pre>\r\n</div>\r\n<p>And the response comes in application/json format:</p>\r\n<div class=\"code\">\r\n<pre>[\r\n  {\r\n    dbId: 141398,\r\n    displayName: \"Kinetochore Complex [cytosol]\",\r\n    stId: \"R-HSA-141398\",\r\n    name: [\r\n      \"Kinetochore Complex\"\r\n    ],\r\n    speciesName: \"Homo sapiens\",\r\n    schemaClass: \"GenomeEncodedEntity\"\r\n  },\r\n  {\r\n    dbId: 141433,\r\n    displayName: \"MAD1L1 [cytosol]\",\r\n    stId: \"R-HSA-141433\",\r\n    name: [\r\n      \"MAD1L1\",\r\n      \"HsMad1\"\r\n    ],\r\n    speciesName: \"Homo sapiens\",\r\n    endCoordinate: 718,\r\n    referenceType: \"ReferenceGeneProduct\",\r\n    startCoordinate: 1,\r\n    schemaClass: \"EntityWithAccessionedSequence\"\r\n  },\r\n  {\r\n    dbId: 141441,\r\n    displayName: \"Mad1:kinetochore complex [cytosol]\",\r\n    stId: \"R-HSA-141441\",\r\n    name: [\r\n      \"Mad1:kinetochore complex\"\r\n    ],\r\n    speciesName: \"Homo sapiens\",\r\n    hasComponent: [\r\n      \r\n    ],\r\n    schemaClass: \"Complex\"\r\n  }\r\n]</pre>\r\n</div>\r\n<p>Check out the&nbsp;<a href=\"/ContentService/\">ContentService API</a>&nbsp;to find out more methods and see which ones cover your needs. Please&nbsp;<a href=\"mailto:help@reactome.org?subject=Question%20about%20the%20ContentService%20API\">let us know</a>&nbsp;if your needs are not covered and you think extra methods should be added to our API.</p>\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<p><a href=\"/ContentService/\">API Documentation</a></p>\r\n<p><a href=\"index.php?option=com_content&amp;view=article&amp;id=96&amp;catid=29&amp;lang=en-GB\" hreflang=\"en\">Diagram exporter</a></p>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-31 16:15:13','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,10,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(49,621,'Pathways Overview','pathways-overview','<p>The Pathways Overview is a genome-wide, hierarchical visualization of Reactome pathways in a space-filling graph. Each node represents a pathway, and the size of the node reflects the number of entities (proteins, small molecules, genes, etc) belonging to that pathway. The edges represent the connection between the different pathway nodes within the Reactome pathway hierarchy.</p>\r\n<p>{loadposition dev-guide-po-1}</p>\r\n<div>\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p>The central node within each pathway burst is the parent (or top-level) pathway node. Each child (or sub level) pathway belonging to the pathway hierarchy is represented as a node, which is projected outwards from the central node and connected to the subsequent pathway in the hierarchy, until the lowest level pathway is reached. When the graph initially loads, the Pathways Overview display automatically fits all the nodes and edges into the existing area.</p>\r\n<p>The pathways overview retrieves the information directly from Reactome server, which mean that for third party resources it will be straight forward to include it in browser supporting CORS, or it will just work by adding a proxy mechanism in order to avoid SOP. Another important feature of the pathways overview that can also be used from third party applications is the analysis result overlay. If you are interested in learning more about Reactomes Analysis Service, please check out the&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=47\">Analysis Service developers guide</a>&nbsp;and later on have a look to the pathways overview&nbsp;<a href=\"#API\">API</a><a>.</a></p>\r\n<p>&nbsp;</p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>{loadposition dev-guide-po-2}</p>\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>API</a></h3>\r\n<p>{loadposition dev-guide-po-3}</p>\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<ul style=\"list-style: none;\">\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp/fireworks\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Fireworks code</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp/fireworks-js\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Fireworks exporter to JavaSript</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>Reactome Pathways Portal code</a></li>\r\n</ul>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-31 16:18:34','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,9,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(50,622,'Pathway Diagrams','pathway-diagrams','<p>The Reactome Pathway Diagram Viewer provides an intuitive means of viewing and interacting with pathway diagrams. Pathway diagrams represent the steps of a pathway as a series of interconnected molecular events, known in Reactome as reactions.</p>\r\n<p>{loadposition dev-guide-pd-1}</p>\r\n<div>\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p>Reactome is a curated database of pathways and reactions (pathway steps) in human biology. The Reactome definition of a reaction includes many events in biology that are changes in state, such as binding, activation, translocation and degradation, in addition to classical biochemical reactions. Information in the database is authored by expert biologist researchers, maintained by Reactome editorial staff, and extensively cross-referenced to other resources e.g. NCBI, Ensembl, UniProt, UCSC Genome Browser, HapMap, KEGG (Gene and Compound), ChEBI, PubMed and GO. Inferred orthologous reactions are available for a number of non-human species including mouse, rat, chicken, puffer fish, worm, fly, yeast, rice and Arabidopsis.</p>\r\n<p>The diagram viewer retrieves the information directly from Reactome server, which mean that for third party resources it will be straight forward to include it in browser supporting CORS, or it will just work by adding a proxy mechanism in order to avoid SOP. Another important feature of the diagram viewer that can also be used from third party applications is the analysis result overlay. If you are interested in learning more about Reactomes Analysis Service, please check out the&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=47\">Analysis Service developers guide</a>&nbsp;and later on have a look to the Diagram viewer&nbsp;<a href=\"#API\">API</a><a>.</a></p>\r\n<p>&nbsp;</p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>Currently, there are two options for integrating Reactomes pathway diagrams in your web application. The first is to use our&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=58\">DiagramJs widget</a>, while the second option involves using our&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=59\">GWT implementation</a>.</p>\r\n<p>{loadposition dev-guide-pd-2}</p>\r\n<!--Get Started Buttons - close-->\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>API</a></h3>\r\n<p>Whether you are a JavaScript or GWT developer, we have a tool that will suit you. Check out your option:</p>\r\n<p>{loadposition dev-guide-pd-3}</p>\r\n<!-- API Buttons - close-->\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<ul style=\"list-style: none;\">\r\n<li style=\"line-height: 30px;\"><i class=\"fa fa-file-alt\" style=\"padding-right: 5px;\"></i><a href=\"index.php?option=com_content&amp;view=article&amp;id=100&amp;catid=29&amp;lang=en-GB\" hreflang=\"en\">Pathway Diagram Specifications</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp/diagram\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Diagram code</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp/diagram-js\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Diagram exporter to JavaSript</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>Reactome Pathways Portal code</a></li>\r\n</ul>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-31 16:19:32','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,8,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(53,625,'Data Model','data-model','<p>Life on the cellular level is a network of molecular interactions. Molecules are synthesized and degraded, undergo a bewildering array of temporary and permanent modifications, are transported from one location to another, and form complexes with other molecules. Reactome represents all of this complexity as reactions in which input physical entities are converted to output entities. These reactions can occur spontaneously or be facilitated by physical entities acting as catalysts, and their progress can be modulated by regulatory effects of other physical entities. Reactions are linked together by shared physical entities: a product from one reaction may be a substrate in another reaction and may catalyze yet a third. It is often convenient, if sometimes arbitrary, to group such sets of interlinked reactions into pathways.</p>\r\n<p>The functions of macromolecular entities such as proteins are often determined not only by their primary sequences but by chemical modifications they have undergone. In Reactome, unmodified and modified forms of a protein are distinct physical entities and the modification process is treated as an explicit reaction. A macromolecules function may depend on whether the molecule is free or complexed with specific other molecules. Reactome treats complexes as physical entities distinct from their components, and the multimerization events that build up complexes are modeled explicitly as reactions.</p>\r\n<p>Cellular compartments play a key role in biological processes. The segregation of molecules into different compartments often regulates the reactions in which those entities can participate, or can be responsible for driving a reaction forward. In Reactome, a molecule in one compartment is distinct from that molecule in another compartment. Thus, extracellular and cytosolic glucose are different Reactome entities and, e.g., the movement of glucose across the plasma membrane is a reaction that converts the extracellular glucose entity into the cytosolic one.</p>\r\n<p>Many biochemical entities and processes appear redundant: there are two or more chemically distinct entities that can act more or less interchangeably. It is often useful to treat functionally equivalent protein isoforms, splice variants, and paralogues as a single entity, implying that any individual entity from the given set could fulfill the same role in a given situation. The Reactome data model allows this type of generalization, but does so explicitly in a way that allows us to trace specific functions back to the individual molecules covered by the generalization.</p>\r\n<p>The goal of the Reactome knowledgebase is to represent human biological processes, but many of these processes have not been directly studied in humans. Rather, a human event has been inferred from experiments on material from a model organism. In such cases, the model organism reaction is annotated in Reactome, the inferred human reaction is annotated as a separate event, and the inferential link between the two reactions is explicitly noted.</p>\r\n<p>Reactome uses a frame-based knowledge representation. The data model consists of classes (frames) that describe the different concepts (e.g., reaction, simple entity). Knowledge is captured as instances of these classes (e.g., glucose transport across the plasma membrane, cytosolic ATP). Classes have attributes (slots) which hold properties of the instances (e.g., the identities of the molecules that participate as inputs and outputs in a reaction).</p>\r\n<h3>Key data classes</h3>\r\n<h4><a href=\"/content/schema/\">PhysicalEntity</a></h4>\r\n<p>PhysicalEntities include individual molecules, multi-molecular complexes, and sets of molecules or complexes grouped together on the basis of shared characteristics. Molecules are further classified as genome encoded (DNA, RNA, and proteins) or not (all others). Attributes of a PhysicalEntity instance capture the chemical structure of an entity, including any covalent modifications in the case of a macromolecule, and its subcellular localization.</p>\r\n<p>PhysicalEntity instances that represent, e.g., the same chemical in different compartments, or different post-translationally modified forms of a single protein, share numerous invariant features such as names, molecular structure and links to external databases like UniProt or ChEBI. To enable storage of this shared information in a single place, and to create an explicit link among all the variant forms of what can also be seen as a single chemical entity, Reactome creates instances of the separate ReferenceEntity class. A&nbsp;<a href=\"/content/schema/ReferenceEntity\">ReferenceEntity</a>&nbsp;instance captures the invariant features of a molecule. A PhysicalEntity instance is then the combination of a ReferenceEntity attribute (e.g.,&nbsp;<a href=\"/content/detail/R-HSA-71580\">Glycogen phosphorylase UniProt:P06737</a>) and attributes giving specific conditional information (e.g., localization to the cytosol and phosphorylation on serine residue 14).</p>\r\n<p>The PhysicalEntity class has subclasses to distinguish between different kinds of entity and to ensure data integrity while enabling different handling rules for different categories:</p>\r\n<p><strong><a href=\"/content/schema/EntityWithAccessionedSequence\">EntityWithAccessionedSequence</a>&nbsp;</strong> proteins and nucleic acids with known sequences.</p>\r\n<p><strong><a href=\"/content/schema/GenomeEncodedEntity\">GenomeEncodedEntity</a>&nbsp;</strong> a species-specific protein or nucleic acid whose sequence is unknown, such as an enzyme that has been characterized functionally but not yet purified and sequenced, e.g.&nbsp;<a href=\"/content/detail/R-HSA-2161674\">cytosolic 15-HEDH enzyme</a></p>\r\n<p><strong><a href=\"/content/schema/SimpleEntity\">SimpleEntity</a>&nbsp;</strong> other fully characterized molecules, e.g.&nbsp;<a href=\"/content/detail/R-ALL-29358\">nucleoplasmic ATP</a>&nbsp;or&nbsp;<a href=\"/content/detail/R-ALL-29450\">cytosolic glutathione</a></p>\r\n<p><strong><a href=\"/content/schema/Complex\">Complex</a>&nbsp;</strong> a complex of two or more PhysicalEntities, e.g.&nbsp;<a href=\"/content/detail/R-HSA-71050\">Trimerization of the FASL:FAS receptor complex</a></p>\r\n<p><strong><a href=\"/content/schema/EntitySet\">EntitySet</a>&nbsp;</strong> a set of PhysicalEntities (molecules or complexes) which function interchangeably in a given situation, e.g.&nbsp;<a href=\"/content/detail/R-HSA-157644\">Notch 3 heterodimer binds with a Notch ligand in the extracellular space</a>. This notation allows collective properties of multiple individual entities to be described explicitly.</p>\r\n<p>PhysicalEntities are paired with molecular functions taken from the Gene Ontology molecular function controlled vocabulary to describe instances of biological catalysis. An optional ActiveUnit attribute indicates the specific domain of a protein or subunit of a complex that mediates the catalysis. If a PhysicalEntity has multiple catalytic activities, a separate CatalystActivity is created for each. This strategy allows the association of specific activities with specific variant forms of a protein or complex, and also enables easy retrieval of all activities of a protein, or all proteins capable of mediating a specific molecular function.</p>\r\n<h4><strong><a href=\"/content/schema/Event\">Event</a></strong></h4>\r\n<p>Events  the conversion of input entities to output entities in one or more steps  are the building blocks used in Reactome to represent all biological processes. Two subclasses of Event are recognized,&nbsp;<strong><a href=\"/content/schema/ReactionlikeEvent\">ReactionlikeEvent</a></strong>and&nbsp;<strong><a href=\"/content/schema/Pathway\">Pathway</a></strong>. A ReactionlikeEvent is an event that converts inputs into outputs. A Pathway is any grouping of related Events. An event may be a member of more than one Pathway.</p>\r\n<p>The ReactionlikeEvent class is further divided into&nbsp;<strong><a href=\"/content/schema/Reaction\">Reaction</a></strong>,&nbsp;<strong><a href=\"/content/schema/BlackBoxEvent\">BlackBoxEvent</a></strong>,&nbsp;<strong><a href=\"/content/schema/Polymerisation\">Polymerisation</a></strong>&nbsp;and&nbsp;<strong><a href=\"/content/schema/Depolymerisation\">Depolymerisation</a></strong>. The Reaction class holds bona fide reactions with balanced inputs and outputs. The BlackBoxEvent class is used for unbalanced reactions like protein synthesis or degradation, as well as shortcut reactions for more complex processes that essentially convert inputs into outputs, e.g. the series of cyclical reactions involved in fatty acid biosynthesis. The De-/Polymerisation classes can hold reactions that describe the mechanics of a de-/polymerisation reaction, which is inherently unbalanced due to the nature of a Polymer (that remains the same entity even after adding or subtracting a unit).</p>\r\n<h3>Full specification of the Reactome data model</h3>\r\n<p>A full specification of all Reactome classes, slots and a listing of all instances of each class is accessible from the&nbsp;<strong><a href=\"/content/schema/\">Schema</a></strong>&nbsp;page on the top menu bar. There is also a Data Model Glossary, giving more details on the usage of the various classes and slots.</p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-02 08:24:17','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(54,628,'Tutorial: Extracting participating molecules using the Graph Database','tutorial-participating-molecules','<style type=\"text/css\">.rcypher, .rtip, .rresult {\r\n	border: 1px solid #dddddd;\r\n	border-radius: 5px;\r\n	background: #f9f9f9;\r\n 	margin-left: 15px;\r\n 	margin-right: 15px;\r\n	overflow-x: scroll;\r\n	padding: 10px;\r\n}\r\n.rtip {\r\n 	border: 1px solid pink;\r\n 	background-color: #ffe6e6;\r\n}\r\n.rresult{\r\n 	font-family: monospace;\r\n 	font-size: x-small;\r\n 	color: darkgray;\r\n        word-wrap: initial;jq\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n.rcypher code, .rtip code {\r\n        word-wrap: initial;\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n@media only screen and (max-width: 768px) {\r\n    .rcypher, .rtip, .rresult {\r\n 	margin-left: 0px;\r\n        margin-right: 0px;\r\n    }\r\n    .rresult {\r\n        overflow-y: auto;\r\n        white-space: pre;\r\n    }\r\n}</style>\r\n<p>The Reactome Graph Database, also called a graph-oriented database, is a type of&nbsp;NoSQL&nbsp;database that uses&nbsp;graph theory&nbsp;to store, map and query relationships relating to our data content. Each node represents an entity (such as a pathway, reaction or proteins) and each edge represents a relationship between two nodes. Every node in our graph database is defined by a&nbsp;unique identifier, a set of outgoing edges and/or incoming edges and a set of properties expressed as&nbsp;key/value pairs. Each edge is defined by a unique identifier, a starting-place and/or ending-place node and a set of properties.</p>\r\n<p><a href=\"http://neo4j.com/developer/cypher/\" target=\"_blank\" rel=\"noopener\">Cypher</a> is Neo4js open graph query language. Cyphers syntax provides a familiar way to match patterns of nodes and relationships in a graph. If you want to learn more about what is Cypher, please visit the following <a href=\"http://neo4j.com/docs/developer-manual/current/#cypher-introduction\" target=\"_blank\" rel=\"noopener\">link</a>.</p>\r\n<p>Graph databases are well-suited for analyzing interconnections, which is why there has been a lot of interest in using graph databases to&nbsp;mine data&nbsp;from&nbsp;biological pathways and reactions. The Reactome Graph database has many advantages, but one is its responsiveness in managing data. Furthermore, even though data queries increase exponentially, the performance of a graph database&nbsp;does not drop, compared to what happens with relational databases. When software developers work with data, they are looking for flexibility and scalability. Our Graph Database contributes a lot in this regard because when needs increase,&nbsp;the possibilities of adding more nodes and relationships to an existing graph are huge.</p>\r\n<h3><a href=\"#participating-molecules-use-case\" id=\"participating-molecules-use-case\">The participating molecules use case</a></h3>\r\n<p>This tutorial explains how to query Reactome using Cypher. It\'s assumed that the reader has a basic knowledge of Cypher as well as an understanding of our&nbsp;<a href=\"/documentation/data-model/\">data model</a> and how data is stored in our&nbsp;<a href=\"/content/schema/\">schema</a>.</p>\r\n<p>Even though it is not possible to cover all possible queries of the Reactome Graph Database in a single tutorial, the sections in this document build up a query, which will retrieve the resource and identifier of each participating molecule of a given Pathway. There are several intermediate stages to be explained before reaching that point:</p>\r\n<ol>\r\n<li>How to retrieve objects like proteins, reactions, pathways, etc.</li>\r\n<li>How to get the identifier of proteins or chemicals</li>\r\n<li>How to deconstruct complexes or sets to get their participants</li>\r\n<li>How to retrieve the subpathways for a given pathway</li>\r\n<li>How to retrieve the reactions of a pathway</li>\r\n<li>How to retrieve the participants of a reaction</li>\r\n</ol>\r\n<p>The enumeration above represents the <b>basic bricks</b> from which to construct the final query that retrieves the <b>participating molecules</b> for a given <b>pathway</b>.</p>\r\n<h3><a href=\"#retrieving-objects\" id=\"retrieving-objects\">Retrieving objects based on their identifier</a></h3>\r\n<p>To retrieve the Pathway \"<b>Antigen processing-Cross presentation</b>\" with identifier <b>R-HSA-1236975</b>, the query is as follows:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Selecting an Pathway by its stable identifier\r\nMATCH (pathway:Pathway{stId:\"R-HSA-1236975\"})\r\nRETURN pathway\r\n</code></pre>\r\n<p>The result of the query is:</p>\r\n<pre class=\"rresult\">\r\npathway                                                                                                              \r\n\r\n{speciesName: Homo sapiens, oldStId: REACT_111119, isInDisease: false, releaseDate: 2011-09-20, displayName: Antigen \r\n processing-Cross presentation, stIdVersion: R-HSA-1236975.1, dbId: 1236975, releaseStatus: UPDATED, name: [Antigen  \r\n processing-Cross presentation], stId: R-HSA-1236975, hasDiagram: false, isInferred: false}                          \r\n\r\n</pre>\r\n<p>In the same way, let\'s focus on an EntityWithAccessionedSequence (EWAS) which corresponds to a protein in Reactome. For this example we use one form of <b>PTEN</b> in the <b>cytosol</b> with identifier <b>R-HSA-199420</b></p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Selecting an EWAS by its stable identifier\r\nMATCH (ewas:EntityWithAccessionedSequence{stId:\"R-HSA-199420\"})\r\nRETURN ewas\r\n</code></pre>\r\n<p>The result is one node of the database:</p>\r\n<pre class=\"rresult\">\r\newas                                                                                                                        \r\n\r\n{speciesName: Homo sapiens, startCoordinate: 1, isInDisease: false, displayName: PTEN [cytosol], dbId: 199420, name: [PTEN, \r\n Phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase PTEN, PTEN_HUMAN, MMAC1, TEP1], referenceType: ReferenceGeneProduct,\r\n endCoordinate: 403, stId: R-HSA-199420}                                                                                    \r\n\r\n</pre>\r\n<h3><a href=\"#identifiers-proteins-or-chemicals\" id=\"identifiers-proteins-or-chemicals\">Identifiers for proteins or chemicals</a></h3>\r\n<p>Continuing with the same form of PTEN, another use case could be to retrieve only a couple of fields for the target node. The following query retrieves the EWAS <b>display name</b> and its <b>identifier</b> (from the reference entity):</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Following the reference entity link in order to get the identifier\r\nMATCH (ewas:EntityWithAccessionedSequence{stId:\"R-HSA-199420\"}),\r\n      (ewas)-[:referenceEntity]-&gt;(re:ReferenceEntity)\r\nRETURN ewas.displayName AS EWAS, re.identifier AS Identifier\r\n</code></pre>\r\n<p>Please note that the <b>identifier</b> is not directly stored in the node for the EWAS but is a property of another node pointed from the EWAS which is a <b>ReferenceEntity</b>. The previous query accesses that node from the EWAS following the <b>referenceEntity</b> edge. The result of this query is as shown below:</p>\r\n<pre class=\"rresult\">\r\nEWAS          Identifier\r\n\r\nPTEN [cytosol]P60484    \r\n\r\n</pre>\r\n<p>TIP: The <i>MATCH</i> part of the previous query could be written in one line as follows:</p>\r\n<pre class=\"rtip\"><code class=\"language-cypher\" data-lang=\"cypher\">//Equivalent to the previous one\r\nMATCH (ewas:EntityWithAccessionedSequence{stId:\"R-HSA-199420\"})-[:referenceEntity]-&gt;(re:ReferenceEntity)\r\nRETURN ewas.displayName AS EWAS, re.identifier AS Identifier\r\n</code></pre>\r\n<p>Continuing on, it is possible to construct a query to retrieve the <b>reference database</b> on top of the previously retrieved fields. Please note that in this case the <b>reference database</b> is a <b>node</b> pointed from <b>ReferenceEntity</b> by an edge called <b>referenceDatabase</b>:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Following the reference entity and database links in order to get the identifier and the database of reference\r\nMATCH (ewas:EntityWithAccessionedSequence{stId:\"R-HSA-199420\"}),\r\n      (ewas)-[:referenceEntity]-&gt;(re:ReferenceEntity)-[:referenceDatabase]-&gt;(rd:ReferenceDatabase)\r\nRETURN ewas.displayName AS EWAS, re.identifier AS Identifier, rd.displayName AS Database\r\n</code></pre>\r\n<pre class=\"rresult\">\r\nEWAS          IdentifierDatabase\r\n\r\nPTEN [cytosol]P60484    UniProt \r\n\r\n</pre>\r\n<h3><a href=\"#complexes-sets-participants\" id=\"complexes-sets-participants\">Breaking down complexes and sets to get their participants</a></h3>\r\n<p>The components of a complex, which are also physical entities, are stored in the \"hasComponent\" slot. Let\'s use the complex \"<b>Ag-substrate:E3:E2:Ub</b>\" with identifier <b>R-HSA-983126</b> as example in this case:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//First level components for the complex with stable identifier R-HSA-983126\r\nMATCH (Complex{stId:\"R-HSA-983126\"})-[:hasComponent]-&gt;(pe:PhysicalEntity)\r\nRETURN pe.stId AS component_stId, pe.displayName AS component\r\n</code></pre>\r\n<p>The result of the query is</p>\r\n<pre class=\"rresult\">\r\ncomponent_stIdcomponent                                      \r\n\r\nR-NUL-983035  antigenic substrate [cytosol]                  \r\n\r\nR-HSA-976075  E3 ligases in proteasomal degradation [cytosol]\r\n\r\nR-HSA-976165  Ubiquitin:E2 conjugating enzymes [cytosol]     \r\n\r\n</pre>\r\n<p>In this example, the \"<b>E3 ligases in proteasomal degradation</b>\" is a Set and \"<b>E3 ligases in proteasomal degradation</b>\" is a Complex. To further deconstruct the initial complex there are some minor changes which should be applied to the Cypher query. Sets can either be DefineSets, OpenSets or CandidateSets. The way to find out which other physical entities are part of them is \"traversing\" through the \"<b>hasMember</b>\" or \"<b>hasCandidate</b>\" slots. The following query will break down the initial complex into ALL its participants:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//All distinct components for the complex with stable identifier R-HSA-983126\r\nMATCH (Complex{stId:\"R-HSA-983126\"})-[:hasComponent|hasMember|hasCandidate*]-&gt;(pe:PhysicalEntity)\r\nRETURN DISTINCT pe.stId AS component_stId, pe.displayName AS component\r\n</code></pre>\r\n<p>This query returns 284 entities for v63:</p>\r\n<pre class=\"rresult\" style=\"height: 250px;\">\r\n component_stId  component              \r\n\r\n R-HSA-141412    CDC20 [cytosol]        \r\n\r\n R-HSA-174242    ANAPC7 [cytosol]       \r\n\r\n R-HSA-174211    ANAPC5 [cytosol]       \r\n\r\n R-HSA-174052    CDC26 [cytosol]        \r\n\r\n R-HSA-174244    UBE2C [cytosol]        \r\n\r\n R-HSA-174126    ANAPC11 [cytosol]      \r\n\r\n R-HSA-174156    CDC16 [cytosol]        \r\n\r\n R-HSA-174189    ANAPC1 [cytosol]       \r\n\r\n R-HSA-174100    UBE2E1 [cytosol]       \r\n\r\n R-HSA-174229    ANAPC2 [cytosol]       \r\n\r\n R-HSA-174168    ANAPC4 [cytosol]       \r\n\r\n R-HSA-174073    CDC27 [cytosol]        \r\n\r\n R-HSA-174137    CDC23 [cytosol]        \r\n\r\n R-HSA-174142    ANAPC10 [cytosol]      \r\n\r\n R-HSA-174236    UBE2D1 [cytosol]       \r\n\r\n R-HSA-976009    CBLB [cytosol]         \r\n\r\n R-HSA-976042    MKRN1 [cytosol]        \r\n\r\n R-HSA-939214    UBB(1-76) [cytosol]    \r\n\r\n R-HSA-939213    UBB(77-152) [cytosol]  \r\n\r\n R-HSA-939239    UBC(533-608) [cytosol] \r\n\r\n...             ...                     \r\n\r\n</pre>\r\n<h3><a href=\"#retrieving-pathways\" id=\"retrieving-pathways\">Retrieving pathways, subpathways and superpathways</a></h3>\r\n<p>In this example, we focus on the pathway \"<b>Class I MHC mediated antigen processing &amp; presentation</b>\" with identifier <b>R-HSA-983169</b>. To find out its subpathways, the slot to query is \"<b>hasEvent</b>\":</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Direct subpathways for the pathway with stable identifier R-HSA-198933\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})-[:hasEvent]-&gt;(sp:Pathway)\r\nRETURN p.stId AS Pathway, sp.stId AS SubPathway, sp.displayName as DisplayName\r\n</code></pre>\r\n<p>The result for v63 returns 3 subpathways:</p>\r\n<pre class=\"rresult\">\r\nPathway     SubPathway   DisplayName                                                               \r\n\r\nR-HSA-983169R-HSA-983168 Antigen processing: Ubiquitination &amp; Proteasome degradation               \r\n\r\nR-HSA-983169R-HSA-1236975Antigen processing-Cross presentation                                     \r\n\r\nR-HSA-983169R-HSA-983170 Antigen Presentation: Folding, assembly and peptide loading of class I MHC\r\n\r\n</pre>\r\n<p>It is important to note that subpathways might contain other subpathways, so to get ALL the supathways of R-HSA-198933, the query is as follows:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//ALL subpathways for the pathway with stable identifier R-HSA-198933\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})-[:hasEvent*]-&gt;(sp:Pathway)\r\nRETURN p.stId AS Pathway, sp.stId AS SubPathway, sp.displayName as DisplayName\r\n</code></pre>\r\n<p>In this case the number of subpathways is increased to 7:</p>\r\n<pre class=\"rresult\">\r\nPathway     SubPathway   DisplayName                                                               \r\n\r\nR-HSA-983169R-HSA-983170 Antigen Presentation: Folding, assembly and peptide loading of class I MHC\r\n\r\nR-HSA-983169R-HSA-1236975Antigen processing-Cross presentation                                     \r\n\r\nR-HSA-983169R-HSA-1236978Cross-presentation of soluble exogenous antigens (endosomes)              \r\n\r\nR-HSA-983169R-HSA-1236977Endosomal/Vacuolar pathway                                                \r\n\r\nR-HSA-983169R-HSA-1236974ER-Phagosome pathway                                                      \r\n\r\nR-HSA-983169R-HSA-1236973Cross-presentation of particulate exogenous antigens (phagosomes)         \r\n\r\nR-HSA-983169R-HSA-983168 Antigen processing: Ubiquitination &amp; Proteasome degradation               \r\n\r\n</pre>\r\n<p>Following the same approach, retrieving the superpathway is as easy as changing the direction of the edge in the query:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Direct superpathway for the pathway with stable identifier R-HSA-198933\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})&lt;-[:hasEvent]-(sp:Pathway)\r\nRETURN p.stId AS Pathway, sp.stId AS SuperPathway, sp.displayName as DisplayName\r\n</code></pre>\r\n<p>It will then retrieve the only one pathway containing R-HSA-198933:</p>\r\n<pre class=\"rresult\">\r\nPathway     SuperPathway DisplayName           \r\n\r\nR-HSA-983169R-HSA-1280218Adaptive Immune System\r\n\r\n</pre>\r\n<p>As subpathways, the superpathways might have other superpathways, so following the \"<b>hasEvent</b>\" slot recursively will show ALL the superpathways up to the root:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//ALL superpathways for the pathway with stable identifier R-HSA-198933\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})&lt;-[:hasEvent*]-(sp:Pathway)\r\nRETURN p.stId AS Pathway, sp.stId AS SuperPathway, sp.displayName as DisplayName\r\n</code></pre>\r\n<p>There are 2 superpathways for R-HSA-198933 in v63:</p>\r\n<pre class=\"rresult\">\r\nPathway     SuperPathway DisplayName           \r\n\r\nR-HSA-983169R-HSA-1280218Adaptive Immune System\r\n\r\nR-HSA-983169R-HSA-168256 Immune System         \r\n\r\n</pre>\r\n<h3><a href=\"#retrieving-reactions\" id=\"retrieving-reactions\">Retrieving the reactions for a given pathway</a></h3>\r\n<p>Continuing with the pathway \"<b>Class I MHC mediated antigen processing &amp; presentation</b>\" with identifier <b>R-HSA-983169</b>, to get ALL the reactions contained either directly in it or as part of any of its subpathways, the query has to recursively traverse the \"<b>hasEvent</b>\" slot:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//All reactions for the pathway with stable identifier R-HSA-198933\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})-[:hasEvent*]-&gt;(rle:ReactionLikeEvent)\r\nRETURN p.stId AS Pathway, rle.stId AS Reaction, rle.displayName AS ReactionName\r\n</code></pre>\r\n<p>As shown in the table below, this pathway contains 51 reactions for v63:</p>\r\n<pre class=\"rresult\" style=\"height: 250px;\">\r\n Pathway       Reaction       ReactionName                                                                      \r\n\r\n R-HSA-983169  R-HSA-983148   Interaction of Erp57 with MHC class I HC                                          \r\n\r\n R-HSA-983169  R-HSA-8951499  Loading of antigenic peptides on to class I MHC                                   \r\n\r\n R-HSA-983169  R-HSA-983146   Interaction of beta-2-microglobulin (B2M) chain with  class I HC                  \r\n\r\n R-HSA-983169  R-HSA-983145   Binding of newly synthesized MHC class I heavy chain (HC) with calnexin           \r\n\r\n R-HSA-983169  R-HSA-983144   Transport of Antigen peptide in to ER                                             \r\n\r\n R-HSA-983169  R-HSA-203979   Coat Assembly                                                                     \r\n\r\n R-HSA-983169  R-HSA-983142   Formation of peptide loading complex (PLC)                                        \r\n\r\n R-HSA-983169  R-HSA-983427   Expression of peptide bound class I MHC on cell surface                           \r\n\r\n R-HSA-983169  R-HSA-983138   Transport of MHC heterotrimer to ER exit site                                     \r\n\r\n R-HSA-983169  R-HSA-983426   Capturing cargo and formation of prebudding complex                               \r\n\r\n R-HSA-983169  R-HSA-983425   Recruitment of Sec31p:Sec13p to prebudding complex and formation of COPII vesicle \r\n\r\n R-HSA-983169  R-HSA-983424   Budding of COPII coated vesicle                                                   \r\n\r\n R-HSA-983169  R-HSA-983422   Disassembly of COPII coated vesicle                                               \r\n\r\n R-HSA-983169  R-HSA-983421   Journey of cargo through Golgi complex                                            \r\n\r\n R-HSA-983169  R-HSA-983161   Dissociation of the Antigenic peptide:MHC:B2M peptide loading complex             \r\n\r\n...           ...            ...                                                                                \r\n\r\n</pre>\r\n<h3><a href=\"#retrieving-participants\" id=\"retrieving-participants\">Retrieving the participants of a Reaction</a></h3>\r\n<p>In this case let\'s use the reaction \"<b>IKKB phosphorylates SNAP23</b>\" with identifier <b>R-HSA-8863895</b>. Reactions have inputs, outputs, catalysts and regulations, so to know the participants of a reaction, all these slots have to be taken into account. Please note that the physical entity acting as catalyst is stored in the \"physicalEntity\" slot of the class \"CatalystActivity\" and the one belonging to the regulation is stored in the \"regulator\" slot of the \"Regulation\" class. So the query is as follows:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//First level paticipating molecules for reaction R-HSA-8863895\r\nMATCH (r:ReactionLikeEvent{stId:\"R-HSA-8863895\"})-[:input|output|catalystActivity|physicalEntity|regulatedBy|regulator*]-&gt;(pe:PhysicalEntity)\r\nRETURN DISTINCT r.stId AS Reaction, pe.stId as Participant, pe.displayName AS DisplayName\r\n</code></pre>\r\n<p>The result of it is 6 physical entities, where two of them are complexes:</p>\r\n<pre class=\"rresult\">\r\nReaction     Participant  DisplayName                                        \r\n\r\nR-HSA-8863895R-HSA-168113 CHUK:IKBKB:IKBKG [cytosol]                         \r\n\r\nR-HSA-8863895R-ALL-113592 ATP [cytosol]                                      \r\n\r\nR-HSA-8863895R-HSA-8863966SNAP23 [phagocytic vesicle membrane]               \r\n\r\nR-HSA-8863895R-HSA-8863923p-S95-SNAP23 [phagocytic vesicle membrane]         \r\n\r\nR-HSA-8863895R-ALL-29370  ADP [cytosol]                                      \r\n\r\nR-HSA-8863895R-HSA-937033 oligo-MyD88:Mal:BTK:activated TLR [plasma membrane]\r\n\r\n</pre>\r\n<p>As shown above, to break down complexes and sets into their participants, the \"hasComponent\", \"hasMember\" and \"hasCandidate\" slots have to be taken into account. Adding them into the previous query will retrieve ALL the participants of the reaction:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//ALL paticipating molecules for reaction R-HSA-8863895\r\nMATCH (r:ReactionLikeEvent{stId:\"R-HSA-8863895\"})-[:input|output|catalystActivity|physicalEntity|regulatedBy|regulator|hasComponent|hasMember|hasCandidate*]-&gt;(pe:PhysicalEntity)\r\nRETURN DISTINCT r.stId AS Reaction, pe.stId as Participant, pe.displayName AS DisplayName\r\n</code></pre>\r\n<p>With the modifications, the result of the query goes up to 43 physical entities in v63:</p>\r\n<pre class=\"rresult\" style=\"height: 250px;\">\r\n Reaction       Participant    DisplayName                                                                  \r\n\r\n R-HSA-8863895  R-HSA-168113   CHUK:IKBKB:IKBKG [cytosol]                                                   \r\n\r\n R-HSA-8863895  R-HSA-168114   IKBKB [cytosol]                                                              \r\n\r\n R-HSA-8863895  R-HSA-168104   CHUK [cytosol]                                                               \r\n\r\n R-HSA-8863895  R-HSA-168108   IKBKG [cytosol]                                                              \r\n\r\n R-HSA-8863895  R-ALL-113592   ATP [cytosol]                                                                \r\n\r\n R-HSA-8863895  R-HSA-8863966  SNAP23 [phagocytic vesicle membrane]                                         \r\n\r\n R-HSA-8863895  R-HSA-8863923  p-S95-SNAP23 [phagocytic vesicle membrane]                                   \r\n\r\n R-HSA-8863895  R-ALL-29370    ADP [cytosol]                                                                \r\n\r\n R-HSA-8863895  R-HSA-937033   oligo-MyD88:Mal:BTK:activated TLR [plasma membrane]                          \r\n\r\n R-HSA-8863895  R-HSA-937013   MyD88 oligomer [plasma membrane]                                             \r\n\r\n R-HSA-8863895  R-HSA-937017   MYD88 [plasma membrane]                                                      \r\n\r\n R-HSA-8863895  R-HSA-2201325  activated TLR2/4:p-4Y-MAL:PI(4,5)P2:BTK [plasma membrane]                    \r\n\r\n R-HSA-8863895  R-HSA-5365824  p-4Y-TIRAP:PI(4,5)P2 [plasma membrane]                                       \r\n\r\n R-HSA-8863895  R-ALL-179856   PI(4,5)P2 [plasma membrane]                                                  \r\n\r\n R-HSA-8863895  R-HSA-2201321  p-4Y-TIRAP [plasma membrane]                                                 \r\n\r\n R-HSA-8863895  R-HSA-181230   Activated TLR1:2 or TLR 2:6 heterodimers or TLR4 homodimer [plasma membrane] \r\n\r\n R-HSA-8863895  R-HSA-181410   TLR6:TLR2:ligand:CD14:CD36 [plasma membrane]                                 \r\n\r\n R-HSA-8863895  R-HSA-2559461  TLR6/2 ligand:CD14:CD36 [plasma membrane]                                    \r\n\r\n R-HSA-8863895  R-HSA-166033   GPIN-CD14(20-345) [plasma membrane]                                          \r\n\r\n...            ...            ...                                                                           \r\n\r\n</pre>\r\n<h3><a href=\"#joining-pieces\" id=\"joining-pieces\">Joining the pieces: Participating molecules for a pathway</a></h3>\r\n<p>The aim of this tutorial was to describe a number of different queries to the Reactome Graph Database that joined together would retrieve the resource and identifier of each participating molecule for a given Pathway. This final example will demonstrate how to concatenate all the individual query described previously into a single query.</p>\r\n<p>Starting from the pathway \"<b>Class I MHC mediated antigen processing &amp; presentation</b>\" with identifier <b>R-HSA-983169</b>, first we need to find out all the reactions contained in it. For each reaction we want to find their participants and for the cases of complexes and sets, we want to break them down into single physical entities. Finally, for each physical entity we are interested in their identifier and resource:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//ALL paticipating molecules for pathway R-HSA-983169\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})-[:hasEvent*]-&gt;(rle:ReactionLikeEvent),\r\n      (rle)-[:input|output|catalystActivity|physicalEntity|regulatedBy|regulator|hasComponent|hasMember|hasCandidate*]-&gt;(pe:PhysicalEntity),\r\n      (pe)-[:referenceEntity]-&gt;(re:ReferenceEntity)-[:referenceDatabase]-&gt;(rd:ReferenceDatabase)\r\nRETURN DISTINCT re.identifier AS Identifier, rd.displayName AS Database\r\n</code></pre>\r\n<p>For version 63, the pathway has 463 participating molecules as shown below:</p>\r\n<pre class=\"rresult\" style=\"height: 250px;\">\r\n Identifier  Database \r\n\r\n P11021      UniProt  \r\n\r\n P27824      UniProt  \r\n\r\n P30501      UniProt  \r\n\r\n P30486      UniProt  \r\n\r\n P01893      UniProt  \r\n\r\n P30447      UniProt  \r\n\r\n P30685      UniProt  \r\n\r\n P18465      UniProt  \r\n\r\n P18464      UniProt  \r\n\r\n P30460      UniProt  \r\n\r\n P30490      UniProt  \r\n\r\n P30495      UniProt  \r\n\r\n P30493      UniProt  \r\n\r\n P13747      UniProt  \r\n\r\n P30488      UniProt  \r\n\r\n P30511      UniProt  \r\n\r\n P30483      UniProt  \r\n\r\n P30462      UniProt  \r\n\r\n P04439      UniProt  \r\n\r\n ...         ...      \r\n\r\n</pre>\r\n<p>This concludes the introductory tutorial on how to build a Cypher query to retrieve all the participating molecules for a given pathway. For questions or suggestions, please get in touch our <a href=\"mailto:help@reactome.org\">help desk</a></p>\r\n<style type=\"text/css\">@import url(\'https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.11.0/codemirror.min.css\');\r\n@import url(\'https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.11.0/theme/neo.min.css\');\r\n@import url(\'https://fonts.googleapis.com/css?family=Open+Sans:400,300,400italic,600,300italic\');</style>\r\n<script src=\"https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.11.0/codemirror.min.js\" type=\"text/javascript\"></script>\r\n<script src=\"https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.11.0/addon/runmode/runmode.min.js\" type=\"text/javascript\"></script>\r\n<script src=\"https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.11.0/mode/cypher/cypher.min.js\" type=\"text/javascript\"></script>\r\n<script src=\"https://neo4j.com/docs/developer-manual/current/javascript/colorize.js\" type=\"text/javascript\"></script>\r\n<script type=\"text/javascript\">function tabTheSource($content) {\r\n    var storedLanguage = getCodeExampleLanguage();\r\n    var LANGUAGES = {\r\n        \'dotnet\': \'C#\',\r\n        \'java\': \'Java\',\r\n        \'javascript\': \'JavaScript\',\r\n        \'python\': \'Python\'\r\n    };\r\n    var $UL = jQuery(\'<ul class=\"nav nav-tabs\" role=\"tablist\"/>\');\r\n    var $LI = jQuery(\'<li role=\"presentation\"/>\');\r\n    var $A = jQuery(\'<a role=\"tab\" data-toggle=\"tab\" style=\"text-decoration:none;\"/>\');\r\n    var $WRAPPER = jQuery(\'<div class=\"tab-content content\"/>\');\r\n    var snippets = [];\r\n    var languageEventElements = {};\r\n\r\n    var focusSelectedExample = function(e) {\r\n        var target = jQuery(e.target);\r\n        var beforeTop = target.offset().top - jQuery(window).scrollTop();\r\n        setTimeout(function(){\r\n            var newTop = target.offset().top - beforeTop;\r\n            jQuery(\'html,body\').scrollTop(newTop);\r\n        }, 1);\r\n    }\r\n\r\n    var selectTab = function (e) {\r\n        var language = jQuery(e.target).data(\'lang\');\r\n        var $elements = languageEventElements[language];\r\n        for (var j = 0; j < $elements.length; j++) {\r\n            $elements[j].tab(\'show\');\r\n        }\r\n        if (storageAvailable(\'sessionStorage\')) {\r\n            sessionStorage.setItem(\'code_example_language\', language);\r\n        }\r\n    }\r\n\r\n    jQuery(\'div.tabbed-example\', $content).each(function () {\r\n        var $exampleBlock = $(this);\r\n        var title = $exampleBlock.children(\'div.example-title\', this).first().text();\r\n        var languages = [];\r\n        var $languageBlocks = {};\r\n        jQuery(this).children(\'div.tabbed-example-contents\').children(\'div.listingblock,div.informalexample[class*=\"include-with\"]\').each(function () {\r\n            var $this = jQuery(this);\r\n            var language = undefined;\r\n            if ($this.hasClass(\'listingblock\')) {\r\n                language = jQuery(\'code\', this).data(\'lang\');\r\n            } else {\r\n                for (var key in LANGUAGES) {\r\n                    if ($this.hasClass(\'include-with-\' + key)) {\r\n                        language = key;\r\n                        break;\r\n                    }\r\n                }\r\n            }\r\n            languages.push(language);\r\n            $languageBlocks[language] = jQuery(this);\r\n        });\r\n        if (languages.length > 1) {\r\n            snippets.push({\r\n                \'$exampleBlock\': $exampleBlock,\r\n                \'languages\': languages,\r\n                \'$languageBlocks\': $languageBlocks\r\n            });\r\n        }\r\n    });\r\n\r\n    var idNum = 0;\r\n    for (var ix = 0; ix < snippets.length; ix++) {\r\n        var snippet = snippets[ix];\r\n        var languages = snippet.languages;\r\n        languages.sort();\r\n        var $languageBlocks = snippet.$languageBlocks;\r\n        var $exampleBlock = snippet.$exampleBlock;\r\n        var idBase = \'tabbed-example-\' + idNum++;\r\n        var $wrapper = $WRAPPER.clone();\r\n        var $ul = $UL.clone();\r\n\r\n        for (var i = 0; i < languages.length; i++) {\r\n            var language = languages[i];\r\n            var $content = jQuery($languageBlocks[language]);\r\n            var id;\r\n            if ($content.attr(\'id\')) {\r\n                id = $content.attr(\'id\');\r\n            } else {\r\n                id = idBase + \'-\' + language;\r\n                $content.attr(\'id\', id);\r\n            }\r\n            $content.addClass(\'tab-pane\').css(\'position\', \'relative\');\r\n            var $li = $LI.clone();\r\n            var $a = $A.clone();\r\n\r\n            $a.attr(\'href\', \'#\' + id).text(LANGUAGES[language]).data(\'lang\', language).on(\'shown.bs.tab\', selectTab).on(\'click\', focusSelectedExample);\r\n\r\n            if (language in languageEventElements) {\r\n                languageEventElements[language].push($a);\r\n            } else {\r\n                languageEventElements[language] = [$a];\r\n            }\r\n            $wrapper.append($content);\r\n\r\n            if (storedLanguage) {\r\n                if (language === storedLanguage) {\r\n                    $li.addClass(\'active\');\r\n                    $content.addClass(\'active\');\r\n                }\r\n            } else if (i === 0) {\r\n                $li.addClass(\'active\');\r\n                $content.addClass(\'active\');\r\n            }\r\n\r\n            $li.append($a);\r\n            $ul.append($li);\r\n        }\r\n        $exampleBlock.children(\'div.example-title\', this).first().after($ul);\r\n        $exampleBlock.append($wrapper);\r\n    }\r\n}\r\n\r\nfunction storageAvailable(type) {\r\n    try {\r\n        var storage = window[type];\r\n        var x = \'__storage_test__\';\r\n        storage.setItem(x, x);\r\n        storage.removeItem(x);\r\n        return true;\r\n    }\r\n    catch(e) {\r\n        return false;\r\n    }\r\n}\r\n\r\nfunction getCodeExampleLanguage() {\r\n    return storageAvailable(\'sessionStorage\') ? sessionStorage.getItem(\'code_example_language\') || false : false;\r\n}</script>\r\n<script type=\"text/javascript\">jQuery(document).ready(function($){\r\nCodeMirror.colorize();\r\ntabTheSource(jQuery(\'#rcypher\'));\r\n});</script>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-02 15:08:40','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,7,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(55,630,'Icons Info ','icons-info','<p>{loadposition icon-lib-pos-1}</p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-05 15:23:33','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"0\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','','Page with 2 options - EHLDs and Icon Library'),(56,631,'Pathways Overview JS','pathways-overview-js','<style type=\"text/css\">pre {\r\n    word-wrap: initial;\r\n    white-space: pre;\r\n    word-break: break-all;\r\n}\r\n}</style>\r\n<div>Reactome\'s Fireworks JS widget is our pathways overview viewer in an ordinary JavaScript API. It is meant to be reused by third party resources in order to display Reactome pathways overview directly in their web pages and enable users to interact with them.</div>\r\n<p>&nbsp;</p>\r\n<div id=\"fireworksHolder\" class=\"diagram\"></div>\r\n<div>\r\n<h3><a href=\"#reuse-pathways-widget\" id=\"reuse-pathways-widget\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Reusing Reactome\'s Pathways Overview Widget?</a></h3>\r\n<p>To reuse our viewer you need to follow the following steps</p>\r\n<p>1. Include the fireworks javascript dependency in you HTML header</p>\r\n<pre class=\"code\">    &lt;script type=\"text/javascript\" language=\"javascript\" src=\"https://reactome.org/FireworksJs/fireworks/fireworks.nocache.js\"&gt;&lt;/script&gt;\r\n        </pre>\r\n<p>2. Add a place holder in the body of your web page</p>\r\n<pre class=\"prettyprint code\">    &lt;div id=\"fireworksHolder\"&gt;&lt;/div&gt;\r\n        </pre>\r\n<p>3. Set a proxy in your server under \"/reactome\" pointing to \"https://reactome.org\"</p>\r\n<p>4. Create and initialise the pathways overview from your javascript code</p>\r\n<pre class=\"prettyprint code\">    //Creating the Reactome pathways overview widget\r\n    function onReactomeFireworksReady(){  //This function is automatically called when the widget code is ready to be used\r\n        var fireworks = Reactome.Fireworks.create({\r\n            \"placeHolder\" : \"fireworksHolder\",\r\n            \"width\" : 930,\r\n            \"height\" : 500\r\n        });\r\n\r\n        //Adding different listeners\r\n\r\n        fireworks.onFireworksLoaded(function (loaded) {\r\n            console.info(\"Loaded \", loaded);\r\n        });\r\n\r\n        fireworks.onNodeHovered(function (hovered){\r\n            console.info(\"Hovered \", hovered);\r\n        });\r\n\r\n        fireworks.onNodeSelected(function (selected){\r\n            console.info(\"Selected \", selected);\r\n        });\r\n    }\r\n        </pre>\r\n<a>&nbsp;</a>\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>FireworksJs API</a></h3>\r\n<p>The current implementation supports the following listeners and methods:</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome\">\r\n<thead>\r\n<tr><th>Method</th><th>Params</th><th>Description</th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td data-label=\"Method\"><b>create</b> :: Constructor<br /> &nbsp;&nbsp;Reactome.Fireworks.create(params);</td>\r\n<td data-label=\"Params\"><b>param</b> :: json object<br /> &nbsp;{<br /> &nbsp;&nbsp;&nbsp;\'proxyPrefix\' : string,<br /> &nbsp;&nbsp;&nbsp;\'placeHolder\' : string,<br /> &nbsp;&nbsp;&nbsp;\'width\' : int (optional),<br /> &nbsp;&nbsp;&nbsp;\'height\' : int (optional)<br /> &nbsp;}</td>\r\n<td data-label=\"Description\">Creates and returns a new Reactome.Fireworks object</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>flagItems(String identifier)</b> :: void</td>\r\n<td data-label=\"Params\">Item identifier <br /> <b>identifier</b> : string</td>\r\n<td data-label=\"Description\">Flags pathways where the identifier is found. It accepts main identifiers but also cross references, gene names and physical entity stable identifiers</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>highlightNode(stId)</b> :: void</td>\r\n<td data-label=\"Params\">Item stable identifier<br /> <b>stId</b> : string</td>\r\n<td data-label=\"Description\">Highlights the specified item if it exists in the fireworks</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetAnalysis()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Resets the analysis overlay</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetHighlight()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Clears the highlights in the fireworks</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetSelection()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Clears the selection in the fireworks</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resize(width, height)</b> :: void</td>\r\n<td data-label=\"Params\"><b>width</b>: int<br /> <b>height</b>: int</td>\r\n<td data-label=\"Description\">Resizes the viewport to the specified with and height</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>selectNode(stId)</b> :: void</td>\r\n<td data-label=\"Params\">Item stable identifier<br /> <b>stId</b> : string</td>\r\n<td data-label=\"Description\">Selects the specified item if it exists in the fireworks</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>setAnalysisToken(token, resource)</b> :: void</td>\r\n<td data-label=\"Params\">Analysis token<br /> <b>token</b> : string<br /> Resource<br /> <b>resource</b> : string</td>\r\n<td data-label=\"Description\">Overlays the analysis result corresponding to the specified (token, resource)</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>showAll()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Shows all the pathways in the viewport</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onNodesFlaggedReset(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters</td>\r\n<td data-label=\"Description\">The function is called when the user resets the flagged items</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onNodeSelected(function(obj))</b> :: void</td>\r\n<td data-label=\"Params\"><b>obj</b> is selected item:<br /> &nbsp;{<br /> &nbsp;&nbsp;&nbsp;\'stId\' : string,<br /> &nbsp;&nbsp;&nbsp;\'displayName\' : string,<br /> &nbsp;&nbsp;&nbsp;\'schemaClass\' : string,<br /> &nbsp;}</td>\r\n<td data-label=\"Description\">The function is called when an object in the fireworks is selected by the user action</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onNodeHovered(function(obj))</b> :: void</td>\r\n<td data-label=\"Params\"><b>obj</b> is hovered item:<br /> &nbsp;{<br /> &nbsp;&nbsp;&nbsp;\'stId\' : string,<br /> &nbsp;&nbsp;&nbsp;\'displayName\' : string,<br /> &nbsp;&nbsp;&nbsp;\'schemaClass\' : string,<br /> &nbsp;}</td>\r\n<td data-label=\"Description\">The function is called when an object in the fireworks is hovered by the user action</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onFireworksLoaded(function(id))</b> :: void</td>\r\n<td data-label=\"Params\">The db identifier of the loaded species in fireworks<br /> <b>id</b>: string</td>\r\n<td data-label=\"Description\">The function is called when a fireworks is loaded in the viewer</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onAnalysisReset(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters</td>\r\n<td data-label=\"Description\">The function is called when the users resets the analysis overlay<br /><br /> fireworks.onAnalysisReset(function(){ /* your code here */ });</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onCanvasNotSupported(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters.</td>\r\n<td data-label=\"Description\">The function is called when the browser doesn\'t support HTML5 Canvas so the viewer cannot be instantiated<br /><br /> fireworks.onCanvasNotSupported(function(){ /* your code here */ });</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<script type=\"text/javascript\">function getMainContentWidth(){\r\n   var DIV = \"fav-maincontent\";\r\n   var style = window.getComputedStyle(document.getElementById(DIV), null);\r\n   var width = style.getPropertyValue(\"width\").replace(\"px\",\"\");\r\n   var paddingLeft = style.getPropertyValue(\"padding-left\").replace(\"px\",\"\"); \r\n   var paddingRight = style.getPropertyValue(\"padding-right\").replace(\"px\",\"\");\r\n   return width - paddingLeft - paddingRight;\r\n}\r\n\r\njQuery( window ).resize(function() {\r\n  fireworks.resize(getMainContentWidth(), 600);  \r\n});\r\n\r\nfunction onReactomeFireworksReady() {\r\n            fireworks = Reactome.Fireworks.create({\r\n                \"placeHolder\": \"fireworksHolder\",\r\n                \"width\": getMainContentWidth(),\r\n                \"height\": 600\r\n            });\r\n            fireworks.onFireworksLoaded(function (loaded) {\r\n                console.info(\"Loaded \", loaded);\r\n            });\r\n            fireworks.onNodeHovered(function (hovered) {\r\n                console.info(\"Hovered \", hovered);\r\n            });\r\n            fireworks.onNodeSelected(function (selected) {\r\n                console.info(\"Selected \", selected);\r\n            });\r\n        }</script>\r\n<script src=\"/FireworksJs/fireworks/fireworks.nocache.js\" type=\"text/javascript\" language=\"javascript\"></script>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-07 10:10:59','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(57,632,'Pathways Overview GWT','pathways-overview-gwt','<p>Reactomes Fireworks GWT widget is our original implementation of the pathways overview in GWT. It is meant to be reused by third party resources in order to display Reactome pathways overview directly in their web pages and enable users to interact with them.</p>\r\n<p>{loadposition dev-guide-po-gwt-1}</p>\r\n<div>\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p><a href=\"http://www.gwtproject.org/\">GWT</a>&nbsp;is a development toolkit for building and optimizing complex browser-based applications. Its goal is to enable productive development of high-performance web applications without the developer having to be an expert in browser quirks, XMLHttpRequest, and JavaScript. GWT is used by many products at Google, including AdWords, AdSense, Flights, Hotel Finder, Offers, Wallet, Blogger. Its open source, completely free, and used by thousands of developers around the world.</p>\r\n<p>&nbsp;</p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>We assume that you already have your <a href=\"http://www.gwtproject.org/gettingstarted.html\">GWT project</a> up and running configured with the <a href=\"http://www.gwtproject.org/doc/latest/RefCommandLineTools.html\"> MAVEN archetype</a>. So the first step is to add the EBI Nexus repository in your pom.xml file:</p>\r\n<pre><code>&lt;repositories&gt;\r\n    ...\r\n    &lt;!-- EBI repo --&gt;\r\n    &lt;repository&gt;\r\n        &lt;id&gt;nexus-ebi-repo&lt;/id&gt;\r\n        &lt;name&gt;The EBI internal repository&lt;/name&gt;\r\n        &lt;url&gt;http://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-repo/&lt;/url&gt;\r\n        &lt;releases&gt;\r\n            &lt;enabled&gt;true&lt;/enabled&gt;\r\n        &lt;/releases&gt;\r\n        &lt;snapshots&gt;\r\n            &lt;enabled&gt;false&lt;/enabled&gt;\r\n        &lt;/snapshots&gt;\r\n    &lt;/repository&gt;\r\n    &lt;!-- EBI SNAPSHOT repo --&gt;\r\n    &lt;repository&gt;\r\n        &lt;id&gt;nexus-ebi-snapshot-repo&lt;/id&gt;\r\n        &lt;name&gt;The EBI internal snapshot repository&lt;/name&gt;\r\n        &lt;url&gt;http://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-snapshots/&lt;/url&gt;\r\n        &lt;releases&gt;\r\n            &lt;enabled&gt;false&lt;/enabled&gt;\r\n        &lt;/releases&gt;\r\n        &lt;snapshots&gt;\r\n            &lt;enabled&gt;true&lt;/enabled&gt;\r\n        &lt;/snapshots&gt;\r\n    &lt;/repository&gt;\r\n&lt;/repositories&gt;\r\n</code></pre>\r\n<p>Once the repository is in place, the next thing to do is adding the Reactome\'s pathways overview dependency:</p>\r\n<pre><code>&lt;dependencies&gt;\r\n    ...\r\n    &lt;dependency&gt;\r\n        &lt;groupId&gt;org.reactome.web&lt;/groupId&gt;\r\n        &lt;artifactId&gt;fireworks&lt;/artifactId&gt;\r\n        &lt;version&gt;1.0.0&lt;/version&gt;\r\n    &lt;/dependency&gt;\r\n&lt;dependencies&gt;\r\n</code></pre>\r\n<p>Before writing the code to use the pathways overview, add the dependency in your <b>gwt.xml</b> file, so the compiler will find the pathways overview:</p>\r\n<pre><code>&lt;inherits name=\"org.reactome.web.fireworks.FireworksViewer\" /&gt;\r\n</code></pre>\r\n<p>And finally, assuming the widget is now going to be displayed by its own, the code in your <b>EntryPoint</b> class would look like:</p>\r\n<pre><code>/**\r\n * For test purposes we assume the pathway overview loads as a single item in the web page.\r\n *\r\n * Entry point classes define onModuleLoad().\r\n */\r\npublic class Main implements EntryPoint {\r\n\r\n    private final FirewowksViewer fireworks;\r\n\r\n    public Main() {\r\n        //We create the fireworks. Please note that different pathways overview cannot be loaded in the same instance\r\n        fireworks = FireowksFactory.createFireworksViewer();\r\n    }\r\n\r\n    @Override\r\n    public void onModuleLoad() {\r\n        Scheduler.get().scheduleDeferred(new Scheduler.ScheduledCommand() {\r\n            @Override\r\n            public void execute() {\r\n                FireworksFactory.CONSOLE_VERBOSE = true;   //This is optional (for dev purposes)\r\n                FireworksFactory.EVENT_BUS_VERBOSE = true; //This is optional (for dev purposes)\r\n\r\n                //For this use case we add the firewoks as the main and only object of the webpage\r\n                RootLayoutPanel.get().add(fireworks);\r\n            }\r\n        });\r\n    }\r\n}\r\n</code></pre>\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>API</a></h3>\r\n<p>The available methods are defined in the <b>DiagramViewer</b> interface in the <b>org.reactome.web.diagram.client</b> package. These are:</p>\r\n<pre><code>    //subscribe to this event to be notified when the user closes the analysis overlay\r\n    HandlerRegistration addAnalysisResetHandler(AnalysisResetHandler handler);\r\n\r\n    //Subscribe to this event to be notified if the browser does not support canvas, so diagrams can not be displayed\r\n    HandlerRegistration addCanvasNotSupportedHandler(CanvasNotSupportedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the overview is displayed in the viewport\r\n    HandlerRegistration addFireworksLoaded(FireworksLoadedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when\r\n    HandlerRegistration addExpressionColumnChangedHandler(ExpressionColumnChangedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user hovers over an node in the overview\r\n    HandlerRegistration addNodeHoverHandler(NodeHoverHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user leave the hovered node in the overview\r\n    HandlerRegistration addNodeHoverResetHandler(NodeHoverResetHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user opens a node in the overview\r\n    HandlerRegistration addNodeOpenedHandler(NodeOpenedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user selects a node in the overview\r\n    HandlerRegistration addNodeSelectedHandler(NodeSelectedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user unselects a node in the overview\r\n    HandlerRegistration addNodeSelectedResetHandler(NodeSelectedResetHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user changes the colour profile\r\n    HandlerRegistration addProfileChangedHandler(ProfileChangedHandler handler);\r\n\r\n    //Returns the selected node\r\n    Node getSelected();\r\n\r\n    //Highlights the node with the provided stable identifier\r\n    void highlightNode(String stableIdentifier);\r\n\r\n    //Highlights the node with the provided stable identifier\r\n    void highlightNode(Long dbIdentifier);\r\n\r\n    //Zooms-in to the node with the provided stable identifier\r\n    void openPathway(String stableIdentifier);\r\n\r\n    //Zooms-in to the node with the provided database identifier\r\n    void openPathway(Long dbIdentifier);\r\n\r\n    //Resets the analysis overlay\r\n    void resetAnalysis();\r\n\r\n    //Resets the highlighted nodes\r\n    void resetHighlight();\r\n\r\n    //Resets the selected nodes\r\n    void resetSelection();\r\n\r\n    //Selects the node with the provided stable identifier\r\n    void selectNode(String stableIdentifier);\r\n\r\n    //Selects the node with the provided database identifier\r\n    void selectNode(Long dbIdentifier);\r\n\r\n    //Sets the analysis overlay for a given \"token\" and \"resource\" (this will query the Analysis Service)\r\n    void setAnalysisToken(String token, String resource);\r\n\r\n    //Show all the nodes in the viewport\r\n    void showAll();</code></pre>\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<ul style=\"list-style: none;\">\r\n<li style=\"line-height: 30px;\"><a href=\"http://github.com/reactome/Fireworks\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Fireworks code</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"http://github.com/reactome-pwp\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>Reactome Pathways Portal code</a></li>\r\n</ul>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-07 10:16:02','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(58,633,'Diagram JS','diagram-js','<style type=\"text/css\">pre, pre code {\r\n    word-wrap: initial;\r\n    white-space: pre;\r\n    word-break: break-all;\r\n}\r\n}</style>\r\n<p>Reactome\'s DiagramJs widget is our diagram viewer in an ordinary JavaScript API. It is meant to be reused by third party resources in order to display Reactome Pathway Diagrams directly in their web pages and enable users to interact with them.</p>\r\n<div id=\"diagramHolder\" class=\"diagram\"></div>\r\n<p>&nbsp;</p>\r\n<h3><a id=\"reuse-diagram-widget\" href=\"#reuse-diagram-widget\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Reusing Reactome\'s Diagram Widget?</a></h3>\r\n<p>To reuse our viewer you need to follow the following steps</p>\r\n<p>1. Include the diagram javascript dependency in you HTML header</p>\r\n<pre><code>&lt;script type=\"text/javascript\" language=\"javascript\" src=\"https://reactome.org/DiagramJs/diagram/diagram.nocache.js\"&gt;&lt;/script&gt;</code></pre>\r\n<p>2. Add a place holder in the body of your web page</p>\r\n<pre><code>&lt;div id=\"diagramHolder\"&gt;&lt;/div&gt;</code></pre>\r\n<p>3. Create and initialise the diagram viewer from your javascript code</p>\r\n<pre><code>    //Creating the Reactome Diagram widget\r\n    //Take into account a proxy needs to be set up in your server side pointing to www.reactome.org\r\n    function onReactomeDiagramReady(){  //This function is automatically called when the widget code is ready to be used\r\n        var diagram = Reactome.Diagram.create({\r\n            \"placeHolder\" : \"diagramHolder\",\r\n            \"width\" : 900,\r\n            \"height\" : 500\r\n        });\r\n\r\n        //Initialising it to the \"Hemostasis\" pathway\r\n        diagram.loadDiagram(\"R-HSA-109582\");\r\n\r\n        //Adding different listeners\r\n\r\n        diagram.onDiagramLoaded(function (loaded) {\r\n            console.info(\"Loaded \", loaded);\r\n            diagram.flagItems(\"FYN\");<br /><code><code>            </code>if (loaded == \"R-HSA-109582\") diagram.selectItem(\"R-HSA-109582\");<br /></code><code>        </code>}); <br /><br /><code>        </code>diagram.onObjectHovered(function (hovered){ <br /><code>            </code>console.info(\"Hovered \", hovered); <br /><code>        </code><code></code>}); <br /><br /><code>        </code>diagram.onObjectSelected(function (selected){<br /><code>            </code>console.info(\"Selected \", selected); <br /><code>        </code><code></code>});<br /> <code>    </code>}</code></pre>\r\n<h3><a id=\"API\" href=\"#API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>DiagramJs API</a></h3>\r\n<p>The current implementation supports the following listeners and methods:</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome\">\r\n<thead>\r\n<tr><th>Method</th><th>Params</th><th>Description</th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td data-label=\"Method\"><b>create</b> :: Constructor<br /> &nbsp;&nbsp;Reactome.Diagram.create(params);</td>\r\n<td data-label=\"Params\"><b>param</b> :: json object<br /> &nbsp;{<br /> &nbsp;&nbsp;&nbsp;\'proxyPrefix\' : string,<br /> &nbsp;&nbsp;&nbsp;\'placeHolder\' : string,<br /> &nbsp;&nbsp;&nbsp;\'width\' : int (optional),<br /> &nbsp;&nbsp;&nbsp;\'height\' : int (optional)<br /> &nbsp;}</td>\r\n<td data-label=\"Description\">Creates and returns a new Reactome.Diagram object</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>loadDiagram(stId)</b> :: void</td>\r\n<td data-label=\"Params\">Pathway stable identifier<br /> <b>stId</b> : string</td>\r\n<td data-label=\"Description\">Loads the specified pathway</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>flagItems(term)</b> :: void</td>\r\n<td data-label=\"Params\">Entity identifier (gene name)<br /> <b>term</b> : string</td>\r\n<td data-label=\"Description\">Flags all entities matching with the \"term\"</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>highlightItem(stId)</b> :: void</td>\r\n<td data-label=\"Params\">Item stable identifier<br /> <b>stId</b> : string</td>\r\n<td data-label=\"Description\">Highlights the specified item if it exists in the diagram</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetAnalysis()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Resets the analysis overlay</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetFlaggedItems()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Resets the flagged items</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetHighlight()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Clears the highlights in the diagram</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetSelection()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Clears the selection in the diagram</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resize(width, height)</b> :: void</td>\r\n<td data-label=\"Params\"><b>widht</b>: int<br /> <b>height</b>: int</td>\r\n<td data-label=\"Description\">Resizes the viewport to the specified width and height</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>selectItem(stId)</b> :: void</td>\r\n<td data-label=\"Params\">Item stable identifier<br /> <b>stId</b> : string</td>\r\n<td data-label=\"Description\">Selects the specified item if it exists in the diagram</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>setAnalysisToken(token, resource)</b> :: void</td>\r\n<td data-label=\"Params\">Analysis token<br /> <b>token</b> : string<br /> Resource<br /> <b>resource</b> : string</td>\r\n<td data-label=\"Description\">Overlays the analysis result corresponding to the specified (token, resource)</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onObjectSelected(function(obj))</b> :: void</td>\r\n<td data-label=\"Params\"><b>obj</b> is selected item:<br /> &nbsp;{<br /> &nbsp;&nbsp;&nbsp;\'stId\' : string,<br /> &nbsp;&nbsp;&nbsp;\'displayName\' : string,<br /> &nbsp;&nbsp;&nbsp;\'schemaClass\' : string,<br /> &nbsp;&nbsp;&nbsp;\'identifier\' : string (optional),<br /> &nbsp;&nbsp;&nbsp;\'geneNames\' : Array&lt;string&gt; (optional)<br /> &nbsp;}</td>\r\n<td data-label=\"Description\">The function is called when an object in the diagram is selected by the user action</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onObjectHovered(function(obj))</b> :: void</td>\r\n<td data-label=\"Params\"><b>obj</b> is hovered item:<br /> &nbsp;{<br /> &nbsp;&nbsp;&nbsp;\'stId\' : string,<br /> &nbsp;&nbsp;&nbsp;\'displayName\' : string,<br /> &nbsp;&nbsp;&nbsp;\'schemaClass\' : string,<br /> &nbsp;&nbsp;&nbsp;\'identifier\' : string (optional),<br /> &nbsp;&nbsp;&nbsp;\'geneNames\' : Array&lt;string&gt; (optional)<br /> &nbsp;}</td>\r\n<td data-label=\"Description\">The function is called when an object in the diagram is hovered by the user action</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onDiagramLoaded(function(stId))</b> :: void</td>\r\n<td data-label=\"Params\">The stable identifier of the loaded diagram<br /> <b>stId</b>: string</td>\r\n<td data-label=\"Description\">The function is called when a diagram is loaded in the viewer</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onFlagsReset(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters</td>\r\n<td data-label=\"Description\">The function is called when the flagged items have been reset by the user action</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onAnalysisReset(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters</td>\r\n<td data-label=\"Description\">The function is called when the users resets the analysis overlay<br /><br /> diagram.onAnalysisReset(function(){ /* your code here */ });</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onCanvasNotSupported(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters.</td>\r\n<td data-label=\"Description\">The function is called when the browser doesn\'t support HTML5 Canvas so the viewer cannot be instantiated<br /><br /> diagram.onCanvasNotSupported(function(){ /* your code here */ });</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<script type=\"text/javascript\">jQuery(\'base\').remove();\r\nfunction getMainContentWidth(){\r\n   var DIV = \"fav-maincontent\";\r\n   var style = window.getComputedStyle(document.getElementById(DIV), null);\r\n   var width = style.getPropertyValue(\"width\").replace(\"px\",\"\");\r\n   var paddingLeft = style.getPropertyValue(\"padding-left\").replace(\"px\",\"\"); \r\n   var paddingRight = style.getPropertyValue(\"padding-right\").replace(\"px\",\"\");\r\n   return width - paddingLeft - paddingRight;\r\n}\r\njQuery( window ).resize(function() {\r\n  diagram.resize(getMainContentWidth(), 600);  \r\n});\r\nfunction onReactomeDiagramReady() {\r\n        diagram = Reactome.Diagram.create({\r\n            \"placeHolder\": \"diagramHolder\",\r\n            \"width\": getMainContentWidth(),\r\n            \"height\": 600\r\n        });\r\n        diagram.loadDiagram(\"R-HSA-109582\");\r\n        diagram.onDiagramLoaded(function (loaded) {\r\n            diagram.flagItems(\"FYN\");\r\n            if(loaded == \"R-HSA-109582\") diagram.selectItem(\"R-HSA-75892\");\r\n        });\r\n    }</script>\r\n<script src=\"/DiagramJs/diagram/diagram.nocache.js\" type=\"text/javascript\" language=\"javascript\"></script>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-07 15:17:14','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(59,634,'Diagram GWT','diagram-gwt','<p>Reactomes Diagram GWT widget is our original implementation of the diagram viewer in <a href=\"http://www.gwtproject.org\" target=\"_blank\" rel=\"nofollow noopener noreferrer\">GWT</a>. It is meant to be reused by third party resources in order to display Reactome Pathway Diagrams directly in their web pages and enable users to interact with them.</p>\r\n<p>{loadposition dev-guide-pd-gwt-1}</p>\r\n<div>\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p><a href=\"http://www.gwtproject.org/\">GWT</a>&nbsp;is a development toolkit for building and optimizing complex browser-based applications. Its goal is to enable productive development of high-performance web applications without the developer having to be an expert in browser quirks, XMLHttpRequest, and JavaScript. GWT is used by many products at Google, including AdWords, AdSense, Flights, Hotel Finder, Offers, Wallet, Blogger. Its open source, completely free, and used by thousands of developers around the world.</p>\r\n<p>&nbsp;</p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>We assume that you already have your <a href=\"http://www.gwtproject.org/gettingstarted.html\">GWT project</a> up and running configured with the <a href=\"http://www.gwtproject.org/doc/latest/RefCommandLineTools.html\"> MAVEN archetype</a>. So the first step is to add the EBI Nexus repository in your pom.xml file:</p>\r\n<pre><code>&lt;repositories&gt;\r\n    ...\r\n    &lt;!-- EBI repo --&gt;\r\n    &lt;repository&gt;\r\n        &lt;id&gt;nexus-ebi-repo&lt;/id&gt;\r\n        &lt;name&gt;The EBI internal repository&lt;/name&gt;\r\n        &lt;url&gt;http://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-repo/&lt;/url&gt;\r\n        &lt;releases&gt;\r\n            &lt;enabled&gt;true&lt;/enabled&gt;\r\n        &lt;/releases&gt;\r\n        &lt;snapshots&gt;\r\n            &lt;enabled&gt;false&lt;/enabled&gt;\r\n        &lt;/snapshots&gt;\r\n    &lt;/repository&gt;\r\n    &lt;!-- EBI SNAPSHOT repo --&gt;\r\n    &lt;repository&gt;\r\n        &lt;id&gt;nexus-ebi-snapshot-repo&lt;/id&gt;\r\n        &lt;name&gt;The EBI internal snapshot repository&lt;/name&gt;\r\n        &lt;url&gt;http://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-snapshots/&lt;/url&gt;\r\n        &lt;releases&gt;\r\n            &lt;enabled&gt;false&lt;/enabled&gt;\r\n        &lt;/releases&gt;\r\n        &lt;snapshots&gt;\r\n            &lt;enabled&gt;true&lt;/enabled&gt;\r\n        &lt;/snapshots&gt;\r\n    &lt;/repository&gt;\r\n&lt;/repositories&gt;\r\n</code></pre>\r\n<p>Once the repository is in place, the next thing to do is adding the Reactome\'s pathways overview dependency:</p>\r\n<pre><code>&lt;dependencies&gt;\r\n    ...\r\n    &lt;dependency&gt;\r\n        &lt;groupId&gt;org.reactome.web&lt;/groupId&gt;\r\n        &lt;artifactId&gt;diagram&lt;/artifactId&gt;\r\n        &lt;version&gt;3.2.1&lt;/version&gt;\r\n    &lt;/dependency&gt;\r\n&lt;dependencies&gt;\r\n</code></pre>\r\n<p>Before writing the code to use the pathways overview, add the dependency in your <b>gwt.xml</b> file, so the compiler will find the pathways overview:</p>\r\n<pre><code>&lt;inherits name=\"org.reactome.web.diagram.DiagramViewer\" /&gt;\r\n</code></pre>\r\n<p>And finally, assuming the widget is now going to be displayed by its own, the code in your <b>EntryPoint</b> class would look like:</p>\r\n<pre><code>/**\r\n * For test purposes we assume the diagram loads as a single item in the web page.\r\n *\r\n * Entry point classes define onModuleLoad().\r\n */\r\npublic class Main implements EntryPoint {\r\n\r\n    private final DiagramViewer diagram;\r\n\r\n    public Main() {\r\n        //We create the diagram. Please note that different pathways diagrams can be loaded in the same\r\n        //instance (the diagram manages the cache of previously loaded diagrams using an LRU)\r\n        diagram = DiagramFactory.createDiagramViewer();\r\n    }\r\n\r\n    @Override\r\n    public void onModuleLoad() {\r\n        Scheduler.get().scheduleDeferred(new Scheduler.ScheduledCommand() {\r\n            @Override\r\n            public void execute() {\r\n                DiagramFactory.CONSOLE_VERBOSE = true;   //This is optional (for dev purposes)\r\n                DiagramFactory.EVENT_BUS_VERBOSE = true; //This is optional (for dev purposes)\r\n\r\n                //For this use case we add the diagram as the main and only object of the webpage\r\n                RootLayoutPanel.get().add(diagram);\r\n                \r\n                //This will run the loadDiagram method asynchronously, ensuring the DOM has been updated\r\n                Scheduler.get().scheduleDeferred(new Scheduler.ScheduledCommand() {\r\n                    @Override\r\n                    public void execute() {\r\n                        diagram.loadDiagram(\"R-HSA-111804\");\r\n                    }\r\n                });\r\n            }\r\n        });\r\n    }\r\n}</code></pre>\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>API</a></h3>\r\n<p>The available methods are defined in the <b>DiagramViewer</b> interface in the <b>org.reactome.web.diagram.client</b> package. These are:</p>\r\n<pre><code>    //subscribe to this event to be notified when the user closes the analysis overlay\r\n    HandlerRegistration addAnalysisResetHandler(AnalysisResetHandler handler);\r\n\r\n    //Subscribe to this event to be notified if the browser does not support canvas, so diagrams can not be displayed\r\n    HandlerRegistration addCanvasNotSupportedEventHandler(CanvasNotSupportedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user selects an object in the diagram\r\n    HandlerRegistration addDatabaseObjectSelectedHandler(GraphObjectSelectedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user hovers over an object in the diagram\r\n    HandlerRegistration addDatabaseObjectHoveredHandler(GraphObjectHoveredHandler handler);\r\n\r\n    //Subscribe to this event to be notified when a new diagram is displayed in the viewport\r\n    HandlerRegistration addDiagramLoadedHandler(DiagramLoadedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when elements in the diagram are flagged\r\n    HandlerRegistration addDiagramObjectsFlaggedHandler(DiagramObjectsFlaggedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when flagged elements in the diagram are not longer visible\r\n    HandlerRegistration addDiagramObjectsFlagResetHandler(DiagramObjectsFlagResetHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user selects the \"Go to overview\" button\r\n    HandlerRegistration addFireworksOpenedHandler(FireworksOpenedHandler handler);\r\n\r\n    //Flags the elements in the diagram which identifiers match with \"identifier\"\r\n    void flagItems(String identifier);\r\n\r\n    //Highlights the element(s) in the diagram with the \"stableIdentifier\"\r\n    void highlightItem(String stableIdentifier);\r\n\r\n    //Highlights the element(s) in the diagram with the \"dbIdentifier\"\r\n    void highlightItem(Long dbIdentifier);\r\n\r\n    //Loads the diagram with the \"stableIdentifier\"\r\n    void loadDiagram(String stId);\r\n\r\n    //Loads the diagram with the \"dbId\"\r\n    void loadDiagram(Long dbId);\r\n\r\n    //Resets the analysis overlay\r\n    void resetAnalysis();\r\n\r\n    //Resets the flagged items\r\n    void resetFlaggedItems();\r\n\r\n    //Resets the highlighted items\r\n    void resetHighlight();\r\n\r\n    //Resets the selected items\r\n    void resetSelection();\r\n\r\n    //Selects the element(s) in the diagram with the \"stableIdentifier\"\r\n    void selectItem(String stableIdentifier);\r\n\r\n    //Selects the element(s) in the diagram with the \"stableIdentifier\"\r\n    void selectItem(Long dbIdentifier);\r\n\r\n    //Sets the analysis overlay for a given \"token\" and \"resource\" (this will query the Analysis Service)\r\n    void setAnalysisToken(String token, String resource);\r\n\r\n    //Set the widget visibility\r\n    void setVisible(boolean visible);</code></pre>\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<ul style=\"list-style: none;\">\r\n<li style=\"line-height: 30px;\"><a href=\"http://github.com/reactome-pwp/diagram\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Diagram code</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"http://github.com/reactome-pwp\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>Reactome Pathways Portal code</a></li>\r\n</ul>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-07 15:17:38','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(60,638,'Identifiers Mapping','identifiers-mapping','<p>The mapping files consist of a tab-separated table that indicates which external protein, gene or small molecule identifiers in the source database were mapped to Reactome pathway and reaction annotations. Our goal with distributing three sets of files for each different identifier type is to provide these mapping files link the source database identifier to:</p>\r\n<ul>\r\n<li style=\"list-style-type: lower-roman;\">the lowest level pathway diagram or subset of the pathway</li>\r\n<li style=\"list-style-type: lower-roman;\">all level pathway diagrams (filename appended with \'All_Levels),</li>\r\n<li style=\"list-style-type: lower-roman;\">all reaction events.</li>\r\n</ul>\r\n<p>The columns within the mappings files follow a similar format:</p>\r\n<ol>\r\n<li>Source database identifier, e.g. UniProt, ENSEMBL, NCBI Gene or ChEBI identifier</li>\r\n<li>&nbsp;Reactome Pathway Stable identifier</li>\r\n<li>URL</li>\r\n<li>Event (Pathway or Reaction) Name</li>\r\n<li>Evidence Code</li>\r\n<li>Species</li>\r\n</ol>\r\n<p>If you have any further questions, please <a href=\"mailto:help@reactome.org?subject=Identifier%20Mapping%20File\">contact us</a>.</p>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-23 09:19:03','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(61,639,'Pathway Information','pathway-information','<p>The pathway information files consist of tab-separated tables that provide information about Reactome pathway hierarchy:</p>\r\n<ol>\r\n<li>The \"Complete list of pathways\" file maps the Reactome Stable identifier (ST_ID) to a pathway name and corresponding species.</li>\r\n<li>The \"Pathway hierarchy relationship\" file consists of two columns of Reactome Stable identifiers (ST_ID), defining the relationship between pathways within the pathway hierarchy. The first column provides the parent pathway stable identifier, whereas the second column provides the child pathway stable identifier.</li>\r\n</ol>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-23 09:37:27','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(62,640,'Tab-Delimiter Format','tab-delimiter-format','<p>Interactions are generated electronically based on the data that we store on complexes and reactions, i.e. the interactions provided by Reactome are not curated and are not experimental data. In addition, the complexes and reactions in species other than human are derived by orthology inference from the correspondinghuman complexes and reactions.</p>\r\n<p>The meaning of \"interaction\" is quite broad: 2 protein sequences occur in the same complex or they occur in the same reaction.</p>\r\n<p>The 1st line begins with \'#\' and gives the number of unique interactions, i.e. unique pairs if UniProt accession numbers.</p>\r\n<p>The rest of the file is tab delimited with columns having following meaning/content:</p>\r\n<ul style=\"list-style: decimal;\">\r\n<li>Protein ID (e.g. UniProt accession) for interactor 1.</li>\r\n<li>Ensembl gene id(s delimited by \'|\') for interactor 1. Note that one UniProt entry may be mapped to multiple Ensembl genes.</li>\r\n<li>Entrez Gene id(s delimited by \'|\') for interactor 1. Note that one UniProt entry may be mapped to multiple Entrez Gene ids.</li>\r\n<li>Protein ID (e.g. UniProt accession) for interactor 2.</li>\r\n<li>Ensembl gene id(s delimited by \'|\') for interactor 2. Note that one UniProt entry may be mapped to multiple Ensembl genes.</li>\r\n<li>Entrez Gene id(s delimited by \'|\') for interactor 2. Note that one UniProt entry may be mapped to multiple Entrez Gene ids.</li>\r\n<li>Type of interaction, either \'direct_complex\', \'indirect_complex\', \'reaction\' or \'neighbouring_reaction\'. Meanings of those below.</li>\r\n<li>Interaction context. This is the class and internal identifier of Reactome instance via which this interaction happens. For \'neighbouring_reaction\' type interaction this contains a pair of class:identifier string separated by \'\'.</li>\r\n<li>Comma-delimited list of PubMed identifiers supporting this interaction. For a reaction these are the PMIDs from the Reaction.literatureReference slot (and not from the \"second tier\" Reaction.summation.literatureReference). For a complex these are from reations which produce the complex. For \'neighbouring_reaction\' type interaction this field is empty.</li>\r\n</ul>\r\n<h3>Interaction types</h3>\r\n<p>In Reactome, complexes can be arbitrarily nested, i.e., contain subcomplexes. Here we have complex C1 consisting of complexes C2 and C3. C2 consist of molecules A and B. C3 consists of molecules C and D:</p>\r\n<pre>           C1 --- C2 --- A\r\n                |      |\r\n                |      |- B\r\n                |\r\n                |- C3 --- C\r\n                |\r\n                |- D\r\n                </pre>\r\n<p>Now really to the interaction types:</p>\r\n<ul>\r\n<li>direct_complex - interactors are directly in the same complex, i.e. w/o further nested complexes. From the example interactions A B and C are of this type.</li>\r\n<li>indirect_complex - interactors in different subcomplexes of a complex. From the example above interactions AC, AD, BC and BD are of this type.</li>\r\n<li>reaction - interactors participate in the same reaction. Only those reactions are reported for which the intreactors are not complexed (with the exception being association dissociation reactions which are reported).</li>\r\n<li>neighbouring_reaction - interactor participate in 2 \"consecutive\" reactions, i.e. when one reaction produces a PhysicalEntity which is either an input or a catalyst for another reaction. However, to avoid the \"trivial\" (i.e. over ATP, ADP etc) interactions, the computation is done using the \'precedingEvent\' attribute used to order reactions in a pathway.</li>\r\n</ul>\r\n<p>The \'direct_complex\' type doesn\'t really mean true pairwise interaction. For example, Reactome\'s representation of ribosomal S60 subunit consists \"directly\" of 40+ proteins and has no sub-complexes. Obviously this can hardly mean that each one of those component proteins directly interacts with every other protein. Hence, the \'direct_complex\' vs. \'indirect_complex\' distinction is quite meaningless.</p>\r\n<h3>Other notes:</h3>\r\n<ul>\r\n<li>If the same interaction (i.e same UniProte accession pair) happens in multiple contexts, each context is reported on a separate row and hence the row number in the file is bigger than the number of unique interactions reported on the 1st line.</li>\r\n<li>The order of interactors on one line has no meaning (they are just ordered according to the Reactome internal identifier...). Every interaction with context is reported once, i.e. AB is not reported as BA.</li>\r\n</ul>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-23 10:02:36','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(63,641,'Curator Tool (version 3.3, build 98, August 6, 2019)','reactome-curator-tool-3','<p>\r\n<style type=\"text/css\">ul{ list-style-type: square; }</style>\r\n</p>\r\n<div id=\"top\"></div>\r\n<center>\r\n<p><span><b>Available Installers</b></span></p>\r\n<table border=\"0\" cellspacing=\"10\" cellpadding=\"0\">\r\n<tbody>\r\n<tr>\r\n<td>&nbsp;</td>\r\n<td><span>Platform</span></td>\r\n<td>&nbsp;</td>\r\n<td>&nbsp;</td>\r\n<td><center><span>Installer</span></center></td>\r\n<td>&nbsp;</td>\r\n<td><center><span>Instructions</span></center></td>\r\n</tr>\r\n<tr>\r\n<td style=\"text-align: center;\" valign=\"center\"><i class=\"fa fa-windows\"></i></td>\r\n<td valign=\"bottom\">\r\n<div style=\"text-align: left;\"><span>Windows</span></div>\r\n</td>\r\n<td valign=\"bottom\">&nbsp;</td>\r\n<td valign=\"bottom\">&nbsp;</td>\r\n<td valign=\"bottom\"><center><a href=\"/download/tools/curatorTool/InstData/win_linux/install.zip\">Download (15.4M)</a></center></td>\r\n<td valign=\"bottom\">&nbsp;</td>\r\n<td valign=\"bottom\"><center><a href=\"#windows\">View</a></center></td>\r\n</tr>\r\n<tr>\r\n<td style=\"text-align: center;\" valign=\"center\"><i class=\"fa fa-apple\"></i></td>\r\n<td valign=\"bottom\">\r\n<div style=\"text-align: left;\"><span>Mac OS X</span></div>\r\n</td>\r\n<td valign=\"bottom\">&nbsp;</td>\r\n<td valign=\"bottom\">&nbsp;</td>\r\n<td valign=\"bottom\"><center><a href=\"/download/tools/curatorTool/InstData/mac/install.zip\">Download (89M)</a></center></td>\r\n<td valign=\"bottom\">&nbsp;</td>\r\n<td valign=\"bottom\"><center><a href=\"#macosx\">View</a></center></td>\r\n</tr>\r\n<tr>\r\n<td style=\"text-align: center;\" valign=\"center\"><i class=\"fa fa-linux\"></i></td>\r\n<td valign=\"bottom\">\r\n<div style=\"text-align: left;\"><span>Linux</span></div>\r\n</td>\r\n<td valign=\"bottom\">&nbsp;</td>\r\n<td valign=\"bottom\">&nbsp;</td>\r\n<td valign=\"bottom\"><center><a href=\"/download/tools/curatorTool/InstData/win_linux/install.zip\">Download (15.4M)</a></center></td>\r\n<td valign=\"bottom\">&nbsp;</td>\r\n<td valign=\"bottom\"><center><a href=\"#linux\">View</a></center></td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</center><hr />\r\n<div id=\"windows\" style=\"text-align: left;\">\r\n<h4><a href=\"#windows\" name=\"windows\">Windows:</a></h4>\r\n<h5>Instructions</h5>\r\n<ul>\r\n<li>After downloading, double-click&nbsp;<b><code>install.zip</code></b>&nbsp;to unzip it to an empty folder. This folder should be named \"ReactomeCuratorTool\" though any name can be used. Please use the latest version of WinZip if you use this application for unzipping.</li>\r\n<li>After unzipping, an application called ReactomeCuratorTool.exe should be found in the named folder. Double click this application to launch the Reactome Curator Tool.</li>\r\n</ul>\r\n<h5>Notes</h5>\r\n<ul>\r\n<li>You need to install a Java 1.8.0 (or later) virtual machine. You can download one from&nbsp;<a href=\"http://java.sun.com/\" target=\"_blank\" rel=\"noopener noreferrer\">Sun\'s Java web site</a>.</li>\r\n</ul>\r\n<p>(<a href=\"#top\">Go To Top</a>)</p>\r\n</div>\r\n<center><hr /></center>\r\n<div id=\"macosx\" style=\"text-align: left;\">\r\n<h4><a href=\"#macosx\" name=\"macosx\">Mac OS X:</a></h4>\r\n<h5>Instructions</h5>\r\n<ul>\r\n<li>After downloading, double-click&nbsp;<b><code>install.zip</code></b>&nbsp;if it is not automatically unzipped.</li>\r\n<li>Double click ReactomeCuratorTool to launch the Reactome Curator Tool.</li>\r\n</ul>\r\n<h5>Notes</h5>\r\n<ul>\r\n<li>Requires Mac OS X 10.4.8 or later, and Java 1.8.0 or later.</li>\r\n<li>The compressed installer should be recognized by Stuffit Expander and should automatically be expanded after downloading. If it is not expanded, you can expand it manually using&nbsp;<a href=\"http://www.aladdinsys.com/expander/index.html\" target=\"_blank\" rel=\"noopener noreferrer\">StuffIt Expander 6.0 or later</a>.</li>\r\n<li>If you have any problems launching the installer once it has been expanded, make sure that the compressed installer was expanded using Stuffit Expander. If you continue to have problems, please contact technical support.</li>\r\n</ul>\r\n<p>(<a href=\"#top\">Go To Top</a>)</p>\r\n</div>\r\n<center><hr /></center>\r\n<div id=\"linux\" style=\"text-align: left;\">\r\n<h4><a href=\"#linux\" name=\"linux\">Linux:</a></h4>\r\n<h5>Instructions</h5>\r\n<ul>\r\n<li>After downloading open a shell, and&nbsp;<span><b><code>cd</code></b></span><span>&nbsp;to the directory where you downloaded the installer.</span></li>\r\n<li>At the prompt, type&nbsp;<span><b><code>unzip install.zip</code></b>&nbsp;to unzip the installer.</span></li>\r\n<li>Use ./ReactomeCuratorTool.sh to launch the Reactome Curator Tool.</li>\r\n</ul>\r\n<h5>Notes</h5>\r\n<ul>\r\n<li><span>You need to install a Java 1.8.0 (or later) virtual machine. You can download one from&nbsp;<a href=\"http://java.sun.com/\" target=\"_blank\" rel=\"noopener noreferrer\">Sun\'s Java web site</a>&nbsp;or contact your OS manufacturer.</span></li>\r\n</ul>\r\n<p>(<a href=\"#top\">Go To Top</a>)</p>\r\n</div>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-23 10:53:19','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(64,668,'Developer\'s Zone','developer-s-zone','<h4>Explore our tools and web services and learn how to include them in your applications</h4>\r\n<p>{loadposition dev-guide-1}{loadposition dev-guide-2}</p>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-29 11:12:14','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(66,675,'Statistics','statistics','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/stats/stats_v70.png\" alt=\"stats v69\" width=\"967\" height=\"576\" /></p>\r\n<div id=\"r-responsive-table\">\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome margin margin0\">\r\n<thead>\r\n<tr><th>SPECIES</th><th>PROTEINS</th><th>COMPLEXES</th><th>REACTIONS</th><th>PATHWAYS</th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td data-label=\"SPECIES\">D. discoideum</td>\r\n<td data-label>2729</td>\r\n<td data-label>2889</td>\r\n<td data-label>2357</td>\r\n<td data-label>981</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">P. falciparum</td>\r\n<td data-label>1162</td>\r\n<td data-label>1160</td>\r\n<td data-label>882</td>\r\n<td data-label>612</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">S. pombe</td>\r\n<td data-label>1823</td>\r\n<td data-label>2055</td>\r\n<td data-label>1498</td>\r\n<td data-label>788</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">S. cerevisiae</td>\r\n<td data-label>1927</td>\r\n<td data-label>1993</td>\r\n<td data-label>1537</td>\r\n<td data-label>783</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">C. elegans</td>\r\n<td data-label>4357</td>\r\n<td data-label>4476</td>\r\n<td data-label>3555</td>\r\n<td data-label>1273</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">S. scrofa</td>\r\n<td data-label>8653</td>\r\n<td data-label>7454</td>\r\n<td data-label>6350</td>\r\n<td data-label>1570</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">B. taurus</td>\r\n<td data-label>8267</td>\r\n<td data-label>7705</td>\r\n<td data-label>6406</td>\r\n<td data-label>1570</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">C. familiaris</td>\r\n<td data-label>8382</td>\r\n<td data-label>8008</td>\r\n<td data-label>6848</td>\r\n<td data-label>1601</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">M. musculus</td>\r\n<td data-label>9665</td>\r\n<td data-label>10068</td>\r\n<td data-label>8837</td>\r\n<td data-label>1654</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">R. norvegicus</td>\r\n<td data-label>8879</td>\r\n<td data-label>9131</td>\r\n<td data-label>7842</td>\r\n<td data-label>1639</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">*H. sapiens</td>\r\n<td data-label>10860</td>\r\n<td data-label>12335</td>\r\n<td data-label>12608</td>\r\n<td data-label>2287</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">G. gallus</td>\r\n<td data-label>7199</td>\r\n<td data-label>6712</td>\r\n<td data-label>5706</td>\r\n<td data-label>1567</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">X. tropicalis</td>\r\n<td data-label>7477</td>\r\n<td data-label>7830</td>\r\n<td data-label>6673</td>\r\n<td data-label>1603</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">D. rerio</td>\r\n<td data-label>8150</td>\r\n<td data-label>8201</td>\r\n<td data-label>6958</td>\r\n<td data-label>1611</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">D. melanogaster</td>\r\n<td data-label>4495</td>\r\n<td data-label>5124</td>\r\n<td data-label>4189</td>\r\n<td data-label>1414</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"SPECIES\">M. tuberculosis</td>\r\n<td data-label>18</td>\r\n<td data-label>46</td>\r\n<td data-label>44</td>\r\n<td data-label>13</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<p></p>\r\n<p>*Reactome annotates to protein isoforms when this information is available. The total number of curated human proteins including isoforms is 11040.</p>\r\n<br />\r\n<p><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"><span style=\"color: #000000; font-style: normal; font-weight: 400; letter-spacing: normal; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; word-spacing: 0px; float: none;\"></span></span></p>\r\n<p></p>\r\n</div>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-07-01 20:24:33','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(67,676,'ORCID Integration Project','orcid-integration-project','<p style=\"text-align: justify;\">Biological information has become so abundant and complex in recent years that it is difficult, if not impossible, even for expert individuals to manage in traditional publication formats and with existing knowledge management tools. It is an ongoing challenge for researchers to keep up-to-date on research developments in their fields, and identify relevant research to support their own studies without devoting too much time collecting unconnected information. The Reactome group has recognized this challenge and is developing a set of novel online resources that use features of the electronic media to organize biological information in ways that provide for more efficient access and that allow new forms of analysis that were not possible with information stored in the traditional printed literature.</p>\r\n<p style=\"text-align: justify;\">The curation process for a pathway is similar to the editing of a scientific review. An external domain expert provides his or her expertise, one of our curators formalizes it into the database structure, and a second external domain expert reviews the representation. A system of evidence tracking ensures that all assertions are backed up by the primary literature. We have created data entry software, called the Author and Curator Tools, and associated web services, which provide interfaces in which the pathway authors can systematize their knowledge and a mechanism by which researchers, students, and clinicians can transform the information about interactions between biomolecules into knowledge about a cellular process.</p>\r\n<p style=\"text-align: justify;\">Reactome has evolved into the one of the largest pathway databases, containing over 19,000 human and inferred model organism pathways, and is accessed by over 15,000 unique visitors per month all around the world. A key challenge for&nbsp;us is to incentivize the unpaid external domain experts to contribute their expertise and time to the curation process. In the scientific process, the key incentive is credit attribution.&nbsp;</p>\r\n<p style=\"text-align: justify;\">As&nbsp;<a href=\"http://orcid.org/\">ORCID</a>&nbsp;is now part of our data model, we are constantly annotating&nbsp;curators, editors, and expert authors and reviewers with their respective ORCID. Thus, if your ORCID isn\'t annotated, please let us know directly from your&nbsp;Authors&nbsp;page (search for you name first) and one of our curators will add it and we will make it be available on the next&nbsp; data release. In the Authors page, once logged in ORCID a button \"Let us know your orcid\" should appear and you can share more information using our internal form.</p>\r\n<hr />\r\n<div class=\"alert alert-success\" role=\"alert\">Our main search allows&nbsp;<strong>Authors</strong> and <strong>Reviewers</strong> to <a href=\"/content/query?q=*&amp;species=Entries+without+species&amp;types=Person&amp;cluster=true\" target=\"_blank\">query</a>&nbsp; for their pathway and reaction contributions using their names, and claim their contributions directly into ORCID.</div>\r\n<hr />\r\n<h3>What is ORCID?</h3>\r\n<section>\r\n<p style=\"text-align: justify;\">ORCID (Open Researcher and Contributor ID) is an open, non-profit, community-driven effort to create and maintain a registry of unique researcher identifiers and a transparent method of linking research activities and outputs to these identifiers.&nbsp;</p>\r\n<p style=\"text-align: justify;\">Individuals can obtain a free ORCID (Open Researcher and Contributor ID) identifier. This digital identifier distinguishes individual researchers from other researchers and enables them to manage their records and search for others in the Registry.</p>\r\n</section>\r\n<h4 style=\"padding-left: 30px;\">Authenticate</h4>\r\n<p style=\"padding-left: 30px; text-align: justify;\"><img style=\"float: left;\" title=\"This integration has been recognized by ORCID in 2019 as verifying ORCID iDs using authentication -- by asking users to sign into their ORCID account.\" src=\"images/others/orcid/orcid_auth.png\" alt=\"\" width=\"43\" height=\"67\" /> Reactome is collecting your ORCID iD so we can add Pathways and Reactions (Works) that you have contributed either as author or reviewer into your ORCID records. When you click the Authorize button, we will ask you to share your iD using an authenticated process: either by <a href=\"https://support.orcid.org/hc/articles/360006897454\">registering for an ORCID iD</a> or, if you already have one, by <a href=\"https://support.orcid.org/hc/articles/360006971613\">signing into your ORCID account</a>, then granting us permission to get your ORCID iD. We do this to ensure that you are correctly identified and securely connecting your ORCID iD. Learn more about <a href=\"https://orcid.org/blog/2017/02/20/whats-so-special-about-signing\">Whats so special about signing in</a></p>\r\n<p></p>\r\n<h4 style=\"padding-left: 30px;\">Display</h4>\r\n<p style=\"padding-left: 30px; text-align: justify;\"><img style=\"float: left;\" title=\"This integration has been recognized by ORCID in 2019 as displaying verified ORCID iDs publicly.\" src=\"images/others/orcid/orcid_display.png\" alt=\"display logo\" width=\"43\" height=\"67\" /> To acknowledge that you have used your iD and that it has been authenticated, we display the ORCID iD icon&nbsp;<img src=\"images/others/orcid_16x16.png\" alt=\"ORCID Logo\" /> alongside your name on our website/in our publications/in our database etc. Learn more in <a href=\"https://orcid.org/blog/2013/02/22/how-should-orcid-id-be-displayed\">How should an ORCID iD be displayed</a>.</p>\r\n<p></p>\r\n<h4 style=\"padding-left: 30px;\">Connect</h4>\r\n<p style=\"padding-left: 30px; text-align: justify;\"><img style=\"float: left;\" title=\"This integration has been recognized by ORCID in 2019 as connecting data that it can uniquely identify to researchers\' ORCID records using the ORCID API.\" src=\"images/others/orcid/orcid_connect.png\" alt=\"connect logo\" width=\"43\" height=\"67\" /> By sharing your iD with Reactome, and giving us permission to <a href=\"https://support.orcid.org/hc/articles/360006896394\">read and update your ORCID record</a>, you enable us to help you keep your record up-to-date with trusted information. Learn more in <a href=\"https://orcid.org/blog/2017/08/10/six-ways-make-your-orcid-id-work-you\">Six ways to make your ORCID iD work for you!</a></p>\r\n<p></p>\r\n<p></p>\r\n<p></p>\r\n<p>For more detailed information about ORCID or to register for an ORCID iD go the&nbsp;<a style=\"text-align: justify; background-color: inherit; font-family: inherit; font-size: 1rem;\" href=\"https://orcid.org/\">ORCID website</a>.</p>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-07-01 20:26:15','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"1\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(68,682,'Computationally Inferred Events','computationally-inferred-events','<p style=\"text-align: justify;\"><span>We use the set of manually curated human reactions to electronically infer reactions in fifteen evolutionarily divergent eukaryotic species for which high-quality whole-genome sequence data are available, and hence a comprehensive and high-quality set of protein predictions exists. These species include the laboratory mouse and rat, the nematode </span><i><span>C. elegans</span></i><span>, and budding and fission yeasts. The estimated success rates of our orthology inference strategy can be stated as the percentage of eligible reactions, defined in step 2 below, in the current human data set for which an event can be inferred in the model organism. By this measure, success rates range from 84.56% for the laboratory mouse to&nbsp;9.60% for the protozoan </span><i><span>P. falciparum</span></i><span>.</span></p>\r\n<p>Electronic inference proceeds in four steps:</p>\r\n<ol>\r\n<li>Protein homology data were obtained from <a href=\"http://www.pantherdb.org/\">PANTHER</a>. PANTHER uses the <a href=\"https://www.ebi.ac.uk/reference_proteomes\">Reference proteome</a> dataset maintained by UniProt to generate phylogenetic trees of protein-coding genes across numerous species. Homologs are derived from these trees and annotated by type (ortholog, paralog). Additionally, in cases with multiple orthologs PANTHER infers the least diverged ortholog based on protein sequence divergence, which is utilized during our inference process. A detailed description of PANTHER\'s methodology can be found in <a href=\"https://academic.oup.com/nar/article/41/D1/D377/1060482\">PANTHER in 2013: modeling the evolution of gene function, and other gene attributes, in the context of phylogenetic trees</a>.</li>\r\n</ol><ol start=\"2\">\r\n<li>All human reactions in the Reactome knowledgebase involving one or more proteins are eligible for electronic inference, with two exceptions. Reactions that were themselves inferred based on data from the model organism, and reactions involving species in addition to human (e.g., HIV infection of human cells) are excluded from electronic inference. Eligible reactions are checked to determine whether each involved protein has at least one homologous protein (HP) in the reaction\'s input, output and (if present) catalyst in the organism undergoing inference. If a human reaction involves a complex, at least 75% of the accessioned protein components of the human complex must have HPs in the model organism.</li>\r\n</ol><ol start=\"3\">\r\n<li>For each reaction that meets these criteria, an equivalent reaction is created for the model organism by replacing each human protein with its model organism HP. If a human protein corresponds to more than one model organism HP, a DefinedSet called Homologues of  is created, with the model organism HPs as members.</li>\r\n</ol><ol start=\"4\">\r\n<li>After all possible reactions have been inferred for the species, any human pathways that contain at least 1 inferred reaction will be inferred for the species as well.</li>\r\n</ol>\r\n<p>These electronically inferred reactions are predictions based on a number of assumptions. Most basically, we assume that if we can find model organism HPs corresponding to all proteins involved in a human reaction, then the proteins mediate the same reaction in the model organism. This may not be true. On the other hand, we may miss a truly homologous reaction in the model organism because it is mediated by structurally divergent proteins that were not identified as such by PANTHERs techniques. Similarly, complexes sharing less than 75% homologous subunits between species may nevertheless continue to perform the same function. The electronically inferred reactions presented in Reactome are thus not data, but hypotheses useful to direct the design of confirmatory experiments.</p>\r\n<p><img src=\"images/stats/inference_stats_v70.png\" alt=\"inference stats v69\" width=\"891\" height=\"697\" /></p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-18 14:47:18','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(69,683,'Papers citing us','papers-citing-us','<p><span >Since 2003, Reactome papers have been cited in hundreds of textbooks and thousands of peer-reviewed articles and reviews, and the citation rate is increasing annually. Several papers cite the use of Reactome data for either integrative data analysis, algorithm development, or the inclusion of Reactome pathway annotations into a secondary bioinformatics database. </span></p>\r\n<p><span >A full listing of all the papers and books citing Reactome is accessible online at </span><a href=\"https://scholar.google.ca/citations?user=gtB1MLcAAAAJ\"><span >Google Scholar</span></a><span >.</span></p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-18 15:21:31','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(70,684,'External Identifiers','external-identifiers','<p><span >Listed below are the external bioinformatics databases that Reactome provides link outs to from our website.</span></p>\r\n<ul>\r\n<li><a href=\"http://pathway.gramene.org/\"><span >AraCyc</span></a></li>\r\n<li><a href=\"http://biogps.org/\"><span >BioGPS</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/biomodels/\"><span >BioModels</span></a></li>\r\n<li><a href=\"http://www.brenda-enzymes.org/\"><span >Brenda</span></a></li>\r\n<li><a href=\"http://www.cas.org/\"><span >CAS</span></a></li>\r\n<li><a href=\"http://www.genome.ad.jp/kegg/\"><span >COMPOUND</span></a></li>\r\n<li><a href=\"http://cancer.sanger.ac.uk/cosmic\"><span >COSMIC</span></a></li>\r\n<li><a href=\"http://ctdbase.org/\"><span >CTD Gene</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/chebi\"><span >ChEBI</span></a></li>\r\n<li><a href=\"https://www.ncbi.nlm.nih.gov/projects/SNP/\"><span >dbSNP Gene</span></a></li>\r\n<li><a href=\"http://www.dictybase.org/\"><span >dictyBase</span></a></li>\r\n<li><a href=\"http://blaster.docking.org/\"><span >DOCK Blaster</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/ols/ontologies/doid\"><span >DOID</span></a></li>\r\n<li><a href=\"http://ca.expasy.org/enzyme/\"><span >EC</span></a></li>\r\n<li><a href=\"http://www.ensembl.org/\"><span >ENSEMBL</span></a></li>\r\n<li><a href=\"http://www.flybase.net/\"><span >Flybase</span></a></li>\r\n<li><a href=\"http://www.geneontology.org/\"><span >GO</span></a></li>\r\n<li><a href=\"http://www.genecards.org/\"><span >GeneCards</span></a></li>\r\n<li><a href=\"http://www.pombase.org/\"><span >GeneDB</span></a></li>\r\n<li><a href=\"http://www.hapmap.org/\"><span >HapMap</span></a></li>\r\n<li><a href=\"http://www.genenames.org/\"><span >HGNC</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/intenz/\"><span >IntEnz</span></a></li>\r\n<li><a href=\"http://www.genome.jp/\"><span >KEGG Gene</span></a></li>\r\n<li><a href=\"http://www.mirbase.org/\"><span >miRBase</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/ols/ontologies/mod\"><span >MOD</span></a></li>\r\n<li><a href=\"http://www.ncbi.nlm.nih.gov/gene\"><span >NCBI Gene</span></a></li>\r\n<li><a href=\"http://www.ncbi.nlm.nih.gov/nuccore\"><span >NCBI Nucleotide</span></a></li>\r\n<li><a href=\"http://www.ncbi.nlm.nih.gov/protein\"><span >NCBI_Protein</span></a></li>\r\n<li><a href=\"http://omim.org/\"><span >OMIM</span></a></li>\r\n<li><a href=\"http://orchid.org/\"><span >ORCID</span></a></li>\r\n<li><a href=\"http://www.orpha.net/\"><span >Orphanet</span></a></li>\r\n<li><a href=\"http://www.genome.jp/dbget-bin/www_bfind?prf\"><span >PRF</span></a></li>\r\n<li><a href=\"http://plasmodb.org/\"><span >PlasmoDB</span></a></li>\r\n<li><a href=\"http://www.pdb.org/\"><span >Protein Data Bank</span></a></li>\r\n<li><a href=\"https://www.ncbi.nlm.nih.gov/pccompound\"><span >PubChem Compound</span></a></li>\r\n<li><a href=\"https://www.ncbi.nlm.nih.gov/pcsubstance\"><span >PubChem Substance</span></a></li>\r\n<li><a href=\"https://www.ncbi.nlm.nih.gov/refseq/\"><span >RefSeq</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/rhea/\"><span >Rhea</span></a></li>\r\n<li><a href=\"http://www.yeastgenome.org/\"><span >SGD</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/ols/ontologies/so\"><span >SO</span></a></li>\r\n<li><a href=\"http://www.arabidopsis.org/index.jsp\"><span >TAIR</span></a></li>\r\n<li><a href=\"http://genome.ucsc.edu/\"><span >UCSC human</span></a></li>\r\n<li><a href=\"http://www.uniprot.org/\"><span >UniProt</span></a></li>\r\n<li><a href=\"http://www.wormbase.org/\"><span >Wormbase</span></a></li>\r\n<li><a href=\"http://zinc15.docking.org/\"><span >ZINC</span></a></li>\r\n</ul>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-19 19:46:26','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(71,685,'Version 57 Released','version-57-released','<p>In version V57, topics with new or revised pathways include: Developmental biology (<a href=\"/PathwayBrowser/#R-HSA-8853659.1\">RET signaling</a>), Disease (<a href=\"/PathwayBrowser/#R-HSA-1226099.3\">Signaling by FGFR in disease</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-5678895.2\">Defective CFTR causes cystic fibrosis</a>), Gene expression (<a href=\"/PathwayBrowser/#R-HSA-6793080.1\">rRNA modification in the mitochondrion</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8864260.1\">Transcriptional regulation by the AP-2 (TFAP2) family of transcription factors</a>), Immune system (<a href=\"/PathwayBrowser/#R-HSA-1236974.2\">ER-Phagosome pathway</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-1266695.3\">Interleukin-7 signaling</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-5686938.1\">Regulation by endogenous TLR ligand</a>, and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-202403.3\">TCR signaling</a>), Metabolism (<a href=\"/PathwayBrowser/#R-HSA-73923.3\">Lipid digestion, mobilization, and transport</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8850843.1\">Phosphate bond hydrolysis by NTPDase proteins</a>). Metabolism of proteins (<a href=\"/PathwayBrowser/#R-HSA-5688426.1\">Deubiquitination</a>), Neuronal system (<a href=\"/PathwayBrowser/#/R-HSA-6794361\">Interactions of neurexins and neuroligins at synapses</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-8849932\">SALM protein interactions at synapse</a>). Signal transduction (<a href=\"/PathwayBrowser/#R-HSA-182971.3\">EGFR downregulation</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-5654736.3\">Signaling by FGFR1</a>, and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-5658623.2\">FGFRL1 modulation of FGFR1 signaling</a>), Transmembrane transport of small molecules (<a href=\"/PathwayBrowser/#R-HSA-382556.2\">ABC-family proteins mediated transport</a>) and Vesicle-mediated transport (<a href=\"/PathwayBrowser/#R-HSA-8856828.1\">Clathrin-mediated endocytosis</a>).</p>\r\n<p>Our external author is&nbsp;<a href=\"http://scholar.google.com/citations?hl=en&amp;user=hdIojjoAAAAJ\">Alba Sanchis</a>.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Antonescu++Costin+N%5BAuthor+-+Full%5D\">Costin Antonescu</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=John++J.M+Bergeron%5BAuthor+-+Full%5D\">John Bergeron</a><span style=\"text-decoration: underline;\">,</span>&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Daniel+F+Bogenhagen%5BAuthor+-+Full%5D\">Daniel Bogenhagen</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nunzio++Bottini%5BAuthor+-+Full%5D\">Nunzio Bottini</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Guang-Chao+Chen%5BAuthor+-+Full%5D\">Guang-Chao Chen</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Igor+B+Dawid%5BAuthor+-+Full%5D\">Igor Dawid</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Regina+Fluhrer%5BAuthor+-+Full%5D\">Regina Fluhrer</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Noriko++Gotoh%5BAuthor+-+Full%5D\">Noriko Gotoh</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Francesca+Granucci%5BAuthor+-+Full%5D\">Francesca Granucci</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Richard+P+Grose%5BAuthor+-+Full%5D\">Richard Grose</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Paul+A+Heppenstall%5BAuthor+-+Full%5D\">Paul Heppenstall</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Hu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26443885\">Jing Hu</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Jan%20Huertas%5BAuthor%20-%20Full%5D\">Jan Huertas</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Luo%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26992078\">Wenqin Luo</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Malay%20Mandal%5BAuthor%20-%20Full%5D\">Malay Mandal</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Birgit+Meldal%5BAuthor+-+Full%5D\">Birgit Meldal</a>,&nbsp;<a href=\"http://scholar.google.ca/citations?user=qux0Tt4AAAAJ&amp;hl=en\">Daniel Morales</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Tatsunori+Nishimura%5BAuthor+-+Full%5D\">Tatsunori Nishimura</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ronald++S+Petralia%5BAuthor+-+Full%5D\">Ronald Petralia</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Gail+K+Seabold%5BAuthor+-+Full%5D\">Gail Seabold</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Sunny+Sharma%5BAuthor+-+Full%5D\">Sunny Sharma</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Stephanie+Stanford%5BAuthor+-+Full%5D\">Stephanie Stanford</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jean+S%C3%A9vigny%5BAuthor+-+Full%5D\">Jean Svigny</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Philip+Washbourne%5BAuthor+-+Full%5D\">Philip Washbourne</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ivan+Zanoni%5BAuthor+-+Full%5D\">Ivan Zanoni</a>, and&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Valeria+Zarelli%5BAuthor+-+Full%5D\">Valeria Zarelli</a>&nbsp;are our external reviewers<strong>.</strong></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:13:06','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,41,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(72,686,'Version 58 Released','version-58-released','<p><strong><a href=\"/PathwayBrowser/#R-HSA-8862803\"><img class=\"aligncenter wp-image-3812\" style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/illustration_deregulated_CDK5_triggers_multiple_neurodegenerative_pathways_72.png\" alt=\"illustration deregulated CDK5 triggers multiple neurodegenerative pathways 72\" width=\"407\" height=\"400\" /></a></strong></p>\r\n<p><strong>New and Updated Pathways.&nbsp;</strong>With version V58, Reactome has annotations for over 10,000 human proteins. New or revised pathways include:&nbsp;Cell cycle (<a href=\"/PathwayBrowser/#R-HSA-8854050\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8854050\">FBXL7 down-regulates AURKA during mitotic entry and in early mitosis</a>), Developmental biology (<a href=\"/PathwayBrowser/#R-HSA-6805567\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6805567\">Keratinization</a>), Disease (<a href=\"/PathwayBrowser/#R-HSA-6802957\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6802957\">Oncogenic MAPK signaling</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8862803\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8862803\">Deregulated CDK5 triggers multiple neurodegenerative pathways in Alzheimers disease models</a>), Gene expression (<a href=\"/PathwayBrowser/#R-HSA-6811555\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6811555\">PI5P Regulates TP53 Acetylation</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8864260\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8864260\">Transcriptional regulation by the AP-2 (TFAP2) family of transcription factors</a>),&nbsp; Immune system&nbsp; (<a href=\"/PathwayBrowser/#R-HSA-6798695\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6798695\">Neutrophil degranulation</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-6803157\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6803157\">Antimicrobial peptides</a>), Metabolism of proteins (<a href=\"/PathwayBrowser/#R-HSA-8866652\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8866652\">Synthesis of active ubiquitin: roles of E1 and E2 enzymes</a>), Signal transduction (<a href=\"/PathwayBrowser/#R-HSA-6806834\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6806834\">Signaling by MET</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8863795\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8863795\">Downregulation of ERBB2 signaling</a>), and Vesicle-mediated transport (<a href=\"/PathwayBrowser/#R-HSA-8876198\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8876198\">RAB GEFs exchange GTP for GDP on RABs</a>).</p>\r\n<p>A pathway illustration is available for&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8862803\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8862803\">Deregulated CDK5 triggers multiple neurodegenerative pathways in Alzheimers disease&nbsp;</a>.</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;Our external author is&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Shah%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26962010\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Shah%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26962010\">Kavita Shah</a>. &nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ayoub%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23897807\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ayoub%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23897807\">Emily Ayoub</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Azevedo%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26235645\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Azevedo%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26235645\">Jorge Azevedo</a>,&nbsp;<a href=\"http://www.mdc-berlin.de/1154093/en/research/research_teams/signal_transduction_\">Walter Birchmeier</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Blumenberg%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26933871\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Blumenberg%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26933871\">Miroslav Blumenberg</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bogachek%20MV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25524900\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bogachek%20MV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25524900\">Maria Bogachek</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nullin+Divecha%5BAuthor%5D\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nullin+Divecha%5BAuthor%5D\">Nullin Divecha</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Roman%20Dziarski%5BAuthor%20-%20Full%5D\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Roman%20Dziarski%5BAuthor%20-%20Full%5D\">Roman Dziarski</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Rhys+Grant%5BAuthor%5D\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Rhys+Grant%5BAuthor%5D\">Rhys Grant</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Hains%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21357198\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Hains%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21357198\">David Hains</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Heegaard%20NH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26608204\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Heegaard%20NH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26608204\">Niels Heegard</a>,&nbsp;<a href=\"http://www.mdc-berlin.de/1154093/en/research/research_teams/signal_transduction_\">Guustaaf Heynen</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Lindon%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26835416\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Lindon%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26835416\">Catherine Lindon</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Andrea+Marat%5BAuthor%5D\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Andrea+Marat%5BAuthor%5D\">Andrea Marat</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Stephens%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27557867\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Stephens%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27557867\">Robert Stephens</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Tremblay%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27020859\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Tremblay%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27020859\">Michel Tremblay</a>, and&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Weigel%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25524900\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Weigel%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25524900\">Ronald Weigel</a>&nbsp;are our external reviewers.</p>\r\n<p>Reactome comprises 10,168 human reactions organized into 2,069 pathways involving 10,461 proteins encoded by 10,221 different human genes, and 1,710 small molecules. These annotations are supported by 24,974 literature references. We have projected these reactions onto 110,710 orthologous proteins, creating 19,991 orthologous pathways in 18 non-human species.</p>\r\n<p>Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, New York University Medical Center, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the&nbsp;<a href=\"/\" data-cke-saved-href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"https://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p>For more information please contact our&nbsp;<a href=\"mailto:help@reactome.org\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:14:30','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,40,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(73,687,'Reactome celebrates release of 10,000th annotated protein','reactome-celebrates-release-of-10-000th-annotated-protein','<p><a href=\"/\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/10K_Reactome.png\" alt=\"10K Reactome\" width=\"440\" height=\"400\" /></a></p>\r\n<p>The Reactome team is pleased to announce that it met a major milestone in October 2016 with the annotation and release of its 10,000th human protein. Reactome (<a href=\"/\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/\">www.reactome.org</a>) is an open access curated knowledgebase which relates human genes, proteins and other biomolecules to the biological pathways and processes in which they participate. It is a key resource for the biomedical research community, and is widely used by researchers around the world to interpret high-throughput experiments in genetics, genomics and proteomics. Given that the human genome contains roughly 20,000 protein-coding genes in total, the annotation of the 10,000th protein means that Reactome now covers half of the protein-coding portion of the genome. This makes Reactome the most comprehensive open access pathway knowledgebase available to the scientific community.</p>\r\n<p>By relating genes and proteins to normal and abnormal biological pathways, Reactome allows researchers to identify patterns in large data sets. For example, researchers can use Reactome to reduce an experiment that identified thousands of genes whose activities are altered in a disease to a manageable number of key biological pathways that are disrupted by these changes. Researchers can then combine Reactome with other databases to find drugs and protein targets that might reverse the pathway alterations, or to devise ways of diagnosing the disease at an early stage. Via its web site, online tools, and specialized visualization and analysis applications, Reactome has been incorporated into more than 400 third-party genome analysis tools, and has been cited more than 4,000 times in the scientific literature.&nbsp;</p>\r\n<p>Reactome has been in continuous operation since 2004 and is an international collaboration among the Ontario Institute for Cancer Research in Canada, New York University School of Medicine in the United States, and the European Bioinformatics Institute in the United Kingdom. It is staffed by expert biological curators, bioinformaticians and computer scientists. Much of its content is provided by community authors and peer reviewers who are assisted by the curatorial staff. The Reactome content, including pathway data and the software infrastructure, are available to all comers free of charge under a Creative Commons open access license. Reactome is supported by grants from the US National Institutes of Health, the Ontario Research Fund, the University of Toronto, OpenTargets, Genome Canada, and the European Molecular Biology Laboratory.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"https://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p>For more information please contact our&nbsp;<a href=\"mailto:help@reactome.org\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:15:27','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,39,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(74,688,'Version 59 Released','version-59-release','<p><strong>New and Updated Pathways.&nbsp;</strong>In version V59, topics with new or revised pathways include: Disease (<a href=\"/PathwayBrowser/#/R-HSA-8876384&amp;PATH=R-HSA-1643685,R-HSA-5663205\">Listeria monocytogenes entry into host cells</a>), Hemostasis (<a href=\"/PathwayBrowser/#R-HSA-202733\">Cell surface interaction at the vascular wall</a>), Immune System (<a href=\"/PathwayBrowser/#R-HSA-8851680\">Butyrophilins</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-6783783\">Interleukin 10 signalling</a>, and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-6785807\">Interleukin-4 and 13 signaling</a>), Metabolism (<a href=\"/PathwayBrowser/#R-HSA-196807\">Nicotinate metabolism</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8847453\">Synthesis of PIPs at the nuclear envelope</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-199220\">Vitamin B5 (pantothenate) metabolism</a>, and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8937144\">Aryl hydrocarbon receptor signalling</a>), Metabolism of proteins (<a href=\"/PathwayBrowser/#R-HSA-8866654\">E3 ubiquitin ligases ubiquitinate target proteins</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-375276\">Peptide-ligand binding receptors</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8876725\">Protein methylation</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8873719\">RAB geranylgeranylation</a>), Signal transduction (<a href=\"/PathwayBrowser/#R-HSA-373076\">Class A/1 (Rhodopsin-like receptors</a>), and Vesicle-mediated transport (<a href=\"/PathwayBrowser/#R-HSA-8854214\">TBC RABGAPs</a>).</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;Our external reviewers are&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Jorge%20E%20Azevedo%5BAuthor%20-%20Full%5D\">Jorge Azevedo</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Ester%20Boix%5BAuthor%20-%20Full%5D\">Ester Boix</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Lu%20Deng%5BAuthor%20-%20Full%5D\">Lu Deng</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Pal%20Falnes%5BAuthor%20-%20Full%5D\">Pl Falnes</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Vardan%20Karamyan%5BAuthor%20-%20Full%5D\">Vardan Karamyan</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Samuel%20J%20Leibovich%5BAuthor%20-%20Full%5D\">Samuel Leibovich</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Weei-Chin%20Lin%5BAuthor%20-%20Full%5D\">Weei-Chin Lin</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Charuta%20Palsuledesai%5BAuthor%20-%20Full%5D\">Charuta Palsuledesai</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Joel%20Pomerantz%5BAuthor%20-%20Full%5D\">Joel Pomerantz</a>,&nbsp;<a href=\"http://www.unige.ch/medecine/pati/en/groupes/282reith/\">Walter Reith</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Sylvie%20Ricard-Blum%5BAuthor%20-%20Full%5D\">Sylvie Ricard-Blum</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Christian%20Schwerk%5BAuthor%20-%20Full%5D\">Christian Schwerk</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Bruce%20Spiegelman\">Bruce Spiegelman</a>, and&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Xiaochun%20Yu%5BAuthor%20-%20Full%5D\">Xiaochun Yu</a>.</p>\r\n<p><strong>Annotation Statistics.</strong>&nbsp;Reactome comprises 10,391 human reactions organized into 2,080 pathways involving 10,624 proteins encoded by 10,381 different human genes, and 1,735 small molecules. These annotations are supported by 25,449 literature references. We have projected these reactions onto 115,881 orthologous proteins, creating 20,164 orthologous pathways in 18 non-human species. Version 59 has annotations for 1,496 protein variants (mutated proteins) and their post-translationally modified forms, derived from 285 proteins. These have been used to annotate 506 disease-specific complexes and 897 disease-specific reactions organized into 447 pathways and subpathways, and tagged with 294 Disease Ontology terms.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, New York University Medical Center, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;<a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"https://twitter.com/reactome\">@reactome</a></strong>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<a href=\"mailto:help@reactome.org\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:16:00','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,38,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(75,689,'New Reactome Publication','new-reactome-publication','<p>A new Reactome paper titled Functional Interaction Network Construction and Analysis for Disease Discovery has been published in&nbsp;<a href=\"http://link.springer.com/protocol/10.1007%2F978-1-4939-6783-4_11\">Methods in Molecular Biology</a>.&nbsp;More publications from the&nbsp;<a href=\"about/team\">Reactome Team</a>&nbsp;can be found&nbsp;<a href=\"https://reactome.org/community/publications\">here</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:16:36','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,37,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(76,690,'New Reactome Paper published','new-reactome-paper-published','<p>A new Reactome paper titled Reactome pathway analysis: a high-performance in-memory approach has been published in&nbsp;<a href=\"https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1559-2\">BMC Bioinformatics</a>.&nbsp;More publications from the&nbsp;<a href=\"about/team\">Reactome Team</a>&nbsp;can be found&nbsp;<a href=\"community/publications\">here</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:17:08','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,36,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(77,691,'Version 60 Released','version-60-released','<p><strong>New and Updated Pathways.</strong><strong>&nbsp;</strong>In version V60, topics with new or revised pathways include: Cell cycle (<a href=\"/PathwayBrowser/#R-HSA-69231.4\">Cyclin D associated events in G1</a>), Cell-cell communication&nbsp;<a href=\"/PathwayBrowser/#R-HSA-373756.1\">(SDK interactions</a>), Cellular response to external stimuli (<a href=\"/PathwayBrowser/#R-HSA-3371497.1\">HSP90 chaperone cycle for steroid hormone receptors (SHR)</a>), Disease (<a href=\"/PathwayBrowser/#R-HSA-5423599.1\">Diseases of Mismatch Repair</a>), Gene Expression (<a href=\"/PathwayBrowser/#R-HSA-6804116.2\">TP53 Regulates Transcription of Genes Involved in G1 Cell Cycle Arrest</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8878159.1\">Transcriptional Regulation by the CBFB:RUNX3 complex</a>), Immune System (<a href=\"/PathwayBrowser/#R-HSA-8851680.2\">Butyrophilins</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-977606.3\">Regulation of complement cascade</a>), Metabolism (<a href=\"/PathwayBrowser/#R-HSA-1855183.4\">Synthesis of IP2, IP, and Ins in the cytosol</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-1660516.6\">Synthesis of PIPs at the early endosome membrane</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-1483248.5\">Synthesis of PIPs at the ER membrane</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-1660517.5\">Synthesis of PIPs at the late endosome membrane</a>, and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-1660499.5\">Synthesis of PIPs at the plasma membrane</a>), Metabolism of proteins (<a href=\"/PathwayBrowser/#R-HSA-8874211.1\">CREB3 factors activate genes</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8951664.1\">Neddylation</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8852135.1\">Protein ubiquitination</a>, and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-4551638.4\">SUMOylation of chromatin organizing proteins</a>), Mitophagy (<a href=\"/PathwayBrowser/#R-HSA-8934903.1\">Receptor Mediated Mitophagy</a>), Neuronal System (<a href=\"/PathwayBrowser/#R-HSA-388844.1\">Receptor protein tyrosine phosphatases interactions</a>), Organelle biogenesis and maintenance (<a href=\"/PathwayBrowser/#R-HSA-8949613.1\">Cristae formation</a>), and Transport of small molecules (<a href=\"/PathwayBrowser/#R-HSA-8949215.1\">Mitochondrial calcium ion transport</a>).</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Wei-Chih+Yang%5BAuthor+-+Full%5D\">Wei-Chih Yang</a>&nbsp;and&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jian+Lu%5BAuthor+-+Full%5D\">Jian Lu</a>&nbsp;are our external &nbsp;authors.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Joseph+Ainscough%5BAuthor+-+Full%5D\">Joseph Ainscough</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Sanjeevani+Arora%5BAuthor+-+Full%5D\">Sanjeevani &nbsp;Arora</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jorge+E+Azevedo%5BAuthor+-+Full%5D\">Jorge E Azevedo</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jeehyeon+Bae%5BAuthor+-+Full%5D\">, &nbsp;Jeehyeon Bae</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Lucia+Banci%5BAuthor+-+Full%5D\">Lucia Banci</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Gautam+Bhave%5BAuthor+-+Full%5D\">Gautam Bhave</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=David+R+Brown%5BAuthor+-+Full%5D\">David R Brown</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Roberta+Bulla%5BAuthor+-+Full%5D\">Roberta Bulla</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Alexandre+M+Carmo%5BAuthor+-+Full%5D\">Alexandre M Carmo</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Linda+Shyue+Huey+Chuang%5BAuthor+-+Full%5D\">Linda Shyue Huey Chuang</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Karlene+A+Cimprich%5BAuthor+-+Full%5D\">Karlene A Cimprich</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Laura+Crisponi%5BAuthor+-+Full%5D\">Laura Crisponi</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Alain+de+Bruin%5BAuthor+-+Full%5D\">Alain de Bruin</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Luisa+Di+Stefano%5BAuthor+-+Full%5D\">, Luisa Di Stefano</a>&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ilaria+Drago%5BAuthor+-+Full%5D\">, Ilaria Drago</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Pablo+C+Echeverria%5BAuthor+-+Full%5D\">Pablo C Echeverria</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Du+Feng%5BAuthor+-+Full%5D\">Du Feng</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Emer+S+Ferro%5BAuthor+-+Full%5D\">Emer S Ferro</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Dianne+Ford%5BAuthor+-+Full%5D\">Dianne Ford</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Frances+V+Fuller-Pace%5BAuthor+-+Full%5D\">Frances V Fuller-Pace</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Cem+Gabay%5BAuthor+-+Full%5D\">Cem Gabay</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Dominique+Gagliardi%5BAuthor+-+Full%5D\">Dominique Gagliardi</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Marcia++Haigis%5BAuthor+-+Full%5D\">Marcia&nbsp; Haigis</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=J+Wade+Harper%5BAuthor+-+Full%5D\">J Wade Harper</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Barry+Honig%5BAuthor+-+Full%5D\">Barry Honig</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Yoshiaki+Ito%5BAuthor+-+Full%5D\">Yoshiaki Ito</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Veerle+Janssens%5BAuthor+-+Full%5D\">, Veerle Janssens</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nathalie+Josso%5BAuthor+-+Full%5D\">Nathalie Josso</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jaewon+Ko+%5BAuthor+-+Full%5D\">Jaewon&nbsp; Ko</a>&nbsp;,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Vera+Kozjak-Pavlovic+%5BAuthor+-+Full%5D\">Vera Kozjak-Pavlovic</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Anastasia+Kralli%5BAuthor+-+Full%5D\">Anastasia Kralli</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Paul+J+Lehner%5BAuthor+-+Full%5D\">Paul J Lehner</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Dominique++Leprince%5BAuthor+-+Full%5D\">Dominique&nbsp; Leprince</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bruce+D+Levy%5BAuthor+-+Full%5D\">Bruce D Levy</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Wei+Li%5BAuthor+-+Full%5D\">Wei Li</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Francisco+Lozano%5BAuthor+-+Full%5D\">Francisco Lozano</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jian+Lu%5BAuthor+-+Full%5D\">Jian Lu</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Michael+J+Matunis%5BAuthor+-+Full%5D\">Michael J Matunis,</a>&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Birgit+Meldal%5BAuthor+-+Full%5D\">Birgit Meldal</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Violaine+Moreau%5BAuthor+-+Full%5D\">Violaine Moreau</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Kyungjae+Myung%5BAuthor+-+Full%5D\">Kyungjae Myung</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Joseph+H+Neale%5BAuthor+-+Full%5D\">Joseph H Neale</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Christian+Obinger%5BAuthor+-+Full%5D\">Christian Obinger</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=R+Jeroen+Pasterkamp%5BAuthor+-+Full%5D\">, &nbsp;R Jeroen Pasterkamp</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Richard+Phipps%5BAuthor+-+Full%5D\">Richard Phipps</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Didier+Picard%5BAuthor+-+Full%5D\">Didier Picard</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Elah+Pick%5BAuthor+-+Full%5D\">Elah Pick</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=David+A+Rhodes%5BAuthor+-+Full%5D\">David A Rhodes</a>,&nbsp;<a href=\"http://iribhm.org/index.php/our-research-group/roger\">Pier e P Roger</a>, Mark G Rush,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Joshua+R+Sanes%5BAuthor+-+Full%5D\">Joshua R Sanes</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Martin+Schr%C3%B6der%5BAuthor+-+Full%5D\">Martin Schrder</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Pierre+Thibault%5BAuthor+-+Full%5D\">Pierre Thibault</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Dick+J+H+van+den+Boomen%5BAuthor+-+Full%5D\">Dick J H van den Boomen</a>&nbsp;,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Thomas+E+Van+Dyke%5BAuthor+-+Full%5D\">Thomas E Van Dyke</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Roberto+M+Vanacore%5BAuthor+-+Full%5D\">Roberto M Vanacore</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nobutaka+Wakamiya%5BAuthor+-+Full%5D\">Nobutaka Wakamiya</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Qinglu+Wang%5BAuthor+-+Full%5D\">Qinglu Wang</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bart+Westendorp%5BAuthor+-+Full%5D\">, Bart Westendorp</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Sandra+E+Wiley%5BAuthor+-+Full%5D\">, Sandra E Wiley</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Miriam+Wittmann%5BAuthor+-+Full%5D\">Miriam Wittmann</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Guilherme+Xavier%5BAuthor+-+Full%5D\">Guilherme Xavier</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Wei-Chih+Yang%5BAuthor+-+Full%5D\">Wei-Chih Yang</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ali+A+Zarrin%5BAuthor+-+Full%5D\">Ali A Zarrin</a>, and&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bing+Zhu+%5BAuthor+-+Full%5D\">Bing Zhu</a>&nbsp;are our external reviewers.</p>\r\n<p><strong>Annotation Statistics.</strong>&nbsp;Reactome comprises 10,754 human reactions organized into 2,132 pathways involving 10,907 proteins encoded by 10,658 different human&nbsp;genes,&nbsp;1,763 small molecules, and 10,748 complexes. These annotations are supported by 26,384 literature references. We have projected these reactions onto 115,881 orthologous proteins, creating 20,701 orthologous pathways in 18 non-human species. Version 60 has annotations for 1,503 protein variants (mutated proteins) and their post-translationally modified forms, derived from 285 proteins. These have been used to annotate 506 disease-specific complexes and 906 disease-specific reactions organized into 453 pathways and&nbsp;subpathways,&nbsp;and tagged with 294 Disease Ontology terms.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;<a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a></strong>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:18:03','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,35,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(78,692,'Version 61 Released','version-61-released','<p><strong><a href=\"/PathwayBrowser/#R-HSA-913531\"><img class=\"aligncenter wp-image-4020\" style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-913531.svg\" alt=\"R-HSA-913531\" width=\"650\" height=\"400\" /></a></strong></p>\r\n<p><strong>New and Updated Pathways.</strong><strong>&nbsp;</strong>In version&nbsp;V61, topics with new or revised pathways include: Gene expression (<span style=\"text-decoration: underline;\"><a href=\"//reactome.org/PathwayBrowser/#/R-HSA-8878171\">Transcriptional regulation by RUNX1</a></span>),&nbsp;Immune System (<a href=\"/PathwayBrowser/#/R-HSA-447115\">Interleukin-12 family signaling</a>), Signal Transduction (<span style=\"text-decoration: underline;\"><a href=\"/PathwayBrowser/#/R-HSA-6807070\">PTEN Regulation</a></span>), and Transport of small molecules (<span style=\"text-decoration: underline;\"><a href=\"/PathwayBrowser/#/R-HSA-8981607\">Intracellular oxygen transport</a></span>).</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Arkaitz%20%20Carracedo%5BAuthor%20-%20Full%5D\">Arkaitz Carracedo</a>&nbsp;and&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Leonardo%20Salmena%5BAuthor%20-%20Full%5D\">Leonardo Salmena</a>&nbsp;were our external authors.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bailly%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26176610\">Sabine Bailly</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=thorsten+burmester%5BAuthor+-+Full%5D\">Thorsten Burmester</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Linda+Shyue+Huey++Chuang%5BAuthor+-+Full%5D\">Linda Shyue Huey Chuang</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ito%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28299675\">Yoshiaki Ito</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nisha+Kriplani%5BAuthor+-+Full%5D\">Nisha Kriplani</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nick+Leslie%5BAuthor+-+Full%5D\">Nick Leslie</a>, and&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Esther+van+de+Vosse%5BAuthor+-+Full%5D\">Esther van de Vosse</a>&nbsp;were our external reviewers.</p>\r\n<p><b>Annotation Statistics.</b>&nbsp;Reactome comprises 11,042 human reactions organized into 2,148 pathways involving 10,940 proteins encoded by 10,691 different human genes, 1,763 small molecules, and 11,041 complexes. These annotations are supported by 26,859 literature references. We have projected these reactions onto 120,804 orthologous proteins, creating 20,780 orthologous pathways in 18 non-human species. Version 61 has annotations for 1,334 protein variants (mutated proteins) and their post-translationally modified forms, derived from 285 proteins. These have been used to annotate 506 disease-specific complexes and 906 disease-specific reactions organized into 453 pathways and subpathways, and tagged with 294 Disease Ontology terms.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:20:12','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,34,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(79,693,'New Protein-protein Interaction files','new-protein-protein-interaction-files','<p><img class=\"aligncenter wp-image-4029\" style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Interaction-nologo.png\" alt=\"Interaction\" width=\"681\" height=\"400\" /></p>\r\n<p>The Reactome team has released new versions of our protein-protein interaction files derived from reactions and complexes. These files were updated following user feedback, with the goal of providing extra annotation features through support from the&nbsp;<a href=\"http://github.com/HUPO-PSI/miTab/blob/master/PSI-MITAB27Format.md\">PSI-MITAB 2.7</a>&nbsp;data format. Interactions are computationally generated based on the data stored on complexes and reactions. The interactions provided by Reactome are not curated and are not experimental data. In addition, the complexes and reactions in species other than human are derived by orthology inference from the corresponding human complexes and reactions. Tab-delimited formatted files are also provided for human and all species.</p>\r\n<p>The four new files are:</p>\r\n<ul>\r\n<li><a href=\"/download/current/interactors/reactome.all_species.interactions.psi-mitab.txt\">Full list of protein-protein interactions, incl. non-human species [PSI-MITAB]</a></li>\r\n<li><a href=\"/download/current/interactors/reactome.all_species.interactions.tab-delimited.txt\">Full list of protein-protein interactions, incl.&nbsp;non-human species [tab-delimited]</a></li>\r\n<li><a href=\"/download/current/interactors/reactome.homo_sapiens.interactions.psi-mitab.txt\">Human protein-protein interactions [PSI-MITAB]</a></li>\r\n<li><a href=\"/download/current/interactors/reactome.homo_sapiens.interactions.tab-delimited.txt\">Human protein-protein interactions [tab-delimited]</a>.</li>\r\n</ul>\r\n<p>Documentation about these files is available from the&nbsp;<a href=\"/download-data/\">Data Download</a>&nbsp;page.</p>\r\n<p>The former versions of the protein-protein interaction files are still available but will be removed from service as of Version 62 in September 2017. We encourage users that programmatically access these files to update the scripts.</p>\r\n<p><strong>About the Reactome&nbsp;Project.</strong>&nbsp;Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Cold Spring Harbor Laboratory, New York University Langone Medical Center, Oregon Health and Science University, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the Reactome website.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"https://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:24:21','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,33,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(80,694,'New SBML Level 3 Version 1 export is now available','new-sbml-level-3-version-1-export-is-now-available','<p><a href=\"/download-data/\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/SBML.png\" alt=\"SBML\" width=\"755\" height=\"400\" /></a></p>\r\n<p>As part of our efforts to give our modeling user community better experience, we have updated the SBML export to Level 3 Version 1, which is organized in a modular manner. &nbsp;Our initial export provides a richer annotation &nbsp;syntax and we will explore supporting other SBML Level 3 Packages in the future. The SBML data export can be used by any tool that supports SBML L3V1.</p>\r\n<p>The SBML Level 3 Version 1 export is available for download here:&nbsp;<a href=\"/download/current/homo_sapiens.3.1.sbml.tgz\">https://reactome.org/download/current/homo_sapiens.3.1.sbml.tgz</a>.</p>\r\n<p>We are also providing a programmatic interface to access these updated SBML files through our&nbsp;<a href=\"dev/content-service/\">ContentService</a>&nbsp;for all species at&nbsp;<a href=\"/ContentService/#!/exporter/toSBMLUsingGET\">https://reactome.org/ContentService/#!/exporter/toSBMLUsingGET</a></p>\r\n<p>Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Cold Spring Harbor Laboratory, New York University Langone Medical Center, Oregon Health and Science University, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the Reactome website.</p>\r\n<p>Follow us on Twitter:&nbsp;<a href=\"https://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:25:06','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,32,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(81,695,'The Pathway Browser','the-pathway-browser','<p><span>The Pathway Browser is the primary means of viewing and interacting with pathways in Reactome. It includes tools for analysing datasets and exploring pathways. These tools allow several types of analysis:</span></p>\r\n<ul>\r\n<li><span>Pathway over-representation analysis and pathway topology-based analysis</span></li>\r\n<li><span>Comparison of a pathway with its equivalent in another species</span></li>\r\n<li><span>The overlay of user-supplied expression data onto a pathway</span></li>\r\n<li><span>The overlay of protein-protein or protein-compound interaction data from external databases or user-supplied data onto a pathway &nbsp;</span></li>\r\n</ul>\r\n<p></p>\r\n<p><span>See the section Reactome Tools for details of the analysis tools.</span></p>\r\n<p><span>The Pathway Browser is launched by clicking the Browse Pathways button on the Homepage:</span></p>\r\n<p><img src=\"images/userguide/pathway_browser/pathway_browser_button.gif\" alt=\"pathway browser button\" /></p>\r\n<p><span>When the Pathway Browser opens, it displays an overview of all Reactome pathways. Pathways are organized hierarchically and often have sub-pathways. At the highest hierarchical level, all pathways are represented in an Overview. At intermediate levels, pathways are often represented as interactive illustrations, with selectable regions that act as links to the lower levels, which are represented as detailed Pathway Diagrams. Reactomes Overview uses a unique graphical visualization of the hierarchy, representing pathways at the uppermost hierarchical level as central nodes (circles) with subpathways arranged concentrically (in rings) around them. Many have further concentric rings of child subpathways. Pathway nodes are connected to their subpathways by edges (lines). This overview displays all pathways and allows rapid zooming to details. As you zoom in, labels appear for the subpathway nodes. This view is also used to display data analysis results. </span></p>\r\n<p><span>Nodes in the overview link to detailed pathway diagrams that use a style based on Systems Biology Graphical Notation. They have several features intended to ensure an appropriate level of information detail, including subpathway shading and fade-in of labels, commonly occurring small molecules and structures.</span></p>\r\n<p><span>To see the details of a specific pathway, select and click (or double-click) the node representing the pathway. Alternatively select and click on the name in the Pathway Hierarchy panel on the left.</span></p>\r\n<p></p>\r\n<p><span>Features of the Pathway Browser are labelled in the diagram below. &nbsp;</span></p>\r\n<p></p>\r\n<p><img src=\"images/userguide/pathway_browser/pathway_browser_labelled.png\" alt=\"pathway browser labelled\" /></p>\r\n<p></p>\r\n<p><span>Home  the Reactome logo is a button linked to the homepage.</span></p>\r\n<p><span>Species  Reactome is a database of curated human biological pathways. These human pathways are used to computationally infer equivalent pathways in model organisms (described in detail </span><a href=\"http://reactomedev.oicr.on.ca/electronic_inference.html\"><span>here</span></a><span>). Use the Species drop-down to select a species and view the predicted pathway. Note that infectious disease pathways that show the interaction of pathogens with human proteins are included in Homo sapiens.</span></p>\r\n<p><span>Layout  The pathway browser is divided into 3 main sections. These are: the Hierarchy Panel, on the left, the Details Panel, bottom right, the Pathway Panel, top right. The layout buttons allow you to show or hide these panels.</span></p>\r\n<p><span>Analyse Data  Opens a panel where analysis can be performed. </span></p>\r\n<p><span>Key  this opens a key explaining the objects present in the pathway panel. </span></p>\r\n<p><span>Zoom/Move Toolbar - Click on the arrows to move the diagram. Click on the plus and minus buttons to zoom in and out. Alternatively, click and drag the diagram, and use your mouse scroll wheel to zoom. </span></p>\r\n<p><span>In-Diagram Search  click the button to open the search panel. When the Overview or a Pathway Diagram is displayed, objects represented in the displayed diagram or all diagrams are searchable.</span></p>\r\n<p><span>Fit to Page - resizes the Pathway Overview or Diagram to fit the available space.</span></p>\r\n<p><span>Open Diagram  when a pathway node is selected in the Pathway Overview, this button opens the corresponding Pathway Diagram</span></p>\r\n<p><span>Illustrations  some Reactome pathways have a corresponding high-quality graphic. Select this button to view.</span></p>\r\n<p><span>Export  use this button to download the visible region of the Pathway Overview or Diagram, including any analysis overlay, as a PNG file or upload it to GenomeSpace. </span></p>\r\n<p><span>Thumbnail  Provides an overview of the entire pathway when zoomed in so that only a region of the pathway is displayed. </span></p>\r\n<p><span>Pathway Panel - This is where Pathway Diagrams are displayed when they are selected in the Pathway Hierarchy or in search results. If no pathway is selected, this panel displays a brief Reactome help guide.</span></p>\r\n<p><span>Details Panel - Contains details of objects when they are selected in the Pathway Diagram. The content depends on the type of object, i.e. pathway, reaction, complex, set, protein or small molecule (see the Details section). </span></p>\r\n<p><span>Hierarchy Panel  Includes the hierarchical organisation of Reactome pathways and pathway events or steps, known as reactions. When the Pathway Browser is opened, only the top level of the hierarchy is shown, as an alphabetical list of Reactomes main topics. Most topics are too large to be represented as a single diagram. Instead they are split into sub-pathways, which may be further divided into sub-sub-pathways. Most topics in Reactome are organised in this hierarchical manner. Sub-pathways can be revealed by clicking on the + icon to the left of the pathway name, and hidden by clicking on the - icon.</span></p>\r\n<p>Settings sidebar  Used for context-sensitive help, to configure colour schemes and the default source of interactors.</p>\r\n<p></p>\r\n<p><span>Pathway names are preceded by the icon &nbsp;</span><span><img src=\"https://lh3.googleusercontent.com/4PsJlyvYJIRSQHfJfY6dc9LrmluDI_06BR_8NjwhdS_GUqhlEJt3IdzAA2eb9n2OGhE95renl08AMs8En3ls4IvZxuLGsk-sFShv_012DZlf8VmJK2Gs7DyZiP-Cg4OP9ONOKzlvy5y8P-4NqA\" alt=\"\" width=\"17\" height=\"19\" /></span><span>. As sub-pathways are revealed, at some point a level is reached where the reactions will become visible. Reactions names are preceded by the icon &nbsp;</span><span><img src=\"https://lh3.googleusercontent.com/Jr4STDpyuWZtkmAqM4mcVE-oaarFOJ7CH_SaZ-y3Mvwiq4ncMIJ1w_v6lde3bSOG9MpKTU2A-ULPhFckGltofmNQQzH-aeXk2b22fkCtasX-1q1b7tCJa5WcZFi5e60jYbG4joTYXnoHkEUCzQ\" alt=\"\" width=\"18\" height=\"17\" /></span><span>. You may also see the icon &nbsp;</span><span><img src=\"https://lh3.googleusercontent.com/-27vLbYfjam8e5NLhevGlr5b1JZE2WJ2I9guUZdRpcZsmGfSduZ59dIKoA81cJDaZfAEadCPzvsYdbgLGr1ava5vK9GhgYXSRbNSJW1cZaDSTzKsUGqDM0d7JoJEwR7N3Tt3k0sgXcHbAWRfqQ\" alt=\"\" width=\"17\" height=\"11\" /></span><span> representing black-box reactions, where the full molecular details have been omitted for clarity, or the details are incompletely determined.</span></p>\r\n<p><span>The order of reactions from top to bottom in the Hierarchy Panel usually follows their order as steps in the pathway so that the preceding reaction is above and the subsequent reaction below, but this is not always the case. Pathways may not be linear; they can be circular or branched. Consequently, reactions often have multiple preceding or subsequent reactions. These connected events are often visible within the pathway diagram but can be part of a different pathway diagram. To view a complete list of the events that precede or follow a reaction, refer to the Preceding and Following Events section in the Details Panel, as described in the Details Panel chapter below.</span></p>\r\n<p></p>\r\n<h2><a id=\"pathway-diagrams\"><b>Pathway Diagrams</b></a></h2>\r\n<p></p>\r\n<p><span>Pathway Diagrams represent pathways as a series of connected molecular events, known in Reactome as reactions, which can be considered as steps in the pathway.</span></p>\r\n<p><span>Cellular compartments are represented as pink/orange boxes with a double boundary. A typical diagram has a box to represent the cytosol, bounded by a double-line that represents the plasma membrane. The white area outside this box represents the extracellular space. Other organelles are represented as additional labelled boxes within the cytosol. </span></p>\r\n<p><span>Molecules, represented in diagrams as blue/green boxes, are placed in the physiologically-correct cellular compartment, or lie on the boundary of a compartment to indicate that they are in the corresponding membrane, e.g. a plasma membrane protein will be placed on the boundary of the cytosol.</span></p>\r\n<p></p>\r\n<p><span>Reactions typically include:</span></p>\r\n<ul>\r\n<li><span>Input and output molecules, and when relevant a catalyst (see diagram A below)</span></li>\r\n<li><span>Inputs, outputs and catalyst are represented as boxes or ovals</span></li>\r\n<li><span>Green boxes with rounded corners are proteins</span></li>\r\n<li><span>Green boxes with square corners are proteins that have no UniProt accession (or did not at the time the reaction was created).</span></li>\r\n<li><span>Green ovals are small molecules or sets of small molecules</span></li>\r\n<li><span>Blue boxes with a double boundary are sets, i.e. proteins or small molecules that are functionally equivalent. </span></li>\r\n<li><span>Blue boxes with cut corners are complexes, i.e. proteins and/or other molecules that are bound in a multimolecular entity.</span></li>\r\n<li><span>Green boxes with a white inner box are sub-pathways.</span></li>\r\n<li><span>Reaction input and output molecules are joined by lines to a central reaction node (surrounded by a green box in Figure A below). Clicking this node selects the reaction. </span></li>\r\n<li><span>The outputs of a reaction have an arrowhead on the line connecting them to the reaction node.</span></li>\r\n<li><span>Reaction inputs/output molecules are often connected by arrows to preceding or subsequent reactions (i.e. the preceding/subsequent steps in the pathway). </span></li>\r\n<li><span>Catalysts are connected to the reaction node by a line ending in a circle. </span></li>\r\n<li><span>Numbered boxes on the line between an input/output and the reaction node indicates the number of molecules of this type in the reaction (when n &gt;1). </span></li>\r\n<li><span>Molecules that regulate a reaction are connected to the reaction node by a line ending in an open triangle for positive regulation or a T-shaped head for negative regulation (see B below).</span></li>\r\n<li><span>A white box labelled P on the boundary of an object indicates a phosphorylation event.</span></li>\r\n<li><span>Proteins or small molecules that are also part of a displayed set may be connected to the set object by a line of short dashes.</span></li>\r\n<li><span>Sets with overlapping content may be connected by a line of long dashes. </span></li>\r\n<li><span>Reactions that represent a disease process use red connecting lines. Objects associated with a disease are bordered in red.</span></li>\r\n</ul>\r\n<p><span><img src=\"https://lh4.googleusercontent.com/s5IzQIdqrdfXxHQ5xkTjeqgDoWb8OwabONfpAU-nHiJZNhopKKj8n9tgXBb1o_uR-GWDJcFI6I_EwneGz7JnHVudRgXT3f-PjPl1dUf5Tnzw-NQ5k_MNpv_rgX8fGjnoLDZyDunp6-BHjcRBTg\" alt=\"\" width=\"487\" height=\"214\" /></span></p>\r\n<h2><a id=\"ehld\"><b>Enhanced high-level diagrams and Subpathway icons</b></a></h2>\r\n<p></p>\r\n<p><span>Many pathways at the higher levels of the hierarchy are too large to represent as a single, detailed pathway diagram. Instead they are often represented as an illustration, known as an enhanced high-level diagram, which graphically represents the subpathways as selectable regions of an illustration. When the mouse pointer is moved over a region, it becomes surrounded by a blue highlighting \'halo\'. When you click on a region it becomes selected and outlined in dark blue, the corresponding subpathway is selected in the Hierarchy panel and the Details panel updates to show details of the subpathway. Double-clicking on a region opens the corresponding detailed Pathway Diagram.</span></p>\r\n<p><span>In the Hierarchy panel, pathways that have an enhanced high-level diagram have a blue icon to the left of their name.</span></p>\r\n<p><img src=\"images/userguide/pathway_browser/EHLD.png\" alt=\"EHLD\" /></p>\r\n<p></p>\r\n<p><span>A few higher level pathways use an older visualization of large topics, where subpathways are represented by a box with a green boundary, the subpathway icon (see example below). Selecting a subpathway icon has the same result as selecting a region of an enhanced-level pathway diagram as described above. </span></p>\r\n<p><img src=\"images/userguide/pathway_browser/green_boxes.png\" alt=\"green boxes\" /></p>\r\n<p><span><span style=\"color: #333333; font-family: \'Open Sans\', Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: #ffffff; float: none;\">In some pathways, Reactome links to other related pathways through green boxes. In order to differentiate between related subpathways that are children&nbsp;</span><span style=\"color: #333333; font-family: \'Open Sans\', Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: #ffffff; float: none; display: inline;\">of the same parent pathway (subpathways) and related subpathways that are children of a different parent pathway</span><span style=\"color: #333333; font-family: \'Open Sans\', Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: #ffffff; float: none; display: inline;\">&nbsp;(interacting pathways), the Pathway Browser uses slightly different glyphs as illustrated in the figure below.</span></span></p>\r\n<p><span><img src=\"images/userguide/pathway_browser/subpathways_interacting.png\" alt=\"subpathways interacting\" width=\"250\" height=\"205\" /></span></p>\r\n<h3><b>Subpathway Shading</b></h3>\r\n<p></p>\r\n<p><span>When a Pathway Diagram contains subpathways, the area containing each subpathway is overlaid with a coloured box, labelled with the subpathway name, to help locate it in the larger diagram. Below is the diagram for ERBB4 signaling. The four subpathways are represented as 4 boxed and shaded areas.</span></p>\r\n<p><span><img src=\"https://lh3.googleusercontent.com/X6noUX07flcHHx8XVIW07fdI09Cp6d8Hn2YNy2q6dyLsCFuBmuAYnWDYbRaXRHcm6TT4AyTvQIxBhHLMxGMBcaTy96Domy_GTaLtSOJRIpqyj6So4W4WJV3DwXd3VAI9wvcM-xfxnK-9DDv4FA\" alt=\"\" width=\"377\" height=\"583\" /></span></p>\r\n<p></p>\r\n<h2><a id=\"pathway-diagram-zoom\"><b>Pathway Diagram zoom detail level</b></a></h2>\r\n<p></p>\r\n<p><span>The pathway diagram represents different levels of detail, depending on the zoom level. When fully zoomed-out, regions corresponding to subpathways are shaded. The boxes representing molecules or groups of molecules have no text labels and trivial molecules, such as water and ATP, are not represented. At the next level of zoom, subpathway shading disappears, while trivial molecules and the circular red icon indicating that a protein has known interactors fade into view. At closer levels of zoom the reaction nodes appear, as do boxes containing a number on the lines connecting molecules to the reaction node if stoichiometry is greater than one. At this level of zoom, if interactors are displayed their names appear. At the highest level of zoom, if available, proteins, small molecules and interactors display structural diagrams from PDBe or ChEBI with associated details.</span></p>\r\n<p><span><img src=\"https://lh3.googleusercontent.com/jMUkTx4gjecv-HECOZZsVxesOeR2y_8KKHbDF7DyfffViCkC4q_OmoiInPSqDqAPFviHKa88A9BPLwvkgTf9wLOjProrTNS6jnZ0EeO0cAF368yExxK6UCedHbygXeUcIHmPrRJimB64qt87bg\" alt=\"\" width=\"358\" height=\"165\" /></span></p>\r\n<p><span><img src=\"https://lh5.googleusercontent.com/ZWlECCkQEM0vBBPWy30jV9A0ahI3o8lYxD5KcIQk4wf0aAqFz190N13eAsbwkPD4dkX0K_av3PJqpq4zn_fPzKOh3DlSXXVutk4rI9O2hVl_IIDqN_Bfz9fw8333hobIzRcTfozZzYEl9e0LwA\" alt=\"\" width=\"238\" height=\"151\" /></span></p>\r\n<p><span><img src=\"https://lh5.googleusercontent.com/zHAs4P2NQVkawfp1UtbqQhKjiXboS2YMLBzzFzgSUystPIsvMuMWrOLr8WkDVZf52FWxIIIAw8AeqUgjWfGzi1OvmiGKmyCqQ-F4tT2nLep5Fx9wDJobdqXox56xOPaHSHwNJuki0RQUr2UPmg\" alt=\"\" width=\"186\" height=\"143\" /></span></p>\r\n<p><span><img src=\"https://lh6.googleusercontent.com/IQorGltHBd2KHFSY5D-I-qm2qvrK4vC0hSsXnrP_hHYm0qppmmUWKtPkpEEzaWDZOaICEnNCfRMZ95-P_vX3KE7Ady3n6DrxzpgwoVIRHigkVEnKXNzUFJRKjnSItZ5xKh_kt2I0Ly1qvH5SBg\" alt=\"\" width=\"203\" height=\"183\" /></span></p>\r\n<p></p>\r\n<h2><a id=\"navigating-pathway-diagrams\"><b>Navigating Pathway Diagrams</b></a></h2>\r\n<p></p>\r\n<p><span>The Pathway Diagram, Hierarchy and Details Panels are interactively connected. Selecting an entity (molecular object) or event in the diagram will reveal relevant information in the Details Panel. Clicking a pathway name in the Hierarchy will open the corresponding Pathway Diagram in the Pathway Diagram Panel, and reveal details of the pathway in the Details Panel. If a subpathway is contained within the diagram that is displayed, clicking its name in the hierarchy will cause all the reactions in the pathway to be highlighted in blue. In the figure below, the diagram for Platelet homeostasis is displayed in the Pathway Panel, and the sub-pathway Prostacyclin signalling through prostacyclin receptor has been selected in the Hierarchy, causing all the reactions in this sub-pathway to be highlighted (blue) on the Pathway Diagram.</span></p>\r\n<p><img src=\"images/userguide/pathway_browser/subpathway_selection.png\" alt=\"subpathway selection\" /></p>\r\n<p></p>\r\n<p><span>Similarly, selecting a reaction in the Hierarchy causes that reaction to be highlighted in the pathway diagram and updates the Details Panel to show details of the reaction. If the reaction is not in the visible region of the diagram, the view will re-centre and zoom to show it. If instead of clicking an event, you hover the mouse pointer over its name, it will be highlighted in yellow. In the figure above, the sub-pathway Platelet calcium homeostasis is highlighted in yellow.</span></p>\r\n<p></p>\r\n<h3><b>Getting Started</b></h3>\r\n<p></p>\r\n<h3><span>Hierarchy Panel </span><b>Exercises</b></h3>\r\n<p></p>\r\n<p><span>This exercise is to check that you understand the organisation of the Hierarchy Panel. You dont need to look at the Diagram Panel. </span></p>\r\n<p><span>From the Home page, search for PDGF signaling.</span></p>\r\n<p><span>In the results page, open the expandable hierarchy in the section Locations in the Pathway Browser by clicking on the + button. </span><b>DONT</b><span> click on the location before expanding the hierarchy! This will open the pathway diagram in the Pathway Browser.</span></p>\r\n<p><span>Look at the Hierarchy Panel on the left. </span></p>\r\n<ol>\r\n<li><span>How many sub-pathways does this pathway have?</span></li>\r\n<li><span>How many reactions are in the first sub-pathway?</span></li>\r\n<li><span>What reaction(s) follow Translocation of PDGF from ER to Golgi? </span><i><span>Hint:Look in the Details Panel, bottom-right. &nbsp;If it is hidden, use the layout buttons in the top right corner, above the diagram, to reveal it.</span></i></li>\r\n</ol>\r\n<h3><span>Subpathway </span><b>Exercises</b></h3>\r\n<p></p>\r\n<p><span>This exercise is to check that you understand Subpathway icons and the relationship between the Pathway Hierarchy and Pathway Diagram.</span></p>\r\n<p></p>\r\n<p><span>Open the pathway Hemostasis in the Pathway Browser. </span></p>\r\n<p><span>Select the subpathway Platelet Hemostasis. </span></p>\r\n<ol>\r\n<li><span>What happens if you hover your mouse pointer in the Hierarchy on the sub-pathway Platelet calcium hemostasis?</span></li>\r\n<li><span>What happens if you click in the Hierarchy on the sub-pathway Platelet calcium hemostasis?</span></li>\r\n<li><span>What happens if you click in the hierarchy on the reaction Binding of ATP to P2X receptors?</span></li>\r\n</ol>\r\n<p></p>\r\n<p><b>More Information</b></p>\r\n<p><span>There are 5 subtypes of reaction node, indicating reaction subclasses.</span></p>\r\n<p></p>\r\n<p><span><img src=\"https://lh6.googleusercontent.com/Uw5AIAabFQ5JikVervywJ3gpHlqkWfMIn73gE4lmzgtJJjD9h74HM3Ws6jlYkqo_E2SpLetnhXjs0DQL_14WtoW3VOgunjS51LO-HohG3NR_1UBwwPbp2VbPTTyj5o3RCtXMJDJh4jDKRLZNhA\" alt=\"\" width=\"602\" height=\"97\" /></span></p>\r\n<p></p>\r\n<ul>\r\n<li><span>Open squares represent transition, i.e. a change of state that is not one of the defined subclasses.</span></li>\r\n<li><span>Solid circles represent association, i.e. binding</span></li>\r\n<li><span>Double-bordered circles represent dissociation </span></li>\r\n<li><span>A square with two slashes represents an omitted process. This is used when the full details of a reaction have been deliberately omitted. This is most commonly used for events that include representative members of a large family to illustrate the general behaviour of the group. It can be used for reactions that occur with no fixed order or stoichiometry, or for degradation events where the output is a random set of fragments.</span></li>\r\n<li><span>Squares containing a question mark represent an uncertain process, where some details of the reaction are known, but the process is thought to be more complex than represented. Explanatory details are typically included in the Description.</span></li>\r\n</ul>\r\n<p></p>\r\n<h3><span>Reaction Node </span><b>Exercises</b></h3>\r\n<p></p>\r\n<p><span>This exercise is to check that you understand reaction node subtypes.</span></p>\r\n<p></p>\r\n<p><span>From the Home page, search for the pathway Effects of PIP2 hydrolysis and open it in the Pathway Browser (in any of the several locations)</span></p>\r\n<ol>\r\n<li><span>What symbol represents the reaction for Binding of IP3 to the IP3 receptor?</span></li>\r\n<li><span>What symbol represents the reaction IP3R tetramer:I(1,4,5)P3:4xCa2+ transports Ca2+ from platelet dense tubular system to cytosol? What subtype of reaction is this?</span></li>\r\n<li><span>Open the subpathway Arachidonate production from DAG. What is the name of the catalyst for 2-AG hydrolysis to arachidonate by MAGL? Can you name the three outputs of this reaction? </span></li>\r\n<li><span>Can you find the UniProt ID for the catalyst? Hint: There are several ways to find this, two require you to select something!</span></li>\r\n</ol>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:48:22','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(82,696,'Details Panel','details-panel','<p><span>The Details Panel is the bottom right panel of the Pathway Browser. It displays the details of molecular objects or events when selected in the Pathway Diagram or Hierarchy. The details shown depend on the type of molecule or event selected. The Details Panel can be revealed/hidden using the middle Layout button. &nbsp;</span></p>\r\n<p></p>\r\n<p><span>The Details Panel has several tabs:</span></p>\r\n<p></p>\r\n<h3><b>The Description tab </b></h3>\r\n<p></p>\r\n<p><span>This tab contains the defining details of the selected diagram object. </span><span>These details vary, depending on what is selected in the diagram. </span><span>For example, reaction details always describe the input and output molecules, and where relevant, the enzyme catalyst. Other items in the Description tab </span><span>may</span><span> include:</span></p>\r\n<p><span>Summation  a summary of the molecular event, often with background information and additional literature citations</span></p>\r\n<p><span>Preceding/Following Events - Links to reactions that precede/follow this reaction.</span></p>\r\n<p><span>Cellular Compartment - Identifies the cellular compartment for the reaction, with the associated Gene Ontology (GO) term.</span></p>\r\n<p><span>Inferred from another species - Reactome represents human biology. Wherever possible key references contain experimental data obtained using human reagents. If the only data available is from model organisms, a human event may be inferred from this data, if the Author, Curator and Reviewer agree that this inference is valid. The inferred event is labelled with an icon Inferred from another species in the Hierarchy and in the Details panel, where there is a link to the molecular details of the event as it occurs in the model organism and the supporting literature references. </span><span>A description of the inference process can be found here.</span></p>\r\n<p></p>\r\n<p><span>Computationally Inferred To  This drop-down list represents the species available for computationally predicted pathways, inferred from Reactomes human pathway. References  all events in Reactome include links to original research publications containing experimental data, the evidence that the event has been shown to exist.</span></p>\r\n<p><span>Authored - an expert biologist who contributed materials allowing construction of the reaction/pathway. All Reactome pathways are authored by an expert biologist.</span></p>\r\n<p><span>Reviewed  all Reactome pathways are peer reviewed by an expert biologist who verifies the content provided by the Author.</span></p>\r\n<p><span>Details are organised into panels. Panels that have a plus icon on the right hand side can be expanded to reveal further levels of detail by clicking on the icon or the detail name.</span></p>\r\n<p><img src=\"images/userguide/details/details_output.gif\" alt=\"details output\" /></p>\r\n<p><span>Clicking the plus icon reveals further details:</span></p>\r\n<p><img src=\"images/userguide/details/further_details.gif\" alt=\"further details\" /></p>\r\n<p><span>Panels that represent molecules have an icon on the left hand side that represents the molecule subtype, e.g. the protein icon is a green circle. Mouse-over the icon to see an explanation of the subtype. The icon is a selection button; clicking the icon replaces the current details with a new panel of details specific to the selected molecule, which is highlighted on the pathway diagram. &nbsp;</span></p>\r\n<p></p>\r\n<h3><b>The Molecules Tab </b></h3>\r\n<p></p>\r\n<p><span>This tab details all the molecules involved in the pathway displayed in the panel above. </span><span>When a pathway name is highlighted in the hierarchy, the total number of molecules in that pathway is shown in a bubble on the molecules tab.</span></p>\r\n<p>Molecules are grouped into the subtypes Protein, Chemical Compound, Sequences (for DNA/RNA) and Other.</p>\r\n<p><span>If an item is selected in the diagram, the corresponding molecules are highlighted in the details panel, and the bubble next to the Molecules tab enumerates the fraction of the total pathway molecules that are represented in that item.</span></p>\r\n<p><img src=\"images/userguide/details/molecules.png\" alt=\"molecules\" /></p>\r\n<p><span>The Download button to the right of the pathway name and species buttons allows details of the molecules to be downloaded in several formats. Click on the buttons to select the fields you want included in the output file, and the format of the file. Click Download to save the file.</span></p>\r\n<p></p>\r\n<h3><b>The Structures Tab</b></h3>\r\n<p></p>\r\n<p><span>The content displayed in this tab depends on the type of object or event selected in the pathway diagram. For Reactions it will display equivalent reaction diagrams from the </span><a href=\"http://www.ebi.ac.uk/rhea/home.xhtml\"><span>Rhea</span></a><span> database if available:</span></p>\r\n<p></p>\r\n<p><span><img src=\"https://lh5.googleusercontent.com/UIB00AQgklytLvqFpkOHDdtJUpLw0unSwy8d8eo760LO6EYjEn9EUR49-YqZxE9Gpw_DBTdG6G-CzjjtFv97yUyb4akZCFbAft4Mfnuw7tOLkT1iIG3Wj0i29l22TScaC64UkAqb\" alt=\"Screen Shot 2017-07-24 at 3.40.03 PM.png\" width=\"624\" height=\"416\" /></span></p>\r\n<p><span>The panel has a link top-left to details in the Rhea database. </span></p>\r\n<p><span>The molecular structures are linked to the </span><a href=\"http://www.ebi.ac.uk/chebi/\"><span>ChEBI</span></a><span> database:</span></p>\r\n<p><span><img src=\"https://lh4.googleusercontent.com/0S8mi7PXc4tBonxbyU5QgoQ6G1gM-kq79NXXQlCwaWFhi60OLNMLrMihfMwxzLUWZrFy0PmPos7fEL2fpaaK89BXhkvClTVLk01FsIaLARlllUJabtSYy0KlE7MzY4j4vPjiN8MT\" alt=\"Screen Shot 2017-07-24 at 3.41.20 PM.png\" width=\"550\" height=\"318\" /></span></p>\r\n<p><span>For Proteins or pathway objects that represent sets or complexes, corresponding structure information from </span><a href=\"http://www.ebi.ac.uk/pdbe/\"><span>PDBe</span></a><span> is represented if available:</span></p>\r\n<p><span><img src=\"https://lh5.googleusercontent.com/ICvr848NcHpWPTLgn-6swJRNi9q5HYZrNp27y8k-zI_yrv0mmY7HbPhaTXQ8bqqKixxKa42b15ogKH66Hnd9wXy6F0FRFwo1FmXjdSl6oHe-7cfHmAuSfpBj7j6DHNQPu9Fbv2oGtTo33RF6Kg\" alt=\"\" width=\"528\" height=\"118\" /></span></p>\r\n<p></p>\r\n<p><span>For simple molecules, the structures tab represents information from the ChEBI &nbsp;database:</span></p>\r\n<p><span><img src=\"https://lh4.googleusercontent.com/6tUwdASMecfHCHbBTinvuHn2vzVJLVRuHObBJnZgGNjmAhBt7z8FmpqZpBfqVmd7PaDmREG4MUPVf4Ap9FX6mciYjFxfb6_TMIBRPc4DJjxwZBx3aXEmpv1qUGFOoJXlj4kfeoXE-P4H8458vA\" alt=\"\" width=\"529\" height=\"140\" /></span></p>\r\n<p></p>\r\n<h3><b>The Expression Tab </b></h3>\r\n<p></p>\r\n<p><span>This tab represents expression information obtained from the </span><a href=\"http://www.ebi.ac.uk/gxa/\"><span>Expression Atlas</span></a><span>. </span><span>Note that at the moment, only information from the first 50 genes in the pathway is displayed or available for download from the Expression Atlas widget. </span></p>\r\n<p><img src=\"images/userguide/details/expression_atlas.gif\" alt=\"expression atlas\" /></p>\r\n<p></p>\r\n<h3><b>The Analysis Tab</b></h3>\r\n<p></p>\r\n<p><span>This tab displays the results of analyses  see the section Reactome Tools for further details</span></p>\r\n<p></p>\r\n<h3><b>The Downloads Tab</b></h3>\r\n<p></p>\r\n<p><span>The Downloads Tab contains buttons to start a download of the currently displayed pathway in several human-readable or computationally reusable formats. &nbsp;</span></p>\r\n<p><img src=\"images/userguide/details/downloads.gif\" alt=\"downloads\" /></p>\r\n<p></p>\r\n<h3><span>Get Started:</span></h3>\r\n<p></p>\r\n<h3><b>Exercises:</b></h3>\r\n<p></p>\r\n<ol>\r\n<li><span>Find the reaction Activated type I receptor phosphorylates SMAD2/3 directly. What pathway does it belong to?</span></li>\r\n<li><span>In which cellular compartment does this reaction take place?</span></li>\r\n<li><span>What is the GO molecular function associated with the catalyst?</span></li>\r\n<li><span>What reference has experimental verification of this reaction?</span></li>\r\n<li><span>Is this reaction predicted to occur in Canis familiaris? In C. elegans?</span></li>\r\n<li><span>Is this event likely to occur in liver?</span></li>\r\n<li><span>Are 3D structures available for TGF-beta1?</span></li>\r\n</ol>\r\n<p></p>\r\n<h3><span>More info:</span></h3>\r\n<p></p>\r\n<p><b>Pathway Description tab:</b></p>\r\n<p><span>The Description tab for a pathway includes:</span></p>\r\n<ul>\r\n<li><span>pathway name</span></li>\r\n<li><span>species</span></li>\r\n<li><span>stable identifier</span></li>\r\n<li><span>pathway summation</span></li>\r\n<li><span>a drop-down menu to select species for which the pathway is computationally predicted to be conserved, where appropriate. A description of the inference process can be found here.</span></li>\r\n<li><span>the GO biological process term for the pathway, where appropriate. Expanding the panel provides a link-out to GO.</span></li>\r\n<li><span>literature references </span></li>\r\n</ul>\r\n<p><b>Reaction Description tab:</b></p>\r\n<p><span>The Description tab for a reaction includes:</span></p>\r\n<ul>\r\n<li><span>reaction name</span></li>\r\n<li><span>a stable identifier</span></li>\r\n<li><span>pathway summation</span></li>\r\n<li><span>links to external identifiers, where appropriate. For instance, a reaction that describes a binding event may link out to IntAct records that corroborate the interaction. </span></li>\r\n<li><span>Input/Output: Identifies the input/output molecules, sets or complexes for this reaction. Icons to the right of these named items link to further information in Reactome.</span></li>\r\n<li><span>Catalyst (when relevant): The protein or complex that catalyzes the reaction. The Gene Ontology molecular function term that represents the activity of a catalyst or transporter within the reaction will be listed. If the catalyst is a complex, the component that enables the reaction to occur will be identified as the active unit.</span></li>\r\n<li><span>Preceding event(s): A list of events that occur immediately before the event being viewed.</span></li>\r\n<li><span>Following event(s): A list of events that occur immediately after the event being viewed.</span></li>\r\n<li><span>Cellular compartment:</span></li>\r\n<li><span>Inferred from another species (when relevant): This indicates that event has not been experimentally demonstrated in humans, but has been inferred on the basis of data acquired for another species. </span>\r\n<ul>\r\n<li><span>Clicking on the + button to the right of the reaction name in this panel reveals the summation and literature references for the supporting reaction in the other species; </span></li>\r\n<li><span>clicking on the reaction icon to the left of the reaction name in this panel switches the user to the corresponding pathway of the species from which the human reaction has been inferred. Note that the manually curated event in the non-human species is not currently displayed in this pathway; instead, a computationally-inferred reaction is shown. This will be addressed in future updates. The human representation of the pathway can be restored either by clicking on the back button of the browser or by re-selecting Homo sapiens in the species selector in the top panel of the Reactome website.</span></li>\r\n</ul>\r\n</li>\r\n<li><span>Computationally inferred to: Links to descriptions of the events in other species that are either confirmed to occur in a very similar way in both species, or have been electronically inferred. </span></li>\r\n<li><span>Positively (or Negatively) regulated by: The protein or complex that regulates the reaction.</span></li>\r\n<li><span>References that contain experimental data verifying the reaction, with a link-out to PubMed, when applicable.</span></li>\r\n<li><span>Authors: The expert biologists that contributed materials that allowed this reaction to be created in Reactome.</span></li>\r\n</ul>\r\n<ul>\r\n<li><span>Reviewers: The expert biologists that verified the content for this pathway.</span></li>\r\n</ul>\r\n<p><b>Physical Entity Description tab:</b></p>\r\n<p><span>The details of any protein, small molecule, complex or set represented in a pathway diagram can be displayed in the Details panel by selecting the object within the diagram. Clicking a + or - icon associated with the different displayed fields within the Detail panel tabs will show or hide more information about the protein, small molecule, complex and set.</span></p>\r\n<p><span>The Description tab may display include the following information depending upon what &nbsp;physical entity is selected:</span></p>\r\n<ul>\r\n<li><span>Link to corresponding entries in other databases: Cross-reference to identifiers used for this molecule in external reference databases, with hyperlinks to the record.</span></li>\r\n<li><span>Cellular compartment: The cellular compartment that contains this molecule, with hyperlink to the corresponding GO term.</span></li>\r\n<li><span>Computationally inferred orthologues: Lists equivalent molecules in other species if these have been inferred to exist. A description of the inference process can be found here.</span></li>\r\n<li><span>Components: If a complex is selected in the diagram, its components are listed here.</span></li>\r\n<li><span>Produced by: If a complex is selected in the diagram, this tab in the details panel will list all the reactions in Reactome that produce it as an output. Clicking on the + reveals summation and literature references for these reactions.</span></li>\r\n<li><span>Consumed by events: If a complex is selected in the diagram, this tab in the details panel will list all reactions in Reactome that use it as an input. Clicking on the + reveals summation and literature references for these reactions.</span></li>\r\n</ul>\r\n<p></p>\r\n<p><b>Download tab:</b></p>\r\n<p><span>The Download tab provides options for the user to download different files types for the selected pathways. These files include:</span></p>\r\n<ul>\r\n<li><span>SBML: An exchange format used by systems biologists for their models.</span></li>\r\n<li><span>SBGN: An exchange format used to represent pathway and network diagrams.</span></li>\r\n<li><span>BioPAX2: An exchange format used by systems biologists for their models.</span></li>\r\n<li><span>BioPAX3: An exchange format used by systems biologists for their models.</span></li>\r\n<li><span>PDF: Text dump of the pathway, organized to look like a research report.</span></li>\r\n<li><span>Word: A document format compatible with Microsoft and other word processor software.</span></li>\r\n<li><span>Protege: A format used for ontology exchange.</span></li>\r\n</ul>\r\n<p><a href=\"http://sbml.org/Main_Page\"><span>SBML</span></a><span>,</span><a href=\"http://www.sbgn.org/Main_Page\"> <span>SBGN</span></a><span> and</span><a href=\"http://www.biopax.org\"> <span>BioPAX</span></a><span> are exchange formats of interest to bioinformaticians.</span><a href=\"http://en.wikipedia.org/wiki/Microsoft_Word\"> <span>Word</span></a><span> and</span><a href=\"http://en.wikipedia.org/wiki/Portable_Document_Format\"> <span>PDF</span></a><span> are familiar document formats, providing you with a convenient \"document\" of the pathway.</span><a href=\"http://protege.stanford.edu\"> <span>Protg</span></a><span> is an extensible, platform-independent environment for creating and editing ontologies and knowledge bases, this download is likely to be of interest to those wishing to extend Reactome functionality.</span></p>\r\n<p><span>The complete Reactome textbook of biological processes in PDF or RTF format, the complete set of human reactions in Reactome (in SBML or BioPAX level 2 or 3 format), and a list of human protein-protein interaction pairs are available to download from the Download page, linked to the Menu Bar on the Reactome homepage.</span></p>\r\n<p><span>To find out about how Reactome generates SBML, see</span><a href=\"http://wiki.reactome.org/index.php/SBML_At_Reactome\"> <span>SBML At Reactome</span></a><span>.</span></p>\r\n<p><span>For general information about SBML,</span><a href=\"http://sbml.org/index.psp\"> <span>click here</span></a><span>. For more information about BioPAX</span><a href=\"http://www.biopax.org\"> <span>click here</span></a><span>.</span></p>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:49:23','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(83,697,'Searching Reactome','searching-reactome','<h3><b>Simple text search</b></h3>\r\n<p></p>\r\n<p><span>The simple text search tool is located top right of the Home page. To search type a word, phrase or identifier in the search box. The search has an auto-complete function; if the text you wanted to use appears in the drop-down list, select it and results will be displayed. For other text click Search.</span></p>\r\n<p></p>\r\n<p><img src=\"images/userguide/search/search_box.gif\" alt=\"search box\" /></p>\r\n<p></p>\r\n<p><span>Search results are presented in categories with grey headers, representing the molecular entity or event types.</span></p>\r\n<p><span>Below the header is a list of items from the named type that match your search terms. Matching words are highlighted in grey. Click on the name of an item to go to a page of details.</span></p>\r\n<p><span>On the left side of the search results page are groups of checkboxes. These can be used to change defaults and add filters. Check or uncheck boxes for types that you do/dont want to see, e.g. if you want to see results for Mus musculus, under Species uncheck Homo sapiens and check Mus musculus. In the Type, Compartment and Reaction Type filter groups, select to only see results that fall into the selected category, e.g. to see only reactions, select Reaction in the Types filter group. To see only Reactions that occur in the cytosol, also select Cytosol in Compartments.</span></p>\r\n<p></p>\r\n<p><img src=\"images/userguide/search/protein_kinase_results.gif\" alt=\"protein kinase results\" /></p>\r\n<p><span>Click on any of the results to go to a page of information about it. The contents of these Results Detail pages will depend on the type, but all have a section named Locations in the PathwayBrowser. </span></p>\r\n<p><img src=\"images/userguide/search/calmodulin_details.gif\" alt=\"calmodulin details\" /></p>\r\n<p><span>This contains an expandable hierarchy that identifies the locations of the item in Reactome pathway diagrams. Click the plus symbol to open the hierarchy.</span></p>\r\n<p>This hierarchy reflects the organization of events in Reactome. Top-level pathways representing a broad area of biology typically contain one or more levels of subpathways, becoming more specific with each level. The levels of the hierarchy are represented here by indentation; the most general pathways are closest to the left-hand edge of the screen, the most specific subpathways have the greatest indentation to the right. Note that some search result items (e.g. proteins) can be in more than one pathway, which may be in different broad areas of biology, resulting in representation in multiple hierarchies. Note that you can open all the hierarchy levels with a single click on the Expand all link, on the right side. Select any event name to open the corresponding Pathway Diagram.&nbsp;The diagram will open with an animation that shows you where the pathway is located in the Pathway Overview. If the selected item was an event it will be highlighted in blue in the Pathway Diagram. If it was a protein, all instances of it will be highlighted in pink.</p>\r\n<p><strong>Icon Search</strong></p>\r\n<p>Elements of the Icon Library are accessible to the main website search. In the screenshot below, the results of a search for the term: \"liver\" are displayed.&nbsp;</p>\r\n<p><img src=\"images/userguide/search/Icon_Search_Liver.png\" alt=\"Icon Search Liver\" width=\"585\" height=\"400\" /></p>\r\n<p>Below the header is a list of items from the \"icon\" type that match your the \"liver\" search terms. Matching words are highlighted in grey. Click on the name of an item to go to a page of details.&nbsp;In the following image, selecting the first search result is shown</p>\r\n<p><img src=\"images/userguide/search/Icon_Liver_Results.png\" alt=\"Icon Liver Results\" width=\"566\" height=\"400\" /></p>\r\n<p>The metadata associated with the selected icon is displayed, including the icon category, the curator and designer, and a brief description of the icon. To download the image icon files, click the links to the fight of the preview image. Further down the page is a series of expandable hierarchy links that identifies the locations of the icon in Reactome pathway diagrams. Click the plus symbol to open the hierarchy. Clicking the external link, towards the bottom of the page, will link out to the referencing database for the named icon.</p>\r\n<p></p>\r\n<p></p>\r\n<p><b>In-Diagram Search (Searching in the Pathway Browser)</b></p>\r\n<p></p>\r\n<p><span>Searches can be performed within the Pathway Browser using the In-Diagram Search panel (top left corner of the Pathway Panel). </span><span>When the Pathway Overview,&nbsp; Diagram is Enhanced HIgh Level Diagram (EHLD) is displayed, all of Reactome is searched.&nbsp;</span></p>\r\n<p><img src=\"images/userguide/search/In-Diagram-Start.png\" alt=\"In Diagram Start\" width=\"550\" height=\"113\" /></p>\r\n<p></p>\r\n<p><b>Searching the Pathway Overview</b></p>\r\n<p></p>\r\n<p><span>In the example below, as the search term, IL6 has been entered. The search tool has predicted results that match this search term and displays a drop-down list of these matches.&nbsp;</span></p>\r\n<p><img src=\"images/userguide/search/Overview_Search.png\" alt=\"Overview Search\" width=\"550\" height=\"400\" /></p>\r\n<p><b>Note:&nbsp;</b><span>the deep blue bar, which indicates that there are 23 results shown. Only the three results are clearly shown in the results window. To view additional results, move the cursor over the window and scroll down. Each result term has a symbol (to the left) representing the type of molecule or event it is. The first result, with a blue circle icon to the left of the name, represent matches to the IL6R protein, located in the plasma membrane. The second result, with a double-helix icon to the left, is the IL6R gene. The third result is a reaction, with <span>icon showing above/below boxes on left connected by a right-pointing arrow to box on right, representing the binding of IL6 to IL6R</span>. The Reactome stable identifier and cell compartment annotations are also provided for each molecule or event type that is displayed.</span></p>\r\n<p><img src=\"images/userguide/search/Overview_Search_Results_Summary.png\" alt=\"Overview Search Results Summary\" width=\"554\" height=\"400\" /></p>\r\n<p><span>In the following image, the recent search results are displayed at the bottom of the search results window. Clicking the \"Clear History\" button will remove the search terms from the history.</span></p>\r\n<p><img src=\"images/userguide/search/Overview_Search_History.png\" alt=\"Overview Search History\" width=\"553\" height=\"400\" /></p>\r\n<p>In the next image, selecting an item in the Search Results Panel also causes a second drop-down Search Results Details Panel to appear. The additional annotations displayed include the molecule or event type, the Reactome stable identifier, an external reference database identifier, and the cell compartment. Clicking the \"Filter\" button (to the right of the search bar) will display options to filter the search results by molecule or event type, e.g. Pathway, Reaction, Complex, Protein, DNA Sequence, Set, etc. Selecting one or more of the filters will restrict the Search results viewed. Deselecting the filter(s) will alter the search results display.&nbsp;Selecting an item in the initial Search Results Panel causes the Overview to zoom and recentre, focusing on the region that represents pathways containing the selected item. This identifies the selected item and lists the pathways that contain the selected item.&nbsp;</p>\r\n<p><img src=\"images/userguide/search/Overview_Search_Results_Detailed.png\" alt=\"Overview Search Results Detailed\" width=\"550\" height=\"350\" /></p>\r\n<p>In the following image, moving the cursor over a pathway name (under the \"Present in X pathway\") will highlight the corresponding in the Overview, if it is visible in the viewport. The tooltip will display the name of the pathway.</p>\r\n<p><img src=\"images/userguide/search/Overview_Search_Results_Selected.png\" alt=\"Overview Search Results Selected\" width=\"550\" height=\"365\" /></p>\r\n<p>Clicking a pathway name (under the \"Present in X pathway\") in the Search Results Detail Panel will select the corresponding object in the Overview. If it is not visible, the Pathway Diagram will re-centre to show it. Double-click a pathway name in the Search Results Detail Panel or pressing the \"GO &gt;&gt;&gt;\" button will open the corresponding Pathway Diagram. If you need to return to the Overview, you can use the browser back button.&nbsp;<b>Note</b>&nbsp;that in the image above, the Search Results Detail Panel extends beyond the limits of the Overview window, consequently some results are hidden. This explains why the results details title says Present in 3 pathways but only 2 are visible. To see all the matching pathways, click in the pathway list in the Search Results Detail Panel.&nbsp;</p>\r\n<p>In the following image, the \"Flag\" button in the Search Results Details Panel (to the right of IL6R) has been clicked and the events that contain the selected molecule are highlighted blue and the connected events are highlighted purple. Clicking the \"Flag\" button for a second time will remove the event highlighting.</p>\r\n<p><img src=\"images/userguide/search/Overview_Search_Results_Highlighting.png\" alt=\"Overview Search Results Highlighting\" width=\"550\" height=\"361\" /></p>\r\n<p></p>\r\n<p><b>Searching a Pathway Diagram</b></p>\r\n<p></p>\r\n<p><span>In the example below, the search has identified IL6R and produced a dropdown list of matches. The first match is to a protein, as indicated by the icon on the left. The second match with the icon showing a double helix is a DNA sequence or gene. Note that if IL6R was represented in two cellular compartments in this diagram it would appear twice in this initial search result.&nbsp;<span>The Reactome stable identifier and cell compartment annotations are also provided for each molecule or event type in the search results. <strong>Note:</strong><span>&nbsp;as the search term is entered, the search tool will predict results that match this search term and displays a drop-down list of these matches. </span></span></span></p>\r\n<p><span><span><span>Although not shown below, when a new search is performed, the recent search results are displayed at the bottom of the search results window. Clicking the \"Clear History\" button will remove the search terms from the history.<br /></span></span></span></p>\r\n<p><span><img src=\"images/userguide/search/In-Diagram_Search.png\" alt=\"In Diagram Search\" width=\"550\" height=\"401\" /></span></p>\r\n<p>The deep blue highlighted bar, which indicates that there are 3 results shown in \"This diagram\". Only the two results are clearly shown in the results window. To view additional results, move the cursor over the window and scroll down. Each result term has a symbol (to the left) representing the type of molecule or event it is. The first result, with a blue circle icon to the left of the name, represent matches to the IL6R protein, located in the plasma membrane. The second result, with a double-helix icon to the left, is the IL6R gene. The Reactome stable identifier and cell compartment annotations are also provided for each molecule or event type that is displayed.</p>\r\n<p>In the next image, clicking the \"Filter\" button (to the right of the search bar) will display options to filter the search results by molecule or event type, e.g. Pathway, Reaction, Complex, Protein, DNA Sequence, Set, etc. Selecting one or more of the filters will restrict the Search results viewed. Deselecting the filter(s) will alter the search results display.</p>\r\n<p><img src=\"images/userguide/search/In-Diagram_Search_Filter.png\" alt=\"In Diagram Search Filter\" width=\"550\" height=\"362\" /></p>\r\n<p><span>In the next image, selecting an item in the Search Results Panel also causes a second drop-down Search Results Details Panel to appear. The at the top of the Details Panel, additional annotations are displayed that include the molecule or event type, the Reactome stable identifier, an external reference database identifier (if applicable), and the cell compartment. Additional annotations are listed below describing the involvement of the protein within complexes, sets and reactions within the diagram. <span>Selecting names in the Search Results Detail Panel will select the corresponding object in the Pathway Diagram. If it is not visible, the Pathway Diagram will re-centre to show it.</span><br /></span></p>\r\n<p><span>In this example query, clicking the \"Flag\" button in the Search Results Details Panel (to the right of IL6R) will highlight all (15) the objects in the diagram that contains IL6R. Clicking the \"Flag\" button for<span> a second time </span>(or the \"X\" button toward the bottom of the pathway viewport) will remove the object highlighting. <span><span>To see all the matches, it is necessary to resize the Pathway Diagram panel.</span></span><br /></span></p>\r\n<p><img src=\"images/userguide/search/In-Diagram_Search-Entity_This_Diagram.png\" alt=\"In Diagram Search Entity This Diagram\" width=\"550\" height=\"349\" /></p>\r\n<p><span><span>In the following image, selecting the second item in the Search Results Panel (IL6 binds IL6R) causes a second drop-down Search Results Details Panel to appear. The at the top of the Details Panel, additional annotations are displayed that include the event type, the Reactome stable identifier, and the cell compartment. Additonal annotations are listed below describing the involvement of the reaction within the diagram, and participants of the reaction. <span>Selecting names in the Search Results Detail Panel will select the corresponding object in the Pathway Diagram. If it is not visible, the Pathway Diagram will re-centre to show it. in this image, the IL6 binds IL6R reaction is blue highlighted.</span></span></span><br /><span></span></p>\r\n<p><img src=\"images/userguide/search/In-Diagram_Search-Event_This_Diagram.png\" alt=\"In Diagram Search Event This Diagram\" width=\"550\" height=\"348\" /></p>\r\n<p><span>In the next image, clicking the \"All diagram\" tab will present a different set of results when selecting an item in the Search Results Panel. As before, at the top of the Details Panel, additional annotations are displayed that include the molecule or event type, the Reactome stable identifier, an external reference database identifier (if applicable), and the cell compartment. Instead of displaying objects within the currently displayed pathway, the pathways not shown in the pathway viewport that contain the search term are displayed. <br /></span></p>\r\n<p><img src=\"images/userguide/search/In-Diagram_Search-Gene_All_Diagrams.png\" alt=\"In Diagram Search Gene All Diagrams\" width=\"550\" height=\"351\" /></p>\r\n<p><span>Selecting the hyperlinked pathway name in the Search Results Detail Panel (\"Present in n pathway diagram\") will open the new pathway diagram in the Pathway Browser and select the corresponding object in the new diagram. </span><span></span></p>\r\n<p><img src=\"images/userguide/search/In-Diagram_Search-Gene_Set_All_Diagrams.png\" alt=\"In Diagram Search Gene Set All Diagrams\" width=\"550\" height=\"349\" /></p>\r\n<p></p>\r\n<p><b>Searching&nbsp;within an Enhanced HIgh Level Diagram (EHLD)<br /></b></p>\r\n<p></p>\r\n<p>In the example below, the search has identified IL6R and produced a dropdown list of matches. The first match is to a protein, as indicated by the icon on the left. The second match with the icon showing a double helix is a DNA sequence or gene. Note that if IL6R was represented in two cellular compartments in this diagram it would appear twice in this initial search result.&nbsp;The Reactome stable identifier and cell compartment annotations are also provided for each molecule or event type in the search results. <strong>Note:</strong>&nbsp;as the search term is entered, the search tool will predict results that match this search term and displays a drop-down list of these matches.</p>\r\n<p>Although not shown below, when a new search is performed, the recent search results are displayed at the bottom of the search results window. Clicking the \"Clear History\" button will remove the search terms from the history.</p>\r\n<p><img src=\"images/userguide/search/EHLD_Search.png\" alt=\"EHLD Search\" width=\"550\" height=\"393\" /></p>\r\n<p>The deep blue highlighted bar, which indicates that there are 23 results shown in \"This diagram\" and 30 results shown in \"All diagrams\". Only the two results are clearly shown in the results window. To view additional results, move the cursor over the window and scroll down. Each result term has a symbol (to the left) representing the type of molecule or event it is. The first result, with a blue circle icon to the left of the name, represent matches to the IL6R protein, located in the plasma membrane. The second result, with a double-helix icon to the left, is the IL6R gene. The Reactome stable identifier and cell compartment annotations are also provided for each molecule or event type that is displayed.</p>\r\n<p><span>In the following image, selecting the first item in the Search Results Panel (IL6R) causes a second drop-down Search Results Details Panel to appear. The at the top of the Details Panel, additional annotations are displayed that include the molecule or event type, the Reactome stable identifier, and the cell compartment. Additional annotations are listed below describing the involvement of the object within the EHLD. <span>Selecting pathway names in the Search Results Detail Panel will select the corresponding object in the EHLD. If it is not visible, the EHLD will re-centre to show it in the </span></span><span></span></p>\r\n<p><img src=\"images/userguide/search/EHLD_Search_Results_Detailed.png\" alt=\"EHLD Search Results Detailed\" width=\"550\" height=\"392\" /></p>\r\n<p></p>\r\n<p></p>\r\n<p><span>Getting Started</span></p>\r\n<p></p>\r\n<h3><b>Search Exercises</b></h3>\r\n<p></p>\r\n<p><span>This exercise encourages you to find information via the menu bar and Search </span></p>\r\n<p></p>\r\n<ol>\r\n<li><span>Whats the latest news item on the Reactome Home page?</span></li>\r\n<li><span>How many human proteins are represented in Reactome?</span></li>\r\n<li><span>Whats the first item listed that will be included in the next release?</span></li>\r\n<li><span>Is VAV2 in Reactome?</span></li>\r\n<li><span>How many reactions involve VAV2?</span></li>\r\n<li><span>Are there any complexes that include VAV2?</span></li>\r\n<li><span>Is CRB2 in Reactome?</span></li>\r\n</ol>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:50:12','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(84,698,'Analysis Tools','analysis-tools','<p><span>Click on the Browse Pathways button on the Homepage. In the next page, click the Analysis button on the top right:</span></p>\r\n<p><span><img src=\"/images/userguide/analysis/analysis_1.png\" alt=\"\" width=\"222\" height=\"81\" /></span></p>\r\n<p><span>Alternatively, select the Analyze Data button on the Homepage:</span></p>\r\n<p><img src=\"images/userguide/analysis/analysis_2.png\" alt=\"analyze data button\" width=\"188\" height=\"206\" /></p>\r\n<p><span>This opens a submission form, where you can select the analysis you want to perform, paste in or browse to a file containing your data, or use an example data set.</span></p>\r\n<p><em><img src=\"images/userguide/analysis/analysis_3.png\" alt=\"analysis 3\" width=\"626\" height=\"400\" /></em></p>\r\n<p></p>\r\n<p></p>\r\n<p></p>\r\n<p><span>There are two sections to the submission form. The Analyse your data section is selected by default to submit your data. Several different analyses can be performed, depending on the format of your data.</span></p>\r\n<p><span>If your data is a single column of identifiers such as UniProt IDs, gene symbols or ChEBI IDs, they are mapped to pathways and over-representation and pathway-topology analyses are run. Over-representation analysis is a statistical (hypergeometric distribution) test that determines whether certain Reactome pathways are over-represented (enriched) in the submitted data. It answers the question Does my list contain more proteins for pathway X than would be expected by chance? This test produces a probability score, which is corrected for false discovery rate using the Benjamani-Hochberg method.</span></p>\r\n<p><span>Pathway topology analysis considers the connectivity between molecules that is represented by the pathway steps (which we refer to as reactions) in the pathway. It groups all the molecules represented in each reaction as a pathway unit. If any of these molecules are represented in your query set, this is considered a match to that reaction. This may give a better indication of the proportion of the pathway that matches your data, rather than the number of molecules that are common between your data and the pathway. It may also indicate that your data matches the start, end or a particular branch of a pathway process. This test does not have a probability score.</span></p>\r\n<p><span>If your data has one or more additional columns of numbers it will be recognized as expression data and expression data overlay will be performed. Note that this data format should include a header row. The first column header should start with the # symbol. The numbers are used to produce a scaled coloured overlay over Reactome pathway diagrams, as a means to visualize relative expression levels. Note that the numeric values do not have to be expression data, for instance by using gene association scores the same analysis can be used to visualize genotyping results. &nbsp;&nbsp;</span></p>\r\n<p></p>\r\n<h3><b>Identifier mapping </b></h3>\r\n<p><span>The submission process recognizes many types of identifiers. As part of the pre-analysis, they are mapped to Reactome molecules. The ideal identifiers to use are UniProt IDs for proteins, ChEBI IDs for small molecules, and either HGNC gene symbols or ENSEMBL IDs for DNA/RNA molecules, as these are our main external reference sources for proteins and small molecules. Many other identifiers are recognized and mapped to appropriate Reactome molecules. Accepted identifiers include HUGO gene symbols, GenBank/EMBL/DDBJ, RefPep, RefSeq, EntrezGene, MIM, InterPro, EnsEMBL protein, EnsEMBL gene, EnsEMBL transcript, and some Affymetrix and Agilent probe IDs. UniProt isoforms may be specified using the format P12345-2. If the -n suffix is omitted, this canonical form and all isoforms of it will be matched. Mixed identifier lists (different protein identifiers or protein/gene identifiers) may be used. Identifiers must be one per line. Protein-specific identifiers will typically map to protein entities, while gene-specific identifiers will map to the gene, transcript and derived proteins. If desired results can be filtered to show protein-specific or gene/transcript-specific results, details below. &nbsp;&nbsp;&nbsp;</span></p>\r\n<p><span>Below is an example of the identifiers-only format:</span></p>\r\n<p><span>Click the Continue button. A second options selection page appears:</span></p>\r\n<p><em><img src=\"images/userguide/analysis/analysis_4.png\" alt=\"analysis 4\" width=\"626\" height=\"400\" /></em></p>\r\n<p><span>Project to human is checked by default. With this option selected, all non-human identifiers in your query are converted by the analysis service to their human equivalents. In general, this maximizes the chances of a successful match to Reactomes curated human pathways. However, if you want to use non-human identifiers and match these to our computationally-inferred non-human pathways, uncheck the box. You may also choose to uncheck this box if your query consists of a mixture of human and microbial identifiers and your goal is to find pathways that represent the processes of infection.</span></p>\r\n<p><span>Include Interactors is unchecked by default. With this box unchecked, your query will consider only Reactome pathways. If you choose to check the box, your query will consider Reactome pathways that have been expanded by including all available protein-protein interactors from the IntAct database. This greatly increases the size of Reactome pathways, which maximizes the chances of matching your submitted identifiers to the expanded pathway, but will include interactors that have not undergone manual curation by Reactome and may include interactors that have no biological significance, or unexplained relevance. In practice it is preferable to query with Include Interactors unchecked in the first instance, followed by a repeated query with Include interactors selected, if a substantial proportion of the submitted identifiers do not match a Reactome pathway, to see if they can be identified as interactors.</span></p>\r\n<p></p>\r\n<h3><b>Results for Identifier lists without associated numeric values</b></h3>\r\n<p></p>\r\n<p><span>If you submit a single column of protein or small molecule identifiers they are mapped to pathways and over-representation and pathway-topology analyses are performed. The results will resemble the example below.</span></p>\r\n<p><span><img src=\"images/userguide/analysis/analysis_6.png\" alt=\"analysis 6\" width=\"965\" height=\"591\" /></span></p>\r\n<p><span>Analysis results are shown in the Analysis tab, within the Details Panel. All Reactome pathways are shown, in blocks of 20 pathways, ranked by the p-value obtained from over-representation analysis. If multiple pathways have the same p-value, they are ranked by the number of identifiers in the query that match the pathway. The number of molecules matched/total number of molecules and FDR values are added to the right side of pathway names in the Hierarchy Panel. The names of reactions that match at least one identifier in the query, representing positive pathway topology analysis hits, are boxed in orange. </span></p>\r\n<p><span>In the Analysis tab, clicking on the name of a pathway will select it in the Hierarchy, which if necessary will expand hidden hierarchical levels to show the pathway, while the name becomes highlighted in dark blue. </span></p>\r\n<p><span>By default, molecules of all types (protein, small molecules, genes, transcripts) are used for over-representation analysis, but it is possible to restrict the analysis results to a specific subtype by using a drop-down list located top-left of the results table. Selecting one of the subsets will display results that consider only the selected molecular subtype.</span></p>\r\n<p><span><img src=\"/images/userguide/analysis/analysis_7.png\" alt=\"\" width=\"529\" height=\"36\" /></span></p>\r\n<p><span>The columns in analysis details represent:</span></p>\r\n<ol>\r\n<li><span>Pathway name: Click the name to open the pathway. </span></li>\r\n<li><span>Entities found: the number of curated molecules of the type selected with Results Type that are common between the submitted data set and the pathway named in column 1. Click on this number to display the matched submitted identifiers and their mapping to Reactome molecules.</span></li>\r\n<li><span>Entities total: The total number of curated molecules of the type selected with Results Type within the pathway named in column 1.</span></li>\r\n<li><span>Interactors found (if this option was selected). The number of interactor molecules of the type selected with Results Type that are common between the submitted data set and the pathway named in column 1. Click on this number to display the matched submitted identifiers and their mapping to Reactome molecules.</span></li>\r\n<li><span>Interactors total (if this option was selected): The total number of interactor molecules of the type selected with Results Type within the pathway named in column 1.</span></li>\r\n<li><span>Entities ratio: Put simply, the proportion of Reactome pathway molecules represented by this pathway. Calculated as the ratio of entities from this pathway that are molecules of the type selected with Results Type Vs. all entities of the type selected with Results Type. </span></li>\r\n<li><span>Entities pvalue: The result of the statistical test for over-representation, for molecules of the results type selected. &nbsp;</span></li>\r\n<li><span>Entities FDR: False discovery rate. Corrected over-representation probability.</span></li>\r\n<li><span>Reactions found: The number of reactions in the pathway that are represented by at least one molecule in the submitted data set, for the molecule type selected with Results Type.</span></li>\r\n<li><span>Reactions Total: The number of reactions in the pathway that contain molecules of the type selected with Results Type.</span></li>\r\n<li><span>Reactions ratio: Put simply, the proportion of Reactome reactions represented by this pathway. Calculated as the ratio of reactions from this pathway that contain molecules of the type selected with Results Type Vs. all Reactome reactions that contain molecules of the type selected with Results Type.</span></li>\r\n<li><span>Species Name.</span></li>\r\n</ol>\r\n<p><span>When an analysis has run, the Pathway Browser will display the Pathway Overview. All pathways that contain identifiers from your submitted list are highlighted, using a coloured scale to indicate the corrected probability (FDR). The colour scheme can be changed using the colour profiles tab (artists easel icon) on the pop-out Settings panel, found on the right-hand edge of the pathway panel. &nbsp;Selecting coverage in the Overexpression panel will alter the Pathway Overview display to show additional event crosslinks corresponding to areas that are heavily covered vs regions that have lower coverage (as shown in the pValue view), and making it easier to visualize the pathways enriched within your dataset.</span></p>\r\n<p><span>The highlighting of pathways in the Overview provides an at-a-glance representation of analysis results for all pathways. To see the details of a specific pathway, double-click the node representing the pathway in the overview or in the Pathway Hierarchy on the left. Alternatively, click it once to select it and use the Show All button (square with outward pointing triangles inside) in the top left corner of the Overview panel. The Overview can be navigated using the mouse scroll wheel to zoom in and out and click and drag to move it around. Alternatively, use the navigation buttons in the bottom right corner of the overview panel. At any level of the pathway, the diagram key can be found by clicking the compass symbol in the top right corner.</span></p>\r\n<p><span><img src=\"images/userguide/analysis/analysis_8.png\" alt=\"analysis 8\" width=\"962\" height=\"591\" /></span></p>\r\n<p></p>\r\n<p><span>Enhanced high-level diagrams represent analysis results within the label for subpathways. The label background changes from blue to white, a yellow band is used to indicate the proportion of the pathway that is represented in the query dataset. A grey bar above the label indicates the number of pathway entities that are represented in the query dataset, the total number of entities in the pathway, and the FDR corrected probability score</span></p>\r\n<p><span><img src=\"https://lh5.googleusercontent.com/jZ7Ukd-MjPcGNyx6RSvU28O3eLkSX0pZgSFOszHppikbW9wqfntjOYQ5kFwL-ObwbpZ2UKoM_x2QFCEqzSCC4LxaahhgnooZ50aTM5Fz7mDNdH8-leWXbHhEB_XUzx3tV08Bq0DG6gsJGKoWKA\" alt=\"\" width=\"602\" height=\"407\" /></span></p>\r\n<p><span>In Pathway Diagrams, entities are re-coloured (yellow in the default colour scheme) if they were represented in the submitted data set. Complexes, Sets and Subpathway Icons are coloured to represent the proportion that is represented in the submitted identifier list. In the figure below, Insulin receptor is yellow indicating that is was in the submitted list. Insulin was not in the submitted list so it is not re-coloured. The complex of insulin:Insulin receptor is part re-coloured, part not, indicating that some molecules in the complex were represented in the submitted dataset while others were not. </span></p>\r\n<p><span><img src=\"https://lh3.googleusercontent.com/bUXkrCjyaY_-ITh0HOmDjGXVFZH2E9OerBgiWOCi9CQtqGEL5jUqtcidSh8mW-g2UibuOWwmU9qYanNpzqYuWgU5FLB0Op7j1prz-VyT_nU4dtBGgwDZzXodkfiDhRRr8IUOTyasGQeIoSy1AQ\" alt=\"\" width=\"367\" height=\"188\" /></span></p>\r\n<p><span>If the Include Interactors option was checked, entities with interactors that were part of the submitted identifier list have a ribbon across the top right corner. In the example below, RASA1 is not a direct match with the submitted list of identifiers but has interactors that match the list. The interactors are displayed; the yellow overlay indicates the matching interactors. PTPN11 is a direct match and has interactors that match the list. A limited number of interactors are displayed to avoid crowding.</span></p>\r\n<p><span><img src=\"https://lh3.googleusercontent.com/Dg_JUFg916QZ4lueNufNjTHHXwnau0B2MPBHxv1l2QBHfJbvdmL3cDW1TLsad6Rxpi9_tbtelLh8Ypmzb-XM0HSyIwqpSYptyT8gmSXnVXIUzJLbPfSYUGmSxVvX2cZasVJITpT5nyP9NTs4iA\" alt=\"\" width=\"529\" height=\"311\" /></span></p>\r\n<p><span>At the right side of the Analysis results details is a button indicating the number of submitted identifiers that were not successfully matched to molecules in Reactome. Click the button to produce a list.</span></p>\r\n<p><span><img src=\"https://lh3.googleusercontent.com/PVNAysf0Ucim_yvhZglD5IFLHTBgbsFhDjm2Z-juGLcot0Lv0NVzsaEINTxMcVhkrbCHuCU7SYsKr_a4g5z6IkI2TZMDj4W4es5NUsfJbtmR1mz4m6xZC9cfvlY3qo5LDlABen6DQTf3h6r1FQ\" alt=\"\" width=\"529\" height=\"134\" /></span></p>\r\n<p></p>\r\n<p><span></span></p>\r\n<p></p>\r\n<h3><b>Results for Identifier lists with associated numeric values (expression representation)</b></h3>\r\n<p></p>\r\n<p><span>To run expression analysis, submit your data in a format that includes a first row of column headers. The header for column 1 must start with the # symbol. The first column must contain protein, compound or other suitable identifiers, such as probe IDs. All other columns must be numeric values, with no alphabetical characters. The analysis tool will interpret your data as expression data. The numeric values are used to colour objects in pathway diagrams. This view was created for microarray data, but any dataset that consists of a list of identifiers with associated numeric values can be used, e.g. quantitative proteomics, GWAS scores.</span></p>\r\n<p></p>\r\n<p><span>The tool is launched using the Analyse data button in the Pathway Browser header bar. Either paste your data into the submission form or browse to a saved file (or select an example file). </span></p>\r\n<p><span>The figure below shows the correct data format. Each row must have an identifier in the first column (a header row is optional). </span></p>\r\n<p><span>The submission process recognized many types of identifiers. As part of the pre-analysis, they are mapped to equivalent UniProt accessions or for small compounds to ChEBI IDs. These are the ideal identifiers to use with Reactome analysis tools. Other identifiers that are recognized and converted to UniProt equivalents include HUGO gene symbols, GenBank/EMBL/DDBJ, RefPep, RefSeq, EntrezGene, MIM and InterPro IDs, some Affymetrix and Agilent probe IDs, Ensembl protein, transcript and gene identifiers. Identifiers that contain only numbers such as those from OMIM and EntrezGene must be prefixed by the source database name and a colon e.g. MIM:602544, EntrezGene:55718. Mixed identifier lists (different protein identifiers or protein/gene identifiers) may be used. Identifiers must be one per line.</span></p>\r\n<p><span>By default, all non-human identifiers are mapped to their human equivalents, unless the Project to the human checkbox is unselected. </span></p>\r\n<p></p>\r\n<p><span>After column 1, all other columns must contain numbers, representing expression or other values. Microsoft Excel and tab separated value (TSV) files can be used. When submitted, columns of numbers are considered as separate samples or experimental conditions. The values are used to overlay colour onto pathway diagrams. An Experiment Browser tool allows you to select and view-overlays for each submitted data column. This tool is the panel that appears in the bottom-centre of both the pathways overview and the diagram viewer when expression data columns have been submitted. In that panel, the user can move through the different time series (and also play it as a \"movie\"). This is particularly useful for visualizing time-points or a disease progression.</span></p>\r\n<p><span><img src=\"images/userguide/analysis/analysis_14.png\" alt=\"analysis 14\" width=\"678\" height=\"400\" /></span></p>\r\n<p><span>The results may take a few seconds to appear. </span></p>\r\n<p></p>\r\n<p><span>The results page is very similar to that seen following submission of a simple one-column list of identifiers, with extra columns in the Analysis details following column 9. &nbsp;These extra columns represent the submitted expression values. </span></p>\r\n<p></p>\r\n<p><span>Clicking on a pathway name launches the Pathway Browser and displays the relevant Pathway Diagram (see example below).</span></p>\r\n<p><span><img src=\"https://lh3.googleusercontent.com/D1rR1J18cP80pHIIC3i2Rr_SXaXtmH3CcPzSDNTTDBW4W0UIfeXJKX16mXl4iHe29byGFYaHwoQGch1Na9-3kVQ7jUCNyfuuiIVtl2TF-ip2myR-mUkX1REFr3mXtlTrYOMDpzuKpztd5uClvw\" alt=\"\" width=\"529\" height=\"164\" /></span></p>\r\n<p><span>Objects in the Pathway Diagram are re-coloured according to the numeric values submitted. The colours are based on a scale as represented in a bar on the right-hand side. There are several colour schemes, selected using the Settings pop-out panel on the right side. The scale (on the right side) automatically adjusts to fit the range of values in the dataset.</span></p>\r\n<p><span>Objects that were not represented in the input data are not re-coloured.</span></p>\r\n<p><span>Objects with bands of colour represent complexes or sets containing more than one molecule. When zoomed out, the colour of the band reflects the average of the values submitted, for molecules represented in the dataset. The size of the band reflects the proportion of molecules that had submitted values. Zoom in to see individual bands, for each molecule that had a submitted value, arranged alphabetically by name. The bands of colour now reflect the values submitted. If multiple columns of values were submitted, representing multiple samples, e.g. time-points or a disease progression, the order of the bands will be the same for each sample.</span></p>\r\n<p><span>To view details of the components of a complex or set right click it. This opens a Contextual Information Panel (CIP) (see figure above). This has two tabs - Molecules and Pathways. Molecules show the participating molecules, and if an expression analysis has been performed, their expression values. Pathways identify whether the selected object is present in any other Reactome pathways, with links to the appropriate Pathway Diagrams.</span></p>\r\n<p><span> &nbsp;&nbsp;&nbsp;Multiple CIPs can be opened and pinned so they remain visible when other entities are selected.</span></p>\r\n<p><span> &nbsp;&nbsp;&nbsp;The browser remembers CIPs that were pinned on the last visit (for up to 5 diagrams).</span></p>\r\n<p></p>\r\n<p><span>The orange Experiment Browser toolbar (bottom left in the figure above) is used to step through the columns of your data, e.g. time-points or disease progression. Move between them by clicking the arrow buttons. The header of the data column (if present) is displayed between the arrows. The Pathway Diagram will re-colour to reflect the new values.</span></p>\r\n<p></p>\r\n<p><span>For an explanation of the results seen in the Identifier results tab see the explanation in the section Results for Identifier lists without associated numeric values.</span></p>\r\n<p></p>\r\n<h3><span><b>Analysis Report</b></span></h3>\r\n<p></p>\r\n<p>For some of our users, it might be important to preserve a pathway analysis for the long-term. Following the pathway analysis, the results can be downloaded as an easy-to-read PDF report by clicking the Report (PDF) button located at the bottom left corner of the Details panel. The first page of the PDF report will look something like this:</p>\r\n<p><img src=\"images/userguide/analysis/analysis_16.png\" alt=\"analysis 16\" width=\"626\" height=\"400\" /></p>\r\n<p>The sections of the analysis report represent:</p>\r\n<ol>\r\n<li>Introduction: An overview of the Reactome project and the Analysis tool.</li>\r\n<li>Properties: A summary details of the analysis output. For example, the number of identifiers that were found in Reactome.&nbsp;&nbsp;</li>\r\n<li>Genome-wide overview: A&nbsp;genome-wide overview of the results of your pathway analysis.</li>\r\n<li>Most significant pathways: A table of the top 25 pathway hits. Results are ranked based upon the most significant FDR value. The embedded links within the pathway name of the table will connect to the Pathway details (Section #5).</li>\r\n<li>Pathway details: A summary view of each pathway hit. The pathway identifier in parenthesis when clicked will connect with the Pathway details page on the Reactome website. A pathway diagram, pathway summation, edit history, list of identifiers found, and references (when available) are also provided.</li>\r\n<li>Identifiers found: A table of the synonyms or identifiers that were found in Reactome based upon the input list.</li>\r\n<li>Identifiers not found:&nbsp;A table of the synonyms or identifiers that were not found in Reactome based upon the input list.</li>\r\n</ol>\r\n<p></p>\r\n<p></p>\r\n<h3><b>Species Comparison</b></h3>\r\n<p></p>\r\n<p><span>The manually-curated human pathways in Reactome are used to predict equivalent pathways in 18 other species. This automated, computational process is based on orthology. A full description of the inference process can be found on the Home page under Documentation, Orthology Prediction. </span></p>\r\n<p><span>The Species Comparison tool allows you to compare human pathways with computationally-predicted pathways in model organisms, highlighting the elements of the pathway that are common to both species and those that may be absent in the model organism. </span></p>\r\n<p><span>Species Comparison is launched using the Analysis button in the Pathway Browser header bar. In the Species Comparison section, select one of the species in the dropdown list. Click the Go button:</span></p>\r\n<p><span><img src=\"images/userguide/analysis/analysis_10.png\" alt=\"analysis 10\" width=\"678\" height=\"400\" /></span></p>\r\n<p><span>The results are ready to view when analysis results appear (or are updated if already present) in the Pathway Hierarchy. </span></p>\r\n<p></p>\r\n<p><span>Clicking on a pathway name launches the Pathway Browser and displays the relevant Pathway Diagram (see example below).</span></p>\r\n<p><span><img src=\"https://lh4.googleusercontent.com/Kue8q__DoMAJ18GVVlNW5sk3ZSnC89GaZIlXRrWhjoZux_aE5FfESnAyoFDYko6KsUxK0COzZXKO4_9H3ZFVA0Q7zNOTX53A85M593i5qcE96_7IWSrUWPyiwCXdNcLnU70z1DznFYETXww6Iw\" alt=\"\" width=\"602\" height=\"140\" /></span></p>\r\n<p><span>The colour of reaction objects indicates the result of the comparison:</span></p>\r\n<ul>\r\n<li><span>Yellow indicates that the protein has an inferred equivalent in the comparison species.</span></li>\r\n<li><span>No overlay indicates that inference was not possible. This is always the case for small molecules, DNA and other objects that have no UniProt entry (or did not at the time the pathway was constructed).</span></li>\r\n<li><span>Objects with bands of colour represent complexes or sets containing more than one molecule. The bands of colour reflect the inference success for the molecules within the complex/set.</span></li>\r\n</ul>\r\n<p></p>\r\n<p><span>To view species comparison results for a complex or set hover your mouse over the object and a small blue triangle will appear on its right side. Select this to open the Contextual information Panel.</span></p>\r\n<p><span>This reveals a table representing all the proteins involved in the complex/set. Each square in the grid represents one component of the complex/set, coloured as described above.</span></p>\r\n<p><span><img src=\"https://lh6.googleusercontent.com/Txu6bW9KWTCfMU42Z2bLTxuAqCsu6CgtuPfG99UPyDTWzzcjOpM1UQ_N0ARCb-PAidCqnTtlUr3uvQHdrMlu_ITZ5rYbDWHcaupDlJOzjMjJf7ER5CwPWA_AUN8OmMAnq3n3FfXh9cpnOjWctQ\" alt=\"\" width=\"560\" height=\"295\" /></span></p>\r\n<p><span>The species bar at the bottom of the Pathway Diagram (see example above) can be used to turn off species comparison colouring, by unchecking the box. </span></p>\r\n<p><span>Refer to the Navigating Pathway Diagrams section for more information on the diagram content.</span></p>\r\n<p><span></span></p>\r\n<h3><b>Tissue Expression</b></h3>\r\n<p>Traditionally, reactions in Reactome represent events that occur within a single generic human cell. It would, however, be useful to classify reactions into different human tissue types, as to provide an evolving picture of the reactions and pathways in different cell- and tissue-specific environments. We have imported protein expression in different cell/tissue types from the <a href=\"https://www.proteinatlas.org/\">Human Protein Atlas</a>&nbsp;(HPA), overlaid these proteins on Reactome data, and extracted the subset of reactions for that particular cell type. The HPA data reflecting the expression of the protein-coding genes in 44 different human tissues can be visualized through the Analysis tools.</p>\r\n<h3><img src=\"images/userguide/analysis/analysis_17.png\" alt=\"analysis 17\" width=\"678\" height=\"400\" /></h3>\r\n<p><span>Tissue Distribution is launched using the Analysis button in the Pathway Browser header bar. In the <span>Tissue Distribution</span> section, select the one experimental tissue dataset [HPA (E-PROT-3) - Expression Atlas] in the dropdown list. Once the window refreshes, select the \'Available Tissues\' in the left panel and hit the \'Add\' button to&nbsp;add tissue expression data to the analysis tool. Press the \"Add All\' button, if you would like to adjoin all the tissue expression data. Use the \'Remove all\' and \'Remove\' buttons to delete tissue expression data from the filter list. Once you have selected all the appropriate tissues, c</span>lick the Go button to start the analysis.</p>\r\n<p><img src=\"images/userguide/analysis/analysis_18.png\" alt=\"analysis 18\" width=\"677\" height=\"400\" /></p>\r\n<p><span><span>The results are ready to view when analysis results appear (or are updated if already present) in the Pathway Hierarchy and the Overview panel.</span></span></p>\r\n<p><img src=\"images/userguide/analysis/analysis_19.png\" alt=\"analysis 19\" width=\"710\" height=\"400\" /></p>\r\n<p>Clicking on a pathway name launches the Pathway Browser and displays the relevant Pathway Diagram (see example below).</p>\r\n<p><img src=\"images/userguide/analysis/analysis_21.png\" alt=\"analysis 21\" width=\"744\" height=\"275\" /></p>\r\n<p><span>Objects in the Pathway Diagram are re-coloured according to the numeric values submitted. The colours are based on a scale as represented in a bar on the right-hand side. There are several colour schemes, selected using the Settings pop-out panel on the right side. The scale (on the right side) automatically adjusts to fit the range of values in the dataset.</span></p>\r\n<p><span>Objects that were not represented in the input data are not re-coloured.</span></p>\r\n<p><span>Objects with bands of colour represent complexes or sets containing more than one molecule. When zoomed out, the colour of the band reflects the average of the values submitted, for molecules represented in the dataset. The size of the band reflects the proportion of molecules that had submitted values. Zoom in to see individual bands, for each molecule that had a submitted value, arranged alphabetically by name. The bands of colour now reflect the values submitted. If multiple columns of values were submitted, representing multiple samples, e.g. time-points or a disease progression, the order of the bands will be the same for each sample.</span></p>\r\n<p><span>To view details of the components of a complex or set right click it. This opens a Contextual Information Panel (CIP) (see figure above). This has two tabs - Molecules and Pathways. Molecules show the participating molecules, and if an expression analysis has been performed, their expression values. Pathways identify whether the selected object is present in any other Reactome pathways, with links to the appropriate Pathway Diagrams.</span></p>\r\n<p><span> &nbsp;&nbsp;&nbsp;Multiple CIPs can be opened and pinned so they remain visible when other entities are selected.</span></p>\r\n<p><span> &nbsp;&nbsp;&nbsp;The browser remembers CIPs that were pinned on the last visit (for up to 5 diagrams).</span></p>\r\n<p><span>The Experiment Browser toolbar (bottom left in the figure above) is used to step through the columns of your data, e.g. time-points or disease progression. Move between them by clicking the arrow buttons. The header of the data column (if present) is displayed between the arrows. The Pathway Diagram will re-colour to reflect the new values.</span></p>\r\n<p>Clicking on the illustration icon (photo-like icon in the top right of the Pathway Browser) displays the relevant Pathway Illustration (see example below).</p>\r\n<p><img src=\"images/userguide/analysis/analysis_20.png\" alt=\"analysis 20\" width=\"744\" height=\"275\" /></p>\r\n<h3></h3>\r\n<h3><span>Getting Started</span></h3>\r\n<p></p>\r\n<h3><span>Pathway Analysis </span><b>Exercises</b></h3>\r\n<p><span>This exercise is to check that you understand the results of Pathway Analysis.</span></p>\r\n<p></p>\r\n<p><span>On the Home page, open the Pathway Browser. Click on the Analyse Data button. When the submission form appears, in the Analysis Tools section select Click here to paste your data or try example data sets</span></p>\r\n<p><span>and click on the UniProt accession list button on the right side. </span></p>\r\n<p></p>\r\n<p><span>Your submission page should look like this:</span></p>\r\n<p><span><img src=\"https://lh6.googleusercontent.com/2lQJh7d9K2spKGan4__i9FAR6uGF_E95v3vOkzXHxKmdklOV5LhdUHLYwaavTnaIo-zwL4sx9YgeEwK-BzhqfLQ_cVHE14Lz-u0j2-VFqaAVzJHtvMan-WbgJ1s0ZIAUGXCnM8IBrfuBx0YIEQ\" alt=\"\" width=\"529\" height=\"342\" /></span></p>\r\n<p></p>\r\n<p><span>Click Continue. Make sure that Project to human is checked, but Include interactors is not. &nbsp;When the results appear:</span></p>\r\n<p></p>\r\n<ol>\r\n<li><span>What pathway(s) is (are) most over-represented in this dataset? </span></li>\r\n<li><span>How many IDs match the pathway at the top of the list?</span></li>\r\n<li><span>Roughly what proportion of molecules in the pathway are matched?</span></li>\r\n<li><span>What proportion of reactions are matched?</span></li>\r\n<li><span>Repeat the analysis but this time check the box to Include interactors. Are the results the same? If not, why?</span></li>\r\n</ol>\r\n<h3></h3>\r\n<h3><span>Expression Analysis </span><b>Exercises</b></h3>\r\n<p><span>This exercise is to check that you can run and interpret the results of Expression Analysis.</span></p>\r\n<p><span>Launch Expression Analysis and load the example dataset. Click GO When the results are displayed, find the pathway DNA Repair, </span></p>\r\n<ol>\r\n<li><span>How many proteins are in this pathway?</span></li>\r\n<li><span>What proportion of these had expression data?</span></li>\r\n<li><span>What was the average expression at 24h?</span></li>\r\n<li><span>Which subpathway has the highest average expression value at 24h? (Hint: move your mouse pointer over the pathway overview diagram, and look at the scale on the right side). </span></li>\r\n</ol>\r\n<p></p>\r\n<h3><span>Species Comparison </span><b>Exercises</b></h3>\r\n<p><span>This exercise is to check that you can run and interpret Species Comparison results.</span></p>\r\n<p><span>Launch Species Comparison and select the species Danio rerio. When the results are displayed, use the Pathway Hierarchy to Navigate to Hemostasis, Dissolution of fibrin clot. &nbsp;</span></p>\r\n<p></p>\r\n<ol>\r\n<li><span>Find PLAT(36-562) top-right of the diagram - what colour is it and why? </span></li>\r\n<li><span>Find HRG - what colour is it and why?</span></li>\r\n<li><span>Why is Zn2+ (upper left) uncoloured?</span></li>\r\n</ol>\r\n<p></p>\r\n<h3><b>More information</b></h3>\r\n<p></p>\r\n<h3><b>Molecular Interaction Overlay</b></h3>\r\n<p></p>\r\n<p><span>If the user is interested in analyzing the data with interactors from other sources (like IntAct), the Include Interactors options must be selected in the Analyze Data section. Once in the pathway diagram, the Molecular Interaction (MI) overlay allows protein-protein or protein-small molecule interactions to be superimposed onto the pathway diagram. The source depends on the currently selected interaction database. The default interaction database is IntAct (Static), which provides fast access to a quarterly updated and locally-hosted version of IntAct data. Access to the IntAct-hosted dataset and other PSICQUIC sources of interaction data (protein-protein and protein-small molecule) can be selected using the PSICQUIC feature of the Interactor Overlays tab within the Settings panel in the middle-right of the Pathway diagram. A list allows selection of the source of interactors. This is automatically populated by querying the PSICQUIC Registry. If a new database is selected while interactors are displayed on the pathway diagram, the proteins represented by those entities will be used as queries in the new database and the display will automatically update. An i (information) button to the top right of the Interactor Overlays tab provides a quick tip about using the MI overlay. Selecting either Cyan or Teal in the Interactors Colour Profile: drop-down menu within the Colour Palette feature of the Settings panel will change the interactor colour.</span></p>\r\n<p><span><img src=\"https://lh5.googleusercontent.com/ucfQbIyTH8Md6LMVlVjE306-gHLmlga8_96TB3h2ab57bwB0eQUVPWNeTXVCVl7wxXTp7f3gm8iLXhlAaNM9fi3lPTxn4Icx4Rs52aKcOdwcZgWngNUrE2rpL44zfOFhZcWPTHOMbvirk0ayMg\" alt=\"\" width=\"307\" height=\"311\" /></span></p>\r\n<p><span>Once a pathway diagram is displayed in the viewport, a small red circle with a white-lettered number in the top right corner of the entity will automatically appear for individual protein and chemical entities that have interactors. The number represents the number of known interactors for the given protein. Pressing the red circle with the cursor will display the interactors. A maximum of 10 interactors is displayed as a ring of blue-bordered boxes (protein interactors) or green-bordered ovals (small molecule interactors) connected by black lines to the selected protein. If the same interactor is connected to more than one entity it is re-used, i.e. connected to all the selected entities in the diagram. Furthermore, if the interactor is a protein or small molecule entity that pre-exists within the pathway diagram, a black line connects the two entities. Pressing the red circle a second time will remove the interactors from the pathway diagram. If an entity interacts with itself, a looping arrow is shown.</span></p>\r\n<p><span><img src=\"https://lh6.googleusercontent.com/lfEz98F3u2gTHU4yNzpPerSnazU218ci24W5aGQv85vXqkB-wk3yVDcD2zKEK61G76l3oHzz3vpFWywdvNu05hJmf6L4tX35DIzKQQoNtAgreaWpJaB44jqyL2EPmS9deXKR6KnU_0x9NxDauA\" alt=\"\" width=\"647\" height=\"427\" /></span></p>\r\n<p><span>A number of actions provide additional information about the interactor and its relationship to the protein entity. These include:</span></p>\r\n<ul>\r\n<li><span>Hover the mouse pointer over a protein interactor to reveal a tool-tip containing its name and UniProt identifier.</span></li>\r\n<li><span>Click an interactor to open its database entry in a new window.</span></li>\r\n<li><span>Click the line that connects the interactor to the pathway molecule to open details of the interaction from the source database, in a new window.</span></li>\r\n<li><span>Right-click a pathway object, or select the blue triangle that appears on the right side when hovering the mouse pointer over it, to open a pop-up panel (the Contextual Information Panel). Select the Interactors (bottom) tab to display a table of all interactors for the selected object. This table is scrollable and provides additional information about the interactors. Click the Interactor name or accession identifier to connect to UniProt or the interaction source database, respectively. Click the id button to toggle the display of either the interactor name or its database identifier. Click the pin button to fix the open table in position on the pathway diagram. Click the X in the top right corner to close the table view.</span></li>\r\n</ul>\r\n<p><span><img src=\"https://lh5.googleusercontent.com/jaq_yPXgVcWvbLnCUnl6-WTBFc5KK5DpPCTCGMnJnzOQwQlupNnIFeh4Eeg6xfKW8KApz9fEMC34zjuMDnMDQMbfEFK-zP8yLXngkSstk4aBbhsIiUpInsMqsqFXqX_3Rb93yQpFEfw4Kdxq8A\" alt=\"\" width=\"664\" height=\"391\" /></span></p>\r\n<p><span>Interactions from the IntAct database have confidence scores that are generated as the cumulated sum of a weighted score that depends on the interaction detection method and the number of observations. The default lower threshold is 0.45. Interactions with a confidence level equal to or above this are shown. Move the slider bar in the Molecular Overlay toolbar (light grey bar with dark grey slider found at the bottom of the pathway diagram) to the right to raise the confidence level threshold. Interactors below the displayed confidence level will progressively disappear. Move the slider to the left and the interactors will reappear. Click the X in the Molecular Overlay toolbar to hide it.</span></p>\r\n<p><span><img src=\"https://lh3.googleusercontent.com/Gq_Obzb2aJ01CCdv_1SCNJnYTsenQq6-NWPsxxNqCcpAHDeeFSTqPtkXDZFnqRL4rtPFy8x7R2PB3uFxefEmKqGHyAFK1x8GTPj3hLNhS6qenlCxv_yRfzkDlW7jIH6zei_z565iYTIS3pMQFQ\" alt=\"\" width=\"400\" height=\"55\" /></span></p>\r\n<p><span>Zooming, either by using the controls in the bottom right corner of the Pathway Diagram or the mouse scroll wheel, increases the size of all objects displayed in the pathway diagram including interactors. When a sufficient size is reached, the crystal structure of proteins or chemical structure of small molecules will appear in the interactor object, if available.</span></p>\r\n<p><span><img src=\"https://lh3.googleusercontent.com/cb__e6qEODh6LN2f9BM_6cGH2V3WTcu2LNiTrS0LPL1OMXqmjSmeWh9R-K7PaeYffboFhaCXsqmfnqvbvhCDdf95cX5IHXP8Aa3LEabKi9LMRNRappmiCiX_eG-Wp15gfPjbNhgoLfA1pNIvPw\" alt=\"\" width=\"672\" height=\"456\" /></span></p>\r\n<p><span>The details of every interactor for every protein in the displayed pathway can be downloaded as a tab-delimited file by clicking the Cloud button on the Molecular Overlay toolbar or from the Interactor Overlays tab in the Settings panel.</span></p>\r\n<p><span></span></p>\r\n<h3><span>Molecular Interaction Overlay </span><b>Exercises</b></h3>\r\n<p></p>\r\n<p><span>This exercise is to check that you can use and interpret Molecular Interaction Overlays</span></p>\r\n<p></p>\r\n<p><span>Open the Pathway Diagram for Netrin-1 Signaling.</span></p>\r\n<ol>\r\n<li><span>Find the protein NEO1 (top left of the cytosol). How many interactors does it have? </span></li>\r\n<li><span>What is the confidence score for the interaction between NEO1 and GD3? How many times has this interaction been documented? Hint: This detail is not in Reactome.</span></li>\r\n<li><span>Find the protein PTPN11 (below and to the right of NEO1). How many interactors does it have? Display interactors for this protein. How many are there? Can you get a list of all of them?</span></li>\r\n<li><span>What is the easiest way to remove interactors?</span></li>\r\n</ol>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:51:54','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(85,699,'Diseases','diseases','<p><span >Reactome can annotate and display pathways associated with disease. </span><span >Reactome disease annotations include cancer, metabolic and immune disease as well as infectious diseases, among others. Where possible, Reactome disease pathways also include the interaction of relevant therapeutic drugs. (Disease pathways are contained in a separate top-level Chapter in the hierarchy, and are designated with a red + symbol to the left of the pathway name.)</span></p>\r\n<p><span >Pathway Diagrams:</span></p>\r\n<p><span >Disease events or processes </span><span >are shown in the context of the normal pathway diagram </span><span >to provide context to the disease event. </span></p>\r\n<p><span >Disease events have red connecting lines; abnormal molecules involved in these processes are outlined in red.</span></p>\r\n<p><span >For infection processes, events involved in the life cycle of the infecting agent and host interactions with the infecting agent have red connecting lines. Molecules derived from the infecting agent are outlined in red, while those derived from the host are outlined as normal in black.</span></p>\r\n<p><span >Disease events may be loss-of-function or gain-of-function compared to the normal process. </span></p>\r\n<p><span >For loss-of-function events (where a protein has lost all or most of its functional activity) the disease reaction is automatically overlaid on top of the corresponding normal reaction. The loss-of-function disease entity is outlined with a dashed red line, and products that are no longer made are greyed out with a superimposed red cross. These loss-of-function events represent stop points in the pathway.</span></p>\r\n<p><span ><img src=\"https://lh3.googleusercontent.com/QQpMa9wCdb-jFFAi6O9w-QZ-fzDDmaal8O6EriwxLmV0aL7BCZnuCvRbqpGtZX9YW57GxL_5vn2O56j6yFMfW6xvvGFSHRxk876wU8dSmRCTkrs-fg2mbnzaCDv-Uksj2zoWhCuWl29AQfCLfQ\" alt=\"\" width=\"486\" height=\"296\" /></span></p>\r\n<p></p>\r\n<p><span >For gain-of-function events (where a protein has acquired a novel function not performed by the wild-type protein), disease events are represented alongside the normal events. </span></p>\r\n<p><span ><img src=\"https://lh3.googleusercontent.com/UpyAnX4H1L3Ly2XTp2SGfSwVRNGsq0OSSb72UFBpXSWxuAFOT0GjZ70dIfJbcsEDJ1ZC7kyR4bShnFSavQB9PAEl4G8w3LWCHSr6ql3bTmYH7NNuNWMuAEPcyvHlBlD082AOB897\" alt=\"Screen Shot 2017-07-19 at 4.08.24 PM.png\" width=\"579\" height=\"413\" /></span></p>\r\n<p><span >When a gain-of-function mutant performs a normal function (either a higher rate or efficiency, or after being activated in an novel manner, for instance), these disease events are overlaid on the corresponding normal event in the pathway diagram.</span></p>\r\n<p><span ><img src=\"https://lh6.googleusercontent.com/6tXYCq69mNJABE4w9TPXpulZoLQhK5PfwhLY0zfmggznxCd-W7lNdgJh4-CNGJ0IcKaRrhrxE-rCyoPHfW8VK-IhxwkUOJXTU7xoMkTvyK5ODXUCMKjY00ObvDo2kIhzoGXHAd7s\" alt=\"Screen Shot 2017-07-20 at 10.25.00 AM.png\" width=\"599\" height=\"364\" /></span></p>\r\n<p><span >Infectious disease events (which dont occur in the absence of the infectious agent and therefore have no normal counterpart) have their own diagrams.</span></p>\r\n<p><span ><img src=\"https://lh3.googleusercontent.com/neErCBw-DAwj7o1Ki7y7ukBTNx-uX_nUVZsLStcq3gz9SfmPKDuEmdpD36gdtjyrbmS5sYgC6GnuP4lYwBWWC7uNCrsYKO-Lr5SemRyv1bM9bZqQJDee7SkJTzI1m0AOk7-c3C-1SMa0BWi_bg\" alt=\"\" width=\"465\" height=\"336\" /></span></p>\r\n<p><span >Drug annotations:</span></p>\r\n<p><span >Where applicable, the effect of drugs on disease pathways is annotated.</span><span ><img src=\"https://lh6.googleusercontent.com/Yz1nbEOLWJDAaHbkzGeZsSSdcVo7L8FC8EqPk4neu0NhsX8gsBNcllNxyRV8Cy24geDo139GiTfxKDyeXUrNKo2sxJ6u4WaA4O_hJinpMZDf_12DUrYhcK8TedXFKMXdNmiL7iMm\" alt=\"Screen Shot 2017-07-20 at 11.55.30 AM.png\" width=\"624\" height=\"447\" /></span></p>\r\n<p></p>\r\n<p><span >Details panel:</span></p>\r\n<p></p>\r\n<p><span >The details panel for a disease pathway/event has a number of extra panels in addition to those displayed in a normal pathway/event.</span></p>\r\n<p></p>\r\n<p><span >Disease tag: </span></p>\r\n<p><span >Disease events and entities are tagged with a disease term, taken from the </span><a href=\"http://www.disease-ontology.org/\"><span >Disease Ontology</span></a><span > and displayed in the Disease panel. </span></p>\r\n<p><span >Individual proteins are labeled with all the relevant disease terms, while sets of disease entities are tagged with the most specific term that represents all of the set members. </span></p>\r\n<p><span ><img src=\"https://lh4.googleusercontent.com/ORDsTaNuzOLlhGKs3pVIhM1cK9E-zyaj3GTn0JHMjpDdX0zs7hKseQp9OOZQV6KcTdedfHS2ukal3qx6CmZdtbmU9kWqOChAFQOMMScLj-hJrTOSGEm74dLfVgePsHwOmprpjayd\" alt=\"Screen Shot 2017-07-19 at 4.26.47 PM.png\" width=\"624\" height=\"132\" /></span></p>\r\n<p><span >Where possible, disease proteins are linked to </span><a href=\"https://www.omim.org/\"><span >OMIM</span></a><span > (Online Inheritance in Man),</span></p>\r\n<p><span ><img src=\"https://lh5.googleusercontent.com/hwzNr5aC4lb0fuXwvksBeYid1oY5c-l6DYFloUfKjq6l0jBVLa_ItV2lAql4tamw5agSVpjmkponL909syIBW-7jdgDZKbKt8hk95MwctEsgZJSJkTycfTXEZDjWpp03_9j_GItn\" alt=\"Screen Shot 2017-07-20 at 10.17.08 AM.png\" width=\"624\" height=\"216\" /></span></p>\r\n<p></p>\r\n<p><span >and cancer variants are linked to </span><a href=\"http://cancer.sanger.ac.uk/cosmic\"><span >COSMIC</span></a><span > (Catalogue of Somatic Mutations in Cancer).</span></p>\r\n<p><span ><img src=\"https://lh5.googleusercontent.com/NOLO0HEQFpq-BKygW-wM-AzfIpNQH1dzGqcbMGiXeZJ8aP0yBBI8igE7ghJwNr3pLo1AdU0M_cSErilZ2CwzPV3Al21T3RyCH60NqvvG92hjDMsd-fzC8cpxVjWhY6oRV95fpT_x\" alt=\"Screen Shot 2017-07-19 at 4.28.09 PM.png\" width=\"624\" height=\"91\" /></span></p>\r\n<p></p>\r\n<p><span >Functional status:</span></p>\r\n<p><span >As described above, disease events are tagged as either gain- or loss-of-function, and this is displayed in the Functional Status panel.</span></p>\r\n<p><span ><img src=\"https://lh6.googleusercontent.com/7dCeUgEph3DuN5gvYWKG4dm9yl0GQkWaTjbXWuoVu6hSwiNBZU1PJTXqrTkF9sNRDJIBoTwqdfJfbViH26xLR8BERy3wYF9Lj_eSvaXdtp5xt417ofC7dRciZ98q7lYcSWHT2KWp\" alt=\"Screen Shot 2017-07-19 at 4.29.28 PM.png\" width=\"624\" height=\"183\" /></span></p>\r\n<p></p>\r\n<p><span >Normal Pathway/Reaction:</span></p>\r\n<p><span >Where applicable, the corresponding normal pathway or reaction is identified in the details panel of the disease event. </span></p>\r\n<p><span ><img src=\"https://lh5.googleusercontent.com/FeceEhlHE21doXqjRo0idbLFJSmjh5fBxSlw-3fXPYb5lAj8qrh_stJl6XFmX-il4px3qUnSRYvxBadD3nCQ1b5Txy7gcEazIQ2Cr6h3ZAf3qn5NycA5BpchSTKp_FVch0XvtD7J\" alt=\"Screen Shot 2017-07-19 at 4.34.53 PM.png\" width=\"624\" height=\"61\" /></span></p>\r\n<p></p>\r\n<p><span >(description of how to switch back and forth from disease to normal, once I remember how it is meant to work).</span></p>\r\n<p><span >Disease entities:</span></p>\r\n<p><span >Disease entities are annotated in molecular detail. Changes to protein sequence that arise as a result of variation in DNA sequence in disease are displayed in the details panel when a disease entity is highlighted in the diagram. Missense, nonsense, frameshift, deletion and fusion mutations are all annotated. See More info, below, for details of these annotations. </span></p>\r\n<p><span >Primary references describing the identification of the mutants proteins are annotated, but this information is not currently displayed on the website. Note also that on the website, these genetic modifications are displayed in the same panel as post-translational modifications such as phosphorylations, and are therefore (mis)labeled as Post-translational modifications.</span></p>\r\n<p></p>\r\n<p><span >Missense mutation:</span></p>\r\n<p><span ><img src=\"https://lh5.googleusercontent.com/Ww805yicGSIvzlmVds7PnLWIidLRU2H0Zl-RWGrLn-V2zjQO4LiwyWBQRXcf1UT7iLeY5JYaT22bVmT6R2CWSCWgCUUf-cPDll4q15EHwGCFFWPjS1NDmVCS_TCMB-noQBAjvoUZ\" alt=\"Screen Shot 2017-07-20 at 10.54.40 AM.png\" width=\"624\" height=\"104\" /></span></p>\r\n<p></p>\r\n<p><span >Deletion mutation:</span></p>\r\n<p><span ><img src=\"https://lh4.googleusercontent.com/gFl_5VQgGz-GjdE2cwQnU4TIwDk3vQQNQ-5ZH2OxSdqm51OUEv-JqxN5BnbkEkkFI7BaUfFhIIR_V3LDF0Pe-JQ3sjCd3e8iGvtMrzc_mY5Ef4N24HLrCfL6v2J9Dl5bJJ5pPdVM\" alt=\"Screen Shot 2017-07-20 at 10.55.35 AM.png\" width=\"624\" height=\"151\" /></span></p>\r\n<p></p>\r\n<p><span >Drugs:</span></p>\r\n<p><span > Drugs are cross-referenced to </span><a href=\"https://www.ebi.ac.uk/chebi/\"><span >ChEBI</span></a><span > and to </span><a href=\"http://www.iuphar.org/\"><span >IUPHAR</span></a><span > where possible and applicable, and this information is displayed in the details panel when a drug is highlighted in the diagram.</span></p>\r\n<p></p>\r\n<p><span >Get started:</span></p>\r\n<p></p>\r\n<p><b>Example 1 - drug-target interaction in disease</b><span >:</span></p>\r\n<p><span >The cystic fibrosis transmembrane conductance regulator (CFTR) is a low conductance chloride-selective channel that mediates the transport of chloride ions in human airway epithelial cells. Chloride ions plays a key role in maintaining homeostasis of epithelial secretions in the lungs. Defects in CFTR can cause cystic fibrosis (CF), resulting in an ionic imbalance that impairs clearance of secretions, not only in the lung, but also in the pancreas, gastrointestinal tract and liver. More than 1500 mutations in the CFTR gene have been identified. </span></p>\r\n<p></p>\r\n<p><span >Gain-of-function events involving CFTR F508del:</span></p>\r\n<p><span >Deletion of phenylalanine 508 in CFTR is the most prevalent mutation causing cystic fibrosis. F508 deletion causes destabilization and subsequent targeting for co-translational degradation by the ER-associated degradation machinery (ERAD). F508del is ubiquitinated by ERAD-associated E3 ligases including RNF5 and RNF185, targeting it for VCP-mediated retrotranslocation and 26S proteasomal degradation. &nbsp;These events are novel for the mutant protein and are represented in the disease diagram ABC transporter disorders.</span></p>\r\n<p></p>\r\n<p><span ><img src=\"https://lh3.googleusercontent.com/GaKhKcPreMQYJKoI9O4HZ0BwbbSj3T3EDOWU5Z0jSMHiW4PwZJe3Aw-kZ5f_JJfYQ8UAbVDWl95cxPTMkNw08pA0OVR_87AQ2wg9iG0opUOEc9LdpIKYHAERnUcp_0DGJb_fdQwr\" alt=\"Screen Shot 2017-07-20 at 4.30.29 PM.png\" width=\"624\" height=\"401\" /></span></p>\r\n<p></p>\r\n<p><span >Loss-of-function events for CFTR mutants:</span></p>\r\n<p><span >Some loss-of-function CFTR mutants are properly transported to the plasma membrane but are unable to transport chloride ions to the extracellular space. &nbsp;These are represented as a loss-of-function event in an overlay of the corresponding WT reaction, and displayed in the ABC transporter disorders disease diagram.</span></p>\r\n<p><span ><img src=\"https://lh6.googleusercontent.com/ZeWg974VqhHybmsaV3IysWQ7dkVoPgV77R7IgK4s81wPZQ8Hp10_2Yp-F7ffuWuA3jgUFXIF4lGXJUhDtOOY82ByCXz-ANOSimlLAa3bbhvqZG_X4URLFNY-EqS_p3at2XNf2XZz\" alt=\"Screen Shot 2017-07-20 at 4.39.41 PM.png\" width=\"624\" height=\"443\" /></span></p>\r\n<p></p>\r\n<p><span >Drug interactions in Cystic Fibrosis:</span></p>\r\n<p><span >CF patients with a particular mutation, G551D, have shown lung function improvements when given the drug Ivacaftor. Reactions showing ivacaftor binding the mutant protein (top right, above) and the following reaction (centre) showing channel functionality restored are displayed in the disease diagram.</span></p>\r\n<p></p>\r\n<p><b>Example 2 - diseases caused by accumulation of substrate over time</b><span >:</span></p>\r\n<p><span >Special case in Reactome where the reaction describing a particular substrate\'s metabolism occurs in normal physiology but over time, accumulation of the substrate leads to toxic consequences which can lead to disease. Examples are neurodegenerative diseases such as Alzheimer\'s and Parkinson\'s diseases, chronic obstructive pulmonary disease and retinal macular degeneration.</span></p>\r\n<p><span >The bisretinoid A2E (di-retinoid-pyridinium-ethanolamine) is a major component of lipofuschin, a yellow-brown pigment grain composed mainly of lipids but also sugars and certain metals whose accumulation is associated with degenerative diseases. In the eye, A2E is the end-product of the condensation of 2 molecules of all-trans-retinal and phosphatidylethanolamine in photoreceptor outer disc membranes. Once formed, A2E is phagocytosed, together with outer segments, to retinal pigment epithelial (RPE) cells where it accumulates. There is no evidence as yet to indicate that A2E can be catabolised. </span></p>\r\n<p><span >The relationship between lipofuscin accumulation and retinal degeneration is illustrated by Stargardt disease type 1. Because the reactions can be considered \"normal\" (they occur as part of the normal metabolism of retinoids) as well as disease-causing, they appear in the normal pathway diagram (Visual phototransduction) and coloured red. </span></p>\r\n<p><span ><img src=\"https://lh3.googleusercontent.com/rMS6cZ3XWezO_q_cBMCT9tHrh0xeOkv9X0X9Kc9CzvmKD15Ik-BCVPcVDPADhIlU-diS6Brh2p-CKwbVZm-0siVnGDxYl0oGAsVma0KLX1zzyX1M4Arh4rVa6bkBchTFJORIxLWo\" alt=\"Screen Shot 2017-07-21 at 10.02.59 AM.png\" width=\"624\" height=\"423\" /></span></p>\r\n<p></p>\r\n<p><span >To display in the disease view of the diagram, these reactions are added as components of the disease pathway \"Diseases associated with visual transduction\". These reactions will now display in both normal and disease diagrams.</span></p>\r\n<p><span ><img src=\"https://lh5.googleusercontent.com/POGdWDmAwxyRdFGTc7mdLDs48GDRpZcxzO_0P5dufaQTgdXxbYrajNyT94NQw7Nwevr6WD_DsO1FFaUCreBZP7E5DvhHiUhgjYTZsa2E_LiOR2ksq9ESRnvA961Iv2o5oiq4TVCb\" alt=\"Screen Shot 2017-07-21 at 10.04.48 AM.png\" width=\"624\" height=\"444\" /></span></p>\r\n<h3><b>Getting Started</b></h3>\r\n<h3><b>Exercises:</b></h3>\r\n<p></p>\r\n<ol>\r\n<li ><span >Search for the pathway FGFR1 mutant receptor activation and open it. What disease(s) is/are associated with this pathway? (</span><i><span >Hint: look at the Description tab in the Details section</span></i><span >). </span></li>\r\n<li ><span >Search for the reaction Defective MMADHC does not bind MMACHC:B12r. What type of defect has caused this loss of function? (</span><i><span >Hint: look at the Functional Status category for the reaction</span></i><span >).</span></li>\r\n<li ><span >How many mutations of MMADHC are represented?</span></li>\r\n</ol>\r\n<p></p>\r\n<p><span >More info:</span></p>\r\n<p><b>Annotating genetic alteration of protein sequence for disease pathways:</b></p>\r\n<p><span >The following image shows the portion of the Reactome data model describing the relationship between possible modifications to protein sequence. The subclass GeneticallyModifiedResidue is used for the annotation of disease entities, while TranslationalModification is used to annotate the consequence of processes such as phosphorylation, acylation, cross-linking and other similar non-genetic events. TranslationalModifications will not be described further here.</span></p>\r\n<p></p>\r\n<p><span ><img src=\"https://lh4.googleusercontent.com/sA8jAiu081GEy5CAdD1qM-LhoEGA1nVIB9MXoTq4uNLbjwHrmg0l0-BdqmTk69sT5Znn26QLEuxBY0qbSjd2e6BR2fcAq2Fp23GylHf4Ie9Dcre-qDR5O-ZyCCmELPdpFSdBV7sO\" alt=\"Screen Shot 2017-07-21 at 10.42.28 AM.png\" width=\"351\" height=\"249\" /></span></p>\r\n<p></p>\r\n<p><span >The ReplacedResidue class is used for amino-acid substitutions. &nbsp;This class is also used for simple nonsense mutations that change a coding amino acid for a stop codon.</span></p>\r\n<p><span >The FragmentModification class describes more extensive changes to the coding sequence through insertions and deletions in the gene, and includes three subclasses:</span></p>\r\n<ul>\r\n<li ><span >FragmentDeletionModification is used for in-frame deletions of amino-acids</span></li>\r\n<li ><span >FragmentInsertionModification is used for in-frame insertions of amino-acids, including genomic events that result in fusion proteins</span></li>\r\n<li ><span >FragmentReplacedModification is used for frameshifts.</span></li>\r\n</ul>\r\n<p></p>\r\n<p><span >Examples of each annotation type are further described below.</span></p>\r\n<p></p>\r\n<p><b>Simple missense mutation: HHAT G287V</b></p>\r\n<p><span >Hedgehog (Hh) is a secreted morphogen that regulates a number of developmental processes in vertebrates, including limb development and neural tube patterning, among others. &nbsp;Maturation of Hh ligand includes a number of proteolytic processing and lipid modification steps. These modifications are required for normal transit of the ligand to the surface of the secreting cell and mutations that affect these processes are associated with decreased Hh ligand secretion, abrogated Hh signaling and disease. &nbsp;</span></p>\r\n<p></p>\r\n<p><span >HHAT is an O-acyltransferase that palmitoylates the N-terminal fragment of Hh. A G287V loss-of-function mutation in HHAT was identified in a rare case of Syndromic 46 XY Disorder of Sex Development, which results in testis dysgenesis. This mutant is not able to palmitoylate the Hh ligand. Details of the amino-acid substitution are displayed in the details panel when the entity is highlighted in the diagram:</span></p>\r\n<p><span ><img src=\"https://lh6.googleusercontent.com/Ejv39liBq90P6sUu_nDZQI5O1Y0UWc3ipGjrBp5oeosWhasulxRj0hQO1sZv3w3zpnzsgaIcDpfr9IgR43sbwaAvtst5h4wBaQx1ANqsadv41_ETixcUvOg5n3frCqIZlXov0jmv\" alt=\"Screen Shot 2017-07-21 at 2.26.44 PM.png\" width=\"624\" height=\"368\" /></span></p>\r\n<p></p>\r\n<p><b>Simple nonsense mutation: </b></p>\r\n<p><span >The disorder Ehlers-Danlos syndrome, musculocontractural type 1 (EDSMC1) is caused by loss-of-function mutations in the carbohydrate sulfotransferase 14 (CHST14) gene. A nonsense mutation causing this disorder is </span><span >a 205A-T transversion in the CHST14 gene, resulting in a lys69-to-ter (K69*) substitution.</span><span > This is represented in the details panel as L-lysine 69 replaced with unknown.</span></p>\r\n<p><span ><img src=\"https://lh5.googleusercontent.com/zOgggk2__8z48ZUcp3M2wMM78lIT3iorQ-cM-ALpU8OGRX796A2MX1dhwv4ANa_SkivQWUyXSuUE-QBLhR8cJueXjaab5hiJTC-XF5A78eSfDhmfXeXZducY5sXhx_TR_JJ4GJ2Y\" alt=\"Screen Shot 2017-07-21 at 2.33.20 PM.png\" width=\"624\" height=\"479\" /></span></p>\r\n<p></p>\r\n<p><b>FragmentDeletionModification:</b></p>\r\n<p><span >This class is used for in-frame deletions of amino-acids leading to internally truncated proteins. These variants are named core protein name [first amino acid of deletion_last amino acid of deletion]del, as shown below for PIK3R1 Y463_L466del, which has a deletion of residues Y463 to L466, inclusive:</span></p>\r\n<p><span ><img src=\"https://lh4.googleusercontent.com/c_bOTttttccZNb372JHVABit5G4rZzlPoS5oNhlZVwY1bKwztVM-Lrl2t1ztPkKQx35fl3KqBGT2XipgAlSw8bzufmZ7TxdVuNPs1O4wvtDMGEav11Qs1aaKln7UG6D1GDajRD_3\" alt=\"Screen Shot 2017-07-21 at 2.40.02 PM.png\" width=\"624\" height=\"508\" /></span></p>\r\n<p></p>\r\n<p><b>FragmentInsertionModification</b></p>\r\n<p><span >This class is used for in-frame insertions of amino-acids and for fusion proteins.</span></p>\r\n<p><b>In-frame insertions:</b></p>\r\n<p><span >These variants are named &nbsp;&nbsp;core protein name [aa prior to insertion_aa following insertion]ins[inserted aas]. The amino-acid string of the inserted residues is added manually to the mutant protein name, as shown below for EGFR V738_K739insKIPVAI. This represents a variant where the amino acids 739-744 (KIPVAI) from EGFR are inserted at aa 739 of EGFR, and is thus a duplication of these residues: </span></p>\r\n<p></p>\r\n<p><span ><img src=\"https://lh4.googleusercontent.com/dMI0l9vzq9aSV1LHJnVU8bCU7yxc2EQZZ9vb2Me4THRFkVJI6LIYTOhJBaOBZJm3X3kSSpAtXbRlKlHUNQnu2tizFxH5bTVADONHPmp-V5BrUf12qic1eNDLQbi3thr_EQ0zODh3\" alt=\"Screen Shot 2017-07-21 at 2.57.20 PM.png\" width=\"624\" height=\"479\" /></span></p>\r\n<p></p>\r\n<p><b>Fusion proteins:</b></p>\r\n<p><span >The FragmentInsertionModification class is also used to annotate proteins that arise as the result of genomic changes that bring two genes together to result in a fusion protein, as in the ZMYM2-FGFR1 fusion described below. &nbsp;This fusion puts the ZMYM2 dimerization region (1-914) together with the kinase domain of the FGFR1 receptor (residues 429-822) and results in constitutive activation of the kinase domain by virtue of ligand-independent dimerization (for reference, the full length aa sequence of these two proteins are 1-1377 and 1-822 for ZMYM2 and FGFR1, respectively).</span></p>\r\n<p><span >By convention, the N-terminal most partner of the fusion is set as the reference protein for the variant protein while the C-terminal fusion partner sequence is captured in the FragmentInsertionModification record, as in the example below for the ZMYM2-FGFR1 fusion:</span></p>\r\n<p><span ><img src=\"https://lh5.googleusercontent.com/qR3RjZR00hjH64eJhgWtGJQaaEXGuRyBUj5rFZ-Zs0Ini27S2dgqB1gmqqvqw5mXpi2vEA3yezaCJ4eKZgeAbXPmfVrBWB1wqTfYYMpShvU8sGTUZvMy-nqRUBrWkxqMh7VKkjn7\" alt=\"Screen Shot 2017-07-21 at 3.04.28 PM.png\" width=\"624\" height=\"456\" /></span></p>\r\n<p></p>\r\n<p><span >Post-translational modifications to either partner in the fusion protein are numbered according to each respective WT reference gene product and do not reflect the aa position in the fusion. &nbsp;For instance, if in the context of the fusion protein, the FGFR1 partner is phosphorylated at (WT FGFR1 position) Y766, the fusion EWAS would be ZMYM2-pY766-FGFR1, despite the fact that in linear sequence the phosphorylation occurs at residue 1250 of the fusion (913+(766-429)). </span></p>\r\n<p></p>\r\n<p><b>FragmentReplacedModification</b></p>\r\n<p><span >This class is used to annotate frameshift mutations that alter the amino-acid sequence of the protein as in the case of EXT1, described below.</span></p>\r\n<p><span >The disorder Hereditary multiple exostoses 1 (EXT1) is caused by loss-of-function mutations in Exostosin 1 (EXT1). One such mutation is a 1-bp deletion at nucleotide 1469 in the EXT1 gene, resulting in a frameshift mutation with a premature stop codon nine amino acids downstream. Variants of this type are named core protein name</span><span > </span><span >[aa prior to insertion_fs*(number of aa to the first stop codon)]. The novel amino acids that occur as a result of the frameshift are identified in the mutant protein record, as shown below for EXT1 L490Rfs*9:</span></p>\r\n<p><span ><img src=\"https://lh3.googleusercontent.com/236RYyP9g8b4tF6NrUIqzBchLbyrHT3vHxsEASqA7L73eD29jr5JMN4uwAURNMvHHoOyi4N96bxkuLg9-2c4mxpHa-CY2qhqfBE9TCTi6BlVHY-tWHgyXLdqAKLOSNwPuLY0nwh7\" alt=\"Screen Shot 2017-07-21 at 3.23.53 PM.png\" width=\"624\" height=\"460\" /></span></p>\r\n<p><span >For cancer-related processes, it is useful to view the altered disease events alongside their normal counterparts in the same diagram. The user can then see where normal processes diverge into ones that are implicated in cancer.</span></p>\r\n<p></p>\r\n<p><span >For metabolic processes, proteins that have lost all or most of their functional activity towards a substrate causes the majority of defects. The diagram displays these disease reactions as \'stop points\' in the pathway. Defective enzyme catalysts are outlined by a red dashed line; products that are no longer made are shown greyed-out with a superimposed red cross.</span></p>\r\n<p><span ><img src=\"https://lh3.googleusercontent.com/kqPeoAgsq5nGZkF6Yg-qlYCKIFptFynhVC2bxyCzj6bjnOIgtdY4KoKD6FFOObSXaMwn9wLx8sNjwddUGJlCuYD36vqiKJM7mzUIyC_7VA_tVy4WDJ-UhhBJxOq7SCreD_v4Ae3lYIArXBzJVg\" alt=\"\" width=\"455\" height=\"205\" /></span></p>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:53:13','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(86,701,'Physical Entity Identifiers Mapping','pe-identifiers-mapping','<p>The mapping files consist of a tab-separated table that indicates which external protein, gene or small molecule identifiers in the source database were mapped to Reactome physical entities and reaction annotations. Our goal with distributing three sets of files for each different identifier type is to provide these mapping files link the source database identifier to:</p>\r\n<ul>\r\n<li style=\"list-style-type: lower-roman;\">the lowest level pathway diagram or subset of the pathway</li>\r\n<li style=\"list-style-type: lower-roman;\">all level pathway diagrams (filename appended with \'All_Levels),</li>\r\n<li style=\"list-style-type: lower-roman;\">all reaction events.</li>\r\n</ul>\r\n<p>The columns within the mappings files follow a similar format:</p>\r\n<ol>\r\n<li>Source database identifier, e.g. UniProt, ENSEMBL, NCBI Gene or ChEBI identifier</li>\r\n<li>Reactome Physical Entity Stable Identifier</li>\r\n<li>Reactome Physical Entity Name</li>\r\n<li>Reactome Pathway Stable identifier</li>\r\n<li>URL</li>\r\n<li>Event (Pathway or Reaction) Name</li>\r\n<li>Evidence Code</li>\r\n<li>Species</li>\r\n</ol>\r\n<p>If you have any further questions, please <a href=\"mailto:help@reactome.org?subject=PE%20Identifier%20Mapping%20File\">contact us</a>.</p>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-23 09:19:03','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(87,703,'Reaction PMIDs','reaction-pmids','<p>The \"Reaction PMIDs\" file comprises of a&nbsp;tab-separated table consisting of two columns that provide information about the research publications associated with Reactome pathways. The first column provides the pathway stable identifier, whereas the second column provides the PubMed citation identifier.</p>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-09-25 14:58:11','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(88,704,'Reactome to OMIM','reactome-to-omim','<p>The \"Reactome to OMIM\" file comprises of a&nbsp;tab-separated table provide supporting annotations to establish links from <a href=\"https://www.omim.org/\">OMIM</a>&nbsp;to Reactome pathways using isoform information in UniProt. The \"Reactome to OMIM\" file consists of three columns defining the pathways in the \"lower levels\" of the pathway hierarchy where the PhysicalEntity for the given UniProt accession can be found. The first column provides the UniProt accession (Acc), whereas the second and&nbsp;third column provides the pathway stable identifier (PathwayId), and the name of the pathway (Pathway), respectively.</p>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-09-25 14:58:50','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(89,705,'Protein Role Reaction','protein-role-reaction','<p>The \"Protein Role Reaction\" file consists of a tab-separated table comprising three columns that provides information about the role of a protein within a pathway. The first column provides the UniProt accession whereas the second and third columns provides the role of the protein within the reaction and the reaction stable identifier, respectively.</p>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-09-25 14:59:25','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(90,709,'Version 62 Released','version-62-released','<p style=\"text-align: justify;\"><strong><a href=\"/PathwayBrowser/#/R-HSA-445717\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-445717.svg\" alt=\"\" width=\"681\" height=\"400\" /></a></strong></p>\r\n<p><strong>New and Updated Pathways.</strong><strong>&nbsp;</strong>In version&nbsp;V62, topics with new or revised pathways include Developmental biology (<a href=\"/PathwayBrowser/#/R-HSA-376176\">Signaling by Robo receptor</a>), Gene expression &nbsp;(<a href=\"/PathwayBrowser/#/R-HSA-8953750\">Transcriptional regulation by E2F6&nbsp;</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-8878166\">Transcriptional regulation by RUNX2</a>), and Immune System (<a href=\"/PathwayBrowser/#/R-HSA-1266695\">Interleukin-7 family signaling</a>,<a href=\"/PathwayBrowser/#/R-HSA-8983432\">&nbsp;Interleukin-15&nbsp; family signaling</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-8984722\">Interleukin-35&nbsp; family signaling</a>, and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-9007892\">Interleukin-38&nbsp; family signaling</a>). Illustrations are now available for&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-445717\">Aquaporin-mediated transport</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-2559583\">Cellular senescence</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-212165\">Epigenetic regulation of gene expression</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-74160\">Gene Expression</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-5250941\">Negative epigenetic regulation of rRNA expression</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-1480926\">O2/CO2 exchange in erythrocytes</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-2980736\">Peptide hormone metabolism,</a>&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-5250913\">Positive epigenetic regulation of rRNA expression</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-597592\">Post-translational protein modification</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-5660526\">Response to metal ions</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-162582\">Signal Transduction</a>, and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-425407\">SLC-mediated transmembrane transport</a>.</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.&nbsp;</strong><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Patricia+Ducy%5BAuthor%5D\">Patricia Ducy</a>&nbsp;is our external author.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Patricia+Ducy%5BAuthor%5D\">Patricia Ducy</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jorg+J+Goronzy%5BAuthor%5D\">Jorg Goronzy</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Anna+Herlihy%5BAuthor%5D\">Anna Herlihy</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Alexander++Jaworski%5BAuthor%5D\">Alexander&nbsp; Jaworski</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Umesh+Kumar%5BAuthor%5D\">Umesh&nbsp; Kumar</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Mora%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26892022\">Javier Francisco Mora</a>,&nbsp;<a href=\"http://scholar.google.co.in/citations?user=MDttKOwAAAAJ&amp;hl=en\">Manoj Patidar</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Yuliya++Pylayeva-Gupta%5BAuthor%5D\">Yuliya&nbsp; Pylayeva-Gupta</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Kailash%20%20Singh%5BAuthor%5D\">Kailash Singh</a>, and Ren Sun are our external reviewers.</p>\r\n<p><b>Annotation Statistics.</b>&nbsp;Reactome comprises 11,302 human reactions organized into 2,176 pathways involving 10,878 proteins encoded by 10,712 different human genes, 1,768 small molecules, and 11,284 complexes. These annotations are supported by 27,526 literature references. We have projected these reactions onto 121,709 orthologous proteins, creating 20,854 orthologous pathways in 18 non-human species. Version 62 has annotations for 1,334 protein variants (mutated proteins) and their post-translationally modified forms, derived from 285 proteins. These have been used to annotate 506 disease-specific complexes and 906 disease-specific reactions organized into 453 pathways and subpathways, and tagged with 294 Disease Ontology terms.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-09-28 15:45:38','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,31,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(91,714,'Introducing our Logo','introducing-our-logo','<style type=\"text/css\">.pure-logo{\r\n    text-align: center;\r\n}\r\n.pure-logo img {\r\n    max-width: 50%;\r\n    padding: 15px;\r\n    margin: 10px;\r\n}\r\n\r\n.pure-logo img.imagotype {\r\n    max-width: 80%;\r\n}\r\n\r\n.pure-logo img.isotype {\r\n    padding: 3px;\r\n}\r\n\r\n.pure-logo img.negative {\r\n    background-color: rgba(47, 158, 194, 0.3) !important;\r\n}\r\n\r\n.intro-logo {\r\n   border: 1px solid #c8c8c8;\r\n    -webkit-border-radius: 10px;\r\n    -moz-border-radius: 10px;\r\n    border-radius: 10px;\r\n    margin: 15px 0;\r\n}\r\n\r\n.formats {\r\n    border-top: 1px solid #c8c8c8;\r\n    margin: 0 10px;\r\n}\r\n\r\n.png-download {\r\n    display: inline-block;\r\n}\r\n\r\n.png-download p a {\r\n    font-weight: bold;\r\n    color: white;\r\n}\r\n\r\n.png-download p a:focus, .png-download p a:hover {\r\n    font-weight: bold;\r\n    text-decoration: underline;\r\n    background-color: #FFF;\r\n    color: #2F9EC2;\r\n    opacity: 0.7;\r\n    -webkit-border-radius: 4px;\r\n    -moz-border-radius: 4px;\r\n    border-radius: 4px;\r\n    padding: 1px 3px 1px 3px;\r\n}\r\n\r\n.png-download .tooltip-sizes {\r\n    visibility: hidden;\r\n    background-color: #2F9EC2;\r\n    color: #fff;\r\n    text-align: center;\r\n    border-radius: 6px;\r\n    padding: 5px 0;\r\n\r\n    /* Position the tooltip */\r\n    position: absolute;\r\n    z-index: 1;\r\n\r\n    /* TOP */\r\n    width: 145px;\r\n    top: 100%;\r\n    /*left: 80%;*/\r\n    margin-left: -54px;\r\n\r\n    -webkit-transition: height .5s; /* For Safari 3.1 to 6.0 */\r\n    transition: height .5s;\r\n    opacity: 0;\r\n    height: 0;\r\n    overflow: hidden;\r\n}\r\n\r\n.png-download .tooltip-sizes p {\r\n    margin: 8px 2px;\r\n    font-size: 12px;\r\n}\r\n\r\n.png-download:hover .tooltip-sizes {\r\n    visibility: visible;\r\n    background-color: #2F9EC2;\r\n    opacity: 1;\r\n    height: 110px;\r\n}</style>\r\n<p>The new Reactome logo brings to the forefront the value and quality of the information and analysis tools that can be found in our curated database of pathways. The layering in our logo highlights the transparency of Reactomes nature while the rounded typography matches our openness.</p>\r\n<p>Derived from a shape that exists in nature and at the same time gives structure, the logo evokes the idea of discovering by unwrapping the layers of knowledge that surround biological events.</p>\r\n<h3><i class=\"fa fa fa-question-circle\" style=\"font-size: 23px;\"></i>How to use our logo</h3>\r\n<p>We invite you to use the brand (<a href=\"#imagotype\">imagotype</a>) or the logo (<a href=\"#isotype\">isotype</a>) as you decide. We suggest the usage of our image against a white background but we understand that sometimes it is just not possible, that is why you have as well the negative of our image, that fits nicely against a coloured background.</p>\r\n<div class=\"favth-row\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<h3><a href=\"#imagotype\" id=\"imagotype\"><i class=\"fa fa fa-file-image-o\" style=\"font-size: 23px;\"></i>Imagotype</a></h3>\r\n</div>\r\n</div>\r\n<div class=\"favth-row\">\r\n<div class=\"favth-col-lg-6 favth-col-md-6 favth-col-sm-6 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"intro-logo favth-clearfix\">\r\n<div class=\"pure-logo\">\r\n<div><img src=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive.svg\" alt=\"Reactome Logo\" class=\"imagotype\" title=\"Reactome Logo\" /></div>\r\n<div class=\"formats favth-clearfix\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-right\">\r\n<div class=\"png-download\"><a>PNG</a>\r\n<div class=\"tooltip-sizes\">\r\n<p><a href=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive_25mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Positive_25mm.png\">Small [297x70]</a></p>\r\n<p><a href=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive_50mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Positive_50mm.png\">Medium [591x138]</a></p>\r\n<p><a href=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive_100mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Positive_100mm.png\">Large [1183x276]</a></p>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix\"><a href=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive.svg\" title=\"Click to Download our logo in SVG format\" download=\"Reactome_Imagotype_Positive.svg\">SVG</a></div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-left\"><a href=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive.emf\" title=\"Click to Download our logo in EMF format\" download=\"Reactome_Imagotype_Positive.emf\">EMF</a></div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-6 favth-col-md-6 favth-col-sm-6 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"intro-logo favth-clearfix\">\r\n<div class=\"pure-logo\">\r\n<div><img src=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative.svg\" alt=\"Reactome Logo\" class=\"imagotype negative\" title=\"Reactome Logo (Transparent Background)\" /></div>\r\n<div class=\"formats favth-clearfix\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-right\">\r\n<div class=\"png-download\"><a>PNG</a>\r\n<div class=\"tooltip-sizes\">\r\n<p><a href=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative_25mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Negative_25mm.png\">Small [297x70]</a></p>\r\n<p><a href=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative_50mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Negative_50mm.png\">Medium [591x138]</a></p>\r\n<p><a href=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative_100mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Negative_100mm.png\">Large [1183x276]</a></p>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix \"><a href=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative.svg\" title=\"Click to Download our logo in SVG format\" download=\"Reactome_Imagotype_Negative.svg\">SVG</a></div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix  favth-text-left\"><a href=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative.emf\" title=\"Click to Download our logo in EMF format\" download=\"Reactome_Imagotype_Negative.emf\">EMF</a></div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-row\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"padding0 top30\"></div>\r\n</div>\r\n</div>\r\n<div class=\"favth-row\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<h3><a href=\"#isotype\" id=\"isotype\"><i class=\"fa fa-file-image-o\" style=\"font-size: 23px;\"></i>Isotype</a></h3>\r\n</div>\r\n</div>\r\n<div class=\"favth-row\">\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"intro-logo favth-clearfix\">\r\n<div class=\"pure-logo\">\r\n<div><img src=\"/images/logo/isotype/positive/Reactome_Isotype_Positive.svg\" alt=\"Reactome Logo (Isotype)\" width=\"80\" height=\"89\" class=\"isotype\" /></div>\r\n<div class=\"formats favth-clearfix\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-right\">\r\n<div class=\"png-download\"><a>PNG</a>\r\n<div class=\"tooltip-sizes\">\r\n<p><a href=\"/images/logo/isotype/positive/Reactome_Isotype_Positive_10mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Positive_25mm.png\">Small [119x138]</a></p>\r\n<p><a href=\"/images/logo/isotype/positive/Reactome_Isotype_Positive_25mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Positive_50mm.png\">Medium [297x342]</a></p>\r\n<p><a href=\"/images/logo/isotype/positive/Reactome_Isotype_Positive_50mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Positive_100mm.png\">Large [591x682]</a></p>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix\"><a href=\"/images/logo/isotype/positive/Reactome_Isotype_Positive.svg\" title=\"Click to Download our logo in SVG format\" download=\"Reactome_Isotype_Positive.svg\">SVG</a></div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-left\"><a href=\"/images/logo/isotype/positive/Reactome_Isotype_Positive.emf\" title=\"Click to Download our logo in EMF format\" download=\"Reactome_Isotype_Positive.emf\">EMF</a></div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"intro-logo favth-clearfix\">\r\n<div class=\"pure-logo\">\r\n<div><img src=\"/images/logo/isotype/negative/Reactome_Isotype_Negative.svg\" alt=\"Reactome Logo (Isotype)\" width=\"80\" height=\"89\" class=\"isotype negative\" /></div>\r\n<div class=\"formats favth-clearfix\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-right\">\r\n<div class=\"png-download\"><a>PNG</a>\r\n<div class=\"tooltip-sizes\">\r\n<p><a href=\"/images/logo/isotype/negative/Reactome_Isotype_Negative_10mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Negative_25mm.png\">Small [119x138]</a></p>\r\n<p><a href=\"/images/logo/isotype/negative/Reactome_Isotype_Negative_25mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Negative_50mm.png\">Medium [297x342]</a></p>\r\n<p><a href=\"/images/logo/isotype/negative/Reactome_Isotype_Negative_50mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Negative_100mm.png\">Large [591x682]</a></p>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix \"><a href=\"/images/logo/isotype/negative/Reactome_Isotype_Negative.svg\" title=\"Click to Download our logo in SVG format\" download=\"Reactome_Isotype_Negative.svg\">SVG</a></div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-left\"><a href=\"/images/logo/isotype/negative/Reactome_Isotype_Negative.emf\" title=\"Click to Download our logo in EMF format\" download=\"Reactome_Isotype_Negative.emf\">EMF</a></div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-row favth-hidden-xs\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"padding0 top30 bottom30\"></div>\r\n</div>\r\n</div>\r\n<script type=\"text/javascript\">var showMsg = 1;\r\nfunction internetExplorerRightClick(){\r\n    if (document.all){\r\n        if (showMsg != 1)return false;\r\n    }\r\n}\r\n\r\nfunction firefoxRightClick(e){\r\n    if ((document.layers) || (document.getElementById && !document.all)) {\r\n        if (e.which==2 || e.which==3){\r\n            if (showMsg != 1)return false;\r\n        }\r\n    }\r\n}\r\n\r\nif (document.layers){\r\n    document.captureEvents(Event.MOUSEDOWN);\r\n    document.onmousedown=firefoxRightClick;\r\n}\r\nelse{\r\n    document.onmouseup=firefoxRightClick;\r\n    document.oncontextmenu=internetExplorerRightClick;\r\n}\r\n\r\ndocument.oncontextmenu=new Function(\"return false\");</script>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-10-13 08:33:16','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(93,718,'New responsive website with a fresh look','reactome-launches-new-website','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Responsive_Reactome_2.png\" alt=\"Responsive Reactome 2\" width=\"650\" height=\"400\" /></p>\r\n<p>Weve launched our new website and are excited to introduce you to our new look! Reactome is inviting its users to explore its new website. The new website has been designed to provide the ultimate user-friendly experience with improved navigation and functionality throughout. Created with the user experience firmly in mind, the new web interface has been designed using the latest technology, so the site is compatible with today\'s browsers and mobile devices. The site includes extensive documentation to help users understand Reactomes complete range of tools for <a href=\"/PathwayBrowser/\">viewing pathway diagrams</a>, <a href=\"/PathwayBrowser/#TOOL=AT\">analyzing experimental data</a> and <a href=\"/user/reactome-fiviz\">exploring protein-protein interaction networks</a>. For software developers, our <a href=\"/dev\">technical documentation and use cases</a> provide a detailed overview of our extensive web services, to access our curated pathway data (<a href=\"/ContentService/\">ContentService</a>) and analytical tools (<a href=\"/AnalysisService/\">AnalysisService</a>). Biologists and bioinformaticians can now benefit from richer online content that is easier to navigate and share with others, assisting with pathway data analysis and visualization.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-11-01 16:25:34','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,30,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(94,719,'Proudly introducing our new logo','proudly-introducing-our-new-logo','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/new_logo.png\" alt=\"new logo\" width=\"650\" height=\"365\" /></p>\r\n<p>As part of the ongoing evolution of our website, we are proud to announce the launch of our new logo. Reactome has grown and evolved over the last 14 years, and we felt it was time for a change.</p>\r\n<p>The new logo brings to the forefront the value and quality of the information and analysis tools that can be found in our curated database of pathways. The layering in our logo highlights the transparency of Reactomes nature while the rounded typography matches our openness.</p>\r\n<p>Derived from a shape that exists in nature and at the same time gives structure, the logo evokes the idea of discovering by unwrapping the layers of knowledge that surround biological events.</p>\r\n<p>We have refreshed our logo to reflect who we are today and to symbolize our dynamic future. Rest assured though: our high-quality data and services, and our dedication to you will remain the same!</p>\r\n<p>For more information, please go to <a href=\"about/logo\">Our Logo</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-11-07 16:44:47','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,29,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(95,721,'New Reactome Publication published in 2018 NAR Database Issue','new-reactome-publication-published-in-2017-nar-database-issue','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20171115_NAR_Paper_2017.png\" alt=\"20171115 NAR Paper 2017\" width=\"650\" height=\"489\" /></p>\r\n<p>A new&nbsp;research article titled <a href=\"https://academic.oup.com/nar/article/doi/10.1093/nar/gkx1132/4626770\">The Reactome Pathway Knowledgebase</a> has been published in the forthcoming&nbsp;2018 NAR Database Issue. The paper describes the&nbsp;deployment&nbsp;of a Neo4J graph database on our production server, development of a new high-performance in-memory implementation of our overrepresentation data analysis tool, improvements to the Pathway Diagram Viewer, and implementation of the new Enhanced&nbsp;High Level&nbsp;Diagrams (EHLDs). More publications from the&nbsp;<a href=\"about/team\">Reactome Team</a>&nbsp;can be found&nbsp;<a href=\"community/publications/\">here</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-11-15 04:02:20','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,28,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(96,722,'Diagram exporter','diagram-exporter','<style type=\"text/css\">pre, pre code {\r\n    word-wrap: initial;\r\n    white-space: pre;\r\n    word-break: break-all;\r\n}</style>\r\n<!-- This script performs the required analyses, gets the 2 tokens back and fills the links in the examples below. \r\nAnnotate the elements with class \'expression\' or \'enrichment\' accordingly-->\r\n<script type=\"text/javascript\">jQuery(document).ready(function() {\r\n		// The URL of the analysis method\r\n		var reactomeURI = \"/AnalysisService/identifiers/url/projection?pageSize=20&page=1\";\r\n		// The URL of our sample to be submitted for enrichment analysis\r\n		var enrichment = window.location.origin + \"/images/devguide/exporter/enrichment.txt\";\r\n		// Perform the ajax call to the analysis service\r\n		jQuery.ajax({\r\n			type: \"POST\",\r\n			url: reactomeURI,\r\n			data: enrichment,\r\n			headers: {\r\n					\"Content-Type\": \"text/plain\"\r\n			},\r\n			dataType: \"json\"\r\n		}).success(function (data, textStatus) { \r\n		   var token = data.summary.token;\r\n\r\n		   jQuery(\'.enrichment\').each(function(i, obj) {\r\n		       var link = jQuery(obj);\r\n		       var url = link.attr(\"href\") + token;\r\n		       link.attr(\"href\", url);\r\n		       link.text(function(index, text){\r\n                                    return text + token;\r\n                                });\r\n		   });\r\n		}).error(function (data, status, headers, config) {\r\n			$(\'#errorHolder\').text(status + \": \" + headers);\r\n		});\r\n\r\n		// The URL of our sample to be submitted for expression analysis\r\n		var expression = window.location.origin + \"/images/devguide/exporter/expression.txt\";\r\n		// Perform the ajax call to the analysis service\r\n		jQuery.ajax({\r\n			type: \"POST\",\r\n			url: reactomeURI,\r\n			data: expression,\r\n			headers: {\r\n					\"Content-Type\": \"text/plain\"\r\n			},\r\n			dataType: \"json\"\r\n		}).success(function (data, textStatus) { \r\n		   var token = data.summary.token;\r\n\r\n		   jQuery(\'.expression\').each(function(i, obj) {\r\n		       var link = jQuery(obj);\r\n		       var url = link.attr(\"href\") + token;\r\n		       link.attr(\"href\", url);\r\n		       link.text(function(index, text){\r\n                                    return text + token;\r\n                                });\r\n		   });\r\n		}).error(function (data, status, headers, config) {\r\n			$(\'#errorHolder\').text(status + \": \" + headers);\r\n		});\r\n		\r\n		// Get all elements annotated with \'addHost\' class and\r\n		// add window.location.origin in the displayed link\r\n		jQuery(\'.addHost\').each(function(i, obj) {\r\n		       var link = jQuery(obj);\r\n		       link.text(function(index, text){\r\n                                    return window.location.origin + text;\r\n                                });\r\n		   });\r\n	});</script>\r\n<h3 id=\"resultsHolder\"><span>Introduction</span></h3>\r\n<p><span>In Reactome, pathway diagrams are stored using a custom format. Aimming to allow researchers to include images of their favourite pathway diagrams into their publications, posters or presentations, we have developed the diagram exporter. This tool makes it easy to export pathway diagrams in bitmap format, allowing users to specify the output format, the quality and the decorators. The diagram exporter has been deployed as part of our <a href=\"/ContentService/#/exporter/toRasterUsingGET\" target=\"_blank\" rel=\"noopener noreferrer\">Content Service</a>, which constitutes an easy API that provides access to the Reactome knowledgebase. To learn more about the Content Service and how you can use it, have a look <a href=\"/dev/content-service\" target=\"&quot;_blank\">here</a>.</span></p>\r\n<p>This tutorial will walk you through the simple steps of using this tool to generate images of your favourite pathway diagram.</p>\r\n<h3><span>Get started</span></h3>\r\n<!-- Annotate the elements with class \'expression\' or \'enrichment\' accordingly -->\r\n<p><span>Generating the image of a diagram using our API is as easy as just calling the service with the identifier of the diagram and the desired extension (for example \".png\", \".jpg\" etc.)</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-169911.png\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-169911.png</code></a></pre>\r\n<p></p>\r\n<p><span><img src=\"images/devguide/exporter/Fig1.png\" alt=\"Figure 1\" width=\"620\" height=\"274\" /></span></p>\r\n<p><span>This will export the image in the default quality. To get an image in print quality, use the </span><i><span>quality&nbsp;</span></i><span><a href=\"#arguments\">argument</a>. Higher quality value results in higher image quality.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-169911.png?quality=7\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-169911.png?quality=7</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig2.png\" alt=\"Figure 2\" width=\"620\" height=\"274\" /></span></p>\r\n<p><span>In case the requested pathway has an interactive illustration (EHLD) associated with it, the diagram exporter will export the illustration as an image. In the following example, the EHLD of Hemostasis is requested in png format. Follow this <a href=\"/icon-info\">link</a> if you are interested to learn more about our EHLDs.&nbsp;</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-109582.png\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-109582.png</code></a></pre>\r\n<p></p>\r\n<p><span><img src=\"images/devguide/exporter/Fig3.png\" alt=\"Figure 3\" width=\"620\" height=\"274\" /></span></p>\r\n\r\n<p>If you are interested in a particular reaction or entity, just write it and the service will show it selected in its containing pathway.</p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-68919.png\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-68919.png</code></a></pre>\r\n<p></p>\r\n<p><img src=\"images/devguide/exporter/R-HSA-68919.png\" alt=\"R-HSA-68919\" title=\"R-HSA-68919\" width=\"620\"/></p>\r\n\r\n<h3><span>Decorators</span></h3>\r\n<p><span>The API allows users to easily highlight diagram elements (molecules and reactions) by either selecting them or flagging them. For instance, to select a reaction, we simply need to add the <i>sel</i> argument and its Reactome Identifier, exactly like in the following example.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-390522.png?sel=R-HSA-390598\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-390522.png?sel=R-HSA-390598</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig4.png\" alt=\"Figure 4\" width=\"620\" height=\"193\" /></span></p>\r\n<p><span>We can also use the <i>flg</i> argument to flag diagram entities (genes, molecules, etc.)</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-428359.png?flg=Q9NZI8\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-428359.png?flg=Q9NZI8</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig5.png\" alt=\"Figure 5\" width=\"620\" height=\"252\" /></span></p>\r\n<p><span>The selection and flagging arguments work in exactly the same way for EHLDs.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-109582.png?sel=R-HSA-983231&amp;flg=THBD\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-109582.png?sel=R-HSA-983231&amp;flg=THBD</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig6.png\" alt=\"Figure 6\" width=\"620\" height=\"193\" /></span></p>\r\n<h3><span>Analysis overlay</span></h3>\r\n<p><span>Reactome offers a pathway analysis service that supports enrichment and expression analysis. The diagram exporter allows you to overlay the results of the analysis on top of the exported diagrams. To do so, use the </span><i><span>token</span></i><span> argument to specify the unique token assosiated with the performed analysis. To learn more about our Analysis Service and how to use it have a look to this <a href=\"/dev/analysis\">page</a>.</span></p>\r\n<p><span>In the next example, we use the token acquired from an overrepresentation analysis, to overlay the results on top of a diagram and export it in high quality (7) jpeg format.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-8937144.jpeg?quality=7&amp;token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"enrichment\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-8937144.jpeg?quality=7&amp;token=</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig7.jpeg\" alt=\"Figure 7\" width=\"620\" height=\"291\" /></span></p>\r\n<p><span>In the same way, we can overlay the results of an expression analysis on top of a diagram and export it in our prefered format and quality. In case the submitted sample (for this type of analysis) includes more than one column, we can specify it by using the <i>column</i> argument, like in the following example.</span></p>\r\n<p><span> Please keep in mind that in case a column is not specified (null), the first one is selected by default. Also in case a column is not specified and the requested format is gif, then an animated image with all the columns is generated.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-432047.jpg?quality=7&amp;column=1&amp;token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"expression\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-432047.jpg?quality=7&amp;column=1&amp;token=</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig8.jpg\" alt=\"Figure 8\" width=\"620\" height=\"504\" /></span></p>\r\n<p><span>Analysis results can also be overlaid on top of our interactive illustrations.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-69278.png?token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"expression\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-69278.png?token=</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig9.png\" alt=\"Figure 9\" width=\"620\" height=\"291\" /></span></p>\r\n<h3><span>Animated GIFs</span></h3>\r\n<p><span>When you run an expression analysis and want to export an animated GIF with a frame per analysis column, set extension to gif and dont specify column:</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-432047.gif?quality=7&amp;sel=R-ALL-879865&amp;token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"expression\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-432047.gif?quality=7&amp;sel=R-ALL-879865&amp;token=</code></a></pre>\r\n<p></p>\r\n<p><span><img src=\"images/devguide/exporter/Fig10.gif\" alt=\"Figure 10\" width=\"620\" height=\"504\" /></span></p>\r\n<p><span>In the same way, you can export animaged gifs of EHLDs</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-69278.gif?quality=7&amp;sel=R-HSA-69242&amp;token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"expression\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-69278.gif?quality=7&amp;sel=R-HSA-69242&amp;token=</code></a></pre>\r\n<p></p>\r\n<p><span><img src=\"images/devguide/exporter/Fig11.gif\" alt=\"Figure 11\" width=\"620\" height=\"504\" /></span></p>\r\n<h3><span>Color profiles</span></h3>\r\n<p><span>Reactome provides several color profiles for diagram and analysis. You can use </span><i><span>diagramProfile</span></i><span> and </span><i><span>analysisProfile</span></i><span>&nbsp;arguments to modify the default ones.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-879415.png?quality=7&amp;sel=R-ALL-87986&amp;diagramProfile=standard&amp;analysisProfile=strosobar&amp;token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"expression\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-879415.png?quality=7&amp;sel=R-ALL-879865&amp;diagramProfile=standard&amp;analysisProfile=strosobar&amp;token=</code></a></pre>\r\n<p></p>\r\n<p><span><img src=\"images/devguide/exporter/Fig12.png\" alt=\"Figure 12\" width=\"620\" height=\"357\" /></span></p>\r\n<p></p>\r\n<p></p>\r\n<h3><a name=\"arguments\"><span>Arguments</span></a></h3>\r\n<p><span>The folowing table presents a full list of the supported arguments alond with their default values.</span></p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome arguments\">\r\n<thead>\r\n<tr><th>Name</th><th>Description</th><th>Values</th><th>Default</th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p><i><span>quality</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>Quality of the image</span></p>\r\n</td>\r\n<td>\r\n<p><span>1 - 10</span></p>\r\n</td>\r\n<td>\r\n<p><span>5</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>format</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>Format of the image</span></p>\r\n</td>\r\n<td>\r\n<p><span>png, jpg, jpeg, gif</span></p>\r\n</td>\r\n<td>\r\n<p><span>png</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>flags</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>List of elements to be flagged</span></p>\r\n</td>\r\n<td>\r\n<p><span>stId, dbId, identifer, geneName</span></p>\r\n</td>\r\n<td>\r\n<p><span>null</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>selection</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>List of elements to be selected</span></p>\r\n</td>\r\n<td>\r\n<p><span>stId, dbId, identifer, geneName</span></p>\r\n</td>\r\n<td>\r\n<p><span>null</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>diagramProfile</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>Color profile for diagram</span></p>\r\n</td>\r\n<td>\r\n<p><span>modern, standard</span></p>\r\n</td>\r\n<td>\r\n<p><span>modern</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>analysisProfile</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>Color profile for analysis</span></p>\r\n</td>\r\n<td>\r\n<p><span>standard, strosobar, copper plus</span></p>\r\n</td>\r\n<td>\r\n<p><span>standard</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>token</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>Analysis token</span></p>\r\n</td>\r\n<td>\r\n<p><span>String</span></p>\r\n</td>\r\n<td>\r\n<p><span>null</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>column</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>The specific expression analysis results column to be overlaid</span></p>\r\n<p><span>If column is not specified (null), the first one is selected.</span></p>\r\n<p><span>If column is not specified (null) and format is gif, then an animated gif is generated with all the columns.</span></p>\r\n</td>\r\n<td>\r\n<p><span>Integers</span></p>\r\n</td>\r\n<td>\r\n<p><span>null</span></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<!-- END OF ARGUMENTS TABLE-->\r\n<h3><span>&nbsp;</span></h3>\r\n<p></p>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-11-16 12:59:30','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(97,728,'Updated License Agreement','updated-license-agreement','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20171205_CC0.png\" alt=\"Creative Commons Zero - CC0\" width=\"650\" height=\"382\" /></p>\r\n<p>Since our inception, we have been an open source and open access resource, &nbsp;free for use by anyone under the terms of a Creative Commons Attribution 4.0 International (CC BY 4.0) license. This license granted parties the non-exclusive right to use, distribute and create derivative works based on Reactome, provided that the works are correctly attributed to OICR, NYUMC, EBI, and OHSU.</p>\r\n<p>In line with the growing movement to provide free open data in the public domain, and to better support the needs of our user community, we are updating the licensing agreement for some of our web content and data to reflect the <a href=\"https://creativecommons.org/share-your-work/public-domain/cc0\">Creative Commons Public Domain (CC0)</a>. CC0 is the \"no copyright reserved\" option in the Creative Commons toolkit. It effectively means relinquishing all copyright and similar rights that we hold in a work and dedicating those rights to the public domain.</p>\r\n<p>A <a href=\"https://creativecommons.org/publicdomain/zero/1.0/\">Creative Commons Public Domain (CC0 1.0 Universal) Licence</a> will now cover all Reactome annotation files, e.g. identifier mapping, specialized data files, and interaction data derived from Reactome.</p>\r\n<p>A <a href=\"https://creativecommons.org/licenses/by/4.0/\">Creative Commons Attribution 4.0 International (CC BY 4.0) Licence</a> will continue to apply to all software and code, e.g. relating to the functionality of the <a href=\"https://reactome.org\">reactome.org</a>,&nbsp;derived websites and webservices, the Curator Tool, the Functional Interaction application, SQL and Graph Database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials.</p>\r\n<p>For information about how to properly credit data use, please review the <a href=\"/license\">Reactome License</a> and the <a href=\"https://creativecommons.org/faq/\">Creative Commons FAQ</a>, or contact the <a href=\"mailto:help@reactome.org\">Reactome Helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-12-07 10:37:34','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,27,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(98,729,'Version 63 Released','version-63-released','<p><a href=\"/PathwayBrowser/#/R-HSA-73894\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-73894.svg\" alt=\"DNA repair\" width=\"650\" height=\"361\" /></a></p>\r\n<p class=\"gmail-m_-8391422641416123832gmail-MsoNormal\"><strong>New and Updated Pathways.&nbsp;</strong>In version&nbsp;V63, topics with new or revised pathways include&nbsp;Immune System&nbsp;<a href=\"PathwayBrowser/#/R-HSA-8854691\" target=\"_blank\">(Interleukin-20&nbsp; family signaling</a>,&nbsp;<a href=\"PathwayBrowser/#/R-HSA-451927&amp;SEL=R-HSA-9020958&amp;PATH=R-HSA-168256,R-HSA-1280215,R-HSA-449147\" target=\"_blank\">Interleukin-21 signaling</a>,&nbsp;<a href=\"PathwayBrowser/#/R-HSA-9008059\" target=\"_blank\">Interleukin-37 signaling</a>, and&nbsp;<a href=\"PathwayBrowser/#/R-HSA-449836\" target=\"_blank\">other Interleukin signaling</a>), Metabolism (<a href=\"PathwayBrowser/#/R-HSA-196791\" target=\"_blank\">Vitamin D (calciferol) metabolism</a>), and Signal Transduction (<a href=\"PathwayBrowser/#/R-HSA-9012852&amp;PATH=R-HSA-162582,R-HSA-157118\" target=\"_blank\">Signaling by NOTCH3</a>).&nbsp;Illustrations are now available for&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-73894\" target=\"_blank\">DNA Repair</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-73884\" target=\"_blank\">Base Excision Repair</a>,&nbsp;<a href=\"PathwayBrowser/#/R-HSA-372790&amp;PATH=R-HSA-162582\" target=\"_blank\">Signaling by GPCR</a>,&nbsp;&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-500792&amp;PATH=R-HSA-162582,R-HSA-372790\" target=\"_blank\">GPCR ligand binding</a>,&nbsp;&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-388396&amp;PATH=R-HSA-162582,R-HSA-372790\" target=\"_blank\">GPCR downstream&nbsp;signaling</a>, and&nbsp;<a href=\"PathwayBrowser/#/R-HSA-72766\" target=\"_blank\">Translation</a>.</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.&nbsp;</strong><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Laurence+Bindoff\" target=\"_blank\" rel=\"noopener\">Laurence Bindoff</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Roberta+Carriero+%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Roberta Carriero</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Sandip+K+Datta%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Sandip Datta</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Cecilia+Garlanda+%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Cecilia Garlanda</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Elzbieta+Glaser%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Elzbieta Glaser</a>,&nbsp;<a href=\"http://drholick.com/\" target=\"_blank\" rel=\"noopener\">Michael Holick</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Tony%20Kouzarides%5BAuthor%20-%20Full%5D\" target=\"_blank\" rel=\"noopener\">Tony Kouzarides</a>&nbsp;,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Alberto+Mantovani%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Alberto Mantovani</a>,&nbsp;and&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Birgit+Meldal+%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Birgit Meldal</a>&nbsp;are our external reviewers.</p>\r\n<p><b>Annotation Statistics.</b>&nbsp;Reactome comprises 11,426 human reactions organized into 2,179 pathways involving 10,996 proteins encoded by 10,739 different human genes, 1,764 small molecules, and 11,366 complexes. These annotations are supported by 27,694 literature references. We have projected these reactions onto 138,985 orthologous proteins, creating 20,932 orthologous pathways in 18 non-human species. Version 63 has annotations for 1,334 protein variants (mutated proteins) and their post-translationally modified forms, derived from 287 proteins. These have been used to annotate 506 disease-specific complexes and 906 disease-specific reactions organized into 453 pathways and subpathways, and tagged with 294 Disease Ontology terms.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome&nbsp;are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence,. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">,.</a>.</p>\r\n<p><strong>Follow us on Twitter:</strong>&nbsp;<a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<span id=\"cloak15d567bd0a84d9471f3b57a7075b6514\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-12-18 17:06:49','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,26,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(99,732,'Help','help','','',1,30,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-01-21 18:59:05','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"0\",\"link_titles\":\"0\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"0\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"0\",\"link_author\":\"0\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',2,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(100,733,'Pathway Diagrams Specifications','pathway-diagrams-specifications','<p>\r\n<script src=\"//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js\" type=\"text/javascript\"></script>\r\n<script type=\"text/javascript\">hljs.initHighlightingOnLoad();</script>\r\n<style type=\"text/css\">ul.tree, fieldset legend {font-family: \"Source Code Pro\", monospace;}\r\nul.tree li {list-style-type: none; position: relative;}\r\nul.tree li ul {display: none;}\r\nul.tree li.open > ul {display: block;}\r\nul.tree li a {color: black;text-decoration: none;}\r\nul.tree li a:before {height: 1em;padding: 0 .1em; font-size: .8em; display: block; position: absolute; left: -1.3em;top: .2em;}\r\nul.tree li > a:not(:last-child):before { content: \'+\';}\r\nul.tree li.open > a:not(:last-child):before {content: \'-\';}\r\n.frame { border: 1px solid #888888; background-color: #f0f0f0;}</style>\r\n</p>\r\n<h2 id=\"introduction\">Introduction</h2>\r\n<p>This document provides a set of guidelines for developers on how to properly render Reactome pathway diagrams. Pathway Diagrams represent pathways as a series of connected molecular events, known in Reactome as reactions, which can be considered as steps in the pathway. For a more detailed guide about diagrams refer to <a href=\"/userguide/pathway-browser\">our user guide</a>.</p>\r\n<p>Currently, there are two projects that follow these guidelines:</p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li>diagram: <a href=\"https://github.com/reactome-pwp/diagram\" target=\"_blank\" rel=\"noopener\"> https://github.com/reactome-pwp/diagram </a></li>\r\n<li>diagram-exporter: <a href=\"https://github.com/reactome-pwp/diagram-exporter\" target=\"_blank\" rel=\"noopener\"> https://github.com/reactome-pwp/diagram-exporter </a></li>\r\n</ul>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/apoptosis.png\" alt=\"Regulation of Apoptosis diagram\" />Regulation of Apoptosis diagram</figure>\r\n<h2 id=\"diagram-layout-and-graph-files\">Diagram layout and graph files</h2>\r\n<p>All the class diagrams can be found as UML in xmi (Umbrello) format. <a href=\"/docs/devguide/diagram/reactome-interfaces.xmi\" download=\"reactome-interfaces.xmi\">Download</a></p>\r\n<p>The layout information of every diagram is stored in a JSON file, named after its stable identifier, i.e. <em> R-HSA-12345.json </em> . This file contains all the information regarding what to draw and how to draw it, including coordinates and sizes. In addition to the layout file, Reactome provides another JSON file per diagram containing a small graph view of the diagram content, i.e. <em> R-HSA-12345.graph.json</em>. We discuss this diagram graph file in more detail further on.</p>\r\n<p><em>The general structure of a diagram <em>layout&nbsp;</em>file: </em></p>\r\n<pre><code class=\"json\">{\r\n    \"dbId\":169911,\r\n    \"stableId\":\"R-HSA-169911\",\r\n    \"displayName\":\"Diagram of Regulation of Apoptosis\",\r\n    \"disease\":false,\r\n    \"minX\":61,\r\n    \"maxX\":1126,\r\n    \"minY\":202,\r\n    \"maxY\":656,\r\n    \"nodes\":[],\r\n    \"edges\":[],\r\n    \"compartments\":[],\r\n    \"notes\":[],\r\n    \"links\":[],\r\n    \"shadows\":[]\r\n}\r\n</code></pre>\r\n<p><em>To get a more detailed description of the diagram&nbsp;<em>layout</em>, expand the following tree:</em></p>\r\n<div class=\"frame\">\r\n<ul class=\"tree\">\r\n<li class=\"parent\"><a>Diagram</a>\r\n<ul>\r\n<li class=\"parent\"><a>nodes : <strong>Node[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>nodeAttachments : <strong>NodeAttachment[]</strong></a>\r\n<ul>\r\n<li>reactomeId : <strong>Long</strong></li>\r\n<li>label : <strong>String</strong></li>\r\n<li>description : <strong>String</strong></li>\r\n<li class=\"parent\"><a>shape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>interactorsSummary : <strong>SummaryItem</strong></a>\r\n<ul>\r\n<li>pressed : <strong>Boolean</strong></li>\r\n<li class=\"parent\"><a>shape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>hit : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n<li>number : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li>trivial : <strong>Boolean</strong></li>\r\n<li class=\"parent\"><a>connectors : <strong>Connector[]</strong></a>\r\n<ul>\r\n<li>edgeId : <strong>Long</strong></li>\r\n<li class=\"parent\"><a>segments : <strong>Segment[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>to : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>from : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>endShape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>stoichiometry : <strong>Stoichiometry</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>shape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>value : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li>isDisease : <strong>Boolean</strong></li>\r\n<li>isFadeOut : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] prop : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] innerProp : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] identifier : <strong>Identifier</strong></a>\r\n<ul>\r\n<li>resource : <strong>String</strong></li>\r\n<li>id : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] textPosition : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] insets : <strong>Bound</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] bgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] fgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>NodeCommon</em>] isCrossed : <strong>Boolean</strong></li>\r\n<li>[<em>NodeCommon</em>] needDashedBorder : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>isDisease : <strong>Boolean</strong></li>\r\n<li>minX : <strong>Integer</strong></li>\r\n<li>minY : <strong>Integer</strong></li>\r\n<li>maxX : <strong>Integer</strong></li>\r\n<li>maxY : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>notes : <strong>Note[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] prop : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] innerProp : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] identifier : <strong>Identifier</strong></a>\r\n<ul>\r\n<li>resource : <strong>String</strong></li>\r\n<li>id : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] textPosition : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] insets : <strong>Bound</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] bgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] fgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>NodeCommon</em>] isCrossed : <strong>Boolean</strong></li>\r\n<li>[<em>NodeCommon</em>] needDashedBorder : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>edges : <strong>Edge[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] segments : <strong>Segment[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>to : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>from : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] endShape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] catalysts : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] inhibitors : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] activators : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>EdgeCommon</em>] precedingEvents : <strong>Long[]</strong></li>\r\n<li>[<em>EdgeCommon</em>] reactionType : <strong>String</strong></li>\r\n<li>[<em>EdgeCommon</em>] followingEvents : <strong>Long[]</strong></li>\r\n<li>[<em>EdgeCommon</em>] interactionType : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] reactionShape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] inputs : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] outputs : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>links : <strong>Link[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] segments : <strong>Segment[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>to : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>from : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] endShape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] catalysts : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] inhibitors : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] activators : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>EdgeCommon</em>] precedingEvents : <strong>Long[]</strong></li>\r\n<li>[<em>EdgeCommon</em>] reactionType : <strong>String</strong></li>\r\n<li>[<em>EdgeCommon</em>] followingEvents : <strong>Long[]</strong></li>\r\n<li>[<em>EdgeCommon</em>] interactionType : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] reactionShape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] inputs : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] outputs : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>compartments : <strong>Compartment[]</strong></a>\r\n<ul>\r\n<li>componentIds : <strong>Long[]</strong></li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] prop : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] innerProp : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] identifier : <strong>Identifier</strong></a>\r\n<ul>\r\n<li>resource : <strong>String</strong></li>\r\n<li>id : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] textPosition : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] insets : <strong>Bound</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] bgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] fgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>NodeCommon</em>] isCrossed : <strong>Boolean</strong></li>\r\n<li>[<em>NodeCommon</em>] needDashedBorder : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>shadows : <strong>Shadow[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>prop : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>colour : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>dbId : <strong>Long</strong></li>\r\n<li>stableId : <strong>String</strong></li>\r\n<li>cPicture : <strong>String</strong></li>\r\n<li>forNormalDraw : <strong>Boolean</strong></li>\r\n<li>displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</div>\r\n<p>&nbsp;</p>\r\n<h2>Diagram objects</h2>\r\n<p>Diagram layout is made up of six types of elements:</p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li><strong>Compartments</strong></li>\r\n<li><strong>Nodes</strong></li>\r\n<li><strong>Edges</strong></li>\r\n<li><strong>Links</strong></li>\r\n<li><strong>Notes</strong></li>\r\n<li><strong>Shadows</strong></li>\r\n</ul>\r\n<p>These elements follow this hierarchy.</p>\r\n<p><img src=\"images/devguide/diagram/diagram_objects.png\" alt=\"Hierarchy of diagram layout objects\" width=\"620\" height=\"133\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<h3>Compartments</h3>\r\n<p><img src=\"images/devguide/diagram/compartment.png\" alt=\"Compartment\" width=\"300\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p>Compartments are rendered in the background and represent where reactions happen. They are rendered as rounded rectangles using <em> NodeCommon.prop</em>. Compartments might have an additional layer when surrounded by a membrane, which must be specified using <em> NodeCommon.innerProp. Text (<em>DiagramObject.displayName</em>) is displayed in one line, using <em> NodeCommon.textPosition.</em> Color is taken from the diagram profile sheet (<em>Profile.compartment</em>). Compartments cannot be rendered on top of any other node.</em></p>\r\n<h3>Nodes</h3>\r\n<p>Nodes represent the participants of reactions, i.e. inputs, outputs, catalysts, regulators and other pathways. The type of each node is coded into <em>renderableClass.</em> The renderable class can be one of <strong>Chemical, ChemicalDrug, Complex, Entity, EntitySet, Gene, ProcessNode, EncapsulatedNode, Protein</strong> and <strong>RNA.</strong></p>\r\n<p>Nodes are laid in 2 layers, background and foreground. The background is rendered using a different shape for each class with dimensions defined in <em> NodeCommon.prop</em>. Some nodes may have a foreground (<strong>ProcessNode</strong>, <strong>EncapsulatedNode</strong> and <strong>EntitySet</strong>). The foreground shape and colour are class specific. The text is centred to the node with a padding of 5 points to the <em>NodeCommon.prop</em>. In case it is too large, then it is wrapped in several lines and, if needed, the size of the font is reduced. When a foreground is present, the text is padded to foreground limits.</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome spec-nodes\">\r\n<thead>\r\n<tr><th scope=\"col\"><strong>Class</strong></th><th scope=\"col\"><strong>Background</strong></th><th scope=\"col\"><strong>Foreground</strong></th><th scope=\"col\"><strong>Example</strong></th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td><em> Chemical </em></td>\r\n<td>ellipse</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/chemical.png\" alt=\"Chemical\" width=\"150\" height=\"75\" title=\"Chemical\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> ChemicalDrug </em></td>\r\n<td>ellipse</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/chemicaldrug.png\" alt=\"Chemical drug\" width=\"150\" height=\"75\" title=\"Chemical drug\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> Complex </em></td>\r\n<td>edged rectangle (octagon)</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/complex.png\" alt=\"Complex\" width=\"150\" height=\"75\" title=\"Complex\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> Entity </em></td>\r\n<td>rectangle</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/entity.png\" alt=\"Entity\" width=\"150\" height=\"75\" title=\"Entity\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> EntitySet </em></td>\r\n<td>rounded rectangle</td>\r\n<td>rounded rectangle (padding = 4)</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/entityset.png\" alt=\"EntitySet\" width=\"150\" height=\"75\" title=\"EntitySet\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em>* Gene </em></td>\r\n<td>recatangle with 2 rounded corners</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/gene.png\" alt=\"Gene\" width=\"150\" height=\"75\" title=\"Gene\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> ProcessNode </em></td>\r\n<td>rectangle</td>\r\n<td>rectangle (padding = 10)</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/processnode.png\" alt=\"ProcessNode\" width=\"150\" height=\"75\" title=\"ProcessNode\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> EncapsulatedNode </em></td>\r\n<td>hexagon</td>\r\n<td>hexagon (padding = 10)</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/encapsulatednode.png\" alt=\"EncapsulatedNode\" width=\"150\" height=\"75\" title=\"EncapsulatedNode\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> Protein </em></td>\r\n<td>rounded rectangle</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/attachment.png\" alt=\"Protein with attachment\" width=\"150\" height=\"75\" title=\"Protein with attachment\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> <em> </em> RNA </em></td>\r\n<td>bone</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/rna.png\" alt=\"RNA\" width=\"150\" height=\"75\" title=\"RNA\" /></td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<p>*Genes are made up using a different approach. They have 2 shapes, a rectangle with 2 rounded corners (y=NodeCommon.prop.y + 25) and a triangle. The rectangle is filled but not stroked. Both shapes are joint using 3 perpendicular lines.</p>\r\n<pre><code class=\"python\">shape = SemiRoundedRectangle(prop.x, prop.y + 25, prop.width. prop.height)\r\narrow = Path()\r\narrow.moveTo(prop.maxX,     prop.getY + 8)\r\narrow.lineTo(prop.maxX,     prop.getY - 8)\r\narrow.lineTo(prop.maxX + 8, prop.getY)\r\narrow.closePath()\r\npath = Path()\r\npath.moveTo(prop.x,        prop.y + 25)\r\npath.lineTo(prop.maxX,     prop.y + 25)\r\npath.moveTo(prop.maxX - 4, prop.y + 25)\r\npath.lineTo(prop.maxX - 4, prop.y)\r\npath.lineTo(prop.maxX,     prop.y)  # already closed\r\nfill(shape, arrow)\r\nstroke(path, arrow)\r\n</code></pre>\r\n<p><em>RNA shape looks after a bone </em> :</p>\r\n<pre><code class=\"python\">x1 = rna.x + 16\r\nx2 = rna.maxX - 16\r\ny1 = rna.y + 8\r\ny2 = rna.maxY - 8\r\npath = Path()\r\npath.moveTo(x1, y1)\r\npath.lineTo(x2, y1)\r\npath.quadTo(rna.maxX, rna.y,    rna.maxX, rna.centerY)\r\npath.quadTo(rna.maxX, rna.maxY, x2,       y2)\r\npath.lineTo(x1, y2)\r\npath.quadTo(rna.x,    rna.maxY, rna.x,    rna.centerY)\r\npath.quadTo(rna.x,    rna.y,    x1,       y1)\r\nfill(path)\r\nstroke(path)\r\n</code></pre>\r\n<p>Some nodes contain attachments, such as post-translational modifications (PTMs) . Attachments are styled after their owner. The border of nodes is rendered after the background and foreground shapes, except for genes, which have a custom border shape. When a node has <em>NodeCommon.needDashedBorder</em> then the border must be dashed.</p>\r\n<p>Drugs use the same shape as chemicals, but with different colors. They also have a small reactangle (14x7) in the bottom right corner with the text Rx (recipe, latin word for prescription).</p>\r\n<h3>Edges</h3>\r\n<p>Edges represent reactions in the diagram. They are layout as lines with shapes. They can have 2 shapes: a <em>reaction shape</em>, to indicate the type of reaction, and an <em>end shape</em>. Both can be omitted. Lines are taken from <em>EdgeCommon.segments</em>. Shape is created using several properties inside <em>EdgeCommon</em>.</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome spec-edges\">\r\n<thead>\r\n<tr><th><strong>Reaction type</strong></th><th><strong>shape.type</strong></th><th><strong>shape.empty</strong></th><th><strong>shape.text</strong></th><th><strong>Example</strong></th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>Association</td>\r\n<td>circle</td>\r\n<td>fill</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/association.png\" alt=\"Association\" width=\"24\" title=\"Association\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Dissociation</td>\r\n<td>double circle</td>\r\n<td>empty</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/dissociation.png\" alt=\"Dissociation\" width=\"24\" title=\"Dissociation\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Omitted process</td>\r\n<td>box</td>\r\n<td>empty</td>\r\n<td>\\\\</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/omitted.png\" alt=\"Omitted\" width=\"24\" title=\"Omitted\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Transition</td>\r\n<td>box</td>\r\n<td>empty</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/transition.png\" alt=\"Transition\" width=\"24\" title=\"Transition\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Uncertain</td>\r\n<td>box</td>\r\n<td>empty</td>\r\n<td>?</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/uncertain.png\" alt=\"Uncertain\" width=\"24\" title=\"Uncertain\" /></td>\r\n</tr>\r\n<tr>\r\n<td>All (end shape)</td>\r\n<td>arrow</td>\r\n<td>fill</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/end-shape.png\" alt=\"End shape\" width=\"24\" title=\"End shape\" /></td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>Shapes are specified in the <em>Shape</em> element inside <em>Edge.reactionType</em> and <em>Edge.endShape</em>.</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome spec-shapes\">\r\n<thead>\r\n<tr><th><strong> <em> Shape.type </em> </strong></th><th><strong> Shape </strong></th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>ARROW</td>\r\n<td>triangle with points: a, b, c</td>\r\n</tr>\r\n<tr>\r\n<td>BOX</td>\r\n<td>rectangle from a (top left) to b (bottom right)</td>\r\n</tr>\r\n<tr>\r\n<td>CIRCLE</td>\r\n<td>circle with center c and radius r</td>\r\n</tr>\r\n<tr>\r\n<td>STOP</td>\r\n<td>line from a to b</td>\r\n</tr>\r\n<tr>\r\n<td>DOUBLE CIRCLE</td>\r\n<td>2 circles with center c and radii r and r1</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>When <em> Shape.s </em> is present, the value of <em>Shape.s</em> is written in the centre of the shape. When <em> Shape.emtpy </em> is true, the Shape is filled with <em> Profile.reaction.fill </em> color (usually white), otherwise, it is filled with <em> Profile.reaction.stroke </em> (usually black).</p>\r\n<h3>Connectors</h3>\r\n<p>As each reaction can have several participants, a connector is created in the participant with <em> Connector.EdgeId </em> = <em> DiagramObject.Id </em>. Connectors are styled after the reaction they belong. Connectors have segments and 2 shapes: <em>end shape</em> and <em>stoichiometry shape</em>. Stoichiometries are always empty boxes with the stoichiometry value as text only if stoichiometry value is greater than 1.</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome spec-connectors\">\r\n<thead>\r\n<tr><th><strong>endShape.type</strong></th><th><strong>Shape</strong></th><th><strong>Empty</strong></th><th><strong>Example</strong></th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>Inhibitor</td>\r\n<td>line</td>\r\n<td>fill</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/inhibitor.png\" alt=\"Inhibitor\" width=\"2\" title=\"Inhibitor\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Catalyst</td>\r\n<td>circle</td>\r\n<td>empty</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/catalyst.png\" alt=\"Catalyst\" width=\"24\" title=\"Catalyst\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Output</td>\r\n<td>arrow</td>\r\n<td>fill</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/end-shape.png\" alt=\"Output\" width=\"24\" title=\"Output\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Activator</td>\r\n<td>arrow</td>\r\n<td>empty</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/activator.png\" alt=\"Activator\" width=\"24\" title=\"Activator\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Input</td>\r\n<td><em>no shape</em></td>\r\n<td>&nbsp;</td>\r\n<td>&nbsp;</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<h3>Links</h3>\r\n<p>Links are used for linking elements which are related, normally subpathways or distant nodes.</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome spec-links\">\r\n<thead>\r\n<tr><th><strong>Renderable class</strong></th><th><strong>Dashed</strong></th><th><strong>Link type</strong></th><th><strong>Shape</strong></th><th><strong>Empty</strong></th><th><strong>Example</strong></th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>EntitySetAndMemberLink</td>\r\n<td>dashed</td>\r\n<td>&nbsp;</td>\r\n<td>&nbsp;</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/link.png\" alt=\"Link\" width=\"75\" title=\"Link\" /></td>\r\n</tr>\r\n<tr>\r\n<td>EntitySetAndEntitySetLink</td>\r\n<td>dashed</td>\r\n<td>&nbsp;</td>\r\n<td>&nbsp;</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/link.png\" alt=\"Link\" width=\"75\" title=\"Link\" /></td>\r\n</tr>\r\n<tr>\r\n<td>FlowLine</td>\r\n<td>false</td>\r\n<td>&nbsp;</td>\r\n<td>ARROW</td>\r\n<td>fill</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/flowline.png\" alt=\"Flow line\" width=\"75\" title=\"Flow Line\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Interaction</td>\r\n<td>false</td>\r\n<td>Activate</td>\r\n<td>ARROW</td>\r\n<td>empty</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/activate_inhibit.png\" alt=\"Activate\" width=\"75\" title=\"Activate\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Interaction</td>\r\n<td>false</td>\r\n<td>Inhibit</td>\r\n<td>ARROW</td>\r\n<td>empty</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/activate_inhibit.png\" alt=\"Inhibit\" width=\"75\" title=\"Inhibit\" /></td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<h3>Notes</h3>\r\n<p>Notes are just texts. The text is displayed in one line beginning at <em>NodeCommon.textPosition. </em> Color is taken from <em> Profile.note.text</em>.</p>\r\n<h3>Shadows</h3>\r\n<p>Shadows are used when there are several subpathways in the same diagram. In particular, these coloured rectangles highlight the specific diagram reactions belonging to each subpathway, allowing further zooming in to the areas of interest within a given pathway diagram.&nbsp;Rendering them is optional, but it is recommended that they are drawn on top of every element, as rectangles, using <em> Shadow.points </em> and <em> Shadow.color </em> for text and filling. It is also adviced to use transparency for the filling so that all subpathway participants are visible. The text contains the name of the subpathway and should be rendered using a bigger font size.&nbsp;</p>\r\n<h2 id=\"texts\">Text</h2>\r\n<p>The default font for texts is Arial black with size 9. For shadows, font size is 24.</p>\r\n<h2 id=\"colouring\">Colouring</h2>\r\n<p>Element colors are taken from a JSON stylesheet. Currently, we have 2 styles: <a href=\"https://github.com/reactome-pwp/diagram/blob/master/src/main/resources/org/reactome/web/diagram/profiles/diagram/profile_01.json\" target=\"_blank\" rel=\"noopener\"> standard </a> and <a href=\"https://github.com/reactome-pwp/diagram/blob/master/src/main/resources/org/reactome/web/diagram/profiles/diagram/profile_02.json\" target=\"_blank\" rel=\"noopener\"> modern</a>.</p>\r\n<p><em>To get a more detailed description of the diagram colour profile, expand the following tree:</em></p>\r\n<div class=\"frame\">\r\n<ul class=\"tree\">\r\n<li class=\"parent\"><a>DiagramProfile</a>\r\n<ul>\r\n<li class=\"parent\"><a>stoichiometry : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>entity : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>complex : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>note : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>interactor : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>attachment : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>flowline : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>gene : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>processnode : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>encapsulatednode : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>protein : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>link : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>otherentity : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>chemical : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>chemicaldrug : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>entityset : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>compartment : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>reaction : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>rna : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>thumbnail : <strong>DiagramProfileThumbnail</strong></a>\r\n<ul>\r\n<li>hovering : <strong>String</strong></li>\r\n<li>highlight : <strong>String</strong></li>\r\n<li>selection : <strong>String</strong></li>\r\n<li>edge : <strong>String</strong></li>\r\n<li>node : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>name : <strong>String</strong></li>\r\n<li class=\"parent\"><a>properties : <strong>DiagramProfileProperties</strong></a>\r\n<ul>\r\n<li>hovering : <strong>String</strong></li>\r\n<li>highlight : <strong>String</strong></li>\r\n<li>selection : <strong>String</strong></li>\r\n<li>halo : <strong>String</strong></li>\r\n<li>flag : <strong>String</strong></li>\r\n<li>trigger : <strong>String</strong></li>\r\n<li>button : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>disease : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>The <strong>DiagramProfileNode</strong> defines 3 colors per each node: <em> fill, stroke </em> and <em> text </em> . If an object is fadeout (<em> DiagramObject.isFadeOut), </em> then we must choose <em>fadeOutFill, fadeOutStroke</em> and <em>fadeOutText</em>. If an analysis is being run, then we use <em>lighterFill, lighterStroke</em> and <em>lighterText</em>. And by default <em>fill, stroke</em> and <em>text</em>.</p>\r\n<p>There is 1 exception: reactions and stoichiometries texts; text color is ignored and stroke is used instead. When <em>DiagramObject.isDisease</em> is true, the color of node border, edge border, edge text and segments is changed to <em>Profile.properties.disease</em> (usually red).</p>\r\n<h3>Decorators</h3>\r\n<p>Some elements can be decorated in the diagram. This information is not in the diagram and has to be inserted from the outside. Reactome supports 2 types of decoration: <strong>selection</strong> and <strong>flagging</strong>.</p>\r\n<p>When a node is selected, the border is changed in 2 ways: the color is changed to <em> Profile.properties.selection</em> and the width is increased. The node, the reactions in which it participates and the nodes that participate in these reactions are haloed. Haloing is made by drawing the border behind the node with a larger width and with <em>Profile.properties.halo</em> color.</p>\r\n<p>When a reaction is selected, its segments, texts and borders change color to <em>Profile.properties.selection</em> and segments increase its width. The reaction and its participants are haloed.</p>\r\n<p>When a node is flagged, the border is drawn in the background with a larger width using <em> Profile.properties.flag.</em> At this moment, edges are not being flagged.</p>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/node-layers.png\" alt=\"Node layers\" title=\"Node layers\" /> <figcaption>Representation of layers for a node. From bottom to top: flag, halo, background, analysis, foreground, border and text.</figcaption></figure>\r\n<p>&nbsp;</p>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/edge-layers.png\" alt=\"Edge layers\" title=\"Edge layers\" /> <figcaption>Edge layers. From bottom to top: halo, segments, filling, border and text.</figcaption></figure>\r\n<h2>Analysis</h2>\r\n<p>When an analysis is overlaid, we must modify the color of elements to <em>lighter</em>. Then, those nodes hit by the analysis must render a new layer between the background and the foreground. At first, we must calculate how much a node is hit by the analysis. We can do it by using the graph JSON file and the analysis results (or the analysis token). We must traverse the graph from each node through its children (<em>EntityNode.children) </em>.</p>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/analysis_hits.png\" alt=\"C1 has 1/2 children hit. S1 has 3/4 leaves hit. P1, P3 and P4 have 1/1 leaves hit.\" /> <figcaption>C1 has 1/2 children hit. S1 has 3/4 leaves hit. P1, P3 and P4 have 1/1 leaves hit.</figcaption></figure>\r\n<p>&nbsp;</p>\r\n<p><em>To get a more detailed description of the diagram graph, expand the following tree:</em></p>\r\n<div class=\"frame\">\r\n<ul class=\"tree\">\r\n<li class=\"parent\"><a>Graph</a>\r\n<ul>\r\n<li class=\"parent\"><a>nodes : <strong>EntityNode[]</strong></a>\r\n<ul>\r\n<li>identifier : <strong>String</strong></li>\r\n<li>parents : <strong>Long[]</strong></li>\r\n<li>children : <strong>Long[]</strong></li>\r\n<li>geneNames : <strong>String[]</strong></li>\r\n<li>diagramIds : <strong>Long[]</strong></li>\r\n<li>[<em>GraphNode</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>GraphNode</em>] dbId : <strong>Long</strong></li>\r\n<li>[<em>GraphNode</em>] stId : <strong>String</strong></li>\r\n<li>[<em>GraphNode</em>] speciesID : <strong>Long</strong></li>\r\n<li>[<em>GraphNode</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>edges : <strong>EventNode[]</strong></a>\r\n<ul>\r\n<li>catalysts : <strong>Long[]</strong></li>\r\n<li>inhibitors : <strong>Long[]</strong></li>\r\n<li>activators : <strong>Long[]</strong></li>\r\n<li>inputs : <strong>Long[]</strong></li>\r\n<li>outputs : <strong>Long[]</strong></li>\r\n<li>diagramIds : <strong>Long[]</strong></li>\r\n<li>preceding : <strong>Long[]</strong></li>\r\n<li>following : <strong>Long[]</strong></li>\r\n<li>requirements : <strong>Long[]</strong></li>\r\n<li>[<em>GraphNode</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>GraphNode</em>] dbId : <strong>Long</strong></li>\r\n<li>[<em>GraphNode</em>] stId : <strong>String</strong></li>\r\n<li>[<em>GraphNode</em>] speciesID : <strong>Long</strong></li>\r\n<li>[<em>GraphNode</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>dbId : <strong>Long</strong></li>\r\n<li>stId : <strong>String</strong></li>\r\n<li class=\"parent\"><a>subpathways : <strong>SubpathwayNode[]</strong></a>\r\n<ul>\r\n<li>dbId : <strong>Long</strong></li>\r\n<li>stId : <strong>String</strong></li>\r\n<li>events : <strong>Long[]</strong></li>\r\n<li>displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>ProcessNodes percentages are taken from the <em>SubpathwaySummary</em>: entity.found / entity.total. For the analysis, there is another stylesheet. We draw enrichment using the <em>AnalysisSheet.enrichment.gradient.max</em> color. There are 3 analysis stylesheets: <a href=\"https://github.com/reactome-pwp/diagram/blob/master/src/main/resources/org/reactome/web/diagram/profiles/analysis/profile_01.json\" target=\"_blank\" rel=\"noopener\"> Standard</a>, <a href=\"https://github.com/reactome-pwp/diagram/blob/master/src/main/resources/org/reactome/web/diagram/profiles/analysis/profile_02.json\" target=\"_blank\" rel=\"noopener\"> Copper Plus</a> and <a href=\"https://github.com/reactome-pwp/diagram/blob/master/src/main/resources/org/reactome/web/diagram/profiles/analysis/profile_03.json\" target=\"_blank\" rel=\"noopener\"> Strosobar</a>.&nbsp;</p>\r\n<p><em>To get a more detailed description of the analysis colour profile, expand the following tree:</em></p>\r\n<div class=\"frame\">\r\n<ul class=\"tree\">\r\n<li class=\"parent\"><a>AnalysisProfile</a>\r\n<ul>\r\n<li class=\"parent\"><a>enrichment : <strong>OverlayNode</strong></a>\r\n<ul>\r\n<li>text : <strong>String</strong></li>\r\n<li class=\"parent\"><a>gradient : <strong>ProfileGradient</strong></a>\r\n<ul>\r\n<li>stop : <strong>String</strong></li>\r\n<li>min : <strong>String</strong></li>\r\n<li>max : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>legend : <strong>OverlayLegend</strong></a>\r\n<ul>\r\n<li>hover : <strong>String</strong></li>\r\n<li>median : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>expression : <strong>OverlayNode</strong></a>\r\n<ul>\r\n<li>text : <strong>String</strong></li>\r\n<li class=\"parent\"><a>gradient : <strong>ProfileGradient</strong></a>\r\n<ul>\r\n<li>stop : <strong>String</strong></li>\r\n<li>min : <strong>String</strong></li>\r\n<li>max : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>legend : <strong>OverlayLegend</strong></a>\r\n<ul>\r\n<li>hover : <strong>String</strong></li>\r\n<li>median : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li>ribbon : <strong>String</strong></li>\r\n<li>name : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>If the analysis is an expression analysis, we must take the individual expression values from the leaves. We can show only 1 column at a time. If we want to show them all we must create an animation (play button in the pathway browser and GIFs in the diagram exporter). Once we have the expression values for an element, we divide it into regions and use <em>AnalysisSheet.expression.gradient</em> to calculate the color of each region, using <em>AnalysisResult.expression.max</em> and <em>AnalysisResult.expression.min</em> as limit values. Leaves are sorted using the identifier from the analysis (<em>FoundEntity.id</em>).</p>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/analysis.png\" alt=\"Example of diagram with analysis enrichment\" /> <figcaption>Subsection of a diagram with an analysis overlaid</figcaption></figure>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/expression.gif\" alt=\"Expression\" title=\"Expression\" /> <figcaption>Example of a diagram with an expression analysis</figcaption></figure>\r\n<h2 id=\"elements-order\">Elements order</h2>\r\n<p>From bottom to top, elements must be rendered in the following order:</p>\r\n<ol>\r\n<li>Compartments</li>\r\n<li>Fade out reactions</li>\r\n<li>Fade out nodes</li>\r\n<li>Flags</li>\r\n<li>Halos</li>\r\n<li>Reactions</li>\r\n<li>Nodes</li>\r\n<li>Notes</li>\r\n<li>Shadows</li>\r\n</ol>\r\n<h2 id=\"more-info\">More Information</h2>\r\n<p>To learn more about our pathway diagrams and the techniques we use to render them efficiently please go through our relevant publications:</p>\r\n<ol>\r\n<li><a href=\"https://doi.org/10.1093/bioinformatics/btx752\" target=\"_blank\" rel=\"noopener\">Reactome diagram viewer: data structures and strategies to boost performance</a></li>\r\n<li><a href=\"https://doi.org/10.1093/bioinformatics/btx441\" target=\"_blank\" rel=\"noopener\">Reactome enhanced pathway visualization</a></li>\r\n</ol>\r\n<p>\r\n<script type=\"text/javascript\">var tree = document.querySelectorAll(\'ul.tree a:not(:last-child)\');\r\nfor (var i = 0; i < tree.length; i++) {\r\n    tree[i].addEventListener(\'click\', function (e) {\r\n            var parent = e.target.parentElement;\r\n            while (parent.nodeName !== \"LI\")\r\n                parent = parent.parentElement;\r\n            var classList = parent.classList;\r\n            if (classList.contains(\"open\")) {\r\n                classList.remove(\'open\');\r\n                var opensubs = parent.querySelectorAll(\':scope .open\');\r\n                for (var i = 0; i < opensubs.length; i++) {\r\n                    opensubs[i].classList.remove(\'open\');\r\n                }\r\n            } else {\r\n                classList.add(\'open\');\r\n            }\r\n        }\r\n    );\r\n}</script>\r\n</p>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-02-05 11:47:57','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(102,736,'Version 64 Released','version-64-released','<p><a href=\"/PathwayBrowser/#/R-HSA-8963743\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-8963743.svg\" alt=\"Digest Absorp\" width=\"650\" height=\"408\" /></a></p>\r\n<p><strong>New and Updated Pathways.&nbsp;</strong>In version&nbsp;V64, topics with new or revised pathways include Signal Transduction (<a href=\"PathwayBrowser/#/R-HSA-8939211\">ESR-mediated signaling</a> and <a href=\"PathwayBrowser/#/R-HSA-9006115\">Signaling by NTRK2</a>), Metabolism (<a href=\"PathwayBrowser/#/R-HSA-9018678\">Biosynthesis of specialized proresolving mediators (SPMs)</a>), and Metabolism of proteins (<a href=\"PathwayBrowser/#/R-HSA-9033241&amp;PATH=R-HSA-392499\">Peroxisomal protein import</a>).</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Hanna+Antila\">Hanna Antila</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Azevedo%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28787099\">Jorge Azevedo</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Kumar%20Belani%5BAuthor%20-%20Full%5D\">Kumar Belani</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Gerry%20Boss%5BAuthor%20-%20Full%5D\">Gerry Boss</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Matthew+Brenner%5BAuthor+-+Full%5D\">Matthew Brenner</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Eero+Castr%C3%A9n%5BAuthor+-+Full%5D\">Eero Castrn</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Arthur+Cooper%5BAuthor+-+Full%5D\">Arthur Cooper</a><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Diana+Downs%5BAuthor+-+Full%5D\">, Diana Downs</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Marc+Fransen%5BAuthor+-+Full%5D\">Marc Fransen</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Trond%20Hansen%5BAuthor%20-%20Full%5D\">Trond Hansen</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Hui-Chih+Hung%5BAuthor+-+Full%5D\">Hui-Chih Hung</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Margaret+James%5BAuthor+-+Full%5D\">Margaret James</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Pidder+Jansen-Duerr%5BAuthor+-+Full%5D\">Pidder Jansen-Duerr</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Hideo+Kimura%5BAuthor+-+Full%5D\">Hideo Kimura</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Luca+Magnani%5BAuthor+-+Full%5D\">Luca Magnani</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Steven+Patterson%5BAuthor+-+Full%5D\">Steven Patterson</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Paul++Van+Veldhoven%5BAuthor+-+Full%5D\">Paul&nbsp; Van Veldhoven</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Alexander+Weiss%5BAuthor+-+Full%5D\">Alexander Weiss</a>, and <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Herman+Wolosker+%5BAuthor+-+Full%5D\">Herman Wolosker</a> are our external reviewers.</p>\r\n<p>Illustrations with embedded navigation features are now available for&nbsp;<a href=\"PathwayBrowser/#/R-HSA-8963743\">Digestion and absorption</a>, <a href=\"PathwayBrowser/#/R-HSA-5693532\">DNA Double-Strand Break Repair</a>, <a href=\"PathwayBrowser/#DIAGRAM=9006934&amp;PATH=162582\">Signaling by receptor tyrosine kinases</a>, <a href=\"PathwayBrowser/#/R-HSA-190236&amp;PATH=R-HSA-162582,R-HSA-9006934\">Signaling by FGFR</a>, <a href=\"PathwayBrowser/#/R-HSA-5683057\">MAPK family signaling cascades</a>, <a href=\"PathwayBrowser/#/R-HSA-5683057\">MAPK1/MAPK3 signaling</a>, <a href=\"PathwayBrowser/#/R-HSA-9006936&amp;PATH=R-HSA-162582\">Signaling by TGF-beta family</a>, <a href=\"PathwayBrowser/#/R-HSA-195258&amp;PATH=R-HSA-162582,R-HSA-194315\">RHO GTPase Effectors</a>, <a href=\"PathwayBrowser/#/R-HSA-195721\">Signaling by Wnt</a>, <a href=\"PathwayBrowser/#/R-HSA-5358351\">Signaling by Hedgehog</a>, <a href=\"PathwayBrowser/#/R-HSA-73887\">Death Receptor Signaling</a>, and <a href=\"PathwayBrowser/#DIAGRAM=9006925&amp;PATH=162582\">Intracellular signaling by second messengers</a>.</p>\r\n<p><b>Annotation Statistics.</b>&nbsp;Reactome comprises 11,754 human reactions organized into 2,216 pathways involving 11,030 proteins encoded by 10,762 different human genes, 1,867 small molecules, and 11,561 complexes. These annotations are supported by 28,254 literature references. We have projected these reactions onto 140,720 orthologous proteins, creating 21,223 orthologous pathways in 18 non-human species. Version 64 has annotations for 1,339 protein variants (mutated proteins) and their post-translationally modified forms, derived from 289 proteins,&nbsp;which have been used to annotate disease-specific complexes,&nbsp;reactions and pathways.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome&nbsp;are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence,&nbsp;A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<span id=\"cloak15d567bd0a84d9471f3b57a7075b6514\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-03-20 20:00:21','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,25,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(103,739,'Version 65 Released','version-65-released','<div><a href=\"/PathwayBrowser/#/R-HSA-71387\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-71387.svg\" alt=\"carbohydrate\" width=\"650\" height=\"359\" /></a></div>\r\n<div>\r\n<p><strong>New and Updated Pathways. </strong><span style=\"font-family: arial, helvetica, sans-serif;\"> In version&nbsp;V65, topics with new or revised pathways include<span style=\"color: #222222;\">&nbsp;Immune System (</span><a href=\"/PathwayBrowser/#/R-HSA-8985947&amp;PATH=R-HSA-168256,R-HSA-1280215,R-HSA-449147,R-HSA-451927\" target=\"_blank\">Interleukin-9 signaling</a>)&nbsp;<span style=\"color: #222222;\"></span><span style=\"color: #222222;\">and Signal Transduction (</span><a href=\"/PathwayBrowser/#/R-HSA-9013694&amp;PATH=R-HSA-162582,R-HSA-157118\" target=\"_blank\">Signaling by NOTCH4</a><span style=\"color: #222222;\">&nbsp;and&nbsp;</span><a href=\"/PathwayBrowser/#/R-HSA-9034015&amp;PATH=R-HSA-162582,R-HSA-9006934,R-HSA-166520\" target=\"_blank\">Signaling by NTRK3 (TRKC)</a><span style=\"color: #222222;\">).&nbsp;</span></span><span style=\"font-family: arial, helvetica, sans-serif;\">Illustrations with embedded navigation features are now available for&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-71387&amp;PATH=R-HSA-1430728\" target=\"_blank\">Carbohydrate Metabolism</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-5693538&amp;PATH=R-HSA-73894,R-HSA-5693532&amp;DTAB=AN\" target=\"_blank\">Homology Directed Repair</a>.</span></p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;<span style=\"font-family: arial, helvetica, sans-serif;\"><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Azevedo%20JE[Author]&amp;cauthor=true&amp;cauthor_uid=26235645\">Jorge Azevedo</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Kumar+G+Belani[Author+-+Full]\"></a><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=gomez-outes\">Antonio Gmez-Outes</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Jan%20Haavik%5BAuthor%20-%20Full%5D\">Jan Haavik</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Eun-Kyeong%20Jo%5BAuthor%20-%20Full%5D\">Eun-Kyeong Jo</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=hideo+kimura%5BAuthor+-+Full%5D\"></a><a href=\"https://www.ifc.unam.mx/investigadores/paula-licona\">Paula Licona-Limon</a>, <a href=\"http://www.biochem.utah.edu/rutter/\">Jared Rutter</a>, and&nbsp;<a href=\"https://www.scopus.com/authid/detail.uri?authorId=6604050486\">Pantelis Tsoulfas</a>, are our external reviewers</span><span style=\"font-family: arial, helvetica, sans-serif;\">.</span><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Hanna+Antila\"><br /></a></p>\r\n<p><b>Annotation Statistics.</b>&nbsp;Reactome comprises 11,896 human reactions organized into 2,222 pathways involving 10,935 proteins encoded by 10,763 different human genes, 1,880 small molecules, and 11,674 complexes. These annotations are supported by 28,436 literature references. We have projected these reactions onto 159,163 orthologous proteins, creating 23,450 orthologous pathways in 18 non-human species. Version 65 has annotations for 1,339 protein variants (mutated proteins) and their post-translationally modified forms, derived from 289 proteins,&nbsp;which have been used to annotate disease-specific complexes,&nbsp;reactions and pathways.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome&nbsp;are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence,&nbsp;A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<span id=\"cloak15d567bd0a84d9471f3b57a7075b6514\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span>.</p>\r\n<p></p>\r\n</div>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-06-06 20:19:48','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,24,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(104,741,'Searching Reactome - New Search','searching-reactome-2','','',0,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:50:12','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',6,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(105,742,'Our data now available through Google Dataset Search','our-data-now-available-through-google-dataset-search-engine','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Google_Data_Search.jpg\" alt=\"Google Data Search\" width=\"650\" height=\"382\" /></p>\r\n<p></p>\r\n<p>We are pleased to announce that we are amongst the early adopters of the new <a href=\"https://toolbox.google.com/datasetsearch\">Google Dataset Search</a>.</p>\r\n<p>Throughout the world, there are many thousands of open access data repositories hosted by public institutions, research projects, local and national governments, not-for-profit organisations, and many others. These online resources provide a variety of datasets, which are available in different formats and support many data standards. The Google Dataset Search engine enables easy access to these datasets, so that researchers,&nbsp;scientists, data journalists, data wranglers, or anyone else can quickly find the data for their work.</p>\r\n<p>An example of how Reactome pathway data can be viewed in the Google DataSet Search is available at this<a href=\"https://toolbox.google.com/datasetsearch/search?query=glycosylation\">&nbsp;link</a>.</p>\r\n<p></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-09-06 17:49:49','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,23,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(106,743,'Version 66 Released','version-66-release','<p style=\"text-align: justify;\"><a href=\"/PathwayBrowser/#/R-HSA-9609507\"><span style=\"font-family: arial, helvetica, sans-serif;\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-9609507.svg\" alt=\"protein localization\" width=\"650\" height=\"408\" /></span></a></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"><strong>New and Updated Pathways. </strong>In version 66, topics with new or revised pathways includeDisease (<a href=\"/PathwayBrowser/#/R-HSA-9005891\">Loss of function of MECP2 in Rett syndrome </a>and <a href=\"/PathwayBrowser/#/R-HSA-9605310\">Defective Base Excision Repair Associated with MUTYH</a>), Gene Expression (<a href=\"/PathwayBrowser/#/R-HSA-8986944\">Transcriptional regulation by MECP2</a>), Immune Response (<a href=\"/PathwayBrowser/#/R-HSA-8983711\">OAS antiviral response</a>), Metabolism of proteins (<a href=\"/PathwayBrowser/#/R-HSA-4655427\">SUMOylation of DNA methylation proteins</a>, <a href=\"/PathwayBrowser/#/R-HSA-4755510\">SUMOylation of immune response proteins</a>, <a href=\"/PathwayBrowser/#/R-HSA-4090294\">SUMOylation of intracellular receptors</a>, <a href=\"/PathwayBrowser/#/R-HSA-4085377\">SUMOylation of SUMOylation proteins</a>, <a href=\"/PathwayBrowser/#/R-HSA-3232118\">SUMOylation of transcription cofactors</a>, and <a href=\"/PathwayBrowser/#/R-HSA-3232142\">SUMOylation of ubiquitinylation proteins</a>), and Signal Transduction (<a href=\"/PathwayBrowser/#/R-HSA-9006335\">Signaling by Erythropoietin</a>).</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"><strong>Thanks to our Contributors.</strong> <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=John+Christodoulou[Author]\">John Christodoulou</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Rahul+Krishnaraj[Author]\">Rahul Krishnaraj</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=McGraw%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30072146\">Kathy L McGraw</a>, <a href=\"https://www.researchgate.net/profile/Marco_Meraz-Rios\">Marco Meras-Rios</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Nakabeppu%20Y[Author]&amp;cauthor=true&amp;cauthor_uid=29027306\">Yusaku Nakabeppu</a>, <a href=\"https://www.ncbi.nlm.nih.gov_pubmed_-3Fterm-3DEinari-2B-2BNiskanen-255BAuthor-255D&amp;d=DwMFaQ&amp;c=j5oPpO0eBH1iio48DtsedbOBGmuw5jHLjgvtN2r4ehE&amp;r=GTiX74hn1lHmDPIJuP2ff67KO0GU_fIQLdlnOo0L9MI&amp;m=_wG9tOrPx9pey5oeCeKklHayWUchukMItCF_csf6fP0&amp;s=Pwq0mh72qIyR6p0LxhZFdII2QSkDXxyy6y8xqvMncm4&amp;e=\">Einari Niskanen</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Silverman%20RH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26858407\">Robert H Silverman</a>, and <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=J%C3%BCrgen+Wienands%5BAuthor%5D\">Jrgen Wienands </a>are our external reviewers.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\">New Ilustrations. Illustrations with embedded navigation features are now available for <a href=\"/PathwayBrowser/#/R-HSA-69306\">DNA replication </a>and <a href=\"/PathwayBrowser/#/R-HSA-9609507\">Protein localization</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"><strong>Annotation Statistics.</strong> Reactome comprises 12,047 human reactions organized into 2,244 pathways involving 11,049 proteins encoded by 10,870 different human genes, 1,948 small molecules, and 11,823 complexes. New in this release are annotations of the functions of 139 drugs, 3 of them proteins and 136 small molecules. These annotations are supported by 28,829 literature references. We have projected these reactions onto 139,072 orthologous proteins, creating 21,283 orthologous pathways in 18 non-human species. Version 66 has annotations for 1,391 protein variants (mutated proteins) and their post-translationally modified forms, derived from 293 proteins, which have been used to annotate disease-specific complexes, reactions and pathways.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactomeare distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the <a href=\"/\">Reactome website</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"><strong>Follow us on Twitter: </strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a><strong>&nbsp;</strong>to get frequent updates about new and updated pathways, feature updates, and more!</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt;\"><span style=\"font-family: arial, helvetica, sans-serif;\"><strong>For more information. </strong>Please contact our</span> <span id=\"cloak15d567bd0a84d9471f3b57a7075b6514\"><span id=\"cloakc08bbc767a3f8ec0f8a519cfb552e5e3\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</span></span></span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-09-27 06:25:28','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":\"\",\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":\"\",\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":\"\",\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,22,'','',1,0,'{\"robots\":\"index, follow\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(107,745,'New tool to generate analysis PDF reports','new-tool-to-generate-analysis-pdf-reports','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Analysis_Report_PDF.png\" alt=\"Analysis Report PDF\" width=\"650\" height=\"217\" /></p>\r\n<p>We now offer a new tool to download your analysis results as a single PDF file. This report contains a genome-wide overview, statistics for the most significant pathways and, for each significant pathway, it includes the corresponding diagram image with the analysis overlaid, a summation of the pathway, the related bibliography and the list of identifiers found.</p>\r\n<p>When performing pathway analysis through our <a href=\"/PathwayBrowser/\">PathwayBrowser</a>, you can download the report by clicking on the Report (PDF) button on the bottom-left corner of the Analysis tab. This feature is also available when implementing programmatic access to our <a href=\"/AnalysisService/\">AnalysisService</a> through <a href=\"/AnalysisService/#/report/generatePdfReportUsingGET\">this method</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-07-12 10:51:41','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,21,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(109,747,'New advanced search available in our PathwayBrowser','new-advanced-search-available-in-our-pathway-browser','<p>\r\n<video class=\"responsive-vid\" style=\"display: block; margin-left: auto; margin-right: auto;\" controls=\"controls\" width=\"650\" height=\"auto\" controlslist=\"nodownload\" type=\"video/mpeg\"> Your browser does not support the video tag. <source src=\"images/news/SearchInDiagram.mp4\" type=\"video/mp4\" /></video>\r\n</p>\r\n<p>&nbsp;</p>\r\n<p style=\"text-align: justify;\">Our <a href=\"/PathwayBrowser/\">PathwayBrowser</a> now features advanced search capabilities powered by Solr to allow finding content throughout the whole knowledgebase. The user interface has been improved adapting to the findings of our last UX testing. The search within the Diagram Viewer widget enables users to define the scope of their search either limiting it to the content of the displayed diagram or expanding it to cover all pathways allowing&nbsp;our users to perform a search against all content without having to go the main search.</p>\r\n<p style=\"text-align: justify;\">This new feature has also been enabled in our <a href=\"/dev/diagram/js\">diagram</a> and <a href=\"/dev/pathways-overview/js\">pathways overview</a> widgets so third party web applications can already take advantage of it, allowing users to search Reactome content without abandoning the page they are in.</p>\r\n<p>The new search features:</p>\r\n<ol>\r\n<li>Suggestions based on the introduced term.</li>\r\n<li>Listing the most recent searches.</li>\r\n<li>Scoping results to either the displayed diagram or the whole database.</li>\r\n<li>Filtering results by one or more entity types (i.e. Proteins, Chemical compounds, Reactions, etc.)</li>\r\n<li>Flagging a given entity to persist its highligthing.</li>\r\n</ol>\r\n<p>To learn more, please check the&nbsp;<a href=\"/userguide/searching\">Searching Reactome</a>&nbsp;section&nbsp;in our <a href=\"/userguide\">User Guide</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-01 09:05:45','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,20,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(110,748,'SBGN files revamp','sbgn-files-revamp','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20181004_SBGN_Revamp.png\" alt=\"SBGN Revamp\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">The <a href=\"http://sbgn.github.io/sbgn/\">Systems Biology Graphical Notation (SBGN)</a> project is an effort to standardise the graphical notation used in maps of biological processes. It aims to communicate biological knowledge more efficiently and accurately between different research communities in the life sciences.</p>\r\n<p style=\"text-align: justify;\">Following this commitment, weve recently revamped our SBGN export methods and tools to provide more accurate representation of our pathway diagrams. Users have the option to either export a pathway diagram in SBGN through the <a href=\"/PathwayBrowser/\">PathwayBrowser</a>, or get <a href=\"/download/current/homo_sapiens.sbgn.tar.gz\">all human pathway diagrams</a> from our <a href=\"/download-data\">downloads section</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-03 12:53:19','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,19,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(111,749,'International Open Access Week: Analysis Tools','international-open-access-week-analysis-tools','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20181022_OpenAccess_week_-_Analysis_Service.png\" alt=\"International Open Access week (Analysis Tools)\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">Following up on the International <a href=\"http://www.openaccessweek.org/\">Open Access week</a>,&nbsp;we would like to remind our users that all our tools, including the <a href=\"/user/guide/analysis\">analysis</a>, are open access.</p>\r\n<p style=\"text-align: justify;\">Pathway analysis methods have a broad range of applications in physiological and biomedical research. Our analysis suite currently implements an overrepresentation analysis, an expression data analysis and a species comparison tool. Using this service, users can submit their sample (list of identifiers) to get as result the most significants pathways. Results are overlaid in the different modules of our Pathway Browser and can be exported to different formats including a <a href=\"/about/news/107-new-tool-to-generate-analysis-pdf-reports\">PDF report</a>.</p>\r\n<p style=\"text-align: justify;\">A light-weight client is integrated in our&nbsp;<a href=\"/PathwayBrowser/#TOOL=AT\">Pathway Browser</a>. The tool suite is available via a <a href=\"/AnalysisService/\">RESTFul Web Service</a>&nbsp;so all the available analysis tools can be easily integrated into third party software. More documentation for developers is available at our <a href=\"/dev/analysis\">developer\'s zone</a><a href=\"https://reactome.org/dev/analysis.\">.</a></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-23 08:25:02','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,18,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(112,750,'International Open Access Week: Network Visualization & Analysis Tool','international-open-access-week-network-analysis-tools','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20181024_OpenAccess_week_-_ReactomeFIViz_2.png\" alt=\"20181024 OpenAccess week ReactomeFIViz 2\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">As part of the International&nbsp;<a href=\"http://www.openaccessweek.org/\">Open Access week</a>,&nbsp;we would like to talk about another one of our open access network visualization and analysis tool  the <a href=\"/tools/reactome-fiviz\">Reactome FIViz app</a>.</p>\r\n<p style=\"text-align: justify;\">In order to improve our understanding of disease mechanisms and develop better personalized precision therapies for patients, many biological and clinical studies employ high-throughput techniques that generate large-scale data sets. Typically, these data sets are gene- or protein-based, and to better understand the relationships among interesting genes or proteins, researchers usually have to project them onto biological network contexts to provide holistic visualization and analysis platform for reducing the dimensionality of data using network modules.</p>\r\n<p style=\"text-align: justify;\">To assist our users who would like to perform network-based analysis, we have constructed the Reactome Functional Interaction (FI) network which, covers 60% of the total human protein-coding genes, and was created by extracting interactions from manually curated pathways and predicting interactions based on a machine learning technique. We have developed the&nbsp;<a href=\"https://cytoscape.org/\">Cytoscape </a>application (or app), called the ReactomeFIViz that uses this highly reliable FI network to support network-based data visualization and analysis. Users of our app can construct an FI subnetwork for a list of genes, perform network clustering to find network modules, annotate the subnetwork and modules, and perform survival analysis for network modules. Furthermore, the app can also perform pathway enrichment analysis using a gene score file, and pathway mathematical modeling based on probabilistic graphical models and Boolean networks. More documentation about the ReactomeFIViz app is available through our <a href=\"/tools/reactome-fiviz\">User Guide</a>&nbsp;and the <a href=\"http://apps.cytoscape.org/apps/reactomefiplugin\">Cytoscape App Store</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-23 16:54:11','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,17,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(113,751,'International Open Access Week: Interactive illustrations and icon library','oaweek-interactive-illustrations-icons','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20181025_OpenAccess_week_-_EHLD_-_Icon_Library.png\" alt=\"OpenAccessWeek Textbook-like illustrations and icon library\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">Textbook-like illustrations aim to improve the graphical representation of higher-level pathways in the Reactome events hierarchy, e.g., <a href=\"/PathwayBrowser/#/R-HSA-162582\">signal transduction</a>, <a href=\"/PathwayBrowser/#/R-HSA-109581\">apoptosis</a>, or <a href=\"/PathwayBrowser/#/R-HSA-392499\">metabolism of proteins</a> whose pathway diagrams consisted of green boxes labeled with the names of sub-events, optionally located in cellular compartments and connected by arrows. These green-box diagrams feature limited navigation: clicking on a green box takes the user to that sub-event.</p>\r\n<p style=\"text-align: justify;\">There was a general agreement that green-box diagrams are not that appealing, and put off users accustomed to textbook-quality illustrations of biological processes with striking, intuitively clear iconography. The project includes generation of scalable vector graphic (SVG) versions of illustrations, and a diagram module that makes these images interactive by enabling actions such as hovering over the items or selecting them to show the associated content in the details panel.</p>\r\n<p style=\"text-align: justify;\">Developing this project has also driven the creation of the <a href=\"/icon-lib\">Icon Library</a>; a consistent iconography compendium that ranges from simple protein labels to representations of organelles, receptors and cell types. The library has now been integrated in the main search as well as in the pages of their associated entities such as proteins or chemicals. Icons can be found by their name, description, designer and/or contributor.</p>\r\n<p style=\"text-align: justify;\">The Icon Library is freely accessible (under a <a href=\"https://creativecommons.org/licenses/by/4.0/deed.ast\">CC-BY 4.0 licence</a>) and it is suitable for a broad range of purposes, from schematic pathway sketches in scientific presentations and publications to grant proposal illustrations. As the library was created to be a community resource, the invitation for third parties to contribute is still open. Aiming to achieve technical and artistic consistency, detailed guidelines are provided at <a href=\"https://reactome.org/icon-info.\">https://reactome.org/icon-info.</a> To acknowledge the community engagement, each icon is attributed to the author through a metadata file linking to a portfolio and/or ORCID id. As of September 2018, the library has considerably grown to 1,150 components.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-25 08:19:18','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,16,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(114,752,'International Open Access Week: GraphDB and Content Service','graphdb-content-service','<p style=\"text-align: justify;\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20181026_OpenAccess_week_-_GraphDB_ContentService.png\" alt=\"OpenAccess Week: GraphDB and ContentService\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">Since version 57 we provide our data in a <a href=\"https://neo4j.com/\">Neo4j</a>&nbsp;<a href=\"/dev/graph-database\">graph database</a>&nbsp;helping to reduce the complexity of the represented knowledgebase and allowing a more straightforward access to our content. Neo4js query language, Cypher, allows queries to be written in a more intuitive way and reduces the average response time per query by 93% (<a href=\"https://doi.org/10.1371/journal.pcbi.1005968\">Fabregat et al., 2018</a>).</p>\r\n<p style=\"text-align: justify;\">The graph database also benefits Reactome in other aspects like (i) the creation of complex data quality assessment (QA) queries or (ii) supporting software that requires data pattern analysis, e.g. <a href=\"https://github.com/reactome/rxn-classifier\">reactions classification</a>. QA queries are executed during each quarterly release to identify instances that need to be checked out and corrected by Reactome curators ensuring that high-quality content is delivered to the final user. The reactions classifier project, for example, uses Cypher to formalise the concepts presented in <a href=\"https://doi.org/10.1093/database/bau060\">Jupe et al. (2014)</a> to generate a series of reports that help curators classify the reactions in Reactome.</p>\r\n<p style=\"text-align: justify;\">The <a href=\"https://reactome.org/dev/content-service\">Content Service</a>&nbsp;constitutes an easy API, based on the Representational State Transfer (REST) protocol, that provides access to the Reactome knowledgebase. It includes a set of methods classified in groups according to their functionality. For instance, expanding the pathways group reveals a set of methods that provide specific information about pathways such as the contained Events or the participating PhysicalEntities.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-26 08:21:29','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,15,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(115,753,'Performing GSEA analysis with ReactomeFIViz app','performing-gsea-analysis-with-reactomefiviz-app','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/ReactomeFIViz_GSEA.png\" alt=\"ReactomeFIViz GSEA\" width=\"650\" height=\"366\" /></p>\r\n<p class=\"p1\" style=\"text-align: justify;\">ReactomeFIViz is a Cytoscape app built upon Reactome pathways to help users perform pathway- and network-based data analysis and visualization. One of the most popular approaches to pathway analysis, which is an alternative to the traditional gene-list based pathway enrichment, is Gene Set Enrichment Analysis (GSEA). GSEA considers all genes with their scores based on a weighted KolmogorovSmirnov-like test and is a&nbsp;representative approach of second-generation pathway analysis. ReactomeFIViz implements features to perform GSEA analysis using Reactome pathways for a gene score file.&nbsp;</p>\r\n<p class=\"p1\" style=\"text-align: justify;\">More details about the GSEA feature and the ReactomeFIViz app can be found <a href=\"https://reactome.org/tools/reactome-fiviz\">here</a></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-11-01 14:00:04','0000-00-00 00:00:00','{}','{}','{}',1,14,'','',1,0,'{}',0,'en-GB','',''),(116,754,'Version 67 Released','version-67-released','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/ER_to_Golgi.png\" alt=\"ER to Golgi\" width=\"650\" height=\"374\" /></p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Pathways.&nbsp;</strong>In version&nbsp;V67, topics with new or revised pathways include Gene Expression (<a href=\"/PathwayBrowser/#R-HSA-9614085\">FOXO-mediated transcription</a>), Immune Response (<a href=\"/PathwayBrowser/#/R-HSA-1222556\">ROS, RNS production in phagocytes</a>), Neuronal System (<a href=\"/PathwayBrowser/#/R-HSA-442755\">Activation of NMDA receptors and postsynaptic events</a>), Programmed Cell Death (<a href=\"/PathwayBrowser/#/R-HSA-111471\">Apoptotic factor-mediated response</a>), and Signal Transduction (<a href=\"/PathwayBrowser/#/R-HSA-388396\">GPCR downstream signaling</a> and <a href=\"/PathwayBrowser/#/R-HSA-5668599\">RHO GTPase activate NADPH oxidases</a>).</p>\r\n<p style=\"text-align: justify;\"><strong>New Illustrations.</strong> Illustrations with embedded navigation features are now available for <a href=\"/PathwayBrowser/#/R-HSA-199977\">ER to Golgi Anterograde Transport</a>, <a href=\"/PathwayBrowser/#/R-HSA-9036866\">Expression and Processing of Neurotrophins</a>, and <a href=\"/PathwayBrowser/#/R-HSA-166520\">Signaling by NTRKs</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors.</strong>&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Enrico+Bertaggia%5BAuthor%5D\">Enrico&nbsp; Bertaggia</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Timothy+Donlon%5BAuthor%5D\">Timothy Donlon</a>, <a href=\"https://www.ncbi.nlm.nih.gov/myncbi/browse/collection/40443499/?sort=date&amp;direction=descending\">Kasper Hansen</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Oliver+N%C3%BCsse+%5BAuthor%5D\">Oliver Nsse</a>, and <a href=\"https://scholar.google.com/citations?user=OyPga9EAAAAJ&amp;hl=en\">Feng Yi</a> are our external reviewers.</p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong> Reactome comprises 12,788 human reactions organized into 2,256 pathways involving 11,066 proteins and modified forms of proteins encoded by 10,792 different human genes, 1,827 small molecules, and 155 drugs. These annotations are supported by 29,454 literature references. We have projected these reactions onto 139,298 orthologous proteins, creating 21,374 orthologous pathways in 18 non-human species. Version 67 has annotations for 1,564 protein variants (mutated proteins) and their post-translationally modified forms, derived from 299 proteins. These have been used to annotate 506 complexes and 962 disease-specific reactions organized into 467 pathways and subpathways, and tagged with 342 Disease Ontology terms.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong>&nbsp;<a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information.&nbsp;</strong>Please contact our&nbsp;<span id=\"cloak15d567bd0a84d9471f3b57a7075b6514\"><span id=\"cloakc08bbc767a3f8ec0f8a519cfb552e5e3\"><span id=\"cloak8926181d9df5dd2e3fcde3c0c30adf60\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span>.</span></span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-12-10 20:18:07','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,13,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(117,755,'Icon Mapping','icon-mapping','<p ><span>The mapping files consist of a tab-separated table that indicates the external molecule or ontology identifiers in the source database that were mapped to elements of the icon library. Our goal with distributing different identifier type is to provide accurate representation of the different icon library components, e.g. proteins, chemicals, tissues, cell, etc. These mapping files link the source database identifier to:</span></p>\r\n<p ><span>The columns within the mappings files follow a similar format:</span></p>\r\n<ol>\r\n<li><span>Source database identifier, e.g. UniProt, PFAM, Uberon, or ChEBI identifier</span></li>\r\n<li><span>Icon Stable Identifier</span></li>\r\n<li><span>Icon Name</span></li>\r\n</ol>\r\n<p><span>If you have any further questions, please&nbsp;<a href=\"mailto:help@reactome.org?subject=PE%20Identifier%20Mapping%20File\"><span>contact us</span></a>.</span></p>','',1,32,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-12-11 03:32:58','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(118,756,'Icons Mapping Download','icons-mapping-download','<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/BTO2Icon.txt\">BTO to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/CHEBI2Icon.txt\">ChEBI to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/CL2Icon.txt\">CL to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/ENA2Icon.txt\">ENA to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/GO2Icon.txt\">GO to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/INTERPRO2Icon.txt\">INTEPRO to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/KEGG2Icon.txt\">KEGG to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/MESH2Icon.txt\">MESH to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/OMIT2Icon.txt\">OMIT to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/PFAM2Icon.txt\">PFAM to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/PUBCHEM2Icon.txt\">PUBCHEM to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/RFAM2Icon.txt\">RFAM to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/SO2Icon.txt\">SO to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/UBERON2Icon.txt\">UBERON to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/UNIPROT2Icon.txt\">UNIPROT to icons</a></li>\r\n</ul>','',1,32,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-12-12 11:43:47','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(119,760,'New reactions layout with nested compartments','new-reactions-layout-with-nested-compartments','<p style=\"text-align: justify;\"><a href=\"/content/detail/R-HSA-164834\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/logo/20190109_Reaction_preview.png\" alt=\"Reactions automatic layout with nested compartments\" width=\"650\" height=\"382\" /></a></p>\r\n<p style=\"text-align: justify;\">Up until now, we have placed existing regular pathway diagrams and <a href=\"https://doi.org/10.1093/bioinformatics/btx441\" target=\"_blank\" rel=\"noopener\">textbook-style illustrations</a>&nbsp;in our pathway detailed pages (e.g. <a title=\"Open &quot;Hemostasis&quot;\" href=\"/content/detail/R-HSA-109582\" target=\"_blank\">Hemostasis</a>, <a title=\"Open &quot;Platelet Adhesion to exposed collagen&quot;\" href=\"/content/detail/R-HSA-75892\" target=\"_blank\">Platelet Adhesion to exposed collagen</a>), leaving single reactions as the only type of events without a self-contained image.&nbsp;To fill the gap, an automatic algorithm to deterministically lay reactions out has been developed. It uses data directly from our <a href=\"/dev/graph-database\">graph-database</a> to generate images without any kind of human intervention.</p>\r\n<p style=\"text-align: justify;\">Long story short, the algorithms strategy places inputs on the left, outputs on the right, catalysts on top and regulators at the bottom. Each element is placed in its corresponding compartment, and these are nested following the&nbsp;<a href=\"https://doi.org/10.1038/75556\" target=\"_blank\" rel=\"noopener\">Gene Ontology hierarchy</a>. The algorithm supports normal and disease* reactions minimizing the space as well as avoiding lines to cross unnecessary elements in the display.</p>\r\n<p style=\"text-align: justify;\">Finally, the reaction visualisation complies with the <a href=\"https://doi.org/10.1038/nbt.1558\" target=\"_blank\" rel=\"noopener\">Systems Biology Graphical Notation (SBGN)</a>.</p>\r\n<p style=\"text-align: justify;\">Some examples:</p>\r\n<ol style=\"text-align: justify;\">\r\n<li style=\"text-align: justify;\"><a title=\"Open &quot;Enzyme-bound ATP is released&quot;\" href=\"/content/detail/R-HSA-164834\" target=\"_blank\">Enzyme-bound ATP is released</a></li>\r\n<li style=\"text-align: justify;\"><a title=\"Open &quot;VCP-catalyzed ATP hydrolysis promotes the translocation of Hh-C into the cytosol&quot;\" href=\"/content/detail/R-HSA-5362459\" target=\"_blank\">VCP-catalyzed ATP hydrolysis promotes the translocation of Hh-C into the cytosol</a></li>\r\n<li style=\"text-align: justify;\"><a title=\"Open &quot;Ubiquitination of PAK-2p34&quot;\" href=\"/content/detail/R-HSA-211734\" target=\"_blank\">Ubiquitination of PAK-2p34</a></li>\r\n<li><a title=\"Open &quot;Defective ABCD1 does not transfer LCFAs from cytosol to peroxisomal matrix&quot;\" href=\"/content/detail/R-HSA-5684043\" target=\"_blank\">Defective ABCD1 does not transfer LCFAs from cytosol to peroxisomal matrix</a></li>\r\n</ol>\r\n<p style=\"text-align: justify;\">*<a href=\"/user/guide/diseases\" target=\"_blank\">Disease reactions</a> can either be&nbsp;classified as infectious, gain-of-function or lost-of-function. In the latter category some participants have to be&nbsp;crossed out.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-01-09 13:52:43','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,12,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(120,761,'New tool to classify pathways into tissue types based on protein expression','classifying-pathways-into-tissue-types-based-on-protein-expression','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20190114_Tissue_distribution.png\" alt=\"Tissue distribution analysis\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">Pathways in Reactome are curated in a generic cell and are agnostic to tissue types. However, different cell types have different functional requirements and consequently the underlying pathway activities also vary. Studying pathways in a tissue-specific manner will help to understand the biology in context.</p>\r\n<p style=\"text-align: justify;\">Reactome now boasts a <a href=\"/PathwayBrowser/#TOOL=AT\">tool</a> that can categorize pathways into different tissue types based on protein expression. We overlay tissue-specific protein expression data from the <a href=\"https://www.ebi.ac.uk/gxa/home\">ExpressionAtlas</a> database on empirically validated pathway information in Reactome. This facilitates the sorting of pathways from a generic cell to different tissue-types. This new feature allows users to select an experiment and analyse Reactome pathways in different tissues.</p>\r\n<p style=\"text-align: justify;\">To try it out, select the \"Tissue Distribution\" tab, choose the tissues of your interest and click the \"Go\" button. The results are overlaid in the pathways overview and pathway diagrams similar to other analysis types. Users can cycle through the selected tissues via the small control panel displayed at the bottom of the viewport.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-01-15 11:02:06','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,11,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(121,762,'Download our biological pathway knowledge in a PDF book','new-pdf-book','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20190130_Reactome_book.png\" alt=\"Reactome book\" width=\"650\" height=\"383\" /></p>\r\n<p style=\"text-align: justify;\">Aiming to make our biological pathway knowledge more accessible, we\'ve been offering for years the option to download the complete content of our knowledgebase as a PDF book. Following users feedback and requests, this book has now been updated to include yet more content in a freshly redesigned look and feel. Organised in 27 volumes, one per each top level pathway, the new version of the book includes our textbook-like lilustrations as well as regular-pathway&nbsp;and single-reaction diagrams.</p>\r\n<p style=\"text-align: justify;\">In some cases the preferred option might be downloading part of Reactome\'s content. To do so we\'ve included this option in our detail pages, the PathwayBrowser and the DiagramViewer.&nbsp;When downloaded from the PathwayBrowser, the resulting PDF will include the user\'s predefined colour profiles (diagram and analysis overlay) and will also include the analysis results when available. In all cases, resulting documents&nbsp;are fully navigable, including links to following and preceding events or any of the parent pathways in the hierarchy. Literarutre references include links to PubMed and every event has a link to its details page in Reactome\'s site.</p>\r\n<p style=\"text-align: justify;\">The full book can be downloaded from our <a href=\"/download-data\">download</a> section. For an example of the on-demand download please visit the&nbsp;<a href=\"/content/detail/R-HSA-109582\">Hemostasis details page</a>&nbsp;or <a href=\"/PathwayBrowser/#/R-HSA-109582\">its view in the PathwayBrowser</a>.&nbsp;All the above can be programmatically generated using&nbsp;<a href=\"/ContentService/#/exporter/eventPdfUsingGET\">this method</a>&nbsp;from our <a href=\"/ContentService\">ContentService</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-01-30 15:26:26','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,10,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(124,765,'Deprecated support for Reactome RESTful API','deprecated-support-for-reactome-restful-api','<p>As of Version 68 (March 2019), the RESTful API is deprecated and will be superceded by our ContentService. The ContentService is currently available through our website at <a href=\"https://reactome.org/ContentService/\">https://reactome.org/ContentService/</a> and is designed for bioinformaticians, computer scientists, and software developers to access our pathway data. See more details about this API at <a href=\"https://reactome.org/dev/content-service.\">https://reactome.org/dev/content-service. </a></p>\r\n<p>If you have questions or concerns regarding this announcement, please email <a href=\"mailto:help@reactome.org.\">help@reactome.org.</a></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-03-21 16:12:39','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,9,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(125,766,'Version 68 Released','version-68-of-reactome-is-now-online','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Signaling_by_Nuclear_Receptors.png\" alt=\"Signaling by Nuclear Receptors\" width=\"650\" height=\"384\" /></p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Pathways. </strong>In version&nbsp;V68, topics with new or revised pathways include Disease (<a href=\"/PathwayBrowser/#/R-HSA-9616333\">Defective Base Excision Repair Associated with NTHL1</a>, <a href=\"/PathwayBrowser/#/R-HSA-9616334\">Defective Base Excision Repair Associated with NEIL1</a>, and <a href=\"/PathwayBrowser/#/R-HSA-9629232\">Defective Base Excision Repair Associated with NEIL3</a>), Metabolism (<a href=\"/PathwayBrowser/#/R-HSA-9033658\">Blood group systems biosynthesis</a>), Neuronal System (<a href=\"/PathwayBrowser/#/R-HSA-9609736\">Assembly and cell surface presentation of NMDA receptors</a>), Protein localization (<a href=\"/PathwayBrowser/#/R-HSA-9603798\">Class I peroxisomal membrane protein import</a> and <a href=\"/PathwayBrowser/#/R-HSA-9609523\">Insertion of tail-anchored proteins into the endoplasmic reticulum membrane</a>), and Signal Transduction (<a href=\"/PathwayBrowser/#/R-HSA-9009391\">Non-genomic estrogen signaling</a>).</p>\r\n<p style=\"text-align: justify;\"><strong>New Illustrations.</strong> Illustrations with embedded navigation features are now available for <a href=\"/PathwayBrowser/#/R-HSA-9006927&amp;PATH=R-HSA-162582&amp;DTAB=AN\">Signaling by Non-Receptor Tyrosine Kinases</a>, <a href=\"/PathwayBrowser/#/R-HSA-9006931&amp;PATH=R-HSA-162582&amp;DTAB=AN\">Signaling by Nuclear Receptors</a>, and <a href=\"/PathwayBrowser/#/R-HSA-3108232&amp;PATH=R-HSA-392499,R-HSA-597592,R-HSA-2990846\">SUMO E3 ligases SUMOylate target proteins</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors.</strong>&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Bhattacharya%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30479490\">Subhrajit Bhattacharya</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Chad+R+Camp%5BAuthor%5D\">Chad R Camp</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Richarda+de+Voer%5BAuthor%5D\">Richarda&nbsp;de Voer</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=evelina+de+laurentiis%5Bauthor%5D\">Evelina DeLaurentis</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Paul+W+Doetsch%5BAuthor%5D\">Paul W Doetsch</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Akos+Farkas%5BAuthor+-+Full%5D\">kos Farkas</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=marc+fransen%5BAuthor+-+Full%5D\">Marc Fransen</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Roland++Kuiper%5BAuthor%5D\">Roland Kuiper</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Ellis%20R%20Levin%5BAuthor%20-%20Full%5D\">Ellis R Levin</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Taei%20%20Matsui%5BAuthor%20-%20Full%5D\">Taei Matsui</a>, <a href=\"https://scholar.google.com/citations?user=Aj651BgAAAAJ&amp;hl=en\">Huaiyu Mi</a>, <a href=\"https://www.ncbi.nlm.nih.gov/sites/myncbi/1XouqVfFZvAke/bibliography/44215904/public/?sort=date&amp;direction=ascending\">Barbara Rivera</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Sampath%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30291284\">Harini Sampath</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Blanche+Schwappach%5BAuthor+-+Full%5D\">Blanche Schwappach</a>, <a href=\"https://orcid.org/0000-0002-4650-631X\">Ralf Stephan</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Stephen++F+Traynelis%5BAuthor+-+Full%5D\">Stephen F Traynelis</a>, and <a href=\"https://www.ncbi.nlm.nih.gov/myncbi/browse/collection/54269459/\">Jia Zhou</a> are our external reviewers.</p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong>&nbsp;Reactome comprises 12,416 human reactions organized into 2,255 pathways involving 11,000 proteins and modified forms of proteins encoded by 10,825 different human genes, 1,854 small molecules, and 202 drugs. These annotations are supported by 29,885 literature references. We have projected these reactions onto 81,875 orthologous proteins, creating 18,505 orthologous pathways in 15 non-human species. Version 68 has annotations for 1,416 protein variants (mutated proteins) and their post-translationally modified forms, derived from 304 proteins. These have been used to annotate 532 complexes and 966 disease-specific reactions organized into 472 pathways and subpathways, and tagged with 349 Disease Ontology terms.</p>\r\n<p style=\"text-align: justify;\"><strong>Tools and Data. </strong>Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"/PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"/download-data\">download</a>, and to access programmatically through our <a href=\"/ContentService/\">Content</a> and <a href=\"/AnalysisService/\">Analysis</a> Services. The <a href=\"/tools/reactome-fiviz\">ReactomeFIViz app</a> for Cytoscape provides tools to find pathways and network patterns related to cancer and other types of diseases.</p>\r\n<p style=\"text-align: justify;\"><strong>Documentation and Training</strong>. Visit our online <a href=\"/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"/community/training\">here</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the&nbsp;Reactome <a href=\"/\">website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong>&nbsp;<a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information</strong>: If you have a question to ask or would like to give us your feedback, please contact our&nbsp;<a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-03-21 16:13:28','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,8,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(126,768,'PANTHER resource now used to identify orthologs of curated human proteins','panther-resource-now-used-to-identify-orthologs-of-curated-human-proteins','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Orthology.png\" alt=\"Orthology\" width=\"650\" height=\"366\" /></p>\r\n<p style=\"text-align: justify;\">Biological processes are remarkably well-conserved over large evolutionary distances. At a practical level, this fact enables the extrapolation of mechanistic insights from species to species. The Reactome project aims to annotate the molecular details of a broad range of human biological processes based on experimental data from human systems. To link these human annotations at a molecular level to their conserved counterparts in model organism systems we use protein sequence orthology relationships to ask, for each human reaction and each model organism, whether the human proteins involved in the reaction have orthologs in the model organism. If the orthologs exist, we computationally infer the corresponding reaction for the model organism and in this way build up a predicted pathway knowledgebase for the organism. If the human protein functions as part of a complex, we search for orthologs of all components of the complex, and computationally infer the existence of the complex in the model organism if model organism counterparts of at least 75% of the human proteins are found.</p>\r\n<p style=\"text-align: justify;\">With our March, 2019 (version 68) release, we have made two changes to improve the quality and usability of these inferred pathways.</p>\r\n<p style=\"text-align: justify;\">First, with the development of Plant Reactome, a substantial body of rice (Oryza sativa) pathways annotated from plant experimental evidence is now available online. That material improves on orthology-based inferences from human data, and provides a better starting point for making such inferences to other plant species, so all inferences for plant species will now be generated, maintained, and made available through <a href=\"http://plantreactome.gramene.org/index.php?lang=en\">Plant Reactome</a>.</p>\r\n<p style=\"text-align: justify;\">Second, we are now using the <a href=\"http://pantherdb.org\">PANTHER</a> resource (Protein Analysis Through Evolutionary Relationships) to identify model organism orthologs of human proteins annotated in Reactome. This change will allow us to exploit features of PANTHER, such as the identification of least-diverged orthologs in model organism protein families, to improve the specificity of our inferences. The change is also part of a larger project to better align the Reactome with the Gene Ontology, PANTHER, the Alliance of Genome Resources, and related resources to generate interactive, expert-curated, actively maintained, and tightly integrated community genomics resources.</p>\r\n<p></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-03-21 16:14:15','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,7,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(127,769,'Come to our release 68 webinar to learn more about Reactome','come-to-our-release-68-webinar-to-learn-more-about-reactome','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Webinar.png\" alt=\"Webinar\" width=\"650\" height=\"366\" /></p>\r\n<p style=\"text-align: justify;\">We are hosting a webinar aimed at new Reactome users who would like to get an idea of our data and tools and at existing Reactome users who would like to learn about the latest Reactome updates. Reactome is a&nbsp;freely available curated database of human biological pathways and reactions, which is updated on a regular basis. Release 68 features new and updated pathway and reaction&nbsp;annotations for 16 species, as well as a suite of tools for data analysis and visualization.</p>\r\n<p style=\"text-align: justify;\">Event Date: 5th April 2019.<br />Event Time: 11.00 a.m. to 12.00 p.m. EDT.<br />Event Contact: Robin Haw at <a href=\"mailto:help@reactome.org\">Help&nbsp;Mailing List</a>.</p>\r\n<p style=\"text-align: justify;\">Requisites: This webinar will be hosted using <a href=\"https://www.webex.ca/en/\">Cisco WebEx</a>. You will need a computer, an Internet connection and a telephone/Skype connection/microphone to join an online session.</p>\r\n<p style=\"text-align: justify;\">Please register at&nbsp;<a href=\"https://forms.gle/QzeG85hcfrX34ivUA\">https://forms.gle/QzeG85hcfrX34ivUA</a></p>\r\n<p style=\"text-align: justify;\">When you register, please make sure to provide your name and valid email address so that we can send you an email with the webinar details and instructions.</p>\r\n<p style=\"text-align: justify;\">Feel free to pass this invitation along to colleagues who may benefit from learning about this valuable resource.</p>\r\n<p style=\"text-align: justify;\"></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-03-28 18:08:26','0000-00-00 00:00:00','{}','{}','{}',1,6,'','',1,0,'{}',0,'en-GB','',''),(129,773,'EHLD Specs & Guidelines','ehld-guidelines','<p><span>The Enhanced High Level Diagrams (EHLD) project aims to improve the graphical representation of higher-level pathways in the Reactome events hierarchy, e.g., signal transduction, apoptosis, or metabolism whose pathway diagrams consist of green boxes labeled with the names of sub-events, optionally located in cellular compartments and connected by arrows. These green-box diagrams feature limited navigation: clicking on a green box takes the user to that sub-event. In some cases, an illustrator has also created a graphic representation of the event. However, these illustrator diagrams offer no navigation functionality.</span></p>\r\n<p><span>There was a general agreement that green-box diagrams are not that appealing, and put off users accustomed to textbook-quality illustrations of biological processes with striking, intuitively clear iconography. Meanwhile, while our illustrations often are up to textbook standards, they are static images. The project included generation of scalable vector graphic (SVG) versions of illustrations, and the development of software that will make these SVG images navigable in the way that the green-box figures are now, and also to provide user interactivity by progressive zoom in/out, actions when hovering over the items or selecting them and showing the associated content in the details panel for the selected items.</span></p>\r\n<p><span>This project has also driven the development of a consistent iconography, a controlled visual vocabulary for biological processes like the controlled word vocabularies we already have for names of physical entities and reactions.</span></p>\r\n<p><span>This document includes two main sections. The first section describes in detail the process that should be followed to create an EHLD. The second section of this document provides guide through the process of creating and including new graphic elements into the Reactome EHLD icons library. </span></p>\r\n<h3><span>Enhanced High Level Diagrams guidelines</span></h3>\r\n<p><span>This section aims to guide you through the process of creating an EHLD. The following paragraphs aim to summarise the set of requirements and guidelines the illustrator should follow in order to produce EHLDs that are compatible with Reactome Pathway Browser v3.4.</span></p>\r\n<h4><span>Generic guidelines</span></h4>\r\n<p><span>In Reactome, an EHLD is an interactive graphical representation of a higher level pathway diagram, typically containing two or more subpathways as active regions. An active region refers to a group of shapes representing a single subpathway. In particular, users can interact with active regions by hovering or selecting them, as well as use them to navigate to the subpathways they represent.</span></p>\r\n<p><span> &nbsp;</span><span><img src=\"images/ehldguide/ehld-with-analysis.png\" alt=\"ehld with analysis\" width=\"602\" height=\"386\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></span></p>\r\n<p style=\"text-align: center;\"><em>EHLD representing the Hemostasis branch of the Reactome event hierarchy.&nbsp;(a) Pathway hierarchy view for Hemostasis. (b) Hemostasis as an EHLD representation. (c) Hemostasis EHLD overlaid with pathway enrichment analysis results.</em></p>\r\n<h3>Generic guidelines</h3>\r\n<p><span>To make all these possible, there is a set of initial requirements to be taken into account during the generation of the EHLDs.</span></p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li><span>EHLDs must be in SVG format. Illustrators are free to use the design tool of their preference but at the end their diagram must be exported to SVG.</span></li>\r\n<li><span>The use of raster graphics is strongly discouraged as it results in larger file sizes and it has a negative impact on software performance. Also, by including bitmap images we cancel out resolution-independent zooming, which is one of SVGs main advantage. &nbsp;</span></li>\r\n<li><span>All exported SVG files should include the styles in an internal CSS stylesheet. Please make use of this option when exporting from Adobe Illustrator: </span><i><span>Export As -&gt; SVG -&gt; Styling: Internal CSS</span></i><span>. If you are using another application to create the EHLDs, you will need to configure it appropriately.</span></li>\r\n<li><span>The name of each EHLD file has to be the Reactome Identifier of the corresponding pathway diagram, followed by .svg. For example, the </span><i><span>Apoptosis</span></i><span> EHLD should be named as </span><i><span>R-HSA-109581.svg</span></i><span>. Please keep in mind that only one EHLD should be created per high level pathway diagram.</span></li>\r\n<li><span>Active regions are annotated using the id attribute of their group element (inside the SVG file) and can be classified in the following two groups: </span>\r\n<ul style=\"list-style-type: circle;\">\r\n<li><span>Regions containing a group of irregular shapes (including text) that can be selected (and hovered) in order to navigate to the respective pathway diagram. These regions should be annotated by setting their id attribute to </span><i><span>REGION-</span></i><span> </span><i><span>+ Reactome Identifier</span></i><span> of their represented subpathway, e.g. </span><i><span>REGION-R-HSA-109581</span></i><span>. This type of active regions may include arrows pointing to or from them and they are not mandatory.</span></li>\r\n<li><span>Regions containing the label of the represented subpathway in a box-like shape that can be not only selected but also overlaid with the results of the analysis. These regions should be annotated by setting their id attribute to </span><i><span>OVERLAY-</span></i><span> </span><i><span>+ Reactome Identifier</span></i><span> of their represented subpathway, e.g. </span><i><span>OVERLAY-R-HSA-109581</span></i><span>. Regions of this category must not include arrows pointing to or from them, and they are mandatory.</span></li>\r\n<li><span>Please keep in mind that if we need to annotate both types of active regions for a given subpathway, the region (REGION-R-HSA-109581) has to include the overlay component (OVERLAY-R-HSA-109581). In other words, the group of the selectable shapes has to include the group of shapes that can be overlaid.</span></li>\r\n</ul>\r\n</li>\r\n<li><span>Any group of shapes, text or graphical elements that are not annotated as active regions are considered decorators and users cannot interact with them. They have pure esthetic value and they cannot be selected, hovered or overlaid. &nbsp;&nbsp;</span></li>\r\n<li><span>If possible, no background covering the whole image should be put in place. Please note that compartments or other necessary backgrounds can be kept, but we suggest to avoid unnecessary backgrounds. In case a background is required, but it is too big to fit in the illustration, it is suggested to draw only a small part of it with clear and well defined boundaries (See the blood vessel in the EHLD of Hemostasis above). </span></li>\r\n<li><span>Text elements should not be converted to groups of shapes. Instead, use the normal &lt;text&gt; SVG element.</span></li>\r\n</ul>\r\n<p><span><img src=\"images/ehldguide/subpathways.png\" alt=\"subpathways\" width=\"407\" height=\"272\" /></span></p>\r\n<p><span>Example illustrating the two types of active regions. Selectable regions (annotated with REGION- + Reactome Identifier) are highlighted in red, while regions that can be overlaid are highlighted in brown. Shapes and text that are not annotated as active regions are considered decorators (presented in blue) and users cannot interact with them.</span></p>\r\n<p>&nbsp;</p>\r\n<p><span>Taking the aforementioned requirements into consideration, the EHLD of the figure above should have the following structure:</span></p>\r\n<p>&nbsp;</p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li><span>REGION-Pathway A </span><i><span># The whole pathway will be selectable #</span></i>\r\n<ul style=\"list-style-type: circle;\">\r\n<li><span>OVERLAY-Pathway A </span><i><span># Only the label will be overlaid #</span></i></li>\r\n</ul>\r\n</li>\r\n<li><span>REGION-Pathway B </span><i><span># The whole pathway will be selectable including the arrow #</span></i>\r\n<ul style=\"list-style-type: circle;\">\r\n<li><span>OVERLAY-Pathway B </span><i><span># Only the label will be overlaid #</span></i></li>\r\n</ul>\r\n</li>\r\n<li><span>REGION-Pathway C </span><i><span># The whole pathway will be selectable including the arrow #</span></i>\r\n<ul style=\"list-style-type: circle;\">\r\n<li><span>OVERLAY-Pathway C </span><i><span># Only the label will be overlaid #</span></i></li>\r\n</ul>\r\n</li>\r\n<li><span>OVERLAY-Pathway D &nbsp;</span><i><span>&nbsp;# The whole pathway will be selectable and overlaid #</span></i></li>\r\n<li><span>DECORATORS </span><i><span># None of the shapes are selectable or can be overlaid #</span></i></li>\r\n</ul>\r\n<h3>Creating an EHLD step by step</h3>\r\n<p><span>We suggest starting a new EHLD by creating an Adobe Illustrator document with </span><b>1366px</b><span> in width and </span><b>768px</b><span> in height and using the </span><b>RGB colour mode</b><span>.</span></p>\r\n<p><span>Pathway Labels should have </span><b>centered,</b> <b>white</b><span>, </span><b>uppercased</b><span>, text using </span><b>Arial Bold</b><span> font at </span><b>12pt</b><span> inside a </span><b>rounded rectangle</b><span> with </span><b>8pt</b> <b>radius</b><span>, minimum </span><b>width of 170px</b><span> and </span><b>height</b><span> of</span><b> 30px </b><span>for single lines or </span><b>43px</b><span> for double lines. The colour of the rounded rectangle should be </span><b>#0F82BC</b><span> (R:15 G:130 B:188). We suggest writing the name of the subpathway it represents as it appears in the Reactome hierarchy. In case of a discrepancy, or any kind of oddity, please contact the author or the curator related to the pathway.</span></p>\r\n<p><span>Please use the same specifications for any other written element except for:</span></p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li><b>Descriptions</b><span> of a process should always be black and in lowercase letters.</span></li>\r\n<li><span>Any </span><b>written element</b><span> that belongs to a compound, protein or cell (generically inside of any coloured shape) should be white and in uppercase letters (when possible).</span></li>\r\n<li><span>Any </span><b>written element</b><span> that belongs to a receptor and/or describes a cell element or a process (generically outside of any shape) should be black and in lowercase letters (when possible, usually receptors are named by acronyms and capital letters should be used in these cases).</span></li>\r\n<li><span>Greek or other Unicode characters must be avoided in text elements. Thus, names with greek characters should include the whole word (e.g. alpha instead of ) written in lowercase.</span></li>\r\n<li><span>Reactome </span><b>logo</b><span> should always have </span><b>50%</b> <b>opacity</b><span>.</span></li>\r\n</ul>\r\n<p><span>Every pathway label should have one </span><b>Analysis Information Label</b><span>. The latter is used to display additional details about the hit elements and the false discovery rate (FDR). All analysis information labels should be annotated as ANALINFO in the Layer Hierarchy Panel. Place XXX/YYY as a text placeholder inside it. Text should be centered and uppercased in </span><b>Arial Bold</b> <b>9pt</b><span>, and in white colour. The box should be a rounded rectangle with </span><b>8px radius</b><span>,</span> <span>minimum </span><b>width of 170px</b><span> and </span><b>20px height</b><span>. Its fill colour should be </span><b>#C6C6C6</b><span> (R:198 G:198 B:198). Also, set the opacity of the group to 0%, but make sure you leave the opacity of the shapes and text inside the group to 100%. Each analysis information label should be placed beside its pathway label and in the opposite side of the content for that pathway. For instance, if the pathway label is above its content, the analysis information label should be placed above the pathway label and vice versa.</span></p>\r\n<p><span>In order to better shape the active region (or the clickable area) of a pathway we can add a white shape with </span><b>0% opacity</b><span> under any element of the Pathway in the Layer Hierarchy Panel. In this way we can extent the clickable area and improve the overall user experience while navigating the EHLDs. In particular, we should try to surround the elements of the subpathway covering the gaps within the individual graphic elements, and thus, avoiding the inconvenient situation where the mouse cursor changes from the usual arrow shape to the hand constantly. Additionally, in EHLDs where elements of a particular subpathway are too thin and users find it difficult to select them, this technique can provide a bigger and more convenient clickable area around them. However, we should use this with caution, as creating too large clickable areas can mislead users.</span></p>\r\n<p><span>It is recommended to place all arrows and neutral text elements in a group at the top of the layer hierarchy. As well, all decorators and elements in the background that we do not want to highlight or be clickable, we recommend to place them in a group at the bottom of the layer hierarchy.</span></p>\r\n<p><span>Reactome displays EHLDs in a blank zoomable space. For a better user experience we suggest to have all shapes and elements portrayed in full. However, we understand that some pathways are large and contain too many and too big elements to fit in one image. In cases where the illustrator needs to cut an element and it is difficult to make a clean representation of the cut, we suggest the addition of a gradient so that elements vanish keeping the limits of the canvas clear.</span></p>\r\n<p><span>In order to export an EHLD we should use the Export as option and save the file as an SVG with the following options:</span></p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li><span>Styling: Internal CSS.</span></li>\r\n<li><span>Font: SVG.</span></li>\r\n<li><span>Images: Preserve.</span></li>\r\n<li><span>Object IDs: Layer Names.</span></li>\r\n<li><span>Decimal Points: 3.</span></li>\r\n<li><span>Minify and Responsive, both ticked.</span></li>\r\n</ul>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-04-12 11:02:44','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(130,774,'Icons Library Guidelines','icon-guidelines','<p>{loadposition icon-lib-pos-1}</p>\r\n<p>\r\n<style type=\"text/css\">table {\r\n        width: 100%;\r\n        border-collapse: collapse;\r\n        margin-left: auto;\r\n        margin-right: auto;\r\n    }\r\n\r\n    table,\r\n    td {\r\n        border: 1px dashed #c9c9c9;\r\n    }\r\n\r\n    td.split {\r\n        width: 50%;\r\n    }\r\n\r\n    td p {\r\n        text-align: center;\r\n        margin-bottom: 0;\r\n    }\r\n\r\n    td p.detail {\r\n        margin: 5px;\r\n        text-align: justify;\r\n    }\r\n\r\n    p.small {\r\n        line-height: normal;\r\n        text-align: center;\r\n        margin-bottom: 0;\r\n    }\r\n\r\n    p.small span {\r\n        font-size: 8pt;\r\n    }</style>\r\n</p>\r\n<p>This section will guide you through the process of creating and including your own graphic elements into the Reactome Library and become a part of this growing community.</p>\r\n<p>Since Reactome is a free and open-source project, the Reactome Library is under a Creative Commons license. As a result, if your creations are included in our library, they will follow the same policy. Feel free to use any tool or application that allows you to create <b>vector graphics</b> and, please, make sure that the following specifications are implemented in the files you will be creating.</p>\r\n<h4>Generic guidelines</h4>\r\n<ul style=\"list-style-type: disc;\">\r\n<li>Keep simplicity in mind. The main goal for this Icon Library is to provide easy to read graphic components for our users and simplicity is key to this matter.</li>\r\n<li>Consistency is important for Reactome and the Icon Library. Please, before submitting a proposal, read the guidelines we have set for the different types of components we use.</li>\r\n<li>Each icon should represent a single element only. After receiving feedback we open this rule to receptors when binding other receptors in order to make it easy for our users.</li>\r\n<li>Every element should be placed into a separate file, <b>200</b> by <b>200px</b> in size, with <b>RGB colour mode</b> and <b>White background</b>.</li>\r\n<li>Use only <b>Arial Bold 12pt</b>. Text colour should be chosen depending on the texts position. In case the text is <b>inside</b> a coloured shape, its colour should be <b>White</b>, while in case the text is <b>outside</b> of a coloured shape, its colour should be <b>Black</b>.</li>\r\n<li>All text should be <b>Uppercased</b> except for words describing <b>greek</b> characters (e.g alpha, beta, gamma, etc.). In that case those words should be written in <b>Lowercase</b>. It should be noted that Greek or other Unicode characters must be avoided, as they are not &nbsp;supported for the moment.</li>\r\n<li>Your elements should only comprise vector graphics. Please make sure that you have not included any bitmap image in it. Including raster images often results in larger file sizes and thus negatively impacts software performance. &nbsp;</li>\r\n<li>You can place your new element in any of the seven categories of the Reactome Library; Cell elements, Cell types, Compounds, Human tissue, Ion channels, Proteins and Receptors.</li>\r\n</ul>\r\n<h4>Cell elements</h4>\r\n<p>Keeping simplicity is one of our targets so we recommend the use of <b>simple figures</b>, such as circles, squares and triangles as a base. For colouring we suggest the use of shades of <b>Primary Colours</b> (Red, Blue and Yellow) and <b>Secondary Colours</b> (Purple, Green and Orange).</p>\r\n<p>Vector graphic applications allow you to use <b>Gradient tools</b> to blend two or more colours for the same shape. We recommend you to use these tools so you can create more appealing graphics. See the Mitochondrion below:</p>\r\n<p style=\"text-align: center;\"><img src=\"images/ehldguide/gradient_example.png\" alt=\"gradient example\" width=\"314\" height=\"232\" /></p>\r\n<p class=\"small\"><span><em>(a) Without the use of gradients, the Mitochondrion element looks plain while in</em></span></p>\r\n<p class=\"small\"><span><em>(b) through the correct use of gradients the element looks more 3D-like,&nbsp;</em></span></p>\r\n<p class=\"small\"><span><em>resulting in a more attractive and playful image.</em></span></p>\r\n<h4>Cell types</h4>\r\n<p>In order to ensure a sense of unity and a consistent look and feel, we suggest applying the same principles; simple shapes and colours.</p>\r\n<p>For representing types of cells, please keep in mind that we are not using scale to portray the different organelles and elements inside the cell cytoplasm. Additionally, to maintain simplicity, try to avoid overloading the the overall design with too much detail. The following table features 3 examples:</p>\r\n<p>&nbsp;</p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-013014.svg\" alt=\"microbe\" width=\"173\" height=\"173\" /></p>\r\n</td>\r\n<td>\r\n<p class=\"detail\"><em>This <b>Microbe</b> element is formed by a circle with a radial gradient of two colours. To represent the membrane, we have added an outline with a different gradient. To convey the idea of danger we use this squares as spikes and a red shade in their gradients.</em></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-012969.svg\" alt=\"macrophage\" width=\"154\" height=\"154\" /></p>\r\n</td>\r\n<td>\r\n<p class=\"detail\"><em>We have drawn this <b>Macrophage</b> element with shaky tentacles and we have used a not uniform outline (membrane) to give it a soft and flexible feeling.</em></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-013155.svg\" alt=\"pathogen\" width=\"166\" height=\"169\" /><img src=\"/icon/R-ICO-013157.svg\" alt=\"pathogen dead\" /></p>\r\n</td>\r\n<td style=\"text-align: justify;\">\r\n<p class=\"detail\"><em>For this <b>Pathogen</b> element we used a large light gradient to represent a shiny and hard capsule.</em></p>\r\n<p class=\"detail\"><em>To show its death we draw holes on its surface and play with a darkened, sad colour.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Compounds</h4>\r\n<p>Compounds are represented by very geometrical elements, following more specific guidelines.</p>\r\n<p>Simple chemical elements or compounds are usually portrayed in the Library with their chemical element symbol (e.g. Ca for Calcium) or the name of the compound (e.g. IFN-gamma) using <b>Arial bold 12pt</b> in <b>White</b> colour.</p>\r\n<p>Simple chemical elements are represented by an <b>octagon</b> of <b>28px</b> height and two plain colours, one for the fill and a different one for the stroke (<b>2pt</b>). In case the text of the compound is too long to fit in the octagon, we suggest stretching the shape until it fits the name, respecting its lateral edges, like follows:</p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-013527.svg\" alt=\"h2o\" width=\"183\" height=\"138\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><em>An example of a simple chemical element.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p>&nbsp;</p>\r\n<p><em></em>Complex compounds can be represented with a variety of shapes, always simple and easy to differentiate one from each other.</p>\r\n<p>&nbsp;</p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-012399.svg\" alt=\"atp\" width=\"123\" height=\"120\" /></p>\r\n</td>\r\n<td>\r\n<p class=\"detail\"><em>We can use circles with or without outline.</em></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-012413.svg\" alt=\"alpha toh\" width=\"124\" height=\"77\" /></p>\r\n</td>\r\n<td>\r\n<p class=\"detail\"><em>We can use hexagons, in the same way as for the simple compounds, just adding a gradient on their stroke.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p>&nbsp;</p>\r\n<h4>Human tissue</h4>\r\n<p>Human tissue elements are mainly used as backgrounds in our diagrams and the way of representing these organs depends on the illustrators skills and taste. To ensure unity in the Reactome Library we suggest to keep these illustrations as simple as possible and visualise them as toys; so they convey a nice, soft feeling, with a plastic-like, very friendly and approachable look.</p>\r\n<p>&nbsp;</p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012959.svg\" alt=\"liver\" width=\"190\" height=\"114\" /></p>\r\n</td>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012472.svg\" alt=\"blood vessel\" width=\"260\" height=\"117\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td colspan=\"2\">\r\n<p><em>Examples of human tissue elements; Liver and Blood vessel.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Ion channels</h4>\r\n<p>These representations are quite simplified and, thus, elements are designed to look like small funnels that cross through the membrane, allowing elements to move from inside of the cell to outside and vice versa. There are hundreds of different ion channels and we suggest differentiating them with the use of colours in various combinations.</p>\r\n<p>&nbsp;</p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012502.svg\" alt=\"calcium channel\" width=\"150\" height=\"192\" /></p>\r\n</td>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-013251.svg\" alt=\"proton channel\" width=\"159\" height=\"186\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td colspan=\"2\">\r\n<p><em>Examples of ion channels.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Proteins</h4>\r\n<p>There are tens of thousands of different proteins, some of them with specific shapes. Proteins in the Reactome Library follow the standard representation as rounded rectangle shapes. As a result, we suggest using the <b>Rounded Rectangle Tool</b> to draw a shape with <b>10px</b> radius for the corners and <b>20px</b> height for single line of text, or <b>30px</b> for a double line. The stroke should be <b>3pt </b>and we suggest choosing an irregular profile (in Adobe Illustrator, use the default <b>Width Profile 2</b>). For the name of the proteins we suggest to follow the same guides as before, <b>White Uppercase Arial Bold 12pt</b>.</p>\r\n<p>The shape will contain the simplified name of the protein following the general instructions for Reactome Library. It should be filled with a simple gradient of two colours. One of the &nbsp;colours of this gradient should also be used as the stroke colour.</p>\r\n<p>&nbsp;</p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-013194.svg\" alt=\"plc beta1\" width=\"195\" height=\"80\" /></p>\r\n</td>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-013189.svg\" alt=\"plasminogen\" width=\"285\" height=\"83\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td colspan=\"2\">\r\n<p><em>Examples of protein elements in their rounded rectangle representation.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p>&nbsp;</p>\r\n<p>In case a protein has a specific shape, we suggest keeping the standard representation as long as it is not too complicated. Otherwise, we suggest opting for a more simplified version.</p>\r\n<p>&nbsp;</p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012968.svg\" alt=\"mac\" width=\"160\" height=\"164\" /></p>\r\n</td>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012425.svg\" alt=\"antibody\" width=\"189\" height=\"167\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td colspan=\"2\">\r\n<p><em>Examples of protein elements represented in their standard representation.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Receptors</h4>\r\n<p>Like proteins, there is a huge number of receptors. In cases where there is a standard, but quite complex, way to represent them, we suggest adopting a more simplified version. If this is too difficult we suggest finding a more simple representation.</p>\r\n<p>&nbsp;</p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012710.svg\" alt=\"FCgammaR\" width=\"111\" height=\"209\" /></p>\r\n</td>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012529.svg\" alt=\"cd40-cd40l\" width=\"97\" height=\"196\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td colspan=\"2\">\r\n<p><em>Examples of receptor elements</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Metadata</h4>\r\n<p>You are encouraged to accompany your element with more information including a short description and extra details about its designer and curator. Upon inclusion of your element into the Reactome Library, all metadata information will be available to the public through our landing page.</p>\r\n<p>In order to do so, you need to create an additional metadata file using your favorite text editor. The file should have the following structure:</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre><code>&lt;metadata&gt;\r\n    &lt;categories&gt;\r\n        &lt;category&gt;<b>{CATEGORY}</b>&lt;/category&gt;\r\n    &lt;/categories&gt;\r\n    &lt;person orcid=\"<b>{ORCID}</b>\" role=\"curator\"&gt;<b>{CURATOR_NAME}</b>&lt;/person&gt;\r\n    &lt;person role=\"designer\" url=\"<b>{DESIGNER_URL}</b>\"&gt;<b>{DESIGNER_NAME}</b>&lt;/person&gt;\r\n    &lt;name&gt;<b>{ICON_NAME}</b>&lt;/name&gt;\r\n    &lt;description&gt;<b>{DESCRIPTION}</b>&lt;/description&gt;\r\n    &lt;references&gt;\r\n        &lt;reference&gt;\r\n            &lt;db&gt;<b>{REFERENCE_NAME}</b>&lt;/db&gt;\r\n            &lt;id&gt;<b>{REFERENCE_ID}</b>&lt;/id&gt;\r\n        &lt;/reference&gt;\r\n    &lt;/references&gt;\r\n    &lt;synonyms&gt;\r\n        &lt;synonym&gt;<b>{SYNONYM}</b>&lt;/synonym&gt;\r\n    &lt;/synonyms&gt;\r\n&lt;/metadata&gt;\r\n</code></pre>\r\n</div>\r\n</div>\r\n<p>Please follow these steps:</p>\r\n<ol>\r\n<li>Create a new file using your favorite text editor (TextEdit, NotePad, etc.).</li>\r\n<li>Copy and Paste the text above into your file.</li>\r\n<li>Edit the file and:<ol>\r\n<li>Fill {CATEGORY} with one or more of the suggested categories: cell_element, cell_type, compound, human_tissue, protein, receptor or transporter. We accept one or more different categories as we think they are just a guide that helps our users to find the element they need.</li>\r\n<li>Fill {ORCID} and/or {URL} for the curator and designer roles. This is <a href=\"https://orcid.org/orcid-search/search\">how to find ORCID</a>. In case both URL and ORCID are provided, the link for that person will point to the URL.</li>\r\n<li>Fill {CURATOR_NAME} and {DESIGNER_NAME} as the curator and designer name, respectively. If the name is the same, please fill both with same information.</li>\r\n<li>Fill {DESCRIPTION} with any information relevant to describe the component. Please be brief.</li>\r\n<li>Fill {REFERENCE_NAME} for the name of the resource you have used to find this component (e.g.:CHEBI, UNIPROT, GO  ). In case of not finding a suitable reference, please delete this line.</li>\r\n<li>Fill {REFERENCE_ID} as it appears in the resources URL (e.g.: CHEBI:1234, Q1234, GO:1234  ). In case of not finding a suitable reference, please delete this line.</li>\r\n<li>Fill {SYNONYM} as any synonyms or alternative names your component might be known of. In case of not finding &nbsp;a suitable synonym, please delete this line.</li>\r\n</ol></li>\r\n<li>Save the file using the same name as the element but make sure to use .xml as its extension. For example if your graphic element is stored with the name Element.svg then your metadata file should be named as Element.xml.</li>\r\n<li>Make sure to submit your metadata file along with the graphics file.</li>\r\n</ol>\r\n<h3>Export Settings</h3>\r\n<p>Currently, every element in the Reactome Library is available in 3 different formats, SVG, EMF and PNG. Therefore, you are kindly requested to export your work into these formats and include them in your submission.</p>\r\n<p>For PNG files we suggest to export with 300 DPI and transparent background.</p>\r\n<h3>How to submit your work</h3>\r\n<p>To include your graphic elements in the Reactome Library, you need to contact our helpdesk (<a href=\"mailto:help@reactome.org\">help@reactome.org</a>). We will then review your submitted files and, depending on the outcome, we will include them in our Library.</p>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-04-12 11:03:36','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(133,777,'ReactomeFI Network v2018 and FIViz App v2.7.0 Released','reactomefiviz-app-version-7-2-0-released','<p style=\"text-align: justify;\"><img src=\"images/news/ReactomeFI_2018.png\" width=\"650\" height=\"366\" alt=\"ReactomeFI 2018\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p style=\"text-align: justify;\">The Reactome project&nbsp;provides a suite of tools for our users to perform pathway- and network-based data analysis via its <a href=\"/\">web site</a> and <a href=\"https://apps.cytoscape.org/apps/reactomefiplugin\">ReactomeFIViz</a>, the functional interaction (FI) network-based <a href=\"https://cytoscape.org/\">Cytoscape</a> app.</p>\r\n<p style=\"text-align: justify;\">The Reactome FI network is built upon curated pathways in Reactome, supplemented with curated pathways from other pathway databases such as <a href=\"https://www.genome.jp/kegg/pathway.html\">KEGG PATHWAY</a> and <a href=\"http://www.pantherdb.org/pathway/\">PANTHER pathways</a>. These curated pathway interactions are then integrated with gene-gene interactions derived from curated and high-throughput sources using machine learning to generate a reliable set of high-probability functional interactions.</p>\r\n<p style=\"text-align: justify;\">Recently, we have released the 2018 version of the Reactome FI network. To construct this&nbsp;FI network, we improved the&nbsp;training of the Naive Bayesian Classifier, which resulted in an increase in the recall rate. Consequently, the SwissProt coverage was increased from 12,441 to 13,469 proteins (8.3% increase) and the total number of functional interactions among genes increased from 241,338 to 262,321 (8.7% increase). We attributed this to an increase in the number of predicted FIs due to the improved recall rate.</p>\r\n<p style=\"text-align: justify;\">In addition,&nbsp;the ReactomeFIViz app (version 7.2.0) now supports the 2018 version of the FI network and contains updated Reactome Pathways dataset (Release 67). The new ReactomeFIViz app is available to download within the Cytoscape software, or directly from the <a href=\"https://apps.cytoscape.org/\">Cytoscape app Store</a>. The <a href=\"https://www.reactome.org/tools/reactome-fiviz\">User Guide</a> provides instructions on using the Reactome FI network and ReactomeFIViz app.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-04-15 17:00:23','0000-00-00 00:00:00','{}','{}','{}',1,5,'','',1,0,'{}',0,'en-GB','',''),(134,778,'Documentation','documentation','<p style=\"text-align: justify;\">&nbsp;{loadposition userguide-cp-1}</p>\r\n<p style=\"text-align: justify;\">Reactome is documented for users at all levels of sophistication. The online <a href=\"/userguide\">Users Guide</a> begins with entry-level information that describes what our resource contains and how its pathways are organized. The guide leads users through the process of browsing Reactome, searching for particular biological and chemical molecules or pathways of interest, interpreting experimental datasets through the website and the more specialized <a href=\"/userguide/reactome-fiviz\">ReactomeFIViz app</a>.</p>\r\n<p style=\"text-align: justify;\">Other online documentation includes complete descriptions of the Reactome <a href=\"/documentation/data-model\">data model</a> and <a href=\"/content/schema/DatabaseObject\">database schema</a>,&nbsp;information for managers of external biological resources on how to <a href=\"/linking-to-us\">link</a>&nbsp;to specific types of Reactome pages, and information on how to&nbsp;<a href=\"/cite\">cite the resource</a>&nbsp;in publications. In addition, users can download specialized documentation that describes how to use the curator tool, a Reactome specific software tool for submitting data directly to the knowledgebase. One goal for Reactome has been the development of&nbsp;<a href=\"/dev\">reusable software tools and data resources</a> for managing and visualizing pathway information. This requires the creation of portable software and careful documentation. We believe in documenting our curatorial practices and policies as well, allowing other groups to understand and adopt our best practices.</p>\r\n<p style=\"text-align: justify;\">As with extensions of Reactome and integration of new data and software, clear documentation of these methods are developed and, when possible, published in peer-reviewed journals. Documentation is actively maintained to at a level of high polish! We ask our broader base of users to join Reactome curators, technical writers, developers and educators beyond our primary institutions: OICR, OHSU, NYU, and EBI in developing, refining, and improving our documentation.</p>\r\n<p style=\"text-align: justify;\"></p>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-04-18 15:33:34','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"0\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','','Home of the userguide'),(136,780,'Season of Docs','season-of-docs','<ul>\r\n<li style=\"text-align: justify;\"><img src=\"images/news/SeasonofDocs_Logo_MainGrey_300ppi.png\" alt=\"SeasonofDocs Logo MainGrey 300ppi\" style=\"display: block; margin-left: auto; margin-right: auto;\" width=\"650\" height=\"467\" /></li>\r\n</ul>\r\n<p style=\"text-align: justify;\"><strong>Fostering open source collaboration with technical writers  Season of Docs</strong></p>\r\n<p style=\"text-align: justify;\">Established upon the reputation of open source community programs like <a href=\"https://summerofcode.withgoogle.com/\">Google Summer of Code</a> and <a href=\"https://codein.withgoogle.com/archive/\">Google Code-in</a>, Google is introducing a new initiative called <a href=\"https://developers.google.com/season-of-docs/\">Season of Docs</a>. The Reactome project has applied to participate in this year\'s Season of Docs.&nbsp;</p>\r\n<p style=\"text-align: justify;\">Season of Docs aims to provide open source projects with an opportunity to engage with the technical writing community and for technical writers an occasion to gain experience in contributing to open source projects.</p>\r\n<p style=\"text-align: justify;\">Together, we will raise community awareness of open documentation, technical writing, and how we can collectively work together to improve open source projects. Reactome looks forward to the opportunity that Season of Docs presents and for the opportunity to bring our entire community closer together.</p>\r\n<p>&nbsp;</p>\r\n<p style=\"text-align: justify;\"><span style=\"text-decoration: underline;\"><strong>Project Information</strong></span></p>\r\n<p style=\"text-align: justify;\"><strong>Project Name:</strong> Reactome</p>\r\n<p style=\"text-align: justify;\"><strong>Project Description:</strong> Reactome is a free, open-source, curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation, and analysis of pathway knowledge to support basic research, genome analysis, modelling, systems biology, and education.</p>\r\n<p style=\"text-align: justify;\"><strong>Project Website:</strong> <a href=\"https://reactome.org\">https://reactome.org</a></p>\r\n<p>&nbsp;</p>\r\n<p style=\"text-align: justify;\"><strong>Project Idea #1 Name:</strong> Revising the Reactome User Guide</p>\r\n<p style=\"text-align: justify;\"><strong>Description:</strong> The Reactome User Guide provides an introduction to Reactome, the user interfaces and the database content. Exercises are also provided to help user practice what they have learned. New functionality and improvements to the core website and software are constantly added to the User Guide. The outcome of this project is the revision of the existing User Guide, with updated and new tutorials,a set of how-to guides to support navigating the website and using the pathway visualization and analysis tools</p>\r\n<p style=\"text-align: justify;\"><strong>Docs Link:</strong> <a href=\"https://reactome.org/userguide\">https://reactome.org/userguide</a></p>\r\n<p style=\"text-align: justify;\"><strong>Contacts:</strong> Robin Haw: <a href=\"mailto:robinhaw@gmail.com\">robinhaw@gmail.com</a> &amp; Marc Gillepsie: <a href=\"mailto:gillespm@gmail.com\">gillespm@gmail.com.</a></p>\r\n<p>&nbsp;</p>\r\n<p><strong>Project Idea #2 Name:</strong> Updating the Reactome Curator Guide</p>\r\n<p style=\"text-align: justify;\"><strong>Description:</strong> The Reactome Curator Guide contains an overview of the curatorial process as well as a step by step guide for annotating Reactome pathways. The curator guide takes new curators through the process of project design, entry, review, and release. Each one of these steps requires that the curator is familiar with the Reactome data model, software entry tools, process pipeline, and project management. In this sense, the curator guide serves as a compendium of learned lessons, curator philosophy, and a step-by-step process blueprint. The outcome of this project is the complete revision of the current guide to reflect new data entry tools, update the current glossary of terms to accommodate changes to data model, and create short tutorials that can be used to reinforce each learning step.</p>\r\n<p style=\"text-align: justify;\"><strong>Contacts:</strong> Marc Gillepsie <a href=\"mailto:gillespm@gmail.com\">gillespm@gmail.com</a> and Lisa Matthews <a href=\"mailto:lmatthews.nyumc@gmail.com\">lmatthews.nyumc@gmail.com.</a></p>\r\n<p>&nbsp;</p>\r\n<p style=\"text-align: justify;\"><strong>More Information&nbsp;</strong></p>\r\n<p style=\"text-align: justify;\">For more information about Reactome and Season of Docs, please refer to the following pages:</p>\r\n<p style=\"text-align: justify;\"><a href=\"/documentation\">Reactome Documentation</a> - Learn more about Reactome docs.</p>\r\n<p style=\"text-align: justify;\"><a href=\"https://developers.google.com/season-of-docs/docs/\">Season of Docs</a> - What is Season of Docs all about?</p>\r\n<p style=\"text-align: justify;\"><a href=\"https://developers.google.com/season-of-docs/docs/\">Guides</a> - How it all comes together.</p>\r\n<p style=\"text-align: justify;\"><a href=\"https://developers.google.com/season-of-docs/docs/timeline\">Timeline</a> - Please be aware of the schedule!</p>\r\n<p style=\"text-align: justify;\"><a href=\"https://developers.google.com/season-of-docs/terms/program-rules\">Rules</a> - The ever important rulebook.</p>\r\n<p>&nbsp;</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-04-22 19:59:12','0000-00-00 00:00:00','{}','{}','{}',1,4,'','',1,0,'{}',0,'en-GB','',''),(137,781,'Version 69 Released','version-69-released','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Autophagy.png\" alt=\"Autophagy\" width=\"650\" height=\"346\" /></p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Pathways.</strong> In version&nbsp;V69, Autophagy, including&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-9613829\">Chaperone Mediated Autophagy</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-9615710&amp;PATH=R-HSA-9612973\">Microautophagy</a>&nbsp;is new.&nbsp; Topics with new or revised pathways include&nbsp;Development (<a href=\"/PathwayBrowser/#/R-HSA-9616222\">Transcriptional Regulation of Granulopoiesis</a>), Disease (<a href=\"/PathwayBrowser/#/R-HSA-9630750&amp;PATH=R-HSA-1643685,R-HSA-9630747\">Evasion of Oncogene Induced Senescence Due to p16-INK4A Defects</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-9632693&amp;PATH=R-HSA-1643685,R-HSA-9630747\">Evasion of Oxidative Stress Induced Senescence Due to p16-INK4A Defects</a>),&nbsp;Immune system (<a href=\"/PathwayBrowser/#/R-HSA-449836&amp;SEL=R-HSA-9607240&amp;PATH=R-HSA-168256,R-HSA-1280215,R-HSA-449147\">FLT3 Signaling</a>),&nbsp;and Signal Transduction (<a href=\"/PathwayBrowser/#/R-HSA-9009391&amp;PATH=R-HSA-162582,R-HSA-9006931,R-HSA-8939211\">Extra-Nuclear Estrogen Signaling</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-1236394\">Signaling by ERBB4</a>).&nbsp;</p>\r\n<p style=\"text-align: justify;\"><strong>New Illustrations.</strong> Illustration with embedded navigation features is now available for <a href=\"/PathwayBrowser/#/R-HSA-9612973\">Autophagy.</a></p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors. </strong><a href=\"http://orcid.org/0000-0003-4018-5348\">David Stern</a>&nbsp;is our external author.&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Filippo%20%20Acconcia%5BAuthor%20-%20Full%5D\">Filippo Acconcia</a>,&nbsp;<a href=\"https://www.researchgate.net/profile/Dorothy_Bennett\">Dorothy Bennett</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Marino%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30945992\">Maria Marino</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Emmanouil+++Metzakopian%5BAuthor%5D\">Emmanouil Metzakopian</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Julia++Skokowa%5BAuthor%5D\">Julia Skokowa</a>, and&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Traer%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30720426\">Elie Traer</a>&nbsp;are our external reviewers.&nbsp;<strong><br /></strong></p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong>&nbsp;Reactome comprises 12,505 human reactions organized into 2,272 pathways involving 11,009 proteins and modified forms of proteins encoded by 10,833 different human genes, 1,857 small molecules, and 196 drugs. These annotations are supported by 30.027 literature references. We have projected these reactions onto 81,631 orthologous proteins, creating 18,610 orthologous pathways in 15 non-human species. Version 69 has annotations for 1,612 protein variants (mutated proteins) and their post-translationally modified forms, derived from 305 proteins. These have been used to annotate 532 complexes and 968 disease-specific reactions organized into 478 pathways and subpathways, and tagged with 374 Disease Ontology terms.</p>\r\n<p style=\"text-align: justify;\"><strong>Tools and Data. </strong>Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"/PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"/download-data\">download</a>, and to access programmatically through our <a href=\"/ContentService/\">Content</a> and <a href=\"/AnalysisService/\">Analysis</a> Services. The <a href=\"/tools/reactome-fiviz\">ReactomeFIViz app</a> for Cytoscape provides tools to find pathways and network patterns related to cancer and other types of diseases.</p>\r\n<p style=\"text-align: justify;\"><strong>Documentation and Training</strong>. Visit our online <a href=\"/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"/community/training\">here</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the&nbsp;Reactome <a href=\"/\">website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong>&nbsp;<a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information</strong>: If you have a question to ask or would like to give us your feedback, please contact our&nbsp;<a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-05-28 17:41:55','0000-00-00 00:00:00','{}','{}','{}',1,3,'','',1,0,'{}',0,'en-GB','',''),(138,783,'ORCID: Claim your works','orcid-claim-your-works','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20190618_ORCID_claim.png\" alt=\"claim your reactome work into your orcid account image\" usemap=\"#cwmap\" width=\"650\" height=\"382\" /> <map name=\"cwmap\"> \r\n<area coords=\"115,182,540,250\" shape=\"rect\" href=\"index.php?option=com_content&amp;view=article&amp;id=139&amp;Itemid=488\" />\r\n </map></p>\r\n<p style=\"text-align: justify;\">Reactome is a curated database, it critically depends on the expertise of curators and domain experts&nbsp;as volunteer pathway authors and reviewers. Over <a href=\"/content/query?q=type%3APerson&amp;species=Entries+without+species&amp;types=Person&amp;cluster=true\">700 scientists</a> have so far contributed to Reactome content. If you are one of them, we have great news for you: Your Reactome contribution can be included directly into your&nbsp;<a href=\"https://orcid.org\" target=\"_blank\" rel=\"noopener\">ORCID</a> profile. ORCID provides a persistent digital identifier that distinguishes you from every other researcher and, through integration in key research workflows such as manuscript and grant submission, supports automated linkages between you and your professional activities ensuring that your work is recognized.</p>\r\n<p style=\"text-align: justify;\">Because we appreciate your work contribution either authoring or reviewing a pathway or reaction, we have now made it easy to claim your contribution in ORCID. Using this new feature that has been integrated into our website, contributors who have provided their ORCID ID can, by simply clicking a button, reference their work in their profile.</p>\r\n<div class=\"leading-8\" itemscope=\"itemscope\" itemtype=\"https://schema.org/BlogPosting\" itemprop=\"blogPost\">\r\n<p style=\"text-align: justify;\">Follow this <a title=\"Click for instructions on how to claim your work\" href=\"index.php?option=com_content&amp;view=article&amp;id=139&amp;Itemid=488\">link</a> to learn more&nbsp;about how you can claim your work.</p>\r\n</div>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-06-06 14:16:00','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(139,784,'Tutorial: How to claim your works','tutorial-how-to-claim-your-works','<style type=\"text/css\">img { margin: 5px 0 15px; 3px; }</style>\r\n<ul>\r\n<li><a href=\"#guide\">Find yourself in Reactome</a></li>\r\n<li><a href=\"#missing_orcid\">I have an ORCID ID but it is not in Reactome. What should I do ?</a></li>\r\n</ul>\r\n<h3 id=\"guide\">Find yourself in Reactome</h3>\r\n<ol>\r\n<li>The simple text search tool is located at top right of the Home page. To search type your name or ORCID. The search has an auto-complete function; if the text you wanted to use appears in the drop-down list, select it and results will be displayed. For other text click Search.<br /><img src=\"images/userguide/search/search_box.gif\" alt=\"search box\" width=\"400\" height=\"93\" /></li>\r\n<li>Locate yourself in the Result List and Click your name.&nbsp;<br /><img src=\"images/userguide/orcid/name_in_result_list.png\" alt=\"name in result list\" width=\"400\" height=\"105\" /></li>\r\n<li>&nbsp;You\'ll land in the Contributor\'s page. You may see up to four categorized tables showing a maximum of 15 rows each.<br /><img style=\"width: 407px; height: 57px;\" src=\"images/userguide/orcid/are_you_orcid_login.png\" alt=\"are you orcid login\" data-mce-width=\"407\" data-mce-height=\"57\" /></li>\r\n<li>In order to claim your work you may need to Authenticate yourself in ORCID, in a similar way like others websites offer \'Sign in with Google\', \'Sign in with Facebook\'. Simply click \"Are you John Doe ?\". A pop-up like to the one below will open. Please input your ORCID credentials, then Click Sign-in.<br /><img style=\"width: 223px; height: 272px;\" src=\"images/userguide/orcid/orcid_login_dialog.png\" alt=\"orcid login dialog\" data-mce-width=\"223\" data-mce-height=\"272\" /></li>\r\n<li>Reactome.org will ask your permission to add works into your ORCID records. Authorize if you wish to grant access, otherwise we can\'t have access to&nbsp;<br /><img style=\"width: 225px; height: 245px;\" src=\"images/userguide/orcid/orcid_authorise_reactome.png\" alt=\"orcid authorise reactome\" data-mce-width=\"225\" data-mce-height=\"245\" /></li>\r\n<li>After authorizing, we will automatically proceed with the authorization process and the pop up will be closed and current page will be refreshed. A button \"Claim all Work\" may appear next to your ORCID number if we are able to match ORCID IDs. Click it to make sure all your contributions are uploaded.<br /><img style=\"width: 381px; height: 179px;\" src=\"images/userguide/orcid/claim_all_work.png\" alt=\"claim all work\" data-mce-width=\"381\" data-mce-height=\"179\" /></li>\r\n<li>Depending on the number of entries you are uploading it may take a few minutes. You may see a Claiming Summary when the entire process is finished.<br /><img style=\"width: 199px; height: 202px;\" src=\"images/userguide/orcid/claiming_summary.png\" alt=\"claiming summary\" data-mce-width=\"199\" data-mce-height=\"202\" /></li>\r\n<li>Visit your ORCID page and Navigate to the Works Section.&nbsp;</li>\r\n<li>All your work now is uploaded.</li>\r\n</ol>\r\n<h3 id=\"missing_orcid\">I have an ORCID ID but it is not in Reactome. What should I do ?</h3>\r\n<ol>\r\n<li>Follow intructions 1 to 5 in the previous section.</li>\r\n<li>You may see a Link \"Let us know your ORCID\". Please Click.<br /><img style=\"width: 371px; height: 80px;\" src=\"images/userguide/orcid/let_us_know_link.png\" alt=\"let us know link\" data-mce-width=\"371\" data-mce-height=\"80\" /></li>\r\n<li>&nbsp;After clicking, use the following form to send your ORCID details to our curators. Our help team will validate and add your ORCID into our database and will be available to you on our next data release. Follow us on Twitter for any updates.<br /><img style=\"width: 340px; height: 289px;\" src=\"images/userguide/orcid/let_us_know_form.png\" alt=\"let us know form\" data-mce-width=\"340\" data-mce-height=\"289\" /></li>\r\n</ol>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-06-14 13:51:38','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(140,785,'External Contribution','external-contribution','<p>Become a Reactome contributor! You will be credited with authorship or reviewership for all of your contributions. Each pathway is associated with a DOI and can be cited as a publication. You can quickly and easily claim your Reactome contributions in ORCID using our new ORCID claiming feature when you search for your name in Reactome as described <a href=\"userguide/claim-your-work\">here</a>.&nbsp;</p>\r\n<p>We are currently seeking reviewers for the following <a href=\"https://docs.google.com/spreadsheets/d/1k9XCVhonqtdhF1AyykulgKXmGwAV7YttKraPHBOa4Ow\" target=\"_blank\" rel=\"noopener\">pathways</a>. If you\'d like to contribute a pathway that is not on this list, please contact us. We would be happy to work with you!&nbsp;For more information and guidelines for reviewing a Reactome pathway module, see this <a href=\"index.php?option=com_content&amp;view=article&amp;id=143&amp;catid=2&amp;lang=en-GB\" hreflang=\"en\">FAQ for prospective Reactome reviewers and authors</a>.&nbsp;</p>\r\n<p><iframe src=\"https://docs.google.com/spreadsheets/d/e/2PACX-1vTQ9pyHZrHgKU9pPqdLzvUZLWdFSYb7bCuEWX5agymkHfh-7qfS9q3rxf31UtuHQv3ZgENYQhZtu8O0/pubhtml?gid=1520907054&amp;single=true&amp;widget=true&amp;headers=false\" width=\"100%\" height=\"500px\"></iframe></p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-06-20 15:12:50','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(141,786,'ReacFoam: Genome-wide pathway overview based on Voronoi tessellation','reacfoam-genome-wide-pathway-overview-based-on-voronoi-tessellation','<p><a href=\"https://reactome.org/reacfoam/?species=48887&color=COPPER%20PLUS&analysis=MjAxOTA5MjMxMDAwNDBfNTczMQ==&resource=TOTAL\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/voroni.png\" alt=\"\" width=\"650\" /></a></p>\r\n<p>As part of our continuous effort to provide visually attractive and more user-friendly access to our biological pathways, we are pleased to announce our new high level pathways overview visualisation based on Voronoi tessellation.</p>\r\n<p>Following any <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">pathway analysis</a>, the <a href=\"https://reactome.org/reacfoam/?species=48887&amp;color=COPPER%20PLUS&amp;analysis=MjAxOTA2MjAxMjE0NTFfMTM0&amp;resource=TOTAL\">pathway overview</a> provides a <a href=\"reacfoam\">new icon<img src=\"images/Screenshot_2019-07-02_at_124156.png\" alt=\"\" width=\"35\" height=\"35\" /></a>&nbsp; which leads to a comprehensive, highly visual, interactive overview of pathway analysis results. The functionality is available on both desktop and mobile platforms.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-07-02 11:38:13','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(142,787,'Version 70 Released','version-70-released','<p style=\"text-align: justify;\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Innate_Immune_System.png\" alt=\"Innate Immune System\" width=\"1168\" height=\"662\" /></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>New and Updated Pathways.</strong> In version 70, topics&nbsp;with new or revised pathways include Cellular response to stress (<a href=\"PathwayBrowser/#/R-HSA-9639288\">Amino acids regulate mTORC1</a>), Disease (<a href=\"PathwayBrowser/#/R-HSA-9630750\">Evasion of oncogene induced senescence due to p16-INK4A defects</a>,&nbsp;<a href=\"PathwayBrowser/#/R-HSA-9632693\">Evasion of oxidative stress induced senescence due to p16-INK4A defects</a>,&nbsp;<a href=\"PathwayBrowser/#/R-HSA-9645722\">Defective intrinsic pathway for apoptosis due to p14ARF loss of function</a>,&nbsp;<a href=\"PathwayBrowser/#/R-HSA-9646303\">Evasion of oncogene induced senescence due to p14ARF defects</a>,&nbsp;<a href=\"PathwayBrowser/#/R-HSA-9646304\">Evasion of oxidative stress induced senescence due to p14ARF defects</a>),&nbsp;Immune System (<a href=\"PathwayBrowser/#/R-HSA-9645460\">Alpha-protein kinase 1 signaling pathway&nbsp;</a>and&nbsp;<a href=\"PathwayBrowser/#/R-HSA-9607240\">FLT3 signaling</a>), and&nbsp;Signal transduction (<a href=\"PathwayBrowser/#/R-HSA-9024446\">NR1H2 and NR1H3-mediated signaling</a>).</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>New and Updated Illustrations.</strong>&nbsp;Topics with new or revised illustrations include&nbsp;<a href=\"PathwayBrowser/#/R-HSA-71291&amp;PATH=R-HSA-1430728\">Metabolism of amino acids and derivatives</a>&nbsp;and&nbsp;<a href=\"PathwayBrowser/#/R-HSA-168249&amp;PATH=R-HSA-168256\">Innate Immune System</a>.&nbsp;</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong><strong>Thanks to our Contributors.</strong>&nbsp;</strong><a href=\"https://orcid.org/0000-0002-3639-7527\">Dorothy Bennett</a>,&nbsp;<a href=\"https://orcid.org/0000-0002-9515-8892\">Kendall Condon</a>,&nbsp;<a href=\"http://www.cumminslab.com\">Carolyn Cummins</a>,&nbsp;<a href=\"https://orcid.org/0000-0003-4760-1033\">Nicholas Hayward</a>,&nbsp;<a href=\"https://orcid.org/0000-0003-2798-5762\">Sunil Joshi</a>,&nbsp;<a href=\"https://orcid.org/0000-0001-7201-6819\">Vaishnavi Nathan</a>,&nbsp;<a href=\"https://orcid.org/0000-0001-5740-1954\">Joyce Repa</a>,&nbsp;<a href=\"https://orcid.org/0000-0002-2094-9198\">Helen Rizos</a>,&nbsp;<a href=\"https://orcid.org/0000-0002-1446-7256\">David Sabatini</a>, and&nbsp;<a href=\"http://www.nibs.ac.cn/en/yjsjyimgshow.php?cid=5&amp;sid=6&amp;id=777\">Feng Shao</a>&nbsp;are our external reviewers.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Annotation Statistics.</strong>&nbsp;Reactome comprises 12,608 human reactions&nbsp;organized into 2,282 pathways involving 11,040 proteins and modified forms of proteins encoded by 10,860 different human genes, 12,335 complexes, 1,856 small molecules, and 222 drugs. These annotations are supported by 30.398 literature references. We have projected these reactions onto 83,183 orthologous proteins, creating 18,679 orthologous pathways in 15 non-human species. Version 70 has annotations for 1,762 protein variants (mutated proteins) and their post-translationally modified forms, derived from 308 proteins. These have been used to annotate 536 complexes and 970 disease-specific reactions organized into 484 pathways and subpathways, and tagged with 387 Disease Ontology terms.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Tools and Data.&nbsp;</strong>Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our&nbsp;<a href=\"PathwayBrowser/\">Pathway Browser</a>, to&nbsp;<a href=\"PathwayBrowser/#TOOL=AT\">analyze</a>&nbsp;your own dataset, to&nbsp;<a href=\"download-data\">download</a>, and to access programmatically through our&nbsp;<a href=\"ContentService/\">Content</a>&nbsp;and&nbsp;<a href=\"AnalysisService/\">Analysis</a>&nbsp;Services. The&nbsp;<a href=\"tools/reactome-fiviz\">ReactomeFIViz app</a>&nbsp;for Cytoscape provides tools to find pathways and network patterns related to cancer and other types of diseases.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Documentation and Training</strong>. Visit our online&nbsp;<a href=\"userguide\">User Guide</a>&nbsp;to access documentation supporting pathway analysis of experimental data. The&nbsp;<a href=\"dev\">Developer\'s Zone</a>&nbsp;provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found&nbsp;<a href=\"community/training\">here</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the&nbsp;Reactome&nbsp;<a href=\"//\">website</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Follow us on Twitter:</strong>&nbsp;<a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>For more information</strong>: If you have a question to ask or would like to give us your feedback, please contact our&nbsp;<span id=\"cloak3c9b6732f53c185a2fdfae327f5e52d3\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span>.</span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-09-05 14:28:19','0000-00-00 00:00:00','{}','{}','{}',1,0,'','',1,0,'{}',0,'en-GB','',''),(143,789,'FAQ for prospective Reactome reviewers and authors','faq-for-prospective-reactome-reviewers-and-authors','<p><strong>What is involved in reviewing a&nbsp; Reactome pathway module?</strong></p>\r\n<p>Reviewing a pathway module is similar to reviewing a review article and involves evaluating both a pathway report in text format and a corresponding online pathway diagram for completeness and accuracy.&nbsp; A Reactome pathway is a hierarchical representation of a biological process. Pathways are broken down into component subpathways. A subpathway is further subdivided into its component biochemical reactions, and each reaction includes input and output molecules as well as any relevant catalyst or regulators. The text report includes a summary of each event (pathway, subpathway or reaction), a list of its supporting references, and a link to the corresponding event in our pathway browser on our development website.&nbsp; This pathway browser web page, shown below, includes an Event Hierarchy as well as a Pathway Diagram and Event Details section.&nbsp;</p>\r\n<p><img src=\"images/pathway_browser.png\" alt=\"Pathway Browser\" /></p>\r\n<p>In the pathway diagram, reactions are manually laid out showing their relationships to one another. The details section of this webpage provides more detailed dynamic descriptions of the molecular composition and hierarchical organization of reactions.&nbsp;</p>\r\n<p>Selecting any event in the hierarchy (left panel) will bring you to its location in the pathway diagram, and the corresponding event(s) will be highlighted in blue in the diagram. A text and molecular description of the reaction or event can be viewed in the Details tab in the panel below the diagram. The description of the reaction may also be accessed by clicking on the reaction node in the diagram. A text description (and cross-references) for individual reaction component molecules can be displayed by selecting the molecule of interest in the diagram. More detailed instructions for navigating the website can be found <a href=\"https://docs.google.com/document/d/10XDgRsZsjDGpWPVDeeAqnBPzRxgHa-sTr-W0YgVHn28/edit#heading=h.akq8x59f82g1\">here</a>.&nbsp;</p>\r\n<p>We are asking reviewers to verify that the pathways and reactions described in the text document are annotated clearly and completely and that the molecular details of the reactions (described in-depth on the webpages) are accurate.&nbsp; Additional instructions for navigating the web pages can be found here. We would appreciate any comments or suggestions on the user interface as well.</p>\r\n<p><strong>How long does it take to review a module?</strong></p>\r\n<p>Depending on the size of the module, reviewers take anywhere from a day to a month for their reviews. Typically, reviewers provide their feedback within two or three weeks. Reactome has a rolling quarterly release cycle though, so if your review takes a bit longer than expected, this is not a problem. We can include the revised module in the next release.&nbsp;</p>\r\n<p><strong>If Ive decided to review a module, how do I get started?</strong></p>\r\n<p>If youre ready to get started reviewing, send us an email at <a href=\"mailto:help@reactome.org\">help@reactome.org</a> and we can put you in contact with the curator of the pathway module. You can communicate with this curator if you have any questions and when youd like to submit your review.&nbsp;</p>\r\n<p><strong>When can I see my work published on the Reactome website?</strong></p>\r\n<p>Reactome has a quarterly database release cycle, generally in March, June, September, and December. For a revised module to be included in a given release, we ask that reviews be submitted 6 weeks (or more) before the planned release date. So for a revised module to be included in a March release, we would need to have the review back in mid-January.</p>\r\n<p><strong>How can I provide feedback and suggestions to Reactome?</strong></p>\r\n<p>Comments may be added directly to the word document provided. Please opt to tracking changes so that your comments and suggested changes are highlighted. If you prefer, you can send your review as a separate document.&nbsp;</p>\r\n<p><strong>What if my work (or other important work) has not been included or cited?</strong></p>\r\n<p>Due to limited resources, we may not have included all of the references that are relevant for a particular pathway or reaction, but we would be happy to add any that you might wish to see included. Likewise, if you would like to expand a pathway module to include events that we have not covered, we would be glad to work with you on this and would cite you as an author for any new content.&nbsp;</p>\r\n<p><strong>What if I have questions while reviewing a pathway module?</strong></p>\r\n<p>We will pair you with the curator that has worked on the pathway module that you are reviewing. You can communicate directly with this curator for any questions. You can also contact us at&nbsp; <a href=\"mailto:help@reactome.org\">help@reactome.org</a> for any technical questions.&nbsp;</p>\r\n<p><strong>How am I acknowledged for my contribution to Reactome?</strong></p>\r\n<p>Your review will be used by a Reactome curator to revise the pathway module. You will be acknowledged as a reviewer of this pathway on the web pages and in the Reactome table of contents.&nbsp; This pathway module will also be associated with a DOI and may be cited as a publication. We ask that you provide us with an ORCID identifier so that we can associate this with the pathways that you have contributed.&nbsp; Our main search allows our authors and reviewers to <a href=\"https://reactome.org/content/query?q=*&amp;species=Entries+without+species&amp;types=Person&amp;cluster=true\">query</a> for their pathway and reaction contributions using their names, and claim these contributions directly into ORCID.</p>\r\n<p><strong>&nbsp;Spread the word!</strong></p>\r\n<p>Reactome is always looking for new contributors! If you know someone that would be interested in contributing a new pathway or reviewing/revising an existing pathway module, please let them know about us and our mission.</p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-09-11 17:14:23','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','','');
/*!40000 ALTER TABLE `rlp_content` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_content_frontpage`
--

DROP TABLE IF EXISTS `rlp_content_frontpage`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_content_frontpage` (
  `content_id` int(11) NOT NULL DEFAULT '0',
  `ordering` int(11) NOT NULL DEFAULT '0',
  PRIMARY KEY (`content_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_content_frontpage`
--

LOCK TABLES `rlp_content_frontpage` WRITE;
/*!40000 ALTER TABLE `rlp_content_frontpage` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_content_frontpage` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_content_rating`
--

DROP TABLE IF EXISTS `rlp_content_rating`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_content_rating` (
  `content_id` int(11) NOT NULL DEFAULT '0',
  `rating_sum` int(10) unsigned NOT NULL DEFAULT '0',
  `rating_count` int(10) unsigned NOT NULL DEFAULT '0',
  `lastip` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  PRIMARY KEY (`content_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_content_rating`
--

LOCK TABLES `rlp_content_rating` WRITE;
/*!40000 ALTER TABLE `rlp_content_rating` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_content_rating` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_content_types`
--

DROP TABLE IF EXISTS `rlp_content_types`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_content_types` (
  `type_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `type_title` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `type_alias` varchar(400) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `table` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `rules` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `field_mappings` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `router` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `content_history_options` varchar(5120) COLLATE utf8mb4_unicode_ci DEFAULT NULL COMMENT 'JSON string for com_contenthistory options',
  PRIMARY KEY (`type_id`),
  KEY `idx_alias` (`type_alias`(100))
) ENGINE=InnoDB AUTO_INCREMENT=17 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_content_types`
--

LOCK TABLES `rlp_content_types` WRITE;
/*!40000 ALTER TABLE `rlp_content_types` DISABLE KEYS */;
INSERT INTO `rlp_content_types` VALUES (1,'Article','com_content.article','{\"special\":{\"dbtable\":\"#__content\",\"key\":\"id\",\"type\":\"Content\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"state\",\"core_alias\":\"alias\",\"core_created_time\":\"created\",\"core_modified_time\":\"modified\",\"core_body\":\"introtext\", \"core_hits\":\"hits\",\"core_publish_up\":\"publish_up\",\"core_publish_down\":\"publish_down\",\"core_access\":\"access\", \"core_params\":\"attribs\", \"core_featured\":\"featured\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"images\", \"core_urls\":\"urls\", \"core_version\":\"version\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"catid\", \"core_xreference\":\"xreference\", \"asset_id\":\"asset_id\", \"note\":\"note\"}, \"special\":{\"fulltext\":\"fulltext\"}}','ContentHelperRoute::getArticleRoute','{\"formFile\":\"administrator\\/components\\/com_content\\/models\\/forms\\/article.xml\", \"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\"],\"ignoreChanges\":[\"modified_by\", \"modified\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"ordering\"],\"convertToInt\":[\"publish_up\", \"publish_down\", \"featured\", \"ordering\"],\"displayLookup\":[{\"sourceColumn\":\"catid\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"created_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"} ]}'),(2,'Contact','com_contact.contact','{\"special\":{\"dbtable\":\"#__contact_details\",\"key\":\"id\",\"type\":\"Contact\",\"prefix\":\"ContactTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"name\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created\",\"core_modified_time\":\"modified\",\"core_body\":\"address\", \"core_hits\":\"hits\",\"core_publish_up\":\"publish_up\",\"core_publish_down\":\"publish_down\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"featured\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"image\", \"core_urls\":\"webpage\", \"core_version\":\"version\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"catid\", \"core_xreference\":\"xreference\", \"asset_id\":\"null\"}, \"special\":{\"con_position\":\"con_position\",\"suburb\":\"suburb\",\"state\":\"state\",\"country\":\"country\",\"postcode\":\"postcode\",\"telephone\":\"telephone\",\"fax\":\"fax\",\"misc\":\"misc\",\"email_to\":\"email_to\",\"default_con\":\"default_con\",\"user_id\":\"user_id\",\"mobile\":\"mobile\",\"sortname1\":\"sortname1\",\"sortname2\":\"sortname2\",\"sortname3\":\"sortname3\"}}','ContactHelperRoute::getContactRoute','{\"formFile\":\"administrator\\/components\\/com_contact\\/models\\/forms\\/contact.xml\",\"hideFields\":[\"default_con\",\"checked_out\",\"checked_out_time\",\"version\",\"xreference\"],\"ignoreChanges\":[\"modified_by\", \"modified\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"ordering\"],\"convertToInt\":[\"publish_up\", \"publish_down\", \"featured\", \"ordering\"], \"displayLookup\":[ {\"sourceColumn\":\"created_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"catid\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"} ] }'),(3,'Newsfeed','com_newsfeeds.newsfeed','{\"special\":{\"dbtable\":\"#__newsfeeds\",\"key\":\"id\",\"type\":\"Newsfeed\",\"prefix\":\"NewsfeedsTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"name\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created\",\"core_modified_time\":\"modified\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"publish_up\",\"core_publish_down\":\"publish_down\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"featured\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"images\", \"core_urls\":\"link\", \"core_version\":\"version\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"catid\", \"core_xreference\":\"xreference\", \"asset_id\":\"null\"}, \"special\":{\"numarticles\":\"numarticles\",\"cache_time\":\"cache_time\",\"rtl\":\"rtl\"}}','NewsfeedsHelperRoute::getNewsfeedRoute','{\"formFile\":\"administrator\\/components\\/com_newsfeeds\\/models\\/forms\\/newsfeed.xml\",\"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\"],\"ignoreChanges\":[\"modified_by\", \"modified\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"ordering\"],\"convertToInt\":[\"publish_up\", \"publish_down\", \"featured\", \"ordering\"],\"displayLookup\":[{\"sourceColumn\":\"catid\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"created_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"} ]}'),(4,'User','com_users.user','{\"special\":{\"dbtable\":\"#__users\",\"key\":\"id\",\"type\":\"User\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"name\",\"core_state\":\"null\",\"core_alias\":\"username\",\"core_created_time\":\"registerdate\",\"core_modified_time\":\"lastvisitDate\",\"core_body\":\"null\", \"core_hits\":\"null\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"access\":\"null\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"null\", \"core_language\":\"null\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"null\", \"core_ordering\":\"null\", \"core_metakey\":\"null\", \"core_metadesc\":\"null\", \"core_catid\":\"null\", \"core_xreference\":\"null\", \"asset_id\":\"null\"}, \"special\":{}}','UsersHelperRoute::getUserRoute',''),(5,'Article Category','com_content.category','{\"special\":{\"dbtable\":\"#__categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\",\"extension\":\"extension\",\"note\":\"note\"}}','ContentHelperRoute::getCategoryRoute','{\"formFile\":\"administrator\\/components\\/com_categories\\/models\\/forms\\/category.xml\", \"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\",\"lft\",\"rgt\",\"level\",\"path\",\"extension\"], \"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"],\"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"parent_id\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}]}'),(6,'Contact Category','com_contact.category','{\"special\":{\"dbtable\":\"#__categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\",\"extension\":\"extension\",\"note\":\"note\"}}','ContactHelperRoute::getCategoryRoute','{\"formFile\":\"administrator\\/components\\/com_categories\\/models\\/forms\\/category.xml\", \"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\",\"lft\",\"rgt\",\"level\",\"path\",\"extension\"], \"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"],\"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"parent_id\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}]}'),(7,'Newsfeeds Category','com_newsfeeds.category','{\"special\":{\"dbtable\":\"#__categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\",\"extension\":\"extension\",\"note\":\"note\"}}','NewsfeedsHelperRoute::getCategoryRoute','{\"formFile\":\"administrator\\/components\\/com_categories\\/models\\/forms\\/category.xml\", \"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\",\"lft\",\"rgt\",\"level\",\"path\",\"extension\"], \"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"],\"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"parent_id\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}]}'),(8,'Tag','com_tags.tag','{\"special\":{\"dbtable\":\"#__tags\",\"key\":\"tag_id\",\"type\":\"Tag\",\"prefix\":\"TagsTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"featured\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"images\", \"core_urls\":\"urls\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"null\", \"core_xreference\":\"null\", \"asset_id\":\"null\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\"}}','TagsHelperRoute::getTagRoute','{\"formFile\":\"administrator\\/components\\/com_tags\\/models\\/forms\\/tag.xml\", \"hideFields\":[\"checked_out\",\"checked_out_time\",\"version\", \"lft\", \"rgt\", \"level\", \"path\", \"urls\", \"publish_up\", \"publish_down\"],\"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"],\"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}, {\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}, {\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}]}'),(9,'Banner','com_banners.banner','{\"special\":{\"dbtable\":\"#__banners\",\"key\":\"id\",\"type\":\"Banner\",\"prefix\":\"BannersTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"name\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created\",\"core_modified_time\":\"modified\",\"core_body\":\"description\", \"core_hits\":\"null\",\"core_publish_up\":\"publish_up\",\"core_publish_down\":\"publish_down\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"images\", \"core_urls\":\"link\", \"core_version\":\"version\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"catid\", \"core_xreference\":\"null\", \"asset_id\":\"null\"}, \"special\":{\"imptotal\":\"imptotal\", \"impmade\":\"impmade\", \"clicks\":\"clicks\", \"clickurl\":\"clickurl\", \"custombannercode\":\"custombannercode\", \"cid\":\"cid\", \"purchase_type\":\"purchase_type\", \"track_impressions\":\"track_impressions\", \"track_clicks\":\"track_clicks\"}}','','{\"formFile\":\"administrator\\/components\\/com_banners\\/models\\/forms\\/banner.xml\", \"hideFields\":[\"checked_out\",\"checked_out_time\",\"version\", \"reset\"],\"ignoreChanges\":[\"modified_by\", \"modified\", \"checked_out\", \"checked_out_time\", \"version\", \"imptotal\", \"impmade\", \"reset\"], \"convertToInt\":[\"publish_up\", \"publish_down\", \"ordering\"], \"displayLookup\":[{\"sourceColumn\":\"catid\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}, {\"sourceColumn\":\"cid\",\"targetTable\":\"#__banner_clients\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}, {\"sourceColumn\":\"created_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"modified_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"} ]}'),(10,'Banners Category','com_banners.category','{\"special\":{\"dbtable\":\"#__categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\": {\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\",\"extension\":\"extension\",\"note\":\"note\"}}','','{\"formFile\":\"administrator\\/components\\/com_categories\\/models\\/forms\\/category.xml\", \"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\",\"lft\",\"rgt\",\"level\",\"path\",\"extension\"], \"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"], \"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"parent_id\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}]}'),(11,'Banner Client','com_banners.client','{\"special\":{\"dbtable\":\"#__banner_clients\",\"key\":\"id\",\"type\":\"Client\",\"prefix\":\"BannersTable\"}}','','','','{\"formFile\":\"administrator\\/components\\/com_banners\\/models\\/forms\\/client.xml\", \"hideFields\":[\"checked_out\",\"checked_out_time\"], \"ignoreChanges\":[\"checked_out\", \"checked_out_time\"], \"convertToInt\":[], \"displayLookup\":[]}'),(12,'User Notes','com_users.note','{\"special\":{\"dbtable\":\"#__user_notes\",\"key\":\"id\",\"type\":\"Note\",\"prefix\":\"UsersTable\"}}','','','','{\"formFile\":\"administrator\\/components\\/com_users\\/models\\/forms\\/note.xml\", \"hideFields\":[\"checked_out\",\"checked_out_time\", \"publish_up\", \"publish_down\"],\"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\"], \"convertToInt\":[\"publish_up\", \"publish_down\"],\"displayLookup\":[{\"sourceColumn\":\"catid\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}, {\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}, {\"sourceColumn\":\"user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}, {\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}]}'),(13,'User Notes Category','com_users.category','{\"special\":{\"dbtable\":\"#__categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\",\"extension\":\"extension\",\"note\":\"note\"}}','','{\"formFile\":\"administrator\\/components\\/com_categories\\/models\\/forms\\/category.xml\", \"hideFields\":[\"checked_out\",\"checked_out_time\",\"version\",\"lft\",\"rgt\",\"level\",\"path\",\"extension\"], \"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"], \"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}, {\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"parent_id\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}]}'),(14,'Hikashop Product','com_hikashop.product','{\"special\":{\"dbtable\":\"#__hikashop_product\",\"key\":\"product_id\",\"type\":\"Hikashop Product\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"CoreContent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":[{\"core_content_item_id\":\"product_id\",\"core_title\":\"product_name\",\"core_state\":\"product_published\",\"core_alias\":\"product_alias\",\"core_created_time\":\"product_created\",\"core_modified_time\":\"product_modified\",\"core_body\":\"product_description\",\"core_hits\":\"product_hit\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"null\",\"core_params\":\"null\",\"core_featured\":\"null\",\"core_metadata\":\"null\",\"core_language\":\"null\",\"core_images\":\"null\",\"core_urls\":\"null\",\"core_version\":\"null\",\"core_ordering\":\"null\",\"core_metakey\":\"product_keywords\",\"core_metadesc\":\"product_meta_description\",\"core_catid\":\"null\",\"core_xreference\":\"null\",\"asset_id\":\"null\"}],\"special\":[[]]}','hikashopTagRouteHelper::getProductRoute',NULL),(15,'jDownloads Download','com_jdownloads.download','{\"special\":{\"dbtable\":\"#__jdownloads_files\",\"key\":\"file_id\",\"type\":\"Download\",\"prefix\":\"JdownloadsTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Download\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"file_id\",\"core_title\":\"file_title\",\"core_state\":\"published\",\"core_alias\":\"file_alias\",\"core_created_time\":\"date_added\",\"core_modified_time\":\"modified_date\",\"core_body\":\"description\", \"core_hits\":\"views\",\"core_publish_up\":\"publish_from\",\"core_publish_down\":\"publish_to\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"featured\", \"core_metadata\":\"null\", \"core_language\":\"language\", \"core_images\":\"images\", \"core_urls\":\"null\", \"core_version\":\"null\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"cat_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"description_long\":\"description_long\"}}','JdownloadsHelperRoute::getDownloadRoute',NULL),(16,'jDownloads Category','com_jdownloads.category','{\"special\":{\"dbtable\":\"#__jdownloads_categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JdownloadsTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"views\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"null\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"null\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"null\",\"extension\":\"null\",\"note\":\"null\"}}','JdownloadsHelperRoute::getCategoryRoute',NULL);
/*!40000 ALTER TABLE `rlp_content_types` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_contentitem_tag_map`
--

DROP TABLE IF EXISTS `rlp_contentitem_tag_map`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_contentitem_tag_map` (
  `type_alias` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `core_content_id` int(10) unsigned NOT NULL COMMENT 'PK from the core content table',
  `content_item_id` int(11) NOT NULL COMMENT 'PK from the content type table',
  `tag_id` int(10) unsigned NOT NULL COMMENT 'PK from the tag table',
  `tag_date` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'Date of most recent save for this tag-item',
  `type_id` mediumint(8) NOT NULL COMMENT 'PK from the content_type table',
  UNIQUE KEY `uc_ItemnameTagid` (`type_id`,`content_item_id`,`tag_id`),
  KEY `idx_tag_type` (`tag_id`,`type_id`),
  KEY `idx_date_id` (`tag_date`,`tag_id`),
  KEY `idx_core_content_id` (`core_content_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Maps items from content tables to tags';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_contentitem_tag_map`
--

LOCK TABLES `rlp_contentitem_tag_map` WRITE;
/*!40000 ALTER TABLE `rlp_contentitem_tag_map` DISABLE KEYS */;
INSERT INTO `rlp_contentitem_tag_map` VALUES ('com_content.article',1,54,5,'0000-00-00 00:00:00',1),('com_content.article',1,54,6,'0000-00-00 00:00:00',1),('com_content.article',7,71,7,'0000-00-00 00:00:00',1),('com_content.article',6,72,7,'0000-00-00 00:00:00',1),('com_content.article',5,74,7,'0000-00-00 00:00:00',1),('com_content.article',4,77,7,'0000-00-00 00:00:00',1),('com_content.article',3,78,7,'0000-00-00 00:00:00',1),('com_content.article',2,90,7,'0000-00-00 00:00:00',1),('com_content.article',8,98,7,'0000-00-00 00:00:00',1),('com_content.article',9,102,7,'0000-00-00 00:00:00',1),('com_content.article',10,103,7,'0000-00-00 00:00:00',1),('com_content.article',11,106,7,'0000-00-00 00:00:00',1),('com_content.article',12,116,7,'0000-00-00 00:00:00',1),('com_content.article',13,125,7,'0000-00-00 00:00:00',1),('com_content.article',14,127,5,'0000-00-00 00:00:00',1),('com_content.article',15,137,7,'0000-00-00 00:00:00',1),('com_content.article',16,142,7,'0000-00-00 00:00:00',1);
/*!40000 ALTER TABLE `rlp_contentitem_tag_map` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_core_log_searches`
--

DROP TABLE IF EXISTS `rlp_core_log_searches`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_core_log_searches` (
  `search_term` varchar(128) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `hits` int(10) unsigned NOT NULL DEFAULT '0'
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_core_log_searches`
--

LOCK TABLES `rlp_core_log_searches` WRITE;
/*!40000 ALTER TABLE `rlp_core_log_searches` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_core_log_searches` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_easyjoomlabackup`
--

DROP TABLE IF EXISTS `rlp_easyjoomlabackup`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_easyjoomlabackup` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `date` datetime NOT NULL,
  `comment` tinytext NOT NULL,
  `type` varchar(32) NOT NULL,
  `size` varchar(12) NOT NULL,
  `duration` varchar(8) NOT NULL,
  `name` tinytext NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `id` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_easyjoomlabackup`
--

LOCK TABLES `rlp_easyjoomlabackup` WRITE;
/*!40000 ALTER TABLE `rlp_easyjoomlabackup` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_easyjoomlabackup` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_extensions`
--

DROP TABLE IF EXISTS `rlp_extensions`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_extensions` (
  `extension_id` int(11) NOT NULL AUTO_INCREMENT,
  `package_id` int(11) NOT NULL DEFAULT '0' COMMENT 'Parent package ID for extensions installed as a package.',
  `name` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL,
  `type` varchar(20) COLLATE utf8mb4_unicode_ci NOT NULL,
  `element` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL,
  `folder` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL,
  `client_id` tinyint(3) NOT NULL,
  `enabled` tinyint(3) NOT NULL DEFAULT '0',
  `access` int(10) unsigned NOT NULL DEFAULT '1',
  `protected` tinyint(3) NOT NULL DEFAULT '0',
  `manifest_cache` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `params` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `custom_data` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `system_data` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `checked_out` int(10) unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `ordering` int(11) DEFAULT '0',
  `state` int(11) DEFAULT '0',
  PRIMARY KEY (`extension_id`),
  KEY `element_clientid` (`element`,`client_id`),
  KEY `element_folder_clientid` (`element`,`folder`,`client_id`),
  KEY `extension` (`type`,`element`,`folder`,`client_id`)
) ENGINE=InnoDB AUTO_INCREMENT=10355 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_extensions`
--

LOCK TABLES `rlp_extensions` WRITE;
/*!40000 ALTER TABLE `rlp_extensions` DISABLE KEYS */;
INSERT INTO `rlp_extensions` VALUES (1,0,'com_mailto','component','com_mailto','',0,1,1,1,'{\"name\":\"com_mailto\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\\t\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_MAILTO_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mailto\"}','','','',0,'0000-00-00 00:00:00',0,0),(2,0,'com_wrapper','component','com_wrapper','',0,1,1,1,'{\"name\":\"com_wrapper\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\\n\\t\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_WRAPPER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"wrapper\"}','','','',0,'0000-00-00 00:00:00',0,0),(3,0,'com_admin','component','com_admin','',1,1,1,1,'{\"name\":\"com_admin\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_ADMIN_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(4,0,'com_banners','component','com_banners','',1,1,1,0,'{\"name\":\"com_banners\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_BANNERS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"banners\"}','{\"purchase_type\":\"3\",\"track_impressions\":\"0\",\"track_clicks\":\"0\",\"metakey_prefix\":\"\",\"save_history\":\"1\",\"history_limit\":10}','','',0,'0000-00-00 00:00:00',0,0),(5,0,'com_cache','component','com_cache','',1,1,1,1,'{\"name\":\"com_cache\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CACHE_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(6,0,'com_categories','component','com_categories','',1,1,1,1,'{\"name\":\"com_categories\",\"type\":\"component\",\"creationDate\":\"December 2007\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CATEGORIES_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(7,0,'com_checkin','component','com_checkin','',1,1,1,1,'{\"name\":\"com_checkin\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CHECKIN_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(8,0,'com_contact','component','com_contact','',1,1,1,0,'{\"name\":\"com_contact\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CONTACT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contact\"}','{\"show_contact_category\":\"hide\",\"save_history\":\"1\",\"history_limit\":10,\"show_contact_list\":\"0\",\"presentation_style\":\"sliders\",\"show_name\":\"1\",\"show_position\":\"1\",\"show_email\":\"0\",\"show_street_address\":\"1\",\"show_suburb\":\"1\",\"show_state\":\"1\",\"show_postcode\":\"1\",\"show_country\":\"1\",\"show_telephone\":\"1\",\"show_mobile\":\"1\",\"show_fax\":\"1\",\"show_webpage\":\"1\",\"show_misc\":\"1\",\"show_image\":\"1\",\"image\":\"\",\"allow_vcard\":\"0\",\"show_articles\":\"0\",\"show_profile\":\"0\",\"show_links\":\"0\",\"linka_name\":\"\",\"linkb_name\":\"\",\"linkc_name\":\"\",\"linkd_name\":\"\",\"linke_name\":\"\",\"contact_icons\":\"0\",\"icon_address\":\"\",\"icon_email\":\"\",\"icon_telephone\":\"\",\"icon_mobile\":\"\",\"icon_fax\":\"\",\"icon_misc\":\"\",\"show_headings\":\"1\",\"show_position_headings\":\"1\",\"show_email_headings\":\"0\",\"show_telephone_headings\":\"1\",\"show_mobile_headings\":\"0\",\"show_fax_headings\":\"0\",\"allow_vcard_headings\":\"0\",\"show_suburb_headings\":\"1\",\"show_state_headings\":\"1\",\"show_country_headings\":\"1\",\"show_email_form\":\"1\",\"show_email_copy\":\"1\",\"banned_email\":\"\",\"banned_subject\":\"\",\"banned_text\":\"\",\"validate_session\":\"1\",\"custom_reply\":\"0\",\"redirect\":\"\",\"show_category_crumb\":\"0\",\"metakey\":\"\",\"metadesc\":\"\",\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(9,0,'com_cpanel','component','com_cpanel','',1,1,1,1,'{\"name\":\"com_cpanel\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CPANEL_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(10,0,'com_installer','component','com_installer','',1,1,1,1,'{\"name\":\"com_installer\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_INSTALLER_XML_DESCRIPTION\",\"group\":\"\"}','{\"show_jed_info\":\"0\",\"cachetimeout\":\"6\",\"minimum_stability\":\"4\"}','','',0,'0000-00-00 00:00:00',0,0),(11,0,'com_languages','component','com_languages','',1,1,1,1,'{\"name\":\"com_languages\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_LANGUAGES_XML_DESCRIPTION\",\"group\":\"\"}','{\"administrator\":\"en-GB\",\"site\":\"en-GB\"}','','',0,'0000-00-00 00:00:00',0,0),(12,0,'com_login','component','com_login','',1,1,1,1,'{\"name\":\"com_login\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_LOGIN_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(13,0,'com_media','component','com_media','',1,1,0,1,'{\"name\":\"com_media\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_MEDIA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"media\"}','{\"upload_extensions\":\"bmp,csv,doc,gif,ico,jpg,jpeg,odg,odp,ods,odt,pdf,png,ppt,swf,txt,xcf,xls,BMP,CSV,DOC,GIF,ICO,JPG,JPEG,ODG,ODP,ODS,ODT,PDF,PNG,PPT,SWF,TXT,XCF,XLS\",\"upload_maxsize\":\"10\",\"file_path\":\"images\",\"image_path\":\"images\",\"restrict_uploads\":\"1\",\"allowed_media_usergroup\":\"3\",\"check_mime\":\"1\",\"image_extensions\":\"bmp,gif,jpg,png\",\"ignore_extensions\":\"\",\"upload_mime\":\"image\\/jpeg,image\\/gif,image\\/png,image\\/bmp,application\\/x-shockwave-flash,application\\/msword,application\\/excel,application\\/pdf,application\\/powerpoint,text\\/plain,application\\/x-zip\",\"upload_mime_illegal\":\"text\\/html\"}','','',0,'0000-00-00 00:00:00',0,0),(14,0,'com_menus','component','com_menus','',1,1,1,1,'{\"name\":\"com_menus\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_MENUS_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(15,0,'com_messages','component','com_messages','',1,1,1,1,'{\"name\":\"com_messages\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_MESSAGES_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(16,0,'com_modules','component','com_modules','',1,1,1,1,'{\"name\":\"com_modules\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_MODULES_XML_DESCRIPTION\",\"group\":\"\"}','{\"redirect_edit\":\"site\"}','','',0,'0000-00-00 00:00:00',0,0),(17,0,'com_newsfeeds','component','com_newsfeeds','',1,1,1,0,'{\"name\":\"com_newsfeeds\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_NEWSFEEDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"newsfeeds\"}','{\"newsfeed_layout\":\"_:default\",\"save_history\":\"1\",\"history_limit\":5,\"show_feed_image\":\"1\",\"show_feed_description\":\"1\",\"show_item_description\":\"1\",\"feed_character_count\":\"0\",\"feed_display_order\":\"des\",\"float_first\":\"right\",\"float_second\":\"right\",\"show_tags\":\"1\",\"category_layout\":\"_:default\",\"show_category_title\":\"1\",\"show_description\":\"1\",\"show_description_image\":\"1\",\"maxLevel\":\"-1\",\"show_empty_categories\":\"0\",\"show_subcat_desc\":\"1\",\"show_cat_items\":\"1\",\"show_cat_tags\":\"1\",\"show_base_description\":\"1\",\"maxLevelcat\":\"-1\",\"show_empty_categories_cat\":\"0\",\"show_subcat_desc_cat\":\"1\",\"show_cat_items_cat\":\"1\",\"filter_field\":\"1\",\"show_pagination_limit\":\"1\",\"show_headings\":\"1\",\"show_articles\":\"0\",\"show_link\":\"1\",\"show_pagination\":\"1\",\"show_pagination_results\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(18,0,'com_plugins','component','com_plugins','',1,1,1,1,'{\"name\":\"com_plugins\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_PLUGINS_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(19,0,'com_search','component','com_search','',1,1,1,0,'{\"name\":\"com_search\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_SEARCH_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"search\"}','{\"enabled\":\"0\",\"search_phrases\":\"0\",\"search_areas\":\"0\",\"show_date\":\"0\",\"opensearch_name\":\"\",\"opensearch_description\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(20,0,'com_templates','component','com_templates','',1,1,1,1,'{\"name\":\"com_templates\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_TEMPLATES_XML_DESCRIPTION\",\"group\":\"\"}','{\"template_positions_display\":\"0\",\"upload_limit\":\"10\",\"image_formats\":\"gif,bmp,jpg,jpeg,png\",\"source_formats\":\"txt,less,ini,xml,js,php,css\",\"font_formats\":\"woff,ttf,otf\",\"compressed_formats\":\"zip\"}','','',0,'0000-00-00 00:00:00',0,0),(22,0,'com_content','component','com_content','',1,1,0,1,'{\"name\":\"com_content\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CONTENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"content\"}','{\"article_layout\":\"_:default\",\"show_title\":\"1\",\"link_titles\":\"1\",\"show_intro\":\"1\",\"info_block_position\":\"0\",\"info_block_show_title\":\"1\",\"show_category\":\"0\",\"link_category\":\"0\",\"show_parent_category\":\"0\",\"link_parent_category\":\"0\",\"show_associations\":\"0\",\"flags\":\"1\",\"show_author\":\"0\",\"link_author\":\"1\",\"show_create_date\":\"0\",\"show_modify_date\":\"0\",\"show_publish_date\":\"0\",\"show_item_navigation\":\"0\",\"show_vote\":\"0\",\"show_readmore\":\"0\",\"show_readmore_title\":\"0\",\"readmore_limit\":\"100\",\"show_tags\":\"0\",\"show_icons\":\"0\",\"show_print_icon\":\"0\",\"show_email_icon\":\"0\",\"show_hits\":\"0\",\"show_noauth\":\"0\",\"urls_position\":\"0\",\"captcha\":\"0\",\"show_publishing_options\":\"1\",\"show_article_options\":\"1\",\"save_history\":\"1\",\"history_limit\":10,\"show_urls_images_frontend\":\"0\",\"show_urls_images_backend\":\"1\",\"targeta\":0,\"targetb\":0,\"targetc\":0,\"float_intro\":\"none\",\"float_fulltext\":\"none\",\"category_layout\":\"_:blog\",\"show_category_heading_title_text\":\"0\",\"show_category_title\":\"0\",\"show_description\":\"0\",\"show_description_image\":\"0\",\"maxLevel\":\"0\",\"show_empty_categories\":\"0\",\"show_no_articles\":\"0\",\"show_subcat_desc\":\"0\",\"show_cat_num_articles\":\"0\",\"show_cat_tags\":\"0\",\"show_base_description\":\"0\",\"maxLevelcat\":\"-1\",\"show_empty_categories_cat\":\"0\",\"show_subcat_desc_cat\":\"0\",\"show_cat_num_articles_cat\":\"0\",\"num_leading_articles\":\"0\",\"num_intro_articles\":\"1\",\"num_columns\":\"1\",\"num_links\":\"4\",\"multi_column_order\":\"0\",\"show_subcategory_content\":\"0\",\"show_pagination_limit\":\"1\",\"filter_field\":\"hide\",\"show_headings\":\"1\",\"list_show_date\":\"0\",\"date_format\":\"\",\"list_show_hits\":\"0\",\"list_show_author\":\"0\",\"list_show_votes\":\"0\",\"list_show_ratings\":\"0\",\"orderby_pri\":\"none\",\"orderby_sec\":\"order\",\"order_date\":\"published\",\"show_pagination\":\"2\",\"show_pagination_results\":\"0\",\"show_featured\":\"hide\",\"show_feed_link\":\"1\",\"feed_summary\":\"0\",\"feed_show_readmore\":\"0\",\"sef_advanced\":0,\"sef_ids\":0,\"custom_fields_enable\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(23,0,'com_config','component','com_config','',1,1,0,1,'{\"name\":\"com_config\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CONFIG_XML_DESCRIPTION\",\"group\":\"\"}','{\"filters\":{\"1\":{\"filter_type\":\"NH\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"9\":{\"filter_type\":\"BL\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"6\":{\"filter_type\":\"BL\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"7\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"2\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"3\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"4\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"11\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"5\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"10\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"8\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"}}}','','',0,'0000-00-00 00:00:00',0,0),(24,0,'com_redirect','component','com_redirect','',1,1,0,1,'{\"name\":\"com_redirect\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_REDIRECT_XML_DESCRIPTION\",\"group\":\"\"}','{\"mode\":\"0\",\"separator\":\"|\"}','','',0,'0000-00-00 00:00:00',0,0),(25,0,'com_users','component','com_users','',1,1,0,1,'{\"name\":\"com_users\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_USERS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"users\"}','{\"allowUserRegistration\":\"0\",\"new_usertype\":\"2\",\"guest_usergroup\":\"9\",\"sendpassword\":\"1\",\"useractivation\":\"1\",\"mail_to_admin\":\"0\",\"captcha\":\"\",\"frontend_userparams\":\"1\",\"site_language\":\"0\",\"change_login_name\":\"0\",\"reset_count\":\"10\",\"reset_time\":\"1\",\"minimum_length\":\"8\",\"minimum_integers\":\"1\",\"minimum_symbols\":\"1\",\"minimum_uppercase\":\"1\",\"save_history\":\"1\",\"history_limit\":5,\"mailSubjectPrefix\":\"\",\"mailBodySuffix\":\"\",\"debugUsers\":\"1\",\"debugGroups\":\"1\",\"custom_fields_enable\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(27,0,'com_finder','component','com_finder','',1,1,0,0,'{\"name\":\"com_finder\",\"type\":\"component\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_FINDER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"finder\"}','{\"enabled\":\"0\",\"show_description\":\"1\",\"description_length\":255,\"allow_empty_query\":\"0\",\"show_url\":\"0\",\"show_autosuggest\":\"1\",\"show_suggested_query\":\"1\",\"show_explained_query\":\"0\",\"show_advanced\":\"0\",\"show_advanced_tips\":\"1\",\"expand_advanced\":\"0\",\"show_date_filters\":\"1\",\"sort_order\":\"relevance\",\"sort_direction\":\"desc\",\"highlight_terms\":\"1\",\"opensearch_name\":\"\",\"opensearch_description\":\"\",\"batch_size\":\"300\",\"memory_table_limit\":20000,\"title_multiplier\":\"1.7\",\"text_multiplier\":\"0.7\",\"meta_multiplier\":\"1.2\",\"path_multiplier\":\"2.0\",\"misc_multiplier\":\"0.3\",\"stem\":\"1\",\"stemmer\":\"snowball\",\"enable_logging\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(28,0,'com_joomlaupdate','component','com_joomlaupdate','',1,1,0,1,'{\"name\":\"com_joomlaupdate\",\"type\":\"component\",\"creationDate\":\"February 2012\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.6.2\",\"description\":\"COM_JOOMLAUPDATE_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(29,0,'com_tags','component','com_tags','',1,1,1,1,'{\"name\":\"com_tags\",\"type\":\"component\",\"creationDate\":\"December 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.1.0\",\"description\":\"COM_TAGS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"tags\"}','{\"tag_layout\":\"_:default\",\"save_history\":\"1\",\"history_limit\":5,\"show_tag_title\":\"0\",\"tag_list_show_tag_image\":\"0\",\"tag_list_show_tag_description\":\"0\",\"tag_list_image\":\"\",\"show_tag_num_items\":\"0\",\"tag_list_orderby\":\"title\",\"tag_list_orderby_direction\":\"ASC\",\"show_headings\":\"0\",\"tag_list_show_date\":\"0\",\"tag_list_show_item_image\":\"0\",\"tag_list_show_item_description\":\"0\",\"tag_list_item_maximum_characters\":0,\"return_any_or_all\":\"1\",\"include_children\":\"0\",\"maximum\":200,\"tag_list_language_filter\":\"all\",\"tags_layout\":\"_:default\",\"all_tags_orderby\":\"title\",\"all_tags_orderby_direction\":\"ASC\",\"all_tags_show_tag_image\":\"0\",\"all_tags_show_tag_descripion\":\"0\",\"all_tags_tag_maximum_characters\":20,\"all_tags_show_tag_hits\":\"0\",\"filter_field\":\"1\",\"show_pagination_limit\":\"1\",\"show_pagination\":\"2\",\"show_pagination_results\":\"1\",\"tag_field_ajax_mode\":\"1\",\"show_feed_link\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(30,0,'com_contenthistory','component','com_contenthistory','',1,1,1,0,'{\"name\":\"com_contenthistory\",\"type\":\"component\",\"creationDate\":\"May 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.0\",\"description\":\"COM_CONTENTHISTORY_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contenthistory\"}','','','',0,'0000-00-00 00:00:00',0,0),(31,0,'com_ajax','component','com_ajax','',1,1,1,1,'{\"name\":\"com_ajax\",\"type\":\"component\",\"creationDate\":\"August 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.0\",\"description\":\"COM_AJAX_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"ajax\"}','','','',0,'0000-00-00 00:00:00',0,0),(32,0,'com_postinstall','component','com_postinstall','',1,1,1,1,'{\"name\":\"com_postinstall\",\"type\":\"component\",\"creationDate\":\"September 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.0\",\"description\":\"COM_POSTINSTALL_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(33,0,'com_fields','component','com_fields','',1,1,1,0,'{\"name\":\"com_fields\",\"type\":\"component\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"COM_FIELDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"fields\"}','','','',0,'0000-00-00 00:00:00',0,0),(34,0,'com_associations','component','com_associations','',1,1,1,0,'{\"name\":\"com_associations\",\"type\":\"component\",\"creationDate\":\"January 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"COM_ASSOCIATIONS_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(35,0,'com_privacy','component','com_privacy','',1,1,1,1,'{\"name\":\"com_privacy\",\"type\":\"component\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"COM_PRIVACY_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"privacy\"}','','','',0,'0000-00-00 00:00:00',0,0),(36,0,'com_actionlogs','component','com_actionlogs','',1,1,1,1,'{\"name\":\"com_actionlogs\",\"type\":\"component\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"COM_ACTIONLOGS_XML_DESCRIPTION\",\"group\":\"\"}','{\"ip_logging\":0,\"csv_delimiter\":\",\",\"loggable_extensions\":[\"com_banners\",\"com_cache\",\"com_categories\",\"com_checkin\",\"com_config\",\"com_contact\",\"com_content\",\"com_installer\",\"com_media\",\"com_menus\",\"com_messages\",\"com_modules\",\"com_newsfeeds\",\"com_plugins\",\"com_redirect\",\"com_tags\",\"com_templates\",\"com_users\"]}','','',0,'0000-00-00 00:00:00',0,0),(102,0,'LIB_PHPUTF8','library','phputf8','',0,1,1,1,'{\"name\":\"LIB_PHPUTF8\",\"type\":\"library\",\"creationDate\":\"2006\",\"author\":\"Harry Fuecks\",\"copyright\":\"Copyright various authors\",\"authorEmail\":\"hfuecks@gmail.com\",\"authorUrl\":\"http:\\/\\/sourceforge.net\\/projects\\/phputf8\",\"version\":\"0.5\",\"description\":\"LIB_PHPUTF8_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"phputf8\"}','','','',0,'0000-00-00 00:00:00',0,0),(103,0,'LIB_JOOMLA','library','joomla','',0,1,1,1,'{\"name\":\"LIB_JOOMLA\",\"type\":\"library\",\"creationDate\":\"2008\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"https:\\/\\/www.joomla.org\",\"version\":\"13.1\",\"description\":\"LIB_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','{\"mediaversion\":\"f5591e67bba898fe2965fe3ff1c6a90d\"}','','',0,'0000-00-00 00:00:00',0,0),(104,0,'LIB_IDNA','library','idna_convert','',0,1,1,1,'{\"name\":\"LIB_IDNA\",\"type\":\"library\",\"creationDate\":\"2004\",\"author\":\"phlyLabs\",\"copyright\":\"2004-2011 phlyLabs Berlin, http:\\/\\/phlylabs.de\",\"authorEmail\":\"phlymail@phlylabs.de\",\"authorUrl\":\"http:\\/\\/phlylabs.de\",\"version\":\"0.8.0\",\"description\":\"LIB_IDNA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"idna_convert\"}','','','',0,'0000-00-00 00:00:00',0,0),(105,0,'FOF','library','fof','',0,1,1,1,'{\"name\":\"FOF\",\"type\":\"library\",\"creationDate\":\"2015-04-22 13:15:32\",\"author\":\"Nicholas K. Dionysopoulos \\/ Akeeba Ltd\",\"copyright\":\"(C)2011-2015 Nicholas K. Dionysopoulos\",\"authorEmail\":\"nicholas@akeebabackup.com\",\"authorUrl\":\"https:\\/\\/www.akeebabackup.com\",\"version\":\"2.4.3\",\"description\":\"LIB_FOF_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"fof\"}','','','',0,'0000-00-00 00:00:00',0,0),(106,0,'LIB_PHPASS','library','phpass','',0,1,1,1,'{\"name\":\"LIB_PHPASS\",\"type\":\"library\",\"creationDate\":\"2004-2006\",\"author\":\"Solar Designer\",\"copyright\":\"\",\"authorEmail\":\"solar@openwall.com\",\"authorUrl\":\"http:\\/\\/www.openwall.com\\/phpass\\/\",\"version\":\"0.3\",\"description\":\"LIB_PHPASS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"phpass\"}','','','',0,'0000-00-00 00:00:00',0,0),(200,0,'mod_articles_archive','module','mod_articles_archive','',0,1,1,0,'{\"name\":\"mod_articles_archive\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_ARTICLES_ARCHIVE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_archive\"}','','','',0,'0000-00-00 00:00:00',0,0),(201,0,'mod_articles_latest','module','mod_articles_latest','',0,1,1,0,'{\"name\":\"mod_articles_latest\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_LATEST_NEWS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_latest\"}','','','',0,'0000-00-00 00:00:00',0,0),(202,0,'mod_articles_popular','module','mod_articles_popular','',0,1,1,0,'{\"name\":\"mod_articles_popular\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_POPULAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_popular\"}','','','',0,'0000-00-00 00:00:00',0,0),(203,0,'mod_banners','module','mod_banners','',0,1,1,0,'{\"name\":\"mod_banners\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_BANNERS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_banners\"}','','','',0,'0000-00-00 00:00:00',0,0),(204,0,'mod_breadcrumbs','module','mod_breadcrumbs','',0,1,1,1,'{\"name\":\"mod_breadcrumbs\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_BREADCRUMBS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_breadcrumbs\"}','','','',0,'0000-00-00 00:00:00',0,0),(205,0,'mod_custom','module','mod_custom','',0,1,1,1,'{\"name\":\"mod_custom\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_CUSTOM_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_custom\"}','','','',0,'0000-00-00 00:00:00',0,0),(206,0,'mod_feed','module','mod_feed','',0,1,1,0,'{\"name\":\"mod_feed\",\"type\":\"module\",\"creationDate\":\"July 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_FEED_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_feed\"}','','','',0,'0000-00-00 00:00:00',0,0),(207,0,'mod_footer','module','mod_footer','',0,1,1,0,'{\"name\":\"mod_footer\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_FOOTER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_footer\"}','','','',0,'0000-00-00 00:00:00',0,0),(208,0,'mod_login','module','mod_login','',0,1,1,1,'{\"name\":\"mod_login\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_LOGIN_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_login\"}','','','',0,'0000-00-00 00:00:00',0,0),(209,0,'mod_menu','module','mod_menu','',0,1,1,1,'{\"name\":\"mod_menu\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_MENU_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_menu\"}','','','',0,'0000-00-00 00:00:00',0,0),(210,0,'mod_articles_news','module','mod_articles_news','',0,1,1,0,'{\"name\":\"mod_articles_news\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_ARTICLES_NEWS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_news\"}','','','',0,'0000-00-00 00:00:00',0,0),(211,0,'mod_random_image','module','mod_random_image','',0,1,1,0,'{\"name\":\"mod_random_image\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_RANDOM_IMAGE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_random_image\"}','','','',0,'0000-00-00 00:00:00',0,0),(212,0,'mod_related_items','module','mod_related_items','',0,1,1,0,'{\"name\":\"mod_related_items\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_RELATED_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_related_items\"}','','','',0,'0000-00-00 00:00:00',0,0),(213,0,'mod_search','module','mod_search','',0,1,1,0,'{\"name\":\"mod_search\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_SEARCH_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_search\"}','','','',0,'0000-00-00 00:00:00',0,0),(214,0,'mod_stats','module','mod_stats','',0,1,1,0,'{\"name\":\"mod_stats\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_STATS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_stats\"}','','','',0,'0000-00-00 00:00:00',0,0),(215,0,'mod_syndicate','module','mod_syndicate','',0,1,1,1,'{\"name\":\"mod_syndicate\",\"type\":\"module\",\"creationDate\":\"May 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_SYNDICATE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_syndicate\"}','','','',0,'0000-00-00 00:00:00',0,0),(216,0,'mod_users_latest','module','mod_users_latest','',0,1,1,0,'{\"name\":\"mod_users_latest\",\"type\":\"module\",\"creationDate\":\"December 2009\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_USERS_LATEST_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_users_latest\"}','','','',0,'0000-00-00 00:00:00',0,0),(218,0,'mod_whosonline','module','mod_whosonline','',0,1,1,0,'{\"name\":\"mod_whosonline\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_WHOSONLINE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_whosonline\"}','','','',0,'0000-00-00 00:00:00',0,0),(219,0,'mod_wrapper','module','mod_wrapper','',0,1,1,0,'{\"name\":\"mod_wrapper\",\"type\":\"module\",\"creationDate\":\"October 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_WRAPPER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_wrapper\"}','','','',0,'0000-00-00 00:00:00',0,0),(220,0,'mod_articles_category','module','mod_articles_category','',0,1,1,0,'{\"name\":\"mod_articles_category\",\"type\":\"module\",\"creationDate\":\"February 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_ARTICLES_CATEGORY_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_category\"}','','','',0,'0000-00-00 00:00:00',0,0),(221,0,'mod_articles_categories','module','mod_articles_categories','',0,1,1,0,'{\"name\":\"mod_articles_categories\",\"type\":\"module\",\"creationDate\":\"February 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_ARTICLES_CATEGORIES_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_categories\"}','','','',0,'0000-00-00 00:00:00',0,0),(222,0,'mod_languages','module','mod_languages','',0,1,1,1,'{\"name\":\"mod_languages\",\"type\":\"module\",\"creationDate\":\"February 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.5.0\",\"description\":\"MOD_LANGUAGES_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_languages\"}','','','',0,'0000-00-00 00:00:00',0,0),(223,0,'mod_finder','module','mod_finder','',0,1,0,0,'{\"name\":\"mod_finder\",\"type\":\"module\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_FINDER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_finder\"}','','','',0,'0000-00-00 00:00:00',0,0),(300,0,'mod_custom','module','mod_custom','',1,1,1,1,'{\"name\":\"mod_custom\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_CUSTOM_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_custom\"}','','','',0,'0000-00-00 00:00:00',0,0),(301,0,'mod_feed','module','mod_feed','',1,1,1,0,'{\"name\":\"mod_feed\",\"type\":\"module\",\"creationDate\":\"July 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_FEED_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_feed\"}','','','',0,'0000-00-00 00:00:00',0,0),(302,0,'mod_latest','module','mod_latest','',1,1,1,0,'{\"name\":\"mod_latest\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_LATEST_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_latest\"}','','','',0,'0000-00-00 00:00:00',0,0),(303,0,'mod_logged','module','mod_logged','',1,1,1,0,'{\"name\":\"mod_logged\",\"type\":\"module\",\"creationDate\":\"January 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_LOGGED_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_logged\"}','','','',0,'0000-00-00 00:00:00',0,0),(304,0,'mod_login','module','mod_login','',1,1,1,1,'{\"name\":\"mod_login\",\"type\":\"module\",\"creationDate\":\"March 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_LOGIN_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_login\"}','','','',0,'0000-00-00 00:00:00',0,0),(305,0,'mod_menu','module','mod_menu','',1,1,1,0,'{\"name\":\"mod_menu\",\"type\":\"module\",\"creationDate\":\"March 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_MENU_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_menu\"}','','','',0,'0000-00-00 00:00:00',0,0),(307,0,'mod_popular','module','mod_popular','',1,1,1,0,'{\"name\":\"mod_popular\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_POPULAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_popular\"}','','','',0,'0000-00-00 00:00:00',0,0),(308,0,'mod_quickicon','module','mod_quickicon','',1,1,1,1,'{\"name\":\"mod_quickicon\",\"type\":\"module\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_QUICKICON_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_quickicon\"}','','','',0,'0000-00-00 00:00:00',0,0),(309,0,'mod_status','module','mod_status','',1,1,1,0,'{\"name\":\"mod_status\",\"type\":\"module\",\"creationDate\":\"February 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_STATUS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_status\"}','','','',0,'0000-00-00 00:00:00',0,0),(310,0,'mod_submenu','module','mod_submenu','',1,1,1,0,'{\"name\":\"mod_submenu\",\"type\":\"module\",\"creationDate\":\"February 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_SUBMENU_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_submenu\"}','','','',0,'0000-00-00 00:00:00',0,0),(311,0,'mod_title','module','mod_title','',1,1,1,0,'{\"name\":\"mod_title\",\"type\":\"module\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_TITLE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_title\"}','','','',0,'0000-00-00 00:00:00',0,0),(312,0,'mod_toolbar','module','mod_toolbar','',1,1,1,1,'{\"name\":\"mod_toolbar\",\"type\":\"module\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_TOOLBAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_toolbar\"}','','','',0,'0000-00-00 00:00:00',0,0),(313,0,'mod_multilangstatus','module','mod_multilangstatus','',1,1,1,0,'{\"name\":\"mod_multilangstatus\",\"type\":\"module\",\"creationDate\":\"September 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_MULTILANGSTATUS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_multilangstatus\"}','{\"cache\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(314,0,'mod_version','module','mod_version','',1,1,1,0,'{\"name\":\"mod_version\",\"type\":\"module\",\"creationDate\":\"January 2012\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_VERSION_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_version\"}','{\"format\":\"short\",\"product\":\"1\",\"cache\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(315,0,'mod_stats_admin','module','mod_stats_admin','',1,1,1,0,'{\"name\":\"mod_stats_admin\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_STATS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_stats_admin\"}','{\"serverinfo\":\"0\",\"siteinfo\":\"0\",\"counter\":\"0\",\"increase\":\"0\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\"}','','',0,'0000-00-00 00:00:00',0,0),(316,0,'mod_tags_popular','module','mod_tags_popular','',0,1,1,0,'{\"name\":\"mod_tags_popular\",\"type\":\"module\",\"creationDate\":\"January 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.1.0\",\"description\":\"MOD_TAGS_POPULAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_tags_popular\"}','{\"maximum\":\"5\",\"timeframe\":\"alltime\",\"owncache\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(317,0,'mod_tags_similar','module','mod_tags_similar','',0,1,1,0,'{\"name\":\"mod_tags_similar\",\"type\":\"module\",\"creationDate\":\"January 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.1.0\",\"description\":\"MOD_TAGS_SIMILAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_tags_similar\"}','{\"maximum\":\"5\",\"matchtype\":\"any\",\"owncache\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(318,0,'mod_sampledata','module','mod_sampledata','',1,1,1,0,'{\"name\":\"mod_sampledata\",\"type\":\"module\",\"creationDate\":\"July 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.8.0\",\"description\":\"MOD_SAMPLEDATA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_sampledata\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(319,0,'mod_latestactions','module','mod_latestactions','',1,1,1,0,'{\"name\":\"mod_latestactions\",\"type\":\"module\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"MOD_LATESTACTIONS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_latestactions\"}','{}','','',0,'1970-01-01 00:00:00',0,0),(320,0,'mod_privacy_dashboard','module','mod_privacy_dashboard','',1,1,1,0,'{\"name\":\"mod_privacy_dashboard\",\"type\":\"module\",\"creationDate\":\"June 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"MOD_PRIVACY_DASHBOARD_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_privacy_dashboard\"}','{}','','',0,'1970-01-01 00:00:00',0,0),(400,0,'plg_authentication_gmail','plugin','gmail','authentication',0,0,1,0,'{\"name\":\"plg_authentication_gmail\",\"type\":\"plugin\",\"creationDate\":\"February 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_GMAIL_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"gmail\"}','{\"applysuffix\":\"0\",\"suffix\":\"\",\"verifypeer\":\"1\",\"user_blacklist\":\"\"}','','',0,'0000-00-00 00:00:00',1,0),(401,0,'plg_authentication_joomla','plugin','joomla','authentication',0,1,1,1,'{\"name\":\"plg_authentication_joomla\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_AUTH_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','','','',0,'0000-00-00 00:00:00',0,0),(402,0,'plg_authentication_ldap','plugin','ldap','authentication',0,0,1,0,'{\"name\":\"plg_authentication_ldap\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_LDAP_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"ldap\"}','{\"host\":\"\",\"port\":\"389\",\"use_ldapV3\":\"0\",\"negotiate_tls\":\"0\",\"no_referrals\":\"0\",\"auth_method\":\"bind\",\"base_dn\":\"\",\"search_string\":\"\",\"users_dn\":\"\",\"username\":\"admin\",\"password\":\"bobby7\",\"ldap_fullname\":\"fullName\",\"ldap_email\":\"mail\",\"ldap_uid\":\"uid\"}','','',0,'0000-00-00 00:00:00',3,0),(403,0,'plg_content_contact','plugin','contact','content',0,1,1,0,'{\"name\":\"plg_content_contact\",\"type\":\"plugin\",\"creationDate\":\"January 2014\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.2\",\"description\":\"PLG_CONTENT_CONTACT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contact\"}','','','',0,'0000-00-00 00:00:00',1,0),(404,0,'plg_content_emailcloak','plugin','emailcloak','content',0,1,1,0,'{\"name\":\"plg_content_emailcloak\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CONTENT_EMAILCLOAK_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"emailcloak\"}','{\"mode\":\"1\"}','','',0,'0000-00-00 00:00:00',1,0),(406,0,'plg_content_loadmodule','plugin','loadmodule','content',0,1,1,0,'{\"name\":\"plg_content_loadmodule\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_LOADMODULE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"loadmodule\"}','{\"style\":\"xhtml\"}','','',0,'0000-00-00 00:00:00',0,0),(407,0,'plg_content_pagebreak','plugin','pagebreak','content',0,1,1,0,'{\"name\":\"plg_content_pagebreak\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CONTENT_PAGEBREAK_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"pagebreak\"}','{\"title\":\"1\",\"article_index\":\"1\",\"article_index_text\":\"\",\"multipage_toc\":\"1\",\"showall\":\"1\",\"style\":\"sliders\"}','','',0,'0000-00-00 00:00:00',4,0),(408,0,'plg_content_pagenavigation','plugin','pagenavigation','content',0,1,1,0,'{\"name\":\"plg_content_pagenavigation\",\"type\":\"plugin\",\"creationDate\":\"January 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_PAGENAVIGATION_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"pagenavigation\"}','{\"position\":\"1\"}','','',0,'0000-00-00 00:00:00',5,0),(409,0,'plg_content_vote','plugin','vote','content',0,1,1,0,'{\"name\":\"plg_content_vote\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_VOTE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"vote\"}','','','',0,'0000-00-00 00:00:00',6,0),(410,0,'plg_editors_codemirror','plugin','codemirror','editors',0,1,1,1,'{\"name\":\"plg_editors_codemirror\",\"type\":\"plugin\",\"creationDate\":\"28 March 2011\",\"author\":\"Marijn Haverbeke\",\"copyright\":\"Copyright (C) 2014 - 2017 by Marijn Haverbeke <marijnh@gmail.com> and others\",\"authorEmail\":\"marijnh@gmail.com\",\"authorUrl\":\"http:\\/\\/codemirror.net\\/\",\"version\":\"5.40.0\",\"description\":\"PLG_CODEMIRROR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"codemirror\"}','{\"lineNumbers\":\"1\",\"lineWrapping\":\"1\",\"matchTags\":\"1\",\"matchBrackets\":\"1\",\"marker-gutter\":\"1\",\"autoCloseTags\":\"1\",\"autoCloseBrackets\":\"1\",\"autoFocus\":\"1\",\"theme\":\"default\",\"tabmode\":\"indent\"}','','',0,'0000-00-00 00:00:00',1,0),(411,0,'plg_editors_none','plugin','none','editors',0,1,1,1,'{\"name\":\"plg_editors_none\",\"type\":\"plugin\",\"creationDate\":\"September 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_NONE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"none\"}','','','',0,'0000-00-00 00:00:00',2,0),(412,0,'plg_editors_tinymce','plugin','tinymce','editors',0,1,1,0,'{\"name\":\"plg_editors_tinymce\",\"type\":\"plugin\",\"creationDate\":\"2005-2019\",\"author\":\"Tiny Technologies, Inc\",\"copyright\":\"Tiny Technologies, Inc\",\"authorEmail\":\"N\\/A\",\"authorUrl\":\"https:\\/\\/www.tiny.cloud\",\"version\":\"4.5.11\",\"description\":\"PLG_TINY_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"tinymce\"}','{\"configuration\":{\"toolbars\":{\"2\":{\"toolbar1\":[\"bold\",\"underline\",\"strikethrough\",\"|\",\"undo\",\"redo\",\"|\",\"bullist\",\"numlist\",\"|\",\"pastetext\"]},\"1\":{\"menu\":[\"edit\",\"insert\",\"view\",\"format\",\"table\",\"tools\"],\"toolbar1\":[\"bold\",\"italic\",\"underline\",\"strikethrough\",\"|\",\"alignleft\",\"aligncenter\",\"alignright\",\"alignjustify\",\"|\",\"formatselect\",\"|\",\"bullist\",\"numlist\",\"|\",\"outdent\",\"indent\",\"|\",\"undo\",\"redo\",\"|\",\"link\",\"unlink\",\"anchor\",\"code\",\"|\",\"hr\",\"table\",\"|\",\"subscript\",\"superscript\",\"|\",\"charmap\",\"pastetext\",\"preview\"]},\"0\":{\"menu\":[\"edit\",\"insert\",\"view\",\"format\",\"table\",\"tools\"],\"toolbar1\":[\"bold\",\"italic\",\"underline\",\"strikethrough\",\"|\",\"alignleft\",\"aligncenter\",\"alignright\",\"alignjustify\",\"|\",\"styleselect\",\"|\",\"formatselect\",\"fontselect\",\"fontsizeselect\",\"|\",\"searchreplace\",\"|\",\"bullist\",\"numlist\",\"|\",\"outdent\",\"indent\",\"|\",\"undo\",\"redo\",\"|\",\"link\",\"unlink\",\"anchor\",\"image\",\"|\",\"code\",\"|\",\"forecolor\",\"backcolor\",\"|\",\"fullscreen\",\"|\",\"table\",\"|\",\"subscript\",\"superscript\",\"|\",\"charmap\",\"emoticons\",\"media\",\"hr\",\"ltr\",\"rtl\",\"|\",\"cut\",\"copy\",\"paste\",\"pastetext\",\"|\",\"visualchars\",\"visualblocks\",\"nonbreaking\",\"blockquote\",\"template\",\"|\",\"print\",\"preview\",\"codesample\",\"insertdatetime\",\"removeformat\"]}},\"setoptions\":{\"2\":{\"access\":[\"1\"],\"skin\":\"0\",\"skin_admin\":\"0\",\"mobile\":\"0\",\"drag_drop\":\"1\",\"path\":\"\",\"entity_encoding\":\"raw\",\"lang_mode\":\"1\",\"text_direction\":\"ltr\",\"content_css\":\"1\",\"content_css_custom\":\"\",\"relative_urls\":\"1\",\"newlines\":\"0\",\"use_config_textfilters\":\"0\",\"invalid_elements\":\"script,applet,iframe\",\"valid_elements\":\"\",\"extended_elements\":\"\",\"resizing\":\"1\",\"resize_horizontal\":\"1\",\"element_path\":\"1\",\"wordcount\":\"1\",\"image_advtab\":\"1\",\"advlist\":\"1\",\"contextmenu\":\"1\",\"custom_plugin\":\"\",\"custom_button\":\"\"},\"1\":{\"access\":[\"6\",\"2\",\"3\"],\"skin\":\"0\",\"skin_admin\":\"0\",\"mobile\":\"0\",\"drag_drop\":\"1\",\"path\":\"\",\"entity_encoding\":\"raw\",\"lang_mode\":\"1\",\"text_direction\":\"ltr\",\"content_css\":\"1\",\"content_css_custom\":\"\",\"relative_urls\":\"1\",\"newlines\":\"0\",\"use_config_textfilters\":\"0\",\"invalid_elements\":\"applet\",\"valid_elements\":\"\",\"extended_elements\":\"\",\"resizing\":\"1\",\"resize_horizontal\":\"1\",\"element_path\":\"1\",\"wordcount\":\"1\",\"image_advtab\":\"1\",\"advlist\":\"1\",\"contextmenu\":\"1\",\"custom_plugin\":\"\",\"custom_button\":\"\"},\"0\":{\"access\":[\"7\",\"4\",\"8\"],\"skin\":\"0\",\"skin_admin\":\"0\",\"mobile\":\"0\",\"drag_drop\":\"1\",\"path\":\"\",\"entity_encoding\":\"raw\",\"lang_mode\":\"1\",\"text_direction\":\"ltr\",\"content_css\":\"1\",\"content_css_custom\":\"\",\"relative_urls\":\"1\",\"newlines\":\"0\",\"use_config_textfilters\":\"0\",\"invalid_elements\":\"applet\",\"valid_elements\":\"\",\"extended_elements\":\"\",\"resizing\":\"1\",\"resize_horizontal\":\"1\",\"element_path\":\"1\",\"wordcount\":\"1\",\"image_advtab\":\"1\",\"advlist\":\"1\",\"contextmenu\":\"1\",\"custom_plugin\":\"\",\"custom_button\":\"\"}}},\"sets_amount\":3,\"html_height\":\"550\",\"html_width\":\"750\"}','','',0,'0000-00-00 00:00:00',3,0),(413,0,'plg_editors-xtd_article','plugin','article','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_article\",\"type\":\"plugin\",\"creationDate\":\"October 2009\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_ARTICLE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"article\"}','','','',0,'0000-00-00 00:00:00',1,0),(414,0,'plg_editors-xtd_image','plugin','image','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_image\",\"type\":\"plugin\",\"creationDate\":\"August 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_IMAGE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"image\"}','','','',0,'0000-00-00 00:00:00',2,0),(415,0,'plg_editors-xtd_pagebreak','plugin','pagebreak','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_pagebreak\",\"type\":\"plugin\",\"creationDate\":\"August 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_EDITORSXTD_PAGEBREAK_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"pagebreak\"}','','','',0,'0000-00-00 00:00:00',3,0),(416,0,'plg_editors-xtd_readmore','plugin','readmore','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_readmore\",\"type\":\"plugin\",\"creationDate\":\"March 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_READMORE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"readmore\"}','','','',0,'0000-00-00 00:00:00',4,0),(417,0,'plg_search_categories','plugin','categories','search',0,1,1,0,'{\"name\":\"plg_search_categories\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEARCH_CATEGORIES_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"categories\"}','{\"search_limit\":\"50\",\"search_content\":\"1\",\"search_archived\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(418,0,'plg_search_contacts','plugin','contacts','search',0,1,1,0,'{\"name\":\"plg_search_contacts\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEARCH_CONTACTS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contacts\"}','{\"search_limit\":\"50\",\"search_content\":\"1\",\"search_archived\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(419,0,'plg_search_content','plugin','content','search',0,1,1,0,'{\"name\":\"plg_search_content\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEARCH_CONTENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"content\"}','{\"search_limit\":\"50\",\"search_content\":\"1\",\"search_archived\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(420,0,'plg_search_newsfeeds','plugin','newsfeeds','search',0,1,1,0,'{\"name\":\"plg_search_newsfeeds\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEARCH_NEWSFEEDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"newsfeeds\"}','{\"search_limit\":\"50\",\"search_content\":\"1\",\"search_archived\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(422,0,'plg_system_languagefilter','plugin','languagefilter','system',0,1,1,1,'{\"name\":\"plg_system_languagefilter\",\"type\":\"plugin\",\"creationDate\":\"July 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SYSTEM_LANGUAGEFILTER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"languagefilter\"}','{\"detect_browser\":\"0\",\"automatic_change\":\"1\",\"item_associations\":\"1\",\"remove_default_prefix\":\"1\",\"lang_cookie\":\"0\",\"alternate_meta\":\"0\"}','','',0,'0000-00-00 00:00:00',16,0),(423,0,'plg_system_p3p','plugin','p3p','system',0,0,1,0,'{\"name\":\"plg_system_p3p\",\"type\":\"plugin\",\"creationDate\":\"September 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_P3P_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"p3p\"}','{\"headers\":\"NOI ADM DEV PSAi COM NAV OUR OTRo STP IND DEM\"}','','',0,'0000-00-00 00:00:00',17,0),(424,0,'plg_system_cache','plugin','cache','system',0,0,1,1,'{\"name\":\"plg_system_cache\",\"type\":\"plugin\",\"creationDate\":\"February 2007\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CACHE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"cache\"}','{\"browsercache\":\"0\",\"cachetime\":\"15\"}','','',0,'0000-00-00 00:00:00',25,0),(425,0,'plg_system_debug','plugin','debug','system',0,1,1,0,'{\"name\":\"plg_system_debug\",\"type\":\"plugin\",\"creationDate\":\"December 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_DEBUG_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"debug\"}','{\"profile\":\"1\",\"queries\":\"1\",\"memory\":\"1\",\"language_files\":\"1\",\"language_strings\":\"1\",\"strip-first\":\"1\",\"strip-prefix\":\"\",\"strip-suffix\":\"\"}','','',0,'0000-00-00 00:00:00',19,0),(426,0,'plg_system_log','plugin','log','system',0,1,1,1,'{\"name\":\"plg_system_log\",\"type\":\"plugin\",\"creationDate\":\"April 2007\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_LOG_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"log\"}','','','',0,'0000-00-00 00:00:00',20,0),(427,0,'plg_system_redirect','plugin','redirect','system',0,0,1,1,'{\"name\":\"plg_system_redirect\",\"type\":\"plugin\",\"creationDate\":\"April 2009\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SYSTEM_REDIRECT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"redirect\"}','','','',0,'0000-00-00 00:00:00',21,0),(428,0,'plg_system_remember','plugin','remember','system',0,1,1,1,'{\"name\":\"plg_system_remember\",\"type\":\"plugin\",\"creationDate\":\"April 2007\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_REMEMBER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"remember\"}','','','',0,'0000-00-00 00:00:00',22,0),(429,0,'plg_system_sef','plugin','sef','system',0,1,1,0,'{\"name\":\"plg_system_sef\",\"type\":\"plugin\",\"creationDate\":\"December 2007\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEF_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"sef\"}','','','',0,'0000-00-00 00:00:00',24,0),(430,0,'plg_system_logout','plugin','logout','system',0,1,1,1,'{\"name\":\"plg_system_logout\",\"type\":\"plugin\",\"creationDate\":\"April 2009\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SYSTEM_LOGOUT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"logout\"}','','','',0,'0000-00-00 00:00:00',18,0),(431,0,'plg_user_contactcreator','plugin','contactcreator','user',0,0,1,0,'{\"name\":\"plg_user_contactcreator\",\"type\":\"plugin\",\"creationDate\":\"August 2009\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CONTACTCREATOR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contactcreator\"}','{\"autowebpage\":\"\",\"category\":\"34\",\"autopublish\":\"0\"}','','',0,'0000-00-00 00:00:00',1,0),(432,0,'plg_user_joomla','plugin','joomla','user',0,1,1,0,'{\"name\":\"plg_user_joomla\",\"type\":\"plugin\",\"creationDate\":\"December 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_USER_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','{\"autoregister\":\"1\",\"mail_to_user\":\"1\",\"forceLogout\":\"1\"}','','',0,'0000-00-00 00:00:00',2,0),(433,0,'plg_user_profile','plugin','profile','user',0,0,1,0,'{\"name\":\"plg_user_profile\",\"type\":\"plugin\",\"creationDate\":\"January 2008\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_USER_PROFILE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"profile\"}','{\"register-require_address1\":\"1\",\"register-require_address2\":\"1\",\"register-require_city\":\"1\",\"register-require_region\":\"1\",\"register-require_country\":\"1\",\"register-require_postal_code\":\"1\",\"register-require_phone\":\"1\",\"register-require_website\":\"1\",\"register-require_favoritebook\":\"1\",\"register-require_aboutme\":\"1\",\"register-require_tos\":\"2\",\"register_tos_article\":\"\",\"register-require_dob\":\"1\",\"profile-require_address1\":\"1\",\"profile-require_address2\":\"1\",\"profile-require_city\":\"1\",\"profile-require_region\":\"1\",\"profile-require_country\":\"1\",\"profile-require_postal_code\":\"1\",\"profile-require_phone\":\"1\",\"profile-require_website\":\"1\",\"profile-require_favoritebook\":\"1\",\"profile-require_aboutme\":\"1\",\"profile-require_dob\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(434,0,'plg_extension_joomla','plugin','joomla','extension',0,1,1,1,'{\"name\":\"plg_extension_joomla\",\"type\":\"plugin\",\"creationDate\":\"May 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_EXTENSION_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','','','',0,'0000-00-00 00:00:00',1,0),(435,0,'plg_content_joomla','plugin','joomla','content',0,1,1,0,'{\"name\":\"plg_content_joomla\",\"type\":\"plugin\",\"creationDate\":\"November 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CONTENT_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','','','',0,'0000-00-00 00:00:00',0,0),(436,0,'plg_system_languagecode','plugin','languagecode','system',0,0,1,0,'{\"name\":\"plg_system_languagecode\",\"type\":\"plugin\",\"creationDate\":\"November 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SYSTEM_LANGUAGECODE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"languagecode\"}','','','',0,'0000-00-00 00:00:00',26,0),(437,0,'plg_quickicon_joomlaupdate','plugin','joomlaupdate','quickicon',0,1,1,1,'{\"name\":\"plg_quickicon_joomlaupdate\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_QUICKICON_JOOMLAUPDATE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomlaupdate\"}','','','',0,'0000-00-00 00:00:00',0,0),(438,0,'plg_quickicon_extensionupdate','plugin','extensionupdate','quickicon',0,1,1,1,'{\"name\":\"plg_quickicon_extensionupdate\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_QUICKICON_EXTENSIONUPDATE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"extensionupdate\"}','','','',0,'0000-00-00 00:00:00',0,0),(439,0,'plg_captcha_recaptcha','plugin','recaptcha','captcha',0,0,1,0,'{\"name\":\"plg_captcha_recaptcha\",\"type\":\"plugin\",\"creationDate\":\"December 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.4.0\",\"description\":\"PLG_CAPTCHA_RECAPTCHA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"recaptcha\"}','{\"public_key\":\"\",\"private_key\":\"\",\"theme\":\"clean\"}','','',0,'0000-00-00 00:00:00',0,0),(440,0,'plg_system_highlight','plugin','highlight','system',0,1,1,0,'{\"name\":\"plg_system_highlight\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SYSTEM_HIGHLIGHT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"highlight\"}','','','',0,'0000-00-00 00:00:00',23,0),(441,0,'plg_content_finder','plugin','finder','content',0,1,1,0,'{\"name\":\"plg_content_finder\",\"type\":\"plugin\",\"creationDate\":\"December 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CONTENT_FINDER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"finder\"}','','','',0,'0000-00-00 00:00:00',0,0),(442,0,'plg_finder_categories','plugin','categories','finder',0,0,1,0,'{\"name\":\"plg_finder_categories\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_FINDER_CATEGORIES_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"categories\"}','','','',0,'0000-00-00 00:00:00',1,0),(443,0,'plg_finder_contacts','plugin','contacts','finder',0,0,1,0,'{\"name\":\"plg_finder_contacts\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_FINDER_CONTACTS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contacts\"}','','','',0,'0000-00-00 00:00:00',2,0),(444,0,'plg_finder_content','plugin','content','finder',0,0,1,0,'{\"name\":\"plg_finder_content\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_FINDER_CONTENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"content\"}','','','',0,'0000-00-00 00:00:00',3,0),(445,0,'plg_finder_newsfeeds','plugin','newsfeeds','finder',0,0,1,0,'{\"name\":\"plg_finder_newsfeeds\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_FINDER_NEWSFEEDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"newsfeeds\"}','','','',0,'0000-00-00 00:00:00',4,0),(447,0,'plg_finder_tags','plugin','tags','finder',0,0,1,0,'{\"name\":\"plg_finder_tags\",\"type\":\"plugin\",\"creationDate\":\"February 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_FINDER_TAGS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"tags\"}','','','',0,'0000-00-00 00:00:00',0,0),(448,0,'plg_twofactorauth_totp','plugin','totp','twofactorauth',0,0,1,0,'{\"name\":\"plg_twofactorauth_totp\",\"type\":\"plugin\",\"creationDate\":\"August 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.0\",\"description\":\"PLG_TWOFACTORAUTH_TOTP_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"totp\"}','','','',0,'0000-00-00 00:00:00',0,0),(449,0,'plg_authentication_cookie','plugin','cookie','authentication',0,1,1,0,'{\"name\":\"plg_authentication_cookie\",\"type\":\"plugin\",\"creationDate\":\"July 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_AUTH_COOKIE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"cookie\"}','','','',0,'0000-00-00 00:00:00',0,0),(450,0,'plg_twofactorauth_yubikey','plugin','yubikey','twofactorauth',0,0,1,0,'{\"name\":\"plg_twofactorauth_yubikey\",\"type\":\"plugin\",\"creationDate\":\"September 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.0\",\"description\":\"PLG_TWOFACTORAUTH_YUBIKEY_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"yubikey\"}','','','',0,'0000-00-00 00:00:00',0,0),(451,0,'plg_search_tags','plugin','tags','search',0,1,1,0,'{\"name\":\"plg_search_tags\",\"type\":\"plugin\",\"creationDate\":\"March 2014\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEARCH_TAGS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"tags\"}','{\"search_limit\":\"50\",\"show_tagged_items\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(455,0,'plg_installer_packageinstaller','plugin','packageinstaller','installer',0,1,1,1,'{\"name\":\"plg_installer_packageinstaller\",\"type\":\"plugin\",\"creationDate\":\"May 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.6.0\",\"description\":\"PLG_INSTALLER_PACKAGEINSTALLER_PLUGIN_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"packageinstaller\"}','','','',0,'0000-00-00 00:00:00',1,0),(456,0,'PLG_INSTALLER_FOLDERINSTALLER','plugin','folderinstaller','installer',0,1,1,1,'{\"name\":\"PLG_INSTALLER_FOLDERINSTALLER\",\"type\":\"plugin\",\"creationDate\":\"May 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.6.0\",\"description\":\"PLG_INSTALLER_FOLDERINSTALLER_PLUGIN_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"folderinstaller\"}','','','',0,'0000-00-00 00:00:00',2,0),(457,0,'PLG_INSTALLER_URLINSTALLER','plugin','urlinstaller','installer',0,1,1,1,'{\"name\":\"PLG_INSTALLER_URLINSTALLER\",\"type\":\"plugin\",\"creationDate\":\"May 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.6.0\",\"description\":\"PLG_INSTALLER_URLINSTALLER_PLUGIN_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"urlinstaller\"}','','','',0,'0000-00-00 00:00:00',3,0),(458,0,'plg_quickicon_phpversioncheck','plugin','phpversioncheck','quickicon',0,1,1,1,'{\"name\":\"plg_quickicon_phpversioncheck\",\"type\":\"plugin\",\"creationDate\":\"August 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_QUICKICON_PHPVERSIONCHECK_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"phpversioncheck\"}','','','',0,'0000-00-00 00:00:00',0,0),(459,0,'plg_editors-xtd_menu','plugin','menu','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_menu\",\"type\":\"plugin\",\"creationDate\":\"August 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_EDITORS-XTD_MENU_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"menu\"}','','','',0,'0000-00-00 00:00:00',0,0),(460,0,'plg_editors-xtd_contact','plugin','contact','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_contact\",\"type\":\"plugin\",\"creationDate\":\"October 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_EDITORS-XTD_CONTACT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contact\"}','','','',0,'0000-00-00 00:00:00',0,0),(461,0,'plg_system_fields','plugin','fields','system',0,1,1,0,'{\"name\":\"plg_system_fields\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_SYSTEM_FIELDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"fields\"}','','','',0,'0000-00-00 00:00:00',15,0),(462,0,'plg_fields_calendar','plugin','calendar','fields',0,1,1,0,'{\"name\":\"plg_fields_calendar\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_CALENDAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"calendar\"}','','','',0,'0000-00-00 00:00:00',0,0),(463,0,'plg_fields_checkboxes','plugin','checkboxes','fields',0,1,1,0,'{\"name\":\"plg_fields_checkboxes\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_CHECKBOXES_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"checkboxes\"}','','','',0,'0000-00-00 00:00:00',0,0),(464,0,'plg_fields_color','plugin','color','fields',0,1,1,0,'{\"name\":\"plg_fields_color\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_COLOR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"color\"}','','','',0,'0000-00-00 00:00:00',0,0),(465,0,'plg_fields_editor','plugin','editor','fields',0,1,1,0,'{\"name\":\"plg_fields_editor\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_EDITOR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"editor\"}','','','',0,'0000-00-00 00:00:00',0,0),(466,0,'plg_fields_imagelist','plugin','imagelist','fields',0,1,1,0,'{\"name\":\"plg_fields_imagelist\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_IMAGELIST_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"imagelist\"}','','','',0,'0000-00-00 00:00:00',0,0),(467,0,'plg_fields_integer','plugin','integer','fields',0,1,1,0,'{\"name\":\"plg_fields_integer\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_INTEGER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"integer\"}','{\"multiple\":\"0\",\"first\":\"1\",\"last\":\"100\",\"step\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(468,0,'plg_fields_list','plugin','list','fields',0,1,1,0,'{\"name\":\"plg_fields_list\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_LIST_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"list\"}','','','',0,'0000-00-00 00:00:00',0,0),(469,0,'plg_fields_media','plugin','media','fields',0,1,1,0,'{\"name\":\"plg_fields_media\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_MEDIA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"media\"}','','','',0,'0000-00-00 00:00:00',0,0),(470,0,'plg_fields_radio','plugin','radio','fields',0,1,1,0,'{\"name\":\"plg_fields_radio\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_RADIO_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"radio\"}','','','',0,'0000-00-00 00:00:00',0,0),(471,0,'plg_fields_sql','plugin','sql','fields',0,1,1,0,'{\"name\":\"plg_fields_sql\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_SQL_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"sql\"}','','','',0,'0000-00-00 00:00:00',0,0),(472,0,'plg_fields_text','plugin','text','fields',0,1,1,0,'{\"name\":\"plg_fields_text\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_TEXT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"text\"}','','','',0,'0000-00-00 00:00:00',0,0),(473,0,'plg_fields_textarea','plugin','textarea','fields',0,1,1,0,'{\"name\":\"plg_fields_textarea\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_TEXTAREA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"textarea\"}','','','',0,'0000-00-00 00:00:00',0,0),(474,0,'plg_fields_url','plugin','url','fields',0,1,1,0,'{\"name\":\"plg_fields_url\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_URL_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"url\"}','','','',0,'0000-00-00 00:00:00',0,0),(475,0,'plg_fields_user','plugin','user','fields',0,1,1,0,'{\"name\":\"plg_fields_user\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_USER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"user\"}','','','',0,'0000-00-00 00:00:00',0,0),(476,0,'plg_fields_usergrouplist','plugin','usergrouplist','fields',0,1,1,0,'{\"name\":\"plg_fields_usergrouplist\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_USERGROUPLIST_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"usergrouplist\"}','','','',0,'0000-00-00 00:00:00',0,0),(477,0,'plg_content_fields','plugin','fields','content',0,1,1,0,'{\"name\":\"plg_content_fields\",\"type\":\"plugin\",\"creationDate\":\"February 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_CONTENT_FIELDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"fields\"}','','','',0,'0000-00-00 00:00:00',0,0),(478,0,'plg_editors-xtd_fields','plugin','fields','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_fields\",\"type\":\"plugin\",\"creationDate\":\"February 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_EDITORS-XTD_FIELDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"fields\"}','','','',0,'0000-00-00 00:00:00',0,0),(479,0,'plg_sampledata_blog','plugin','blog','sampledata',0,0,1,0,'{\"name\":\"plg_sampledata_blog\",\"type\":\"plugin\",\"creationDate\":\"July 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.8.0\",\"description\":\"PLG_SAMPLEDATA_BLOG_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"blog\"}','','','',0,'0000-00-00 00:00:00',0,0),(480,0,'plg_system_sessiongc','plugin','sessiongc','system',0,1,1,0,'{\"name\":\"plg_system_sessiongc\",\"type\":\"plugin\",\"creationDate\":\"February 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.8.6\",\"description\":\"PLG_SYSTEM_SESSIONGC_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"sessiongc\"}','','','',0,'0000-00-00 00:00:00',0,0),(481,0,'plg_fields_repeatable','plugin','repeatable','fields',0,1,1,0,'{\"name\":\"plg_fields_repeatable\",\"type\":\"plugin\",\"creationDate\":\"April 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_FIELDS_REPEATABLE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"repeatable\"}','','','',0,'0000-00-00 00:00:00',0,0),(482,0,'plg_content_confirmconsent','plugin','confirmconsent','content',0,0,1,0,'{\"name\":\"plg_content_confirmconsent\",\"type\":\"plugin\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_CONTENT_CONFIRMCONSENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"confirmconsent\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(483,0,'PLG_SYSTEM_ACTIONLOGS','plugin','actionlogs','system',0,0,1,0,'{\"name\":\"PLG_SYSTEM_ACTIONLOGS\",\"type\":\"plugin\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_SYSTEM_ACTIONLOGS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"actionlogs\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(484,0,'PLG_ACTIONLOG_JOOMLA','plugin','joomla','actionlog',0,1,1,0,'{\"name\":\"PLG_ACTIONLOG_JOOMLA\",\"type\":\"plugin\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_ACTIONLOG_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(485,0,'plg_system_privacyconsent','plugin','privacyconsent','system',0,0,1,0,'{\"name\":\"plg_system_privacyconsent\",\"type\":\"plugin\",\"creationDate\":\"April 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_SYSTEM_PRIVACYCONSENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"privacyconsent\"}','{\"privacy_note\":\"\",\"privacy_article\":\"\",\"messageOnRedirect\":\"\",\"enabled\":0,\"cachetimeout\":30,\"consentexpiration\":360,\"remind\":30,\"lastrun\":0}','','',0,'0000-00-00 00:00:00',0,0),(486,0,'plg_system_logrotation','plugin','logrotation','system',0,1,1,0,'{\"name\":\"plg_system_logrotation\",\"type\":\"plugin\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_SYSTEM_LOGROTATION_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"logrotation\"}','{\"lastrun\":1571574433}','','',0,'0000-00-00 00:00:00',0,0),(487,0,'plg_privacy_user','plugin','user','privacy',0,1,1,0,'{\"name\":\"plg_privacy_user\",\"type\":\"plugin\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_USER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"user\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(488,0,'plg_quickicon_privacycheck','plugin','privacycheck','quickicon',0,1,1,0,'{\"name\":\"plg_quickicon_privacycheck\",\"type\":\"plugin\",\"creationDate\":\"June 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_QUICKICON_PRIVACYCHECK_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"privacycheck\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(489,0,'plg_user_terms','plugin','terms','user',0,0,1,0,'{\"name\":\"plg_user_terms\",\"type\":\"plugin\",\"creationDate\":\"June 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_USER_TERMS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"terms\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(490,0,'plg_privacy_contact','plugin','contact','privacy',0,1,1,0,'{\"name\":\"plg_privacy_contact\",\"type\":\"plugin\",\"creationDate\":\"July 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_CONTACT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contact\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(491,0,'plg_privacy_content','plugin','content','privacy',0,1,1,0,'{\"name\":\"plg_privacy_content\",\"type\":\"plugin\",\"creationDate\":\"July 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_CONTENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"content\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(492,0,'plg_privacy_message','plugin','message','privacy',0,1,1,0,'{\"name\":\"plg_privacy_message\",\"type\":\"plugin\",\"creationDate\":\"July 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_MESSAGE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"message\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(493,0,'plg_privacy_actionlogs','plugin','actionlogs','privacy',0,1,1,0,'{\"name\":\"plg_privacy_actionlogs\",\"type\":\"plugin\",\"creationDate\":\"July 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_ACTIONLOGS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"actionlogs\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(494,0,'plg_captcha_recaptcha_invisible','plugin','recaptcha_invisible','captcha',0,0,1,0,'{\"name\":\"plg_captcha_recaptcha_invisible\",\"type\":\"plugin\",\"creationDate\":\"November 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.8\",\"description\":\"PLG_CAPTCHA_RECAPTCHA_INVISIBLE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"recaptcha_invisible\"}','{\"public_key\":\"\",\"private_key\":\"\",\"theme\":\"clean\"}','','',0,'0000-00-00 00:00:00',0,0),(495,0,'plg_privacy_consents','plugin','consents','privacy',0,1,1,0,'{\"name\":\"plg_privacy_consents\",\"type\":\"plugin\",\"creationDate\":\"July 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_CONSENTS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"consents\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(503,0,'beez3','template','beez3','',0,1,1,0,'{\"name\":\"beez3\",\"type\":\"template\",\"creationDate\":\"25 November 2009\",\"author\":\"Angie Radtke\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.\",\"authorEmail\":\"a.radtke@derauftritt.de\",\"authorUrl\":\"http:\\/\\/www.der-auftritt.de\",\"version\":\"3.1.0\",\"description\":\"TPL_BEEZ3_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"templateDetails\"}','{\"wrapperSmall\":\"53\",\"wrapperLarge\":\"72\",\"sitetitle\":\"\",\"sitedescription\":\"\",\"navposition\":\"center\",\"templatecolor\":\"nature\"}','','',0,'0000-00-00 00:00:00',0,0),(504,0,'hathor','template','hathor','',1,1,1,0,'{\"name\":\"hathor\",\"type\":\"template\",\"creationDate\":\"May 2010\",\"author\":\"Andrea Tarr\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"\",\"version\":\"3.0.0\",\"description\":\"TPL_HATHOR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"templateDetails\"}','{\"showSiteName\":\"0\",\"colourChoice\":\"0\",\"boldText\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(506,0,'protostar','template','protostar','',0,1,1,0,'{\"name\":\"protostar\",\"type\":\"template\",\"creationDate\":\"4\\/30\\/2012\",\"author\":\"Kyle Ledbetter\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"\",\"version\":\"1.0\",\"description\":\"TPL_PROTOSTAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"templateDetails\"}','{\"templateColor\":\"\",\"logoFile\":\"\",\"googleFont\":\"1\",\"googleFontName\":\"Open+Sans\",\"fluidContainer\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(507,0,'isis','template','isis','',1,1,1,0,'{\"name\":\"isis\",\"type\":\"template\",\"creationDate\":\"3\\/30\\/2012\",\"author\":\"Kyle Ledbetter\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"\",\"version\":\"1.0\",\"description\":\"TPL_ISIS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"templateDetails\"}','{\"templateColor\":\"\",\"logoFile\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(600,802,'English (en-GB)','language','en-GB','',0,1,1,1,'{\"name\":\"English (en-GB)\",\"type\":\"language\",\"creationDate\":\"August 2019\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.11\",\"description\":\"en-GB site language\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(601,802,'English (en-GB)','language','en-GB','',1,1,1,1,'{\"name\":\"English (en-GB)\",\"type\":\"language\",\"creationDate\":\"August 2019\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.11\",\"description\":\"en-GB administrator language\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(700,0,'files_joomla','file','joomla','',0,1,1,1,'{\"name\":\"files_joomla\",\"type\":\"file\",\"creationDate\":\"August 2019\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 - 2019 Open Source Matters. All rights reserved\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.11\",\"description\":\"FILES_JOOMLA_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(802,0,'English (en-GB) Language Pack','package','pkg_en-GB','',0,1,1,1,'{\"name\":\"English (en-GB) Language Pack\",\"type\":\"package\",\"creationDate\":\"August 2019\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.11.1\",\"description\":\"en-GB language pack\",\"group\":\"\",\"filename\":\"pkg_en-GB\"}','','','',0,'0000-00-00 00:00:00',0,0),(10180,0,'FavGlyph','module','mod_favglyph','',0,1,0,0,'{\"name\":\"FavGlyph\",\"type\":\"module\",\"creationDate\":\"2014\",\"author\":\"FavThemes\",\"copyright\":\"Copyright (C) FavThemes. All rights reserved.\",\"authorEmail\":\"\",\"authorUrl\":\"http:\\/\\/www.favthemes.com\",\"version\":\"2.2\",\"description\":\"\\n  \\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favglyph\\\" target=\\\"_blank\\\"><strong>FavGlyph<\\/strong><\\/a> is a free responsive Joomla! module that uses any icon from Font Awesome and assigns it a short description and a title.\\n    <br\\/>\\n    For more info, please see the <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favglyph\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">demo and documentation<\\/a> for the FavGlyph module.\\n    <br\\/><br\\/>\\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favglyph\\\" target=\\\"_blank\\\">\\n      <img src=\\\"..\\/modules\\/mod_favglyph\\/theme\\/img\\/favglyph.jpg\\\" alt=\\\"FavGlyph Responsive Module\\\">\\n    <\\/a>\\n\\n    <br\\/><br\\/>\\n    Copyright &#169; 2012-2018 <a href=\\\"http:\\/\\/www.favthemes.com\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">FavThemes<\\/a>.\\n    <br\\/><br\\/>\\n\\n    <link rel=\\\"stylesheet\\\" href=\\\"..\\/modules\\/mod_favglyph\\/theme\\/css\\/admin.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/use.fontawesome.com\\/releases\\/v5.1.0\\/css\\/all.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/fonts.googleapis.com\\/css?family=Open+Sans:400\\\" type=\\\"text\\/css\\\" \\/>\\n    <script src=\\\"..\\/modules\\/mod_favglyph\\/theme\\/js\\/jscolor\\/jscolor.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n    <script src=\\\"..\\/modules\\/mod_favglyph\\/theme\\/js\\/admin\\/admin.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n\\n  \\n  \",\"group\":\"\",\"filename\":\"mod_favglyph\"}','{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect1\",\"icon_width\":\"77%\",\"icon_border_radius\":\"50%\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_size\":\"21px\",\"title_margin\":\"28px 0px 14px\",\"description_description_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_layout1\":\"center\",\"icon_name1\":\"fa-globe\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"CCCCCC\",\"icon_link1\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target1\":\"blank\",\"icon_font_size1\":\"6em\",\"title_text1\":\"Title Text\",\"title_color1\":\"444444\",\"description_layout1\":\"0\",\"description_text1\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_layout2\":\"center\",\"icon_name2\":\"fa-html5\",\"icon_color2\":\"ff5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"CCCCCC\",\"icon_link2\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target2\":\"blank\",\"icon_font_size2\":\"6em\",\"title_text2\":\"Title Text\",\"title_color2\":\"444444\",\"description_layout2\":\"0\",\"description_text2\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_layout3\":\"center\",\"icon_name3\":\"fa-recycle\",\"icon_color3\":\"8bc34a\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"CCCCCC\",\"icon_link3\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target3\":\"blank\",\"icon_font_size3\":\"6em\",\"title_text3\":\"Title Text\",\"title_color3\":\"444444\",\"description_layout3\":\"0\",\"description_text3\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_layout4\":\"center\",\"icon_name4\":\"fa-lightbulb-o\",\"icon_color4\":\"ffc107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"CCCCCC\",\"icon_link4\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target4\":\"blank\",\"icon_font_size4\":\"6em\",\"title_text4\":\"Title Text\",\"title_color4\":\"444444\",\"description_layout4\":\"0\",\"description_text4\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_layout5\":\"center\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673ab7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"description_layout5\":\"0\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_layout6\":\"center\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00bcd4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"description_layout6\":\"0\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\"}','','',0,'0000-00-00 00:00:00',0,0),(10182,0,'FavPromote','module','mod_favpromote','',0,1,0,0,'{\"name\":\"FavPromote\",\"type\":\"module\",\"creationDate\":\"2013\",\"author\":\"FavThemes\",\"copyright\":\"Copyright (C) FavThemes. All rights reserved.\",\"authorEmail\":\"\",\"authorUrl\":\"http:\\/\\/www.favthemes.com\",\"version\":\"2.2\",\"description\":\"\\n  \\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\"><strong>FavPromote<\\/strong><\\/a> is a free responsive Joomla! module that lets you upload any image and assign it a short description and a title with a Font Awesome icon.\\n    <br\\/>\\n    For more info, please see the <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">demo and documentation<\\/a> for the FavPromote module.\\n    <br\\/><br\\/>\\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\">\\n      <img src=\\\"..\\/modules\\/mod_favpromote\\/theme\\/img\\/favpromote.jpg\\\" alt=\\\"FavPromote Responsive Module\\\">\\n    <\\/a>\\n\\n    <br\\/><br\\/>\\n    Copyright &#169; 2012-2018 <a href=\\\"http:\\/\\/www.favthemes.com\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">FavThemes<\\/a>.\\n    <br\\/><br\\/>\\n\\n    <link rel=\\\"stylesheet\\\" href=\\\"..\\/modules\\/mod_favpromote\\/theme\\/css\\/admin.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/use.fontawesome.com\\/releases\\/v5.1.0\\/css\\/all.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/fonts.googleapis.com\\/css?family=Open+Sans:400\\\" type=\\\"text\\/css\\\" \\/>\\n    <script src=\\\"..\\/modules\\/mod_favpromote\\/theme\\/js\\/jscolor\\/jscolor.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n    <script src=\\\"..\\/modules\\/mod_favpromote\\/theme\\/js\\/admin\\/admin.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n\\n  \\n\\t\",\"group\":\"\",\"filename\":\"mod_favpromote\"}','{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect1\",\"title_google_font\":\"Open Sans\",\"title_padding\":\"10px 20px\",\"title_font_size\":\"18px\",\"title_line_height\":\"1.4em\",\"title_text_align\":\"left\",\"title_icon_font_size\":\"18px\",\"title_icon_vertical_align\":\"baseline\",\"description_text_font_size\":\"14px\",\"description_text_line_height\":\"1.5em\",\"description_text_align\":\"justify\",\"show_column1\":\"1\",\"column_border_color1\":\"DDDDDD\",\"column_border_radius1\":\"4px\",\"show_image_link1\":\"1\",\"image_link1\":\"http:\\/\\/www.favthemes.com\",\"image_target1\":\"blank\",\"image_alt1\":\"\",\"title_text1\":\"Title Text\",\"title_color1\":\"FFFFFF\",\"title_bg_color1\":\"D63E52\",\"show_title_link1\":\"1\",\"title_link1\":\"http:\\/\\/www.favthemes.com\",\"title_target1\":\"blank\",\"title_icon1\":\"fa-coffee\",\"description_text1\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color1\":\"444444\",\"show_column2\":\"1\",\"column_border_color2\":\"DDDDDD\",\"column_border_radius2\":\"4px\",\"show_image_link2\":\"1\",\"image_link2\":\"http:\\/\\/www.favthemes.com\",\"image_target2\":\"blank\",\"image_alt2\":\"\",\"title_text2\":\"Title Text\",\"title_color2\":\"FFFFFF\",\"title_bg_color2\":\"D63E52\",\"show_title_link2\":\"1\",\"title_link2\":\"http:\\/\\/www.favthemes.com\",\"title_target2\":\"blank\",\"title_icon2\":\"fa-puzzle-piece\",\"description_text2\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color2\":\"444444\",\"show_column3\":\"1\",\"column_border_color3\":\"DDDDDD\",\"column_border_radius3\":\"4px\",\"show_image_link3\":\"1\",\"image_link3\":\"http:\\/\\/www.favthemes.com\",\"image_target3\":\"blank\",\"image_alt3\":\"\",\"title_text3\":\"Title Text\",\"title_color3\":\"FFFFFF\",\"title_bg_color3\":\"D63E52\",\"show_title_link3\":\"1\",\"title_link3\":\"http:\\/\\/www.favthemes.com\",\"title_target3\":\"blank\",\"title_icon3\":\"fa-bookmark\",\"description_text3\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color3\":\"444444\",\"show_column4\":\"1\",\"column_border_color4\":\"DDDDDD\",\"column_border_radius4\":\"4px\",\"show_image_link4\":\"1\",\"image_link4\":\"http:\\/\\/www.favthemes.com\",\"image_target4\":\"blank\",\"image_alt4\":\"\",\"title_text4\":\"Title Text\",\"title_color4\":\"FFFFFF\",\"title_bg_color4\":\"D63E52\",\"show_title_link4\":\"1\",\"title_link4\":\"http:\\/\\/www.favthemes.com\",\"title_target4\":\"blank\",\"title_icon4\":\"fa-magic\",\"description_text4\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color4\":\"444444\",\"show_column5\":\"0\",\"column_border_color5\":\"DDDDDD\",\"column_border_radius5\":\"4px\",\"show_image_link5\":\"1\",\"image_link5\":\"http:\\/\\/www.favthemes.com\",\"image_target5\":\"blank\",\"image_alt5\":\"\",\"title_text5\":\"Title Text\",\"title_color5\":\"FFFFFF\",\"title_bg_color5\":\"D63E52\",\"show_title_link5\":\"1\",\"title_link5\":\"http:\\/\\/www.favthemes.com\",\"title_target5\":\"blank\",\"title_icon5\":\"fa-quote-right\",\"description_text5\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color5\":\"444444\",\"show_column6\":\"0\",\"column_border_color6\":\"DDDDDD\",\"column_border_radius6\":\"4px\",\"show_image_link6\":\"1\",\"image_link6\":\"http:\\/\\/www.favthemes.com\",\"image_target6\":\"blank\",\"image_alt6\":\"\",\"title_text6\":\"Title Text\",\"title_color6\":\"FFFFFF\",\"title_bg_color6\":\"D63E52\",\"show_title_link6\":\"1\",\"title_link6\":\"http:\\/\\/www.favthemes.com\",\"title_target6\":\"blank\",\"title_icon6\":\"fa-recycle\",\"description_text6\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color6\":\"444444\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\"}','','',0,'0000-00-00 00:00:00',0,0),(10184,0,'FavSocial','module','mod_favsocial','',0,1,0,0,'{\"name\":\"FavSocial\",\"type\":\"module\",\"creationDate\":\"2013\",\"author\":\"FavThemes\",\"copyright\":\"Copyright (C) FavThemes. All rights reserved.\",\"authorEmail\":\"\",\"authorUrl\":\"http:\\/\\/www.favthemes.com\",\"version\":\"1.6\",\"description\":\"\\n  \\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favsocial\\\" target=\\\"_blank\\\"><strong>FavSocial<\\/strong><\\/a> is a free responsive Joomla! module that lets you choose up to 10 social media icons from Font Awesome and link each one to your social media accounts.\\n    <br\\/>\\n    For more info, please see the <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favsocial\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">demo and documentation<\\/a> for the FavSocial module.\\n    <br\\/><br\\/>\\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favsocial\\\" target=\\\"_blank\\\">\\n      <img src=\\\"..\\/modules\\/mod_favsocial\\/theme\\/img\\/favsocial.jpg\\\" alt=\\\"FavSocial Responsive Module\\\">\\n    <\\/a>\\n\\n    <br\\/><br\\/>\\n    Copyright &#169; 2012-2016 <a href=\\\"http:\\/\\/www.favthemes.com\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">FavThemes<\\/a>.\\n    <br\\/><br\\/>\\n\\n    <link rel=\\\"stylesheet\\\" href=\\\"..\\/modules\\/mod_favsocial\\/theme\\/css\\/admin.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/maxcdn.bootstrapcdn.com\\/font-awesome\\/4.7.0\\/css\\/font-awesome.min.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/fonts.googleapis.com\\/css?family=Open+Sans:400\\\" type=\\\"text\\/css\\\" \\/>\\n    <script src=\\\"..\\/modules\\/mod_favsocial\\/theme\\/js\\/jscolor\\/jscolor.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n\\n  \\n  \",\"group\":\"\",\"filename\":\"mod_favsocial\"}','{\"module_align\":\"favsocial-left\",\"show_icon1\":\"1\",\"icon_name1\":\"fa-twitter\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"00ACED\",\"icon_font_size1\":\"21px\",\"icon_link1\":\"https:\\/\\/twitter.com\\/\",\"icon_link_target1\":\"blank\",\"icon_padding1\":\"0.5em\",\"icon_border_radius1\":\"50%\",\"show_icon2\":\"1\",\"icon_name2\":\"fa-facebook\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"3B5998\",\"icon_font_size2\":\"21px\",\"icon_link2\":\"https:\\/\\/www.facebook.com\\/\",\"icon_link_target2\":\"blank\",\"icon_padding2\":\"0.5em\",\"icon_border_radius2\":\"50%\",\"show_icon3\":\"1\",\"icon_name3\":\"fa-dribbble\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"EA4C89\",\"icon_font_size3\":\"21px\",\"icon_link3\":\"https:\\/\\/dribbble.com\\/\",\"icon_link_target3\":\"blank\",\"icon_padding3\":\"0.5em\",\"icon_border_radius3\":\"50%\",\"show_icon4\":\"0\",\"icon_name4\":\"\",\"icon_color4\":\"\",\"icon_bg_color4\":\"\",\"icon_font_size4\":\"\",\"icon_link4\":\"\",\"icon_link_target4\":\"blank\",\"icon_padding4\":\"\",\"icon_border_radius4\":\"\",\"show_icon5\":\"0\",\"icon_name5\":\"\",\"icon_color5\":\"\",\"icon_bg_color5\":\"\",\"icon_font_size5\":\"\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_padding5\":\"\",\"icon_border_radius5\":\"\",\"show_icon6\":\"0\",\"icon_name6\":\"\",\"icon_color6\":\"\",\"icon_bg_color6\":\"\",\"icon_font_size6\":\"\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_padding6\":\"\",\"icon_border_radius6\":\"\",\"show_icon7\":\"0\",\"icon_name7\":\"\",\"icon_color7\":\"\",\"icon_bg_color7\":\"\",\"icon_font_size7\":\"\",\"icon_link7\":\"\",\"icon_link_target7\":\"blank\",\"icon_padding7\":\"\",\"icon_border_radius7\":\"\",\"show_icon8\":\"0\",\"icon_name8\":\"\",\"icon_color8\":\"\",\"icon_bg_color8\":\"\",\"icon_font_size8\":\"\",\"icon_link8\":\"\",\"icon_link_target8\":\"blank\",\"icon_padding8\":\"\",\"icon_border_radius8\":\"\",\"show_icon9\":\"0\",\"icon_name9\":\"\",\"icon_color9\":\"\",\"icon_bg_color9\":\"\",\"icon_font_size9\":\"\",\"icon_link9\":\"\",\"icon_link_target9\":\"blank\",\"icon_padding9\":\"\",\"icon_border_radius9\":\"\",\"show_icon10\":\"0\",\"icon_name10\":\"\",\"icon_color10\":\"\",\"icon_bg_color10\":\"\",\"icon_font_size10\":\"\",\"icon_link10\":\"\",\"icon_link_target10\":\"blank\",\"icon_padding10\":\"\",\"icon_border_radius10\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\"}','','',0,'0000-00-00 00:00:00',0,0),(10186,0,'Favourite','template','favourite','',0,1,1,0,'{\"name\":\"Favourite\",\"type\":\"template\",\"creationDate\":\"September 2012\",\"author\":\"FavThemes\",\"copyright\":\"Copyright (C) 2012-2016 FavThemes. All rights reserved.\",\"authorEmail\":\"hello@favthemes.com\",\"authorUrl\":\"http:\\/\\/www.favthemes.com\",\"version\":\"3.4\",\"description\":\"\\n\\n  <p style=\\\"max-width: 900px; margin-bottom: 21px; text-align: justify;\\\"><strong>Favourite<\\/strong> is a free responsive Joomla! template with over 200 settings for an easy and fast customization to suit your clients\\u2019 needs as quickly as possible and is built using Bootstrap, HTML5, CSS3, Google Fonts and Font Awesome. Design amazing responsive websites with this smart template that adapts and resizes itself to desktop and mobile devices, making your website looking great for everyone!\\n\\n  <\\/br><\\/br>\\n\\n  This free template can be used on unlimited personal or commercial websites, just don\'t resell it! Once you downloaded the template, you have free and unlimited access to all future Joomla! versions of the product.<\\/p>\\n\\n<a href=\\\"http:\\/\\/demo.favthemes.com\\/favourite\\/\\\" class=\\\"btn btn-success\\\" target=\\\"_blank\\\">Demo<\\/a>\\n\\n<a href=\\\"http:\\/\\/www.favthemes.com\\/docs.html\\\" class=\\\"btn btn-info\\\" target=\\\"_blank\\\">Documentation<\\/a>\\n\\n<\\/br><\\/br>\\n\\n<a href=\\\"http:\\/\\/demo.favthemes.com\\/favourite\\/\\\" target=\\\"_blank\\\">\\n  <img class=\\\"fav-admin-img\\\" style=\\\"padding: 4px; border: 1px solid #DDD; border-radius: 4px;\\\" src=\\\"..\\/templates\\/favourite\\/template_preview.png\\\" alt=\\\"Favourite Responsive Template\\\">\\n<\\/a>\\n\\n<\\/br><\\/br>\\nCopyright &#169; 2012-2016 <a href=\\\"http:\\/\\/www.favthemes.com\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">FavThemes<\\/a>.\\n<\\/br><\\/br>\\n\\n<script src=\\\"..\\/templates\\/favourite\\/admin\\/jscolor\\/jscolor.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n<link rel=\\\"stylesheet\\\" href=\\\"..\\/templates\\/favourite\\/admin\\/admin.css\\\" type=\\\"text\\/css\\\" \\/>\\n\\n\",\"group\":\"\",\"filename\":\"templateDetails\"}','{\"template_styles\":\"style1\",\"max_width\":\"\",\"show_copyright\":\"1\",\"copyright_text\":\"FavThemes\",\"copyright_text_link\":\"www.favthemes.com\",\"header_style\":\"fav-dark\",\"header_position\":\"fav-fixed\",\"header_fixed_height\":\"\",\"header_fixed_height_tablet\":\"\",\"nav_style\":\"navstyle-1\",\"nav_style_icon\":\"navstyle-icon-horizontal\",\"nav_style_color\":\"\",\"nav_google_font\":\"\",\"nav_google_font_weight\":\"400\",\"nav_google_font_style\":\"normal\",\"nav_text_transform\":\"uppercase\",\"nav_font_size\":\"\",\"nav_link_padding\":\"\",\"nav_icons_color\":\"\",\"nav_icons_font_size\":\"\",\"product_img_padding\":\"0\",\"link_color\":\"\",\"link_color_hover\":\"\",\"titles_google_font\":\"\",\"titles_google_font_weight\":\"400\",\"titles_google_font_style\":\"normal\",\"titles_text_align\":\"left\",\"titles_text_transform\":\"none\",\"article_title_color\":\"\",\"article_title_link_color\":\"\",\"article_title_link_hover_color\":\"\",\"article_title_font_size\":\"\",\"article_title_line_height\":\"\",\"module_title_color\":\"\",\"module_title_font_size\":\"\",\"module_title_line_height\":\"\",\"module_title_icon_color\":\"\",\"module_title_icon_font_size\":\"\",\"module_title_icon_padding\":\"\",\"module_title_icon_border_color\":\"\",\"module_title_icon_border_radius\":\"\",\"variation_style_color\":\"\",\"variation_style_icon_font_size\":\"\",\"variation_style_icon_padding\":\"\",\"button_color\":\"\",\"button_bg_color\":\"\",\"button_hover_color\":\"\",\"button_bg_hover_color\":\"\",\"button_google_font\":\"\",\"button_google_font_weight\":\"400\",\"button_google_font_style\":\"normal\",\"button_text_transform\":\"\",\"vertical_menus_style_color\":\"\",\"vertical_menus_text_transform\":\"none\",\"horizontal_menus_style_color\":\"\",\"horizontal_menus_google_font\":\"\",\"horizontal_menus_google_font_weight\":\"400\",\"horizontal_menus_google_font_style\":\"normal\",\"horizontal_menus_text_transform\":\"none\",\"list_style_color\":\"\",\"show_back_to_top\":\"1\",\"error_page_article_id\":\"3\",\"offline_page_style\":\"offline-dark\",\"offline_page_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"responsive_k2_image\":\"100%!important\",\"body_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"body_bg_image_overlay\":\"fav-transparent\",\"body_bg_color\":\"\",\"body_color\":\"\",\"body_title_color\":\"\",\"body_link_color\":\"\",\"body_link_hover_color\":\"\",\"show_advert_button\":\"1\",\"advert_bg_color\":\"\",\"advert_color\":\"\",\"advert_title_color\":\"\",\"advert_link_color\":\"\",\"advert_link_hover_color\":\"\",\"topbar_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"topbar_bg_image_overlay\":\"fav-transparent\",\"topbar_bg_color\":\"\",\"topbar_color\":\"\",\"topbar_title_color\":\"\",\"topbar_link_color\":\"\",\"topbar_link_hover_color\":\"\",\"slide_width\":\"\",\"slide_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"slide_bg_image_overlay\":\"fav-transparent\",\"slide_bg_color\":\"\",\"slide_color\":\"\",\"slide_title_color\":\"\",\"slide_link_color\":\"\",\"slide_link_hover_color\":\"\",\"intro_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"intro_bg_image_overlay\":\"fav-overlay\",\"intro_bg_color\":\"\",\"intro_color\":\"\",\"intro_title_color\":\"\",\"intro_link_color\":\"\",\"intro_link_hover_color\":\"\",\"lead_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"lead_bg_image_overlay\":\"fav-transparent\",\"lead_bg_color\":\"\",\"lead_color\":\"\",\"lead_title_color\":\"\",\"lead_link_color\":\"\",\"lead_link_hover_color\":\"\",\"promo_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"promo_bg_image_overlay\":\"fav-transparent\",\"promo_bg_color\":\"\",\"promo_color\":\"\",\"promo_title_color\":\"\",\"promo_link_color\":\"\",\"promo_link_hover_color\":\"\",\"prime_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"prime_bg_image_overlay\":\"fav-overlay\",\"prime_bg_color\":\"\",\"prime_color\":\"\",\"prime_title_color\":\"\",\"prime_link_color\":\"\",\"prime_link_hover_color\":\"\",\"showcase_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"showcase_bg_image_overlay\":\"fav-transparent\",\"showcase_bg_color\":\"\",\"showcase_color\":\"\",\"showcase_title_color\":\"\",\"showcase_link_color\":\"\",\"showcase_link_hover_color\":\"\",\"feature_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"feature_bg_image_overlay\":\"fav-transparent\",\"feature_bg_color\":\"\",\"feature_color\":\"\",\"feature_title_color\":\"\",\"feature_link_color\":\"\",\"feature_link_hover_color\":\"\",\"focus_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"focus_bg_image_overlay\":\"fav-transparent\",\"focus_bg_color\":\"\",\"focus_color\":\"\",\"focus_title_color\":\"\",\"focus_link_color\":\"\",\"focus_link_hover_color\":\"\",\"portfolio_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"portfolio_bg_image_overlay\":\"fav-transparent\",\"portfolio_bg_color\":\"\",\"portfolio_color\":\"\",\"portfolio_title_color\":\"\",\"portfolio_link_color\":\"\",\"portfolio_link_hover_color\":\"\",\"screen_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"screen_bg_image_overlay\":\"fav-overlay\",\"screen_bg_color\":\"\",\"screen_color\":\"\",\"screen_title_color\":\"\",\"screen_link_color\":\"\",\"screen_link_hover_color\":\"\",\"top_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"top_bg_image_overlay\":\"fav-transparent\",\"top_bg_color\":\"\",\"top_color\":\"\",\"top_title_color\":\"\",\"top_link_color\":\"\",\"top_link_hover_color\":\"\",\"maintop_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"maintop_bg_image_overlay\":\"fav-transparent\",\"maintop_bg_color\":\"\",\"maintop_color\":\"\",\"maintop_title_color\":\"\",\"maintop_link_color\":\"\",\"maintop_link_hover_color\":\"\",\"mainbottom_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"mainbottom_bg_image_overlay\":\"fav-transparent\",\"mainbottom_bg_color\":\"\",\"mainbottom_color\":\"\",\"mainbottom_title_color\":\"\",\"mainbottom_link_color\":\"\",\"mainbottom_link_hover_color\":\"\",\"bottom_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"bottom_bg_image_overlay\":\"fav-transparent\",\"bottom_bg_color\":\"\",\"bottom_color\":\"\",\"bottom_title_color\":\"\",\"bottom_link_color\":\"\",\"bottom_link_hover_color\":\"\",\"note_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"note_bg_image_overlay\":\"fav-transparent\",\"note_bg_color\":\"\",\"note_color\":\"\",\"note_title_color\":\"\",\"note_link_color\":\"\",\"note_link_hover_color\":\"\",\"base_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"base_bg_image_overlay\":\"fav-overlay\",\"base_bg_color\":\"\",\"base_color\":\"\",\"base_title_color\":\"\",\"base_link_color\":\"\",\"base_link_hover_color\":\"\",\"block_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"block_bg_image_overlay\":\"fav-transparent\",\"block_bg_color\":\"\",\"block_color\":\"\",\"block_title_color\":\"\",\"block_link_color\":\"\",\"block_link_hover_color\":\"\",\"user_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"user_bg_image_overlay\":\"fav-transparent\",\"user_bg_color\":\"\",\"user_color\":\"\",\"user_title_color\":\"\",\"user_link_color\":\"\",\"user_link_hover_color\":\"\",\"footer_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"footer_bg_image_overlay\":\"fav-transparent\",\"footer_bg_color\":\"\",\"footer_color\":\"\",\"footer_title_color\":\"\",\"footer_link_color\":\"\",\"footer_link_hover_color\":\"\",\"show_default_logo\":\"1\",\"default_logo\":\"logo.png\",\"default_logo_img_alt\":\"Favourite template\",\"default_logo_padding\":\"\",\"default_logo_margin\":\"\",\"show_media_logo\":\"0\",\"media_logo_img_alt\":\"Favourite template\",\"media_logo_padding\":\"\",\"media_logo_margin\":\"\",\"show_text_logo\":\"0\",\"text_logo\":\"Favourite\",\"text_logo_color\":\"\",\"text_logo_font_size\":\"\",\"text_logo_google_font\":\"\",\"text_logo_google_font_weight\":\"400\",\"text_logo_google_font_style\":\"normal\",\"text_logo_line_height\":\"\",\"text_logo_padding\":\"\",\"text_logo_margin\":\"\",\"show_slogan\":\"0\",\"slogan\":\"slogan text here\",\"slogan_color\":\"\",\"slogan_font_size\":\"\",\"slogan_line_height\":\"\",\"slogan_padding\":\"\",\"slogan_margin\":\"\",\"show_retina_logo\":\"0\",\"retina_logo_height\":\"52px\",\"retina_logo_width\":\"188px\",\"retina_logo_img_alt\":\"Favourite template\",\"retina_logo_padding\":\"0px\",\"retina_logo_margin\":\"0px\",\"mobile_nav_color\":\"navbar\",\"show_mobile_submenu\":\"none\",\"show_mobile_menu_text\":\"1\",\"mobile_menu_text\":\"Menu\",\"mobile_show_images\":\"inline-base\",\"paragraph_mobile_font_size\":\"\",\"article_mobile_title_font_size\":\"\",\"module_mobile_title_font_size\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(10187,0,'plg_editors-xtd_module','plugin','module','editors-xtd',0,0,1,0,'{\"name\":\"plg_editors-xtd_module\",\"type\":\"plugin\",\"creationDate\":\"October 2015\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.5.0\",\"description\":\"PLG_MODULE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"module\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10188,0,'plg_system_stats','plugin','stats','system',0,0,1,0,'{\"name\":\"plg_system_stats\",\"type\":\"plugin\",\"creationDate\":\"November 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.5.0\",\"description\":\"PLG_SYSTEM_STATS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"stats\"}','{\"interval\":\"12\",\"mode\":\"1\",\"lastrun\":\"\",\"debug\":\"0\"}','','',0,'0000-00-00 00:00:00',4,0),(10189,0,'plg_system_updatenotification','plugin','updatenotification','system',0,0,1,0,'{\"name\":\"plg_system_updatenotification\",\"type\":\"plugin\",\"creationDate\":\"May 2015\",\"author\":\"Joomla! Project\",\"copyright\":\"Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.5.0\",\"description\":\"PLG_SYSTEM_UPDATENOTIFICATION_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"updatenotification\"}','{\"email\":\"\",\"language_override\":\"\",\"lastrun\":\"0\"}','','',0,'0000-00-00 00:00:00',8,0),(10190,0,'PRO Twitter Widget','module','mod_pro_twitter_widget','',0,1,0,0,'{\"name\":\"PRO Twitter Widget\",\"type\":\"module\",\"creationDate\":\"January 2016\",\"author\":\"Joe Morrison\",\"copyright\":\"Copyright (C) 2016. All rights reserved by highschooldiplomaonlinefast.com.\",\"authorEmail\":\"joemorrisonyt@yahoo.com\",\"authorUrl\":\"http:\\/\\/www.highschooldiplomaonlinefast.com\\/extensions\",\"version\":\"1.0\",\"description\":\"Thanks for installing PRO Twitter Widget.\",\"group\":\"\",\"filename\":\"mod_pro_twitter_widget\"}','{\"twitter_id\":\"470475991895138304\",\"twitter_name\":\"BarackObama\",\"width\":\"300\",\"height\":\"150px\",\"scrollbar\":\"true\",\"color_scheme\":\"0\",\"backgroundColor\":\"#f5784f\",\"header\":\"true\",\"footer\":\"true\",\"border\":\"true\",\"tranparent\":\"false\",\"backgroundColors\":\"#e6e1e2\"}','','',0,'0000-00-00 00:00:00',0,0),(10193,10200,'COM_JCE','component','com_jce','',1,1,0,0,'{\"name\":\"COM_JCE\",\"type\":\"component\",\"creationDate\":\"25-10-2019\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2019 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"www.joomlacontenteditor.net\",\"version\":\"2.7.19\",\"description\":\"COM_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"jce\"}','{\"help_url\":\"http:\\/\\/www.joomlacontenteditor.net\",\"feed\":\"0\",\"feed_limit\":\"2\",\"browser_width\":\"780\",\"browser_height\":\"560\",\"replace_media_manager\":\"1\",\"secureparams\":\"0\",\"updates_key\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(10194,10200,'plg_editors_jce','plugin','jce','editors',0,1,1,0,'{\"name\":\"plg_editors_jce\",\"type\":\"plugin\",\"creationDate\":\"25-10-2019\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2019 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"http:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.7.19\",\"description\":\"WF_EDITOR_PLUGIN_DESC\",\"group\":\"\",\"filename\":\"jce\"}','{\"verify_html\":\"1\",\"schema\":\"mixed\",\"entity_encoding\":\"raw\",\"keep_nbsp\":\"0\",\"pad_empty_tags\":\"0\",\"cleanup_pluginmode\":\"0\",\"forced_root_block\":\"p\",\"content_style_reset\":\"0\",\"content_css\":\"1\",\"content_css_custom\":\"\",\"body_class\":\"\",\"compress_javascript\":\"0\",\"compress_css\":\"0\",\"compress_gzip\":\"0\",\"custom_config\":\"\",\"callback_file\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(10195,10200,'plg_content_jce','plugin','jce','content',0,1,1,0,'{\"name\":\"plg_content_jce\",\"type\":\"plugin\",\"creationDate\":\"25-10-2019\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2019 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"http:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.7.19\",\"description\":\"PLG_CONTENT_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"jce\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10196,10200,'plg_extension_jce','plugin','jce','extension',0,1,1,0,'{\"name\":\"plg_extension_jce\",\"type\":\"plugin\",\"creationDate\":\"25-10-2019\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2019 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"http:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.7.19\",\"description\":\"PLG_EXTENSION_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"jce\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10197,10200,'plg_installer_jce','plugin','jce','installer',0,1,1,0,'{\"name\":\"plg_installer_jce\",\"type\":\"plugin\",\"creationDate\":\"25-10-2019\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2019 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"http:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.7.19\",\"description\":\"PLG_INSTALLER_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"jce\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10198,10200,'plg_quickicon_jce','plugin','jce','quickicon',0,1,1,0,'{\"name\":\"plg_quickicon_jce\",\"type\":\"plugin\",\"creationDate\":\"25-10-2019\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2019 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"http:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.7.19\",\"description\":\"PLG_QUICKICON_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"jce\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10199,10200,'plg_system_jce','plugin','jce','system',0,1,1,0,'{\"name\":\"plg_system_jce\",\"type\":\"plugin\",\"creationDate\":\"25-10-2019\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2019 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"http:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.7.19\",\"description\":\"PLG_SYSTEM_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"jce\"}','{}','','',0,'0000-00-00 00:00:00',11,0),(10200,0,'PKG_JCE','package','pkg_jce','',0,1,1,0,'{\"name\":\"PKG_JCE\",\"type\":\"package\",\"creationDate\":\"25-10-2019\",\"author\":\"Ryan Demmer\",\"copyright\":\"\",\"authorEmail\":\"\",\"authorUrl\":\"\",\"version\":\"2.7.19\",\"description\":\"PKG_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"pkg_jce\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10208,0,'COM_EASYJOOMLABACKUP','component','com_easyjoomlabackup','',1,1,0,0,'{\"name\":\"COM_EASYJOOMLABACKUP\",\"type\":\"component\",\"creationDate\":\"2019-06-30\",\"author\":\"Viktor Vogel\",\"copyright\":\"Copyright 2019 Viktor Vogel. All rights reserved.\",\"authorEmail\":\"admin@kubik-rubik.de\",\"authorUrl\":\"https:\\/\\/kubik-rubik.de\\/\",\"version\":\"3.2.6\",\"description\":\"COM_EASYJOOMLABACKUP_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"com_easyjoomlabackup\"}','{\"exclude_files\":\".DS_Store\\r\\n.gitignore\",\"exclude_folders\":\".git\\r\\n.idea\\r\\nehld-icons\\r\\ndownload\\r\\nfigures\\r\\nimg-tmp\\r\\ncgi-tmp\",\"add_drop_statement\":\"1\",\"add_db_tables\":\"\",\"max_number_backups\":8,\"prefix_archive\":\"\",\"add_suffix_archive\":\"1\",\"donation_code\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(10209,0,'PLG_EASYJOOMLABACKUPCRONJOB','plugin','easyjoomlabackupcronjob','system',0,1,1,0,'{\"name\":\"PLG_EASYJOOMLABACKUPCRONJOB\",\"type\":\"plugin\",\"creationDate\":\"2019-06-30\",\"author\":\"Viktor Vogel\",\"copyright\":\"Copyright 2019 Viktor Vogel. All rights reserved.\",\"authorEmail\":\"\",\"authorUrl\":\"\",\"version\":\"3.2.6\",\"description\":\"PLG_EASYJOOMLABACKUPCRONJOB_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"easyjoomlabackupcronjob\"}','{\"token\":\"\",\"type\":\"1\",\"donation_code\":\"\"}','','',0,'0000-00-00 00:00:00',14,0),(10220,10226,'plg_installer_jeventsinstaller','plugin','jeventsinstaller','installer',0,1,1,0,'{\"name\":\"plg_installer_jeventsinstaller\",\"type\":\"plugin\",\"creationDate\":\"April 2017\",\"author\":\"GWE Systems Ltd\",\"copyright\":\"(C) 2016 GWE Systems Ltd. All rights reserved.\",\"authorEmail\":\"via website\",\"authorUrl\":\"www.gwesystems.com\",\"version\":\"3.4.36\",\"description\":\"PLG_INSTALLER_JEVENTSINSTALLER\",\"group\":\"\",\"filename\":\"jeventsinstaller\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10225,0,'plg_system_regularlabs','plugin','regularlabs','system',0,1,1,0,'{\"name\":\"plg_system_regularlabs\",\"type\":\"plugin\",\"creationDate\":\"October 2019\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2018 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/www.regularlabs.com\",\"version\":\"19.10.11711\",\"description\":\"PLG_SYSTEM_REGULARLABS_DESC\",\"group\":\"\",\"filename\":\"regularlabs\"}','{\"combine_admin_menu\":\"0\",\"show_help_menu\":\"1\",\"max_list_count\":\"2500\"}','','',0,'0000-00-00 00:00:00',7,0),(10226,0,'plg_editors-xtd_tabs','plugin','tabs','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_tabs\",\"type\":\"plugin\",\"creationDate\":\"August 2019\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2018 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/www.regularlabs.com\",\"version\":\"7.5.10\",\"description\":\"PLG_EDITORS-XTD_TABS_DESC\",\"group\":\"\",\"filename\":\"tabs\"}','[]','','',0,'0000-00-00 00:00:00',0,0),(10227,0,'plg_system_tabs','plugin','tabs','system',0,1,1,0,'{\"name\":\"plg_system_tabs\",\"type\":\"plugin\",\"creationDate\":\"August 2019\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2018 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/www.regularlabs.com\",\"version\":\"7.5.10\",\"description\":\"PLG_SYSTEM_TABS_DESC\",\"group\":\"\",\"filename\":\"tabs\"}','{\"load_stylesheet\":\"1\",\"mainclass\":\"\",\"alignment\":\"\",\"color_inactive_handles\":\"0\",\"outline_handles\":\"1\",\"outline_content\":\"1\",\"tag_open\":\"tab\",\"tag_close\":\"tabs\",\"tag_delimiter\":\"space\",\"tag_characters\":\"{.}\",\"title_tag\":\"h2\",\"use_hash\":\"0\",\"reload_iframes\":\"0\",\"init_timeout\":\"0\",\"enable_admin\":\"0\",\"place_comments\":\"1\",\"media_versioning\":\"1\",\"load_bootstrap_framework\":\"1\",\"load_jquery\":\"0\",\"button_text\":\"Tabs\",\"enable_frontend\":\"1\",\"button_use_simple_button\":\"0\",\"button_max_count\":\"20\",\"button_use_custom_code\":\"0\",\"button_custom_code\":\"{tab Tab Title 1}<br \\/>[:SELECTION:]<br \\/>{tab Tab Title 2}<br \\/>Tab text...<br \\/>{\\/tabs}\"}','','',0,'0000-00-00 00:00:00',3,0),(10230,0,'plg_editors-xtd_sliders','plugin','sliders','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_sliders\",\"type\":\"plugin\",\"creationDate\":\"October 2019\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2018 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/www.regularlabs.com\",\"version\":\"7.7.8\",\"description\":\"PLG_EDITORS-XTD_SLIDERS_DESC\",\"group\":\"\",\"filename\":\"sliders\"}','[]','','',0,'0000-00-00 00:00:00',0,0),(10231,0,'plg_system_sliders','plugin','sliders','system',0,1,1,0,'{\"name\":\"plg_system_sliders\",\"type\":\"plugin\",\"creationDate\":\"October 2019\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2018 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/www.regularlabs.com\",\"version\":\"7.7.8\",\"description\":\"PLG_SYSTEM_SLIDERS_DESC\",\"group\":\"\",\"filename\":\"sliders\"}','{\"load_stylesheet\":\"1\",\"mainclass\":\"\",\"state_first\":\"open\",\"tag_open\":\"slider\",\"tag_close\":\"sliders\",\"tag_delimiter\":\"space\",\"tag_characters\":\"{.}\",\"title_tag\":\"h2\",\"use_hash\":\"0\",\"reload_iframes\":\"0\",\"init_timeout\":\"0\",\"enable_admin\":\"0\",\"place_comments\":\"1\",\"media_versioning\":\"1\",\"load_bootstrap_framework\":\"1\",\"load_jquery\":\"0\",\"button_text\":\"Sliders\",\"enable_frontend\":\"1\",\"button_use_simple_button\":\"0\",\"button_max_count\":\"20\",\"button_use_custom_code\":\"0\",\"button_custom_code\":\"{slider Slider Title 1}<br \\/>[:SELECTION:]<br \\/>{slider Slider Title 2}<br \\/>Slider text...<br \\/>{\\/sliders}\"}','','',0,'0000-00-00 00:00:00',6,0),(10232,0,'plg_system_directalias','plugin','directalias','system',0,1,1,0,'{\"name\":\"plg_system_directalias\",\"type\":\"plugin\",\"creationDate\":\"01\\/09\\/2019\",\"author\":\"AlterBrains\",\"copyright\":\"(C) 2013-2019 AlterBrains. All rights reserved.\",\"authorEmail\":\"alterbrains@gmail.com\",\"authorUrl\":\"http:\\/\\/alterbrains.com\",\"version\":\"2.0.0\",\"description\":\"PLG_SYSTEM_DIRECTALIAS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"directalias\"}','{\"shorten_all\":\"0\"}','','',0,'0000-00-00 00:00:00',13,0),(10234,0,'Footer Menu','module','mod_footermenu','',0,1,0,0,'{\"name\":\"Footer Menu\",\"type\":\"module\",\"creationDate\":\"January 2015\",\"author\":\"KMO Design\",\"copyright\":\"Copyright (C) 2013-2014 KMO Design Pty Ltd. All rights reserved.\",\"authorEmail\":\"support@kmo.com.au\",\"authorUrl\":\"www.kmo.com.au\",\"version\":\"1.0.1\",\"description\":\"MOD_FOOTERMENU_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_footermenu\"}','{\"startLevel\":\"1\",\"endLevel\":\"0\",\"showAllChildren\":\"0\",\"use_default_stylesheet\":\"0\",\"columns\":\"0\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"itemid\"}','','',0,'0000-00-00 00:00:00',0,0),(10235,0,'System - JU Sticky Panel','plugin','justickypanel','system',0,0,1,0,'{\"name\":\"System - JU Sticky Panel\",\"type\":\"plugin\",\"creationDate\":\"Mar 2014\",\"author\":\"JoomUltra\",\"copyright\":\"Copyright (C) 2010 - 2014 JoomUltra. All rights reserved.\",\"authorEmail\":\"admin@joomultra.com\",\"authorUrl\":\"www.joomultra.com\",\"version\":\"1.2\",\"description\":\"\\n\\t\\t\\n\\t\\t<div style=\'font-weight: normal\'>\\n\\t\\t<h2 style=\'color: #0A71A5;\'>JU Sticky Panel Plugin<\\/strong><\\/h2>\\n\\t\\t<span style=\'color: #ff0000;\'><strong>Features:<\\/strong><\\/span>\\n\\t\\t<br\\/><br\\/>\\n\\t\\tStick any element in your site, the sticked elements will be keep always visible when you scroll the page down.\\n\\t\\t<br\\/><br\\/>\\n\\t\\t<span style=\'color: #ff0000;\'><strong>Usage:<\\/strong><\\/span>\\n\\t\\t<br\\/><br\\/>\\n\\t\\tYou can use <a href=\\\"http:\\/\\/getfirebug.com\\/\\\" title=\\\"Download Firebug\\\" target=\\\"_blank\\\">Firebug<\\/a> on Firefox to inspect the element you want to stick, then put the class or id or name,...<i>(See <a href=\\\"http:\\/\\/api.jquery.com\\/category\\/selectors\\/\\\" title=\\\"Read more about jQuery Selectors\\\" target=\\\"_blank\\\">jQuery selectors<\\/a>)<\\/i> of elements into the selectors config.\\n\\t\\tYou can put as many selectors as you want, seperate by commas, all elements with selectors you put in selectors config will be sticked by JU Sticky Panel.\\n\\t\\t<br\\/><br\\/>\\n\\t\\tHere is an example how to stick a module:<br\\/>\\n\\t\\t<ul>\\n\\t\\t\\t<li>1. In the module config, set <b>Module class suffix:<\\/b> \\\"<b> sticky<\\/b>\\\" <i>(without quote, and remember we have a white space preceded)<\\/i><\\/li>\\n\\t\\t\\t<li>2. In the JU Sticky Panel Plugin set <b>Selectors:<\\/b> <b>.sticky<\\/b><\\/li>\\n\\t\\t\\t<li>3. That\'s all, now check the frontend, scroll down your site, you will see the module with class suffix \\\"<b> sticky<\\/b>\\\" is ticked, and when it has been sticking it will be add new class in <b>After detach css class<\\/b>, you can use this class to format sticked element if you want. And now you can adjust other config in the plugin to fit your needs. Remember that JU Sticky Panel support multi selectors, that means you can stick many elements in your site.<\\/li>\\n\\t\\t<\\/ul>\\n\\t\\t<br\\/><br\\/>\\n\\t\\t<strong><span style=\'color: #ff0000;\'>Requirements:<\\/span><\\/strong>\\n\\t\\t<br\\/><br \\/>\\n\\t\\tJoomla! 2.5.x, Joomla! 3.x<br\\/>\\n\\t\\tThis plugin require jQuery javascript library to work. If your site gets jQuery conflict issue, you should install <b>Plugin <a href=\\\"http:\\/\\/extensions.joomla.org\\/extensions\\/core-enhancements\\/performance\\/jquery-scripts\\/18327\\\" target=\\\"_blank\\\" title=\\\"Download jQuery Easy\\\">jQuery Easy<\\/a><\\/b> <i>(Remember activate this plugin after installing it)<\\/i>.<br\\/><br\\/>\\n\\t\\tIf you use JU Sticky Panel, please post a review for <a href=\\\"http:\\/\\/extensions.joomla.org\\/extensions\\/style-a-design\\/modules-panel\\/25232\\\" target=\\\"_blank\\\" title=\\\"Post a review for JU Sticky Panel\\\">JU Sticky Panel on JED<\\/a>\\n\\t\\t<p>Copyright 2010 - 2014 <a href=\\\"http:\\/\\/www.joomultra.com\\/\\\" title=\\\"Visit JoomUltra.com!\\\" target=\\\"_blank\\\">JoomUltra.com<\\/a>.<\\/p>\\n\\t\\t<\\/div>\\n\\t\\t\\n\\t\",\"group\":\"\",\"filename\":\"justickypanel\"}','{\"loadjquery\":\"2\",\"loadjqueryfrom\":\"1\",\"selectors\":\".sticky\",\"topPadding\":\"0\",\"afterDetachCSSClass\":\"detached\",\"savePanelSpace\":\"true\",\"parentSelector\":\"\",\"onDetached\":\"null\",\"onReAttached\":\"null\",\"enablepaths\":\"\",\"disablepaths\":\"\"}','','',0,'0000-00-00 00:00:00',9,0),(10238,0,'plg_editors-xtd_sourcerer','plugin','sourcerer','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_sourcerer\",\"type\":\"plugin\",\"creationDate\":\"September 2019\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2018 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/www.regularlabs.com\",\"version\":\"8.2.0\",\"description\":\"PLG_EDITORS-XTD_SOURCERER_DESC\",\"group\":\"\",\"filename\":\"sourcerer\"}','[]','','',0,'0000-00-00 00:00:00',0,0),(10239,0,'plg_system_sourcerer','plugin','sourcerer','system',0,1,1,0,'{\"name\":\"plg_system_sourcerer\",\"type\":\"plugin\",\"creationDate\":\"September 2019\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2018 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/www.regularlabs.com\",\"version\":\"8.2.0\",\"description\":\"PLG_SYSTEM_SOURCERER_DESC\",\"group\":\"\",\"filename\":\"sourcerer\"}','{\"syntax_word\":\"source\",\"tag_characters\":\"{.}\",\"trim\":\"0\",\"enable_in_head\":\"0\",\"remove_from_search\":\"0\",\"include_path\":\"\\/\",\"place_comments\":\"1\",\"enable_css\":\"1\",\"enable_js\":\"1\",\"enable_php\":\"1\",\"forbidden_php\":\"dl, escapeshellarg, escapeshellcmd, exec, passthru, popen, proc_close, proc_open, shell_exec, symlink, system\",\"forbidden_tags\":\"\",\"@notice_articles_security_level\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_articles_enable_css\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_articles_enable_js\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_articles_enable_php\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_components_enable_css\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_components_enable_js\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_components_enable_php\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@wizard\":\"0\",\"@noticeother_enable_css\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_other_enable_js\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_other_enable_php\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"button_text\":\"Code\",\"enable_frontend\":\"1\",\"addsourcetags\":\"1\",\"@notice_use_example_code\":\"RL_ONLY_AVAILABLE_IN_PRO\"}','','',0,'0000-00-00 00:00:00',5,0),(10254,0,'plg_system_modals','plugin','modals','system',0,1,1,0,'{\"name\":\"plg_system_modals\",\"type\":\"plugin\",\"creationDate\":\"October 2019\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2018 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/www.regularlabs.com\",\"version\":\"11.5.6\",\"description\":\"PLG_SYSTEM_MODALS_DESC\",\"group\":\"\",\"filename\":\"modals\"}','{\"load_stylesheet\":\"1\",\"style\":\"bootstrap\",\"autoconvert_classnames\":\"1\",\"classnames\":\"modal\",\"opacity\":\"0.8\",\"width\":\"\",\"height\":\"\",\"externalwidth\":\"95%\",\"externalheight\":\"95%\",\"initialwidth\":\"600\",\"initialheight\":\"450\",\"maxwidth\":\"95%\",\"maxheight\":\"95%\",\"tag\":\"modal\",\"tag_characters\":\"{.}\",\"tmpl\":\"modal\",\"iframe\":\"0\",\"load_head\":\"0\",\"mediafiles\":\"jpg,png,gif,eps,pdf\",\"iframefiles\":\"pdf\",\"exclude_urls\":\"\",\"auto_titles\":\"1\",\"title_case\":\"titlecase\",\"lowercase_words\":\"a,the,to,at,in,with,and,but,or\",\"enable_admin\":\"0\",\"place_comments\":\"1\",\"add_redirect\":\"1\",\"media_versioning\":\"1\",\"load_jquery\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(10259,0,'FavPromote4col','module','mod_favpromote4col','',0,1,0,0,'{\"name\":\"FavPromote4col\",\"type\":\"module\",\"creationDate\":\"2013\",\"author\":\"FavThemes\",\"copyright\":\"Copyright (C) FavThemes. All rights reserved.\",\"authorEmail\":\"\",\"authorUrl\":\"http:\\/\\/www.favthemes.com\",\"version\":\"2.1\",\"description\":\"\\n  \\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\"><strong>FavPromote<\\/strong><\\/a> is a free responsive Joomla! module that lets you upload any image and assign it a short description and a title with a Font Awesome icon.\\n    <br\\/>\\n    It works exactly in the same way as the pure FavPromote, however it is fixed in 4 cols or less. It means, if only one column is present it will occupy 3 empty spaces (12\\/4columns).\\n    For more info, please see the <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">demo and documentation<\\/a> for the FavPromote module.\\n    <br\\/><br\\/>\\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\">\\n      <img src=\\\"..\\/modules\\/mod_favpromote4col\\/theme\\/img\\/favpromote4col.jpg\\\" alt=\\\"FavPromote Responsive Module\\\">\\n    <\\/a>\\n\\n    <br\\/><br\\/>\\n    Copyright &#169; 2012-2017 <a href=\\\"http:\\/\\/www.favthemes.com\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">FavThemes<\\/a>.\\n    <br\\/><br\\/>\\n\\n\\n    <link rel=\\\"stylesheet\\\" href=\\\"..\\/modules\\/mod_favpromote4col\\/theme\\/css\\/admin.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/maxcdn.bootstrapcdn.com\\/font-awesome\\/4.7.0\\/css\\/font-awesome.min.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/fonts.googleapis.com\\/css?family=Open+Sans:400\\\" type=\\\"text\\/css\\\" \\/>\\n    <script src=\\\"..\\/modules\\/mod_favpromote4col\\/theme\\/js\\/jscolor\\/jscolor.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n    <script src=\\\"..\\/modules\\/mod_favpromote4col\\/theme\\/js\\/admin\\/admin.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n\\n  \\n\\t\",\"group\":\"\",\"filename\":\"mod_favpromote4col\"}','{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect1\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_padding\":\"10px 20px\",\"title_font_size\":\"18px\",\"title_line_height\":\"1.4em\",\"title_text_align\":\"left\",\"title_icon_font_size\":\"18px\",\"title_icon_vertical_align\":\"baseline\",\"description_text_font_size\":\"14px\",\"description_text_line_height\":\"1.5em\",\"description_text_align\":\"justify\",\"show_column1\":\"1\",\"column_border_color1\":\"DDDDDD\",\"column_border_radius1\":\"4px\",\"show_image_link1\":\"1\",\"image_link1\":\"\",\"image_target1\":\"blank\",\"image_alt1\":\"\",\"title_text1\":\"Title Text\",\"title_color1\":\"FFFFFF\",\"title_bg_color1\":\"D63E52\",\"show_title_link1\":\"1\",\"title_link1\":\"\",\"title_target1\":\"blank\",\"title_icon1\":\"fa-coffee\",\"description_text1\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color1\":\"444444\",\"show_column2\":\"1\",\"column_border_color2\":\"DDDDDD\",\"column_border_radius2\":\"4px\",\"show_image_link2\":\"1\",\"image_link2\":\"\",\"image_target2\":\"blank\",\"image_alt2\":\"\",\"title_text2\":\"Title Text\",\"title_color2\":\"FFFFFF\",\"title_bg_color2\":\"D63E52\",\"show_title_link2\":\"1\",\"title_link2\":\"\",\"title_target2\":\"blank\",\"title_icon2\":\"fa-puzzle-piece\",\"description_text2\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color2\":\"444444\",\"show_column3\":\"1\",\"column_border_color3\":\"DDDDDD\",\"column_border_radius3\":\"4px\",\"show_image_link3\":\"1\",\"image_link3\":\"\",\"image_target3\":\"blank\",\"image_alt3\":\"\",\"title_text3\":\"Title Text\",\"title_color3\":\"FFFFFF\",\"title_bg_color3\":\"D63E52\",\"show_title_link3\":\"1\",\"title_link3\":\"\",\"title_target3\":\"blank\",\"title_icon3\":\"fa-bookmark\",\"description_text3\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color3\":\"444444\",\"show_column4\":\"1\",\"column_border_color4\":\"DDDDDD\",\"column_border_radius4\":\"4px\",\"show_image_link4\":\"1\",\"image_link4\":\"\",\"image_target4\":\"blank\",\"image_alt4\":\"\",\"title_text4\":\"Title Text\",\"title_color4\":\"FFFFFF\",\"title_bg_color4\":\"D63E52\",\"show_title_link4\":\"1\",\"title_link4\":\"\",\"title_target4\":\"blank\",\"title_icon4\":\"fa-magic\",\"description_text4\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color4\":\"444444\",\"show_column5\":\"0\",\"column_border_color5\":\"DDDDDD\",\"column_border_radius5\":\"4px\",\"show_image_link5\":\"1\",\"image_link5\":\"\",\"image_target5\":\"blank\",\"image_alt5\":\"\",\"title_text5\":\"Title Text\",\"title_color5\":\"FFFFFF\",\"title_bg_color5\":\"D63E52\",\"show_title_link5\":\"1\",\"title_link5\":\"\",\"title_target5\":\"blank\",\"title_icon5\":\"fa-quote-right\",\"description_text5\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color5\":\"444444\",\"show_column6\":\"0\",\"column_border_color6\":\"DDDDDD\",\"column_border_radius6\":\"4px\",\"show_image_link6\":\"1\",\"image_link6\":\"\",\"image_target6\":\"blank\",\"image_alt6\":\"\",\"title_text6\":\"Title Text\",\"title_color6\":\"FFFFFF\",\"title_bg_color6\":\"D63E52\",\"show_title_link6\":\"1\",\"title_link6\":\"\",\"title_target6\":\"blank\",\"title_icon6\":\"fa-recycle\",\"description_text6\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color6\":\"444444\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\"}','','',0,'0000-00-00 00:00:00',0,0),(10269,0,'plg_editors-xtd_modals','plugin','modals','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_modals\",\"type\":\"plugin\",\"creationDate\":\"October 2019\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2018 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/www.regularlabs.com\",\"version\":\"11.5.6\",\"description\":\"PLG_EDITORS-XTD_MODALS_DESC\",\"group\":\"\",\"filename\":\"modals\"}','[]','','',0,'0000-00-00 00:00:00',0,0),(10302,0,'mod_jsocialfeed','module','mod_jsocialfeed','',0,0,1,0,'{\"name\":\"mod_jsocialfeed\",\"type\":\"module\",\"creationDate\":\"03\\/06\\/2014\",\"author\":\"Garda Informatica\",\"copyright\":\"Garda Informatica\",\"authorEmail\":\"info@gardainformatica.it\",\"authorUrl\":\"http:\\/\\/www.gardainformatica.it\\/\",\"version\":\"1.5.4\",\"description\":\"MOD_JSOCIALFEED_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_jsocialfeed\"}','{}','','',0,'0000-00-00 00:00:00',0,-1),(10303,0,'jsocialfeed','library','jsocialfeed','',0,0,1,0,'{\"name\":\"JSocialFeed\",\"type\":\"library\",\"creationDate\":\"03\\/06\\/2014\",\"author\":\"Garda Informatica\",\"copyright\":\"Garda Informatica\",\"authorEmail\":\"info@gardainformatica.it\",\"authorUrl\":\"http:\\/\\/www.gardainformatica.it\\/\",\"version\":\"1.5.4\",\"description\":\"\",\"group\":\"\",\"filename\":\"jsocialfeed\"}','{}','','',0,'0000-00-00 00:00:00',0,-1),(10304,0,'com_jsocialfeed','component','com_jsocialfeed','',1,0,1,0,'{\"name\":\"COM_JSOCIALFEED\",\"type\":\"component\",\"creationDate\":\"03\\/06\\/2014\",\"author\":\"Garda Informatica\",\"copyright\":\"Garda Informatica\",\"authorEmail\":\"info@gardainformatica.it\",\"authorUrl\":\"http:\\/\\/www.gardainformatica.it\\/\",\"version\":\"1.5.4\",\"description\":\"COM_JSOCIALFEED_DESCRIPTION\",\"group\":\"\",\"filename\":\"jsocialfeed\"}','{}','','',0,'0000-00-00 00:00:00',0,-1),(10307,10200,'plg_fields_mediajce','plugin','mediajce','fields',0,1,1,0,'{\"name\":\"plg_fields_mediajce\",\"type\":\"plugin\",\"creationDate\":\"25-10-2019\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2019 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"https:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.7.19\",\"description\":\"PLG_FIELDS_MEDIAJCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mediajce\"}','[]','','',0,'0000-00-00 00:00:00',0,0),(10354,0,'Regular Labs Library','library','regularlabs','',0,1,1,0,'{\"name\":\"Regular Labs Library\",\"type\":\"library\",\"creationDate\":\"October 2019\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2018 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/www.regularlabs.com\",\"version\":\"19.10.11711\",\"description\":\"\",\"group\":\"\",\"filename\":\"regularlabs\"}','{}','','',0,'0000-00-00 00:00:00',0,0);
/*!40000 ALTER TABLE `rlp_extensions` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_fields`
--

DROP TABLE IF EXISTS `rlp_fields`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_fields` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `asset_id` int(10) unsigned NOT NULL DEFAULT '0',
  `context` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `group_id` int(10) unsigned NOT NULL DEFAULT '0',
  `title` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `name` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `label` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `default_value` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `type` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT 'text',
  `note` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `description` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `state` tinyint(1) NOT NULL DEFAULT '0',
  `required` tinyint(1) NOT NULL DEFAULT '0',
  `checked_out` int(11) NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `ordering` int(11) NOT NULL DEFAULT '0',
  `params` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `fieldparams` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `language` char(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `created_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `modified_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int(10) unsigned NOT NULL DEFAULT '0',
  `access` int(11) NOT NULL DEFAULT '1',
  PRIMARY KEY (`id`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_state` (`state`),
  KEY `idx_created_user_id` (`created_user_id`),
  KEY `idx_access` (`access`),
  KEY `idx_context` (`context`(191)),
  KEY `idx_language` (`language`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_fields`
--

LOCK TABLES `rlp_fields` WRITE;
/*!40000 ALTER TABLE `rlp_fields` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_fields` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_fields_categories`
--

DROP TABLE IF EXISTS `rlp_fields_categories`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_fields_categories` (
  `field_id` int(11) NOT NULL DEFAULT '0',
  `category_id` int(11) NOT NULL DEFAULT '0',
  PRIMARY KEY (`field_id`,`category_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_fields_categories`
--

LOCK TABLES `rlp_fields_categories` WRITE;
/*!40000 ALTER TABLE `rlp_fields_categories` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_fields_categories` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_fields_groups`
--

DROP TABLE IF EXISTS `rlp_fields_groups`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_fields_groups` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `asset_id` int(10) unsigned NOT NULL DEFAULT '0',
  `context` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `title` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `note` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `description` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `state` tinyint(1) NOT NULL DEFAULT '0',
  `checked_out` int(11) NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `ordering` int(11) NOT NULL DEFAULT '0',
  `params` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `language` char(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by` int(10) unsigned NOT NULL DEFAULT '0',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int(10) unsigned NOT NULL DEFAULT '0',
  `access` int(11) NOT NULL DEFAULT '1',
  PRIMARY KEY (`id`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_state` (`state`),
  KEY `idx_created_by` (`created_by`),
  KEY `idx_access` (`access`),
  KEY `idx_context` (`context`(191)),
  KEY `idx_language` (`language`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_fields_groups`
--

LOCK TABLES `rlp_fields_groups` WRITE;
/*!40000 ALTER TABLE `rlp_fields_groups` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_fields_groups` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_fields_values`
--

DROP TABLE IF EXISTS `rlp_fields_values`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_fields_values` (
  `field_id` int(10) unsigned NOT NULL,
  `item_id` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'Allow references to items which have strings as ids, eg. none db systems.',
  `value` text COLLATE utf8mb4_unicode_ci NOT NULL,
  KEY `idx_field_id` (`field_id`),
  KEY `idx_item_id` (`item_id`(191))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_fields_values`
--

LOCK TABLES `rlp_fields_values` WRITE;
/*!40000 ALTER TABLE `rlp_fields_values` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_fields_values` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_filters`
--

DROP TABLE IF EXISTS `rlp_finder_filters`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_filters` (
  `filter_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `title` varchar(255) NOT NULL,
  `alias` varchar(255) NOT NULL,
  `state` tinyint(1) NOT NULL DEFAULT '1',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by` int(10) unsigned NOT NULL,
  `created_by_alias` varchar(255) NOT NULL,
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int(10) unsigned NOT NULL DEFAULT '0',
  `checked_out` int(10) unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `map_count` int(10) unsigned NOT NULL DEFAULT '0',
  `data` mediumtext NOT NULL,
  `params` longtext,
  PRIMARY KEY (`filter_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_filters`
--

LOCK TABLES `rlp_finder_filters` WRITE;
/*!40000 ALTER TABLE `rlp_finder_filters` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_filters` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links`
--

DROP TABLE IF EXISTS `rlp_finder_links`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links` (
  `link_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `url` varchar(255) NOT NULL,
  `route` varchar(255) NOT NULL,
  `title` varchar(400) DEFAULT NULL,
  `description` text,
  `indexdate` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `md5sum` varchar(32) DEFAULT NULL,
  `published` tinyint(1) NOT NULL DEFAULT '1',
  `state` int(5) DEFAULT '1',
  `access` int(5) DEFAULT '0',
  `language` varchar(8) NOT NULL,
  `publish_start_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_end_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `start_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `end_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `list_price` double unsigned NOT NULL DEFAULT '0',
  `sale_price` double unsigned NOT NULL DEFAULT '0',
  `type_id` int(11) NOT NULL,
  `object` mediumblob NOT NULL,
  PRIMARY KEY (`link_id`),
  KEY `idx_type` (`type_id`),
  KEY `idx_md5` (`md5sum`),
  KEY `idx_url` (`url`(75)),
  KEY `idx_published_list` (`published`,`state`,`access`,`publish_start_date`,`publish_end_date`,`list_price`),
  KEY `idx_published_sale` (`published`,`state`,`access`,`publish_start_date`,`publish_end_date`,`sale_price`),
  KEY `idx_title` (`title`(100))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links`
--

LOCK TABLES `rlp_finder_links` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms0`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms0`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_terms0` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms0`
--

LOCK TABLES `rlp_finder_links_terms0` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms0` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms0` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms1`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms1`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_terms1` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms1`
--

LOCK TABLES `rlp_finder_links_terms1` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms1` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms1` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms2`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms2`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_terms2` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms2`
--

LOCK TABLES `rlp_finder_links_terms2` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms2` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms2` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms3`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms3`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_terms3` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms3`
--

LOCK TABLES `rlp_finder_links_terms3` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms3` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms3` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms4`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms4`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_terms4` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms4`
--

LOCK TABLES `rlp_finder_links_terms4` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms4` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms4` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms5`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms5`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_terms5` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms5`
--

LOCK TABLES `rlp_finder_links_terms5` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms5` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms5` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms6`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms6`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_terms6` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms6`
--

LOCK TABLES `rlp_finder_links_terms6` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms6` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms6` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms7`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms7`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_terms7` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms7`
--

LOCK TABLES `rlp_finder_links_terms7` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms7` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms7` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms8`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms8`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_terms8` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms8`
--

LOCK TABLES `rlp_finder_links_terms8` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms8` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms8` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms9`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms9`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_terms9` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms9`
--

LOCK TABLES `rlp_finder_links_terms9` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms9` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms9` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termsa`
--

DROP TABLE IF EXISTS `rlp_finder_links_termsa`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_termsa` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termsa`
--

LOCK TABLES `rlp_finder_links_termsa` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termsa` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termsa` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termsb`
--

DROP TABLE IF EXISTS `rlp_finder_links_termsb`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_termsb` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termsb`
--

LOCK TABLES `rlp_finder_links_termsb` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termsb` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termsb` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termsc`
--

DROP TABLE IF EXISTS `rlp_finder_links_termsc`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_termsc` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termsc`
--

LOCK TABLES `rlp_finder_links_termsc` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termsc` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termsc` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termsd`
--

DROP TABLE IF EXISTS `rlp_finder_links_termsd`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_termsd` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termsd`
--

LOCK TABLES `rlp_finder_links_termsd` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termsd` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termsd` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termse`
--

DROP TABLE IF EXISTS `rlp_finder_links_termse`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_termse` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termse`
--

LOCK TABLES `rlp_finder_links_termse` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termse` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termse` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termsf`
--

DROP TABLE IF EXISTS `rlp_finder_links_termsf`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_links_termsf` (
  `link_id` int(10) unsigned NOT NULL,
  `term_id` int(10) unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termsf`
--

LOCK TABLES `rlp_finder_links_termsf` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termsf` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termsf` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_taxonomy`
--

DROP TABLE IF EXISTS `rlp_finder_taxonomy`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_taxonomy` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `parent_id` int(10) unsigned NOT NULL DEFAULT '0',
  `title` varchar(255) NOT NULL,
  `state` tinyint(1) unsigned NOT NULL DEFAULT '1',
  `access` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `ordering` tinyint(1) unsigned NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`),
  KEY `parent_id` (`parent_id`),
  KEY `state` (`state`),
  KEY `ordering` (`ordering`),
  KEY `access` (`access`),
  KEY `idx_parent_published` (`parent_id`,`state`,`access`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_taxonomy`
--

LOCK TABLES `rlp_finder_taxonomy` WRITE;
/*!40000 ALTER TABLE `rlp_finder_taxonomy` DISABLE KEYS */;
INSERT INTO `rlp_finder_taxonomy` VALUES (1,1,'Type',1,1,0);
/*!40000 ALTER TABLE `rlp_finder_taxonomy` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_taxonomy_map`
--

DROP TABLE IF EXISTS `rlp_finder_taxonomy_map`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_taxonomy_map` (
  `link_id` int(10) unsigned NOT NULL,
  `node_id` int(10) unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`node_id`),
  KEY `link_id` (`link_id`),
  KEY `node_id` (`node_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_taxonomy_map`
--

LOCK TABLES `rlp_finder_taxonomy_map` WRITE;
/*!40000 ALTER TABLE `rlp_finder_taxonomy_map` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_taxonomy_map` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_terms`
--

DROP TABLE IF EXISTS `rlp_finder_terms`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_terms` (
  `term_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `term` varchar(75) NOT NULL,
  `stem` varchar(75) NOT NULL,
  `common` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `phrase` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `weight` float unsigned NOT NULL DEFAULT '0',
  `soundex` varchar(75) NOT NULL,
  `links` int(10) NOT NULL DEFAULT '0',
  `language` char(3) NOT NULL DEFAULT '',
  PRIMARY KEY (`term_id`),
  UNIQUE KEY `idx_term` (`term`),
  KEY `idx_term_phrase` (`term`,`phrase`),
  KEY `idx_stem_phrase` (`stem`,`phrase`),
  KEY `idx_soundex_phrase` (`soundex`,`phrase`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_terms`
--

LOCK TABLES `rlp_finder_terms` WRITE;
/*!40000 ALTER TABLE `rlp_finder_terms` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_terms` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_terms_common`
--

DROP TABLE IF EXISTS `rlp_finder_terms_common`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_terms_common` (
  `term` varchar(75) NOT NULL,
  `language` varchar(3) NOT NULL,
  KEY `idx_word_lang` (`term`,`language`),
  KEY `idx_lang` (`language`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_terms_common`
--

LOCK TABLES `rlp_finder_terms_common` WRITE;
/*!40000 ALTER TABLE `rlp_finder_terms_common` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_terms_common` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_tokens`
--

DROP TABLE IF EXISTS `rlp_finder_tokens`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_tokens` (
  `term` varchar(75) NOT NULL,
  `stem` varchar(75) NOT NULL,
  `common` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `phrase` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `weight` float unsigned NOT NULL DEFAULT '1',
  `context` tinyint(1) unsigned NOT NULL DEFAULT '2',
  `language` char(3) NOT NULL DEFAULT '',
  KEY `idx_word` (`term`),
  KEY `idx_context` (`context`)
) ENGINE=MEMORY DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_tokens`
--

LOCK TABLES `rlp_finder_tokens` WRITE;
/*!40000 ALTER TABLE `rlp_finder_tokens` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_tokens` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_tokens_aggregate`
--

DROP TABLE IF EXISTS `rlp_finder_tokens_aggregate`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_tokens_aggregate` (
  `term_id` int(10) unsigned NOT NULL,
  `map_suffix` char(1) NOT NULL,
  `term` varchar(75) NOT NULL,
  `stem` varchar(75) NOT NULL,
  `common` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `phrase` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `term_weight` float unsigned NOT NULL,
  `context` tinyint(1) unsigned NOT NULL DEFAULT '2',
  `context_weight` float unsigned NOT NULL,
  `total_weight` float unsigned NOT NULL,
  `language` char(3) NOT NULL DEFAULT '',
  KEY `token` (`term`),
  KEY `keyword_id` (`term_id`)
) ENGINE=MEMORY DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_tokens_aggregate`
--

LOCK TABLES `rlp_finder_tokens_aggregate` WRITE;
/*!40000 ALTER TABLE `rlp_finder_tokens_aggregate` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_tokens_aggregate` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_types`
--

DROP TABLE IF EXISTS `rlp_finder_types`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_finder_types` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `title` varchar(100) NOT NULL,
  `mime` varchar(100) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `title` (`title`)
) ENGINE=InnoDB AUTO_INCREMENT=12 DEFAULT CHARSET=utf8mb4;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_types`
--

LOCK TABLES `rlp_finder_types` WRITE;
/*!40000 ALTER TABLE `rlp_finder_types` DISABLE KEYS */;
INSERT INTO `rlp_finder_types` VALUES (6,'Tag',''),(7,'Category',''),(8,'Contact',''),(9,'Article',''),(10,'News Feed',''),(11,'Event','');
/*!40000 ALTER TABLE `rlp_finder_types` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_languages`
--

DROP TABLE IF EXISTS `rlp_languages`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_languages` (
  `lang_id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `asset_id` int(10) unsigned NOT NULL DEFAULT '0',
  `lang_code` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
  `title` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL,
  `title_native` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL,
  `sef` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL,
  `image` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL,
  `description` varchar(512) COLLATE utf8mb4_unicode_ci NOT NULL,
  `metakey` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadesc` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `sitename` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `published` int(11) NOT NULL DEFAULT '0',
  `access` int(10) unsigned NOT NULL DEFAULT '0',
  `ordering` int(11) NOT NULL DEFAULT '0',
  PRIMARY KEY (`lang_id`),
  UNIQUE KEY `idx_sef` (`sef`),
  UNIQUE KEY `idx_langcode` (`lang_code`),
  KEY `idx_access` (`access`),
  KEY `idx_ordering` (`ordering`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_languages`
--

LOCK TABLES `rlp_languages` WRITE;
/*!40000 ALTER TABLE `rlp_languages` DISABLE KEYS */;
INSERT INTO `rlp_languages` VALUES (1,0,'en-GB','English (UK)','English (UK)','en','en','','','','',1,1,-1);
/*!40000 ALTER TABLE `rlp_languages` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_menu`
--

DROP TABLE IF EXISTS `rlp_menu`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_menu` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `menutype` varchar(24) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The type of menu this item belongs to. FK to #__menu_types.menutype',
  `title` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The display title of the menu item.',
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL COMMENT 'The SEF alias of the menu item.',
  `note` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `path` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The computed path of the menu item based on the alias field.',
  `link` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The actually link the menu item refers to.',
  `type` varchar(16) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The type of link: Component, URL, Alias, Separator',
  `published` tinyint(4) NOT NULL DEFAULT '0' COMMENT 'The published state of the menu link.',
  `parent_id` int(10) unsigned NOT NULL DEFAULT '1' COMMENT 'The parent menu item in the menu tree.',
  `level` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'The relative level in the tree.',
  `component_id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'FK to #__extensions.id',
  `checked_out` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'FK to #__users.id',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00' COMMENT 'The time the menu item was checked out.',
  `browserNav` tinyint(4) NOT NULL DEFAULT '0' COMMENT 'The click behaviour of the link.',
  `access` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'The access level required to view the menu item.',
  `img` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The image of the menu item.',
  `template_style_id` int(10) unsigned NOT NULL DEFAULT '0',
  `params` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'JSON encoded data for the menu item.',
  `lft` int(11) NOT NULL DEFAULT '0' COMMENT 'Nested set lft.',
  `rgt` int(11) NOT NULL DEFAULT '0' COMMENT 'Nested set rgt.',
  `home` tinyint(3) unsigned NOT NULL DEFAULT '0' COMMENT 'Indicates if this menu item is the home or default page.',
  `language` char(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `client_id` tinyint(4) NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`),
  UNIQUE KEY `idx_client_id_parent_id_alias_language` (`client_id`,`parent_id`,`alias`(100),`language`),
  KEY `idx_componentid` (`component_id`,`menutype`,`published`,`access`),
  KEY `idx_menutype` (`menutype`),
  KEY `idx_left_right` (`lft`,`rgt`),
  KEY `idx_language` (`language`),
  KEY `idx_alias` (`alias`(100)),
  KEY `idx_path` (`path`(100))
) ENGINE=InnoDB AUTO_INCREMENT=509 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_menu`
--

LOCK TABLES `rlp_menu` WRITE;
/*!40000 ALTER TABLE `rlp_menu` DISABLE KEYS */;
INSERT INTO `rlp_menu` VALUES (1,'','Menu_Item_Root','root','','','','',1,0,0,0,0,'0000-00-00 00:00:00',0,0,'',0,'',0,249,0,'*',0),(2,'main','com_banners','Banners','','Banners','index.php?option=com_banners','component',1,1,1,4,0,'0000-00-00 00:00:00',0,0,'class:banners',0,'',1,10,0,'*',1),(3,'main','com_banners','Banners','','Banners/Banners','index.php?option=com_banners','component',1,2,2,4,0,'0000-00-00 00:00:00',0,0,'class:banners',0,'',2,3,0,'*',1),(4,'main','com_banners_categories','Categories','','Banners/Categories','index.php?option=com_categories&extension=com_banners','component',1,2,2,6,0,'0000-00-00 00:00:00',0,0,'class:banners-cat',0,'',4,5,0,'*',1),(5,'main','com_banners_clients','Clients','','Banners/Clients','index.php?option=com_banners&view=clients','component',1,2,2,4,0,'0000-00-00 00:00:00',0,0,'class:banners-clients',0,'',6,7,0,'*',1),(6,'main','com_banners_tracks','Tracks','','Banners/Tracks','index.php?option=com_banners&view=tracks','component',1,2,2,4,0,'0000-00-00 00:00:00',0,0,'class:banners-tracks',0,'',8,9,0,'*',1),(7,'main','com_contact','Contacts','','Contacts','index.php?option=com_contact','component',1,1,1,8,0,'0000-00-00 00:00:00',0,0,'class:contact',0,'',193,198,0,'*',1),(8,'main','com_contact','Contacts','','Contacts/Contacts','index.php?option=com_contact','component',1,7,2,8,0,'0000-00-00 00:00:00',0,0,'class:contact',0,'',194,195,0,'*',1),(9,'main','com_contact_categories','Categories','','Contacts/Categories','index.php?option=com_categories&extension=com_contact','component',1,7,2,6,0,'0000-00-00 00:00:00',0,0,'class:contact-cat',0,'',196,197,0,'*',1),(10,'main','com_messages','Messaging','','Messaging','index.php?option=com_messages','component',1,1,1,15,0,'0000-00-00 00:00:00',0,0,'class:messages',0,'',199,204,0,'*',1),(11,'main','com_messages_add','New Private Message','','Messaging/New Private Message','index.php?option=com_messages&task=message.add','component',1,10,2,15,0,'0000-00-00 00:00:00',0,0,'class:messages-add',0,'',200,201,0,'*',1),(12,'main','com_messages_read','Read Private Message','','Messaging/Read Private Message','index.php?option=com_messages','component',1,10,2,15,0,'0000-00-00 00:00:00',0,0,'class:messages-read',0,'',202,203,0,'*',1),(13,'main','com_newsfeeds','News Feeds','','News Feeds','index.php?option=com_newsfeeds','component',1,1,1,17,0,'0000-00-00 00:00:00',0,0,'class:newsfeeds',0,'',205,210,0,'*',1),(14,'main','com_newsfeeds_feeds','Feeds','','News Feeds/Feeds','index.php?option=com_newsfeeds','component',1,13,2,17,0,'0000-00-00 00:00:00',0,0,'class:newsfeeds',0,'',206,207,0,'*',1),(15,'main','com_newsfeeds_categories','Categories','','News Feeds/Categories','index.php?option=com_categories&extension=com_newsfeeds','component',1,13,2,6,0,'0000-00-00 00:00:00',0,0,'class:newsfeeds-cat',0,'',208,209,0,'*',1),(16,'main','com_redirect','Redirect','','Redirect','index.php?option=com_redirect','component',1,1,1,24,0,'0000-00-00 00:00:00',0,0,'class:redirect',0,'',211,212,0,'*',1),(17,'main','com_search','Basic Search','','Basic Search','index.php?option=com_search','component',1,1,1,19,0,'0000-00-00 00:00:00',0,0,'class:search',0,'',213,214,0,'*',1),(18,'main','com_finder','Smart Search','','Smart Search','index.php?option=com_finder','component',1,1,1,27,0,'0000-00-00 00:00:00',0,0,'class:finder',0,'',215,216,0,'*',1),(20,'main','com_tags','Tags','','Tags','index.php?option=com_tags','component',1,1,1,29,0,'0000-00-00 00:00:00',0,1,'class:tags',0,'',217,218,0,'',1),(21,'main','com_postinstall','Post-installation messages','','Post-installation messages','index.php?option=com_postinstall','component',1,1,1,32,0,'0000-00-00 00:00:00',0,1,'class:postinstall',0,'',219,220,0,'*',1),(101,'mainmenu','Home','homepage','','homepage','index.php?option=com_content&view=featured','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"0\",\"info_block_show_title\":\"\",\"show_category\":\"0\",\"link_category\":\"0\",\"show_parent_category\":\"0\",\"link_parent_category\":\"0\",\"show_associations\":\"\",\"show_author\":\"0\",\"link_author\":\"0\",\"show_create_date\":\"0\",\"show_modify_date\":\"0\",\"show_publish_date\":\"0\",\"show_item_navigation\":\"0\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"0\",\"show_print_icon\":\"0\",\"show_email_icon\":\"0\",\"show_hits\":\"0\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-home\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',25,26,1,'*',0),(102,'usermenu','Your Profile','your-profile','','your-profile','index.php?option=com_users&view=profile&layout=edit','component',1,1,1,25,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"0\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',11,12,0,'*',0),(103,'usermenu','Site Administrator','2013-11-16-23-26-41','','2013-11-16-23-26-41','administrator','url',1,1,1,0,0,'0000-00-00 00:00:00',0,6,'',7,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1}',17,18,0,'*',0),(104,'usermenu','Submit an Article','submit-an-article','','submit-an-article','index.php?option=com_content&view=form&layout=edit','component',1,1,1,22,0,'0000-00-00 00:00:00',0,3,' ',0,'{\"enable_category\":\"0\",\"catid\":\"2\",\"redirect_menuitem\":\"\",\"custom_cancel_redirect\":\"0\",\"cancel_redirect_menuitem\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"0\",\"page_heading\":\"\",\"pageclass_sfx\":\" submit-an-article-show-help\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',13,14,0,'*',0),(106,'usermenu','Template Settings','template-settings','','template-settings','index.php?option=com_config&view=templates&controller=config.display.templates','component',1,1,1,23,0,'0000-00-00 00:00:00',0,6,'',7,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"page_title\":\"\",\"show_page_heading\":0,\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',19,20,0,'*',0),(107,'usermenu','Site Settings','site-settings','','site-settings','index.php?option=com_config&view=config&controller=config.display.config','component',1,1,1,23,0,'0000-00-00 00:00:00',0,6,'',7,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"page_title\":\"\",\"show_page_heading\":0,\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',21,22,0,'*',0),(152,'mainmenu','Docs','documentation','','documentation','index.php?option=com_content&view=article&id=134','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-graduation-cap\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',63,132,0,'*',0),(229,'main','com_joomlaupdate','com-joomlaupdate','','com-joomlaupdate','index.php?option=com_joomlaupdate','component',1,1,1,28,0,'0000-00-00 00:00:00',0,1,'class:joomlaupdate',0,'{}',221,222,0,'',1),(234,'mainmenu','About','about','','about','','heading',1,1,1,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-info-circle\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',27,50,0,'*',0),(235,'mainmenu','What is Reactome ?','what-is-reactome','','about/what-is-reactome','index.php?option=com_content&view=article&id=11','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',28,29,0,'*',0),(236,'mainmenu','News','news','','about/news','index.php?option=com_content&view=category&layout=blog&id=26','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"layout_type\":\"blog\",\"show_category_heading_title_text\":\"\",\"show_category_title\":\"\",\"show_description\":\"\",\"show_description_image\":\"\",\"maxLevel\":\"\",\"show_empty_categories\":\"\",\"show_no_articles\":\"\",\"show_subcat_desc\":\"\",\"show_cat_num_articles\":\"\",\"show_cat_tags\":\"\",\"page_subheading\":\"\",\"num_leading_articles\":\"9999\",\"num_intro_articles\":\"1\",\"num_columns\":\"2\",\"num_links\":\"\",\"multi_column_order\":\"1\",\"show_subcategory_content\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"rdate\",\"order_date\":\"\",\"show_pagination\":\"0\",\"show_pagination_results\":\"\",\"show_featured\":\"\",\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"0\",\"absent_alias\":\"\"}',30,31,0,'*',0),(237,'mainmenu','Team','team','','about/team','index.php?option=com_content&view=article&id=13','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',32,33,0,'*',0),(238,'mainmenu','Tools','tools','','tools','','heading',1,1,1,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-cogs\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',133,152,0,'*',0),(239,'mainmenu','Pathway Browser','pathway-browser','','tools/pathway-browser','/PathwayBrowser','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1}',134,135,0,'*',0),(240,'mainmenu','Analyze Data','analyze-data','','tools/analyze-data','/PathwayBrowser/#TOOL=AT','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1}',136,137,0,'*',0),(241,'mainmenu','Community','community','','community','','heading',1,1,1,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-users\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',153,174,0,'*',0),(242,'mainmenu','Download','download-data','','download-data','index.php?option=com_content&view=article&id=18','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-download\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',175,178,0,'*',0),(244,'main','com_associations','multilingual-associations','','multilingual-associations','index.php?option=com_associations','component',1,1,1,34,0,'0000-00-00 00:00:00',0,1,'class:associations',0,'{}',223,224,0,'*',1),(245,'mainmenu','Species Comparison','species-comparison','','tools/species-comparison','/PathwayBrowser/#TOOL=AT','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1}',138,139,0,'*',0),(246,'mainmenu','ReactomeFIViz','reactome-fiviz','','tools/reactome-fiviz','index.php?option=com_content&view=article&id=44','component',1,238,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',144,145,0,'*',0),(247,'mainmenu','Advanced Data Search','advanced-search','','tools/advanced-search','/content/advanced','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',146,147,0,'*',0),(248,'mainmenu','Analysis Service','analysis-service-tool','','tools/analysis-service-tool','/AnalysisService','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',140,141,0,'*',0),(249,'mainmenu','Content Service','content-service','','tools/content-service','/ContentService','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1}',142,143,0,'*',0),(251,'mainmenu','Scientific Advisory Board','sab','','about/sab','index.php?option=com_content&view=article&id=14','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',34,35,0,'*',0),(252,'mainmenu','Other Reactomes','other-reactomes','','about/other-reactomes','index.php?option=com_content&view=article&id=15','component',0,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',36,37,0,'*',0),(253,'mainmenu','License Agreement','license','','about/license','index.php?option=com_content&view=article&id=16','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',44,45,0,'*',0),(254,'mainmenu','Disclaimer','disclaimer','','about/disclaimer','index.php?option=com_content&view=article&id=17','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',46,47,0,'*',0),(259,'mainmenu','Outreach','outreach','','community/outreach','index.php?option=com_content&view=article&id=22','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',156,157,0,'*',0),(260,'mainmenu','Events','events','','community/events','index.php?option=com_content&view=article&id=23','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',158,159,0,'*',0),(261,'mainmenu','Training','training','','community/training','index.php?option=com_content&view=article&id=25','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',160,161,0,'*',0),(262,'mainmenu','Publications','publications','','community/publications','index.php?option=com_content&view=article&id=24','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',162,163,0,'*',0),(263,'mainmenu','Partners','partners','','community/partners','index.php?option=com_content&view=article&id=26','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',164,165,0,'*',0),(264,'mainmenu','Papers Citing Us','citation-list','','community/citation-list','index.php?option=com_content&view=article&id=69','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',166,167,0,'*',0),(265,'mainmenu','Resources Guide','resources','','community/resources','index.php?option=com_content&view=article&id=27','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',168,169,0,'*',0),(266,'mainmenu','Icon Library','icon-lib','','community/icon-lib','index.php?option=com_content&view=featured','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"Community library of icons that are used in the Reactome Enhanced High Level Diagrams (EHLD). We would like to develop the EHLD library further as a community resource. If you use library elements, and design similar elements that are still missing, we would be very happy to incorporate them into the library, naming you as the author\",\"menu-anchor_css\":\"icons-library-mi\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',154,155,0,'*',0),(267,'mainmenu','User','user','','documentation/user','user/guide','url',0,152,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',80,81,0,'*',0),(268,'mainmenu','Developer\'s Zone','dev','','documentation/dev','index.php?option=com_content&view=article&id=64','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',82,111,0,'*',0),(269,'mainmenu','Data Model','data-model','','documentation/data-model','index.php?option=com_content&view=article&id=53','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',118,119,0,'*',0),(270,'mainmenu','Computationally inferred events','inferred-events','','documentation/inferred-events','index.php?option=com_content&view=article&id=68','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',120,121,0,'*',0),(271,'mainmenu','Linking to Us','linking-to-us','','documentation/linking-to-us','index.php?option=com_content&view=article&id=28','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',122,125,0,'*',0),(272,'mainmenu','Citing us','cite','','documentation/cite','index.php?option=com_content&view=article&id=29','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',126,127,0,'*',0),(277,'usermenu','Logout','logout','','logout','index.php?option=com_users&view=login&layout=logout&task=user.menulogout','component',1,1,1,25,0,'0000-00-00 00:00:00',0,2,' ',7,'{\"logout\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',23,24,0,'*',0),(279,'mainmenu','Account','reactome','','reactome','','heading',1,1,1,0,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-users\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',179,192,0,'*',0),(280,'mainmenu','Logout','userlogout','','reactome/userlogout','index.php?Itemid=','alias',1,279,2,0,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"aliasoptions\":\"277\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1}',190,191,0,'*',0),(281,'mainmenu','Publish an Article','publish-an-article','','reactome/publish-an-article','index.php?Itemid=','alias',1,279,2,0,0,'0000-00-00 00:00:00',0,2,' ',4,'{\"aliasoptions\":\"104\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',184,185,0,'*',0),(282,'mainmenu','Your Profile','user-profile','','reactome/user-profile','index.php?Itemid=','alias',1,279,2,0,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"aliasoptions\":\"102\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1}',180,181,0,'*',0),(284,'mainmenu','Staff','staff','This menu entry is visible only in ReactomeRelease. SyncTool unpublished when syncing to production','about/staff','index.php?option=com_users&view=login','component',1,234,2,25,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"loginredirectchoice\":\"1\",\"login_redirect_url\":\"\",\"login_redirect_menuitem\":\"101\",\"logindescription_show\":\"1\",\"login_description\":\"Note: Login is only available for Reactome Staff Members.\",\"login_image\":\"\",\"logoutredirectchoice\":\"1\",\"logout_redirect_url\":\"\",\"logout_redirect_menuitem\":\"101\",\"logoutdescription_show\":\"1\",\"logout_description\":\"\",\"logout_image\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"Reactome Staff Member Login\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"noindex, nofollow\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',48,49,0,'*',0),(288,'hidden-menu','Template Search Page','template-datacontent','','template-datacontent','index.php?option=com_content&view=featured','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"0\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"0\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"Reactome\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"index, follow\",\"secure\":-1,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',225,226,0,'*',0),(290,'mainmenu','Graph Database','graph-database','','documentation/dev/graph-database','index.php?option=com_content&view=article&id=45','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"0\",\"absent_alias\":\"\"}',83,86,0,'*',0),(291,'mainmenu','Teleconferences','ateleconferences','','reactome/ateleconferences','index.php?Itemid=','alias',0,279,2,0,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"aliasoptions\":\"398\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"1\",\"absent_alias\":\"0\"}',186,187,0,'*',0),(292,'mainmenu','Analysis Service ','analysis','','documentation/dev/analysis','index.php?option=com_content&view=article&id=47','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',87,88,0,'*',0),(293,'mainmenu','Content Service','content-service','','documentation/dev/content-service','index.php?option=com_content&view=article&id=48','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',89,92,0,'*',0),(294,'mainmenu','Pathways Overview','pathways-overview','','documentation/dev/pathways-overview','index.php?option=com_content&view=article&id=49','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',93,98,0,'*',0),(295,'mainmenu','Pathway Diagrams','diagram','','documentation/dev/diagram','index.php?option=com_content&view=article&id=50','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',99,106,0,'*',0),(296,'mainmenu','RESTFul API','restful-api','','documentation/dev/restful-api','index.php?option=com_content&view=article&id=51','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',107,108,0,'*',0),(297,'mainmenu','Simplified DB','simplified-db','','documentation/dev/simplified-db','index.php?option=com_content&view=article&id=52','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',109,110,0,'*',0),(299,'mainmenu','Tutorial: Extracting participating molecules using the Graph Database','extract-participating-molecules','','documentation/dev/graph-database/extract-participating-molecules','index.php?option=com_content&view=article&id=54','component',1,290,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"0\",\"absent_alias\":\"\"}',84,85,0,'*',0),(301,'mainmenu','Site map','site-map','Search for contents in the WebSite. It does not support data content search, like the one provided by SolR','tools/site-map','index.php?option=com_search&view=search&searchword=pathway','component',1,238,2,19,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"search_phrases\":\"\",\"search_areas\":\"\",\"show_date\":\"\",\"searchphrase\":\"0\",\"ordering\":\"alpha\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',150,151,0,'*',0),(302,'mainmenu','Icon Info','icon-info','','documentation/icon-info','index.php?option=com_content&view=article&id=55','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',112,117,0,'*',0),(303,'mainmenu','Userguide','userguide','user guide','documentation/userguide','index.php?option=com_content&view=article&id=39','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"0\",\"info_block_position\":\"\",\"info_block_show_title\":\"0\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',64,79,0,'*',0),(304,'mainmenu','Fireworks JS','js','','documentation/dev/pathways-overview/js','index.php?option=com_content&view=article&id=56','component',1,294,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',94,95,0,'*',0),(305,'mainmenu','Fireworks GWT','gwt','','documentation/dev/pathways-overview/gwt','index.php?option=com_content&view=article&id=57','component',1,294,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',96,97,0,'*',0),(306,'mainmenu','Diagram JS','js','','documentation/dev/diagram/js','index.php?option=com_content&view=article&id=58','component',1,295,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',100,101,0,'*',0),(307,'mainmenu','Diagram GWT','gwt','','documentation/dev/diagram/gwt','index.php?option=com_content&view=article&id=59','component',1,295,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',102,103,0,'*',0),(355,'mainmenu','Author/Reviewer Search','author-reviewer-search','','tools/author-reviewer-search','cgi-bin/author_contributions/','url',0,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',148,149,0,'*',0),(356,'mainmenu','Content','content','','content','','heading',1,1,1,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-pencil-square-o\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',51,62,0,'*',0),(357,'mainmenu','Data Schema','data-schema','','content/data-schema','/content/schema','url',1,356,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',56,57,0,'*',0),(358,'mainmenu','Table of Contents','table-of-contents','','content/table-of-contents','cgi-bin/toc?DB=current','url',1,356,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',52,53,0,'*',0),(359,'mainmenu','DOIs','dois','','content/dois','cgi-bin/doi_toc?DB=current','url',1,356,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',54,55,0,'*',0),(360,'mainmenu','Statistics','statistics','','about/statistics','index.php?option=com_content&view=article&id=66','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',40,41,0,'*',0),(361,'mainmenu','ORCID Integration Project','orcid','','content/orcid','index.php?option=com_content&view=article&id=67','component',1,356,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"1\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',58,59,0,'*',0),(362,'mainmenu','Editorial Calendar','editorial-calendar','','about/editorial-calendar','index.php?option=com_wrapper&view=wrapper','component',1,234,2,2,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"url\":\"https:\\/\\/docs.google.com\\/spreadsheets\\/d\\/1pWa6r5URoX5BtMc4_mRnvUZ4xKC6G8u_XZypid3JtQE\\/htmlembed?widget=true#gid=5\",\"scrolling\":\"auto\",\"width\":\"100%\",\"height\":\"500\",\"height_auto\":\"0\",\"add_scheme\":\"1\",\"frameborder\":\"1\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',38,39,0,'*',0),(371,'hidden-menu','Template Swagger Page','template-swagger','','template-swagger','index.php?option=com_content&view=featured','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"Reactome\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"index, follow\",\"secure\":-1,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',227,228,0,'*',0),(377,'mainmenu','External Identifiers','identifiers','','documentation/linking-to-us/identifiers','index.php?option=com_content&view=article&id=70','component',1,271,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',123,124,0,'*',0),(378,'mainmenu','Pathway Browser','pathway-browser','','documentation/userguide/pathway-browser','index.php?option=com_content&view=article&id=81','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',65,66,0,'*',0),(379,'mainmenu','Details Panel','details-panel','','documentation/userguide/details-panel','index.php?option=com_content&view=article&id=82','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',69,70,0,'*',0),(380,'mainmenu','How do I search ?','searching','','documentation/userguide/searching','index.php?option=com_content&view=article&id=83','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',67,68,0,'*',0),(381,'mainmenu','Analysis Tools','analysis','','documentation/userguide/analysis','index.php?option=com_content&view=article&id=84','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',71,72,0,'*',0),(382,'mainmenu','Diseases','diseases','','documentation/userguide/diseases','index.php?option=com_content&view=article&id=85','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',73,74,0,'*',0),(383,'hidden-menu','Template CGI','template-cgi','','template-cgi','index.php?option=com_content&view=featured','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"Reactome\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"noindex, nofollow\",\"secure\":-1,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',229,230,0,'*',0),(384,'mainmenu','Reactome Curator Tool 3','reactome-curator-tool','','download-data/reactome-curator-tool','index.php?option=com_content&view=article&id=63','component',1,242,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',176,177,0,'*',0),(397,'mainmenu','Our Logo','logo','','about/logo','index.php?option=com_content&view=article&id=91','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"0\",\"absent_alias\":\"\"}',42,43,0,'*',0),(398,'usermenu','Teleconferences','teleconferences','','teleconferences','index.php?option=com_content&view=article&id=46','component',0,1,1,22,0,'0000-00-00 00:00:00',0,3,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',15,16,0,'*',0),(405,'hidden-menu','Synchronization Tool','p2p','','p2p','index.php?option=com_content&view=featured','component',1,1,1,22,0,'0000-00-00 00:00:00',0,7,' ',24,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"noindex, nofollow\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',231,232,0,'*',0),(406,'mainmenu','Synchronization Tool','sync-tool','','reactome/sync-tool','index.php?option=com_content&view=featured','component',1,279,2,22,0,'0000-00-00 00:00:00',0,7,' ',0,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',188,189,0,'*',0),(407,'mainmenu','Diagram exporter','diagram-exporter','','documentation/dev/content-service/diagram-exporter','index.php?option=com_content&view=article&id=96','component',1,293,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',90,91,0,'*',0),(412,'mainmenu','Release','release','Hidden Menu for the Release Articles','tag/release','index.php?option=com_tags&view=tag&id[0]=7&types[0]=1','component',1,413,2,29,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_tag_title\":\"\",\"tag_list_show_tag_image\":\"\",\"tag_list_show_tag_description\":\"\",\"tag_list_image\":\"\",\"tag_list_description\":\"\",\"tag_list_orderby\":\"c.core_created_time\",\"tag_list_orderby_direction\":\"DESC\",\"tag_list_show_item_image\":\"\",\"tag_list_show_item_description\":\"\",\"tag_list_item_maximum_characters\":\"\",\"filter_field\":\"0\",\"show_pagination_limit\":\"0\",\"show_pagination\":\"0\",\"show_pagination_results\":\"0\",\"return_any_or_all\":\"\",\"include_children\":\"\",\"show_feed_link\":\"0\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"Release\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',234,235,0,'*',0),(413,'mainmenu','Tag','tag','','tag','','heading',1,1,1,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',233,236,0,'*',0),(422,'usermenu','Help','staff-help','','staff-help','index.php?option=com_content&view=article&id=99','component',1,1,1,22,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"noindex, nofollow\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',237,238,0,'*',0),(423,'mainmenu','Help','help','This is an Alias to a menu item in the Users Menu','reactome/help','index.php?Itemid=','alias',1,279,2,0,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"aliasoptions\":\"422\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',182,183,0,'*',0),(424,'mainmenu','Pathway Diagrams Specifications','pathway-diagram-specs','','documentation/dev/diagram/pathway-diagram-specs','index.php?option=com_content&view=article&id=100','component',1,295,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',104,105,0,'*',0),(471,'mainmenu','EHLD Specs & Guidelines','ehld-specs-guideline','','documentation/icon-info/ehld-specs-guideline','index.php?option=com_content&view=article&id=129','component',1,302,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',113,114,0,'*',0),(472,'mainmenu','Icon Library Guidelines','icons-guidelines','','documentation/icon-info/icons-guidelines','index.php?option=com_content&view=article&id=130','component',1,302,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',115,116,0,'*',0),(477,'mainmenu','User 2','user2','','documentation/user2','','heading',-2,152,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',128,131,0,'*',0),(478,'mainmenu','Guide2','guide2','','documentation/user2/guide2','index.php?Itemid=','alias',-2,477,3,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"aliasoptions\":\"303\",\"alias_redirect\":0,\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',129,130,0,'*',0),(479,'mainmenu','ReactomeFIViz','reactome-fiviz','','documentation/userguide/reactome-fiviz','index.php?option=com_content&view=article&id=44','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',75,76,0,'*',0),(488,'mainmenu','Claim your work','claim-your-work','','documentation/userguide/claim-your-work','index.php?option=com_content&view=article&id=139','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',77,78,0,'*',0),(493,'mainmenu','Collaboration','collaboration','','community/collaboration','index.php?option=com_content&view=article&id=140','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',170,173,0,'*',0),(494,'main','COM_EASYJOOMLABACKUP','com-easyjoomlabackup','','com-easyjoomlabackup','index.php?option=com_easyjoomlabackup','component',1,1,1,10208,0,'0000-00-00 00:00:00',0,1,'components/com_easyjoomlabackup/images/ejb-menu.png',0,'{}',239,240,0,'',1),(499,'mainmenu','Disease Overlay','disease-overlay','','content/disease-overlay','/content/disease','url',0,356,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',60,61,0,'*',0),(504,'mainmenu','FAQ for prospective reviewers and authors','faq-for-prospective-reviewers-and-authors','','community/collaboration/faq-for-prospective-reviewers-and-authors','index.php?option=com_content&view=article&id=143','component',1,493,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"0\",\"absent_alias\":\"\"}',171,172,0,'*',0),(505,'main','COM_JCE','com-jce','','com-jce','index.php?option=com_jce&view=cpanel','component',1,1,1,10193,0,'0000-00-00 00:00:00',0,1,'components/com_jce/media/img/menu/logo.png',0,'{}',241,248,0,'',1),(506,'main','COM_JCE_MENU_CPANEL','com-jce-menu-cpanel','','com-jce/com-jce-menu-cpanel','index.php?option=com_jce','component',1,505,2,10193,0,'0000-00-00 00:00:00',0,1,'class:component',0,'{}',242,243,0,'',1),(507,'main','COM_JCE_MENU_CONFIG','com-jce-menu-config','','com-jce/com-jce-menu-config','index.php?option=com_jce&view=config','component',1,505,2,10193,0,'0000-00-00 00:00:00',0,1,'class:component',0,'{}',244,245,0,'',1),(508,'main','COM_JCE_MENU_PROFILES','com-jce-menu-profiles','','com-jce/com-jce-menu-profiles','index.php?option=com_jce&view=profiles','component',1,505,2,10193,0,'0000-00-00 00:00:00',0,1,'class:component',0,'{}',246,247,0,'',1);
/*!40000 ALTER TABLE `rlp_menu` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_menu_types`
--

DROP TABLE IF EXISTS `rlp_menu_types`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_menu_types` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `asset_id` int(10) unsigned NOT NULL DEFAULT '0',
  `menutype` varchar(24) COLLATE utf8mb4_unicode_ci NOT NULL,
  `title` varchar(48) COLLATE utf8mb4_unicode_ci NOT NULL,
  `description` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `client_id` int(11) NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`),
  UNIQUE KEY `idx_menutype` (`menutype`)
) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_menu_types`
--

LOCK TABLES `rlp_menu_types` WRITE;
/*!40000 ALTER TABLE `rlp_menu_types` DISABLE KEYS */;
INSERT INTO `rlp_menu_types` VALUES (1,552,'mainmenu','Main Menu','Landing Page menu',0),(2,568,'usermenu','User Menu','A Menu for logged-in Users',0),(4,599,'hidden-menu','Hidden Menu','Menu to hold hidden options',0);
/*!40000 ALTER TABLE `rlp_menu_types` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_messages`
--

DROP TABLE IF EXISTS `rlp_messages`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_messages` (
  `message_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `user_id_from` int(10) unsigned NOT NULL DEFAULT '0',
  `user_id_to` int(10) unsigned NOT NULL DEFAULT '0',
  `folder_id` tinyint(3) unsigned NOT NULL DEFAULT '0',
  `date_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `state` tinyint(1) NOT NULL DEFAULT '0',
  `priority` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `subject` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `message` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`message_id`),
  KEY `useridto_state` (`user_id_to`,`state`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_messages`
--

LOCK TABLES `rlp_messages` WRITE;
/*!40000 ALTER TABLE `rlp_messages` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_messages` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_messages_cfg`
--

DROP TABLE IF EXISTS `rlp_messages_cfg`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_messages_cfg` (
  `user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `cfg_name` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `cfg_value` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  UNIQUE KEY `idx_user_var_name` (`user_id`,`cfg_name`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_messages_cfg`
--

LOCK TABLES `rlp_messages_cfg` WRITE;
/*!40000 ALTER TABLE `rlp_messages_cfg` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_messages_cfg` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_modules`
--

DROP TABLE IF EXISTS `rlp_modules`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_modules` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `asset_id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'FK to the #__assets table.',
  `title` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `note` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `content` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `ordering` int(11) NOT NULL DEFAULT '0',
  `position` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `checked_out` int(10) unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `published` tinyint(1) NOT NULL DEFAULT '0',
  `module` varchar(50) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `access` int(10) unsigned NOT NULL DEFAULT '0',
  `showtitle` tinyint(3) unsigned NOT NULL DEFAULT '1',
  `params` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `client_id` tinyint(4) NOT NULL DEFAULT '0',
  `language` char(7) COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`id`),
  KEY `published` (`published`,`access`),
  KEY `newsfeeds` (`module`,`published`),
  KEY `idx_language` (`language`)
) ENGINE=InnoDB AUTO_INCREMENT=587 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_modules`
--

LOCK TABLES `rlp_modules` WRITE;
/*!40000 ALTER TABLE `rlp_modules` DISABLE KEYS */;
INSERT INTO `rlp_modules` VALUES (1,39,'Main Menu','','',1,'nav',158,'2019-09-11 14:46:04','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_menu',1,0,'{\"menutype\":\"mainmenu\",\"base\":\"\",\"startLevel\":\"1\",\"endLevel\":\"0\",\"showAllChildren\":\"1\",\"tag_id\":\"\",\"class_sfx\":\"\",\"window_open\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"itemid\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(2,40,'Login','','',1,'login',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_login',1,1,'',1,'*'),(3,41,'Popular Articles','','',3,'cpanel',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_popular',3,1,'{\"count\":\"5\",\"catid\":\"\",\"user_id\":\"0\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\",\"automatic_title\":\"1\"}',1,'*'),(4,42,'Recently Added Articles','','',4,'cpanel',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_latest',3,1,'{\"count\":\"5\",\"ordering\":\"c_dsc\",\"catid\":\"\",\"user_id\":\"0\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\",\"automatic_title\":\"1\"}',1,'*'),(8,43,'Toolbar','','',1,'toolbar',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_toolbar',3,1,'',1,'*'),(9,44,'Quick Icons','','',1,'icon',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_quickicon',3,1,'',1,'*'),(10,45,'Logged-in Users','','',2,'cpanel',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_logged',3,1,'{\"count\":\"5\",\"name\":\"1\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\",\"automatic_title\":\"1\"}',1,'*'),(12,46,'Admin Menu','','',1,'menu',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_menu',3,1,'{\"layout\":\"\",\"moduleclass_sfx\":\"\",\"shownew\":\"1\",\"showhelp\":\"1\",\"cache\":\"0\"}',1,'*'),(13,47,'Admin Submenu','','',1,'submenu',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_submenu',3,1,'',1,'*'),(14,48,'User Status','','',2,'status',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_status',3,1,'',1,'*'),(15,49,'Title','','',1,'title',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_title',3,1,'',1,'*'),(17,51,'Breadcrumb','','',1,'breadcrumbs',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_breadcrumbs',1,0,'{\"showHere\":0,\"showHome\":1,\"homeText\":\"Home\",\"showLast\":1,\"separator\":\">\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" hidden-phone\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"itemid\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(79,52,'Multilanguage status','','',1,'status',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',0,'mod_multilangstatus',3,1,'{\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\"}',1,'*'),(86,53,'Joomla Version','','',1,'footer',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_version',3,1,'{\"format\":\"short\",\"product\":\"1\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\"}',1,'*'),(88,55,'Site Information','','',3,'cpanel',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_stats_admin',3,1,'{\"serverinfo\":\"1\",\"siteinfo\":\"1\",\"counter\":\"0\",\"increase\":\"0\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',1,'*'),(89,56,'Release News','','',0,'postinstall',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_feed',1,1,'{\"rssurl\":\"http:\\/\\/www.joomla.org\\/announcements\\/release-news.feed\",\"rssrtl\":\"0\",\"rsstitle\":\"1\",\"rssdesc\":\"1\",\"rssimage\":\"1\",\"rssitems\":\"3\",\"rssitemdesc\":\"1\",\"word_count\":\"0\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',1,'*'),(475,483,'FavGlyph - Home Page','Glyphs for PathwayBrowser, Analyze Data, ReactomeFIViz and Documentation','',1,'lead1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"uppercase\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-sitemap\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"C8C8C8\",\"icon_link1\":\"\\/PathwayBrowser\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Pathway Browser\",\"title_color1\":\"444444\",\"title_link1\":\"\\/PathwayBrowser\",\"title_link_target1\":\"self\",\"description_text1\":\"Visualize and interact with Reactome biological pathways\",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-bar-chart\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"\\/PathwayBrowser\\/#TOOL=AT\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Analyze Data\",\"title_color2\":\"444444\",\"title_link2\":\"\\/PathwayBrowser\\/#TOOL=AT\",\"title_link_target2\":\"self\",\"description_text2\":\"Merges pathway identifier mapping, <br\\/> over-representation, and expression analysis\",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-share-alt-square\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"index.php?option=com_content&view=article&id=44\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"ReactomeFIViz\",\"title_color3\":\"444444\",\"title_link3\":\"index.php?option=com_content&view=article&id=44\",\"title_link_target3\":\"self\",\"description_text3\":\"Designed to find pathways and network patterns related to cancer and other types of diseases\",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-file-text-o\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"documentation\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Documentation\",\"title_color4\":\"444444\",\"title_link4\":\"documentation\",\"title_link_target4\":\"self\",\"description_text4\":\"Information to browse the database and use its principal tools for data analysis\",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(479,487,'FavSocial - Reactome Social Medias','Module to create social media icons. Facebook, Twitter and YouTube','',1,'copyright1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favsocial',1,0,'{\"module_align\":\"favsocial-center\",\"show_icon1\":\"1\",\"icon_name1\":\"fa-twitter\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"AAAAAA\",\"icon_font_size1\":\"18px\",\"icon_link1\":\"https:\\/\\/twitter.com\\/intent\\/user?screen_name=reactome\",\"icon_link_target1\":\"blank\",\"icon_padding1\":\"0.2em\",\"icon_border_radius1\":\"4px\",\"show_icon2\":\"1\",\"icon_name2\":\"fa-facebook\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"AAAAAA\",\"icon_font_size2\":\"18px\",\"icon_link2\":\"https:\\/\\/www.facebook.com\\/groups\\/244908260192\\/\",\"icon_link_target2\":\"blank\",\"icon_padding2\":\"0.2em\",\"icon_border_radius2\":\"4px\",\"show_icon3\":\"1\",\"icon_name3\":\"fa-youtube\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"AAAAAA\",\"icon_font_size3\":\"18px\",\"icon_link3\":\"https:\\/\\/www.youtube.com\\/channel\\/UCXKPDGH34lyyMAjPUXubykw\",\"icon_link_target3\":\"blank\",\"icon_padding3\":\"0.2em\",\"icon_border_radius3\":\"4px\",\"show_icon4\":\"0\",\"icon_name4\":\"\",\"icon_color4\":\"\",\"icon_bg_color4\":\"\",\"icon_font_size4\":\"\",\"icon_link4\":\"\",\"icon_link_target4\":\"blank\",\"icon_padding4\":\"\",\"icon_border_radius4\":\"\",\"show_icon5\":\"0\",\"icon_name5\":\"\",\"icon_color5\":\"\",\"icon_bg_color5\":\"\",\"icon_font_size5\":\"\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_padding5\":\"\",\"icon_border_radius5\":\"\",\"show_icon6\":\"0\",\"icon_name6\":\"\",\"icon_color6\":\"\",\"icon_bg_color6\":\"\",\"icon_font_size6\":\"\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_padding6\":\"\",\"icon_border_radius6\":\"\",\"show_icon7\":\"0\",\"icon_name7\":\"\",\"icon_color7\":\"\",\"icon_bg_color7\":\"\",\"icon_font_size7\":\"\",\"icon_link7\":\"\",\"icon_link_target7\":\"blank\",\"icon_padding7\":\"\",\"icon_border_radius7\":\"\",\"show_icon8\":\"0\",\"icon_name8\":\"\",\"icon_color8\":\"\",\"icon_bg_color8\":\"\",\"icon_font_size8\":\"\",\"icon_link8\":\"\",\"icon_link_target8\":\"blank\",\"icon_padding8\":\"\",\"icon_border_radius8\":\"\",\"show_icon9\":\"0\",\"icon_name9\":\"\",\"icon_color9\":\"\",\"icon_bg_color9\":\"\",\"icon_font_size9\":\"\",\"icon_link9\":\"\",\"icon_link_target9\":\"blank\",\"icon_padding9\":\"\",\"icon_border_radius9\":\"\",\"show_icon10\":\"0\",\"icon_name10\":\"\",\"icon_color10\":\"\",\"icon_bg_color10\":\"\",\"icon_font_size10\":\"\",\"icon_link10\":\"\",\"icon_link_target10\":\"blank\",\"icon_padding10\":\"\",\"icon_border_radius10\":\"\",\"moduleclass_sfx\":\"pull-center favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(484,505,'Use Reactome Graph Database in your Project','Download Reactome Graph Database','<div id=\"fav-download\">\r\n<p id=\"fav-download-message\"><span style=\"color: #ffffff;\">USE REACTOME GRAPH DATABASE IN YOUR PROJECT</span></p>\r\n<p id=\"fav-download-btn\"><a href=\"dev/graph-database\" class=\"favth-btn\">LEARN MORE</a></p>\r\n</div>',1,'promo1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(496,518,'Tweets','','',1,'prime2',0,'0000-00-00 00:00:00','2017-04-19 13:41:08','0000-00-00 00:00:00',1,'mod_pro_twitter_widget',1,1,'{\"twitter_id\":\"344458797739294721\",\"twitter_name\":\"reactome\",\"width\":\"600\",\"height\":\"496\",\"scrollbar\":\"false\",\"color_scheme\":\"1\",\"backgroundColor\":\"#58c3e5\",\"header\":\"false\",\"footer\":\"true\",\"border\":\"false\",\"tranparent\":\"true\",\"backgroundColors\":\"#292f33\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-clear favth-shadow tweet-widget\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-twitter\",\"style\":\"0\"}',0,'*'),(499,521,'Find Reactions, Proteins and Pathways','SolR Search input text. Always visible','<h3 class=\"visible-desktop\">Find Reactions, Proteins and Pathways</h3>\r\n<div class=\"nav-collapse-search search\"><form action=\"/content/query\" method=\"get\" id=\"search_form\" class=\"form-inline\"><label for=\"local-searchbox\" class=\"element-invisible\">Search ...</label> <input maxlength=\"200\" name=\"q\" size=\"10\" type=\"search\" id=\"local-searchbox\" class=\"inputbox search-query\" autofocus=\"\" placeholder=\"e.g. O95631, NTN1, signaling by EGFR, glucose\" /> <input name=\"species\" type=\"hidden\" value=\"Homo sapiens\" /> <input name=\"species\" type=\"hidden\" value=\"Entries without species\" /> <input name=\"cluster\" type=\"hidden\" value=\"true\" /> <button class=\"button btn btn-primary btn-info\">Go!</button></form></div>',1,'slide',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" text-center main-search\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-search text-center\",\"style\":\"0\"}',0,'*'),(500,522,'Notice','Warning this is Reactome Dev / Release...','<div class=\"hidden-phone\">\r\n<p class=\"rlp-noticemsg\" style=\"margin-top: 5px;\"><i class=\"fa fa-exclamation-triangle fa-1\"></i><span style=\"font-size: 8pt;\">THIS SITE IS USED FOR SOFTWARE DEVELOPMENT AND TESTING&nbsp;</span><br /> <span style=\"font-size: 8pt;\">IT IS NOT STABLE, IS LINKED TO AN INCOMPLETE DATA SET, AND IS NOT MONITORED FOR PERFORMANCE. WE STRONGLY RECOMMEND THE USE OF OUR&nbsp;<a href=\"https://reactome.org/\">PUBLIC SITE</a></span></p>\r\n</div>\r\n<div class=\"hidden-tablet hidden-desktop\">\r\n<p class=\"rlp-noticemsg\"><i class=\"fa fa-exclamation-triangle fa-1\"></i><span style=\"font-size: 8pt;\">THIS SITE IS USED FOR SOFTWARE DEVELOPMENT AND TESTING&nbsp;</span><span style=\"font-size: 8pt;\">WE STRONGLY RECOMMEND THE USE OF OUR&nbsp;<a href=\"https://reactome.org/\">PUBLIC SITE</a></span></p>\r\n</div>',1,'notice',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(505,539,'Why Reactome','Short description for Landing Page. Be careful when adding text here. It may misalign the Twitter Widget. To fix, adjust the .twitter-timeline height in custom.css','<p style=\"text-align: left;\">Reactome is a free, open-source, curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology and education.&nbsp;</p>\r\n<p class=\"text-center\" style=\"font-size: 12px;\">If you use Reactome in Asia, we suggest using our Chinese mirror site at <a href=\"http://reactome.ncpsb.org/\">reactome.ncpsb.org</a>.</p>\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 pull-center  favth-hidden-sm favth-hidden-xs\" style=\"background-color: ghostwhite; margin: 0; padding: 10px; border-radius: 8px; border: 2px solid #58C3E5;\">\r\n<div class=\"favth-col-lg-3 favth-col-md-3 favth-col-sm-1 text-center\"><a href=\"http://www.ebi.ac.uk/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 34px; margin-top: 7px;\" src=\"images/sponsor/embl_ebi_logo.png\" alt=\"European Bioinformatics Institute (EMBL-EBI)\" /></a></div>\r\n<div class=\"favth-col-lg-3 favth-col-md-3 favth-col-sm-1 text-right\" style=\"padding-right: 0;\"><a href=\"https://med.nyu.edu/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px; padding-right: 0;\" src=\"images/sponsor/nyul_health_logo.png\" alt=\"NYU Langone Health\" /></a></div>\r\n<div class=\"favth-col-lg-3 favth-col-md-3 favth-col-sm-1 text-center\"><a href=\"http://www.ohsu.edu/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px;\" src=\"images/sponsor/ohsu_logo.png\" alt=\"Oregon Health &amp; Science University\" /></a></div>\r\n<div class=\"favth-col-lg-3 favth-col-md-3 favth-col-sm-1 text-center\"><a href=\"https://oicr.on.ca/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px;\" src=\"images/sponsor/oicr_logo.png\" alt=\"Ontario Institute for Cancer Research\" /></a></div>\r\n</div>\r\n<p style=\"text-align: center; line-height: 13px; clear: both; padding-top: 10px;\"><span style=\"font-size: 8pt;\">The development of Reactome is supported by grants from the US National Institutes of Health (P41 HG003751 and 1U54GM114833-01), CFREF Medicine by Design, and the European Molecular Biology Laboratory.</span></p>',1,'prime1',0,'0000-00-00 00:00:00','2017-10-03 13:44:12','0000-00-00 00:00:00',1,'mod_custom',1,1,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-clear favth-shadow\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-search\",\"style\":\"0\"}',0,'*'),(508,544,'Latest News','','',1,'prime1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_articles_category',1,1,'{\"mode\":\"normal\",\"show_on_article_page\":0,\"count\":5,\"show_front\":\"hide\",\"category_filtering_type\":1,\"catid\":[26],\"show_child_category_articles\":0,\"levels\":1,\"author_filtering_type\":1,\"author_alias_filtering_type\":1,\"excluded_articles\":\"\",\"date_filtering\":\"off\",\"date_field\":\"a.created\",\"start_date_range\":\"\",\"end_date_range\":\"\",\"relative_date\":30,\"article_ordering\":\"publish_up\",\"article_ordering_direction\":\"DESC\",\"article_grouping\":\"none\",\"month_year_format\":\"F Y\",\"article_grouping_direction\":\"ksort\",\"link_titles\":1,\"show_date\":0,\"show_date_field\":\"created\",\"show_date_format\":\"Y-m-d H:i:s\",\"show_category\":0,\"show_hits\":0,\"show_author\":0,\"show_introtext\":0,\"introtext_limit\":100,\"show_readmore\":0,\"show_readmore_title\":0,\"readmore_limit\":15,\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-clear favth-shadow latest-news\",\"owncache\":1,\"cache_time\":900,\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-newspaper-o\",\"style\":\"0\"}',0,'*'),(518,571,'Maintop Blank Space','Blank Space between nav and main-content','',1,'maintop3',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(519,572,'Find Reactions, Proteins and Pathways - Others','SolR Search input text. Short version without description. To be used in all other pages.','<div class=\"search\"><form action=\"/content/query\" method=\"get\" class=\"clean-form form-inline\"><label for=\"local-searchbox\" class=\"element-invisible\">Search ...</label> <input maxlength=\"200\" name=\"q\" size=\"10\" type=\"search\" id=\"local-searchbox\" class=\"inputbox search-query alt-searchbox\" autofocus=\"\" placeholder=\"e.g. O95631, NTN1, signaling by EGFR, glucose\" /> <input name=\"species\" type=\"hidden\" value=\"Homo sapiens\" /> <input name=\"species\" type=\"hidden\" value=\"Entries without species\" /> <input name=\"cluster\" type=\"hidden\" value=\"true\" /> <button class=\"button btn btn-primary btn-info\">Go!</button></form></div>',1,'slide',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" text-center alt-search\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-search text-center\",\"style\":\"0\"}',0,'*'),(524,591,'Articles Sidebar Menu','','',1,'sidebar1',0,'0000-00-00 00:00:00','2017-04-26 13:39:34','0000-00-00 00:00:00',1,'mod_menu',1,0,'{\"menutype\":\"mainmenu\",\"base\":\"\",\"startLevel\":2,\"endLevel\":2,\"showAllChildren\":1,\"tag_id\":\"\",\"class_sfx\":\"favth-side\",\"window_open\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"itemid\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(525,592,'Icon Library','','',1,'icon-lib',0,'0000-00-00 00:00:00','2017-05-17 11:12:38','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(526,595,'[TEMPLATE] data-content / swagger placeholder','Div placeholder for template-page. This is hidden from the normal website','<!-- template-placeholder -->',1,'lead1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":0,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(527,596,'[TEMPLATE] search placeholder','Search Form Placeholder, To be used as template in the DataContent Project. This is hidden from the normal website','<!--SearchForm-->\r\n<p>&nbsp;</p>\r\n<!--/SearchForm-->',1,'slide',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" text-center alt-search\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-search text-center\",\"style\":\"0\"}',0,'*'),(533,603,'Footer','','',1,'footer1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_menu',1,0,'{\"menutype\":\"mainmenu\",\"base\":\"101\",\"startLevel\":\"1\",\"endLevel\":\"2\",\"showAllChildren\":\"1\",\"tag_id\":\"\",\"class_sfx\":\" footer-menu\",\"window_open\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-visible-md favth-visible-lg favth-visible-sm\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"itemid\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(540,615,'Developers Zone - Row 1','','',1,'dev-guide-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"28px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"40%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-bar-chart\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"CCCCCC\",\"icon_link1\":\"index.php?option=com_content&view=article&id=47\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4em\",\"title_text1\":\"Analysis Service\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=47\",\"title_link_target1\":\"self\",\"description_text1\":\"Use the Analysis Service to analyse your data against Reactome\\u2019s content\",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"40%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-code\",\"icon_color2\":\"FF5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"CCCCCC\",\"icon_link2\":\"index.php?option=com_content&view=article&id=48\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4em\",\"title_text2\":\"Content Service\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=48\",\"title_link_target2\":\"self\",\"description_text2\":\"Use the Content Service to access all our knowledgebase content from your client\",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"40%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-database\",\"icon_color3\":\"8BC34A\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"CCCCCC\",\"icon_link3\":\"index.php?option=com_content&view=article&id=45\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4em\",\"title_text3\":\"Graph Database\",\"title_color3\":\"444444\",\"title_link3\":\"index.php?option=com_content&view=article&id=45\",\"title_link_target3\":\"self\",\"description_text3\":\"Access to the Reactome knowledgebase content as an interconnected graph database\",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-lightbulb-o\",\"icon_color4\":\"FFC107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"CCCCCC\",\"icon_link4\":\"\",\"icon_link_target4\":\"blank\",\"icon_font_size4\":\"4em\",\"title_text4\":\"<<<< READ DESCRIPTION >>>>\",\"title_color4\":\"444444\",\"title_link4\":\"\",\"title_link_target4\":\"blank\",\"description_text4\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4em\",\"title_text5\":\"<<< READ DESCRIPTION >>>\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"4em\",\"title_text6\":\"<<< READ DESCRIPTION >>>\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"-sfx favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(544,627,'Documentation','','',1,'userguide-cp-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favpromote',1,1,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_padding\":\"10px 20px\",\"title_font_size\":\"18px\",\"title_line_height\":\"1.4em\",\"title_text_align\":\"left\",\"title_icon_font_size\":\"18px\",\"title_icon_vertical_align\":\"baseline\",\"description_text_font_size\":\"14px\",\"description_text_line_height\":\"1.5em\",\"description_text_align\":\"justify\",\"show_column1\":\"1\",\"column_border_color1\":\"DDDDDD\",\"column_border_radius1\":\"4px\",\"upload_image1\":\"images\\/docs\\/for_users_110x110.png\",\"image_link1\":\"index.php?option=com_content&view=article&id=39\",\"image_target1\":\"self\",\"image_alt1\":\"\",\"title_text1\":\"For Users\",\"title_color1\":\"FFFFFF\",\"title_bg_color1\":\"58C3E5\",\"title_link1\":\"index.php?option=com_content&view=article&id=39\",\"title_link_target1\":\"self\",\"title_icon1\":\"fa fa-users\",\"description_text1\":\"Take the most out of our tools and data analysis. All you need is our User Guide.\",\"description_text_color1\":\"444444\",\"show_column2\":\"1\",\"column_border_color2\":\"DDDDDD\",\"column_border_radius2\":\"4px\",\"upload_image2\":\"images\\/docs\\/for_developers_110x110.png\",\"image_link2\":\"dev\",\"image_target2\":\"self\",\"image_alt2\":\"\",\"title_text2\":\"For Developers\",\"title_color2\":\"FFFFFF\",\"title_bg_color2\":\"58C3E5\",\"title_link2\":\"dev\",\"title_link_target2\":\"self\",\"title_icon2\":\"fa fa-code\",\"description_text2\":\"Are you interested in programatically querying our data or integrating our Widgets ?\",\"description_text_color2\":\"444444\",\"show_column3\":\"1\",\"column_border_color3\":\"DDDDDD\",\"column_border_radius3\":\"4px\",\"upload_image3\":\"images\\/docs\\/citing_us_110x110.png\",\"image_link3\":\"cite\",\"image_target3\":\"self\",\"image_alt3\":\"\",\"title_text3\":\"Citing us\",\"title_color3\":\"FFFFFF\",\"title_bg_color3\":\"58C3E5\",\"title_link3\":\"cite\",\"title_link_target3\":\"self\",\"title_icon3\":\"fa fa-pencil\",\"description_text3\":\"Have our data been useful in your research or experiment ? Please, remember to cite us!\",\"description_text_color3\":\"444444\",\"show_column4\":\"0\",\"column_border_color4\":\"DDDDDD\",\"column_border_radius4\":\"4px\",\"upload_image4\":\"\",\"image_link4\":\"\",\"image_target4\":\"blank\",\"image_alt4\":\"\",\"title_text4\":\"Title Text\",\"title_color4\":\"FFFFFF\",\"title_bg_color4\":\"D63E52\",\"title_link4\":\"\",\"title_link_target4\":\"blank\",\"title_icon4\":\"fa-magic\",\"description_text4\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color4\":\"444444\",\"show_column5\":\"0\",\"column_border_color5\":\"DDDDDD\",\"column_border_radius5\":\"4px\",\"upload_image5\":\"\",\"image_link5\":\"\",\"image_target5\":\"blank\",\"image_alt5\":\"\",\"title_text5\":\"Title Text\",\"title_color5\":\"FFFFFF\",\"title_bg_color5\":\"D63E52\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"title_icon5\":\"fa-quote-right\",\"description_text5\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color5\":\"444444\",\"show_column6\":\"0\",\"column_border_color6\":\"DDDDDD\",\"column_border_radius6\":\"4px\",\"upload_image6\":\"\",\"image_link6\":\"\",\"image_target6\":\"blank\",\"image_alt6\":\"\",\"title_text6\":\"Title Text\",\"title_color6\":\"FFFFFF\",\"title_bg_color6\":\"D63E52\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"title_icon6\":\"fa-recycle\",\"description_text6\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"System-none\"}',0,'*'),(551,647,'Partners','','',1,'block1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favpromote4col',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_padding\":\"10px 20px\",\"title_font_size\":\"15px\",\"title_line_height\":\"1.4em\",\"title_text_align\":\"left\",\"title_icon_font_size\":\"15px\",\"title_icon_vertical_align\":\"baseline\",\"description_text_font_size\":\"14px\",\"description_text_line_height\":\"1.5em\",\"description_text_align\":\"justify\",\"order1\":\"1\",\"show_column1\":\"1\",\"column_border_color1\":\"DDDDDD\",\"column_border_radius1\":\"4px\",\"upload_image1\":\"images\\/partners\\/openTargets_logo.png\",\"image_link1\":\"http:\\/\\/www.targetvalidation.org\\/target\\/ENSG00000157764\",\"image_target1\":\"blank\",\"image_alt1\":\"\",\"title_text1\":\"Open Targets\",\"title_color1\":\"FFFFFF\",\"title_bg_color1\":\"58C3E5\",\"title_link1\":\"http:\\/\\/www.targetvalidation.org\\/target\\/ENSG00000157764\",\"title_link_target1\":\"blank\",\"title_icon1\":\"fa-empty\",\"description_text1\":\"For biomedical researchers who need to identify a biological target for a new therapy, Open Targets is a public-private initiative to generate evidence on the validity of therapeutic targets based on genome-scale experiments and analysis.\",\"widget_text1\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color1\":\"444444\",\"order2\":\"2\",\"show_column2\":\"1\",\"column_border_color2\":\"DDDDDD\",\"column_border_radius2\":\"4px\",\"upload_image2\":\"images\\/partners\\/The_Human_Protein_Atlas.png\",\"image_link2\":\"http:\\/\\/www.proteinatlas.org\\/ENSG00000132155-RAF1\\/cell\",\"image_target2\":\"blank\",\"image_alt2\":\"\",\"title_text2\":\"The Human Protein Atlas\",\"title_color2\":\"FFFFFF\",\"title_bg_color2\":\"58C3E5\",\"title_link2\":\"http:\\/\\/www.proteinatlas.org\\/ENSG00000132155-RAF1\\/cell\",\"title_link_target2\":\"blank\",\"title_icon2\":\"fa-empty\",\"description_text2\":\"Contains information for a large majority of all human protein-coding genes regarding the expression and localization of the corresponding proteins based on both RNA and protein data.\",\"widget_text2\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color2\":\"444444\",\"order3\":\"5\",\"show_column3\":\"1\",\"column_border_color3\":\"DDDDDD\",\"column_border_radius3\":\"4px\",\"upload_image3\":\"images\\/partners\\/complexportal.png\",\"image_link3\":\"https:\\/\\/www.ebi.ac.uk\\/complexportal\\/complex\\/EBI-9008420\",\"image_target3\":\"blank\",\"image_alt3\":\"\",\"title_text3\":\"Complex Portal\",\"title_color3\":\"FFFFFF\",\"title_bg_color3\":\"58C3E5\",\"title_link3\":\"https:\\/\\/www.ebi.ac.uk\\/complexportal\\/complex\\/EBI-9008420\",\"title_link_target3\":\"blank\",\"title_icon3\":\"fa-empty\",\"description_text3\":\"The Complex Portal is a manually curated, encyclopaedic resource of macromolecular complexes from a number of key model organisms. The majority of complexes are made up of proteins but may also include nucleic acids or small molecules.\",\"widget_text3\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Content Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color3\":\"444444\",\"order4\":\"4\",\"show_column4\":\"1\",\"column_border_color4\":\"DDDDDD\",\"column_border_radius4\":\"4px\",\"upload_image4\":\"images\\/partners\\/blueprint_logo.png\",\"image_link4\":\"http:\\/\\/dcc.blueprint-epigenome.eu\\/#\\/experiments\\/ERX641310\",\"image_target4\":\"blank\",\"image_alt4\":\"\",\"title_text4\":\"BLUEPRINT\",\"title_color4\":\"FFFFFF\",\"title_bg_color4\":\"58C3E5\",\"title_link4\":\"http:\\/\\/dcc.blueprint-epigenome.eu\\/#\\/experiments\\/ERX641310\",\"title_link_target4\":\"blank\",\"title_icon4\":\"fa-empty\",\"description_text4\":\"The BLUEPRINT consortium has been formed with the aim to further the understanding of how genes are activated or repressed in both healthy and diseased human cells.\\r\\n\",\"widget_text4\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color4\":\"444444\",\"order5\":\"6\",\"show_column5\":\"1\",\"column_border_color5\":\"DDDDDD\",\"column_border_radius5\":\"4px\",\"upload_image5\":\"images\\/partners\\/ChEBI_logo.png\",\"image_link5\":\"http:\\/\\/www.ebi.ac.uk\\/chebi\\/pathway.do?chebiId=CHEBI:11230\",\"image_target5\":\"blank\",\"image_alt5\":\"\",\"title_text5\":\"ChEBI\",\"title_color5\":\"FFFFFF\",\"title_bg_color5\":\"58C3E5\",\"title_link5\":\"http:\\/\\/www.ebi.ac.uk\\/chebi\\/pathway.do?chebiId=CHEBI:11230\",\"title_link_target5\":\"blank\",\"title_icon5\":\"fa-empty\",\"description_text5\":\"Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on \\u2018small\\u2019 chemical compounds. The term \\u2018molecular entity\\u2019 refers to any constitutionally or isotopically distinct atom, molecule, ion, ion pair, radical, radical ion, complex, conformer, etc., identifiable as a separately distinguishable entity.\",\"widget_text5\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color5\":\"444444\",\"order6\":\"8\",\"show_column6\":\"1\",\"column_border_color6\":\"DDDDDD\",\"column_border_radius6\":\"4px\",\"upload_image6\":\"images\\/partners\\/pride_logo.png\",\"image_link6\":\"http:\\/\\/www.ebi.ac.uk\\/pride\\/archive\\/projects\\/PXD000764\",\"image_target6\":\"blank\",\"image_alt6\":\"\",\"title_text6\":\"PRIDE\",\"title_color6\":\"FFFFFF\",\"title_bg_color6\":\"58C3E5\",\"title_link6\":\"http:\\/\\/www.ebi.ac.uk\\/pride\\/archive\\/projects\\/PXD000764\",\"title_link_target6\":\"blank\",\"title_icon6\":\"fa-empty\",\"description_text6\":\"PRoteomics IDEntifications (PRIDE) database is a centralized, standards compliant, public data repository for proteomics data, including protein and peptide identifications, post-translational modifications and supporting spectral evidence.\",\"widget_text6\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color6\":\"444444\",\"order7\":\"9\",\"show_column7\":\"1\",\"column_border_color7\":\"DDDDDD\",\"column_border_radius7\":\"4px\",\"upload_image7\":\"images\\/partners\\/logo.png\",\"image_link7\":\"http:\\/\\/iomics.ugent.be\\/tabloidproteome\",\"image_target7\":\"blank\",\"image_alt7\":\"\",\"title_text7\":\"Tabloid Proteome\",\"title_color7\":\"FFFFFF\",\"title_bg_color7\":\"2F9EC2\",\"title_link7\":\"http:\\/\\/iomics.ugent.be\\/tabloidproteome\",\"title_link_target7\":\"blank\",\"title_icon7\":\"fa-empty\",\"description_text7\":\"Tabloid Proteome is a database of protein association network generated using publicly available mass spectrometry based experiments in PRIDE. These associations represent a broad range of biological associations between pairs of proteins that goes well beyond mere binary protein interactions.\",\"widget_text7\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color7\":\"444444\",\"order8\":\"10\",\"show_column8\":\"1\",\"column_border_color8\":\"DDDDDD\",\"column_border_radius8\":\"4px\",\"upload_image8\":\"images\\/partners\\/DCIC_500x375_cropped.png\",\"image_link8\":\"http:\\/\\/www.ilincs.org\\/ilincs\\/signature\\/GDS_5917\",\"image_target8\":\"blank\",\"image_alt8\":\"\",\"title_text8\":\"iLINCS\",\"title_color8\":\"FFFFFF\",\"title_bg_color8\":\"2F9EC2\",\"title_link8\":\"http:\\/\\/www.ilincs.org\\/ilincs\\/signature\\/GDS_5917\",\"title_link_target8\":\"blank\",\"title_icon8\":\"fa-empty\",\"description_text8\":\"Integrative LINCS is an integrative web platform for analysis of LINCS data and signatures.The BD2K-LINCS Data Coordination and Integration Center is part of the Big Data to Knowledge (BD2K) NIH initiative.\",\"widget_text8\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color8\":\"444444\",\"order9\":\"7\",\"show_column9\":\"1\",\"column_border_color9\":\"DDDDDD\",\"column_border_radius9\":\"4px\",\"upload_image9\":\"images\\/partners\\/EPD.png\",\"image_link9\":\"https:\\/\\/peptracker.com\\/epd\\/analytics\\/?section_id=20202\",\"image_target9\":\"blank\",\"image_alt9\":\"\",\"title_text9\":\"PepTracker: EPD\",\"title_color9\":\"FFFFFF\",\"title_bg_color9\":\"2F9EC2\",\"title_link9\":\"https:\\/\\/peptracker.com\\/epd\\/analytics\\/?section_id=20202\",\"title_link_target9\":\"blank\",\"title_icon9\":\"fa-empty\",\"description_text9\":\"The Encyclopedia of Proteome Dynamics is a polyglot persistent database and web-application that provides open access to integrated proteomics data from published studies on human cells and model organisms featuring graphical navigation with interactive visualizations that facilitate powerful data exploration in an intuitive manner.\",\"widget_text9\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color9\":\"444444\",\"order10\":\"11\",\"show_column10\":\"1\",\"column_border_color10\":\"DDDDDD\",\"column_border_radius10\":\"4px\",\"upload_image10\":\"images\\/partners\\/ipa_logo.png\",\"image_link10\":\"http:\\/\\/www.integratedproteomics.com\\/\",\"image_target10\":\"blank\",\"image_alt10\":\"\",\"title_text10\":\"IPA\",\"title_color10\":\"FFFFFF\",\"title_bg_color10\":\"2F9EC2\",\"title_link10\":\"http:\\/\\/www.integratedproteomics.com\\/\",\"title_link_target10\":\"blank\",\"title_icon10\":\"fa-empty\",\"description_text10\":\"Integrated Proteomics Applications provides a comprehensive proteomic data analysis solution, the Integrated Proteomics Pipeline (IP2), which allows researchers to identify, quantify, and functionally analyze all of the proteins in a biological sample on one, easy to use interface.\",\"widget_text10\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color10\":\"444444\",\"order11\":\"12\",\"show_column11\":\"1\",\"column_border_color11\":\"DDDDDD\",\"column_border_radius11\":\"4px\",\"upload_image11\":\"images\\/partners\\/scripps_logo.png\",\"image_link11\":\"http:\\/\\/sealion.scripps.edu\\/pint\\/\",\"image_target11\":\"blank\",\"image_alt11\":\"\",\"title_text11\":\"PINT\",\"title_color11\":\"FFFFFF\",\"title_bg_color11\":\"2F9EC2\",\"title_link11\":\"http:\\/\\/sealion.scripps.edu\\/pint\\/\",\"title_link_target11\":\"blank\",\"title_icon11\":\"fa-empty\",\"description_text11\":\"PINT, the Proteomics INTegrator, is an online experiment repository for final results coming from different qualitative and\\/or quantitative proteomics assays. PINT is a new comprehensive system to store, visualize, and analyze data for proteomics results obtained under different experimental conditions.\",\"widget_text11\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\\r\\n\",\"description_text_color11\":\"444444\",\"order12\":\"13\",\"show_column12\":\"1\",\"column_border_color12\":\"DDDDDD\",\"column_border_radius12\":\"4px\",\"upload_image12\":\"images\\/partners\\/COPaKB_logo.png\",\"image_link12\":\"https:\\/\\/amino.heartproteome.org\\/\",\"image_target12\":\"blank\",\"image_alt12\":\"\",\"title_text12\":\"COPaKB\",\"title_color12\":\"FFFFFF\",\"title_bg_color12\":\"2F9EC2\",\"title_link12\":\"https:\\/\\/amino.heartproteome.org\\/\",\"title_link_target12\":\"blank\",\"title_icon12\":\"fa-empty\",\"description_text12\":\"The COPa Knowledgebase (COPaKB) has been created as a unique resource to facilitate the discovery of novel biological insights from proteomic datasets. COPaKB was developed under the NHLBI Proteomics Centers Program (2010-2015) and is continuously supported through the NIGMS BD2K Centers Program (2014-2018).\",\"widget_text12\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color12\":\"444444\",\"order13\":\"3\",\"show_column13\":\"1\",\"column_border_color13\":\"DDDDDD\",\"column_border_radius13\":\"4px\",\"upload_image13\":\"images\\/partners\\/ensembl.png\",\"image_link13\":\"http:\\/\\/www.ensembl.org\\/Homo_sapiens\\/Gene\\/Pathway?db=core;g=ENSG00000139618;r=13:32315474-32400266\",\"image_target13\":\"blank\",\"image_alt13\":\"\",\"title_text13\":\"Ensembl\",\"title_color13\":\"FFFFFF\",\"title_bg_color13\":\"2F9EC2\",\"title_link13\":\"http:\\/\\/www.ensembl.org\\/Homo_sapiens\\/Gene\\/Pathway?db=core;g=ENSG00000139618;r=13:32315474-32400266\",\"title_link_target13\":\"blank\",\"title_icon13\":\"fa-empty\",\"description_text13\":\"Ensembl is a genome browser for vertebrate genomes that supports research in comparative genomics, evolution, sequence variation and transcriptional regulation. Ensembl annotate genes, computes multiple alignments, predicts regulatory function and collects disease data. \",\"widget_text13\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color13\":\"444444\",\"order14\":\"14\",\"show_column14\":\"1\",\"column_border_color14\":\"DDDDDD\",\"column_border_radius14\":\"4px\",\"upload_image14\":\"images\\/partners\\/targetpathogen.png\",\"image_link14\":\"http:\\/\\/target.sbg.qb.fcen.uba.ar\\/patho\\/protein\\/56425909be737e6c7a9fd658#patho_host_row\",\"image_target14\":\"blank\",\"image_alt14\":\"\",\"title_text14\":\"Target-Pathogen\",\"title_color14\":\"FFFFFF\",\"title_bg_color14\":\"2F9EC2\",\"title_link14\":\"http:\\/\\/target.sbg.qb.fcen.uba.ar\\/patho\\/protein\\/56425909be737e6c7a9fd658#patho_host_row\",\"title_link_target14\":\"blank\",\"title_icon14\":\"fa-empty\",\"description_text14\":\"Target-Pathogen database is a bioinformatic approach to prioritize and identify candidates drug targets for pathogens. Designed and developed as an online resource to allow genome wide data consolidation from diverse sources focusing on structural druggability, essentiality and metabolic role of proteins. \\r\\n\",\"widget_text14\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color14\":\"444444\",\"order15\":\"15\",\"show_column15\":\"0\",\"column_border_color15\":\"DDDDDD\",\"column_border_radius15\":\"4px\",\"upload_image15\":\"\",\"image_link15\":\"\",\"image_target15\":\"blank\",\"image_alt15\":\"\",\"title_text15\":\"Title Text\",\"title_color15\":\"FFFFFF\",\"title_bg_color15\":\"2F9EC2\",\"title_link15\":\"\",\"title_link_target15\":\"blank\",\"title_icon15\":\"fa-recycle\",\"description_text15\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"widget_text15\":\"\",\"description_text_color15\":\"444444\",\"order16\":\"16\",\"show_column16\":\"0\",\"column_border_color16\":\"DDDDDD\",\"column_border_radius16\":\"4px\",\"upload_image16\":\"\",\"image_link16\":\"\",\"image_target16\":\"blank\",\"image_alt16\":\"\",\"title_text16\":\"Title Text\",\"title_color16\":\"FFFFFF\",\"title_bg_color16\":\"2F9EC2\",\"title_link16\":\"\",\"title_link_target16\":\"blank\",\"title_icon16\":\"fa-recycle\",\"description_text16\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"widget_text16\":\"\",\"description_text_color16\":\"444444\",\"order17\":\"17\",\"show_column17\":\"0\",\"column_border_color17\":\"DDDDDD\",\"column_border_radius17\":\"4px\",\"upload_image17\":\"\",\"image_link17\":\"\",\"image_target17\":\"blank\",\"image_alt17\":\"\",\"title_text17\":\"Title Text\",\"title_color17\":\"FFFFFF\",\"title_bg_color17\":\"2F9EC2\",\"title_link17\":\"\",\"title_link_target17\":\"blank\",\"title_icon17\":\"fa-recycle\",\"description_text17\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"widget_text17\":\"\",\"description_text_color17\":\"444444\",\"order18\":\"18\",\"show_column18\":\"0\",\"column_border_color18\":\"DDDDDD\",\"column_border_radius18\":\"4px\",\"upload_image18\":\"\",\"image_link18\":\"\",\"image_target18\":\"blank\",\"image_alt18\":\"\",\"title_text18\":\"Title Text\",\"title_color18\":\"FFFFFF\",\"title_bg_color18\":\"2F9EC2\",\"title_link18\":\"\",\"title_link_target18\":\"blank\",\"title_icon18\":\"fa-recycle\",\"description_text18\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"widget_text18\":\"\",\"description_text_color18\":\"444444\",\"order19\":\"19\",\"show_column19\":\"0\",\"column_border_color19\":\"DDDDDD\",\"column_border_radius19\":\"4px\",\"upload_image19\":\"\",\"image_link19\":\"\",\"image_target19\":\"blank\",\"image_alt19\":\"\",\"title_text19\":\"Title Text\",\"title_color19\":\"FFFFFF\",\"title_bg_color19\":\"2F9EC2\",\"title_link19\":\"\",\"title_link_target19\":\"blank\",\"title_icon19\":\"fa-recycle\",\"description_text19\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"widget_text19\":\"\",\"description_text_color19\":\"444444\",\"order20\":\"20\",\"show_column20\":\"0\",\"column_border_color20\":\"DDDDDD\",\"column_border_radius20\":\"4px\",\"upload_image20\":\"\",\"image_link20\":\"\",\"image_target20\":\"blank\",\"image_alt20\":\"\",\"title_text20\":\"Title Text\",\"title_color20\":\"FFFFFF\",\"title_bg_color20\":\"2F9EC2\",\"title_link20\":\"\",\"title_link_target20\":\"blank\",\"title_icon20\":\"fa-recycle\",\"description_text20\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"widget_text20\":\"\",\"description_text_color20\":\"444444\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(555,651,'Do you need help ?','','',1,'screen1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,1,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"50%\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"300\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"20px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-heart\",\"icon_color1\":\"006FFF\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"C8C8C8\",\"icon_link1\":\"index.php?option=com_content&view=article&id=39\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4em\",\"title_text1\":\"Use Reactome!\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=39\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-code\",\"icon_color2\":\"0099FF\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"C8C8C8\",\"icon_link2\":\"dev\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4em\",\"title_text2\":\"For Developers\",\"title_color2\":\"444444\",\"title_link2\":\"dev\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-pencil\",\"icon_color3\":\"0099FF\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"C8C8C8\",\"icon_link3\":\"index.php?option=com_content&view=article&id=29\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4em\",\"title_text3\":\"Citing us\",\"title_color3\":\"444444\",\"title_link3\":\"index.php?option=com_content&view=article&id=29\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-envelope\",\"icon_color4\":\"0099FF\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"DDDDDD\",\"icon_link4\":\"mailto:help@reactome.org\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4em\",\"title_text4\":\"Contact Us\",\"title_color4\":\"444444\",\"title_link4\":\"mailto:help@reactome.org\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"DDDDDD\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4em\",\"title_text5\":\"Title Text\",\"title_color5\":\"111111\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"DDDDDD\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"4em\",\"title_text6\":\"Title Text\",\"title_color6\":\"111111\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favcustom\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h4\",\"header_class\":\"fa-question-circle\",\"style\":\"0\"}',0,'*'),(556,652,'API and Data access','','',1,'top1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,1,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"50%\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"300\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"20px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-laptop\",\"icon_color1\":\"006FFF\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"C8C8C8\",\"icon_link1\":\"\\/ContentService\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4em\",\"title_text1\":\"Content Service\",\"title_color1\":\"444444\",\"title_link1\":\"\\/ContentService\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-line-chart\",\"icon_color2\":\"0099FF\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"C8C8C8\",\"icon_link2\":\"\\/AnalysisService\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4em\",\"title_text2\":\"Analysis Service\",\"title_color2\":\"444444\",\"title_link2\":\"\\/AnalysisService\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-archive\",\"icon_color3\":\"0099FF\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"C8C8C8\",\"icon_link3\":\"icon-lib\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4em\",\"title_text3\":\"Icon Library\",\"title_color3\":\"444444\",\"title_link3\":\"icon-lib\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-database\",\"icon_color4\":\"0099FF\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"C8C8C8\",\"icon_link4\":\"index.php?option=com_content&view=article&id=45\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4em\",\"title_text4\":\"Graph Database\",\"title_color4\":\"444444\",\"title_link4\":\"index.php?option=com_content&view=article&id=45\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"DDDDDD\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4em\",\"title_text5\":\"Title Text\",\"title_color5\":\"111111\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"DDDDDD\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"4em\",\"title_text6\":\"Title Text\",\"title_color6\":\"111111\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favcustom\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa-bookmark\",\"style\":\"0\"}',0,'*'),(559,655,'Pathway Diagram - Dev Guide, 4 buttons','4 buttons in Pathway Diagram Dev Guide Page','',1,'dev-guide-pd-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(560,656,'Pathway Overview - Dev Guide, 4 buttons','4 buttons in Pathway Overview Dev Guide Page','',1,'dev-guide-po-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(561,657,'Pathway Diagram - Dev Guide - Get Started, 2 buttons ','Get Started 2 buttons in Pathway Diagram Dev Guide Page','',1,'dev-guide-pd-2',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"25%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"rglyph-gs-diagram-js\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"index.php?option=com_content&view=article&id=58\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Diagram JS\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=58\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"25%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"rglyph-gs-diagram-gwt\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"index.php?option=com_content&view=article&id=59\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Diagram GWT\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=59\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"0\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle fav2buttons\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(563,659,'Pathway Diagram - Dev Guide - API, 2 buttons','API 2 buttons in Pathway Diagram Dev Guide Page','',1,'dev-guide-pd-3',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"25%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"rglyph-diagram-js\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"index.php?option=com_content&view=article&id=58#API\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Diagram JS API\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=58#API\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"25%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"rglyph-diagram-gwt\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"index.php?option=com_content&view=article&id=59#API\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Diagram GWT API\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=59#API\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"0\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle fav2buttons\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(564,660,'Pathway Overview - Dev Guide - Get Started, 2 buttons','Get Started 2 buttons in Pathway Overview Dev Guide Page','',1,'dev-guide-po-2',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"25%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"rglyph-gs-fireworks-js\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"index.php?option=com_content&view=article&id=56\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Fireworks JS\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=56\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"25%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"rglyph-gs-fireworks-gwt\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"index.php?option=com_content&view=article&id=57\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Fireworks GWT\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=57\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"0\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle fav2buttons\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(565,661,'Pathway Overview - Dev Guide - API, 2 buttons','API 2 buttons in Pathway Overview Dev Guide Page','',1,'dev-guide-po-3',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"25%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"rglyph-fireworks-js\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"index.php?option=com_content&view=article&id=56#API\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Fireworks JS API\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=56#API\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"25%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"rglyph-fireworks-gwt\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"index.php?option=com_content&view=article&id=57#API\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Fireworks GWT API\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=57#API\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"0\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle fav2buttons\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(566,662,'Pathway Overview GWT - Dev Guide, 4 buttons','4 buttons in Pathway Overview GWT Dev Guide Page','',1,'dev-guide-po-gwt-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(567,663,'Pathway Diagram GWT - Dev Guide, 4 buttons','4 buttons in Pathway Diagram GWT Dev Guide Page','',1,'dev-guide-pd-gwt-1',0,'0000-00-00 00:00:00','2017-06-29 10:09:22','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(568,664,'Content Service - Dev Guide, 4 buttons','4 buttons in Content Service Dev Guide Page','',1,'dev-guide-cs-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"\\/ContentService\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"\\/ContentService\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(569,665,'Analysis Service - Dev Guide, 4 buttons','4 buttons in Analysis Service Dev Guide Page','',1,'dev-guide-as-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"\\/AnalysisService\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"\\/AnalysisService\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(570,666,'Graph Database - Dev Guide, 4 buttons','4 buttons in Graph Database Dev Guide Page','',1,'dev-guide-gd-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"50%\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-download\",\"icon_color5\":\"FFFFFF\",\"icon_bg_color5\":\"0099FF\",\"icon_border_color5\":\"0099FF\",\"icon_link5\":\"\\/download\\/current\\/reactome.graphdb.tgz\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4.8em\",\"title_text5\":\"Download\",\"title_color5\":\"444444\",\"title_link5\":\"\\/download\\/current\\/reactome.graphdb.tgz\",\"title_link_target5\":\"blank\",\"description_text5\":\" \",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(571,667,'Developers Zone - Row 2','','',1,'dev-guide-2',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"28px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"40%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-sitemap\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"CCCCCC\",\"icon_link1\":\"index.php?option=com_content&view=article&id=49\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Pathways Overview\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=49\",\"title_link_target1\":\"self\",\"description_text1\":\"Use this widget to include our pathways overview in your web application\",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"40%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-television\",\"icon_color2\":\"FF5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"CCCCCC\",\"icon_link2\":\"index.php?option=com_content&view=article&id=50\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4em\",\"title_text2\":\"Pathway Diagrams\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=50\",\"title_link_target2\":\"self\",\"description_text2\":\"Use this widget to include our pathway diagrams in your web application\",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"40%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-handshake-o\",\"icon_color3\":\"8BC34A\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"CCCCCC\",\"icon_link3\":\"index.php?option=com_content&view=article&id=26\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4em\",\"title_text3\":\"Reactome Partners\",\"title_color3\":\"444444\",\"title_link3\":\"index.php?option=com_content&view=article&id=26\",\"title_link_target3\":\"self\",\"description_text3\":\"Check out who is currently using Reactome web services and widgets\",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-lightbulb-o\",\"icon_color4\":\"FFC107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"CCCCCC\",\"icon_link4\":\"\",\"icon_link_target4\":\"blank\",\"icon_font_size4\":\"4em\",\"title_text4\":\"<<<< READ DESCRIPTION >>>>\",\"title_color4\":\"444444\",\"title_link4\":\"\",\"title_link_target4\":\"blank\",\"description_text4\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4em\",\"title_text5\":\"<<< READ DESCRIPTION >>>\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"4em\",\"title_text6\":\"<<< READ DESCRIPTION >>>\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"-sfx last favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(574,674,'Version 70 released on September 9, 2019','Release Statistics (stats) for Landing Page','',1,'portfolio1',169,'2019-09-10 19:05:53','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,1,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"1px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"0px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"600\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"28px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"25%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-male\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"FFFFFF\",\"icon_link1\":\"\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"2.5em\",\"title_text1\":\"2,287\",\"title_color1\":\"444444\",\"title_link1\":\"\",\"title_link_target1\":\"self\",\"description_text1\":\"Human Pathways\",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"25%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"rglyph-reaction\",\"icon_color2\":\"FF5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"FFFFFF\",\"icon_link2\":\"\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"2.5em\",\"title_text2\":\"12,608\",\"title_color2\":\"444444\",\"title_link2\":\"\",\"title_link_target2\":\"self\",\"description_text2\":\"Reactions\",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"25%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"rglyph-protein\",\"icon_color3\":\"8BC34A\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"FFFFFF\",\"icon_link3\":\"\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"2.5em\",\"title_text3\":\"10,860\",\"title_color3\":\"444444\",\"title_link3\":\"\",\"title_link_target3\":\"self\",\"description_text3\":\"Proteins\",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"25%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"rglyph-small-molecule\",\"icon_color4\":\"FFC107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"FFFFFF\",\"icon_link4\":\"\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"2.5em\",\"title_text4\":\"1,856\",\"title_color4\":\"444444\",\"title_link4\":\"\",\"title_link_target4\":\"self\",\"description_text4\":\"Small Molecules\",\"description_text_color4\":\"444444\",\"show_icon5\":\"1\",\"icon_width5\":\"25%\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa fa-medkit\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"FFFFFF\",\"icon_link5\":\"\",\"icon_link_target5\":\"self\",\"icon_font_size5\":\"2.5em\",\"title_text5\":\"222\",\"title_color5\":\"111111\",\"title_link5\":\"\",\"title_link_target5\":\"self\",\"description_text5\":\"Drugs\",\"description_text_color5\":\"444444\",\"show_icon6\":\"1\",\"icon_width6\":\"25%\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa fa-book\",\"icon_color6\":\"673AB7\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"DDDDDD\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"2.5em\",\"title_text6\":\"30,398\",\"title_color6\":\"111111\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Literature References\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstats\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa-calendar\",\"style\":\"0\"}',0,'*'),(579,681,'Footer (For Small Devices)','','',1,'footer1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',0,'mod_menu',1,0,'{\"menutype\":\"mainmenu\",\"base\":\"101\",\"startLevel\":\"1\",\"endLevel\":\"1\",\"showAllChildren\":\"0\",\"tag_id\":\"\",\"class_sfx\":\" footer-menu\",\"window_open\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-visible- favth-visible-xs\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"itemid\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(580,700,'User Guide Glyphs','','',1,'user-guide-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"50%\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"600\",\"title_font_style\":\"normal\",\"title_font_size\":\"16px\",\"title_margin\":\"28px 0px 14px\",\"title_text_transform\":\"uppercase\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-sitemap\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"DDDDDD\",\"icon_link1\":\"userguide\\/pathway-browser\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"3em\",\"title_text1\":\"Pathway Browser\",\"title_color1\":\"2196F3\",\"title_link1\":\"user\\/guide\\/pathway-browser\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-laptop\",\"icon_color2\":\"FF5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"DDDDDD\",\"icon_link2\":\"userguide\\/details-panel\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"3em\",\"title_text2\":\"Details Panel\",\"title_color2\":\"FF5252\",\"title_link2\":\"user\\/guide\\/details-panel\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-search\",\"icon_color3\":\"8BC34A\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"DDDDDD\",\"icon_link3\":\"userguide\\/searching\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"3em\",\"title_text3\":\"How do I search?\",\"title_color3\":\"8BC34A\",\"title_link3\":\"user\\/guide\\/searching\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-area-chart\",\"icon_color4\":\"FFC107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"DDDDDD\",\"icon_link4\":\"userguide\\/analysis\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"3em\",\"title_text4\":\"Analysis Tools\",\"title_color4\":\"FFC107\",\"title_link4\":\"user\\/guide\\/analysis\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"1\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa fa-stethoscope\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"DDDDDD\",\"icon_link5\":\"userguide\\/diseases\",\"icon_link_target5\":\"self\",\"icon_font_size5\":\"3em\",\"title_text5\":\"Diseases\",\"title_color5\":\"673AB7\",\"title_link5\":\"user\\/guide\\/diseases\",\"title_link_target5\":\"self\",\"description_text5\":\" \",\"description_text_color5\":\"444444\",\"show_icon6\":\"1\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa fa-share-alt-square\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"DDDDDD\",\"icon_link6\":\"tools\\/reactome-fiviz\",\"icon_link_target6\":\"self\",\"icon_font_size6\":\"3em\",\"title_text6\":\"ReactomeFIVIz\",\"title_color6\":\"00BCD4\",\"title_link6\":\"user\\/reactome-fiviz\",\"title_link_target6\":\"self\",\"description_text6\":\" \",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(581,712,'Warning','Banner - Temporary Warnings. IMPORTANT: It is duplicated to address responsive design. Less text is shown on the small devices.','',1,'topbar1',0,'0000-00-00 00:00:00','1000-01-01 00:00:00','1000-01-01 00:00:00',0,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(582,713,'Why Reactome','CHINA :: Short description for Landing Page. Be careful when adding text here. It may misalign the Twitter Widget. To fix, adjust the .twitter-timeline height in custom.css','<p style=\"text-align: justify;\">Reactome is a free, open-source, curated and peer reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology and education.</p>\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 pull-center  favth-hidden-sm favth-hidden-xs\" style=\"background-color: ghostwhite; margin: 18px 0; padding: 10px; border-radius: 8px; border: 2px solid #58C3E5;\">\r\n<div class=\"favth-col-lg-2-4 favth-col-md-2 favth-col-sm-1 text-center\" style=\"padding: 0 4px;\"><a href=\"http://www.ebi.ac.uk/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px;\" src=\"images/sponsor/embl_ebi_logo.png\" alt=\"European Bioinformatics Institute (EMBL-EBI)\" /></a></div>\r\n<div class=\"favth-col-lg-2-4 favth-col-md-2 favth-col-sm-1 text-center\"><a href=\"http://www.ncpsb.org/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px;\" src=\"images/sponsor/ncpsb.png\" alt=\"National Center for Protein Sciences - Beijing\" /></a></div>\r\n<div class=\"favth-col-lg-2-4 favth-col-md-2 favth-col-sm-1 text-center\" style=\"padding-right: 0;\"><a href=\"https://med.nyu.edu/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px; padding-right: 0;\" src=\"images/sponsor/nyul_health_logo.png\" alt=\"NYU Langone Health\" /></a></div>\r\n<div class=\"favth-col-lg-2-4 favth-col-md-2 favth-col-sm-1 text-center\"><a href=\"http://www.ohsu.edu/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px;\" src=\"images/sponsor/ohsu_logo.png\" alt=\"Oregon Health &amp; Science University\" /></a></div>\r\n<div class=\"favth-col-lg-2-4 favth-col-md-2 favth-col-sm-1 text-center\"><a href=\"https://oicr.on.ca/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px;\" src=\"images/sponsor/oicr_logo.png\" alt=\"Ontario Institute for Cancer Research\" /></a></div>\r\n</div>\r\n<p style=\"text-align: center; line-height: 13px; clear: both; padding-top: 10px;\"><span style=\"font-size: 8pt;\">The development of Reactome is supported by grants from the US National Institutes of Health (P41 HG003751 and 1U54GM114833-01), Ontario Research Fund, and the European Molecular Biology Laboratory.</span></p>',1,'prime1',0,'0000-00-00 00:00:00','2018-06-22 09:58:12','0000-00-00 00:00:00',0,'mod_custom',1,1,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-clear favth-shadow\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-search\",\"style\":\"0\"}',0,'*'),(583,715,'[TEMPLATE] CGI placeholder','Div placeholder for template-page. This is hidden from the normal website. CGI pages','<!-- template-placeholder -->\r\n<p>&nbsp;</p>',1,'lead1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(584,720,'Synchronization Tool','Synchronization Tool: Joomla Content only','',1,'sync-tool',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',7,1,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(585,738,'Elixir Banner','Includes and customises Elixir logo to fit nicely in the Reactome Page.','<style type=\"text/css\">.elixir-ribbon {\r\n    padding-bottom: 0px !important;\r\n    padding-top: 16px !important;\r\n}.elixir-ribbon .row {\r\nmargin: 0 !important; \r\n}.elixir-ribbon .elixir-logo-kite {\r\ntop: -12px !important;\r\n}</style>\r\n<script type=\"text/javascript\">jQuery(document).ready(function() {jQuery(\'#elixir-ribbon a\').each(function(){\r\n  jQuery(this).attr(\'target\', \'_blank\');\r\n });});</script>\r\n<div id=\"elixir-banner\" data-color=\"none\" data-name=\"Reactome\" data-description=\"Reactome is an Elixir Service\" data-use-basic-styles=\"false\"></div>\r\n<script src=\"https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/js/elixirBanner.js\" defer=\"defer\" type=\"text/javascript\"></script>',1,'user1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(586,772,'EHLD / Icon Library (Specs & Guidelines)','','',1,'icon-lib-pos-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"28px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"30%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-magic\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"CCCCCC\",\"icon_link1\":\"index.php?option=com_content&view=article&id=129\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4em\",\"title_text1\":\"EHLD Specs & Guidelines\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=129\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"30%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-archive\",\"icon_color2\":\"FF5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"CCCCCC\",\"icon_link2\":\"index.php?option=com_content&view=article&id=130\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4em\",\"title_text2\":\"Icon Library guidelines\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=130\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"0\",\"icon_width3\":\"40%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-database\",\"icon_color3\":\"8BC34A\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"CCCCCC\",\"icon_link3\":\"index.php?option=com_content&view=article&id=45\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4em\",\"title_text3\":\"Graph Database\",\"title_color3\":\"444444\",\"title_link3\":\"index.php?option=com_content&view=article&id=45\",\"title_link_target3\":\"self\",\"description_text3\":\"Access to the Reactome knowledgebase content as an interconnected graph database\",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-lightbulb-o\",\"icon_color4\":\"FFC107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"CCCCCC\",\"icon_link4\":\"\",\"icon_link_target4\":\"blank\",\"icon_font_size4\":\"4em\",\"title_text4\":\"<<<< READ DESCRIPTION >>>>\",\"title_color4\":\"444444\",\"title_link4\":\"\",\"title_link_target4\":\"blank\",\"description_text4\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4em\",\"title_text5\":\"<<< READ DESCRIPTION >>>\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"4em\",\"title_text6\":\"<<< READ DESCRIPTION >>>\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"-sfx favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*');
/*!40000 ALTER TABLE `rlp_modules` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_modules_menu`
--

DROP TABLE IF EXISTS `rlp_modules_menu`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_modules_menu` (
  `moduleid` int(11) NOT NULL DEFAULT '0',
  `menuid` int(11) NOT NULL DEFAULT '0',
  PRIMARY KEY (`moduleid`,`menuid`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_modules_menu`
--

LOCK TABLES `rlp_modules_menu` WRITE;
/*!40000 ALTER TABLE `rlp_modules_menu` DISABLE KEYS */;
INSERT INTO `rlp_modules_menu` VALUES (0,0),(1,0),(2,0),(3,0),(4,0),(6,0),(7,0),(8,0),(9,0),(10,0),(12,0),(13,0),(14,0),(15,0),(17,235),(17,236),(17,237),(17,251),(17,252),(17,253),(17,254),(17,259),(17,260),(17,261),(17,262),(17,263),(17,264),(17,265),(17,269),(17,270),(17,271),(17,272),(17,290),(17,292),(17,293),(17,294),(17,295),(17,296),(17,297),(17,299),(17,301),(17,302),(17,303),(17,304),(17,305),(17,306),(17,307),(17,360),(17,361),(17,362),(17,377),(17,378),(17,379),(17,380),(17,381),(17,382),(17,384),(17,407),(17,424),(17,471),(17,472),(79,0),(86,0),(88,0),(89,0),(475,101),(479,-281),(479,-280),(479,-278),(479,-107),(479,-106),(479,-104),(479,-102),(484,101),(496,101),(499,101),(500,0),(505,101),(508,101),(518,-288),(518,-266),(518,-246),(518,-101),(519,-288),(519,-284),(519,-266),(519,-101),(523,267),(524,-422),(524,-406),(524,-398),(524,-397),(524,-371),(524,-361),(524,-288),(524,-284),(524,-277),(524,-266),(524,-263),(524,-246),(524,-242),(524,-152),(524,-107),(524,-106),(524,-104),(524,-102),(524,-101),(525,266),(526,288),(526,371),(527,288),(533,0),(540,268),(544,152),(551,263),(555,101),(556,101),(559,295),(560,294),(561,295),(563,295),(564,294),(565,294),(566,305),(567,307),(568,293),(569,292),(570,290),(571,268),(573,266),(574,101),(579,0),(580,303),(581,0),(582,101),(583,383),(584,406),(585,101),(586,302);
/*!40000 ALTER TABLE `rlp_modules_menu` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_newsfeeds`
--

DROP TABLE IF EXISTS `rlp_newsfeeds`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_newsfeeds` (
  `catid` int(11) NOT NULL DEFAULT '0',
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `name` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `link` varchar(2048) COLLATE utf8mb4_unicode_ci NOT NULL,
  `published` tinyint(1) NOT NULL DEFAULT '0',
  `numarticles` int(10) unsigned NOT NULL DEFAULT '1',
  `cache_time` int(10) unsigned NOT NULL DEFAULT '3600',
  `checked_out` int(10) unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `ordering` int(11) NOT NULL DEFAULT '0',
  `rtl` tinyint(4) NOT NULL DEFAULT '0',
  `access` int(10) unsigned NOT NULL DEFAULT '0',
  `language` char(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `params` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by` int(10) unsigned NOT NULL DEFAULT '0',
  `created_by_alias` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int(10) unsigned NOT NULL DEFAULT '0',
  `metakey` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadesc` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadata` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `xreference` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `description` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `version` int(10) unsigned NOT NULL DEFAULT '1',
  `hits` int(10) unsigned NOT NULL DEFAULT '0',
  `images` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`id`),
  KEY `idx_access` (`access`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_state` (`published`),
  KEY `idx_catid` (`catid`),
  KEY `idx_createdby` (`created_by`),
  KEY `idx_language` (`language`),
  KEY `idx_xreference` (`xreference`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_newsfeeds`
--

LOCK TABLES `rlp_newsfeeds` WRITE;
/*!40000 ALTER TABLE `rlp_newsfeeds` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_newsfeeds` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_overrider`
--

DROP TABLE IF EXISTS `rlp_overrider`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_overrider` (
  `id` int(10) NOT NULL AUTO_INCREMENT COMMENT 'Primary Key',
  `constant` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
  `string` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `file` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_overrider`
--

LOCK TABLES `rlp_overrider` WRITE;
/*!40000 ALTER TABLE `rlp_overrider` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_overrider` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_postinstall_messages`
--

DROP TABLE IF EXISTS `rlp_postinstall_messages`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_postinstall_messages` (
  `postinstall_message_id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
  `extension_id` bigint(20) NOT NULL DEFAULT '700' COMMENT 'FK to #__extensions',
  `title_key` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'Lang key for the title',
  `description_key` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'Lang key for description',
  `action_key` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `language_extension` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT 'com_postinstall' COMMENT 'Extension holding lang keys',
  `language_client_id` tinyint(3) NOT NULL DEFAULT '1',
  `type` varchar(10) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT 'link' COMMENT 'Message type - message, link, action',
  `action_file` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT '' COMMENT 'RAD URI to the PHP file containing action method',
  `action` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT '' COMMENT 'Action method name or URL',
  `condition_file` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL COMMENT 'RAD URI to file holding display condition method',
  `condition_method` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL COMMENT 'Display condition method, must return boolean',
  `version_introduced` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '3.2.0' COMMENT 'Version when this message was introduced',
  `enabled` tinyint(3) NOT NULL DEFAULT '1',
  PRIMARY KEY (`postinstall_message_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_postinstall_messages`
--

LOCK TABLES `rlp_postinstall_messages` WRITE;
/*!40000 ALTER TABLE `rlp_postinstall_messages` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_postinstall_messages` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_privacy_consents`
--

DROP TABLE IF EXISTS `rlp_privacy_consents`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_privacy_consents` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `state` int(10) NOT NULL DEFAULT '1',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `subject` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `body` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `remind` tinyint(4) NOT NULL DEFAULT '0',
  `token` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  PRIMARY KEY (`id`),
  KEY `idx_user_id` (`user_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_privacy_consents`
--

LOCK TABLES `rlp_privacy_consents` WRITE;
/*!40000 ALTER TABLE `rlp_privacy_consents` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_privacy_consents` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_privacy_requests`
--

DROP TABLE IF EXISTS `rlp_privacy_requests`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_privacy_requests` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `email` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `requested_at` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `status` tinyint(4) NOT NULL DEFAULT '0',
  `request_type` varchar(25) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `confirm_token` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `confirm_token_created_at` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_privacy_requests`
--

LOCK TABLES `rlp_privacy_requests` WRITE;
/*!40000 ALTER TABLE `rlp_privacy_requests` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_privacy_requests` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_redirect_links`
--

DROP TABLE IF EXISTS `rlp_redirect_links`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_redirect_links` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `old_url` varchar(2048) COLLATE utf8mb4_unicode_ci NOT NULL,
  `new_url` varchar(2048) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `referer` varchar(2048) COLLATE utf8mb4_unicode_ci NOT NULL,
  `comment` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `hits` int(10) unsigned NOT NULL DEFAULT '0',
  `published` tinyint(4) NOT NULL,
  `created_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `header` smallint(3) NOT NULL DEFAULT '301',
  PRIMARY KEY (`id`),
  KEY `idx_link_modifed` (`modified_date`),
  KEY `idx_old_url` (`old_url`(100))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_redirect_links`
--

LOCK TABLES `rlp_redirect_links` WRITE;
/*!40000 ALTER TABLE `rlp_redirect_links` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_redirect_links` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_schemas`
--

DROP TABLE IF EXISTS `rlp_schemas`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_schemas` (
  `extension_id` int(11) NOT NULL,
  `version_id` varchar(20) COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`extension_id`,`version_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_schemas`
--

LOCK TABLES `rlp_schemas` WRITE;
/*!40000 ALTER TABLE `rlp_schemas` DISABLE KEYS */;
INSERT INTO `rlp_schemas` VALUES (700,'3.9.10-2019-07-09'),(10208,'3.2.3');
/*!40000 ALTER TABLE `rlp_schemas` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_session`
--

DROP TABLE IF EXISTS `rlp_session`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_session` (
  `session_id` varbinary(192) NOT NULL,
  `client_id` tinyint(3) unsigned DEFAULT NULL,
  `guest` tinyint(3) unsigned DEFAULT '1',
  `time` int(11) NOT NULL DEFAULT '0',
  `data` longtext COLLATE utf8mb4_unicode_ci,
  `userid` int(11) DEFAULT '0',
  `username` varchar(150) COLLATE utf8mb4_unicode_ci DEFAULT '',
  PRIMARY KEY (`session_id`),
  KEY `userid` (`userid`),
  KEY `time` (`time`),
  KEY `client_id_guest` (`client_id`,`guest`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_session`
--

LOCK TABLES `rlp_session` WRITE;
/*!40000 ALTER TABLE `rlp_session` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_session` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_tags`
--

DROP TABLE IF EXISTS `rlp_tags`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_tags` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `parent_id` int(10) unsigned NOT NULL DEFAULT '0',
  `lft` int(11) NOT NULL DEFAULT '0',
  `rgt` int(11) NOT NULL DEFAULT '0',
  `level` int(10) unsigned NOT NULL DEFAULT '0',
  `path` varchar(400) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `title` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `note` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `description` longtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `published` tinyint(1) NOT NULL DEFAULT '0',
  `checked_out` int(11) unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `access` int(10) unsigned NOT NULL DEFAULT '0',
  `params` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadesc` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The meta description for the page.',
  `metakey` varchar(1024) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The meta keywords for the page.',
  `metadata` varchar(2048) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'JSON encoded metadata properties.',
  `created_user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `created_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by_alias` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `modified_user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `modified_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `images` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `urls` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `hits` int(10) unsigned NOT NULL DEFAULT '0',
  `language` char(7) COLLATE utf8mb4_unicode_ci NOT NULL,
  `version` int(10) unsigned NOT NULL DEFAULT '1',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  PRIMARY KEY (`id`),
  KEY `tag_idx` (`published`,`access`),
  KEY `idx_access` (`access`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_left_right` (`lft`,`rgt`),
  KEY `idx_language` (`language`),
  KEY `idx_path` (`path`(100)),
  KEY `idx_alias` (`alias`(100))
) ENGINE=InnoDB AUTO_INCREMENT=8 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_tags`
--

LOCK TABLES `rlp_tags` WRITE;
/*!40000 ALTER TABLE `rlp_tags` DISABLE KEYS */;
INSERT INTO `rlp_tags` VALUES (1,0,0,7,0,'','ROOT','root','','',1,0,'1000-01-01 00:00:00',1,'{}','','','',156,'2011-01-01 00:00:01','',0,'0000-00-00 00:00:00','','',0,'*',1,'0000-00-00 00:00:00','0000-00-00 00:00:00'),(5,1,1,2,1,'tutorial','tutorial','tutorial','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',156,'2017-06-05 09:55:40','',0,'2017-06-05 09:55:40','{}','{}',0,'*',1,'2017-06-05 09:55:40','2017-06-05 09:55:40'),(6,1,3,4,1,'graph','graph','graph','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',156,'2017-06-05 09:55:40','',0,'2017-06-05 09:55:40','{}','{}',0,'*',1,'2017-06-05 09:55:40','2017-06-05 09:55:40'),(7,1,5,6,1,'release','release','release','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',156,'2017-09-28 21:26:13','',0,'2017-09-28 21:26:13','{}','{}',0,'*',1,'2017-09-28 21:26:13','2017-09-28 21:26:13');
/*!40000 ALTER TABLE `rlp_tags` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_template_styles`
--

DROP TABLE IF EXISTS `rlp_template_styles`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_template_styles` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `template` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `client_id` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `home` char(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '0',
  `title` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `params` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`id`),
  KEY `idx_template` (`template`),
  KEY `idx_client_id` (`client_id`),
  KEY `idx_client_id_home` (`client_id`,`home`)
) ENGINE=InnoDB AUTO_INCREMENT=25 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_template_styles`
--

LOCK TABLES `rlp_template_styles` WRITE;
/*!40000 ALTER TABLE `rlp_template_styles` DISABLE KEYS */;
INSERT INTO `rlp_template_styles` VALUES (4,'beez3',0,'0','Beez3 - Default','{\"wrapperSmall\":\"53\",\"wrapperLarge\":\"72\",\"logo\":\"images\\/joomla_black.gif\",\"sitetitle\":\"Joomla!\",\"sitedescription\":\"Open Source Content Management\",\"navposition\":\"left\",\"templatecolor\":\"personal\",\"html5\":\"0\"}'),(5,'hathor',1,'0','Hathor - Default','{\"showSiteName\":\"0\",\"colourChoice\":\"\",\"boldText\":\"0\"}'),(7,'protostar',0,'0','protostar - Default','{\"templateColor\":\"#0088cc\",\"templateBackgroundColor\":\"#f7f2f2\",\"logoFile\":\"\",\"sitetitle\":\"\",\"sitedescription\":\"\",\"googleFont\":\"1\",\"googleFontName\":\"Open+Sans\",\"fluidContainer\":\"0\"}'),(8,'isis',1,'1','isis - Default','{\"templateColor\":\"\",\"logoFile\":\"\"}'),(24,'favourite',0,'1','Favourite - Default','{\"template_styles\":\"style2\",\"custom_style\":\"\",\"max_width\":\"1190px\",\"show_copyright\":\"1\",\"copyright_text\":\"Reactome\",\"copyright_text_link\":\"reactome.org\",\"nav_link_padding\":\"\",\"nav_font_size\":\"\",\"nav_text_transform\":\"uppercase\",\"nav_icons_color\":\"\",\"nav_icons_font_size\":\"\",\"nav_google_font\":\"Open Sans\",\"nav_google_font_weight\":\"400\",\"nav_google_font_style\":\"normal\",\"titles_font_size\":\"\",\"titles_line_height\":\"\",\"titles_text_align\":\"left\",\"titles_text_transform\":\"uppercase\",\"module_title_icon_font_size\":\"\",\"module_title_icon_padding\":\"\",\"module_title_icon_border_radius\":\"\",\"titles_google_font\":\"Open Sans\",\"titles_google_font_weight\":\"300\",\"titles_google_font_style\":\"normal\",\"error_page_article_id\":\"43\",\"offline_page_style\":\"offline-light\",\"offline_page_bg_image\":\"\",\"offline_page_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"show_back_to_top\":\"1\",\"back_to_top_style_color\":\"\",\"back_to_top_text\":\"Back to Top\",\"body_bg_image\":\"\",\"body_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"body_bg_image_overlay\":\"fav-transparent\",\"body_bg_color\":\"\",\"body_color\":\"\",\"body_title_color\":\"\",\"body_link_color\":\"\",\"body_link_hover_color\":\"\",\"notice_bg_style\":\"fav-module-block-color\",\"notice_bg_color\":\"FCF8E3\",\"notice_color\":\"8A6D3B\",\"notice_title_color\":\"\",\"notice_link_color\":\"\",\"notice_link_hover_color\":\"\",\"topbar_bg_style\":\"fav-module-block-color\",\"topbar_bg_image\":\"\",\"topbar_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"topbar_bg_image_overlay\":\"fav-transparent\",\"topbar_bg_color\":\"F2DEDE\",\"topbar_color\":\"A94442\",\"topbar_title_color\":\"\",\"topbar_link_color\":\"\",\"topbar_link_hover_color\":\"\",\"slide_width\":\"\",\"slide_bg_style\":\"fav-module-block-light\",\"slide_bg_image\":\"\",\"slide_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"slide_bg_image_overlay\":\"fav-transparent\",\"slide_bg_color\":\"F0F0F0\",\"slide_color\":\"\",\"slide_title_color\":\"\",\"slide_link_color\":\"\",\"slide_link_hover_color\":\"\",\"intro_bg_style\":\"fav-module-block-light\",\"intro_bg_image\":\"\",\"intro_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"intro_bg_image_overlay\":\"fav-overlay\",\"intro_bg_color\":\"\",\"intro_color\":\"\",\"intro_title_color\":\"\",\"intro_link_color\":\"\",\"intro_link_hover_color\":\"\",\"breadcrumbs_bg_style\":\"fav-module-block-light\",\"breadcrumbs_bg_image\":\"\",\"breadcrumbs_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"breadcrumbs_bg_image_overlay\":\"fav-transparent\",\"breadcrumbs_bg_color\":\"\",\"breadcrumbs_color\":\"\",\"breadcrumbs_title_color\":\"\",\"breadcrumbs_link_color\":\"\",\"breadcrumbs_link_hover_color\":\"\",\"lead_bg_style\":\"fav-module-block-light\",\"lead_bg_image\":\"\",\"lead_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"lead_bg_image_overlay\":\"fav-transparent\",\"lead_bg_color\":\"\",\"lead_color\":\"\",\"lead_title_color\":\"\",\"lead_link_color\":\"\",\"lead_link_hover_color\":\"\",\"promo_bg_style\":\"fav-module-block-light\",\"promo_bg_image\":\"\",\"promo_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"promo_bg_image_overlay\":\"fav-transparent\",\"promo_bg_color\":\"\",\"promo_color\":\"\",\"promo_title_color\":\"\",\"promo_link_color\":\"\",\"promo_link_hover_color\":\"\",\"prime_bg_style\":\"fav-module-block-light\",\"prime_bg_image\":\"\",\"prime_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"prime_bg_image_overlay\":\"fav-overlay\",\"prime_bg_color\":\"FFFFFF\",\"prime_color\":\"\",\"prime_title_color\":\"\",\"prime_link_color\":\"\",\"prime_link_hover_color\":\"\",\"showcase_bg_style\":\"fav-module-block-light\",\"showcase_bg_image\":\"\",\"showcase_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"showcase_bg_image_overlay\":\"fav-transparent\",\"showcase_bg_color\":\"111111\",\"showcase_color\":\"\",\"showcase_title_color\":\"\",\"showcase_link_color\":\"\",\"showcase_link_hover_color\":\"\",\"feature_bg_style\":\"fav-module-block-light\",\"feature_bg_image\":\"\",\"feature_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"feature_bg_image_overlay\":\"fav-transparent\",\"feature_bg_color\":\"\",\"feature_color\":\"\",\"feature_title_color\":\"\",\"feature_link_color\":\"\",\"feature_link_hover_color\":\"\",\"focus_bg_style\":\"fav-module-block-color\",\"focus_bg_image\":\"\",\"focus_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"focus_bg_image_overlay\":\"fav-transparent\",\"focus_bg_color\":\"\",\"focus_color\":\"\",\"focus_title_color\":\"\",\"focus_link_color\":\"\",\"focus_link_hover_color\":\"\",\"portfolio_bg_style\":\"fav-module-block-light\",\"portfolio_bg_image\":\"\",\"portfolio_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"portfolio_bg_image_overlay\":\"fav-transparent\",\"portfolio_bg_color\":\"\",\"portfolio_color\":\"\",\"portfolio_title_color\":\"\",\"portfolio_link_color\":\"\",\"portfolio_link_hover_color\":\"\",\"screen_bg_style\":\"fav-module-block-color\",\"screen_bg_image\":\"\",\"screen_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"screen_bg_image_overlay\":\"fav-transparent\",\"screen_bg_color\":\"F0F0F0\",\"screen_color\":\"\",\"screen_title_color\":\"444444\",\"screen_link_color\":\"\",\"screen_link_hover_color\":\"\",\"top_bg_style\":\"fav-module-block-light\",\"top_bg_image\":\"\",\"top_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"top_bg_image_overlay\":\"fav-transparent\",\"top_bg_color\":\"\",\"top_color\":\"\",\"top_title_color\":\"444444\",\"top_link_color\":\"\",\"top_link_hover_color\":\"\",\"maintop_bg_style\":\"fav-module-block-light\",\"maintop_bg_image\":\"\",\"maintop_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"maintop_bg_image_overlay\":\"fav-transparent\",\"maintop_bg_color\":\"\",\"maintop_color\":\"\",\"maintop_title_color\":\"\",\"maintop_link_color\":\"\",\"maintop_link_hover_color\":\"\",\"mainbottom_bg_style\":\"fav-module-block-light\",\"mainbottom_bg_image\":\"\",\"mainbottom_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"mainbottom_bg_image_overlay\":\"fav-transparent\",\"mainbottom_bg_color\":\"\",\"mainbottom_color\":\"\",\"mainbottom_title_color\":\"\",\"mainbottom_link_color\":\"\",\"mainbottom_link_hover_color\":\"\",\"bottom_bg_style\":\"fav-module-block-light\",\"bottom_bg_image\":\"\",\"bottom_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"bottom_bg_image_overlay\":\"fav-transparent\",\"bottom_bg_color\":\"\",\"bottom_color\":\"\",\"bottom_title_color\":\"\",\"bottom_link_color\":\"\",\"bottom_link_hover_color\":\"\",\"note_bg_style\":\"fav-module-block-dark\",\"note_bg_image\":\"\",\"note_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"note_bg_image_overlay\":\"fav-transparent\",\"note_bg_color\":\"\",\"note_color\":\"\",\"note_title_color\":\"\",\"note_link_color\":\"\",\"note_link_hover_color\":\"\",\"base_bg_style\":\"fav-module-block-light\",\"base_bg_image\":\"\",\"base_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"base_bg_image_overlay\":\"fav-overlay\",\"base_bg_color\":\"\",\"base_color\":\"\",\"base_title_color\":\"\",\"base_link_color\":\"\",\"base_link_hover_color\":\"\",\"block_bg_style\":\"fav-module-block-light\",\"block_bg_image\":\"\",\"block_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"block_bg_image_overlay\":\"fav-transparent\",\"block_bg_color\":\"\",\"block_color\":\"\",\"block_title_color\":\"\",\"block_link_color\":\"\",\"block_link_hover_color\":\"\",\"user_bg_style\":\"fav-module-block-color\",\"user_bg_image\":\"\",\"user_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"user_bg_image_overlay\":\"fav-transparent\",\"user_bg_color\":\"F0F0F0\",\"user_color\":\"\",\"user_title_color\":\"\",\"user_link_color\":\"\",\"user_link_hover_color\":\"\",\"footer_bg_style\":\"fav-module-block-color\",\"footer_bg_image\":\"\",\"footer_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"footer_bg_image_overlay\":\"fav-transparent\",\"footer_bg_color\":\"58C3E5\",\"footer_color\":\"FFFFFF\",\"footer_title_color\":\"\",\"footer_link_color\":\"FFFFFF\",\"footer_link_hover_color\":\"\",\"show_default_logo\":\"1\",\"default_logo\":\"logo.png\",\"default_logo_img_alt\":\"Reactome Pathway Database\",\"default_logo_padding\":\"\",\"default_logo_margin\":\"\",\"show_media_logo\":\"0\",\"media_logo\":\"\",\"media_logo_img_alt\":\"Reactome Pathway Database\",\"media_logo_padding\":\"\",\"media_logo_margin\":\"\",\"show_text_logo\":\"0\",\"text_logo\":\"REACTOME\",\"text_logo_color\":\"58C3E5\",\"text_logo_font_size\":\"42px\",\"text_logo_google_font\":\"Source Sans Pro\",\"text_logo_google_font_weight\":\"700\",\"text_logo_google_font_style\":\"normal\",\"text_logo_line_height\":\"\",\"text_logo_padding\":\"\",\"text_logo_margin\":\"6px 0 0 0\",\"show_slogan\":\"0\",\"slogan\":\"A curated Pathway Database\",\"slogan_color\":\"\",\"slogan_font_size\":\"\",\"slogan_line_height\":\"14px\",\"slogan_padding\":\"\",\"slogan_margin\":\"\",\"show_retina_logo\":\"0\",\"retina_logo\":\"\",\"retina_logo_height\":\"52px\",\"retina_logo_width\":\"188px\",\"retina_logo_img_alt\":\"Reactome Pathway Database\",\"retina_logo_padding\":\"0px\",\"retina_logo_margin\":\"0px\",\"mobile_nav_color\":\"favth-navbar-default\",\"show_mobile_menu_text\":\"0\",\"mobile_menu_text\":\"MENU\",\"mobile_paragraph_font_size\":\"12px\",\"mobile_paragraph_line_height\":\"\",\"mobile_title_font_size\":\"\",\"analytics_code\":\"\"}');
/*!40000 ALTER TABLE `rlp_template_styles` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_ucm_base`
--

DROP TABLE IF EXISTS `rlp_ucm_base`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_ucm_base` (
  `ucm_id` int(10) unsigned NOT NULL,
  `ucm_item_id` int(10) NOT NULL,
  `ucm_type_id` int(11) NOT NULL,
  `ucm_language_id` int(11) NOT NULL,
  PRIMARY KEY (`ucm_id`),
  KEY `idx_ucm_item_id` (`ucm_item_id`),
  KEY `idx_ucm_type_id` (`ucm_type_id`),
  KEY `idx_ucm_language_id` (`ucm_language_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_ucm_base`
--

LOCK TABLES `rlp_ucm_base` WRITE;
/*!40000 ALTER TABLE `rlp_ucm_base` DISABLE KEYS */;
INSERT INTO `rlp_ucm_base` VALUES (1,54,1,0),(2,90,1,0),(3,78,1,0),(4,77,1,0),(5,74,1,0),(6,72,1,0),(7,71,1,0),(8,98,1,1),(9,102,1,1),(10,103,1,1),(11,106,1,1),(12,116,1,1),(13,125,1,1),(14,127,1,1),(15,137,1,1),(16,142,1,1);
/*!40000 ALTER TABLE `rlp_ucm_base` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_ucm_content`
--

DROP TABLE IF EXISTS `rlp_ucm_content`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_ucm_content` (
  `core_content_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `core_type_alias` varchar(400) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'FK to the content types table',
  `core_title` varchar(400) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `core_alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `core_body` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `core_state` tinyint(1) NOT NULL DEFAULT '0',
  `core_checked_out_time` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '0000-00-00 00:00:00',
  `core_checked_out_user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `core_access` int(10) unsigned NOT NULL DEFAULT '0',
  `core_params` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `core_featured` tinyint(4) unsigned NOT NULL DEFAULT '0',
  `core_metadata` varchar(2048) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'JSON encoded metadata properties.',
  `core_created_user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `core_created_by_alias` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `core_created_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `core_modified_user_id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'Most recent user that modified',
  `core_modified_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `core_language` char(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `core_publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `core_publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `core_content_item_id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'ID from the individual type table',
  `asset_id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'FK to the #__assets table.',
  `core_images` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `core_urls` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `core_hits` int(10) unsigned NOT NULL DEFAULT '0',
  `core_version` int(10) unsigned NOT NULL DEFAULT '1',
  `core_ordering` int(11) NOT NULL DEFAULT '0',
  `core_metakey` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `core_metadesc` text COLLATE utf8mb4_unicode_ci NOT NULL,
  `core_catid` int(10) unsigned NOT NULL DEFAULT '0',
  `core_xreference` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'A reference to enable linkages to external data sets.',
  `core_type_id` int(10) unsigned NOT NULL DEFAULT '0',
  PRIMARY KEY (`core_content_id`),
  KEY `tag_idx` (`core_state`,`core_access`),
  KEY `idx_access` (`core_access`),
  KEY `idx_language` (`core_language`),
  KEY `idx_modified_time` (`core_modified_time`),
  KEY `idx_created_time` (`core_created_time`),
  KEY `idx_core_modified_user_id` (`core_modified_user_id`),
  KEY `idx_core_checked_out_user_id` (`core_checked_out_user_id`),
  KEY `idx_core_created_user_id` (`core_created_user_id`),
  KEY `idx_core_type_id` (`core_type_id`),
  KEY `idx_alias` (`core_alias`(100)),
  KEY `idx_title` (`core_title`(100)),
  KEY `idx_content_type` (`core_type_alias`(100))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Contains core content data in name spaced fields';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_ucm_content`
--

LOCK TABLES `rlp_ucm_content` WRITE;
/*!40000 ALTER TABLE `rlp_ucm_content` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_ucm_content` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_ucm_history`
--

DROP TABLE IF EXISTS `rlp_ucm_history`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_ucm_history` (
  `version_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `ucm_item_id` int(10) unsigned NOT NULL,
  `ucm_type_id` int(10) unsigned NOT NULL,
  `version_note` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'Optional version name',
  `save_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `editor_user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `character_count` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'Number of characters in this version.',
  `sha1_hash` varchar(50) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'SHA1 hash of the version_data column.',
  `version_data` longtext COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'json-encoded string of version data',
  `keep_forever` tinyint(4) NOT NULL DEFAULT '0' COMMENT '0=auto delete; 1=keep',
  PRIMARY KEY (`version_id`),
  KEY `idx_ucm_item_id` (`ucm_type_id`,`ucm_item_id`),
  KEY `idx_save_date` (`save_date`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_ucm_history`
--

LOCK TABLES `rlp_ucm_history` WRITE;
/*!40000 ALTER TABLE `rlp_ucm_history` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_ucm_history` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_update_sites`
--

DROP TABLE IF EXISTS `rlp_update_sites`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_update_sites` (
  `update_site_id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(100) COLLATE utf8mb4_unicode_ci DEFAULT '',
  `type` varchar(20) COLLATE utf8mb4_unicode_ci DEFAULT '',
  `location` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `enabled` int(11) DEFAULT '0',
  `last_check_timestamp` bigint(20) DEFAULT '0',
  `extra_query` varchar(1000) COLLATE utf8mb4_unicode_ci DEFAULT '',
  PRIMARY KEY (`update_site_id`)
) ENGINE=InnoDB AUTO_INCREMENT=63 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Update Sites';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_update_sites`
--

LOCK TABLES `rlp_update_sites` WRITE;
/*!40000 ALTER TABLE `rlp_update_sites` DISABLE KEYS */;
INSERT INTO `rlp_update_sites` VALUES (1,'Joomla! Core','collection','https://update.joomla.org/core/list.xml',1,NULL,''),(3,'Accredited Joomla! Translations','collection','https://update.joomla.org/language/translationlist_3.xml',1,NULL,''),(4,'Joomla! Update Component Update Site','extension','https://update.joomla.org/core/extensions/com_joomlaupdate.xml',1,NULL,''),(9,'COM_EASYJOOMLABACKUP','extension','https://check.kubik-rubik.de/updates/com_easyjoomlabackup.xml',1,NULL,''),(12,'Regular Labs - Tabs','extension','https://download.regularlabs.com/updates.xml?e=tabs&type=.xml',1,NULL,''),(13,'Regular Labs - Sliders','extension','https://download.regularlabs.com/updates.xml?e=sliders&type=.xml',1,NULL,''),(14,'AlterBrains Updates','collection','http://www.alterbrains.com/update.xml',1,NULL,''),(16,'','collection','http://www.joomultra.com/service/tracking/list.xml',0,NULL,''),(17,'Regular Labs - Sourcerer','extension','https://download.regularlabs.com/updates.xml?e=sourcerer&type=.xml',1,NULL,''),(23,'Alledia Framework','extension','https://deploy.ostraining.com/client/update/free/stable/lib_allediaframework',1,NULL,''),(24,'Regular Labs - Modals','extension','https://download.regularlabs.com/updates.xml?e=modals&type=.xml',1,NULL,''),(25,'FavPromote Updates','extension','http://updates.favthemes.com/extensions/favpromote.xml',1,NULL,''),(27,'FavGlyph Updates','extension','http://updates.favthemes.com/extensions/favglyph.xml',1,NULL,''),(62,'JCE Editor Package','extension','https://cdn.joomlacontenteditor.net/updates/xml/editor/pkg_jce.xml',1,NULL,'');
/*!40000 ALTER TABLE `rlp_update_sites` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_update_sites_extensions`
--

DROP TABLE IF EXISTS `rlp_update_sites_extensions`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_update_sites_extensions` (
  `update_site_id` int(11) NOT NULL DEFAULT '0',
  `extension_id` int(11) NOT NULL DEFAULT '0',
  PRIMARY KEY (`update_site_id`,`extension_id`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Links extensions to update sites';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_update_sites_extensions`
--

LOCK TABLES `rlp_update_sites_extensions` WRITE;
/*!40000 ALTER TABLE `rlp_update_sites_extensions` DISABLE KEYS */;
INSERT INTO `rlp_update_sites_extensions` VALUES (1,700),(3,802),(4,28),(9,10208),(12,10227),(13,10231),(14,10232),(16,10235),(17,10239),(23,10250),(24,10254),(25,10182),(27,10180),(62,10200);
/*!40000 ALTER TABLE `rlp_update_sites_extensions` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_updates`
--

DROP TABLE IF EXISTS `rlp_updates`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_updates` (
  `update_id` int(11) NOT NULL AUTO_INCREMENT,
  `update_site_id` int(11) DEFAULT '0',
  `extension_id` int(11) DEFAULT '0',
  `name` varchar(100) COLLATE utf8mb4_unicode_ci DEFAULT '',
  `description` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `element` varchar(100) COLLATE utf8mb4_unicode_ci DEFAULT '',
  `type` varchar(20) COLLATE utf8mb4_unicode_ci DEFAULT '',
  `folder` varchar(20) COLLATE utf8mb4_unicode_ci DEFAULT '',
  `client_id` tinyint(3) DEFAULT '0',
  `version` varchar(32) COLLATE utf8mb4_unicode_ci DEFAULT '',
  `data` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `detailsurl` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `infourl` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `extra_query` varchar(1000) COLLATE utf8mb4_unicode_ci DEFAULT '',
  PRIMARY KEY (`update_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Available Updates';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_updates`
--

LOCK TABLES `rlp_updates` WRITE;
/*!40000 ALTER TABLE `rlp_updates` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_updates` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_user_keys`
--

DROP TABLE IF EXISTS `rlp_user_keys`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_user_keys` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `user_id` varchar(150) COLLATE utf8mb4_unicode_ci NOT NULL,
  `token` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
  `series` varchar(191) COLLATE utf8mb4_unicode_ci NOT NULL,
  `invalid` tinyint(4) NOT NULL,
  `time` varchar(200) COLLATE utf8mb4_unicode_ci NOT NULL,
  `uastring` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `series` (`series`),
  KEY `user_id` (`user_id`)
) ENGINE=InnoDB AUTO_INCREMENT=4 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_user_keys`
--

LOCK TABLES `rlp_user_keys` WRITE;
/*!40000 ALTER TABLE `rlp_user_keys` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_user_keys` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_user_notes`
--

DROP TABLE IF EXISTS `rlp_user_notes`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_user_notes` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `catid` int(10) unsigned NOT NULL DEFAULT '0',
  `subject` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `body` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `state` tinyint(3) NOT NULL DEFAULT '0',
  `checked_out` int(10) unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_user_id` int(10) unsigned NOT NULL DEFAULT '0',
  `created_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_user_id` int(10) unsigned NOT NULL,
  `modified_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `review_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  PRIMARY KEY (`id`),
  KEY `idx_user_id` (`user_id`),
  KEY `idx_category_id` (`catid`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_user_notes`
--

LOCK TABLES `rlp_user_notes` WRITE;
/*!40000 ALTER TABLE `rlp_user_notes` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_user_notes` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_user_profiles`
--

DROP TABLE IF EXISTS `rlp_user_profiles`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_user_profiles` (
  `user_id` int(11) NOT NULL,
  `profile_key` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL,
  `profile_value` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `ordering` int(11) NOT NULL DEFAULT '0',
  UNIQUE KEY `idx_user_id_profile_key` (`user_id`,`profile_key`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Simple user profile storage table';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_user_profiles`
--

LOCK TABLES `rlp_user_profiles` WRITE;
/*!40000 ALTER TABLE `rlp_user_profiles` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_user_profiles` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_user_usergroup_map`
--

DROP TABLE IF EXISTS `rlp_user_usergroup_map`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_user_usergroup_map` (
  `user_id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'Foreign Key to #__users.id',
  `group_id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'Foreign Key to #__usergroups.id',
  PRIMARY KEY (`user_id`,`group_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_user_usergroup_map`
--

LOCK TABLES `rlp_user_usergroup_map` WRITE;
/*!40000 ALTER TABLE `rlp_user_usergroup_map` DISABLE KEYS */;
INSERT INTO `rlp_user_usergroup_map` VALUES (156,8),(159,2),(162,2),(163,2),(165,2),(166,2),(167,2),(168,2),(169,2),(170,2),(171,2),(172,2),(173,2),(174,2),(175,2),(176,2),(177,2),(178,2),(179,2),(180,2);
/*!40000 ALTER TABLE `rlp_user_usergroup_map` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_usergroups`
--

DROP TABLE IF EXISTS `rlp_usergroups`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_usergroups` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Primary Key',
  `parent_id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'Adjacency List Reference Id',
  `lft` int(11) NOT NULL DEFAULT '0' COMMENT 'Nested set lft.',
  `rgt` int(11) NOT NULL DEFAULT '0' COMMENT 'Nested set rgt.',
  `title` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  PRIMARY KEY (`id`),
  UNIQUE KEY `idx_usergroup_parent_title_lookup` (`parent_id`,`title`),
  KEY `idx_usergroup_title_lookup` (`title`),
  KEY `idx_usergroup_adjacency_lookup` (`parent_id`),
  KEY `idx_usergroup_nested_set_lookup` (`lft`,`rgt`) USING BTREE
) ENGINE=InnoDB AUTO_INCREMENT=12 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_usergroups`
--

LOCK TABLES `rlp_usergroups` WRITE;
/*!40000 ALTER TABLE `rlp_usergroups` DISABLE KEYS */;
INSERT INTO `rlp_usergroups` VALUES (1,0,1,22,'Public'),(2,1,8,19,'Registered'),(3,2,9,18,'Author'),(4,3,10,17,'Editor'),(5,4,13,14,'Publisher'),(6,1,4,7,'Manager'),(7,6,5,6,'Administrator'),(8,1,20,21,'Super Users'),(9,1,2,3,'Guest'),(10,4,15,16,'SyncTool'),(11,4,11,12,'FrontEnd Editor');
/*!40000 ALTER TABLE `rlp_usergroups` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_users`
--

DROP TABLE IF EXISTS `rlp_users`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_users` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(400) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `username` varchar(150) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `email` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `password` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `block` tinyint(4) NOT NULL DEFAULT '0',
  `sendEmail` tinyint(4) DEFAULT '0',
  `registerDate` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `lastvisitDate` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `activation` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `params` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
  `lastResetTime` datetime NOT NULL DEFAULT '0000-00-00 00:00:00' COMMENT 'Date of last password reset',
  `resetCount` int(11) NOT NULL DEFAULT '0' COMMENT 'Count of password resets since lastResetTime',
  `otpKey` varchar(1000) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'Two factor authentication encrypted keys',
  `otep` varchar(1000) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'One time emergency passwords',
  `requireReset` tinyint(4) NOT NULL DEFAULT '0' COMMENT 'Require user to reset password on next login',
  PRIMARY KEY (`id`),
  KEY `idx_block` (`block`),
  KEY `username` (`username`),
  KEY `email` (`email`),
  KEY `idx_name` (`name`(100))
) ENGINE=InnoDB AUTO_INCREMENT=181 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_users`
--

LOCK TABLES `rlp_users` WRITE;
/*!40000 ALTER TABLE `rlp_users` DISABLE KEYS */;
INSERT INTO `rlp_users` VALUES (156,'Joomla Admin','admin','change-me@nodomain.com','$2y$10$suDIc1tF2MApag9nthYrOez4G9mMDT.ushhi6CeEspDp5aKQjLnlC',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','0','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(158,'Name 158','unknown158','invalid158@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,1,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(159,'Name 159','unknown159','invalid159@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(161,'Name 161','unknown161','invalid161@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(162,'Name 162','unknown162','invalid162@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(163,'Name 163','unknown163','invalid163@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(165,'Name 165','unknown165','invalid165@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(166,'Name 166','unknown166','invalid166@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(167,'Name 167','unknown167','invalid167@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(168,'Name 168','unknown168','invalid168@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(169,'Name 169','unknown169','invalid169@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(170,'Name 170','unknown170','invalid170@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(171,'Name 171','unknown171','invalid171@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(172,'Name 172','unknown172','invalid172@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(173,'Name 173','unknown173','invalid173@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(174,'Name 174','unknown174','invalid174@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(175,'Name 175','unknown175','invalid175@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(176,'Name 176','unknown176','invalid176@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(177,'Name 177','unknown177','invalid177@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(178,'Name 178','unknown178','invalid178@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(179,'Name 179','unknown179','invalid179@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0),(180,'Name 180','unknown180','invalid180@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0);
/*!40000 ALTER TABLE `rlp_users` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_utf8_conversion`
--

DROP TABLE IF EXISTS `rlp_utf8_conversion`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_utf8_conversion` (
  `converted` tinyint(4) NOT NULL DEFAULT '0'
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_utf8_conversion`
--

LOCK TABLES `rlp_utf8_conversion` WRITE;
/*!40000 ALTER TABLE `rlp_utf8_conversion` DISABLE KEYS */;
INSERT INTO `rlp_utf8_conversion` VALUES (2);
/*!40000 ALTER TABLE `rlp_utf8_conversion` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_viewlevels`
--

DROP TABLE IF EXISTS `rlp_viewlevels`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_viewlevels` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Primary Key',
  `title` varchar(100) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `ordering` int(11) NOT NULL DEFAULT '0',
  `rules` varchar(5120) COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'JSON encoded access control.',
  PRIMARY KEY (`id`),
  UNIQUE KEY `idx_assetgroup_title_lookup` (`title`)
) ENGINE=InnoDB AUTO_INCREMENT=8 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_viewlevels`
--

LOCK TABLES `rlp_viewlevels` WRITE;
/*!40000 ALTER TABLE `rlp_viewlevels` DISABLE KEYS */;
INSERT INTO `rlp_viewlevels` VALUES (1,'Public',1,'[1]'),(2,'Registered',3,'[6,2,8]'),(3,'Special',4,'[6,3,8]'),(5,'Guest',2,'[9]'),(6,'Super Users',5,'[8]'),(7,'Synchronization Tool',6,'[10]');
/*!40000 ALTER TABLE `rlp_viewlevels` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_wf_profiles`
--

DROP TABLE IF EXISTS `rlp_wf_profiles`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `rlp_wf_profiles` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(255) NOT NULL,
  `description` text NOT NULL,
  `users` text NOT NULL,
  `types` text NOT NULL,
  `components` text NOT NULL,
  `area` tinyint(3) NOT NULL,
  `device` varchar(255) NOT NULL,
  `rows` text NOT NULL,
  `plugins` text NOT NULL,
  `published` tinyint(3) NOT NULL,
  `ordering` int(11) NOT NULL,
  `checked_out` tinyint(3) NOT NULL,
  `checked_out_time` datetime NOT NULL,
  `params` text NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_wf_profiles`
--

LOCK TABLES `rlp_wf_profiles` WRITE;
/*!40000 ALTER TABLE `rlp_wf_profiles` DISABLE KEYS */;
INSERT INTO `rlp_wf_profiles` VALUES (1,'Default','Default Profile for all users','','6,7,2,3,4,5,8','',0,'phone,tablet,desktop','help,newdocument,undo,redo,spacer,bold,italic,underline,strikethrough,justifyfull,justifycenter,justifyleft,justifyright,spacer,blockquote,formatselect,styleselect,removeformat,table,cleanup;searchreplace,fontselect,fontsizeselect,fontcolor,spacer,clipboard,indent,outdent,lists,sub,sup,textcase,directionality,charmap,style,hr,fullscreen,xhtmlxtras;visualaid,visualchars,visualblocks,nonbreaking,anchor,unlink,link,imgmanager,spellchecker,article,emotions,layer,autosave','formatselect,styleselect,table,cleanup,searchreplace,fontselect,fontsizeselect,fontcolor,clipboard,lists,textcase,directionality,charmap,style,hr,fullscreen,xhtmlxtras,visualchars,visualblocks,nonbreaking,anchor,link,imgmanager,spellchecker,article,emotions,layer,autosave,browser,contextmenu,inlinepopups,media,preview,source',1,1,0,'0000-00-00 00:00:00','{\"fontselect\":{\"fonts\":\"{\\\"Andale Mono\\\":\\\"andale mono,times\\\",\\\"Arial\\\":\\\"arial,helvetica,sans-serif\\\",\\\"Arial Black\\\":\\\"arial black,avant garde\\\",\\\"Book Antiqua\\\":\\\"book antiqua,palatino\\\",\\\"Comic Sans MS\\\":\\\"comic sans ms,sans-serif\\\",\\\"Courier New\\\":\\\"courier new,courier\\\",\\\"Georgia\\\":\\\"georgia,palatino\\\",\\\"Helvetica\\\":\\\"helvetica\\\",\\\"Impact\\\":\\\"impact,chicago\\\",\\\"Symbol\\\":\\\"symbol\\\",\\\"Tahoma\\\":\\\"tahoma,arial,helvetica,sans-serif\\\",\\\"Terminal\\\":\\\"terminal,monaco\\\",\\\"Times New Roman\\\":\\\"times new roman,times\\\",\\\"Trebuchet MS\\\":\\\"trebuchet ms,geneva\\\",\\\"Verdana\\\":\\\"verdana,geneva\\\",\\\"Webdings\\\":\\\"webdings\\\",\\\"Wingdings\\\":\\\"wingdings,zapf dingbats\\\"}\"},\"editor\":{\"verify_html\":\"\",\"schema\":\"\",\"allow_css\":\"1\",\"allow_javascript\":\"1\",\"allow_php\":\"1\",\"forced_root_block\":\"\",\"profile_content_css\":\"0\",\"profile_content_css_custom\":\"\\/media\\/font-awesome-4.7.0\\/css\\/font-awesome.css\"},\"media\":{\"iframes\":\"1\"}}'),(2,'Front End','Sample Front-end Profile','','3,4,5','',1,'desktop,tablet,phone','help,newdocument,undo,redo,spacer,bold,italic,underline,strikethrough,justifyfull,justifycenter,justifyleft,justifyright,spacer,formatselect,styleselect;clipboard,searchreplace,indent,outdent,lists,cleanup,charmap,removeformat,hr,sub,sup,textcase,nonbreaking,visualchars,visualblocks;fullscreen,print,visualaid,style,xhtmlxtras,anchor,unlink,link,imgmanager,spellchecker,article','charmap,contextmenu,inlinepopups,help,clipboard,searchreplace,fullscreen,preview,print,style,textcase,nonbreaking,visualchars,visualblocks,xhtmlxtras,imgmanager,anchor,link,spellchecker,article,lists,formatselect,styleselect,hr',0,2,0,'0000-00-00 00:00:00',''),(3,'Blogger','Simple Blogging Profile','','3,4,5,6,8,7','',0,'desktop,tablet,phone','bold,italic,strikethrough,lists,blockquote,spacer,justifyleft,justifycenter,justifyright,spacer,link,unlink,imgmanager,article,spellchecker,fullscreen,kitchensink;formatselect,styleselect,underline,justifyfull,clipboard,removeformat,charmap,indent,outdent,undo,redo,help','link,imgmanager,article,spellchecker,fullscreen,kitchensink,clipboard,contextmenu,inlinepopups,lists,formatselect,styleselect,textpattern',0,3,0,'0000-00-00 00:00:00','{\"editor\":{\"toggle\":\"0\"}}'),(4,'Mobile','Sample Mobile Profile','','3,4,5,6,8,7','',0,'tablet,phone','undo,redo,spacer,bold,italic,underline,formatselect,spacer,justifyleft,justifycenter,justifyfull,justifyright,spacer,fullscreen,kitchensink;styleselect,lists,spellchecker,article,link,unlink','fullscreen,kitchensink,spellchecker,article,link,inlinepopups,lists,formatselect,styleselect,textpattern',0,4,0,'0000-00-00 00:00:00','{\"editor\":{\"toolbar_theme\":\"mobile\",\"resizing\":\"0\",\"resize_horizontal\":\"0\",\"resizing_use_cookie\":\"0\",\"toggle\":\"0\",\"links\":{\"popups\":{\"default\":\"\",\"jcemediabox\":{\"enable\":\"0\"},\"window\":{\"enable\":\"0\"}}}}}'),(5,'Markdown','Sample Markdown Profile','','6,7,3,4,5,8','',0,'desktop,tablet,phone','fullscreen,justifyleft,justifycenter,justifyfull,justifyright,link,unlink,imgmanager,styleselect','fullscreen,link,imgmanager,styleselect,inlinepopups,media,textpattern',0,5,0,'0000-00-00 00:00:00','{\"editor\":{\"toolbar_theme\":\"mobile\"}}');
/*!40000 ALTER TABLE `rlp_wf_profiles` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed
