-- MySQL dump 10.13  Distrib 8.0.39, for Linux (x86_64)
--
-- Host: localhost    Database: website
-- ------------------------------------------------------
-- Server version	8.0.39-0ubuntu0.20.04.1

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!50503 SET NAMES utf8mb4 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `rlp_action_log_config`
--

DROP TABLE IF EXISTS `rlp_action_log_config`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_action_log_config` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `type_title` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `type_alias` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `id_holder` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `title_holder` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `table_name` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `text_prefix` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=20 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_action_log_config`
--

LOCK TABLES `rlp_action_log_config` WRITE;
/*!40000 ALTER TABLE `rlp_action_log_config` DISABLE KEYS */;
INSERT INTO `rlp_action_log_config` VALUES (1,'article','com_content.article','id','title','#__content','PLG_ACTIONLOG_JOOMLA'),(2,'article','com_content.form','id','title','#__content','PLG_ACTIONLOG_JOOMLA'),(3,'banner','com_banners.banner','id','name','#__banners','PLG_ACTIONLOG_JOOMLA'),(4,'user_note','com_users.note','id','subject','#__user_notes','PLG_ACTIONLOG_JOOMLA'),(5,'media','com_media.file','','name','','PLG_ACTIONLOG_JOOMLA'),(6,'category','com_categories.category','id','title','#__categories','PLG_ACTIONLOG_JOOMLA'),(7,'menu','com_menus.menu','id','title','#__menu_types','PLG_ACTIONLOG_JOOMLA'),(8,'menu_item','com_menus.item','id','title','#__menu','PLG_ACTIONLOG_JOOMLA'),(9,'newsfeed','com_newsfeeds.newsfeed','id','name','#__newsfeeds','PLG_ACTIONLOG_JOOMLA'),(10,'link','com_redirect.link','id','old_url','#__redirect_links','PLG_ACTIONLOG_JOOMLA'),(11,'tag','com_tags.tag','id','title','#__tags','PLG_ACTIONLOG_JOOMLA'),(12,'style','com_templates.style','id','title','#__template_styles','PLG_ACTIONLOG_JOOMLA'),(13,'plugin','com_plugins.plugin','extension_id','name','#__extensions','PLG_ACTIONLOG_JOOMLA'),(14,'component_config','com_config.component','extension_id','name','','PLG_ACTIONLOG_JOOMLA'),(15,'contact','com_contact.contact','id','name','#__contact_details','PLG_ACTIONLOG_JOOMLA'),(16,'module','com_modules.module','id','title','#__modules','PLG_ACTIONLOG_JOOMLA'),(17,'access_level','com_users.level','id','title','#__viewlevels','PLG_ACTIONLOG_JOOMLA'),(18,'banner_client','com_banners.client','id','name','#__banner_clients','PLG_ACTIONLOG_JOOMLA'),(19,'application_config','com_config.application','','name','','PLG_ACTIONLOG_JOOMLA');
/*!40000 ALTER TABLE `rlp_action_log_config` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_action_logs`
--

DROP TABLE IF EXISTS `rlp_action_logs`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_action_logs` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `message_language_key` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `message` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `log_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `extension` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `user_id` int NOT NULL DEFAULT '0',
  `item_id` int NOT NULL DEFAULT '0',
  `ip_address` varchar(40) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '0.0.0.0',
  PRIMARY KEY (`id`),
  KEY `idx_user_id` (`user_id`),
  KEY `idx_user_id_logdate` (`user_id`,`log_date`),
  KEY `idx_user_id_extension` (`user_id`,`extension`),
  KEY `idx_extension_item_id` (`extension`,`item_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_action_logs`
--

LOCK TABLES `rlp_action_logs` WRITE;
/*!40000 ALTER TABLE `rlp_action_logs` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_action_logs` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_action_logs_extensions`
--

DROP TABLE IF EXISTS `rlp_action_logs_extensions`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_action_logs_extensions` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `extension` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=19 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_action_logs_extensions`
--

LOCK TABLES `rlp_action_logs_extensions` WRITE;
/*!40000 ALTER TABLE `rlp_action_logs_extensions` DISABLE KEYS */;
INSERT INTO `rlp_action_logs_extensions` VALUES (1,'com_banners'),(2,'com_cache'),(3,'com_categories'),(4,'com_config'),(5,'com_contact'),(6,'com_content'),(7,'com_installer'),(8,'com_media'),(9,'com_menus'),(10,'com_messages'),(11,'com_modules'),(12,'com_newsfeeds'),(13,'com_plugins'),(14,'com_redirect'),(15,'com_tags'),(16,'com_templates'),(17,'com_users'),(18,'com_checkin');
/*!40000 ALTER TABLE `rlp_action_logs_extensions` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_action_logs_users`
--

DROP TABLE IF EXISTS `rlp_action_logs_users`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_action_logs_users` (
  `user_id` int unsigned NOT NULL,
  `notify` tinyint unsigned NOT NULL,
  `extensions` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`user_id`),
  KEY `idx_notify` (`notify`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_action_logs_users`
--

LOCK TABLES `rlp_action_logs_users` WRITE;
/*!40000 ALTER TABLE `rlp_action_logs_users` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_action_logs_users` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_assets`
--

DROP TABLE IF EXISTS `rlp_assets`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_assets` (
  `id` int unsigned NOT NULL AUTO_INCREMENT COMMENT 'Primary Key',
  `parent_id` int NOT NULL DEFAULT '0' COMMENT 'Nested set parent.',
  `lft` int NOT NULL DEFAULT '0' COMMENT 'Nested set lft.',
  `rgt` int NOT NULL DEFAULT '0' COMMENT 'Nested set rgt.',
  `level` int unsigned NOT NULL COMMENT 'The cached level in the nested tree.',
  `name` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The unique name for the asset.\n',
  `title` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The descriptive title for the asset.',
  `rules` varchar(5120) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'JSON encoded access control.',
  PRIMARY KEY (`id`),
  UNIQUE KEY `idx_asset_name` (`name`),
  KEY `idx_lft_rgt` (`lft`,`rgt`),
  KEY `idx_parent_id` (`parent_id`)
) ENGINE=InnoDB AUTO_INCREMENT=972 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_assets`
--

LOCK TABLES `rlp_assets` WRITE;
/*!40000 ALTER TABLE `rlp_assets` DISABLE KEYS */;
INSERT INTO `rlp_assets` VALUES (1,0,0,819,0,'root.1','Root Asset','{\"core.login.site\":{\"6\":1,\"2\":1},\"core.login.admin\":{\"6\":1,\"10\":0},\"core.login.offline\":{\"6\":1,\"10\":0},\"core.admin\":{\"8\":1},\"core.manage\":{\"7\":1},\"core.create\":{\"6\":1,\"3\":1,\"10\":1},\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1},\"core.edit.own\":{\"6\":1,\"3\":1}}'),(2,1,1,2,1,'com_admin','com_admin','{}'),(3,1,3,6,1,'com_banners','com_banners','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1},\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(4,1,7,8,1,'com_cache','com_cache','{\"core.admin\":{\"7\":1},\"core.manage\":{\"7\":1}}'),(5,1,9,10,1,'com_checkin','com_checkin','{\"core.admin\":{\"7\":1},\"core.manage\":{\"7\":1}}'),(6,1,11,12,1,'com_config','com_config','{}'),(7,1,13,16,1,'com_contact','com_contact','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1},\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"core.edit.own\":[]}'),(8,1,17,506,1,'com_content','com_content','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(9,1,507,508,1,'com_cpanel','com_cpanel','{}'),(10,1,509,510,1,'com_installer','com_installer','{\"core.admin\":[],\"core.manage\":{\"7\":0},\"core.delete\":{\"7\":0},\"core.edit.state\":{\"7\":0}}'),(11,1,511,512,1,'com_languages','com_languages','{\"core.admin\":{\"7\":1},\"core.manage\":[],\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(12,1,513,514,1,'com_login','com_login','{}'),(13,1,515,516,1,'com_mailto','com_mailto','{}'),(14,1,517,518,1,'com_massmail','com_massmail','{}'),(15,1,519,520,1,'com_media','com_media','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":{\"5\":1}}'),(16,1,521,528,1,'com_menus','com_menus','{\"core.admin\":{\"7\":1},\"core.manage\":[],\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(17,1,529,530,1,'com_messages','com_messages','{\"core.admin\":{\"7\":1},\"core.manage\":{\"7\":1}}'),(18,1,531,684,1,'com_modules','com_modules','{\"core.admin\":{\"7\":1},\"core.manage\":[],\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(19,1,685,688,1,'com_newsfeeds','com_newsfeeds','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1},\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"core.edit.own\":[]}'),(20,1,689,690,1,'com_plugins','com_plugins','{\"core.admin\":{\"7\":1},\"core.manage\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(21,1,691,692,1,'com_redirect','com_redirect','{\"core.admin\":{\"7\":1},\"core.manage\":[]}'),(22,1,693,694,1,'com_search','com_search','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1}}'),(23,1,695,696,1,'com_templates','com_templates','{\"core.admin\":{\"7\":1},\"core.manage\":[],\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(24,1,697,700,1,'com_users','com_users','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":[],\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(26,1,701,702,1,'com_wrapper','com_wrapper','{}'),(27,8,18,45,2,'com_content.category.2','Uncategorised','{\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"core.edit.own\":[]}'),(28,3,4,5,2,'com_banners.category.3','Uncategorised','{\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(29,7,14,15,2,'com_contact.category.4','Uncategorised','{\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"core.edit.own\":[]}'),(30,19,686,687,2,'com_newsfeeds.category.5','Uncategorised','{\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"core.edit.own\":[]}'),(32,24,698,699,1,'com_users.category.7','Uncategorised','{\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(33,1,703,704,1,'com_finder','com_finder','{\"core.admin\":{\"7\":1},\"core.manage\":{\"6\":1},\"core.create\":[],\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(34,1,705,706,1,'com_joomlaupdate','com_joomlaupdate','{\"core.admin\":[],\"core.manage\":[],\"core.delete\":[],\"core.edit.state\":[]}'),(35,1,707,708,1,'com_tags','com_tags','{\"core.admin\":[],\"core.manage\":[],\"core.manage\":[],\"core.delete\":[],\"core.edit.state\":[]}'),(36,1,709,710,1,'com_contenthistory','com_contenthistory','{}'),(37,1,711,712,1,'com_ajax','com_ajax','{}'),(38,1,713,714,1,'com_postinstall','com_postinstall','{}'),(39,18,532,533,2,'com_modules.module.1','Main Menu','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(40,18,534,535,2,'com_modules.module.2','Login','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(41,18,536,537,2,'com_modules.module.3','Popular Articles','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(42,18,538,539,2,'com_modules.module.4','Recently Added Articles','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(43,18,540,541,2,'com_modules.module.8','Toolbar','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(44,18,542,543,2,'com_modules.module.9','Quick Icons','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(45,18,544,545,2,'com_modules.module.10','Logged-in Users','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(46,18,546,547,2,'com_modules.module.12','Admin Menu','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(47,18,548,549,2,'com_modules.module.13','Admin Submenu','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(48,18,550,551,2,'com_modules.module.14','User Status','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(49,18,552,553,2,'com_modules.module.15','Title','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(51,18,554,555,2,'com_modules.module.17','Breadcrumb','{}'),(52,18,556,557,2,'com_modules.module.79','Multilanguage status','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(53,18,558,559,2,'com_modules.module.86','Joomla Version','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[]}'),(55,18,560,561,2,'com_modules.module.88','Site Information','{\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1}}'),(56,18,562,563,2,'com_modules.module.89','Release News','{\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1}}'),(63,8,46,47,2,'com_content.category.8','Home','{\"core.create\":{\"6\":1,\"3\":1},\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1},\"core.edit.own\":{\"6\":1,\"3\":1}}'),(69,8,48,71,2,'com_content.category.14','Docs','{\"core.create\":{\"6\":1,\"3\":1},\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1},\"core.edit.own\":{\"6\":1,\"3\":1}}'),(296,18,564,564,2,'com_modules.module.300','Image Center','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(483,18,566,567,2,'com_modules.module.475','FavGlyph - Home Page','{}'),(487,18,568,569,2,'com_modules.module.479','FavSocial - Reactome Social Medias','{}'),(505,18,570,571,2,'com_modules.module.484','Homepage black banner - alternative','{\"core.delete\":{\"6\":1},\"core.edit\":{\"6\":1,\"4\":1},\"core.edit.state\":{\"6\":1,\"5\":1}}'),(518,18,572,573,2,'com_modules.module.496','Tweets','{}'),(521,18,574,575,2,'com_modules.module.499','Find Reactions, Proteins and Pathways','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(522,18,576,577,2,'com_modules.module.500','Notice','{\"core.edit\":{\"10\":0}}'),(523,1,715,716,1,'com_jce','COM_JCE','{}'),(524,8,72,259,2,'com_content.category.25','About','{}'),(525,524,73,74,3,'com_content.article.11','What is Reactome ?','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(526,524,75,236,3,'com_content.category.26','News','{}'),(528,524,237,238,3,'com_content.article.13','Reactome Team','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(529,524,239,240,3,'com_content.article.14','Scientific Advisory Board','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(530,524,245,246,3,'com_content.article.15','Other Reactomes','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(531,524,241,242,3,'com_content.article.16','License Agreement','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(532,524,243,244,3,'com_content.article.17','Disclaimer','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(537,637,347,348,3,'com_content.article.18','Downloads','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(539,18,578,579,2,'com_modules.module.505','Why Reactome','{\"module.edit.frontend\":{\"11\":1}}'),(544,18,580,581,2,'com_modules.module.508','Latest News','{}'),(548,1,717,718,1,'com_fields','com_fields','{}'),(549,1,719,720,1,'com_associations','com_associations','{}'),(552,16,522,523,2,'com_menus.menu.1','Main Menu','{}'),(555,8,260,279,2,'com_content.category.27','Community','{}'),(556,555,261,262,3,'com_content.article.22','Outreach','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(557,555,263,264,3,'com_content.article.23','Events','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(558,555,265,266,3,'com_content.article.24','Publications','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(559,555,267,268,3,'com_content.article.25','Training Materials','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(560,555,269,270,3,'com_content.article.26','Partners','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(561,555,271,272,3,'com_content.article.27','Resources Guide','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(562,69,49,50,3,'com_content.article.28','Linking to us','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(563,69,51,52,3,'com_content.article.29','Referencing our Publications','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(568,16,524,525,2,'com_menus.menu.2','User Menu','{}'),(571,18,582,583,2,'com_modules.module.518','Maintop Blank Space','{}'),(572,18,584,585,2,'com_modules.module.519','Find Reactions, Proteins and Pathways - Others','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(573,8,280,309,2,'com_content.category.28','User Guide','{}'),(586,18,586,587,2,'com_modules.module.523','User Guide Anchor Menu','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(587,573,281,282,3,'com_content.article.39','User Guide','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(591,18,588,589,2,'com_modules.module.524','Articles Sidebar Menu','{}'),(592,18,590,591,2,'com_modules.module.525','Icon Library','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(594,27,19,20,3,'com_content.article.43','404','{\"core.admin\":{\"7\":1},\"core.options\":{},\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":{},\"core.edit.own\":{},\"core.edit\":{\"4\":0},\"core.edit.state\":{\"5\":0}}'),(595,18,592,593,2,'com_modules.module.526','[TEMPLATE] data-content / swagger placeholder','{}'),(596,18,594,595,2,'com_modules.module.527','[TEMPLATE] search placeholder','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(599,16,526,527,2,'com_menus.menu.4','Hidden Menu','{}'),(603,18,596,597,2,'com_modules.module.533','Footer','{}'),(604,573,283,284,3,'com_content.article.44','ReactomeFIVIz','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(605,606,311,312,3,'com_content.article.45','Graph Database','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(606,8,310,337,2,'com_content.category.29','Developers Guide','{}'),(612,613,339,340,3,'com_content.article.46','Teleconferences','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(613,8,338,343,2,'com_content.category.30','Reactome Staff','{}'),(615,18,598,599,2,'com_modules.module.540','Developers Zone - Row 1','{}'),(619,606,313,314,3,'com_content.article.47','Analysis Service','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(620,606,315,316,3,'com_content.article.48','Content Service','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(621,606,317,318,3,'com_content.article.49','Pathways Overview','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(622,606,319,320,3,'com_content.article.50','Pathway Diagrams','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(625,69,53,54,3,'com_content.article.53','Data Model','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(626,8,344,345,2,'com_content.category.31','Release','{}'),(627,18,600,601,2,'com_modules.module.544','Documentation','{}'),(628,606,321,322,3,'com_content.article.54','Tutorial: Extracting participating molecules using the Graph Database','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(629,1,721,722,1,'#__ucm_content.1','#__ucm_content.1','{}'),(630,69,55,56,3,'com_content.article.55','Icons Info ','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(631,606,323,324,3,'com_content.article.56','Pathways Overview JS','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(632,606,325,326,3,'com_content.article.57','Pathways Overview GWT','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(633,606,327,328,3,'com_content.article.58','Diagram JS','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(634,606,329,330,3,'com_content.article.59','Diagram GWT','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(637,8,346,353,2,'com_content.category.32','Download','{}'),(638,702,355,356,3,'com_content.article.60','Identifiers Mapping','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(639,27,21,22,3,'com_content.article.61','Pathway Information','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(640,702,365,366,3,'com_content.article.62','Tab-Delimiter Format','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(641,69,61,62,3,'com_content.article.63','Curator Tool (version 4.0.3, build 118, March 3, 2024)','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(647,18,602,603,2,'com_modules.module.551','Partners','{}'),(651,18,604,605,2,'com_modules.module.555','Do you need help ?','{}'),(652,18,606,607,2,'com_modules.module.556','API and Data access','{}'),(655,18,608,609,2,'com_modules.module.559','Pathway Diagram - Dev Guide, 4 buttons','{}'),(656,18,610,611,2,'com_modules.module.560','Pathway Overview - Dev Guide, 4 buttons','{}'),(657,18,612,613,2,'com_modules.module.561','Pathway Diagram - Dev Guide - Get Started, 2 buttons ','{}'),(659,18,614,615,2,'com_modules.module.563','Pathway Diagram - Dev Guide - API, 2 buttons','{}'),(660,18,616,617,2,'com_modules.module.564','Pathway Overview - Dev Guide - Get Started, 2 buttons','{}'),(661,18,618,619,2,'com_modules.module.565','Pathway Overview - Dev Guide - API, 2 buttons','{}'),(662,18,620,621,2,'com_modules.module.566','Pathway Overview GWT - Dev Guide, 4 buttons','{}'),(663,18,622,623,2,'com_modules.module.567','Pathway Diagram GWT - Dev Guide, 4 buttons','{}'),(664,18,624,625,2,'com_modules.module.568','Content Service - Dev Guide, 4 buttons','{}'),(665,18,626,627,2,'com_modules.module.569','Analysis Service - Dev Guide, 4 buttons','{}'),(666,18,628,629,2,'com_modules.module.570','Graph Database - Dev Guide, 4 buttons','{}'),(667,18,630,631,2,'com_modules.module.571','Developers Zone - Row 2','{}'),(668,27,23,24,3,'com_content.article.64','Developer\'s Zone','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(673,18,632,633,2,'com_modules.module.573','Icon Lib Menu','{\"core.delete\":[],\"core.edit\":[],\"core.edit.state\":[],\"module.edit.frontend\":[]}'),(674,18,634,635,2,'com_modules.module.574','Version 86 released on September 13th, 2023','{\"module.edit.frontend\":{\"11\":1}}'),(675,27,25,26,3,'com_content.article.66','Statistics','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(676,819,369,370,3,'com_content.article.67','ORCID Integration Project','{\"core.admin\":{\"7\":1},\"core.options\":[],\"core.manage\":{\"6\":1},\"core.create\":{\"3\":1},\"core.delete\":[],\"core.edit\":{\"4\":1},\"core.edit.state\":{\"5\":1},\"core.edit.own\":[]}'),(681,18,636,637,2,'com_modules.module.579','Footer (For Small Devices)','{}'),(682,69,57,58,3,'com_content.article.68','Computationally Inferred Events','{}'),(683,555,273,274,3,'com_content.article.69','Papers citing us','{}'),(684,69,59,60,3,'com_content.article.70','External Identifiers','{}'),(685,526,76,77,4,'com_content.article.71','Version 57 Released','{}'),(686,526,94,95,4,'com_content.article.72','Version 58 Released','{}'),(687,526,78,79,4,'com_content.article.73','Reactome celebrates release of 10,000th annotated protein','{}'),(688,526,80,81,4,'com_content.article.74','Version 59 Released','{}'),(689,526,82,83,4,'com_content.article.75','New Reactome Publication','{}'),(690,526,84,85,4,'com_content.article.76','New Reactome Paper published','{}'),(691,526,86,87,4,'com_content.article.77','Version 60 Released','{}'),(692,526,88,89,4,'com_content.article.78','Version 61 Released','{}'),(693,526,90,91,4,'com_content.article.79','New Protein-protein Interaction files','{}'),(694,526,92,93,4,'com_content.article.80','New SBML Level 3 Version 1 export is now available','{}'),(695,573,285,286,3,'com_content.article.81','The Pathway Browser','{}'),(696,573,287,288,3,'com_content.article.82','Details Panel','{}'),(697,573,289,290,3,'com_content.article.83','Searching Reactome','{}'),(698,573,291,292,3,'com_content.article.84','Analysis Tools','{}'),(699,573,293,294,3,'com_content.article.85','Diseases','{}'),(700,18,638,639,2,'com_modules.module.580','User Guide Glyphs Row 1','{\"module.edit.frontend\":{\"5\":1}}'),(701,702,357,358,3,'com_content.article.86','Physical Entity Identifiers Mapping','{}'),(702,8,354,367,2,'com_content.category.33','PopUps and Modals','{}'),(703,702,359,360,3,'com_content.article.87','Reaction PMIDs','{}'),(704,702,361,362,3,'com_content.article.88','Reactome to OMIM','{}'),(705,702,363,364,3,'com_content.article.89','Protein Role Reaction','{}'),(709,526,96,97,4,'com_content.article.90','Version 62 Released','{}'),(710,1,723,724,1,'#__ucm_content.2','#__ucm_content.2','{}'),(712,18,640,641,2,'com_modules.module.581','(*** WORK ON THE TAB CODE *** ) BANNER','{\"core.edit\":{\"10\":0},\"core.edit.state\":{\"11\":1},\"module.edit.frontend\":{\"11\":1}}'),(713,18,642,643,2,'com_modules.module.582','Why Reactome','{}'),(714,524,247,248,3,'com_content.article.91','Introducing our Logo','{}'),(715,18,644,645,2,'com_modules.module.583','[TEMPLATE] CGI placeholder','{}'),(718,526,98,99,4,'com_content.article.93','New responsive website with a fresh look','{}'),(719,526,100,101,4,'com_content.article.94','Proudly introducing our new logo','{}'),(720,18,646,647,2,'com_modules.module.584','Synchronization Tool','{}'),(721,526,102,103,4,'com_content.article.95','New Reactome Publication published in 2018 NAR Database Issue','{}'),(722,606,331,332,3,'com_content.article.96','Diagram exporter','{}'),(723,1,725,726,1,'#__ucm_content.3','#__ucm_content.3','{}'),(724,1,727,728,1,'#__ucm_content.4','#__ucm_content.4','{}'),(725,1,729,730,1,'#__ucm_content.5','#__ucm_content.5','{}'),(726,1,731,732,1,'#__ucm_content.6','#__ucm_content.6','{}'),(728,526,104,105,4,'com_content.article.97','Updated License Agreement','{}'),(729,526,106,107,4,'com_content.article.98','Version 63 Released','{}'),(731,1,733,734,1,'#__ucm_content.8','#__ucm_content.8','{}'),(732,613,341,342,3,'com_content.article.99','Help','{}'),(733,606,333,334,3,'com_content.article.100','Pathway Diagrams Specifications','{}'),(736,526,108,109,4,'com_content.article.102','Version 64 Released','{}'),(737,1,735,736,1,'#__ucm_content.9','#__ucm_content.9','{}'),(738,18,648,649,2,'com_modules.module.585','Elixir Banner','{}'),(739,526,110,111,4,'com_content.article.103','Version 65 Released','{}'),(740,1,737,738,1,'#__ucm_content.10','#__ucm_content.10','{}'),(741,573,295,296,3,'com_content.article.104','Searching Reactome - New Search','{}'),(742,526,112,113,4,'com_content.article.105','Our data now available through Google Dataset Search','{}'),(743,526,114,115,4,'com_content.article.106','Version 66 Released','{}'),(744,1,739,740,1,'#__ucm_content.11','#__ucm_content.11','{}'),(745,526,116,117,4,'com_content.article.107','New tool to generate analysis PDF reports','{}'),(747,526,118,119,4,'com_content.article.109','New advanced search available in our PathwayBrowser','{}'),(748,526,120,121,4,'com_content.article.110','SBGN files revamp','{}'),(749,526,122,123,4,'com_content.article.111','International Open Access Week: Analysis Tools','{}'),(750,526,124,125,4,'com_content.article.112','International Open Access Week: Network Visualization & Analysis Tool','{}'),(751,526,126,127,4,'com_content.article.113','International Open Access Week: Interactive illustrations and icon library','{}'),(752,526,128,129,4,'com_content.article.114','International Open Access Week: GraphDB and Content Service','{}'),(753,526,130,131,4,'com_content.article.115','Performing GSEA analysis with ReactomeFIViz app','{}'),(754,526,132,133,4,'com_content.article.116','Version 67 Released','{}'),(755,637,349,350,3,'com_content.article.117','Icon Mapping','{}'),(756,637,351,352,3,'com_content.article.118','Icons Mapping Download','{}'),(757,1,741,742,1,'#__ucm_content.12','#__ucm_content.12','{}'),(758,1,743,744,1,'com_privacy','com_privacy','{}'),(759,1,745,746,1,'com_actionlogs','com_actionlogs','{}'),(760,526,134,135,4,'com_content.article.119','New reactions layout with nested compartments','{}'),(761,526,136,137,4,'com_content.article.120','New tool to classify pathways into tissue types based on protein expression','{}'),(762,526,138,139,4,'com_content.article.121','Download our biological pathway knowledge in a PDF book','{}'),(765,526,140,141,4,'com_content.article.124','Deprecated support for Reactome RESTful API','{}'),(766,526,142,143,4,'com_content.article.125','Version 68 Released','{}'),(767,1,747,748,1,'#__ucm_content.13','#__ucm_content.13','{}'),(768,526,144,145,4,'com_content.article.126','PANTHER resource now used to identify orthologs of curated human proteins','{}'),(769,526,146,147,4,'com_content.article.127','Come to our release 68 webinar to learn more about Reactome','{}'),(770,1,749,750,1,'#__ucm_content.14','#__ucm_content.14','{}'),(772,18,650,651,2,'com_modules.module.586','EHLD / Icon Library (Specs & Guidelines)','{}'),(773,27,27,28,3,'com_content.article.129','EHLD Specs & Guidelines','{}'),(774,573,299,300,3,'com_content.article.130','Icons Library Guidelines','{}'),(777,526,148,149,4,'com_content.article.133','ReactomeFI Network v2018 and FIViz App v2.7.0 Released','{}'),(778,573,297,298,3,'com_content.article.134','Documentation','{}'),(780,526,150,151,4,'com_content.article.136','Season of Docs','{}'),(781,526,152,153,4,'com_content.article.137','Version 69 Released','{}'),(782,1,751,752,1,'#__ucm_content.15','#__ucm_content.15','{}'),(783,526,154,155,4,'com_content.article.138','ORCID: Claim your works','{}'),(784,573,301,302,3,'com_content.article.139','Tutorial: How to claim your works','{}'),(785,555,275,276,3,'com_content.article.140','Contribute Pathway Knowledge','{}'),(786,526,156,157,4,'com_content.article.141','ReacFoam: Genome-wide pathway overview based on Voronoi tessellation','{}'),(787,526,158,159,4,'com_content.article.142','Version 70 Released','{}'),(789,555,277,278,3,'com_content.article.143','FAQ for prospective Reactome reviewers and authors','{}'),(790,526,160,161,4,'com_content.article.144',' New Reactome Publication published in 2020 NAR Database Issue','{}'),(791,526,162,163,4,'com_content.article.145','Version 71 Released','{}'),(792,1,753,754,1,'#__ucm_content.17','#__ucm_content.17','{}'),(793,1,755,756,1,'#__ucm_content.18','#__ucm_content.18','{}'),(794,526,164,165,4,'com_content.article.146',' New Paper published in Database (Oxford)','{}'),(795,1,757,758,1,'#__ucm_content.19','#__ucm_content.19','{}'),(796,18,652,653,2,'com_modules.module.587','Cite us','{}'),(797,526,166,167,4,'com_content.article.147','Version 72 Released','{}'),(798,1,759,760,1,'#__ucm_content.20','#__ucm_content.20','{}'),(799,524,249,250,3,'com_content.article.148','Privacy Notice','{}'),(801,526,168,169,4,'com_content.article.150','New Paper published in eLife','{}'),(802,1,761,762,1,'#__ucm_content.21','#__ucm_content.21','{}'),(803,526,170,171,4,'com_content.article.151','New reactome2py package links our web services and data with python.','{}'),(804,1,763,764,1,'#__ucm_content.22','#__ucm_content.22','{}'),(806,18,654,655,2,'com_modules.module.588','userguide sidebar test','{}'),(808,18,656,657,2,'com_modules.module.589','Userguide Module','{}'),(809,18,658,659,2,'com_modules.module.590','USERGGUIDE MODULE CATEGORY','{}'),(810,27,29,30,3,'com_content.article.152','User Guide Revised','{}'),(811,18,660,661,2,'com_modules.module.591','User Guide TOC','{}'),(812,69,63,64,3,'com_content.article.153','Data Model Glossary','{}'),(813,526,172,173,4,'com_content.article.154','New Paper published in Autophagy','{}'),(814,526,174,175,4,'com_content.article.155','Version 73 Released','{}'),(815,573,303,304,3,'com_content.article.156','ReactomeGSA','{}'),(816,69,65,66,3,'com_content.article.157','User Guide','{}'),(817,1,765,766,1,'#__ucm_content.23','#__ucm_content.23','{}'),(818,819,371,372,3,'com_content.article.158','COVID-19 Disease Pathways','{}'),(819,8,368,429,2,'com_content.category.34','Content','{}'),(823,526,176,177,4,'com_content.article.161','COVID-19: SARS-CoV-2 infection pathway Released','{}'),(824,1,767,768,1,'#__ucm_content.25','#__ucm_content.25','{}'),(825,18,662,663,2,'com_modules.module.592','Use Reactome Graph Database in your Project','{}'),(826,526,178,179,4,'com_content.article.162','New Paper published in Molecular & Cellular Proteomics','{}'),(827,1,769,770,1,'#__ucm_content.26','#__ucm_content.26','{}'),(828,526,180,181,4,'com_content.article.163','Version 75 Released','{}'),(829,1,771,772,1,'#__ucm_content.27','#__ucm_content.27','{}'),(830,18,664,665,2,'com_modules.module.593','Banner','{}'),(831,18,666,667,2,'com_modules.module.594','New training material','{}'),(832,18,668,669,2,'com_modules.module.595','Please Take The EBI Impact Survey','{}'),(833,526,182,183,4,'com_content.article.164','Version 76 Released','{}'),(834,1,773,774,1,'#__ucm_content.28','#__ucm_content.28','{}'),(835,1,775,776,1,'#__ucm_content.29','#__ucm_content.29','{}'),(836,1,777,778,1,'#__ucm_content.30','#__ucm_content.30','{}'),(837,526,184,185,4,'com_content.article.165','The Reactome IDG Portal is released','{}'),(838,1,779,780,1,'#__ucm_content.31','#__ucm_content.31','{}'),(839,526,186,187,4,'com_content.article.166','Version 77 Released','{}'),(840,1,781,782,1,'#__ucm_content.32','#__ucm_content.32','{}'),(841,526,188,189,4,'com_content.article.167','Reactome Multi-Omics Pathway Analysis webinar reaches record attendance','{}'),(842,1,783,784,1,'#__ucm_content.33','#__ucm_content.33','{}'),(843,606,335,336,3,'com_content.article.168','Neo4j Desktop','{}'),(844,526,190,191,4,'com_content.article.169','Version 78 Released','{}'),(845,1,785,786,1,'#__ucm_content.34','#__ucm_content.34','{}'),(846,8,430,505,2,'com_content.category.35','FAQ','{}'),(847,27,31,32,3,'com_content.article.170','Reactome Frequently Asked Questions','{}'),(848,27,33,34,3,'com_content.article.171','FAQ','{}'),(849,526,192,193,4,'com_content.article.172','We want to hear your Success Story!','{}'),(850,1,787,788,1,'#__ucm_content.35','#__ucm_content.35','{}'),(851,524,251,252,3,'com_content.article.173','FAQ','{}'),(852,526,194,195,4,'com_content.article.174','Version 79 Released','{}'),(853,1,789,790,1,'#__ucm_content.36','#__ucm_content.36','{}'),(854,1,791,792,1,'com_easyjoomlabackup','COM_EASYJOOMLABACKUP','{}'),(855,526,196,197,4,'com_content.article.175','Version 80 Released','{}'),(856,1,793,794,1,'#__ucm_content.37','#__ucm_content.37','{}'),(857,526,198,199,4,'com_content.article.176','Version 82 Released','{}'),(858,526,200,201,4,'com_content.article.177','New Paper pulished in Database (Oxford)','{}'),(859,1,795,796,1,'#__ucm_content.38','#__ucm_content.38','{}'),(860,526,202,203,4,'com_content.article.178','Version 81 Released','{}'),(861,526,204,205,4,'com_content.article.179','Collaboration with PharmGKB','{}'),(862,819,373,428,3,'com_content.category.36','Spotlight','{}'),(863,18,670,671,2,'com_modules.module.596','Reactome Research Spotlight','{\"module.edit.frontend\":{\"11\":1}}'),(864,862,374,375,4,'com_content.article.180','Post-infusion CAR TReg cells identify patients resistant to CD19-CAR therapy','{}'),(865,862,376,377,4,'com_content.article.181','Reactome Research Spotlight test','{}'),(866,862,378,379,4,'com_content.article.182','reactome spotlight test','{}'),(867,526,206,207,4,'com_content.article.183','Reactome Research Spotlight','{}'),(868,1,797,798,1,'#__ucm_content.39','#__ucm_content.39','{}'),(869,1,799,800,1,'#__ucm_content.40','#__ucm_content.40','{}'),(870,526,208,209,4,'com_content.article.184','Version 83 Released','{}'),(871,1,801,802,1,'#__ucm_content.41','#__ucm_content.41','{}'),(872,862,380,381,4,'com_content.article.185','Circadian transcriptional pathway atlas highlights a proteasome switch in intermittent fasting','{}'),(873,862,382,383,4,'com_content.article.186','Circadian transcriptional pathway atlas highlights a proteasome switch in intermittent fasting','{}'),(874,526,210,211,4,'com_content.article.187','Reactome named as a Global Core Biodata Resource','{}'),(875,18,672,673,2,'com_modules.module.597','Elixir and GCBR Banner','{}'),(876,862,384,385,4,'com_content.article.188','Gene set enrichment analysis (GSEA) identifies upregulated carbohydrate metabolism pathways in tumor','{}'),(877,862,386,387,4,'com_content.article.189','Common targetable inflammatory pathways in brain transcriptome of autism spectrum disorders and Tour','{}'),(878,862,388,389,4,'com_content.article.190','Probable Treatment Targets for Diabetic Retinopathy Based on an Integrated Proteomic and Genomic Ana','{}'),(879,526,212,213,4,'com_content.article.191','Reactome is hiring!','{}'),(880,573,305,306,3,'com_content.article.192','Review Status of Reactome Events ','{}'),(881,526,214,215,4,'com_content.article.193','Version 84 released','{}'),(882,526,216,217,4,'com_content.article.194','V84 released','{}'),(883,1,803,804,1,'#__ucm_content.42','#__ucm_content.42','{}'),(884,846,431,446,3,'com_content.category.37','General website','{}'),(885,846,447,450,3,'com_content.category.38','Illustrations/Figures','{}'),(886,846,451,490,3,'com_content.category.39','Analysis','{}'),(887,886,452,465,4,'com_content.category.40','General','{}'),(888,886,466,473,4,'com_content.category.41','Reactome GSA','{}'),(889,886,474,479,4,'com_content.category.42','FIViz','{}'),(890,846,491,504,3,'com_content.category.43','Graph Database and cypher query','{}'),(891,886,480,487,4,'com_content.category.44','API and R','{}'),(892,18,674,675,2,'com_modules.module.598','FAQ Sidebar Menu','{}'),(893,884,432,433,4,'com_content.article.195','I searched for (...) and didn’t get any results.','{}'),(894,885,448,449,4,'com_content.article.196','Is there a way to download high level images which can be zoomed in locally (after download)?','{}'),(895,887,453,454,5,'com_content.article.197','I used the “Analyse gene expression” tool with my mouse proteomics data set. I notice that the datab','{}'),(896,890,492,493,4,'com_content.article.198','I am having trouble installing a local Reactome Neo4J server.','{}'),(897,884,434,435,4,'com_content.article.199','Does Reactome contain pathway information from non-human species?','{}'),(898,862,390,391,4,'com_content.article.200','Patient-derived cell-based pharmacogenomic assessment to unveil underlying resistance mechanisms and','{}'),(899,884,436,437,4,'com_content.article.201','Explain the identifiers associated with entities and events.','{}'),(900,884,438,439,4,'com_content.article.202','Where can I find earlier versions of the Reactome database?','{}'),(901,884,440,441,4,'com_content.article.203','How can I find all pathways reported for one organ? For example, I want to extract all reported path','{}'),(902,884,442,443,4,'com_content.article.204','Is there a mapping file between KEGG and Reactome pathways?','{}'),(903,884,444,445,4,'com_content.article.205','Is it possible to download the inferred pathways for mouse (or another species), similar to the GMT ','{}'),(904,886,488,489,4,'com_content.article.206','DELETE:I used the “Analyse gene expression” tool with my mouse proteomics data set. I notice that th','{}'),(905,887,455,456,5,'com_content.article.207','The analysis report includes lists of significantly up- and down-regulated single genes. Which stati','{}'),(906,887,457,458,5,'com_content.article.208','Can I visualize gene expression data (color nodes) according to gene expression level with Reactome?','{}'),(907,887,459,460,5,'com_content.article.209','After submitting a list of genes for Reactome Analysis, how do I download the fully expanded \'Event ','{}'),(908,887,461,462,5,'com_content.article.210','Is it possible to know how many of my uploaded genes are found within each specific hierarchical pat','{}'),(909,887,463,464,5,'com_content.article.211','Is there a permanent link available for my analysis results?','{}'),(910,888,467,468,5,'com_content.article.212','I have a proteomics data set that I would like to use for quantitative pathway analysis. Is there tr','{}'),(911,888,469,470,5,'com_content.article.213','Can I submit a list of differentially expressed genes, as log2 fold change values, coming from two d','{}'),(912,888,471,472,5,'com_content.article.214','I submitted a list for expression analysis using the PADOG or CAMERA tools and the output Excel file','{}'),(913,891,481,482,5,'com_content.article.215','Is there an API for Reactome? Can I do pathway analysis through the API?','{}'),(914,891,483,484,5,'com_content.article.216','Should I use the gene symbol or the NCBI identifiers to make requests to the identifiers endpoint of','{}'),(915,891,485,486,5,'com_content.article.217','Can I do pathway analysis using R?','{}'),(916,889,475,476,5,'com_content.article.218','I need to do a network analysis for (species) from RNA seq data. Can you please guide me on how to g','{}'),(917,889,477,478,5,'com_content.article.219','Can I submit a list of differentially expressed genes, as log2 fold change values, coming from two d','{}'),(918,890,494,495,4,'com_content.article.220','DELETE:I am having trouble installing a local Reactome Neo4J server.','{}'),(919,890,496,497,4,'com_content.article.221','I have a list of input genes. I want to retrieve all the pathways in which the genes are involved. H','{}'),(920,890,498,499,4,'com_content.article.222','I have a list of Reactome pathway IDs. I want to find whether any of them are disease pathways and a','{}'),(921,890,500,501,4,'com_content.article.223','How can I use Neo4J to identify all the genes for a given pathway?','{}'),(922,890,502,503,4,'com_content.article.224','Is it possible to directly connect my protein protein interaction network with the Reactome database','{}'),(923,526,218,219,4,'com_content.article.225','V85 released','{}'),(924,1,805,806,1,'#__ucm_content.43','#__ucm_content.43','{}'),(925,862,392,393,4,'com_content.article.226','Label-Free Mass Spectrometry Proteomics Reveals Different Pathways Modulated in THP‐1 Cells Infected','{}'),(926,862,394,395,4,'com_content.article.227','Severe COVID-19 in pregnancy has a distinct serum profile, including greater complement activation a','{}'),(927,862,396,397,4,'com_content.article.228','In Vitro Zika Virus Infection of Human Neural Progenitor Cells: Meta-Analysis of RNA-Seq Assays','{}'),(928,862,398,399,4,'com_content.article.229','Genetic Networks of Alzheimer’s Disease, Aging, and Longevity in Humans','{}'),(929,526,220,221,4,'com_content.article.230','V86 Released  ','{}'),(930,1,807,808,1,'#__ucm_content.44','#__ucm_content.44','{}'),(931,526,222,223,4,'com_content.article.231','V86','{}'),(932,1,809,810,1,'#__ucm_content.45','#__ucm_content.45','{}'),(933,862,400,401,4,'com_content.article.232','Computational drug repositioning of clopidogrel as a novel therapeutic option for focal segmental gl','{}'),(934,862,402,403,4,'com_content.article.233','DNA methylation and 28-year cardiovascular disease risk in type 1 diabetes: the Epidemiology of Diab','{}'),(935,27,35,36,3,'com_content.article.234','XMR: an explainable multimodal neural network for drug response prediction','{}'),(936,862,404,405,4,'com_content.article.235','XMR: an explainable multimodal neural network for drug response prediction','{}'),(937,18,676,677,2,'com_modules.module.599','Version 90 released on October 2, 2024','{\"module.edit.frontend\":{\"11\":1}}'),(938,27,37,38,3,'com_content.article.236','New Insights into Clinical Management for Sickle Cell Disease: Uncovering the Significant Pathways A','{}'),(939,862,406,407,4,'com_content.article.237','New Insights into Clinical Management for Sickle Cell Disease: Uncovering the Significant Pathways A','{}'),(940,526,224,225,4,'com_content.article.238','V87 released','{}'),(941,1,811,812,1,'#__ucm_content.46','#__ucm_content.46','{}'),(942,524,253,254,3,'com_content.article.239','Contact us','{}'),(943,862,408,409,4,'com_content.article.240','Machine learning-based analysis of cancer cell-derived vesicular proteins revealed significant tumor','{}'),(944,524,255,256,3,'com_content.article.241','Digital Preservation','{}'),(945,862,410,411,4,'com_content.article.242','Discovering the anti-cancer phytochemical rutin against breast cancer through the methodical platfor','{}'),(946,27,39,40,3,'com_content.article.243','Identification of potential biological processes and key genes in diabetes-related stroke through we','{}'),(947,862,412,413,4,'com_content.article.244','Identification of potential biological processes and key genes in diabetes-related stroke through we','{}'),(948,18,678,679,2,'com_modules.module.600','User Guide Glyphs Row 2','{}'),(949,526,226,227,4,'com_content.article.245','V88 Released','{}'),(950,526,228,229,4,'com_content.article.246','V88 Released','{}'),(951,573,307,308,3,'com_content.article.247','Cytomics','{}'),(952,1,813,814,1,'#__ucm_content.47','#__ucm_content.47','{}'),(953,862,414,415,4,'com_content.article.248','Nickel-induced transcriptional memory in lung epithelial cells promotes interferon signaling upon ni','{}'),(954,862,416,417,4,'com_content.article.249','IBPGNET: lung adenocarcinoma recurrence prediction based on neural network interpretability','{}'),(955,526,230,231,4,'com_content.article.250','New Publication in Database (Oxford)! ','{}'),(956,18,680,681,2,'com_modules.module.601','EMBL-EBI User Survey 2024','{}'),(957,526,232,233,4,'com_content.article.251','V89 Released','{}'),(958,1,815,816,1,'#__ucm_content.48','#__ucm_content.48','{}'),(959,862,418,419,4,'com_content.article.252','Acquired resistance to immunotherapy and chemoradiation in MYC amplified head and neck cancer','{}'),(960,27,41,42,3,'com_content.article.253','Drug target prediction through deep learning functional representation of gene signatures','{}'),(961,862,420,421,4,'com_content.article.254','Drug target prediction through deep learning functional representation of gene signatures','{}'),(962,862,422,423,4,'com_content.article.255','The landscape of cancer-rewired GPCR signaling axes','{}'),(963,862,424,425,4,'com_content.article.256','Engineering Toxoplasma gondii secretion systems for intracellular delivery of multiple large therape','{}'),(964,18,682,683,2,'com_modules.module.602','Community','{}'),(965,27,43,44,3,'com_content.article.257','Community','{}'),(966,524,257,258,3,'com_content.article.258','Funding','{}'),(967,69,67,68,3,'com_content.article.259','Curator Guide','{}'),(968,69,69,70,3,'com_content.article.260','Release Documentation','{}'),(969,526,234,235,4,'com_content.article.261','V90 Released','{}'),(970,1,817,818,1,'#__ucm_content.49','#__ucm_content.49','{}'),(971,862,426,427,4,'com_content.article.262','Chemical coverage of human biological pathways','{}');
/*!40000 ALTER TABLE `rlp_assets` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_associations`
--

DROP TABLE IF EXISTS `rlp_associations`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_associations` (
  `id` int NOT NULL COMMENT 'A reference to the associated item.',
  `context` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The context of the associated item.',
  `key` char(32) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The key for the association computed from an md5 on associated ids.',
  PRIMARY KEY (`context`,`id`),
  KEY `idx_key` (`key`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_associations`
--

LOCK TABLES `rlp_associations` WRITE;
/*!40000 ALTER TABLE `rlp_associations` DISABLE KEYS */;
INSERT INTO `rlp_associations` VALUES (208,'com_menus.item','d8de03296c20dc0e00a095d1bdf9d908'),(209,'com_menus.item','d8de03296c20dc0e00a095d1bdf9d908'),(214,'com_menus.item','d8de03296c20dc0e00a095d1bdf9d908');
/*!40000 ALTER TABLE `rlp_associations` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_banner_clients`
--

DROP TABLE IF EXISTS `rlp_banner_clients`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_banner_clients` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `contact` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `email` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `extrainfo` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `state` tinyint NOT NULL DEFAULT '0',
  `checked_out` int unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `metakey` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `own_prefix` tinyint NOT NULL DEFAULT '0',
  `metakey_prefix` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `purchase_type` tinyint NOT NULL DEFAULT '-1',
  `track_clicks` tinyint NOT NULL DEFAULT '-1',
  `track_impressions` tinyint NOT NULL DEFAULT '-1',
  PRIMARY KEY (`id`),
  KEY `idx_own_prefix` (`own_prefix`),
  KEY `idx_metakey_prefix` (`metakey_prefix`(100))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_banner_clients`
--

LOCK TABLES `rlp_banner_clients` WRITE;
/*!40000 ALTER TABLE `rlp_banner_clients` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_banner_clients` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_banner_tracks`
--

DROP TABLE IF EXISTS `rlp_banner_tracks`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_banner_tracks` (
  `track_date` datetime NOT NULL,
  `track_type` int unsigned NOT NULL,
  `banner_id` int unsigned NOT NULL,
  `count` int unsigned NOT NULL DEFAULT '0',
  PRIMARY KEY (`track_date`,`track_type`,`banner_id`),
  KEY `idx_track_date` (`track_date`),
  KEY `idx_track_type` (`track_type`),
  KEY `idx_banner_id` (`banner_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_banner_tracks`
--

LOCK TABLES `rlp_banner_tracks` WRITE;
/*!40000 ALTER TABLE `rlp_banner_tracks` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_banner_tracks` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_banners`
--

DROP TABLE IF EXISTS `rlp_banners`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_banners` (
  `id` int NOT NULL AUTO_INCREMENT,
  `cid` int NOT NULL DEFAULT '0',
  `type` int NOT NULL DEFAULT '0',
  `name` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `imptotal` int NOT NULL DEFAULT '0',
  `impmade` int NOT NULL DEFAULT '0',
  `clicks` int NOT NULL DEFAULT '0',
  `clickurl` varchar(200) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `state` tinyint NOT NULL DEFAULT '0',
  `catid` int unsigned NOT NULL DEFAULT '0',
  `description` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `custombannercode` varchar(2048) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `sticky` tinyint unsigned NOT NULL DEFAULT '0',
  `ordering` int NOT NULL DEFAULT '0',
  `metakey` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `params` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `own_prefix` tinyint(1) NOT NULL DEFAULT '0',
  `metakey_prefix` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `purchase_type` tinyint NOT NULL DEFAULT '-1',
  `track_clicks` tinyint NOT NULL DEFAULT '-1',
  `track_impressions` tinyint NOT NULL DEFAULT '-1',
  `checked_out` int unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `reset` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `language` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `created_by` int unsigned NOT NULL DEFAULT '0',
  `created_by_alias` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int unsigned NOT NULL DEFAULT '0',
  `version` int unsigned NOT NULL DEFAULT '1',
  PRIMARY KEY (`id`),
  KEY `idx_state` (`state`),
  KEY `idx_own_prefix` (`own_prefix`),
  KEY `idx_banner_catid` (`catid`),
  KEY `idx_language` (`language`),
  KEY `idx_metakey_prefix` (`metakey_prefix`(100))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_banners`
--

LOCK TABLES `rlp_banners` WRITE;
/*!40000 ALTER TABLE `rlp_banners` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_banners` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_categories`
--

DROP TABLE IF EXISTS `rlp_categories`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_categories` (
  `id` int NOT NULL AUTO_INCREMENT,
  `asset_id` int unsigned NOT NULL DEFAULT '0' COMMENT 'FK to the #__assets table.',
  `parent_id` int unsigned NOT NULL DEFAULT '0',
  `lft` int NOT NULL DEFAULT '0',
  `rgt` int NOT NULL DEFAULT '0',
  `level` int unsigned NOT NULL DEFAULT '0',
  `path` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `extension` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `title` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `note` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `description` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `published` tinyint(1) NOT NULL DEFAULT '0',
  `checked_out` int unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `access` int unsigned NOT NULL DEFAULT '0',
  `params` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `metadesc` varchar(1024) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'The meta description for the page.',
  `metakey` varchar(1024) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'The meta keywords for the page.',
  `metadata` varchar(2048) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'JSON encoded metadata properties.',
  `created_user_id` int unsigned NOT NULL DEFAULT '0',
  `created_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_user_id` int unsigned NOT NULL DEFAULT '0',
  `modified_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `hits` int unsigned NOT NULL DEFAULT '0',
  `language` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `version` int unsigned NOT NULL DEFAULT '1',
  PRIMARY KEY (`id`),
  KEY `cat_idx` (`extension`,`published`,`access`),
  KEY `idx_access` (`access`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_left_right` (`lft`,`rgt`),
  KEY `idx_language` (`language`),
  KEY `idx_path` (`path`(100)),
  KEY `idx_alias` (`alias`(100))
) ENGINE=InnoDB AUTO_INCREMENT=45 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_categories`
--

LOCK TABLES `rlp_categories` WRITE;
/*!40000 ALTER TABLE `rlp_categories` DISABLE KEYS */;
INSERT INTO `rlp_categories` VALUES (1,0,0,0,55,0,'','system','ROOT','root','','',1,0,'0000-00-00 00:00:00',1,'{}','','','{}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(2,27,1,1,2,1,'uncategorised','com_content','Uncategorised','uncategorised','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(3,28,1,19,20,1,'uncategorised','com_banners','Uncategorised','uncategorised','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(4,29,1,21,22,1,'uncategorised','com_contact','Uncategorised','uncategorised','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(5,30,1,23,24,1,'uncategorised','com_newsfeeds','Uncategorised','uncategorised','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(7,32,1,25,26,1,'uncategorised','com_users','Uncategorised','uncategorised','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2011-01-01 00:00:01',0,'0000-00-00 00:00:00',0,'*',1),(8,63,1,3,4,1,'home','com_content','Home','home','','',1,184,'2024-04-12 16:53:19',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2015-04-12 12:45:31',0,'2015-04-12 12:45:31',0,'*',1),(14,69,1,11,12,1,'docs','com_content','Docs','docs','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2015-04-12 13:36:52',851,'2015-06-05 11:15:55',0,'*',1),(25,524,1,5,8,1,'about','com_content','About','about','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-04-20 15:00:05',0,'2017-04-20 15:00:05',0,'*',1),(26,526,25,6,7,2,'about/news','com_content','News','news','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-04-20 15:08:31',184,'2024-05-14 21:44:21',0,'*',1),(27,555,1,9,10,1,'community','com_content','Community','community','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-04-27 15:43:12',0,'2017-04-27 15:43:12',0,'*',1),(28,573,1,13,14,1,'user-guide','com_content','User Guide','user-guide','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-05-08 14:42:40',0,'2017-05-08 14:42:40',0,'*',1),(29,606,1,15,16,1,'developers-guide','com_content','Developers Guide','developers-guide','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-05-26 08:41:01',0,'2017-05-26 08:41:01',0,'*',1),(30,613,1,17,18,1,'reactome-staff','com_content','Reactome Staff','reactome-staff','','',1,0,'0000-00-00 00:00:00',2,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-05-30 09:59:41',0,'2017-05-30 09:59:41',0,'*',1),(31,626,1,27,28,1,'release','com_content','Release','release','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-06-02 12:44:42',0,'2017-06-02 12:44:42',0,'*',1),(32,637,1,35,36,1,'download','com_content','Download','download','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-06-20 11:18:43',0,'2017-06-20 11:18:43',0,'*',1),(33,702,1,29,30,1,'popups-and-modals','com_content','PopUps and Modals','popups-and-modals','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"_:blog\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',156,'2017-09-25 13:53:42',0,'2017-09-25 13:53:42',0,'*',1),(34,819,1,31,34,1,'content','com_content','Content','content','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',169,'2020-09-18 20:22:55',0,'2020-09-18 20:22:55',0,'*',1),(35,846,1,37,54,1,'faq','com_content','FAQ','faq','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',166,'2021-10-21 03:39:00',181,'2023-04-11 09:16:09',0,'*',1),(36,862,34,32,33,2,'content/spotlight','com_content','Spotlight','spotlight','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',181,'2022-10-18 14:11:59',0,'2022-10-18 14:11:59',0,'*',1),(37,884,35,38,39,2,'faq/general-website','com_content','General website','general-website','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',181,'2023-04-11 09:16:01',0,'2023-04-11 09:16:01',0,'*',1),(38,885,35,40,41,2,'faq/illustrations-figures','com_content','Illustrations/Figures','illustrations-figures','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',181,'2023-04-11 09:16:34',0,'2023-04-11 09:16:34',0,'*',1),(39,886,35,42,51,2,'faq/analysis','com_content','Analysis','analysis','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',181,'2023-04-11 09:16:52',0,'2023-04-11 09:16:52',0,'*',1),(40,887,39,43,44,3,'faq/analysis/general','com_content','General','general','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',181,'2023-04-11 09:17:14',0,'2023-04-11 09:17:14',0,'*',1),(41,888,39,45,46,3,'faq/analysis/reactome-gsa','com_content','Reactome GSA','reactome-gsa','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',181,'2023-04-11 09:17:30',0,'2023-04-11 09:17:30',0,'*',1),(42,889,39,49,50,3,'faq/analysis/fiviz','com_content','FIViz','fiviz','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',181,'2023-04-11 09:17:55',0,'2023-04-11 09:17:55',0,'*',1),(43,890,35,52,53,2,'faq/graph-database-and-cypher-query','com_content','Graph Database and cypher query','graph-database-and-cypher-query','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',181,'2023-04-11 09:19:23',0,'2023-04-11 09:19:23',0,'*',1),(44,891,39,47,48,3,'faq/analysis/api-and-r','com_content','API and R','api-and-r','','',1,0,'0000-00-00 00:00:00',1,'{\"category_layout\":\"\",\"image\":\"\",\"image_alt\":\"\"}','','','{\"author\":\"\",\"robots\":\"\"}',181,'2023-04-11 09:20:07',0,'2023-04-11 09:20:07',0,'*',1);
/*!40000 ALTER TABLE `rlp_categories` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_contact_details`
--

DROP TABLE IF EXISTS `rlp_contact_details`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_contact_details` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `con_position` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `address` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `suburb` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `state` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `country` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `postcode` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `telephone` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `fax` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `misc` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `image` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `email_to` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `default_con` tinyint unsigned NOT NULL DEFAULT '0',
  `published` tinyint(1) NOT NULL DEFAULT '0',
  `checked_out` int unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `ordering` int NOT NULL DEFAULT '0',
  `params` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `user_id` int NOT NULL DEFAULT '0',
  `catid` int NOT NULL DEFAULT '0',
  `access` int unsigned NOT NULL DEFAULT '0',
  `mobile` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `webpage` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `sortname1` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `sortname2` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `sortname3` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `language` varchar(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by` int unsigned NOT NULL DEFAULT '0',
  `created_by_alias` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int unsigned NOT NULL DEFAULT '0',
  `metakey` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadesc` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadata` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `featured` tinyint unsigned NOT NULL DEFAULT '0' COMMENT 'Set if article is featured.',
  `xreference` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'A reference to enable linkages to external data sets.',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `version` int unsigned NOT NULL DEFAULT '1',
  `hits` int unsigned NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`),
  KEY `idx_access` (`access`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_state` (`published`),
  KEY `idx_catid` (`catid`),
  KEY `idx_createdby` (`created_by`),
  KEY `idx_featured_catid` (`featured`,`catid`),
  KEY `idx_language` (`language`),
  KEY `idx_xreference` (`xreference`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_contact_details`
--

LOCK TABLES `rlp_contact_details` WRITE;
/*!40000 ALTER TABLE `rlp_contact_details` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_contact_details` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_content`
--

DROP TABLE IF EXISTS `rlp_content`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_content` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `asset_id` int unsigned NOT NULL DEFAULT '0' COMMENT 'FK to the #__assets table.',
  `title` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `introtext` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `fulltext` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `state` tinyint NOT NULL DEFAULT '0',
  `catid` int unsigned NOT NULL DEFAULT '0',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by` int unsigned NOT NULL DEFAULT '0',
  `created_by_alias` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int unsigned NOT NULL DEFAULT '0',
  `checked_out` int unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `images` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `urls` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `attribs` varchar(5120) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `version` int unsigned NOT NULL DEFAULT '1',
  `ordering` int NOT NULL DEFAULT '0',
  `metakey` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadesc` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `access` int unsigned NOT NULL DEFAULT '0',
  `hits` int unsigned NOT NULL DEFAULT '0',
  `metadata` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `featured` tinyint unsigned NOT NULL DEFAULT '0' COMMENT 'Set if article is featured.',
  `language` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The language code for the article.',
  `xreference` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `note` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  PRIMARY KEY (`id`),
  KEY `idx_access` (`access`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_state` (`state`),
  KEY `idx_catid` (`catid`),
  KEY `idx_createdby` (`created_by`),
  KEY `idx_featured_catid` (`featured`,`catid`),
  KEY `idx_language` (`language`),
  KEY `idx_xreference` (`xreference`),
  KEY `idx_alias` (`alias`(191))
) ENGINE=InnoDB AUTO_INCREMENT=263 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_content`
--

LOCK TABLES `rlp_content` WRITE;
/*!40000 ALTER TABLE `rlp_content` DISABLE KEYS */;
INSERT INTO `rlp_content` VALUES (11,525,'What is Reactome ?','about-reactome','<h3>Mission Statement</h3>\r\n<p><span>REACTOME is an open-source, open access, manually curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic and clinical research, genome analysis, modeling, systems biology and education. Founded in 2003, the Reactome project is led by Lincoln Stein of </span><a href=\"http://oicr.on.ca/\"><span>OICR</span></a><span>, Peter D’Eustachio of <a href=\"https://nyulangone.org/\">NYU Langone Health</a></span><span>, Henning Hermjakob of </span><a href=\"http://www.ebi.ac.uk/\"><span>EMBL-EBI</span></a><span>, and Guanming Wu of </span><a href=\"http://www.ohsu.edu/\"><span>OHSU</span></a><span>.</span></p>\r\n<h3>The Reactome Project</h3>\r\n<p><span>Biological information has become so abundant and complex in recent years that it is difficult, if not impossible, even for expert individuals to manage in traditional publication formats and with existing knowledge management tools. It is an ongoing challenge for researchers to keep up-to-date on research developments in their fields, and identify relevant research to support their own studies without devoting too much time collecting unconnected information. The Reactome group has recognized this challenge and is developing a set of novel online resources that use features of the electronic media to organize biological pathway  information in ways that provide for more efficient access and that allow new forms of analysis that were not possible with information stored in the traditional printed literature.</span></p>\r\n<p><span>The cornerstone of Reactome is a freely available, open source relational database of signaling and metabolic molecules and their relations organized into biological pathways and processes. The core unit of the Reactome data model is the reaction. Entities (nucleic acids, proteins, complexes, vaccines, anti-cancer therapeutics and small molecules) participating in reactions form a network of biological interactions and are grouped into pathways. Examples of biological pathways in Reactome include </span><a href=\"PathwayBrowser/#/R-HSA-1430728\"><span>classical intermediary metabolism</span></a><span>, </span><a href=\"PathwayBrowser/#/R-HSA-162582\"><span>signaling</span></a><span>, </span><a href=\"PathwayBrowser/#/R-HSA-74160\"><span>transcriptional regulation</span></a><span>, </span><a href=\"PathwayBrowser/#/R-HSA-109581&amp;PATH=R-HSA-5357801\"><span>apoptosis</span></a><span> and </span><a href=\"PathwayBrowser/#/R-HSA-1643685\"><span>disease</span></a><span>. The Reactome curation process for a pathway is similar to the editing of a scientific review. An external domain expert provides his or her expertise, a curator formalizes it into the database structure, and an external domain expert reviews the representation. A system of evidence tracking ensures that all assertions are backed up by the primary literature.</span></p>\r\n<p><span>The Reactome website is designed to literally give the user a graphical map of known biological processes and pathways that is also an interface which the user can ‘click through’ to authoritative detailed information on components and their relations. The Reactome database and website enable scientists, researchers, students, and educators to find, organize, and utilize biological information to support </span><a href=\"PathwayBrowser/\"><span>data visualization</span></a><span>, </span><a href=\"dev/\"><span>integration</span></a><span> and </span><a href=\"PathwayBrowser/#TOOL=AT\"><span>analysis</span></a><span>. Reactome pathway, reaction and molecules pages extensively cross-reference to over 100 different online bioinformatics resources, including NCBI Gene, Ensembl and UniProt databases, the UCSC Genome Browser, ChEBI small molecule databases, and the PubMed literature database.</span></p>\r\n<p><span>Reactome is used by clinicians, geneticists, genomics researchers, and molecular biologists to interpret the results of high-throughput experimental studies, by bioinformaticians seeking to develop novel algorithms for mining knowledge from genomic studies, and by systems biologists building predictive models of normal and disease variant pathways.</span></p>\r\n<p><span>All data and software are freely available for </span><a href=\"download-data\"><span>download</span></a><span>. Interaction, reaction and pathway data are provided as downloadable flat, </span><a href=\"http://neo4j.com/\"><span>Neo4j GraphDB</span></a><span>, </span><a href=\"http://www.mysql.com/\"><span>MySQL</span></a><span>, </span><a href=\"http://www.biopax.org/\"><span>BioPAX</span></a><span>, </span><a href=\"https://synonym.caltech.edu\"><span>SBML </span></a><span>and </span><a href=\"https://github.com/HUPO-PSI/miTab/blob/master/PSI-MITAB27Format.md\"><span>PSI-MITAB</span></a><span> files and are also accessible through our Web Services APIs. Software and instructions for local installation of the Reactome database, website, and data entry tools are also available to support independent pathway curation.</span></p>\r\n<h3>For more information:</h3>\r\n<p>Visit our Youtube Video for an <a href=\"https://youtu.be/cA7lQACsgZk\">Introduction to Reactome</a>! </p>\r\n<p><span>If you have any feedback or questions, please contact us at the Reactome </span><a href=\"mailto:help@reactome.org\"><span>HelpDesk</span></a><span>.</span></p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 15:01:50','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,11,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(13,528,'Reactome Team','team','<p style=\"text-align: justify;\">The Reactome group consists of an international multidisciplinary team from OICR, OHSU, EMBL-EBI and NYULMC, with expertise in pathway curation and annotation, software development, and training and outreach, dedicated to providing the research community with openly accessible biological pathway knowledge.</p>\r\n<p> </p>\r\n<p><b>Principal Investigators</b></p>\r\n<p><a href=\"https://oicr.on.ca/investigators/lincoln-stein/\" target=\"_blank\" rel=\"noopener\">Lincoln Stein</a> (<a href=\"http://oicr.on.ca/\" target=\"_blank\" rel=\"noopener\">OICR</a>)</p>\r\n<p><a href=\"https://med.nyu.edu/faculty/peter-g-d-eustachio\" target=\"_blank\" rel=\"noopener\">Peter D\'Eustachio</a> (<a href=\"http://nyulangone.org/\" target=\"_blank\" rel=\"noopener\">NYULMC</a>)</p>\r\n<p><a href=\"https://www.ebi.ac.uk/about/people/henning-hermjakob\" target=\"_blank\" rel=\"noopener\">Henning Hermjakob</a> (<a href=\"http://www.ebi.ac.uk/\" target=\"_blank\" rel=\"noopener\">EMBL-EBI</a>)</p>\r\n<p><a href=\"http://www.ohsu.edu/xd/education/schools/school-of-medicine/academic-programs/graduate-studies/faculty/grad-studies-faculty.cfm?facultyid=818\" target=\"_blank\" rel=\"noopener\">Guanming Wu</a> (<a href=\"http://www.ohsu.edu/\" target=\"_blank\" rel=\"noopener\">OHSU</a>)</p>\r\n<p> </p>\r\n<p><b>Ontario Institute for Cancer Research</b></p>\r\n<p>Lincoln Stein</p>\r\n<p>Marc Gillespie</p>\r\n<p>Nancy Li</p>\r\n<p>Bruce May</p>\r\n<p>Marija Orlic-Milacic</p>\r\n<p>Robert Petryszak</p>\r\n<p>Karen Rothfels</p>\r\n<p>Ralf Stephan</p>\r\n<p>Joel Weiser</p>\r\n<p>Adam Wright</p>\r\n<p> </p>\r\n<p><b>New York University Langone Medical Center</b></p>\r\n<p>Peter D\'Eustachio</p>\r\n<p>Lisa Matthews</p>\r\n<p>Veronica Shamovsky</p>\r\n<p>  </p>\r\n<p><b>European Bioinformatics Institute</b></p>\r\n<p>Henning Hermjakob</p>\r\n<p>Chuqiao Gong</p>\r\n<p>Eliot Ragueneau</p>\r\n<p>Cristoffer Sevilla</p>\r\n<p>Krishna Tiwari</p>\r\n<p> </p>\r\n<p><b>Oregon Health &amp; Science University</b></p>\r\n<p>Guanming Wu</p>\r\n<p>Deidre Beavers</p>\r\n<p></p>\r\n<p> </p>\r\n<p><b>Alumni</b></p>\r\n<p>Ewan Birney</p>\r\n<p>Bernard de Bono</p>\r\n<p></p>\r\n<p>Liam Beckman</p>\r\n<p>Tim Brunson</p>\r\n<p>Michael Caudy</p>\r\n<p>Patrick Conley</p>\r\n<p>Justin Cook</p>\r\n<p>David Croft</p>\r\n<p>Corina Duenas</p>\r\n<p>Antonio Fabregat</p>\r\n<p>Phani Garapati</p>\r\n<p>Gopal Gopinath</p>\r\n<p>Yusra Haider</p>\r\n<p>Kerstin Hausmann</p>\r\n<p>Robin Haw</p>\r\n<p>Jill Hemish</p>\r\n<p>Bijay Jassal</p>\r\n<p>Geeta Joshi-Tope</p>\r\n<p>Steve Jupe</p>\r\n<p>Sarah Keating</p>\r\n<p>Maximilian Koch</p>\r\n<p>Florian Korninger</p>\r\n<p>Pascual Lorente</p>\r\n<p>Shahana Mahajan</p>\r\n<p>Sheldon McKay</p>\r\n<p>Nelson Ndegwa</p>\r\n<p>Elizabeth Nickerson</p>\r\n<p>Gavin O\'Kelly</p>\r\n<p>Nasim Sanati</p>\r\n<p>Carl Schmidt</p>\r\n<p>Esther Schmidt</p>\r\n<p>Solomon Shorser</p>\r\n<p>Konstantinos Sidiropoulos</p>\r\n<p>Heeyeon Song</p>\r\n<p>Thawfeek Varusai</p>\r\n<p>Imre Vastrik</p>\r\n<p>Guilherme Viteri</p>\r\n<p>Marissa Webber</p>\r\n<p>Mark Williams</p>\r\n<p>Florent Yvon</p>\r\n<p>Christina Yung</p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 15:59:20','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,10,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(14,529,'Scientific Advisory Board','scientific-advisory-board','<p><span>Our Scientific Advisory Board members are internationally recognized, researchers. The SAB meets annually to i) discuss the scientific agenda, ii) explore ways to expand our research efforts, and iii) critically review our database, curation practices and software development programs.</span></p>\r\n<ul style=\"list-style-type: square;\">\r\n<li>Evan Bolton <a href=\"https://www.ncbi.nlm.nih.gov\">NCBI</a></li>\r\n<li>Fiona Brinkman <a href=\"https://www.brinkman.mbb.sfu.ca\">Simon Fraser University</a> </li>\r\n<li>Ethan Cerami <a href=\"https://www.dana-farber.org/find-a-doctor/ethan-cerami\">Dana Farber Cancer Institute</a></li>\r\n<li><span>Anna Niarakis <a href=\"https://www.genhotel.univ-evry.fr/en/team/members/anna-niarakis.html/\">University of Evry Val d\'Essonne</a></span></li>\r\n<li><span>Tudor Oprea <a href=\"https://www.roivant.com/science\">Roivant Sciences</a></span></li>\r\n<li>Paul Sternberg <a href=\"https://wormlab.caltech.edu\">CalTech</a></li>\r\n</ul>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 11:59:59','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":\"\",\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":\"\",\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":\"\",\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,8,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(15,530,'Other Reactomes','other-reactomes','<p>The following Reactomes are known to us. Clicking on the name of any one of these will take you either to the home page or a description of planned work.</p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 16:00:24','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,9,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(16,531,'License Agreement','license-agreement','<p>Reactome is an open source and open access resource, available to anyone. Usage of Reactome material is covered by two Creative Commons licenses:</p>\r\n<p>The terms of the <a href=\"https://creativecommons.org/publicdomain/zero/1.0/\">Creative Commons&nbsp;Public Domain (CC0)&nbsp;License<strong>&nbsp;</strong></a>apply to all Reactome annotation files, e.g. identifier mapping data, specialized data files, and interaction data derived from Reactome.</p>\r\n<p>The terms of the <a href=\"https://creativecommons.org/licenses/by/4.0/\">Creative Commons&nbsp;Attribution 4.0 International (CC BY 4.0)&nbsp;License</a><strong>&nbsp;</strong>apply to all&nbsp;software and code, e.g. relating to the functionality of the <a href=\"https://reactome.org\">reactome.org</a>,&nbsp;derived websites and webservices, the Curator Tool, the Functional Interaction application, SQL and Graph Database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials.</p>\r\n<p style=\"text-align: justify;\"><span>KEGG gene and pathway annotations, used to construct the Reactome Functional Interaction (FI) Network, are provided under license to OICR by Pathway Solutions, Inc. The Reactome FI Network data set is intended for use only by the original recipient. The recipient may not distribute the Reactome Functional interaction data files to other users without a license from Pathway Solutions, Inc.</span></p>\r\n<p>The FoamTree (Voronoi treemaps), used to construct the ReacFoam genome-wide pathway overview, is provided under license to EMBL-EBI by Carrot Search, Inc. A fully functional demo version, which displays the FoamTree logo in one of the cells, is free of charge.</p>\r\n<p>For information about how to properly credit data use, please review the <a href=\"https://creativecommons.org/faq/\">Creative Commons FAQ</a> or contact the <a href=\"mailto:help@reactome.org\">Reactome Helpdesk.</a></p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 16:01:10','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,7,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(17,532,'Disclaimer','disclaimer','<p style=\"text-align: justify;\"><b>DISCLAIMER OF LIABILITY AND OF WARRANTIES </b></p>\r\n<p style=\"text-align: justify;\"><span>In no event shall Ontario Institute for Cancer Research (OICR), New York University (NYU), Oregon Health &amp; Science University (OHSU), and/or EMBL-European Bioinformatics Institute (EBI) be liable for any use of the Reactome website, its contents or information derived thereof. The content of and information contained in the Reactome website is the opinion of the contributor and/or the author of such content and/or information and is not supplied as medical information. OICR, NYU, OHSU, and/or EBI accepts no responsibility or liability for any loss, cost, claim or expense arising from any reliance on such content or information. The content of this site is intended for educational and scientific research purposes only and not as a source of medical advice or consultation.</span></p>\r\n<p style=\"text-align: justify;\"><span>BY USING THIS SITE YOU AGREE TO ASSUME ALL RISKS ASSOCIATED WITH YOUR USE OR TRANSFER OF ANY AND ALL INFORMATION CONTAINED ON THIS SITE AND TO HOLD OICR, NYU, OHSU, AND/OR EBI HARMLESS FROM ANY CLAIMS RELATING TO CONTENT OR INFORMATION IN EXCHANGE FOR YOUR USE OF THE SITE. THE REACTOME SITE IS PROVIDED “AS IS” WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, OR NON-INFRINGEMENT. OICR, NYU, OHSU, and/or EBI MAKES NO WARRANTIES ABOUT THE ACCURACY, RELIABILITY, COMPLETENESS, OR TIMELINESS OF THE MATERIAL, SERVICES, SOFTWARE, TEXT, GRAPHICS, AND LINKS. DESCRIPTION OF, OR REFERENCES TO, PRODUCTS OR PUBLICATIONS DOES NOT IMPLY ENDORSEMENT OF THAT PRODUCT OR PUBLICATION.</span></p>\r\n<p style=\"text-align: justify;\"><span>OICR, NYU, OHSU, and/or EBI DOES NOT WARRANT THAT THE SITE WILL OPERATE ERROR-FREE OR THAT THIS SITE AND ITS SERVER ARE FREE OF COMPUTER VIRUSES OR OTHER HARMFUL MECHANISMS. IF YOUR USE OR TRANSFER OF THE SITE OR THE MATERIALS RESULTS IN THE NEED FOR SERVICING OR REPLACING EQUIPMENT OR DATA, OICR, NYU, OHSU, AND/OR EBI IS NOT RESPONSIBLE FOR THOSE COSTS.</span></p>\r\n<p style=\"text-align: justify;\"><b>INDEMNITY</b></p>\r\n<p style=\"text-align: justify;\"><span>You agree to defend, indemnify, and hold harmless OICR, NYU, OHSU, and/or EBI, its officers, directors, employees and agents, from and against any and all claims, actions or demands, (including without limitation all legal and accounting fees) which may arise due to your use or transfer of the Reactome web site, its contents, or information thereof.</span></p>\r\n<p style=\"text-align: justify;\"><b>PRIVACY</b></p>\r\n<p style=\"text-align: justify;\"><span>Cookies are used by the search pages in order to remember your search settings. Some cookies persist after you exit the browser, but they are never used for either identification or tracking purposes.</span></p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-20 16:02:46','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(18,537,'Downloads','downloads','<p>Reactome provides <a href=\"http://en.wikipedia.org/wiki/Open_source\">open-source</a> and <a href=\"http://en.wikipedia.org/wiki/Open_data\">open-data</a>. We have continuously supported the major open-data standards, including <a href=\"http://www.biopax.org/\">BioPAX</a>, <a href=\"http://www.psidev.info/groups/molecular-interactions\">PSI-MITAB</a>, <a href=\"http://sbml.org/\">SBML</a> and <a href=\"https://sbgn.github.io/\">SBGN</a> export formats. The Reactome data and source code continue to be publicly accessible under the terms of Creative Commons Licenses. Please refer to our <a href=\"index.php?option=com_content&amp;view=article&amp;id=16:license-agreement&amp;catid=25\">License Agreement</a>.</p>\r\n<p>Please contact the Reactome <a href=\"mailto:help@reactome.org\">Helpdesk</a> if you have questions about the files. </p>\r\n<blockquote>\r\n<h3>Services and Repositories</h3>\r\n</blockquote>\r\n<table style=\"width: 1259px; height: 136px;\">\r\n<tbody>\r\n<tr>\r\n<td>\r\n<ul>\r\n<li style=\"list-style-type: circle;\"><span style=\"font-size: 10pt;\">Content Service: Our <a href=\"index.php?option=com_content&amp;view=article&amp;id=48\">Content Service</a> provides outside users with direct access to pathway data in Reactome via an API. </span></li>\r\n<li style=\"list-style-type: circle;\"><span style=\"font-size: 10pt;\">Analysis Service: The Reactome Pathway <a href=\"index.php?option=com_content&amp;view=article&amp;id=47\" style=\"text-align: justify; font-size: 14px;\">Analysis Tools</a><span style=\"text-align: justify; color: #333333;\"> offer direct access to pathway analysis via an API, and are also available for integration into third party websites.</span></span></li>\r\n<li style=\"list-style-type: circle;\"><a href=\"https://zenodo.org/records/12775583\" target=\"_blank\" rel=\"noopener\">Zenodo Repository</a>: The Zenodo download directory is created for every Reactome (reactome.org) release quarterly, starting from version 89 (06/2024), containing download files. </li>\r\n<li style=\"list-style-type: circle;\"><span style=\"font-size: 10pt;\"><a href=\"https://github.com/reactome\">GitHub Repository</a>   </span></li>\r\n</ul>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p> </p>\r\n<blockquote>\r\n<h3>Database</h3>\r\n</blockquote>\r\n<table style=\"width: 1416px; height: 257px;\">\r\n<tbody>\r\n<tr>\r\n<td style=\"text-align: left; vertical-align: top;\">\r\n<h4>Reactome Knowledge Graph</h4>\r\n<p>For more information on the installation of the Reactome Graph Database,</p>\r\n<p>please refer to the <a href=\"index.php?option=com_content&amp;view=article&amp;id=45#GetStarted\">Get Started</a> section of the <a href=\"index.php?option=com_content&amp;view=article&amp;id=45\">Graph Database</a> </p>\r\n<p>documentation in the <a href=\"dev\">Developer Guide</a>.</p>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/reactome.graphdb.tgz\">Reactome Graph Database</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/reactome.graphdb.dump\" download=\"\">Reactome Graph Database Dump File</a> </li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"https://hub.docker.com/r/reactome/graphdb/tags\" target=\"_blank\" rel=\"noopener\">Reactome Graph Database Docker Container</a></li>\r\n</ul>\r\n</td>\r\n<td style=\"text-align: left; vertical-align: top;\">\r\n<h4>MySQL dumps of Reactome database                                 </h4>\r\n<ul>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/databases/gk_current.sql.gz\">Main database</a></li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/databases/gk_stable_ids.sql.gz\">Stable identifiers database</a></li>\r\n</ul>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p></p>\r\n<blockquote>\r\n<h3>Mapping Files</h3>\r\n</blockquote>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td>\r\n<h4>Identifier mapping files</h4>\r\n<p>Mapping files link the source database identifier to the lowest level pathway diagram or subset of the pathway, all levels of the pathway hierarchy or database identifier to all reactions. {modal url=\"index.php?option=com_content&amp;view=article&amp;id=60\"}<span style=\"font-size: 8pt;\"><i class=\"fa fa-info-circle\" style=\"font-size: 23px;\"></i> {/modal} </span></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<table style=\"width: 1399px; height: 78px;\">\r\n<tbody>\r\n<tr>\r\n<td style=\"vertical-align: top;\">\r\n<p><i class=\"fa fa-folder\"><br /></i><span style=\"font-family: FontAwesome;\"></span>Lowest level pathway diagram / Subset of the pathway</p>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2Reactome.txt\">UniProt to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2Reactome.txt\">ChEBI to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2Reactome.txt\">ENSEMBL to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2Reactome.txt\">miRBase to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2Reactome.txt\">NCBI to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/GtoP2Reactome.txt\">GtoP to pathways</a></li>\r\n</ul>\r\n</td>\r\n<td style=\"vertical-align: top;\">\r\n<p><i class=\"fa fa-folder\"></i>All levels of the pathway hierarchy.     </p>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2Reactome_All_Levels.txt\">UniProt to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2Reactome_All_Levels.txt\">ChEBI to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2Reactome_All_Levels.txt\">ENSEMBL to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2Reactome_All_Levels.txt\">miRBase to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2Reactome_All_Levels.txt\">NCBI to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/GtoP2Reactome_All_Levels.txt\">GtoP to All pathways</a></li>\r\n</ul>\r\n</td>\r\n<td style=\"vertical-align: top;\">\r\n<p><i class=\"fa fa-folder\"></i>Database identifier to all reactions.         </p>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2ReactomeReactions.txt\">UniProt to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2ReactomeReactions.txt\">ChEBI to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2ReactomeReactions.txt\">ENSEMBL to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2ReactomeReactions.txt\">miRBase to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2ReactomeReactions.txt\">NCBI to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/GtoP2ReactomeReactions.txt\">GtoP to All reactions</a></li>\r\n</ul>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Physical Entity (PE) Identifier mapping files </h4>\r\n<p id=\"downloads-page\" class=\"moduletable-sfx4\">Mapping files link the source database identifier to the lowest level pathway diagram or subset of the pathway. <span style=\"font-size: 8pt;\">{modal url=\"index.php?option=com_content&amp;view=article&amp;id=86\"}  <span style=\"font-size: 8pt;\"><i class=\"fa fa-info-circle\" style=\"font-size: 23px;\"></i> {/modal} </span></span></p>\r\n<table style=\"width: 1409px; height: 158px;\">\r\n<tbody>\r\n<tr>\r\n<td style=\"text-align: left; vertical-align: top;\">\r\n<p><i class=\"fa fa-folder\"></i>Lowest level pathway diagram / Subset of the pathway</p>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2Reactome_PE_Pathway.txt\">UniProt to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2Reactome_PE_Pathway.txt\">ChEBI to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2Reactome_PE_Pathway.txt\">ENSEMBL to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2Reactome_PE_Pathway.txt\">miRBase to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2Reactome_PE_Pathway.txt\">NCBI to pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/GtoP2Reactome_PE_Pathway.txt\">GtoP to pathways</a></li>\r\n</ul>\r\n</td>\r\n<td style=\"text-align: left; vertical-align: top;\">\r\n<p><i class=\"fa fa-folder\"></i>All levels of the pathway hierarchy.     </p>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2Reactome_PE_All_Levels.txt\">UniProt to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2Reactome_PE_All_Levels.txt\">ChEBI to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2Reactome_PE_All_Levels.txt\">ENSEMBL to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2Reactome_PE_All_Levels.txt\">miRBase to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2Reactome_PE_All_Levels.txt\">NCBI to All pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/GtoP2Reactome_PE_All_Levels.txt\">GtoP to All pathways</a></li>\r\n</ul>\r\n</td>\r\n<td style=\"text-align: left; vertical-align: top;\">\r\n<p><i class=\"fa fa-folder\"></i>Database identifier to all reactions           </p>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/UniProt2Reactome_PE_Reactions.txt\">UniProt to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ChEBI2Reactome_PE_Reactions.txt\">ChEBI to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ensembl2Reactome_PE_Reactions.txt\">ENSEMBL to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/miRBase2Reactome_PE_Reactions.txt\">miRBase to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/NCBI2Reactome_PE_Reactions.txt\">NCBI to All reactions</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/GtoP2Reactome_PE_Reactions.txt\">GtoP to All reactions</a></li>\r\n</ul>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Other Mappings</h4>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/models2pathways.tsv\">BioModels to Pathways</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ReactionPMIDS.txt\">Reaction PMIDs</a> {modal url=\"index.php?option=com_content&amp;view=article&amp;id=87\"}<span style=\"font-size: 8pt;\"> <i class=\"fa fa-info-circle\" style=\"font-size: 23px;\"></i> {/modal} </span></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Reactome2OMIM.txt\">Reactome to OMIM</a> {modal url=\"index.php?option=com_content&amp;view=article&amp;id=88\"}<span style=\"font-size: 8pt;\"><i class=\"fa fa-info-circle\" style=\"font-size: 23px;\"></i> {/modal} </span></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ProteinRoleReaction.txt\">Protein Role Reaction</a> {modal url=\"index.php?option=com_content&amp;view=article&amp;id=89\"}<span style=\"font-size: 8pt;\"><i class=\"fa fa-info-circle\" style=\"font-size: 23px;\"></i> {/modal} </span></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"https://reactome.org/download/current/disease_variant_ewas_mapping.tsv\">Disease Variant EWAS mapping file</a> (Read more about our disease variant annotations<a href=\"https://doi.org/10.1093/database/baae031\"> here</a>).</li>\r\n</ul>\r\n<h4>Icons</h4>\r\n<p class=\"p1\"><span class=\"s1\">The <a href=\"index.php?option=com_content&amp;view=article&amp;id=118\" hreflang=\"en\">icon mapping files</a> link the source database identifier to elements of the icon library. This folder contains a number of different mapping files to reflect the different source databases. {modal url=\"index.php?option=com_content&amp;view=article&amp;id=117\"}<span style=\"font-size: 8pt;\"><i class=\"fa fa-info-circle\" style=\"font-size: 23px;\"></i> {/modal} </span></span></p>\r\n<h4>Complexes</h4>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ComplexParticipantsPubMedIdentifiers_human.txt\">Human complexes with their participating protein molecules</a></li>\r\n</ul>\r\n<h4>Reactome External Identifiers</h4>\r\n<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/reactome_stable_ids.txt\">Reactome Stable Identifiers</a> </li>\r\n</ul>\r\n<p>  </p>\r\n<blockquote>\r\n<h3> <span style=\"font-size: 1.17em; background-color: inherit; color: inherit; font-family: inherit; caret-color: auto;\">Pathway Information</span></h3>\r\n</blockquote>\r\n<table style=\"width: 2104px; height: 468px;\">\r\n<tbody>\r\n<tr>\r\n<td style=\"vertical-align: top;\">\r\n<h4>Pathway Information</h4>\r\n<p>These files describe the Reactome pathway hierarchy {modal url=\"index.php?option=com_content&amp;view=article&amp;id=61\"}<span style=\"font-size: 8pt;\"><i class=\"fa fa-info-circle\" style=\"font-size: 23px;\"></i> {/modal} </span></p>\r\n<p></p>\r\n<p><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ReactomePathways.txt\">Complete List of Pathways</a></p>\r\n<p><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/ReactomePathwaysRelation.txt\">Pathways hierarchy relationship</a></p>\r\n<p><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Complex_2_Pathway_human.txt\">Complexes to Pathways (Human)</a></p>\r\n<p><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/Ewas2Pathway_human.txt\">EWAS to Pathways (Human)</a></p>\r\n</td>\r\n<td style=\"vertical-align: top;\">\r\n<h4>Pathway Diagrams &amp; Figures</h4>\r\n<p><i class=\"fa fa-file-zip-o\"></i>Human pathway diagrams &amp; figures: <a href=\"/download/current/diagrams.svg.tgz\">SVG</a> or <a href=\"/download/current/diagrams.png.tgz\">PNG</a></p>\r\n</td>\r\n<td style=\"vertical-align: top;\">\r\n<h4>Gene Ontology Association Files</h4>\r\n<p>Read more about our GO-CAM project<a href=\"https://doi.org/10.1093/bioinformatics/btab325\"> here</a></p>\r\n<p><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/gene_association.reactome.gz\">Gene association file</a></p>\r\n<p><i class=\"fa fa-file-text-o\"></i><a href=\"https://reactome.org/download/current/Pathways2GoTerms_human.txt\">Pathways to Gene Ontology</a></p>\r\n<p><i class=\"fa fa-file-text-o\"></i><a href=\"https://reactome.org/download/current/Reactions2GoTerms_human.txt\">Reactions to Gene Ontology</a> </p>\r\n<p></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td style=\"vertical-align: top;\">\r\n<h4>Systems Biology Formats</h4>\r\n<p><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/homo_sapiens.3.1.sbml.tgz\">Human reactions</a> in <a href=\"http://sbml.org/\">SBML</a> (level 3, v1)</p>\r\n<p><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/all_species.3.1.sbml.tgz\">All Species reactions</a> in <a href=\"http://sbml.org/\">SBML</a> (level 3, v1)</p>\r\n<p><i class=\"fa fa-file-zip-o\"></i>Events in the <a href=\"/download/current/biopax2.zip\">BioPAX level 2 </a>format</p>\r\n<p><i class=\"fa fa-file-zip-o\"></i>Events in the <a href=\"/download/current/biopax.zip\">BioPAX level 3 </a>format</p>\r\n<p><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/homo_sapiens.sbgn.tar.gz\">Human pathway diagrams</a> in the <a href=\"https://sbgn.github.io/\">SBGN</a> format</p>\r\n</td>\r\n<td style=\"vertical-align: top;\">\r\n<h4>Reactome Pathways Gene Set</h4>\r\n<p><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/ReactomePathways.gmt.zip\">Reactome Pathways Gene Set</a></p>\r\n<p></p>\r\n</td>\r\n<td style=\"vertical-align: top;\">\r\n<h4>Ontology Files</h4>\r\n<p><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/protege_files.tar.gz\">Protégé 2.0 ontology files</a> </p>\r\n</td>\r\n<td style=\"vertical-align: top;\">\r\n<h4></h4>\r\n</td>\r\n<td style=\"vertical-align: top;\"></td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p> </p>\r\n<blockquote>\r\n<h3>Protein-Protein Interaction Files </h3>\r\n</blockquote>\r\n<table>\r\n<tbody>\r\n<tr><th style=\"text-align: center;\"></th><th style=\"text-align: center;\"></th></tr>\r\n<tr>\r\n<td style=\"vertical-align: top;\">\r\n<h4>Protein-Protein Interactions derived from Reactome pathways</h4>\r\n<p>Interactions are inferred between all protein components of a complex. To limit the creation of false-positive inferences, only complexes with four or fewer protein components are considered. <a href=\"https://github.com/reactome/interaction-exporter/wiki/Methods\" target=\"_blank\" rel=\"noindex,nofollow noopener\">More information</a></p>\r\n<p></p>\r\n<p><i class=\"fa fa-folder\"></i>{modal url=\"https://psicquic.github.io/MITAB27Format.html\"}PSI-MITAB format{/modal} interaction files</p>\r\n<ul>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/interactors/reactome.all_species.interactions.psi-mitab.txt\">All interactions, including non-human species</a></li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/interactors/reactome.homo_sapiens.interactions.psi-mitab.txt\" title=\"Human protein-protein interactions\">Human protein-protein interactions</a></li>\r\n<li></li>\r\n</ul>\r\n<p><i class=\"fa fa-folder\"></i>{modal url=\"index.php?option=com_content&amp;view=article&amp;id=62\"}TAB-delimited format{/modal} interaction files</p>\r\n<ul>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/interactors/reactome.all_species.interactions.tab-delimited.txt\">All interactions, including non-human species</a></li>\r\n<li><i class=\"fa fa-file-zip-o\"></i><a href=\"/download/current/interactors/reactome.homo_sapiens.interactions.tab-delimited.txt\">Human protein-protein interactions</a></li>\r\n</ul>\r\n<p></p>\r\n<p></p>\r\n</td>\r\n<td style=\"vertical-align: top;\"></td>\r\n</tr>\r\n<tr>\r\n<td style=\"vertical-align: top;\">\r\n<h4>Functional interactions (FIs) derived from Reactome, and other pathway and interaction databases</h4>\r\n<p>For details about FIs, please see <a href=\"https://genomebiology.biomedcentral.com/articles/10.1186/gb-2010-11-5-r53\">Wu G, Feng X, Stein L</a>. To use these FI networks, please try our <a href=\"index.php?option=com_content&amp;view=article&amp;id=44\" title=\"ReactomeFIViz Cytoscape app\">ReactomeFIViz Cytoscape app</a>.</p>\r\n<p></p>\r\n<p><i class=\"fa fa-file-zip-o\"></i><a href=\"https://reactome.org/download/tools/ReatomeFIs/FIsInGene_070323_with_annotations.txt.zip\" target=\"_blank\" rel=\"noopener noreferrer\">Version 2022</a> (Constructed with a new random forest classifier. For more information, refer to <a href=\"https://www.biorxiv.org/content/10.1101/2023.06.05.543335v1\">Illuminating Dark Proteins using Reactome Pathways</a>)</p>\r\n<p><i class=\"fa fa-file-zip-o\"></i><a href=\"https://reactome.org/download/tools/ReatomeFIs/FIsInGene_122921_with_annotations.txt.zip\" target=\"_blank\" rel=\"noopener noreferrer\">Version 2021</a></p>\r\n<p><i class=\"fa fa-file-zip-o\"></i><a href=\"https://reactome.org/download/tools/ReatomeFIs/FIsInGene_122220_with_annotations.txt.zip\" target=\"_blank\" rel=\"noopener noreferrer\">Version 2020</a> (See note below)</p>\r\n<p><i class=\"fa fa-file-zip-o\"></i><a href=\"https://reactome.org/download/tools/ReatomeFIs/FIsInGene_020720_with_annotations.txt.zip\" target=\"_blank\" rel=\"noopener noreferrer\">Version 2019</a></p>\r\n<p><i class=\"fa fa-file-zip-o\"></i><a href=\"https://reactome.org/download/tools/ReatomeFIs/FIsInGene_122718_with_annotations.txt.zip\" target=\"_blank\" rel=\"noopener noreferrer\">Version 2018</a></p>\r\n<p></p>\r\n<p><em><strong>Note: </strong>The number of predicted FIs has dropped about 50% from Version 2019 to Version 2020 most likely because we excluded BioGrid protein-protein interactions collected in iRefIndex. We found an abnormal high odds ratio for mapped worm protein-protein interaction feature from BioGrid, which was much higher than human PPIs (120 vs. 80) and other no-human PPI features.</em></p>\r\n<h5 class=\"moduletable-sfx4\"></h5>\r\n</td>\r\n<td style=\"vertical-align: top;\"></td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<blockquote>\r\n<h3>Curation Team Software</h3>\r\n</blockquote>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p>The Curator tool is meant for use by curators to annotate biological pathways based on the Reactome schema. Please note that this tool will only work with a database using our internal database format; the database available from this web page will not work with them.</p>\r\n<ul>\r\n<li><i class=\"fa fa-book\"></i><a href=\"index.php?option=com_content&amp;view=article&amp;id=63\">Curator Tool (version 4.0.2, build 117, October 14, 2023)</a></li>\r\n</ul>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p> </p>\r\n<blockquote>\r\n<h3>Our textbook of biological pathways and processes</h3>\r\n</blockquote>\r\n<ul>\r\n<li><i class=\"fa fa-file-pdf-o\"></i><a href=\"/download/current/TheReactomeBook.pdf.tgz\">PDF format</a></li>\r\n</ul>\r\n<script type=\"text/javascript\">\r\njQuery.get( \"/ContentService/data/database/version\", function( data ) {\r\n// or use build the anchor and replace properly.\r\n  jQuery(\".reactome-db-version\").html( data );\r\n  // it will print 77\r\n});\r\n</script>','',1,32,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-21 14:47:36','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(22,556,'Outreach','outreach','<p style=\"text-align: justify;\"><span>Reactome’s continued success will be determined by the extent to which we reach out to current and new potential users. We will reach out to bioinformaticians, traditional molecular and cellular biologists, computational biologists, software developers, geneticists, clinicians, educators and students to ensure that all potential uses are aware of the resource and take full advantage of the services we offer. Therefore, we offer a series of outreach and training programs which will: </span></p>\r\n<ol style=\"list-style-type: lower-roman; text-align: justify;\">\r\n<li><span>foster the training of the next generation of scientists</span></li>\r\n<li><span>collaborate with data providers</span></li>\r\n<li><span>actively reach out to present and new users and collaborators. </span></li>\r\n</ol>\r\n<p style=\"text-align: justify;\"><span>Simultaneously, we encourage the reuse of the Reactome data model and software by supporting its adoption by outside groups, in particular by the cancer, clinical and disease research communities.</span></p>\r\n<p style=\"text-align: justify;\"><span>Outreach takes on a variety of forms, such as public talks, lectures, visiting universities and colleges and supporting traditional science events (meetings, conferences, and workshops). Education is a major focus of Reactome community outreach and aims to inform biologists, clinicians, and bioinformaticians about what Reactome can do. </span></p>\r\n<p style=\"text-align: justify;\"><span>Reactome training teaches new and current users how to use Reactome resources to help meet the efforts of their organizations, groups, or specific audience.</span></p>\r\n<p></p>\r\n<p style=\"text-align: justify;\"><b>Training and Community Outreach</b><span>. If you have a particular outreach question or request, you can contact our </span><a href=\"mailto:help@reactome.org\"><span>HelpDesk</span></a>.<span> All our training and outreach materials are available under a </span><a href=\"license\"><span>Creative Commons Attribution 4.0 Unported License</span></a><span>. We have just two simple requests, please attribute Reactome, and let us know if you use our presentations, posters or training resources.</span></p>\r\n<p style=\"text-align: justify;\"><b>Following Reactome.</b><span> </span><span>Reactome has embraced the social networking scene. You can follow Reactome on</span><span> </span><a href=\"http://www.linkedin.com/groups?gid=2118372\"><span>LinkedIn</span></a><span> and </span><a href=\"http://twitter.com/reactome\"><span>Twitter</span></a><span>.</span></p>\r\n<p style=\"text-align: justify;\"><b>Getting Involved. </b><span>Reactome is an open-source project that promotes community participation. We are eager for contributions from the community and would welcome inquiries from researchers interested in contributing information on pathways in their area of expertise. </span></p>\r\n<p style=\"text-align: justify;\"><a href=\"mailto:help@reactome.org\"><span>Contact us</span></a><span> with your ideas.</span></p>\r\n<p>If you would like to meet with our Outreach Lead, please feel free to book a meeting here:</p>\r\n<div class=\"meetings-iframe-container\" data-src=\"https://meetings.hubspot.com/oicrntl?embed=true\"></div>\r\n<script src=\"https://static.hsappstatic.net/MeetingsEmbed/ex/MeetingsEmbedCode.js\" type=\"text/javascript\"></script>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 15:45:28','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,7,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(23,557,'Events','events','<p><strong>Here is a list of all the Reactome talks, poster presentations, training events and workshops</strong></p>\r\n<p>{tab title=\"2024\"}</p>\r\n<h2>Upcoming:</h2>\r\n<p><a href=\"https://www.ashg.org/meetings/2024meeting/\" target=\"_blank\" rel=\"noopener\">American Society of Health Genomics,</a> November 5-9th, 2024.</p>\r\n<h2>Past Events:</h2>\r\n<p><span style=\"color: inherit;\">NIH Bioinformatics \"CCR Collaborative Bioinformatics Resource (CCBR)\" Virtual Meeting, Jan 9 2024</span></p>\r\n<p><span style=\"color: inherit;\">Keystone Symposia: Single-Cell Biology: Tissue Genomics, Technology and Disease, Jan 21-24 2024. <span class=\"s1\" style=\"color: #0000ff;\"><a href=\"images/events/Reactome_GSA.png\" target=\"_blank\" style=\"color: #0000ff;\">Poster</a></span></span></p>\r\n<p><span style=\"color: inherit;\">NIH Bioinformatics Training and Education Program Virtual Meeting, Feb14 2024<span style=\"color: #0000ff;\"><a href=\"https://cbiit.webex.com/recordingservice/sites/cbiit/recording/50f4927dad80103cbfd5005056812f37/playback\" target=\"_blank\" rel=\"noopener\" style=\"color: #0000ff;\"> Link to recording.</a></span></span></p>\r\n<p><span class=\"s1\" style=\"color: inherit;\"><span class=\"s1\">EMBL-EBI Training: </span>Introduction to RNA-seq and functional interpretation, Feb 24 2024</span></p>\r\n<p><span style=\"color: inherit;\">Keystone Symposia: Systems and Engineered Biology. The Reactome Knowledgebase: a resource for systems immunology and innate immunity research April 9, 2024. <span style=\"color: #0000ff;\"><span class=\"s1\"><a href=\"images/events/Reactome_Keystone_Immunology.png\" target=\"_blank\" style=\"color: #0000ff;\">Poster</a></span><span class=\"s2\">.</span></span></span></p>\r\n<p><span style=\"color: inherit;\">National Library of Medicine Bioinformatics and Clinical Informatics Program Fellowship, April 18, 2024</span></p>\r\n<p><span class=\"s1\" style=\"color: inherit;\">17th Annual <span style=\"color: #0000ff;\"><a href=\"https://ibdc.rcb.res.in/biocuration2024/\" target=\"_blank\" rel=\"noopener\" style=\"color: #0000ff;\">International Biocuration Conference</a></span>, March 5-8 2024 </span></p>\r\n<p><span style=\"color: inherit;\">The Annual Genetics Conference: TAGC Washington DC, March 6-10 2024 </span></p>\r\n<p><span style=\"color: inherit;\"><span class=\"s1\" style=\"background-color: inherit; font-family: inherit; caret-color: auto;\">EMBL-EBI Training: </span><span style=\"background-color: inherit; font-family: inherit; caret-color: auto;\">Introduction to multi-omics data integration and visualisation,  March 7 2024</span></span></p>\r\n<p><span class=\"s1\" style=\"color: inherit;\">EMBL-EBI Training: Reactome: from hierarchical pathway diagrams to multi-omics analysis of public data, March 25 2024</span></p>\r\n<p><span class=\"s1\" style=\"background-color: inherit; color: inherit; font-family: inherit; caret-color: auto;\">African Society for Bioinformatics and Computational Biology: Introduction of Reactome and IntAct, April 2024</span></p>\r\n<p><span class=\"s1\" style=\"color: inherit;\"><span class=\"s1\">EMBL-EBI Training: </span>Data-driven approaches to understanding dementia, April 12 2024</span></p>\r\n<p><span class=\"s1\" style=\"color: inherit;\"><span class=\"s1\">EMBL-EBI Training: </span>Introduction to metabolomics analysis, May 17 2024</span></p>\r\n<p>CANSSI Research Day, May 22 2024 </p>\r\n<p>Microbiology and Infectious Disease Day, June 10-11, 2024.</p>\r\n<p><span class=\"s1\" style=\"font-size: 10pt;\">Boston Festival of Genomics, June 12-13, 2024.</span></p>\r\n<p class=\"p2\"><span class=\"s1\" style=\"font-size: 10pt;\"><span class=\"s1\">EMBL-EBI Training: </span>Bioinformatics for immunologists, July 3 2024</span></p>\r\n<p><span style=\"font-size: 10pt;\">ISMB 2024, BOSC track, July 12-17 2024</span></p>\r\n<p>ISMB 2024: BioInfo Core \"AI and LLMs in cores: how are we using them now?\" July 12-17 2024</p>\r\n<p class=\"p2\"><span class=\"s1\" style=\"font-size: 10pt;\"><span class=\"s1\"><span style=\"font-size: 10pt;\">ISMB 2024, NIH ODSS track, July 12-17 2024</span></span></span></p>\r\n<p class=\"p2\"><span class=\"s1\" style=\"font-size: 10pt;\"><span class=\"s1\">EMBL-EBI Training: </span>Proteomics bioinformatics, July 19 2024</span></p>\r\n<p>Society of Developmental Biology Meeting, July 17 2024. <a href=\"images/events/2024_DevBio_116x120_no-abstract.jpg\" target=\"_blank\">Poster</a></p>\r\n<p>Online Webinar for Front Line Genomics: Integrative Multi-omics using July 17, 2024, Reactome GSA. <a href=\"https://app.livestorm.co/front-line-genomics/multi-omics-unleashed-webinar-2-enhancing-your-multi-omic-workflows-with-data-integration/live?s=7f203310-f5d0-4727-b1e3-dbbc83a6b7ca\" target=\"_blank\" rel=\"noopener\">Link to Recording.</a></p>\r\n<p>Bits in Bio Virtual Mixer, August 15th, 2024.<a href=\"https://lu.ma/wumm374h\" target=\"_blank\" rel=\"noopener\"> Link to register</a>. <a href=\"https://www.youtube.com/watch?v=O_DialfoboU\" target=\"_blank\" rel=\"noopener\">Link to recording.</a></p>\r\n<p>ACM<a href=\"https://kdd2024.kdd.org/\" target=\"_blank\" rel=\"noopener\"> KDD 2024</a>, Barcelona, August 25-29, 2024. </p>\r\n<p class=\"p2\"><span class=\"s1\" style=\"font-size: 10pt;\"></span></p>\r\n<p class=\"p2\"></p>\r\n<p>{tab title=\"2023\"}</p>\r\n<p class=\"p1\"><span class=\"s1\">London Festival of Genomics: A FAIR Biodata Analysis Resource in Practice: The Reactome Database of Curated Biomolecular Pathways</span></p>\r\n<p class=\"p2\"><span class=\"s1\">EMBL-EBI Training Workshop: Introduction of RNA-seq and functional interpretation: Exploring biological pathways</span></p>\r\n<p class=\"p2\"><span class=\"s1\">EMBL-EBI Training Workshop: Reactome Introduction</span></p>\r\n<p class=\"p2\"><span class=\"s1\">EMBL-EBI Training Workshop: Multiomics comparative pathway analysis using Reactome analysis tools and pathway browser</span></p>\r\n<p class=\"p2\"><span class=\"s1\">EMBL-EBI Training Workshop: Network Context for Large Scale Biology</span></p>\r\n<p class=\"p2\"><span class=\"s1\">Maastricht Centre for Systems Biology: Disease representation in Reactome</span></p>\r\n<p class=\"p2\"><span class=\"s1\">University of Padova: Community SARS-CoV-2 Curation Driven Emergent Experiences- Increased Curation Efficiency and Learned Lessons for the Future</span></p>\r\n<p class=\"p2\"><span class=\"s1\">EMBL-EBI Training Workshop: Bioinformatics resources for protein biology</span></p>\r\n<p class=\"p2\"><span class=\"s1\">EMBL-EBI Training Workshop: Reactome Introduction</span></p>\r\n<p class=\"p2\"><span class=\"s1\">HUPO 2023: Reactome Disease Representation</span></p>\r\n<p class=\"p2\"><span class=\"s1\">CNHUPO: Network Context for Large Scale Biology</span></p>\r\n<p class=\"p2\"><span class=\"s1\">UniAndes: Reactome Introduction</span></p>\r\n<p class=\"p2\"><span class=\"s1\">EMBL-EBI network resources</span></p>\r\n<p>ISCB RSGDream2024 Conference.<span style=\"color: #0000ff;\"> <a href=\"https://www.youtube.com/watch?v=cXuHUgdIILE&amp;ab_channel=ReactomeKnowledgebase\" target=\"_blank\" rel=\"noopener\" style=\"color: #0000ff;\">Link</a>. <span class=\"s1\"><a href=\"images/events/Reactome_IDG.png\" target=\"_blank\" style=\"color: #0000ff;\">Poster</a></span><span class=\"s2\">. </span></span></p>\r\n<p class=\"p2\"><span class=\"s1\"><span class=\"s1\">University of Toronto: Medicine By Design </span></span></p>\r\n<p class=\"p2\"><span class=\"s1\">CSHL: Genome Informatics Meeting</span></p>\r\n<p class=\"p2\"></p>\r\n<p>{tab title=\"2022\"}</p>\r\n<p>Virtual IDG meeting</p>\r\n<p>Bioinformatics Resources for Protein Biology</p>\r\n<p>Introduction to RNA-Seq and functional interpretation</p>\r\n<p>VIZBI</p>\r\n<p>{tab title=\"2021\"}</p>\r\n<p>Virtual IDG meeting</p>\r\n<p>CSHL Network Biology</p>\r\n<p>GO Consortium Meeting</p>\r\n<p>CBW Pathway and Network Analysis</p>\r\n<p>CBW Cancer Analysis</p>\r\n<p>ISMB 2021</p>\r\n<p>FASEB 2021</p>\r\n<p>ReactomeGSA training workshop</p>\r\n<p>GO Consortium Meeting</p>\r\n<p>COVID-19 Disease Map Group Meeting</p>\r\n<p>{tab title=\"2020\"}</p>\r\n<p>Face-to-Face IDG meeting</p>\r\n<p>Introduction to RNA-Seq and Functional Interpretation</p>\r\n<p>Bioinformatics Resources for Protein Biology</p>\r\n<p>CABANA seminar</p>\r\n<p>GO Consortium meeting</p>\r\n<p>Mathematics of Life</p>\r\n<p>Bioinformatics Community Conference</p>\r\n<p>Proteomics Society of India</p>\r\n<p>HUPO conference Training day</p>\r\n<p>ASHG 2020 Meeting</p>\r\n<p>COVID-19 Disease Map Group</p>\r\n<p>2020 Disease Maps Community Meeting</p>\r\n<p>ASCB/EMBO 2020</p>\r\n<p>ISMB 2020</p>\r\n<p>CBW</p>\r\n<p>ECCB 2020</p>\r\n<p>{tab title=\"2019\"}</p>\r\n<p>Bioinformatics Resources for Protein Biology, EMBL-EBI, Cambridge, UK, 28 February</p>\r\n<p>Society of Toxicology, Baltimore, MD, USA, 12 March</p>\r\n<p>VIZBI, EMBO, Heidelberg, Germany, 13-15 March</p>\r\n<p>Bioinformatics Resources for Protein Biology, EMBL-EBI, Cambridge, UK, 28 March</p>\r\n<p>GO Consortium Meeting, Cambridge, UK, 11-12 April</p>\r\n<p>Biocuration 2019, Cambridge, UK, 8 April</p>\r\n<p>Bioinformatics resources for protein biology, Cambridge University, Cambridge, UK, 1 May</p>\r\n<p>Biophysical Society Meeting, Baltimore, USA, 3 May</p>\r\n<p>ProteoNet Milan, ProteoNet Milan, Milan, Italy, 4 May</p>\r\n<p>American Society of Human Genetics, Houston, TX, USA, 15-19 Oct</p>\r\n<p>Cancer Moonshot Collaborative Meeting, NCI, DC, USA, 19-20 Nov</p>\r\n<p>ASCB-EMBO 2019 Meeting, NCI, DC, USA, 7-11 Dec</p>\r\n<p>{tab title=\"2018\"}</p>\r\n<p>EMBL-EBI Industry Programme Workshop, Pfizer, La Jolla, CA, USA, 7-8 February</p>\r\n<p>CBW Bioinformatics for Cancer Genomics, CSHL, NY 12-17 March</p>\r\n<p>Interactions and Pathways, EMBL-EBI, Cambridge, UK, 13 March</p>\r\n<p>Biocuration 2018, Fudan University, Shanghai, China, 04-08 April</p>\r\n<p>GO Consortium Meeting, NYUMC, New York, USA, 12-14 May 2018</p>\r\n<p>International Proteogenome Workshop, University of Kyoto, Kyoto, Japan, 13 May</p>\r\n<p>University of Chongqing, Chongqing, China, 13 May</p>\r\n<p>Network and Pathways, EMBL-EBI, Cambridge, UK, 17 May</p>\r\n<p>Disease Maps Community Meeting, Institut Curie, Paris, France, 21 June</p>\r\n<p>CBW Pathway and Network Analysis of Omics Data, Toronto, ON, Canada, 25-27 June</p>\r\n<p>ISMB 2018, N/A, Chicago, IL, USA, 6-10 July</p>\r\n<p>British Society for Proteome Research, Bradford, UK, 10 July</p>\r\n<p>Proteomics Bioinformatics, EMBL-EBI, Cambridge, UK, 20 July</p>\r\n<p>Universitat de València, Valencia, Spain, 27 July</p>\r\n<p>GlyGen consortium, USA, September 4</p>\r\n<p>Italian Proteomics Association 2018, Como, Italy, 5-7 September</p>\r\n<p>ECCB 2018, Athens, Greece, 8-12 September</p>\r\n<p>EMBL-EBI Industry Programme Workshop, EMBL-EBI, Cambridge, UK, 17 September</p>\r\n<p>Computational systems biology of cancer, Institut Curie, Paris, France, 24 September</p>\r\n<p>HUPO 2018, Orlando, USA, 30 September - 3 October</p>\r\n<p>Bioinformatics &amp; Functional Genomics in Zebrafish, EMBL-EBI, Cambridge, UK, 17 September</p>\r\n<p>CABANA, EMBL-EBI, Cambridge, UK, 6 November</p>\r\n<p>WikiPathways Summit 2018, UCSF, San Francisco, USA, 8-10 November</p>\r\n<p>Chinese Human Proteome Organisation Conference, Guangzhou, China, 16 November</p>\r\n<p>GO Consortium Meeting, Montreal, Canada, 17-19 November 2018</p>\r\n<p>Train Malta, EMBL-EBI, Cambridge, UK, 19 November</p>\r\n<p>Boehringer Ingelheim, Ridge Field, CT, USA, 31 November 2018</p>\r\n<p>GO Ontology Editors, Geneva, Switzerland, 10-13 December</p>\r\n<p>{tab title=\"2017\"}</p>\r\n<p><span>Pathway and Network Analysis of -omics Data (2017), OICR, Toronto, Canada, 26-28 June</span></p>\r\n<p><span>Neo4j Life &amp; Health Sciences Day, Berlin, Germany, 21 June</span></p>\r\n<p><span>Bioinformatics for Cancer Genomics (2017), OICR, Toronto, Canada, 29 May – 2 June</span></p>\r\n<p><span>3rd BiVi Annual Meeting, Edinburgh Napier University, UK, 20-21 April</span></p>\r\n<p><span>Biocuration 2017, Stanford University, US, 26-29 March</span></p>\r\n<p><span>High-Throughput Biology: From Sequence to Networks, CSHL, Cold Spring Harbor, US, 20-26 March</span></p>\r\n<p><span>EBI Training Workshop, Hinxton, UK, 9 March</span></p>\r\n<p><span>Interactions and Pathways – Reactome Workshop, Cambridge University, UK, 2 February</span></p>\r\n<p>{tab title=\"2016\"}</p>\r\n<p><span>Proteomics bioinformatics, Hinxton, UK, 5 December</span></p>\r\n<p><span>Data sharing Best Practice workshop, EMBL-ABR, Australia, 24-28 October</span></p>\r\n<p><span>ECCB 2016, Amsterdam, Netherlands, 5-6 September</span></p>\r\n<p><span>BioNetVisA Workshop, Amsterdam, Netherlands, 4 September</span></p>\r\n<p><span>Integrating Interactions and Pathways, Hinxton, UK, 22 July</span></p>\r\n<p><span>PI seminar series, National Center for Protein Sciences Beijing, China, 21 July</span></p>\r\n<p><span>Reactome webinar (live), Hinxton, UK, 22 June</span></p>\r\n<p><span>SAC seminar series: Harvesting and analysing proteomics big data, Hinxton, UK, 15 June</span></p>\r\n<p><span>Pathway and Network Analysis of -omics Data, OICR, Toronto, 13-15 June</span></p>\r\n<p><span>Bioinformatics for Cancer Genomics, OICR, Toronto, 30 May – 2 June</span></p>\r\n<p><span>CNHUPO, Xiamen University, China, 23 May</span></p>\r\n<p><span>Reactome, IMEx, ProteomeXchange, Tsinghua University, China, 17 May</span></p>\r\n<p><span>GLBIO/CCBC 2016, University of Toronto, Canada, 16-19 May</span></p>\r\n<p><span>Reactome web services and widgets for third-party integration, UV, Spain, 12 May</span></p>\r\n<p><span>Networks and Pathways 2016, Hinxton, 12 May</span></p>\r\n<p><span>Data integration, analysis and visualisation. Experiences from a biological pathway database, UDC, Spain, 25 April</span></p>\r\n<p><span>251st American Chemical Society National Meeting Exposition, San Diego, 13-16 March</span></p>\r\n<p><span>SME Industry Forum, Hinxton, UK, 7 March</span></p>\r\n<p><span>Bioinformatics Resources for Protein Biology, Hinxton, UK, 2 February</span></p>\r\n<p><span>Pathways Overview Widget, Hinxton, 30 January</span></p>\r\n<p><span>Diagram Widget Training Session, Hinxton, 30 January</span></p>\r\n<p>{tab title=\"2015\"}</p>\r\n<p><span>WGC Advanced Course: Proteomics Bioinformatics, Hinxton, UK, 11 December</span></p>\r\n<p><span>MiMS / NDPIA Bioinformatics Workshop – Umea, Hinxton, UK, 10-14 November</span></p>\r\n<p><span>Networks &amp; Pathways, Hinxton, UK, 5 November</span></p>\r\n<p><span>Interactions &amp; Pathways – Reactome, Cambridge, UK, 3 November</span></p>\r\n<p><span>Pathway and Network Analysis Workshop, Liverpool, UK, 1 November</span></p>\r\n<p><span>EBI Open Day, Hinxton, UK, 29 October</span></p>\r\n<p><span>EBI VI Argentinian Conference of Bioinformatics and Computational Biology (CAB2C), Bahía Blanca, Argentina, 13 October</span></p>\r\n<p><span>EBI Bioinformatics Roadshow, Campinas, Brazil, 5 August</span></p>\r\n<p><span>EBI Bioinformatics Roadshow, Sao Paulo, Brazil, 31 July</span></p>\r\n<p><span>Reactome Webinar, Toronto, Canada, July 24</span></p>\r\n<p><span>Reactome SOT Webinar, New York, USA, 15 June</span></p>\r\n<p><span>Pathway and Network Analysis of –omics Data, Toronto, Canada, 1-3 June</span></p>\r\n<p><span>Royal Vet College, London, UK, 28 May</span></p>\r\n<p><span>Bioinformatics for Cancer Genomics, Toronto, Canada, 25-29 May</span></p>\r\n<p><span>Reactome Exploring and Analysing Biological Pathways, Hinxton, UK, 12 May</span></p>\r\n<p><span>Reactome: Networks and Pathways, Toronto, Canada, 6 May</span></p>\r\n<p><span>Reactome Webinar, Hinxton, UK, 6 May</span></p>\r\n<p><span>High-Throughput Biology: From Sequence to Networks, Cold Spring Harbor Laboratory and the New York Genome Center, New York, USA, 27 April – 3 May</span></p>\r\n<p><span>EBI Industry Workshop, Boston, USA, April 1-2, 2015</span></p>\r\n<p><span>Reactome Webinar, Toronto, Canada, March 26, 2015</span></p>\r\n<p><span>Introduction to EBI, Cambridge, UK, 16 February</span></p>\r\n<p>{tab title=\"2014\"}</p>\r\n<p><span>ACSB 2014, Philadelphia, USA, 6-10 December</span></p>\r\n<p><span>Wellcome Trust Bioinformatics Course, Hinxton, UK, 14 November</span></p>\r\n<p><span>Cambridge University Computer Training, University of Cambridge, UK, 22 October</span></p>\r\n<p><span>EBI Open Day, Hinxton, UK, 16 October</span></p>\r\n<p><span>Argentinian Bioinformatics Workshop, University of San Martin, Buenos Aires, Argentina, 1 October</span></p>\r\n<p><span>ECCB 2014: The 13th European Conference on Computational Biology, Strasbourg, France, 7-10 September</span></p>\r\n<p><span>BioNetVisA 2014, Strasbourg, France, 6 September</span></p>\r\n<p><span>Proteomics Data, Functional Analysis and Dissemination, Strasbourg, France, 6 September</span></p>\r\n<p><span>Aveiro Roadshow, University of Aveiro, Portugal, 23 July</span></p>\r\n<p><span>ProteomeXchange/Reactome, NIDDK, USA, 19 June</span></p>\r\n<p><span>Functional Genomics and Systems Biology, WTSI, Hinxton, UK, 12 June</span></p>\r\n<p><span>Networks &amp; Pathways, Hinxton, UK, 12 June</span></p>\r\n<p><span>ProteomeXchange/Reactome, NHLBI, USA, 12 June</span></p>\r\n<p><span>IOCB AS CR, Prague, Czech Republic, 4 June</span></p>\r\n<p><span>Networks and Pathways Course, University of Lisbon, Portugal, 23 May</span></p>\r\n<p><span>ORCID Outreach Meeting, University of Illinois, Chicago, USA, 21-22 May</span></p>\r\n<p><span>PAG Asia, Singapore, Republic of Singapore, 21 May</span></p>\r\n<p><span>Leiden Students Day, Hinxton, UK, 28 April</span></p>\r\n<p><span>diXa: Functional Interpretation of Toxicogenomic Data, Hinxton, UK, 10 April</span></p>\r\n<p><span>Biocuration 2014, Toronto University, Canada, 9 April</span></p>\r\n<p><span>SME forum, Hinxton, UK, 6 March</span></p>\r\n<p><span>Reactome Course, Cambridge University Dept. of Genetics, UK, 24 January</span></p>\r\n<p><span>Plant &amp; Animal Genome Conference, San Diego, USA, 12 January</span></p>\r\n<p>{tab title=\"Archive\"}</p>\r\n<h4>2013</h4>\r\n<p><span>Canadian Cancer Research Conference, CCRC, 3-6 November, Toronto, Canada</span></p>\r\n<p><span>GOC Meeting, Bar Harbor, Maine, USA, October 4-6, 2013</span></p>\r\n<p><span>ISSX 10th International Meeting, ISSX, Toronto, Canada, 29 October-3 November</span></p>\r\n<p><span>EMBO Meeting, EMBO, Amsterdam, The Netherlands, 21-23 October</span></p>\r\n<p><span>Bioinformatics for Immunologists, University of Cambridge, Cambridge, UK, 5 September</span></p>\r\n<p><span>SciKnowMine Release Workshop – Bridging BioNLP and Biocuration, USC Viterbi ISI, Los Angeles, USA, 19 August</span></p>\r\n<p><span>ISMB/ISMB – The Art and Science Exhibit, ISMB, Berlin, Germany, 21-23 July</span></p>\r\n<p><span>NetBioSIG, Berlin, Germany, 19 July</span></p>\r\n<p><span>Networks &amp; Pathways, EBI, Cambridge, UK, 11 July</span></p>\r\n<p><span>Gordon Conference Elastin &amp; Elastic Fibers, University of New England, Biddeford, USA, 22 July</span></p>\r\n<p><span>Summer School of Bioinformatics, EBI, Cambridge, UK, 12 June</span></p>\r\n<p><span>UCL, London, UCL, London, UK, 30 May</span></p>\r\n<p><span>University of Rome, Tor Vergata (Rome 2), Rome, Italy, 7 May</span></p>\r\n<p><span>University of Singapore, Singapore, 20 March</span></p>\r\n<p><span>CAGE-KID Cancer Genomics Workshop, CAGE-KID/EBI, 22 March, UK</span></p>\r\n<p><span>SBGN Workshop, University of Edinburgh, Edinburgh, UK, 29 April-2 May</span></p>\r\n<p><span>GenomeSpace Collaboration, BROAD Institute, Cambridge, USA, 26 March</span></p>\r\n<p><span>CAGE-KID Cancer Genomics Workshop, EBI, Cambridge, UK, 22 March</span></p>\r\n<p><span>EBI Open day, EBI, Cambridge, UK, 14 March</span></p>\r\n<p><span>International Plant &amp; Animal Genome XXI, San Diego, USA, 13 January</span></p>\r\n<p><span>Nimbios Workshop, University of Tennesse, Tennesse, USA, 8-10 January</span></p>\r\n<h4>2012</h4>\r\n<p><span>EBI Open Day, EBI, Cambridge, UK, 20 December</span></p>\r\n<p><span>HPH 2012, Center for Genetic Engineering and Biotechnology, Havana, Cuba, 9 December</span></p>\r\n<p><span>TCAG Analysis Group, TCAG, Washington, DC, USA, 27 November</span></p>\r\n<p><span>Toronto Bioinformatics User Group, University of Toronto, Toronto, Canada, 31 October</span></p>\r\n<p><span>Cambridge University, Cambridge University, Cambridge, UK, 24 October</span></p>\r\n<p><span>Randall Lad, KCL, Randall Lad, King’s College London, London, UK, 23 October</span></p>\r\n<p><span>Google Summer of Code Open Software Meeting, Google, Mountain View, USA, 21 October</span></p>\r\n<p><span>The Jackson Laboratory, Maine, The Jackson Laboratory, Maine, Bar Harbor, USA, 19 October</span></p>\r\n<p><span>Norwestern University Chicago, Norwestern University, Chicago, USA, 17 October</span></p>\r\n<p><span>Workshop on the Concept and Tools for Pathways of Toxicity (PoT), John Hopkins University, Baltimore, USA, 10 October</span></p>\r\n<p><span>Programmatic Access to EBI Proteomics Resources, EBI, Hinxton, UK, 4 October</span></p>\r\n<p><span>University of Leicester/B/BASH, University of Leicester, Leicester, UK, 12 September</span></p>\r\n<p><span>Animal Biotechnology Workshop, EBI, Hinxton, UK, 13 September</span></p>\r\n<p><span>COMBINE 2012, University of Toronto, Toronto, Canada, 18 August</span></p>\r\n<p><span>University of Toronto, Toronto, Canada, 25 July</span></p>\r\n<p><span>Division of Translational Research, National Cancer Center Hospital East, Kashiwanoha, Tokyo University, Kashiwanoha, Japan, 13 July</span></p>\r\n<p><span>Department of Medical Genome Sciences, Kashiwanoha campus, Tokyo University, Kashiwanoha, Japan, 12 July</span></p>\r\n<p><span>Human Genome Center, Institute of Medical Science, Shirokanedai campus, Tokyo University, Tokyo, Japan, 11 July</span></p>\r\n<p><span>Current and Future in Pathway Research International Workshop, Korea Institute of Science and Technology (KISTI), Daejeon, South Korea, 6 July</span></p>\r\n<p><span>Erasmus MC, Rotterdam, Erasmus MC, Rotterdam, The Netherlands, 13 June</span></p>\r\n<p><span>OICR, Toronto, Canada, 12 June</span></p>\r\n<p><span>Ontologies for Immunity and Infectious Disease, University at Buffalo, Buffalo, USA, 11-13 June</span></p>\r\n<p><span>HARMONY Hackathon, Department of Bioinformatics – BiGCaT, Maastricht, The Netherlands, 21 May</span></p>\r\n<p><span>Networks and Pathways Bioinformatics for Biologists, EBI, Hinxton, UK, 18 May</span></p>\r\n<p><span>6th Biocuration Conference, University of Georgetown, Washington, USA, 2-4 April</span></p>\r\n<p><span>Perspectives in Clinical Proteomics Training Workshop, EBI, Hinxton, UK, 17 March</span></p>\r\n<p><span>Fobiotech Meeting, Fobiotech, Torino, Italy, 5 March</span></p>\r\n<p><span>GO Consortium Annotation Meeting, Stanford University, Palo Alto, USA, 26-28 February</span></p>\r\n<p><span>Reactome Outreach, University of Toronto, Toronto, Canada, 17 February</span></p>\r\n<h4>2011</h4>\r\n<p><span>Canadian Cancer Research Conference, Toronto, Canada, 27-30 November</span></p>\r\n<p><span>X-Meeting 2011, Florianopolis, Brazil, 12-15 October</span></p>\r\n<p><span>PSIMEx Workshop: Interactions and Pathways, Lausanne, Switzerland, 6-7 October</span></p>\r\n<p><span>AlzForum / PRO workshop, Buffalo, USA, 4-5 October</span></p>\r\n<p><span>COMBINE 2011, Heidelberg, Germany, 3-7 September</span></p>\r\n<p><span>OICR Reactome Workshop, Ottawa Hospital Research Institute, University of Ottawa, Ottawa, Canada, 4 August</span></p>\r\n<p><span>OICR Reactome Workshop, University of Kingston, Kingston, Canada, 3 August.</span></p>\r\n<p><span>Network Biology SIG, ISMB/ECCB 2011, Vienna, Austria, 15 July.</span></p>\r\n<p><span>International Institute of Molecular and Cell Biology, Warsaw, Poland, 15 July</span></p>\r\n<p><span>9th Poznan Summer School of Bioinformatics, Poznan, Poland, 11-15 July</span></p>\r\n<p><span>OICR-OGI Pathway Workshop, Hospital for Sick Children, Toronto, Canada, 2-3 June</span></p>\r\n<p><span>EBI Roadshow, University of Düsseldorf &amp; Center for Neuronal Regenaration, Düsseldorf, Germany, 15-17 March.</span></p>\r\n<p><span>EBI Roadshow, University of the Western Cape, Cape Town, South Africa, 7-8 March</span></p>\r\n<p><span>EBI Roadshow, International Livestock Research Institute, Nairobi, Kenya, 2-3 March</span></p>\r\n<p><span>EBI Roadshow, Centro de Edafología y Biología Aplicada del Segura (CEBAS)-CSIC, Murcia, Spain, 1-3 February</span></p>\r\n<p><span>EBI Workshop, City of Hope, Duarte, California, United States, 20-21 January</span></p>\r\n<h4>2010</h4>\r\n<p><span>Joint EBI-Wellcome Trust Proteomics Workshop, Hinxton, UK, 13-17 December</span></p>\r\n<p><span>Introduction to Bioinformatics at the EBI, Dept. of Genetics Cambridge University, Cambridge, UK, 30 November</span></p>\r\n<p><span>EBI Roadshow, Murdoch University, Perth, Australia, 15-18 November</span></p>\r\n<p><span>EBI Roadshow, MRC National Institute for Medical Research, London, UK, 15-19 November</span></p>\r\n<p><span>EBI Roadshow, Vilnius Institute of Biotechnology, Vilnius, Liuania, 4-5 November</span></p>\r\n<p><span>Predoc Training Course, EMBL Heidelberg, Heidelberg, Germany, 2 November</span></p>\r\n<p><span>EMBL-EBI Open Day, Hinxton, UK, 2 November</span></p>\r\n<p><span>EBI Roadshow, Dokuz Eylül University, Izmir, Turkey, 12-14 October</span></p>\r\n<p><span>COMBINE 2010, University of Edinburgh, Edinburgh, UK, 6-9 October</span></p>\r\n<p><span>EBI Roadshow, Institute of Molecular Biology, Bratislava, Slovakia, 6-7 October</span></p>\r\n<p><span>EBI Roadshow, University of Rostock, Rostock, Germany, 24-25 September</span></p>\r\n<p><span>Protein databases and tools from the EBI, Dept. of Genetics Cambridge University, Cambridge, UK, 20-21 September</span></p>\r\n<p><span>OICR Scientific Information Resource Interoperability Symposium (OSIRIS), Toronto, Canada, 15 October</span></p>\r\n<p><span>7th International Conference Data Integration in e Life Sciences conference, Goenburg, Sweden, 25-27 August</span></p>\r\n<p><span>12th UK Meeting on Platelets/12th Erfurt Conference on Platelets, Nottingham, UK, 15-16 July</span></p>\r\n<p><span>BSPR/EBI Educational Workshop – Quantitative Proteomics, Hinxton, UK, 15-16 July</span></p>\r\n<p><span>18 Annual International Conference on Intelligent Systems for Molecular Biology (ISMB 2010), Boston, USA, July 11 – 13</span></p>\r\n<p><span>EBI Roadshow, Teleon Institute of Genetics and Medicine, Naples, Italy, 1-2 July</span></p>\r\n<p><span>35 FEBS Congress, Gotenborg Convention Centre, Goenburg, Sweden, 26 June – 1 July</span></p>\r\n<p><span>Joint EBI-Wellcome Trust Summer School in Bioinformatics, Hinxton, UK, 14 – 18 June</span></p>\r\n<p><span>EBI Roadshow, Charles University, Prague, Czech Republic, 2 June</span></p>\r\n<p><span>5th Canadian Conference on Ovarian Cancer Research, Toronto, Canada, 15 – 18 May</span></p>\r\n<p><span>ELLS LearningLab, EMBL Heidelberg, Heidelberg, Germany, 10-12 May</span></p>\r\n<p><span>97th AAI Annual Meeting Immunology 2010, Baltimore, USA, 7 – 11 May</span></p>\r\n<p><span>EBI Roadshow, North East USA, Chicago, USA, 26-27 April</span></p>\r\n<p><span>LipidomicNet Bioinformatics Workshop, Cambridge, UK, 26-27 April</span></p>\r\n<p><span>3nd Annual Protein Ontology Meeting, Newark, USA, April 26 – 28</span></p>\r\n<p><span>EBI Roadshow, University of Tor Vergata, Rome, Italy, April 15-16</span></p>\r\n<p><span>EBI Roadshow, Institut für Molekulare Enzymtechnologie (IMET), Aachen, Germany, 13-14 April</span></p>\r\n<p><span>EBI Roadshow, Università degli Studi di Firenze, Florence, Italy, 7-9 April</span></p>\r\n<p><span>Gene Ontology Consortium Meeting, Palo Alto, USA, 30 – 31 March</span></p>\r\n<p><span>Plant Bioinformatics, Hinxton, UK, 29-31 March</span></p>\r\n<p><span>International Symposium for Integrative Bioinformatics, Cambridge, UK, 22-24 March</span></p>\r\n<p><span>Perspectives in Clinical Proteomics Training Workshop, Hinxton, UK, 18-19 March</span></p>\r\n<p><span>Keystone Symposia: Biomolecular Interaction Networks: Function and Disease, Quebec City, Canada,  7 – 12 March</span></p>\r\n<p><span>49th Annual Meeting of the Society of Toxicology, Salt Lake City, USA, 7 – 11 March</span></p>\r\n<p><span>EBI Open Day, Hinxton, UK, 4 March</span></p>\r\n<p><span>Therapeutic Applications of Computational Biology and Chemistry: TACBAC 2010, Hinxton, UK, 1-3 March</span></p>\r\n<p><span>OICR Annual Scientific Meeting 2010, Nottawasaga, Canada, 28 February – 2 March</span></p>\r\n<p><span>EBI Roadshow, Jozef Stefan Institute and University of Ljubljana, Llubljana, Slovenia, 16-18 February</span></p>\r\n<p><span>EBI Roadshow, Gulbenkian Institute, Lisbon, Portugal, 9-11 February</span></p>\r\n<p><span>Bloodomics 2 Workshop &amp; Meeting Madingley Hall, Cambridge, UK, 5 February</span></p>\r\n<p><span>ENFIN Fif Annual General Meeting, Berlin, Germany, 3-5 February</span></p>\r\n<p><span>EBI Roadshow, University of Aberdeen, Aberdeen, UK, 3-4 February</span></p>\r\n<p><span>CIP-MCMM Bioinformatics Workshop, Toronto, Canada, 21 January</span></p>\r\n<p><span>Plant &amp; Animal Genomes XVIII Conference, San Diego, USA, 9 – 13 January</span></p>\r\n<h4>2009</h4>\r\n<p><span>OGI-OICR Reactome Webinar, Toronto, Canada, 17 December</span></p>\r\n<p><span>NIH 7th Symposium on the Functional Genomics of Critical Illness &amp; Injury, Bethesda, USA, 7 December</span></p>\r\n<p><span>Functional Genomics &amp; Systems Biology Workshop, Hinxton, UK, 30 November – 2 December</span></p>\r\n<p><span>Rb International Meeting, Toronto, Canada, 19 – 21 November</span></p>\r\n<p><span>BioPAX Hackathon and Workshop, New York, USA, 11 – 13 November</span></p>\r\n<p><span>Bloodomics 2 Student Workshop &amp; Meeting, Cambridge, UK, 11 – 12 November</span></p>\r\n<p><span>Joint EBI-Wellcome Trust Proteomics Workshop, Hinxton, UK, 9 – 13 November</span></p>\r\n<p><span>CSHL Genome Informatics Meeting, Cold Spring Harbor, USA, 27 – 30 October</span></p>\r\n<p><span>Oeiras – PaProt II Workshop, Lisbon, Portugal, 13 – 15 October</span></p>\r\n<p><span>Enfin EMBRACE Workshop, Helsinki, Finland, 4 – 6 October</span></p>\r\n<p><span>Gene Ontology Consortium Meeting, Cambridge UK, 23 – 25 September</span></p>\r\n<p><span>5th Metabolomics Meeting, Edmonton, Canada, 30 August – 2 September</span></p>\r\n<p><span>Cancer Proteomics 2009, Dublin, Ireland, 8 – 11 June</span></p>\r\n<p><span>TCGA Ovarian Analysis Workshop, Berkeley, USA, 21 – 22 May</span></p>\r\n<p><span>3rd International Biocuration Conference, Berlin, Germany, 16 – 19 April</span></p>\r\n<p><span>Gene Ontology Consortium Meeting, Eugene, USA, 30 – 31 March</span></p>\r\n<p><span>Semantic Enrichment of the Scientific Literature 2009 (SESL 2009), 30 – 31 Hinxton, UK, March</span></p>\r\n<p><span>OICR Annual Scientific Meeting, Nottawasaga, Canada, 22 – 24 February</span></p>\r\n<p>{/tabs}</p>\r\n<p> </p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 15:45:48','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(24,558,'Publications','publications','<div class=\"article-publications\">Since 2003, Reactome papers have been cited in hundreds of textbooks and thousands of peer-reviewed articles and reviews, and the citation rate is increasing annually. Several papers cite the use of Reactome data for either integrative data analysis, algorithm development, or the inclusion of Reactome pathway annotations into a secondary bioinformatics database. A full listing of all the papers and books citing Reactome is accessible online at <a href=\"https://scholar.google.ca/citations?user=gtB1MLcAAAAJ\">Google Scholar</a>.<br /><br />\r\n<p><strong>2024</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Gretner A, Ragueneau A, Gong C, Prinz A, Gansberger S, Oyarzun I, Hermjakob H, Griss J, ReactomeGSA: new features to simplify public data reuse. Bioinformatics, Volume 40, Issue 6, June 2024; 10.1093/bioinformatics/btae338. <a href=\"https://doi.org/10.1093/bioinformatics/btae338\">Link</a></li>\r\n<li>Arora C, Matic M, DiChiaro P, Rosa NO, Carli F, Clubb L, Fard LAN, Kargas G, Diaferia G, Vukotic R, Licata L, Wu G, Natoli G, Gutkind JS, Raimondi F. The landscape of cancer rewired GPCR signaling axes. Cell Genomics, 2024. DOI: 10.1016/j.xgen.2024.100557. <a href=\"https://doi.org/10.1016/j.xgen.2024.100557\">Link</a></li>\r\n<li>Orlic-Milacic M, Rothfels K, Matthews L, Wright A, Jassal B, Shamovsky V, Trinh Q, Gillespie M, Sevilla C, Tiwari K, Ragueneau E, Gong C, Stephan R, May B, Haw R, Weiser J, Beavers D, Conley P, Hermjakob H, Stein LD, D\'Eustachio P, Wu G. Pathway-based, reaction-specific annotation of disease variants for elucidation of molecular phenotypes. Database, 2024. doi:10.1093/database/baae03. <a href=\"https://doi.org/10.1093/database/baae031\">Link</a></li>\r\n<li>R. Stephan, Interleaved snowballing: Reducing the workload of literature curators. Arxiv, 2024 DOI: 10.48550/arXiv.2402.08339. <a href=\"https://arxiv.org/pdf/2402.08339\">Link</a>.</li>\r\n<li>Niarakis A, Ostaszewski M, Mazein A, Kuperstein I, Kutmon M, Gillespie ME, Funahashi A, Acencio ML, Hemedan A, Aichem M, Klein K, Czauderna T, Burtscher F, Yamada TG, Hiki Y, Hiroi NF, Hu F, Pham N, Ehrhart F, Willighagen EL, Valdeolivas A, Dugourd A, Messina F, Esteban-Medina M, Peña-Chilet M, Rian K, Soliman S, Aghamiri SS, Puniya BL, Naldi A, Helikar T, Singh V, Fernández MF, Bermudez V, Tsirvouli E, Montagud A, Noël V, Ponce-de-Leon M, Maier D, Bauch A, Gyori BM, Bachman JA, Luna A, Piñero J, Furlong LI, Balaur I, Rougny A, Jarosz Y, Overall RW, Phair R, Perfetto L, Matthews L, Rex DAB, Orlic-Milacic M, Gomez LCM, De Meulder B, Ravel JM, Jassal B, Satagopam V, Wu G, Golebiewski M, Gawron P, Calzone L, Beckmann JS, Evelo CT, D\'Eustachio P, Schreiber F, Saez-Rodriguez J, Dopazo J, Kuiper M, Valencia A, Wolkenhauer O, Kitano H, Barillot E, Auffray C, Balling R, Schneider R; COVID-19 Disease Map Community. Drug-target identification in COVID-19 disease mechanisms using computational systems biology approaches. Front Immunol. 2024 Feb 13;14:1282859. doi: 10.3389/fimmu.2023.1282859. <a href=\"https://doi.org/10.3389/fimmu.2023.1282859\">Link</a></li>\r\n<li>Milacic M, Beavers D, Conley P, Gong C, Gillespie M, Griss J, Haw R, Jassal B, Matthews L, May B, Petryszak R, Ragueneau E, Rothfels K, Sevilla C, Shamovsky V, Stephan R, Tiwari K, Varusai T, Weiser J, Wright A, Wu G, Stein L, Hermjakob H, D’Eustachio P. The Reactome Pathway Knowledgebase 2024. Nucleic Acids Research, 2024. doi: 10.1093/nar/gkad1025. <a href=\"https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkad1025/7369850?utm_source=advanceaccess&amp;utm_campaign=nar&amp;utm_medium=email\">Link</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2023</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Tiwari K, Matthews L, May B, Shamovsky V, Orlic-Milacic M, Rothfels K, Ragueneau E, Gong C, Stephan R, Li N, Wu G, Stein L, D\'Eustachio P, Hermjakob H. ChatGPT usage in the Reactome curation process. bioRxiv [Preprint]. 2023 Nov 8:2023.11.08.566195. doi: 10.1101/2023.11.08.566195. <a href=\"https://doi.org/10.1101/2023.11.08.566195\">Link</a></li>\r\n<li>Beavers D, Brunson T, Sanati N, Matthews L, Haw R, Shorser S, Sevilla C, Viteri G, Conley P, Rothfels K, Hermjakob H, Stein L, D\'Eustachio P, Wu G. Illuminate the Functions of Dark Proteins Using the Reactome-IDG Web Portal. Curr Protoc. 2023 Jul;3(7):e845. doi: 10.1002/cpz1.845. <a href=\"https://pubmed.ncbi.nlm.nih.gov/37467006/\">PubMed</a></li>\r\n<li>Brunson T, Sanati N, Matthews L, Haw R, Beavers D, Shorser S, Sevilla C, Viteri G, Conley P, Rothfels K, Hermjakob H, Stein L, D\'Eustachio P, Wu G. Illuminating Dark Proteins using Reactome Pathways. bioRxiv [Preprint]. 2023 Jun 5:2023.06.05.543335. doi: 10.1101/2023.06.05.543335. <a href=\"https://pubmed.ncbi.nlm.nih.gov/37333417/\">PubMed</a></li>\r\n<li>Rothfels K, Milacic M, Matthews L, Haw R, Sevilla C, Gillespie M, Stephan R, Gong C, Ragueneau E, May B, Shamovsky V, Wright A, Weiser J, Beavers D, Conley P, Tiwari K, Jassal B, Griss J, Senff-Ribeiro A, Brunson T, Petryszak R, Hermjakob H, D\'Eustachio P, Wu G, Stein L. Using the Reactome Database. Curr Protoc. 2023 Apr;3(4):e722. doi: 10.1002/cpz1.722. <a href=\"https://pubmed.ncbi.nlm.nih.gov/37053306/\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2022</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Wright AJ, Orlic-Milacic M, Rothfels K, Weiser J, Trinh QM, Jassal B, Haw RA, Stein LD. Evaluating the predictive accuracy of curated biological pathways in a public knowledgebase. Database (Oxford). 2022 Mar 28;2022:baac009. doi: 10.1093/database/baac009. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/35348650\">PubMed</a></li>\r\n<li>Gillespie M, Jassal B, Stephan R, Milacic M, Rothfels K, Senff-Ribeiro A, Griss J, Sevilla C, Matthews L, Gong C, Deng C, Varusai T, Ragueneau E, Haider Y, May B, Shamovsky V, Weiser J, Brunson T, Sanati N, Beckman L, Shao X, Fabregat A, Sidiropoulos K, Murillo J, Viteri G, Cook J, Shorser S, Bader G, Demir E, Sander C, Haw R, Wu G, Stein L, Hermjakob H, D\'Eustachio P. The reactome pathway knowledgebase 2022. Nucleic Acids Res. 2022 Jan 7;50(D1):D687-D692. doi: 10.1093/nar/gkab1028. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/34788843\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2021</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Ostaszewski M, Niarakis A, Mazein A, Kuperstein I, Phair R, Orta-Resendiz A, Singh V, Aghamiri SS, Acencio ML, Glaab E, Ruepp A, Fobo G, Montrone C, Brauner B, Frishman G, Monraz Gómez LC, Somers J, Hoch M, Kumar Gupta S, Scheel J, Borlinghaus H, Czauderna T, Schreiber F, Montagud A, Ponce de Leon M, Funahashi A, Hiki Y, Hiroi N, Yamada TG, Dräger A, Renz A, Naveez M, Bocskei Z, Messina F, Börnigen D, Fergusson L, Conti M, Rameil M, Nakonecnij V, Vanhoefer J, Schmiester L, Wang M, Ackerman EE, Shoemaker JE, Zucker J, Oxford K, Teuton J, Kocakaya E, Summak GY, Hanspers K, Kutmon M, Coort S, Eijssen L, Ehrhart F, Rex DAB, Slenter D, Martens M, Pham N, Haw R, Jassal B, Matthews L, Orlic-Milacic M, Senff Ribeiro A, Rothfels K, Shamovsky V, Stephan R, Sevilla C, Varusai T, Ravel JM, Fraser R, Ortseifen V, Marchesi S, Gawron P, Smula E, Heirendt L, Satagopam V, Wu G, Riutta A, Golebiewski M, Owen S, Goble C, Hu X, Overall RW, Maier D, Bauch A, Gyori BM, Bachman JA, Vega C, Grouès V, Vazquez M, Porras P, Licata L, Iannuccelli M, Sacco F, Nesterova A, Yuryev A, de Waard A, Turei D, Luna A, Babur O, Soliman S, Valdeolivas A, Esteban-Medina M, Peña-Chilet M, Rian K, Helikar T, Puniya BL, Modos D, Treveil A, Olbei M, De Meulder B, Ballereau S, Dugourd A, Naldi A, Noël V, Calzone L, Sander C, Demir E, Korcsmaros T, Freeman TC, Augé F, Beckmann JS, Hasenauer J, Wolkenhauer O, Wilighagen EL, Pico AR, Evelo CT, Gillespie ME, Stein LD, Hermjakob H, D\'Eustachio P, Saez-Rodriguez J, Dopazo J, Valencia A, Kitano H, Barillot E, Auffray C, Balling R, Schneider R; COVID-19 Disease Map Community. COVID19 Disease Map, a computational knowledge repository of virus-host interaction mechanisms. Mol Syst Biol. 2021 Oct;17(10):e10387. doi: 10.15252/msb.202110387. Erratum in: Mol Syst Biol. 2021 Dec;17(12):e10851. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/34664389\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2020</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Griss J, Viteri G, Sidiropoulos K, Nguyen V, Fabregat A, Hermjakob H. ReactomeGSA - Efficient Multi-Omics Comparative Pathway Analysis. Mol Cell Proteomics. 2020 Sep 9:mcp.TIR120.002155. doi: 10.1074/mcp.TIR120.002155. <a href=\"https://pubmed.ncbi.nlm.nih.gov/32907876\">PubMed</a>.</li>\r\n<li>Varusai TM, Jupe S, Sevilla C, et al. Using Reactome to build an autophagy mechanism knowledgebase. <i>Autophagy</i>. 2020;1‐12. doi:10.1080/15548627.2020.1761659. <a href=\"https://pubmed.ncbi.nlm.nih.gov/32486891\">PubMed</a>.</li>\r\n<li>Waagmeester A, Stupp G, Burgstaller-Muehlbacher S, Good BM, Griffith M, Griffith OL, Hanspers K, Hermjakob H, Hudson TS, Hybiske K, Keating SM, Manske M, Mayers M, Mietchen D, Mitraka E, Pico AR, Putman T, Riutta A, Queralt-Rosinach N, Schriml LM, Shafee T, Slenter D, Stephan R, Thornton K, Tsueng G, Tu R, Ul-Hasan S, Willighagen E, Wu C, Su AI. Wikidata as a knowledge graph for the life sciences. Elife. 2020 Mar 17;9. pii: e52614. doi: 10.7554/eLife.52614. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/32180547\">PubMed</a>.</li>\r\n<li>Jassal B, Matthews L, Viteri G, Gong C, Lorente P, Fabregat A, Sidiropoulos K, Cook J, Gillespie M, Haw R, Loney F, May B, Milacic M, Rothfels K, Sevilla C, Shamovsky V, Shorser S, Varusai T, Weiser J, Wu G, Stein L, Hermjakob H, D\'Eustachio P. The reactome pathway knowledgebase. <em>Nucleic Acids Res<em>. </em></em>pii: gkz1031. doi:10.1093/nar/gkz1031.<em><em> </em></em><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/31691815\">PubMed.</a></li>\r\n<li>Naithani S, Gupta P, Preece J, D\'Eustachio P, Elser J, Kiff J, Dikeman D, Olson AJ, Wei S, Tello-Ruiz M, Cook J, Fabregat A, Cheng T, Bolton E, Muñoz-Pomer Fuentes A, Mohammed S, Papa-theodorou I, Stein L, Ware D, Jaiswal P. 2020. Plant Reactome: A knowledgebase and resource for comparative pathway analysis. Nucleic Acids Res. gkz996, doi.org/10.1093/nar/gkz996. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/31680153\">PubMed</a>.</li>\r\n<li>Haw R, Loney F, Ong E, He Y, Wu G. Perform Pathway Enrichment Analysis Using ReactomeFIViz. <i>Methods Mol Biol</i>. 2020;2074:165–179. doi:10.1007/978-1-4939-9873-9_13. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/31583638\">PubMed</a>.</li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2019</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Blucher AS, McWeeney SK, Stein L, Wu G. Visualization of drug target interactions in the contexts of pathways and networks with ReactomeFIViz. <em>F1000Res</em>. 2019 Jun 20. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/31372215\">PubMed.</a></li>\r\n<li>Viteri G, Matthews L, Varusai T, Gillespie M, Milacic M, Cook J, Weiser J, Shorser S, Sidiropoulos K, Fabregat A, Haw R, Wu G, Stein L, D\'Eustachio P, Hermjakob H. Reactome and ORCID-fine-grained credit attribution for community curation. Database (Oxford). 2019 Jan 1. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/31802127\">PubMed</a>.</li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2018</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Jupe S, Ray K, Roca CD, Varusai T, Shamovsky V, Stein L, D\'Eustachio P, Hermjakob H. Interleukins and their signaling pathways in the Reactome biological pathway database. <em>J Allergy Clin Immunol</em>. Apr 14. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29378288\">PubMed</a>.</li>\r\n<li>Fabregat A, Sidiropoulos K, Viteri G, Marin-Garcia P, Ping P, Stein L, D\'Eustachio P, Hermjakob H. Reactome diagram viewer: data structures and strategies to boost performance.<em> Bioinformatics. </em>Apr 1. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29186351\">PubMed.</a></li>\r\n<li>Fabregat A, Korninger F, Viteri G, Sidiropoulos K, Marin-Garcia P, Ping P, Wu G, Stein L, D\'Eustachio P, Hermjakob H. Reactome graph database: Efficient access to complex pathway data. <em>PLoS Comput Biol.</em>Jan 29. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29377902\">PubMed.</a></li>\r\n<li>Fabregat A, Jupe S, Matthews L, Sidiropoulos K, Gillespie M, Garapati P, Haw R, Jassal B, Korninger F, May B, Milacic M, Duenas Roca C, Rothfels K, Sevilla C, Shamovsky V, Shorser S, Varusai T, Viteri G, Weiser J, Wu G, Stein L, Hermjakob H, D’Eustachio P. The Reactome Pathway Knowledgebase.<em>Nucleic Acids Research.</em> Jan 4. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29145629\">PubMed.</a></li>\r\n<li>Loney F, Wu G. Automation of ReactomeFIViz via CyREST API. F1000Res. May 23. 2018. doi: 10.12688/f1000research.14776.2 <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5998009/\">PubMed.</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2017</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Sidiropoulos K, Viteri G, Sevilla C, Jupe S, Webber M, Orlic-Milacic M, Jassal B, May B, Shamovsky V, Duenas C, Rothfels K, Matthews L, Song H, Stein L, Haw R, D’Eustachio P, Ping P, Hermjakob H, Fabregat A. Reactome enhanced pathway visualization. <em>Oxford Bioinformatics.</em> July 6. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29077811\">PubMed</a>.</li>\r\n<li>Fabregat A, Sidiropoulos K, Viteri G, Forner O, Marin-Garcia P, Arnau V, D’Eustachio P, Stein L, Hermjakob H. Reactome pathway analysis: a high-performance in-memory approach.<em>BMC Bioinformatics.</em>Mar 2. <a href=\"http://ncbi.nlm.nih.gov/pubmed/28249561\">PubMed</a>.</li>\r\n<li>Wu G, Haw R. Functional Interaction Network Construction and Analysis for Disease Discovery.<em>Methods Mol Biol. </em><a href=\"http://ncbi.nlm.nih.gov/pubmed/28150241\">PubMed</a>.</li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2016</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Hill DP, D’Eustachio P, Berardini TZ, Mungall CJ, Renedo N, Blake JA. Modeling biochemical pathways in the gene ontology.<em>Database (Oxford)</em>. 2016 Sep 1. <a href=\"http://ncbi.nlm.nih.gov/pubmed/27589964\">PubMed</a>.</li>\r\n<li>Bohler A, Wu G, Kutmon M, Pradhana LA, Coort SL, Hanspers K, Haw R, Pico AR, Evelo CT. Reactome from a WikiPathways Perspective. PLoS Comput Biol. May. <a href=\"http://ncbi.nlm.nih.gov/pubmed/27203685\">PubMed</a>.</li>\r\n<li>Fabregat A, Sidiropoulos K, Garapati P, Gillespie M, Hausmann K, Haw R, Jassal B, Jupe S, Korninger F, McKay S, Matthews L, May B, Milacic M, Rothfels K, Shamovsky V, Webber M, Weiser J, Williams M, Wu G, Stein L, Hermjakob H, D’Eustachio P. The Reactome pathway Knowledgebase.<em>Nucleic Acids Res</em>. Dec 9. <a href=\"http://ncbi.nlm.nih.gov/pubmed/26656494\">PubMed</a>.</li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2015</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li class=\"title\">McKay SJ, Weiser J. Installing a Local Copy of the Reactome Web Site and Knowledgebase. Curr Protoc<em>Bioinformatics</em>. 50:9.10.1-9.10.10. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/26087747\">PubMed</a>.</li>\r\n<li class=\"title\">Jupe S, Fabregat A, and Hermjakob H.Expression data analysis with reactome.<em><span class=\"jrnl\" title=\"Current protocols in bioinformatics / editoral board, Andreas D. Baxevanis ... [et al.]\">Curr Protoc Bioinformatics</span></em>. Chapter 8 Unit8.20. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/25754994\">PubMed</a>.</li>\r\n<li class=\"title\">Porras P, Duesbury M, Fabregat A, Ueffing M, Orchard S, Gloeckner CJ, and Hermjakob H.A visual review of the interactome of LRRK2: Using deep-curated molecular interactions data to represent biology.<em>Proteomics. </em><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/25648416\">PubMed</a>.</li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2014</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li><span class=\"article-title-and-info in-popup\">Wu G, Dawson E, Duong A, Haw R and Stein L.ReactomeFIViz: the Reactome FI Cytoscape app for pathway and network-based data analysis.3:146</span><em>F1000Research</em>. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/25309732\">PubMed</a>.</li>\r\n<li>Jupe S, Jassal B, Williams M, and Wu G. A controlled vocabulary for pathway entities and events.<em>Database (Oxford)</em><em>Jun. </em><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/24951798\">PubMed</a>.</li>\r\n<li>Backman S, Kollara A, Haw R, Stein L, Brown TJ. Glucocorticoid-Induced Reversal of Interleukin-1β-Stimulated Inflammatory Gene Expression in Human Oviductal Cells. PLoS ONE 9(5). <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/24848801\">PubMed</a>.</li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2013</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Croft D, Fabregat Mundo A, Haw R, Milacic, M, Weiser J, Wu G, Caudy M,Garapati P, Gillespie M, Kamdar MR,Jassal B, Jupe S, Matthews L, May B, Palatnik S, Rothfels K, Shamovsky V, Song H, Williams M, Birney E, Hermjakob H, Stein L and D’Eustachio P.The Reactome pathway knowledgebase.<em>Nucleic Acids Res. </em><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/24243840\">PubMed</a></li>\r\n<li>Monaco MK, Stein J, Naithani S, Wei S, Dharmawardhana P, Kumari S, Amarasinghe V, Youens-Clark K, Thomason J, Preece J, Pasternak S, Olson A, Jiao Y, Lu Z, Bolser D, Kerhornou A, Staines D, Walts B, Wu G, D’Eustachio P, Haw R, Croft D, Kersey PJ, Stein L, Jaiswal P, Ware D. Gramene 2013: comparative plant genomics resources.<em>Nucleic Acids Res</em>.<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/24217918\">PubMed</a> Croft, D. Building models using reactome pathways as templates.<em>Methods Mol Biol</em>. 1021:273-83. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/23715990\">PubMed</a></li>\r\n<li>D’Eustachio, P. Pathway databases: making chemical and biological sense of the genomic data flood.<em>Chem Biol</em>. 20(5):629-35. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/23706629\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2012</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Milacic M, Haw R, Rothfels K, Wu G, Croft D, Hermjakob H, D’Eustachio P and Stein L. Annotating Cancer Variants and Anti-Cancer Therapeutics in Reactome.<em>Cancers.</em>4 (4), 1180-1211. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/24213504\">PubMed.</a></li>\r\n<li>Wu G and Stein L. A network module-based method for identifying cancer prognostic signatures.<em>Genome Biol</em>. 13(12):R112. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/23228031\">PubMed</a></li>\r\n<li>Haw R and Stein L. Using the reactome database.<em>Curr Protoc Bioinformatics</em>. Chapter 8:Unit 8.7. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22700314\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2011</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Gieger C, Radhakrishnan A, Cvejic A, Tang W, Porcu E, Pistis G, Serbanovic-Canic J, Elling U,Goodall AH,……….. Soranzo N. New gene functions in megakaryopoiesis and platelet formation.<em>Nature</em>. 480(7376):201-8. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22139419\">PubMed</a></li>\r\n<li>Ndegwa N, Côté RG, Ovelleiro D, D’Eustachio P, Hermjakob H, Vizcaíno JA, Croft D. Critical amino acid residues in proteins: a BioMart integration of Reactome protein annotations with PRIDE mass spectrometry data and COSMIC somatic mutations.<em>Database (Oxford)</em>. bar047. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22025670\">PubMed</a></li>\r\n<li>Haw RA, Croft D, Yung CK, Ndegwa N, D’Eustachio P, Hermjakob H, Stein LD. The Reactome BioMart.<em>Database (Oxford)</em>. bar047. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22012987\">PubMed</a></li>\r\n<li>Bult CJ, Drabkin HJ, Evsikov A, Natale D, Arighi C, Roberts N, Ruttenberg A, D’Eustachio P, Smith B, Blake JA, Wu C.<em>The representation of protein complexes in the Protein Ontology (PRO)</em>. BMC Bioinformatics. 12:371. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21929785\">PubMed</a></li>\r\n<li>Haw R, Hermjakob H, D’Eustachio P, Stein L. Reactome pathway analysis to enrich biological discovery in proteomics data sets.<em>Proteomics</em>. 11(18):3598-613. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21751369\">PubMed</a></li>\r\n<li>Sawey ET, Chanrion M, Cai C, Wu G, Zhang J, Zender L, Zhao A, Busuttil RW, Yee H, Stein L, French DM, Finn RS, Lowe SW, and Powers S. Identification of a therapeutic strategy targeting amplified FGF19 in liver cancer by oncogenomic screening.<em>Cancer Cell.</em>19(3): 347-58. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21397858\">PubMed</a></li>\r\n<li>Jassal B. Pathway annotation and analysis with Reactome: The solute carrier class of membrane transporters.<em>Human Genomics</em>. 5(4):310-5. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21712192\">PubMed</a></li>\r\n<li>D’Eustachio P. Reactome knowledgebase of human biological pathways and processes.<em>Methods Mol Biol.</em>694:49-61. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21082427\">PubMed</a></li>\r\n<li>Gillespie M, Shamovsky V, D’Eustachio P. Human and chicken TLR pathways: manual curation and computer-based orthology analysis.<em>Mammalian Genome.</em>22(1-2):130-8. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21052677\">PubMed</a></li>\r\n<li>Croft D, O’Kelly G, Wu G, Haw R, Gillespie M, Matthews L, Caudy M, Garapati P, Gopinath G, Jassal B, Jupe S, Kalatskaya I, Mahajan S, May B, Ndegwa N, Schmidt E, Shamovsky V, Yung C, Birney E, Hermjakob H, D’Eustachio P, Stein L. Reactome: a database of reactions, pathways and biological processes.<em>Nucleic Acids Res. (Database issue).</em>D691-7. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21067998\">PubMed</a></li>\r\n<li>Dall’olio GM, Jassal B, Montanucci L, Gagneux P, Bertranpetit J, Laayouni H. The annotation of the Asparagine N-linked Glycosylation pathway in the Reactome Database.<em>Glycobiology</em>. 21(11):1395-400. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/21199820\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2010</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Voight BF, Scott LJ, Steinthorsdottir V, Morris AP, Dina C, Welch RP, Zeggini E, Huth C, Aulchenko YS, Thorleifsson G, McCulloch LJ, Ferreira T, Grallert H, Amin N, Wu G, ……. McCarthy MI. Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis.<em>Nat Genet.</em>42(7): 579-89. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/20581827\">PubMed</a></li>\r\n<li>Jassal B, Jupe S, Caudy M, Birney E, Stein L, Hermjakob H, D’Eustachio P. The systematic annotation of the three main GPCR families in Reactome.<em>Database (Oxford)</em>.baq018. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/20671204\">PubMed</a></li>\r\n<li>Antonov AV, Schmidt EE, Dietmann S, Krestyaninova M, Hermjakob H. R spider: a network-based analysis of gene lists by combining signaling and metabolic pathways from Reactome and KEGG databases.<em>Nucleic Acids Res. (Web Server issue)</em>38:W78-83. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/20519200\">PubMed</a></li>\r\n<li>Wu G, Feng X, Stein L. A human functional protein interaction network and its application to cancer data analysis.<em>Genome Biol.</em>11(5):R53. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/20482850\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2009</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Matthews L, Gopinath G, Gillespie M, Caudy M, Croft D, de Bono B, Garapati P, Hemish J, Hermjakob H, Jassal B, Kanapin A, Lewis S, Mahajan S, May B, Schmidt E, Vastrik I, Wu G, Birney E, Stein L, D’Eustachio P. Reactome knowledgebase of human biological pathways and processes.<em>Nucleic Acids Res.</em>37:D619-22. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/18981052\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2007</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Vastrik I, D’Eustachio P, Schmidt E, Gopinath G, Croft D, de Bono B, Gillespie M, Jassal B, Lewis S, Matthews L, Wu G, Birney E, Stein L.Reactome: a knowledge base of biologic pathways and processes.<em>Genome Biol</em>. 8:R39. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/17367534\">PubMed</a></li>\r\n<li>Matthews L, D’Eustachio P, Gillespie M, Croft D, de Bono B, Gopinath G, Jassal B, Lewis S, Schmidt E, Vastrik I, Wu G, Birney E, Stein L. An Introduction to the Reactome Knowledgebase of Human Biological Pathways and Processes.<em>Bioinformatics Primer, NCI/Nature Pathway Interaction Database. </em><a href=\"http://pid.nci.nih.gov/PID/2007/071211/full/pid.2007.3.shtml\">doi:10.1038/pid.2007.3.</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2005</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Joshi-Tope G, Gillespie M, Vastrik I, D’Eustachio P, Schmidt E, de Bono B, Jassal B, Gopinath GR, Wu GR, Matthews L, Lewis S, Birney E, Stein L. Reactome: a knowledgebase of biological pathways.<em>Nucleic Acids Res.</em>1:D428-32. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/15608231\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2004</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Stein LD. Using the Reactome database.<em>Curr Protoc Bioinformatics.</em>Chapter 8:Unit 8.7. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/18428737\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n<p><strong>2003</strong></p>\r\n<ul>\r\n<ul>\r\n<ul>\r\n<li>Joshi-Tope G, Vastrik I, Gopinath GR, Matthews L, Schmidt E, Gillespie M, D’Eustachio P, Jassal B, Lewis S, Wu G, Birney E, Stein L. The Genome Knowledgebase: a resource for biologists and bioinformaticists.<em>Cold Spring Harb Symp Quant Biol.</em>68:237-43. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/15338623\">PubMed</a></li>\r\n</ul>\r\n</ul>\r\n</ul>\r\n</div>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 15:46:05','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(25,559,'Training Materials','training','<h3>Online Tutorial</h3>\r\n<h5>Available via the EBI Train Online system</h5>\r\n<div style=\"padding: 0; margin: 0; border: 0;\">{loadposition new-training-1}</div>\r\n<h3>Supplemental material</h3>\r\n<p style=\"text-align: justify;\"><b>User’s Guide. </b><span>One of our most valuable user documents is the </span><a href=\"user/guide\"><span>Reactome User’s Guide</span></a><span>. Please look through the guide and </span><span>contact us</span><span> if our guide needs something new.</span></p>\r\n<p style=\"text-align: justify;\"><b>Curator’s Guide</b><span>. The document contains an overview of the curatorial process as well as a step by step guide for annotating Reactome pathways. </span></p>\r\n<p style=\"text-align: justify;\"><b>Reactome Tutorial Material. </b><span>All our training and outreach materials are available under a </span><a href=\"license\"><span>Creative Commons Attribution 4.0 Unported License</span></a><span>. We have just two simple requests, please attribute Reactome, and let us know if you use our presentations, posters, or training resources. Here are some of the material that we make available to download:</span></p>\r\n<ul style=\"padding-left: 30px;\">\r\n<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><a href=\"docs/training/Reactome_Website.pdf\"><span>Tutorial workshop</span></a><span> slides. </span>This document provides additional information about the Reactome website and suite of data analysis tools, including exercises to allow you to check your understanding. Answers are provided and were correct when written, but please note that answers may change over time as more pathways are added to Reactome.</li>\r\n<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><span><a href=\"docs/training/Pathways_&amp;_Networks_Overview.pdf\">Pathways and Networks Overview</a>. These slides provide an overview of pathway and network resources for data visualisation, analysis, and integration.</span></li>\r\n<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><span><a href=\"docs/training/ReactomeFIVizapp.pdf\">Reactome FI Viz app</a>. These slides provide an introduction to the Reactome Functional Interaction Network and Cytoscape app. These Reactome </span><a href=\"docs/training/ReactomeFIVizapp_Lab_Questions.pdf\"><span>Questions</span></a><span> and </span><a href=\"docs/training/ReactomeFIVizapp_Lab_Answers.pdf\"><span>Answers</span></a><span> provide a series of questions and answers to demonstrate the functionality of the Reactome FI Viz app. </span></li>\r\n<!--<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><span>Online Training courses. Available via the EBI Train Online system - </span><a href=\"https://www.ebi.ac.uk/training/online/course/reactome-quick-tour\"><span>Reactome:Quick Tour</span></a><span>, </span><a href=\"https://www.ebi.ac.uk/training/online/course/reactome-exploring-biological-pathways\"><span>Reactome: Exploring Biological Pathways</span></a><span>, </span><span><a href=\"https://www.ebi.ac.uk/training/online/course/reactome-tools-analysis-biological-pathways\">Reactome: Tools for Analysis of Biological Pathways</a></span></li>--></ul>\r\n<h3>Introduction to the Reactome Pathway Knowledgebase</h3>\r\n<p>{source}&lt;style&gt;.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }&lt;/style&gt;&lt;div class=\'embed-container\'&gt;&lt;iframe src=\'https://www.youtube.com/embed/videoseries?si=ZgK9G3Ws-Zxin16h&amp;list=PLLc8-qElSrCxhMsCVjYIO8OmO-Im_lQ6D\' frameborder=\'0\' allowfullscreen&gt;&lt;/iframe&gt;&lt;/div&gt;{/source}<br /><br /></p>\r\n<p></p>','',0,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 15:46:13','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"0\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(26,560,'Partners','partners','<style type=\"text/css\">#fav-maincontent {\r\n    margin-bottom: 5px;\r\n}\r\np {\r\n    margin: 0 0 0 0;\r\n}</style>\r\n<p>The following table contains the known resources that have integrated Reactome by using the&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=47&amp;catid=29\">Analysis Service</a>, the <a href=\"index.php?option=com_content&amp;view=article&amp;id=45&amp;catid=29\">Graph Database</a>&nbsp;or any of our widgets (<a href=\"index.php?option=com_content&amp;view=article&amp;id=50&amp;catid=29\">diagram</a>&nbsp;or <a href=\"index.php?option=com_content&amp;view=article&amp;id=49&amp;catid=29\">pathways overview</a>). If your resource is using our services, widgets or content and it is not in the list,&nbsp;please <a href=\"mailto:help@reactome.org?subject=Including%20a%20Reactome%20Partner\">let us know</a>&nbsp;and we will add it.</p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 16:46:52','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":\"\",\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":\"\",\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":\"\",\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(27,561,'Resources Guide','resources-guide','<style type=\"text/css\">@media only screen and (max-width: 767px) {div[id^=\"r-responsive-table\"] table.reactome.resource-guide td:nth-of-type(1):before { content: \"Name\"; }\r\ndiv[id^=\"r-responsive-table\"] table.reactome.resource-guide td:nth-of-type(2):before { content: \"Description\"; }}</style>\r\n<p><span>We actively seek collaborations with other data resources and users to improve data integration, share efforts, and make our data maximally useful to biologists and bioinformaticians. Here are a list of software tools, websites, databases and research projects that: i) support the use of Reactome data, ii) have integrated Reactome data, or iii) provide data linkages to the Reactome website. </span></p>\r\n<p><span>If we are missing your website, database, software tool and project, please contact our </span><a href=\"mailto: &lt;a href=\"></a><a href=\"mailto:help@reactome.org\"><span>Help Desk</span></a><span>.</span></p>\r\n<p></p>\r\n<p><iframe src=\"https://docs.google.com/spreadsheets/d/e/2PACX-1vRbRm3dc7Ms8AmDm4zR67jSWhFSb-Jgkf0vfyBdnMF-wfNC0aZJfK3ltVOe5lVwhHDgdtkYRHbtM4q9/pubhtml?gid=0&amp;single=true&amp;widget=false&amp;headers=false&amp;chrome=false\" width=\"100%\" height=\"1000px\"></iframe></p>\r\n<p></p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 15:48:13','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(28,562,'Linking to us','linking-to-us','<p><span>Reciprocal linking between related bioinformatics resources not only encourages mutual traffic but ensures that Reactome users can quickly linkout to the source reference material as they navigate through our website. Furthermore, many Reactome visitors are referred to this web site by many bioinformatics resources that integrate Reactome data. These resources include UniProt, Genecards, OMIM, NCBI Biosystems, MSigDB, HGNC, NextProt, BioGPS, Pathway Commons, and Wikipathways.</span></p>\r\n<p><span>Our goal with distributing different sets of files for each identifier type (i.e. protein (</span><a href=\"https://www.uniprot.org\"><span>UniProt</span></a><span>), gene (Ensembl and NCBI), small molecule (ChEBI), and microRNA (miRBase)) is to link the source database identifier to: </span></p>\r\n<ol style=\"list-style-type: lower-alpha;\">\r\n<li><span>the lowest level pathway diagram or subset of the pathway,</span></li>\r\n<li><span>all level pathway diagrams, and all reaction events.</span></li>\r\n</ol>\r\n<p><span>The identifier mapping files can be obtained from our </span><a href=\"download-data\"><span>Downloads</span></a><span> section.</span></p>\r\n<p><span>An alternative strategy to link to Reactome can be achieved by creating URLs containing the name of and an identifier from an “external” database in the following format: </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=identifier\"><i><span>https://reactome.org/content/query?cluster=true&amp;q=identifier</span></i></a><i><span>.</span></i></p>\r\n<p><span>Click </span><a href=\"index.php?option=com_content&amp;view=article&amp;id=70\"><span>here</span></a><span> to see the external databases and list of identifiers available in the current release of Reactome</span></p>\r\n<p><span>Below are few concrete examples:</span></p>\r\n<ul style=\"list-style-type: circle;\">\r\n<li><a href=\"https://www.uniprot.org/\"><span>UniProt</span></a><span> accession numbers (the ‘AC’ line, preferred) and identifiers (the ‘ID’ line), e.g. UNIPROT:P30304, </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=P30304\"><span>https://reactome.org/content/query?cluster=true&amp;q=P30304</span></a></li>\r\n<li><a href=\"https://www.ebi.ac.uk/chebi/\"><span>ChEBI</span></a><span> identifiers, e.g. CHEBI:15422, </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=15422\"><span>https://reactome.org/content/query?cluster=true&amp;q=15422</span></a></li>\r\n<li><a href=\"http://www.genome.jp/kegg/compound/\"><span>COMPOUND</span></a><span> identifiers, e.g. COMPOUND:C00002, </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=C00002\"><span>https://reactome.org/content/query?cluster=true&amp;q=C00002</span></a></li>\r\n<li><a href=\"http://www.geneontology.org/\"><span>Gene Ontology</span></a><span> (GO) accession numbers, e.g. GO:0030060, </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=0030060\"><span>https://reactome.org/content/query?cluster=true&amp;q=0030060</span></a></li>\r\n<li><a href=\"https://www.brenda-enzymes.info/index.php\"><span>Enzyme Classification</span></a><span> (EC) numbers, e.g. EC:1.1.1.37, </span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=1.1.1.37\"><span>https://reactome.org/content/query?cluster=true&amp;q=1.1.1.37</span></a></li>\r\n<li><a href=\"https://www.ncbi.nlm.nih.gov/gene\"><span>NCBI Gene</span></a><span>, e.g. NCBI Gene:4171, </span><span><a href=\"https://reactome.org/content/query?cluster=true&amp;q=4171\">https://reactome.org/content/query?cluster=true&amp;q=4171</a></span></li>\r\n</ul>\r\n<p> </p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 16:18:42','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,10,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(29,563,'Referencing our Publications','referencing-our-publications','<h4 style=\"text-align: justify;\"><strong>To cite the use of Reactome in your work,</strong> please reference one or more of the following publications:</h4>\r\n<ul>\r\n<li style=\"text-align: justify;\">Milacic M, Beavers D, Conley P, Gong C, Gillespie M, Griss J, Haw R, Jassal B, Matthews L, May B, Petryszak R, Ragueneau E, Rothfels K, Sevilla C, Shamovsky V, Stephan R, Tiwari K, Varusai T, Weiser J, Wright A, Wu G, Stein L, Hermjakob H, D’Eustachio P. The Reactome Pathway Knowledgebase 2024. Nucleic Acids Research. 2024. doi: 10.1093/nar/gkad1025. <a href=\"https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkad1025/7369850?utm_source=advanceaccess&amp;utm_campaign=nar&amp;utm_medium=email\">Link</a></li>\r\n<li style=\"text-align: justify;\">Griss J, Viteri G, Sidiropoulos K, Nguyen V, Fabregat A, Hermjakob H. ReactomeGSA - Efficient Multi-Omics Comparative Pathway Analysis. Mol Cell Proteomics. 2020 Sep 9. doi: 10.1074/mcp. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/32907876\">PubMed PMID: 32907876</a>.</li>\r\n<li style=\"text-align: justify;\">Jassal B, Matthews L, Viteri G, Gong C, Lorente P, Fabregat A, Sidiropoulos K, Cook J, Gillespie M, Haw R, Loney F, May B, Milacic M, Rothfels K, Sevilla C, Shamovsky V, Shorser S, Varusai T, Weiser J, Wu G, Stein L, Hermjakob H, D\'Eustachio P. The reactome pathway knowledgebase. <em>Nucleic Acids Res.</em> 2020 Jan 8;48(D1):D498-D503. doi: 10.1093/nar/gkz1031. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/31691815\">PubMed PMID: 31691815</a>.</li>\r\n<li style=\"text-align: justify;\">Fabregat A, Korninger F, Viteri G, Sidiropoulos K, Marin-Garcia P, Ping P, Wu G, Stein L, D\'Eustachio P, Hermjakob H. Reactome graph database: Efficient accessto complex pathway data. <em>PLoS Comput Biol.</em> 2018 Jan 29;14(1):e1005968. doi: 10.1371/journal.pcbi.1005968. eCollection 2018 Jan. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29377902\">PubMed PMID: 29377902</a>.</li>\r\n<li style=\"text-align: justify;\">Fabregat A, Sidiropoulos K, Viteri G, Marin-Garcia P, Ping P, Stein L, D\'Eustachio P, Hermjakob H. Reactome diagram viewer: data structures and strategies to boost performance. <em>Bioinformatics.</em> 2018 Apr 1;34(7):1208-1214. doi: 10.1093/bioinformatics/btx752. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29186351\">PubMed PMID: 29186351</a>.</li>\r\n<li style=\"text-align: justify;\">Sidiropoulos K, Viteri G, Sevilla C, Jupe S, Webber M, Orlic-Milacic M, Jassal B, May B, Shamovsky V, Duenas C, Rothfels K, Matthews L, Song H, Stein L, Haw R, D\'Eustachio P, Ping P, Hermjakob H, Fabregat A. Reactome enhanced pathway visualization. <em>Bioinformatics.</em> 2017 Nov 1;33(21):3461-3467. doi: 10.1093/bioinformatics/btx441. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/29077811\">PubMed PMID: 29077811</a>.</li>\r\n<li style=\"text-align: justify;\">Fabregat A, Sidiropoulos K, Viteri G, Forner O, Marin-Garcia P, Arnau V, D\'Eustachio P, Stein L, Hermjakob H. Reactome pathway analysis: a high-performance in-memory approach. <em>BMC Bioinformatics.</em> 2017 Mar 2;18(1):142. doi: 10.1186/s12859-017-1559-2. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/28249561\">PubMed PMID: 28249561</a>.</li>\r\n<li style=\"text-align: justify;\">Wu G, Haw R. Functional Interaction Network Construction and Analysis for Disease Discovery. <em>Methods Mol Biol.</em> 2017;1558:235-253. doi: 10.1007/978-1-4939-6783-4_11. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/28150241\">PubMed PMID: 28150241</a>. </li>\r\n</ul>\r\n<p></p>\r\n<p></p>\r\n<h4 style=\"text-align: justify;\"><strong>To cite a pathway.</strong> </h4>\r\n<p style=\"text-align: justify;\">Please use the appropriate DOI from the <a href=\"/content/doi\" rel=\"nofollow\">Table of Contents</a> within the citation, when it is available. Otherwise, use the stable identifier of the pathway. You can add a DOI to the end of your citation following the appropriate style. Generally, these citations follow this format: Author, A. (year). “Title of pathway\". Reactome, release#, URL with doi:xxxxxx (date of access). If a DOI is unavailable, please follow this format: “Title of pathway\". Reactome, release#, URL with StableID: R-HSA-xxxxxx.x (date of access).</p>\r\n<p style=\"text-align: justify;\">Please find other citing styles at APA Style: <a href=\"https://owl.purdue.edu/owl/research_and_citation/apa_style/apa_formatting_and_style_guide/reference_list_electronic_sources.html\">Purdue U Online Writing Lab</a>.</p>\r\n<p></p>\r\n<p></p>\r\n<h4><strong>To reference an image.</strong></h4>\r\n<p>Within your citation, please use, where available, the DOI associated with the pathway of interest. This information can be found on our <a href=\"/content/doi\">DOI page</a>, when it is available. Otherwise, use the stable identifier of the pathway associated with the image. You can add a DOI to the end of your citation following the appropriate style. Generally, these citations follow this format:  Image for “Title of pathway\". Reactome, release#, URL with doi:xxxxxx (date of access). If a DOI is unavailable, please follow this format: Image for “Title of pathway\". Reactome, release#, URL with StableID: R-HSA-xxxxxx.x (date of access).</p>\r\n<p></p>\r\n<p></p>\r\n<h4 style=\"text-align: justify;\"><strong>To cite our files available for download.</strong></h4>\r\n<p style=\"text-align: justify;\">Please use the following format: \"Name of file\", Reactome, release#, <a href=\"https://reactome.org/download-data/\">https://reactome.org/download-data/</a> (date of access).</p>\r\n<p></p>\r\n<p></p>\r\n<h4 style=\"text-align: justify;\"><strong>When citing information obtained in a search.</strong></h4>\r\n<p style=\"text-align: justify;\">It should be remembered that while we strive to contain the most current and accurate data, Reactome should not be used in citations where other primary sources of information are available.</p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-04-27 16:25:22','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,8,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(39,587,'User Guide','user-guide','<h3>What is Reactome?</h3>\r\n<p><span>Reactome is a curated database of pathways and reactions in human biology. Reactions can be considered as pathway ‘steps’. Reactome defines a ‘reaction’ as any event in biology that changes the state of a biological molecule. Binding, activation, translocation, degradation and classical biochemical events involving a catalyst are all reactions. Information in the database is authored by expert biologists, entered and maintained by Reactome’s team of Curators and Editorial staff. Reactome content frequently cross-references other resources e.g. </span><a href=\"http://www.ncbi.nlm.nih.gov\" target=\"_blank\" rel=\"noopener noreferrer\"><span>NCBI</span></a><span>, </span><a href=\"http://www.ensembl.org\" target=\"_blank\" rel=\"noopener noreferrer\"><span>Ensembl</span></a><span>, </span><a href=\"http://www.uniprot.org/\" target=\"_blank\" rel=\"noopener noreferrer\"><span>UniProt</span></a><span>, KEGG (</span><a href=\"http://www.genome.jp/kegg/genes.html\" target=\"_blank\" rel=\"noopener noreferrer\"><span>Gene </span></a><span>and </span><a href=\"http://www.genome.jp/dbget-bin/www_bfind?compound\" target=\"_blank\" rel=\"noopener noreferrer\"><span>Compound</span></a><span>), </span><a href=\"http://www.ebi.ac.uk/chebi/init.do\"><span>ChEBI</span></a><span>, </span><a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?CMD=&amp;DB=PubMed\"><span>PubMed</span></a><span> and </span><a href=\"http://www.geneontology.org\" target=\"_blank\" rel=\"noopener noreferrer\"><span>GO</span></a><span>. </span><a href=\"documentation/inferred-events\"><span>Inferred orthologous reactions</span></a><span> Inferred orthologous reactions are available for 15 non-human species including mouse, rat, chicken, puffer fish, worm, fly, yeast, rice, and Arabidopsis</span></p>\r\n<h3>What is this guide For?</h3>\r\n<p><span>This guide introduces features of the Reactome website using a combination of short explanations and exercises. You will learn how to search, interpret the views, use the tools and if necessary find documentation or <a href=\"mailto:help@reactome.org\">contact us</a> for help.</span></p>\r\n<p>{loadposition user-guide-1}</p>\r\n<h3>Online Tutorial</h3>\r\n<p><span>Available via the EBI Train Online system</span></p>\r\n<div>{loadposition new-training-1}</div>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-08 19:51:22','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"0\",\"link_titles\":\"0\",\"show_tags\":\"\",\"show_intro\":\"0\",\"info_block_position\":\"\",\"info_block_show_title\":\"0\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"1\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,12,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(43,594,'404','404','<h3 class=\"fav404\">404</h3>\r\n<p class=\"fav404\">Oops! The page you requested was not found!</p>\r\n<p class=\"fav404\"><a href=\"/\" class=\"btn\">Back to Home Page</a></p>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-17 12:15:17','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"0\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,12,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(44,604,'ReactomeFIVIz','reactome-fiviz','<div id=\"toc\" class=\"toc\">\r\n<div id=\"toctitle\">\r\n<h2>Contents</h2>\r\n</div>\r\n<ul>\r\n<li class=\"toclevel-1 tocsection-1\"><a href=\"#Overview\"><span class=\"tocnumber\">1.</span> <span class=\"toctext\">Overview</span></a></li>\r\n<li class=\"toclevel-1 tocsection-2\"><a href=\"#Download_and_Launch_ReactomeFIViz\"><span class=\"tocnumber\">2.</span> <span class=\"toctext\">Download and Launch ReactomeFIViz</span></a></li>\r\n<li class=\"toclevel-1 tocsection-3\"><a href=\"#Use_Reactome_Pathways\"><span class=\"tocnumber\">3.</span> <span class=\"toctext\">Use Reactome Pathways</span></a>\r\n<ul>\r\n<li class=\"toclevel-2 tocsection-4\">        <a href=\"#Explore_Reactome_Pathways\"><span class=\"tocnumber\">3.1.</span> <span class=\"toctext\">Explore Reactome Pathways</span></a></li>\r\n<li class=\"toclevel-2 tocsection-5\">        <a href=\"#Display_Reactome_Pathways_in_the_FI_Network_View\"><span class=\"tocnumber\">3.2.</span> <span class=\"toctext\">Display Reactome Pathways in the FI Network View</span></a></li>\r\n<li class=\"toclevel-2 tocsection-6\">        <a href=\"#Pathway_Enrichment_Analysis\"><span class=\"tocnumber\">3.3.</span> <span class=\"toctext\">Pathway Enrichment Analysis</span></a></li>\r\n<li class=\"toclevel-2 tocsection-7\">        <a href=\"#Probabilistic_Graphical_Model_based_Pathway_Analysis\"><span class=\"tocnumber\">3.4.</span> <span class=\"toctext\">Probabilistic Graphical Model Based Pathway Analysis</span></a></li>\r\n<li class=\"toclevel-2 tocsection-8\">        <a href=\"#Boolean_Network_based_Pathway_Analysis\"><span class=\"tocnumber\">3.5.</span> <span class=\"toctext\">Boolean Network Based Pathway Analysis</span></a></li>\r\n<li class=\"toclevel-2 tocsection-8\">        <a href=\"#Structural_Variants_Visualization\"><span class=\"tocnumber\">3.6.</span> <span class=\"toctext\">Visualization of Structural Variants in the Context of Reactome Pathways</span></a></li>\r\n</ul>\r\n</li>\r\n<li class=\"toclevel-1 tocsection-9\"><a href=\"#Use_the_Reactome_Functional_Interaction_.28FI.29_Network\"><span class=\"tocnumber\">4.</span> <span class=\"toctext\">Use the Reactome Functional Interaction (FI) Network</span></a>\r\n<ul>\r\n<li class=\"toclevel-2 tocsection-10\">       <a href=\"#Gene_Set.2FMutation_Analysis\"><span class=\"tocnumber\">4.1.</span> <span class=\"toctext\">Gene Set/Mutation Analysis</span></a></li>\r\n<li class=\"toclevel-2 tocsection-11\">       <a href=\"#PGM_Impact_Analysis\"><span class=\"tocnumber\">4.2.</span> <span class=\"toctext\">PGM Impact Analysis</span></a></li>\r\n<li class=\"toclevel-2 tocsection-13\">       <a href=\"#Microarray_Data_Analysis\"><span class=\"tocnumber\">4.3.</span> <span class=\"toctext\">Microarray Data Analysis</span></a></li>\r\n</ul>\r\n</li>\r\n<li class=\"toclevel-1 tocsection-14\"><a href=\"#Visualize_Cancer_Drugs_in_Contexts_of_Reactome_Pathways_and_FI_Network\"><span class=\"tocnumber\">5.</span> <span class=\"toctext\">Visualize Drugs in the Contexts of Reactome Pathways and FI Network</span></a>\r\n<ul>\r\n<li class=\"toclevel-2 tocsection-15\">       <a href=\"#Visualize_Cancer_Drugs_in_Reactome_Pathways\"><span class=\"tocnumber\">5.1.</span> <span class=\"toctext\">Visualize Cancer Drugs in Reactome Pathways</span></a></li>\r\n<li class=\"toclevel-2 tocsection-16\">       <a href=\"#Visualize_Cancer_Drugs_in_the_FI_Network\"><span class=\"tocnumber\">5.2.</span> <span class=\"toctext\">Visualize Cancer Drugs in the FI Network</span></a></li>\r\n<li class=\"toclevel-2 tocsection-16\">       <a href=\"#Visualize_DrugCentral_Drugs\"><span class=\"tocnumber\">5.3.</span> <span class=\"toctext\">Visualize Drug Central Drugs</span></a></li>\r\n<li class=\"toclevel-2 tocsection-17\">       <a href=\"#Simulate_Impact_of_Cancer_Drugs_on_Pathway_Activities\"><span class=\"tocnumber\">5.4.</span> <span class=\"toctext\">Simulate Impact Drugs on Pathway Activities</span></a></li>\r\n</ul>\r\n</li>\r\n<li class=\"toclevel-1 tocsection-18\"><a href=\"#scRNA_seq\"><span class=\"tocnumber\">6.</span> <span class=\"toctext\">Perform scRNA-seq Data Analysis and Visualization</span></a>\r\n<ul>\r\n<li class=\"toclevel-2 tocsection-19\">       <a href=\"#scanpy_analysis\"><span class=\"tocnumber\">6.1.</span> <span class=\"toctext\">Standard Analysis via scanpy</span></a></li>\r\n<li class=\"toclevel-2 tocsection-20\">       <a href=\"#scvelo_analysis\"><span class=\"tocnumber\">6.2.</span> <span class=\"toctext\">RNA Velocity Analysis via scVelo</span></a></li>\r\n</ul>\r\n</li>\r\n<li class=\"toclevel-1 tocsection-18\"><a href=\"#Other_Features_Related_to_the_FI_Network\"><span class=\"tocnumber\">7.</span> <span class=\"toctext\">Other Features Related to the FI Network</span></a>\r\n<ul>\r\n<li class=\"toclevel-2 tocsection-19\">       <a href=\"#Query_FI_Source\"><span class=\"tocnumber\">7.1.</span> <span class=\"toctext\">Query FI Source</span></a></li>\r\n<li class=\"toclevel-2 tocsection-20\">       <a href=\"#Fetch_FIs_for_Node\"><span class=\"tocnumber\">7.2.</span> <span class=\"toctext\">Fetch FIs for Node</span></a></li>\r\n<li class=\"toclevel-2 tocsection-21\">       <a href=\"#Show_Pathway_Diagram\"><span class=\"tocnumber\">7.3.</span> <span class=\"toctext\">Show Pathway Diagram</span></a></li>\r\n<li class=\"toclevel-2 tocsection-22\">       <a href=\"#Load_Cancer_Gene_Index_Annotations\"><span class=\"tocnumber\">7.4.</span> <span class=\"toctext\">Load Cancer Gene Index Annotations</span></a></li>\r\n<li class=\"toclevel-2 tocsection-23\">       <a href=\"#Survival_Analysis\"><span class=\"tocnumber\">7.5.</span> <span class=\"toctext\">Survival Analysis</span></a></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</div>\r\n<h3><span id=\"Overview\">Overview</span></h3>\r\n<p>The <a href=\"http://f1000research.com/articles/3-146/v2\" rel=\"nofollow\">ReactomeFIViz</a> app is designed to find pathways and network patterns related to cancer and other types of diseases. This app accesses the <a href=\"https://reactome.org/\" rel=\"nofollow\">Reactome</a> pathways stored in the database, help you to do pathway enrichment analysis for a set of genes, visualize hit pathways using manually laid-out pathway diagrams directly in Cytoscape, and investigate functional relationships among genes in hit pathways. The app can also access the Reactome Functional Interaction (FI) network, a highly reliable, manually curated pathway-based protein functional interaction network covering over 60% of human proteins, and allows you to construct a FI sub-network based on a set of genes, query the FI data source for the underlying evidence for the interaction, build and analyze network modules of highly-interacting groups of genes, perform functional enrichment analysis to annotate the modules, expand the network by finding genes related to the experimental data set, display pathway diagrams, and overlay with a variety of information sources such as cancer gene index annotations. Recently we have also added features to help users visualize FDA-approved cancer drugs in the contexts of the FI network and Reactome pathways, and use Boolean network models directly built from Reactome pathways to investigate potential functional impacts of displayed cancer drugs.</p>\r\n<p>For an example how we use Reactome FIs for cancer data analysis, please see our publication: <a href=\"https://pubmed.ncbi.nlm.nih.gov/20482850/\" rel=\"nofollow\">A human functional protein interaction network and its application to cancer data analysis</a>.</p>\r\n<h3><span id=\"Download_and_Launch_ReactomeFIViz\">Download and Launch ReactomeFIViz</span></h3>\r\n<p>ReactomeFIViz app 6 needs Cytoscape 3.7.0 or above. If you have not installed Cytoscape 3.7.0 or above, please download it from Cytoscape\'s web site: <a href=\"http://www.cytoscape.org/\" rel=\"nofollow\">http://www.cytoscape.org</a>. After launching Cytoscape, use menu \"Apps/App Manager\" to open the \"App Manager\" dialog, and search for \"ReactomeFI\". You should see the ReactomeFIViz app listed in the middle panel (See the Figure below. You may see a different version number. <b>Note: The listed name of this app is \"ReactomeFIPlugIn\"</b>, which is the original name of the app.). Choose the app, and then click the \"Install\" button at the bottom of the dialog. Follow the procedures to finish the installation.</p>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/InstallReactomeFIVizFromAppstore.png\" alt=\"\" width=\"663\" height=\"540\" />\r\n<div>Install ReactomeFIViz app From App Store</div>\r\n</div>\r\n</div>\r\n</div>\r\n<h3><span id=\"Use_Reactome_Pathways\">Use Reactome Pathways</span></h3>\r\n<p>Using the pathway visualization and analysis features, you can load pathways in the Reactome database into Cytoscape, visualize Reactome pathways in either the native pathway diagram view or the FI network view, do pathway enrichment analysis for a set of genes, and check genes from your list in hit pathways.</p>\r\n<h4><span id=\"Explore_Reactome_Pathways\">Explore Reactome Pathways</span></h4>\r\n<ol>\r\n<li>Load Reactome pathways: Use menu \"Apps/Reactome FI/Reactome Pathways\" to load pathways into Cytoscape. The loaded pathways are organized in a hierarchical way as in the Reactome web application (<a href=\"/PathwayBrowser\" rel=\"nofollow\">https://reactome.org/PathwayBrowser/</a>), and listed in the left side \"Control Panel\" in the tab called \"Reactome\".\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ReactomePathways_4.png\" alt=\"\" width=\"429\" height=\"602\" />\r\n<div>Reactome Pathways</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li>View pathways in Reactome: After selecting a pathway in the pathway hierarchy, you can choose \"View Reactome Source\" from the popup menu (right click in Windows or Control-click in Macs to get the popup menu)\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/PathwayPopup_4.png\" alt=\"\" width=\"340\" height=\"248\" />\r\n<div>Pathway Popup Menu</div>\r\n</div>\r\n</div>\r\nto view its detailed annotation in Reactome. Or you can choose \"View in Reactome\" to view the detailed information in the Reactome web application. <br /> <b>Note</b>: The ancestor pathways (container pathways) for a selected pathway are displayed in the middle panel, \"Selected Event Branch\", in the Reactome tab. You can click an ancestor pathway in this middle panel to view the clicked pathway\'s location in the original pathway hierarchical tree. However, the ancestor pathway will not be selected in the original tree. This is a designed behavior to keep the selection in the original tree.</li>\r\n<li>Search pathways: Choose \"Search\" in the popup menu to bring up the search dialog. The found pathway(s) will be highlighted in blue in the pathway tree. <br /> <b>Note</b>: Search will be against all loaded pathways, not limited to the selected pathway and its contained sub-pathways.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/PathwaySearch_4.png\" alt=\"\" width=\"369\" height=\"199\" />\r\n<div>Search Pathways</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li>Open Reactome Reacfoam: The Reacfoam view provides a holistic view of all (exclude disease) human pathways in the Reactome database. Choose \"Open Reactome Reacfoam\" in the popup menu to open the Reactome Reacfoam in the default browser. <br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/OpenReacfoamPopup.png\" alt=\"Open Reactome Reacfoam\" width=\"400\" />\r\n<div>Open Reacfoam</div>\r\n</div>\r\n</div>\r\n</div>\r\n<b>Note</b>: Pressing your mouse and then holding it to a pathway box will select the pathway in the tree of ReactomeFIV automatically.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/Reacfoam.png\" alt=\"\" width=\"1000\" />\r\n<div>Reactome Reacfoam</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li>Open pathway diagram: Pathways in Reactome are organized in a hierarchical way. Not all pathways have their own pathway diagrams. A smaller pathway (called sub-pathway) may be drawn in a bigger pathway, which has its own pathway diagram. Most of top-level pathways (called modules or super pathways) are used to organize related pathways (e.g. Disease, Signaling Transduction), and therefore contain only rectangle boxes representing canonical pathways.<ol>\r\n<li><b>Show Diagram</b>: If a selected pathway has its own pathway diagram, you can choose \"Show Diagram\" in the popup menu to open its pathway diagram into the central Cytoscape desktop.</li>\r\n<li><b>View in Diagram</b>: If a selected pathway is laid-out as a sub-pathway in a bigger one, you can choose \"View in Diagram\" in the popup menu to view its drawing in its container pathway. Reactions contained by the selected pathway will be highlighted in blue after the diagram is opened. For example, see pathway \"G1/S DNA Damage Checkpoints\" opened in pathway \"Cell Cycle Checkpoints\" below:<br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/PathwayDiagram_4.png\" alt=\"\" width=\"800\" height=\"540\" />\r\n<div>Pathway Diagram</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n</ol></li>\r\n<li>Search diagram: Objects displayed in a pathway diagram can be searched using \"Search Diagram\" from the popup menu (Right click in Windows or Control click in Macs without selecting any object in the pathway diagram to get the popup menu). The found objects will be selected and highlighted in blue. <br /> <b>Note</b>: Reactions will not be searched in the diagram. Use the search feature in the pathway tree to search for reactions.</li>\r\n<li>Export diagram: Displayed diagram can be exported as a PDF, JPG or PNG file. Use \"Export Diagram\" from the popup menu to export the displayed diagram.</li>\r\n<li>View Reactome Source or View in Reactome: Select an object, and then right-click (or control click) to get the popup menu. Choose \"View Reactome Source\" to view the detailed annotation for the selected object in a table (See Figure below for an example). Or choose \"View in Reactome\" to view the selected object in the Reactome web application.<br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ReactomeInstanceView.png\" alt=\"\" width=\"497\" height=\"393\" />\r\n<div>Reactome Instance View</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li>List Genes: Genes contained by a complex or protein set, or a gene related by a displayed protein can be viewed by using a menu item \"List Genes\" after selecting an object. For example, the following dialog shows genes contained by complex hBUBR1:hBUB3:MAD2*:CDC20. Clicking a gene symbol will bring you to the web page for that gene in the GeneCard web site.<br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ListGenes_4.png\" alt=\"\" width=\"448\" height=\"406\" />\r\n<div>List Genes</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n</ol>\r\n<h4><span id=\"Display_Reactome_Pathways_in_the_FI_Network_View\">Display Reactome Pathways in the FI Network View</span></h4>\r\n<ol>\r\n<li>Display pathway in the FI network view: A Reactome pathway can be converted into a functional interaction network using the method we have established (see <a href=\"https://doi.org/10.1186/gb-2010-11-5-r53\" rel=\"nofollow\">A human functional protein interaction network and its application to cancer dat analysis</a>). Use \"Convert to FI Network\" in the popup menu brought up by right-clicking (Windows) or control-clicking (Macs) an empty area without any selection in the pathway diagram panel. The original pathway diagram will be moved to the bottom-left corner, and a new FI network will be generated based on the original pathway diagram, which will be displayed in a new network panel. <br /> <b>Note</b>: sub-pathways contained by the displayed pathway will be extracted into the FI network too.<br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/PathwayInFINetworkView_4.png\" alt=\"\" width=\"800\" height=\"486\" />\r\n<div>Pathway in the FI Network View</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li>Explore objects in the pathway and network views: Object selection in three views has been synchronized. Objects that can be selected include: events in the pathway tree view, objects in the pathway view at the bottom-left corner, and genes and FIs in the network view. You can select an object in one of three views, and corresponding objects in other two views should be selected too. Also you should use features implemented in popup menus in each individual view to explore objects as in a single view.</li>\r\n</ol>\r\n<p><br /> <b>Note</b>: Using Cytoscape\'s built-in \"Saving Session\" feature can save the converted FI networks from pathways. However, displayed pathways cannot be saved into a session file for the time being. We will implement this function in a future release.</p>\r\n<h4><span id=\"Pathway_Enrichment_Analysis\">Pathway Enrichment Analysis</span></h4>\r\n<ol>\r\n<li><b>Pathway enrichment analysis</b>: A list of genes can be used to check if any of Reactome pathways have been enriched. To do this, use the popup menu item, \"Analyze Pathway Enrichment\" (below left figure), to get the dialog for choosing a gene set file (below right figure). You can use a gene set file in one of three file formats: one gene per line, all genes in the same line and delimited by commas, or all genes in the same line and delimited by tabs. You can also manually input genes by clicking the \"Click to Enter\" button (one line for one gene).<br /> <b>Note</b>: Dependent on the size of your gene list, it may take over 1 minute for running the pathway enrichment analysis. Pathways used in this feature are different from Reactome pathways for annotating a FI network or network modules. Here all over 2,000 pathways are used. For annotation, only a subset of Reactome pathways, which have been pre-selected for a certain size, are used.<br /> <img src=\"images/userguide/fiviz/AnalyzeGeneEnrichment_4.png\" alt=\"Analyze Pathway Enrichment\" height=\"233\" /><img src=\"images/userguide/fiviz/DialogForAnalyzeGeneEnrichment_4.png\" alt=\"Dialog for Analyzing Pathway Enrichment\" width=\"400\" height=\"233\" /><br /> <b>Note</b>: To get a holistic view of the pathway enrichment analysis results, open the Reactome Reacfoam after the analysis using the popup menu \"Open Reactome Reacfoam\" for the pathway tree. You may also download the Reacfoam view by clicking the download button at the top-right corner. For windows 10 users, to open the Reacfoam view, you need to allow \"public\" access to Cytoscape by checking \"public\" in the settings for \"Allow an app through Windows Firewall\" in the \"System and Security\" control settings.<br /> <img src=\"images/userguide/fiviz/Reacfoam_Enrichment.png\" alt=\"Reacfoam View for Enrichment Analysis\" width=\"1000\" /></li>\r\n<li><b>View enrichment analysis results</b>: Pathway enrichment results are displayed as a table labeled as \"Reactome Pathway Enrichment\" in the \"Table Panel\" at the bottom of the main Cytoscape window. You can use \"Views in Diagram\" to view hit pathways in the pathway diagram view, and use \"Export Annotations\" to save the results in the table. Pathways in the Reactome pathway tree are highlighted in different colors based on their FDR values. Objects containing genes from your gene list are highlighted in a purple background with a white font in the pathway diagram view. Hit genes are displayed in a thick purple border in the FI network view for a hit pathway. <br /> <b>Note</b>: Hit genes are displayed with same colors in the \"Gene List\" dialog from the \"List Genes\" feature.\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/PathwayEnrichmentResults_4.png\" alt=\"\" width=\"800\" height=\"460\" />\r\n<div>Pathway Enrichment Results</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li><b>Perform GSEA analysis</b>: Gene Set Enrichment Analysis (<a href=\"http://software.broadinstitute.org/gsea/index.jsp\">GSEA</a>) is a rank-based pathway enrichment analysis approach, widely used in pathway-based data analysis. ReactomeFIViz provides support to perform GSEA analysis for Reactome pathways using a gene score file. Gene score may be t-score from differential gene expression analysis or other type of scores that can be ranked. To perform the GSEA pathway enrichment analysis, you need to provide a tab-delimited text file containing two columns: the first for gene symbols (human only) and the second for gene scores. The first row is reserved for the column headers, and will not be imported for analysis. To perform GSEA analysis, use popup menu \"Perform GSEA Analysis\" in the pathway tree to bring up the GSEA configuration dialog, where you can enter the gene score file and choose the minimum and maximum size of pathways along with the permutation number.\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/PerformGSEAAnalysis.png\" alt=\"Perform GSEA Analysis\" width=\"400\" />\r\n<div>Perform GSEA Analysis</div>\r\n<img src=\"images/userguide/fiviz/ConfigureGSEAAnalysis.png\" alt=\"Congigure GSEA Analysis\" width=\"400\" />\r\n<div>Configure GSEA Analysis</div>\r\n</div>\r\n<div>\r\n<p>The GSEA analysis results are displayed in the table labeled as \"Reactome GSEA Analysis\" in Cytoscape Table Panel. Pathways subject to GSEA analysis in the pathway tree are highlighted based on FDR values as in the gene set-based pathway enrichment analysis (See above). For details about the meanings of columns shown in the results table, please consult the original GSEA document: <a href=\"http://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Main_Page\">GSEA Document</a>.</p>\r\n</div>\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/GSEAAnalysisResults.png\" alt=\"GSEA Analysis Results\" width=\"700\" />\r\n<div>GSEA Analysis Results</div>\r\n</div>\r\n</li>\r\n<li><b>Overlay Gene Scores onto Pathways</b>: For significant pathways produced from the GSEA analysis, you can overlay gene scores to investigate locations of products of genes having significant high or low scores, therefore to understand potential pathway activity impact caused by these extreme scores. To do this, use popup menu \"Overlay Gene Scores\" in pathway diagram view to choose the gene score file in the configuration dialog. After the file loading, entities in pathway diagrams will be highlighted based on scores. You may choose one or more genes in the right gene scores Table View to visualize related entities in the pathway diagram.\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/OverlayGeneScores.png\" alt=\"Overlay Gene Scores\" width=\"375\" />\r\n<div>Overlay Gene Scores</div>\r\n<img src=\"images/userguide/fiviz/GeneScoreOverlayResults.png\" alt=\"Gene Score Overlay Results\" width=\"900\" />\r\n<div>Gene Score Overlay Results</div>\r\n</div>\r\n<div>\r\n<p><b>Note</b>: If an entitiy (e.g. a complex or an EntitySet) is composed of more than one gene, the score for the entity is the mean of all genes annotated for that entity. To remove overlaid gene scores in the pathway diagram, use popup menu \"Remove Gene Scores\". To view the distribution of scores for genes annotated in the displayed pathway diagram, choose \"Plot View\" in the \"Gene Scores\" tab in Cytoscape Results Panel (see below).</p>\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/GeneScoreDistribution.png\" alt=\"Gene Score Distribution\" width=\"350\" />\r\n<div>Gene Score Distribution</div>\r\n</div>\r\n</div>\r\n</li>\r\n</ol>\r\n<h4><span id=\"Probabilistic_Graphical_Model_based_Pathway_Analysis\">Probabilistic Graphical Model based Pathway Analysis</span></h4>\r\n<p>We adapted the PARADIGM approach for Reactome pathways by converting reactions drawn in pathway diagrams into factors in factor graphs, a type of probabilistic graphical models (PGMs). For details about the PARADIGM approach, see: <a href=\"https://pubmed.ncbi.nlm.nih.gov/20529912/\" rel=\"nofollow\">Inference of patient-specific pathway activities from multi-dimensional cancer genomics data using PARADIGM</a>. For introduction to factor graphs, see this wikipedia entry: <a href=\"http://en.wikipedia.org/wiki/Factor_graph\" rel=\"nofollow\">Factor Graph</a>. For test purposes, you can download two sample data files for 100 TCGA ovarian cancer patients: <a href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/ImpactAnalysis/ov.CNV.100.txt.zip\" title=\"ov.CNV.100.txt.zip\">CNVs</a> and <a href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/ImpactAnalysis/ov.mRNA.100.txt.zip\" title=\"ov.mRNA.100.txt.zip\">mRNA gene expression</a>. The original TCGA OV files were downloaded from <a href=\"https://gdac.broadinstitute.org\" rel=\"nofollow\">the Broad GDAC</a><a href=\"https://gdac.broadinstitute.org\" rel=\"nofollow\"> Firehose</a> web site.</p>\r\n<ol>\r\n<li><b>Run graphical model analysis in batch:</b> This feature is used to perform a batch graphical model analysis for all Reactome pathways having manual layout diagrams.<ol>\r\n<li><b>Start the analysis:</b> Choose the popup menu, \"Run Graphical Model Analysis\", in the pathway hierarchical tree. After choosing this menu, you will be asked to choose data files and provide parameters for inference algorithms in the following two tabs in the \"Run Graphical Model Analysis\" dialog:<br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/LoadData.png\" alt=\"\" width=\"500\" height=\"509\" />\r\n<div>LoadData</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/SetUpAlgorithms.png\" alt=\"\" width=\"500\" height=\"504\" />\r\n<div>SetUpAlgorithms</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> <b>Notes</b>: <br />1). If you choose \"Use empirical distribution\" in the data loading dialog, your loaded data will be used directly to construct factor functions without discretizing. At present, we recommend to use \"Choose threshold values for discretizing\". <br />2). It is recommended to use the default parameters for inference algorithms for a batch analysis for quick performance. You can try different parameters for some specific pathways after you find interesting pathways from the batch analysis. <i><b>If you want to perform two-case study (e.g. case-control, drug sensitive/insensitive, etc), check the checkbox, \"Used for pathway analysis for samples with two cases\", and provide a sample information file as required. For two-case analysis, a random data set will not be generated. Results will be presented by comparing two types of samples in your uploaded data files.</b></i></li>\r\n<li><b>Run the analysis:</b> Click the \"OK\" button to start the batch analysis. Depending on your sample size, it may take hours to finish the whole analysis.</li>\r\n<li><b>Finish the analysis:</b> After the batch analysis done, you may see the following list if some of pathways cannot be analyzed because the inference algorithm cannot converge. Please make sure the following list is small (probably less than 10 pathways) so that you can get enough results.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/FailedPathwaysList.png\" alt=\"\" width=\"523\" height=\"289\" />\r\n<div>FailedPathwaysList</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li><b>View the results:</b> The results from the batch analysis are displayed at the bottom table panel of the Cytoscape desktop as the following:\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/BatchResults.png\" alt=\"\" width=\"800\" height=\"280\" />\r\n<div>BatchResults</div>\r\n</div>\r\n</div>\r\n</div>\r\n<b>Note</b>: There are 7 columns in this table: ReactomePathway for pathway names analyzed by the App; AverageUpIPA shows how much a pathway is up-perturbated by comparing to a random background (IPA: integrated pathway activity. See the above PARADIGM paper for details); AverageDownIPA shows how much a pathway is down-perturbated; CombinedPValue is a p-value indicating how significant this pathway is perturbed based on pathway outputs and the Fisher\'s method; MinimumPValue is the minimum p-value for pathway outputs; the last two columns are FDRs for two p-values based on the Benjamini–Hochberg method. AverageUpIPA or AverageDownIPA may be NaN, which indicates there is no detected up or down perturbation based on this analysis. The FDR filtering works based on the FDR values displayed in the last two columns with \"OR\" operation.</li>\r\n<li><b>Save the results:</b> To keep the results, use the popup menu in the table, \"Export Annotations\", to save the results into an external text file. The saved results can be loaded later on by using popup menu, \"Load Graphical Model Results\", in the pathway tree.</li>\r\n</ol></li>\r\n<li><b>Run graphical model analysis for a single pathway:</b> This feature is used to perform a graphical model analysis for a pathway displayed in the Cytoscape desktop.<ol>\r\n<li><b>Open a pathway:</b> As before, you can choose a pathway in the pathway tree, and open its diagram in the Cytoscape desktop. Or you can choose an interesting pathway from the batch analysis results table by choosing popup menu, \"View in Diagram\".</li>\r\n<li><b>Start the analysis:</b> Choose popup menu, \"Run Graphical Model Analysis\", from the popup menu list in the pathway diagram window. You will be asked to provide data files and set up inference algorithms as in the batch analysis. After clicking the \"OK\" button, you will be asked to provide a list of escape names for entities in the pathway that will not be considered in the graphical model (e.g. ATP, ADP, etc) in the following dialog:\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/EscapeNamesDialog.png\" alt=\"\" width=\"401\" height=\"277\" />\r\n<div>EscapeNameDialog</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> <b>Note</b>: If you have loaded data files, you may choose to use the loaded data files without displaying the data loading dialog.</li>\r\n<li><b>View the results:</b> After the analysis is done, three tabs are displayed in the table pane of Cytoscape: IPA Pathway Analysis, IPA Sample Analysis and IPA Node Values. IPA Pathway analysis displays inference results for entities in the pathway by comparing samples in your data files and in a random data set generated dynamically by the App based on your data files. IPA Sample Analysis shows results for each individual samples as up or down perturbation. You may choose to show/hide p-values and FDR values for samples in the table. IPA Node Values show inference results for selected entities in the pathway diagram for each sample. Entities in the pathway diagram are highlighted based on values in the MeanDiff column in the IPA Pathway Analysis tab.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/CellCycleCheckPointsResults.png\" alt=\"\" width=\"800\" height=\"452\" />\r\n<div>CellCycleCheckPointsResults</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> <b>Note</b>: You may change the color spectrum mapping for pathway diagram highlighting by double-clicking the color spectrum bar at the bottom of pathway diagram window to get the dialog for setting min/max values. You can save the analysis results for a pathway by using popup menu, \"Save Analysis Results\", and load the results back later on by \"Open Analysis Results\".</li>\r\n<li><b>Analyze gene level results:</b> The up or down perturbation results are inferred based on genomic data files for individual genes. The App provides features to analyze observation results and inference results for individual genes. You can view gene-level observation and inference results for the whole pathway by using popup menus, \"Show Gene Level Analysis Results\" and \"Show Observations\". You can also view these results for genes contained by an entity displayed in the pathway diagram after selecting that entity and then using these two popup menus. The following two dialogs show gene level observations and inference results for genes whose products are contained by complex \"hBUBR1:hBUB3:MAD2*:CDC20 complex [cytosol]\" in the cell cycle checkpoints pathway:\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ObservationsForEntity.png\" alt=\"\" width=\"799\" height=\"474\" />\r\n<div>ObservationsForEntity</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/GeneLevelResultsForEntity.png\" alt=\"\" width=\"799\" height=\"479\" />\r\n<div>GeneLevelResultsForEntity</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li><b>Analyze and visualize results for individual samples:</b> The inference results and loaded observation data are displayed in the right \"Results Panel\" (see below). By checking \"Highlight pathway for sample\", entities in the displayed pathway diagram will be highlighted based on inferred IPA values for the selected sample displayed in the \"Choose sample\" box. You can also enable animation by clicking the play button. There are two tabs in the \"Results Panel\": \"Inference\" for showing inferred IPA values, and \"Observation\" tab for loaded observed data related to entities in the pathway (Note: if you choose \"discretizing\", the displayed observation values are discretized: 0 for lower than normal, 1 for normal, and 2 for higher normal). Objects in three views (pathway diagram, inference table, and observation table) are synchronized for selection.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/PGMSampleViewWithInference.png\" alt=\"\" width=\"799\" height=\"321\" />\r\n<div>PGM Sample View: Inference</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/PGMSampleViewWithObservation.png\" alt=\"\" width=\"799\" height=\"320\" />\r\n<div>PGM Sample View: Observation</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li><b>Compare analysis results for two samples:</b> You can compare observation data and inference results for two samples. To do this, choose two samples in the \"IPA Sample Analysis\" tab in the bottom results pane, and use popup menu \"Compare Samples\" to bring out another tab called \"Sample Comparison\". You can view comparing results for inference and observation data.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/TwoSampleComparison.png\" alt=\"\" width=\"799\" height=\"649\" />\r\n<div>Two Sample Comparison</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n</ol></li>\r\n</ol>\r\n<h4><span id=\"Boolean_Network_based_Pathway_Analysis\">Boolean Network Based Pathway Analysis</span></h4>\r\n<p>We have developed an approach (Manuscript in preparation) to convert biochemical reactions-based Reactome pathways into Boolean networks and then perform pathway simulation based on the constrained fuzzy logic method according to <a href=\"http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1001099\" rel=\"nofollow\">Training Signaling Pathway Maps to Biochemical Data with Constrained Fuzzy Logic: Quantitative Analysis of Liver Cell Responses to Inflammatory Stimuli</a> and <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3292705/\" rel=\"nofollow\">Querying quantitative logic models (Q2LM) to study intracellular signaling networks and cell-cytokine interactions</a>. Based on this approach, the user can perform pathway simulation inside Cytoscape using rich Reactome pathways based on fuzzy logic built upon Boolean networks.</p>\r\n<ol>\r\n<li><b>Set up and run logic model simulation</b>: Choose popup menu \"Run Logic Model Analysis\" in the Pathway Diagram View to get the New Simulation dialog. Enter a name for the simulation and the default value, which usually should be 1.0 to enable that the simulation can proceed, and then choose either PROD or MIN for the AND gate mode (the default choice PROD usually should be fine). <br /> <b>Note</b>: You may also choose an Transfer Function and adjust parameters for Hill function. However, for simplicity, it is suggested to use \"Identity Function\" first. For how to apply drugs for logic model simulation, see below. <br />\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/RunBooleanNetworkAnalysis.png\" alt=\"\" width=\"288\" />\r\n<div>Run Boolean Network Analysis</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/NewBNSimulation.png\" alt=\"\" width=\"400\" />\r\n<div>New BN Simulation</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> After clicking the OK button in the New Simulation dialog, the default initial configuration will be displayed in the Results Panel. You may change the variable Type and Modification in the set up table by clicking the cell for the selected variable. To run the simulation, click the Simulate button in the Results Panel. <br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/SetupBNSimulation.png\" alt=\"\" width=\"400\" height=\"359\" />\r\n<div>Set up Boolean Network Analysis</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/ChooseBNType.png\" alt=\"\" width=\"400\" height=\"72\" />\r\n<div>Choose BN Variable Type</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/ChooseBNModification.png\" alt=\"\" width=\"400\" height=\"103\" />\r\n<div>Choose BN Modification Type</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li><b>Visualize the simulation results</b>: After the simulation is done, entities in the pathway diagram will be highlighted based on simulated values, which should be between 0 and 1. You may choose one or more entities to visualize their temporal behaviors inside the Table Panel at the bottom of Cytoscape. Attractors computed from the simulation are also listed in the right columns in the original set up table inside the Results Panel. <b>Note</b>: After simulation, you will not be able to modify any initial configuration.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/BNSimulationResults_Default.png\" alt=\"\" width=\"800\" height=\"532\" />\r\n<div>Boolean Network Simulation Results</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> <b>Note</b>: To avoid clutter, if too many time steps have been generated for a logic model simulation, only the last 20 time steps are displayed in the table. However, the plot shows all time steps. You may choose different columns to display in the table by use popup menu \"Configure Columns\" after selecting any variables in the table.</li>\r\n<li><b>Perform pathway simulation via modification</b>: Simulation with Boolean network can help users uncover the impact of modification of entity activities (e.g. inhibition or activation caused by somatic mutation) on the pathway behaviors. For example, in PIP3 activates AKT signaling, Complex AKT:PIP3 forms a complex with EntitySet THEM4/TRIB3 to form another complex, which inhibits the activation of AKT (For details see <a href=\"/PathwayBrowser/#/R-HSA-1257604\" rel=\"nofollow\">Reactome PIP3 activates AKT signaling</a>). To perform simulation with modification, choose modification type in the simulation set up table and assign the strength to the modification. <br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/SetBNInhibition.png\" alt=\"\" width=\"750\" height=\"253\" />\r\n<div>Choose Inhibition for Boolean Network Simulation</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br />Clicking the Simulate button invokes the constrained fuzzy logic model simulation with this configured inhibition. You can compare the simulation results between the two configurations by selecting an entity and then toggling the simulation result tables at the bottom of Cytoscape. <br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ActiveAKT_Inhibition.png\" alt=\"\" width=\"800\" height=\"136\" />\r\n<div>Active AKT in Inihibition</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ActiveAKT_Default.png\" alt=\"\" width=\"800\" height=\"136\" />\r\n<div>Active AKT in Default</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br />You can also use the \"Compare\" button in the Results Panel to get the comparison dialog and then choose two simulations for comparison. The comparison results are displayed in a new table listed at the bottom Table Panel. <br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/CompareTwoBNResults.png\" alt=\"\" width=\"450\" height=\"210\" />\r\n<div>Comparison Dialog</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/AktiveAKT_Comparison.png\" alt=\"\" width=\"850\" />\r\n<div>Active AKT in Comparison</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> <b>Note:</b> The RelativeDifference in the comparison result table is calculated based on relative change for each fuzzy logic variable, calculated as (valueInSim2 - valueInSim1) / (valueInSim2 + valueInSim1). The time course may be interpolated based on the attractor until this relative difference converges.</li>\r\n<li>To remove all displayed constrained fuzzy logic simulation results, use popup menu, \"Remove Analysis Results\" under \"Run Logic Model Analysis\". The pathway diagram should be reset to the original colors, and all tables related to logic model simulations will be deleted.</li>\r\n</ol>\r\n<h4><span id=\"Structural_Variants_Visualization\">Visualization of Structural Variants in the Context of Reactome Pathways</span></h4>\r\n<p>By collaborating with Drs. Francesco Raimondi and Rob Russell at the University of Heidelberg to utilize protein-protein interaction 3D structures provided by <a href=\"https://pubmed.ncbi.nlm.nih.gov/25392414/\">Mechismo</a>, a platform developed by <a href=\"http://www.russelllab.org\">Dr. Russell\'s group</a> to study the contributions of individual amino acid residues to protein structure and function, we have systematically analyzed mutations in the TCGA dataset and collected a set of reactions and functional interactions that involve proteins significantly enriched with mutations in their interaction interfaces (manuscript in preparation). We have added features to ReactomeFIViz to visualize these 3D structures and mutated residues in the contexts of Reactome pathways, reactions, and interactions.</p>\r\n<ol>\r\n<li><b>Visualize analysis results in the context of a pathway and its reactions</b>: In the opened pathway diagram view (See <a href=\"#Explore_Reactome_Pathways\">Explore Reactome Pathways</a> for how to open a pathway diagram), use the popup menu \"Load Mechismo Results\" to load the analysis results into the opened pathway diagram. After the results are loaded, reactions are colored based on FDR values as listed in the bottom table labeled as \"Mechismo Reaction\".\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/MechismoReactionView.png\" alt=\"Mechismo Reaction View\" width=\"1000\" />\r\n<div>Mechismo Reaction View</div>\r\n</div>\r\n<b>Note</b>: You may choose results from a different cancer type or pancancer by clicking the list at the top of the bottom tab labeled as \"Choose a cancer type to highlight reactions based on FDRs in the table\". Some of reactions are not highlighted by any color since there are no structural variants found for proteins annotated for these reactions. To remove the loaded results from the pathway diagram, use another popup menu \"Remove Mechismo Results\".</li>\r\n<li><b>Visualize analysis results in the context of a pathway FI network</b>: After the Mechismo results are loaded into the pathway diagram, you can convert the diagram into a FI network by using popup menu \"Convert to FI Network\" as usual. The edges displayed in the FI network view are highlighted based on FDR values listed in the bottom table labeled as \"Mechismo Interaction\".\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/MechismoInteractionView.png\" alt=\"Mechismo Interaction View\" width=\"1000\" />\r\n<div>Mechismo Interaction View</div>\r\n</div>\r\n<b>Note</b>: To make the FI network view simplier, check \"Show FIs Only for Selected\" at the left-bottom corner for the pathway diagram view. You may choose a reaction or complex and then view extracted FIs for the selected object in the pathway diagram view. Some of reactions and complexes may not contain any FI (e.g. a complex composed of a protein and a chemical or a reaction between a protein and a chemical).</li>\r\n<li><b>Visualize structural variants in the protein-protein 3D structures</b>: In the FI network view, choose the popup menu called \"Fetch Mechismo Results\" to bring the view for protein-protein interaction 3D models. Structural variants collected from the TCGA data set are mapped to the original protein-protein interaction 3D structures based on the <a href=\"https://pubmed.ncbi.nlm.nih.gov/25392414/\">Mechismo</a> platform.\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/MechismoStructureView.png\" alt=\"Mechismo Structure View\" width=\"900\" />\r\n<div>Mechismo Structure View</div>\r\n</div>\r\n<b>Note</b>: In the structure view, amino acid residues whose coordinates are mapped to structural variants are displayed in balls. The residues that are mapped to the selected rows in the bottom table are highlighted in yellow. ReactomeFIViz uses <a href=\"http://jmol.sourceforge.net\">Jmol</a> for protein 3D structure visualization. For more information about how to use Jmol, see its documentation: <a href=\"http://jmol.sourceforge.net/docs/\">http://jmol.sourceforge.net/docs/</a>.</li>\r\n</ol>\r\n<h3><span id=\"Use_the_Reactome_Functional_Interaction_.28FI.29_Network\">Use the Reactome Functional Interaction (FI) Network</span></h3>\r\n<p>After the ReactomeFIViz app installed, you should see a menu item called \"Reactome FI\" under the Apps menu. Clicking this menu, you will see 6 sub-menus: <a href=\"#Gene_Set.2FMutation_Analysis\">Gene Set/Mutation Analysis</a>, <a href=\"#PGM_Impact_Analysis\">PGM Impact Analysis</a>, <a href=\"#Microarray_Data_Analysis\">Microarray Data Analysis</a>, <a href=\"#Use_Reactome_Pathways\">Reactome Pathways</a> and <a href=\"userguide\">User Guide.</a> Gene set/mutation analysis is for doing FI network-based data analysis for a set of genes or a mutation data file, PGM Impact analysis for performing functional impact analysis based on a probabilistic graphical model for the Reactome FI network using multiple omics data types, HotNet mutation analysis for the HotNet algorithm to search for network modules (see <a href=\"http://compbio.cs.brown.edu/projects/hotnet/\" rel=\"nofollow\">http://compbio.cs.brown.edu/projects/hotnet/</a>), microarray data analysis for doing MCL (Markov Graph Clustering, <a href=\"http://micans.org/mcl/\" rel=\"nofollow\">http://micans.org/mcl/</a>) based FI network clustering analysis by converting a non-weighted FI network to weighted network using correlations among genes in the network, Reactome pathways for loading pathways from the Reactome database, visualizing Reactome pathways directly in Cytoscape in a their native way, and doing pathway enrichment analysis, and user guide brings you to this user guide.</p>\r\n<div class=\"text-center\">\r\n<div class=\"floatnone\"><img src=\"images/userguide/fiviz/ReactomeFIMenu_4.png\" alt=\"ReactomeFIViz app Menu\" width=\"441\" height=\"176\" /></div>\r\n</div>\r\n<h4><span id=\"Gene_Set.2FMutation_Analysis\">Gene Set/Mutation Analysis</span></h4>\r\n<ol>\r\n<li>You can enter a list of genes directly into ReactomeFIViz by clicking the \"Enter\" button, or load it from a local file. Currently ReactomeFIViz supports three file formats for gene set/mutation analysis:<ol>\r\n<li><b>Simple gene set</b>: one line per gene. For example, <a href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/GeneList/GWASFuzzyGenes.txt\" rel=\"nofollow\">GWASFuzzyGenes.txt</a>, a list of T2D GWAS genes.</li>\r\n<li><b>Gene/sample number pair</b>. For example, <a href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/GeneList/GeneSampleNumber.txt\" rel=\"nofollow\">GeneSampleNumber.txt</a>, which contains two required columns, gene and number of samples having gene mutated, and an optional third column listing sample names (delimited by \";\").</li>\r\n<li><b>NCI MAF (mutation annotation file)</b>. For example, <a href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/GeneList/GlioblastomaMutationTable.txt\" rel=\"nofollow\">GlioblastomaMutationTable.txt</a>, the mutation file from the TCGA GBM project.</li>\r\n</ol></li>\r\n<li>Choose a FI network version from listed three versions. <br /> <b>Note</b>: you may get different results using different FI network versions because a later version may contain more proteins/genes and more FIs. But based on our experience, a significant FI network module is usually stable across multiple versions.</li>\r\n<li>Enter genes directly by clicking the \"Enter\" button or choose a file containing genes you want to use to construct a functional interaction network. To choose a file, select an appropriate file format and parameters to load genes and construct FI network in the dialog. Click the \"OK\" button to start the FI network building process.\r\n<div class=\"text-center\">\r\n<div class=\"floatnone\"><img src=\"images/userguide/fiviz/GeneSetAnalysis_4.png\" alt=\"Gene Set/Mutation Analysis\" width=\"496\" /></div>\r\n</div>\r\n</li>\r\n<li>The constructed FI network will be displayed in the network view panel. A FI specific visual style will be created automatically for the FI network.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/FISubNetwork.png\" alt=\"\" width=\"470\" height=\"375\" />\r\n<div>Reactome FI Sub-Network</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li>The main features of Reactome FI plug-in should be invoked from a popup menu, which can be displayed by right clicking an empty space in the network view panel.\r\n<div class=\"text-center\">\r\n<div class=\"floatnone\"><img src=\"images/userguide/fiviz/PopupMenu_4.png\" alt=\"Popup Menu for Network\" width=\"526\" height=\"239\" /></div>\r\n</div>\r\n</li>\r\n</ol>\r\n<ul>\r\n<li><b>Fetch FI annotations</b>: query detailed information on selected FIs. Three FI related edge attribues will be created: FI Annotation, FI Direction, and FI Score. Edges will be displayed based on FI direction attribute values. In the following screenshot, \"-&gt;\" for activating/catalyzing, \"-|\" for inhibition, \"-\" for FIs extracted from complexes or inputs, and \"---\" for predicted FIs. See the \"VizMapper\" tab, Edge Source Arrow Shape and Edge Target Arrow Shape values for details.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/FIAnnotations.png\" alt=\"\" width=\"515\" height=\"508\" />\r\n<div>FI Annotations</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> <b>Note</b>: Here is a short explanation about displayed columns: GeneSet for pathways collected in the Reactome FI network hit by the query gene list; RatioOfProteinInGeneSet for ratios of numbers of genes contained in pathways to total genes in the Reactome FI network; NumberOfProteinInGeneSet for numbers of genes in pathways; ProteinFromNetwork for numbers of hit genes from the query gene list; P-value for pvalues calculated based on binomial test; FDR for FDRs calculated based on p-values using Benjamini-Hocherberg method; Nodes for hit genes in pathways.</li>\r\n<li><b>Analyze network functions</b>: pathway or GO term ennrichment analysis for the displayed network. You can choose to filter enrichment results by a FDR cutoff value. Also you can choose to display nodes in the network panel for a selected row or rows by checking \"Hide nodes in not selected rows\". The letter in parentheses after each pathway gene set name corresponds to the source of the pathway annotations: C - CellMap, R – Reactome, K – KEGG, N – NCI PID, P - Panther, and B – BioCarta. The following screenshot shows results from a pathway enrichment analysis.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/PathwayAnnotations.png\" alt=\"\" width=\"799\" height=\"251\" />\r\n<div>Pathways in FI Sub-Network</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> <i>Tip: To analyze pathway or GO term enrichment on a set of genes that are not linked together, select the \"Show genes not linked to others\" option in the \"Set Parameters for FI Network\" dialog.</i></li>\r\n<li><b>Cluster FI network</b>: run a network clustering algorithm (spectral partition based network clustering by <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/16723398\" rel=\"nofollow\">Newman 2006</a>) on the displayed FI network. Nodes in different network modules will be shown in different colors (different colors used only for first 15 modules based on sizes).\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/NetworkModules.png\" alt=\"\" width=\"679\" height=\"421\" />\r\n<div>Network Modules</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li><b>Analyze module functions</b>: pathway or GO term enrichment analysis for each individual network modules. You can select a size cutoff to filter out network modules that are too small, choose a FDR cutoff to view enriched pathways or GO terms under a certain FDR value, and view nodes in a selected row or rows only in the network diagram.</li>\r\n<li><b>Analyze functions for a set of selected genes</b>: select a set of nodes displayed in the network view and then choose the popup menu, Analyze Nodes Functions, to perform pathway or GO term enrichment analysis. The results will be displayed in a dialog.</li>\r\n<li><b>Load Cancer Gene Index</b>: load cancer gene index annotations. For details, see section <a href=\"#Load_Cancer_Gene_Index_Annotations\">Load Cancer Gene Index</a>.</li>\r\n</ul>\r\n<h4><span id=\"PGM_Impact_Analysis\">PGM Impact Analysis</span></h4>\r\n<ol>\r\n<li>We have developed a probabilistic graphical model (PGM)-based functional impact analysis using the Reactome FI network by integrating multiple omics data types together. The current version of ReactomeFIViz supports four omics data types: CNV, mRNA expression, DNA methylation, and somatic mutation. PGMs used for this analysis are based on <a href=\"https://en.wikipedia.org/wiki/Markov_random_field\" rel=\"nofollow\">Markov random field (MRF)</a>. Currently we support two types of MRFs: <a href=\"https://class.coursera.org/pgm/lecture/31\" rel=\"nofollow\">Pairwise MRF</a> and <a href=\"http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1001353\" rel=\"nofollow\">Nearest neighbor Gibbs MRF</a>. You can choose one of these two models. We recommend pair-wise MRF for its simplicity.</li>\r\n<li>To perform PGM-based functional impact analysis, choose menu, Apps/Rectome FI/PGM Impact Analysis/Analyze. You can enter your omics data by using the PGM configuration dialog.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ReactomeFIPGMInput.png\" alt=\"\" width=\"786\" />\r\n<div>FI-PGM Configuration Dialog</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> <b>Notes</b>:<br />1). ReactomeFIViz supports continuous observation variables without discretizing by choosing \"Use empirical distribution\" in the configuration. For somatic mutation, currently it supports NCI MAF file format only and requires a specific column named \"MA_FI.score\" for mutation function impact score collected from <a href=\"http://mutationassessor.org/\" rel=\"nofollow\">Mutation Assessor</a> or from some other sources. <br />2). The default MRF model used is PairwiseMRF. You can choose the default setting for the first test. <br />3). Several parameters are needed for MRF models. We have tuned these parameters based on a small toy model. In the current version of ReactomeFIViz, these parameters cannot be changed.</li>\r\n<li>It may take several hours to finish the whole analysis. The actual running time will be dependent on the size of your data. The progress of the job running is displayed in the following progress pane. You can cancel the running at any time.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ProgressPaneOfFIPGM.png\" alt=\"\" width=\"296\" height=\"197\" />\r\n<div>Progress of FI PGM Running</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li>After the analysis is finished, you should see the result dialog similar to the following screenshot. You can use filtering features to filter to a list of genes that you want to use to construct a FI subnetwork for further analysis.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/FIPGMResultDialog.png\" alt=\"\" width=\"799\" height=\"590\" />\r\n<div>FI-PGM Result Dialog</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> <b>Note</b>: If you select one or more genes in the result table, only these selected genes will be used to construct a FI sub-network. You can save the full analysis results by clicking the \"Save\" button (Results for all genes, not just displayed ones, will be saved). We strongly recommend to save your results first before clicking the \"OK\" button so that you can visit your results back.</li>\r\n<li>After choosing the \"OK\" button, a FI subnetwork will be constructed and displayed in a network view. The sizes of displayed nodes are proportional to impact scores inferred from the FI-PGM model. To view impact scores and loaded observation data, you can choose a sample in the Sample List tab in the Results Panel. You can also enable sample-based network visualization of the network by checking \"Highlight network for sample\" in the Sample List tab. If you want to review the original results used to construct the FI subnetwork, click \"Show All Results\" in the \"Impact Gene Values\" tab in \"Table Panel\".\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/FIPGMResultNetwork.png\" alt=\"\" width=\"799\" height=\"670\" />\r\n<div>FI-PGM Result Network</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n</ol>\r\n<h4><span id=\"Microarray_Data_Analysis\">Microarray Data Analysis</span></h4>\r\n<p>The ReactomeFIViz app can load gene expression data file, calculate correlations among genes involved in the same FIs, use the calculated correlations as weights for edges (i.e. FIs) in the whole FI network, apply MCL graph clustering algorithm to the weighted FI network, and generate a sub-network for a list of selected network modules based on module size and average correlation. The generated FI sub-network will be displayed in the network panel, and can be used for analysis as in Gene Set/Mutation Analysis. For details about this method, please see our publication: <a href=\"http://genomebiology.com/2012/13/12/R112\" rel=\"nofollow\">A network module-based method for identifying cancer prognostic signatures</a>.</p>\r\n<p>An array data file should be a tab-delimited text file with table headers. The first column should be gene names. All other columns should be expression values in different samples. <b>The data set in the file should be pre-normalized.</b> For example, see this gene expression file for breast cancer: <a href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/GeneExpression/NejmLogRatioNormGlobalZScore_070111.txt.zip\" rel=\"nofollow\">NejmLogRatioNormGlobalZScore_070111.txt.zip</a>. This data set was download from <a href=\"http://www.nejm.org/doi/full/10.1056/NEJMoa021967\" rel=\"nofollow\">van de Vijver et al in 2002</a>, and has been normalized.</p>\r\n<ol>\r\n<li><b>Select a microarray data file and run MCL network clustering</b>: After selecting sub-menu \"Microarray Data Analysis\" from menu Plugins/Reactome FIs, you should see the following dialog. Choose a microarray data file, check if you want to use absolute values as weights for edges, and input an inflation parameter (-I) for the MCL clustering algorithm. The smaller the inflation parameter is, the bigger the average size of generated network modules. Based on our own experience, we use 5.0 for the inflation parameter, the highest recommended value, and choose the absolute value for edge weights. For more details on how to choose the inflation parameter, please see <a href=\"http://micans.org/mcl/\" rel=\"nofollow\">http://micans.org/mcl/</a>. After you have set these parameters, click the OK button to load the data file, calculate correlations, and apply the MCL clustering algorithm.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/MicroarrayAnalysis_4.png\" alt=\"\" width=\"498\" />\r\n<div>Set Parameters for Microarray Data Analysis</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li><b>Select network modules and build a FI sub-network</b>: The generated network modules are listed in the MCL clustering results dialog (see below). Only modules having more than 2 genes can be listed, and used in the FI sub-network building. You can choose a module size or an average correlation value (absolute value if absolute has been checked before) to filter out modules that may not be significant (Note: after set these cutoff values, please press the \"Enter\" key to commit your changes.). In our analysis, we choose modules having 7 or more genes with average correlation values no less than 0.25. These values have been used as default in the dialog. In the dialog, you can see how many modules and genes will be chosen for building FI sub-network under your selected filter values. Click the OK button to start the sub-network building. The built sub-network will be displayed, and can be analyzed as with sub-networks generated from the gene set/mutation analysis.\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/MCLClusteringResultsDialog.png\" alt=\"\" width=\"705\" height=\"508\" />\r\n<div>Choose MCL Modules</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n</ol>\r\n<h3><span id=\"Visualize_Cancer_Drugs_in_Contexts_of_Reactome_Pathways_and_FI_Network\">Visualize Drugs in the Contexts of Reactome Pathways and FI Network</span></h3>\r\n<p>ReactomeFIViz provide a suite of features to assist users to visualize drugs in the contexts of Reactome pathways and networks. The drug data sources include two: Cancer Targetome (<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/28964549\">Blucher et al 2017</a>), which collected all FDA-approved cancer drugs (prior to 2018) and their target interactions from four sources, including DrugBank, Therapeutic Targets Database, IUPHAR, and BindingDB; <a href=\"http://drugcentral.org\">DrugCentral</a>, a comprehensive drug database supported by <a href=\"https://ncats.nih.gov/idg\">NIH IDG program</a>. </p>\r\n<h4><span id=\"Visualize_Cancer_Drugs_in_Reactome_Pathways\">Visualize Drugs in Reactome Pathways</span></h4>\r\n<ol>\r\n<li><b>List all FDA approved cancer drugs</b>: Use popup menu \"View Cancer Drugs\" in the Reactome pathway tree to get the list of all FDA approved cancer drugs collected in Cancer Targetome in a table dialog. <br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ViewCancerDrugs.png\" alt=\"ViewCancerDrugs\" width=\"400\" height=\"444\" />\r\n<div>View Cancer Drugs</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br />A screenshot of this drug list is shown below:<br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/CancerDrugsTable.png\" alt=\"\" width=\"800\" height=\"417\" />\r\n<div>List of Cancer Drugs</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li><b>View drug/target interactions</b>: Select a drug in the drug table (See Figure above) to perform Google search, or view its targets by clicking one of buttons in the dialog. Two views are provided for drug/target interactions. Drug targets table view (below) shows targets for the selected drug with collected binding affinities, which are divided into different categories (e.g.): KD, IC50, Ki, and EC50. The displayed targets are pre-filtered using a set of default filters. You may change the default filters in the Drug/Target Interaction Filter dialog by clicking the \"Filter\" button.<br />\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/DrugTargetsView.png\" alt=\"DrugTargetsTableView\" width=\"600\" height=\"300\" />\r\n<div>Drug Targets Table View</div>\r\n</div>\r\n<div></div>\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/DrugTargetInteractionFilter.png\" alt=\"\" width=\"350\" height=\"316\" />\r\n<div>Drug Target Interaction Filter</div>\r\n<div></div>\r\n</div>\r\n<div>In the Drug targets table view, you can choose an interaction and then click the \"View Details\" button to view the detailed information for the selected interaction. In the details view, you can browse the original pubmed references for experimental evidence, Gene Card entry for target, or Google drug by clicking the hyper links.\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/CancerDrugDetailsView.png\" alt=\"\" width=\"450\" height=\"377\" />\r\n<div>Cancer Drug Interaction Details View</div>\r\n</div>\r\n</div>\r\n<div></div>\r\n<div>You can also select multiple rows in the table to perform pathway enrichment analysis by clicking \"Overlay Targets to Pathways\". See <a href=\"#Pathway_Enrichment_Analysis\">Pathway Enrichment Analysis</a> for details about pathway enrichment analysis results.</div>\r\n<div></div>\r\n<div>Drug targets plot view (below) provides a stagged plot of bar charts for evidence-supported interactions between the selected drug and all its targets, suggesting primary target(s) and secondary targets in a single graphical view.</div>\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/DrugTargetsPlotView.png\" alt=\"DrugTargetsPlotView\" width=\"600\" />\r\n<div>Drug Targets Plot View</div>\r\n</div>\r\n<div></div>\r\n</li>\r\n<li>\r\n<div><b>View cancer drugs in pathway diagrams</b>: Cancer drugs can be overlaid directly onto displayed pathway diagrams using the popup menu, \"Fetch Cancer Drugs\", in the <a href=\"http://wiki.reactome.org/index.php/ReactomeFIViz#Explore_Reactome_Pathways\">Pathway Diagram View</a>. The fetched cancer drugs are displayed in the pathway diagram, linked to entities containing targets for the displayed drugs.</div>\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/FetchCancerDrugs.png\" alt=\"\" width=\"300\" />\r\n<div>Fetch Cancer Drugs</div>\r\n</div>\r\n<div></div>\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/DrugsInPathway.png\" alt=\"\" width=\"450\" />\r\n<div>Drugs in Pathway Diagram</div>\r\n</div>\r\n<div></div>\r\n<div><b>Note:</b> Use Popup menu \"Filter Drugs\" to adjust filters to select drugs and interactions in the pathway diagram view; Use \"Remove Overlaid Interactions\" to remove displayed drugs; You may choose a displayed entity in the pathway diagram and then use popup menu \"Fetch Cancer Drugs\" to display drugs targeted to the selected entity only. </div>\r\n<br />\r\n<div>To view the detailed information about the interaction between the displayed drug and its targeted entity, choose the link and use pupup menu \"Show Details\" to bring up the drug targets view. </div>\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/ViewDrugDetailsInPathway.png\" alt=\"\" width=\"400\" height=\"169\" />\r\n<div>View Details</div>\r\n</div>\r\n</li>\r\n<li>\r\n<div><b>Perform systems pathway impact analysis</b>: In the cancer drug list table (see above), you may perform pathway impact analysis for all Reactome pathways having entity level view drawn by choosing the \"Run Pathway Impact Analysis\" button. After the analysis is done, the impact results are displayed in the table labled with the selected drug in the Cytoscape Table Panel. See below for an example of the results for Imatinib:</div>\r\n<div class=\"text-center\"><img src=\"images/userguide/fiviz/PathwayImpactAnalysisTable.png\" alt=\"PathwayImpactAnalysisTable\" width=\"700\" />\r\n<div>Pathway Impact Analysis Results</div>\r\n</div>\r\n<div><b>Note</b>: You may open the pathway diagram using popup menu \"View in Diagram\" and export the table into a text file using menu \"Export Table\" in the results table.</div>\r\n</li>\r\n</ol>\r\n<h4><span id=\"Visualize_Cancer_Drugs_in_the_FI_Network\">Visualize Cancer Drugs in the FI Network</span></h4>\r\n<p>The Reactome FI network provides a network-based view among proteins/genes, where each gene/protein is displayed only once. Visualizing cancer drugs in a FI network context shows a simplified relationship between cancer drugs and their targets. In the <a href=\"#Gene_Set.2FMutation_Analysis\">FI Network View</a>, use popup menu, \"Reactome FI/Overlay Cancer Drugs/Fetch Cancer Drugs\", to load cancer drugs for proteins/genes displayed in the FI network. The loaded drugs and interactions between drugs and proteins/genes are rendered in green diamonds and blue edges, respectively.</p>\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/FetchCancerDrugsInFINetwork.png\" alt=\"\" width=\"400\" height=\"247\" />\r\n<div>Fetch Cancer Drugs in FI Network</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ShowDrugInteractionDetailsInFI.png\" alt=\"\" width=\"400\" height=\"265\" />\r\n<div>Show Drug Target In FI Network</div>\r\n</div>\r\n</div>\r\n</div>\r\n<p><br /> <b>Note</b>: To remove overlaid drugs in the FI network view, use popup menu, \"Reactome FI/Overlay Cancer Drugs/Remove Drugs\"; As in the Pathway diagram view, you can also apply filters by using \"Filte Drugs\" popup menu; To view the details about an interaction between a drug and a target, select the edge, and then use popup menu \"Reactome FI/Show Drug/Target Interaction Details\". (You may need to zoom into the selected edge.)</p>\r\n<h4><span id=\"Visualize_DrugCentral_Drugs\">Visualize DrugCentral Drugs</span></h4>\r\n<p>Visualize drugs collected in DrugCentral is simlar with cancer drugs collected in Cancer Targetome. You can use popup menu \"View DrugCentral Drugs\" in the pathway tree to list all drugs collected in the DrugCentral database, \"Fetch DrugCentral Drugs\" popup menu in the pathway diagram view to overlay drugs onto pathways, or \"ReactomeFI/Overlay Drugs/Fetch DrugCentral Drugs\" to overlay drugs onto the FI network in the network view.</p>\r\n<h4><span id=\"Simulate_Impact_of_Cancer_Drugs_on_Pathway_Activities\">Simulate Impact of Drugs on Pathway Activities</span></h4>\r\n<p>Overlaying cancer drugs onto the contexts of Reactome pathways and its FI network helps users to understand the potential impact of applying drugs on the pathway activities and network behavior. However, the actual perturbation of drugs on pathways may be much more complicated. Performing pathway simulation may help users to understand the actual impact. ReactomeFIViz implements features to assist users to perform Boolean network-based drug simulation. Before you do drug simulation, please read section <a href=\"#Boolean_Network_based_Pathway_Analysis\">Boolean Network Based Pathway Analysis</a> first. In this section, we will use pathway <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-5693567\" rel=\"nofollow\">HDR through Homologous Recombination (HR) or Single Strand Annealing (SSA)</a> and drug Imatinib as an example (Note: To get the following screenshot, open diagram for this pathway, and then fetch cancer drugs and filter drugs to Imatinib using the name filter).</p>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ImatinibInHDRThroughHROrSSA.png\" alt=\"\" width=\"600\" height=\"276\" />\r\n<div>Imatinib in Pathway</div>\r\n</div>\r\n</div>\r\n</div>\r\n<ol>\r\n<li><b>Set up new simulation for drug</b>: In the pathway diagram view, use popup menu, \"Run Logic Model Analysis\". In the New Simulation dialog, enter a name (e.g. Imtatinib) and default value for the simulation, and then choose the mode for the AND gate as in the regular logic model simulation. To perform drug simulation, choose the Drug Application tab and a drug data source, and then click the \"...\" button to bring up the Drug Selection dialog. <b><br />Note</b>: You will need to adjust the drug filters to show all targets for Imatinib as displayed in the following screenshot. Based on the collected annotations for interactions between cancer drugs and their targets, default modification types will be selected (as expected, most of them are inhibition). Strengths of modifications are pre-configured based on affinities collected in our aggregated drug/target database. <br />\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/ImatinibNewSimulation_1.png\" alt=\"\" width=\"400\" />\r\n<div>Imatinib BN Simulation I</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ImatinibNewSimulation_2.png\" alt=\"\" width=\"400\" />\r\n<div>Imatinib BN Simulation II</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/SelectDrugsForBN.png\" alt=\"\" width=\"750\" height=\"291\" />\r\n<div>Select Drugs for BN Simulation</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> <b>Note:</b> Since no affinities can be found for interactions between Imatinib and CHEK1 or CDK2, CHEK1 and CDK2 are not listed in the New Simulation dialog. Checking \"Filter members in sets to drug targets\" will select members in EntitySet instances that are targeted by drugs to force showing of potential pathway impact caused by drugs.</li>\r\n<li><b>Perform simulation with drug</b>: The configuration for drug in the Drug Selection dialog will be copied into the Boolean Network configuration table displayed in the Results Panel. Click the \"Simulate\" button to perform simulation. After the simulation is done, Entities in the pathway diagrams are highlighted in different colors based on values in the attractor with detailed temporal values displayed in the table at the bottom Table Panel. <br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ImatinibBNSetup.png\" alt=\"\" width=\"400\" height=\"339\" />\r\n<div>Imatinib BN Setup</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ImatinibBNResults.png\" alt=\"\" width=\"850\" height=\"569\" />\r\n<div>Imatinib BN Results</div>\r\n</div>\r\n</div>\r\n</div>\r\n</li>\r\n<li><b>Investigate the drug impact on pathway activities</b>: To see the impact of a drug on the pathway activities, perform another Boolean network simulation without applying cancer drugs (here as Default) (For details, see <a href=\"#Boolean_Network_based_Pathway_Analysis\">Boolean Network Based Pathway Analysis</a>). The screenshot for the logic model simulation results with the default initial configuration for pathway \"HDR through Homologous Recombination (HR) or Single Strand Annealing (SSA)\" is displayed below: <br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/DefaultHDRThroughHROrSSABNResults.png\" alt=\"\" width=\"850\" height=\"567\" />\r\n<div>HDR Through HR or SSA Default Results</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br />To see the drug impact to the activity of an entity displayed in the pathway, choose that entity and check its temporal behavior in both BN:Default and BN:Imatinib tables. For example, below a complex (see above screenshot) related to ABL1 is selected (Up for imatinib applied and down for default without drug): <br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ImatinibOneVariable.png\" alt=\"\" width=\"800\" height=\"165\" />\r\n<div>Imatinib One Variable</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/DefaultBNOneVariable.png\" alt=\"\" width=\"800\" height=\"171\" />\r\n<div>Default One Variable</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br />You may also use the \"Compare\" button to check the detailed difference in the computed attractors from two simulations. <br />\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/PS456ABL1Comparison.png\" alt=\"\" width=\"800\" height=\"203\" />\r\n<div>p-S456-ABL1 Values</div>\r\n</div>\r\n</div>\r\n</div>\r\n<br /> <b>Note</b>: From the above comparison, we can see that application of imatinib will significantly impact the formation of the complex in HDR, an effect of imanitib on DNA repair pathway has been reported by others (e.g. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/15840387\" rel=\"nofollow\">Imatinib (STI571) induces DNA damage in BCR/ABL-expressing leukemic cells but not in normal lymphocytes</a>). You may see a little bit different simulation results because of update in pathway annotations in new versions of ReactomeFIViz.</li>\r\n</ol>\r\n<h3><span id=\"scRNA_seq\">Perform scRNA-seq Data Analysis and Visualization</span></h3>\r\n<p>ReactomeFIViz implements a suite of features for users to conduct scRNA-seq data analysis and visualization. To do this, we have packaged several popuplar Python packages developed for scRNA-seq data analysis and visualization together into a Python standalone application. These packages include <a href=\"https://scanpy.readthedocs.io/en/stable/\">scanpy</a> for routine scRNA-seq data analysis and visualization and <a href=\"https://scvelo.readthedocs.io\">scVelo</a> for RNA velocity based data analysis and visualization. <br /><b>Note</b>: For scRNA-seq data analysis and visualization, you need to have Python 3.7 installed at your computer. If you have not installed Python at your computer, you can do so by downloading an installer from <a href=\"https://www.python.org/downloads/\">https://www.python.org/downloads</a> for your computer. We have tested Python 3.7 only and thefore suggest that you use 3.7 for these features. However, you don\'t need to install our standalone Python application indepedently from ReactomeFIViz. When needed, ReactomeFIViz will automatically download and update the application for you as long as you point to the correct Python application path (i.e. directory and application file).</p>\r\n<h4><span id=\"scanpy_analysis\">Standard Analysis via Scanpy</span></h4>\r\n<ol>\r\n<li><b>Set up the analysis:</b>The Python package, <a href=\"https://scanpy.readthedocs.io/en/stable/\">scanpy</a>, provides a set of powerful analysis and visualization features for scRNA-seq data. ReactomeFIViz wraps these features for users to take advantage of pathway and network analysis and visualization facilities provided by Cytoscape in general and ReactomeFIViz in particular. To conduct a scRNA-seq analysis using scanpy, choose menu Apps/Reactome FI/Single Cell Analysis/Analyze to get the configuration window as shown below:\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/scRNASeqConfig.png\" alt=\"scRNA-seq Analysis Configuration\" width=\"450\" /></div>\r\n<div>scRNA-seq Analysis Configuration</div>\r\n</div>\r\nReactomeFIViz supports scRNA-seq data generated from mouse and human. You should choose the species for your data and the format. If your data is in the 10x-Genomics-mtx format, you should choose the directory containing the files in that format. You may check an imputation method. Currently, ReactomeFIViz supports the <a href=\"https://github.com/KrishnaswamyLab/MAGIC\">MAGIC</a> approach only. You may also check total_counts and/or pct_counts_mt for <a href=\"https://scanpy.readthedocs.io/en/stable/api/scanpy.pp.regress_out.html?highlight=regressout#scanpy.pp.regress_out\">regress out</a> to control unwanted variations. <br /><b>Note</b>: All analysis steps and their paramters are logged into CytoscapeConfiguration/ReactomeFIViz/ReactomeFIViz.{date}.log in your user folder for your review.</li>\r\n<li><b>Configure Python for ReactomeFIViz:</b> If you have not done so, you will be asked to set up Python for ReactomeFIViz using the following configuration dialog when ReactomeFIViz downloads the Reactome Python app for scRNA-seq data analysis and visualization.\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/SetupPython.png\" alt=\"Set up Python\" width=\"450\" /></div>\r\n<div>Set up Python</div>\r\n</div>\r\n<b>Note</b>: Currently only Python 3.7 is supported.<br /> ReactomeFIViz uses the functions provided by scanpy for pre-processing, normalization, UMAP analysis, cell clustering and all other scRNA-seq analysis except imputation, which is handled by <a href=\"https://github.com/KrishnaswamyLab/MAGIC\">MAGIC</a>, if checked. See details in the scanpy document: <a href=\"https://scanpy.readthedocs.io/en/stable/api/index.html\">https://scanpy.readthedocs.io/en/stable/api/index.html</a>. For paramters used for these functions, open the ReactomeFIViz.{data}.log file (see above).</li>\r\n<li><b>Visuzlize Cell Networks</b>: Dependent on the sample size and the computing power, it may take several minutes to finish the analysis. After that, two networks, one for cell clusters and another for single cells, are displayed in Cytoscape and listed under \"SingleCellClusterNetwork\" and \"SingleCellNetwork\" in the left-side, Network tab, respectively.\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/ScClusterNetwork.png\" alt=\"ScRNA-seq Cluster Network\" width=\"800\" /></div>\r\n<div>ScRNA-seq Cluster Network</div>\r\n<div><img src=\"images/userguide/fiviz/ScCellNetwork.png\" alt=\"ScRNA-seq Cell Network\" width=\"800\" /></div>\r\n<div>ScRNA-seq Cell Network</div>\r\n</div>\r\n<b>Note</b>: You may use the built-in Cytoscape Style features and other configuration properties to adjust the rendering of these two networks. See Cytoscape\'s user manual by clicking menu Help/User Manual. To show or hide edges in the networks, use popup menu Reactome FI/Show Edges (see below for a screenshot). Cluster in the cell cluster network are named based on the rank of cell clusters sorted by cell numbers in the clusters. For example, cluster0 has the largest number of cells.</li>\r\n<li><b>Analyze scRNA-seq Data</b>: To explore the loaded scRNA-seq data and perform further analysis, you can use the popup menu provided in the cell cluster or single cell network view as shown below:\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/ScStandardPopupMenus.png\" alt=\"ScRNA-seq Standard Analysis Popup Menus\" width=\"600\" /></div>\r\n<div>ScRNA-seq Standard Analysis Popup Menus</div>\r\n</div>\r\n<ul>\r\n<li><b>Load Gene Expression</b>: Overlay expression value for a gene onto the network. You may enter the gene name from the input dialog after clicking this menu. <br /><b>Note</b>: Cell clusters use the median values of cells in the clusters for coloring for gene expression and cell features (below).</li>\r\n<li><b>Load Cell Features</b>: Overlay cell features by choosing a sub-menu, e.g., n_genes (total genes), n_genes_by_counts (total genes having counts), total_counts, total_count_mt (for mitonchorian genes), pct_counts_mt (percent of mitochondria genes), and leiden (network clustering results based on the <a href=\"https://arxiv.org/abs/1810.08473\">Leiden</a> algorithm).<br /><b>Note</b>: Both the cell cluster network and the single cell network are colored based on the leiden clustering results when they are rendered after the analysis. To get back to the orignal colors, choose Load Cell Feature/leiden.</li>\r\n<li><b>CytoTrace Analysis</b>: Perform CytoTrace analysis to predict the differential state of cells based on the number of detected expressed genes per cell. ReactomeFIViz provides a Python implementation of CytoTrace based on the original R code published in <a href=\"https://cytotrace.stanford.edu\">https://cytotrace.stanford.edu</a>. For details about CytoTrace, see the original paper: <a href=\"https://science.sciencemag.org/content/367/6476/405.long\">Single-cell transcriptional diversity is a hallmark of developmental potential</a>. You can find more information in the CytoTrace\'s web site: <a href=\"https://cytotrace.stanford.edu\">https://cytotrace.stanford.edu</a>. <br /><b>Note</b>: The analysis may take several minutes. After the analysis, cells will be colored based on predicted differential state values scaled between 0 and 1: 0 for most differentiated (yellow) and 1 for least differentiated (blue). The analysis results are cached in ReactomeFIViz and listed in a new column called \"cytotrace\" in the Node Table at the bottom. When you choose this menu again after the analysis, the results will be overlaid without performing another analysis. A new menu item called \"cytotrace\" will be added to the \"Load Cell Features\" popup menu too so that you can load these results directly. <br /><b>Note</b>: You may not see this menu item after the analysis. Try to switch to another network view and then come back to refresh the menu items.</li>\r\n<li><b>Diffusion Pseudotime Analysis (DPT)</b>: Perform cell trajectory inference based on network diffusion. For details about the algorithm, see <a href=\"https://scanpy.readthedocs.io/en/stable/api/scanpy.tl.dpt.html\">scanpy.tl.dpt</a> and the original paper: <a href=\"https://doi.org/10.1186/s13059-019-1663-x\">PAGA: graph abstraction reconciles clustering with trajectory inference through a topology preserving map of single cells</a>. To conduct this analysis, the id of a cell that should be regarded as the root of the trajectory is needed. If you have some idea what this cell is, you may enter it directly in the following dialog. If you don\'t know what it is but have some idea in which cluster or clusters the root may reside, you can enter the cluster(s) in the second text field. ReactomeFIViz will try to infer a possibe cell root for you in your specified cluster(s) based on <a href=\"https://en.wikipedia.org/wiki/PageRank\">PageRank</a>. For details about the cell root inference algorithm, see <a href=\"https://github.com/reactome-fi/fi_sc_analysis/blob/5cd26c200d6860e4a27df12fd1716709685bfe63/python/scpy4reactome/ScanpyWrapper.py#L455\">infer_cell_root</a>.\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/ConfigCellRootForDPT.png\" alt=\"Configure Cell Root for DPT\" width=\"400\" /></div>\r\n<div>Configure Cell Root for DPT</div>\r\n</div>\r\n<b>Note</b>: If you don\'t have any idea what the cell root is, you may try several approaches. If you have conducted a CytoTrace analysis, you may choose the cell having the largest CytoTrace value as the cell root. You may also try the cell having the largest number of detected genes as the root for exploration data analysis. To choose cell clusters for inferring the cell root, you should choose clusters having the largest CytoTrace values or gene numbers. If you have enter values into both text fields in the above dialog, the value in the first text field will be used as the cell root.<br /> After the DPT analysis, cells will be colored based on DPT values ranged from 0 to 1 with 0 as the earliest cell (yellow) in the trajectory and 1 as the latest (blue). A new column called \"dpt_pseudotime\" will be added into the Node Table at the bottom and \"dpt_pseudotime\" will be registered as a new item under \"Load Cell Feature\" popup menu for loading without repeating the analysis. <br /> <b>Note</b>: You may not see this menu item after the analysis. Try to switch to another network view and then come back to refresh the menu items.</li>\r\n<li><b>Differential Expression Analysis</b>: Perform differential gene expression analysis between a cell cluster (group) and another cell cluster or all other cell clusters using <a href=\"https://scanpy.readthedocs.io/en/stable/api/scanpy.tl.rank_genes_groups.html\">t-test_overestim_var</a>. To conduct this analysis, you need to choose two groups of cells first: The group of cell for analysis based on clustering results and another group as the reference. You may choose another cell cluster or all other cells as the reference.\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/ScDiffChooseGroups.png\" alt=\"Choose Cell Groups for Differential Expression Analysis\" width=\"400\" /></div>\r\n<div>Choose Cell Groups for Differentila Expression Analysis</div>\r\n</div>\r\nThe differential expression analysis result is displayed in the following table. You may choose one or more filteres by clicking the \"Add\" button to filter genes displayed in the table. To create a FI network for the filtered genes displayed in the table, click the \"Build FI Network\" button. To conduct a pathway enrichment analysis, you can choose Binomial_test or <a href=\"#Gene_Set.2FMutation_Analysis\">GSEA</a>. The Binomial_test will use the filtered, displayed genes in the table while the GSEA analysis will use gene rank by the score, including genes that are not displayed in the table.\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/ScDiffExpResultTable.png\" alt=\"Differential Expression Analysis Result\" width=\"800\" /></div>\r\n<div>Differential Expression Analysis Result</div>\r\n</div>\r\n<b>Note</b>: Genes in the FI network constructed from the selected genes are colored based on gene scores. For more information on how to use the features for the FI network, see <a href=\"#Gene_Set.2FMutation_Analysis\">Gene Set/Mutation Analysis</a>. You may open a pathway diagram when a network view is shown. To get back to the previous network view, close all displayed pathway diagrams and then select the network in the Network tab in the left, control panel. Reactome mouse pathways are predicted from human pathways based on the panther orthologous mapping file. For details, see: <a href=\"https://reactome.org/documentation/inferred-events\">Inferred Events in Reactome</a>. The mouse human functional interaction network is predicted from the human functional interaction network using the mapping file provided by <a href=\"http://www.informatics.jax.org\">MGI</a>, downloaded using this link: <a href=\"http://www.informatics.jax.org/downloads/reports/HOM_MouseHumanSequence.rpt\">http://www.informatics.jax.org/downloads/reports/HOM_MouseHumanSequence.rpt</a>. One human gene may be mapped to multiple mouse genes. Therefore, the mouse FI network may show a node that is annotated with multiple mouse genes. For example, see below: The original human KLK3 is mapped to Klk1b9, Klk1b21, and many others, which are all listed in the node table at the bottom and in the network view as the label for that node. Currently links to these nodes in the FI network point to human genes (e.g. GeneCard).\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/OneHumanGeneToMultipleMouseGenes.png\" alt=\"One Human Gene Mapped to Multiple Mouse Genes\" width=\"1000\" /></div>\r\n<div>One Human Gene Mapped to Multiple Mouse Genes</div>\r\n</div>\r\n</li>\r\n<li><b>Build Regulatory Network</b>: Infer an underlying gene regulatory network between transcriptional factors (TFs) and their targets for one or more cell clusters. This approach is inspirted by Qiu et al\'s <a href=\"https://www.sciencedirect.com/science/article/abs/pii/S2405471220300363\">Inferring Causal Gene Regulatory Networks from Coupled Single-Cell Expression Dynamics Using Scribe</a>. However, the current implementation provided by ReactomeFIViz uses time-delayed gene co-expression to infer potential causal relationships between TFs and their targers instead of \"restricted directed information (RDI)\" and limits the causal relationships search between TFs and their targets based on TF/target interactions provided by <a href=\"https://saezlab.github.io/dorothea/\">dorothea</a>. To perform this analysis, set up parameters in the following dialog:\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/ScRegNetSetup.png\" alt=\"Gene Regulatory Network Infernece Setup\" width=\"400\" /></div>\r\n<div>Gene Regulatory Network Inference Setup</div>\r\n</div>\r\n<b>Note</b>: You may choose Spearman, Pearson, or Kendal for gene expression correlaiton calculation. The cell time type is one of latent_time, velocity_pseudotime, cytotrace or dpt_pseudotime, which are cell properties calculated during trajectory inference. The first two types are generated during an RNA velocity analysis (See below). If ReactomeFIViz cannot find any of these variables, you will be asked to perform the dpt_pseudotime analysis first. The time delay is used to conduct a delayed gene co-expression calculation. For example, the expression of a TF is [1, 2, 3, 4, 5, 6, 7, 8, 9, 10] and the expresison of one of its target is [11, 12, 13, 14, 15, 16, 17, 18, 19, 20]. If the time delay = 4, the correlation between [1, 2, 3, 4, 5, 6] and [15, 16, 17, 18, 19, 20] is calculated. The time delay in the above dialog is for the maximum time delay. Therefore, if you enter 7 for the time delay, the actual time delay values used for correlation calculation are [1, 2, 3, 4, 5, 6, 7]. The correlation is calculated 7 times and the maximum value of those calculated correlation values is used. For the cell groups, you may choose all, one of cell clusters, or multiple cell clusters (hold the command key under Mac or the control key under Windows). <br /> It may take several minutes to calculate the correlation for all TF/target interactions and then build the regulatory network. The following is an example showing how a final gene regulatory network looks like:\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/PartialGeneRegNet.png\" alt=\"Partial Gene Regulatory Network\" width=\"1000\" /></div>\r\n<div>Partial Gene Regulatory Network</div>\r\n</div>\r\n<b>Note</b>: A style called \"Regulatory Network Style\" is created for rendering the generated gene regulatory network. With this style, TFs are rendered as diamonds while their targets as circles. The original <a href=\"https://saezlab.github.io/dorothea/\">dorothea</a> interactions provide annotation: -&gt; for activation and -| for inhibition. Red for positive correlation while blue for negative correlaiton. The calculated correlation may not match the original annotation in dorothea. For example, in the above figure, the interaction between Insm1, a TF and one of its target, Scg3, is annotated as an inhibition. However, the actual calculated Spearman correlation is positive. The edge width is proportional to the absolute correlation value. You may investigate this style inside Cytoscape for more information.</li>\r\n<li><b>Project New Data</b>: Project another data set onto the displayed network. This feature uses the <a href=\"https://scanpy.readthedocs.io/en/stable/api/scanpy.tl.ingest.html\">ingest</a> function in scanpy to integrate a new dataset onto the dataset used to generate the network, showing where cells in the new dataset can be mapped to the displayed network. To perform this analysis, enter the data file in the following dialog:\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/ProjectNewDataConfig.png\" alt=\"Project New Data Configuration\" width=\"400\" /></div>\r\n<div>Project New Data Configuration</div>\r\n</div>\r\n<b>Note</b>: You cannot choose the approach in this dialog. It is expected that the same approach should be used to project a new dataset onto the existing network. After projecting, a new popup menu called \"Toggle Projected Data\" is added under the \"Project New Data\" menu for toggling the display of the projected cells. The following two figures show the cell network before (left) and after (right) a new dataset is projected. As shown, the majority of cells in the new dataset are projected onto the cell clusters displayed in green, salmon and rosy brown around the middle and top-right regions.\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/CellNetworkBeforeProject.png\" alt=\"Before Projecting\" width=\"500\" /><img src=\"images/userguide/fiviz/CellNetworkAfterProject.png\" alt=\"After Projecting\" width=\"500\" /></div>\r\n</div>\r\n</li>\r\n<li><b>Save Analysis Results</b>: All analysis results may be saved into a local file in the <a href=\"https://github.com/theislab/anndata/issues/180\">h5ad</a> format. The saved file can be opened using the main menu, Apps/Reactome FI/Single Cell Analysis/Open.</li>\r\n</ul>\r\n</li>\r\n</ol>\r\n<h4><span id=\"scvelo_analysis\">RNA Velocity Analysis via scVelo</span></h4>\r\n<p>RNA velocity analysis is a powerful approach to quantitatively model dynamic behavior of mRNA transcription of genes based on count ratios between unspliced and spliced forms of mRNAs (<a href=\"https://www.nature.com/articles/s41586-018-0414-6\">La Manno et al 2018</a>). <a href=\"https://doi.org/10.1038/s41587-020-0591-3\">scVelo</a> provides an enhanced implementation of the original approach in Python. ReactomeFIViz packages scVelo for you to conduct this analysis in Cytoscape using graphic user interfaces without scripting. For more information about RNA velocity and scVelo, see the original scVelo document: <a href=\"https://scvelo.readthedocs.io\">https://scvelo.readthedocs.io</a>.</p>\r\n<ol>\r\n<li><b>Set up the analysis</b>: Select \"RNA Velocity Analysis via scVelo\" for Approach in the following scRNA-seq analysis action dialog and choose one of RNA velocity modes. To see the differences among the three modes listed in dialog, see the original scVelo paper, <a href=\"https://doi.org/10.1038/s41587-020-0591-3\">Generalizing RNA velocity to transient cell states through dynamical modeling</a>. The default is \"stochastic\". However, if you want to get more dynamic information, choose \"dynamical\". For what you can do with the dynamical model, see <a href=\"https://scvelo.readthedocs.io/DynamicalModeling.html\">Dynamical Modeling with scVelo</a>. The data file required by this analysis should be pre-processed specically by using velocyto or loompy/kallisto pipeline and contain two matrices for unspliced and spliced abundances. For more information on how to get started, see this scVelo tutorial: <a href=\"https://scvelo.readthedocs.io/getting_started.html\">Getting Started</a>.\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/RNAVelocitySetup.png\" alt=\"RNA Velocity Analysis Configuration\" width=\"400\" /></div>\r\n<div>RNA Velocity Analysis Configuration</div>\r\n</div>\r\n</li>\r\n<li><b>Visualize the RNA velocity analysis results</b>: Dependent on the mode you choose, it may take a while to conduct the RNA velocity analysis. The outputs are the same as ones from a standard analysis using scanpy except that the single cell cluster network is displayed as directed, weighted network with directions corresponding to times in the inferred trajectory based on the <a href=\"https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1663-x\">PAGA</a> approach and weights for the connectivities between cell clusters. The following figure shows an example of such a weighted, directed cell cluster network.\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/RNAVelocityNetwork.png\" alt=\"RNA Velocity Cluster Network\" width=\"500\" /></div>\r\n<div>RNA Velocity Cluster Network</div>\r\n</div>\r\n<b>Note</b>: It is expected that you see different results from the RNA velocity analysis than ones from the standard analysis.</li>\r\n<li><b>Analyze the RNA velocity results</b>: Most of analysis features for the displayed networks generated from the RNA velocity analysis are the same as ones from the standard analysis via scanpy. However, the RNA velocity analysis provides much more cell features than the standard analysis as shown in the following popup menu:\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/RNAVelocityCellFeatures.png\" alt=\"RNA Velocity Cell Features\" width=\"600\" /></div>\r\n<div>RNA Velocity Cell Features Popup Menu</div>\r\n</div>\r\n<b>Note</b>: To understand the meanings of these RNA velocity specific cell features, please refer to the original scVelo tutorials: <a href=\"https://scvelo.readthedocs.io/getting_started.html\">scVelo tutorials</a>. In addition to the above RNA velocity specific cell features, a new popup menu group is added for you to conduct some RNA velocity specific data analysis and visualization as shown below:\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/RNAVelocityPopupMenu.png\" alt=\"RNA Velocity Popup Menu\" width=\"500\" /></div>\r\n<div>RNA Velocity Popup Menu</div>\r\n</div>\r\n<b>Note</b>: Refer to this scVelo tutorial for Embedding, Embedding Grid, Embedding Stream, and Gene Velocity: <a href=\"https://scvelo.readthedocs.io/VelocityBasics.html\">RNA Velocity Basics</a>. ReactomeFIViz utilizes scVelo\'s visualization features to generate image files for these plots and then automatically open them. To keep these files for your record, you may have to save them into your designated files. Otherwise, they will be automatically deleted when you close Cytoscape.\r\n<ul>\r\n<li><b>Rank Velocity Genes</b>: Ranks genes in individual cell clusters based on differential expression analysis using scVelo\'s rank_velocity_genes function: <a href=\"https://scvelo.readthedocs.io/scvelo.tl.rank_velocity_genes.html#scvelo.tl.rank_velocity_genes\"> rank_velocity_genes</a>. Top 250 genes for individual clusters returned from this analysis are displayed in a table as shown in the following figure. You may conduct pathway enrichment analysis using a binomial test or build a FI network for a selected cell cluster.\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/RNAVelocityRankGeneTable.png\" alt=\"RNA Velocity Rank Gene Table\" width=\"1000\" /></div>\r\n<div>RNA Velocity Rank Gene Table</div>\r\n</div>\r\n<b>Note</b>: You may filter genes displayed in the table. However, for pathway enrichment analysis or building a FI network, all 250 genes for a selected cell cluster are used.</li>\r\n<li><b>Rank Dynamic Genes</b>: If you choose the dynamic mode for your RNA velocity analysis, you can also do \"Rank Dynamic Genes\". This feature is based on scVelo function, <a href=\"https://scvelo.readthedocs.io/scvelo.tl.rank_dynamical_genes.html#scvelo.tl.rank_dynamical_genes\">rank_dynamical_genes</a>. The output and the functions are the same as \"Rank Velocity Genes\".</li>\r\n</ul>\r\n</li>\r\n</ol>\r\n<h3><span id=\"Other_Features_Related_to_the_FI_Network\">Other Features Related to the FI Network</span></h3>\r\n<h4><span id=\"Query_FI_Source\">Query FI Source</span></h4>\r\n<p>Select an edge and right click it to get the popup menu for edge. Select a menu called \"Reactome FI/Query FI Source\". If a FI is extracted from curated pathways or reactions, a dialog for the original data source(s) will be displayed. Double click a row in the displayed table to show a detailed web page for the source of the FI. If the selected FI is a predicted one, the evidence for this FI should be displayed.</p>\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/QueryFISource.png\" alt=\"\" width=\"400\" height=\"150\" />\r\n<div>Query FI Source</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ShowFISource.png\" alt=\"\" width=\"350\" height=\"177\" />\r\n<div>ReactomeFIViz app Menu</div>\r\n</div>\r\n</div>\r\n</div>\r\n<p> </p>\r\n<h4><span id=\"Fetch_FIs_for_Node\">Fetch FIs for Node</span></h4>\r\n<p>All FIs for a node can be queried. Select a node in the network panel, and right click it to get the popup menu for node. Select a menu called \"Reactome FI/Fetch FIs\". FI partners for the selected node will be displayed in two sections: partners have been displayed in the network and partners not displayed in the network. You can select partners from the second sections to expand the displayed network.</p>\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/FetchFIs.png\" alt=\"\" width=\"350\" height=\"238\" />\r\n<div>Query Node FIs</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ShowNodeFIs.png\" alt=\"\" width=\"398\" height=\"276\" />\r\n<div>Show Node FIs</div>\r\n</div>\r\n</div>\r\n</div>\r\n<p> </p>\r\n<h4><span id=\"Show_Pathway_Diagram\">Show Pathway Diagram</span></h4>\r\n<p>Pathway diagrams can be shown for pathway hits. Select a pathway in the \"Pathways in Network\" or \"Pathways in Modules\" tab, and right click to get the popup menu for pathway. Select \"Show Pathway Diagram\" from the popup menu</p>\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ShowPathwayDiagram.png\" alt=\"\" width=\"340\" height=\"102\" />\r\n<div>Show Pathway Diagram</div>\r\n</div>\r\n</div>\r\n<p>. If pathways are imported from KEGG, KEGG pathway diagram pages will be shown in a browser with node genes listed in the \"Nodes\" column highlighted in red (for text and borders in pathway diagrams). If pathways are from Reactome or other non-KEGG databases, pathway diagrams should be shown in a separated window. If pathways are curated by the Reactome project, human laid-out diagrams should be displayed if any. Otherwise, auto-laid-out diagrams should be displayed. Genes or proteins from the displayed network should be highlighted in blue. Detailed annotations for nodes and reactions displayed in the diagram window can be viewed by using a popup menu called \"View Instance\". Diagrams displayed can be zoomed in/out using the zoom slider at the bottom of the window. The diagram can be panned by the overview window at the top-right corner.</p>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/KEGGDiagram.png\" alt=\"\" width=\"610\" height=\"377\" />\r\n<div>KEGG Focal Adhesion</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/ReactomeDiagram.png\" alt=\"\" width=\"610\" height=\"422\" />\r\n<div>Reactome Signaling by PDGF</div>\r\n</div>\r\n</div>\r\n</div>\r\n<p> </p>\r\n<h4><span id=\"Load_Cancer_Gene_Index_Annotations\">Load Cancer Gene Index Annotations</span></h4>\r\n<p>Reactome FI plug-in can load NCI cancer <a href=\"https://wiki.nci.nih.gov/display/cageneindex/Cancer+Gene+Index+End+User+Documentation\" rel=\"nofollow\">gene index annotations</a> for genes/proteins displayed in the network. There are two ways to show these annotations: use a popup menu called \"Load Cancer Gene Index\" when no object is selected (left figure), and use another popup menu \"Fetch Cancer Gene Index\" for a selected node (right figure).</p>\r\n<p> </p>\r\n<div class=\"text-center\">\r\n<div><img src=\"images/userguide/fiviz/LoadCGI.png\" alt=\"\" width=\"200\" height=\"126\" />\r\n<div>Load Gene Index</div>\r\n</div>\r\n</div>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/LoadNodeCGI.png\" alt=\"\" width=\"400\" height=\"227\" />\r\n<div>Load Node Cancer Gene Index</div>\r\n</div>\r\n</div>\r\n</div>\r\n<p><br />By using the first method, the user can load the tree of NCI disease terms and display the tree in the left panel. The user can select disease term in the tree, all genes or proteins have been annotated for the selected disease and its sub-terms will be selected.</p>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/CGIOverlay.png\" alt=\"\" width=\"700\" height=\"527\" />\r\n<div>Cancer Gene Index Overlay</div>\r\n</div>\r\n</div>\r\n</div>\r\n<p> </p>\r\n<p>By using the second method, the user can view detailed annotations for the selected gene or protein. The user can sort these annotations based on PubMedID, Cancer type, and annotation status, and also filter annotations based on several criteria.</p>\r\n<p> </p>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/CGIAnnotationsForNode.png\" alt=\"\" width=\"700\" height=\"541\" />\r\n<div>Cancer Gene Index Annotations for Node</div>\r\n</div>\r\n</div>\r\n</div>\r\n<p> </p>\r\n<h4><span id=\"Survival_Analysis\">Survival Analysis</span></h4>\r\n<p>Survival analysis is based on a server-side R script to do either coxph or Kaplan-Meier survival analysis. To do survival analysis, a tab-delimited text file containing at least three columns should be provided. The names of three columns should be: Samples, OSDURATION, and OSEVENT. For example, see this survival information file downloaded from <a href=\"http://www.nejm.org/doi/full/10.1056/NEJMoa021967\" rel=\"nofollow\">van de Vijver et al in 2002</a>: <a href=\"https://github.com/reactome-fi/CytoscapePlugIn/raw/master/example_data/GeneExpression/Nejm_Clin_Simple.txt\" rel=\"nofollow\">Nejm_Clin_Simple.txt</a>, which has been simplified for our analysis purpose. To do survival analysis, use the popup menu \"Analyze Module Functions/Survival Analysis...\" (see below)</p>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/SurvivalAnalysisMenu.png\" alt=\"\" width=\"464\" height=\"205\" />\r\n<div>Survival Analysis Menu</div>\r\n</div>\r\n</div>\r\n</div>\r\n<p> </p>\r\n<p>In the survival analysis dialog (below), double click the text field to select a file containing survival information for samples used to build the displayed FI sub-network (Note: you cannot do survival analysis if you use a gene set file only to construct the displayed FI subnetweork). You can choose either coxph or Kaplan-Meier model to do survival analysis. If you choose the Kaplan-Meier model, you have to select a module for analysis. In the Kaplan-Meier analysis, all samples will be divided into two groups: samples having no mutated genes in the selected module (group 1) and samples having mutated genes in module (group 2). It is recommended to run the coxph module first without selecting any module in order to see which module is most significantly related to survival times. After that, you can focus on some specific modules for survival analysis.</p>\r\n<p> </p>\r\n<div class=\"text-center\">\r\n<div>\r\n<div><img src=\"images/userguide/fiviz/SurvivalAnalysisDialog.png\" alt=\"\" width=\"605\" height=\"332\" />\r\n<div>Survival Analysis Dialog</div>\r\n</div>\r\n</div>\r\n</div>\r\n<p> </p>\r\n<p>The results from survival analysis will be displayed in the right Results Panel with a tab labeled \"Survival Analysis\" (below left). You can do multiple survival analyses. All results returned from the server-side R script will be displayed in this panel with labels based on your parameter selections in the survival analysis dialog. The last result will be selected as default. At most three sections are displayed in the result panel for each analysis: Output, Error, and Plot. If no warning or error returned from an analysis, the error section may not be shown. Rows for modules having p-values less than 0.05 from coxph (all modules) analysis are displayed in blue with text underlined. You can click these modules to do a quick single-module based survival analysis without going through the above steps. Single module-based Kaplan-Meier analysis will show a plot file. You can click the file to view the actual plot (below right). You may need to save the plot file for your future use.</p>\r\n<div class=\"text-center\">\r\n<p><img src=\"images/userguide/fiviz/SurvivalAnalysisResult.png\" alt=\"Survival Analysis Results\" width=\"418\" height=\"812\" /></p>\r\n<p> </p>\r\n<p><img src=\"images/userguide/fiviz/KaplanMeyerPlot.png\" alt=\"Kaplan-Meier Survival Plot\" width=\"500\" height=\"500\" /></p>\r\n</div>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-25 16:02:02','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,13,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(45,605,'Graph Database','graph-database','<h4>The Reactome Graph Database models the Reactome knowledgebase as an interconnected graph database.</h4>\r\n<p>{loadposition dev-guide-gd-1}</p>\r\n<div style=\"text-align: justify;\">\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p>At the cellular level, life is a network of molecular reactions. In Reactome, these processes are systematically described in molecular detail to generate an ordered network of molecular transformations <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/26656494\">(Fabregat et al. 2015)</a>. This amounts to millions of interconnected terms naturally forming a <a href=\"http://en.wikipedia.org/wiki/Graph_(abstract_data_type)\">graph</a> of biological knowledge. The Reactome Graph provides an intuitive way for data retrieval as well as interpretation and analysis of pathway knowledge.</p>\r\n<p>Retrieving, and especially analysing such complex data becomes tedious when using relational databases. Queries across the pathway knowledgebase are composed by a number of expensive join operations resulting in poor performance and a hard-to-maintain project. Due to the schema-based approach, relational databases are limited in how information is stored and thus are difficult to scale for new requirements. In order to overcome these problems the Reactome database is imported in <a href=\"http://neo4j.com/\" target=\"_blank\" rel=\"noopener\">Neo4j</a>, creating one large interconnected graph. Graph database technology is an effective tool for modelling highly connected data.</p>\r\n<p>Storing Reactome data in this form has many benefits. No denormalisation is required so data can be stored in its natural form. Nodes in the vicinity of a starting point can quickly be traversed giving the user the possibility to not only retrieve data but also perform fast analysis of these neighbour networks. Thus, knowledge that previously was unavailable due to the limitations of relational data storage can now be retrieved.</p>\r\n<p>To easily access and benefit from the graph database, we have developed the GraphCore; an open source library implemented in Java. This project uses <a href=\"http://projects.spring.io/spring-data-neo4j/\" target=\"_blank\" rel=\"noopener\">Spring Data Neo4j</a>, which provides an automatic object graph mapping on top of Neo4j and tightly integrates with other parts of the spring framework used across the project.</p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p class=\"text-center\"><a href=\"/download/current/reactome.graphdb.tgz\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"favth-btn\"><i class=\"fa fa-download\"></i>Download</a></p>\r\n<div class=\"alert alert-info\"><span>To run our graph database on your personal computer, please choose the installation option that suits your need. Our team recommends <a href=\"/dev/graph-database/neo4j-desktop\">Neo4j Desktop</a>, but we can\'t compete with developers and their love for terminal console.</span></div>\r\n<div></div>\r\n<div>\r\n<h4>1. Docker</h4>\r\n<p>If you are comfortable working with docker,  you can build a <a href=\"https://hub.docker.com/r/reactome/graphdb/tags\">docker image</a> that contains Neo4j and a Reactome graph database</p>\r\n<p>You can use the Neo4j graph database: </p>\r\n<ol>\r\n<li aria-level=\"1\">Download <a href=\"https://www.docker.com/products/docker-desktop/\">Docker</a> on your desktop. </li>\r\n<li aria-level=\"1\">Pull a <a href=\"https://hub.docker.com/r/reactome/graphdb\">docker image</a> that contains Neo4j and a Reactome graph database. </li>\r\n<li aria-level=\"1\">Run the docker image:  </li>\r\n</ol>\r\n<p><code><span style=\"font-family: courier new, courier, monospace;\">docker run -p 7474:7474 -p 7687:7687 -e NEO4J_dbms_memory_heap_maxSize=8g reactome/graphdb:${VERSION}</span></code></p>\r\n<ol start=\"3\">\r\n<li aria-level=\"1\">Go to localhost:7474 in your browser. </li>\r\n</ol>\r\n<p>You now have a docker image containing Neo4j and the Reactome graph database. You can create custom queries using Cypher and submit your own queries. Please refer to our <a href=\"https://reactome.org/dev/graph-database/extract-participating-molecules\">extracting pathway participating molecules</a> tutorial to introduce yourself to using Cypher to query the Reactome Graph Database.</p>\r\n</div>\r\n<h4>2. Neo4j Desktop</h4>\r\n<p style=\"padding-left: 30px;\">If you would like to use Neo4j Desktop, we have created a dedicated page for it. Please see the instructions <a href=\"/dev/graph-database/neo4j-desktop\">here</a>.</p>\r\n<h4>3. Neo4j Community manual installation</h4>\r\n<p>If you have trouble using the Neo4j 3.5.X with our data, we strongly recommend upgrading your neo4j version to 4.X.X and following the instructions above.</p>\r\n<p>Our Graph Database is available in our <a href=\"download-data\">download data section</a>. It is possible to use it in your local environment by following these steps:</p>\r\n<ol>\r\n<li aria-level=\"1\">Download and install the <a href=\"https://neo4j.com/download/other-releases/\">Neo4j V4</a>.<ol style=\"list-style-type: lower-alpha;\">\r\n<li aria-level=\"2\">Untar/unzip Neo4j tar/zip file.</li>\r\n</ol></li>\r\n<li aria-level=\"1\">Download the <a href=\"download/current/reactome.graphdb.tgz\">Graph Database</a> for the latest data release.</li>\r\n<li aria-level=\"1\">Install the Graph Database for Mac/Linux users.<ol style=\"list-style-type: lower-alpha;\">\r\n<li aria-level=\"2\">Extract the Reactome.graph.db after downloading.</li>\r\n<li aria-level=\"2\">Move graph.db folder to /path/to/neo4j/data/databases/<ol style=\"list-style-type: lower-roman;\">\r\n<li aria-level=\"3\">If graph.db already exists, remove it or rename it.</li>\r\n</ol></li>\r\n<li aria-level=\"2\">Rename the folder to the graph.db.</li>\r\n<li aria-level=\"2\">Config Neo4j<ol style=\"list-style-type: lower-roman;\">\r\n<li aria-level=\"3\">Edit Neo4j confirmation file in /path/to/neo4j/conf/neo4j.conf</li>\r\n<li aria-level=\"3\">We recommend having all these settings:<ol style=\"list-style-type: upper-alpha;\">\r\n<li aria-level=\"4\"><em>dbms.default_database=graph.db</em></li>\r\n<li aria-level=\"4\"><em>dbms.recovery.fail_on_missing_files=false</em></li>\r\n<li aria-level=\"4\"><em>unsupported.dbms.tx_log.fail_on_corrupted_log_files=false</em></li>\r\n</ol></li>\r\n</ol></li>\r\n<li aria-level=\"2\">Start Neo4j  ./path/to/neo4j/bin/neo4j start</li>\r\n</ol></li>\r\n<li aria-level=\"1\">Install the Graph Database for Windows user<ol style=\"list-style-type: lower-alpha;\">\r\n<li aria-level=\"2\">Please see the instructions<a href=\"https://neo4j.com/docs/operations-manual/current/installation/windows/\"> here</a>.</li>\r\n</ol></li>\r\n<li aria-level=\"1\">If the standard procedure has been followed, the graph database should be accessible via the Neo4j browser at your <a href=\"http://localhost:7474/\">localhost</a>. More instructions are available in the <a href=\"http://neo4j.com/docs/operations-manual/current/\">Neo4j operations tutorial</a>, specifically the sections “<a href=\"#file-locations\">file locations</a>” and “<a href=\"#backup-restoring\">restoring a backup</a>“.</li>\r\n</ol>\r\n<p>You can also <a href=\"https://neo4j.com/docs/operations-manual/current/backup-restore/restore-dump/\">restore a graph database </a>dump file to Neo4j Community if you have Java 11 installed on your local.</p>\r\n<ol>\r\n<li aria-level=\"1\">Download and install the <a href=\"https://neo4j.com/download/other-releases/\">Neo4j V4</a>.<ol style=\"list-style-type: lower-alpha;\">\r\n<li aria-level=\"1\">Untar/unzip Neo4j tar/zip file.</li>\r\n</ol></li>\r\n<li>Download the Graph Database Dump file.</li>\r\n<li>Run ./path/to/neo4j/bin/neo4j-admin load --force --from=/path/to/reactome.graphdb.dump --database=graph.db</li>\r\n<li>Start Neo4j  ./path/to/neo4j/bin/neo4j start</li>\r\n</ol>\r\n<h4>4. Using Neo4j V5: </h4>\r\n<p>Reactome data dumps are in Neo4j version 4 format. To use them with Neo4j version 5 the Reactome database needs to be downloaded as a dump and converted. The following instructions are for CLI installation of Neo4j (Linux): </p>\r\n<ol>\r\n<li aria-level=\"1\">Download and install any 5.* version of Neo4j </li>\r\n<li aria-level=\"1\">Download the Reactome <a href=\"https://reactome.org/download/current/reactome.graphdb.dump\">database dump</a></li>\r\n<li aria-level=\"1\">cd to installation directory. From there, run all the commands provided below</li>\r\n<li aria-level=\"1\">cp &lt;PATH OF DOWNLOADED DUMP&gt; reactome.dump </li>\r\n<li aria-level=\"1\">./bin/neo4j-admin database migrate --force-btree-indexes-to-range reactome</li>\r\n<li aria-level=\"1\">Now there should be a new directory created under data/databases</li>\r\n<li aria-level=\"1\">Config Neo4j:<ol>\r\n<li aria-level=\"2\">We recommend having all these settings in conf/neo4j.conf:<ol>\r\n<li aria-level=\"3\">initial.dbms.default_database=reactome</li>\r\n<li aria-level=\"3\">db.recovery.fail_on_missing_files=false</li>\r\n<li aria-level=\"3\">unsupported.dbms.tx_log.fail_on_corrupted_log_files=false</li>\r\n</ol></li>\r\n</ol></li>\r\n</ol>\r\n<h4>Troubleshooting</h4>\r\n<h5>Not able to access graph.db in Mac/Linux</h5>\r\n<span>DatabaseUnavailable：</span>\r\n<p><code>Database \"`graph.db`\" is unavailable, its status is \"offline.\"</code></p>\r\n<p>There may be a few reasons for being unable to access the graph database. Some areas you can check include:</p>\r\n<ol>\r\n<li>Ensure the \"graph.db\" folder is accessible under your Neo4j installation folder. It should be located at \"/path/to/neo4j/data/databases/graph.db\".</li>\r\n<li>Ensure the user and group owner are recursively set to \"neo4j:adm\" using the command \"chown -R neo4j:adm /path/to/neo4j\".</li>\r\n<li>Depending on how you are using Neo4j, the database may need to be started using the \"cypher-shell\" or a configuration value may need to be changed to allow access to the database.</li>\r\n</ol>\r\n<p>If you still face any problems please send an email to <a href=\"mailto:help@reactome.org\"></a><a href=\"mailto:help@reactome.org\">help@reactome.org</a> with information about the following to help debug the problem:</p>\r\n<ol>\r\n<li>The version of Neo4j you are using</li>\r\n<li>The working environment you are using, Mac? Linux? Windows?</li>\r\n<li>The software you are using to run and access Neo4j <ol style=\"list-style-type: lower-alpha;\">\r\n<li>Neo4j Docker image vs. Neo4j community edition installed to your system</li>\r\n<li>Neo4j Desktop vs. Web Browser vs. Cypher-Shell</li>\r\n</ol></li>\r\n<li>Path of the Neo4j installation, if installed locally</li>\r\n</ol>\r\n<p></p>\r\n<p>Great! Now you have your own copy of the current version of the Reactome data content in your instance of Neo4j, so let’s see how you can take advantage of it either with direct queries to the graph database or using our GraphCore java library.</p>\r\n<h4>Directly querying to the Reactome Graph Database</h4>\r\n<p>The Neo4j browser offers a nice interface to submit your own queries to the graph database. We recommend using this platform for the first interaction with the Reactome Graph database to see how easy is to use the <a href=\"http://neo4j.com/developer/cypher/\">Cypher query language</a>.</p>\r\n<p>Please refer to our <a href=\"index.php?option=com_content&amp;view=article&amp;id=54\">extracting pathway participating molecules</a> tutorial to introduce yourself to using Cypher to query the Reactome Graph Database.</p>\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>API</a></h3>\r\n<p>The API for the Reactome GraphCore Java library is available on our<a href=\"https://github.com/reactome/graph-core\"><i class=\"fa fa-github\" style=\"padding-right: 1px;\"> GitHub</i></a> repository.</p>\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<p><a href=\"index.php?option=com_content&amp;view=article&amp;id=54&amp;catid=29\">Tutorial: Extracting participating molecules using the Graph Database</a>.</p>\r\n<p>To learn more about our graph database, have a look at our relevant publication entitled <a href=\"https://doi.org/10.1371/journal.pcbi.1005968\" target=\"_blank\" rel=\"noopener\">Reactome graph database: Efficient access to complex pathway data</a>.</p>\r\n<p></p>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-25 16:27:19','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,12,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(46,612,'Teleconferences','teleconferences','','',0,30,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-29 20:01:45','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',2,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(47,619,'Analysis Service','analysis-service','<h4>Explore our tools and web services and learn how to include them in your applications</h4>\r\n<p>{loadposition dev-guide-as-1}</p>\r\n<div>The analysis tool suite contains an overrepresentation analysis, an expression data analysis and a species comparison tool. The tool suite is available via a&nbsp;<a title=\"Web Service\" href=\"https://reactome.org/AnalysisService/\">Web Service</a>&nbsp;so all the available analysis tools can be easily integrated into third party software.</div>\r\n<div style=\"text-align: justify;\">\r\n<h3><a id=\"MoreInfo\" href=\"#MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p>The Analysis Web-Service is token based, so for every analysis request a TOKEN is associated to the result. From this moment on, the results can be accessed via the API “token” method in order to retrieve more detailed information. Taking advantage of the token produced during every analysis, it is possible to link back to the PathwayBrowser and browse the results overlaid on either the pathways overview or a selected pathway.</p>\r\n<p>As mentioned afore, Reactome provides an overrepresentation analysis, an expression data analysis and a species comparison tool. For overrepresentation and expression analysis, the service can automatically detect which one to perform depending on the format of the submitted data (see&nbsp;<a href=\"#AnalysisFormat\">Table 1</a>). If the data contains expression values in a tab-separated-values (TSV) format, then the expression analysis will be executed, otherwise the overrepresentation analysis is the executed one.</p>\r\n<div class=\"text-center\"><a id=\"AnalysisFormat\"></a>Table 1: Analysis input data formats<br /><img src=\"images/devguide/analysis/analysis_format.png\" alt=\"analysis format\" width=\"1252\" /></div>\r\n<p>Data must be submitted in a format that includes a first row of column headers. The header for column 1 must start with the # symbol. Column 1 must contain protein, compound or other suitable identifiers, such as OMIM IDs. Identifiers can only be placed in column 1. A one-column file is sufficient for pathway over-representation analysis. For expression analysis, columns 2 and onwards must contain numeric values, with no alphabetical characters. Decimals must use the full stop (period) symbol, not commas. Columns must be tab-delimited (this format is an option when saving from most spreadsheet programs such as Excel).</p>\r\n<p>The first line must start with “#” to indicate the name of the sample (first column) and the names of each expression column when expression data is submitted. Please note that while this line is optional for the overrepresentation format, it is mandatory for the expression data format, because the name of each column needs to be provided. It is recommended though to have this first line defined in the submitted data, so later on it is easier to identify what kind of experiment the data is related to.</p>\r\n<p>Every other line starts with the identifier of the gene/protein/chemical. For expression analysis, the rest of the columns should contain numbers corresponding to the expression values.</p>\r\n<h4><a id=\"DataSubmission\" href=\"#DataSubmission\">Data Submission</a></h4>\r\n<p>The first use case would be when the data to be analysed is submitted using the method:</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/AnalysisService/identifiers/</pre>\r\n</div>\r\n<p>In this case the data must be sent via POST (please refer to the API documentation for more details) so the analysis will be performed against the Reactome database content.</p>\r\n<p>A second use case is when a text flat file containing the data to be analysed (in the specified format mentioned afore) is sent. In this case the method to be used is:</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/AnalysisService/identifiers/form/</pre>\r\n</div>\r\n<p>A third use case would be when the data to be analysed is accessible through the Internet. In this case, there is a specific method that indicates to the analysis service where to get the data:</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/AnalysisService/identifiers/url/</pre>\r\n</div>\r\n<p>Please note all these methods have an option to project the identifiers to human and only show the result in this species (by adding the “projection/” suffix to any of them). Please refer to the API documentation for more details.</p>\r\n<h4><a id=\"analysis-result-handling\" href=\"#analysis-result-handling\">Analysis result handling</a></h4>\r\n<p>Once the analysis is finished, the result is sent back to the user in json format (please refer to the API documentation for more details). One of the fields in the analysis summary object is called “token”. The particular analysis result is associated to this token and it can be retrieved later on using the following method without having to send the same data again:</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/AnalysisService/token/</pre>\r\n</div>\r\n<p>Reactome ensures that the token will be available for the 7 next days after the analysis. After this period it goes into an LRU queue, so may be available for longer but this cannot be ensured as it depends on the frequency of its usage.</p>\r\n<p>The “pathways” field in the analysis result contains a list of the most significant pathways with the corresponding statistics results. By default, these pathways are sorted from the most to the least significant, so the first item would be the most statistically significant. However the user is advised to take into account the statistical values (pValue and FDR) and also the results coverage in the pathway, indicated by entities and reactions found.</p>\r\n<h4><a id=\"linking-back\" href=\"#linking-back\">Linking back to Reactome Pathway Browser</a></h4>\r\n<p>If linking back to Reactome to visualise the results is required, there are two methods. Providing the token is enough to link back to the PathwayBrowser and get the Fireworks results overlay view. To do so please replace {TOKEN} in the following URL by the one provided in the result.</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/PathwayBrowser/#DTAB=AN&amp;ANALYSIS=<b>{TOKEN}</b></pre>\r\n</div>\r\n<p>To build a link back pointing to a specific pathway to overlay the result on its diagram, please replace&nbsp;<b>{ST_ID}</b>&nbsp;by the pathway stable identifier (provided in the analysis result for each pathway) and&nbsp;<b>{TOKEN}</b>&nbsp;by the one provided in the result.</p>\r\n<div class=\"code\">\r\n<pre>https://reactome.org/PathwayBrowser/#<b>{ST_ID}</b>&amp;DTAB=AN&amp;ANALYSIS=<b>{TOKEN}</b></pre>\r\n</div>\r\n<p>&nbsp;</p>\r\n<h3><a id=\"GetStarted\" href=\"#GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>OK! You’ve reached this far :) so let’s see how to start using the analysis service.</p>\r\n<p>There are two ways to submit your sample of identifiers for analysis. The first method is to POST all the identifiers (or the file containing them). The second involves letting the analysis service know where the sample is by providing its URL. Many types of identifiers that can be submitted, including UniProt, chEBI, Ensembl, miRBase, GenBank/EMBL/DDBJ, RefPep, RefSeq, EntrezGene, OMIM, InterPro, Affymetrix, Agilent, Compound, Illumina, etc. Please get in touch with our&nbsp;<a href=\"mailto:help@reactome.org?subject=Sample%20Identifiers%20are%20not%20supported\">help desk</a>&nbsp;if your sample identifiers are not supported.</p>\r\n<p>In the following examples we use&nbsp;<a href=\"http://curl.haxx.se/docs/manual.html\">curl</a>&nbsp;to query the analysis service from the command line. They show how to send some gene names (PIK3C2A, PTEN and UNC5B) to be analysed and also how to provide the location of your data via the&nbsp;<b>url</b>&nbsp;interface.</p>\r\n<p>The simplest approach is to send your gene names via POST to the&nbsp;<b><a href=\"https://reactome.org/AnalysisService/identifiers/projection\">https://reactome.org/AnalysisService/identifiers/projection</a>&nbsp;</b>method.</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>curl -H \"Content-Type: text/plain\" -d \"$(printf \'<b>#Genes</b>\\n<b>PIK3C2A</b>\\n<b>PTEN</b>\\n<b>UNC5B</b>\')\" -X POST --url https://reactome.org/AnalysisService/identifiers/projection/\\?pageSize\\=1\\&amp;page\\=1</pre>\r\n</div>\r\n</div>\r\n<p>For long lists, the previous example might be impractical. A more convenient way is to POST the content of a file containing the sample to be analysed. Let’s assume you have a file called&nbsp;<b>genes.txt</b>&nbsp;that contains the following set of genes to be analysed:</p>\r\n<div class=\"code\">\r\n<pre>#Genes<br />PIK3C2A<br />PTEN<br />UNC5B</pre>\r\n</div>\r\n<p>The command changes to indicate that the data is now taken from the file (but please note that the method of the analysis service remains the same):</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>curl -H \"Content-Type: text/plain\" --data-binary @genes.txt -X POST --url https://reactome.org/AnalysisService/identifiers/projection/\\?pageSize\\=1\\&amp;page\\=1</pre>\r\n</div>\r\n</div>\r\n<p>Both of the previous commands will produce the result shown in&nbsp;<a href=\"#AnalysisResult\">Figure 1</a></p>\r\n<div><a id=\"AnalysisResult\"></a>Figure 1: Analysis result example</div>\r\n<p><img src=\"images/devguide/analysis/analysis_result_example_01.png\" alt=\"analysis result example 01\" /></p>\r\n<p>Let’s focus on the retrieved&nbsp;<b>pathways</b>&nbsp;list. It only contains one pathway because in the command we have specified&nbsp;<b>\\?pageSize\\=1\\&amp;page\\=1</b>&nbsp;and that forces the analysis to provide&nbsp;<b>ONLY</b>&nbsp;the most significant pathway for the submitted data. If you want the first 10 results, the way of doing it would be&nbsp;<b>\\?pageSize\\=10\\&amp;page\\=1</b>. To see the results from 11 to 20 use&nbsp;<b>\\?pageSize\\=10\\&amp;page\\=2</b>. The reason for this&nbsp;<b>paging</b>&nbsp;mechanism is to avoid overloading the client with the full set of results. Here we are querying from the command line and probably the memory usage isn’t an issue, but please consider web-clients.&nbsp;<b>Important note:</b>&nbsp;If&nbsp;<b>pageSize</b>&nbsp;and&nbsp;<b>page</b>&nbsp;are not specified, then the whole set of results is retrieved to the client.</p>\r\n<p>Other important fields in the result are&nbsp;<b>pathwaysFound</b>,&nbsp;<b>identifiersNotFound</b>,&nbsp;<b>summary.token</b>&nbsp;and&nbsp;<b>summary.type</b>. The first two are self-descriptive, so let’s focus on the&nbsp;<b>summary</b>. As already mentioned, the&nbsp;<b>summary.token</b>&nbsp;can be used to retrieve the results of a previously performed analysis without the need to submit the sample again. For example, the results of the previous analysis can be easily accessed by simply calling the&nbsp;<b>https://reactome.org/AnalysisService/token</b>&nbsp;method and providing the token.</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>curl https://reactome.org/AnalysisService/token/<b>MjAxNTEwMjAwNjU0MDBfMzMw</b>\\?pageSize\\=1\\&amp;page\\=1</pre>\r\n</div>\r\n</div>\r\n<p>As illustrated in&nbsp;<a href=\"#AnalysisTokenMethods\">Figure 2</a>, the analysis service provides a thorough collection of token-based methods that can be used to access the results of a previously performed analysis.</p>\r\n<div class=\"text-center\"><a id=\"AnalysisTokenMethods\"></a>Figure 2: All the token-based methods provided by the analysis service</div>\r\n<p class=\"text-center\"><img src=\"images/devguide/analysis/token_methods.png\" alt=\"token methods\" width=\"972\" /></p>\r\n<p>The&nbsp;<b>summary.type</b>&nbsp;provides information about the type of the analysis performed and it can be OVERREPRESENTATION, EXPRESSION or SPECIES_COMPARISON.</p>\r\n<p>Every item in the&nbsp;<b>pathways</b>&nbsp;array, contains information about a specific pathway, i.e its name, stable identifier, the species it belongs to, the number of entities matching the submitted sample etc.&nbsp;<b>pValue</b>&nbsp;and&nbsp;<b>fdr</b>&nbsp;show the significance of the pathway as the result of the analysis.</p>\r\n<h4>Pointing to your resource as data provider for the analysis</h4>\r\n<p>Let’s suppose that you have some data to be analysed on your server and you want to point Reactome analysis to retrieve it directly. We use&nbsp;<a href=\"https://www.ebi.ac.uk/pride\">PRIDE</a>&nbsp;data for our examples. More specifically we analyse PRIDE data stored in&nbsp;<a href=\"https://www.ebi.ac.uk/pride/ws/archive/protein/list/assay/27929.acc\">https://www.ebi.ac.uk/pride/ws/archive/protein/list/assay/27929.acc</a>. This can be done by using the&nbsp;<b>/identifiers/url</b>&nbsp;methods:</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>curl -H \"Content-Type: text/plain\" -d \"https://www.ebi.ac.uk/pride/ws/archive/protein/list/assay/27929.acc\" -X POST --url https://reactome.org/AnalysisService/identifiers/url/projection/\\?pageSize\\=1\\&amp;page\\=1</pre>\r\n</div>\r\n</div>\r\n<p>Just&nbsp;<b>POST</b>ing the&nbsp;<b>URL</b>, where the data is, to the&nbsp;<b>https://reactome.org/AnalysisService/identifiers/url/projection/</b>&nbsp;method is enough to perform the analysis. The rest of the parameters work exactly as explained above. It is also important to take into account that&nbsp;<b>URL</b>s sent to this method can either be&nbsp;<b>HTTP</b>&nbsp;or&nbsp;<b>HTTPS</b>, if your service uses&nbsp;<a href=\"http://en.wikipedia.org/wiki/HTTPS\">secure HTTP</a>, we can deal with it ;).</p>\r\n<h4><a id=\"how-do-i-query\" href=\"#how-do-i-query\">I have a JavaScript client. How do I query your service?</a></h4>\r\n<p>First we will create a simple HTML page with one button and a place holder to show a table with the results of the query. Please note we have included the&nbsp;<a href=\"http://jquery.com/\" target=\"_blank\" rel=\"nofollow noopener noreferrer\">jQuery library</a>.</p>\r\n<a id=\"HTMLExample01\"></a>\r\n<div class=\"text-center\">HTML: Base example</div>\r\n<div class=\"code\">\r\n<div>\r\n<pre>&lt;html&gt;<br />  &lt;head&gt;<br />	&lt;title&gt;Connection to the Reactome Analysis Service&lt;/title&gt;<br />	//We are using jQuery for this example<br />	&lt;script src=\"//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js\"&gt;&lt;/script&gt;<br />	&lt;script type=\"text/javascript\"&gt;\r\n		//Here is where we are going to include the magic\r\n	&lt;/script&gt;\r\n  &lt;/head&gt;\r\n  &lt;body&gt;\r\n	&lt;h1&gt;Connection to the Reactome Analysis Service&lt;/h1&gt;\r\n	&lt;p&gt;Please click the button to execute the analysis:&lt;/p&gt;\r\n	&lt;button id=\"analyseURL\"&gt;Analyse data&lt;/button&gt;\r\n	&lt;br&gt; &lt;br&gt;\r\n	&lt;div id=\"resultsHolder\"&gt;&lt;/div&gt;\r\n	&lt;div id=\"errorHolder\" style=\"color:red\"&gt;&lt;/div&gt;\r\n  &lt;/body&gt;\r\n&lt;/html&gt;<br />\r\n</pre>\r\n</div>\r\n</div>\r\n<p>As you can see in the&nbsp;<a href=\"#HTMLExample01Figure\">Figure 3</a>, the resulting page is quite a simple one, but enough for our&nbsp;<b>get started</b>&nbsp;purposes.</p>\r\n<a id=\"HTMLExample01Figure\"></a>\r\n<div class=\"text-center\">Figure 3: Example HTML</div>\r\n<p class=\"text-center\"><img style=\"border: solid 1px grey;\" src=\"images/devguide/analysis/example_01.png\" alt=\"example 01\" width=\"923\" /></p>\r\n<p>The next step is to write the code that connects to the analysis service and presents the data back to the client. Please note that we will use the&nbsp;<b>/identifiers/url/projection</b>&nbsp;method in order to request the analysis of&nbsp;<a href=\"https://www.ebi.ac.uk/pride/ws/archive/protein/list/assay/27929.acc\">data</a>&nbsp;stored in the PRIDE repository.</p>\r\n<a id=\"HTMLExample02\"></a>\r\n<div class=\"text-center\">HTML: Example with JavaScript code included</div>\r\n<div class=\"code\">\r\n<div>\r\n<pre>&lt;html&gt;\r\n  &lt;head&gt;\r\n	&lt;title&gt;Connection to the Reactome Analysis Service&lt;/title&gt;\r\n	&lt;script src=\"//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js\"&gt;&lt;/script&gt;\r\n	&lt;script type=\"text/javascript\"&gt;\r\n		$( document ).ready(function() {\r\n			$(\"#analyseURL\").click(function() {\r\n				// The URL of the analysis method\r\n				var reactomeURI = \'https://reactome.org/AnalysisService/identifiers/url/projection?pageSize=20&amp;page=1\';\r\n				// The URL of our sample to be submitted for analysis\r\n				var dataUrl = \"http://www.ebi.ac.uk/pride/ws/archive/protein/list/assay/27929.acc\";\r\n				// Perform the ajax call to the analysis service\r\n				jQuery.ajax({\r\n					type: \"POST\",\r\n					url: reactomeURI,\r\n					data: dataUrl,\r\n					headers: {\r\n						\"Content-Type\": \"text/plain\"\r\n					},\r\n					dataType: \"json\"\r\n				}).success(function (data, textStatus) {\r\n					if (textStatus === \'success\') {\r\n						var table = \"&lt;table width=\'100%\'&gt;&lt;tr&gt;&lt;td&gt;Name&lt;/td&gt;&lt;td&gt;pValue&lt;/td&gt;&lt;td&gt;FDR&lt;/td&gt;&lt;/tr&gt;\";\r\n						$.each(data.pathways, function(num, pathway){\r\n							table += \"&lt;tr&gt;&lt;td&gt;&lt;a href=\'https://reactome.org/PathwayBrowser/#/\"+  pathway.stId +\"&amp;DTAB=AN&amp;ANALYSIS=\" + data.summary.token +\"\'&gt;\" + pathway.name + \"&lt;/a&gt;&lt;/td&gt;&lt;td&gt;\" + pathway.entities.pValue + \"&lt;/td&gt;&lt;td&gt;\" + pathway.entities.fdr + \"&lt;/td&gt;\";\r\n						});\r\n						table += \"&lt;/table&gt;\";\r\n						$(\"#resultsHolder\").append(table);\r\n					} else {\r\n						$(\'#errorHolder\').text(\'An error occurred\');\r\n					}\r\n				}).error(function (data, status, headers, config) {\r\n					$(\'#errorHolder\').text(status + \": \" + headers);\r\n				});\r\n			});\r\n		});\r\n	&lt;/script&gt;\r\n  &lt;/head&gt;\r\n  &lt;body&gt;\r\n	&lt;h1&gt;Connection to the Reactome Analysis Service&lt;/h1&gt;\r\n	&lt;p&gt;Please click the button to execute the analysis:&lt;/p&gt;\r\n	&lt;button id=\"analyseURL\"&gt;Analyse data&lt;/button&gt;\r\n	&lt;br&gt; &lt;br&gt;\r\n	&lt;div id=\"resultsHolder\"&gt;&lt;/div&gt;\r\n	&lt;div id=\"errorHolder\" style=\"color:red\"&gt;&lt;/div&gt;\r\n  &lt;/body&gt;\r\n&lt;/html&gt;<br />\r\n</pre>\r\n</div>\r\n</div>\r\n<p>Quickly analysing the main parts of the&nbsp;<a href=\"#HTMLExample02\">JavaScript code</a>&nbsp;body, we see two main methods:&nbsp;<a href=\"http://learn.jquery.com/using-jquery-core/document-ready/\"><b>$( document ).ready</b></a>&nbsp;and&nbsp;<a href=\"http://api.jquery.com/jquery.ajax/\"><b>jQuery.ajax</b></a>. The first one ensures the content of the associated callback function will be executed when all the HTML is fully loaded in the&nbsp;<a href=\"http://www.w3.org/TR/WD-DOM/introduction.html\">DOM</a>. The second is responsible for performing the analysis request against Reactome’s analysis service.</p>\r\n<p>Focusing on the&nbsp;<b>.success</b>&nbsp;function in the ajax query, we see how the results for a “success” connection with a “success” response are handled (please note that a success connection differs from a success response, since there can be different reasons why a success connection can end up retrieving an error, i.e. data content is not in the right format). The results can be accessed within the&nbsp;<b>data</b>&nbsp;object passed to the&nbsp;<b>.success(function(data, textStatus){…})</b>&nbsp;method following the format shown in&nbsp;<a href=\"#AnalysisResult\">Figure 1</a>.</p>\r\n<p>In the example, a table is created with the retrieved set of pathways (<a href=\"#HTMLExample02Figure\">Figure 4</a>). Each row of the table contains the pathway name, the pValue and the FDR. It’s also important to see how we’ve built the hyperlink from the name to the view of the pathway (with the analysis result overlay) directly to the&nbsp;<a href=\"/PathwayBrowser/\">Pathway Browser</a>. Please note that it is also possible to link to the Pathway Browser in order to show the general result overlaid on the pathways overview. The link for this is:</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>https://reactome.org/PathwayBrowser/#/DTAB=AN&amp;ANALYSIS=<b>{TOKEN}</b></pre>\r\n</div>\r\n</div>\r\n<a id=\"HTMLExample02Figure\"></a>\r\n<div class=\"text-center\">Figure 4: Example HTML with results already loaded</div>\r\n<p class=\"text-center\"><img style=\"border-right-color: grey; border-bottom-color: grey;\" src=\"images/devguide/analysis/example_02.png\" alt=\"example 02\" width=\"928\" /></p>\r\n<p>But this example doesn’t look pretty! Yes, we know but we are explaining how to access Reactome data. Please have a look to the&nbsp;<a href=\"http://www.w3schools.com/css/\">CSS</a>&nbsp;documentation and start learning how to apply the styles to make the look fit with your expectations.</p>\r\n<p>&nbsp;</p>\r\n<h3><a id=\"Resources\" href=\"#Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<p><a href=\"/AnalysisService/\">API Documentation</a></p>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-31 15:55:45','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"1\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,11,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(48,620,'Content Service','content-service','<h4>Explore our tools and web services and learn how to include them in your applications</h4>\r\n<p>{loadposition dev-guide-cs-1}</p>\r\n<div style=\"text-align: justify;\">\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p>The Content Service constitutes an easy <a href=\"/ContentService/#/\" target=\"_blank\" rel=\"noopener noreferrer\">API</a> that provides access to the Reactome knowledgebase. Information in Reactome is authored by expert biologist researchers, maintained by Reactome editorial staff, and extensively cross-referenced to other resources e.g. NCBI, Ensembl, UniProt, UCSC Genome Browser, HapMap, KEGG (Gene and Compound), ChEBI, PubMed and GO. It also incorporates inferred orthologous reactions for over 14 non-human species including mouse, rat, chicken, puffer fish, worm, fly, yeast, rice, Arabidopsis and E.coli. Additionally, the Content Service provides access to molecular interactions integrated from PSICQUIC.</p>\r\n<p>The Content Service is based on the Representational State Transfer (REST) protocol. This eliminates the need for complex clients and renders the service simpler, more lightweight, more flexible, and, thus, easier to integrate into third party software compared to its SOAP/WSDL counterparts.</p>\r\n<p>The <a href=\"/ContentService/\" target=\"_blank\" rel=\"noopener noreferrer\">API</a> includes a set of methods classified in groups according to their functionality. For instance, expanding the <a href=\"/ContentService/#/pathways\" target=\"_blank\" rel=\"noopener noreferrer\">pathways group</a> reveals a set of methods that provide specific information about pathways such as the contained Events or the participating PhysicalEntities. All methods are specified as either GET or POST requests. GET and POST are not interchangeable. <a href=\"/ContentService/#!/pathways/getParticipatingPhysicalEntitiesUsingGET\" target=\"_blank\" rel=\"noopener noreferrer\">Expanding a particular method</a> provides more information about its input and output parameters as well as a simple test case for users to try out. In addition, users can also use any command line client such as <a href=\"http://www.gnu.org/software/wget/\" target=\"_blank\" rel=\"noopener noreferrer\">wget</a> or <a href=\"http://curl.haxx.se/docs/manual.html\" target=\"_blank\" rel=\"noopener noreferrer\">curl</a> to access the methods.</p>\r\n<p>It is worth mentioning that to use this API efficiently, the user needs to understand the Reactome <a href=\"/documentation/data-model\" target=\"_blank\" rel=\"noopener noreferrer\">data model</a> and <a href=\"/content/schema/\" target=\"_blank\" rel=\"noopener noreferrer\">schema</a>.</p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>Let’s start with one of the most basic examples; retrieving the version of the database. It can be addressed by querying the “/data/database/version” method:</p>\r\n<div class=\"code\">\r\n<pre>curl -X GET --header \'Accept: text/plain\' \'https://reactome.org/ContentService/data/database/version\'</pre>\r\n</div>\r\n<p>The response will be a “text/plain” file containing just the number of the release.</p>\r\n<p>To retrieve the information for the reaction <b>Mad1 binds kinetochore</b> which identifier is <b>R-HSA-141409</b>, the query would be as follows:</p>\r\n<div class=\"code\">\r\n<pre>curl -X GET --header \'Accept: application/json\' \'https://reactome.org/ContentService/data/query/R-HSA-141409\'</pre>\r\n</div>\r\n<p>And the response comes in “application/json” format:</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre>{\r\n  dbId: 141409,\r\n  displayName: \"Mad1 binds kinetochore\",\r\n  stId: \"R-HSA-141409\",\r\n  created: {\r\n    dbId: 143430,\r\n    displayName: \"Yen, T, 2004-05-05 00:00:00\",\r\n    dateTime: \"2004-05-05 05:00:00.0\",\r\n    schemaClass: \"InstanceEdit\"\r\n  },\r\n  modified: {\r\n    dbId: 1591212,\r\n    displayName: \"Matthews, L, 2011-09-08\",\r\n    dateTime: \"2011-09-08 21:45:40.0\",\r\n    schemaClass: \"InstanceEdit\"\r\n  },\r\n  isInDisease: false,\r\n  isInferred: false,\r\n  name: [\r\n    \"Mad1 binds kinetochore\"\r\n  ],\r\n  speciesName: \"Homo sapiens\",\r\n  authored: [\r\n    143430\r\n  ],\r\n  compartment: [\r\n    {\r\n      dbId: 70101,\r\n      displayName: \"cytosol\",\r\n      accession: \"0005829\",\r\n      databaseName: \"GO\",\r\n      definition: \"The part of the cytoplasm that does not contain organelles but which does contain other particulate matter, such as protein complexes.\",\r\n      name: \"cytosol\",\r\n      url: \"http://www.ebi.ac.uk/ego/QuickGO?mode=display&amp;entry=GO:0005829\",\r\n      schemaClass: \"EntityCompartment\"\r\n    }\r\n  ],\r\n  literatureReference: [\r\n    {\r\n      dbId: 143441,\r\n      displayName: \"Mitotic checkpoint proteins HsMAD1 and HsMAD2 are associated with nuclear pore complexes in interphase\",\r\n      title: \"Mitotic checkpoint proteins HsMAD1 and HsMAD2 are associated with nuclear pore complexes in interphase\",\r\n      journal: \"J Cell Sci\",\r\n      pages: \"953-63\",\r\n      pubMedIdentifier: 11181178,\r\n      volume: 114,\r\n      year: 2001,\r\n      url: \"http://www.ncbi.nlm.nih.gov/pubmed/11181178\",\r\n      schemaClass: \"LiteratureReference\"\r\n    }\r\n  ],\r\n  species: [\r\n    {\r\n      dbId: 48887,\r\n      displayName: \"Homo sapiens\",\r\n      name: [\r\n        \"Homo sapiens\",\r\n        \"H. sapiens\",\r\n        \"Hs\",\r\n        \"human\",\r\n        \"man\"\r\n      ],\r\n      taxId: \"9606\",\r\n      schemaClass: \"Species\"\r\n    }\r\n  ],\r\n  summation: [\r\n    {\r\n      dbId: 143355,\r\n      displayName: \"\",\r\n      text: \"The association of Mad1 with the kinetochore is the first step in the process of Mad2 mediated amplification of the signal from defective kinetochores.\",\r\n      schemaClass: \"Summation\"\r\n    }\r\n  ],\r\n  input: [\r\n    {\r\n      dbId: 141433,\r\n      displayName: \"MAD1L1 [cytosol]\",\r\n      stId: \"R-HSA-141433\",\r\n      name: [\r\n        \"MAD1L1\",\r\n        \"HsMad1\"\r\n      ],\r\n      speciesName: \"Homo sapiens\",\r\n      consumedByEvent: [\r\n        141409\r\n      ],\r\n      endCoordinate: 718,\r\n      referenceType: \"ReferenceGeneProduct\",\r\n      startCoordinate: 1,\r\n      schemaClass: \"EntityWithAccessionedSequence\"\r\n    },\r\n    {\r\n      dbId: 141398,\r\n      displayName: \"Kinetochore Complex [cytosol]\",\r\n      stId: \"R-HSA-141398\",\r\n      name: [\r\n        \"Kinetochore Complex\"\r\n      ],\r\n      speciesName: \"Homo sapiens\",\r\n      consumedByEvent: [\r\n        141409\r\n      ],\r\n      schemaClass: \"GenomeEncodedEntity\"\r\n    }\r\n  ],\r\n  output: [\r\n    {\r\n      dbId: 141441,\r\n      displayName: \"Mad1:kinetochore complex [cytosol]\",\r\n      stId: \"R-HSA-141441\",\r\n      name: [\r\n        \"Mad1:kinetochore complex\"\r\n      ],\r\n      speciesName: \"Homo sapiens\",\r\n      producedByEvent: [\r\n        141409\r\n      ],\r\n      hasComponent: [\r\n        \r\n      ],\r\n      schemaClass: \"Complex\"\r\n    }\r\n  ],\r\n  schemaClass: \"Reaction\"\r\n}</pre>\r\n</div>\r\n</div>\r\n<p>In the previous result, there are several json field keys such as dbId, displayName, stId, name, compartment, literatureReference and so on. In case we are only interested in a particular field, e.g compartment, the query would be as follows:</p>\r\n<div class=\"code\">\r\n<pre>curl -X GET --header \'Accept: text/plain\' \'https://reactome.org/ContentService/data/query/R-HSA-141409/compartment\'</pre>\r\n</div>\r\n<p>In this case, because <b>compartment</b> is an object, the result will be a TSV file where the first column is the identifier of the object, the second column is the displayName of the object and the third column is the schemaClass of the object:</p>\r\n<div class=\"code\">\r\n<pre>70101	cytosol	EntityCompartment</pre>\r\n</div>\r\n<p>In case the queried attribute is a <b>primitive</b> type:</p>\r\n<div class=\"code\">\r\n<pre>curl -X GET --header \'Accept: text/plain\' \'https://reactome.org/ContentService/data/query/R-HSA-141409/displayName\'</pre>\r\n</div>\r\n<p>Then the returned value is the content of it in a “text/plain” response:</p>\r\n<div class=\"code\">\r\n<pre>Mad1 binds kinetochore</pre>\r\n</div>\r\n<p>As the last example in the section, let’s check out how to retrieve the participanting molecules of the reaction used in the previous examples. To do so, we call the <a href=\"/ContentService/#!/events/getParticipatingPhysicalEntitiesUsingGET\">getParticipatingPhysicalEntities</a> method in the API:</p>\r\n<div class=\"code\">\r\n<pre>curl -X GET --header \'Accept: application/json\' \'https://www.reactome.org/ContentService/data/event/R-HSA-141409/participatingPhysicalEntities\'</pre>\r\n</div>\r\n<p>And the response comes in “application/json” format:</p>\r\n<div class=\"code\">\r\n<pre>[\r\n  {\r\n    dbId: 141398,\r\n    displayName: \"Kinetochore Complex [cytosol]\",\r\n    stId: \"R-HSA-141398\",\r\n    name: [\r\n      \"Kinetochore Complex\"\r\n    ],\r\n    speciesName: \"Homo sapiens\",\r\n    schemaClass: \"GenomeEncodedEntity\"\r\n  },\r\n  {\r\n    dbId: 141433,\r\n    displayName: \"MAD1L1 [cytosol]\",\r\n    stId: \"R-HSA-141433\",\r\n    name: [\r\n      \"MAD1L1\",\r\n      \"HsMad1\"\r\n    ],\r\n    speciesName: \"Homo sapiens\",\r\n    endCoordinate: 718,\r\n    referenceType: \"ReferenceGeneProduct\",\r\n    startCoordinate: 1,\r\n    schemaClass: \"EntityWithAccessionedSequence\"\r\n  },\r\n  {\r\n    dbId: 141441,\r\n    displayName: \"Mad1:kinetochore complex [cytosol]\",\r\n    stId: \"R-HSA-141441\",\r\n    name: [\r\n      \"Mad1:kinetochore complex\"\r\n    ],\r\n    speciesName: \"Homo sapiens\",\r\n    hasComponent: [\r\n      \r\n    ],\r\n    schemaClass: \"Complex\"\r\n  }\r\n]</pre>\r\n</div>\r\n<p>Check out the <a href=\"/ContentService/\">ContentService API</a> to find out more methods and see which ones cover your needs. Please <a href=\"mailto:help@reactome.org?subject=Question%20about%20the%20ContentService%20API\">let us know</a> if your needs are not covered and you think extra methods should be added to our API.</p>\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<p><a href=\"/ContentService/\">API Documentation</a></p>\r\n<p><a href=\"index.php?option=com_content&amp;view=article&amp;id=96&amp;catid=29&amp;lang=en-GB\" hreflang=\"en\">Diagram exporter</a></p>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-31 16:15:13','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,10,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(49,621,'Pathways Overview','pathways-overview','<p>The Pathways Overview is a genome-wide, hierarchical visualization of Reactome pathways in a space-filling graph. Each node represents a pathway, and the size of the node reflects the number of entities (proteins, small molecules, genes, etc) belonging to that pathway. The edges represent the connection between the different pathway nodes within the Reactome pathway hierarchy.</p>\r\n<p>{loadposition dev-guide-po-1}</p>\r\n<div>\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p>The central node within each pathway burst is the parent (or top-level) pathway node. Each child (or sub level) pathway belonging to the pathway hierarchy is represented as a node, which is projected outwards from the central node and connected to the subsequent pathway in the hierarchy, until the lowest level pathway is reached. When the graph initially loads, the Pathways Overview display automatically fits all the nodes and edges into the existing area.</p>\r\n<p>The pathways overview retrieves the information directly from Reactome server, which mean that for third party resources it will be straight forward to include it in browser supporting CORS, or it will just work by adding a proxy mechanism in order to avoid SOP. Another important feature of the pathways overview that can also be used from third party applications is the analysis result overlay. If you are interested in learning more about Reactome’s Analysis Service, please check out the&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=47\">Analysis Service developers’ guide</a>&nbsp;and later on have a look to the pathways overview&nbsp;<a href=\"#API\">API</a><a>.</a></p>\r\n<p>&nbsp;</p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>{loadposition dev-guide-po-2}</p>\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>API</a></h3>\r\n<p>{loadposition dev-guide-po-3}</p>\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<ul style=\"list-style: none;\">\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp/fireworks\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Fireworks code</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp/fireworks-js\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Fireworks exporter to JavaSript</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>Reactome Pathways Portal code</a></li>\r\n</ul>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-31 16:18:34','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,9,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(50,622,'Pathway Diagrams','pathway-diagrams','<p>The Reactome Pathway Diagram Viewer provides an intuitive means of viewing and interacting with pathway diagrams. Pathway diagrams represent the steps of a pathway as a series of interconnected molecular events, known in Reactome as ‘reactions’.</p>\r\n<p>{loadposition dev-guide-pd-1}</p>\r\n<div>\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p>Reactome is a curated database of pathways and reactions (pathway steps) in human biology. The Reactome definition of a ‘reaction’ includes many events in biology that are changes in state, such as binding, activation, translocation and degradation, in addition to classical biochemical reactions. Information in the database is authored by expert biologist researchers, maintained by Reactome editorial staff, and extensively cross-referenced to other resources e.g. NCBI, Ensembl, UniProt, UCSC Genome Browser, HapMap, KEGG (Gene and Compound), ChEBI, PubMed and GO. Inferred orthologous reactions are available for a number of non-human species including mouse, rat, chicken, puffer fish, worm, fly, yeast, rice and Arabidopsis.</p>\r\n<p>The diagram viewer retrieves the information directly from Reactome server, which mean that for third party resources it will be straight forward to include it in browser supporting CORS, or it will just work by adding a proxy mechanism in order to avoid SOP. Another important feature of the diagram viewer that can also be used from third party applications is the analysis result overlay. If you are interested in learning more about Reactome’s Analysis Service, please check out the&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=47\">Analysis Service developers’ guide</a>&nbsp;and later on have a look to the Diagram viewer&nbsp;<a href=\"#API\">API</a><a>.</a></p>\r\n<p>&nbsp;</p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>Currently, there are two options for integrating Reactome’s pathway diagrams in your web application. The first is to use our&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=58\">DiagramJs widget</a>, while the second option involves using our&nbsp;<a href=\"index.php?option=com_content&amp;view=article&amp;id=59\">GWT implementation</a>.</p>\r\n<p>{loadposition dev-guide-pd-2}</p>\r\n<!--Get Started Buttons - close-->\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>API</a></h3>\r\n<p>Whether you are a JavaScript or GWT developer, we have a tool that will suit you. Check out your option:</p>\r\n<p>{loadposition dev-guide-pd-3}</p>\r\n<!-- API Buttons - close-->\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<ul style=\"list-style: none;\">\r\n<li style=\"line-height: 30px;\"><i class=\"fa fa-file-alt\" style=\"padding-right: 5px;\"></i><a href=\"index.php?option=com_content&amp;view=article&amp;id=100&amp;catid=29&amp;lang=en-GB\" hreflang=\"en\">Pathway Diagram Specifications</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp/diagram\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Diagram code</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp/diagram-js\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Diagram exporter to JavaSript</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>Reactome Pathways Portal code</a></li>\r\n</ul>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-05-31 16:19:32','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,8,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(53,625,'Data Model','data-model','<p>Life on the cellular level is a network of molecular interactions. Molecules are synthesized and degraded, undergo a bewildering array of temporary and permanent modifications, are transported from one location to another, and form complexes with other molecules. Reactome represents all of this complexity as reactions in which input physical entities are converted to output entities. These reactions can occur spontaneously or be facilitated by physical entities acting as catalysts, and their progress can be modulated by regulatory effects of other physical entities. Reactions are linked together by shared physical entities: a product from one reaction may be a substrate in another reaction and may catalyze yet a third. It is often convenient, if sometimes arbitrary, to group such sets of interlinked reactions into pathways.</p>\r\n<p>The functions of macromolecular entities such as proteins are often determined not only by their primary sequences but by chemical modifications they have undergone. In Reactome, unmodified and modified forms of a protein are distinct physical entities and the modification process is treated as an explicit reaction. A macromolecule’s function may depend on whether the molecule is free or complexed with specific other molecules. Reactome treats complexes as physical entities distinct from their components, and the multimerization events that build up complexes are modeled explicitly as reactions.</p>\r\n<p>Cellular compartments play a key role in biological processes. The segregation of molecules into different compartments often regulates the reactions in which those entities can participate or can be responsible for driving a reaction forward. In Reactome, a molecule in one compartment is distinct from that molecule in another compartment. Thus, extracellular and cytosolic glucose are different Reactome entities and, e.g., the movement of glucose across the plasma membrane is a reaction that converts the extracellular glucose entity into the cytosolic one.</p>\r\n<p>Many biochemical entities and processes appear redundant: there are two or more chemically distinct entities that can act more or less interchangeably. It is often useful to treat functionally equivalent protein isoforms, splice variants, and paralogues as a single entity, implying that any individual entity from the given set could fulfill the same role in a given situation. The Reactome data model allows this type of generalization, but does so explicitly in a way that allows us to trace specific functions back to the individual molecules covered by the generalization.</p>\r\n<p>The goal of the Reactome knowledgebase is to represent human biological processes, but many of these processes have not been directly studied in humans. Rather, a human event has been inferred from experiments on material from a model organism. In such cases, the model organism reaction is annotated in Reactome, the inferred human reaction is annotated as a separate event, and the inferential link between the two reactions is explicitly noted.</p>\r\n<p>Reactome uses a frame-based knowledge representation. The data model consists of classes (frames) that describe the different concepts (e.g., reaction, simple entity). Knowledge is captured as instances of these classes (e.g., “glucose transport across the plasma membrane”, “cytosolic ATP”). Classes have attributes (slots) that hold properties of the instances (e.g., the identities of the molecules that participate as inputs and outputs in a reaction).</p>\r\n<h3>Key data classes</h3>\r\n<h4><a href=\"/content/schema/\">PhysicalEntity</a></h4>\r\n<p>PhysicalEntities include individual molecules, multi-molecular complexes, and sets of molecules or complexes grouped together on the basis of shared characteristics. Molecules are further classified as genome-encoded (DNA, RNA, and proteins) or not (all others). Attributes of a PhysicalEntity instance capture the chemical structure of an entity, including any covalent modifications in the case of a macromolecule and its subcellular localization.</p>\r\n<p>PhysicalEntity instances that represent, e.g., the same chemical in different compartments, or different post-translationally modified forms of a single protein, share numerous invariant features such as names, molecular structure and links to external databases like UniProt or ChEBI. To enable storage of this shared information in a single place, and to create an explicit link among all the variant forms of what can also be seen as a single chemical entity, Reactome creates instances of the separate ReferenceEntity class. A <a href=\"/content/schema/ReferenceEntity\">ReferenceEntity</a> instance captures the invariant features of a molecule. A PhysicalEntity instance is then the combination of a ReferenceEntity attribute (e.g., <a href=\"/content/detail/R-HSA-71580\">Glycogen phosphorylase UniProt:P06737</a>) and attributes giving specific conditional information (e.g., localization to the cytosol and phosphorylation on serine residue 14).</p>\r\n<p>The PhysicalEntity class has subclasses to distinguish between different kinds of entity and to ensure data integrity while enabling different handling rules for different categories:</p>\r\n<p><strong><a href=\"/content/schema/EntityWithAccessionedSequence\">EntityWithAccessionedSequence</a> </strong>– proteins and nucleic acids with known sequences.</p>\r\n<p><strong><a href=\"/content/schema/GenomeEncodedEntity\">GenomeEncodedEntity</a> </strong>– a species-specific protein or nucleic acid whose sequence is unknown, such as an enzyme that has been characterized functionally but not yet purified and sequenced, e.g. <a href=\"/content/detail/R-HSA-2161674\">cytosolic 15-HEDH enzyme</a></p>\r\n<p><strong><a href=\"/content/schema/SimpleEntity\">SimpleEntity</a> </strong>– other fully characterized molecules, e.g. <a href=\"/content/detail/R-ALL-29358\">nucleoplasmic ATP</a> or <a href=\"/content/detail/R-ALL-29450\">cytosolic glutathione</a></p>\r\n<p><strong><a href=\"/content/schema/Complex\">Complex</a> </strong>– a complex of two or more PhysicalEntities, e.g. <a href=\"/content/detail/R-HSA-71050\">Trimerization of the FASL:FAS receptor complex</a></p>\r\n<p><a href=\"/content/schema/EntitySet\" target=\"_blank\" style=\"background-color: inherit; font-family: inherit; font-size: 1rem;\"><strong>EntitySet</strong></a><strong style=\"background-color: inherit; color: inherit; font-family: inherit; font-size: 1rem; caret-color: auto;\"> </strong><span style=\"background-color: inherit; color: inherit; font-family: inherit; font-size: 1rem; caret-color: auto;\">– a set of PhysicalEntities (molecules or complexes) that function interchangeably in a given situation, e.g., </span><a href=\"/content/detail/R-HSA-1912382\" target=\"_blank\" style=\"background-color: inherit; font-family: inherit; font-size: 1rem;\">Mature NOTCH heterodimer traffics to the plasma membrane</a><span style=\"background-color: inherit; color: inherit; font-family: inherit; font-size: 1rem; caret-color: auto;\">.</span> This notation allows the collective properties of multiple individual entities to be described explicitly.</p>\r\n<p>PhysicalEntities are paired with molecular functions taken from the Gene Ontology molecular function controlled vocabulary to describe instances of biological catalysis. An optional ActiveUnit attribute indicates the specific domain of a protein or subunit of a complex that mediates the catalysis. If a PhysicalEntity has multiple catalytic activities, a separate CatalystActivity is created for each. This strategy allows the association of specific activities with specific variant forms of a protein or complex and also enables easy retrieval of all activities of a protein or all proteins capable of mediating a specific molecular function.</p>\r\n<h4><strong><a href=\"/content/schema/Event\">Event</a></strong></h4>\r\n<p>Events – the conversion of input entities to output entities in one or more steps – are the building blocks used in Reactome to represent all biological processes. Two subclasses of Event are recognized, <strong><a href=\"/content/schema/ReactionLikeEvent\">ReactionLikeEvent</a> </strong>and <strong><a href=\"/content/schema/Pathway\">Pathway</a></strong>. A ReactionlikeEvent is an event that converts inputs into outputs. A Pathway is any grouping of related Events. An event may be a member of more than one Pathway.</p>\r\n<p>The ReactionlikeEvent class is further divided into <strong><a href=\"/content/schema/Reaction\">Reaction</a></strong>, <strong><a href=\"/content/schema/BlackBoxEvent\">BlackBoxEvent</a></strong>, <strong><a href=\"/content/schema/Polymerisation\">Polymerisation</a>, </strong>and <strong><a href=\"/content/schema/Depolymerisation\">Depolymerisation</a></strong>. The Reaction class holds bona fide reactions with balanced inputs and outputs. The BlackBoxEvent class is used for ‘unbalanced’ reactions like protein synthesis or degradation, as well as ‘shortcut’ reactions for more complex processes that essentially convert inputs into outputs, e.g. the series of cyclical reactions involved in fatty acid biosynthesis. The De-/Polymerisation classes can hold reactions that describe the mechanics of a de-/polymerisation reaction, which is inherently ‘unbalanced’ due to the nature of a Polymer (that remains the ‘same’ entity even after adding or subtracting a unit).</p>\r\n<h3>Full specification of the Reactome data model</h3>\r\n<p>A full specification of all Reactome classes, slots and a listing of all instances of each class is accessible from the <strong><a href=\"/content/schema/\">Schema</a></strong> page on the top menu bar. There is also a <a href=\"https://download.reactome.org/documentation/DataModelGlossary_V90.pdf\">Data Model Glossary</a>, giving more details on the usage of the various classes and slots.</p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-02 08:24:17','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,7,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(54,628,'Tutorial: Extracting participating molecules using the Graph Database','tutorial-participating-molecules','<style type=\"text/css\">.rcypher, .rtip, .rresult {\r\n	border: 1px solid #dddddd;\r\n	border-radius: 5px;\r\n	background: #f9f9f9;\r\n 	margin-left: 15px;\r\n 	margin-right: 15px;\r\n	overflow-x: scroll;\r\n	padding: 10px;\r\n}\r\n.rtip {\r\n 	border: 1px solid pink;\r\n 	background-color: #ffe6e6;\r\n}\r\n.rresult{\r\n 	font-family: monospace;\r\n 	font-size: x-small;\r\n 	color: darkgray;\r\n        word-wrap: initial;jq\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n.rcypher code, .rtip code {\r\n        word-wrap: initial;\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n@media only screen and (max-width: 768px) {\r\n    .rcypher, .rtip, .rresult {\r\n 	margin-left: 0px;\r\n        margin-right: 0px;\r\n    }\r\n    .rresult {\r\n        overflow-y: auto;\r\n        white-space: pre;\r\n    }\r\n}</style>\r\n<p></p>\r\n<ul>\r\n<li><a href=\"#participating-molecules-use-case\">The participating molecules use case</a></li>\r\n<li>\r\n<p><span><a href=\"#retrieving-objects\">Retrieving objects based on their identifier</a></span></p>\r\n</li>\r\n<li>\r\n<p><a href=\"#identifiers-proteins-or-chemicals\">Identifiers for proteins or chemicals</a></p>\r\n</li>\r\n<li>\r\n<p><a href=\"#complexes-sets-participants\">Breaking down complexes and sets to get their participants</a></p>\r\n</li>\r\n<li>\r\n<p><a href=\"#retrieving-pathways\">Retrieving pathways,subpathways and superpathways</a></p>\r\n</li>\r\n<li>\r\n<p><a href=\"#retrieving-reactions\">Retrieving the reactions for a given pathway</a></p>\r\n</li>\r\n<li>\r\n<p><a href=\"#retrieving-participants\">Retrieving the participants of a Reaction</a></p>\r\n</li>\r\n<li>\r\n<p><a href=\"#joining-pieces\">Joining the pieces: Participating molecules for a pathway</a></p>\r\n</li>\r\n</ul>\r\n<p></p>\r\n<p>The Reactome Graph Database, also called a graph-oriented database, is a type of NoSQL database that uses graph theory to store, map and query relationships relating to our data content. Each node represents an entity (such as a pathway, reaction or proteins) and each edge represents a relationship between two nodes. Every node in our graph database is defined by a unique identifier, a set of outgoing edges and/or incoming edges and a set of properties expressed as key/value pairs. Each edge is defined by a unique identifier, a starting-place and/or ending-place node and a set of properties.</p>\r\n<p><a href=\"http://neo4j.com/developer/cypher/\" target=\"_blank\" rel=\"noopener\">Cypher</a> is Neo4j’s open graph query language. Cypher’s syntax provides a familiar way to match patterns of nodes and relationships in a graph. If you want to learn more about what is Cypher, please visit the following <a href=\"http://neo4j.com/docs/developer-manual/current/#cypher-introduction\" target=\"_blank\" rel=\"noopener\">link</a>.</p>\r\n<p>Graph databases are well-suited for analyzing interconnections, which is why there has been a lot of interest in using graph databases to mine data from biological pathways and reactions. The Reactome Graph database has many advantages, but one is its responsiveness in managing data. Furthermore, even though data queries increase exponentially, the performance of a graph database does not drop, compared to what happens with relational databases. When software developers work with data, they are looking for flexibility and scalability. Our Graph Database contributes a lot in this regard because when needs increase, the possibilities of adding more nodes and relationships to an existing graph are huge.</p>\r\n<h3><a href=\"#participating-molecules-use-case\" id=\"participating-molecules-use-case\">The participating molecules use case</a></h3>\r\n<p>This tutorial explains how to query Reactome using Cypher. It\'s assumed that the reader has a basic knowledge of Cypher as well as an understanding of our <a href=\"/documentation/data-model/\">data model</a> and how data is stored in our <a href=\"/content/schema/\">schema</a>.</p>\r\n<p>Even though it is not possible to cover all possible queries of the Reactome Graph Database in a single tutorial, the sections in this document build up a query, which will retrieve the resource and identifier of each participating molecule of a given Pathway. There are several intermediate stages to be explained before reaching that point:</p>\r\n<ol>\r\n<li>How to retrieve objects like proteins, reactions, pathways, etc.</li>\r\n<li>How to get the identifier of proteins or chemicals</li>\r\n<li>How to deconstruct complexes or sets to get their participants</li>\r\n<li>How to retrieve the subpathways for a given pathway</li>\r\n<li>How to retrieve the reactions of a pathway</li>\r\n<li>How to retrieve the participants of a reaction</li>\r\n</ol>\r\n<p>The enumeration above represents the <b>basic bricks</b> from which to construct the final query that retrieves the <b>participating molecules</b> for a given <b>pathway</b>.</p>\r\n<h3><a href=\"#retrieving-objects\" id=\"retrieving-objects\">Retrieving objects based on their identifier</a></h3>\r\n<p>To retrieve the Pathway \"<b>Antigen processing-Cross presentation</b>\" with identifier <b>R-HSA-1236975</b>, the query is as follows:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Selecting an Pathway by its stable identifier\r\nMATCH (pathway:Pathway{stId:\"R-HSA-1236975\"})\r\nRETURN pathway\r\n</code></pre>\r\n<p>The result of the query is:</p>\r\n<pre class=\"rresult\">╒═════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╕\r\n│pathway                                                                                                              │\r\n╞═════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╡\r\n│{speciesName: Homo sapiens, oldStId: REACT_111119, isInDisease: false, releaseDate: 2011-09-20, displayName: Antigen │\r\n│ processing-Cross presentation, stIdVersion: R-HSA-1236975.1, dbId: 1236975, releaseStatus: UPDATED, name: [Antigen  │\r\n│ processing-Cross presentation], stId: R-HSA-1236975, hasDiagram: false, isInferred: false}                          │\r\n└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘\r\n</pre>\r\n<p>In the same way, let\'s focus on an EntityWithAccessionedSequence (EWAS) which corresponds to a protein in Reactome. For this example we use one form of <b>PTEN</b> in the <b>cytosol</b> with identifier <b>R-HSA-199420</b></p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Selecting an EWAS by its stable identifier\r\nMATCH (ewas:EntityWithAccessionedSequence{stId:\"R-HSA-199420\"})\r\nRETURN ewas\r\n</code></pre>\r\n<p>The result is one node of the database:</p>\r\n<pre class=\"rresult\">╒════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╕\r\n│ewas                                                                                                                        │\r\n╞════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╡\r\n│{speciesName: Homo sapiens, startCoordinate: 1, isInDisease: false, displayName: PTEN [cytosol], dbId: 199420, name: [PTEN, │\r\n│ Phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase PTEN, PTEN_HUMAN, MMAC1, TEP1], referenceType: ReferenceGeneProduct,│\r\n│ endCoordinate: 403, stId: R-HSA-199420}                                                                                    │\r\n└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘\r\n</pre>\r\n<h3><a href=\"#identifiers-proteins-or-chemicals\" id=\"identifiers-proteins-or-chemicals\">Identifiers for proteins or chemicals</a></h3>\r\n<p>Continuing with the same form of PTEN, another use case could be to retrieve only a couple of fields for the target node. The following query retrieves the EWAS <b>display name</b> and its <b>identifier</b> (from the reference entity):</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Following the reference entity link in order to get the identifier\r\nMATCH (ewas:EntityWithAccessionedSequence{stId:\"R-HSA-199420\"}),\r\n      (ewas)-[:referenceEntity]-&gt;(re:ReferenceEntity)\r\nRETURN ewas.displayName AS EWAS, re.identifier AS Identifier\r\n</code></pre>\r\n<p>Please note that the <b>identifier</b> is not directly stored in the node for the EWAS but is a property of another node pointed from the EWAS which is a <b>ReferenceEntity</b>. The previous query accesses that node from the EWAS following the <b>referenceEntity</b> edge. The result of this query is as shown below:</p>\r\n<pre class=\"rresult\">╒══════════════╤══════════╕\r\n│EWAS          │Identifier│\r\n╞══════════════╪══════════╡\r\n│PTEN [cytosol]│P60484    │\r\n└──────────────┴──────────┘\r\n</pre>\r\n<p>TIP: The <i>MATCH</i> part of the previous query could be written in one line as follows:</p>\r\n<pre class=\"rtip\"><code class=\"language-cypher\" data-lang=\"cypher\">//Equivalent to the previous one\r\nMATCH (ewas:EntityWithAccessionedSequence{stId:\"R-HSA-199420\"})-[:referenceEntity]-&gt;(re:ReferenceEntity)\r\nRETURN ewas.displayName AS EWAS, re.identifier AS Identifier\r\n</code></pre>\r\n<p>Continuing on, it is possible to construct a query to retrieve the <b>reference database</b> on top of the previously retrieved fields. Please note that in this case the <b>reference database</b> is a <b>node</b> pointed from <b>ReferenceEntity</b> by an edge called <b>referenceDatabase</b>:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Following the reference entity and database links in order to get the identifier and the database of reference\r\nMATCH (ewas:EntityWithAccessionedSequence{stId:\"R-HSA-199420\"}),\r\n      (ewas)-[:referenceEntity]-&gt;(re:ReferenceEntity)-[:referenceDatabase]-&gt;(rd:ReferenceDatabase)\r\nRETURN ewas.displayName AS EWAS, re.identifier AS Identifier, rd.displayName AS Database\r\n</code></pre>\r\n<pre class=\"rresult\">╒══════════════╤══════════╤════════╕\r\n│EWAS          │Identifier│Database│\r\n╞══════════════╪══════════╪════════╡\r\n│PTEN [cytosol]│P60484    │UniProt │\r\n└──────────────┴──────────┴────────┘\r\n</pre>\r\n<h3><a href=\"#complexes-sets-participants\" id=\"complexes-sets-participants\">Breaking down complexes and sets to get their participants</a></h3>\r\n<p>The components of a complex, which are also physical entities, are stored in the \"hasComponent\" slot. Let\'s use the complex \"<b>Ag-substrate:E3:E2:Ub</b>\" with identifier <b>R-HSA-983126</b> as example in this case:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//First level components for the complex with stable identifier R-HSA-983126\r\nMATCH (Complex{stId:\"R-HSA-983126\"})-[:hasComponent]-&gt;(pe:PhysicalEntity)\r\nRETURN pe.stId AS component_stId, pe.displayName AS component\r\n</code></pre>\r\n<p>The result of the query is</p>\r\n<pre class=\"rresult\">╒══════════════╤═══════════════════════════════════════════════╕\r\n│component_stId│component                                      │\r\n╞══════════════╪═══════════════════════════════════════════════╡\r\n│R-NUL-983035  │antigenic substrate [cytosol]                  │\r\n├──────────────┼───────────────────────────────────────────────┤\r\n│R-HSA-976075  │E3 ligases in proteasomal degradation [cytosol]│\r\n├──────────────┼───────────────────────────────────────────────┤\r\n│R-HSA-976165  │Ubiquitin:E2 conjugating enzymes [cytosol]     │\r\n└──────────────┴───────────────────────────────────────────────┘\r\n</pre>\r\n<p>In this example, the \"<b>E3 ligases in proteasomal degradation</b>\" is a Set and \"<b>E3 ligases in proteasomal degradation</b>\" is a Complex. To further deconstruct the initial complex there are some minor changes which should be applied to the Cypher query. Sets can either be DefineSets, OpenSets or CandidateSets. The way to find out which other physical entities are part of them is \"traversing\" through the \"<b>hasMember</b>\" or \"<b>hasCandidate</b>\" slots. The following query will break down the initial complex into ALL its participants:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//All distinct components for the complex with stable identifier R-HSA-983126\r\nMATCH (Complex{stId:\"R-HSA-983126\"})-[:hasComponent|hasMember|hasCandidate*]-&gt;(pe:PhysicalEntity)\r\nRETURN DISTINCT pe.stId AS component_stId, pe.displayName AS component\r\n</code></pre>\r\n<p>This query returns 284 entities for v63:</p>\r\n<pre class=\"rresult\" style=\"height: 250px;\">╒════════════════╤════════════════════════╕\r\n│ component_stId │ component              │\r\n╞════════════════╪════════════════════════╡\r\n│ R-HSA-141412   │ CDC20 [cytosol]        │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174242   │ ANAPC7 [cytosol]       │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174211   │ ANAPC5 [cytosol]       │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174052   │ CDC26 [cytosol]        │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174244   │ UBE2C [cytosol]        │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174126   │ ANAPC11 [cytosol]      │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174156   │ CDC16 [cytosol]        │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174189   │ ANAPC1 [cytosol]       │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174100   │ UBE2E1 [cytosol]       │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174229   │ ANAPC2 [cytosol]       │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174168   │ ANAPC4 [cytosol]       │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174073   │ CDC27 [cytosol]        │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174137   │ CDC23 [cytosol]        │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174142   │ ANAPC10 [cytosol]      │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-174236   │ UBE2D1 [cytosol]       │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-976009   │ CBLB [cytosol]         │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-976042   │ MKRN1 [cytosol]        │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-939214   │ UBB(1-76) [cytosol]    │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-939213   │ UBB(77-152) [cytosol]  │\r\n├────────────────┼────────────────────────┤\r\n│ R-HSA-939239   │ UBC(533-608) [cytosol] │\r\n├────────────────┼────────────────────────┤\r\n│...             │...                     │\r\n└────────────────┴────────────────────────┘\r\n</pre>\r\n<h3><a href=\"#retrieving-pathways\" id=\"retrieving-pathways\">Retrieving pathways, subpathways and superpathways</a></h3>\r\n<p>In this example, we focus on the pathway \"<b>Class I MHC mediated antigen processing &amp; presentation</b>\" with identifier <b>R-HSA-983169</b>. To find out its subpathways, the slot to query is \"<b>hasEvent</b>\":</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Direct subpathways for the pathway with stable identifier R-HSA-198933\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})-[:hasEvent]-&gt;(sp:Pathway)\r\nRETURN p.stId AS Pathway, sp.stId AS SubPathway, sp.displayName as DisplayName\r\n</code></pre>\r\n<p>The result for v63 returns 3 subpathways:</p>\r\n<pre class=\"rresult\">╒════════════╤═════════════╤══════════════════════════════════════════════════════════════════════════╕\r\n│Pathway     │SubPathway   │DisplayName                                                               │\r\n╞════════════╪═════════════╪══════════════════════════════════════════════════════════════════════════╡\r\n│R-HSA-983169│R-HSA-983168 │Antigen processing: Ubiquitination &amp; Proteasome degradation               │\r\n├────────────┼─────────────┼──────────────────────────────────────────────────────────────────────────┤\r\n│R-HSA-983169│R-HSA-1236975│Antigen processing-Cross presentation                                     │\r\n├────────────┼─────────────┼──────────────────────────────────────────────────────────────────────────┤\r\n│R-HSA-983169│R-HSA-983170 │Antigen Presentation: Folding, assembly and peptide loading of class I MHC│\r\n└────────────┴─────────────┴──────────────────────────────────────────────────────────────────────────┘\r\n</pre>\r\n<p>It is important to note that subpathways might contain other subpathways, so to get ALL the supathways of R-HSA-198933, the query is as follows:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//ALL subpathways for the pathway with stable identifier R-HSA-198933\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})-[:hasEvent*]-&gt;(sp:Pathway)\r\nRETURN p.stId AS Pathway, sp.stId AS SubPathway, sp.displayName as DisplayName\r\n</code></pre>\r\n<p>In this case the number of subpathways is increased to 7:</p>\r\n<pre class=\"rresult\">╒════════════╤═════════════╤══════════════════════════════════════════════════════════════════════════╕\r\n│Pathway     │SubPathway   │DisplayName                                                               │\r\n╞════════════╪═════════════╪══════════════════════════════════════════════════════════════════════════╡\r\n│R-HSA-983169│R-HSA-983170 │Antigen Presentation: Folding, assembly and peptide loading of class I MHC│\r\n├────────────┼─────────────┼──────────────────────────────────────────────────────────────────────────┤\r\n│R-HSA-983169│R-HSA-1236975│Antigen processing-Cross presentation                                     │\r\n├────────────┼─────────────┼──────────────────────────────────────────────────────────────────────────┤\r\n│R-HSA-983169│R-HSA-1236978│Cross-presentation of soluble exogenous antigens (endosomes)              │\r\n├────────────┼─────────────┼──────────────────────────────────────────────────────────────────────────┤\r\n│R-HSA-983169│R-HSA-1236977│Endosomal/Vacuolar pathway                                                │\r\n├────────────┼─────────────┼──────────────────────────────────────────────────────────────────────────┤\r\n│R-HSA-983169│R-HSA-1236974│ER-Phagosome pathway                                                      │\r\n├────────────┼─────────────┼──────────────────────────────────────────────────────────────────────────┤\r\n│R-HSA-983169│R-HSA-1236973│Cross-presentation of particulate exogenous antigens (phagosomes)         │\r\n├────────────┼─────────────┼──────────────────────────────────────────────────────────────────────────┤\r\n│R-HSA-983169│R-HSA-983168 │Antigen processing: Ubiquitination &amp; Proteasome degradation               │\r\n└────────────┴─────────────┴──────────────────────────────────────────────────────────────────────────┘\r\n</pre>\r\n<p>Following the same approach, retrieving the superpathway is as easy as changing the direction of the edge in the query:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//Direct superpathway for the pathway with stable identifier R-HSA-198933\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})&lt;-[:hasEvent]-(sp:Pathway)\r\nRETURN p.stId AS Pathway, sp.stId AS SuperPathway, sp.displayName as DisplayName\r\n</code></pre>\r\n<p>It will then retrieve the only one pathway containing R-HSA-198933:</p>\r\n<pre class=\"rresult\">╒════════════╤═════════════╤══════════════════════╕\r\n│Pathway     │SuperPathway │DisplayName           │\r\n╞════════════╪═════════════╪══════════════════════╡\r\n│R-HSA-983169│R-HSA-1280218│Adaptive Immune System│\r\n└────────────┴─────────────┴──────────────────────┘\r\n</pre>\r\n<p>As subpathways, the superpathways might have other superpathways, so following the \"<b>hasEvent</b>\" slot recursively will show ALL the superpathways up to the root:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//ALL superpathways for the pathway with stable identifier R-HSA-198933\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})&lt;-[:hasEvent*]-(sp:Pathway)\r\nRETURN p.stId AS Pathway, sp.stId AS SuperPathway, sp.displayName as DisplayName\r\n</code></pre>\r\n<p>There are 2 superpathways for R-HSA-198933 in v63:</p>\r\n<pre class=\"rresult\">╒════════════╤═════════════╤══════════════════════╕\r\n│Pathway     │SuperPathway │DisplayName           │\r\n╞════════════╪═════════════╪══════════════════════╡\r\n│R-HSA-983169│R-HSA-1280218│Adaptive Immune System│\r\n├────────────┼─────────────┼──────────────────────┤\r\n│R-HSA-983169│R-HSA-168256 │Immune System         │\r\n└────────────┴─────────────┴──────────────────────┘\r\n</pre>\r\n<h3><a href=\"#retrieving-reactions\" id=\"retrieving-reactions\">Retrieving the reactions for a given pathway</a></h3>\r\n<p>Continuing with the pathway \"<b>Class I MHC mediated antigen processing &amp; presentation</b>\" with identifier <b>R-HSA-983169</b>, to get ALL the reactions contained either directly in it or as part of any of its subpathways, the query has to recursively traverse the \"<b>hasEvent</b>\" slot:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//All reactions for the pathway with stable identifier R-HSA-198933\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})-[:hasEvent*]-&gt;(rle:ReactionLikeEvent)\r\nRETURN p.stId AS Pathway, rle.stId AS Reaction, rle.displayName AS ReactionName\r\n</code></pre>\r\n<p>As shown in the table below, this pathway contains 51 reactions for v63:</p>\r\n<pre class=\"rresult\" style=\"height: 250px;\">╒══════════════╤═══════════════╤═══════════════════════════════════════════════════════════════════════════════════╕\r\n│ Pathway      │ Reaction      │ ReactionName                                                                      │\r\n╞══════════════╪═══════════════╪═══════════════════════════════════════════════════════════════════════════════════╡\r\n│ R-HSA-983169 │ R-HSA-983148  │ Interaction of Erp57 with MHC class I HC                                          │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-8951499 │ Loading of antigenic peptides on to class I MHC                                   │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983146  │ Interaction of beta-2-microglobulin (B2M) chain with  class I HC                  │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983145  │ Binding of newly synthesized MHC class I heavy chain (HC) with calnexin           │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983144  │ Transport of Antigen peptide in to ER                                             │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-203979  │ Coat Assembly                                                                     │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983142  │ Formation of peptide loading complex (PLC)                                        │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983427  │ Expression of peptide bound class I MHC on cell surface                           │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983138  │ Transport of MHC heterotrimer to ER exit site                                     │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983426  │ Capturing cargo and formation of prebudding complex                               │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983425  │ Recruitment of Sec31p:Sec13p to prebudding complex and formation of COPII vesicle │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983424  │ Budding of COPII coated vesicle                                                   │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983422  │ Disassembly of COPII coated vesicle                                               │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983421  │ Journey of cargo through Golgi complex                                            │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-983169 │ R-HSA-983161  │ Dissociation of the Antigenic peptide:MHC:B2M peptide loading complex             │\r\n├──────────────┼───────────────┼───────────────────────────────────────────────────────────────────────────────────┤\r\n│...           │...            │...                                                                                │\r\n└──────────────┴───────────────┴───────────────────────────────────────────────────────────────────────────────────┘\r\n</pre>\r\n<h3><a href=\"#retrieving-participants\" id=\"retrieving-participants\">Retrieving the participants of a Reaction</a></h3>\r\n<p>In this case let\'s use the reaction \"<b>IKKB phosphorylates SNAP23</b>\" with identifier <b>R-HSA-8863895</b>. Reactions have inputs, outputs, catalysts and regulations, so to know the participants of a reaction, all these slots have to be taken into account. Please note that the physical entity acting as catalyst is stored in the \"physicalEntity\" slot of the class \"CatalystActivity\" and the one belonging to the regulation is stored in the \"regulator\" slot of the \"Regulation\" class. So the query is as follows:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//First level paticipating molecules for reaction R-HSA-8863895\r\nMATCH (r:ReactionLikeEvent{stId:\"R-HSA-8863895\"})-[:input|output|catalystActivity|physicalEntity|regulatedBy|regulator*]-&gt;(pe:PhysicalEntity)\r\nRETURN DISTINCT r.stId AS Reaction, pe.stId as Participant, pe.displayName AS DisplayName\r\n</code></pre>\r\n<p>The result of it is 6 physical entities, where two of them are complexes:</p>\r\n<pre class=\"rresult\">╒═════════════╤═════════════╤═══════════════════════════════════════════════════╕\r\n│Reaction     │Participant  │DisplayName                                        │\r\n╞═════════════╪═════════════╪═══════════════════════════════════════════════════╡\r\n│R-HSA-8863895│R-HSA-168113 │CHUK:IKBKB:IKBKG [cytosol]                         │\r\n├─────────────┼─────────────┼───────────────────────────────────────────────────┤\r\n│R-HSA-8863895│R-ALL-113592 │ATP [cytosol]                                      │\r\n├─────────────┼─────────────┼───────────────────────────────────────────────────┤\r\n│R-HSA-8863895│R-HSA-8863966│SNAP23 [phagocytic vesicle membrane]               │\r\n├─────────────┼─────────────┼───────────────────────────────────────────────────┤\r\n│R-HSA-8863895│R-HSA-8863923│p-S95-SNAP23 [phagocytic vesicle membrane]         │\r\n├─────────────┼─────────────┼───────────────────────────────────────────────────┤\r\n│R-HSA-8863895│R-ALL-29370  │ADP [cytosol]                                      │\r\n├─────────────┼─────────────┼───────────────────────────────────────────────────┤\r\n│R-HSA-8863895│R-HSA-937033 │oligo-MyD88:Mal:BTK:activated TLR [plasma membrane]│\r\n└─────────────┴─────────────┴───────────────────────────────────────────────────┘\r\n</pre>\r\n<p>As shown above, to break down complexes and sets into their participants, the \"hasComponent\", \"hasMember\" and \"hasCandidate\" slots have to be taken into account. Adding them into the previous query will retrieve ALL the participants of the reaction:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//ALL paticipating molecules for reaction R-HSA-8863895\r\nMATCH (r:ReactionLikeEvent{stId:\"R-HSA-8863895\"})-[:input|output|catalystActivity|physicalEntity|regulatedBy|regulator|hasComponent|hasMember|hasCandidate*]-&gt;(pe:PhysicalEntity)\r\nRETURN DISTINCT r.stId AS Reaction, pe.stId as Participant, pe.displayName AS DisplayName\r\n</code></pre>\r\n<p>With the modifications, the result of the query goes up to 43 physical entities in v63:</p>\r\n<pre class=\"rresult\" style=\"height: 250px;\">╒═══════════════╤═══════════════╤══════════════════════════════════════════════════════════════════════════════╕\r\n│ Reaction      │ Participant   │ DisplayName                                                                  │\r\n╞═══════════════╪═══════════════╪══════════════════════════════════════════════════════════════════════════════╡\r\n│ R-HSA-8863895 │ R-HSA-168113  │ CHUK:IKBKB:IKBKG [cytosol]                                                   │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-168114  │ IKBKB [cytosol]                                                              │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-168104  │ CHUK [cytosol]                                                               │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-168108  │ IKBKG [cytosol]                                                              │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-ALL-113592  │ ATP [cytosol]                                                                │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-8863966 │ SNAP23 [phagocytic vesicle membrane]                                         │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-8863923 │ p-S95-SNAP23 [phagocytic vesicle membrane]                                   │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-ALL-29370   │ ADP [cytosol]                                                                │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-937033  │ oligo-MyD88:Mal:BTK:activated TLR [plasma membrane]                          │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-937013  │ MyD88 oligomer [plasma membrane]                                             │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-937017  │ MYD88 [plasma membrane]                                                      │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-2201325 │ activated TLR2/4:p-4Y-MAL:PI(4,5)P2:BTK [plasma membrane]                    │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-5365824 │ p-4Y-TIRAP:PI(4,5)P2 [plasma membrane]                                       │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-ALL-179856  │ PI(4,5)P2 [plasma membrane]                                                  │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-2201321 │ p-4Y-TIRAP [plasma membrane]                                                 │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-181230  │ Activated TLR1:2 or TLR 2:6 heterodimers or TLR4 homodimer [plasma membrane] │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-181410  │ TLR6:TLR2:ligand:CD14:CD36 [plasma membrane]                                 │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-2559461 │ TLR6/2 ligand:CD14:CD36 [plasma membrane]                                    │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│ R-HSA-8863895 │ R-HSA-166033  │ GPIN-CD14(20-345) [plasma membrane]                                          │\r\n├───────────────┼───────────────┼──────────────────────────────────────────────────────────────────────────────┤\r\n│...            │...            │...                                                                           │\r\n└───────────────┴───────────────┴──────────────────────────────────────────────────────────────────────────────┘\r\n</pre>\r\n<h3><a href=\"#joining-pieces\" id=\"joining-pieces\">Joining the pieces: Participating molecules for a pathway</a></h3>\r\n<p>The aim of this tutorial was to describe a number of different queries to the Reactome Graph Database that joined together would retrieve the resource and identifier of each participating molecule for a given Pathway. This final example will demonstrate how to concatenate all the individual query described previously into a single query.</p>\r\n<p>Starting from the pathway \"<b>Class I MHC mediated antigen processing &amp; presentation</b>\" with identifier <b>R-HSA-983169</b>, first we need to find out all the reactions contained in it. For each reaction we want to find their participants and for the cases of complexes and sets, we want to break them down into single physical entities. Finally, for each physical entity we are interested in their identifier and resource:</p>\r\n<pre class=\"rcypher\"><code class=\"language-cypher\" data-lang=\"cypher\">//ALL paticipating molecules for pathway R-HSA-983169\r\nMATCH (p:Pathway{stId:\"R-HSA-983169\"})-[:hasEvent*]-&gt;(rle:ReactionLikeEvent),\r\n      (rle)-[:input|output|catalystActivity|physicalEntity|regulatedBy|regulator|hasComponent|hasMember|hasCandidate*]-&gt;(pe:PhysicalEntity),\r\n      (pe)-[:referenceEntity]-&gt;(re:ReferenceEntity)-[:referenceDatabase]-&gt;(rd:ReferenceDatabase)\r\nRETURN DISTINCT re.identifier AS Identifier, rd.displayName AS Database\r\n</code></pre>\r\n<p>For version 63, the pathway has 463 participating molecules as shown below:</p>\r\n<pre class=\"rresult\" style=\"height: 250px;\">╒════════════╤══════════╕\r\n│ Identifier │ Database │\r\n╞════════════╪══════════╡\r\n│ P11021     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P27824     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30501     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30486     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P01893     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30447     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30685     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P18465     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P18464     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30460     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30490     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30495     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30493     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P13747     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30488     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30511     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30483     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P30462     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ P04439     │ UniProt  │\r\n├────────────┼──────────┤\r\n│ ...        │ ...      │\r\n└────────────┴──────────┘\r\n</pre>\r\n<p>This concludes the introductory tutorial on how to build a Cypher query to retrieve all the participating molecules for a given pathway. For questions or suggestions, please get in touch our <a href=\"mailto:help@reactome.org\">help desk</a></p>\r\n<style type=\"text/css\">@import url(\'https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.11.0/codemirror.min.css\');\r\n@import url(\'https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.11.0/theme/neo.min.css\');\r\n@import url(\'https://fonts.googleapis.com/css?family=Open+Sans:400,300,400italic,600,300italic\');</style>\r\n<script src=\"https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.11.0/codemirror.min.js\" type=\"text/javascript\"></script>\r\n<script src=\"https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.11.0/addon/runmode/runmode.min.js\" type=\"text/javascript\"></script>\r\n<script src=\"https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.11.0/mode/cypher/cypher.min.js\" type=\"text/javascript\"></script>\r\n<script src=\"https://neo4j.com/docs/developer-manual/current/javascript/colorize.js\" type=\"text/javascript\"></script>\r\n<script type=\"text/javascript\">function tabTheSource($content) {\r\n    var storedLanguage = getCodeExampleLanguage();\r\n    var LANGUAGES = {\r\n        \'dotnet\': \'C#\',\r\n        \'java\': \'Java\',\r\n        \'javascript\': \'JavaScript\',\r\n        \'python\': \'Python\'\r\n    };\r\n    var $UL = jQuery(\'<ul class=\"nav nav-tabs\" role=\"tablist\"/>\');\r\n    var $LI = jQuery(\'<li role=\"presentation\"/>\');\r\n    var $A = jQuery(\'<a role=\"tab\" data-toggle=\"tab\" style=\"text-decoration:none;\"/>\');\r\n    var $WRAPPER = jQuery(\'<div class=\"tab-content content\"/>\');\r\n    var snippets = [];\r\n    var languageEventElements = {};\r\n\r\n    var focusSelectedExample = function(e) {\r\n        var target = jQuery(e.target);\r\n        var beforeTop = target.offset().top - jQuery(window).scrollTop();\r\n        setTimeout(function(){\r\n            var newTop = target.offset().top - beforeTop;\r\n            jQuery(\'html,body\').scrollTop(newTop);\r\n        }, 1);\r\n    }\r\n\r\n    var selectTab = function (e) {\r\n        var language = jQuery(e.target).data(\'lang\');\r\n        var $elements = languageEventElements[language];\r\n        for (var j = 0; j < $elements.length; j++) {\r\n            $elements[j].tab(\'show\');\r\n        }\r\n        if (storageAvailable(\'sessionStorage\')) {\r\n            sessionStorage.setItem(\'code_example_language\', language);\r\n        }\r\n    }\r\n\r\n    jQuery(\'div.tabbed-example\', $content).each(function () {\r\n        var $exampleBlock = $(this);\r\n        var title = $exampleBlock.children(\'div.example-title\', this).first().text();\r\n        var languages = [];\r\n        var $languageBlocks = {};\r\n        jQuery(this).children(\'div.tabbed-example-contents\').children(\'div.listingblock,div.informalexample[class*=\"include-with\"]\').each(function () {\r\n            var $this = jQuery(this);\r\n            var language = undefined;\r\n            if ($this.hasClass(\'listingblock\')) {\r\n                language = jQuery(\'code\', this).data(\'lang\');\r\n            } else {\r\n                for (var key in LANGUAGES) {\r\n                    if ($this.hasClass(\'include-with-\' + key)) {\r\n                        language = key;\r\n                        break;\r\n                    }\r\n                }\r\n            }\r\n            languages.push(language);\r\n            $languageBlocks[language] = jQuery(this);\r\n        });\r\n        if (languages.length > 1) {\r\n            snippets.push({\r\n                \'$exampleBlock\': $exampleBlock,\r\n                \'languages\': languages,\r\n                \'$languageBlocks\': $languageBlocks\r\n            });\r\n        }\r\n    });\r\n\r\n    var idNum = 0;\r\n    for (var ix = 0; ix < snippets.length; ix++) {\r\n        var snippet = snippets[ix];\r\n        var languages = snippet.languages;\r\n        languages.sort();\r\n        var $languageBlocks = snippet.$languageBlocks;\r\n        var $exampleBlock = snippet.$exampleBlock;\r\n        var idBase = \'tabbed-example-\' + idNum++;\r\n        var $wrapper = $WRAPPER.clone();\r\n        var $ul = $UL.clone();\r\n\r\n        for (var i = 0; i < languages.length; i++) {\r\n            var language = languages[i];\r\n            var $content = jQuery($languageBlocks[language]);\r\n            var id;\r\n            if ($content.attr(\'id\')) {\r\n                id = $content.attr(\'id\');\r\n            } else {\r\n                id = idBase + \'-\' + language;\r\n                $content.attr(\'id\', id);\r\n            }\r\n            $content.addClass(\'tab-pane\').css(\'position\', \'relative\');\r\n            var $li = $LI.clone();\r\n            var $a = $A.clone();\r\n\r\n            $a.attr(\'href\', \'#\' + id).text(LANGUAGES[language]).data(\'lang\', language).on(\'shown.bs.tab\', selectTab).on(\'click\', focusSelectedExample);\r\n\r\n            if (language in languageEventElements) {\r\n                languageEventElements[language].push($a);\r\n            } else {\r\n                languageEventElements[language] = [$a];\r\n            }\r\n            $wrapper.append($content);\r\n\r\n            if (storedLanguage) {\r\n                if (language === storedLanguage) {\r\n                    $li.addClass(\'active\');\r\n                    $content.addClass(\'active\');\r\n                }\r\n            } else if (i === 0) {\r\n                $li.addClass(\'active\');\r\n                $content.addClass(\'active\');\r\n            }\r\n\r\n            $li.append($a);\r\n            $ul.append($li);\r\n        }\r\n        $exampleBlock.children(\'div.example-title\', this).first().after($ul);\r\n        $exampleBlock.append($wrapper);\r\n    }\r\n}\r\n\r\nfunction storageAvailable(type) {\r\n    try {\r\n        var storage = window[type];\r\n        var x = \'__storage_test__\';\r\n        storage.setItem(x, x);\r\n        storage.removeItem(x);\r\n        return true;\r\n    }\r\n    catch(e) {\r\n        return false;\r\n    }\r\n}\r\n\r\nfunction getCodeExampleLanguage() {\r\n    return storageAvailable(\'sessionStorage\') ? sessionStorage.getItem(\'code_example_language\') || false : false;\r\n}</script>\r\n<script type=\"text/javascript\">jQuery(document).ready(function($){\r\nCodeMirror.colorize();\r\ntabTheSource(jQuery(\'#rcypher\'));\r\n});</script>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-02 15:08:40','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,7,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(55,630,'Icons Info ','icons-info','<p>{loadposition icon-lib-pos-1}</p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-05 15:23:33','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"0\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','','Page with 2 options - EHLDs and Icon Library'),(56,631,'Pathways Overview JS','pathways-overview-js','<style type=\"text/css\">pre {\r\n    word-wrap: initial;\r\n    white-space: pre;\r\n    word-break: break-all;\r\n}\r\n}</style>\r\n<div>Reactome\'s Fireworks JS widget is our pathways overview viewer in an ordinary JavaScript API. It is meant to be reused by third party resources in order to display Reactome pathways overview directly in their web pages and enable users to interact with them.</div>\r\n<p>&nbsp;</p>\r\n<div id=\"fireworksHolder\" class=\"diagram\"></div>\r\n<div>\r\n<h3><a href=\"#reuse-pathways-widget\" id=\"reuse-pathways-widget\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Reusing Reactome\'s Pathways Overview Widget?</a></h3>\r\n<p>To reuse our viewer you need to follow the following steps</p>\r\n<p>1. Include the fireworks javascript dependency in you HTML header</p>\r\n<pre class=\"code\">    &lt;script type=\"text/javascript\" language=\"javascript\" src=\"https://reactome.org/FireworksJs/fireworks/fireworks.nocache.js\"&gt;&lt;/script&gt;\r\n        </pre>\r\n<p>2. Add a place holder in the body of your web page</p>\r\n<pre class=\"prettyprint code\">    &lt;div id=\"fireworksHolder\"&gt;&lt;/div&gt;\r\n        </pre>\r\n<p>3. Set a proxy in your server under \"/reactome\" pointing to \"https://reactome.org\"</p>\r\n<p>4. Create and initialise the pathways overview from your javascript code</p>\r\n<pre class=\"prettyprint code\">    //Creating the Reactome pathways overview widget\r\n    function onReactomeFireworksReady(){  //This function is automatically called when the widget code is ready to be used\r\n        var fireworks = Reactome.Fireworks.create({\r\n            \"placeHolder\" : \"fireworksHolder\",\r\n            \"width\" : 930,\r\n            \"height\" : 500\r\n        });\r\n\r\n        //Adding different listeners\r\n\r\n        fireworks.onFireworksLoaded(function (loaded) {\r\n            console.info(\"Loaded \", loaded);\r\n        });\r\n\r\n        fireworks.onNodeHovered(function (hovered){\r\n            console.info(\"Hovered \", hovered);\r\n        });\r\n\r\n        fireworks.onNodeSelected(function (selected){\r\n            console.info(\"Selected \", selected);\r\n        });\r\n    }\r\n        </pre>\r\n<a>&nbsp;</a>\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>FireworksJs API</a></h3>\r\n<p>The current implementation supports the following listeners and methods:</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome\">\r\n<thead>\r\n<tr><th>Method</th><th>Params</th><th>Description</th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td data-label=\"Method\"><b>create</b> :: Constructor<br /> &nbsp;&nbsp;Reactome.Fireworks.create(params);</td>\r\n<td data-label=\"Params\"><b>param</b> :: json object<br /> &nbsp;{<br /> &nbsp;&nbsp;&nbsp;\'proxyPrefix\' : string,<br /> &nbsp;&nbsp;&nbsp;\'placeHolder\' : string,<br /> &nbsp;&nbsp;&nbsp;\'width\' : int (optional),<br /> &nbsp;&nbsp;&nbsp;\'height\' : int (optional)<br /> &nbsp;}</td>\r\n<td data-label=\"Description\">Creates and returns a new Reactome.Fireworks object</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>flagItems(String identifier)</b> :: void</td>\r\n<td data-label=\"Params\">Item identifier <br /> <b>identifier</b> : string</td>\r\n<td data-label=\"Description\">Flags pathways where the identifier is found. It accepts main identifiers but also cross references, gene names and physical entity stable identifiers</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>highlightNode(stId)</b> :: void</td>\r\n<td data-label=\"Params\">Item stable identifier<br /> <b>stId</b> : string</td>\r\n<td data-label=\"Description\">Highlights the specified item if it exists in the fireworks</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetAnalysis()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Resets the analysis overlay</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetHighlight()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Clears the highlights in the fireworks</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetSelection()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Clears the selection in the fireworks</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resize(width, height)</b> :: void</td>\r\n<td data-label=\"Params\"><b>width</b>: int<br /> <b>height</b>: int</td>\r\n<td data-label=\"Description\">Resizes the viewport to the specified with and height</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>selectNode(stId)</b> :: void</td>\r\n<td data-label=\"Params\">Item stable identifier<br /> <b>stId</b> : string</td>\r\n<td data-label=\"Description\">Selects the specified item if it exists in the fireworks</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>setAnalysisToken(token, resource)</b> :: void</td>\r\n<td data-label=\"Params\">Analysis token<br /> <b>token</b> : string<br /> Resource<br /> <b>resource</b> : string</td>\r\n<td data-label=\"Description\">Overlays the analysis result corresponding to the specified (token, resource)</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>showAll()</b> :: void</td>\r\n<td data-label=\"Params\">&nbsp;</td>\r\n<td data-label=\"Description\">Shows all the pathways in the viewport</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onNodesFlaggedReset(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters</td>\r\n<td data-label=\"Description\">The function is called when the user resets the flagged items</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onNodeSelected(function(obj))</b> :: void</td>\r\n<td data-label=\"Params\"><b>obj</b> is selected item:<br /> &nbsp;{<br /> &nbsp;&nbsp;&nbsp;\'stId\' : string,<br /> &nbsp;&nbsp;&nbsp;\'displayName\' : string,<br /> &nbsp;&nbsp;&nbsp;\'schemaClass\' : string,<br /> &nbsp;}</td>\r\n<td data-label=\"Description\">The function is called when an object in the fireworks is selected by the user action</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onNodeHovered(function(obj))</b> :: void</td>\r\n<td data-label=\"Params\"><b>obj</b> is hovered item:<br /> &nbsp;{<br /> &nbsp;&nbsp;&nbsp;\'stId\' : string,<br /> &nbsp;&nbsp;&nbsp;\'displayName\' : string,<br /> &nbsp;&nbsp;&nbsp;\'schemaClass\' : string,<br /> &nbsp;}</td>\r\n<td data-label=\"Description\">The function is called when an object in the fireworks is hovered by the user action</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onFireworksLoaded(function(id))</b> :: void</td>\r\n<td data-label=\"Params\">The db identifier of the loaded species in fireworks<br /> <b>id</b>: string</td>\r\n<td data-label=\"Description\">The function is called when a fireworks is loaded in the viewer</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onAnalysisReset(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters</td>\r\n<td data-label=\"Description\">The function is called when the users resets the analysis overlay<br /><br /> fireworks.onAnalysisReset(function(){ /* your code here */ });</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onCanvasNotSupported(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters.</td>\r\n<td data-label=\"Description\">The function is called when the browser doesn\'t support HTML5 Canvas so the viewer cannot be instantiated<br /><br /> fireworks.onCanvasNotSupported(function(){ /* your code here */ });</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<script type=\"text/javascript\">function getMainContentWidth(){\r\n   var DIV = \"fav-maincontent\";\r\n   var style = window.getComputedStyle(document.getElementById(DIV), null);\r\n   var width = style.getPropertyValue(\"width\").replace(\"px\",\"\");\r\n   var paddingLeft = style.getPropertyValue(\"padding-left\").replace(\"px\",\"\"); \r\n   var paddingRight = style.getPropertyValue(\"padding-right\").replace(\"px\",\"\");\r\n   return width - paddingLeft - paddingRight;\r\n}\r\n\r\njQuery( window ).resize(function() {\r\n  fireworks.resize(getMainContentWidth(), 600);  \r\n});\r\n\r\nfunction onReactomeFireworksReady() {\r\n            fireworks = Reactome.Fireworks.create({\r\n                \"placeHolder\": \"fireworksHolder\",\r\n                \"width\": getMainContentWidth(),\r\n                \"height\": 600\r\n            });\r\n            fireworks.onFireworksLoaded(function (loaded) {\r\n                console.info(\"Loaded \", loaded);\r\n            });\r\n            fireworks.onNodeHovered(function (hovered) {\r\n                console.info(\"Hovered \", hovered);\r\n            });\r\n            fireworks.onNodeSelected(function (selected) {\r\n                console.info(\"Selected \", selected);\r\n            });\r\n            //fireworks.setAnalysisToken(\"MjAyMDEwMjAxMTU4NDFfMjkxMDA%253D\",\"TOTAL\");\r\n        }</script>\r\n<script src=\"/FireworksJs/fireworks/fireworks.nocache.js\" type=\"text/javascript\" language=\"javascript\"></script>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-07 10:10:59','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(57,632,'Pathways Overview GWT','pathways-overview-gwt','<p>Reactome’s Fireworks GWT widget is our original implementation of the pathways overview in GWT. It is meant to be reused by third party resources in order to display Reactome pathways overview directly in their web pages and enable users to interact with them.</p>\r\n<p>{loadposition dev-guide-po-gwt-1}</p>\r\n<div>\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p><a href=\"http://www.gwtproject.org/\">GWT</a> is a development toolkit for building and optimizing complex browser-based applications. Its goal is to enable productive development of high-performance web applications without the developer having to be an expert in browser quirks, XMLHttpRequest, and JavaScript. GWT is used by many products at Google, including AdWords, AdSense, Flights, Hotel Finder, Offers, Wallet, Blogger. It’s open source, completely free, and used by thousands of developers around the world.</p>\r\n<p> </p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>We assume that you already have your <a href=\"http://www.gwtproject.org/gettingstarted.html\">GWT project</a> up and running configured with the <a href=\"http://www.gwtproject.org/doc/latest/RefCommandLineTools.html\"> MAVEN archetype</a>. So the first step is to add the EBI Nexus repository in your pom.xml file:</p>\r\n<pre><code>&lt;repositories&gt;\r\n    ...\r\n    &lt;!-- EBI repo --&gt;\r\n    &lt;repository&gt;\r\n        &lt;id&gt;nexus-ebi-repo&lt;/id&gt;\r\n        &lt;name&gt;The EBI internal repository&lt;/name&gt;\r\n        &lt;url&gt;http://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-repo/&lt;/url&gt;\r\n        &lt;releases&gt;\r\n            &lt;enabled&gt;true&lt;/enabled&gt;\r\n        &lt;/releases&gt;\r\n        &lt;snapshots&gt;\r\n            &lt;enabled&gt;false&lt;/enabled&gt;\r\n        &lt;/snapshots&gt;\r\n    &lt;/repository&gt;\r\n    &lt;!-- EBI SNAPSHOT repo --&gt;\r\n    &lt;repository&gt;\r\n        &lt;id&gt;nexus-ebi-snapshot-repo&lt;/id&gt;\r\n        &lt;name&gt;The EBI internal snapshot repository&lt;/name&gt;\r\n        &lt;url&gt;http://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-snapshots/&lt;/url&gt;\r\n        &lt;releases&gt;\r\n            &lt;enabled&gt;false&lt;/enabled&gt;\r\n        &lt;/releases&gt;\r\n        &lt;snapshots&gt;\r\n            &lt;enabled&gt;true&lt;/enabled&gt;\r\n        &lt;/snapshots&gt;\r\n    &lt;/repository&gt;\r\n&lt;/repositories&gt;\r\n</code></pre>\r\n<p>Once the repository is in place, the next thing to do is adding the Reactome\'s pathways overview dependency:</p>\r\n<pre><code>&lt;dependencies&gt;\r\n    ...\r\n    &lt;dependency&gt;\r\n        &lt;groupId&gt;org.reactome.web&lt;/groupId&gt;\r\n        &lt;artifactId&gt;fireworks&lt;/artifactId&gt;\r\n        &lt;version&gt;1.0.0&lt;/version&gt;\r\n    &lt;/dependency&gt;\r\n&lt;dependencies&gt;\r\n</code></pre>\r\n<p>Before writing the code to use the pathways overview, add the dependency in your <b>gwt.xml</b> file, so the compiler will find the pathways overview:</p>\r\n<pre><code>&lt;inherits name=\"org.reactome.web.fireworks.FireworksViewer\" /&gt;\r\n</code></pre>\r\n<p>And finally, assuming the widget is now going to be displayed by its own, the code in your <b>EntryPoint</b> class would look like:</p>\r\n<pre><code>/**\r\n * For test purposes we assume the pathway overview loads as a single item in the web page.\r\n *\r\n * Entry point classes define onModuleLoad().\r\n */\r\npublic class Main implements EntryPoint {\r\n\r\n    private final FirewowksViewer fireworks;\r\n\r\n    public Main() {\r\n        //We create the fireworks. Please note that different pathways overview cannot be loaded in the same instance\r\n        fireworks = FireowksFactory.createFireworksViewer();\r\n    }\r\n\r\n    @Override\r\n    public void onModuleLoad() {\r\n        Scheduler.get().scheduleDeferred(new Scheduler.ScheduledCommand() {\r\n            @Override\r\n            public void execute() {\r\n                FireworksFactory.CONSOLE_VERBOSE = true;   //This is optional (for dev purposes)\r\n                FireworksFactory.EVENT_BUS_VERBOSE = true; //This is optional (for dev purposes)\r\n\r\n                //For this use case we add the firewoks as the main and only object of the webpage\r\n                RootLayoutPanel.get().add(fireworks);\r\n            }\r\n        });\r\n    }\r\n}\r\n</code></pre>\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>API</a></h3>\r\n<p>The available methods are defined in the <b>DiagramViewer</b> interface in the <b>org.reactome.web.diagram.client</b> package. These are:</p>\r\n<pre><code>    //subscribe to this event to be notified when the user closes the analysis overlay\r\n    HandlerRegistration addAnalysisResetHandler(AnalysisResetHandler handler);\r\n\r\n    //Subscribe to this event to be notified if the browser does not support canvas, so diagrams can not be displayed\r\n    HandlerRegistration addCanvasNotSupportedHandler(CanvasNotSupportedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the overview is displayed in the viewport\r\n    HandlerRegistration addFireworksLoaded(FireworksLoadedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when\r\n    HandlerRegistration addExpressionColumnChangedHandler(ExpressionColumnChangedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user hovers over an node in the overview\r\n    HandlerRegistration addNodeHoverHandler(NodeHoverHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user leave the hovered node in the overview\r\n    HandlerRegistration addNodeHoverResetHandler(NodeHoverResetHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user opens a node in the overview\r\n    HandlerRegistration addNodeOpenedHandler(NodeOpenedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user selects a node in the overview\r\n    HandlerRegistration addNodeSelectedHandler(NodeSelectedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user unselects a node in the overview\r\n    HandlerRegistration addNodeSelectedResetHandler(NodeSelectedResetHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user changes the colour profile\r\n    HandlerRegistration addProfileChangedHandler(ProfileChangedHandler handler);\r\n\r\n    //Returns the selected node\r\n    Node getSelected();\r\n\r\n    //Highlights the node with the provided stable identifier\r\n    void highlightNode(String stableIdentifier);\r\n\r\n    //Highlights the node with the provided stable identifier\r\n    void highlightNode(Long dbIdentifier);\r\n\r\n    //Zooms-in to the node with the provided stable identifier\r\n    void openPathway(String stableIdentifier);\r\n\r\n    //Zooms-in to the node with the provided database identifier\r\n    void openPathway(Long dbIdentifier);\r\n\r\n    //Resets the analysis overlay\r\n    void resetAnalysis();\r\n\r\n    //Resets the highlighted nodes\r\n    void resetHighlight();\r\n\r\n    //Resets the selected nodes\r\n    void resetSelection();\r\n\r\n    //Selects the node with the provided stable identifier\r\n    void selectNode(String stableIdentifier);\r\n\r\n    //Selects the node with the provided database identifier\r\n    void selectNode(Long dbIdentifier);\r\n\r\n    //Sets the analysis overlay for a given \"token\" and \"resource\" (this will query the Analysis Service)\r\n    void setAnalysisToken(String token, String resource);\r\n\r\n    //Show all the nodes in the viewport\r\n    void showAll();</code></pre>\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<ul style=\"list-style: none;\">\r\n<li style=\"line-height: 30px;\"><a href=\"https://github.com/reactome-pwp/fireworks\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Fireworks code</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"http://github.com/reactome-pwp\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>Reactome Pathways Portal code</a></li>\r\n</ul>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-07 10:16:02','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(58,633,'Diagram JS','diagram-js','<style type=\"text/css\">pre, pre code {\r\n    word-wrap: initial;\r\n    white-space: pre;\r\n    word-break: break-all;\r\n}\r\n}</style>\r\n<p>Reactome\'s DiagramJs widget is our diagram viewer in an ordinary JavaScript API. It is meant to be reused by third party resources in order to display Reactome Pathway Diagrams directly in their web pages and enable users to interact with them.</p>\r\n<div id=\"diagramHolder\" class=\"diagram\"></div>\r\n<p> </p>\r\n<h3><a href=\"#reuse-diagram-widget\" id=\"reuse-diagram-widget\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Reusing Reactome\'s Diagram Widget?</a></h3>\r\n<p>To reuse our viewer you need to follow the following steps</p>\r\n<p>1. Include the diagram javascript dependency in you HTML header</p>\r\n<pre><code>&lt;script type=\"text/javascript\" language=\"javascript\" src=\"https://reactome.org/DiagramJs/diagram/diagram.nocache.js\"&gt;&lt;/script&gt;</code></pre>\r\n<p>2. Add a place holder in the body of your web page</p>\r\n<pre><code>&lt;div id=\"diagramHolder\"&gt;&lt;/div&gt;</code></pre>\r\n<p>3. Create and initialise the diagram viewer from your javascript code</p>\r\n<pre><code>    //Creating the Reactome Diagram widget\r\n    //Take into account a proxy needs to be set up in your server side pointing to www.reactome.org\r\n    function onReactomeDiagramReady(){  //This function is automatically called when the widget code is ready to be used\r\n        var diagram = Reactome.Diagram.create({\r\n            \"placeHolder\" : \"diagramHolder\",\r\n            \"width\" : 950, // minimum recommended width\r\n            \"height\" : 500\r\n        });\r\n\r\n        //Initialising it to the \"Hemostasis\" pathway\r\n        diagram.loadDiagram(\"R-HSA-109582\");\r\n\r\n        //Adding different listeners\r\n\r\n        diagram.onDiagramLoaded(function (loaded) {\r\n            console.info(\"Loaded \", loaded);\r\n            diagram.flagItems(\"FYN\");<br /><code><code>            </code>if (loaded == \"R-HSA-109582\") diagram.selectItem(\"R-HSA-109582\");<br /></code><code>        </code>}); <br /><br /><code>        </code>diagram.onObjectHovered(function (hovered){ <br /><code>            </code>console.info(\"Hovered \", hovered); <br /><code>        </code><code></code>}); <br /><br /><code>        </code>diagram.onObjectSelected(function (selected){<br /><code>            </code>console.info(\"Selected \", selected); <br /><code>        </code><code></code>});<br /> <code>    </code>}</code></pre>\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>DiagramJs API</a></h3>\r\n<p>The current implementation supports the following listeners and methods:</p>\r\n<table class=\"reactome\">\r\n<thead>\r\n<tr><th>Method</th><th>Params</th><th>Description</th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td data-label=\"Method\"><b>create</b> :: Constructor<br />   Reactome.Diagram.create(params);</td>\r\n<td data-label=\"Params\"><b>param</b> :: json object<br />  {<br />    \'proxyPrefix\' : string,<br />    \'placeHolder\' : string,<br />    \'width\' : int (optional),<br />    \'height\' : int (optional)<br />  }</td>\r\n<td data-label=\"Description\">Creates and returns a new Reactome.Diagram object</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>loadDiagram(stId)</b> :: void</td>\r\n<td data-label=\"Params\">Pathway stable identifier<br /> <b>stId</b> : string</td>\r\n<td data-label=\"Description\">Loads the specified pathway</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>flagItems(term)</b> :: void</td>\r\n<td data-label=\"Params\">Entity identifier (gene name)<br /> <b>term</b> : string</td>\r\n<td data-label=\"Description\">Flags all entities matching with the \"term\"</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>highlightItem(stId)</b> :: void</td>\r\n<td data-label=\"Params\">Item stable identifier<br /> <b>stId</b> : string</td>\r\n<td data-label=\"Description\">Highlights the specified item if it exists in the diagram</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetAnalysis()</b> :: void</td>\r\n<td data-label=\"Params\"> </td>\r\n<td data-label=\"Description\">Resets the analysis overlay</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetFlaggedItems()</b> :: void</td>\r\n<td data-label=\"Params\"> </td>\r\n<td data-label=\"Description\">Resets the flagged items</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetHighlight()</b> :: void</td>\r\n<td data-label=\"Params\"> </td>\r\n<td data-label=\"Description\">Clears the highlights in the diagram</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resetSelection()</b> :: void</td>\r\n<td data-label=\"Params\"> </td>\r\n<td data-label=\"Description\">Clears the selection in the diagram</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>resize(width, height)</b> :: void</td>\r\n<td data-label=\"Params\"><b>widht</b>: int<br /> <b>height</b>: int</td>\r\n<td data-label=\"Description\">Resizes the viewport to the specified width and height</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>selectItem(stId)</b> :: void</td>\r\n<td data-label=\"Params\">Item stable identifier<br /> <b>stId</b> : string</td>\r\n<td data-label=\"Description\">Selects the specified item if it exists in the diagram</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>setAnalysisToken(token, resultFilter)</b> :: void</td>\r\n<td data-label=\"Params\">\r\n<p>Analysis token<br /> <b>token</b> : string<br /> Resource<br /><b>resultFilter</b> : <strong>obj</strong></p>\r\n<p><b>obj</b> resultFilter:</p>\r\n<div> {<br />    \'resource\' : string,<br />    \'pValue\' : double,(optional)<br />    \'includeDisease\' : boolean,(optional)<br />    \'min\' : Integer, (optional)</div>\r\n<div>   \'max\':Integer, (optioinal)</div>\r\n<div>   \'speciesList\': [List&lt;String&gt;]<br />  }</div>\r\n</td>\r\n<td data-label=\"Description\">Overlays the analysis result corresponding to the specified (token, resultFilter)</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onObjectSelected(function(obj))</b> :: void</td>\r\n<td data-label=\"Params\"><b>obj</b> is selected item:<br />  {<br />    \'stId\' : string,<br />    \'displayName\' : string,<br />    \'schemaClass\' : string,<br />    \'identifier\' : string (optional),<br />    \'geneNames\' : Array&lt;string&gt; (optional)<br />  }</td>\r\n<td data-label=\"Description\">The function is called when an object in the diagram is selected by the user action</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onObjectHovered(function(obj))</b> :: void</td>\r\n<td data-label=\"Params\"><b>obj</b> is hovered item:<br />  {<br />    \'stId\' : string,<br />    \'displayName\' : string,<br />    \'schemaClass\' : string,<br />    \'identifier\' : string (optional),<br />    \'geneNames\' : Array&lt;string&gt; (optional)<br />  }</td>\r\n<td data-label=\"Description\">The function is called when an object in the diagram is hovered by the user action</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onDiagramLoaded(function(stId))</b> :: void</td>\r\n<td data-label=\"Params\">The stable identifier of the loaded diagram<br /> <b>stId</b>: string</td>\r\n<td data-label=\"Description\">The function is called when a diagram is loaded in the viewer</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onFlagsReset(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters</td>\r\n<td data-label=\"Description\">The function is called when the flagged items have been reset by the user action</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onAnalysisReset(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters</td>\r\n<td data-label=\"Description\">The function is called when the users resets the analysis overlay<br /><br /> diagram.onAnalysisReset(function(){ /* your code here */ });</td>\r\n</tr>\r\n<tr>\r\n<td data-label=\"Method\"><b>onCanvasNotSupported(function())</b> :: void</td>\r\n<td data-label=\"Params\">The function receives no parameters.</td>\r\n<td data-label=\"Description\">The function is called when the browser doesn\'t support HTML5 Canvas so the viewer cannot be instantiated<br /><br /> diagram.onCanvasNotSupported(function(){ /* your code here */ });</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<script type=\"text/javascript\">jQuery(\'base\').remove();\r\nfunction getMainContentWidth(){\r\n   var DIV = \"fav-maincontent\";\r\n   var style = window.getComputedStyle(document.getElementById(DIV), null);\r\n   var width = style.getPropertyValue(\"width\").replace(\"px\",\"\");\r\n   var paddingLeft = style.getPropertyValue(\"padding-left\").replace(\"px\",\"\"); \r\n   var paddingRight = style.getPropertyValue(\"padding-right\").replace(\"px\",\"\");\r\n   return width - paddingLeft - paddingRight;\r\n}\r\njQuery( window ).resize(function() {\r\n  diagram.resize(getMainContentWidth(), 600);  \r\n});\r\nfunction onReactomeDiagramReady() {\r\n        diagram = Reactome.Diagram.create({\r\n            \"placeHolder\": \"diagramHolder\",\r\n            \"width\": getMainContentWidth(),\r\n            \"height\": 600\r\n        });\r\n        diagram.loadDiagram(\"R-HSA-69620\");\r\n        diagram.onDiagramLoaded(function (loaded) {\r\n            diagram.flagItems(\"FYN\");\r\n            if(loaded == \"R-HSA-109582\") diagram.selectItem(\"R-HSA-75892\");\r\n        });\r\n    }\r\n</script>\r\n<script src=\"/DiagramJs/diagram/diagram.nocache.js\" type=\"text/javascript\" language=\"javascript\"></script>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-07 15:17:14','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(59,634,'Diagram GWT','diagram-gwt','<p>Reactome’s Diagram GWT widget is our original implementation of the diagram viewer in <a href=\"http://www.gwtproject.org\" target=\"_blank\" rel=\"nofollow noopener noreferrer\">GWT</a>. It is meant to be reused by third party resources in order to display Reactome Pathway Diagrams directly in their web pages and enable users to interact with them.</p>\r\n<p>{loadposition dev-guide-pd-gwt-1}</p>\r\n<div>\r\n<h3><a href=\"#MoreInfo\" id=\"MoreInfo\"><i class=\"fa fa-question-circle\" style=\"font-size: 23px;\"></i>More Info</a></h3>\r\n<p><a href=\"http://www.gwtproject.org/\">GWT</a>&nbsp;is a development toolkit for building and optimizing complex browser-based applications. Its goal is to enable productive development of high-performance web applications without the developer having to be an expert in browser quirks, XMLHttpRequest, and JavaScript. GWT is used by many products at Google, including AdWords, AdSense, Flights, Hotel Finder, Offers, Wallet, Blogger. It’s open source, completely free, and used by thousands of developers around the world.</p>\r\n<p>&nbsp;</p>\r\n<h3><a href=\"#GetStarted\" id=\"GetStarted\"><i class=\"fa fa-rocket\" style=\"font-size: 23px;\"></i>Get Started</a></h3>\r\n<p>We assume that you already have your <a href=\"http://www.gwtproject.org/gettingstarted.html\">GWT project</a> up and running configured with the <a href=\"http://www.gwtproject.org/doc/latest/RefCommandLineTools.html\"> MAVEN archetype</a>. So the first step is to add the EBI Nexus repository in your pom.xml file:</p>\r\n<pre><code>&lt;repositories&gt;\r\n    ...\r\n    &lt;!-- EBI repo --&gt;\r\n    &lt;repository&gt;\r\n        &lt;id&gt;nexus-ebi-repo&lt;/id&gt;\r\n        &lt;name&gt;The EBI internal repository&lt;/name&gt;\r\n        &lt;url&gt;http://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-repo/&lt;/url&gt;\r\n        &lt;releases&gt;\r\n            &lt;enabled&gt;true&lt;/enabled&gt;\r\n        &lt;/releases&gt;\r\n        &lt;snapshots&gt;\r\n            &lt;enabled&gt;false&lt;/enabled&gt;\r\n        &lt;/snapshots&gt;\r\n    &lt;/repository&gt;\r\n    &lt;!-- EBI SNAPSHOT repo --&gt;\r\n    &lt;repository&gt;\r\n        &lt;id&gt;nexus-ebi-snapshot-repo&lt;/id&gt;\r\n        &lt;name&gt;The EBI internal snapshot repository&lt;/name&gt;\r\n        &lt;url&gt;http://www.ebi.ac.uk/Tools/maven/repos/content/groups/ebi-snapshots/&lt;/url&gt;\r\n        &lt;releases&gt;\r\n            &lt;enabled&gt;false&lt;/enabled&gt;\r\n        &lt;/releases&gt;\r\n        &lt;snapshots&gt;\r\n            &lt;enabled&gt;true&lt;/enabled&gt;\r\n        &lt;/snapshots&gt;\r\n    &lt;/repository&gt;\r\n&lt;/repositories&gt;\r\n</code></pre>\r\n<p>Once the repository is in place, the next thing to do is adding the Reactome\'s pathways overview dependency:</p>\r\n<pre><code>&lt;dependencies&gt;\r\n    ...\r\n    &lt;dependency&gt;\r\n        &lt;groupId&gt;org.reactome.web&lt;/groupId&gt;\r\n        &lt;artifactId&gt;diagram&lt;/artifactId&gt;\r\n        &lt;version&gt;3.2.1&lt;/version&gt;\r\n    &lt;/dependency&gt;\r\n&lt;dependencies&gt;\r\n</code></pre>\r\n<p>Before writing the code to use the pathways overview, add the dependency in your <b>gwt.xml</b> file, so the compiler will find the pathways overview:</p>\r\n<pre><code>&lt;inherits name=\"org.reactome.web.diagram.DiagramViewer\" /&gt;\r\n</code></pre>\r\n<p>And finally, assuming the widget is now going to be displayed by its own, the code in your <b>EntryPoint</b> class would look like:</p>\r\n<pre><code>/**\r\n * For test purposes we assume the diagram loads as a single item in the web page.\r\n *\r\n * Entry point classes define onModuleLoad().\r\n */\r\npublic class Main implements EntryPoint {\r\n\r\n    private final DiagramViewer diagram;\r\n\r\n    public Main() {\r\n        //We create the diagram. Please note that different pathways diagrams can be loaded in the same\r\n        //instance (the diagram manages the cache of previously loaded diagrams using an LRU)\r\n        diagram = DiagramFactory.createDiagramViewer();\r\n    }\r\n\r\n    @Override\r\n    public void onModuleLoad() {\r\n        Scheduler.get().scheduleDeferred(new Scheduler.ScheduledCommand() {\r\n            @Override\r\n            public void execute() {\r\n                DiagramFactory.CONSOLE_VERBOSE = true;   //This is optional (for dev purposes)\r\n                DiagramFactory.EVENT_BUS_VERBOSE = true; //This is optional (for dev purposes)\r\n\r\n                //For this use case we add the diagram as the main and only object of the webpage\r\n                RootLayoutPanel.get().add(diagram);\r\n                \r\n                //This will run the loadDiagram method asynchronously, ensuring the DOM has been updated\r\n                Scheduler.get().scheduleDeferred(new Scheduler.ScheduledCommand() {\r\n                    @Override\r\n                    public void execute() {\r\n                        diagram.loadDiagram(\"R-HSA-111804\");\r\n                    }\r\n                });\r\n            }\r\n        });\r\n    }\r\n}</code></pre>\r\n<h3><a href=\"#API\" id=\"API\"><i class=\"fa fa-code\" style=\"font-size: 23px;\"></i>API</a></h3>\r\n<p>The available methods are defined in the <b>DiagramViewer</b> interface in the <b>org.reactome.web.diagram.client</b> package. These are:</p>\r\n<pre><code>    //subscribe to this event to be notified when the user closes the analysis overlay\r\n    HandlerRegistration addAnalysisResetHandler(AnalysisResetHandler handler);\r\n\r\n    //Subscribe to this event to be notified if the browser does not support canvas, so diagrams can not be displayed\r\n    HandlerRegistration addCanvasNotSupportedEventHandler(CanvasNotSupportedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user selects an object in the diagram\r\n    HandlerRegistration addDatabaseObjectSelectedHandler(GraphObjectSelectedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user hovers over an object in the diagram\r\n    HandlerRegistration addDatabaseObjectHoveredHandler(GraphObjectHoveredHandler handler);\r\n\r\n    //Subscribe to this event to be notified when a new diagram is displayed in the viewport\r\n    HandlerRegistration addDiagramLoadedHandler(DiagramLoadedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when elements in the diagram are flagged\r\n    HandlerRegistration addDiagramObjectsFlaggedHandler(DiagramObjectsFlaggedHandler handler);\r\n\r\n    //Subscribe to this event to be notified when flagged elements in the diagram are not longer visible\r\n    HandlerRegistration addDiagramObjectsFlagResetHandler(DiagramObjectsFlagResetHandler handler);\r\n\r\n    //Subscribe to this event to be notified when the user selects the \"Go to overview\" button\r\n    HandlerRegistration addFireworksOpenedHandler(FireworksOpenedHandler handler);\r\n\r\n    //Flags the elements in the diagram which identifiers match with \"identifier\"\r\n    void flagItems(String identifier);\r\n\r\n    //Highlights the element(s) in the diagram with the \"stableIdentifier\"\r\n    void highlightItem(String stableIdentifier);\r\n\r\n    //Highlights the element(s) in the diagram with the \"dbIdentifier\"\r\n    void highlightItem(Long dbIdentifier);\r\n\r\n    //Loads the diagram with the \"stableIdentifier\"\r\n    void loadDiagram(String stId);\r\n\r\n    //Loads the diagram with the \"dbId\"\r\n    void loadDiagram(Long dbId);\r\n\r\n    //Resets the analysis overlay\r\n    void resetAnalysis();\r\n\r\n    //Resets the flagged items\r\n    void resetFlaggedItems();\r\n\r\n    //Resets the highlighted items\r\n    void resetHighlight();\r\n\r\n    //Resets the selected items\r\n    void resetSelection();\r\n\r\n    //Selects the element(s) in the diagram with the \"stableIdentifier\"\r\n    void selectItem(String stableIdentifier);\r\n\r\n    //Selects the element(s) in the diagram with the \"stableIdentifier\"\r\n    void selectItem(Long dbIdentifier);\r\n\r\n    //Sets the analysis overlay for a given \"token\" and \"resource\" (this will query the Analysis Service)\r\n    void setAnalysisToken(String token, String resource);\r\n\r\n    //Set the widget visibility\r\n    void setVisible(boolean visible);</code></pre>\r\n<h3><a href=\"#Resources\" id=\"Resources\"><i class=\"fa fa-book\" style=\"font-size: 23px;\"></i>Resources</a></h3>\r\n<ul style=\"list-style: none;\">\r\n<li style=\"line-height: 30px;\"><a href=\"http://github.com/reactome-pwp/diagram\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>GWT Diagram code</a></li>\r\n<li style=\"line-height: 30px;\"><a href=\"http://github.com/reactome-pwp\"><i class=\"fa fa-github\" style=\"padding-right: 5px;\"></i>Reactome Pathways Portal code</a></li>\r\n</ul>\r\n</div>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-07 15:17:38','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(60,638,'Identifiers Mapping','identifiers-mapping','<p>The mapping files consist of a tab-separated table that indicates which external protein, gene or small molecule identifiers in the source database were mapped to Reactome pathway and reaction annotations. Our goal with distributing three sets of files for each different identifier type is to provide these mapping files link the source database identifier to:</p>\r\n<ul>\r\n<li style=\"list-style-type: lower-roman;\">the lowest level pathway diagram or subset of the pathway</li>\r\n<li style=\"list-style-type: lower-roman;\">all level pathway diagrams (filename appended with \'All_Levels’),</li>\r\n<li style=\"list-style-type: lower-roman;\">all reaction events.</li>\r\n</ul>\r\n<p>The columns within the mappings files follow a similar format:</p>\r\n<ol>\r\n<li>Source database identifier, e.g. UniProt, ENSEMBL, NCBI Gene or ChEBI identifier</li>\r\n<li>&nbsp;Reactome Pathway Stable identifier</li>\r\n<li>URL</li>\r\n<li>Event (Pathway or Reaction) Name</li>\r\n<li>Evidence Code</li>\r\n<li>Species</li>\r\n</ol>\r\n<p>If you have any further questions, please <a href=\"mailto:help@reactome.org?subject=Identifier%20Mapping%20File\">contact us</a>.</p>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-23 09:19:03','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(61,639,'Pathway Information','pathway-information','<p>The pathway information files consist of tab-separated tables that provide information about Reactome pathway hierarchy:</p>\r\n<ol>\r\n<li>The \"Complete list of pathways\" file maps the Reactome Stable identifier (ST_ID) to a pathway name and corresponding species.</li>\r\n<li>The \"Pathway hierarchy relationship\" file consists of two columns of Reactome Stable identifiers (ST_ID), defining the relationship between pathways within the pathway hierarchy. The first column provides the parent pathway stable identifier, whereas the second column provides the child pathway stable identifier.</li>\r\n</ol>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-23 09:37:27','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,11,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(62,640,'Tab-Delimiter Format','tab-delimiter-format','<p>Interactions are generated electronically based on the data that we store on complexes and reactions, i.e. the interactions provided by Reactome are not curated and are not experimental data. In addition, the complexes and reactions in species other than human are derived by orthology inference from the correspondinghuman complexes and reactions.</p>\r\n<p>The meaning of \"interaction\" is quite broad: 2 protein sequences occur in the same complex or they occur in the same reaction.</p>\r\n<p>The 1st line begins with \'#\' and gives the number of unique interactions, i.e. unique pairs if UniProt accession numbers.</p>\r\n<p>The rest of the file is tab delimited with columns having following meaning/content:</p>\r\n<ul style=\"list-style: decimal;\">\r\n<li>Protein ID (e.g. UniProt accession) for interactor 1.</li>\r\n<li>Ensembl gene id(s delimited by \'|\') for interactor 1. Note that one UniProt entry may be mapped to multiple Ensembl genes.</li>\r\n<li>Entrez Gene id(s delimited by \'|\') for interactor 1. Note that one UniProt entry may be mapped to multiple Entrez Gene ids.</li>\r\n<li>Protein ID (e.g. UniProt accession) for interactor 2.</li>\r\n<li>Ensembl gene id(s delimited by \'|\') for interactor 2. Note that one UniProt entry may be mapped to multiple Ensembl genes.</li>\r\n<li>Entrez Gene id(s delimited by \'|\') for interactor 2. Note that one UniProt entry may be mapped to multiple Entrez Gene ids.</li>\r\n<li>Type of interaction, either \'direct_complex\', \'indirect_complex\', \'reaction\' or \'neighbouring_reaction\'. Meanings of those below.</li>\r\n<li>Interaction context. This is the class and internal identifier of Reactome instance via which this interaction happens. For \'neighbouring_reaction\' type interaction this contains a pair of class:identifier string separated by \'\'.</li>\r\n<li>Comma-delimited list of PubMed identifiers supporting this interaction. For a reaction these are the PMIDs from the Reaction.literatureReference slot (and not from the \"second tier\" Reaction.summation.literatureReference). For a complex these are from reations which produce the complex. For \'neighbouring_reaction\' type interaction this field is empty.</li>\r\n</ul>\r\n<h3>Interaction types</h3>\r\n<p>In Reactome, complexes can be arbitrarily nested, i.e., contain subcomplexes. Here we have complex C1 consisting of complexes C2 and C3. C2 consist of molecules A and B. C3 consists of molecules C and D:</p>\r\n<pre>           C1 --- C2 --- A\r\n                |      |\r\n                |      |- B\r\n                |\r\n                |- C3 --- C\r\n                |\r\n                |- D\r\n                </pre>\r\n<p>Now really to the interaction types:</p>\r\n<ul>\r\n<li>direct_complex - interactors are directly in the same complex, i.e. w/o further nested complexes. From the example interactions A B and C are of this type.</li>\r\n<li>indirect_complex - interactors in different subcomplexes of a complex. From the example above interactions AC, AD, BC and BD are of this type.</li>\r\n<li>reaction - interactors participate in the same reaction. Only those reactions are reported for which the intreactors are not complexed (with the exception being association dissociation reactions which are reported).</li>\r\n<li>neighbouring_reaction - interactor participate in 2 \"consecutive\" reactions, i.e. when one reaction produces a PhysicalEntity which is either an input or a catalyst for another reaction. However, to avoid the \"trivial\" (i.e. over ATP, ADP etc) interactions, the computation is done using the \'precedingEvent\' attribute used to order reactions in a pathway.</li>\r\n</ul>\r\n<p>The \'direct_complex\' type doesn\'t really mean true pairwise interaction. For example, Reactome\'s representation of ribosomal S60 subunit consists \"directly\" of 40+ proteins and has no sub-complexes. Obviously this can hardly mean that each one of those component proteins directly interacts with every other protein. Hence, the \'direct_complex\' vs. \'indirect_complex\' distinction is quite meaningless.</p>\r\n<h3>Other notes:</h3>\r\n<ul>\r\n<li>If the same interaction (i.e same UniProte accession pair) happens in multiple contexts, each context is reported on a separate row and hence the row number in the file is bigger than the number of unique interactions reported on the 1st line.</li>\r\n<li>The order of interactors on one line has no meaning (they are just ordered according to the Reactome internal identifier...). Every interaction with context is reported once, i.e. AB is not reported as BA.</li>\r\n</ul>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-23 10:02:36','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(63,641,'Curator Tool (version 4.0.3, build 118, March 3, 2024)','reactome-curator-tool-3','<p></p>\r\n<style type=\"text/css\">ul{ list-style-type: square; }</style>\r\n<p></p>\r\n<div id=\"top\"></div>\r\n<center>\r\n<p><span><b>Available Installers</b></span></p>\r\n<table border=\"0\" cellspacing=\"10\" cellpadding=\"0\">\r\n<tbody>\r\n<tr>\r\n<td> </td>\r\n<td><span>Platform</span></td>\r\n<td> </td>\r\n<td> </td>\r\n<td><center><span>Installer</span></center></td>\r\n<td> </td>\r\n<td><center><span>Instructions</span></center></td>\r\n</tr>\r\n<tr>\r\n<td style=\"text-align: center;\" valign=\"center\"><i class=\"fa fa-windows\"></i></td>\r\n<td valign=\"bottom\">\r\n<div style=\"text-align: left;\"><span>Windows</span></div>\r\n</td>\r\n<td valign=\"bottom\"> </td>\r\n<td valign=\"bottom\"> </td>\r\n<td valign=\"bottom\"><center><a href=\"https://reactome.org/download/tools/curatorTool/InstData/win_linux/install.zip\">Download (15.4M)</a></center></td>\r\n<td valign=\"bottom\"> </td>\r\n<td valign=\"bottom\"><center><a href=\"#windows\">View</a></center></td>\r\n</tr>\r\n<tr>\r\n<td style=\"text-align: center;\" valign=\"center\"><i class=\"fa fa-apple\"></i></td>\r\n<td valign=\"bottom\">\r\n<div style=\"text-align: left;\"><span>Mac OS X</span></div>\r\n</td>\r\n<td valign=\"bottom\"> </td>\r\n<td valign=\"bottom\"> </td>\r\n<td valign=\"bottom\"><center><a href=\"https://reactome.org/download/tools/curatorTool/InstData/mac/install.zip\">Download (15.4)</a></center></td>\r\n<td valign=\"bottom\"> </td>\r\n<td valign=\"bottom\"><center><a href=\"#macosx\">View</a></center></td>\r\n</tr>\r\n<tr>\r\n<td style=\"text-align: center;\" valign=\"center\"><i class=\"fa fa-linux\"></i></td>\r\n<td valign=\"bottom\">\r\n<div style=\"text-align: left;\"><span>Linux</span></div>\r\n</td>\r\n<td valign=\"bottom\"> </td>\r\n<td valign=\"bottom\"> </td>\r\n<td valign=\"bottom\"><center><a href=\"https://reactome.org/download/tools/curatorTool/InstData/win_linux/install.zip\">Download (15.4M)</a></center></td>\r\n<td valign=\"bottom\"> </td>\r\n<td valign=\"bottom\"><center><a href=\"#linux\">View</a></center></td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</center><hr />\r\n<div id=\"windows\" style=\"text-align: left;\">\r\n<h4><a href=\"#windows\" name=\"windows\">Windows:</a></h4>\r\n<h5>Instructions</h5>\r\n<ul>\r\n<li>After downloading, double-click <b><code>install.zip</code></b> to unzip it to an empty folder. This folder should be named \"ReactomeCuratorTool\" though any name can be used. Please use the latest version of WinZip if you use this application for unzipping.</li>\r\n<li>After unzipping, an application called ReactomeCuratorTool.exe should be found in the named folder. Double click this application to launch the Reactome Curator Tool.</li>\r\n</ul>\r\n<h5>Notes</h5>\r\n<ul>\r\n<li>You need to install Java 11. You can download one from <a href=\"https://adoptium.net/temurin/releases/?version=11\">OpenJDK</a>.</li>\r\n</ul>\r\n<p>(<a href=\"#top\">Go To Top</a>)</p>\r\n</div>\r\n<center><hr /></center>\r\n<div id=\"macosx\" style=\"text-align: left;\">\r\n<h4><a href=\"#macosx\" name=\"macosx\">Mac OS X:</a></h4>\r\n<h5>Instructions</h5>\r\n<ul>\r\n<li>Please refer this document for updated information: <a href=\"https://devwiki.reactome.org/index.php/New_Curators_Guide#Downloading.2C_Installing.2C_And_Maintaining_The_Curator_Tool\">https://devwiki.reactome.org/index.php/New_Curators_Guide#Downloading.2C_Installing.2C_And_Maintaining_The_Curator_Tool</a></li>\r\n<li>After downloading, double-click <b><code>install.zip</code></b> to unzip the folder.</li>\r\n<li>Follow the instruction for the linux version below.</li>\r\n</ul>\r\n<h5>Notes</h5>\r\n<ul>\r\n<li>Requires Mac OS X 10.4.8 or later, and Java 11.</li>\r\n</ul>\r\n<p>(<a href=\"#top\">Go To Top</a>)</p>\r\n</div>\r\n<center><hr /></center>\r\n<div id=\"linux\" style=\"text-align: left;\">\r\n<h4><a href=\"#linux\" name=\"linux\">Linux:</a></h4>\r\n<h5>Instructions</h5>\r\n<ul>\r\n<li>After downloading open a shell, and <span><b><code>cd</code></b></span><span> to the directory where you downloaded the installer.</span></li>\r\n<li>At the prompt, type <span><b><code>unzip install.zip</code></b> to unzip the installer.</span></li>\r\n<li>Use ./ReactomeCuratorTool.sh to launch the Reactome Curator Tool.</li>\r\n</ul>\r\n<h5>Notes</h5>\r\n<ul>\r\n<li><span>You need to install Java 11. You can download one from <a href=\"https://adoptium.net/temurin/releases/?version=11\">OpenJDK</a>.</span></li>\r\n</ul>\r\n<p>(<a href=\"#top\">Go To Top</a>)</p>\r\n</div>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-23 10:53:19','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,9,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(64,668,'Developer\'s Zone','developer-s-zone','<h4>Explore our tools and web services and learn how to include them in your applications</h4>\r\n<p>{loadposition dev-guide-1}{loadposition dev-guide-2}</p>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-29 11:12:14','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,10,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(66,675,'Statistics','statistics','<p>{source}<span style=\"font-family: courier new, courier, monospace;\">&lt;?php </span><span style=\"font-family: courier new, courier, monospace;\">$host =</span><code class=\"hljs language-php\"><span class=\"hljs-variable\">$_SERVER</span>[<span class=\"hljs-string\">\'SERVER_NAME\'</span>]</code><span style=\"font-family: courier new, courier, monospace;\">;</span></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">    IF ($host=== \"localhost\"|| $host=== \"127.0.1\") {</span></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\"><span style=\"font-family: courier new, courier, monospace;\">     $host = \"dev.reactome.org\";</span></span><span style=\"font-family: courier new, courier, monospace;\"><br /><img src=\"/Website/media/sourcerer/images/tab.png\" alt=\"\" />}</span></p>\r\n<p>      $url = \"https://\" .$host . \'/ContentService/data/database/version\';</p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">   $version = file_get_contents(<span style=\"font-family: courier new, courier, monospace;\">$url</span>); </span></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\"><span style=\"font-family: courier new, courier, monospace;\">?&gt;</span></span>{/source}</p>\r\n<p>{source}<span style=\"font-family: courier new, courier, monospace;\">&lt;iframe</span></p>\r\n<pre class=\"c-mrkdwn__pre\" data-stringify-type=\"pre\"> style=\"height: 500px; width:100%;\"</pre>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">src=\"https://download.reactome.org/{source}<span style=\"font-family: courier new, courier, monospace;\">&lt;?php echo $version ?&gt;</span>{/source}/stats/release_stats.html\" <span class=\"hljs-tag\"><span class=\"hljs-attr\">frameborder</span>=<span class=\"hljs-string\">\"0\"</span> </span>&gt;&lt;/iframe&gt;<span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"></span></span></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">&lt;iframe </span>style=\"height: 595px; width:100%\"</p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">src=\"https://download.reactome.org/{source}<span style=\"font-family: courier new, courier, monospace;\">&lt;?php echo $version ?&gt;</span>{/source}/stats/ordered_release_stats.html\" <span class=\"hljs-tag\"><span class=\"hljs-attr\">frameborder</span>=<span class=\"hljs-string\">\"0\"</span> </span><span style=\"font-family: courier new, courier, monospace;\"></span> &gt;&lt;/iframe&gt;<span style=\"font-family: courier new, courier, monospace;\"><span style=\"font-family: courier new, courier, monospace;\"></span></span></span>{/source}</p>\r\n<p></p>\r\n<p></p>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-07-01 20:24:33','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,9,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(67,676,'ORCID Integration Project','orcid-integration-project','<p style=\"text-align: justify;\">Biological information has become so abundant and complex in recent years that it is difficult, if not impossible, even for expert individuals to manage in traditional publication formats and with existing knowledge management tools. It is an ongoing challenge for researchers to keep up-to-date on research developments in their fields and to identify relevant research to support their own studies without devoting too much time to collecting unconnected information. The Reactome group has recognized this challenge and is developing a set of novel online resources that use features of electronic media to organize biological information in ways that provide for more efficient access and that allow new forms of analysis that were not possible with information stored in the traditional printed literature.</p>\r\n<p style=\"text-align: justify;\">The curation process for a pathway is similar to the editing of a scientific review. An external domain expert provides his or her expertise, one of our curators formalizes it into the database structure, and a second external domain expert reviews the representation. An evidence-tracking system ensures that all assertions are backed up by the primary literature. We have created data entry software, called the Author and Curator Tools, and associated web services, which provide interfaces in which the pathway authors can systematize their knowledge and a mechanism by which researchers, students, and clinicians can transform the information about interactions between biomolecules into knowledge about a cellular process.</p>\r\n<p style=\"text-align: justify;\">Reactome has evolved into the one of the largest pathway databases, containing over 19,000 human and inferred model organism pathways, and is accessed by over 15,000 unique visitors per month all around the world. A key challenge for us is incentivizing the unpaid external domain experts to contribute their expertise and time to the curation process. In the scientific process, the key incentive is credit attribution. </p>\r\n<p style=\"text-align: justify;\">As <a href=\"http://orcid.org/\">ORCID</a> is now part of our data model, we are constantly annotating curators, editors, and expert authors and reviewers with their respective ORCID. Thus, if your ORCID isn\'t annotated, please let us know directly from your Authors page (search for your name first), and one of our curators will add it, and we will make it available on the next data release. On the Authors page, once logged in ORCID, a button \"Let us know your orchid\" should appear, and you can share more information using our internal form.</p>\r\n<hr />\r\n<div class=\"alert alert-success\" role=\"alert\">Our main search allows <strong>Authors</strong> and <strong>Reviewers</strong> to <a href=\"/content/query?q=*&amp;species=Entries+without+species&amp;types=Person&amp;cluster=true\" target=\"_blank\">query</a> for their pathway and reaction contributions using their names, and claim their contributions directly into ORCID.</div>\r\n<hr />\r\n<h3>What is ORCID?</h3>\r\n<section>\r\n<p style=\"text-align: justify;\">ORCID (Open Researcher and Contributor ID) is an open, non-profit, community-driven effort to create and maintain a registry of unique researcher identifiers and a transparent method of linking research activities and outputs to these identifiers. </p>\r\n<p style=\"text-align: justify;\">Individuals can obtain a free ORCID (Open Researcher and Contributor ID) identifier. This digital identifier distinguishes individual researchers from other researchers and enables them to manage their records and search for others in the Registry.</p>\r\n</section>\r\n<h4 style=\"padding-left: 30px;\">Authenticate</h4>\r\n<p style=\"padding-left: 30px; text-align: justify;\"><img src=\"images/others/orcid/orcid_auth.png\" alt=\"\" width=\"43\" height=\"67\" style=\"float: left;\" title=\"This integration has been recognized by ORCID in 2019 as verifying ORCID iDs using authentication -- by asking users to sign into their ORCID account.\" /> Reactome is collecting your ORCID iD so we can add Pathways and Reactions (Works) that you have contributed either as author or reviewer into your ORCID records. When you click the “Authorize” button, we will ask you to share your iD using an authenticated process: either by <a href=\"https://support.orcid.org/hc/articles/360006897454\">registering for an ORCID iD</a> or, if you already have one, by <a href=\"https://support.orcid.org/hc/articles/360006971613\">signing into your ORCID account</a>, then granting us permission to get your ORCID iD. We do this to ensure that you are correctly identified and securely connecting your ORCID iD. Learn more about <a href=\"https://orcid.org/blog/2017/02/20/whats-so-special-about-signing\">What’s so special about signing in</a>.</p>\r\n<p></p>\r\n<h4 style=\"padding-left: 30px;\">Display</h4>\r\n<p style=\"padding-left: 30px; text-align: justify;\"><img src=\"images/others/orcid/orcid_display.png\" alt=\"display logo\" width=\"43\" height=\"67\" style=\"float: left;\" title=\"This integration has been recognized by ORCID in 2019 as displaying verified ORCID iDs publicly.\" /> To acknowledge that you have used your iD and that it has been authenticated, we display the ORCID iD icon <img src=\"images/others/orcid_16x16.png\" alt=\"ORCID Logo\" /> alongside your name on our website/in our publications/in our database etc. Learn more in <a href=\"https://orcid.org/blog/2013/02/22/how-should-orcid-id-be-displayed\">How should an ORCID iD be displayed</a>.</p>\r\n<p></p>\r\n<h4 style=\"padding-left: 30px;\">Connect</h4>\r\n<p style=\"padding-left: 30px; text-align: justify;\"><img src=\"images/others/orcid/orcid_connect.png\" alt=\"connect logo\" width=\"43\" height=\"67\" style=\"float: left;\" title=\"This integration has been recognized by ORCID in 2019 as connecting data that it can uniquely identify to researchers\' ORCID records using the ORCID API.\" /> By sharing your iD with Reactome, and giving us permission to <a href=\"https://support.orcid.org/hc/articles/360006896394\">read and update your ORCID record</a>, you enable us to help you keep your record up-to-date with trusted information. Learn more in <a href=\"https://info.orcid.org/six-ways-to-make-your-orcid-id-work-for-you/\">Six ways to make your ORCID iD work for you!</a></p>\r\n<p></p>\r\n<p></p>\r\n<p>For more detailed information about ORCID or to register for an ORCID iD, go the <a href=\"https://orcid.org/\" style=\"text-align: justify; background-color: inherit; font-family: inherit; font-size: 1rem;\">ORCID website</a>.</p>','',1,34,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-07-01 20:26:15','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(68,682,'Computationally Inferred Events','computationally-inferred-events','<p style=\"text-align: justify;\">We use the set of manually curated human reactions to electronically infer reactions in fourteen evolutionarily divergent eukaryotic species for which high-quality whole-genome sequence data are available, and hence a comprehensive and high-quality set of protein predictions exists. These species include the laboratory mouse and rat, the nematode <i>C. elegans</i>, and budding and fission yeasts. The estimated success rates of our orthology inference strategy can be stated as ‘the percentage of eligible reactions, defined in step 2 below, in the current human data set for which an event can be inferred in the model organism. By this measure, success rates range from 82.4% for the laboratory mouse to 9.8% for <em>P. falciparum.</em></p>\r\n<p style=\"text-align: justify;\">Electronic inference proceeds in four steps:</p>\r\n<ol style=\"text-align: justify;\">\r\n<li>Protein homology data were obtained from <a href=\"http://www.pantherdb.org/\">PANTHER</a>. PANTHER uses the <a href=\"https://www.ebi.ac.uk/reference_proteomes\">Reference proteome</a> dataset maintained by UniProt to generate phylogenetic trees of protein-coding genes across numerous species. Homologs are derived from these trees and annotated by type (ortholog, paralog). Additionally, in cases with multiple orthologs PANTHER infers the least diverged ortholog based on protein sequence divergence, which is utilized during our inference process. A detailed description of PANTHER\'s methodology can be found in <a href=\"https://academic.oup.com/nar/article/41/D1/D377/1060482\">PANTHER in 2013: modeling the evolution of gene function, and other gene attributes, in the context of phylogenetic trees</a>.</li>\r\n</ol><ol start=\"2\" style=\"text-align: justify;\">\r\n<li>All human reactions in the Reactome knowledgebase involving one or more proteins are eligible for electronic inference, with two exceptions. Reactions that were themselves inferred based on data from the model organism, and reactions involving species in addition to human (e.g., HIV infection of human cells) are excluded from electronic inference. Eligible reactions are checked to determine whether each involved protein has at least one homologous protein (HP) in the reaction\'s input, output and (if present) catalyst in the organism undergoing inference. If a human reaction involves a complex, at least 75% of the accessioned protein components of the human complex must have HPs in the model organism.</li>\r\n</ol><ol start=\"3\" style=\"text-align: justify;\">\r\n<li>For each reaction that meets these criteria, an equivalent reaction is created for the model organism by replacing each human protein with its model organism HP. If a human protein corresponds to more than one model organism HP, a DefinedSet called ‘Homologues of …’ is created, with the model organism HPs as members.</li>\r\n</ol><ol start=\"4\" style=\"text-align: justify;\">\r\n<li>After all possible reactions have been inferred for the species, any human pathways that contain at least 1 inferred reaction will be inferred for the species as well.</li>\r\n</ol>\r\n<p style=\"text-align: justify;\">These electronically inferred reactions are predictions based on a number of assumptions. Most basically, we assume that if we can find model organism HPs corresponding to all proteins involved in a human reaction, then the proteins mediate the same reaction in the model organism. This may not be true. On the other hand, we may miss a truly homologous reaction in the model organism because it is mediated by structurally divergent proteins that were not identified as such by PANTHER’s techniques. Similarly, complexes sharing less than 75% homologous subunits between species may nevertheless continue to perform the same function. The electronically inferred reactions presented in Reactome are thus not data, but hypotheses useful to direct the design of confirmatory experiments.</p>\r\n<p style=\"text-align: justify;\">A modified version of the orthoinference process was used to create a first draft of the <a href=\"/PathwayBrowser/#/R-HSA-9694516\">SARS-CoV-2 infection pathway</a>. Events in the SARS-Cov-2 pathway corresponding to each event in the <a href=\"/PathwayBrowser/#/R-HSA-9678108\">SARS-CoV-1 pathway</a> were created and populated with SARS-CoV-2 protein-containing physical entities based on orthology to SARS-CoV-1 proteins. We continue to replace predicted SARS-Cov-2 events with experimentally validated events when the relevant experimental evidence becomes available.</p>\r\n<p></p>\r\n<p>{source}<span style=\"font-family: courier new, courier, monospace;\">&lt;?php $host =$_SERVER[\'SERVER_NAME\'];</span></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">IF ($host=== \"localhost\"|| $host=== \"127.0.1\") {<br /><br />$host = \"dev.reactome.org\";<br />}<br /><br />$url = \"https://\" .$host . \'/ContentService/data/database/version\';<br />$version = file_get_contents($url); ?&gt;</span>{/source}</p>\r\n<p>{source}<span style=\"font-family: courier new, courier, monospace;\">&lt;img src=\"https://download.reactome.org/{source}<span style=\"font-family: courier new, courier, monospace;\">&lt;?php echo $version ?&gt;</span>{/source}/stats/reaction_release_stats.png\" height=\"800\" width=\"900\"&gt;&lt;/iframe&gt;</span>{/source}</p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-18 14:47:18','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(69,683,'Papers citing us','papers-citing-us','<p><span>Since 2003, Reactome papers have been cited in hundreds of textbooks and thousands of peer-reviewed articles and reviews, and the citation rate is increasing annually. Several papers cite the use of Reactome data for either integrative data analysis, algorithm development, or the inclusion of Reactome pathway annotations into a secondary bioinformatics database. </span></p>\r\n<p><span>A full listing of all the papers and books citing Reactome is accessible online at </span><a href=\"https://scholar.google.ca/citations?user=gtB1MLcAAAAJ\"><span>Google Scholar</span></a><span>.</span></p>','',0,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-18 15:21:31','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(70,684,'External Identifiers','external-identifiers','<p><span>Listed below are the external bioinformatics databases that Reactome provides link outs to from our website.</span></p>\r\n<ul>\r\n<li><a href=\"http://pathway.gramene.org/\"><span>AraCyc</span></a></li>\r\n<li><a href=\"http://biogps.org/\"><span>BioGPS</span></a></li>\r\n<li><span><a href=\"http://www.ebi.ac.uk/biomodels/\">BioModels</a></span></li>\r\n<li><a href=\"http://www.cas.org/\"><span>CAS</span></a></li>\r\n<li><a href=\"http://www.genome.ad.jp/kegg/\"><span>COMPOUND</span></a></li>\r\n<li><a href=\"http://cancer.sanger.ac.uk/cosmic\"><span>COSMIC</span></a></li>\r\n<li><a href=\"http://ctdbase.org/\"><span>CTD Gene</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/chebi\"><span>ChEBI</span></a></li>\r\n<li><a href=\"https://clinicalgenome.org/\">ClinGen</a> </li>\r\n<li><a href=\"https://www.ncbi.nlm.nih.gov/projects/SNP/\"><span>dbSNP Gene</span></a></li>\r\n<li><a href=\"http://www.dictybase.org/\"><span>dictyBase</span></a></li>\r\n<li><a href=\"http://blaster.docking.org/\"><span>DOCK Blaster</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/ols/ontologies/doid\"><span>DOID</span></a></li>\r\n<li><a href=\"http://ca.expasy.org/enzyme/\"><span>EC</span></a></li>\r\n<li><a href=\"http://www.ensembl.org/\"><span>ENSEMBL</span></a></li>\r\n<li><a href=\"http://www.flybase.net/\"><span>Flybase</span></a></li>\r\n<li><a href=\"http://www.geneontology.org/\"><span>GO</span></a></li>\r\n<li><a href=\"http://www.genecards.org/\"><span>GeneCards</span></a></li>\r\n<li><a href=\"https://www.guidetopharmacology.org\">Guide to Pharmacology</a></li>\r\n<li><a href=\"http://www.genenames.org/\"><span>HGNC</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/intenz/\"><span>IntEnz</span></a></li>\r\n<li><a href=\"http://www.genome.jp/\"><span>KEGG Gene</span></a></li>\r\n<li><a href=\"http://www.mirbase.org/\"><span>miRBase</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/ols/ontologies/mod\"><span>MOD</span></a></li>\r\n<li><a href=\"http://www.ncbi.nlm.nih.gov/gene\"><span>NCBI Gene</span></a></li>\r\n<li><a href=\"http://www.ncbi.nlm.nih.gov/nuccore\"><span>NCBI Nucleotide</span></a></li>\r\n<li><a href=\"http://www.ncbi.nlm.nih.gov/protein\"><span>NCBI_Protein</span></a></li>\r\n<li><a href=\"http://omim.org/\"><span>OMIM</span></a></li>\r\n<li><a href=\"http://orchid.org/\"><span>ORCID</span></a></li>\r\n<li><a href=\"http://www.orpha.net/\"><span>Orphanet</span></a></li>\r\n<li><a href=\"http://www.genome.jp/dbget-bin/www_bfind?prf\"><span>PRF</span></a></li>\r\n<li><a href=\"http://plasmodb.org/\"><span>PlasmoDB</span></a></li>\r\n<li><a href=\"http://www.pdb.org/\"><span>Protein Data Bank</span></a></li>\r\n<li><a href=\"https://www.ncbi.nlm.nih.gov/pccompound\"><span>PubChem Compound</span></a></li>\r\n<li><a href=\"https://www.ncbi.nlm.nih.gov/pcsubstance\"><span>PubChem Substance</span></a></li>\r\n<li><a href=\"https://www.ncbi.nlm.nih.gov/refseq/\"><span>RefSeq</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/rhea/\"><span>Rhea</span></a></li>\r\n<li><a href=\"http://www.yeastgenome.org/\"><span>SGD</span></a></li>\r\n<li><a href=\"http://www.ebi.ac.uk/ols/ontologies/so\"><span>SO</span></a></li>\r\n<li><a href=\"http://www.arabidopsis.org/index.jsp\"><span>TAIR</span></a></li>\r\n<li><a href=\"http://genome.ucsc.edu/\"><span>UCSC human</span></a></li>\r\n<li><a href=\"http://www.uniprot.org/\"><span>UniProt</span></a></li>\r\n<li><a href=\"http://www.wormbase.org/\"><span>Wormbase</span></a></li>\r\n<li><a href=\"http://zinc15.docking.org/\"><span>ZINC</span></a></li>\r\n</ul>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-19 19:46:26','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(71,685,'Version 57 Released','version-57-released','<p>In version V57, topics with new or revised pathways include: Developmental biology (<a href=\"/PathwayBrowser/#R-HSA-8853659.1\">RET signaling</a>), Disease (<a href=\"/PathwayBrowser/#R-HSA-1226099.3\">Signaling by FGFR in disease</a> and <a href=\"/PathwayBrowser/#R-HSA-5678895.2\">Defective CFTR causes cystic fibrosis</a>), Gene expression (<a href=\"/PathwayBrowser/#R-HSA-6793080.1\">rRNA modification in the mitochondrion</a> and <a href=\"/PathwayBrowser/#R-HSA-8864260.1\">Transcriptional regulation by the AP-2 (TFAP2) family of transcription factors</a>), Immune system (<a href=\"/PathwayBrowser/#R-HSA-1236974.2\">ER-Phagosome pathway</a>, <a href=\"/PathwayBrowser/#R-HSA-1266695.3\">Interleukin-7 signaling</a>, <a href=\"/PathwayBrowser/#R-HSA-5686938.1\">Regulation by endogenous TLR ligand</a>, and <a href=\"/PathwayBrowser/#R-HSA-202403.3\">TCR signaling</a>), Metabolism (<a href=\"/PathwayBrowser/#R-HSA-73923.3\">Lipid digestion, mobilization, and transport</a> and <a href=\"/PathwayBrowser/#R-HSA-8850843.1\">Phosphate bond hydrolysis by NTPDase proteins</a>). Metabolism of proteins (<a href=\"/PathwayBrowser/#R-HSA-5688426.1\">Deubiquitination</a>), Neuronal system (<a href=\"/PathwayBrowser/#/R-HSA-6794361\">Interactions of neurexins and neuroligins at synapses</a> and <a href=\"/PathwayBrowser/#/R-HSA-8849932\">SALM protein interactions at synapse</a>). Signal transduction (<a href=\"/PathwayBrowser/#R-HSA-182971.3\">EGFR downregulation</a>, <a href=\"/PathwayBrowser/#R-HSA-5654736.3\">Signaling by FGFR1</a>, and <a href=\"/PathwayBrowser/#R-HSA-5658623.2\">FGFRL1 modulation of FGFR1 signaling</a>), Transmembrane transport of small molecules (<a href=\"/PathwayBrowser/#R-HSA-382556.2\">ABC-family proteins mediated transport</a>) and Vesicle-mediated transport (<a href=\"/PathwayBrowser/#R-HSA-8856828.1\">Clathrin-mediated endocytosis</a>).</p>\r\n<p>Our external author is <a href=\"http://scholar.google.com/citations?hl=en&amp;user=hdIojjoAAAAJ\">Alba Sanchis</a>. <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Antonescu++Costin+N%5BAuthor+-+Full%5D\">Costin Antonescu</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=John++J.M+Bergeron%5BAuthor+-+Full%5D\">John Bergeron</a><span style=\"text-decoration: underline;\">,</span> <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Daniel+F+Bogenhagen%5BAuthor+-+Full%5D\">Daniel Bogenhagen</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nunzio++Bottini%5BAuthor+-+Full%5D\">Nunzio Bottini</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Guang-Chao+Chen%5BAuthor+-+Full%5D\">Guang-Chao Chen</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Igor+B+Dawid%5BAuthor+-+Full%5D\">Igor Dawid</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Regina+Fluhrer%5BAuthor+-+Full%5D\">Regina Fluhrer</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Noriko++Gotoh%5BAuthor+-+Full%5D\">Noriko Gotoh</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Francesca+Granucci%5BAuthor+-+Full%5D\">Francesca Granucci</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Richard+P+Grose%5BAuthor+-+Full%5D\">Richard Grose</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Paul+A+Heppenstall%5BAuthor+-+Full%5D\">Paul Heppenstall</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Hu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26443885\">Jing Hu</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Jan%20Huertas%5BAuthor%20-%20Full%5D\">Jan Huertas</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Luo%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26992078\">Wenqin Luo</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Malay%20Mandal%5BAuthor%20-%20Full%5D\">Malay Mandal</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Birgit+Meldal%5BAuthor+-+Full%5D\">Birgit Meldal</a>, <a href=\"http://scholar.google.ca/citations?user=qux0Tt4AAAAJ&amp;hl=en\">Daniel Morales</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Tatsunori+Nishimura%5BAuthor+-+Full%5D\">Tatsunori Nishimura</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ronald++S+Petralia%5BAuthor+-+Full%5D\">Ronald Petralia</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Gail+K+Seabold%5BAuthor+-+Full%5D\">Gail Seabold</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Sunny+Sharma%5BAuthor+-+Full%5D\">Sunny Sharma</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Stephanie+Stanford%5BAuthor+-+Full%5D\">Stephanie Stanford</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jean+S%C3%A9vigny%5BAuthor+-+Full%5D\">Jean Sévigny</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Philip+Washbourne%5BAuthor+-+Full%5D\">Philip Washbourne</a>, <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ivan+Zanoni%5BAuthor+-+Full%5D\">Ivan Zanoni</a>, and <a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Valeria+Zarelli%5BAuthor+-+Full%5D\">Valeria Zarelli</a> are our external reviewers<strong>.</strong></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:13:06','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,76,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(72,686,'Version 58 Released','version-58-released','<p><strong><a href=\"/PathwayBrowser/#R-HSA-8862803\"><img class=\"aligncenter wp-image-3812\" style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/illustration_deregulated_CDK5_triggers_multiple_neurodegenerative_pathways_72.png\" alt=\"illustration deregulated CDK5 triggers multiple neurodegenerative pathways 72\" width=\"407\" height=\"400\" /></a></strong></p>\r\n<p><strong>New and Updated Pathways.&nbsp;</strong>With version V58, Reactome has annotations for over 10,000 human proteins. New or revised pathways include:&nbsp;Cell cycle (<a href=\"/PathwayBrowser/#R-HSA-8854050\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8854050\">FBXL7 down-regulates AURKA during mitotic entry and in early mitosis</a>), Developmental biology (<a href=\"/PathwayBrowser/#R-HSA-6805567\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6805567\">Keratinization</a>), Disease (<a href=\"/PathwayBrowser/#R-HSA-6802957\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6802957\">Oncogenic MAPK signaling</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8862803\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8862803\">Deregulated CDK5 triggers multiple neurodegenerative pathways in Alzheimer’s disease models</a>), Gene expression (<a href=\"/PathwayBrowser/#R-HSA-6811555\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6811555\">PI5P Regulates TP53 Acetylation</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8864260\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8864260\">Transcriptional regulation by the AP-2 (TFAP2) family of transcription factors</a>),&nbsp; Immune system&nbsp; (<a href=\"/PathwayBrowser/#R-HSA-6798695\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6798695\">Neutrophil degranulation</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-6803157\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6803157\">Antimicrobial peptides</a>), Metabolism of proteins (<a href=\"/PathwayBrowser/#R-HSA-8866652\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8866652\">Synthesis of active ubiquitin: roles of E1 and E2 enzymes</a>), Signal transduction (<a href=\"/PathwayBrowser/#R-HSA-6806834\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-6806834\">Signaling by MET</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8863795\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8863795\">Downregulation of ERBB2 signaling</a>), and Vesicle-mediated transport (<a href=\"/PathwayBrowser/#R-HSA-8876198\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8876198\">RAB GEFs exchange GTP for GDP on RABs</a>).</p>\r\n<p>A pathway illustration is available for&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8862803\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/PathwayBrowser/#R-HSA-8862803\">Deregulated CDK5 triggers multiple neurodegenerative pathways in Alzheimer’s disease&nbsp;</a>.</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;Our external author is&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Shah%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26962010\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Shah%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26962010\">Kavita Shah</a>. &nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ayoub%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23897807\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ayoub%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23897807\">Emily Ayoub</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Azevedo%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26235645\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Azevedo%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26235645\">Jorge Azevedo</a>,&nbsp;<a href=\"http://www.mdc-berlin.de/1154093/en/research/research_teams/signal_transduction_\">Walter Birchmeier</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Blumenberg%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26933871\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Blumenberg%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26933871\">Miroslav Blumenberg</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bogachek%20MV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25524900\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bogachek%20MV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25524900\">Maria Bogachek</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nullin+Divecha%5BAuthor%5D\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nullin+Divecha%5BAuthor%5D\">Nullin Divecha</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Roman%20Dziarski%5BAuthor%20-%20Full%5D\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Roman%20Dziarski%5BAuthor%20-%20Full%5D\">Roman Dziarski</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Rhys+Grant%5BAuthor%5D\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Rhys+Grant%5BAuthor%5D\">Rhys Grant</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Hains%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21357198\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Hains%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21357198\">David Hains</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Heegaard%20NH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26608204\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Heegaard%20NH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26608204\">Niels Heegard</a>,&nbsp;<a href=\"http://www.mdc-berlin.de/1154093/en/research/research_teams/signal_transduction_\">Guustaaf Heynen</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Lindon%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26835416\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Lindon%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26835416\">Catherine Lindon</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Andrea+Marat%5BAuthor%5D\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Andrea+Marat%5BAuthor%5D\">Andrea Marat</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Stephens%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27557867\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Stephens%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27557867\">Robert Stephens</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Tremblay%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27020859\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Tremblay%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27020859\">Michel Tremblay</a>, and&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Weigel%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25524900\" data-cke-saved-href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Weigel%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25524900\">Ronald Weigel</a>&nbsp;are our external reviewers.</p>\r\n<p>Reactome comprises 10,168 human reactions organized into 2,069 pathways involving 10,461 proteins encoded by 10,221 different human genes, and 1,710 small molecules. These annotations are supported by 24,974 literature references. We have projected these reactions onto 110,710 orthologous proteins, creating 19,991 orthologous pathways in 18 non-human species.</p>\r\n<p>Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, New York University Medical Center, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the&nbsp;<a href=\"/\" data-cke-saved-href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"https://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p>For more information please contact our&nbsp;<a href=\"mailto:help@reactome.org\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:14:30','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,75,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(73,687,'Reactome celebrates release of 10,000th annotated protein','reactome-celebrates-release-of-10-000th-annotated-protein','<p><a href=\"/\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/10K_Reactome.png\" alt=\"10K Reactome\" width=\"440\" height=\"400\" /></a></p>\r\n<p>The Reactome team is pleased to announce that it met a major milestone in October 2016 with the annotation and release of its 10,000th human protein. Reactome (<a href=\"/\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"/\">www.reactome.org</a>) is an open access curated knowledgebase which relates human genes, proteins and other biomolecules to the biological pathways and processes in which they participate. It is a key resource for the biomedical research community, and is widely used by researchers around the world to interpret high-throughput experiments in genetics, genomics and proteomics. Given that the human genome contains roughly 20,000 protein-coding genes in total, the annotation of the 10,000th protein means that Reactome now covers half of the protein-coding portion of the genome. This makes Reactome the most comprehensive open access pathway knowledgebase available to the scientific community.</p>\r\n<p>By relating genes and proteins to normal and abnormal biological pathways, Reactome allows researchers to identify patterns in large data sets. For example, researchers can use Reactome to reduce an experiment that identified thousands of genes whose activities are altered in a disease to a manageable number of key biological pathways that are disrupted by these changes. Researchers can then combine Reactome with other databases to find drugs and protein targets that might reverse the pathway alterations, or to devise ways of diagnosing the disease at an early stage. Via its web site, online tools, and specialized visualization and analysis applications, Reactome has been incorporated into more than 400 third-party genome analysis tools, and has been cited more than 4,000 times in the scientific literature.&nbsp;</p>\r\n<p>Reactome has been in continuous operation since 2004 and is an international collaboration among the Ontario Institute for Cancer Research in Canada, New York University School of Medicine in the United States, and the European Bioinformatics Institute in the United Kingdom. It is staffed by expert biological curators, bioinformaticians and computer scientists. Much of its content is provided by community authors and peer reviewers who are assisted by the curatorial staff. The Reactome content, including pathway data and the software infrastructure, are available to all comers free of charge under a Creative Commons open access license. Reactome is supported by grants from the US National Institutes of Health, the Ontario Research Fund, the University of Toronto, OpenTargets, Genome Canada, and the European Molecular Biology Laboratory.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"https://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p>For more information please contact our&nbsp;<a href=\"mailto:help@reactome.org\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:15:27','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,74,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(74,688,'Version 59 Released','version-59-release','<p><strong>New and Updated Pathways.&nbsp;</strong>In version V59, topics with new or revised pathways include: Disease (<a href=\"/PathwayBrowser/#/R-HSA-8876384&amp;PATH=R-HSA-1643685,R-HSA-5663205\">Listeria monocytogenes entry into host cells</a>), Hemostasis (<a href=\"/PathwayBrowser/#R-HSA-202733\">Cell surface interaction at the vascular wall</a>), Immune System (<a href=\"/PathwayBrowser/#R-HSA-8851680\">Butyrophilins</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-6783783\">Interleukin 10 signalling</a>, and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-6785807\">Interleukin-4 and 13 signaling</a>), Metabolism (<a href=\"/PathwayBrowser/#R-HSA-196807\">Nicotinate metabolism</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8847453\">Synthesis of PIPs at the nuclear envelope</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-199220\">Vitamin B5 (pantothenate) metabolism</a>, and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8937144\">Aryl hydrocarbon receptor signalling</a>), Metabolism of proteins (<a href=\"/PathwayBrowser/#R-HSA-8866654\">E3 ubiquitin ligases ubiquitinate target proteins</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-375276\">Peptide-ligand binding receptors</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8876725\">Protein methylation</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8873719\">RAB geranylgeranylation</a>), Signal transduction (<a href=\"/PathwayBrowser/#R-HSA-373076\">Class A/1 (Rhodopsin-like receptors</a>), and Vesicle-mediated transport (<a href=\"/PathwayBrowser/#R-HSA-8854214\">TBC RABGAPs</a>).</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;Our external reviewers are&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Jorge%20E%20Azevedo%5BAuthor%20-%20Full%5D\">Jorge Azevedo</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Ester%20Boix%5BAuthor%20-%20Full%5D\">Ester Boix</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Lu%20Deng%5BAuthor%20-%20Full%5D\">Lu Deng</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Pal%20Falnes%5BAuthor%20-%20Full%5D\">Pål Falnes</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Vardan%20Karamyan%5BAuthor%20-%20Full%5D\">Vardan Karamyan</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Samuel%20J%20Leibovich%5BAuthor%20-%20Full%5D\">Samuel Leibovich</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Weei-Chin%20Lin%5BAuthor%20-%20Full%5D\">Weei-Chin Lin</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Charuta%20Palsuledesai%5BAuthor%20-%20Full%5D\">Charuta Palsuledesai</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Joel%20Pomerantz%5BAuthor%20-%20Full%5D\">Joel Pomerantz</a>,&nbsp;<a href=\"http://www.unige.ch/medecine/pati/en/groupes/282reith/\">Walter Reith</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Sylvie%20Ricard-Blum%5BAuthor%20-%20Full%5D\">Sylvie Ricard-Blum</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Christian%20Schwerk%5BAuthor%20-%20Full%5D\">Christian Schwerk</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Bruce%20Spiegelman\">Bruce Spiegelman</a>, and&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Xiaochun%20Yu%5BAuthor%20-%20Full%5D\">Xiaochun Yu</a>.</p>\r\n<p><strong>Annotation Statistics.</strong>&nbsp;Reactome comprises 10,391 human reactions organized into 2,080 pathways involving 10,624 proteins encoded by 10,381 different human genes, and 1,735 small molecules. These annotations are supported by 25,449 literature references. We have projected these reactions onto 115,881 orthologous proteins, creating 20,164 orthologous pathways in 18 non-human species. Version 59 has annotations for 1,496 protein variants (mutated proteins) and their post-translationally modified forms, derived from 285 proteins. These have been used to annotate 506 disease-specific complexes and 897 disease-specific reactions organized into 447 pathways and subpathways, and tagged with 294 Disease Ontology terms.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, New York University Medical Center, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;<a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"https://twitter.com/reactome\">@reactome</a></strong>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<a href=\"mailto:help@reactome.org\" target=\"_blank\" rel=\"noopener noreferrer\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:16:00','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,73,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(75,689,'New Reactome Publication','new-reactome-publication','<p>A new Reactome paper titled “Functional Interaction Network Construction and Analysis for Disease Discovery” has been published in&nbsp;<a href=\"http://link.springer.com/protocol/10.1007%2F978-1-4939-6783-4_11\">Methods in Molecular Biology</a>.&nbsp;More publications from the&nbsp;<a href=\"about/team\">Reactome Team</a>&nbsp;can be found&nbsp;<a href=\"https://reactome.org/community/publications\">here</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:16:36','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,72,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(76,690,'New Reactome Paper published','new-reactome-paper-published','<p>A new Reactome paper titled “Reactome pathway analysis: a high-performance in-memory approach” has been published in&nbsp;<a href=\"https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1559-2\">BMC Bioinformatics</a>.&nbsp;More publications from the&nbsp;<a href=\"about/team\">Reactome Team</a>&nbsp;can be found&nbsp;<a href=\"community/publications\">here</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:17:08','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,71,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(77,691,'Version 60 Released','version-60-released','<p><strong>New and Updated Pathways.</strong><strong>&nbsp;</strong>In version V60, topics with new or revised pathways include: Cell cycle (<a href=\"/PathwayBrowser/#R-HSA-69231.4\">Cyclin D associated events in G1</a>), Cell-cell communication&nbsp;<a href=\"/PathwayBrowser/#R-HSA-373756.1\">(SDK interactions</a>), Cellular response to external stimuli (<a href=\"/PathwayBrowser/#R-HSA-3371497.1\">HSP90 chaperone cycle for steroid hormone receptors (SHR)</a>), Disease (<a href=\"/PathwayBrowser/#R-HSA-5423599.1\">Diseases of Mismatch Repair</a>), Gene Expression (<a href=\"/PathwayBrowser/#R-HSA-6804116.2\">TP53 Regulates Transcription of Genes Involved in G1 Cell Cycle Arrest</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8878159.1\">Transcriptional Regulation by the CBFB:RUNX3 complex</a>), Immune System (<a href=\"/PathwayBrowser/#R-HSA-8851680.2\">Butyrophilins</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-977606.3\">Regulation of complement cascade</a>), Metabolism (<a href=\"/PathwayBrowser/#R-HSA-1855183.4\">Synthesis of IP2, IP, and Ins in the cytosol</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-1660516.6\">Synthesis of PIPs at the early endosome membrane</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-1483248.5\">Synthesis of PIPs at the ER membrane</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-1660517.5\">Synthesis of PIPs at the late endosome membrane</a>, and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-1660499.5\">Synthesis of PIPs at the plasma membrane</a>), Metabolism of proteins (<a href=\"/PathwayBrowser/#R-HSA-8874211.1\">CREB3 factors activate genes</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8951664.1\">Neddylation</a>,&nbsp;<a href=\"/PathwayBrowser/#R-HSA-8852135.1\">Protein ubiquitination</a>, and&nbsp;<a href=\"/PathwayBrowser/#R-HSA-4551638.4\">SUMOylation of chromatin organizing proteins</a>), Mitophagy (<a href=\"/PathwayBrowser/#R-HSA-8934903.1\">Receptor Mediated Mitophagy</a>), Neuronal System (<a href=\"/PathwayBrowser/#R-HSA-388844.1\">Receptor protein tyrosine phosphatases interactions</a>), Organelle biogenesis and maintenance (<a href=\"/PathwayBrowser/#R-HSA-8949613.1\">Cristae formation</a>), and Transport of small molecules (<a href=\"/PathwayBrowser/#R-HSA-8949215.1\">Mitochondrial calcium ion transport</a>).</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Wei-Chih+Yang%5BAuthor+-+Full%5D\">Wei-Chih Yang</a>&nbsp;and&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jian+Lu%5BAuthor+-+Full%5D\">Jian Lu</a>&nbsp;are our external &nbsp;authors.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Joseph+Ainscough%5BAuthor+-+Full%5D\">Joseph Ainscough</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Sanjeevani+Arora%5BAuthor+-+Full%5D\">Sanjeevani &nbsp;Arora</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jorge+E+Azevedo%5BAuthor+-+Full%5D\">Jorge E Azevedo</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jeehyeon+Bae%5BAuthor+-+Full%5D\">, &nbsp;Jeehyeon Bae</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Lucia+Banci%5BAuthor+-+Full%5D\">Lucia Banci</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Gautam+Bhave%5BAuthor+-+Full%5D\">Gautam Bhave</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=David+R+Brown%5BAuthor+-+Full%5D\">David R Brown</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Roberta+Bulla%5BAuthor+-+Full%5D\">Roberta Bulla</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Alexandre+M+Carmo%5BAuthor+-+Full%5D\">Alexandre M Carmo</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Linda+Shyue+Huey+Chuang%5BAuthor+-+Full%5D\">Linda Shyue Huey Chuang</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Karlene+A+Cimprich%5BAuthor+-+Full%5D\">Karlene A Cimprich</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Laura+Crisponi%5BAuthor+-+Full%5D\">Laura Crisponi</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Alain+de+Bruin%5BAuthor+-+Full%5D\">Alain de Bruin</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Luisa+Di+Stefano%5BAuthor+-+Full%5D\">, Luisa Di Stefano</a>&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ilaria+Drago%5BAuthor+-+Full%5D\">, Ilaria Drago</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Pablo+C+Echeverria%5BAuthor+-+Full%5D\">Pablo C Echeverria</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Du+Feng%5BAuthor+-+Full%5D\">Du Feng</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Emer+S+Ferro%5BAuthor+-+Full%5D\">Emer S Ferro</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Dianne+Ford%5BAuthor+-+Full%5D\">Dianne Ford</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Frances+V+Fuller-Pace%5BAuthor+-+Full%5D\">Frances V Fuller-Pace</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Cem+Gabay%5BAuthor+-+Full%5D\">Cem Gabay</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Dominique+Gagliardi%5BAuthor+-+Full%5D\">Dominique Gagliardi</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Marcia++Haigis%5BAuthor+-+Full%5D\">Marcia&nbsp; Haigis</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=J+Wade+Harper%5BAuthor+-+Full%5D\">J Wade Harper</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Barry+Honig%5BAuthor+-+Full%5D\">Barry Honig</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Yoshiaki+Ito%5BAuthor+-+Full%5D\">Yoshiaki Ito</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Veerle+Janssens%5BAuthor+-+Full%5D\">, Veerle Janssens</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nathalie+Josso%5BAuthor+-+Full%5D\">Nathalie Josso</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jaewon+Ko+%5BAuthor+-+Full%5D\">Jaewon&nbsp; Ko</a>&nbsp;,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Vera+Kozjak-Pavlovic+%5BAuthor+-+Full%5D\">Vera Kozjak-Pavlovic</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Anastasia+Kralli%5BAuthor+-+Full%5D\">Anastasia Kralli</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Paul+J+Lehner%5BAuthor+-+Full%5D\">Paul J Lehner</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Dominique++Leprince%5BAuthor+-+Full%5D\">Dominique&nbsp; Leprince</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bruce+D+Levy%5BAuthor+-+Full%5D\">Bruce D Levy</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Wei+Li%5BAuthor+-+Full%5D\">Wei Li</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Francisco+Lozano%5BAuthor+-+Full%5D\">Francisco Lozano</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jian+Lu%5BAuthor+-+Full%5D\">Jian Lu</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Michael+J+Matunis%5BAuthor+-+Full%5D\">Michael J Matunis,</a>&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Birgit+Meldal%5BAuthor+-+Full%5D\">Birgit Meldal</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Violaine+Moreau%5BAuthor+-+Full%5D\">Violaine Moreau</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Kyungjae+Myung%5BAuthor+-+Full%5D\">Kyungjae Myung</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Joseph+H+Neale%5BAuthor+-+Full%5D\">Joseph H Neale</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Christian+Obinger%5BAuthor+-+Full%5D\">Christian Obinger</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=R+Jeroen+Pasterkamp%5BAuthor+-+Full%5D\">, &nbsp;R Jeroen Pasterkamp</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Richard+Phipps%5BAuthor+-+Full%5D\">Richard Phipps</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Didier+Picard%5BAuthor+-+Full%5D\">Didier Picard</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Elah+Pick%5BAuthor+-+Full%5D\">Elah Pick</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=David+A+Rhodes%5BAuthor+-+Full%5D\">David A Rhodes</a>,&nbsp;<a href=\"http://iribhm.org/index.php/our-research-group/roger\">Pier e P Roger</a>, Mark G Rush,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Joshua+R+Sanes%5BAuthor+-+Full%5D\">Joshua R Sanes</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Martin+Schr%C3%B6der%5BAuthor+-+Full%5D\">Martin Schröder</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Pierre+Thibault%5BAuthor+-+Full%5D\">Pierre Thibault</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Dick+J+H+van+den+Boomen%5BAuthor+-+Full%5D\">Dick J H van den Boomen</a>&nbsp;,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Thomas+E+Van+Dyke%5BAuthor+-+Full%5D\">Thomas E Van Dyke</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Roberto+M+Vanacore%5BAuthor+-+Full%5D\">Roberto M Vanacore</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nobutaka+Wakamiya%5BAuthor+-+Full%5D\">Nobutaka Wakamiya</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Qinglu+Wang%5BAuthor+-+Full%5D\">Qinglu Wang</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bart+Westendorp%5BAuthor+-+Full%5D\">, Bart Westendorp</a><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Sandra+E+Wiley%5BAuthor+-+Full%5D\">, Sandra E Wiley</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Miriam+Wittmann%5BAuthor+-+Full%5D\">Miriam Wittmann</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Guilherme+Xavier%5BAuthor+-+Full%5D\">Guilherme Xavier</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Wei-Chih+Yang%5BAuthor+-+Full%5D\">Wei-Chih Yang</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ali+A+Zarrin%5BAuthor+-+Full%5D\">Ali A Zarrin</a>, and&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bing+Zhu+%5BAuthor+-+Full%5D\">Bing Zhu</a>&nbsp;are our external reviewers.</p>\r\n<p><strong>Annotation Statistics.</strong>&nbsp;Reactome comprises 10,754 human reactions organized into 2,132 pathways involving 10,907 proteins encoded by 10,658 different human&nbsp;genes,&nbsp;1,763 small molecules, and 10,748 complexes. These annotations are supported by 26,384 literature references. We have projected these reactions onto 115,881 orthologous proteins, creating 20,701 orthologous pathways in 18 non-human species. Version 60 has annotations for 1,503 protein variants (mutated proteins) and their post-translationally modified forms, derived from 285 proteins. These have been used to annotate 506 disease-specific complexes and 906 disease-specific reactions organized into 453 pathways and&nbsp;subpathways,&nbsp;and tagged with 294 Disease Ontology terms.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;<a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a></strong>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:18:03','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,70,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(78,692,'Version 61 Released','version-61-released','<p><strong><a href=\"/PathwayBrowser/#R-HSA-913531\"><img class=\"aligncenter wp-image-4020\" style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-913531.svg\" alt=\"R-HSA-913531\" width=\"650\" height=\"400\" /></a></strong></p>\r\n<p><strong>New and Updated Pathways.</strong><strong>&nbsp;</strong>In version&nbsp;V61, topics with new or revised pathways include: Gene expression (<span style=\"text-decoration: underline;\"><a href=\"//reactome.org/PathwayBrowser/#/R-HSA-8878171\">Transcriptional regulation by RUNX1</a></span>),&nbsp;Immune System (<a href=\"/PathwayBrowser/#/R-HSA-447115\">Interleukin-12 family signaling</a>), Signal Transduction (<span style=\"text-decoration: underline;\"><a href=\"/PathwayBrowser/#/R-HSA-6807070\">PTEN Regulation</a></span>), and Transport of small molecules (<span style=\"text-decoration: underline;\"><a href=\"/PathwayBrowser/#/R-HSA-8981607\">Intracellular oxygen transport</a></span>).</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Arkaitz%20%20Carracedo%5BAuthor%20-%20Full%5D\">Arkaitz Carracedo</a>&nbsp;and&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Leonardo%20Salmena%5BAuthor%20-%20Full%5D\">Leonardo Salmena</a>&nbsp;were our external authors.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Bailly%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26176610\">Sabine Bailly</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=thorsten+burmester%5BAuthor+-+Full%5D\">Thorsten Burmester</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Linda+Shyue+Huey++Chuang%5BAuthor+-+Full%5D\">Linda Shyue Huey Chuang</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Ito%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28299675\">Yoshiaki Ito</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nisha+Kriplani%5BAuthor+-+Full%5D\">Nisha Kriplani</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Nick+Leslie%5BAuthor+-+Full%5D\">Nick Leslie</a>, and&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Esther+van+de+Vosse%5BAuthor+-+Full%5D\">Esther van de Vosse</a>&nbsp;were our external reviewers.</p>\r\n<p><b>Annotation Statistics.</b>&nbsp;Reactome comprises 11,042 human reactions organized into 2,148 pathways involving 10,940 proteins encoded by 10,691 different human genes, 1,763 small molecules, and 11,041 complexes. These annotations are supported by 26,859 literature references. We have projected these reactions onto 120,804 orthologous proteins, creating 20,780 orthologous pathways in 18 non-human species. Version 61 has annotations for 1,334 protein variants (mutated proteins) and their post-translationally modified forms, derived from 285 proteins. These have been used to annotate 506 disease-specific complexes and 906 disease-specific reactions organized into 453 pathways and subpathways, and tagged with 294 Disease Ontology terms.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:20:12','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,69,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(79,693,'New Protein-protein Interaction files','new-protein-protein-interaction-files','<p><img class=\"aligncenter wp-image-4029\" style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Interaction-nologo.png\" alt=\"Interaction\" width=\"681\" height=\"400\" /></p>\r\n<p>The Reactome team has released new versions of our protein-protein interaction files derived from reactions and complexes. These files were updated following user feedback, with the goal of providing extra annotation features through support from the&nbsp;<a href=\"http://github.com/HUPO-PSI/miTab/blob/master/PSI-MITAB27Format.md\">PSI-MITAB 2.7</a>&nbsp;data format. Interactions are computationally generated based on the data stored on complexes and reactions. The interactions provided by Reactome are not curated and are not experimental data. In addition, the complexes and reactions in species other than human are derived by orthology inference from the corresponding human complexes and reactions. Tab-delimited formatted files are also provided for human and all species.</p>\r\n<p>The four new files are:</p>\r\n<ul>\r\n<li><a href=\"/download/current/interactors/reactome.all_species.interactions.psi-mitab.txt\">Full list of protein-protein interactions, incl. non-human species [PSI-MITAB]</a></li>\r\n<li><a href=\"/download/current/interactors/reactome.all_species.interactions.tab-delimited.txt\">Full list of protein-protein interactions, incl.&nbsp;non-human species [tab-delimited]</a></li>\r\n<li><a href=\"/download/current/interactors/reactome.homo_sapiens.interactions.psi-mitab.txt\">Human protein-protein interactions [PSI-MITAB]</a></li>\r\n<li><a href=\"/download/current/interactors/reactome.homo_sapiens.interactions.tab-delimited.txt\">Human protein-protein interactions [tab-delimited]</a>.</li>\r\n</ul>\r\n<p>Documentation about these files is available from the&nbsp;<a href=\"/download-data/\">Data Download</a>&nbsp;page.</p>\r\n<p>The former versions of the protein-protein interaction files are still available but will be removed from service as of Version 62 in September 2017. We encourage users that programmatically access these files to update the scripts.</p>\r\n<p><strong>About the Reactome&nbsp;Project.</strong>&nbsp;Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Cold Spring Harbor Laboratory, New York University Langone Medical Center, Oregon Health and Science University, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the Reactome website.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"https://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:24:21','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,68,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(80,694,'New SBML Level 3 Version 1 export is now available','new-sbml-level-3-version-1-export-is-now-available','<p><a href=\"/download-data/\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/SBML.png\" alt=\"SBML\" width=\"755\" height=\"400\" /></a></p>\r\n<p>As part of our efforts to give our modeling user community better experience, we have updated the SBML export to Level 3 Version 1, which is organized in a modular manner. &nbsp;Our initial export provides a richer annotation &nbsp;syntax and we will explore supporting other SBML Level 3 Packages in the future. The SBML data export can be used by any tool that supports SBML L3V1.</p>\r\n<p>The SBML Level 3 Version 1 export is available for download here:&nbsp;<a href=\"/download/current/homo_sapiens.3.1.sbml.tgz\">https://reactome.org/download/current/homo_sapiens.3.1.sbml.tgz</a>.</p>\r\n<p>We are also providing a programmatic interface to access these updated SBML files through our&nbsp;<a href=\"dev/content-service/\">ContentService</a>&nbsp;for all species at&nbsp;<a href=\"/ContentService/#!/exporter/toSBMLUsingGET\">https://reactome.org/ContentService/#!/exporter/toSBMLUsingGET</a></p>\r\n<p>Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Cold Spring Harbor Laboratory, New York University Langone Medical Center, Oregon Health and Science University, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the Reactome website.</p>\r\n<p>Follow us on Twitter:&nbsp;<a href=\"https://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-20 17:25:06','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,67,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(81,695,'The Pathway Browser','the-pathway-browser','<p><span>The Pathway Browser is the primary means of viewing and interacting with pathways in Reactome. </span></p>\r\n<p><span>See our Youtube Video explaining <a href=\"https://youtu.be/zfdGmFnsvIU\">navigating the Pathway Browser</a> for more information. </span></p>\r\n<p><span>The Pathway Browser includes tools for analysing datasets and exploring pathways. These tools allow several types of analysis:</span></p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li><span>Pathway over-representation analysis and pathway topology-based analysis</span></li>\r\n<li><span>Comparison of a pathway with its equivalent in another species</span></li>\r\n<li><span>The overlay of user-supplied expression data onto a pathway</span></li>\r\n<li><span>The overlay of protein-protein or protein-compound interaction data from external databases or user-supplied data onto a pathway  </span></li>\r\n</ul>\r\n<p><span>See the section <a href=\"https://reactome.org/userguide/analysis\">Reactome Analysis Tools </a>for details of the analysis tools.</span></p>\r\n<p></p>\r\n<p><span>The Pathway Browser is launched by clicking the Browse Pathways button on the Homepage:</span></p>\r\n<p><img src=\"images/userguide/pathway_browser/pathway_browser_button.gif\" alt=\"pathway browser button\" /></p>\r\n<p><span>When the Pathway Browser opens, it displays an overview of all Reactome pathways. Pathways are organized hierarchically and often have sub-pathways. At the highest hierarchical level, all pathways are represented in an Overview. At intermediate levels, pathways are often represented as interactive illustrations, with selectable regions that act as links to the lower levels, which are represented as detailed Pathway Diagrams. Reactome’s Overview uses a unique graphical visualization of the hierarchy, representing pathways at the uppermost hierarchical level as central nodes (circles) with subpathways arranged concentrically (in rings) around them. Many have further concentric rings of ‘child’ subpathways. Pathway nodes are connected to their subpathways by edges (lines). This overview displays all pathways and allows rapid zooming to details. As you zoom in, labels appear for the subpathway nodes. This view is also used to display data analysis results. </span></p>\r\n<p><span>Nodes in the overview link to detailed pathway diagrams that use a style based on Systems Biology Graphical Notation. They have several features intended to ensure an appropriate level of information detail, including subpathway shading and fade-in of labels, commonly occurring small molecules and structures.</span></p>\r\n<p><span>To see the details of a specific pathway, select and click (or double-click) the node representing the pathway. Alternatively select and click on the name in the Pathway Hierarchy panel on the left.</span></p>\r\n<p></p>\r\n<p><span>Features of the Pathway Browser are labelled in the diagram below.  </span></p>\r\n<p></p>\r\n<p><img src=\"images/userguide/pathway_browser/pathway_browser_labelled.png\" alt=\"pathway browser labelled\" /></p>\r\n<p></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Home</strong></span> – the Reactome logo is a button linked to the homepage.</span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Species</strong> </span>– Reactome is a database of curated human biological pathways. These human pathways are used to computationally infer equivalent pathways in model organisms (described in detail </span><a href=\"http://reactomedev.oicr.on.ca/electronic_inference.html\"><span>here</span></a><span>). Use the Species drop-down to select a species and view the predicted pathway. Note that infectious disease pathways that show the interaction of pathogens with human proteins are included in Homo sapiens.</span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Layout</strong></span> – The pathway browser is divided into 3 main sections. These are: the Hierarchy Panel, on the left, the Details Panel, bottom right, the Pathway Panel, top right. The layout buttons allow you to show or hide these panels.</span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Analyse Data</strong> </span>– Opens a panel where analysis can be performed. </span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Key</strong></span> – this opens a key explaining the objects present in the pathway panel. </span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Zoom/Move Toolbar</strong> </span>- Click on the arrows to move the diagram. Click on the plus and minus buttons to zoom in and out. Alternatively, click and drag the diagram, and use your mouse scroll wheel to zoom. </span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>In-Diagram Search</strong> </span>– click the button to open the search panel. When the Overview or a Pathway Diagram is displayed, objects represented in the displayed diagram or all diagrams are searchable.</span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Fit to Page</strong></span> - resizes the Pathway Overview or Diagram to fit the available space.</span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Open Diagram</strong> </span>– when a pathway node is selected in the Pathway Overview, this button opens the corresponding Pathway Diagram</span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Illustrations</strong></span> – some Reactome pathways have a corresponding high-quality graphic. Select this button to view.</span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Export</strong></span> – use this button to download the visible region of the Pathway Overview or Diagram, including any analysis overlay, as a PNG file or upload it to GenomeSpace. </span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Thumbnail</strong></span> – Provides an overview of the entire pathway when zoomed in so that only a region of the pathway is displayed. </span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Pathway Panel</strong></span> - This is where Pathway Diagrams are displayed when they are selected in the Pathway Hierarchy or in search results. If no pathway is selected, this panel displays a brief Reactome help guide.</span></p>\r\n<p><span><strong>Details Panel</strong> - Contains details of objects when they are selected in the Pathway Diagram. The content depends on the type of object, i.e. pathway, reaction, complex, set, protein or small molecule (see the Details section). </span></p>\r\n<p><span><span style=\"text-decoration: underline;\"><strong>Event Hierarchy Panel</strong></span> – Shows the hierarchical organisation of Reactome pathways and pathway events or steps, known as reactions. When the Pathway Browser is opened, only the top level of the hierarchy is shown, as an alphabetical list of Reactome’s main topics (superpathways). Most topics are split into sub-pathways, which may be further divided into sub-sub-pathways. Most topics in Reactome are organised in this hierarchical manner. Sub-pathways can be revealed by clicking on the + icon to the left of the pathway name, and hidden by clicking on the - icon.</span></p>\r\n<p><span style=\"text-decoration: underline;\"><strong>Settings Sidebar</strong></span> – Used for context-sensitive help, to configure colour schemes and the default source of interactors.</p>\r\n<h2><a id=\"event-hierarchy\">Event Hierarchy</a></h2>\r\n<p>The order of reactions from top to bottom in the Event Hierarchy Panel usually follows their order as steps in the pathway so that the preceding reaction is above and the subsequent reaction below, but this is not always the case. Pathways may not be linear; they can be circular or branched. Consequently, reactions often have multiple preceding or subsequent reactions. To view a complete list of the events that precede or follow a reaction, refer to the Preceding and Following Events section in the Details Panel, as described in the Details Panel chapter below.</p>\r\n<p>The event hierarchy panel provides a nested structure containing large topics that are organised as pathways and subpathways and can be expanded to show individual events and steps. The icons next to the event describe the event. </p>\r\n<p>You can hover over the icon in the Event Hierarchy panel to show a tool-tip that describes the event, as illustrated in the figure below. </p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/LdcnK4AoSQ9wjJQ-39V1Nk4ysKCVezmULy2cSMWnQuV2XvWAKkvoRVg4BHU2L-Nxr3w2Dk6fCTuNUHLuzH7WHDU0SHMp-7QkFKbKFalfaf1kE1XDg_q_QJYWtZyx4iyl0W9cV9sWQSGse2LSFgB3yug\" alt=\"\" width=\"552\" height=\"302\" /></p>\r\n<p>This table shows a legend of the icons and their respective descriptions in Reactome: </p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/Y0LFxxY9pFsdsmugJ8ftNn8f0F8eT7pve1aPBkhYopTUV_ejcwT26QqdLE1_ZF3MmXnYER7Rn_c7NGfgprtCMsDZ0z5_Ari6OcnTGLjQbbsKmZ8ySxjXdjeFHSSiaTecyTVcd2Zb58Oefgxx6ruYzX8\" alt=\"\" width=\"528\" height=\"555\" /></p>\r\n<p></p>\r\n<h2><a id=\"pathway-diagrams\"><b>Pathway Diagrams</b></a></h2>\r\n<p></p>\r\n<p><span>Pathway Diagrams represent pathways as a series of connected molecular events, known in Reactome as ‘reactions’, which can be considered as steps in the pathway.</span></p>\r\n<p><span>Cellular compartments are represented as pink/orange boxes with a double boundary. A typical diagram has a box to represent the cytosol, bounded by a double-line that represents the plasma membrane. The white area outside this box represents the extracellular space. Other organelles are represented as additional labelled boxes within the cytosol. </span></p>\r\n<p><span>Molecules, represented in diagrams as blue/green boxes, are placed in the physiologically-correct cellular compartment, or lie on the boundary of a compartment to indicate that they are in the corresponding membrane, e.g. a plasma membrane protein will be placed on the boundary of the cytosol.</span></p>\r\n<p></p>\r\n<p><span>Reactions typically include:</span></p>\r\n<ul>\r\n<li><span>Input and output molecules, and when relevant a catalyst (see diagram A below)</span></li>\r\n<li><span>Inputs, outputs and catalyst are represented as boxes or ovals</span></li>\r\n<li><span>Green boxes with rounded corners are proteins</span></li>\r\n<li><span>Green boxes with square corners are proteins that have no UniProt accession (or did not at the time the reaction was created).</span></li>\r\n<li><span>Green ovals are small molecules or sets of small molecules</span></li>\r\n<li><span>Blue boxes with a double boundary are sets, i.e. proteins or small molecules that are functionally equivalent. </span></li>\r\n<li><span>Blue boxes with cut corners are complexes, i.e. proteins and/or other molecules that are bound in a multimolecular entity.</span></li>\r\n<li><span>Green boxes with a white inner box are sub-pathways.</span></li>\r\n<li><span>Reaction input and output molecules are joined by lines to a central ‘reaction node’ (surrounded by a green box in Figure A below). Clicking this node selects the reaction. </span></li>\r\n<li><span>The outputs of a reaction have an arrowhead on the line connecting them to the reaction node.</span></li>\r\n<li><span>Reaction inputs/output molecules are often connected by arrows to preceding or subsequent reactions (i.e. the preceding/subsequent steps in the pathway). </span></li>\r\n<li><span>Catalysts are connected to the reaction node by a line ending in a circle. </span></li>\r\n<li><span>Numbered boxes on the line between an input/output and the reaction node indicates the number of molecules of this type in the reaction (when n &gt;1). </span></li>\r\n<li><span>Molecules that regulate a reaction are connected to the reaction node by a line ending in an open triangle for positive regulation or a ‘T’-shaped head for negative regulation (see B below).</span></li>\r\n<li><span>A white box labelled P on the boundary of an object indicates a phosphorylation event.</span></li>\r\n<li><span>Proteins or small molecules that are also part of a displayed set may be connected to the set object by a line of short dashes.</span></li>\r\n<li><span>Sets with overlapping content may be connected by a line of long dashes. </span></li>\r\n<li><span>Reactions that represent a disease process use red connecting lines. Objects associated with a disease are bordered in red.</span></li>\r\n</ul>\r\n<p><span><img src=\"https://lh7-us.googleusercontent.com/dVO9EBIDoI0NUzmqhLqrZ8Gq6tTPliYvK6tv8CXzclZ5gnQ-0JOev-7uUXNNlKbV8eD98vz8lCk3AUU7HKWPFOQOWRbL5BrHJw6KVHA2utYd2G1BcMwx73Wd3ylc0LWZDxwdXiFQ5YSBFJvQ2Mq-1A\" alt=\"\" width=\"487\" height=\"214\" /></span></p>\r\n<h2><a id=\"ehld\"><b>Enhanced high-level diagrams and Subpathway icons</b></a></h2>\r\n<p></p>\r\n<p><span>Many pathways at the higher levels of the hierarchy are too large to represent as a single, detailed pathway diagram. Instead they are often represented as an illustration, known as an enhanced high-level diagram, which graphically represents the subpathways as selectable regions of an illustration. When the mouse pointer is moved over a region, it becomes surrounded by a blue highlighting \'halo\'. When you click on a region it becomes selected and outlined in dark blue, the corresponding subpathway is selected in the Hierarchy panel and the Details panel updates to show details of the subpathway. Double-clicking on a region opens the corresponding detailed Pathway Diagram.</span></p>\r\n<p><span>In the Hierarchy panel, pathways that have an enhanced high-level diagram have a blue icon to the left of their name.</span></p>\r\n<p><img src=\"images/userguide/pathway_browser/EHLD.png\" alt=\"EHLD\" /></p>\r\n<p></p>\r\n<p><span>A few higher level pathways use an older visualization of large topics, where subpathways are represented by a box with a green boundary, the subpathway icon (see example below). Selecting a subpathway icon has the same result as selecting a region of an enhanced-level pathway diagram as described above. </span></p>\r\n<p><img src=\"images/userguide/pathway_browser/green_boxes.png\" alt=\"green boxes\" /></p>\r\n<p><span><span style=\"color: #333333; font-family: \'Open Sans\', Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: #ffffff; float: none;\">In some pathways, Reactome links to other related pathways through green boxes. In order to differentiate between related subpathways that are children </span><span style=\"color: #333333; font-family: \'Open Sans\', Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: #ffffff; float: none; display: inline;\">of the same parent pathway (subpathways) and related subpathways that are children of a different parent pathway</span><span style=\"color: #333333; font-family: \'Open Sans\', Helvetica, Arial, sans-serif; font-size: 14px; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: #ffffff; float: none; display: inline;\"> (interacting pathways), the Pathway Browser uses slightly different glyphs as illustrated in the figure below.</span></span></p>\r\n<p><span><img src=\"images/userguide/pathway_browser/subpathways_interacting.png\" alt=\"subpathways interacting\" width=\"250\" height=\"205\" /></span></p>\r\n<h3><b>Subpathway Shading</b></h3>\r\n<p></p>\r\n<p><span>When a Pathway Diagram contains subpathways, the area containing each subpathway is overlaid with a coloured box, labelled with the subpathway name, to help locate it in the larger diagram. Below is the diagram for ERBB4 signaling. The four subpathways are represented as 4 boxed and shaded areas.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/gM-xLON2Aa-T8Ax7evA5kB_uKazCb7qlXwF9kKwQUmsVzJGxvDs3tohzGV_YglupRksZwE1c0C51gHfUAvSVKEfvVE4-uw5mvqLfacvgAlv78Ws8CTTKT0JU4B7ivRQc_Z_fKnvGBfNudDZnmtrwiA\" alt=\"\" width=\"377\" height=\"583\" /></p>\r\n<p></p>\r\n<h2><a id=\"pathway-diagram-zoom\"><b>Pathway Diagram zoom detail level</b></a></h2>\r\n<p></p>\r\n<p><span>The pathway diagram represents different levels of detail, depending on the zoom level. When fully zoomed-out, regions corresponding to subpathways are shaded. The boxes representing molecules or groups of molecules have no text labels and trivial molecules, such as water and ATP, are not represented. At the next level of zoom, subpathway shading disappears, while trivial molecules and the circular red icon indicating that a protein has known interactors fade into view. At closer levels of zoom the reaction nodes appear, as do boxes containing a number on the lines connecting molecules to the reaction node if stoichiometry is greater than one. At this level of zoom, if interactors are displayed their names appear. At the highest level of zoom, if available, proteins, small molecules and interactors display structural diagrams from PDBe or ChEBI with associated details.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/DxdyJhN-zdmOurblU7Ad2S_B2sbFoxyaASJ3tYMqVSn4XEY30iVau1F_whhOMT8YdakKB9ofAyG1eZYs9krLEfbiJ15NJSH1jt6vkSDBqT_AQ0Ac_KZQTMNgvPQgNBzpqzeApodZ9q8K6xx4rM2IpA\" alt=\"\" width=\"645\" height=\"297\" /></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/jw2FfgyBdzz2XUy959bmIUVW9m5ZDVbAiYOrJYoVYuQPgUWAfIcZ0NSc8stZKJjxEGf6R06n5j9zZUD0L2JW2uUunWT7poJKMtrrCE002bOtZnTuJ2OoWU8fmpqv88rbz4BKKa2AEPakwhsnR55LBQ\" alt=\"\" width=\"343\" height=\"218\" /></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/fh4FjWeNi6tZs6U2Jpim1RqTvQpwNjUXKWxD2JGUcLDXyOGehlNsO8YFX6B9wNx4m67hvjjdNeFm1mf5igw74HYRFgSRlmP6MPr0qSXfT_qyMKUeWV7oDkYQh_6jdKP-JVG9d3bGTGbS2LVcY000ZA\" alt=\"\" width=\"343\" height=\"264\" /></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/5rqZfLPJoFOHUD9zELUG3ZKQfGwqOlLP0bAXpzVl_J-J16jdOwG9rjztEN84NZTFSHhbzeBJPi1_jKXxAE5dODm3md7eJKhEuLHDEJfbDk0N2p_Orl2g1y_xerHwMfbZi987s_wI_1jzeobrg0PtqQ\" alt=\"\" width=\"337\" height=\"304\" /></p>\r\n<p></p>\r\n<h2><a id=\"navigating-pathway-diagrams\"><b>Navigating Pathway Diagrams</b></a></h2>\r\n<p></p>\r\n<p><span>The Pathway Diagram, Hierarchy and Details Panels are interactively connected. Selecting an entity (molecular object) or event in the diagram will reveal relevant information in the Details Panel. Clicking a pathway name in the Hierarchy will open the corresponding Pathway Diagram in the Pathway Diagram Panel, and reveal details of the pathway in the Details Panel. If a subpathway is contained within the diagram that is displayed, clicking its name in the hierarchy will cause all the reactions in the pathway to be highlighted in blue. In the figure below, the diagram for Platelet homeostasis is displayed in the Pathway Panel, and the sub-pathway Prostacyclin signalling through prostacyclin receptor has been selected in the Hierarchy, causing all the reactions in this sub-pathway to be highlighted (blue) on the Pathway Diagram.</span></p>\r\n<p><img src=\"images/userguide/pathway_browser/subpathway_selection.png\" alt=\"subpathway selection\" width=\"848\" height=\"517\" /></p>\r\n<p></p>\r\n<p><span>Similarly, selecting a reaction in the Hierarchy causes that reaction to be highlighted in the pathway diagram and updates the Details Panel to show details of the reaction. If the reaction is not in the visible region of the diagram, the view will re-centre and zoom to show it. If instead of clicking an event, you hover the mouse pointer over its name, it will be highlighted in yellow. In the figure above, the sub-pathway Platelet calcium homeostasis is highlighted in yellow.</span></p>\r\n<p></p>\r\n<h3><b>Getting Started</b></h3>\r\n<p></p>\r\n<h3><span>Hierarchy Panel </span><b>Exercises</b></h3>\r\n<p></p>\r\n<p><span>This exercise is to check that you understand the organisation of the Hierarchy Panel. You don’t need to look at the Diagram Panel. </span></p>\r\n<p><span>From the Home page, search for PDGF signaling.</span></p>\r\n<p><span>In the results page, open the expandable hierarchy in the section Locations in the Pathway Browser by clicking on the + button. </span><b>DON’T</b><span> click on the location before expanding the hierarchy! This will open the pathway diagram in the Pathway Browser.</span></p>\r\n<p><span>Look at the Hierarchy Panel on the left. </span></p>\r\n<ol>\r\n<li><span>How many sub-pathways does this pathway have?</span></li>\r\n<li><span>How many reactions are in the first sub-pathway?</span></li>\r\n<li><span>What reaction(s) follow ‘Translocation of PDGF from ER to Golgi’? </span><i><span>Hint:Look in the Details Panel, bottom-right.  If it is hidden, use the layout buttons in the top right corner, above the diagram, to reveal it.</span></i></li>\r\n</ol>\r\n<h3><span>Subpathway </span><b>Exercises</b></h3>\r\n<p></p>\r\n<p><span>This exercise is to check that you understand Subpathway icons and the relationship between the Pathway Hierarchy and Pathway Diagram.</span></p>\r\n<p></p>\r\n<p><span>Open the pathway Hemostasis in the Pathway Browser. </span></p>\r\n<p><span>Select the subpathway ‘Platelet Hemostasis’. </span></p>\r\n<ol>\r\n<li><span>What happens if you hover your mouse pointer in the Hierarchy on the sub-pathway ‘Platelet calcium hemostasis’?</span></li>\r\n<li><span>What happens if you click in the Hierarchy on the sub-pathway ‘Platelet calcium hemostasis’?</span></li>\r\n<li><span>What happens if you click in the hierarchy on the reaction ‘Binding of ATP to P2X receptors’?</span></li>\r\n</ol>\r\n<p></p>\r\n<p><b>More Information</b></p>\r\n<p><span>There are 5 subtypes of reaction node, indicating reaction subclasses.</span></p>\r\n<p></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/YXfWEaVjFMFQ5HdH96MvWb4gupfJw2C7JWYCgWZ0LQx1Ec_Pyu7Y5N8S8w424oRfBOjgd0PO78hVRq4MrJvK8V0P0p5n_bJk7fzMLbUyookLSS7Es0tiTlv3koxsL6J13m0j8JFqkjHJlDA6YBah7w\" alt=\"\" width=\"602\" height=\"97\" /></p>\r\n<p></p>\r\n<ul>\r\n<li><span>Open squares represent ‘transition’, i.e. a change of state that is not one of the defined subclasses.</span></li>\r\n<li><span>Solid circles represent ‘association’, i.e. binding</span></li>\r\n<li><span>Double-bordered circles represent ‘dissociation’ </span></li>\r\n<li><span>A square with two slashes represents an ‘omitted process’. This is used when the full details of a reaction have been deliberately omitted. This is most commonly used for events that include representative members of a large family to illustrate the general behaviour of the group. It can be used for reactions that occur with no fixed order or stoichiometry, or for degradation events where the output is a random set of fragments.</span></li>\r\n<li><span>Squares containing a question mark represent an ‘uncertain process’, where some details of the reaction are known, but the process is thought to be more complex than represented. Explanatory details are typically included in the Description.</span></li>\r\n</ul>\r\n<p></p>\r\n<h3><span>Reaction Node </span><b>Exercises</b></h3>\r\n<p></p>\r\n<p><span>This exercise is to check that you understand reaction node subtypes.</span></p>\r\n<p></p>\r\n<p><span>From the Home page, search for the pathway ‘Effects of PIP2 hydrolysis’ and open it in the Pathway Browser (in any of the several locations)</span></p>\r\n<ol>\r\n<li><span>What symbol represents the reaction for ‘Binding of IP3 to the IP3 receptor’?</span></li>\r\n<li><span>What symbol represents the reaction ‘IP3R tetramer:I(1,4,5)P3:4xCa2+ transports Ca2+ from platelet dense tubular system to cytosol? What subtype of reaction is this?</span></li>\r\n<li><span>Open the subpathway ‘Arachidonate production from DAG’. What is the name of the catalyst for ‘2-AG hydrolysis to arachidonate by MAGL’? Can you name the three outputs of this reaction? </span></li>\r\n<li><span>Can you find the UniProt ID for the catalyst? Hint: There are several ways to find this, two require you to select something!</span></li>\r\n</ol>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:48:22','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,11,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(82,696,'Details Panel','details-panel','<p><span>The Details Panel is the bottom right panel of the Pathway Browser. It displays the details of molecular objects or events when selected in the Pathway Diagram or Hierarchy. The details shown depend on the type of molecule or event selected. The Details Panel can be revealed/hidden using the middle Layout button.  </span></p>\r\n<p></p>\r\n<p><span>The Details Panel has several tabs:</span></p>\r\n<p></p>\r\n<h3><b>The Description tab </b></h3>\r\n<p></p>\r\n<p><span>This tab contains the defining details of the selected diagram object. </span><span>These details vary, depending on what is selected in the diagram. </span><span>For example, reaction details always describe the input and output molecules, and where relevant, the enzyme catalyst. Other items in the Description tab </span><span>may</span><span> include:</span></p>\r\n<p><span>Summation – a summary of the molecular event, often with background information and additional literature citations</span></p>\r\n<p><span>Preceding/Following Events - Links to reactions that precede/follow this reaction.</span></p>\r\n<p><span>Cellular Compartment - Identifies the cellular compartment for the reaction, with the associated Gene Ontology (GO) term.</span></p>\r\n<p><span>Inferred from another species - Reactome represents human biology. Wherever possible key references contain experimental data obtained using human reagents. If the only data available is from model organisms, a human event may be inferred from this data, if the Author, Curator and Reviewer agree that this inference is valid. The inferred event is labelled with an icon ‘Inferred from another species’ in the Hierarchy and in the Details panel, where there is a link to the molecular details of the event as it occurs in the model organism and the supporting literature references. </span><span>A description of the inference process can be found here.</span></p>\r\n<p></p>\r\n<p><span>Computationally Inferred To – This drop-down list represents the species available for computationally predicted pathways, inferred from Reactome’s human pathway. References – all events in Reactome include links to original research publications containing experimental data, the evidence that the event has been shown to exist.</span></p>\r\n<p><span>Authored - an expert biologist who contributed materials allowing construction of the reaction/pathway. All Reactome pathways are authored by an expert biologist.</span></p>\r\n<p><span>Reviewed – all Reactome pathways are peer reviewed by an expert biologist who verifies the content provided by the Author.</span></p>\r\n<p><span>Details are organised into panels. Panels that have a plus icon on the right hand side can be expanded to reveal further levels of detail by clicking on the icon or the detail name.</span></p>\r\n<p><img src=\"images/userguide/details/details_output.gif\" alt=\"details output\" /></p>\r\n<p><span>Clicking the plus icon reveals further details:</span></p>\r\n<p><img src=\"images/userguide/details/further_details.gif\" alt=\"further details\" /></p>\r\n<p><span>Panels that represent molecules have an icon on the left hand side that represents the molecule subtype, e.g. the protein icon is a green circle. Mouse-over the icon to see an explanation of the subtype. The icon is a selection button; clicking the icon replaces the current details with a new panel of details specific to the selected molecule, which is highlighted on the pathway diagram.  </span></p>\r\n<p></p>\r\n<h3><b>The Molecules Tab </b></h3>\r\n<p></p>\r\n<p><span>This tab details all the molecules involved in the pathway displayed in the panel above. </span><span>When a pathway name is highlighted in the hierarchy, the total number of molecules in that pathway is shown in a bubble on the molecules tab.</span></p>\r\n<p>Molecules are grouped into the subtypes Protein, Chemical Compound, Sequences (for DNA/RNA) and Other.</p>\r\n<p><span>If an item is selected in the diagram, the corresponding molecules are highlighted in the details panel, and the bubble next to the ‘Molecules’ tab enumerates the fraction of the total pathway molecules that are represented in that item.</span></p>\r\n<p><img src=\"images/userguide/details/molecules.png\" alt=\"molecules\" /></p>\r\n<p><span>The Download button to the right of the pathway name and species buttons allows details of the molecules to be downloaded in several formats. Click on the buttons to select the fields you want included in the output file, and the format of the file. Click Download to save the file.</span></p>\r\n<p></p>\r\n<h3><b>The Structures Tab</b></h3>\r\n<p></p>\r\n<p><span>The content displayed in this tab depends on the type of object or event selected in the pathway diagram. For Reactions it will display equivalent reaction diagrams from the </span><a href=\"http://www.ebi.ac.uk/rhea/home.xhtml\"><span>Rhea</span></a><span> database if available:</span></p>\r\n<p></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/g6iumaN6NuhJGuh0Lt2VlaAWYvtWXWkDAHAFydu-LqjKjXtiUzvVuAdIflUs6TRNAnksDhtMsfHuiBGYHBoednc7mOcuemJ--VZZRhBGzfLmbwl0Fj0EbjKl2ttq1RU7ftfYM4cNQr-FYkMXIRJBRw\" alt=\"Screen Shot 2017-07-24 at 3.40.03 PM.png\" width=\"624\" height=\"416\" /></p>\r\n<p><span>The panel has a link top-left to details in the Rhea database. </span></p>\r\n<p><span>The molecular structures are linked to the </span><a href=\"http://www.ebi.ac.uk/chebi/\"><span>ChEBI</span></a><span> database:</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/6k3XSs45-pLn5HxU-u6P1gvTEmV61rKJDW278KuDMnRqeWS8HLhv05dbUwVVA1GflKWZ-iVufwn68vCFoYvYhd6rVZN_7GEikOLMcO6WHUCMCS1ck0EbYxzv1_SO8BD3n4fgOZNWDWHQyeP5LKjiGw\" alt=\"Screen Shot 2017-07-24 at 3.41.20 PM.png\" width=\"550\" height=\"318\" /></p>\r\n<p><span>For Proteins or pathway objects that represent sets or complexes, corresponding structure information from </span><a href=\"http://www.ebi.ac.uk/pdbe/\"><span>PDBe</span></a><span> is represented if available:</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/8GbH3m-qGBUHN56s2oTSlOTzUy5WSQvtXhou9wb2a32LeTU0Q-bEB7SFpeIoMHj5sSc-0Cp_YQ0MDKI3Zvj2Uy3Ray8xHyiRtBkzQCgYs7b5Eow9TBGwKHyVvpDLQp7ESW_VnzzD2YTg5kwNcPSYhQ\" alt=\"\" width=\"528\" height=\"118\" /></p>\r\n<p></p>\r\n<p><span>For simple molecules, the structures tab represents information from the ChEBI  database:</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/7_W66YJKtBA9pvDOPgM_8h2LMgTuQ4GOaaxbUqPieS27oBiACQ0R6oxL1lUKuwoF6gBCvGBGLiOYkf9fkXsnQM5ihus1-MgTw6epeeC42aaORf2KJqfpW5NDmsiTv-JOOkuaeUGLE3EBPcbknUUv3w\" alt=\"\" width=\"529\" height=\"140\" /></p>\r\n<p></p>\r\n<h3><b>The Expression Tab </b></h3>\r\n<p></p>\r\n<p><span>This tab represents expression information obtained from the </span><a href=\"http://www.ebi.ac.uk/gxa/\"><span>Expression Atlas</span></a><span>. </span><span>Note that at the moment, only information from the first 50 genes in the pathway is displayed or available for download from the Expression Atlas widget. </span></p>\r\n<p><img src=\"images/userguide/details/expression_atlas.gif\" alt=\"expression atlas\" /></p>\r\n<p></p>\r\n<h3><b>The Analysis Tab</b></h3>\r\n<p></p>\r\n<p><span>This tab displays the results of analyses – see the section Reactome Tools for further details</span></p>\r\n<p></p>\r\n<h3><b>The Downloads Tab</b></h3>\r\n<p></p>\r\n<p><span>The Downloads Tab contains buttons to start a download of the currently displayed pathway in several human-readable or computationally reusable formats.  </span></p>\r\n<p><img src=\"images/userguide/details/downloads.gif\" alt=\"downloads\" /></p>\r\n<p></p>\r\n<h3><span>Get Started:</span></h3>\r\n<p></p>\r\n<h3><b>Exercises:</b></h3>\r\n<p></p>\r\n<ol>\r\n<li><span>Find the reaction ‘Activated type I receptor phosphorylates SMAD2/3 directly’. What pathway does it belong to?</span></li>\r\n<li><span>In which cellular compartment does this reaction take place?</span></li>\r\n<li><span>What is the GO molecular function associated with the catalyst?</span></li>\r\n<li><span>What reference has experimental verification of this reaction?</span></li>\r\n<li><span>Is this reaction predicted to occur in Canis familiaris? In C. elegans?</span></li>\r\n<li><span>Is this event likely to occur in liver?</span></li>\r\n<li><span>Are 3D structures available for TGF-beta1?</span></li>\r\n</ol>\r\n<p></p>\r\n<h3><span>More info:</span></h3>\r\n<p></p>\r\n<p><b>Pathway Description tab:</b></p>\r\n<p><span>The Description tab for a pathway includes:</span></p>\r\n<ul>\r\n<li><span>pathway name</span></li>\r\n<li><span>species</span></li>\r\n<li><span>stable identifier</span></li>\r\n<li><span>pathway summation</span></li>\r\n<li><span>a drop-down menu to select species for which the pathway is computationally predicted to be conserved, where appropriate. A description of the inference process can be found here.</span></li>\r\n<li><span>the GO biological process term for the pathway, where appropriate. Expanding the panel provides a link-out to GO.</span></li>\r\n<li><span>literature references </span></li>\r\n</ul>\r\n<p><b>Reaction Description tab:</b></p>\r\n<p><span>The Description tab for a reaction includes:</span></p>\r\n<ul>\r\n<li><span>reaction name</span></li>\r\n<li><span>a stable identifier</span></li>\r\n<li><span>pathway summation</span></li>\r\n<li><span>links to external identifiers, where appropriate. For instance, a reaction that describes a binding event may link out to IntAct records that corroborate the interaction. </span></li>\r\n<li><span>Input/Output: Identifies the input/output molecules, sets or complexes for this reaction. Icons to the right of these named items link to further information in Reactome.</span></li>\r\n<li><span>Catalyst (when relevant): The protein or complex that catalyzes the reaction. The Gene Ontology molecular function term that represents the activity of a catalyst or transporter within the reaction will be listed. If the catalyst is a complex, the component that enables the reaction to occur will be identified as the ‘active unit’.</span></li>\r\n<li><span>Preceding event(s): A list of events that occur immediately before the event being viewed.</span></li>\r\n<li><span>Following event(s): A list of events that occur immediately after the event being viewed.</span></li>\r\n<li><span>Cellular compartment:</span></li>\r\n<li><span>Inferred from another species (when relevant): This indicates that event has not been experimentally demonstrated in humans, but has been inferred on the basis of data acquired for another species. </span>\r\n<ul>\r\n<li><span>Clicking on the ‘+’ button to the right of the reaction name in this panel reveals the summation and literature references for the supporting reaction in the other species; </span></li>\r\n<li><span>clicking on the reaction icon to the left of the reaction name in this panel switches the user to the corresponding pathway of the species from which the human reaction has been inferred. Note that the manually curated event in the non-human species is not currently displayed in this pathway; instead, a computationally-inferred reaction is shown. This will be addressed in future updates. The human representation of the pathway can be restored either by clicking on the back button of the browser or by re-selecting ‘Homo sapiens’ in the species selector in the top panel of the Reactome website.</span></li>\r\n</ul>\r\n</li>\r\n<li><span>Computationally inferred to: Links to descriptions of the events in other species that are either confirmed to occur in a very similar way in both species, or have been electronically inferred. </span></li>\r\n<li><span>Positively (or Negatively) regulated by: The protein or complex that regulates the reaction.</span></li>\r\n<li><span>References that contain experimental data verifying the reaction, with a link-out to PubMed, when applicable.</span></li>\r\n<li><span>Authors: The expert biologists that contributed materials that allowed this reaction to be created in Reactome.</span></li>\r\n</ul>\r\n<ul>\r\n<li><span>Reviewers: The expert biologists that verified the content for this pathway.</span></li>\r\n</ul>\r\n<p><b>Physical Entity Description tab:</b></p>\r\n<p><span>The details of any protein, small molecule, complex or set represented in a pathway diagram can be displayed in the Details panel by selecting the object within the diagram. Clicking a ‘+’ or ‘-‘ icon associated with the different displayed fields within the Detail panel tabs will show or hide more information about the protein, small molecule, complex and set.</span></p>\r\n<p><span>The Description tab may display include the following information depending upon what  physical entity is selected:</span></p>\r\n<ul>\r\n<li><span>Link to corresponding entries in other databases: Cross-reference to identifiers used for this molecule in external reference databases, with hyperlinks to the record.</span></li>\r\n<li><span>Cellular compartment: The cellular compartment that contains this molecule, with hyperlink to the corresponding GO term.</span></li>\r\n<li><span>Computationally inferred orthologues: Lists equivalent molecules in other species if these have been inferred to exist. A description of the inference process can be found here.</span></li>\r\n<li><span>Components: If a complex is selected in the diagram, its components are listed here.</span></li>\r\n<li><span>Produced by: If a complex is selected in the diagram, this tab in the details panel will list all the reactions in Reactome that produce it as an output. Clicking on the ‘+’ reveals summation and literature references for these reactions.</span></li>\r\n<li><span>Consumed by events: If a complex is selected in the diagram, this tab in the details panel will list all reactions in Reactome that use it as an input. Clicking on the ‘+’ reveals summation and literature references for these reactions.</span></li>\r\n</ul>\r\n<p></p>\r\n<p><b>Download tab:</b></p>\r\n<p><span>The Download tab provides options for the user to download different files types for the selected pathways. These files include:</span></p>\r\n<ul>\r\n<li><span>SBML: An exchange format used by systems biologists for their models.</span></li>\r\n<li><span>SBGN: An exchange format used to represent pathway and network diagrams.</span></li>\r\n<li><span>BioPAX2: An exchange format used by systems biologists for their models.</span></li>\r\n<li><span>BioPAX3: An exchange format used by systems biologists for their models.</span></li>\r\n<li><span>PDF: Text dump of the pathway, organized to look like a research report.</span></li>\r\n<li><span>Word: A document format compatible with Microsoft and other word processor software.</span></li>\r\n<li><span>Protege: A format used for ontology exchange.</span></li>\r\n</ul>\r\n<p><a href=\"http://sbml.org/Main_Page\"><span>SBML</span></a><span>,</span><a href=\"http://www.sbgn.org/Main_Page\"> <span>SBGN</span></a><span> and</span><a href=\"http://www.biopax.org\"> <span>BioPAX</span></a><span> are exchange formats of interest to bioinformaticians.</span><a href=\"http://en.wikipedia.org/wiki/Microsoft_Word\"> <span>Word</span></a><span> and</span><a href=\"http://en.wikipedia.org/wiki/Portable_Document_Format\"> <span>PDF</span></a><span> are familiar document formats, providing you with a convenient \"document\" of the pathway.</span><a href=\"http://protege.stanford.edu\"> <span>Protégé</span></a><span> is an extensible, platform-independent environment for creating and editing ontologies and knowledge bases, this download is likely to be of interest to those wishing to extend Reactome functionality.</span></p>\r\n<p><span>The complete Reactome textbook of biological processes in PDF or RTF format, the complete set of human reactions in Reactome (in SBML or BioPAX level 2 or 3 format), and a list of human protein-protein interaction pairs are available to download from the Download page, linked to the Menu Bar on the Reactome homepage.</span></p>\r\n<p><span>To find out about how Reactome generates SBML, see</span><a href=\"http://wiki.reactome.org/index.php/SBML_At_Reactome\"> <span>SBML At Reactome</span></a><span>.</span></p>\r\n<p><span>For general information about SBML,</span><a href=\"http://sbml.org/index.psp\"> <span>click here</span></a><span>. For more information about BioPAX</span><a href=\"http://www.biopax.org\"> <span>click here</span></a><span>.</span></p>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:49:23','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,10,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(83,697,'Searching Reactome','searching-reactome','<p>See our Youtube Video explaining <a href=\"https://youtu.be/3lXWcPgP8iQ\">Reactome\'s Main Search Feature</a>! </p>\r\n<h3><b>Simple text search</b></h3>\r\n<p></p>\r\n<p><span>The simple text search tool is located top right of the Home page. To search type a word, phrase or identifier in the search box. The search has an auto-complete function; if the text you wanted to use appears in the drop-down list, select it and results will be displayed. For other text click Search.</span></p>\r\n<p></p>\r\n<p><img src=\"images/userguide/search/search_box.gif\" alt=\"search box\" /></p>\r\n<p></p>\r\n<p><span>Search results are presented in categories with grey headers, representing the molecular entity or event types.</span></p>\r\n<p><span>Below the header is a list of items from the named type that match your search terms. Matching words are highlighted in grey. Click on the name of an item to go to a page of details.</span></p>\r\n<p><span>On the left side of the search results page are groups of checkboxes. These can be used to change defaults and add filters. Check or uncheck boxes for types that you do/don’t want to see, e.g. if you want to see results for Mus musculus, under Species uncheck Homo sapiens and check Mus musculus. In the Type, Compartment and Reaction Type filter groups, select to only see results that fall into the selected category, e.g. to see only reactions, select Reaction in the Types filter group. To see only Reactions that occur in the cytosol, also select Cytosol in Compartments.</span></p>\r\n<p></p>\r\n<p><img src=\"images/userguide/search/protein_kinase_results.gif\" alt=\"protein kinase results\" /></p>\r\n<p><span>Click on any of the results to go to a page of information about it. The contents of these Results Detail pages will depend on the type, but all have a section named ‘Locations in the PathwayBrowser’. </span></p>\r\n<p><img src=\"images/userguide/search/calmodulin_details.gif\" alt=\"calmodulin details\" /></p>\r\n<p><span>This contains an expandable hierarchy that identifies the locations of the item in Reactome pathway diagrams. Click the plus symbol to open the hierarchy.</span></p>\r\n<p>This hierarchy reflects the organization of events in Reactome. ‘Top-level’ pathways representing a broad area of biology typically contain one or more levels of subpathways, becoming more specific with each level. The levels of the hierarchy are represented here by indentation; the most general pathways are closest to the left-hand edge of the screen, the most specific subpathways have the greatest indentation to the right. Note that some search result items (e.g. proteins) can be in more than one pathway, which may be in different broad areas of biology, resulting in representation in multiple hierarchies. Note that you can open all the hierarchy levels with a single click on the Expand all link, on the right side. Select any event name to open the corresponding Pathway Diagram. The diagram will open with an animation that shows you where the pathway is located in the Pathway Overview. If the selected item was an event it will be highlighted in blue in the Pathway Diagram. If it was a protein, all instances of it will be highlighted in pink.</p>\r\n<p><strong>Icon Search</strong></p>\r\n<p>Elements of the Icon Library are accessible to the main website search. In the screenshot below, the results of a search for the term: \"liver\" are displayed. </p>\r\n<p><img src=\"images/userguide/search/Icon_Search_Liver.png\" alt=\"Icon Search Liver\" width=\"585\" height=\"400\" /></p>\r\n<p>Below the header is a list of items from the \"icon\" type that match your the \"liver\" search terms. Matching words are highlighted in grey. Click on the name of an item to go to a page of details. In the following image, selecting the first search result is shown</p>\r\n<p><img src=\"images/userguide/search/Icon_Liver_Results.png\" alt=\"Icon Liver Results\" width=\"566\" height=\"400\" /></p>\r\n<p>The metadata associated with the selected icon is displayed, including the icon category, the curator and designer, and a brief description of the icon. To download the image icon files, click the links to the fight of the preview image. Further down the page is a series of expandable hierarchy links that identifies the locations of the icon in Reactome pathway diagrams. Click the plus symbol to open the hierarchy. Clicking the external link, towards the bottom of the page, will link out to the referencing database for the named icon.</p>\r\n<p></p>\r\n<p></p>\r\n<p><b>In-Diagram Search (Searching in the Pathway Browser)</b></p>\r\n<p></p>\r\n<p><span>Searches can be performed within the Pathway Browser using the In-Diagram Search panel (top left corner of the Pathway Panel). </span><span>When the Pathway Overview,  Diagram is Enhanced HIgh Level Diagram (EHLD) is displayed, all of Reactome is searched. </span></p>\r\n<p><img src=\"images/userguide/search/In-Diagram-Start.png\" alt=\"In Diagram Start\" width=\"550\" height=\"113\" /></p>\r\n<p></p>\r\n<p><b>Searching the Pathway Overview</b></p>\r\n<p></p>\r\n<p><span>In the example below, as the search term, ‘IL6’ has been entered. The search tool has predicted results that match this search term and displays a drop-down list of these matches. </span></p>\r\n<p><img src=\"images/userguide/search/Overview_Search.png\" alt=\"Overview Search\" width=\"550\" height=\"400\" /></p>\r\n<p><b>Note: </b><span>the deep blue bar, which indicates that there are 23 results shown. Only the three results are clearly shown in the results window. To view additional results, move the cursor over the window and scroll down. Each result term has a symbol (to the left) representing the type of molecule or event it is. The first result, with a blue circle icon to the left of the name, represent matches to the IL6R protein, located in the plasma membrane. The second result, with a double-helix icon to the left, is the IL6R gene. The third result is a reaction, with <span>icon showing above/below boxes on left connected by a right-pointing arrow to box on right, representing the binding of IL6 to IL6R</span>. The Reactome stable identifier and cell compartment annotations are also provided for each molecule or event type that is displayed.</span></p>\r\n<p><img src=\"images/userguide/search/Overview_Search_Results_Summary.png\" alt=\"Overview Search Results Summary\" width=\"554\" height=\"400\" /></p>\r\n<p><span>In the following image, the recent search results are displayed at the bottom of the search results window. Clicking the \"Clear History\" button will remove the search terms from the history.</span></p>\r\n<p><img src=\"images/userguide/search/Overview_Search_History.png\" alt=\"Overview Search History\" width=\"553\" height=\"400\" /></p>\r\n<p>In the next image, selecting an item in the Search Results Panel also causes a second drop-down Search Results Details Panel to appear. The additional annotations displayed include the molecule or event type, the Reactome stable identifier, an external reference database identifier, and the cell compartment. Clicking the \"Filter\" button (to the right of the search bar) will display options to filter the search results by molecule or event type, e.g. Pathway, Reaction, Complex, Protein, DNA Sequence, Set, etc. Selecting one or more of the filters will restrict the Search results viewed. Deselecting the filter(s) will alter the search results display. Selecting an item in the initial Search Results Panel causes the Overview to zoom and recentre, focusing on the region that represents pathways containing the selected item. This identifies the selected item and lists the pathways that contain the selected item. </p>\r\n<p><img src=\"images/userguide/search/Overview_Search_Results_Detailed.png\" alt=\"Overview Search Results Detailed\" width=\"550\" height=\"350\" /></p>\r\n<p>In the following image, moving the cursor over a pathway name (under the \"Present in X pathway\") will highlight the corresponding in the Overview, if it is visible in the viewport. The tooltip will display the name of the pathway.</p>\r\n<p><img src=\"images/userguide/search/Overview_Search_Results_Selected.png\" alt=\"Overview Search Results Selected\" width=\"550\" height=\"365\" /></p>\r\n<p>Clicking a pathway name (under the \"Present in X pathway\") in the Search Results Detail Panel will select the corresponding object in the Overview. If it is not visible, the Pathway Diagram will re-centre to show it. Double-click a pathway name in the Search Results Detail Panel or pressing the \"GO &gt;&gt;&gt;\" button will open the corresponding Pathway Diagram. If you need to return to the Overview, you can use the browser back button. <b>Note</b> that in the image above, the Search Results Detail Panel extends beyond the limits of the Overview window, consequently some results are hidden. This explains why the results details title says Present in 3 pathways but only 2 are visible. To see all the matching pathways, click in the pathway list in the Search Results Detail Panel. </p>\r\n<p>In the following image, the \"Flag\" button in the Search Results Details Panel (to the right of IL6R) has been clicked and the events that contain the selected molecule are highlighted blue and the connected events are highlighted purple. Clicking the \"Flag\" button for a second time will remove the event highlighting.</p>\r\n<p><img src=\"images/userguide/search/Overview_Search_Results_Highlighting.png\" alt=\"Overview Search Results Highlighting\" width=\"550\" height=\"361\" /></p>\r\n<p></p>\r\n<p><b>Searching a Pathway Diagram</b></p>\r\n<p></p>\r\n<p><span>In the example below, the search has identified IL6R and produced a dropdown list of matches. The first match is to a protein, as indicated by the icon on the left. The second match with the icon showing a double helix is a DNA sequence or gene. Note that if IL6R was represented in two cellular compartments in this diagram it would appear twice in this initial search result. <span>The Reactome stable identifier and cell compartment annotations are also provided for each molecule or event type in the search results. <strong>Note:</strong><span> as the search term is entered, the search tool will predict results that match this search term and displays a drop-down list of these matches. </span></span></span></p>\r\n<p><span><span><span>Although not shown below, when a new search is performed, the recent search results are displayed at the bottom of the search results window. Clicking the \"Clear History\" button will remove the search terms from the history.<br /></span></span></span></p>\r\n<p><span><img src=\"images/userguide/search/In-Diagram_Search.png\" alt=\"In Diagram Search\" width=\"550\" height=\"401\" /></span></p>\r\n<p>The deep blue highlighted bar, which indicates that there are 3 results shown in \"This diagram\". Only the two results are clearly shown in the results window. To view additional results, move the cursor over the window and scroll down. Each result term has a symbol (to the left) representing the type of molecule or event it is. The first result, with a blue circle icon to the left of the name, represent matches to the IL6R protein, located in the plasma membrane. The second result, with a double-helix icon to the left, is the IL6R gene. The Reactome stable identifier and cell compartment annotations are also provided for each molecule or event type that is displayed.</p>\r\n<p>In the next image, clicking the \"Filter\" button (to the right of the search bar) will display options to filter the search results by molecule or event type, e.g. Pathway, Reaction, Complex, Protein, DNA Sequence, Set, etc. Selecting one or more of the filters will restrict the Search results viewed. Deselecting the filter(s) will alter the search results display.</p>\r\n<p><img src=\"images/userguide/search/In-Diagram_Search_Filter.png\" alt=\"In Diagram Search Filter\" width=\"550\" height=\"362\" /></p>\r\n<p><span>In the next image, selecting an item in the Search Results Panel also causes a second drop-down Search Results Details Panel to appear. The at the top of the Details Panel, additional annotations are displayed that include the molecule or event type, the Reactome stable identifier, an external reference database identifier (if applicable), and the cell compartment. Additional annotations are listed below describing the involvement of the protein within complexes, sets and reactions within the diagram. <span>Selecting names in the Search Results Detail Panel will select the corresponding object in the Pathway Diagram. If it is not visible, the Pathway Diagram will re-centre to show it.</span><br /></span></p>\r\n<p><span>In this example query, clicking the \"Flag\" button in the Search Results Details Panel (to the right of IL6R) will highlight all (15) the objects in the diagram that contains IL6R. Clicking the \"Flag\" button for<span> a second time </span>(or the \"X\" button toward the bottom of the pathway viewport) will remove the object highlighting. <span><span>To see all the matches, it is necessary to resize the Pathway Diagram panel.</span></span><br /></span></p>\r\n<p><img src=\"images/userguide/search/In-Diagram_Search-Entity_This_Diagram.png\" alt=\"In Diagram Search Entity This Diagram\" width=\"550\" height=\"349\" /></p>\r\n<p><span><span>In the following image, selecting the second item in the Search Results Panel (IL6 binds IL6R) causes a second drop-down Search Results Details Panel to appear. The at the top of the Details Panel, additional annotations are displayed that include the event type, the Reactome stable identifier, and the cell compartment. Additonal annotations are listed below describing the involvement of the reaction within the diagram, and participants of the reaction. <span>Selecting names in the Search Results Detail Panel will select the corresponding object in the Pathway Diagram. If it is not visible, the Pathway Diagram will re-centre to show it. in this image, the IL6 binds IL6R reaction is blue highlighted.</span></span></span><br /><span></span></p>\r\n<p><img src=\"images/userguide/search/In-Diagram_Search-Event_This_Diagram.png\" alt=\"In Diagram Search Event This Diagram\" width=\"550\" height=\"348\" /></p>\r\n<p><span>In the next image, clicking the \"All diagram\" tab will present a different set of results when selecting an item in the Search Results Panel. As before, at the top of the Details Panel, additional annotations are displayed that include the molecule or event type, the Reactome stable identifier, an external reference database identifier (if applicable), and the cell compartment. Instead of displaying objects within the currently displayed pathway, the pathways not shown in the pathway viewport that contain the search term are displayed. <br /></span></p>\r\n<p><img src=\"images/userguide/search/In-Diagram_Search-Gene_All_Diagrams.png\" alt=\"In Diagram Search Gene All Diagrams\" width=\"550\" height=\"351\" /></p>\r\n<p><span>Selecting the hyperlinked pathway name in the Search Results Detail Panel (\"Present in n pathway diagram\") will open the new pathway diagram in the Pathway Browser and select the corresponding object in the new diagram. </span><span></span></p>\r\n<p><img src=\"images/userguide/search/In-Diagram_Search-Gene_Set_All_Diagrams.png\" alt=\"In Diagram Search Gene Set All Diagrams\" width=\"550\" height=\"349\" /></p>\r\n<p></p>\r\n<p><b>Searching within an Enhanced HIgh Level Diagram (EHLD)<br /></b></p>\r\n<p></p>\r\n<p>In the example below, the search has identified IL6R and produced a dropdown list of matches. The first match is to a protein, as indicated by the icon on the left. The second match with the icon showing a double helix is a DNA sequence or gene. Note that if IL6R was represented in two cellular compartments in this diagram it would appear twice in this initial search result. The Reactome stable identifier and cell compartment annotations are also provided for each molecule or event type in the search results. <strong>Note:</strong> as the search term is entered, the search tool will predict results that match this search term and displays a drop-down list of these matches.</p>\r\n<p>Although not shown below, when a new search is performed, the recent search results are displayed at the bottom of the search results window. Clicking the \"Clear History\" button will remove the search terms from the history.</p>\r\n<p><img src=\"images/userguide/search/EHLD_Search.png\" alt=\"EHLD Search\" width=\"550\" height=\"393\" /></p>\r\n<p>The deep blue highlighted bar, which indicates that there are 23 results shown in \"This diagram\" and 30 results shown in \"All diagrams\". Only the two results are clearly shown in the results window. To view additional results, move the cursor over the window and scroll down. Each result term has a symbol (to the left) representing the type of molecule or event it is. The first result, with a blue circle icon to the left of the name, represent matches to the IL6R protein, located in the plasma membrane. The second result, with a double-helix icon to the left, is the IL6R gene. The Reactome stable identifier and cell compartment annotations are also provided for each molecule or event type that is displayed.</p>\r\n<p><span>In the following image, selecting the first item in the Search Results Panel (IL6R) causes a second drop-down Search Results Details Panel to appear. The at the top of the Details Panel, additional annotations are displayed that include the molecule or event type, the Reactome stable identifier, and the cell compartment. Additional annotations are listed below describing the involvement of the object within the EHLD. <span>Selecting pathway names in the Search Results Detail Panel will select the corresponding object in the EHLD. If it is not visible, the EHLD will re-centre to show it in the </span></span><span></span></p>\r\n<p><img src=\"images/userguide/search/EHLD_Search_Results_Detailed.png\" alt=\"EHLD Search Results Detailed\" width=\"550\" height=\"392\" /></p>\r\n<p></p>\r\n<p></p>\r\n<p><span>Getting Started</span></p>\r\n<p></p>\r\n<h3><b>Search Exercises</b></h3>\r\n<p></p>\r\n<p><span>This exercise encourages you to find information via the menu bar and Search </span></p>\r\n<p></p>\r\n<ol>\r\n<li><span>What’s the latest news item on the Reactome Home page?</span></li>\r\n<li><span>How many human proteins are represented in Reactome?</span></li>\r\n<li><span>What’s the first item listed that will be included in the next release?</span></li>\r\n<li><span>Is VAV2 in Reactome?</span></li>\r\n<li><span>How many reactions involve VAV2?</span></li>\r\n<li><span>Are there any complexes that include VAV2?</span></li>\r\n<li><span>Is CRB2 in Reactome?</span></li>\r\n</ol>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:50:12','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,9,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(84,698,'Analysis Tools','analysis-tools','<h4>Jump to section:</h4>\r\n<ul>\r\n<li><a href=\"#analysis\">Analysis Data</a></li>\r\n<li><a href=\"index.php?option=com_content&amp;view=article&amp;id=156&amp;Itemid=560\">Analysis Gene Expression</a></li>\r\n<li><a href=\"#species\">Species Comparison</a></li>\r\n<li><a href=\"#tissue\">Tissue Distribution</a></li>\r\n</ul>\r\n<p>See our Youtube Video that introduces our <a href=\"https://youtu.be/rxEpA-lJguE\">Analysis Tools</a>! </p>\r\n<h4><a id=\"analysis\"></a><strong>Analysis Data</strong></h4>\r\n<p>Click on the ‘Browse Pathways’ button on the Homepage. In the next page, click the ‘Analysis’ button on the top right:</p>\r\n<p><span><img src=\"/images/userguide/analysis/analysis_1.png\" alt=\"\" width=\"222\" height=\"81\" /></span></p>\r\n<p><span>Alternatively, select the ‘Analyze Data’ button on the Homepage:</span></p>\r\n<p><img src=\"images/userguide/analysis/analysis_2.png\" alt=\"analyze data button\" width=\"188\" height=\"206\" /></p>\r\n<p><span>This opens a submission form, where you can select the analysis you want to perform, paste in or browse to a file containing your data, or use an example data set.</span></p>\r\n<p><em><img src=\"images/userguide/analysis/analysis_3.png\" alt=\"analysis 3\" width=\"626\" height=\"400\" /></em></p>\r\n<p></p>\r\n<p></p>\r\n<p></p>\r\n<p><span>There are two sections to the submission form. The ‘Analyse your data’ section is selected by default to submit your data. Several different analyses can be performed, depending on the format of your data.</span></p>\r\n<p><span>If your data is a single column of identifiers such as UniProt IDs, gene symbols or ChEBI IDs, they are mapped to pathways and over-representation and pathway-topology analyses are run. Over-representation analysis is a statistical (hypergeometric distribution) test that determines whether certain Reactome pathways are over-represented (enriched) in the submitted data. It answers the question ‘Does my list contain more proteins for pathway X than would be expected by chance?’ This test produces a probability score, which is corrected for false discovery rate using the Benjamani-Hochberg method.</span></p>\r\n<p><span>Pathway topology analysis considers the connectivity between molecules that is represented by the pathway steps (which we refer to as reactions) in the pathway. It groups all the molecules represented in each reaction as a pathway ‘unit’. If any of these molecules are represented in your query set, this is considered a match to that reaction. This may give a better indication of the proportion of the pathway that matches your data, rather than the number of molecules that are common between your data and the pathway. It may also indicate that your data matches the start, end or a particular branch of a pathway process. This test does not have a probability score.</span></p>\r\n<p><span>If your data has one or more additional columns of numbers it will be recognized as expression data and expression data overlay will be performed. Note that this data format should include a header row. The first column header should start with the # symbol. The numbers are used to produce a scaled coloured overlay over Reactome pathway diagrams, as a means to visualize relative expression levels. Note that the numeric values do not have to be expression data, for instance by using gene association scores the same analysis can be used to visualize genotyping results.   </span></p>\r\n<p></p>\r\n<h4><b>Identifier mapping </b></h4>\r\n<p><span>The submission process recognizes many types of identifiers. As part of the pre-analysis, they are mapped to Reactome molecules. The ideal identifiers to use are UniProt IDs for proteins, ChEBI IDs for small molecules, and either HGNC gene symbols or ENSEMBL IDs for DNA/RNA molecules, as these are our main external reference sources for proteins and small molecules. Many other identifiers are recognized and mapped to appropriate Reactome molecules. Accepted identifiers include HUGO gene symbols, GenBank/EMBL/DDBJ, RefPep, RefSeq, EntrezGene, MIM, InterPro, EnsEMBL protein, EnsEMBL gene, EnsEMBL transcript, and some Affymetrix and Agilent probe IDs. UniProt isoforms may be specified using the format P12345-2. If the -n suffix is omitted, this canonical form and all isoforms of it will be matched. Mixed identifier lists (different protein identifiers or protein/gene identifiers) may be used. Identifiers must be one per line. Protein-specific identifiers will typically map to protein entities, while gene-specific identifiers will map to the gene, transcript and derived proteins. If desired results can be filtered to show protein-specific or gene/transcript-specific results, details below.    </span></p>\r\n<p><span>Below is an example of the identifiers-only format:</span></p>\r\n<p><span>Click the Continue button. A second options selection page appears:</span></p>\r\n<p><em><img src=\"images/userguide/analysis/analysis_4.png\" alt=\"analysis 4\" width=\"626\" height=\"400\" /></em></p>\r\n<p><span>Project to human is checked by default. With this option selected, all non-human identifiers in your query are converted by the analysis service to their human equivalents. In general, this maximizes the chances of a successful match to Reactome’s curated human pathways. However, if you want to use non-human identifiers and match these to our computationally-inferred non-human pathways, uncheck the box. You may also choose to uncheck this box if your query consists of a mixture of human and microbial identifiers and your goal is to find pathways that represent the processes of infection.</span></p>\r\n<p><span>‘Include Interactors’ is unchecked by default. With this box unchecked, your query will consider only Reactome pathways. If you choose to check the box, your query will consider Reactome pathways that have been expanded by including all available protein-protein interactors from the IntAct database. This greatly increases the size of Reactome pathways, which maximizes the chances of matching your submitted identifiers to the expanded pathway, but will include interactors that have not undergone manual curation by Reactome and may include interactors that have no biological significance, or unexplained relevance. In practice it is preferable to query with ‘Include Interactors’ unchecked in the first instance, followed by a repeated query with ‘Include interactors’ selected, if a substantial proportion of the submitted identifiers do not match a Reactome pathway, to see if they can be identified as interactors.</span></p>\r\n<p></p>\r\n<h4><b>Results for Identifier lists without associated numeric values</b></h4>\r\n<p></p>\r\n<p><span>If you submit a single column of protein or small molecule identifiers they are mapped to pathways and over-representation and pathway-topology analyses are performed. The results will resemble the example below.</span></p>\r\n<p><span><img src=\"images/userguide/analysis/analysis_6.png\" alt=\"analysis 6\" width=\"965\" height=\"591\" /></span></p>\r\n<p><span>Analysis results are shown in the Analysis tab, within the Details Panel. All Reactome pathways are shown, in blocks of 20 pathways, ranked by the p-value obtained from over-representation analysis. If multiple pathways have the same p-value, they are ranked by the number of identifiers in the query that match the pathway. The number of molecules matched/total number of molecules and FDR values are added to the right side of pathway names in the Hierarchy Panel. The names of reactions that match at least one identifier in the query, representing positive pathway topology analysis hits, are boxed in orange. </span></p>\r\n<p><span>In the Analysis tab, clicking on the name of a pathway will select it in the Hierarchy, which if necessary will expand hidden hierarchical levels to show the pathway, while the name becomes highlighted in dark blue. </span></p>\r\n<p><span>By default, molecules of all types (protein, small molecules, genes, transcripts) are used for over-representation analysis, but it is possible to restrict the analysis results to a specific subtype by using a drop-down list located top-left of the results table. Selecting one of the subsets will display results that consider only the selected molecular subtype.</span></p>\r\n<p><span><img src=\"/images/userguide/analysis/analysis_7.png\" alt=\"\" width=\"529\" height=\"36\" /></span></p>\r\n<p><span>The columns in analysis details represent:</span></p>\r\n<ol>\r\n<li><span>Pathway name: Click the name to open the pathway. </span></li>\r\n<li><span>Entities found: the number of curated molecules of the type selected with Results Type that are common between the submitted data set and the pathway named in column 1. Click on this number to display the matched submitted identifiers and their mapping to Reactome molecules.</span></li>\r\n<li><span>Entities total: The total number of curated molecules of the type selected with Results Type within the pathway named in column 1.</span></li>\r\n<li><span>Interactors found (if this option was selected). The number of interactor molecules of the type selected with Results Type that are common between the submitted data set and the pathway named in column 1. Click on this number to display the matched submitted identifiers and their mapping to Reactome molecules.</span></li>\r\n<li><span>Interactors total (if this option was selected): The total number of interactor molecules of the type selected with Results Type within the pathway named in column 1.</span></li>\r\n<li><span>Entities ratio: Put simply, the proportion of Reactome pathway molecules represented by this pathway. Calculated as the ratio of entities from this pathway that are molecules of the type selected with Results Type Vs. all entities of the type selected with Results Type. </span></li>\r\n<li><span>Entities pvalue: The result of the statistical test for over-representation, for molecules of the results type selected.  </span></li>\r\n<li><span>Entities FDR: False discovery rate. Corrected over-representation probability.</span></li>\r\n<li><span>Reactions found: The number of reactions in the pathway that are represented by at least one molecule in the submitted data set, for the molecule type selected with Results Type.</span></li>\r\n<li><span>Reactions Total: The number of reactions in the pathway that contain molecules of the type selected with Results Type.</span></li>\r\n<li><span>Reactions ratio: Put simply, the proportion of Reactome reactions represented by this pathway. Calculated as the ratio of reactions from this pathway that contain molecules of the type selected with Results Type Vs. all Reactome reactions that contain molecules of the type selected with Results Type.</span></li>\r\n<li><span>Species Name.</span></li>\r\n</ol>\r\n<p><span>When an analysis has run, the Pathway Browser will display the Pathway Overview. All pathways that contain identifiers from your submitted list are highlighted, using a coloured scale to indicate the corrected probability (FDR). The colour scheme can be changed using the colour profiles tab (artists easel icon) on the pop-out Settings panel, found on the right-hand edge of the pathway panel.  Selecting coverage in the Overexpression panel will alter the Pathway Overview display to show additional event crosslinks corresponding to areas that are heavily covered vs regions that have lower coverage (as shown in the pValue view), and making it easier to visualize the pathways enriched within your dataset.</span></p>\r\n<p><span>The highlighting of pathways in the Overview provides an at-a-glance representation of analysis results for all pathways. To see the details of a specific pathway, double-click the node representing the pathway in the overview or in the Pathway Hierarchy on the left. Alternatively, click it once to select it and use the Show All button (square with outward pointing triangles inside) in the top left corner of the Overview panel. The Overview can be navigated using the mouse scroll wheel to zoom in and out and click and drag to move it around. Alternatively, use the navigation buttons in the bottom right corner of the overview panel. At any level of the pathway, the diagram key can be found by clicking the compass symbol in the top right corner.</span></p>\r\n<p><span><img src=\"images/userguide/analysis/analysis_8.png\" alt=\"analysis 8\" width=\"962\" height=\"591\" /></span></p>\r\n<p></p>\r\n<p><span>Enhanced high-level diagrams represent analysis results within the label for subpathways. The label background changes from blue to white, a yellow band is used to indicate the proportion of the pathway that is represented in the query dataset. A grey bar above the label indicates the number of pathway entities that are represented in the query dataset, the total number of entities in the pathway, and the FDR corrected probability score</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/GlDAvsPmWt5whkyOHX6QCY-aNHyPO6c1HdDzaRGopnB3Jp3kM7R978cQJOMx52Nw4UFO3Nb7d-NLsFXD9hFUxx926SC0bnPDkretnUJ384h9gh1lWojOctJ7QN36DzuIiC-5z95V0gHgDjFCaiEMNQ\" alt=\"\" width=\"602\" height=\"407\" /></p>\r\n<p><span>In Pathway Diagrams, entities are re-coloured (yellow in the default colour scheme) if they were represented in the submitted data set. Complexes, Sets and Subpathway Icons are coloured to represent the proportion that is represented in the submitted identifier list. In the figure below, Insulin receptor is yellow indicating that is was in the submitted list. Insulin was not in the submitted list so it is not re-coloured. The complex of insulin:Insulin receptor is part re-coloured, part not, indicating that some molecules in the complex were represented in the submitted dataset while others were not. </span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/u2NGQ9IAuHxRfBRuDUU_dl15xGEumbTtfPSmRS8NpiSFXd4RhhiaPjlpvk5Vs4Sq5gjDkAwCcbYQDWvs0x3V7PsST5NaYnAMhfoXOYIAdalzxrnvGlQ0cWIIAPrljG9pyHpBp85wigSNqGu7TvXAjQ\" alt=\"\" width=\"367\" height=\"188\" /></p>\r\n<p><span>If the Include Interactors option was checked, entities with interactors that were part of the submitted identifier list have a ribbon across the top right corner. In the example below, RASA1 is not a direct match with the submitted list of identifiers but has interactors that match the list. The interactors are displayed; the yellow overlay indicates the matching interactors. PTPN11 is a direct match and has interactors that match the list. A limited number of interactors are displayed to avoid crowding.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/_ajYcYFnr08f37kKzWlV4wD8Shw_GuB3u5wsPILxr5zWT667o1wGUzLjR15ZxbPM2RZ5_WqhZE1sDIsoNzCXLRso28_Zb-ExWmu3P3qRlZ0CPqjTVCwrXXqDEud0CoAIHpDBQVNkroI5IyZ7tXv5OQ\" alt=\"\" width=\"529\" height=\"311\" /></p>\r\n<p><span>At the right side of the Analysis results details is a button indicating the number of submitted identifiers that were not successfully matched to molecules in Reactome. Click the button to produce a list.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/mDsHg7HIuPXY1nUcQ-iEVdN9mMxnHtM_lPLJvYr8DF2AudA0NwPf-pM6uCc_1PnKMPNKojcPHbQz_8CcL_H8S8W_5g6MPs9Q2pMlnTAkHDftc32R5v6kes3CPjIWWxVVHUCT-VW1otfLF6otJy_PGA\" alt=\"\" width=\"529\" height=\"134\" /></p>\r\n<p></p>\r\n<p><span></span></p>\r\n<p></p>\r\n<h4><b>Results for Identifier lists with associated numeric values (expression representation)</b></h4>\r\n<p></p>\r\n<p><span>To run expression analysis, submit your data in a format that includes a first row of column headers. The header for column 1 must start with the # symbol. The first column must contain protein, compound or other suitable identifiers, such as probe IDs. All other columns must be numeric values, with no alphabetical characters. The analysis tool will interpret your data as expression data. The numeric values are used to colour objects in pathway diagrams. This view was created for microarray data, but any dataset that consists of a list of identifiers with associated numeric values can be used, e.g. quantitative proteomics, GWAS scores.</span></p>\r\n<p></p>\r\n<p><span>The tool is launched using the Analyse data button in the Pathway Browser header bar. Either paste your data into the submission form or browse to a saved file (or select an example file). </span></p>\r\n<p><span>The figure below shows the correct data format. Each row must have an identifier in the first column (a header row is optional). </span></p>\r\n<p><span>The submission process recognized many types of identifiers. As part of the pre-analysis, they are mapped to equivalent UniProt accessions or for small compounds to ChEBI IDs. These are the ideal identifiers to use with Reactome analysis tools. Other identifiers that are recognized and converted to UniProt equivalents include HUGO gene symbols, GenBank/EMBL/DDBJ, RefPep, RefSeq, EntrezGene, MIM and InterPro IDs, some Affymetrix and Agilent probe IDs, Ensembl protein, transcript and gene identifiers. Identifiers that contain only numbers such as those from OMIM and EntrezGene must be prefixed by the source database name and a colon e.g. MIM:602544, EntrezGene:55718. Mixed identifier lists (different protein identifiers or protein/gene identifiers) may be used. Identifiers must be one per line.</span></p>\r\n<p><span>By default, all non-human identifiers are mapped to their human equivalents, unless the Project to the human checkbox is unselected. </span></p>\r\n<p></p>\r\n<p><span>After column 1, all other columns must contain numbers, representing expression or other values. Comma-separated values(CSV) and tab separated value (TSV) files can be used. When submitted, columns of numbers are considered as separate samples or experimental conditions. The values are used to overlay colour onto pathway diagrams. An Experiment Browser tool allows you to select and view-overlays for each submitted data column. This tool is the panel that appears in the bottom-centre of both the pathways overview and the diagram viewer when expression data columns have been submitted. In that panel, the user can move through the different time series (and also play it as a \"movie\"). This is particularly useful for visualizing time-points or a disease progression.</span></p>\r\n<p><span><img src=\"images/userguide/analysis/analysis_14.png\" alt=\"analysis 14\" width=\"678\" height=\"400\" /></span></p>\r\n<p><span>The results may take a few seconds to appear. </span></p>\r\n<p></p>\r\n<p><span>The results page is very similar to that seen following submission of a simple one-column list of identifiers, with extra columns in the Analysis details following column 9.  These extra columns represent the submitted expression values. </span></p>\r\n<p></p>\r\n<p><span>Clicking on a pathway name launches the Pathway Browser and displays the relevant Pathway Diagram (see example below).</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/orjrg14PWCLt0fEaDvf9lItRpNgIM8UJ3mzPy_e7ChidXP5fMPH48j5cxRC-UGX84tJSlB1sxdA6uZI0-31CBaFpX0bDCWP0nx6H43rrt1RRYJmtvbvxon_o58u0wz5OhhCMWb9sADOzaSjaT8N89g\" alt=\"\" width=\"529\" height=\"164\" /></p>\r\n<p><span>Objects in the Pathway Diagram are re-coloured according to the numeric values submitted. The colours are based on a scale as represented in a bar on the right-hand side. There are several colour schemes, selected using the Settings pop-out panel on the right side. The scale (on the right side) automatically adjusts to fit the range of values in the dataset.</span></p>\r\n<p><span>Objects that were not represented in the input data are not re-coloured.</span></p>\r\n<p><span>Objects with bands of colour represent complexes or sets containing more than one molecule. When zoomed out, the colour of the band reflects the average of the values submitted, for molecules represented in the dataset. The size of the band reflects the proportion of molecules that had submitted values. Zoom in to see individual bands, for each molecule that had a submitted value, arranged alphabetically by name. The bands of colour now reflect the values submitted. If multiple columns of values were submitted, representing multiple samples, e.g. time-points or a disease progression, the order of the bands will be the same for each sample.</span></p>\r\n<p><span>To view details of the components of a complex or set right click it. This opens a Contextual Information Panel (CIP) (see figure above). This has two tabs - Molecules and Pathways. Molecules show the participating molecules, and if an expression analysis has been performed, their expression values. Pathways identify whether the selected object is present in any other Reactome pathways, with links to the appropriate Pathway Diagrams.</span></p>\r\n<p><span>    Multiple CIPs can be opened and pinned so they remain visible when other entities are selected.</span></p>\r\n<p><span>    The browser remembers CIPs that were pinned on the last visit (for up to 5 diagrams).</span></p>\r\n<p></p>\r\n<p><span>The orange Experiment Browser toolbar (bottom left in the figure above) is used to step through the columns of your data, e.g. time-points or disease progression. Move between them by clicking the arrow buttons. The header of the data column (if present) is displayed between the arrows. The Pathway Diagram will re-colour to reflect the new values.</span></p>\r\n<p></p>\r\n<p><span>For an explanation of the results seen in the Identifier results tab see the explanation in the section Results for Identifier lists without associated numeric values.</span></p>\r\n<h4><strong>Analysis Token</strong></h4>\r\n<p>Once the analysis is finished, this data structure can also be serialised to a bin file to persist the result. In addition, associating the file to a token provides an easy way to access the data.You can keep the token once the initial analysis is finished and the results are only temporarily available for 7 days unless you re-perform your analysis with the same data. However, the analysis results will disappear after each quarterly update of the Reactome data. At this time, analysis result links and tokens will no longer work anymore duo to the data update, we recommend you to save the analysis results report(PDF) for future use.</p>\r\n<h4><span><b>Analysis Report</b></span></h4>\r\n<p></p>\r\n<p>For some of our users, it might be important to preserve a pathway analysis for the long-term. Following the pathway analysis, the results can be downloaded as an easy-to-read PDF report by clicking the ‘Report (PDF)’ button located at the bottom left corner of the Details panel. The first page of the PDF report will look something like this:</p>\r\n<p><img src=\"images/userguide/analysis/analysis_16.png\" alt=\"analysis 16\" width=\"626\" height=\"400\" /></p>\r\n<p>The sections of the analysis report represent:</p>\r\n<ol>\r\n<li>Introduction: An overview of the Reactome project and the Analysis tool.</li>\r\n<li>Properties: A summary details of the analysis output. For example, the number of identifiers that were found in Reactome.  </li>\r\n<li>Genome-wide overview: A genome-wide overview of the results of your pathway analysis.</li>\r\n<li>Most significant pathways: A table of the top 25 pathway hits. Results are ranked based upon the most significant FDR value. The embedded links within the pathway name of the table will connect to the Pathway details (Section #5).</li>\r\n<li>Pathway details: A summary view of each pathway hit. The pathway identifier in parenthesis when clicked will connect with the Pathway details page on the Reactome website. A pathway diagram, pathway summation, edit history, list of identifiers found, and references (when available) are also provided.</li>\r\n<li>Identifiers found: A table of the synonyms or identifiers that were found in Reactome based upon the input list.</li>\r\n<li>Identifiers not found: A table of the synonyms or identifiers that were not found in Reactome based upon the input list.</li>\r\n</ol>\r\n<p></p>\r\n<h4><strong>Analysis Gene Expression</strong></h4>\r\n<p>ReactomeGSA is a new pathway analysis tool integrated into the Reactome ecosystem. Its main feature is that it performs quantitative pathway analyses (so-called gene set analyses). This increases the statistical power of the differential expression analysis, which is directly performed on the pathway level. More information click <a href=\"index.php?option=com_content&amp;view=article&amp;id=156&amp;Itemid=560\">here</a>.</p>\r\n<h4><b><a id=\"species\"></a>Species Comparison</b></h4>\r\n<p>The manually-curated human pathways in Reactome are used to predict equivalent pathways in 18 other species. This automated, computational process is based on orthology. A full description of the inference process can be found on the Home page under Documentation, Orthology Prediction.</p>\r\n<p><span>The Species Comparison tool allows you to compare human pathways with computationally-predicted pathways in model organisms, highlighting the elements of the pathway that are common to both species and those that may be absent in the model organism. </span></p>\r\n<p><span>Species Comparison is launched using the Analysis button in the Pathway Browser header bar. In the Species Comparison section, select one of the species in the dropdown list. Click the Go button:</span></p>\r\n<p><span><img src=\"images/userguide/analysis/analysis_10.png\" alt=\"analysis 10\" width=\"678\" height=\"400\" /></span></p>\r\n<p><span>The results are ready to view when analysis results appear (or are updated if already present) in the Pathway Hierarchy. </span></p>\r\n<p></p>\r\n<p><span>Clicking on a pathway name launches the Pathway Browser and displays the relevant Pathway Diagram (see example below).</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/XDSC8LoPLGyALKNnGZYmucqZADPRZRzP-YShLpFFEzjrRmv11pHrmO1MuKbCU_yVSfSKpR1imlw1RQW3enE7ODi6DBbEyxTiXKxJuOUfj5GEJJZIDSUvOkcwGLe4ux3PwsgIKKmzqZl___V2vIcI5w\" alt=\"\" width=\"602\" height=\"140\" /></p>\r\n<p><span>The colour of reaction objects indicates the result of the comparison:</span></p>\r\n<ul>\r\n<li><span>Yellow indicates that the protein has an inferred equivalent in the comparison species.</span></li>\r\n<li><span>No overlay indicates that inference was not possible. This is always the case for small molecules, DNA and other objects that have no UniProt entry (or did not at the time the pathway was constructed).</span></li>\r\n<li><span>Objects with bands of colour represent complexes or sets containing more than one molecule. The bands of colour reflect the inference success for the molecules within the complex/set.</span></li>\r\n</ul>\r\n<p></p>\r\n<p><span>To view species comparison results for a complex or set hover your mouse over the object and a small blue triangle will appear on its right side. Select this to open the Contextual information Panel.</span></p>\r\n<p><span>This reveals a table representing all the proteins involved in the complex/set. Each square in the grid represents one component of the complex/set, coloured as described above.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/FOrq-w7mqvFTEM9xHMaREoXw4e1ZMc-Fv6kAjk2deku4nduY3UDyHYUAXdViDJsG4N0PZpP51lzritSoUsKO0Yb-ZgFWNX3QVOVGzfqTx-TvaZpGC6F2Imw7P28p14asvKVhyeSL8nZBQALzIL2cgQ\" alt=\"\" width=\"560\" height=\"295\" /></p>\r\n<p><span>The species bar at the bottom of the Pathway Diagram (see example above) can be used to turn off species comparison colouring, by unchecking the box. </span></p>\r\n<p><span>Refer to the Navigating Pathway Diagrams section for more information on the diagram content.</span></p>\r\n<p><span></span></p>\r\n<h4><b><a id=\"tissue\"></a>Tissue Distribution </b></h4>\r\n<p>Traditionally, reactions in Reactome represent events that occur within a single generic human cell. It would, however, be useful to classify reactions into different human tissue types, as to provide an evolving picture of the reactions and pathways in different cell- and tissue-specific environments. We have imported protein expression in different cell/tissue types from the <a href=\"https://www.proteinatlas.org/\">Human Protein Atlas</a> (HPA), overlaid these proteins on Reactome data, and extracted the subset of reactions for that particular cell type. The HPA data reflecting the expression of the protein-coding genes in 44 different human tissues can be visualized through the Analysis tools.</p>\r\n<h3><img src=\"images/userguide/analysis/analysis_17.png\" alt=\"analysis 17\" width=\"678\" height=\"400\" /></h3>\r\n<p><span>Tissue Distribution is launched using the Analysis button in the Pathway Browser header bar. In the <span>Tissue Distribution</span> section, select the one experimental tissue dataset [HPA (E-PROT-3) - Expression Atlas] in the dropdown list. Once the window refreshes, select the \'Available Tissues\' in the left panel and hit the \'Add\' button to add tissue expression data to the analysis tool. Press the \"Add All\' button, if you would like to adjoin all the tissue expression data. Use the \'Remove all\' and \'Remove\' buttons to delete tissue expression data from the filter list. Once you have selected all the appropriate tissues, c</span>lick the Go button to start the analysis.</p>\r\n<p><img src=\"images/userguide/analysis/analysis_18.png\" alt=\"analysis 18\" width=\"677\" height=\"400\" /></p>\r\n<p><span><span>The results are ready to view when analysis results appear (or are updated if already present) in the Pathway Hierarchy and the Overview panel.</span></span></p>\r\n<p><img src=\"images/userguide/analysis/analysis_19.png\" alt=\"analysis 19\" width=\"710\" height=\"400\" /></p>\r\n<p>Clicking on a pathway name launches the Pathway Browser and displays the relevant Pathway Diagram (see example below).</p>\r\n<p><img src=\"images/userguide/analysis/analysis_21.png\" alt=\"analysis 21\" width=\"744\" height=\"275\" /></p>\r\n<p><span>Objects in the Pathway Diagram are re-coloured according to the numeric values submitted. The colours are based on a scale as represented in a bar on the right-hand side. There are several colour schemes, selected using the Settings pop-out panel on the right side. The scale (on the right side) automatically adjusts to fit the range of values in the dataset.</span></p>\r\n<p><span>Objects that were not represented in the input data are not re-coloured.</span></p>\r\n<p><span>Objects with bands of colour represent complexes or sets containing more than one molecule. When zoomed out, the colour of the band reflects the average of the values submitted, for molecules represented in the dataset. The size of the band reflects the proportion of molecules that had submitted values. Zoom in to see individual bands, for each molecule that had a submitted value, arranged alphabetically by name. The bands of colour now reflect the values submitted. If multiple columns of values were submitted, representing multiple samples, e.g. time-points or a disease progression, the order of the bands will be the same for each sample.</span></p>\r\n<p><span>To view details of the components of a complex or set right click it. This opens a Contextual Information Panel (CIP) (see figure above). This has two tabs - Molecules and Pathways. Molecules show the participating molecules, and if an expression analysis has been performed, their expression values. Pathways identify whether the selected object is present in any other Reactome pathways, with links to the appropriate Pathway Diagrams.</span></p>\r\n<p><span>    Multiple CIPs can be opened and pinned so they remain visible when other entities are selected.</span></p>\r\n<p><span>    The browser remembers CIPs that were pinned on the last visit (for up to 5 diagrams).</span></p>\r\n<p><span>The Experiment Browser toolbar (bottom left in the figure above) is used to step through the columns of your data, e.g. time-points or disease progression. Move between them by clicking the arrow buttons. The header of the data column (if present) is displayed between the arrows. The Pathway Diagram will re-colour to reflect the new values.</span></p>\r\n<p>Clicking on the illustration icon (photo-like icon in the top right of the Pathway Browser) displays the relevant Pathway Illustration (see example below).</p>\r\n<p><img src=\"images/userguide/analysis/analysis_20.png\" alt=\"analysis 20\" width=\"744\" height=\"275\" /></p>\r\n<h3></h3>\r\n<h3><span>Getting Started</span></h3>\r\n<p></p>\r\n<h3><span>Pathway Analysis </span><b>Exercises</b></h3>\r\n<p><span>This exercise is to check that you understand the results of Pathway Analysis.</span></p>\r\n<p></p>\r\n<p><span>On the Home page, open the Pathway Browser. Click on the Analyse Data button. When the submission form appears, in the Analysis Tools section select ‘Click here to paste your data or try example data sets…’</span></p>\r\n<p><span>…and click on the UniProt accession list button on the right side. </span></p>\r\n<p></p>\r\n<p><span>Your submission page should look like this:</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/dBXvqrzfG5X_f4IjDdOt_kR1erNp83Jc7Ep-xjoktrS0go6Xxr_t5jaZOAf8XEatzqrgen7dh-PIGSvpNtvHzkp4cjDn6fzUm7RTSdcrWw2FdA67n4P-45zHF0DYPM-PyE2SO94f38qrrxaSJ5A6Gw\" alt=\"\" width=\"529\" height=\"342\" /></p>\r\n<p></p>\r\n<p><span>Click Continue. Make sure that Project to human is checked, but Include interactors is not.  When the results appear:</span></p>\r\n<p></p>\r\n<ol>\r\n<li><span>What pathway(s) is (are) most over-represented in this dataset? </span></li>\r\n<li><span>How many IDs match the pathway at the top of the list?</span></li>\r\n<li><span>Roughly what proportion of molecules in the pathway are matched?</span></li>\r\n<li><span>What proportion of reactions are matched?</span></li>\r\n<li><span>Repeat the analysis but this time check the box to Include interactors. Are the results the same? If not, why?</span></li>\r\n</ol>\r\n<h3></h3>\r\n<h3><span>Expression Analysis </span><b>Exercises</b></h3>\r\n<p><span>This exercise is to check that you can run and interpret the results of Expression Analysis.</span></p>\r\n<p><span>Launch Expression Analysis and load the example dataset. Click GO When the results are displayed, find the pathway DNA Repair, </span></p>\r\n<ol>\r\n<li><span>How many proteins are in this pathway?</span></li>\r\n<li><span>What proportion of these had expression data?</span></li>\r\n<li><span>What was the average expression at 24h?</span></li>\r\n<li><span>Which subpathway has the highest average expression value at 24h? (Hint: move your mouse pointer over the pathway overview diagram, and look at the scale on the right side). </span></li>\r\n</ol>\r\n<p></p>\r\n<h3><span>Species Comparison </span><b>Exercises</b></h3>\r\n<p><span>This exercise is to check that you can run and interpret Species Comparison results.</span></p>\r\n<p><span>Launch Species Comparison and select the species Danio rerio. When the results are displayed, use the Pathway Hierarchy to Navigate to ‘Hemostasis’, ‘Dissolution of fibrin clot’.  </span></p>\r\n<p></p>\r\n<ol>\r\n<li><span>Find PLAT(36-562) top-right of the diagram - what colour is it and why? </span></li>\r\n<li><span>Find HRG - what colour is it and why?</span></li>\r\n<li><span>Why is Zn2+ (upper left) uncoloured?</span></li>\r\n</ol>\r\n<p></p>\r\n<h3><b>More information</b></h3>\r\n<p></p>\r\n<h3><b>Molecular Interaction Overlay</b></h3>\r\n<p></p>\r\n<p><span>If the user is interested in analyzing the data with interactors from other sources (like IntAct), the ‘Include Interactors’ options must be selected in the ‘Analyze Data’ section. Once in the pathway diagram, the Molecular Interaction (MI) overlay allows protein-protein or protein-small molecule interactions to be superimposed onto the pathway diagram. The source depends on the currently selected interaction database. The default interaction database is IntAct (Static), which provides fast access to a quarterly updated and locally-hosted version of IntAct data. Access to the IntAct-hosted dataset and other PSICQUIC sources of interaction data (protein-protein and protein-small molecule) can be selected using the ‘PSICQUIC’ feature of the Interactor Overlays tab within the Settings panel in the middle-right of the Pathway diagram. A list allows selection of the source of interactors. This is automatically populated by querying the PSICQUIC Registry. If a new database is selected while interactors are displayed on the pathway diagram, the proteins represented by those entities will be used as queries in the new database and the display will automatically update. An ‘i’ (‘information’) button to the top right of the Interactor Overlays tab provides a quick tip about using the MI overlay. Selecting either ‘Cyan’ or ‘Teal’ in the ‘Interactors Colour Profile:’ drop-down menu within the ‘Colour Palette’ feature of the Settings panel will change the interactor colour.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/xH_T-TtBBWGoPtjw1k624JQmYElHOmxuoyIp0BU1kTZInmBaet8nBD9ON-zwX4ZXR7DhLm-CxF4fgtonOdgigLMBtNT2GTY6tUsiwEep_mYbPC1z2ytkNVQpoXO7sCGtg7p9UvZlolhla5BAsEVV_g\" alt=\"\" width=\"307\" height=\"311\" /></p>\r\n<p><span>Once a pathway diagram is displayed in the viewport, a small red circle with a white-lettered number in the top right corner of the entity will automatically appear for individual protein and chemical entities that have interactors. The number represents the number of known interactors for the given protein. Pressing the red circle with the cursor will display the interactors. A maximum of 10 interactors is displayed as a ring of blue-bordered boxes (protein interactors) or green-bordered ovals (small molecule interactors) connected by black lines to the selected protein. If the same interactor is connected to more than one entity it is re-used, i.e. connected to all the selected entities in the diagram. Furthermore, if the interactor is a protein or small molecule entity that pre-exists within the pathway diagram, a black line connects the two entities. Pressing the red circle a second time will remove the interactors from the pathway diagram. If an entity interacts with itself, a looping arrow is shown.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/MrRSfPJ61nz8BgXn_Jz6tORuPhNsVVilXQfaO_IznC50hLmr97RFlEgsS5yi6qCu8r-0VUqzQYMhTqLAgtcmkbNCEmMXiWEj_KW2R513v3o6tbCPFV8hFKwJYarE7HMw3y1y97Cvx747WMltVHv77A\" alt=\"\" width=\"647\" height=\"427\" /></p>\r\n<p><span>A number of actions provide additional information about the interactor and its relationship to the protein entity. These include:</span></p>\r\n<ul>\r\n<li><span>Hover the mouse pointer over a protein interactor to reveal a tool-tip containing its name and UniProt identifier.</span></li>\r\n<li><span>Click an interactor to open its database entry in a new window.</span></li>\r\n<li><span>Click the line that connects the interactor to the pathway molecule to open details of the interaction from the source database, in a new window.</span></li>\r\n<li><span>Right-click a pathway object, or select the blue triangle that appears on the right side when hovering the mouse pointer over it, to open a pop-up panel (the Contextual Information Panel). Select the Interactors (bottom) tab to display a table of all interactors for the selected object. This table is scrollable and provides additional information about the interactors. Click the ‘Interactor’ name or accession identifier to connect to UniProt or the interaction source database, respectively. Click the ‘id’ button to toggle the display of either the interactor name or its database identifier. Click the pin button to fix the open table in position on the pathway diagram. Click the ‘X’ in the top right corner to close the table view.</span></li>\r\n</ul>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/wpQ--wcVMOV4Nu2u745uSwz8n0g0hQbcN3wJ-8MiGYeoOKcBZkTLQwBlOO4FQXt2m_6gPSf1wMmpAHSOrPeJans1pm1wvyKsoeq7_HdQscm3TvWrhD6Kc6PT4_hXjSs77CJeF4cBGX0syAWILhIQHw\" alt=\"\" width=\"664\" height=\"391\" /></p>\r\n<p><span>Interactions from the IntAct database have confidence scores that are generated as the cumulated sum of a weighted score that depends on the interaction detection method and the number of observations. The default lower threshold is 0.45. Interactions with a confidence level equal to or above this are shown. Move the slider bar in the Molecular Overlay toolbar (light grey bar with dark grey slider found at the bottom of the pathway diagram) to the right to raise the confidence level threshold. Interactors below the displayed confidence level will progressively disappear. Move the slider to the left and the interactors will reappear. Click the ‘X’ in the Molecular Overlay toolbar to hide it.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/0rpwIeHiu4Dh81Rh6FEjtnp5hZDfaGcp2hZkMCczq-vCUeRDPS4_3yz9zktbPZQlkD6SXqMyEKCgRw_MndlWtrggXF4Ou5JA077n921kXNJ8fxN49jg08fB7KT3D7F8AYFjOyy4jsT9PODHukPQuQw\" alt=\"\" width=\"400\" height=\"55\" /></p>\r\n<p><span>Zooming, either by using the controls in the bottom right corner of the Pathway Diagram or the mouse scroll wheel, increases the size of all objects displayed in the pathway diagram including interactors. When a sufficient size is reached, the crystal structure of proteins or chemical structure of small molecules will appear in the interactor object, if available.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/pUz7ovdDJ6mXTxE3w0RI0eb3k_fOifotSRsSXNhkRkS-9QGkg1zlKGlWaQox7rCxvloL1mQvOJ4KraXK25ebN9613dwOPsYi9pTTVuPC7ezI8eJK30FKuzp2s_HBTN-tdsb213fS2jR1t_TILLDs_g\" alt=\"\" width=\"672\" height=\"456\" /></p>\r\n<p><span>The details of every interactor for every protein in the displayed pathway can be downloaded as a tab-delimited file by clicking the ‘Cloud’ button on the Molecular Overlay toolbar or from the Interactor Overlays tab in the Settings panel.</span></p>\r\n<p><span></span></p>\r\n<h3><span>Molecular Interaction Overlay </span><b>Exercises</b></h3>\r\n<p></p>\r\n<p><span>This exercise is to check that you can use and interpret Molecular Interaction Overlays</span></p>\r\n<p></p>\r\n<p><span>Open the Pathway Diagram for Netrin-1 Signaling.</span></p>\r\n<ol>\r\n<li><span>Find the protein NEO1 (top left of the cytosol). How many interactors does it have? </span></li>\r\n<li><span>What is the confidence score for the interaction between NEO1 and GD3? How many times has this interaction been documented? Hint: This detail is not in Reactome.</span></li>\r\n<li><span>Find the protein PTPN11 (below and to the right of NEO1). How many interactors does it have? Display interactors for this protein. How many are there? Can you get a list of all of them?</span></li>\r\n<li><span>What is the easiest way to remove interactors?</span></li>\r\n</ol>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:51:54','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,8,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(85,699,'Diseases','diseases','<p><span>Reactome can annotate and display pathways associated with disease. </span><span>Reactome disease annotations include cancer, metabolic and immune disease as well as infectious diseases, among others. Where possible, Reactome disease pathways also include the interaction of relevant therapeutic drugs. (Disease pathways are contained in a separate top-level Chapter in the hierarchy, and are designated with a red “+” symbol to the left of the pathway name.)</span></p>\r\n<p><span>Pathway Diagrams:</span></p>\r\n<p><span>Disease events or processes </span><span>are shown in the context of the normal pathway diagram </span><span>to provide context to the disease event. </span></p>\r\n<p><span>Disease events have red connecting lines; abnormal molecules involved in these processes are outlined in red.</span></p>\r\n<p><span>For infection processes, events involved in the life cycle of the infecting agent and host interactions with the infecting agent have red connecting lines. Molecules derived from the infecting agent are outlined in red, while those derived from the host are outlined as normal in black.</span></p>\r\n<p><span>Disease events may be loss-of-function or gain-of-function compared to the normal process. </span></p>\r\n<p><span>For loss-of-function events (where a protein has lost all or most of its functional activity) the disease reaction is automatically overlaid on top of the corresponding normal reaction. The loss-of-function disease entity is outlined with a dashed red line, and products that are no longer made are greyed out with a superimposed red cross. These loss-of-function events represent “stop points” in the pathway.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/ECIWCGjnQT7a_dYG8865g-SPo_xa6Qw2f7OHcqQ_1q39n5A3hLbnaDpTPK0WfuhUflahdbU5tCZEXOWMc1DhaMYU1V_ey36YSYqFSkZo3g_YBXshR6A1GQ9KB6TBm09iuA_of-v530yXfeyWWcTj9A\" alt=\"\" width=\"486\" height=\"296\" /></p>\r\n<p></p>\r\n<p><span>For gain-of-function events (where a protein has acquired a novel function not performed by the wild-type protein), disease events are represented alongside the normal events. </span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/x0_f13MZ2lzcr_e0_yBTW3ZsmGkr9-lb7Ae155D2SNXFLnY1i_BPuBXlj1G7qG6_a3EsnembSAlW2Do8CY_iqvOn94prdLyR6FISvqJahqBd6iHZK0xMXOJLVMH4DjElwj6-qNuF-cgZWuLHQ0EO1Q\" alt=\"Screen Shot 2017-07-19 at 4.08.24 PM.png\" width=\"579\" height=\"413\" /></p>\r\n<p><span>When a gain-of-function mutant performs a normal function (either a higher rate or efficiency, or after being activated in an novel manner, for instance), these disease events are overlaid on the corresponding normal event in the pathway diagram.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/TrGR3G_GHlIya3fYLCxDRVvLnAa8dv_FQtQ8Mj9GknggE4jtSpMImhC6vlnGM-uzvpC3OFuwNqCz9bBaw2VaXjfK1RQArVi2omJ7_Ig9jGAs2jINBnF2kyiRkAQh_aW3TtA14L4XezAgEBPxztn21A\" alt=\"Screen Shot 2017-07-20 at 10.25.00 AM.png\" width=\"599\" height=\"364\" /></p>\r\n<p><span>Infectious disease events (which don’t occur in the absence of the infectious agent and therefore have no normal counterpart) have their own diagrams.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/w6D9a8jgIPKbGQa9gVx_rOwzIqM84CMQj-wBXPTFpGmC0lRfRkGRDkouvkOYpHGUj8TXCjj6l4GzWGTW__e2RO7cDrQsV7cxy6-7IqoWK04DadxKL2svzpvzk-CEF1O2cvYJ8aLIoYLkUGrJOn6AqQ\" alt=\"\" width=\"465\" height=\"336\" /></p>\r\n<p><span>Drug annotations:</span></p>\r\n<p><span>Where applicable, the effect of drugs on disease pathways is annotated.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/9uTs4SmYtvoR-HeWGOJhRlsvOtW9TGhFYuDkz279gaLNBThtluib39Fp9TLUZ9YXbYoC37el44M4fDpKmAdVvZhiLAkfu7iL12NEOAGLtZ9UXQT_XsZGEX3mAcujV0V_PzLgiz5acHH_OUQE1swoRw\" alt=\"Screen Shot 2017-07-20 at 11.55.30 AM.png\" width=\"624\" height=\"447\" /></p>\r\n<p></p>\r\n<p><span>Details panel:</span></p>\r\n<p></p>\r\n<p><span>The details panel for a disease pathway/event has a number of extra panels in addition to those displayed in a normal pathway/event.</span></p>\r\n<p></p>\r\n<p><span>Disease tag: </span></p>\r\n<p><span>Disease events and entities are tagged with a disease term, taken from the </span><a href=\"http://www.disease-ontology.org/\"><span>Disease Ontology</span></a><span> and displayed in the “Disease” panel. </span></p>\r\n<p><span>Individual proteins are labeled with all the relevant disease terms, while sets of disease entities are tagged with the most specific term that represents all of the set members. </span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/YqHl3yPS8L6zpSZ3eqAeAxIHg_qwLJaQsyiiCdD6cQogRZtHGM8L9qXyMuUPfrlhWWedgJY5SnRyyvKsnMMql54jR1dCUUzm0psh2axQUYpDuhCEs7V31XH74JyhVmfRqN-hnY8CIaiLH-Klac3AQQ\" alt=\"Screen Shot 2017-07-19 at 4.26.47 PM.png\" width=\"624\" height=\"132\" /></p>\r\n<p><span>Where possible, disease proteins are linked to </span><a href=\"https://www.omim.org/\"><span>OMIM</span></a><span> (Online Inheritance in Man),</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/4WHSE0z8bLefDjATxs27SafbW0ygEgtAdp0U2RTt9dp1m6LeTgYGE6JyrpFNfoxqX6T3jdrwjoI_NI250mCB7tXW-Cfd1yti7fTMlKA0ZL4T_N4RPR4_retmkMi69GuOx-yEUPp0GFos0cDTORA4Cg\" alt=\"Screen Shot 2017-07-20 at 10.17.08 AM.png\" width=\"624\" height=\"216\" /></p>\r\n<p></p>\r\n<p><span>and cancer variants are linked to </span><a href=\"http://cancer.sanger.ac.uk/cosmic\"><span>COSMIC</span></a><span> (Catalogue of Somatic Mutations in Cancer).</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/0suvelWQ7ww5WCwEcEWhDIOzf4AjoLwZO9aEtWTMYAg-sJFNKLsfXSwLhf-UcmH0pIBCORh_Y0AS1OWV9Uc7U1wzCehwEBWFUQFl8NiHsoSCRvhauVqXwITajHCvORjRaHYgboktrXsXWYrgH_WEfQ\" alt=\"Screen Shot 2017-07-19 at 4.28.09 PM.png\" width=\"624\" height=\"91\" /></p>\r\n<p></p>\r\n<p><span>Functional status:</span></p>\r\n<p><span>As described above, disease events are tagged as either gain- or loss-of-function, and this is displayed in the “Functional Status” panel.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/Z6AJ_-hYyxzGEnCgqn34Ec1xiNV5GEmYXy_ki91BEa7_MyU5bsQP_--0fhJJfIv3SbQJAw5QFvVXqzOdUKBm5GUor7WO9Me8ySFYtZ3QOTUi_0uFDiSPz5r8M6zGM2q085s_I3mx8woHCmGc0HSJpA\" alt=\"Screen Shot 2017-07-19 at 4.29.28 PM.png\" width=\"624\" height=\"183\" /></p>\r\n<p></p>\r\n<p><span>Normal Pathway/Reaction:</span></p>\r\n<p><span>Where applicable, the corresponding normal pathway or reaction is identified in the details panel of the disease event. </span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/jEdbE7DGDAR4PuvM4Oa6-r-ECAO1osZpnrK6PyWr22gPjihMnFC8AZhYw4Qb0JsN5cTKpbfk94JfZdSw0D_5pnj_2rXuGZKu69RQYIthP0QbSZjN0bqU1Aq5Y72tIQHYh3UfO2ioPUZOGRoo3L_Tsw\" alt=\"Screen Shot 2017-07-19 at 4.34.53 PM.png\" width=\"624\" height=\"61\" /></p>\r\n<p></p>\r\n<p><span>Disease entities:</span></p>\r\n<p><span>Disease entities are annotated in molecular detail. Changes to protein sequence that arise as a result of variation in DNA sequence in disease are displayed in the details panel when a disease entity is highlighted in the diagram. Missense, nonsense, frameshift, deletion and fusion mutations are all annotated. See “More info”, below, for details of these annotations. </span></p>\r\n<p><span>Primary references describing the identification of the mutants proteins are annotated, but this information is not currently displayed on the website. Note also that on the website, these genetic modifications are displayed in the same panel as post-translational modifications such as phosphorylations, and are therefore (mis)labeled as ‘Post-translational modifications’.</span></p>\r\n<p></p>\r\n<p><span>Missense mutation:</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/lvDCMvglV_YrIfaxpn6zvzPTGtUhWQChGUWM72fyYeiXFa3_guPyyegpB81zIjF-LMvlBj-tbU23rg3fm7_NYKVevs5RwStYMRk1Pz67jfZIv79Ld0qM-LFTMaH3fSusQqYWxoag7_OwJHN_O0nW5w\" alt=\"Screen Shot 2017-07-20 at 10.54.40 AM.png\" width=\"624\" height=\"104\" /></p>\r\n<p></p>\r\n<p><span>Deletion mutation:</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/6vuG52aeNoW4uMtvxeGq1SIW8MpPY1med1xVbYhgiK145vo-DhlyipCJxtJW-U3CgoLC7qfdna69vJxQjCed1vq2Oy2EFpeHx18JiXC4R0NBbkO4V6FeuiATdH0k4DHGGh6XQWOHTVSHfxURdfnm2A\" alt=\"Screen Shot 2017-07-20 at 10.55.35 AM.png\" width=\"624\" height=\"151\" /></p>\r\n<p></p>\r\n<p><span>Drugs:</span></p>\r\n<p><span> Drugs are cross-referenced to </span><a href=\"https://www.ebi.ac.uk/chebi/\"><span>ChEBI</span></a><span> and to </span><a href=\"http://www.iuphar.org/\"><span>IUPHAR</span></a><span> where possible and applicable, and this information is displayed in the details panel when a drug is highlighted in the diagram.</span></p>\r\n<p></p>\r\n<p><span>Get started:</span></p>\r\n<p></p>\r\n<p><b>Example 1 - drug-target interaction in disease</b><span>:</span></p>\r\n<p><span>The cystic fibrosis transmembrane conductance regulator (CFTR) is a low conductance chloride-selective channel that mediates the transport of chloride ions in human airway epithelial cells. Chloride ions plays a key role in maintaining homeostasis of epithelial secretions in the lungs. Defects in CFTR can cause cystic fibrosis (CF), resulting in an ionic imbalance that impairs clearance of secretions, not only in the lung, but also in the pancreas, gastrointestinal tract and liver. More than 1500 mutations in the CFTR gene have been identified. </span></p>\r\n<p></p>\r\n<p><span>Gain-of-function events involving CFTR F508del:</span></p>\r\n<p><span>Deletion of phenylalanine 508 in CFTR is the most prevalent mutation causing cystic fibrosis. F508 deletion causes destabilization and subsequent targeting for co-translational degradation by the ER-associated degradation machinery (ERAD). F508del is ubiquitinated by ERAD-associated E3 ligases including RNF5 and RNF185, targeting it for VCP-mediated retrotranslocation and 26S proteasomal degradation.  These events are novel for the mutant protein and are represented in the disease diagram “ABC transporter disorders”.</span></p>\r\n<p></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/1G_7ddhZUkBnzlCLLM3cQLPKuR_4tZYNFPFU04Bn4PzqDUF7X2leVCGeIq7fZ_8YP4QGrqThj6eXY1CLdQQJP9luVqJqKHlPZbYn7ym5tErLgvT4hosOjgRp6DetCbrQWIQdP-q2vArOwbezfPcqFA\" alt=\"Screen Shot 2017-07-20 at 4.30.29 PM.png\" width=\"624\" height=\"401\" /></p>\r\n<p></p>\r\n<p><span>Loss-of-function events for CFTR mutants:</span></p>\r\n<p><span>Some loss-of-function CFTR mutants are properly transported to the plasma membrane but are unable to transport chloride ions to the extracellular space.  These are represented as a loss-of-function event in an overlay of the corresponding WT reaction, and displayed in the “ABC transporter disorders” disease diagram.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/XtdPcxApQv0zSOsQPf0rbheZLavc6YVWxz92oEE-p6PF8NP45ae3r66cl_WztdGgifKPQINZNWVKytylN2bFeC-yBk8zjdU7fASwaS2zzyTFpvn_fEeWL1ioKRWgl4kJPCnzdeIxYs_cCE9tCGXzlA\" alt=\"Screen Shot 2017-07-20 at 4.39.41 PM.png\" width=\"624\" height=\"443\" /></p>\r\n<p></p>\r\n<p><span>Drug interactions in Cystic Fibrosis:</span></p>\r\n<p><span>CF patients with a particular mutation, G551D, have shown lung function improvements when given the drug Ivacaftor. Reactions showing ivacaftor binding the mutant protein (top right, above) and the following reaction (centre) showing channel functionality restored are displayed in the disease diagram.</span></p>\r\n<p></p>\r\n<p><b>Example 2 - diseases caused by accumulation of substrate over time</b><span>:</span></p>\r\n<p><span>Special case in Reactome where the reaction describing a particular substrate\'s metabolism occurs in normal physiology but over time, accumulation of the substrate leads to toxic consequences which can lead to disease. Examples are neurodegenerative diseases such as Alzheimer\'s and Parkinson\'s diseases, chronic obstructive pulmonary disease and retinal macular degeneration.</span></p>\r\n<p><span>The bisretinoid A2E (di-retinoid-pyridinium-ethanolamine) is a major component of lipofuschin, a yellow-brown pigment grain composed mainly of lipids but also sugars and certain metals whose accumulation is associated with degenerative diseases. In the eye, A2E is the end-product of the condensation of 2 molecules of all-trans-retinal and phosphatidylethanolamine in photoreceptor outer disc membranes. Once formed, A2E is phagocytosed, together with outer segments, to retinal pigment epithelial (RPE) cells where it accumulates. There is no evidence as yet to indicate that A2E can be catabolised. </span></p>\r\n<p><span>The relationship between lipofuscin accumulation and retinal degeneration is illustrated by Stargardt disease type 1. Because the reactions can be considered \"normal\" (they occur as part of the normal metabolism of retinoids) as well as disease-causing, they appear in the normal pathway diagram (Visual phototransduction) and coloured red. </span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/3iicpYVHwl6-h11WHNquSh9f6NEyoOvQSgdL1_aY5t6psjzW2zSlkOTmgQG4Sha3CwcTA1OOq1tDs9yz5hxRu-yIlnUyBmaurzVrtpkzSQD9yMTUGcEBOlt5mcjST8EiV-NDqpaADt1FmW8n0bC5dw\" alt=\"Screen Shot 2017-07-21 at 10.02.59 AM.png\" width=\"624\" height=\"423\" /></p>\r\n<p></p>\r\n<p><span>To display in the disease view of the diagram, these reactions are added as components of the disease pathway \"Diseases associated with visual transduction\". These reactions will now display in both normal and disease diagrams.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/1hqo-MMWOSN8SqRE5Xo5mOGlNkKAWxebRQgRbktjaQzajGaPxgQecETS_tf4kjoKk0Bu7RjUM7_rlwVJAD0f5Y_UXJXsqkW-60fDZDKyoX0jyk49_3uk_EuVwoaeoJrQnOhpN1K62V6EE6_NpRATjg\" alt=\"Screen Shot 2017-07-21 at 10.04.48 AM.png\" width=\"624\" height=\"444\" /></p>\r\n<h3><b>Getting Started</b></h3>\r\n<h3><b>Exercises:</b></h3>\r\n<p></p>\r\n<ol>\r\n<li><span>Search for the pathway FGFR1 mutant receptor activation and open it. What disease(s) is/are associated with this pathway? (</span><i><span>Hint: look at the Description tab in the Details section</span></i><span>). </span></li>\r\n<li><span>Search for the reaction Defective MMADHC does not bind MMACHC:B12r. What type of defect has caused this loss of function? (</span><i><span>Hint: look at the Functional Status category for the reaction</span></i><span>).</span></li>\r\n<li><span>How many mutations of MMADHC are represented?</span></li>\r\n</ol>\r\n<p></p>\r\n<p><span>More info:</span></p>\r\n<p><b>Annotating genetic alteration of protein sequence for disease pathways:</b></p>\r\n<p><span>The following image shows the portion of the Reactome data model describing the relationship between possible modifications to protein sequence. The subclass “GeneticallyModifiedResidue” is used for the annotation of disease entities, while “TranslationalModification” is used to annotate the consequence of processes such as phosphorylation, acylation, cross-linking and other similar non-genetic events. TranslationalModifications will not be described further here.</span></p>\r\n<p></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/jG-CrhgbwJZrzulDipna6TqymnFCZc-3f3CCxIiX33RWB5SOc1bPs6n-ApaqJTkehDUlkptUVV0fpc_S6ifXxa4I3H9b7Yt7u0AXDegbjgn-ZTt6XJoFbCjB1Q94g3__fvWWTLhH0RectEhzPit9fw\" alt=\"Screen Shot 2017-07-21 at 10.42.28 AM.png\" width=\"351\" height=\"249\" /></p>\r\n<p></p>\r\n<p><span>The ReplacedResidue class is used for amino-acid substitutions.  This class is also used for “simple” nonsense mutations that change a coding amino acid for a stop codon.</span></p>\r\n<p><span>The FragmentModification class describes more extensive changes to the coding sequence through insertions and deletions in the gene, and includes three subclasses:</span></p>\r\n<ul>\r\n<li><span>FragmentDeletionModification is used for in-frame deletions of amino-acids</span></li>\r\n<li><span>FragmentInsertionModification is used for in-frame insertions of amino-acids, including genomic events that result in fusion proteins</span></li>\r\n<li><span>FragmentReplacedModification is used for frameshifts.</span></li>\r\n</ul>\r\n<p></p>\r\n<p><span>Examples of each annotation type are further described below.</span></p>\r\n<p></p>\r\n<p><b>Simple missense mutation: HHAT G287V</b></p>\r\n<p><span>Hedgehog (Hh) is a secreted morphogen that regulates a number of developmental processes in vertebrates, including limb development and neural tube patterning, among others.  Maturation of Hh ligand includes a number of proteolytic processing and lipid modification steps. These modifications are required for normal transit of the ligand to the surface of the secreting cell and mutations that affect these processes are associated with decreased Hh ligand secretion, abrogated Hh signaling and disease.  </span></p>\r\n<p></p>\r\n<p><span>HHAT is an O-acyltransferase that palmitoylates the N-terminal fragment of Hh. A G287V loss-of-function mutation in HHAT was identified in a rare case of Syndromic 46 XY Disorder of Sex Development, which results in testis dysgenesis. This mutant is not able to palmitoylate the Hh ligand. Details of the amino-acid substitution are displayed in the details panel when the entity is highlighted in the diagram:</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/-c7-uFziAnI43NgAjmA-Kp3BiYPF8qiBnUKKIrGTybCdp2nwdhEjlnNfV6uWNyleXAw0n1Z-Y1MjgM0_wI9oOQHn8ZuvM5dhUgfgw4wa02IXu_j6yJghQ3Gs7oXhsD4OEU4a0EjlPE5_PAjeL1p7lA\" alt=\"Screen Shot 2017-07-21 at 2.26.44 PM.png\" width=\"624\" height=\"368\" /></p>\r\n<p></p>\r\n<p><b>Simple nonsense mutation: </b></p>\r\n<p><span>The disorder “Ehlers-Danlos syndrome, musculocontractural type 1” (EDSMC1) is caused by loss-of-function mutations in the carbohydrate sulfotransferase 14 (CHST14) gene. A nonsense mutation causing this disorder is </span><span>a 205A-T transversion in the CHST14 gene, resulting in a lys69-to-ter (K69*) substitution.</span><span> This is represented in the details panel as “L-lysine 69 replaced with unknown”.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/NTzHie9iEHLQztIdfC0fVFgwXeDJ42qYbZNxfmxayTuBL8VFGrxGPnCyJ2zZf2CI-lALxV-JoccfXMQ49LZ39JSkQz0rREajieGXF7d6OJ47y8lOrpzx7mnvgcyy93ocGjm9NPjJvTAR2ufmT3wpJg\" alt=\"Screen Shot 2017-07-21 at 2.33.20 PM.png\" width=\"624\" height=\"479\" /></p>\r\n<p></p>\r\n<p><b>FragmentDeletionModification:</b></p>\r\n<p><span>This class is used for in-frame deletions of amino-acids leading to internally truncated proteins. These variants are named “core protein name” [first amino acid of deletion_last amino acid of deletion]del, as shown below for PIK3R1 Y463_L466del, which has a deletion of residues Y463 to L466, inclusive:</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/bpm3TU53KCjGSZ1V7nlukxUQ4gGQjbw2CgjWhlMI1preT6XpsbE1lOf8JDdcjxMITyqlpYUObJduzp-pB94ehjI8dDiGAlgYEJf4mpOTp4nit5meGKQNXliJkNVrIHz5PRa8RuYpT2wVjsMDCFU9Yw\" alt=\"Screen Shot 2017-07-21 at 2.40.02 PM.png\" width=\"624\" height=\"508\" /></p>\r\n<p></p>\r\n<p><b>FragmentInsertionModification</b></p>\r\n<p><span>This class is used for in-frame insertions of amino-acids and for fusion proteins.</span></p>\r\n<p><b>In-frame insertions:</b></p>\r\n<p><span>These variants are named   “core protein name” [aa prior to insertion_aa following insertion]ins[inserted aa’s]. The amino-acid string of the inserted residues is added manually to the mutant protein name, as shown below for EGFR V738_K739insKIPVAI. This represents a variant where the amino acids 739-744 (KIPVAI) from EGFR are inserted at aa 739 of EGFR, and is thus a duplication of these residues: </span></p>\r\n<p></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/HzXhutKYw3XLYgK4OzNmEZjyPW1P5HVleNoqHlOykRn9k0gBnoNHkRd_8QnFU5bgk0PPDD3_yOz7EFHXzuBSyFqWMBoy6RRbMm9aM0X9FaK3VNKxZySgAyJnAwjI_8DZWerLBZeS0jjYNcXfk5b2AA\" alt=\"Screen Shot 2017-07-21 at 2.57.20 PM.png\" width=\"624\" height=\"479\" /></p>\r\n<p></p>\r\n<p><b>Fusion proteins:</b></p>\r\n<p><span>The FragmentInsertionModification class is also used to annotate proteins that arise as the result of genomic changes that bring two genes together to result in a fusion protein, as in the ZMYM2-FGFR1 fusion described below.  This fusion puts the ZMYM2 dimerization region (1-914) together with the kinase domain of the FGFR1 receptor (residues 429-822) and results in constitutive activation of the kinase domain by virtue of ligand-independent dimerization (for reference, the full length aa sequence of these two proteins are 1-1377 and 1-822 for ZMYM2 and FGFR1, respectively).</span></p>\r\n<p><span>By convention, the N-terminal most partner of the fusion is set as the reference protein for the variant protein while the C-terminal fusion partner sequence is captured in the FragmentInsertionModification record, as in the example below for the ZMYM2-FGFR1 fusion:</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/vSmj8cdDGv5McQY6zk-bhWOFjglRTdM1G5-01pgWtuSTpE2Uu7zs6q90a2Q7ulYws4RiPMKOOlypNVTkwI-0Cm-2E_9xn5Cdhp2IysaxWXzRZyZMW86Tc5WNFPuyGckbmriaSn8_-otpT9kvbuh_1w\" alt=\"Screen Shot 2017-07-21 at 3.04.28 PM.png\" width=\"624\" height=\"456\" /></p>\r\n<p></p>\r\n<p><span>Post-translational modifications to either partner in the fusion protein are numbered according to each respective WT reference gene product and do not reflect the aa position in the fusion.  For instance, if in the context of the fusion protein, the FGFR1 partner is phosphorylated at (WT FGFR1 position) Y766, the fusion EWAS would be ZMYM2-pY766-FGFR1, despite the fact that in linear sequence the phosphorylation occurs at residue 1250 of the fusion (913+(766-429)). </span></p>\r\n<p></p>\r\n<p><b>FragmentReplacedModification</b></p>\r\n<p><span>This class is used to annotate frameshift mutations that alter the amino-acid sequence of the protein as in the case of EXT1, described below.</span></p>\r\n<p><span>The disorder “Hereditary multiple exostoses 1” (EXT1) is caused by loss-of-function mutations in Exostosin 1 (EXT1). One such mutation is a 1-bp deletion at nucleotide 1469 in the EXT1 gene, resulting in a frameshift mutation with a premature stop codon nine amino acids downstream. Variants of this type are named “core protein name</span><span>” </span><span>[aa prior to insertion_fs*(number of aa to the first stop codon)]. The novel amino acids that occur as a result of the frameshift are identified in the mutant protein record, as shown below for EXT1 L490Rfs*9:</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/Jadn77kIybJhazFC-460IJkD71ALsqLWn9Ca7o3ihEybJBH1Cgw-Rojh9FMjarx-Pc5UXwIx3UH8FmF95lm_tivcaA74O4JUJ5PeopDdU0wmU70sKr1EA3f3kKrsZObwEM6v8Ugqo1hsUSqKz5FuZA\" alt=\"Screen Shot 2017-07-21 at 3.23.53 PM.png\" width=\"624\" height=\"460\" /></p>\r\n<p><span>For cancer-related processes, it is useful to view the altered disease events alongside their normal counterparts in the same diagram. The user can then see where normal processes diverge into ones that are implicated in cancer.</span></p>\r\n<p></p>\r\n<p><span>For metabolic processes, proteins that have lost all or most of their functional activity towards a substrate causes the majority of defects. The diagram displays these disease reactions as \'stop points\' in the pathway. Defective enzyme catalysts are outlined by a red dashed line; products that are no longer made are shown greyed-out with a superimposed red cross.</span></p>\r\n<p><img src=\"https://lh7-us.googleusercontent.com/SW9rceePmuPkuHNCR734-ekj8YIj8fEAoKRzlXXWA428BPXEdnVh9uQi37CJSG4DCNgp3PuBvs6wzOb-8mfhpLlKLevwseeBznoBOedAoHAOscBEUy0y41bdu_c-l2C_QKTaagDoe1tHNshWcAxm0g\" alt=\"\" width=\"455\" height=\"205\" /></p>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:53:13','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,7,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(86,701,'Physical Entity Identifiers Mapping','pe-identifiers-mapping','<p>The mapping files consist of a tab-separated table that indicates which external protein, gene or small molecule identifiers in the source database were mapped to Reactome physical entities and reaction annotations. Our goal with distributing three sets of files for each different identifier type is to provide these mapping files link the source database identifier to:</p>\r\n<ul>\r\n<li style=\"list-style-type: lower-roman;\">the lowest level pathway diagram or subset of the pathway</li>\r\n<li style=\"list-style-type: lower-roman;\">all level pathway diagrams (filename appended with \'All_Levels’),</li>\r\n<li style=\"list-style-type: lower-roman;\">all reaction events.</li>\r\n</ul>\r\n<p>The columns within the mappings files follow a similar format:</p>\r\n<ol>\r\n<li>Source database identifier, e.g. UniProt, ENSEMBL, NCBI Gene or ChEBI identifier</li>\r\n<li>Reactome Physical Entity Stable Identifier</li>\r\n<li>Reactome Physical Entity Name</li>\r\n<li>Reactome Pathway Stable identifier</li>\r\n<li>URL</li>\r\n<li>Event (Pathway or Reaction) Name</li>\r\n<li>Evidence Code</li>\r\n<li>Species</li>\r\n</ol>\r\n<p>If you have any further questions, please <a href=\"mailto:help@reactome.org?subject=PE%20Identifier%20Mapping%20File\">contact us</a>.</p>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-06-23 09:19:03','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(87,703,'Reaction PMIDs','reaction-pmids','<p>The \"Reaction PMIDs\" file comprises of a&nbsp;tab-separated table consisting of two columns that provide information about the research publications associated with Reactome pathways. The first column provides the pathway stable identifier, whereas the second column provides the PubMed citation identifier.</p>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-09-25 14:58:11','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(88,704,'Reactome to OMIM','reactome-to-omim','<p>The \"Reactome to OMIM\" file comprises of a&nbsp;tab-separated table provide supporting annotations to establish links from <a href=\"https://www.omim.org/\">OMIM</a>&nbsp;to Reactome pathways using isoform information in UniProt. The \"Reactome to OMIM\" file consists of three columns defining the pathways in the \"lower levels\" of the pathway hierarchy where the PhysicalEntity for the given UniProt accession can be found. The first column provides the UniProt accession (Acc), whereas the second and&nbsp;third column provides the pathway stable identifier (PathwayId), and the name of the pathway (Pathway), respectively.</p>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-09-25 14:58:50','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(89,705,'Protein Role Reaction','protein-role-reaction','<p>The \"Protein Role Reaction\" file consists of a tab-separated table comprising three columns that provides information about the role of a protein within a pathway. The first column provides the UniProt accession whereas the second and third columns provides the role of the protein within the reaction and the reaction stable identifier, respectively.</p>','',1,33,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-09-25 14:59:25','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(90,709,'Version 62 Released','version-62-released','<p style=\"text-align: justify;\"><strong><a href=\"/PathwayBrowser/#/R-HSA-445717\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-445717.svg\" alt=\"\" width=\"681\" height=\"400\" /></a></strong></p>\r\n<p><strong>New and Updated Pathways.</strong><strong>&nbsp;</strong>In version&nbsp;V62, topics with new or revised pathways include Developmental biology (<a href=\"/PathwayBrowser/#/R-HSA-376176\">Signaling by Robo receptor</a>), Gene expression &nbsp;(<a href=\"/PathwayBrowser/#/R-HSA-8953750\">Transcriptional regulation by E2F6&nbsp;</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-8878166\">Transcriptional regulation by RUNX2</a>), and Immune System (<a href=\"/PathwayBrowser/#/R-HSA-1266695\">Interleukin-7 family signaling</a>,<a href=\"/PathwayBrowser/#/R-HSA-8983432\">&nbsp;Interleukin-15&nbsp; family signaling</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-8984722\">Interleukin-35&nbsp; family signaling</a>, and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-9007892\">Interleukin-38&nbsp; family signaling</a>). Illustrations are now available for&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-445717\">Aquaporin-mediated transport</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-2559583\">Cellular senescence</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-212165\">Epigenetic regulation of gene expression</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-74160\">Gene Expression</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-5250941\">Negative epigenetic regulation of rRNA expression</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-1480926\">O2/CO2 exchange in erythrocytes</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-2980736\">Peptide hormone metabolism,</a>&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-5250913\">Positive epigenetic regulation of rRNA expression</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-597592\">Post-translational protein modification</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-5660526\">Response to metal ions</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-162582\">Signal Transduction</a>, and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-425407\">SLC-mediated transmembrane transport</a>.</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.&nbsp;</strong><a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Patricia+Ducy%5BAuthor%5D\">Patricia Ducy</a>&nbsp;is our external author.&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Patricia+Ducy%5BAuthor%5D\">Patricia Ducy</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Jorg+J+Goronzy%5BAuthor%5D\">Jorg Goronzy</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Anna+Herlihy%5BAuthor%5D\">Anna Herlihy</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Alexander++Jaworski%5BAuthor%5D\">Alexander&nbsp; Jaworski</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Umesh+Kumar%5BAuthor%5D\">Umesh&nbsp; Kumar</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Mora%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26892022\">Javier Francisco Mora</a>,&nbsp;<a href=\"http://scholar.google.co.in/citations?user=MDttKOwAAAAJ&amp;hl=en\">Manoj Patidar</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/?term=Yuliya++Pylayeva-Gupta%5BAuthor%5D\">Yuliya&nbsp; Pylayeva-Gupta</a>,&nbsp;<a href=\"http://www.ncbi.nlm.nih.gov/pubmed?term=Kailash%20%20Singh%5BAuthor%5D\">Kailash Singh</a>, and Ren Sun are our external reviewers.</p>\r\n<p><b>Annotation Statistics.</b>&nbsp;Reactome comprises 11,302 human reactions organized into 2,176 pathways involving 10,878 proteins encoded by 10,712 different human genes, 1,768 small molecules, and 11,284 complexes. These annotations are supported by 27,526 literature references. We have projected these reactions onto 121,709 orthologous proteins, creating 20,854 orthologous pathways in 18 non-human species. Version 62 has annotations for 1,334 protein variants (mutated proteins) and their post-translationally modified forms, derived from 285 proteins. These have been used to annotate 506 disease-specific complexes and 906 disease-specific reactions organized into 453 pathways and subpathways, and tagged with 294 Disease Ontology terms.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-09-28 15:45:38','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,66,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(91,714,'Introducing our Logo','introducing-our-logo','<style type=\"text/css\">.pure-logo{\r\n    text-align: center;\r\n}\r\n.pure-logo img {\r\n    max-width: 50%;\r\n    padding: 15px;\r\n    margin: 10px;\r\n}\r\n\r\n.pure-logo img.imagotype {\r\n    max-width: 80%;\r\n}\r\n\r\n.pure-logo img.isotype {\r\n    padding: 3px;\r\n}\r\n\r\n.pure-logo img.negative {\r\n    background-color: rgba(47, 158, 194, 0.3) !important;\r\n}\r\n\r\n.intro-logo {\r\n   border: 1px solid #c8c8c8;\r\n    -webkit-border-radius: 10px;\r\n    -moz-border-radius: 10px;\r\n    border-radius: 10px;\r\n    margin: 15px 0;\r\n}\r\n\r\n.formats {\r\n    border-top: 1px solid #c8c8c8;\r\n    margin: 0 10px;\r\n}\r\n\r\n.png-download {\r\n    display: inline-block;\r\n}\r\n\r\n.png-download p a {\r\n    font-weight: bold;\r\n    color: white;\r\n}\r\n\r\n.png-download p a:focus, .png-download p a:hover {\r\n    font-weight: bold;\r\n    text-decoration: underline;\r\n    background-color: #FFF;\r\n    color: #2F9EC2;\r\n    opacity: 0.7;\r\n    -webkit-border-radius: 4px;\r\n    -moz-border-radius: 4px;\r\n    border-radius: 4px;\r\n    padding: 1px 3px 1px 3px;\r\n}\r\n\r\n.png-download .tooltip-sizes {\r\n    visibility: hidden;\r\n    background-color: #2F9EC2;\r\n    color: #fff;\r\n    text-align: center;\r\n    border-radius: 6px;\r\n    padding: 5px 0;\r\n\r\n    /* Position the tooltip */\r\n    position: absolute;\r\n    z-index: 1;\r\n\r\n    /* TOP */\r\n    width: 145px;\r\n    top: 100%;\r\n    /*left: 80%;*/\r\n    margin-left: -54px;\r\n\r\n    -webkit-transition: height .5s; /* For Safari 3.1 to 6.0 */\r\n    transition: height .5s;\r\n    opacity: 0;\r\n    height: 0;\r\n    overflow: hidden;\r\n}\r\n\r\n.png-download .tooltip-sizes p {\r\n    margin: 8px 2px;\r\n    font-size: 12px;\r\n}\r\n\r\n.png-download:hover .tooltip-sizes {\r\n    visibility: visible;\r\n    background-color: #2F9EC2;\r\n    opacity: 1;\r\n    height: 110px;\r\n}</style>\r\n<p>The new Reactome logo brings to the forefront the value and quality of the information and analysis tools that can be found in our curated database of pathways. The layering in our logo highlights the transparency of Reactome’s nature while the rounded typography matches our openness.</p>\r\n<p>Derived from a shape that exists in nature and at the same time gives structure, the logo evokes the idea of discovering by unwrapping the layers of knowledge that surround biological events.</p>\r\n<h3><i class=\"fa fa fa-question-circle\" style=\"font-size: 23px;\"></i>How to use our logo</h3>\r\n<p>We invite you to use the brand (<a href=\"#imagotype\">imagotype</a>) or the logo (<a href=\"#isotype\">isotype</a>) as you decide. We suggest the usage of our image against a white background but we understand that sometimes it is just not possible, that is why you have as well the negative of our image, that fits nicely against a coloured background.</p>\r\n<div class=\"favth-row\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<h3><a href=\"#imagotype\" id=\"imagotype\"><i class=\"fa fa fa-file-image-o\" style=\"font-size: 23px;\"></i>Imagotype</a></h3>\r\n</div>\r\n</div>\r\n<div class=\"favth-row\">\r\n<div class=\"favth-col-lg-6 favth-col-md-6 favth-col-sm-6 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"intro-logo favth-clearfix\">\r\n<div class=\"pure-logo\">\r\n<div><img src=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive.svg\" alt=\"Reactome Logo\" class=\"imagotype\" title=\"Reactome Logo\" /></div>\r\n<div class=\"formats favth-clearfix\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-right\">\r\n<div class=\"png-download\"><a>PNG</a>\r\n<div class=\"tooltip-sizes\">\r\n<p><a href=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive_25mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Positive_25mm.png\">Small [297x70]</a></p>\r\n<p><a href=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive_50mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Positive_50mm.png\">Medium [591x138]</a></p>\r\n<p><a href=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive_100mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Positive_100mm.png\">Large [1183x276]</a></p>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix\"><a href=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive.svg\" title=\"Click to Download our logo in SVG format\" download=\"Reactome_Imagotype_Positive.svg\">SVG</a></div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-left\"><a href=\"/images/logo/imagotype/positive/Reactome_Imagotype_Positive.emf\" title=\"Click to Download our logo in EMF format\" download=\"Reactome_Imagotype_Positive.emf\">EMF</a></div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-6 favth-col-md-6 favth-col-sm-6 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"intro-logo favth-clearfix\">\r\n<div class=\"pure-logo\">\r\n<div><img src=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative.svg\" alt=\"Reactome Logo\" class=\"imagotype negative\" title=\"Reactome Logo (Transparent Background)\" /></div>\r\n<div class=\"formats favth-clearfix\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-right\">\r\n<div class=\"png-download\"><a>PNG</a>\r\n<div class=\"tooltip-sizes\">\r\n<p><a href=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative_25mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Negative_25mm.png\">Small [297x70]</a></p>\r\n<p><a href=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative_50mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Negative_50mm.png\">Medium [591x138]</a></p>\r\n<p><a href=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative_100mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Imagotype_Negative_100mm.png\">Large [1183x276]</a></p>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix \"><a href=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative.svg\" title=\"Click to Download our logo in SVG format\" download=\"Reactome_Imagotype_Negative.svg\">SVG</a></div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix  favth-text-left\"><a href=\"/images/logo/imagotype/negative/Reactome_Imagotype_Negative.emf\" title=\"Click to Download our logo in EMF format\" download=\"Reactome_Imagotype_Negative.emf\">EMF</a></div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-row\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"padding0 top30\"></div>\r\n</div>\r\n</div>\r\n<div class=\"favth-row\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<h3><a href=\"#isotype\" id=\"isotype\"><i class=\"fa fa-file-image-o\" style=\"font-size: 23px;\"></i>Isotype</a></h3>\r\n</div>\r\n</div>\r\n<div class=\"favth-row\">\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"intro-logo favth-clearfix\">\r\n<div class=\"pure-logo\">\r\n<div><img src=\"/images/logo/isotype/positive/Reactome_Isotype_Positive.svg\" alt=\"Reactome Logo (Isotype)\" width=\"80\" height=\"89\" class=\"isotype\" /></div>\r\n<div class=\"formats favth-clearfix\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-right\">\r\n<div class=\"png-download\"><a>PNG</a>\r\n<div class=\"tooltip-sizes\">\r\n<p><a href=\"/images/logo/isotype/positive/Reactome_Isotype_Positive_10mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Positive_25mm.png\">Small [119x138]</a></p>\r\n<p><a href=\"/images/logo/isotype/positive/Reactome_Isotype_Positive_25mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Positive_50mm.png\">Medium [297x342]</a></p>\r\n<p><a href=\"/images/logo/isotype/positive/Reactome_Isotype_Positive_50mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Positive_100mm.png\">Large [591x682]</a></p>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix\"><a href=\"/images/logo/isotype/positive/Reactome_Isotype_Positive.svg\" title=\"Click to Download our logo in SVG format\" download=\"Reactome_Isotype_Positive.svg\">SVG</a></div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-left\"><a href=\"/images/logo/isotype/positive/Reactome_Isotype_Positive.emf\" title=\"Click to Download our logo in EMF format\" download=\"Reactome_Isotype_Positive.emf\">EMF</a></div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"intro-logo favth-clearfix\">\r\n<div class=\"pure-logo\">\r\n<div><img src=\"/images/logo/isotype/negative/Reactome_Isotype_Negative.svg\" alt=\"Reactome Logo (Isotype)\" width=\"80\" height=\"89\" class=\"isotype negative\" /></div>\r\n<div class=\"formats favth-clearfix\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-right\">\r\n<div class=\"png-download\"><a>PNG</a>\r\n<div class=\"tooltip-sizes\">\r\n<p><a href=\"/images/logo/isotype/negative/Reactome_Isotype_Negative_10mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Negative_25mm.png\">Small [119x138]</a></p>\r\n<p><a href=\"/images/logo/isotype/negative/Reactome_Isotype_Negative_25mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Negative_50mm.png\">Medium [297x342]</a></p>\r\n<p><a href=\"/images/logo/isotype/negative/Reactome_Isotype_Negative_50mm.png\" title=\"Click to Download our logo in PNG format\" download=\"Reactome_Isotype_Negative_100mm.png\">Large [591x682]</a></p>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix \"><a href=\"/images/logo/isotype/negative/Reactome_Isotype_Negative.svg\" title=\"Click to Download our logo in SVG format\" download=\"Reactome_Isotype_Negative.svg\">SVG</a></div>\r\n<div class=\"favth-col-lg-4 favth-col-md-4 favth-col-sm-4 favth-col-xs-4 favth-clearfix favth-text-left\"><a href=\"/images/logo/isotype/negative/Reactome_Isotype_Negative.emf\" title=\"Click to Download our logo in EMF format\" download=\"Reactome_Isotype_Negative.emf\">EMF</a></div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n</div>\r\n<div class=\"favth-row favth-hidden-xs\">\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 favth-clearfix\">\r\n<div class=\"padding0 top30 bottom30\"></div>\r\n</div>\r\n</div>\r\n<script type=\"text/javascript\">var showMsg = 1;\r\nfunction internetExplorerRightClick(){\r\n    if (document.all){\r\n        if (showMsg != 1)return false;\r\n    }\r\n}\r\n\r\nfunction firefoxRightClick(e){\r\n    if ((document.layers) || (document.getElementById && !document.all)) {\r\n        if (e.which==2 || e.which==3){\r\n            if (showMsg != 1)return false;\r\n        }\r\n    }\r\n}\r\n\r\nif (document.layers){\r\n    document.captureEvents(Event.MOUSEDOWN);\r\n    document.onmousedown=firefoxRightClick;\r\n}\r\nelse{\r\n    document.onmouseup=firefoxRightClick;\r\n    document.oncontextmenu=internetExplorerRightClick;\r\n}\r\n\r\ndocument.oncontextmenu=new Function(\"return false\");</script>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-10-13 08:33:16','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(93,718,'New responsive website with a fresh look','reactome-launches-new-website','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Responsive_Reactome_2.png\" alt=\"Responsive Reactome 2\" width=\"650\" height=\"400\" /></p>\r\n<p>We’ve launched our new website and are excited to introduce you to our new look! Reactome is inviting its users to explore its new website. The new website has been designed to provide the ultimate user-friendly experience with improved navigation and functionality throughout. Created with the user experience firmly in mind, the new web interface has been designed using the latest technology, so the site is compatible with today\'s browsers and mobile devices. The site includes extensive documentation to help users understand Reactome’s complete range of tools for <a href=\"/PathwayBrowser/\">viewing pathway diagrams</a>, <a href=\"/PathwayBrowser/#TOOL=AT\">analyzing experimental data</a> and <a href=\"/user/reactome-fiviz\">exploring protein-protein interaction networks</a>. For software developers, our <a href=\"/dev\">technical documentation and use cases</a> provide a detailed overview of our extensive web services, to access our curated pathway data (<a href=\"/ContentService/\">ContentService</a>) and analytical tools (<a href=\"/AnalysisService/\">AnalysisService</a>). Biologists and bioinformaticians can now benefit from richer online content that is easier to navigate and share with others, assisting with pathway data analysis and visualization.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-11-01 16:25:34','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,65,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(94,719,'Proudly introducing our new logo','proudly-introducing-our-new-logo','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/new_logo.png\" alt=\"new logo\" width=\"650\" height=\"365\" /></p>\r\n<p>As part of the ongoing evolution of our website, we are proud to announce the launch of our new logo. Reactome has grown and evolved over the last 14 years, and we felt it was time for a change.</p>\r\n<p>The new logo brings to the forefront the value and quality of the information and analysis tools that can be found in our curated database of pathways. The layering in our logo highlights the transparency of Reactome’s nature while the rounded typography matches our openness.</p>\r\n<p>Derived from a shape that exists in nature and at the same time gives structure, the logo evokes the idea of discovering by unwrapping the layers of knowledge that surround biological events.</p>\r\n<p>We have refreshed our logo to reflect who we are today and to symbolize our dynamic future. Rest assured though: our high-quality data and services, and our dedication to you will remain the same!</p>\r\n<p>For more information, please go to <a href=\"about/logo\">Our Logo</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-11-07 16:44:47','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,64,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(95,721,'New Reactome Publication published in 2018 NAR Database Issue','new-reactome-publication-published-in-2017-nar-database-issue','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20171115_NAR_Paper_2017.png\" alt=\"20171115 NAR Paper 2017\" width=\"650\" height=\"489\" /></p>\r\n<p>A new&nbsp;research article titled “<a href=\"https://academic.oup.com/nar/article/doi/10.1093/nar/gkx1132/4626770\">The Reactome Pathway Knowledgebase</a>” has been published in the forthcoming&nbsp;2018 NAR Database Issue. The paper describes the&nbsp;deployment&nbsp;of a Neo4J graph database on our production server, development of a new high-performance in-memory implementation of our overrepresentation data analysis tool, improvements to the Pathway Diagram Viewer, and implementation of the new Enhanced&nbsp;High Level&nbsp;Diagrams (EHLDs). More publications from the&nbsp;<a href=\"about/team\">Reactome Team</a>&nbsp;can be found&nbsp;<a href=\"community/publications/\">here</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-11-15 04:02:20','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,63,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(96,722,'Diagram exporter','diagram-exporter','<style type=\"text/css\">pre, pre code {\r\n    word-wrap: initial;\r\n    white-space: pre;\r\n    word-break: break-all;\r\n}</style>\r\n<!-- This script performs the required analyses, gets the 2 tokens back and fills the links in the examples below. \r\nAnnotate the elements with class \'expression\' or \'enrichment\' accordingly-->\r\n<script type=\"text/javascript\">jQuery(document).ready(function() {\r\n		// The URL of the analysis method\r\n		var reactomeURI = \"/AnalysisService/identifiers/url/projection?pageSize=20&page=1\";\r\n		// The URL of our sample to be submitted for enrichment analysis\r\n		var enrichment = window.location.origin + \"/images/devguide/exporter/enrichment.txt\";\r\n		// Perform the ajax call to the analysis service\r\n		jQuery.ajax({\r\n			type: \"POST\",\r\n			url: reactomeURI,\r\n			data: enrichment,\r\n			headers: {\r\n					\"Content-Type\": \"text/plain\"\r\n			},\r\n			dataType: \"json\"\r\n		}).success(function (data, textStatus) { \r\n		   var token = data.summary.token;\r\n\r\n		   jQuery(\'.enrichment\').each(function(i, obj) {\r\n		       var link = jQuery(obj);\r\n		       var url = link.attr(\"href\") + token;\r\n		       link.attr(\"href\", url);\r\n		       link.text(function(index, text){\r\n                                    return text + token;\r\n                                });\r\n		   });\r\n		}).error(function (data, status, headers, config) {\r\n			$(\'#errorHolder\').text(status + \": \" + headers);\r\n		});\r\n\r\n		// The URL of our sample to be submitted for expression analysis\r\n		var expression = window.location.origin + \"/images/devguide/exporter/expression.txt\";\r\n		// Perform the ajax call to the analysis service\r\n		jQuery.ajax({\r\n			type: \"POST\",\r\n			url: reactomeURI,\r\n			data: expression,\r\n			headers: {\r\n					\"Content-Type\": \"text/plain\"\r\n			},\r\n			dataType: \"json\"\r\n		}).success(function (data, textStatus) { \r\n		   var token = data.summary.token;\r\n\r\n		   jQuery(\'.expression\').each(function(i, obj) {\r\n		       var link = jQuery(obj);\r\n		       var url = link.attr(\"href\") + token;\r\n		       link.attr(\"href\", url);\r\n		       link.text(function(index, text){\r\n                                    return text + token;\r\n                                });\r\n		   });\r\n		}).error(function (data, status, headers, config) {\r\n			$(\'#errorHolder\').text(status + \": \" + headers);\r\n		});\r\n		\r\n		// Get all elements annotated with \'addHost\' class and\r\n		// add window.location.origin in the displayed link\r\n		jQuery(\'.addHost\').each(function(i, obj) {\r\n		       var link = jQuery(obj);\r\n		       link.text(function(index, text){\r\n                                    return window.location.origin + text;\r\n                                });\r\n		   });\r\n	});</script>\r\n<h3 id=\"resultsHolder\"><span>Introduction</span></h3>\r\n<p><span>In Reactome, pathway diagrams are stored using a custom format. Aimming to allow researchers to include images of their favourite pathway diagrams into their publications, posters or presentations, we have developed the diagram exporter. This tool makes it easy to export pathway diagrams in bitmap format, allowing users to specify the output format, the quality and the decorators. The diagram exporter has been deployed as part of our <a href=\"/ContentService/#/exporter/toRasterUsingGET\" target=\"_blank\" rel=\"noopener noreferrer\">Content Service</a>, which constitutes an easy API that provides access to the Reactome knowledgebase. To learn more about the Content Service and how you can use it, have a look <a href=\"/dev/content-service\" target=\"&quot;_blank\">here</a>.</span></p>\r\n<p>This tutorial will walk you through the simple steps of using this tool to generate images of your favourite pathway diagram.</p>\r\n<h3><span>Get started</span></h3>\r\n<!-- Annotate the elements with class \'expression\' or \'enrichment\' accordingly -->\r\n<p><span>Generating the image of a diagram using our API is as easy as just calling the service with the identifier of the diagram and the desired extension (for example \".png\", \".jpg\" etc.)</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-169911.png\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-169911.png</code></a></pre>\r\n<p></p>\r\n<p><span><img src=\"images/devguide/exporter/Fig1.png\" alt=\"Figure 1\" width=\"620\" height=\"274\" /></span></p>\r\n<p><span>This will export the image in the default quality. To get an image in print quality, use the </span><i><span>quality&nbsp;</span></i><span><a href=\"#arguments\">argument</a>. Higher quality value results in higher image quality.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-169911.png?quality=7\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-169911.png?quality=7</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig2.png\" alt=\"Figure 2\" width=\"620\" height=\"274\" /></span></p>\r\n<p><span>In case the requested pathway has an interactive illustration (EHLD) associated with it, the diagram exporter will export the illustration as an image. In the following example, the EHLD of Hemostasis is requested in png format. Follow this <a href=\"/icon-info\">link</a> if you are interested to learn more about our EHLDs.&nbsp;</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-109582.png\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-109582.png</code></a></pre>\r\n<p></p>\r\n<p><span><img src=\"images/devguide/exporter/Fig3.png\" alt=\"Figure 3\" width=\"620\" height=\"274\" /></span></p>\r\n\r\n<p>If you are interested in a particular reaction or entity, just write it and the service will show it selected in its containing pathway.</p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-68919.png\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-68919.png</code></a></pre>\r\n<p></p>\r\n<p><img src=\"images/devguide/exporter/R-HSA-68919.png\" alt=\"R-HSA-68919\" title=\"R-HSA-68919\" width=\"620\"/></p>\r\n\r\n<h3><span>Decorators</span></h3>\r\n<p><span>The API allows users to easily highlight diagram elements (molecules and reactions) by either selecting them or flagging them. For instance, to select a reaction, we simply need to add the <i>sel</i> argument and its Reactome Identifier, exactly like in the following example.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-390522.png?sel=R-HSA-390598\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-390522.png?sel=R-HSA-390598</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig4.png\" alt=\"Figure 4\" width=\"620\" height=\"193\" /></span></p>\r\n<p><span>We can also use the <i>flg</i> argument to flag diagram entities (genes, molecules, etc.)</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-428359.png?flg=Q9NZI8\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-428359.png?flg=Q9NZI8</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig5.png\" alt=\"Figure 5\" width=\"620\" height=\"252\" /></span></p>\r\n<p><span>The selection and flagging arguments work in exactly the same way for EHLDs.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-109582.png?sel=R-HSA-983231&amp;flg=THBD\" target=\"_blank\" rel=\"noopener noreferrer\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-109582.png?sel=R-HSA-983231&amp;flg=THBD</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig6.png\" alt=\"Figure 6\" width=\"620\" height=\"193\" /></span></p>\r\n<h3><span>Analysis overlay</span></h3>\r\n<p><span>Reactome offers a pathway analysis service that supports enrichment and expression analysis. The diagram exporter allows you to overlay the results of the analysis on top of the exported diagrams. To do so, use the </span><i><span>token</span></i><span> argument to specify the unique token assosiated with the performed analysis. To learn more about our Analysis Service and how to use it have a look to this <a href=\"/dev/analysis\">page</a>.</span></p>\r\n<p><span>In the next example, we use the token acquired from an overrepresentation analysis, to overlay the results on top of a diagram and export it in high quality (7) jpeg format.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-8937144.jpeg?quality=7&amp;token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"enrichment\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-8937144.jpeg?quality=7&amp;token=</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig7.jpeg\" alt=\"Figure 7\" width=\"620\" height=\"291\" /></span></p>\r\n<p><span>In the same way, we can overlay the results of an expression analysis on top of a diagram and export it in our prefered format and quality. In case the submitted sample (for this type of analysis) includes more than one column, we can specify it by using the <i>column</i> argument, like in the following example.</span></p>\r\n<p><span> Please keep in mind that in case a column is not specified (null), the first one is selected by default. Also in case a column is not specified and the requested format is gif, then an animated image with all the columns is generated.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-432047.jpg?quality=7&amp;column=1&amp;token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"expression\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-432047.jpg?quality=7&amp;column=1&amp;token=</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig8.jpg\" alt=\"Figure 8\" width=\"620\" height=\"504\" /></span></p>\r\n<p><span>Analysis results can also be overlaid on top of our interactive illustrations.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-69278.png?token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"expression\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-69278.png?token=</code></a></pre>\r\n<p><span><img src=\"images/devguide/exporter/Fig9.png\" alt=\"Figure 9\" width=\"620\" height=\"291\" /></span></p>\r\n<h3><span>Animated GIFs</span></h3>\r\n<p><span>When you run an expression analysis and want to export an animated GIF with a frame per analysis column, set extension to gif and don’t specify column:</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-432047.gif?quality=7&amp;sel=R-ALL-879865&amp;token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"expression\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-432047.gif?quality=7&amp;sel=R-ALL-879865&amp;token=</code></a></pre>\r\n<p></p>\r\n<p><span><img src=\"images/devguide/exporter/Fig10.gif\" alt=\"Figure 10\" width=\"620\" height=\"504\" /></span></p>\r\n<p><span>In the same way, you can export animaged gifs of EHLDs</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-69278.gif?quality=7&amp;sel=R-HSA-69242&amp;token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"expression\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-69278.gif?quality=7&amp;sel=R-HSA-69242&amp;token=</code></a></pre>\r\n<p></p>\r\n<p><span><img src=\"images/devguide/exporter/Fig11.gif\" alt=\"Figure 11\" width=\"620\" height=\"504\" /></span></p>\r\n<h3><span>Color profiles</span></h3>\r\n<p><span>Reactome provides several color profiles for diagram and analysis. You can use </span><i><span>diagramProfile</span></i><span> and </span><i><span>analysisProfile</span></i><span>&nbsp;arguments to modify the default ones.</span></p>\r\n<pre><a href=\"/ContentService/exporter/diagram/R-HSA-879415.png?quality=7&amp;sel=R-ALL-87986&amp;diagramProfile=standard&amp;analysisProfile=strosobar&amp;token=\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"expression\"><code class=\"addHost\">/ContentService/exporter/diagram/R-HSA-879415.png?quality=7&amp;sel=R-ALL-879865&amp;diagramProfile=standard&amp;analysisProfile=strosobar&amp;token=</code></a></pre>\r\n<p></p>\r\n<p><span><img src=\"images/devguide/exporter/Fig12.png\" alt=\"Figure 12\" width=\"620\" height=\"357\" /></span></p>\r\n<p></p>\r\n<p></p>\r\n<h3><a name=\"arguments\"><span>Arguments</span></a></h3>\r\n<p><span>The folowing table presents a full list of the supported arguments alond with their default values.</span></p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome arguments\">\r\n<thead>\r\n<tr><th>Name</th><th>Description</th><th>Values</th><th>Default</th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p><i><span>quality</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>Quality of the image</span></p>\r\n</td>\r\n<td>\r\n<p><span>1 - 10</span></p>\r\n</td>\r\n<td>\r\n<p><span>5</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>format</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>Format of the image</span></p>\r\n</td>\r\n<td>\r\n<p><span>png, jpg, jpeg, gif</span></p>\r\n</td>\r\n<td>\r\n<p><span>png</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>flags</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>List of elements to be flagged</span></p>\r\n</td>\r\n<td>\r\n<p><span>stId, dbId, identifer, geneName</span></p>\r\n</td>\r\n<td>\r\n<p><span>null</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>selection</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>List of elements to be selected</span></p>\r\n</td>\r\n<td>\r\n<p><span>stId, dbId, identifer, geneName</span></p>\r\n</td>\r\n<td>\r\n<p><span>null</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>diagramProfile</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>Color profile for diagram</span></p>\r\n</td>\r\n<td>\r\n<p><span>modern, standard</span></p>\r\n</td>\r\n<td>\r\n<p><span>modern</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>analysisProfile</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>Color profile for analysis</span></p>\r\n</td>\r\n<td>\r\n<p><span>standard, strosobar, copper plus</span></p>\r\n</td>\r\n<td>\r\n<p><span>standard</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>token</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>Analysis token</span></p>\r\n</td>\r\n<td>\r\n<p><span>String</span></p>\r\n</td>\r\n<td>\r\n<p><span>null</span></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><i><span>column</span></i></p>\r\n</td>\r\n<td>\r\n<p><span>The specific expression analysis results column to be overlaid</span></p>\r\n<p><span>If column is not specified (null), the first one is selected.</span></p>\r\n<p><span>If column is not specified (null) and format is gif, then an animated gif is generated with all the columns.</span></p>\r\n</td>\r\n<td>\r\n<p><span>Integers</span></p>\r\n</td>\r\n<td>\r\n<p><span>null</span></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<!-- END OF ARGUMENTS TABLE-->\r\n<h3><span>&nbsp;</span></h3>\r\n<p></p>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-11-16 12:59:30','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(97,728,'Updated License Agreement','updated-license-agreement','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20171205_CC0.png\" alt=\"Creative Commons Zero - CC0\" width=\"650\" height=\"382\" /></p>\r\n<p>Since our inception, we have been an open source and open access resource, &nbsp;free for use by anyone under the terms of a Creative Commons Attribution 4.0 International (CC BY 4.0) license. This license granted parties the non-exclusive right to use, distribute and create derivative works based on Reactome, provided that the works are correctly attributed to OICR, NYUMC, EBI, and OHSU.</p>\r\n<p>In line with the growing movement to provide free open data in the public domain, and to better support the needs of our user community, we are updating the licensing agreement for some of our web content and data to reflect the <a href=\"https://creativecommons.org/share-your-work/public-domain/cc0\">Creative Commons Public Domain (CC0)</a>. CC0 is the \"no copyright reserved\" option in the Creative Commons toolkit. It effectively means relinquishing all copyright and similar rights that we hold in a work and dedicating those rights to the public domain.</p>\r\n<p>A <a href=\"https://creativecommons.org/publicdomain/zero/1.0/\">Creative Commons Public Domain (CC0 1.0 Universal) Licence</a> will now cover all Reactome annotation files, e.g. identifier mapping, specialized data files, and interaction data derived from Reactome.</p>\r\n<p>A <a href=\"https://creativecommons.org/licenses/by/4.0/\">Creative Commons Attribution 4.0 International (CC BY 4.0) Licence</a> will continue to apply to all software and code, e.g. relating to the functionality of the <a href=\"https://reactome.org\">reactome.org</a>,&nbsp;derived websites and webservices, the Curator Tool, the Functional Interaction application, SQL and Graph Database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials.</p>\r\n<p>For information about how to properly credit data use, please review the <a href=\"/license\">Reactome License</a> and the <a href=\"https://creativecommons.org/faq/\">Creative Commons FAQ</a>, or contact the <a href=\"mailto:help@reactome.org\">Reactome Helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-12-07 10:37:34','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,62,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(98,729,'Version 63 Released','version-63-released','<p><a href=\"/PathwayBrowser/#/R-HSA-73894\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-73894.svg\" alt=\"DNA repair\" width=\"650\" height=\"361\" /></a></p>\r\n<p class=\"gmail-m_-8391422641416123832gmail-MsoNormal\"><strong>New and Updated Pathways.&nbsp;</strong>In version&nbsp;V63, topics with new or revised pathways include&nbsp;Immune System&nbsp;<a href=\"PathwayBrowser/#/R-HSA-8854691\" target=\"_blank\">(Interleukin-20&nbsp; family signaling</a>,&nbsp;<a href=\"PathwayBrowser/#/R-HSA-451927&amp;SEL=R-HSA-9020958&amp;PATH=R-HSA-168256,R-HSA-1280215,R-HSA-449147\" target=\"_blank\">Interleukin-21 signaling</a>,&nbsp;<a href=\"PathwayBrowser/#/R-HSA-9008059\" target=\"_blank\">Interleukin-37 signaling</a>, and&nbsp;<a href=\"PathwayBrowser/#/R-HSA-449836\" target=\"_blank\">other Interleukin signaling</a>), Metabolism (<a href=\"PathwayBrowser/#/R-HSA-196791\" target=\"_blank\">Vitamin D (calciferol) metabolism</a>), and Signal Transduction (<a href=\"PathwayBrowser/#/R-HSA-9012852&amp;PATH=R-HSA-162582,R-HSA-157118\" target=\"_blank\">Signaling by NOTCH3</a>).&nbsp;Illustrations are now available for&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-73894\" target=\"_blank\">DNA Repair</a>,&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-73884\" target=\"_blank\">Base Excision Repair</a>,&nbsp;<a href=\"PathwayBrowser/#/R-HSA-372790&amp;PATH=R-HSA-162582\" target=\"_blank\">Signaling by GPCR</a>,&nbsp;&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-500792&amp;PATH=R-HSA-162582,R-HSA-372790\" target=\"_blank\">GPCR ligand binding</a>,&nbsp;&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-388396&amp;PATH=R-HSA-162582,R-HSA-372790\" target=\"_blank\">GPCR downstream&nbsp;signaling</a>, and&nbsp;<a href=\"PathwayBrowser/#/R-HSA-72766\" target=\"_blank\">Translation</a>.</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.&nbsp;</strong><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Laurence+Bindoff\" target=\"_blank\" rel=\"noopener\">Laurence Bindoff</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Roberta+Carriero+%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Roberta Carriero</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Sandip+K+Datta%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Sandip Datta</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Cecilia+Garlanda+%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Cecilia Garlanda</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Elzbieta+Glaser%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Elzbieta Glaser</a>,&nbsp;<a href=\"http://drholick.com/\" target=\"_blank\" rel=\"noopener\">Michael Holick</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Tony%20Kouzarides%5BAuthor%20-%20Full%5D\" target=\"_blank\" rel=\"noopener\">Tony Kouzarides</a>&nbsp;,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Alberto+Mantovani%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Alberto Mantovani</a>,&nbsp;and&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Birgit+Meldal+%5BAuthor+-+Full%5D\" target=\"_blank\" rel=\"noopener\">Birgit Meldal</a>&nbsp;are our external reviewers.</p>\r\n<p><b>Annotation Statistics.</b>&nbsp;Reactome comprises 11,426 human reactions organized into 2,179 pathways involving 10,996 proteins encoded by 10,739 different human genes, 1,764 small molecules, and 11,366 complexes. These annotations are supported by 27,694 literature references. We have projected these reactions onto 138,985 orthologous proteins, creating 20,932 orthologous pathways in 18 non-human species. Version 63 has annotations for 1,334 protein variants (mutated proteins) and their post-translationally modified forms, derived from 287 proteins. These have been used to annotate 506 disease-specific complexes and 906 disease-specific reactions organized into 453 pathways and subpathways, and tagged with 294 Disease Ontology terms.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome&nbsp;are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence,. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">,.</a>.</p>\r\n<p><strong>Follow us on Twitter:</strong>&nbsp;<a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<span id=\"cloak15d567bd0a84d9471f3b57a7075b6514\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-12-18 17:06:49','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,61,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(99,732,'Help','help','','',1,30,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-01-21 18:59:05','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"0\",\"link_titles\":\"0\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"0\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"0\",\"link_author\":\"0\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',2,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(100,733,'Pathway Diagrams Specifications','pathway-diagrams-specifications','<p>\r\n<script src=\"//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js\" type=\"text/javascript\"></script>\r\n<script type=\"text/javascript\">hljs.initHighlightingOnLoad();</script>\r\n<style type=\"text/css\">ul.tree, fieldset legend {font-family: \"Source Code Pro\", monospace;}\r\nul.tree li {list-style-type: none; position: relative;}\r\nul.tree li ul {display: none;}\r\nul.tree li.open > ul {display: block;}\r\nul.tree li a {color: black;text-decoration: none;}\r\nul.tree li a:before {height: 1em;padding: 0 .1em; font-size: .8em; display: block; position: absolute; left: -1.3em;top: .2em;}\r\nul.tree li > a:not(:last-child):before { content: \'+\';}\r\nul.tree li.open > a:not(:last-child):before {content: \'-\';}\r\n.frame { border: 1px solid #888888; background-color: #f0f0f0;}</style>\r\n</p>\r\n<h2 id=\"introduction\">Introduction</h2>\r\n<p>This document provides a set of guidelines for developers on how to properly render Reactome pathway diagrams. Pathway Diagrams represent pathways as a series of connected molecular events, known in Reactome as ‘reactions’, which can be considered as steps in the pathway. For a more detailed guide about diagrams refer to <a href=\"/userguide/pathway-browser\">our user guide</a>.</p>\r\n<p>Currently, there are two projects that follow these guidelines:</p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li>diagram: <a href=\"https://github.com/reactome-pwp/diagram\" target=\"_blank\" rel=\"noopener\"> https://github.com/reactome-pwp/diagram </a></li>\r\n<li>diagram-exporter: <a href=\"https://github.com/reactome-pwp/diagram-exporter\" target=\"_blank\" rel=\"noopener\"> https://github.com/reactome-pwp/diagram-exporter </a></li>\r\n</ul>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/apoptosis.png\" alt=\"Regulation of Apoptosis diagram\" />Regulation of Apoptosis diagram</figure>\r\n<h2 id=\"diagram-layout-and-graph-files\">Diagram layout and graph files</h2>\r\n<p>All the class diagrams can be found as UML in xmi (Umbrello) format. <a href=\"/docs/devguide/diagram/reactome-interfaces.xmi\" download=\"reactome-interfaces.xmi\">Download</a></p>\r\n<p>The layout information of every diagram is stored in a JSON file, named after its stable identifier, i.e. <em> R-HSA-12345.json </em> . This file contains all the information regarding what to draw and how to draw it, including coordinates and sizes. In addition to the layout file, Reactome provides another JSON file per diagram containing a small graph view of the diagram content, i.e. <em> R-HSA-12345.graph.json</em>. We discuss this diagram graph file in more detail further on.</p>\r\n<p><em>The general structure of a diagram <em>layout&nbsp;</em>file: </em></p>\r\n<pre><code class=\"json\">{\r\n    \"dbId\":169911,\r\n    \"stableId\":\"R-HSA-169911\",\r\n    \"displayName\":\"Diagram of Regulation of Apoptosis\",\r\n    \"disease\":false,\r\n    \"minX\":61,\r\n    \"maxX\":1126,\r\n    \"minY\":202,\r\n    \"maxY\":656,\r\n    \"nodes\":[],\r\n    \"edges\":[],\r\n    \"compartments\":[],\r\n    \"notes\":[],\r\n    \"links\":[],\r\n    \"shadows\":[]\r\n}\r\n</code></pre>\r\n<p><em>To get a more detailed description of the diagram&nbsp;<em>layout</em>, expand the following tree:</em></p>\r\n<div class=\"frame\">\r\n<ul class=\"tree\">\r\n<li class=\"parent\"><a>Diagram</a>\r\n<ul>\r\n<li class=\"parent\"><a>nodes : <strong>Node[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>nodeAttachments : <strong>NodeAttachment[]</strong></a>\r\n<ul>\r\n<li>reactomeId : <strong>Long</strong></li>\r\n<li>label : <strong>String</strong></li>\r\n<li>description : <strong>String</strong></li>\r\n<li class=\"parent\"><a>shape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>interactorsSummary : <strong>SummaryItem</strong></a>\r\n<ul>\r\n<li>pressed : <strong>Boolean</strong></li>\r\n<li class=\"parent\"><a>shape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>hit : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n<li>number : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li>trivial : <strong>Boolean</strong></li>\r\n<li class=\"parent\"><a>connectors : <strong>Connector[]</strong></a>\r\n<ul>\r\n<li>edgeId : <strong>Long</strong></li>\r\n<li class=\"parent\"><a>segments : <strong>Segment[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>to : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>from : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>endShape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>stoichiometry : <strong>Stoichiometry</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>shape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>value : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li>isDisease : <strong>Boolean</strong></li>\r\n<li>isFadeOut : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] prop : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] innerProp : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] identifier : <strong>Identifier</strong></a>\r\n<ul>\r\n<li>resource : <strong>String</strong></li>\r\n<li>id : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] textPosition : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] insets : <strong>Bound</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] bgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] fgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>NodeCommon</em>] isCrossed : <strong>Boolean</strong></li>\r\n<li>[<em>NodeCommon</em>] needDashedBorder : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>isDisease : <strong>Boolean</strong></li>\r\n<li>minX : <strong>Integer</strong></li>\r\n<li>minY : <strong>Integer</strong></li>\r\n<li>maxX : <strong>Integer</strong></li>\r\n<li>maxY : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>notes : <strong>Note[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] prop : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] innerProp : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] identifier : <strong>Identifier</strong></a>\r\n<ul>\r\n<li>resource : <strong>String</strong></li>\r\n<li>id : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] textPosition : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] insets : <strong>Bound</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] bgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] fgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>NodeCommon</em>] isCrossed : <strong>Boolean</strong></li>\r\n<li>[<em>NodeCommon</em>] needDashedBorder : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>edges : <strong>Edge[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] segments : <strong>Segment[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>to : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>from : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] endShape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] catalysts : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] inhibitors : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] activators : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>EdgeCommon</em>] precedingEvents : <strong>Long[]</strong></li>\r\n<li>[<em>EdgeCommon</em>] reactionType : <strong>String</strong></li>\r\n<li>[<em>EdgeCommon</em>] followingEvents : <strong>Long[]</strong></li>\r\n<li>[<em>EdgeCommon</em>] interactionType : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] reactionShape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] inputs : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] outputs : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>links : <strong>Link[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] segments : <strong>Segment[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>to : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>from : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] endShape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] catalysts : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] inhibitors : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] activators : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>EdgeCommon</em>] precedingEvents : <strong>Long[]</strong></li>\r\n<li>[<em>EdgeCommon</em>] reactionType : <strong>String</strong></li>\r\n<li>[<em>EdgeCommon</em>] followingEvents : <strong>Long[]</strong></li>\r\n<li>[<em>EdgeCommon</em>] interactionType : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] reactionShape : <strong>Shape</strong></a>\r\n<ul>\r\n<li>r : <strong>Double</strong></li>\r\n<li class=\"parent\"><a>b : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>a : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>c : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>s : <strong>String</strong></li>\r\n<li>r1 : <strong>Double</strong></li>\r\n<li>empty : <strong>Boolean</strong></li>\r\n<li>type : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] inputs : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>EdgeCommon</em>] outputs : <strong>ReactionPart[]</strong></a>\r\n<ul>\r\n<li>stoichiometry : <strong>Integer</strong></li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>id : <strong>Long</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>compartments : <strong>Compartment[]</strong></a>\r\n<ul>\r\n<li>componentIds : <strong>Long[]</strong></li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] prop : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] innerProp : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] identifier : <strong>Identifier</strong></a>\r\n<ul>\r\n<li>resource : <strong>String</strong></li>\r\n<li>id : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] textPosition : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] insets : <strong>Bound</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] bgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>[<em>NodeCommon</em>] fgColor : <strong>Color</strong></a>\r\n<ul>\r\n<li>r : <strong>Integer</strong></li>\r\n<li>g : <strong>Integer</strong></li>\r\n<li>b : <strong>Integer</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>NodeCommon</em>] isCrossed : <strong>Boolean</strong></li>\r\n<li>[<em>NodeCommon</em>] needDashedBorder : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>shadows : <strong>Shadow[]</strong></a>\r\n<ul>\r\n<li class=\"parent\"><a>prop : <strong>NodeProperties</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n<li>width : <strong>Double</strong></li>\r\n<li>height : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>points : <strong>Coordinate[]</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>colour : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] reactomeId : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>DiagramObject</em>] renderableClass : <strong>String</strong></li>\r\n<li class=\"parent\"><a>[<em>DiagramObject</em>] position : <strong>Coordinate</strong></a>\r\n<ul>\r\n<li>x : <strong>Double</strong></li>\r\n<li>y : <strong>Double</strong></li>\r\n</ul>\r\n</li>\r\n<li>[<em>DiagramObject</em>] isDisease : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] isFadeOut : <strong>Boolean</strong></li>\r\n<li>[<em>DiagramObject</em>] minX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] minY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxX : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] maxY : <strong>Double</strong></li>\r\n<li>[<em>DiagramObject</em>] id : <strong>Long</strong></li>\r\n<li>[<em>DiagramObject</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>dbId : <strong>Long</strong></li>\r\n<li>stableId : <strong>String</strong></li>\r\n<li>cPicture : <strong>String</strong></li>\r\n<li>forNormalDraw : <strong>Boolean</strong></li>\r\n<li>displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</div>\r\n<p>&nbsp;</p>\r\n<h2>Diagram objects</h2>\r\n<p>Diagram layout is made up of six types of elements:</p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li><strong>Compartments</strong></li>\r\n<li><strong>Nodes</strong></li>\r\n<li><strong>Edges</strong></li>\r\n<li><strong>Links</strong></li>\r\n<li><strong>Notes</strong></li>\r\n<li><strong>Shadows</strong></li>\r\n</ul>\r\n<p>These elements follow this hierarchy.</p>\r\n<p><img src=\"images/devguide/diagram/diagram_objects.png\" alt=\"Hierarchy of diagram layout objects\" width=\"620\" height=\"133\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<h3>Compartments</h3>\r\n<p><img src=\"images/devguide/diagram/compartment.png\" alt=\"Compartment\" width=\"300\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p>Compartments are rendered in the background and represent where reactions happen. They are rendered as rounded rectangles using <em> NodeCommon.prop</em>. Compartments might have an additional layer when surrounded by a membrane, which must be specified using <em> NodeCommon.innerProp. Text (<em>DiagramObject.displayName</em>) is displayed in one line, using <em> NodeCommon.textPosition.</em> Color is taken from the diagram profile sheet (<em>Profile.compartment</em>). Compartments cannot be rendered on top of any other node.</em></p>\r\n<h3>Nodes</h3>\r\n<p>Nodes represent the participants of reactions, i.e. inputs, outputs, catalysts, regulators and other pathways. The type of each node is coded into <em>renderableClass.</em> The renderable class can be one of <strong>Chemical, ChemicalDrug, Complex, Entity, EntitySet, Gene, ProcessNode, EncapsulatedNode, Protein</strong> and <strong>RNA.</strong></p>\r\n<p>Nodes are laid in 2 layers, background and foreground. The background is rendered using a different shape for each class with dimensions defined in <em> NodeCommon.prop</em>. Some nodes may have a foreground (<strong>ProcessNode</strong>, <strong>EncapsulatedNode</strong> and <strong>EntitySet</strong>). The foreground shape and colour are class specific. The text is centred to the node with a padding of 5 points to the <em>NodeCommon.prop</em>. In case it is too large, then it is wrapped in several lines and, if needed, the size of the font is reduced. When a foreground is present, the text is padded to foreground limits.</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome spec-nodes\">\r\n<thead>\r\n<tr><th scope=\"col\"><strong>Class</strong></th><th scope=\"col\"><strong>Background</strong></th><th scope=\"col\"><strong>Foreground</strong></th><th scope=\"col\"><strong>Example</strong></th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td><em> Chemical </em></td>\r\n<td>ellipse</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/chemical.png\" alt=\"Chemical\" width=\"150\" height=\"75\" title=\"Chemical\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> ChemicalDrug </em></td>\r\n<td>ellipse</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/chemicaldrug.png\" alt=\"Chemical drug\" width=\"150\" height=\"75\" title=\"Chemical drug\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> Complex </em></td>\r\n<td>edged rectangle (octagon)</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/complex.png\" alt=\"Complex\" width=\"150\" height=\"75\" title=\"Complex\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> Entity </em></td>\r\n<td>rectangle</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/entity.png\" alt=\"Entity\" width=\"150\" height=\"75\" title=\"Entity\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> EntitySet </em></td>\r\n<td>rounded rectangle</td>\r\n<td>rounded rectangle (padding = 4)</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/entityset.png\" alt=\"EntitySet\" width=\"150\" height=\"75\" title=\"EntitySet\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em>* Gene </em></td>\r\n<td>recatangle with 2 rounded corners</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/gene.png\" alt=\"Gene\" width=\"150\" height=\"75\" title=\"Gene\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> ProcessNode </em></td>\r\n<td>rectangle</td>\r\n<td>rectangle (padding = 10)</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/processnode.png\" alt=\"ProcessNode\" width=\"150\" height=\"75\" title=\"ProcessNode\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> EncapsulatedNode </em></td>\r\n<td>hexagon</td>\r\n<td>hexagon (padding = 10)</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/encapsulatednode.png\" alt=\"EncapsulatedNode\" width=\"150\" height=\"75\" title=\"EncapsulatedNode\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> Protein </em></td>\r\n<td>rounded rectangle</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/attachment.png\" alt=\"Protein with attachment\" width=\"150\" height=\"75\" title=\"Protein with attachment\" /></td>\r\n</tr>\r\n<tr>\r\n<td><em> <em> </em> RNA </em></td>\r\n<td>bone</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/rna.png\" alt=\"RNA\" width=\"150\" height=\"75\" title=\"RNA\" /></td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<p>*Genes are made up using a different approach. They have 2 shapes, a rectangle with 2 rounded corners (y=NodeCommon.prop.y + 25) and a triangle. The rectangle is filled but not stroked. Both shapes are joint using 3 perpendicular lines.</p>\r\n<pre><code class=\"python\">shape = SemiRoundedRectangle(prop.x, prop.y + 25, prop.width. prop.height)\r\narrow = Path()\r\narrow.moveTo(prop.maxX,     prop.getY + 8)\r\narrow.lineTo(prop.maxX,     prop.getY - 8)\r\narrow.lineTo(prop.maxX + 8, prop.getY)\r\narrow.closePath()\r\npath = Path()\r\npath.moveTo(prop.x,        prop.y + 25)\r\npath.lineTo(prop.maxX,     prop.y + 25)\r\npath.moveTo(prop.maxX - 4, prop.y + 25)\r\npath.lineTo(prop.maxX - 4, prop.y)\r\npath.lineTo(prop.maxX,     prop.y)  # already closed\r\nfill(shape, arrow)\r\nstroke(path, arrow)\r\n</code></pre>\r\n<p><em>RNA shape looks after a bone </em> :</p>\r\n<pre><code class=\"python\">x1 = rna.x + 16\r\nx2 = rna.maxX - 16\r\ny1 = rna.y + 8\r\ny2 = rna.maxY - 8\r\npath = Path()\r\npath.moveTo(x1, y1)\r\npath.lineTo(x2, y1)\r\npath.quadTo(rna.maxX, rna.y,    rna.maxX, rna.centerY)\r\npath.quadTo(rna.maxX, rna.maxY, x2,       y2)\r\npath.lineTo(x1, y2)\r\npath.quadTo(rna.x,    rna.maxY, rna.x,    rna.centerY)\r\npath.quadTo(rna.x,    rna.y,    x1,       y1)\r\nfill(path)\r\nstroke(path)\r\n</code></pre>\r\n<p>Some nodes contain attachments, such as post-translational modifications (PTMs) . Attachments are styled after their owner. The border of nodes is rendered after the background and foreground shapes, except for genes, which have a custom border shape. When a node has <em>NodeCommon.needDashedBorder</em> then the border must be dashed.</p>\r\n<p>Drugs use the same shape as chemicals, but with different colors. They also have a small reactangle (14x7) in the bottom right corner with the text Rx (recipe, latin word for prescription).</p>\r\n<h3>Edges</h3>\r\n<p>Edges represent reactions in the diagram. They are layout as lines with shapes. They can have 2 shapes: a <em>reaction shape</em>, to indicate the type of reaction, and an <em>end shape</em>. Both can be omitted. Lines are taken from <em>EdgeCommon.segments</em>. Shape is created using several properties inside <em>EdgeCommon</em>.</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome spec-edges\">\r\n<thead>\r\n<tr><th><strong>Reaction type</strong></th><th><strong>shape.type</strong></th><th><strong>shape.empty</strong></th><th><strong>shape.text</strong></th><th><strong>Example</strong></th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>Association</td>\r\n<td>circle</td>\r\n<td>fill</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/association.png\" alt=\"Association\" width=\"24\" title=\"Association\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Dissociation</td>\r\n<td>double circle</td>\r\n<td>empty</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/dissociation.png\" alt=\"Dissociation\" width=\"24\" title=\"Dissociation\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Omitted process</td>\r\n<td>box</td>\r\n<td>empty</td>\r\n<td>\\\\</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/omitted.png\" alt=\"Omitted\" width=\"24\" title=\"Omitted\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Transition</td>\r\n<td>box</td>\r\n<td>empty</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/transition.png\" alt=\"Transition\" width=\"24\" title=\"Transition\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Uncertain</td>\r\n<td>box</td>\r\n<td>empty</td>\r\n<td>?</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/uncertain.png\" alt=\"Uncertain\" width=\"24\" title=\"Uncertain\" /></td>\r\n</tr>\r\n<tr>\r\n<td>All (end shape)</td>\r\n<td>arrow</td>\r\n<td>fill</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/end-shape.png\" alt=\"End shape\" width=\"24\" title=\"End shape\" /></td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>Shapes are specified in the <em>Shape</em> element inside <em>Edge.reactionType</em> and <em>Edge.endShape</em>.</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome spec-shapes\">\r\n<thead>\r\n<tr><th><strong> <em> Shape.type </em> </strong></th><th><strong> Shape </strong></th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>ARROW</td>\r\n<td>triangle with points: a, b, c</td>\r\n</tr>\r\n<tr>\r\n<td>BOX</td>\r\n<td>rectangle from a (top left) to b (bottom right)</td>\r\n</tr>\r\n<tr>\r\n<td>CIRCLE</td>\r\n<td>circle with center c and radius r</td>\r\n</tr>\r\n<tr>\r\n<td>STOP</td>\r\n<td>line from a to b</td>\r\n</tr>\r\n<tr>\r\n<td>DOUBLE CIRCLE</td>\r\n<td>2 circles with center c and radii r and r1</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>When <em> Shape.s </em> is present, the value of <em>Shape.s</em> is written in the centre of the shape. When <em> Shape.emtpy </em> is true, the Shape is filled with <em> Profile.reaction.fill </em> color (usually white), otherwise, it is filled with <em> Profile.reaction.stroke </em> (usually black).</p>\r\n<h3>Connectors</h3>\r\n<p>As each reaction can have several participants, a connector is created in the participant with <em> Connector.EdgeId </em> = <em> DiagramObject.Id </em>. Connectors are styled after the reaction they belong. Connectors have segments and 2 shapes: <em>end shape</em> and <em>stoichiometry shape</em>. Stoichiometries are always empty boxes with the stoichiometry value as text only if stoichiometry value is greater than 1.</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome spec-connectors\">\r\n<thead>\r\n<tr><th><strong>endShape.type</strong></th><th><strong>Shape</strong></th><th><strong>Empty</strong></th><th><strong>Example</strong></th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>Inhibitor</td>\r\n<td>line</td>\r\n<td>fill</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/inhibitor.png\" alt=\"Inhibitor\" width=\"2\" title=\"Inhibitor\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Catalyst</td>\r\n<td>circle</td>\r\n<td>empty</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/catalyst.png\" alt=\"Catalyst\" width=\"24\" title=\"Catalyst\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Output</td>\r\n<td>arrow</td>\r\n<td>fill</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/end-shape.png\" alt=\"Output\" width=\"24\" title=\"Output\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Activator</td>\r\n<td>arrow</td>\r\n<td>empty</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/activator.png\" alt=\"Activator\" width=\"24\" title=\"Activator\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Input</td>\r\n<td><em>no shape</em></td>\r\n<td>&nbsp;</td>\r\n<td>&nbsp;</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<h3>Links</h3>\r\n<p>Links are used for linking elements which are related, normally subpathways or distant nodes.</p>\r\n<div id=\"r-responsive-table\">\r\n<table class=\"reactome spec-links\">\r\n<thead>\r\n<tr><th><strong>Renderable class</strong></th><th><strong>Dashed</strong></th><th><strong>Link type</strong></th><th><strong>Shape</strong></th><th><strong>Empty</strong></th><th><strong>Example</strong></th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>EntitySetAndMemberLink</td>\r\n<td>dashed</td>\r\n<td>&nbsp;</td>\r\n<td>&nbsp;</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/link.png\" alt=\"Link\" width=\"75\" title=\"Link\" /></td>\r\n</tr>\r\n<tr>\r\n<td>EntitySetAndEntitySetLink</td>\r\n<td>dashed</td>\r\n<td>&nbsp;</td>\r\n<td>&nbsp;</td>\r\n<td>&nbsp;</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/link.png\" alt=\"Link\" width=\"75\" title=\"Link\" /></td>\r\n</tr>\r\n<tr>\r\n<td>FlowLine</td>\r\n<td>false</td>\r\n<td>&nbsp;</td>\r\n<td>ARROW</td>\r\n<td>fill</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/flowline.png\" alt=\"Flow line\" width=\"75\" title=\"Flow Line\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Interaction</td>\r\n<td>false</td>\r\n<td>Activate</td>\r\n<td>ARROW</td>\r\n<td>empty</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/activate_inhibit.png\" alt=\"Activate\" width=\"75\" title=\"Activate\" /></td>\r\n</tr>\r\n<tr>\r\n<td>Interaction</td>\r\n<td>false</td>\r\n<td>Inhibit</td>\r\n<td>ARROW</td>\r\n<td>empty</td>\r\n<td style=\"text-align: center;\"><img src=\"images/devguide/diagram/activate_inhibit.png\" alt=\"Inhibit\" width=\"75\" title=\"Inhibit\" /></td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n<h3>Notes</h3>\r\n<p>Notes are just texts. The text is displayed in one line beginning at <em>NodeCommon.textPosition. </em> Color is taken from <em> Profile.note.text</em>.</p>\r\n<h3>Shadows</h3>\r\n<p>Shadows are used when there are several subpathways in the same diagram. In particular, these coloured rectangles highlight the specific diagram reactions belonging to each subpathway, allowing further zooming in to the areas of interest within a given pathway diagram.&nbsp;Rendering them is optional, but it is recommended that they are drawn on top of every element, as rectangles, using <em> Shadow.points </em> and <em> Shadow.color </em> for text and filling. It is also adviced to use transparency for the filling so that all subpathway participants are visible. The text contains the name of the subpathway and should be rendered using a bigger font size.&nbsp;</p>\r\n<h2 id=\"texts\">Text</h2>\r\n<p>The default font for texts is Arial black with size 9. For shadows, font size is 24.</p>\r\n<h2 id=\"colouring\">Colouring</h2>\r\n<p>Element colors are taken from a JSON stylesheet. Currently, we have 2 styles: <a href=\"https://github.com/reactome-pwp/diagram/blob/master/src/main/resources/org/reactome/web/diagram/profiles/diagram/profile_01.json\" target=\"_blank\" rel=\"noopener\"> standard </a> and <a href=\"https://github.com/reactome-pwp/diagram/blob/master/src/main/resources/org/reactome/web/diagram/profiles/diagram/profile_02.json\" target=\"_blank\" rel=\"noopener\"> modern</a>.</p>\r\n<p><em>To get a more detailed description of the diagram colour profile, expand the following tree:</em></p>\r\n<div class=\"frame\">\r\n<ul class=\"tree\">\r\n<li class=\"parent\"><a>DiagramProfile</a>\r\n<ul>\r\n<li class=\"parent\"><a>stoichiometry : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>entity : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>complex : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>note : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>interactor : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>attachment : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>flowline : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>gene : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>processnode : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>encapsulatednode : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>protein : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>link : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>otherentity : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>chemical : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>chemicaldrug : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>entityset : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>compartment : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>reaction : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>rna : <strong>DiagramProfileNode</strong></a>\r\n<ul>\r\n<li>stroke : <strong>String</strong></li>\r\n<li>lineWidth : <strong>String</strong></li>\r\n<li>fadeOutFill : <strong>String</strong></li>\r\n<li>fadeOutStroke : <strong>String</strong></li>\r\n<li>lighterFill : <strong>String</strong></li>\r\n<li>lighterStroke : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>fadeOutText : <strong>String</strong></li>\r\n<li>lighterText : <strong>String</strong></li>\r\n<li>fill : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>thumbnail : <strong>DiagramProfileThumbnail</strong></a>\r\n<ul>\r\n<li>hovering : <strong>String</strong></li>\r\n<li>highlight : <strong>String</strong></li>\r\n<li>selection : <strong>String</strong></li>\r\n<li>edge : <strong>String</strong></li>\r\n<li>node : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>name : <strong>String</strong></li>\r\n<li class=\"parent\"><a>properties : <strong>DiagramProfileProperties</strong></a>\r\n<ul>\r\n<li>hovering : <strong>String</strong></li>\r\n<li>highlight : <strong>String</strong></li>\r\n<li>selection : <strong>String</strong></li>\r\n<li>halo : <strong>String</strong></li>\r\n<li>flag : <strong>String</strong></li>\r\n<li>trigger : <strong>String</strong></li>\r\n<li>button : <strong>String</strong></li>\r\n<li>text : <strong>String</strong></li>\r\n<li>disease : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>The <strong>DiagramProfileNode</strong> defines 3 colors per each node: <em> fill, stroke </em> and <em> text </em> . If an object is fadeout (<em> DiagramObject.isFadeOut), </em> then we must choose <em>fadeOutFill, fadeOutStroke</em> and <em>fadeOutText</em>. If an analysis is being run, then we use <em>lighterFill, lighterStroke</em> and <em>lighterText</em>. And by default <em>fill, stroke</em> and <em>text</em>.</p>\r\n<p>There is 1 exception: reactions and stoichiometries’ texts; text color is ignored and stroke is used instead. When <em>DiagramObject.isDisease</em> is true, the color of node border, edge border, edge text and segments is changed to <em>Profile.properties.disease</em> (usually red).</p>\r\n<h3>Decorators</h3>\r\n<p>Some elements can be decorated in the diagram. This information is not in the diagram and has to be inserted from the outside. Reactome supports 2 types of decoration: <strong>selection</strong> and <strong>flagging</strong>.</p>\r\n<p>When a node is selected, the border is changed in 2 ways: the color is changed to <em> Profile.properties.selection</em> and the width is increased. The node, the reactions in which it participates and the nodes that participate in these reactions are haloed. Haloing is made by drawing the border behind the node with a larger width and with <em>Profile.properties.halo</em> color.</p>\r\n<p>When a reaction is selected, its segments, texts and borders change color to <em>Profile.properties.selection</em> and segments increase its width. The reaction and its participants are haloed.</p>\r\n<p>When a node is flagged, the border is drawn in the background with a larger width using <em> Profile.properties.flag.</em> At this moment, edges are not being flagged.</p>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/node-layers.png\" alt=\"Node layers\" title=\"Node layers\" /> <figcaption>Representation of layers for a node. From bottom to top: flag, halo, background, analysis, foreground, border and text.</figcaption></figure>\r\n<p>&nbsp;</p>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/edge-layers.png\" alt=\"Edge layers\" title=\"Edge layers\" /> <figcaption>Edge layers. From bottom to top: halo, segments, filling, border and text.</figcaption></figure>\r\n<h2>Analysis</h2>\r\n<p>When an analysis is overlaid, we must modify the color of elements to <em>lighter</em>. Then, those nodes hit by the analysis must render a new layer between the background and the foreground. At first, we must calculate how much a node is hit by the analysis. We can do it by using the graph JSON file and the analysis results (or the analysis token). We must traverse the graph from each node through its children (<em>EntityNode.children) </em>.</p>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/analysis_hits.png\" alt=\"C1 has 1/2 children hit. S1 has 3/4 leaves hit. P1, P3 and P4 have 1/1 leaves hit.\" /> <figcaption>C1 has 1/2 children hit. S1 has 3/4 leaves hit. P1, P3 and P4 have 1/1 leaves hit.</figcaption></figure>\r\n<p>&nbsp;</p>\r\n<p><em>To get a more detailed description of the diagram graph, expand the following tree:</em></p>\r\n<div class=\"frame\">\r\n<ul class=\"tree\">\r\n<li class=\"parent\"><a>Graph</a>\r\n<ul>\r\n<li class=\"parent\"><a>nodes : <strong>EntityNode[]</strong></a>\r\n<ul>\r\n<li>identifier : <strong>String</strong></li>\r\n<li>parents : <strong>Long[]</strong></li>\r\n<li>children : <strong>Long[]</strong></li>\r\n<li>geneNames : <strong>String[]</strong></li>\r\n<li>diagramIds : <strong>Long[]</strong></li>\r\n<li>[<em>GraphNode</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>GraphNode</em>] dbId : <strong>Long</strong></li>\r\n<li>[<em>GraphNode</em>] stId : <strong>String</strong></li>\r\n<li>[<em>GraphNode</em>] speciesID : <strong>Long</strong></li>\r\n<li>[<em>GraphNode</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>edges : <strong>EventNode[]</strong></a>\r\n<ul>\r\n<li>catalysts : <strong>Long[]</strong></li>\r\n<li>inhibitors : <strong>Long[]</strong></li>\r\n<li>activators : <strong>Long[]</strong></li>\r\n<li>inputs : <strong>Long[]</strong></li>\r\n<li>outputs : <strong>Long[]</strong></li>\r\n<li>diagramIds : <strong>Long[]</strong></li>\r\n<li>preceding : <strong>Long[]</strong></li>\r\n<li>following : <strong>Long[]</strong></li>\r\n<li>requirements : <strong>Long[]</strong></li>\r\n<li>[<em>GraphNode</em>] schemaClass : <strong>String</strong></li>\r\n<li>[<em>GraphNode</em>] dbId : <strong>Long</strong></li>\r\n<li>[<em>GraphNode</em>] stId : <strong>String</strong></li>\r\n<li>[<em>GraphNode</em>] speciesID : <strong>Long</strong></li>\r\n<li>[<em>GraphNode</em>] displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li>dbId : <strong>Long</strong></li>\r\n<li>stId : <strong>String</strong></li>\r\n<li class=\"parent\"><a>subpathways : <strong>SubpathwayNode[]</strong></a>\r\n<ul>\r\n<li>dbId : <strong>Long</strong></li>\r\n<li>stId : <strong>String</strong></li>\r\n<li>events : <strong>Long[]</strong></li>\r\n<li>displayName : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>ProcessNodes percentages are taken from the <em>SubpathwaySummary</em>: entity.found / entity.total. For the analysis, there is another stylesheet. We draw enrichment using the <em>AnalysisSheet.enrichment.gradient.max</em> color. There are 3 analysis stylesheets: <a href=\"https://github.com/reactome-pwp/diagram/blob/master/src/main/resources/org/reactome/web/diagram/profiles/analysis/profile_01.json\" target=\"_blank\" rel=\"noopener\"> Standard</a>, <a href=\"https://github.com/reactome-pwp/diagram/blob/master/src/main/resources/org/reactome/web/diagram/profiles/analysis/profile_02.json\" target=\"_blank\" rel=\"noopener\"> Copper Plus</a> and <a href=\"https://github.com/reactome-pwp/diagram/blob/master/src/main/resources/org/reactome/web/diagram/profiles/analysis/profile_03.json\" target=\"_blank\" rel=\"noopener\"> Strosobar</a>.&nbsp;</p>\r\n<p><em>To get a more detailed description of the analysis colour profile, expand the following tree:</em></p>\r\n<div class=\"frame\">\r\n<ul class=\"tree\">\r\n<li class=\"parent\"><a>AnalysisProfile</a>\r\n<ul>\r\n<li class=\"parent\"><a>enrichment : <strong>OverlayNode</strong></a>\r\n<ul>\r\n<li>text : <strong>String</strong></li>\r\n<li class=\"parent\"><a>gradient : <strong>ProfileGradient</strong></a>\r\n<ul>\r\n<li>stop : <strong>String</strong></li>\r\n<li>min : <strong>String</strong></li>\r\n<li>max : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>legend : <strong>OverlayLegend</strong></a>\r\n<ul>\r\n<li>hover : <strong>String</strong></li>\r\n<li>median : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>expression : <strong>OverlayNode</strong></a>\r\n<ul>\r\n<li>text : <strong>String</strong></li>\r\n<li class=\"parent\"><a>gradient : <strong>ProfileGradient</strong></a>\r\n<ul>\r\n<li>stop : <strong>String</strong></li>\r\n<li>min : <strong>String</strong></li>\r\n<li>max : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n<li class=\"parent\"><a>legend : <strong>OverlayLegend</strong></a>\r\n<ul>\r\n<li>hover : <strong>String</strong></li>\r\n<li>median : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</li>\r\n<li>ribbon : <strong>String</strong></li>\r\n<li>name : <strong>String</strong></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n</div>\r\n<p>&nbsp;</p>\r\n<p>If the analysis is an expression analysis, we must take the individual expression values from the leaves. We can show only 1 column at a time. If we want to show them all we must create an animation (play button in the pathway browser and GIFs in the diagram exporter). Once we have the expression values for an element, we divide it into regions and use <em>AnalysisSheet.expression.gradient</em> to calculate the color of each region, using <em>AnalysisResult.expression.max</em> and <em>AnalysisResult.expression.min</em> as limit values. Leaves are sorted using the identifier from the analysis (<em>FoundEntity.id</em>).</p>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/analysis.png\" alt=\"Example of diagram with analysis enrichment\" /> <figcaption>Subsection of a diagram with an analysis overlaid</figcaption></figure>\r\n<figure class=\"text-center\"><img src=\"images/devguide/diagram/expression.gif\" alt=\"Expression\" title=\"Expression\" /> <figcaption>Example of a diagram with an expression analysis</figcaption></figure>\r\n<h2 id=\"elements-order\">Elements order</h2>\r\n<p>From bottom to top, elements must be rendered in the following order:</p>\r\n<ol>\r\n<li>Compartments</li>\r\n<li>Fade out reactions</li>\r\n<li>Fade out nodes</li>\r\n<li>Flags</li>\r\n<li>Halos</li>\r\n<li>Reactions</li>\r\n<li>Nodes</li>\r\n<li>Notes</li>\r\n<li>Shadows</li>\r\n</ol>\r\n<h2 id=\"more-info\">More Information</h2>\r\n<p>To learn more about our pathway diagrams and the techniques we use to render them efficiently please go through our relevant publications:</p>\r\n<ol>\r\n<li><a href=\"https://doi.org/10.1093/bioinformatics/btx752\" target=\"_blank\" rel=\"noopener\">Reactome diagram viewer: data structures and strategies to boost performance</a></li>\r\n<li><a href=\"https://doi.org/10.1093/bioinformatics/btx441\" target=\"_blank\" rel=\"noopener\">Reactome enhanced pathway visualization</a></li>\r\n</ol>\r\n<p>\r\n<script type=\"text/javascript\">var tree = document.querySelectorAll(\'ul.tree a:not(:last-child)\');\r\nfor (var i = 0; i < tree.length; i++) {\r\n    tree[i].addEventListener(\'click\', function (e) {\r\n            var parent = e.target.parentElement;\r\n            while (parent.nodeName !== \"LI\")\r\n                parent = parent.parentElement;\r\n            var classList = parent.classList;\r\n            if (classList.contains(\"open\")) {\r\n                classList.remove(\'open\');\r\n                var opensubs = parent.querySelectorAll(\':scope .open\');\r\n                for (var i = 0; i < opensubs.length; i++) {\r\n                    opensubs[i].classList.remove(\'open\');\r\n                }\r\n            } else {\r\n                classList.add(\'open\');\r\n            }\r\n        }\r\n    );\r\n}</script>\r\n</p>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-02-05 11:47:57','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(102,736,'Version 64 Released','version-64-released','<p><a href=\"/PathwayBrowser/#/R-HSA-8963743\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-8963743.svg\" alt=\"Digest Absorp\" width=\"650\" height=\"408\" /></a></p>\r\n<p><strong>New and Updated Pathways.&nbsp;</strong>In version&nbsp;V64, topics with new or revised pathways include Signal Transduction (<a href=\"PathwayBrowser/#/R-HSA-8939211\">ESR-mediated signaling</a> and <a href=\"PathwayBrowser/#/R-HSA-9006115\">Signaling by NTRK2</a>), Metabolism (<a href=\"PathwayBrowser/#/R-HSA-9018678\">Biosynthesis of specialized proresolving mediators (SPMs)</a>), and Metabolism of proteins (<a href=\"PathwayBrowser/#/R-HSA-9033241&amp;PATH=R-HSA-392499\">Peroxisomal protein import</a>).</p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Hanna+Antila\">Hanna Antila</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Azevedo%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28787099\">Jorge Azevedo</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Kumar%20Belani%5BAuthor%20-%20Full%5D\">Kumar Belani</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Gerry%20Boss%5BAuthor%20-%20Full%5D\">Gerry Boss</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Matthew+Brenner%5BAuthor+-+Full%5D\">Matthew Brenner</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Eero+Castr%C3%A9n%5BAuthor+-+Full%5D\">Eero Castrén</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Arthur+Cooper%5BAuthor+-+Full%5D\">Arthur Cooper</a><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Diana+Downs%5BAuthor+-+Full%5D\">, Diana Downs</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Marc+Fransen%5BAuthor+-+Full%5D\">Marc Fransen</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Trond%20Hansen%5BAuthor%20-%20Full%5D\">Trond Hansen</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Hui-Chih+Hung%5BAuthor+-+Full%5D\">Hui-Chih Hung</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Margaret+James%5BAuthor+-+Full%5D\">Margaret James</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Pidder+Jansen-Duerr%5BAuthor+-+Full%5D\">Pidder Jansen-Duerr</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Hideo+Kimura%5BAuthor+-+Full%5D\">Hideo Kimura</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Luca+Magnani%5BAuthor+-+Full%5D\">Luca Magnani</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Steven+Patterson%5BAuthor+-+Full%5D\">Steven Patterson</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Paul++Van+Veldhoven%5BAuthor+-+Full%5D\">Paul&nbsp; Van Veldhoven</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Alexander+Weiss%5BAuthor+-+Full%5D\">Alexander Weiss</a>, and <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Herman+Wolosker+%5BAuthor+-+Full%5D\">Herman Wolosker</a> are our external reviewers.</p>\r\n<p>Illustrations with embedded navigation features are now available for&nbsp;<a href=\"PathwayBrowser/#/R-HSA-8963743\">Digestion and absorption</a>, <a href=\"PathwayBrowser/#/R-HSA-5693532\">DNA Double-Strand Break Repair</a>, <a href=\"PathwayBrowser/#DIAGRAM=9006934&amp;PATH=162582\">Signaling by receptor tyrosine kinases</a>, <a href=\"PathwayBrowser/#/R-HSA-190236&amp;PATH=R-HSA-162582,R-HSA-9006934\">Signaling by FGFR</a>, <a href=\"PathwayBrowser/#/R-HSA-5683057\">MAPK family signaling cascades</a>, <a href=\"PathwayBrowser/#/R-HSA-5683057\">MAPK1/MAPK3 signaling</a>, <a href=\"PathwayBrowser/#/R-HSA-9006936&amp;PATH=R-HSA-162582\">Signaling by TGF-beta family</a>, <a href=\"PathwayBrowser/#/R-HSA-195258&amp;PATH=R-HSA-162582,R-HSA-194315\">RHO GTPase Effectors</a>, <a href=\"PathwayBrowser/#/R-HSA-195721\">Signaling by Wnt</a>, <a href=\"PathwayBrowser/#/R-HSA-5358351\">Signaling by Hedgehog</a>, <a href=\"PathwayBrowser/#/R-HSA-73887\">Death Receptor Signaling</a>, and <a href=\"PathwayBrowser/#DIAGRAM=9006925&amp;PATH=162582\">Intracellular signaling by second messengers</a>.</p>\r\n<p><b>Annotation Statistics.</b>&nbsp;Reactome comprises 11,754 human reactions organized into 2,216 pathways involving 11,030 proteins encoded by 10,762 different human genes, 1,867 small molecules, and 11,561 complexes. These annotations are supported by 28,254 literature references. We have projected these reactions onto 140,720 orthologous proteins, creating 21,223 orthologous pathways in 18 non-human species. Version 64 has annotations for 1,339 protein variants (mutated proteins) and their post-translationally modified forms, derived from 289 proteins,&nbsp;which have been used to annotate disease-specific complexes,&nbsp;reactions and pathways.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome&nbsp;are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence,&nbsp;A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<span id=\"cloak15d567bd0a84d9471f3b57a7075b6514\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-03-20 20:00:21','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,60,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(103,739,'Version 65 Released','version-65-released','<div><a href=\"/PathwayBrowser/#/R-HSA-71387\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-71387.svg\" alt=\"carbohydrate\" width=\"650\" height=\"359\" /></a></div>\r\n<div>\r\n<p><strong>New and Updated Pathways. </strong><span style=\"font-family: arial, helvetica, sans-serif;\"> In version&nbsp;V65, topics with new or revised pathways include<span style=\"color: #222222;\">&nbsp;Immune System (</span><a href=\"/PathwayBrowser/#/R-HSA-8985947&amp;PATH=R-HSA-168256,R-HSA-1280215,R-HSA-449147,R-HSA-451927\" target=\"_blank\">Interleukin-9 signaling</a>)&nbsp;<span style=\"color: #222222;\"></span><span style=\"color: #222222;\">and Signal Transduction (</span><a href=\"/PathwayBrowser/#/R-HSA-9013694&amp;PATH=R-HSA-162582,R-HSA-157118\" target=\"_blank\">Signaling by NOTCH4</a><span style=\"color: #222222;\">&nbsp;and&nbsp;</span><a href=\"/PathwayBrowser/#/R-HSA-9034015&amp;PATH=R-HSA-162582,R-HSA-9006934,R-HSA-166520\" target=\"_blank\">Signaling by NTRK3 (TRKC)</a><span style=\"color: #222222;\">).&nbsp;</span></span><span style=\"font-family: arial, helvetica, sans-serif;\">Illustrations with embedded navigation features are now available for&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-71387&amp;PATH=R-HSA-1430728\" target=\"_blank\">Carbohydrate Metabolism</a>&nbsp;and&nbsp;<a href=\"/PathwayBrowser/#/R-HSA-5693538&amp;PATH=R-HSA-73894,R-HSA-5693532&amp;DTAB=AN\" target=\"_blank\">Homology Directed Repair</a>.</span></p>\r\n<p><strong>Thanks&nbsp;to our Contributors.</strong>&nbsp;<span style=\"font-family: arial, helvetica, sans-serif;\"><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Azevedo%20JE[Author]&amp;cauthor=true&amp;cauthor_uid=26235645\">Jorge Azevedo</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Kumar+G+Belani[Author+-+Full]\"></a><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=gomez-outes\">Antonio Gómez-Outes</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Jan%20Haavik%5BAuthor%20-%20Full%5D\">Jan Haavik</a>,&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Eun-Kyeong%20Jo%5BAuthor%20-%20Full%5D\">Eun-Kyeong Jo</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=hideo+kimura%5BAuthor+-+Full%5D\"></a><a href=\"https://www.ifc.unam.mx/investigadores/paula-licona\">Paula Licona-Limon</a>, <a href=\"http://www.biochem.utah.edu/rutter/\">Jared Rutter</a>, and&nbsp;<a href=\"https://www.scopus.com/authid/detail.uri?authorId=6604050486\">Pantelis Tsoulfas</a>, are our external reviewers</span><span style=\"font-family: arial, helvetica, sans-serif;\">.</span><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Hanna+Antila\"><br /></a></p>\r\n<p><b>Annotation Statistics.</b>&nbsp;Reactome comprises 11,896 human reactions organized into 2,222 pathways involving 10,935 proteins encoded by 10,763 different human genes, 1,880 small molecules, and 11,674 complexes. These annotations are supported by 28,436 literature references. We have projected these reactions onto 159,163 orthologous proteins, creating 23,450 orthologous pathways in 18 non-human species. Version 65 has annotations for 1,339 protein variants (mutated proteins) and their post-translationally modified forms, derived from 289 proteins,&nbsp;which have been used to annotate disease-specific complexes,&nbsp;reactions and pathways.</p>\r\n<p><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome&nbsp;are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence,&nbsp;A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p><strong>Follow us on Twitter:&nbsp;</strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information.</strong>&nbsp;Please contact our&nbsp;<span id=\"cloak15d567bd0a84d9471f3b57a7075b6514\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span>.</p>\r\n<p></p>\r\n</div>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-06-06 20:19:48','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,59,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(104,741,'Searching Reactome - New Search','searching-reactome-2','','',0,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2017-08-22 12:50:12','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',6,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(105,742,'Our data now available through Google Dataset Search','our-data-now-available-through-google-dataset-search-engine','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Google_Data_Search.jpg\" alt=\"Google Data Search\" width=\"650\" height=\"382\" /></p>\r\n<p></p>\r\n<p>We are pleased to announce that we are amongst the early adopters of the new <a href=\"https://toolbox.google.com/datasetsearch\">Google Dataset Search</a>.</p>\r\n<p>Throughout the world, there are many thousands of open access data repositories hosted by public institutions, research projects, local and national governments, not-for-profit organisations, and many others. These online resources provide a variety of datasets, which are available in different formats and support many data standards. The Google Dataset Search engine enables easy access to these datasets, so that researchers,&nbsp;scientists, data journalists, data wranglers, or anyone else can quickly find the data for their work.</p>\r\n<p>An example of how Reactome pathway data can be viewed in the Google DataSet Search is available at this<a href=\"https://toolbox.google.com/datasetsearch/search?query=glycosylation\">&nbsp;link</a>.</p>\r\n<p></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-09-06 17:49:49','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,58,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(106,743,'Version 66 Released','version-66-release','<p style=\"text-align: justify;\"><a href=\"/PathwayBrowser/#/R-HSA-9609507\"><span style=\"font-family: arial, helvetica, sans-serif;\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"/download/current/ehld/R-HSA-9609507.svg\" alt=\"protein localization\" width=\"650\" height=\"408\" /></span></a></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"><strong>New and Updated Pathways. </strong>In version 66, topics with new or revised pathways includeDisease (<a href=\"/PathwayBrowser/#/R-HSA-9005891\">Loss of function of MECP2 in Rett syndrome </a>and <a href=\"/PathwayBrowser/#/R-HSA-9605310\">Defective Base Excision Repair Associated with MUTYH</a>), Gene Expression (<a href=\"/PathwayBrowser/#/R-HSA-8986944\">Transcriptional regulation by MECP2</a>), Immune Response (<a href=\"/PathwayBrowser/#/R-HSA-8983711\">OAS antiviral response</a>), Metabolism of proteins (<a href=\"/PathwayBrowser/#/R-HSA-4655427\">SUMOylation of DNA methylation proteins</a>, <a href=\"/PathwayBrowser/#/R-HSA-4755510\">SUMOylation of immune response proteins</a>, <a href=\"/PathwayBrowser/#/R-HSA-4090294\">SUMOylation of intracellular receptors</a>, <a href=\"/PathwayBrowser/#/R-HSA-4085377\">SUMOylation of SUMOylation proteins</a>, <a href=\"/PathwayBrowser/#/R-HSA-3232118\">SUMOylation of transcription cofactors</a>, and <a href=\"/PathwayBrowser/#/R-HSA-3232142\">SUMOylation of ubiquitinylation proteins</a>), and Signal Transduction (<a href=\"/PathwayBrowser/#/R-HSA-9006335\">Signaling by Erythropoietin</a>).</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"><strong>Thanks to our Contributors.</strong> <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=John+Christodoulou[Author]\">John Christodoulou</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Rahul+Krishnaraj[Author]\">Rahul Krishnaraj</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=McGraw%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30072146\">Kathy L McGraw</a>, <a href=\"https://www.researchgate.net/profile/Marco_Meraz-Rios\">Marco Meras-Rios</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Nakabeppu%20Y[Author]&amp;cauthor=true&amp;cauthor_uid=29027306\">Yusaku Nakabeppu</a>, <a href=\"https://www.ncbi.nlm.nih.gov_pubmed_-3Fterm-3DEinari-2B-2BNiskanen-255BAuthor-255D&amp;d=DwMFaQ&amp;c=j5oPpO0eBH1iio48DtsedbOBGmuw5jHLjgvtN2r4ehE&amp;r=GTiX74hn1lHmDPIJuP2ff67KO0GU_fIQLdlnOo0L9MI&amp;m=_wG9tOrPx9pey5oeCeKklHayWUchukMItCF_csf6fP0&amp;s=Pwq0mh72qIyR6p0LxhZFdII2QSkDXxyy6y8xqvMncm4&amp;e=\">Einari Niskanen</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Silverman%20RH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26858407\">Robert H Silverman</a>, and <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=J%C3%BCrgen+Wienands%5BAuthor%5D\">Jürgen Wienands </a>are our external reviewers.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\">New Ilustrations. Illustrations with embedded navigation features are now available for <a href=\"/PathwayBrowser/#/R-HSA-69306\">DNA replication </a>and <a href=\"/PathwayBrowser/#/R-HSA-9609507\">Protein localization</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"><strong>Annotation Statistics.</strong> Reactome comprises 12,047 human reactions organized into 2,244 pathways involving 11,049 proteins encoded by 10,870 different human genes, 1,948 small molecules, and 11,823 complexes. New in this release are annotations of the functions of 139 drugs, 3 of them proteins and 136 small molecules. These annotations are supported by 28,829 literature references. We have projected these reactions onto 139,072 orthologous proteins, creating 21,283 orthologous pathways in 18 non-human species. Version 66 has annotations for 1,391 protein variants (mutated proteins) and their post-translationally modified forms, derived from 293 proteins, which have been used to annotate disease-specific complexes, reactions and pathways.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactomeare distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the <a href=\"/\">Reactome website</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt;\"><strong>Follow us on Twitter: </strong><a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a><strong>&nbsp;</strong>to get frequent updates about new and updated pathways, feature updates, and more!</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt;\"><span style=\"font-family: arial, helvetica, sans-serif;\"><strong>For more information. </strong>Please contact our</span> <span id=\"cloak15d567bd0a84d9471f3b57a7075b6514\"><span id=\"cloakc08bbc767a3f8ec0f8a519cfb552e5e3\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</span></span></span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-09-27 06:25:28','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":\"\",\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":\"\",\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":\"\",\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,57,'','',1,0,'{\"robots\":\"index, follow\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(107,745,'New tool to generate analysis PDF reports','new-tool-to-generate-analysis-pdf-reports','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Analysis_Report_PDF.png\" alt=\"Analysis Report PDF\" width=\"650\" height=\"217\" /></p>\r\n<p>We now offer a new tool to download your analysis results as a single PDF file. This report contains a genome-wide overview, statistics for the most significant pathways and, for each significant pathway, it includes the corresponding diagram image with the analysis overlaid, a summation of the pathway, the related bibliography and the list of identifiers found.</p>\r\n<p>When performing pathway analysis through our <a href=\"/PathwayBrowser/\">PathwayBrowser</a>, you can download the report by clicking on the “Report (PDF)” button on the bottom-left corner of the Analysis tab. This feature is also available when implementing programmatic access to our <a href=\"/AnalysisService/\">AnalysisService</a> through <a href=\"/AnalysisService/#/report/generatePdfReportUsingGET\">this method</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-07-12 10:51:41','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,56,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(109,747,'New advanced search available in our PathwayBrowser','new-advanced-search-available-in-our-pathway-browser','<p>\r\n<video class=\"responsive-vid\" style=\"display: block; margin-left: auto; margin-right: auto;\" controls=\"controls\" width=\"650\" height=\"auto\" controlslist=\"nodownload\" type=\"video/mpeg\"> Your browser does not support the video tag. <source src=\"images/news/SearchInDiagram.mp4\" type=\"video/mp4\" /></video>\r\n</p>\r\n<p>&nbsp;</p>\r\n<p style=\"text-align: justify;\">Our <a href=\"/PathwayBrowser/\">PathwayBrowser</a> now features advanced search capabilities powered by Solr to allow finding content throughout the whole knowledgebase. The user interface has been improved adapting to the findings of our last UX testing. The search within the Diagram Viewer widget enables users to define the scope of their search either limiting it to the content of the displayed diagram or expanding it to cover all pathways allowing&nbsp;our users to perform a search against all content without having to go the main search.</p>\r\n<p style=\"text-align: justify;\">This new feature has also been enabled in our <a href=\"/dev/diagram/js\">diagram</a> and <a href=\"/dev/pathways-overview/js\">pathways overview</a> widgets so third party web applications can already take advantage of it, allowing users to search Reactome content without abandoning the page they are in.</p>\r\n<p>The new search features:</p>\r\n<ol>\r\n<li>Suggestions based on the introduced term.</li>\r\n<li>Listing the most recent searches.</li>\r\n<li>Scoping results to either the displayed diagram or the whole database.</li>\r\n<li>Filtering results by one or more entity types (i.e. Proteins, Chemical compounds, Reactions, etc.)</li>\r\n<li>Flagging a given entity to persist its highligthing.</li>\r\n</ol>\r\n<p>To learn more, please check the&nbsp;<a href=\"/userguide/searching\">Searching Reactome</a>&nbsp;section&nbsp;in our <a href=\"/userguide\">User Guide</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-01 09:05:45','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,55,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(110,748,'SBGN files revamp','sbgn-files-revamp','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20181004_SBGN_Revamp.png\" alt=\"SBGN Revamp\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">The <a href=\"http://sbgn.github.io/sbgn/\">Systems Biology Graphical Notation (SBGN)</a> project is an effort to standardise the graphical notation used in maps of biological processes. It aims to communicate biological knowledge more efficiently and accurately between different research communities in the life sciences.</p>\r\n<p style=\"text-align: justify;\">Following this commitment, we’ve recently revamped our SBGN export methods and tools to provide more accurate representation of our pathway diagrams. Users have the option to either export a pathway diagram in SBGN through the <a href=\"/PathwayBrowser/\">PathwayBrowser</a>, or get <a href=\"/download/current/homo_sapiens.sbgn.tar.gz\">all human pathway diagrams</a> from our <a href=\"/download-data\">downloads section</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-03 12:53:19','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,54,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(111,749,'International Open Access Week: Analysis Tools','international-open-access-week-analysis-tools','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20181022_OpenAccess_week_-_Analysis_Service.png\" alt=\"International Open Access week (Analysis Tools)\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">Following up on the International <a href=\"http://www.openaccessweek.org/\">Open Access week</a>,&nbsp;we would like to remind our users that all our tools, including the <a href=\"/user/guide/analysis\">analysis</a>, are open access.</p>\r\n<p style=\"text-align: justify;\">Pathway analysis methods have a broad range of applications in physiological and biomedical research. Our analysis suite currently implements an overrepresentation analysis, an expression data analysis and a species comparison tool. Using this service, users can submit their sample (list of identifiers) to get as result the most significants pathways. Results are overlaid in the different modules of our Pathway Browser and can be exported to different formats including a <a href=\"/about/news/107-new-tool-to-generate-analysis-pdf-reports\">PDF report</a>.</p>\r\n<p style=\"text-align: justify;\">A light-weight client is integrated in our&nbsp;<a href=\"/PathwayBrowser/#TOOL=AT\">Pathway Browser</a>. The tool suite is available via a <a href=\"/AnalysisService/\">RESTFul Web Service</a>&nbsp;so all the available analysis tools can be easily integrated into third party software. More documentation for developers is available at our <a href=\"/dev/analysis\">developer\'s zone</a><a href=\"https://reactome.org/dev/analysis.\">.</a></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-23 08:25:02','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,53,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(112,750,'International Open Access Week: Network Visualization & Analysis Tool','international-open-access-week-network-analysis-tools','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20181024_OpenAccess_week_-_ReactomeFIViz_2.png\" alt=\"20181024 OpenAccess week ReactomeFIViz 2\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">As part of the International&nbsp;<a href=\"http://www.openaccessweek.org/\">Open Access week</a>,&nbsp;we would like to talk about another one of our open access network visualization and analysis tool – the <a href=\"/tools/reactome-fiviz\">Reactome FIViz app</a>.</p>\r\n<p style=\"text-align: justify;\">In order to improve our understanding of disease mechanisms and develop better personalized precision therapies for patients, many biological and clinical studies employ high-throughput techniques that generate large-scale data sets. Typically, these data sets are gene- or protein-based, and to better understand the relationships among interesting genes or proteins, researchers usually have to project them onto biological network contexts to provide holistic visualization and analysis platform for reducing the dimensionality of data using network modules.</p>\r\n<p style=\"text-align: justify;\">To assist our users who would like to perform network-based analysis, we have constructed the Reactome Functional Interaction (FI) network which, covers 60% of the total human protein-coding genes, and was created by extracting interactions from manually curated pathways and predicting interactions based on a machine learning technique. We have developed the&nbsp;<a href=\"https://cytoscape.org/\">Cytoscape </a>application (or app), called the “ReactomeFIViz” that uses this highly reliable FI network to support network-based data visualization and analysis. Users of our app can construct an FI subnetwork for a list of genes, perform network clustering to find network modules, annotate the subnetwork and modules, and perform survival analysis for network modules. Furthermore, the app can also perform pathway enrichment analysis using a gene score file, and pathway mathematical modeling based on probabilistic graphical models and Boolean networks. More documentation about the ReactomeFIViz app is available through our <a href=\"/tools/reactome-fiviz\">User Guide</a>&nbsp;and the <a href=\"http://apps.cytoscape.org/apps/reactomefiplugin\">Cytoscape App Store</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-23 16:54:11','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,52,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(113,751,'International Open Access Week: Interactive illustrations and icon library','oaweek-interactive-illustrations-icons','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20181025_OpenAccess_week_-_EHLD_-_Icon_Library.png\" alt=\"OpenAccessWeek Textbook-like illustrations and icon library\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">Textbook-like illustrations aim to improve the graphical representation of higher-level pathways in the Reactome events hierarchy, e.g., “<a href=\"/PathwayBrowser/#/R-HSA-162582\">signal transduction</a>”, “<a href=\"/PathwayBrowser/#/R-HSA-109581\">apoptosis</a>”, or “<a href=\"/PathwayBrowser/#/R-HSA-392499\">metabolism of proteins</a>” whose pathway diagrams consisted of green boxes labeled with the names of sub-events, optionally located in cellular compartments and connected by arrows. These green-box diagrams feature limited navigation: clicking on a green box takes the user to that sub-event.</p>\r\n<p style=\"text-align: justify;\">There was a general agreement that green-box diagrams are not that appealing, and put off users accustomed to textbook-quality illustrations of biological processes with striking, intuitively clear iconography. The project includes generation of scalable vector graphic (SVG) versions of illustrations, and a diagram module that makes these images interactive by enabling actions such as hovering over the items or selecting them to show the associated content in the details panel.</p>\r\n<p style=\"text-align: justify;\">Developing this project has also driven the creation of the <a href=\"/icon-lib\">Icon Library</a>; a consistent iconography compendium that ranges from simple protein labels to representations of organelles, receptors and cell types. The library has now been integrated in the main search as well as in the pages of their associated entities such as proteins or chemicals. Icons can be found by their name, description, designer and/or contributor.</p>\r\n<p style=\"text-align: justify;\">The Icon Library is freely accessible (under a <a href=\"https://creativecommons.org/licenses/by/4.0/deed.ast\">CC-BY 4.0 licence</a>) and it is suitable for a broad range of purposes, from schematic pathway sketches in scientific presentations and publications to grant proposal illustrations. As the library was created to be a community resource, the invitation for third parties to contribute is still open. Aiming to achieve technical and artistic consistency, detailed guidelines are provided at <a href=\"https://reactome.org/icon-info.\">https://reactome.org/icon-info.</a> To acknowledge the community engagement, each icon is attributed to the author through a metadata file linking to a portfolio and/or ORCID id. As of September 2018, the library has considerably grown to 1,150 components.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-25 08:19:18','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,51,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(114,752,'International Open Access Week: GraphDB and Content Service','graphdb-content-service','<p style=\"text-align: justify;\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20181026_OpenAccess_week_-_GraphDB_ContentService.png\" alt=\"OpenAccess Week: GraphDB and ContentService\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">Since version 57 we provide our data in a <a href=\"https://neo4j.com/\">Neo4j</a>&nbsp;<a href=\"/dev/graph-database\">graph database</a>&nbsp;helping to reduce the complexity of the represented knowledgebase and allowing a more straightforward access to our content. Neo4j’s query language, Cypher, allows queries to be written in a more intuitive way and reduces the average response time per query by 93% (<a href=\"https://doi.org/10.1371/journal.pcbi.1005968\">Fabregat et al., 2018</a>).</p>\r\n<p style=\"text-align: justify;\">The graph database also benefits Reactome in other aspects like (i) the creation of complex data quality assessment (QA) queries or (ii) supporting software that requires data pattern analysis, e.g. <a href=\"https://github.com/reactome/rxn-classifier\">reactions classification</a>. QA queries are executed during each quarterly release to identify instances that need to be checked out and corrected by Reactome curators ensuring that high-quality content is delivered to the final user. The reactions classifier project, for example, uses Cypher to formalise the concepts presented in <a href=\"https://doi.org/10.1093/database/bau060\">Jupe et al. (2014)</a> to generate a series of reports that help curators classify the reactions in Reactome.</p>\r\n<p style=\"text-align: justify;\">The <a href=\"https://reactome.org/dev/content-service\">Content Service</a>&nbsp;constitutes an easy API, based on the Representational State Transfer (REST) protocol, that provides access to the Reactome knowledgebase. It includes a set of methods classified in groups according to their functionality. For instance, expanding the pathways group reveals a set of methods that provide specific information about pathways such as the contained Events or the participating PhysicalEntities.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-10-26 08:21:29','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,50,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(115,753,'Performing GSEA analysis with ReactomeFIViz app','performing-gsea-analysis-with-reactomefiviz-app','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/ReactomeFIViz_GSEA.png\" alt=\"ReactomeFIViz GSEA\" width=\"650\" height=\"366\" /></p>\r\n<p class=\"p1\" style=\"text-align: justify;\">ReactomeFIViz is a Cytoscape app built upon Reactome pathways to help users perform pathway- and network-based data analysis and visualization. One of the most popular approaches to pathway analysis, which is an alternative to the traditional gene-list based pathway enrichment, is Gene Set Enrichment Analysis (GSEA). GSEA considers all genes with their scores based on a weighted Kolmogorov–Smirnov-like test and is a&nbsp;representative approach of second-generation pathway analysis. ReactomeFIViz implements features to perform GSEA analysis using Reactome pathways for a gene score file.&nbsp;</p>\r\n<p class=\"p1\" style=\"text-align: justify;\">More details about the GSEA feature and the ReactomeFIViz app can be found <a href=\"https://reactome.org/tools/reactome-fiviz\">here</a></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-11-01 14:00:04','0000-00-00 00:00:00','{}','{}','{}',1,49,'','',1,0,'{}',0,'en-GB','',''),(116,754,'Version 67 Released','version-67-released','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/ER_to_Golgi.png\" alt=\"ER to Golgi\" width=\"650\" height=\"374\" /></p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Pathways.&nbsp;</strong>In version&nbsp;V67, topics with new or revised pathways include Gene Expression (<a href=\"/PathwayBrowser/#R-HSA-9614085\">FOXO-mediated transcription</a>), Immune Response (<a href=\"/PathwayBrowser/#/R-HSA-1222556\">ROS, RNS production in phagocytes</a>), Neuronal System (<a href=\"/PathwayBrowser/#/R-HSA-442755\">Activation of NMDA receptors and postsynaptic events</a>), Programmed Cell Death (<a href=\"/PathwayBrowser/#/R-HSA-111471\">Apoptotic factor-mediated response</a>), and Signal Transduction (<a href=\"/PathwayBrowser/#/R-HSA-388396\">GPCR downstream signaling</a> and <a href=\"/PathwayBrowser/#/R-HSA-5668599\">RHO GTPase activate NADPH oxidases</a>).</p>\r\n<p style=\"text-align: justify;\"><strong>New Illustrations.</strong> Illustrations with embedded navigation features are now available for <a href=\"/PathwayBrowser/#/R-HSA-199977\">ER to Golgi Anterograde Transport</a>, <a href=\"/PathwayBrowser/#/R-HSA-9036866\">Expression and Processing of Neurotrophins</a>, and <a href=\"/PathwayBrowser/#/R-HSA-166520\">Signaling by NTRKs</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors.</strong>&nbsp;<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Enrico+Bertaggia%5BAuthor%5D\">Enrico&nbsp; Bertaggia</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Timothy+Donlon%5BAuthor%5D\">Timothy Donlon</a>, <a href=\"https://www.ncbi.nlm.nih.gov/myncbi/browse/collection/40443499/?sort=date&amp;direction=descending\">Kasper Hansen</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Oliver+N%C3%BCsse+%5BAuthor%5D\">Oliver Nüsse</a>, and <a href=\"https://scholar.google.com/citations?user=OyPga9EAAAAJ&amp;hl=en\">Feng Yi</a> are our external reviewers.</p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong> Reactome comprises 12,788 human reactions organized into 2,256 pathways involving 11,066 proteins and modified forms of proteins encoded by 10,792 different human genes, 1,827 small molecules, and 155 drugs. These annotations are supported by 29,454 literature references. We have projected these reactions onto 139,298 orthologous proteins, creating 21,374 orthologous pathways in 18 non-human species. Version 67 has annotations for 1,564 protein variants (mutated proteins) and their post-translationally modified forms, derived from 299 proteins. These have been used to annotate 506 complexes and 962 disease-specific reactions organized into 467 pathways and subpathways, and tagged with 342 Disease Ontology terms.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the&nbsp;<a href=\"/\">Reactome website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong>&nbsp;<a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a>&nbsp;to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information.&nbsp;</strong>Please contact our&nbsp;<span id=\"cloak15d567bd0a84d9471f3b57a7075b6514\"><span id=\"cloakc08bbc767a3f8ec0f8a519cfb552e5e3\"><span id=\"cloak8926181d9df5dd2e3fcde3c0c30adf60\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span>.</span></span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-12-10 20:18:07','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,48,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(117,755,'Icon Mapping','icon-mapping','<p ><span>The mapping files consist of a tab-separated table that indicates the external molecule or ontology identifiers in the source database that were mapped to elements of the icon library. Our goal with distributing different identifier type is to provide accurate representation of the different icon library components, e.g. proteins, chemicals, tissues, cell, etc. These mapping files link the source database identifier to:</span></p>\r\n<p ><span>The columns within the mappings files follow a similar format:</span></p>\r\n<ol>\r\n<li><span>Source database identifier, e.g. UniProt, PFAM, Uberon, or ChEBI identifier</span></li>\r\n<li><span>Icon Stable Identifier</span></li>\r\n<li><span>Icon Name</span></li>\r\n</ol>\r\n<p><span>If you have any further questions, please&nbsp;<a href=\"mailto:help@reactome.org?subject=PE%20Identifier%20Mapping%20File\"><span>contact us</span></a>.</span></p>','',1,32,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-12-11 03:32:58','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(118,756,'Icons Mapping Download','icons-mapping-download','<ul>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/BTO2Icon.txt\">BTO to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/CHEBI2Icon.txt\">ChEBI to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/CL2Icon.txt\">CL to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/COMPLEXPORTAL2Icon.txt\">COMPLEX PORTAL to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/DOID2Icon.txt\">DOID to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/ENA2Icon.txt\">ENA to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/ENSEMBL2Icon.txt\">ENSEMBL to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/GO2Icon.txt\">GO to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/INTERPRO2Icon.txt\">INTERPRO to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/KEGG2Icon.txt\">KEGG to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/MESH2Icon.txt\">MESH to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/NCBI2Icon.txt\">NCBI to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/NCIT2Icon.txt\">NCIT to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/NP2Icon.txt\">NP to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/OMIT2Icon.txt\">OMIT to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/OPL2Icon.txt\">OPL to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/PFAM2Icon.txt\">PFAM to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/PUBCHEM2Icon.txt\">PUBCHEM to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/RFAM2Icon.txt\">RFAM to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/SO2Icon.txt\">SO to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/UBERON2Icon.txt\">UBERON to icons</a></li>\r\n<li><i class=\"fa fa-file-text-o\"></i><a href=\"/download/current/icons/UNIPROT2Icon.txt\">UNIPROT to icons</a></li>\r\n</ul>','',1,32,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2018-12-12 11:43:47','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(119,760,'New reactions layout with nested compartments','new-reactions-layout-with-nested-compartments','<p style=\"text-align: justify;\"><a href=\"/content/detail/R-HSA-164834\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/logo/20190109_Reaction_preview.png\" alt=\"Reactions automatic layout with nested compartments\" width=\"650\" height=\"382\" /></a></p>\r\n<p style=\"text-align: justify;\">Up until now, we have placed existing regular pathway diagrams and <a href=\"https://doi.org/10.1093/bioinformatics/btx441\" target=\"_blank\" rel=\"noopener\">textbook-style illustrations</a>&nbsp;in our pathway detailed pages (e.g. <a title=\"Open &quot;Hemostasis&quot;\" href=\"/content/detail/R-HSA-109582\" target=\"_blank\">Hemostasis</a>, <a title=\"Open &quot;Platelet Adhesion to exposed collagen&quot;\" href=\"/content/detail/R-HSA-75892\" target=\"_blank\">Platelet Adhesion to exposed collagen</a>), leaving single reactions as the only type of events without a self-contained image.&nbsp;To fill the gap, an automatic algorithm to deterministically lay reactions out has been developed. It uses data directly from our <a href=\"/dev/graph-database\">graph-database</a> to generate images without any kind of human intervention.</p>\r\n<p style=\"text-align: justify;\">Long story short, the algorithm’s strategy places inputs on the left, outputs on the right, catalysts on top and regulators at the bottom. Each element is placed in its corresponding compartment, and these are nested following the&nbsp;<a href=\"https://doi.org/10.1038/75556\" target=\"_blank\" rel=\"noopener\">Gene Ontology hierarchy</a>. The algorithm supports normal and disease* reactions minimizing the space as well as avoiding lines to cross unnecessary elements in the display.</p>\r\n<p style=\"text-align: justify;\">Finally, the reaction visualisation complies with the <a href=\"https://doi.org/10.1038/nbt.1558\" target=\"_blank\" rel=\"noopener\">Systems Biology Graphical Notation (SBGN)</a>.</p>\r\n<p style=\"text-align: justify;\">Some examples:</p>\r\n<ol style=\"text-align: justify;\">\r\n<li style=\"text-align: justify;\"><a title=\"Open &quot;Enzyme-bound ATP is released&quot;\" href=\"/content/detail/R-HSA-164834\" target=\"_blank\">Enzyme-bound ATP is released</a></li>\r\n<li style=\"text-align: justify;\"><a title=\"Open &quot;VCP-catalyzed ATP hydrolysis promotes the translocation of Hh-C into the cytosol&quot;\" href=\"/content/detail/R-HSA-5362459\" target=\"_blank\">VCP-catalyzed ATP hydrolysis promotes the translocation of Hh-C into the cytosol</a></li>\r\n<li style=\"text-align: justify;\"><a title=\"Open &quot;Ubiquitination of PAK-2p34&quot;\" href=\"/content/detail/R-HSA-211734\" target=\"_blank\">Ubiquitination of PAK-2p34</a></li>\r\n<li><a title=\"Open &quot;Defective ABCD1 does not transfer LCFAs from cytosol to peroxisomal matrix&quot;\" href=\"/content/detail/R-HSA-5684043\" target=\"_blank\">Defective ABCD1 does not transfer LCFAs from cytosol to peroxisomal matrix</a></li>\r\n</ol>\r\n<p style=\"text-align: justify;\">*<a href=\"/user/guide/diseases\" target=\"_blank\">Disease reactions</a> can either be&nbsp;classified as infectious, gain-of-function or lost-of-function. In the latter category some participants have to be&nbsp;crossed out.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-01-09 13:52:43','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,47,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(120,761,'New tool to classify pathways into tissue types based on protein expression','classifying-pathways-into-tissue-types-based-on-protein-expression','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20190114_Tissue_distribution.png\" alt=\"Tissue distribution analysis\" width=\"650\" height=\"382\" /></p>\r\n<p style=\"text-align: justify;\">Pathways in Reactome are curated in a generic cell and are agnostic to tissue types. However, different cell types have different functional requirements and consequently the underlying pathway activities also vary. Studying pathways in a tissue-specific manner will help to understand the biology in context.</p>\r\n<p style=\"text-align: justify;\">Reactome now boasts a <a href=\"/PathwayBrowser/#TOOL=AT\">tool</a> that can categorize pathways into different tissue types based on protein expression. We overlay tissue-specific protein expression data from the <a href=\"https://www.ebi.ac.uk/gxa/home\">ExpressionAtlas</a> database on empirically validated pathway information in Reactome. This facilitates the sorting of pathways from a generic cell to different tissue-types. This new feature allows users to select an experiment and analyse Reactome pathways in different tissues.</p>\r\n<p style=\"text-align: justify;\">To try it out, select the \"Tissue Distribution\" tab, choose the tissues of your interest and click the \"Go\" button. The results are overlaid in the pathways overview and pathway diagrams similar to other analysis types. Users can cycle through the selected tissues via the small control panel displayed at the bottom of the viewport.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-01-15 11:02:06','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,46,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(121,762,'Download our biological pathway knowledge in a PDF book','new-pdf-book','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20190130_Reactome_book.png\" alt=\"Reactome book\" width=\"650\" height=\"383\" /></p>\r\n<p style=\"text-align: justify;\">Aiming to make our biological pathway knowledge more accessible, we\'ve been offering for years the option to download the complete content of our knowledgebase as a PDF book. Following users feedback and requests, this book has now been updated to include yet more content in a freshly redesigned look and feel. Organised in 27 volumes, one per each top level pathway, the new version of the book includes our textbook-like lilustrations as well as regular-pathway&nbsp;and single-reaction diagrams.</p>\r\n<p style=\"text-align: justify;\">In some cases the preferred option might be downloading part of Reactome\'s content. To do so we\'ve included this option in our detail pages, the PathwayBrowser and the DiagramViewer.&nbsp;When downloaded from the PathwayBrowser, the resulting PDF will include the user\'s predefined colour profiles (diagram and analysis overlay) and will also include the analysis results when available. In all cases, resulting documents&nbsp;are fully navigable, including links to following and preceding events or any of the parent pathways in the hierarchy. Literarutre references include links to PubMed and every event has a link to its details page in Reactome\'s site.</p>\r\n<p style=\"text-align: justify;\">The full book can be downloaded from our <a href=\"/download-data\">download</a> section. For an example of the on-demand download please visit the&nbsp;<a href=\"/content/detail/R-HSA-109582\">Hemostasis details page</a>&nbsp;or <a href=\"/PathwayBrowser/#/R-HSA-109582\">its view in the PathwayBrowser</a>.&nbsp;All the above can be programmatically generated using&nbsp;<a href=\"/ContentService/#/exporter/eventPdfUsingGET\">this method</a>&nbsp;from our <a href=\"/ContentService\">ContentService</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-01-30 15:26:26','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,45,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(124,765,'Deprecated support for Reactome RESTful API','deprecated-support-for-reactome-restful-api','<p>As of Version 68 (March 2019), the RESTful API is deprecated and will be superceded by our ContentService. The ContentService is currently available through our website at <a href=\"https://reactome.org/ContentService/\">https://reactome.org/ContentService/</a> and is designed for bioinformaticians, computer scientists, and software developers to access our pathway data. See more details about this API at <a href=\"https://reactome.org/dev/content-service.\">https://reactome.org/dev/content-service. </a></p>\r\n<p>If you have questions or concerns regarding this announcement, please email <a href=\"mailto:help@reactome.org.\">help@reactome.org.</a></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-03-21 16:12:39','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,44,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(125,766,'Version 68 Released','version-68-of-reactome-is-now-online','<p><img src=\"images/news/Signaling_by_Nuclear_Receptors.png\" alt=\"Signaling by Nuclear Receptors\" width=\"650\" height=\"384\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Pathways. </strong>In version V68, topics with new or revised pathways include Disease (<a href=\"/PathwayBrowser/#/R-HSA-9616333\">Defective Base Excision Repair Associated with NTHL1</a>, <a href=\"/PathwayBrowser/#/R-HSA-9616334\">Defective Base Excision Repair Associated with NEIL1</a>, and <a href=\"/PathwayBrowser/#/R-HSA-9629232\">Defective Base Excision Repair Associated with NEIL3</a>), Metabolism (<a href=\"/PathwayBrowser/#/R-HSA-9033658\">Blood group systems biosynthesis</a>), Neuronal System (<a href=\"/PathwayBrowser/#/R-HSA-9609736\">Assembly and cell surface presentation of NMDA receptors</a>), Protein localization (<a href=\"/PathwayBrowser/#/R-HSA-9603798\">Class I peroxisomal membrane protein import</a> and <a href=\"/PathwayBrowser/#/R-HSA-9609523\">Insertion of tail-anchored proteins into the endoplasmic reticulum membrane</a>), and Signal Transduction (<a href=\"/PathwayBrowser/#/R-HSA-9009391\">Non-genomic estrogen signaling</a>).</p>\r\n<p style=\"text-align: justify;\"><strong>New Illustrations.</strong> Illustrations with embedded navigation features are now available for <a href=\"/PathwayBrowser/#/R-HSA-9006927&amp;PATH=R-HSA-162582&amp;DTAB=AN\">Signaling by Non-Receptor Tyrosine Kinases</a>, <a href=\"/PathwayBrowser/#/R-HSA-9006931&amp;PATH=R-HSA-162582&amp;DTAB=AN\">Signaling by Nuclear Receptors</a>, and <a href=\"/PathwayBrowser/#/R-HSA-3108232&amp;PATH=R-HSA-392499,R-HSA-597592,R-HSA-2990846\">SUMO E3 ligases SUMOylate target proteins</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors.</strong> <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Bhattacharya%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30479490\">Subhrajit Bhattacharya</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Chad+R+Camp%5BAuthor%5D\">Chad R Camp</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Richarda+de+Voer%5BAuthor%5D\">Richarda de Voer</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=evelina+de+laurentiis%5Bauthor%5D\">Evelina DeLaurentis</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Paul+W+Doetsch%5BAuthor%5D\">Paul W Doetsch</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Akos+Farkas%5BAuthor+-+Full%5D\">Ákos Farkas</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=marc+fransen%5BAuthor+-+Full%5D\">Marc Fransen</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Roland++Kuiper%5BAuthor%5D\">Roland Kuiper</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Ellis%20R%20Levin%5BAuthor%20-%20Full%5D\">Ellis R Levin</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Taei%20%20Matsui%5BAuthor%20-%20Full%5D\">Taei Matsui</a>, <a href=\"https://scholar.google.com/citations?user=Aj651BgAAAAJ&amp;hl=en\">Huaiyu Mi</a>, <a href=\"https://www.ncbi.nlm.nih.gov/sites/myncbi/1XouqVfFZvAke/bibliography/44215904/public/?sort=date&amp;direction=ascending\">Barbara Rivera</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Sampath%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30291284\">Harini Sampath</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Blanche+Schwappach%5BAuthor+-+Full%5D\">Blanche Schwappach</a>, <a href=\"https://orcid.org/0000-0002-4650-631X\">Ralf Stephan</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Stephen++F+Traynelis%5BAuthor+-+Full%5D\">Stephen F Traynelis</a>, and <a href=\"https://www.ncbi.nlm.nih.gov/myncbi/browse/collection/54269459/\">Jia Zhou</a> are our external reviewers.</p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong> Reactome comprises 12,416 human reactions organized into 2,255 pathways involving 11,000 proteins and modified forms of proteins encoded by 10,825 different human genes, 1,854 small molecules, and 202 drugs. These annotations are supported by 29,885 literature references. We have projected these reactions onto 81,875 orthologous proteins, creating 18,505 orthologous pathways in 15 non-human species. Version 68 has annotations for 1,416 protein variants (mutated proteins) and their post-translationally modified forms, derived from 304 proteins. These have been used to annotate 532 complexes and 966 disease-specific reactions organized into 472 pathways and subpathways, and tagged with 349 Disease Ontology terms.</p>\r\n<p style=\"text-align: justify;\"><strong>Tools and Data. </strong>Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"/PathwayBrowser/#/TOOL=AT\">analyze</a> your own dataset, to <a href=\"/download-data\">download</a>, and to access programmatically through our <a href=\"/ContentService/\">Content</a> and <a href=\"/AnalysisService/\">Analysis</a> Services. The <a href=\"/tools/reactome-fiviz\">ReactomeFIViz app</a> for Cytoscape provides tools to find pathways and network patterns related to cancer and other types of diseases.</p>\r\n<p style=\"text-align: justify;\"><strong>Documentation and Training</strong>. Visit our online <a href=\"/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"/community/training\">here</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"/\">website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong> <a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information</strong>: If you have a question to ask or would like to give us your feedback, please contact our <a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-03-21 16:13:28','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,43,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(126,768,'PANTHER resource now used to identify orthologs of curated human proteins','panther-resource-now-used-to-identify-orthologs-of-curated-human-proteins','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Orthology.png\" alt=\"Orthology\" width=\"650\" height=\"366\" /></p>\r\n<p style=\"text-align: justify;\">Biological processes are remarkably well-conserved over large evolutionary distances. At a practical level, this fact enables the extrapolation of mechanistic insights from species to species. The Reactome project aims to annotate the molecular details of a broad range of human biological processes based on experimental data from human systems. To link these human annotations at a molecular level to their conserved counterparts in model organism systems we use protein sequence orthology relationships to ask, for each human reaction and each model organism, whether the human proteins involved in the reaction have orthologs in the model organism. If the orthologs exist, we computationally infer the corresponding reaction for the model organism and in this way build up a predicted pathway knowledgebase for the organism. If the human protein functions as part of a complex, we search for orthologs of all components of the complex, and computationally infer the existence of the complex in the model organism if model organism counterparts of at least 75% of the human proteins are found.</p>\r\n<p style=\"text-align: justify;\">With our March, 2019 (version 68) release, we have made two changes to improve the quality and usability of these inferred pathways.</p>\r\n<p style=\"text-align: justify;\">First, with the development of Plant Reactome, a substantial body of rice (Oryza sativa) pathways annotated from plant experimental evidence is now available online. That material improves on orthology-based inferences from human data, and provides a better starting point for making such inferences to other plant species, so all inferences for plant species will now be generated, maintained, and made available through <a href=\"http://plantreactome.gramene.org/index.php?lang=en\">Plant Reactome</a>.</p>\r\n<p style=\"text-align: justify;\">Second, we are now using the <a href=\"http://pantherdb.org\">PANTHER</a> resource (Protein Analysis Through Evolutionary Relationships) to identify model organism orthologs of human proteins annotated in Reactome. This change will allow us to exploit features of PANTHER, such as the identification of least-diverged orthologs in model organism protein families, to improve the specificity of our inferences. The change is also part of a larger project to better align the Reactome with the Gene Ontology, PANTHER, the Alliance of Genome Resources, and related resources to generate interactive, expert-curated, actively maintained, and tightly integrated community genomics resources.</p>\r\n<p></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-03-21 16:14:15','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,42,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(127,769,'Come to our release 68 webinar to learn more about Reactome','come-to-our-release-68-webinar-to-learn-more-about-reactome','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Webinar.png\" alt=\"Webinar\" width=\"650\" height=\"366\" /></p>\r\n<p style=\"text-align: justify;\">We are hosting a webinar aimed at new Reactome users who would like to get an idea of our data and tools and at existing Reactome users who would like to learn about the latest Reactome updates. Reactome is a&nbsp;freely available curated database of human biological pathways and reactions, which is updated on a regular basis. Release 68 features new and updated pathway and reaction&nbsp;annotations for 16 species, as well as a suite of tools for data analysis and visualization.</p>\r\n<p style=\"text-align: justify;\">Event Date: 5th April 2019.<br />Event Time: 11.00 a.m. to 12.00 p.m. EDT.<br />Event Contact: Robin Haw at <a href=\"mailto:help@reactome.org\">Help&nbsp;Mailing List</a>.</p>\r\n<p style=\"text-align: justify;\">Requisites: This webinar will be hosted using <a href=\"https://www.webex.ca/en/\">Cisco WebEx</a>. You will need a computer, an Internet connection and a telephone/Skype connection/microphone to join an online session.</p>\r\n<p style=\"text-align: justify;\">Please register at&nbsp;<a href=\"https://forms.gle/QzeG85hcfrX34ivUA\">https://forms.gle/QzeG85hcfrX34ivUA</a></p>\r\n<p style=\"text-align: justify;\">When you register, please make sure to provide your name and valid email address so that we can send you an email with the webinar details and instructions.</p>\r\n<p style=\"text-align: justify;\">Feel free to pass this invitation along to colleagues who may benefit from learning about this valuable resource.</p>\r\n<p style=\"text-align: justify;\"></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-03-28 18:08:26','0000-00-00 00:00:00','{}','{}','{}',1,41,'','',1,0,'{}',0,'en-GB','',''),(129,773,'EHLD Specs & Guidelines','ehld-guidelines','<p><span>The Enhanced High Level Diagrams (EHLD) project aims to improve the graphical representation of higher-level pathways in the Reactome events hierarchy, e.g., “signal transduction”, “apoptosis”, or “metabolism” whose pathway diagrams consist of green boxes labeled with the names of sub-events, optionally located in cellular compartments and connected by arrows. These green-box diagrams feature limited navigation: clicking on a green box takes the user to that sub-event. In some cases, an illustrator has also created a graphic representation of the event. However, these illustrator diagrams offer no navigation functionality.</span></p>\r\n<p><span>There was a general agreement that green-box diagrams are not that appealing, and put off users accustomed to textbook-quality illustrations of biological processes with striking, intuitively clear iconography. Meanwhile, while our illustrations often are up to textbook standards, they are static images. The project included generation of scalable vector graphic (SVG) versions of illustrations, and the development of software that will make these SVG images navigable in the way that the green-box figures are now, and also to provide user interactivity by progressive zoom in/out, actions when hovering over the items or selecting them and showing the associated content in the details panel for the selected items.</span></p>\r\n<p><span>This project has also driven the development of a consistent iconography, a controlled visual vocabulary for biological processes like the controlled word vocabularies we already have for names of physical entities and reactions.</span></p>\r\n<p><span>This document includes two main sections. The first section describes in detail the process that should be followed to create an EHLD. The second section of this document provides guide through the process of creating and including new graphic elements into the Reactome EHLD icons library. </span></p>\r\n<h3><span>Enhanced High Level Diagrams guidelines</span></h3>\r\n<p><span>This section aims to guide you through the process of creating an EHLD. The following paragraphs aim to summarise the set of requirements and guidelines the illustrator should follow in order to produce EHLDs that are compatible with Reactome Pathway Browser v3.4.</span></p>\r\n<h4><span>Generic guidelines</span></h4>\r\n<p><span>In Reactome, an EHLD is an interactive graphical representation of a higher level pathway diagram, typically containing two or more subpathways as active regions. An active region refers to a group of shapes representing a single subpathway. In particular, users can interact with active regions by hovering or selecting them, as well as use them to navigate to the subpathways they represent.</span></p>\r\n<p><span> &nbsp;</span><span><img src=\"images/ehldguide/ehld-with-analysis.png\" alt=\"ehld with analysis\" width=\"602\" height=\"386\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></span></p>\r\n<p style=\"text-align: center;\"><em>EHLD representing the Hemostasis branch of the Reactome event hierarchy.&nbsp;(a) Pathway hierarchy view for Hemostasis. (b) Hemostasis as an EHLD representation. (c) Hemostasis EHLD overlaid with pathway enrichment analysis results.</em></p>\r\n<h3>Generic guidelines</h3>\r\n<p><span>To make all these possible, there is a set of initial requirements to be taken into account during the generation of the EHLDs.</span></p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li><span>EHLDs must be in SVG format. Illustrators are free to use the design tool of their preference but at the end their diagram must be exported to SVG.</span></li>\r\n<li><span>The use of raster graphics is strongly discouraged as it results in larger file sizes and it has a negative impact on software performance. Also, by including bitmap images we cancel out resolution-independent zooming, which is one of SVG’s main advantage. &nbsp;</span></li>\r\n<li><span>All exported SVG files should include the styles in an internal CSS stylesheet. Please make use of this option when exporting from Adobe Illustrator: </span><i><span>Export As -&gt; SVG -&gt; Styling: Internal CSS</span></i><span>. If you are using another application to create the EHLDs, you will need to configure it appropriately.</span></li>\r\n<li><span>The name of each EHLD file has to be the Reactome Identifier of the corresponding pathway diagram, followed by “.svg”. For example, the </span><i><span>Apoptosis</span></i><span> EHLD should be named as </span><i><span>R-HSA-109581.svg</span></i><span>. Please keep in mind that only one EHLD should be created per high level pathway diagram.</span></li>\r\n<li><span>Active regions are annotated using the id attribute of their group element (inside the SVG file) and can be classified in the following two groups: </span>\r\n<ul style=\"list-style-type: circle;\">\r\n<li><span>Regions containing a group of irregular shapes (including text) that can be selected (and hovered) in order to navigate to the respective pathway diagram. These regions should be annotated by setting their id attribute to “</span><i><span>REGION-</span></i><span>” </span><i><span>+ Reactome Identifier</span></i><span> of their represented subpathway, e.g. “</span><i><span>REGION-R-HSA-109581</span></i><span>”. This type of active regions may include arrows pointing to or from them and they are not mandatory.</span></li>\r\n<li><span>Regions containing the label of the represented subpathway in a box-like shape that can be not only selected but also overlaid with the results of the analysis. These regions should be annotated by setting their id attribute to “</span><i><span>OVERLAY-</span></i><span>” </span><i><span>+ Reactome Identifier</span></i><span> of their represented subpathway, e.g. “</span><i><span>OVERLAY-R-HSA-109581</span></i><span>”. Regions of this category must not include arrows pointing to or from them, and they are mandatory.</span></li>\r\n<li><span>Please keep in mind that if we need to annotate both types of active regions for a given subpathway, the region (“REGION-R-HSA-109581”) has to include the overlay component (“OVERLAY-R-HSA-109581”). In other words, the group of the selectable shapes has to include the group of shapes that can be overlaid.</span></li>\r\n</ul>\r\n</li>\r\n<li><span>Any group of shapes, text or graphical elements that are not annotated as active regions are considered decorators and users cannot interact with them. They have pure esthetic value and they cannot be selected, hovered or overlaid. &nbsp;&nbsp;</span></li>\r\n<li><span>If possible, no background covering the whole image should be put in place. Please note that compartments or other necessary backgrounds can be kept, but we suggest to avoid “unnecessary” backgrounds. In case a background is required, but it is too big to fit in the illustration, it is suggested to draw only a small part of it with clear and well defined boundaries (See the blood vessel in the EHLD of Hemostasis above). </span></li>\r\n<li><span>Text elements should not be converted to groups of shapes. Instead, use the normal &lt;text&gt; SVG element.</span></li>\r\n</ul>\r\n<p><span><img src=\"images/ehldguide/subpathways.png\" alt=\"subpathways\" width=\"407\" height=\"272\" /></span></p>\r\n<p><span>Example illustrating the two types of active regions. Selectable regions (annotated with “REGION-” + Reactome Identifier) are highlighted in red, while regions that can be overlaid are highlighted in brown. Shapes and text that are not annotated as active regions are considered decorators (presented in blue) and users cannot interact with them.</span></p>\r\n<p>&nbsp;</p>\r\n<p><span>Taking the aforementioned requirements into consideration, the EHLD of the figure above should have the following structure:</span></p>\r\n<p>&nbsp;</p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li><span>REGION-Pathway A </span><i><span># The whole pathway will be selectable #</span></i>\r\n<ul style=\"list-style-type: circle;\">\r\n<li><span>OVERLAY-Pathway A </span><i><span># Only the label will be overlaid #</span></i></li>\r\n</ul>\r\n</li>\r\n<li><span>REGION-Pathway B </span><i><span># The whole pathway will be selectable including the arrow #</span></i>\r\n<ul style=\"list-style-type: circle;\">\r\n<li><span>OVERLAY-Pathway B </span><i><span># Only the label will be overlaid #</span></i></li>\r\n</ul>\r\n</li>\r\n<li><span>REGION-Pathway C </span><i><span># The whole pathway will be selectable including the arrow #</span></i>\r\n<ul style=\"list-style-type: circle;\">\r\n<li><span>OVERLAY-Pathway C </span><i><span># Only the label will be overlaid #</span></i></li>\r\n</ul>\r\n</li>\r\n<li><span>OVERLAY-Pathway D &nbsp;</span><i><span>&nbsp;# The whole pathway will be selectable and overlaid #</span></i></li>\r\n<li><span>DECORATORS </span><i><span># None of the shapes are selectable or can be overlaid #</span></i></li>\r\n</ul>\r\n<h3>Creating an EHLD step by step</h3>\r\n<p><span>We suggest starting a new EHLD by creating an Adobe Illustrator document with </span><b>1366px</b><span> in width and </span><b>768px</b><span> in height and using the </span><b>RGB colour mode</b><span>.</span></p>\r\n<p><span>Pathway Labels should have </span><b>centered,</b> <b>white</b><span>, </span><b>uppercased</b><span>, text using </span><b>Arial Bold</b><span> font at </span><b>12pt</b><span> inside a </span><b>rounded rectangle</b><span> with </span><b>8pt</b> <b>radius</b><span>, minimum </span><b>width of 170px</b><span> and </span><b>height</b><span> of</span><b> 30px </b><span>for single lines or </span><b>43px</b><span> for double lines. The colour of the rounded rectangle should be </span><b>#0F82BC</b><span> (R:15 G:130 B:188). We suggest writing the name of the subpathway it represents as it appears in the Reactome hierarchy. In case of a discrepancy, or any kind of oddity, please contact the author or the curator related to the pathway.</span></p>\r\n<p><span>Please use the same specifications for any other written element except for:</span></p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li><b>Descriptions</b><span> of a process should always be black and in lowercase letters.</span></li>\r\n<li><span>Any </span><b>written element</b><span> that belongs to a compound, protein or cell (generically inside of any coloured shape) should be white and in uppercase letters (when possible).</span></li>\r\n<li><span>Any </span><b>written element</b><span> that belongs to a receptor and/or describes a cell element or a process (generically outside of any shape) should be black and in lowercase letters (when possible, usually receptors are named by acronyms and capital letters should be used in these cases).</span></li>\r\n<li><span>Greek or other Unicode characters must be avoided in text elements. Thus, names with greek characters should include the whole word (e.g. alpha instead of α) written in lowercase.</span></li>\r\n<li><span>Reactome </span><b>logo</b><span> should always have </span><b>50%</b> <b>opacity</b><span>.</span></li>\r\n</ul>\r\n<p><span>Every pathway label should have one </span><b>Analysis Information Label</b><span>. The latter is used to display additional details about the hit elements and the false discovery rate (FDR). All analysis information labels should be annotated as “ANALINFO” in the Layer Hierarchy Panel. Place “XXX/YYY” as a text placeholder inside it. Text should be centered and uppercased in </span><b>Arial Bold</b> <b>9pt</b><span>, and in white colour. The box should be a rounded rectangle with </span><b>8px radius</b><span>,</span> <span>minimum </span><b>width of 170px</b><span> and </span><b>20px height</b><span>. Its fill colour should be </span><b>#C6C6C6</b><span> (R:198 G:198 B:198). Also, set the opacity of the group to 0%, but make sure you leave the opacity of the shapes and text inside the group to 100%. Each analysis information label should be placed beside its pathway label and in the opposite side of the content for that pathway. For instance, if the pathway label is above its content, the analysis information label should be placed above the pathway label and vice versa.</span></p>\r\n<p><span>In order to better shape the active region (or the clickable area) of a pathway we can add a white shape with </span><b>0% opacity</b><span> under any element of the Pathway in the Layer Hierarchy Panel. In this way we can extent the clickable area and improve the overall user experience while navigating the EHLDs. In particular, we should try to surround the elements of the subpathway covering the gaps within the individual graphic elements, and thus, avoiding the inconvenient situation where the mouse cursor changes from the usual arrow shape to the hand constantly. Additionally, in EHLDs where elements of a particular subpathway are too thin and users find it difficult to select them, this technique can provide a bigger and more convenient clickable area around them. However, we should use this with caution, as creating too large clickable areas can mislead users.</span></p>\r\n<p><span>It is recommended to place all arrows and neutral text elements in a group at the top of the layer hierarchy. As well, all decorators and elements in the background that we do not want to highlight or be clickable, we recommend to place them in a group at the bottom of the layer hierarchy.</span></p>\r\n<p><span>Reactome displays EHLDs in a blank zoomable space. For a better user experience we suggest to have all shapes and elements portrayed in full. However, we understand that some pathways are large and contain too many and too big elements to fit in one image. In cases where the illustrator needs to cut an element and it is difficult to make a clean representation of the cut, we suggest the addition of a gradient so that elements vanish keeping the limits of the canvas clear.</span></p>\r\n<p><span>In order to export an EHLD we should use the “Export as” option and save the file as an SVG with the following options:</span></p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li><span>Styling: Internal CSS.</span></li>\r\n<li><span>Font: SVG.</span></li>\r\n<li><span>Images: Preserve.</span></li>\r\n<li><span>Object IDs: Layer Names.</span></li>\r\n<li><span>Decimal Points: 3.</span></li>\r\n<li><span>Minify and Responsive, both ticked.</span></li>\r\n</ul>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-04-12 11:02:44','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,8,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(130,774,'Icons Library Guidelines','icon-guidelines','<p>{loadposition icon-lib-pos-1}</p>\r\n<p></p>\r\n<style type=\"text/css\">table {\r\n        width: 100%;\r\n        border-collapse: collapse;\r\n        margin-left: auto;\r\n        margin-right: auto;\r\n    }\r\n\r\n    table,\r\n    td {\r\n        border: 1px dashed #c9c9c9;\r\n    }\r\n\r\n    td.split {\r\n        width: 50%;\r\n    }\r\n\r\n    td p {\r\n        text-align: center;\r\n        margin-bottom: 0;\r\n    }\r\n\r\n    td p.detail {\r\n        margin: 5px;\r\n        text-align: justify;\r\n    }\r\n\r\n    p.small {\r\n        line-height: normal;\r\n        text-align: center;\r\n        margin-bottom: 0;\r\n    }\r\n\r\n    p.small span {\r\n        font-size: 8pt;\r\n    }</style>\r\n<p></p>\r\n<p>This section will guide you through the process of creating and including your own graphic elements into the <a href=\"https://reactome.org/icon-lib\">Reactome Icon Library</a> and become a part of this growing community.</p>\r\n<p>Since Reactome is a free and open-source project, the Reactome Library is under a Creative Commons license. As a result, if your creations are included in our library, they will follow the same policy. Feel free to use any tool or application that allows you to create <b>vector graphics</b> and, please, make sure that the following specifications are implemented in the files you will be creating.</p>\r\n<h4>Generic guidelines</h4>\r\n<ul style=\"list-style-type: disc;\">\r\n<li>Keep simplicity in mind. The main goal for this Icon Library is to provide easy to read graphic components for our users and simplicity is key to this matter.</li>\r\n<li>Consistency is important for Reactome and the Icon Library. Please, before submitting a proposal, read the guidelines we have set for the different types of components we use.</li>\r\n<li>Each icon should represent a single element only. After receiving feedback we open this rule to receptors when binding other receptors in order to make it easy for our users.</li>\r\n<li>Every element should be placed into a separate file, <b>200</b> by <b>200px</b> in size, with <b>RGB colour mode</b> and <b>White background</b>.</li>\r\n<li>Use only <b>Arial Bold 12pt</b>. Text colour should be chosen depending on the text’s position. In case the text is <b>inside</b> a coloured shape, its colour should be <b>White</b>, while in case the text is <b>outside</b> of a coloured shape, its colour should be <b>Black</b>.</li>\r\n<li>All text should be <b>Uppercased</b> except for words describing <b>greek</b> characters (e.g alpha, beta, gamma, etc.). In that case those words should be written in <b>Lowercase</b>. It should be noted that Greek or other Unicode characters must be avoided, as they are not  supported for the moment.</li>\r\n<li>Your elements should only comprise vector graphics. Please make sure that you have not included any bitmap image in it. Including raster images often results in larger file sizes and thus negatively impacts software performance.  </li>\r\n<li>You can place your new element in any of the seven categories of the Reactome Library; Cell elements, Cell types, Compounds, Human tissue, Ion channels, Proteins and Receptors.</li>\r\n</ul>\r\n<h4>Cell elements</h4>\r\n<p>Keeping simplicity is one of our targets so we recommend the use of <b>simple figures</b>, such as circles, squares and triangles as a base. For colouring we suggest the use of shades of <b>Primary Colours</b> (Red, Blue and Yellow) and <b>Secondary Colours</b> (Purple, Green and Orange).</p>\r\n<p>Vector graphic applications allow you to use <b>Gradient tools</b> to blend two or more colours for the same shape. We recommend you to use these tools so you can create more appealing graphics. See the Mitochondrion below:</p>\r\n<p style=\"text-align: center;\"><img src=\"images/ehldguide/gradient_example.png\" alt=\"gradient example\" width=\"314\" height=\"232\" /></p>\r\n<p class=\"small\"><span><em>(a) Without the use of gradients, the Mitochondrion element looks plain while in</em></span></p>\r\n<p class=\"small\"><span><em>(b) through the correct use of gradients the element looks more 3D-like, </em></span></p>\r\n<p class=\"small\"><span><em>resulting in a more attractive and playful image.</em></span></p>\r\n<h4>Cell types</h4>\r\n<p>In order to ensure a sense of unity and a consistent look and feel, we suggest applying the same principles; simple shapes and colours.</p>\r\n<p>For representing types of cells, please keep in mind that we are not using scale to portray the different organelles and elements inside the cell cytoplasm. Additionally, to maintain simplicity, try to avoid overloading the the overall design with too much detail. The following table features 3 examples:</p>\r\n<p> </p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-013014.svg\" alt=\"microbe\" width=\"173\" height=\"173\" /></p>\r\n</td>\r\n<td>\r\n<p class=\"detail\"><em>This <b>Microbe</b> element is formed by a circle with a radial gradient of two colours. To represent the membrane, we have added an outline with a different gradient. To convey the idea of danger we use this squares as spikes and a red shade in their gradients.</em></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-012969.svg\" alt=\"macrophage\" width=\"154\" height=\"154\" /></p>\r\n</td>\r\n<td>\r\n<p class=\"detail\"><em>We have drawn this <b>Macrophage</b> element with shaky tentacles and we have used a not uniform outline (membrane) to give it a soft and flexible feeling.</em></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-013155.svg\" alt=\"pathogen\" width=\"166\" height=\"169\" /><img src=\"/icon/R-ICO-013157.svg\" alt=\"pathogen dead\" /></p>\r\n</td>\r\n<td style=\"text-align: justify;\">\r\n<p class=\"detail\"><em>For this <b>Pathogen</b> element we used a large light gradient to represent a shiny and hard capsule.</em></p>\r\n<p class=\"detail\"><em>To show its death we draw holes on its surface and play with a darkened, sad colour.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Compounds</h4>\r\n<p>Compounds are represented by very geometrical elements, following more specific guidelines.</p>\r\n<p>Simple chemical elements or compounds are usually portrayed in the Library with their chemical element symbol (e.g. Ca for Calcium) or the name of the compound (e.g. IFN-gamma) using <b>Arial bold 12pt</b> in <b>White</b> colour.</p>\r\n<p>Simple chemical elements are represented by an <b>octagon</b> of <b>28px</b> height and two plain colours, one for the fill and a different one for the stroke (<b>2pt</b>). In case the text of the compound is too long to fit in the octagon, we suggest stretching the shape until it fits the name, respecting its lateral edges, like follows:</p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-013527.svg\" alt=\"h2o\" width=\"183\" height=\"138\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><em>An example of a simple chemical element.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p> </p>\r\n<p><em></em>Complex compounds can be represented with a variety of shapes, always simple and easy to differentiate one from each other.</p>\r\n<p> </p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-012399.svg\" alt=\"atp\" width=\"123\" height=\"120\" /></p>\r\n</td>\r\n<td>\r\n<p class=\"detail\"><em>We can use circles with or without outline.</em></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td>\r\n<p><img src=\"/icon/R-ICO-012413.svg\" alt=\"alpha toh\" width=\"124\" height=\"77\" /></p>\r\n</td>\r\n<td>\r\n<p class=\"detail\"><em>We can use hexagons, in the same way as for the simple compounds, just adding a gradient on their stroke.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p> </p>\r\n<h4>Human tissue</h4>\r\n<p>Human tissue elements are mainly used as backgrounds in our diagrams and the way of representing these organs depends on the illustrator’s skills and taste. To ensure unity in the Reactome Library we suggest to keep these illustrations as simple as possible and visualise them as toys; so they convey a nice, soft feeling, with a plastic-like, very friendly and approachable look.</p>\r\n<p> </p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012959.svg\" alt=\"liver\" width=\"190\" height=\"114\" /></p>\r\n</td>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012472.svg\" alt=\"blood vessel\" width=\"260\" height=\"117\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td colspan=\"2\">\r\n<p><em>Examples of human tissue elements; Liver and Blood vessel.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Ion channels</h4>\r\n<p>These representations are quite simplified and, thus, elements are designed to look like small funnels that cross through the membrane, allowing elements to move from inside of the cell to outside and vice versa. There are hundreds of different ion channels and we suggest differentiating them with the use of colours in various combinations.</p>\r\n<p> </p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012502.svg\" alt=\"calcium channel\" width=\"150\" height=\"192\" /></p>\r\n</td>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-013251.svg\" alt=\"proton channel\" width=\"159\" height=\"186\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td colspan=\"2\">\r\n<p><em>Examples of ion channels.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Proteins</h4>\r\n<p>There are tens of thousands of different proteins, some of them with specific shapes. Proteins in the Reactome Library follow the standard representation as rounded rectangle shapes. As a result, we suggest using the <b>Rounded Rectangle Tool</b> to draw a shape with <b>10px</b> radius for the corners and <b>20px</b> height for single line of text, or <b>30px</b> for a double line. The stroke should be <b>3pt </b>and we suggest choosing an irregular profile (in Adobe Illustrator, use the default <b>Width Profile 2</b>). For the name of the proteins we suggest to follow the same guides as before, <b>White Uppercase Arial Bold 12pt</b>.</p>\r\n<p>The shape will contain the simplified name of the protein following the general instructions for Reactome Library. It should be filled with a simple gradient of two colours. One of the  colours of this gradient should also be used as the stroke colour.</p>\r\n<p> </p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-013194.svg\" alt=\"plc beta1\" width=\"195\" height=\"80\" /></p>\r\n</td>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-013189.svg\" alt=\"plasminogen\" width=\"285\" height=\"83\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td colspan=\"2\">\r\n<p><em>Examples of protein elements in their rounded rectangle representation.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p> </p>\r\n<p>In case a protein has a specific shape, we suggest keeping the standard representation as long as it is not too complicated. Otherwise, we suggest opting for a more simplified version.</p>\r\n<p> </p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012968.svg\" alt=\"mac\" width=\"160\" height=\"164\" /></p>\r\n</td>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012425.svg\" alt=\"antibody\" width=\"189\" height=\"167\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td colspan=\"2\">\r\n<p><em>Examples of protein elements represented in their standard representation.</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Receptors</h4>\r\n<p>Like proteins, there is a huge number of receptors. In cases where there is a standard, but quite complex, way to represent them, we suggest adopting a more simplified version. If this is too difficult we suggest finding a more simple representation.</p>\r\n<p> </p>\r\n<table>\r\n<tbody>\r\n<tr>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012710.svg\" alt=\"FCgammaR\" width=\"111\" height=\"209\" /></p>\r\n</td>\r\n<td class=\"split\">\r\n<p><img src=\"/icon/R-ICO-012529.svg\" alt=\"cd40-cd40l\" width=\"97\" height=\"196\" /></p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td colspan=\"2\">\r\n<p><em>Examples of receptor elements</em></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<h4>Metadata</h4>\r\n<p>You are encouraged to accompany your element with more information including a short description and extra details about its designer and curator. Upon inclusion of your element into the Reactome Library, all metadata information will be available to the public through our landing page.</p>\r\n<p>In order to do so, you need to create an additional metadata file using your favorite text editor. The file should have the following structure:</p>\r\n<div class=\"code\">\r\n<div>\r\n<pre><code>&lt;metadata&gt;\r\n    &lt;categories&gt;\r\n        &lt;category&gt;<b>{CATEGORY}</b>&lt;/category&gt;\r\n    &lt;/categories&gt;\r\n    &lt;person orcid=\"<b>{ORCID}</b>\" role=\"curator\"&gt;<b>{CURATOR_NAME}</b>&lt;/person&gt;\r\n    &lt;person role=\"designer\" url=\"<b>{DESIGNER_URL}</b>\"&gt;<b>{DESIGNER_NAME}</b>&lt;/person&gt;\r\n    &lt;name&gt;<b>{ICON_NAME}</b>&lt;/name&gt;\r\n    &lt;description&gt;<b>{DESCRIPTION}</b>&lt;/description&gt;\r\n    &lt;references&gt;\r\n        &lt;reference&gt;\r\n            &lt;db&gt;<b>{REFERENCE_NAME}</b>&lt;/db&gt;\r\n            &lt;id&gt;<b>{REFERENCE_ID}</b>&lt;/id&gt;\r\n        &lt;/reference&gt;\r\n    &lt;/references&gt;\r\n    &lt;synonyms&gt;\r\n        &lt;synonym&gt;<b>{SYNONYM}</b>&lt;/synonym&gt;\r\n    &lt;/synonyms&gt;\r\n&lt;/metadata&gt;\r\n</code></pre>\r\n</div>\r\n</div>\r\n<p>Please follow these steps:</p>\r\n<ol>\r\n<li>Create a new file using your favorite text editor (TextEdit, NotePad, etc.).</li>\r\n<li>Copy and Paste the text above into your file.</li>\r\n<li>Edit the file and:<ol>\r\n<li>Fill {CATEGORY} with one or more of the suggested categories: cell_element, cell_type, compound, human_tissue, protein, receptor or transporter. We accept one or more different categories as we think they are just a guide that helps our users to find the element they need.</li>\r\n<li>Fill {ORCID} and/or {URL} for the curator and designer roles. This is <a href=\"https://orcid.org/orcid-search/search\">how to find ORCID</a>. In case both URL and ORCID are provided, the link for that person will point to the URL.</li>\r\n<li>Fill {CURATOR_NAME} and {DESIGNER_NAME} as the curator and designer name, respectively. If the name is the same, please fill both with same information.</li>\r\n<li>Fill {DESCRIPTION} with any information relevant to describe the component. Please be brief.</li>\r\n<li>Fill {REFERENCE_NAME} for the name of the resource you have used to find this component (e.g.:CHEBI, UNIPROT, GO … ). In case of not finding a suitable reference, please delete this line.</li>\r\n<li>Fill {REFERENCE_ID} as it appears in the resource’s URL (e.g.: CHEBI:1234, Q1234, GO:1234 … ). In case of not finding a suitable reference, please delete this line.</li>\r\n<li>Fill {SYNONYM} as any synonyms or alternative names your component might be known of. In case of not finding  a suitable synonym, please delete this line.</li>\r\n</ol></li>\r\n<li>Save the file using the same name as the element but make sure to use .xml as its extension. For example if your graphic element is stored with the name “Element.svg” then your metadata file should be named as “Element.xml”.</li>\r\n<li>Make sure to submit your metadata file along with the graphics file.</li>\r\n</ol>\r\n<h3>Export Settings</h3>\r\n<p>Currently, every element in the Reactome Library is available in 3 different formats, SVG, EMF and PNG. Therefore, you are kindly requested to export your work into these formats and include them in your submission.</p>\r\n<p>For PNG files we suggest to export with 300 DPI and transparent background.</p>\r\n<h3>How to submit your work</h3>\r\n<p>To include your graphic elements in the Reactome Library, you need to contact our helpdesk (<a href=\"mailto:help@reactome.org\">help@reactome.org</a>). We will then review your submitted files and, depending on the outcome, we will include them in our Library.</p>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-04-12 11:03:36','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(133,777,'ReactomeFI Network v2018 and FIViz App v2.7.0 Released','reactomefiviz-app-version-7-2-0-released','<p style=\"text-align: justify;\"><img src=\"images/news/ReactomeFI_2018.png\" width=\"650\" height=\"366\" alt=\"ReactomeFI 2018\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p style=\"text-align: justify;\">The Reactome project&nbsp;provides a suite of tools for our users to perform pathway- and network-based data analysis via its <a href=\"/\">web site</a> and <a href=\"https://apps.cytoscape.org/apps/reactomefiplugin\">ReactomeFIViz</a>, the functional interaction (FI) network-based <a href=\"https://cytoscape.org/\">Cytoscape</a> app.</p>\r\n<p style=\"text-align: justify;\">The Reactome FI network is built upon curated pathways in Reactome, supplemented with curated pathways from other pathway databases such as <a href=\"https://www.genome.jp/kegg/pathway.html\">KEGG PATHWAY</a> and <a href=\"http://www.pantherdb.org/pathway/\">PANTHER pathways</a>. These curated pathway interactions are then integrated with gene-gene interactions derived from curated and high-throughput sources using machine learning to generate a reliable set of high-probability functional interactions.</p>\r\n<p style=\"text-align: justify;\">Recently, we have released the 2018 version of the Reactome FI network. To construct this&nbsp;FI network, we improved the&nbsp;training of the Naive Bayesian Classifier, which resulted in an increase in the recall rate. Consequently, the SwissProt coverage was increased from 12,441 to 13,469 proteins (8.3% increase) and the total number of functional interactions among genes increased from 241,338 to 262,321 (8.7% increase). We attributed this to an increase in the number of predicted FIs due to the improved recall rate.</p>\r\n<p style=\"text-align: justify;\">In addition,&nbsp;the ReactomeFIViz app (version 7.2.0) now supports the 2018 version of the FI network and contains updated Reactome Pathways dataset (Release 67). The new ReactomeFIViz app is available to download within the Cytoscape software, or directly from the <a href=\"https://apps.cytoscape.org/\">Cytoscape app Store</a>. The <a href=\"https://www.reactome.org/tools/reactome-fiviz\">User Guide</a> provides instructions on using the Reactome FI network and ReactomeFIViz app.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-04-15 17:00:23','0000-00-00 00:00:00','{}','{}','{}',1,40,'','',1,0,'{}',0,'en-GB','',''),(134,778,'Documentation','documentation','<p style=\"text-align: justify;\"> {loadposition userguide-cp-1}</p>\r\n<p style=\"text-align: justify;\">The online <a href=\"/userguide\">User’s Guide</a> begins with entry-level information that describes what our resource contains and how its pathways are organized. The guide leads users through the process of browsing Reactome, searching for particular biological and chemical molecules or pathways of interest, interpreting experimental datasets through the website and the more specialized <a href=\"/userguide/reactome-fiviz\">ReactomeFIViz app</a>.</p>\r\n<h3>Introduction to the Reactome Pathway Knowledgebase</h3>\r\n<p>Please visit our \"Getting Started with Reactome\" playlist to get an overview of Reactome! </p>\r\n<p>{source}&lt;style&gt;.embed-container { position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; } .embed-container iframe, .embed-container object, .embed-container embed { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }&lt;/style&gt;&lt;div class=\'embed-container\'&gt;&lt;iframe src=\'https://www.youtube.com/embed/videoseries?si=ZgK9G3Ws-Zxin16h&amp;list=PLLc8-qElSrCxhMsCVjYIO8OmO-Im_lQ6D\' frameborder=\'0\' allowfullscreen&gt;&lt;/iframe&gt;&lt;/div&gt;{/source}</p>\r\n<h3>Reactome Training Materials</h3>\r\n<p style=\"text-align: justify;\"><span>All our training and outreach materials are available under a </span><a href=\"license\"><span>Creative Commons Attribution 4.0 Unported License</span></a><span>. We have just two simple requests, please attribute Reactome, and let us know if you use our presentations, posters, or training resources. Here are some of the material that we make available to download:</span></p>\r\n<ul style=\"padding-left: 30px;\">\r\n<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><a href=\"docs/training/Reactome_Website.pdf\"><span>Tutorial workshop</span></a><span> slides. </span>This document provides additional information about the Reactome website and suite of data analysis tools, including exercises to allow you to check your understanding. Answers are provided and were correct when written, but please note that answers may change over time as more pathways are added to Reactome.</li>\r\n<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><span><a href=\"docs/training/Pathways_&amp;_Networks_Overview.pdf\">Pathways and Networks Overview</a>. These slides provide an overview of pathway and network resources for data visualisation, analysis, and integration.</span></li>\r\n<li style=\"padding-bottom: 10px; padding-left: 15px; list-style-type: circle; text-align: justify;\"><span><a href=\"docs/training/ReactomeFIVizapp.pdf\">Reactome FI Viz app</a>. These slides provide an introduction to the Reactome Functional Interaction Network and Cytoscape app. These Reactome </span><a href=\"docs/training/ReactomeFIVizapp_Lab_Questions.pdf\"><span>Questions</span></a><span> and </span><a href=\"docs/training/ReactomeFIVizapp_Lab_Answers.pdf\"><span>Answers</span></a><span> provide a series of questions and answers to demonstrate the functionality of the Reactome FI Viz app.</span></li>\r\n</ul>\r\n<p>Other online documentation includes complete descriptions of the Reactome <a href=\"/documentation/data-model\">data model</a> and <a href=\"/content/schema/DatabaseObject\">database schema</a>, information for managers of external biological resources on how to <a href=\"/linking-to-us\">link</a> to specific types of Reactome pages, and information on how to <a href=\"/cite\">cite the resource</a> in publications. In addition, users can download specialized documentation that describes how to use the curator tool, a Reactome specific software tool for submitting data directly to the knowledgebase. One goal for Reactome has been the development of <a href=\"/dev\">reusable software tools and data resources</a> for managing and visualizing pathway information. This requires the creation of portable software and careful documentation. We believe in documenting our curatorial practices and policies as well, allowing other groups to understand and adopt our best practices.</p>\r\n<p style=\"text-align: justify;\">As with extensions of Reactome and integration of new data and software, clear documentation of these methods are developed and, when possible, published in peer-reviewed journals. Documentation is actively maintained. We ask our broader base of users to join Reactome curators, technical writers, developers and educators beyond our primary institutions: OICR, OHSU, NYU, and EBI in developing, refining, and improving our documentation.</p>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-04-18 15:33:34','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"0\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','','Home of the userguide'),(136,780,'Season of Docs','season-of-docs','<ul>\r\n<li style=\"text-align: justify;\"><img src=\"images/news/SeasonofDocs_Logo_MainGrey_300ppi.png\" alt=\"SeasonofDocs Logo MainGrey 300ppi\" style=\"display: block; margin-left: auto; margin-right: auto;\" width=\"650\" height=\"467\" /></li>\r\n</ul>\r\n<p style=\"text-align: justify;\"><strong>“Fostering open source collaboration with technical writers” – Season of Docs</strong></p>\r\n<p style=\"text-align: justify;\">Established upon the reputation of open source community programs like <a href=\"https://summerofcode.withgoogle.com/\">Google Summer of Code</a> and <a href=\"https://codein.withgoogle.com/archive/\">Google Code-in</a>, Google is introducing a new initiative called <a href=\"https://developers.google.com/season-of-docs/\">Season of Docs</a>. The Reactome project has applied to participate in this year\'s Season of Docs.&nbsp;</p>\r\n<p style=\"text-align: justify;\">Season of Docs aims to provide open source projects with an opportunity to engage with the technical writing community and for technical writers an occasion to gain experience in contributing to open source projects.</p>\r\n<p style=\"text-align: justify;\">Together, we will raise community awareness of open documentation, technical writing, and how we can collectively work together to improve open source projects. Reactome looks forward to the opportunity that Season of Docs presents and for the opportunity to bring our entire community closer together.</p>\r\n<p>&nbsp;</p>\r\n<p style=\"text-align: justify;\"><span style=\"text-decoration: underline;\"><strong>Project Information</strong></span></p>\r\n<p style=\"text-align: justify;\"><strong>Project Name:</strong> Reactome</p>\r\n<p style=\"text-align: justify;\"><strong>Project Description:</strong> Reactome is a free, open-source, curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation, and analysis of pathway knowledge to support basic research, genome analysis, modelling, systems biology, and education.</p>\r\n<p style=\"text-align: justify;\"><strong>Project Website:</strong> <a href=\"https://reactome.org\">https://reactome.org</a></p>\r\n<p>&nbsp;</p>\r\n<p style=\"text-align: justify;\"><strong>Project Idea #1 Name:</strong> Revising the Reactome User Guide</p>\r\n<p style=\"text-align: justify;\"><strong>Description:</strong> The Reactome User Guide provides an introduction to Reactome, the user interfaces and the database content. Exercises are also provided to help user practice what they have learned. New functionality and improvements to the core website and software are constantly added to the User Guide. The outcome of this project is the revision of the existing User Guide, with updated and new tutorials,a set of how-to guides to support navigating the website and using the pathway visualization and analysis tools</p>\r\n<p style=\"text-align: justify;\"><strong>Docs Link:</strong> <a href=\"https://reactome.org/userguide\">https://reactome.org/userguide</a></p>\r\n<p style=\"text-align: justify;\"><strong>Contacts:</strong> Robin Haw: <a href=\"mailto:robinhaw@gmail.com\">robinhaw@gmail.com</a> &amp; Marc Gillepsie: <a href=\"mailto:gillespm@gmail.com\">gillespm@gmail.com.</a></p>\r\n<p>&nbsp;</p>\r\n<p><strong>Project Idea #2 Name:</strong> Updating the Reactome Curator Guide</p>\r\n<p style=\"text-align: justify;\"><strong>Description:</strong> The Reactome Curator Guide contains an overview of the curatorial process as well as a step by step guide for annotating Reactome pathways. The curator guide takes new curators through the process of project design, entry, review, and release. Each one of these steps requires that the curator is familiar with the Reactome data model, software entry tools, process pipeline, and project management. In this sense, the curator guide serves as a compendium of learned lessons, curator philosophy, and a step-by-step process blueprint. The outcome of this project is the complete revision of the current guide to reflect new data entry tools, update the current glossary of terms to accommodate changes to data model, and create short tutorials that can be used to reinforce each learning step.</p>\r\n<p style=\"text-align: justify;\"><strong>Contacts:</strong> Marc Gillepsie <a href=\"mailto:gillespm@gmail.com\">gillespm@gmail.com</a> and Lisa Matthews <a href=\"mailto:lmatthews.nyumc@gmail.com\">lmatthews.nyumc@gmail.com.</a></p>\r\n<p>&nbsp;</p>\r\n<p style=\"text-align: justify;\"><strong>More Information&nbsp;</strong></p>\r\n<p style=\"text-align: justify;\">For more information about Reactome and Season of Docs, please refer to the following pages:</p>\r\n<p style=\"text-align: justify;\"><a href=\"/documentation\">Reactome Documentation</a> - Learn more about Reactome docs.</p>\r\n<p style=\"text-align: justify;\"><a href=\"https://developers.google.com/season-of-docs/docs/\">Season of Docs</a> - What is Season of Docs all about?</p>\r\n<p style=\"text-align: justify;\"><a href=\"https://developers.google.com/season-of-docs/docs/\">Guides</a> - How it all comes together.</p>\r\n<p style=\"text-align: justify;\"><a href=\"https://developers.google.com/season-of-docs/docs/timeline\">Timeline</a> - Please be aware of the schedule!</p>\r\n<p style=\"text-align: justify;\"><a href=\"https://developers.google.com/season-of-docs/terms/program-rules\">Rules</a> - The ever important rulebook.</p>\r\n<p>&nbsp;</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-04-22 19:59:12','0000-00-00 00:00:00','{}','{}','{}',1,39,'','',1,0,'{}',0,'en-GB','',''),(137,781,'Version 69 Released','version-69-released','<p><img src=\"images/news/Autophagy.png\" alt=\"Autophagy\" width=\"650\" height=\"346\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Pathways.</strong> In version V69, Autophagy, including <a href=\"/PathwayBrowser/#/R-HSA-9613829\">Chaperone Mediated Autophagy</a> and <a href=\"/PathwayBrowser/#/R-HSA-9615710&amp;PATH=R-HSA-9612973\">Microautophagy</a> is new.  Topics with new or revised pathways include Development (<a href=\"/PathwayBrowser/#/R-HSA-9616222\">Transcriptional Regulation of Granulopoiesis</a>), Disease (<a href=\"/PathwayBrowser/#/R-HSA-9630750&amp;PATH=R-HSA-1643685,R-HSA-9630747\">Evasion of Oncogene Induced Senescence Due to p16-INK4A Defects</a> and <a href=\"/PathwayBrowser/#/R-HSA-9632693&amp;PATH=R-HSA-1643685,R-HSA-9630747\">Evasion of Oxidative Stress Induced Senescence Due to p16-INK4A Defects</a>), Immune system (<a href=\"/PathwayBrowser/#/R-HSA-449836&amp;SEL=R-HSA-9607240&amp;PATH=R-HSA-168256,R-HSA-1280215,R-HSA-449147\">FLT3 Signaling</a>), and Signal Transduction (<a href=\"/PathwayBrowser/#/R-HSA-9009391&amp;PATH=R-HSA-162582,R-HSA-9006931,R-HSA-8939211\">Extra-Nuclear Estrogen Signaling</a> and <a href=\"/PathwayBrowser/#/R-HSA-1236394\">Signaling by ERBB4</a>). </p>\r\n<p style=\"text-align: justify;\"><strong>New Illustrations.</strong> Illustration with embedded navigation features is now available for <a href=\"/PathwayBrowser/#/R-HSA-9612973\">Autophagy.</a></p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors. </strong><a href=\"http://orcid.org/0000-0003-4018-5348\">David Stern</a> is our external author. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed?term=Filippo%20%20Acconcia%5BAuthor%20-%20Full%5D\">Filippo Acconcia</a>, <a href=\"https://www.researchgate.net/profile/Dorothy_Bennett\">Dorothy Bennett</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Marino%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30945992\">Maria Marino</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Emmanouil+++Metzakopian%5BAuthor%5D\">Emmanouil Metzakopian</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Julia++Skokowa%5BAuthor%5D\">Julia Skokowa</a>, and <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Traer%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30720426\">Elie Traer</a> are our external reviewers. <strong><br /></strong></p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong> Reactome comprises 12,505 human reactions organized into 2,272 pathways involving 11,009 proteins and modified forms of proteins encoded by 10,833 different human genes, 1,857 small molecules, and 196 drugs. These annotations are supported by 30.027 literature references. We have projected these reactions onto 81,631 orthologous proteins, creating 18,610 orthologous pathways in 15 non-human species. Version 69 has annotations for 1,612 protein variants (mutated proteins) and their post-translationally modified forms, derived from 305 proteins. These have been used to annotate 532 complexes and 968 disease-specific reactions organized into 478 pathways and subpathways, and tagged with 374 Disease Ontology terms.</p>\r\n<p style=\"text-align: justify;\"><strong>Tools and Data. </strong>Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"/PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"/download-data\">download</a>, and to access programmatically through our <a href=\"/ContentService/\">Content</a> and <a href=\"/AnalysisService/\">Analysis</a> Services. The <a href=\"/tools/reactome-fiviz\">ReactomeFIViz app</a> for Cytoscape provides tools to find pathways and network patterns related to cancer and other types of diseases.</p>\r\n<p style=\"text-align: justify;\"><strong>Documentation and Training</strong>. Visit our online <a href=\"/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"/community/training\">here</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org\">website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong> <a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information</strong>: If you have a question to ask or would like to give us your feedback, please contact our <a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-05-28 17:41:55','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,38,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(138,783,'ORCID: Claim your works','orcid-claim-your-works','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/20190618_ORCID_claim.png\" alt=\"claim your reactome work into your orcid account image\" usemap=\"#cwmap\" width=\"650\" height=\"382\" /> <map name=\"cwmap\"> \r\n<area coords=\"115,182,540,250\" shape=\"rect\" href=\"index.php?option=com_content&amp;view=article&amp;id=139&amp;Itemid=488\" />\r\n </map></p>\r\n<p style=\"text-align: justify;\">Reactome is a curated database, it critically depends on the expertise of curators and domain experts&nbsp;as volunteer pathway authors and reviewers. Over <a href=\"/content/query?q=type%3APerson&amp;species=Entries+without+species&amp;types=Person&amp;cluster=true\">700 scientists</a> have so far contributed to Reactome content. If you are one of them, we have great news for you: Your Reactome contribution can be included directly into your&nbsp;<a href=\"https://orcid.org\" target=\"_blank\" rel=\"noopener\">ORCID</a> profile. ORCID provides a persistent digital identifier that distinguishes you from every other researcher and, through integration in key research workflows such as manuscript and grant submission, supports automated linkages between you and your professional activities ensuring that your work is recognized.</p>\r\n<p style=\"text-align: justify;\">Because we appreciate your work contribution either authoring or reviewing a pathway or reaction, we have now made it easy to claim your contribution in ORCID. Using this new feature that has been integrated into our website, contributors who have provided their ORCID ID can, by simply clicking a button, reference their work in their profile.</p>\r\n<div class=\"leading-8\" itemscope=\"itemscope\" itemtype=\"https://schema.org/BlogPosting\" itemprop=\"blogPost\">\r\n<p style=\"text-align: justify;\">Follow this <a title=\"Click for instructions on how to claim your work\" href=\"index.php?option=com_content&amp;view=article&amp;id=139&amp;Itemid=488\">link</a> to learn more&nbsp;about how you can claim your work.</p>\r\n</div>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-06-06 14:16:00','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,37,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(139,784,'Tutorial: How to claim your works','tutorial-how-to-claim-your-works','<style type=\"text/css\">img { margin: 5px 0 15px; 3px; }</style>\r\n<ul>\r\n<li><a href=\"#guide\">Find yourself in Reactome</a></li>\r\n<li><a href=\"#missing_orcid\">I have an ORCID ID but it is not in Reactome. What should I do ?</a></li>\r\n</ul>\r\n<h3 id=\"guide\">Find yourself in Reactome</h3>\r\n<ol>\r\n<li>The simple text search tool is located at top right of the Home page. To search type your name or ORCID. The search has an auto-complete function; if the text you wanted to use appears in the drop-down list, select it and results will be displayed. For other text click Search.<br /><img src=\"images/userguide/search/search_box.gif\" alt=\"search box\" width=\"400\" height=\"93\" /></li>\r\n<li>Locate yourself in the Result List and Click your name.&nbsp;<br /><img src=\"images/userguide/orcid/name_in_result_list.png\" alt=\"name in result list\" width=\"400\" height=\"105\" /></li>\r\n<li>&nbsp;You\'ll land in the Contributor\'s page. You may see up to four categorized tables showing a maximum of 15 rows each.<br /><img style=\"width: 407px; height: 57px;\" src=\"images/userguide/orcid/are_you_orcid_login.png\" alt=\"are you orcid login\" data-mce-width=\"407\" data-mce-height=\"57\" /></li>\r\n<li>In order to claim your work you may need to Authenticate yourself in ORCID, in a similar way like others websites offer \'Sign in with Google\', \'Sign in with Facebook\'. Simply click \"Are you John Doe ?\". A pop-up like to the one below will open. Please input your ORCID credentials, then Click Sign-in.<br /><img style=\"width: 223px; height: 272px;\" src=\"images/userguide/orcid/orcid_login_dialog.png\" alt=\"orcid login dialog\" data-mce-width=\"223\" data-mce-height=\"272\" /></li>\r\n<li>Reactome.org will ask your permission to add works into your ORCID records. Authorize if you wish to grant access, otherwise we can\'t have access to&nbsp;<br /><img style=\"width: 225px; height: 245px;\" src=\"images/userguide/orcid/orcid_authorise_reactome.png\" alt=\"orcid authorise reactome\" data-mce-width=\"225\" data-mce-height=\"245\" /></li>\r\n<li>After authorizing, we will automatically proceed with the authorization process and the pop up will be closed and current page will be refreshed. A button \"Claim all Work\" may appear next to your ORCID number if we are able to match ORCID IDs. Click it to make sure all your contributions are uploaded.<br /><img style=\"width: 381px; height: 179px;\" src=\"images/userguide/orcid/claim_all_work.png\" alt=\"claim all work\" data-mce-width=\"381\" data-mce-height=\"179\" /></li>\r\n<li>Depending on the number of entries you are uploading it may take a few minutes. You may see a Claiming Summary when the entire process is finished.<br /><img style=\"width: 199px; height: 202px;\" src=\"images/userguide/orcid/claiming_summary.png\" alt=\"claiming summary\" data-mce-width=\"199\" data-mce-height=\"202\" /></li>\r\n<li>Visit your ORCID page and Navigate to the Works Section.&nbsp;</li>\r\n<li>All your work now is uploaded.</li>\r\n</ol>\r\n<h3 id=\"missing_orcid\">I have an ORCID ID but it is not in Reactome. What should I do ?</h3>\r\n<ol>\r\n<li>Follow intructions 1 to 5 in the previous section.</li>\r\n<li>You may see a Link \"Let us know your ORCID\". Please Click.<br /><img style=\"width: 371px; height: 80px;\" src=\"images/userguide/orcid/let_us_know_link.png\" alt=\"let us know link\" data-mce-width=\"371\" data-mce-height=\"80\" /></li>\r\n<li>&nbsp;After clicking, use the following form to send your ORCID details to our curators. Our help team will validate and add your ORCID into our database and will be available to you on our next data release. Follow us on Twitter for any updates.<br /><img style=\"width: 340px; height: 289px;\" src=\"images/userguide/orcid/let_us_know_form.png\" alt=\"let us know form\" data-mce-width=\"340\" data-mce-height=\"289\" /></li>\r\n</ol>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-06-14 13:51:38','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(140,785,'Contribute Pathway Knowledge','external-contribution','<p>Become a Reactome contributor! You will be credited with authorship or reviewership for all of your contributions. Each pathway is associated with a DOI and can be cited as a publication. You can quickly and easily claim your Reactome contributions in ORCID using our new ORCID claiming feature when you search for your name in Reactome as described <a href=\"userguide/claim-your-work\">here</a>. </p>\r\n<p>We are currently seeking reviewers for the following <a href=\"https://docs.google.com/spreadsheets/d/e/2PACX-1vRorn50RGvdhBflvIAMwr7zgsj3GQrsgBBOuzM6MaKN5ntPcTYRQiCNk5OcYt0aumeLjcUTbhDb_omc/pubhtml?widget=true&amp;amp;headers=false\" target=\"_blank\" rel=\"noopener\">pathways</a>. If you\'d like to contribute a pathway that is not on this list, please contact us. We would be happy to work with you! For more information and guidelines for reviewing a Reactome pathway module, see this <a href=\"index.php?option=com_content&amp;view=article&amp;id=143&amp;catid=2&amp;lang=en-GB\" hreflang=\"en\">FAQ for prospective Reactome reviewers and authors</a>. </p>\r\n<p><iframe src=\"https://docs.google.com/spreadsheets/d/e/2PACX-1vRorn50RGvdhBflvIAMwr7zgsj3GQrsgBBOuzM6MaKN5ntPcTYRQiCNk5OcYt0aumeLjcUTbhDb_omc/pubhtml?widget=false&amp;headers=false&amp;chrome=false\" width=\"100%\" height=\"700px\"></iframe></p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-06-20 15:12:50','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(141,786,'ReacFoam: Genome-wide pathway overview based on Voronoi tessellation','reacfoam-genome-wide-pathway-overview-based-on-voronoi-tessellation','<p><a href=\"https://reactome.org/reacfoam/?species=48887&color=COPPER%20PLUS&analysis=MjAxOTA5MjMxMDAwNDBfNTczMQ==&resource=TOTAL\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/voroni.png\" alt=\"\" width=\"650\" /></a></p>\r\n<p>As part of our continuous effort to provide visually attractive and more user-friendly access to our biological pathways, we are pleased to announce our new high level pathways overview visualisation based on Voronoi tessellation.</p>\r\n<p>Following any <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">pathway analysis</a>, the <a href=\"https://reactome.org/reacfoam/?species=48887&amp;color=COPPER%20PLUS&amp;analysis=MjAxOTA2MjAxMjE0NTFfMTM0&amp;resource=TOTAL\">pathway overview</a> provides a <a href=\"reacfoam\">new icon<img src=\"images/Screenshot_2019-07-02_at_124156.png\" alt=\"\" width=\"35\" height=\"35\" /></a>&nbsp; which leads to a comprehensive, highly visual, interactive overview of pathway analysis results. The functionality is available on both desktop and mobile platforms.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-07-02 11:38:13','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,36,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(142,787,'Version 70 Released','version-70-released','<p style=\"text-align: justify;\"><img src=\"images/news/Innate_Immune_System.png\" alt=\"Innate Immune System\" width=\"1168\" height=\"662\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>New and Updated Pathways.</strong> In version 70, topics with new or revised pathways include Cellular response to stress (<a href=\"PathwayBrowser/#/R-HSA-9639288\">Amino acids regulate mTORC1</a>), Disease (<a href=\"PathwayBrowser/#/R-HSA-9630750\">Evasion of oncogene induced senescence due to p16-INK4A defects</a>, <a href=\"PathwayBrowser/#/R-HSA-9632693\">Evasion of oxidative stress induced senescence due to p16-INK4A defects</a>, <a href=\"PathwayBrowser/#/R-HSA-9645722\">Defective intrinsic pathway for apoptosis due to p14ARF loss of function</a>, <a href=\"PathwayBrowser/#/R-HSA-9646303\">Evasion of oncogene induced senescence due to p14ARF defects</a>, <a href=\"PathwayBrowser/#/R-HSA-9646304\">Evasion of oxidative stress induced senescence due to p14ARF defects</a>), Immune System (<a href=\"PathwayBrowser/#/R-HSA-9645460\">Alpha-protein kinase 1 signaling pathway </a>and <a href=\"PathwayBrowser/#/R-HSA-9607240\">FLT3 signaling</a>), and Signal transduction (<a href=\"PathwayBrowser/#/R-HSA-9024446\">NR1H2 and NR1H3-mediated signaling</a>).</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>New and Updated Illustrations.</strong> Topics with new or revised illustrations include <a href=\"PathwayBrowser/#/R-HSA-71291&amp;PATH=R-HSA-1430728\">Metabolism of amino acids and derivatives</a> and <a href=\"PathwayBrowser/#/R-HSA-168249&amp;PATH=R-HSA-168256\">Innate Immune System</a>. </span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong><strong>Thanks to our Contributors.</strong> </strong><a href=\"https://orcid.org/0000-0002-3639-7527\">Dorothy Bennett</a>, <a href=\"https://orcid.org/0000-0002-9515-8892\">Kendall Condon</a>, <a href=\"http://www.cumminslab.com\">Carolyn Cummins</a>, <a href=\"https://orcid.org/0000-0003-4760-1033\">Nicholas Hayward</a>, <a href=\"https://orcid.org/0000-0003-2798-5762\">Sunil Joshi</a>, <a href=\"https://orcid.org/0000-0001-7201-6819\">Vaishnavi Nathan</a>, <a href=\"https://orcid.org/0000-0001-5740-1954\">Joyce Repa</a>, <a href=\"https://orcid.org/0000-0002-2094-9198\">Helen Rizos</a>, <a href=\"https://orcid.org/0000-0002-1446-7256\">David Sabatini</a>, and <a href=\"http://www.nibs.ac.cn/en/yjsjyimgshow.php?cid=5&amp;sid=6&amp;id=777\">Feng Shao</a> are our external reviewers.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Annotation Statistics.</strong> Reactome comprises 12,608 human reactions organized into 2,282 pathways involving 11,040 proteins and modified forms of proteins encoded by 10,860 different human genes, 12,335 complexes, 1,856 small molecules, and 222 drugs. These annotations are supported by 30.398 literature references. We have projected these reactions onto 83,183 orthologous proteins, creating 18,679 orthologous pathways in 15 non-human species. Version 70 has annotations for 1,762 protein variants (mutated proteins) and their post-translationally modified forms, derived from 308 proteins. These have been used to annotate 536 complexes and 970 disease-specific reactions organized into 484 pathways and subpathways, and tagged with 387 Disease Ontology terms.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Tools and Data. </strong>Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"PathwayBrowser/\">Pathway Browser</a>, to <a href=\"PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"download-data\">download</a>, and to access programmatically through our <a href=\"ContentService/\">Content</a> and <a href=\"AnalysisService/\">Analysis</a> Services. The <a href=\"tools/reactome-fiviz\">ReactomeFIViz app</a> for Cytoscape provides tools to find pathways and network patterns related to cancer and other types of diseases.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Documentation and Training</strong>. Visit our online <a href=\"userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"community/training\">here</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org\">website</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Follow us on Twitter:</strong> <a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>For more information</strong>: If you have a question to ask or would like to give us your feedback, please contact our <span id=\"cloak3c9b6732f53c185a2fdfae327f5e52d3\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span>.</span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-09-05 14:28:19','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,24,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(143,789,'FAQ for prospective Reactome reviewers and authors','faq-for-prospective-reactome-reviewers-and-authors','<p><strong>What is involved in reviewing a&nbsp; Reactome pathway module?</strong></p>\r\n<p>Reviewing a pathway module is similar to reviewing a review article and involves evaluating both a pathway report in text format and a corresponding online pathway diagram for completeness and accuracy.&nbsp; A Reactome pathway is a hierarchical representation of a biological process. Pathways are broken down into component subpathways. A subpathway is further subdivided into its component biochemical reactions, and each reaction includes input and output molecules as well as any relevant catalyst or regulators. The text report includes a summary of each event (pathway, subpathway or reaction), a list of its supporting references, and a link to the corresponding event in our pathway browser on our development website.&nbsp; This pathway browser web page, shown below, includes an Event Hierarchy as well as a Pathway Diagram and Event Details section.&nbsp;</p>\r\n<p><img src=\"images/pathway_browser.png\" alt=\"Pathway Browser\" /></p>\r\n<p>In the pathway diagram, reactions are manually laid out showing their relationships to one another. The details section of this webpage provides more detailed dynamic descriptions of the molecular composition and hierarchical organization of reactions.&nbsp;</p>\r\n<p>Selecting any event in the hierarchy (left panel) will bring you to its location in the pathway diagram, and the corresponding event(s) will be highlighted in blue in the diagram. A text and molecular description of the reaction or event can be viewed in the Details tab in the panel below the diagram. The description of the reaction may also be accessed by clicking on the reaction “node” in the diagram. A text description (and cross-references) for individual reaction component molecules can be displayed by selecting the molecule of interest in the diagram. More detailed instructions for navigating the website can be found <a href=\"https://docs.google.com/document/d/10XDgRsZsjDGpWPVDeeAqnBPzRxgHa-sTr-W0YgVHn28/edit#heading=h.akq8x59f82g1\">here</a>.&nbsp;</p>\r\n<p>We are asking reviewers to verify that the pathways and reactions described in the text document are annotated clearly and completely and that the molecular details of the reactions (described in-depth on the webpages) are accurate.&nbsp; Additional instructions for navigating the web pages can be found here. We would appreciate any comments or suggestions on the user interface as well.</p>\r\n<p><strong>How long does it take to review a module?</strong></p>\r\n<p>Depending on the size of the module, reviewers take anywhere from a day to a month for their reviews. Typically, reviewers provide their feedback within two or three weeks. Reactome has a rolling quarterly release cycle though, so if your review takes a bit longer than expected, this is not a problem. We can include the revised module in the next release.&nbsp;</p>\r\n<p><strong>If I’ve decided to review a module, how do I get started?</strong></p>\r\n<p>If you’re ready to get started reviewing, send us an email at <a href=\"mailto:help@reactome.org\">help@reactome.org</a> and we can put you in contact with the curator of the pathway module. You can communicate with this curator if you have any questions and when you’d like to submit your review.&nbsp;</p>\r\n<p><strong>When can I see my work published on the Reactome website?</strong></p>\r\n<p>Reactome has a quarterly database release cycle, generally in March, June, September, and December. For a revised module to be included in a given release, we ask that reviews be submitted 6 weeks (or more) before the planned release date. So for a revised module to be included in a March release, we would need to have the review back in mid-January.</p>\r\n<p><strong>How can I provide feedback and suggestions to Reactome?</strong></p>\r\n<p>Comments may be added directly to the word document provided. Please opt to “tracking changes” so that your comments and suggested changes are highlighted. If you prefer, you can send your review as a separate document.&nbsp;</p>\r\n<p><strong>What if my work (or other important work) has not been included or cited?</strong></p>\r\n<p>Due to limited resources, we may not have included all of the references that are relevant for a particular pathway or reaction, but we would be happy to add any that you might wish to see included. Likewise, if you would like to expand a pathway module to include events that we have not covered, we would be glad to work with you on this and would cite you as an author for any new content.&nbsp;</p>\r\n<p><strong>What if I have questions while reviewing a pathway module?</strong></p>\r\n<p>We will pair you with the curator that has worked on the pathway module that you are reviewing. You can communicate directly with this curator for any questions. You can also contact us at&nbsp; <a href=\"mailto:help@reactome.org\">help@reactome.org</a> for any technical questions.&nbsp;</p>\r\n<p><strong>How am I acknowledged for my contribution to Reactome?</strong></p>\r\n<p>Your review will be used by a Reactome curator to revise the pathway module. You will be acknowledged as a reviewer of this pathway on the web pages and in the Reactome table of contents.&nbsp; This pathway module will also be associated with a DOI and may be cited as a publication. We ask that you provide us with an ORCID identifier so that we can associate this with the pathways that you have contributed.&nbsp; Our main search allows our authors and reviewers to <a href=\"https://reactome.org/content/query?q=*&amp;species=Entries+without+species&amp;types=Person&amp;cluster=true\">query</a> for their pathway and reaction contributions using their names, and claim these contributions directly into ORCID.</p>\r\n<p><strong>&nbsp;Spread the word!</strong></p>\r\n<p>Reactome is always looking for new contributors! If you know someone that would be interested in contributing a new pathway or reviewing/revising an existing pathway module, please let them know about us and our mission.</p>','',1,27,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-09-11 17:14:23','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(144,790,' New Reactome Publication published in 2020 NAR Database Issue','new-reactome-publication-published-in-2020-nar-database-issue','<p><img src=\"images/news/reactome_nar_2020.jpeg\" alt=\"reactome nar 2020\" width=\"804\" height=\"400\" /></p>\r\n<p class=\"p1\"><span class=\"s1\">A new&nbsp;research article titled “<a href=\"https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz1031/5613674\"><span class=\"s2\">The reactome pathway knowledgebase</span></a>” has been published in the forthcoming&nbsp;2020 NAR Database Issue. The paper describes annotating the molecular mechanisms of drug action, facilitating community involvement in annotation, and improvements to the reaction and pathway visualization. More publications from the&nbsp;<a href=\"about/team\"><span class=\"s2\">Reactome Team</span></a>&nbsp;can be found&nbsp;<a href=\"community/publications/\"><span class=\"s3\">here</span></a>.</span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-11-29 15:00:57','0000-00-00 00:00:00','{}','{}','{}',1,35,'','',1,0,'{}',0,'en-GB','',''),(145,791,'Version 71 Released','version-71-releases','<p><strong><img src=\"images/news/FA_Metabolism.png\" alt=\"FA Metabolism\" width=\"639\" height=\"400\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></strong></p>\r\n<p><span style=\"font-size: 10pt;\"><strong>New and Updated Pathways.</strong> In version 71, Topics with new or revised pathways include Autophagy (<a href=\"PathwayBrowser/#/R-HSA-9646399\" target=\"_blank\">Aggrephagy</a>), Cell Cycle (<a href=\"PathwayBrowser/#/R-HSA-9648025\" target=\"_blank\">EML4 and NUDC in mitotic spindle formation</a>), Cellular responses to external stimuli (<a href=\"PathwayBrowser/#/R-HSA-9633012\" target=\"_blank\">EIK2AK4 (GCN2) modifies gene expression in response to amino acid deficiency</a> and <a href=\"PathwayBrowser/#/R-HSA-9648895\" target=\"_blank\">Response of EIF2AK1 (HRI) to heme deficiency</a>), Disease (<a href=\"PathwayBrowser/#/R-HSA-9656249\" target=\"_blank\">Defective base excision repair associated with OGG1</a>, <a href=\"PathwayBrowser/#/R-HSA-9609646\" target=\"_blank\">HCMV infection</a>, <a href=\"PathwayBrowser/#/R-HSA-9635486\" target=\"_blank\">Infection with Mycobacterium tuberculosis</a>, and <a href=\"PathwayBrowser/#/R-HSA-1227990\" target=\"_blank\">Signaling by ERBB2 in Cancer</a>), Generic transcription pathway (<a href=\"PathwayBrowser/#/R-HSA-8853884\" target=\"_blank\">Transcriptional regulation by VENTX</a>), Metabolism (<a href=\"PathwayBrowser/#/R-HSA-189445\" target=\"_blank\">Metabolism of porphyrins</a>), and Signal Transduction (<a href=\"PathwayBrowser/#/R-HSA-1227986&amp;SEL=R-HSA-9652282&amp;PATH=R-HSA-162582,R-HSA-9006934\">Drug-mediated inhibition of ERBB2 signaling</a>).</span></p>\r\n<p><span style=\"font-size: 10pt;\"><strong>New and Updated Illustrations</strong>.<span style=\"background-color: inherit; font-family: inherit;\"> </span></span><span style=\"font-size: 10pt;\"><span style=\"background-color: inherit; color: inherit; font-family: inherit;\"><a href=\"PathwayBrowser/#/R-HSA-9018678\" target=\"_blank\">Biosynthesis of specialized proresolving mediators (SPMs)</a>, <a href=\"PathwayBrowser/#/R-HSA-8978868\" target=\"_blank\">Fatty acid metabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-1430728\" target=\"_blank\">Metabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-556833\" target=\"_blank\">Metabolism of lipids</a>, <a href=\"PathwayBrowser/#/R-HSA-196854\" target=\"_blank\">Metabolism of vitamins and cofactors</a>, and <a href=\"PathwayBrowser/#/R-HSA-1483257\" target=\"_blank\">Phospholipid metabolism</a> have new Illustrations with embedded navigation features. <a href=\"PathwayBrowser/#/R-HSA-2262752\" target=\"_blank\">Cellular responses to stress</a>, <a href=\"PathwayBrowser/#/R-HSA-8963743\" target=\"_blank\">Digestion and absorption</a>, and <a href=\"PathwayBrowser/#/R-HSA-9612973\" target=\"_blank\">Autophagy</a> have revised Illustrations with embedded navigation features.</span></span></p>\r\n<p><span style=\"font-size: 10pt;\"><strong><strong>Thanks to our Contributors.</strong> </strong><a href=\"https://orcid.org/0000-0002-4650-631X\" target=\"_blank\" rel=\"noopener\">Ralf Stephan</a> is our external author. <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Susanne++Bechstedt%5BAuthor%5D\" target=\"_blank\" rel=\"noopener\">Susanne Bechstedt</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Boldogh%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30240635\" target=\"_blank\" rel=\"noopener\">Istvan Boldogh</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Bruhat%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27297692\" target=\"_blank\" rel=\"noopener\">Alain Bruhat</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Patrizia++Caposio%5BAuthor%5D\" target=\"_blank\" rel=\"noopener\">Patrizia Caposio</a>, <a href=\"https://orcid.org/0000-0002-4372-6907\" target=\"_blank\" rel=\"noopener\">Jane-Jane Chen</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Armin++Deffur%5BAuthor%5D\" target=\"_blank\" rel=\"noopener\">Armin Deffur</a>, <a href=\"https://orcid.org/0000-0003-4417-7329\" target=\"_blank\" rel=\"noopener\">Andrew Fry</a>, <a href=\"https://orcid.org/0000-0001-8146-1551\" target=\"_blank\" rel=\"noopener\">Rama Krishna Kancha</a>, <a href=\"https://orcid.org/0000-0001-6175-7697\" target=\"_blank\" rel=\"noopener\">Kellie Lucken</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Metzakopian%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30622252\" target=\"_blank\" rel=\"noopener\">Emmanouil Metzakopian</a>, <a href=\"https://orcid.org/0000-0002-6101-6300\" target=\"_blank\" rel=\"noopener\">Laura O\'Regan</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Streblow%20DN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=31431555\" target=\"_blank\" rel=\"noopener\">Daniel Streblow</a>, <a href=\"https://orcid.org/0000-0001-7665-4447\" target=\"_blank\" rel=\"noopener\">Naidu M Vegi</a>, <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Vlahopoulos%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=30240635\" target=\"_blank\" rel=\"noopener\">Spiros Vlahopoulos</a>, and <a href=\"https://www.ncbi.nlm.nih.gov/pmc/?term=Robert++J+Wilkinson%5BAuthor%5D\" target=\"_blank\" rel=\"noopener\">Robert Wilkinson</a> are our external reviewers.</span></p>\r\n<p><span style=\"font-size: 10pt;\"><strong>Annotation Statistics.</strong> Reactome comprises 12,608 human reactions organized into 2,282 pathways involving 11,053 proteins and modified forms of proteins encoded by 10,870 different human genes, 12,489 complexes, 1,863 small molecules, and 225 drugs. These annotations are supported by 30,721 literature references. We have projected these reactions onto 83,392 orthologous proteins, creating 18,697 orthologous pathways in 15 non-human species. Version 71 has annotations for 1,816 protein variants (mutated proteins) and their post-translationally modified forms, derived from 310 proteins, which have been used to annotate disease-specific reactions and pathways.</span></p>\r\n<p><span style=\"font-size: 10pt;\"><strong>Tools and Data. </strong>Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"PathwayBrowser/\">Pathway Browser</a>, to <a href=\"PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"download-data\">download</a>, and to access programmatically through our <a href=\"ContentService/\">Content</a> and <a href=\"AnalysisService/\">Analysis</a> Services. The <a href=\"tools/reactome-fiviz\">ReactomeFIViz app</a> for Cytoscape provides tools to find pathways and network patterns related to cancer and other types of diseases.</span></p>\r\n<p><span style=\"font-size: 10pt;\"><strong>Documentation and Training</strong>. Visit our online <a href=\"userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"community/training\">here</a>.</span></p>\r\n<p><span style=\"font-size: 10pt;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https:\">website</a>.</span></p>\r\n<p><span style=\"font-size: 10pt;\"><strong>Follow us on Twitter:</strong> <a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</span></p>\r\n<p><span style=\"font-size: 10pt;\"><strong>For more information</strong>: If you have a question to ask or would like to give us your feedback, please contact our <span id=\"cloak3c9b6732f53c185a2fdfae327f5e52d3\"><span id=\"cloakb0e4e4ab26f934c647cd429c106e907e\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span></span>.</span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-12-02 05:43:55','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,34,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(146,794,' New Paper published in Database (Oxford)','new-paper-published-in-database-oxford','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Reactome-ORCID.png\" alt=\"Reactome ORCID\" width=\"650\" height=\"339\" /></p>\r\n<p>A new&nbsp;Perspective/Opinion article titled “<a href=\"https://doi.org/10.1093/database/baz123\">Reactome and ORCID-fine-grained credit attribution for community curation</a>” has been published in the Database (Oxford) journal. The paper describes how we are using ORCID identifiers to provide clear credit attribution for authors, curators and reviewers to support community engagement. More publications from the&nbsp;<a href=\"about/team\"><span class=\"s2\">Reactome Team</span></a>&nbsp;can be found&nbsp;<a href=\"community/publications/\"><span class=\"s3\">here</span></a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2019-12-19 04:36:50','0000-00-00 00:00:00','{}','{}','{}',1,33,'','',1,0,'{}',0,'en-GB','',''),(147,797,'Version 72 Released','version-72-released','<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><img src=\"images/news/infectious_events.png\" alt=\"infectious events\" width=\"650\" height=\"381\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt; font-family: helvetica;\"><strong>New and Updated Pathways.</strong> In version 72, topics with new or revised pathways include Autophagy (<a href=\"PathwayBrowser/#/R-HSA-9664873\">Pexophagy</a>), Cell Cycle (<a href=\"PathwayBrowser/#/R-HSA-2555396&amp;SEL=R-HSA-2995410&amp;PATH=R-HSA-1640170,R-HSA-69278,R-HSA-68886\">Nuclear Envelope Reassembly</a>), Developmental Biology (<a href=\"PathwayBrowser/#/R-HSA-9619665\">EGR2- and SOX10-mediated initiation of Schwann cell myelination</a>), Disease (<a href=\"PathwayBrowser/#/R-HSA-1227990\" target=\"_blank\" rel=\"noopener\">Signaling by ERBB2 in Cancer</a>, <a href=\"PathwayBrowser/#/R-HSA-9665348\">Signaling by ERBB2 ECD Mutants</a>, <a href=\"PathwayBrowser/#/R-HSA-9665686\">Signaling by ERBB2 TMD/JMD Mutants</a>, <a href=\"PathwayBrowser/#/R-HSA-9671555\">Signaling by PDGFR in disease</a>, and <a href=\"PathwayBrowser/#/R-HSA-9658195&amp;PATH=R-HSA-1643685,R-HSA-5663205\">Leishmania infection</a>),  and Signal Transduction (<a href=\"PathwayBrowser/#/R-HSA-9031628\">NGF-stimulated transcription</a> and <a href=\"PathwayBrowser/#/R-HSA-9652282\">Drug-mediated inhibition of ERBB2 signaling</a>).</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt; font-family: helvetica;\"><strong>New and Updated Illustrations.</strong> Illustrations with embedded navigation features have been added or revised for <a href=\"PathwayBrowser/#/R-HSA-1643685\">Disease</a>, <a href=\"PathwayBrowser/#/R-HSA-5663205&amp;PATH=R-HSA-1643685\">Infectious disease</a>, and <a href=\"PathwayBrowser/#/R-HSA-9658195&amp;PATH=R-HSA-1643685,R-HSA-5663205\">Leishmania infection</a> and <a href=\"PathwayBrowser/#/R-HSA-1266738\">Developmental Biology</a>. New static illustrations are now available for <a href=\"PathwayBrowser/#/R-HSA-376176&amp;SEL=R-HSA-428542&amp;PATH=R-HSA-1266738,R-HSA-422475\">Regulation of commissural axon pathfinding by SLIT and ROBO</a>, <a href=\"PathwayBrowser/#/R-HSA-376176&amp;PATH=R-HSA-1266738,R-HSA-422475\">Signaling by ROBO receptors</a>, and <a href=\"PathwayBrowser/#/R-HSA-202403\">TCR signaling</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt; font-family: helvetica;\"><strong>Thanks to our Contributors.</strong><span class=\"apple-converted-space\"> <span style=\"text-decoration: underline;\"><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=John+Aletta[Author]&amp;sort=\" target=\"_blank\" rel=\"noopener\">John Aletta</a>,</span> <span style=\"text-decoration: underline;\"><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Ron+Bose%5BAuthor%5D&amp;sort=\" target=\"_blank\" rel=\"noopener\">Ron Bose</a></span>, <span style=\"text-decoration: underline;\"><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Larry+Gerace&amp;sort=\" target=\"_blank\" rel=\"noopener\">Larry Gerace</a>,</span> <span style=\"text-decoration: underline;\"><a href=\"https://orcid.org/0000-0001-6534-7150\" target=\"_blank\" rel=\"noopener\">David Gregory</a>,</span> <span style=\"text-decoration: underline;\"><a href=\"https://orcid.org/0000-0002-6672-5982\" target=\"_blank\" rel=\"noopener\">Carman Ip</a>,</span> <span style=\"text-decoration: underline;\"><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Kancha,+Rama+Krishna\" target=\"_blank\" rel=\"noopener\">Rama Krishna Kancha</a>,</span> <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Kinsella%2C+B+Therese%5BAuthor%5D&amp;sort=\">T</a><span style=\"text-decoration: underline;\"><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Kinsella%2C+B+Therese%5BAuthor%5D&amp;sort=\">herese Kinsella</a></span>, Anagha Krishna, <span style=\"text-decoration: underline;\"><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Liu%2C+Xiangguo%5BAuthor%5D&amp;sort=\" target=\"_blank\" rel=\"noopener\">Xiangguo Liu</a>,</span> <span style=\"text-decoration: underline;\"><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Martin%2C+E+Anne%5BAuthor%5D&amp;sort=\" target=\"_blank\" rel=\"noopener\">Anne Martin</a>,</span> <span style=\"text-decoration: underline;\"><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Emmanouil+Metzakopian&amp;sort=\" target=\"_blank\" rel=\"noopener\">Emmanouil Metzakopian</a>,</span> <span style=\"text-decoration: underline;\"><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Miller%2C+Adam+C&amp;sort=\" target=\"_blank\" rel=\"noopener\">Adam Miller</a></span>, <span style=\"text-decoration: underline;\"><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Mulvaney%2C+Eamon+P%5BAuthor%5D&amp;sort=\" target=\"_blank\" rel=\"noopener\">Eamon Mulvaney</a></span>, <span style=\"text-decoration: underline;\"><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Staschke%2C+Kirk+A%5BAuthor%5D&amp;sort=\" target=\"_blank\" rel=\"noopener\">Kirk Staschke</a></span>, and <span style=\"text-decoration: underline;\"><a href=\"https://www.ncbi.nlm.nih.gov/pubmed/?term=Liu%2C+Xiangguo+and+Yao%2C+Shun\" target=\"_blank\" rel=\"noopener\">Shun Yao</a></span> are our external reviewers.</span></span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt; font-family: helvetica;\"><strong style=\"font-family: arial, helvetica, sans-serif; font-size: 10pt; color: inherit; caret-color: auto;\">Annotation Statistics.</strong><span style=\"color: inherit; background-color: inherit; caret-color: auto;\"> Reactome comprises 12,986 human reactions organized into 2,362 pathways involving 11,096 proteins and modified forms of proteins encoded by 10,908 different human genes, 12,728 complexes, 1,865 small molecules, and 237 drugs. These annotations are supported by 31,237 literature references. We have projected these reactions onto 83,698 orthologous proteins, creating 18,996 orthologous pathways in 15 non-human species. Version 72 has annotations for 1,890 protein variants (mutated proteins) and their post-translationally modified forms, derived from 315 proteins, which have been used to annotate disease-specific reactions and pathways.</span></span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt; font-family: helvetica;\"><strong>Tools and Data. </strong>Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"PathwayBrowser/\">Pathway Browser</a>, to <a href=\"PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"download-data\">download</a>, and to access programmatically through our <a href=\"ContentService/\">Content</a> and <a href=\"AnalysisService/\">Analysis</a> Services. The <a href=\"tools/reactome-fiviz\">ReactomeFIViz app</a> for Cytoscape provides tools to find pathways and network patterns related to cancer and other types of diseases.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt; font-family: helvetica;\"><strong>Documentation and Training</strong>. Visit our online <a href=\"userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"community/training\">here</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt; font-family: helvetica;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https:\">website</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt; font-family: helvetica;\"><strong>Follow us on Twitter:</strong> <a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt; font-family: helvetica;\"><strong>For more information</strong>: If you have a question to ask or would like to give us your feedback, please contact our <span id=\"cloak3c9b6732f53c185a2fdfae327f5e52d3\"><span id=\"cloakb0e4e4ab26f934c647cd429c106e907e\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span></span>.</span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-03-13 01:38:36','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,32,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(148,799,'Privacy Notice','privacy-notice','<p><em>March 6, 2024</em></p>\r\n<p>Reactome is an open-source, open access, manually curated and peer-reviewed pathway database. The Reactome website is administered by the Ontario Institute for Cancer Research.   This Privacy Notice sets out how we collect, store and use your personal information that you provide when you use our website or contact us. </p>\r\n<p><strong>Collection and Use</strong></p>\r\n<p>Aggregate information is information from which individual user identities or characteristics have been removed. When you browse or download information from the Reactome website, our servers automatically collect limited amounts of standard information for traffic monitoring and statistical purposes (e.g. browser version, pages visited, dates and times of access, etc.). The information is analyzed for operational trends, performance, ways to improve our site, and for the day-to-day administration of Reactome. None of this information is analyzed in a personally identifiable manner.</p>\r\n<p>Personal information is any information about an identifiable individual. When we collect personal information about you through the website, we do so only if you consent and provide it to us. Specifically, there are four ways we may collect and use personal information about you:</p>\r\n<ol start=\"1\" type=\"1\">\r\n<li>if you agree to act as a reviewer or author of a Reactome pathway, we collect your reviewer or author information (e.g. name, ORCID ID, email address and institutional affiliation) for the purpose of credit attribution on the published pathway;  </li>\r\n</ol><ol start=\"2\">\r\n<li>if you contact the Reactome help desk, we collect your name, email address and information related to your request, question or concern for the purposes of responding and, if applicable, troubleshooting;</li>\r\n</ol><ol start=\"3\">\r\n<li>if you provide your email address when using the Reactome GSA analysis service, we use it to notify you when your analysis is complete; and/or</li>\r\n</ol><ol start=\"4\">\r\n<li>if you register to join our mailing list, we collect your email address in order to send you our publications and information about events. You may unsubscribe from receiving any communication from Reactome at any time by sending a request to our <a href=\"mailto:help@reactome.org\">HelpDesk</a> or by using the unsubscribe option provided in the email you have received.</li>\r\n</ol>\r\n<p><strong>Who will have access to your personal information?</strong></p>\r\n<p>Personal information may only be accessed by authorized staff at OICR and at other institutions that are contractually bound to help administer the Reactome database. All such access is provided only as necessary for the purposes described in this Privacy Notice.</p>\r\n<p>Please note that authorized staff may be located outside Canada, including in the United States and Europe, and therefore, your personal information may become subject to the laws of those countries.</p>\r\n<p><strong>How long do we keep your personal information?</strong></p>\r\n<p>We will keep the personal information only as long as necessary for the fulfilment of the purposes for which it was collected and for which consent was received, unless otherwise authorized or required by law.</p>\r\n<p><strong>Links to Third Party Websites </strong></p>\r\n<p>Our website contains links from and to other websites. Please note that our Privacy Notice applies to our website only and not other linked websites. If you click on a link to other websites, it is incumbent on you to read the applicable Privacy Notice.</p>\r\n<p><strong>Controlling your personal information</strong></p>\r\n<p>To access, correct, delete or request that your Personal Information be sent to you or someone else, please contact us at our <a href=\"mailto:help@reactome.org\">HelpDesk</a>.</p>\r\n<p><strong>How to contact us?</strong></p>\r\n<p>If you have any queries related to our data privacy practices or this Privacy Notice, feel free to contact us at our <a href=\"mailto:help@reactome.org\">HelpDesk</a>.</p>\r\n<p>.</p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-03-19 03:18:15','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(150,801,'New Paper published in eLife','new-paper-published-in-elife','<p class=\"p1\"><span class=\"s1\"><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/wikidata_reactome.jpg\" alt=\"wikidata reactome\" width=\"650\" height=\"460\" /></span></p>\r\n<p class=\"p1\" style=\"text-align: justify;\"><span class=\"s1\">A new&nbsp;research article entitled “<a href=\"https://elifesciences.org/articles/52614\">Science Forum: Wikidata as a knowledge graph for the life sciences</a>” has been published in the <a href=\"https://elifesciences.org/\">eLife</a> journal. The paper describes the&nbsp;use of Wikidata as a platform for the integration of biological knowledge. </span><span class=\"s2\">Over 2,200 pathways from Reactome were integrated into the Wikidata repository.</span><span class=\"s1\"> More publications from the&nbsp;<a href=\"about/team\"><span class=\"s3\">Reactome Team</span></a>&nbsp;can be found&nbsp;<a href=\"community/publications/\"><span class=\"s3\">here</span></a>.</span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-04-06 16:48:52','0000-00-00 00:00:00','{}','{}','{}',1,31,'','',1,0,'{}',0,'en-GB','',''),(151,803,'New reactome2py package links our web services and data with python.','new-package-facilitates-communication-between-reactome-services-and-data-with-python','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/reactome2py_package.jpeg\" alt=\"reactome2py package\" width=\"650\" height=\"488\" /></p>\r\n<p style=\"text-align: justify;\"><span class=\"s1\">Data science represents a new and evolving way of doing scientific research with critical elements including&nbsp;<span class=\"s1\">reproducibility, </span>open access to data, methods and source code, and&nbsp;<span class=\"s1\">highly reusable and modular services on the web.</span>&nbsp;</span></p>\r\n<p style=\"text-align: justify;\"><span class=\"s2\">Given the popularity of the Python programming language among data scientists, we have created the <a href=\"https://reactome.github.io/reactome2py/\">reactome2py package</a>, which facilitates communication between our tools and web services with Python. </span><span class=\"s1\">The reactome2py package consists of a library of helper functions that wrap calls to the Reactome RESTful API and a utility module that simplifies access to our open-data.<span class=\"Apple-converted-space\">&nbsp; </span></span></p>\r\n<p style=\"text-align: justify;\"><span class=\"s1\">The <a href=\"/dev/analysis\">Pathway Analysis Service</a> pathway over-representation and expression analysis as well as a species comparison tool. Further details regarding the AnalysisService API calls are available <a href=\"/AnalysisService/\">here</a></span>. <span class=\"s4\">The <a href=\"/dev/content-service\">Content Service</a> provides access to our data via an easy&nbsp;<a href=\"/ContentService/\"><span class=\"s5\">API</span></a>&nbsp;based on the Representational State Transfer (REST) protocol.&nbsp;</span><span class=\"s6\"> Finally, </span><span class=\"s7\">Utility (<span class=\"s1\"><a href=\"https://reactome.github.io/reactome2py/utils.html\">Utils</a></span>) provides functions for fetching pathway, drug, and drug-target data, other annotations, mapping information, and overlay data from human networks. </span><span class=\"s8\">Further details regarding the </span><span class=\"s9\">Data model key classes are available <a href=\"/documentation/data-model\"><span class=\"s1\">here</span></a>. </span><span class=\"s10\">Explore our tools and web services and learn how to include them in your applications at our <span class=\"s11\"><a href=\"/dev\">Developer\'s Zone</a>.</span></span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-04-06 17:59:35','0000-00-00 00:00:00','{}','{}','{}',1,30,'','',1,0,'{}',0,'en-GB','',''),(152,810,'User Guide Revised','userguide-toc','','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-04-22 10:05:48','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',2,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(153,812,'Data Model Glossary','data-model-glossary','','',0,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',3,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(154,813,'New Paper published in Autophagy','new-paper-published-in-autophagy','<p><img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"images/news/Autophagy.png\" alt=\"Autophagy\" width=\"650\" height=\"346\" />A new&nbsp;research article entitled “<a href=\"https://doi.org/10.1080/15548627.2020.1761659\">Using Reactome to build an autophagy mechanism knowledgebase</a>” has been published in the <a href=\"https://www.tandfonline.com/toc/kaup20/current\">Autophagy</a>&nbsp;journal. The paper discusses the curation and annotation of the molecular mechanisms of autophagy.&nbsp;The autophagy pathway can be viewed <a href=\"/PathwayBrowser/#/R-HSA-9612973\">here</a>. More publications from the&nbsp;<a href=\"about/team\">Reactome Team</a>&nbsp;can be found&nbsp;<a href=\"community/publications/\">here</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-06-11 19:40:50','0000-00-00 00:00:00','{}','{}','{}',1,29,'','',1,0,'{}',0,'en-GB','',''),(155,814,'Version 73 Released','version-73-released','<p><span style=\"font-family: helvetica; font-size: 10pt;\"><img src=\"images/news/SARS-CoV-1.png\" alt=\"SARS-CoV-1\" width=\"650\" height=\"357\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>New Feature. </strong>In response to the COVID-19 pandemic, Reactome is fast-tracking the annotation of Human Coronavirus infection pathways in collaboration with the <a href=\"https://covid.pages.uni.lu/\">COVID-19 Disease Map</a> group. V73 includes descriptions of the <a href=\"PathwayBrowser/#/R-HSA-9678108.1\">SARS-CoV-1 Infection pathway</a> as well as <a href=\"PathwayBrowser/#/R-HSA-9679191.1\">Potential therapeutics for SARS</a>. In the coming months, we will use these descriptions as starting points to annotate the SARS-CoV-2 infection pathway and interactions with host proteins that affect the severity of COVID-19 disease.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>New and Updated Pathways. </strong>Other topics with new or revised pathways in V73 include Cell Cycle (<a href=\"PathwayBrowser/#/R-HSA-9670095.1\">Inhibition of DNA recombination at telomeres</a>, <a href=\"PathwayBrowser/#/R-HSA-157579.4\">Telomere Maintenance</a>), Disease (<a href=\"PathwayBrowser/#/R-HSA-9687139.1\">Aberrant regulation of mitotic cell cycle due to RB1 defects</a>, <a href=\"PathwayBrowser/#/R-HSA-9651496.1\">Defects of contact activation system (CAS) and kallikrein/kinin system (KKS)</a>, <a href=\"PathwayBrowser/#/R-HSA-6802957&amp;PATH=R-HSA-1643685,R-HSA-5663202\">Oncogenic MAPK signaling, </a><a href=\"PathwayBrowser/#/R-HSA-9669938.1\">Signaling by KIT in disease</a>, and Signal Transduction (<a href=\"PathwayBrowser/#/R-HSA-5673001.9\">RAF/MAP kinase cascade</a>,<a href=\"PathwayBrowser/#/R-HSA-9648002.1\"> RAS processing</a>).</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>New and Updated Illustrations. </strong>Illustrations with embedded navigation features have been added for<a href=\"PathwayBrowser/#/R-HSA-9675108\"> Nervous system development</a>, <a href=\"PathwayBrowser/#/R-HSA-3296482&amp;PATH=R-HSA-1643685,R-HSA-5668914\">Defects in vitamin and cofactor metabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-5663084&amp;PATH=R-HSA-1643685,R-HSA-5668914\">Diseases of Carbohydrate metabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-5668914&amp;PATH=R-HSA-1643685\">Diseases of Metabolism</a>, and <a href=\"PathwayBrowser/#/R-HSA-9679506&amp;PATH=R-HSA-1643685,R-HSA-5663205\">SARS-CoV infections</a>.  New static illustrations are now available for <a href=\"PathwayBrowser/#/R-HSA-388841\">Costimulation by the CD28 family</a>, <a href=\"PathwayBrowser/#/R-HSA-389356\">CD28 co-stimulation</a>, <a href=\"PathwayBrowser/#/R-HSA-202424\">Downstream TCR signaling</a>, <a href=\"PathwayBrowser/#/R-HSA-69473\">G2/M DNA damage checkpoint</a>, <a href=\"PathwayBrowser/#/R-HSA-69478\">G2/M DNA replication checkpoint</a>, <a href=\"PathwayBrowser/#/R-HSA-202433\">Generation of second messenger molecules</a>, <a href=\"PathwayBrowser/#/R-HSA-69610\">p53-Independent DNA Damage Response</a>, <a href=\"PathwayBrowser/#/R-HSA-202427\">Phosphorylation of CD3 and TCR zeta chains</a>, <a href=\"PathwayBrowser/#/R-HSA-9678108\">SARS-CoV-1 Infection</a>, and <a href=\"PathwayBrowser/#/R-HSA-202430\">Translocation of ZAP-70 to Immunological synapse</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Thanks to our Contributors.</strong><span class=\"apple-converted-space\"> <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Marcio+Luis+Acencio%5BAuthor%5D&amp;sort=\">Marcio Luis Acencio</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Frederick+A+Dick%5BAuthor%5D&amp;sort=\">Frederick A Dick</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Alfonso+Garc%C3%ADa-Valverde%5BAuthor%5D\">Alfonso García-Valverde</a>, <a href=\"http://www.einstein.yu.edu/faculty/12958/evripidis-gavathiotis/\">Evripidis Gavathiotis</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Makoto+T+Hayashi%5BAuthor%5D\">Makoto T Hayashi</a>,<a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Alexander+Mazein%5BAuthor%5D\"> Alexander Mazein</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/31182063/?from_single_result=Daniel+Pilco-Janeta%5BAuthor%5D\">Daniel Pilco-Janeta</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Cesar+Serrano%5BAuthor%5D\">Cesar Serrano</a>, <a href=\"http://www.bkslab.org/overview\">Brian Shoichet</a>, and <a href=\"https://orcid.org/0000-0002-7786-7580\">Bin Zhang</a> are our external reviewers.</span></span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><span class=\"s1\"><span style=\"font-family: helvetica;\"><span style=\"font-family: arial, helvetica, sans-serif;\"><b>Annotation Statistics. </b></span></span><span style=\"font-family: helvetica;\">Reactome comprises </span></span><span class=\"s2\" style=\"font-family: helvetica;\">13,248</span><span class=\"s1\" style=\"font-family: helvetica;\"> human reactions organized into </span><span class=\"s2\" style=\"font-family: helvetica;\">2,423</span><span class=\"s1\" style=\"font-family: helvetica;\"> pathways involving 11,</span><span class=\"s2\" style=\"font-family: helvetica;\">111</span><span class=\"s1\" style=\"font-family: helvetica;\"> proteins and modified forms of proteins encoded by </span><span class=\"s2\" style=\"font-family: helvetica;\">10,923</span><span class=\"s1\" style=\"font-family: helvetica;\"> different human genes, 12,728 complexes, 1,869 small molecules, and 369 drugs. These annotations are supported by </span><span class=\"s3\" style=\"font-family: helvetica;\">32,150</span><span class=\"s1\" style=\"font-family: helvetica;\"> literature references. We have projected these reactions onto </span><span class=\"s2\" style=\"font-family: helvetica;\">81,835</span><span class=\"s1\" style=\"font-family: helvetica;\"> orthologous proteins, creating </span><span class=\"s2\" style=\"font-family: helvetica;\">18,654</span><span class=\"s1\" style=\"font-family: helvetica;\"> orthologous pathways in 15 non-human species. Version 73 has annotations for </span><span class=\"s3\" style=\"font-family: helvetica;\">2,620</span><span class=\"s1\" style=\"font-family: helvetica;\"> protein variants (mutated proteins) and their post-translationally modified forms, derived from 327 proteins, which have been used to annotate disease-specific </span><span class=\"s3\" style=\"font-family: helvetica;\">1,297 </span><span class=\"s1\" style=\"font-family: helvetica;\">reactions and </span><span class=\"s3\" style=\"font-family: helvetica;\">605 </span><span class=\"s1\" style=\"font-family: helvetica;\">pathways.</span></span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Tools and Data. </strong>Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"PathwayBrowser/\">Pathway Browser</a>, to <a href=\"PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"download-data\">download</a>, and to access programmatically through our <a href=\"ContentService/\">Content</a> and <a href=\"AnalysisService/\">Analysis</a> Services. The <a href=\"tools/reactome-fiviz\">ReactomeFIViz app</a> for Cytoscape provides tools to find pathways and network patterns related to cancer and other types of diseases.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Documentation and Training</strong>. Visit our online <a href=\"userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"community/training\">here</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https:\">website</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Follow us on Twitter:</strong> <a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>For more information</strong>: If you have a question to ask or would like to give us your feedback, please contact our <span id=\"cloak3c9b6732f53c185a2fdfae327f5e52d3\"><span id=\"cloakb0e4e4ab26f934c647cd429c106e907e\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span></span>.</span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-06-11 20:56:40','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,28,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(156,815,'ReactomeGSA','gsa','<h4></h4>\r\n<h4>Quantitative, multi-dataset Pathway Analysis (ReactomeGSA)</h4>\r\n<p>ReactomeGSA is a new pathway analysis tool integrated into the Reactome ecosystem. Its main feature is that it performs quantitative pathway analyses (so-called gene set analyses). This increases the statistical power of the differential expression analysis, which is directly performed on the pathway level.</p>\r\n<p>ReactomeGSA can analyse multiple datasets simultaneously resulting in a comparative pathway analysis. Thereby, it is possible to quickly assess whether the same effect was observed in independent experiments or studies.</p>\r\n<p>ReactomeGSA currently supports quantitative proteomics, transcriptomics, and microarray data. Datasets from all of these methods can be combined in a single analysis. Thereby, ReactomeGSA can perform multi-omics pathway analyses.</p>\r\n<h4>Using ReactomeGSA</h4>\r\n<p>We currently offer three ways to access ReactomeGSA:</p>\r\n<ol>\r\n<li>Reactome’s web-based pathway browser (see below)</li>\r\n<li>From R using our ReactomeGSA Bioconductor R package (see <a href=\"https://bioconductor.org/packages/release/bioc/html/ReactomeGSA.html\" target=\"_blank\" rel=\"noopener\">here</a>)</li>\r\n<li>Programmatically, using the ReactomeGSA API at <a href=\"https://gsa.reactome.org\"></a><a href=\"https://gsa.reactome.org\" target=\"_blank\" rel=\"noopener\">https://gsa.reactome.org</a></li>\r\n</ol>\r\n<blockquote>\r\n<h4><strong>Analyze Gene Expression using ReactomeGSA</strong> </h4>\r\n</blockquote>\r\n<p>You can visit ReactomeGSA <a href=\"https://reactome.org/gsa/home\">here</a>. When you click on the “Analyse gene expression” tab of the Analysis Tools, you will be redirected to ReactomeGSA, where you can perform analyses.</p>\r\n<p>The following tutorial will show you how to: navigate to ReactomeGSA, submit data for analysis, annotate your data, and perform your first analysis. This following tutorial shows a simplified analysis workflow.</p>\r\n<p><iframe src=\"https://docs.google.com/presentation/d/e/2PACX-1vS7wHSSEXSYVI4Ur-7_Sdr9FI8n7-yjl5GbI3INwJUXeqBtLIfzKiTRQ4ueL7uzmA/embed?start=false&amp;loop=true&amp;delayms=3000\" width=\"960\" height=\"629\" frameborder=\"0\" allowfullscreen=\"allowfullscreen\"></iframe></p>\r\n<p></p>\r\n<p></p>\r\n<h4>ReactomeGSA Analysis Algorithms </h4>\r\n<p>In this first screen, you need to select the algorithm to use for the differential pathway analysis. At the time of writing, ReactomeGSA offers three algorithms. PADOG and Camera perform a differential expression analysis between two groups of samples. ssGSEA is a so-called gene set variation approach that returns pathway-level quantitative data for each sample.</p>\r\n<p>The detailed parameters for each algorithm can be adapted by clicking the blue icon on the left of the algorithm’s box.</p>\r\n<h4>Adding datasets</h4>\r\n<p>After clicking “Next”, you are presented with the now empty list of datasets. Click the “+ Add dataset” button to add a new dataset.</p>\r\n<p>First, you have to select the type of dataset you want to load.</p>\r\n<p>To upload your own data, select one of the options under “Select a file from a local folder”. The file must be a tab-delimited text file (CSV or TSV file) where the first column contains the gene or protein identifiers and all subsequent columns the respective samples. The first row contains the sample names and all subsequent rows the genes / proteins.</p>\r\n<p>To test the tool, it is possible to quickly load example data. Currently, ReactomeGSA provides three example datasets, two (matched) datasets on melanoma associated B cells (proteomics and transcriptomics measurements) and one scRNA-seq dataset on B cells.</p>\r\n<h4>Annotating experimental metadata</h4>\r\n<p>Once a dataset is added, you need to annotate the experimental metadata. This is necessary in order to define the groups for the differential expression analysis in the next step.</p>\r\n<p>You can adapt the dataset’s name in the “Dataset name” box at the top. This name will be used for all results. To increase the readability, we suggest to use as short names as possible.</p>\r\n<p>In case you load data from ExpressionAtlas or choose one of the example datasets (as shown in the screenshot) the sample annotation table will already be pre-filled with certain metadata. In case you uploaded your own dataset, the table will only show the orange sample labels on the left.</p>\r\n<p>To add an annotation, click the “plus” symbol on the right. This will add a new empty column to the table. First, add a heading to define the name of the property (for example, “treatment”). Next, add the values for every sample that you want to include in your comparisons. Samples without any values will simply be ignored.</p>\r\n<h4>Defining the experimental design</h4>\r\n<p>In the final step of adding a dataset, you have to define which groups to compare. The “comparison factor” drop-down menu contains all parameters that were annotated in the sample table before (if they contain at least two different values). “1st group” and “2nd group” define which groups of samples to compare against each other. The 1st group is the control or baseline. </p>\r\n<p>Depending on which “comparison factor” you select, the available values for the “1st group” and “2nd group” will change automatically.</p>\r\n<p>Additionally, some gene set analysis methods allow you to define so-called “covariates”. These are parameters that might cause a bias in your result (ie. the sequencing facility used) that you would like to correct for. Simply select the relevant ones for your experiment.</p>\r\n<p>Once you click “Continue” you will be returned to the list of datasets where you will now see your annotated dataset in the list. If you want, you can add any number of datasets to a single request.</p>\r\n<h4>Starting the analysis</h4>\r\n<p>“Create REACTOME visualizations” is always selected. If it is de-selected, the result cannot be visualised in Reactome’s pathway browser. This option is generally only relevant to users of the <a href=\"https://bioconductor.org/packages/release/bioc/html/ReactomeGSA.html\" target=\"_blank\" rel=\"noopener\">ReactomeGSA R package</a>.</p>\r\n<p>If you select “Create reports” ReactomeGSA will automatically create a Microsoft Excel and PDF report of your results. Additionally, it will create a short R script that allows you to load your data directly into an R session.</p>\r\n<p>In case you provide your email address, you are automatically notified as soon as the analysis is complete. The mail will contain direct links to the generated reports (if you chose to create them) and a link to the visualisation in the PathwayBrowser.</p>\r\n<p>Launch the analysis by clicking the “GO” button.</p>\r\n<p>If you provided an email address, you can also close your browser. The analysis will still continue on our servers and you will be notified as soon as it is done. Some analysis with many large datasets (for example comparing five TCGA datasets in a single analysis) may require up to an hour to complete.</p>\r\n<p></p>\r\n<blockquote>\r\n<h4>Loading Public Datasets</h4>\r\n</blockquote>\r\n<p>The following tutorial will show you how to: load public data for analysis from EBI Expression Atlas, EBI Single Cell Expression Atlas, GREIN and GEO data. </p>\r\n<p><iframe src=\"https://docs.google.com/presentation/d/e/2PACX-1vQlrliw-2HHwAOOTltJsQ-gyeAEQwo1tDOj2kLrnf5SZB-FCOIk4jc1-zvnC5D3pw/embed?start=false&amp;loop=true&amp;delayms=3000\" width=\"960\" height=\"629\" frameborder=\"0\" allowfullscreen=\"allowfullscreen\"></iframe></p>\r\n<p>Finally, it is possible to directly load datasets from several sources.</p>\r\n<ol>\r\n<li>ExpressionAtlas. To do so, first navigate to ExpressionAtlas (in a separate tab or window) at <a href=\"https://www.ebi.ac.uk/gxa\"></a><a href=\"https://www.ebi.ac.uk/gxa\" target=\"_blank\" rel=\"noopener\">https://www.ebi.ac.uk/gxa</a>. Once you have identified a dataset of interest, you can get the dataset’s id by opening the dataset and copying the portion after “<a href=\"http://www.ebi.ac.uk/\" target=\"_blank\" rel=\"noopener\">www.ebi.ac.uk/</a>” and before the next “/”. For example, if you opened <a href=\"https://www.ebi.ac.uk/gxa/experiments/E-MTAB-6592/Results\"></a><a href=\"https://www.ebi.ac.uk/gxa/experiments/E-MTAB-6592/Results\" target=\"_blank\" rel=\"noopener\">https://www.ebi.ac.uk/gxa/experiments/E-MTAB-6592/Results</a> the identifier of this dataset would be “E-MTAB-6592”.<br /><br /></li>\r\n<li>Single Cell Expression Atlas. For single-cell experiments you additionally have to define the parameter “k” in order to define which clusters should be used. The effect of “k” on the results can be visualised on the first page of the respective single cell experiment in ExpressionAtlas (see <a href=\"https://www.google.com/url?q=https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7008/results/&amp;sa=D&amp;source=docs&amp;ust=1704903663981717&amp;usg=AOvVaw07jZb6GFrx9exWFVK5YO1W\" target=\"_blank\" rel=\"noopener\" data-rawhref=\"https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7008/results/\"></a><a href=\"https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7008/results/\">https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7008/results/</a>). In this case, the dataset identifier would be “E-MTAB-7008\" and a possible value for “k” would be 10. <br /><br /></li>\r\n<li>GREIN data. The accession ID is provided in the URL. </li>\r\n<li>GEO Microarray data. After choosing the Pathway algorithm the user can choose public datasets in the \"Public datasets\" panel and the \"GEO query\" section to fetch public microarray data via the GEO accession id. e.g. GSE1563. In the process the data is fetched directly from the GEO database and can be annotated, downloaded and analysed within the ReactomeGSA workflow. </li>\r\n</ol>\r\n<blockquote>\r\n<h4>Citing ReactomeGSA</h4>\r\n</blockquote>\r\n<h4><strong>If you use ReactomeGSA in your research, please cite the following publication:</strong></h4>\r\n<p>ReactomeGSA - Efficient Multi-Omics Comparative Pathway Analysis</p>\r\n<p>Johannes Griss, Guilherme Viteri, Konstantinos Sidiropoulos, Vy Nguyen, Antonio Fabregat, Henning Hermjakob</p>\r\n<p><em>Mol Cell Proteomics.</em> 2020 Dec; 19(12): 2115–2124; <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7710148/\">Link</a></p>\r\n<p></p>\r\n<h4><strong>If you use ReactomeGSA\'s data integration, please cite the following publication:</strong></h4>\r\n<p>ReactomeGSA: new features to simplify public data reuse</p>\r\n<p>Alexander Grentner, Eliot Ragueneau, Chuqiao Gong, Adrian Prinz, Sabina Gansberger, Inigo Oyarzun, Henning Hermjakob, Johannes Griss</p>\r\n<p><em>Bioinformatics</em>, Volume 40, Issue 6, June 2024; <a href=\"https://doi.org/10.1093/bioinformatics/btae338\">Link</a> </p>\r\n<p></p>\r\n<p></p>\r\n<p></p>\r\n<p></p>\r\n<p></p>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-07-20 13:01:31','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(157,816,'User Guide','userguide-f','','',0,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00','{}','{}','{}',1,2,'','',2,0,'{}',0,'en-GB','',''),(158,818,'COVID-19 Disease Pathways','covid-19','<p style=\"text-align: justify;\"><strong>Rapid and Precise Molecular Pathway Modeling of the SARS-CoV-1 and SARS-CoV-2 Infection Cycle with Human Host Protein and Therapeutic Interactions</strong></p>\r\n<p style=\"text-align: justify;\">In response to the COVID-19 pandemic, Reactome fast-tracked the annotation of Human Coronavirus infection pathways in collaboration with the COVID-19 Disease Map group. We have described the molecular annotations of the COVID-19 infection process mediated by the SARS-CoV-2 coronavirus, interactions between viral components and human host proteins that mediate the severity of viral infection, and the effects of therapeutics and drug-like compounds on both viral and host proteins. The SARS-CoV-2 pathway annotations will provide a framework for pathway- and network-based data analysis and visualization, which will be critical for interpreting numerous COVID-19 studies now and in the future. </p>\r\n<p style=\"text-align: justify;\">In collaboration with a team of community experts in virology, drug design, and infectious disease, we assembled the information in two stages. First, a draft annotation associated relevant SARS-CoV-1 and SARS-CoV-2 viral and host cell proteins with each stage of the infection process and the host\'s response. These annotations will be immediately useful for identifying additional relevant interacting proteins, for assessing possible effects of variation in the host or viral proteins on specific steps of viral infection, and for identifying possible drug targets. In the second stage, the SARS-CoV-2 map was annotated more extensively to fill in molecular details of each step in these processes highlights differences in the processes mediated by SARS-CoV-2 virus and related coronaviruses. Reactome will continue to incorporate newly validated molecular details as they are uncovered by the research community.</p>\r\n<p style=\"text-align: justify;\">All the data, code, and tools developed within this COVID Disease pathways project will be open source and open access, and freely available for use and re-use to support clinical and translational research projects.</p>\r\n<p style=\"text-align: justify;\">This accelerated annotation project was supported by an administrative supplement grant U41 HG003751-13S1 from the National Human Genome Research Institute.</p>','',1,34,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-09-18 16:41:57','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','','');
INSERT INTO `rlp_content` VALUES (161,823,'COVID-19: SARS-CoV-2 infection pathway Released','version-74-released','<p><span style=\"font-size: 10pt;\"><img src=\"images/news/sars-cov-2-pwy-72.png\" alt=\"SARS-CoV-2\" width=\"650\" height=\"379\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></span></p>\r\n<!-- here -->\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt;\"><strong>New Feature. In response to the COVID-19 pandemic, Reactome is fast-tracking the annotation of Human Coronavirus infection pathways in collaboration with the COVID-19 Disease Map group. </strong>Reactome release 74 features the <a href=\"PathwayBrowser/#/R-HSA-9694516&amp;PATH=R-HSA-1643685,R-HSA-5663205,R-HSA-9679506\">SARS-CoV-2 (COVID-19) infection pathway</a>. To generate this pathway, we started with a rough draft generated computationally via <a href=\"documentation/inferred-events\">our orthoinference process</a> from the previously manually curated, peer-reviewed Reactome SARS-CoV-1 (Human SARS coronavirus) infection pathway. The draft SARS-CoV-2 (COVID-19) infection pathway events and entities were then reviewed by Reactome curators and curated using published SARS-CoV-2 experimental data. Before finalization, the resulting curated SARS-CoV-2 infection pathway was peer-reviewed by external experts. The current pathway consists of 101 reactions involving 489 molecular entities (279 proteins, 12 RNAs and 198 other), and is supported by citations to 227 publications. In future releases of Reactome, we will extend our annotation of the pathway, indicate drugs and other compounds that modulate steps in infection, and add molecular events that link infectious pathway steps to host immune processes and other aspects of human biology that determine responses to viral infection. This accelerated annotation project is supported by a recently-received supplement  grant U41 HG003751-13S1 from the National Human Genome Research Institute.</span></p>\r\n\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt;\"><strong>New and Updated Pathways. </strong>Other topics with new or revised pathways in release 74 include Disease (<a href=\"PathwayBrowser/#/R-HSA-9693928\">Defective RIPK1-mediated regulated necrosis</a>) and Programmed cell death (<a href=\"PathwayBrowser/#/R-HSA-5213460\">RIPK1-mediated regulated necrosis</a>).</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt;\"><strong>New and Updated Illustrations. </strong>Illustrations with embedded navigation features have been added or revised for <a href=\"PathwayBrowser/#/R-HSA-5668914&amp;PATH=R-HSA-1643685\">Diseases of metabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-9679506\">SARS-CoV infections</a>, <a href=\"PathwayBrowser/#/R-HSA-5619084&amp;PATH=R-HSA-1643685,R-HSA-5619115\">ABC transporter disorders</a>,<a href=\"PathwayBrowser/#/R-HSA-5687613&amp;PATH=R-HSA-1643685,R-HSA-5668914\"> Diseases associated with surfactant metabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-9675132&amp;PATH=R-HSA-1643685\">Diseases of cellular response to stress</a>, <a href=\"PathwayBrowser/#/R-HSA-9675135&amp;PATH=R-HSA-1643685\">Diseases of DNA repair</a>, <a href=\"PathwayBrowser/#/R-HSA-9675126&amp;PATH=R-HSA-1643685\">Diseases of mitotic cell cycle</a>, <a href=\"PathwayBrowser/#/R-HSA-9675143&amp;PATH=R-HSA-1643685\">Diseases of neuronal system</a>, <a href=\"PathwayBrowser/#/R-HSA-5663202&amp;PATH=R-HSA-1643685\">Diseases of signal transduction by growth factor receptors and second messengers</a>, <a href=\"PathwayBrowser/#/R-HSA-9675151&amp;PATH=R-HSA-1643685\">Disorders of developmental biology</a>, <a href=\"PathwayBrowser/#/R-HSA-5619115&amp;PATH=R-HSA-1643685\">Disorders of transmembrane transporters</a>, <a href=\"PathwayBrowser/#/R-HSA-3304351&amp;PATH=R-HSA-1643685,R-HSA-5663202\">Signaling by TGF-beta receptor complex in cancer</a>, and <a href=\"PathwayBrowser/#/R-HSA-5619102&amp;PATH=R-HSA-1643685,R-HSA-5619115\">SLC transporter disorders</a>. New static illustrations are now available for <a href=\"PathwayBrowser/#/R-HSA-166658\">Complement cascade</a>, <a href=\"PathwayBrowser/#/R-HSA-194840\">Rho GTPase cycle</a>, <a href=\"PathwayBrowser/#/R-HSA-9694516\">SARS-CoV-2 infection</a>, and <a href=\"PathwayBrowser/#/R-HSA-168898\">Toll-Like receptors cascades</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-size: 10pt;\"><strong>Thanks to our Contributors. </strong>Our external author is <span style=\"font-size: 10pt;\"><a href=\"https://orcid.org/0000-0002-7967-6307\">Andrea Senff-Ribeiro</a> and o</span>ur external reviewers are <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Marcio+Luis+Acencio%5BAuthor%5D&amp;sort=\">Marcio Luis Acencio, </a><a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Najoua+Lalaoui%5BAuthor%5D&amp;sort=date\">Najoua Lalaoui</a>, <a href=\"https://www.wehi.edu.au/people/james-murphy/1216/james-murphy-publications\">James M Murphy</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><span class=\"s1\"><span style=\"font-family: helvetica;\"><span style=\"font-family: arial, helvetica, sans-serif;\"><b>Annotation Statistics. </b></span></span><span style=\"font-family: helvetica;\">Reactome comprises </span></span><span class=\"s2\" style=\"font-family: helvetica;\">13,416</span><span class=\"s1\" style=\"font-family: helvetica;\"> human reactions organized into </span><span class=\"s2\" style=\"font-family: helvetica;\">2,441 </span><span class=\"s1\" style=\"font-family: helvetica;\">pathways involving 11,</span><span class=\"s2\" style=\"font-family: helvetica;\">110</span><span class=\"s1\" style=\"font-family: helvetica;\"> proteins and modified forms of proteins encoded by </span><span class=\"s2\" style=\"font-family: helvetica;\">10,922</span><span class=\"s1\" style=\"font-family: helvetica;\"> different human genes, 12,976 complexes, 1,854 small molecules, and 428 drugs. These annotations are supported by 32,297</span><span class=\"s1\" style=\"font-family: helvetica;\"> literature references. We have projected these reactions onto 79,190</span><span class=\"s1\" style=\"font-family: helvetica;\"> orthologous proteins, creating 18,462</span><span class=\"s1\" style=\"font-family: helvetica;\"> orthologous pathways in 15 non-human species. Version 74 has annotations for </span><span class=\"s3\" style=\"font-family: helvetica;\">2,624</span><span class=\"s1\" style=\"font-family: helvetica;\"> protein variants (mutated proteins) and their post-translationally modified forms, derived from 328 proteins, which have been used to annotate disease-specific</span><span class=\"s3\" style=\"font-family: helvetica;\"></span><span class=\"s1\" style=\"font-family: helvetica;\">reactions and </span><span class=\"s1\" style=\"font-family: helvetica;\">pathways.</span></span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Tools and Data. </strong>Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"PathwayBrowser/\">Pathway Browser</a>, to <a href=\"PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"download-data\">download</a>, and to access programmatically through our <a href=\"ContentService/\">Content</a> and <a href=\"AnalysisService/\">Analysis</a> Services. The <a href=\"tools/reactome-fiviz\">ReactomeFIViz app</a> for Cytoscape provides tools to find pathways and network patterns related to cancer and other types of diseases.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Documentation and Training</strong>. Visit our online <a href=\"userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"community/training\">here</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>About the Reactome Project</strong>. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https:\">website</a>.</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>Follow us on Twitter:</strong> <a href=\"http://twitter.com/reactome\" data-cke-saved-href=\"http://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</span></p>\r\n<p style=\"text-align: justify;\"><span style=\"font-family: helvetica; font-size: 10pt;\"><strong>For more information</strong>: If you have a question to ask or would like to give us your feedback, please contact our <span id=\"cloak3c9b6732f53c185a2fdfae327f5e52d3\"><span id=\"cloakb0e4e4ab26f934c647cd429c106e907e\"><a href=\"mailto:help@reactome.org\" data-cke-saved-href=\"mailto:help@reactome.org\">helpdesk</a></span></span>.</span></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-09-21 20:18:18','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,27,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(162,826,'New Paper published in Molecular & Cellular Proteomics','new-paper-published-in-molecular-cellular-proteomics','<p><span style=\"background-color: #808080;\"><img src=\"images/news/ReactomeGSA.png\" alt=\"ReactomeGSA\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></span></p>\r\n<p>A new&nbsp;research article entitled “<a href=\"https://doi.org/10.1074/mcp.TIR120.002155\">ReactomeGSA - Efficient Multi-Omics Comparative Pathway Analysis</a>” has been published in the&nbsp;<a href=\"https://www.mcponline.org/\">Molecular &amp; Cellular Proteomics</a>&nbsp;journal. The paper discusses the novel ReactomeGSA resource for comparative pathway analyses of multi-omics datasets using our existing web interface and a novel R Bioconductor package with explicit support for scRNA-seq data.&nbsp;The ReactomeGSA tools are described&nbsp;<a href=\"/userguide/analysis/gsa\">here</a>. More publications from the&nbsp;<a href=\"https://reactome.org/about/team\">Reactome Team</a>&nbsp;can be found&nbsp;<a href=\"https://reactome.org/community/publications/\">here</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-10-20 15:16:35','0000-00-00 00:00:00','{}','{}','{}',1,26,'','',1,0,'{}',0,'en-GB','',''),(163,828,'Version 75 Released','version-75-released','<p style=\"text-align: justify;\"><img src=\"images/news/FLT3_Signaling.png\" alt=\"FLT3 Signaling\" width=\"650\" height=\"399\" /></p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Pathways</strong>. Topics with new or revised pathways in release 75 include Developmental biology (<a href=\"PathwayBrowser/#/R-HSA-9690406\">Transcriptional regulation of testis differentiation</a>), Disease (<a href=\"PathwayBrowser/#/R-HSA-9006821\">Alternative lengthening of telomeres</a>, <a href=\"PathwayBrowser/#/R-HSA-9699150\">Defective DNA double strand break response due to BARD1 loss of function</a>, <a href=\"PathwayBrowser/#/R-HSA-9663199\">Defective DNA double strand break response due to BRCA1 loss of function</a>, <a href=\"PathwayBrowser/#/R-HSA-9682385\">Signaling by FLT3 in disease</a>), DNA Repair (<a href=\"PathwayBrowser/#/R-HSA-5693565\">Recruitment and ATM-mediated phosphorylation of repair and signaling proteins at DNA double strand breaks</a>), Immune system (<a href=\"PathwayBrowser/#/R-HSA-9607240\">FLT3 signaling</a>), and Signal transduction (<a href=\"PathwayBrowser/#/R-HSA-9706574\">RHOBTB GTPase cycle</a>).</p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Illustrations.</strong> Illustrations with embedded navigation features have been added or revised for <a href=\"PathwayBrowser/#/R-HSA-1266738\">Developmental biology</a>, <a href=\"PathwayBrowser/#/R-HSA-2474795\">Diseases associated with visual transduction</a>, <a href=\"PathwayBrowser/#/R-HSA-9675135\">Diseases of DNA repair</a>,<a href=\"PathwayBrowser/#/R-HSA-9671793\"> Diseases of hemostasis</a>, <a href=\"PathwayBrowser/#/R-HSA-5260271\">Diseases of the immune system</a>, <a href=\"PathwayBrowser/#/R-HSA-9675126\">Diseases of mitotic cell cycle</a>, <a href=\"PathwayBrowser/#/R-HSA-5663202\">Diseases of signal transduction by growth factor receptors and second messengers</a>, <a href=\"PathwayBrowser/#/R-HSA-9682385\">FLT3 signaling in disease</a>, <a href=\"PathwayBrowser/#/R-HSA-5387390\">Hh mutants abrogate ligand secretion</a>, <a href=\"PathwayBrowser/#/R-HSA-2219528\">PI3PK/AKT signaling in cancer</a>, <a href=\"PathwayBrowser/#/R-HSA-9663891\">Selective autophagy</a>, <a href=\"PathwayBrowser/#/R-HSA-1227990\">Signaling by ERBB2 in cancer</a>, <a href=\"PathwayBrowser/#/R-HSA-9671555\">Signaling by PDGFR in disease</a>, <a href=\"PathwayBrowser/#/R-HSA-4791275\">Signaling by WNT in cancer</a>,  and <a href=\"PathwayBrowser/#/R-HSA-5339562\">Uptake and actions of bacterial toxins</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors.</strong> Our external authors are <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Baer+RJ&amp;cauthor_id=29426838\">Richard J Baer</a> and <a href=\"https://www.hyms.ac.uk/about/people/francisco-rivero-crespo\">Francisco Rivero Crespo</a>.  Our external reviewers are <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Baer+RJ&amp;cauthor_id=29426838\">Richard J Baer</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Kenya+Imaimatsu&amp;sort=\">Kenya Imaimatsu</a>, <a href=\"https://orcid.org/0000-0003-2116-7806\">Yoshiakira Kanai</a>, <a href=\"https://orcid.org/0000-0002-0719-5336\">Julhash Kazi</a>, <a href=\"https://orcid.org/0000-0003-4601-6481\">Alan Meeker</a>, <a href=\"https://orcid.org/0000-0002-6302-6107\">Roger Reddel</a>, and <a href=\"https://www.hyms.ac.uk/about/people/francisco-rivero-crespo\">Francisco Rivero Crespo</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong> Reactome comprises 13,534 human reactions organized into 2,477 pathways involving 11,118 proteins and modified forms of proteins encoded by 10,929 different human genes, 13,210 complexes, 1,854 small molecules, and 414 drugs. These annotations are supported by 32,493 literature references. We have projected these reactions onto 79,333 orthologous proteins, creating 18,510 orthologous pathways in 15 non-human species. Version 75 has annotations for 2,620 protein variants (mutated proteins) and their post-translationally modified forms, derived from 327 proteins, which have been used to annotate disease-specific 1,297 reactions and 605 pathways.</p>\r\n<p style=\"text-align: justify;\"><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"PathwayBrowser/\">Pathway Browser</a>, to <a href=\"PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"download-data\">download</a>, and to access programmatically through our <a href=\"ContentService/\">Content</a> and <a href=\"AnalysisService/\">Analysis</a> Services. The <a href=\"tools/reactome-fiviz\">ReactomeFIViz app</a> for Cytoscape provides tools to find pathways and network patterns related to cancer and other types of diseases.</p>\r\n<p style=\"text-align: justify;\"><strong>Documentation and Training.</strong> Visit our online <a href=\"userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"community/training\">here</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"about/news\">website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong> <a href=\"http://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information:</strong> If you have a question to ask or would like to give us your feedback, please contact our helpdesk.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2020-12-02 18:58:30','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,25,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(164,833,'Version 76 Released','version-76-released','<p><img src=\"images/news/Sensory_processing_of_sound.png\" alt=\"Sensory processing of sound\" width=\"650\" height=\"411\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p style=\"text-align: justify;\">Ne<strong>w and Updated Topics and Pathways.</strong> Topics with new or revised pathways in V76 include Cellular responses to external stimuli (<a href=\"/PathwayBrowser/#/R-HSA-9707564\">Cytoprotection by HMOX1</a> and <a href=\"/PathwayBrowser/#/R-HSA-9707616\">Heme signaling</a>), Disease (<a href=\"/PathwayBrowser/#/R-HSA-9710421\">Defective pyroptosis</a>), Gene Expression (<a href=\"/PathwayBrowser/#/R-HSA-9708296\">tRNA-derived small RNA (tsRNA) biogenesis</a>), Immune system (<a href=\"/PathwayBrowser/#/R-HSA-168928\">DDX58/IFIH1-mediated induction of interferon-alpha/beta</a> and <a href=\"/PathwayBrowser/#/R-HSA-9674555\">Signaling by CSF3 (G-CSF)</a>), Programmed cell death (<a href=\"/PathwayBrowser/#/R-HSA-5620971\">Pyroptosis</a>), Sensory Perception (<a href=\"/PathwayBrowser/#/R-HSA-9659379\">Sensory processing of sound</a>), and Signal transduction (<a href=\"/PathwayBrowser/#/R-HSA-9715370\">Miro GTPase Cycle</a>, <a href=\"/PathwayBrowser/#/R-HSA-9012999\">RHO GTPase Cycle</a>, and <a href=\"/PathwayBrowser/#/R-HSA-9013407\">RHOH GTPase Cycle</a>).</p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Illustrations.</strong> Illustrations with embedded navigation features have been added or revised for <a href=\"/PathwayBrowser/#/R-HSA-2262752\">Cellular responses to stress</a>, <a href=\"/PathwayBrowser/#/R-HSA-1280215\">Cytokine Signaling in Immune system</a>, <a href=\"/PathwayBrowser/#/R-HSA-5668914\">Diseases of Metabolism</a>, <a href=\"/PathwayBrowser/#/R-HSA-5579029\">Metabolic disorders of biological oxidations enzymes</a>, <a href=\"/PathwayBrowser/#/R-HSA-9709957\">Sensory Perception</a>, and <a href=\"/PathwayBrowser/#/R-HSA-9659379\">Sensory processing of sound</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors.</strong> Our external reviewers are <a href=\"https://orcid.org/0000-0002-9316-6039\">Katia Basso</a>, <a href=\"https://orcid.org/0000-0003-4669-985X\">Ines Castro</a>, <a href=\"https://orcid.org/0000-0002-9731-6591\">Peter Dallos</a>, <a href=\"https://orcid.org/0000-0002-4319-0073\">Anindya Dutta</a>, <a href=\"https://orcid.org/0000-0001-5997-8722\">Philippe Fort</a>, <a href=\"https://orcid.org/0000-0001-5764-4200\">David N. Furness</a>, <a href=\"https://orcid.org/0000-0002-6395-6443\">Thirumala-Devi Kanneganti</a>, <a href=\"https://orcid.org/0000-0002-7246-9081\">Carlos Henrique Inacio Ramos</a>, <a href=\"http://schraderlab.weebly.com/\">Michael Schrader</a>, <a href=\"https://orcid.org/0000-0002-9404-7736\">Hans-Uwe Simon</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Somers+J&amp;cauthor_id=33689451\">Julia Somers</a>, <a href=\"https://orcid.org/0000-0001-6495-3351\">Zhangli Su</a>, <a href=\"https://orcid.org/0000-0002-4773-4074\">Ivo P. Touw</a>, <a href=\"https://orcid.org/0000-0002-4967-6998\">Briana Wilson</a>, and <a href=\"https://orcid.org/0000-0003-1349-2713\">Zhibin Zhang</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong> Reactome comprises 13,732 human reactions organized into 2,516 pathways involving 11,362 proteins and modified forms of proteins encoded by 11,073 different human genes, 13,409 complexes, 1,856 small molecules, and 415 drugs. These annotations are supported by 33,453 literature references. We have projected these reactions onto 74,384 orthologous proteins, creating 18,051 orthologous pathways in 15 non-human species. Version 76 has annotations for 2,949 protein variants (mutated proteins) and their post-translationally modified forms, derived from 337 proteins, which have been used to annotate disease-specific 1,550 reactions and 653 pathways.</p>\r\n<p style=\"text-align: justify;\"><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"PathwayBrowser/\">Pathway Browser</a>, to <a href=\"/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"download-data\">download</a>, and to access programmatically through our <a href=\"ContentService/\">Content</a> and <a href=\"AnalysisService/\">Analysis</a> Services. The <a href=\"tools/reactome-fiviz\">ReactomeFIViz app</a> for Cytoscape and the <a href=\"/PathwayBrowser/#/R-HSA-TOOL=AT\">ReactomeGSA</a> package provides tools for multi-omics data analysis.</p>\r\n<p style=\"text-align: justify;\"><strong>Documentation and Training.</strong> Visit our online <a href=\"userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"community/training\">here</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"about/news\">website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong> <a href=\"http://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information:</strong> If you have a question to ask or would like to give us your feedback, please contact our helpdesk.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2021-03-22 03:18:41','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,23,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(165,837,'The Reactome IDG Portal is released','the-reactome-idg-portal-is-released','<p style=\"text-align: justify;\"><img src=\"images/news/IDG_Reactome.jpeg\" alt=\"IDG Reactome\" width=\"640\" height=\"360\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p style=\"text-align: justify;\">As part of the <a href=\"https://druggablegenome.net/CEIT_OHSU\">Cutting Edge Informatics Tools (CEIT)</a> for the <a href=\"https://druggablegenome.net/index\">Illuminating the Druggable (IDG)</a> Program, the Reactome group has built a new tool, the <a href=\"https://idg.reactome.org/\">Reactome IDG Portal</a>, that utilizes the <a href=\"/\">Reactome knowledgebase</a> to systematically illuminate interactions of dark proteins with other proteins and biological entities, allowing evaluation of these understudied proteins via their localizations and potential interactions, and facilitating the design of experiments to test their functions. The portal enables users to:</p>\r\n<ul style=\"text-align: justify;\">\r\n<li>Search any gene and view its location in Reactome’s pathways based on manual annotation or interactions via one-hop pairwise relationships. </li>\r\n<li>Convert biochemical reaction-based diagrams into simple pairwise networks.</li>\r\n<li>View scored interacting pathways based on functional interactions predicted from a random forest model trained with 106 features. </li>\r\n<li>Construct new overlays and visualizations for protein-protein pairwise relationships or drug-target interactions.</li>\r\n<li>Use an extended diagram viewer to visualize protein knowledge levels, and overlay multiple tissue-specific expression values from 19 data sources from <a href=\"http://juniper.health.unm.edu/tcrd/\">Target Central Resource Database (TCRD)</a>.</li>\r\n</ul>\r\n<p style=\"text-align: justify;\"><a href=\"/\">Reactome</a> is a collaboration between groups at the Ontario Institute for Cancer Research, New York University Langone Medical Center, Oregon Health and Science University, and The European Bioinformatics Institute. Reactome data and software are distributed under the terms of the Creative Commons Attribution 4.0 License. A full description of the new and updated content is available on the Reactome website. Follow us on Twitter: <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more.</p>\r\n<p style=\"text-align: justify;\">The <a href=\"https://druggablegenome.net/\">Illuminating the Druggable Genome (IDG) Program</a>, launched by the <a href=\"https://commonfund.nih.gov/\">National Institutes of Health (NIH) Common Fund</a>, is a multidisciplinary project to improve the scientific understanding of understudied members of three key protein families: non-olfactory G-protein-coupled receptors (GPCRs), ion channels and protein kinases. The overall goal of the IDG Program is to catalyze research in areas of biology that are currently understudied but that have high potential to impact human health. Follow us on Twitter: <a href=\"https://twitter.com/druggablegenome\">@DruggableGenome</a> to receive updates about the program.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2021-06-07 17:47:37','0000-00-00 00:00:00','{}','{}','{}',1,22,'','',1,0,'{}',0,'en-GB','',''),(166,839,'Version 77 Released','version-77-released','<p style=\"text-align: justify;\"></p>\r\n<p style=\"text-align: justify;\"><img src=\"images/stats/cellular_stress_72.jpeg\" alt=\"cellular stress 72\" width=\"650\" height=\"488\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p style=\"text-align: justify;\"></p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Topics and Pathways.</strong> Topics with new or revised pathways in V77 include Disease (<a href=\"/PathwayBrowser/#/R-HSA-9701193\">Defective HDR through Homologous Recombination (HRR) Due to PALB2 Loss of Function</a>, <a href=\"/PathwayBrowser/#/9710421\">Defective pyroptosis</a>, <a href=\"/PathwayBrowser/#/R-HSA-9723907\">Loss of Function of TP53 in Cancer</a>, <a href=\"/PathwayBrowser/#/R-HSA-9694631\">Maturation of nucleoprotein</a>, and <a href=\"/PathwayBrowser/#/R-HSA-9700206\">Signaling by ALK in cancer</a>), DNA repair (<a href=\"/PathwayBrowser/#/R-HSA-5693579\">Homologous DNA Pairing and Strand Exchange</a>), Programmed Cell Death (<a href=\"/PathwayBrowser/#/R-HSA-5620971\">Pyroptosis</a>), and Signal transduction (<a href=\"/PathwayBrowser/#/R-HSA-2454202\">Fc epsilon receptor (FCERI) signaling</a>, <a href=\"/PathwayBrowser/#/R-HSA-9013405\">RHOD GTPase Cycle</a>, <a href=\"/PathwayBrowser/#/R-HSA-9035034\">RHOF GTPase Cycle</a>, <a href=\"/PathwayBrowser/#/R-HSA-9013420\">RHOU GTPase Cycle</a>, <a href=\"/PathwayBrowser/#/R-HSA-9013424\">RHOV GTPase Cycle</a>, and <a href=\"/PathwayBrowser/#/R-HSA-201556\">Signaling by ALK</a>).</p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Illustrations.</strong> Illustrations with embedded navigation features have been added or revised for <a href=\"/PathwayBrowser/#/R-HSA-9711123\">Cellular response to chemical stress</a>,  <a href=\"/PathwayBrowser/#/R-HSA-9711097\">Cellular response to starvation</a>, <a href=\"/PathwayBrowser/#/R-HSA-2262752\">Cellular responses to stress</a>, <a href=\"/PathwayBrowser/#/R-HSA-3781865\">Diseases of glycosylation</a>, <a href=\"/PathwayBrowser/#/R-HSA-5663202\">Diseases of Signal Transduction by growth factor receptors and second messengers</a>, <a href=\"/PathwayBrowser/#/R-HSA-162906\">HIV Infection</a>, <a href=\"/PathwayBrowser/#/R-HSA-392499\">Metabolism of proteins</a>, <a href=\"/PathwayBrowser/#/R-HSA-6802957\">Oncogenic MAPK signaling</a>, <a href=\"/PathwayBrowser/#/R-HSA-5218859\">Regulated Necrosis</a>, <a href=\"/PathwayBrowser/#/R-HSA-9700206\">Signaling by ALK in Cancer</a>, <a href=\"/PathwayBrowser/#/R-HSA-1226099\">Signaling by FGFR in Disease</a>, <a href=\"/PathwayBrowser/#/R-HSA-9669938\">Signaling by KIT in Disease</a>, <a href=\"/PathwayBrowser/#/R-HSA-9006934\">Signaling by receptor tyrosine kinases</a>, and <a href=\"/PathwayBrowser/#/R-HSA-194315\">Signaling by Rho GTPases</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors.</strong> Our external reviewers are <a href=\"https://orcid.org/0000-0002-8278-240X\">Marcio Luis Acencio</a>, <a href=\"https://orcid.org/0000-0002-3870-6151\">Mariano Bisbal</a>, <a href=\"https://orcid.org/0000-0001-5566-0864\">Giorgio Inghirami</a>, <a href=\"https://orcid.org/0000-0002-9687-7426\">Anna Niarakis</a>, <a href=\"https://orcid.org/0000-0002-5255-0747\">Helmut Pospiech</a>, <a href=\"https://orcid.org/0000-0002-8434-4171\">Kazuyasu Sakaguchi</a>, <a href=\"https://publons.com/researcher/3381611/mikhail-shepelev/\">Mikhail V Shepelev</a>, <a href=\"http://orcid.org/0000-0002-9562-7791\">Feng Shao</a>, and <a href=\"https://orcid.org/0000-0003-0932-9104\">Robert Winqvist</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong> Reactome comprises 13,827 human reactions organized into 2,536 pathways involving 11,374 proteins and modified forms of proteins encoded by 11,084 different human genes, 13,662 complexes, 1,857 small molecules, and 432 drugs. These annotations are supported by 33,752 literature references. We have projected these reactions onto 78,539 orthologous proteins, creating 18,659 orthologous pathways in 15 non-human species. Version 77 has annotations for 4,464 protein variants (mutated proteins) and their post-translationally modified forms, derived from 347 proteins, which have been used to annotate disease-specific reactions and pathways.</p>\r\n<p style=\"text-align: justify;\"><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"PathwayBrowser/\">Pathway Browser</a>, to <a href=\"/PathwayBrowser/#/TOOL=AT\">analyze</a> your own dataset, to <a href=\"download-data\">download</a>, and to access programmatically through our <a href=\"ContentService/\">Content</a> and <a href=\"AnalysisService/\">Analysis</a> Services. The <a href=\"/PathwayBrowser/#/TOOL=AT\">ReactomeGSA</a> package provides tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p style=\"text-align: justify;\"><strong>Documentation and Training.</strong> Visit our online <a href=\"userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"community/training\">here</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"index.php?option=com_content&amp;view=article&amp;id=11:about-reactome&amp;catid=25\">website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information:</strong> If you have a question to ask or would like to give us your feedback, please contact our helpdesk.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2021-06-09 19:53:12','0000-00-00 00:00:00','{}','{}','{}',1,21,'','',1,0,'{}',0,'en-GB','',''),(167,841,'Reactome Multi-Omics Pathway Analysis webinar reaches record attendance','reactome-multi-omics-pathway-analysis-webinar-reaches-record-attendance','<p style=\"text-align: justify;\"><img src=\"images/news/Reactome_GSA.png\" alt=\"Reactome GSA\" width=\"650\" height=\"602\" style=\"display: block; margin-left: auto; margin-right: auto;\" />On June 2, 2021, Dr. Johannes Griss presented \"<a href=\"https://www.ebi.ac.uk/training/events/guide-multi-omics-pathway-analysis/\">A guide to multiomics pathway analysis</a>\" to a record audience of 572 participants as part of the <a href=\"https://www.ebi.ac.uk/training/on-demand?facets=type:Recorded%20webinar\">EMBL-EBI webinar series</a>. The <a href=\"https://embl-ebi.cloud.panopto.eu/Panopto/Pages/Viewer.aspx?id=34b9ac41-e7c6-46d5-986c-ad3b00c902d6\">seminar recording</a> and <a href=\"https://drive.google.com/file/d/1DjH91Zv_1GyiAMd5Xq3zlnqeKacaK_UX/view\">training materials</a>, which are now available online, provides insight into performing comparative multi-omics pathway analyses using the ReactomeGSA platform. We want to take this opportunity to thank all those that took part in the training webinar. If you are interested in learning more about Reactome or if you have any additional questions, please <a href=\"mailto:help@reactome.org\">email us</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2021-06-21 18:41:09','0000-00-00 00:00:00','{}','{}','{}',1,20,'','',1,0,'{}',0,'en-GB','',''),(168,843,'Neo4j Desktop','neo4j-desktop','<h3>Introduction</h3>\r\n<p>This article is going to describe step by step how to import our great Graph Database dump file into Neo4j Desktop</p>\r\n<h3>Download</h3>\r\n<ul>\r\n<li><a href=\"https://neo4j.com/download/\" target=\"_blank\" rel=\"noopener\">Neo4j Desktop</a></li>\r\n<li><a href=\"/download/current/reactome.graphdb.dump\" download=\"\">Graph Database Dump file</a></li>\r\n</ul>\r\n<h3>Neo4j Desktop</h3>\r\n<p>Install Neo4j Desktop following the <a href=\"https://neo4j.com/docs/desktop-manual/current/installation/#download-installation\" target=\"_blank\" rel=\"noopener\">official documentation website</a>.</p>\r\n<h3>Importing Graph Database</h3>\r\n<ol>\r\n<li>Create new Project\r\n<p><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_1.png\" alt=\"neo4j desktop guide 1\" width=\"600\" height=\"400\" style=\"display: block;\" /></p>\r\n</li>\r\n<li>Add reactome.graphdb.dump\r\n<p><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_3.png\" alt=\"neo4j desktop guide 3\" width=\"600\" height=\"400\" style=\"display: block;\" /></p>\r\n</li>\r\n<li>Click on the Open button next to the file you have added and select \"Create new DBMS from Dump\".\r\n<p><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_7.png\" alt=\"neo4j desktop guide 7\" width=\"600\" height=\"400\" style=\"display: block;\" /></p>\r\n</li>\r\n<li>Choose a name, password and version (preferably &gt;4.x.x)\r\n<p><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_8.png\" alt=\"neo4j desktop guide 8\" width=\"600\" height=\"400\" style=\"display: block;\" /></p>\r\n</li>\r\n<li> Not mandatory, open Neo4j Settings, edit \"dbms.allow_upgrade=true<ol>\r\n<li>This step is only needed if you are using Reactome Graph Database which is not compatible with v4.X.X. </li>\r\n</ol></li>\r\n</ol>\r\n<p style=\"padding-left: 30px;\"><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_9.png\" alt=\"neo4j desktop guide 9\" width=\"600\" height=\"400\" style=\"display: block;\" /></p>\r\n<p style=\"padding-left: 30px;\"><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_10.png\" alt=\"neo4j desktop guide 10\" width=\"600\" height=\"400\" style=\"display: block;\" /></p>\r\n<ol start=\"6\">\r\n<li>Now Click on `Start` and you are ready to explore the Neo4j Browser or Neo4j Bloom. <a href=\"http://localhost:7474\">http://localhost:7474</a> </li>\r\n</ol>\r\n<p style=\"padding-left: 30px;\"><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_17.png\" alt=\"neo4j desktop guide 17\" width=\"593\" height=\"554\" /></p>\r\n<h3>Adding a different version of the Reactome database</h3>\r\n<p>* Only Neo4j &gt;4.x supports multiple DMBS, make sure you installed Neo4j 4.x.x</p>\r\n<ol>\r\n<li>Add reactome.previous.graphdb.dump\r\n<p><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_3.png\" alt=\"neo4j desktop guide 3\" width=\"600\" height=\"400\" style=\"display: block;\" /></p>\r\n</li>\r\n<li>Click on the Open button next to the file you have added and select \"Import dump into existing DBMS\".</li>\r\n<li>Select Reactome and create database \"reactome\"</li>\r\n</ol>\r\n<p style=\"padding-left: 30px;\"><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_16.png\" alt=\"neo4j desktop guide 16\" width=\"602\" height=\"390\" /></p>\r\n<ol>\r\n<li> Open Neo4j Settings\r\n<p><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_9.png\" alt=\"neo4j desktop guide 9\" width=\"600\" height=\"400\" style=\"display: block;\" /></p>\r\n</li>\r\n<li>Not mandatory, open Neo4j Settings, edit \"dbms.allow_upgrade=true<br /><ol>\r\n<li>This step is only needed if you are using Reactome Graph Database which is not compatible with v4.X.X.</li>\r\n</ol>\r\n<p><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_10.png\" alt=\"neo4j desktop guide 10\" width=\"600\" height=\"400\" style=\"display: block;\" /></p>\r\n</li>\r\n<li>If you want this database to be your default, then use its name in the settings file.\r\n<p><img src=\"images/devguide/neo4jdesktop/neo4j_desktop_guide_15.png\" alt=\"neo4j desktop guide 15\" width=\"600\" height=\"400\" style=\"display: block;\" /></p>\r\n</li>\r\n<li>Now Click on `Start` and you are ready to explore the Neo4j Browser or Neo4j Bloom.<ol>\r\n<li><a href=\"http://localhost:7474\">http://localhost:7474</a></li>\r\n</ol></li>\r\n</ol>','',1,29,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2021-07-06 21:13:41','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(169,844,'Version 78 Released','version-78-released','<p style=\"text-align: justify;\"><img src=\"images/news/notch1_cancer.png\" alt=\"notch1 cancer\" width=\"650\" height=\"374\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p style=\"text-align: justify;\"></p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Topics and Pathways.</strong> Topics with new or revised pathways in V78 include Disease (<a href=\"/PathwayBrowser/#/R-HSA-9701192\">Defective HDR through homologous recombination (HRR) due to BRCA1 loss of function</a> and <a href=\"/PathwayBrowser/#/R-HSA-9701193\">Defective HDR through homologous recombination (HRR) due to PALB2 loss of function</a>), DNA repair (<a href=\"/PathwayBrowser/#/R-HSA-9730737\">Alkylating DNA damage induced by chemotherapeutic drugs</a>, <a href=\"/PathwayBrowser/#/R-HSA-9750126\">Drug-induced formation of DNA interstrand crosslinks</a>, and <a href=\"/PathwayBrowser/#/R-HSA-9729902\">Reversible DNA damage induced by alkylating chemotherapeutic drugs</a>), DNA Replication (<a href=\"/PathwayBrowser/#/R-HSA-68616\">Assembly of the ORC complex at the origin of replication</a>), Sensory Perception (<a href=\"/PathwayBrowser/#/R-HSA-381753\">Olfactory signaling</a> and <a href=\"/PathwayBrowser/#/R-HSA-9717189\">Sensory perception of taste</a>), and Signal transduction (<a href=\"/PathwayBrowser/#/R-HSA-9013148\">CDC42 GTPase Cycle</a>, <a href=\"/PathwayBrowser/#/R-HSA-9734091\">Drug-mediated inhibition of MET activation</a>, <a href=\"/PathwayBrowser/#/R-HSA-6806942\">Met Receptor Activation</a>, <a href=\"/PathwayBrowser/#/R-HSA-5627083\">RHO GTPases regulate CFTR trafficking</a>, <a href=\"/PathwayBrowser/#/R-HSA-9013409\">RHOJ GTPase cycle</a>, <a href=\"/PathwayBrowser/#/R-HSA-9013406\">RHOQ GTPase cycle</a>, and <a href=\"/PathwayBrowser/#/R-HSA-2173789\">TGF-beta receptor signaling activates SMADs</a>).</p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Illustrations.</strong> Illustrations with embedded navigation features have been added or revised for <a href=\"/PathwayBrowser/#/R-HSA-9734009\">Defective intrinsic pathway to apoptosis</a>, <a href=\"/PathwayBrowser/#/R-HSA-3323169\">Defects in biotin (Btn) metabolism</a>, <a href=\"/PathwayBrowser/#/R-HSA-3296469\">Defects in cobalamin (B12) metabolism</a>, <a href=\"/PathwayBrowser/#/R-HSA-9651496\">Defects of contact activation system (CAS) and kallikrein/kinin system (KKS)</a>, <a href=\"/PathwayBrowser/#/R-HSA-3906995\">Diseases associated with O-glycosylation of proteins</a>, <a href=\"/PathwayBrowser/#/R-HSA-3781860\">Diseases associated with N-glycosylation of proteins</a>, <a href=\"/PathwayBrowser/#/R-HSA-5602358\">Diseases associated with the TLR signaling cascade</a>, <a href=\"/PathwayBrowser/#/R-HSA-9605308\">Diseases of base excision repair</a>, <a href=\"/PathwayBrowser/#/R-HSA-9645723\">Diseases of programmed cell death</a>, <a href=\"/PathwayBrowser/#/R-HSA-69306\">DNA replication</a>, <a href=\"/PathwayBrowser/#/R-HSA-3304349\">Loss of function of SMAD2/3 in cancer</a>, <a href=\"/PathwayBrowser/#/R-HSA-2206281\">Mucopolysaccharidoses</a>, <a href=\"/PathwayBrowser/#/R-HSA-6791465\">Pentose phosphate pathway disease</a>, <a href=\"/PathwayBrowser/#/R-HSA-9709957\">Sensory perception</a>, <a href=\"/PathwayBrowser/#/R-HSA-2644603\">Signaling by NOTCH1 in cancer</a>, and <a href=\"/PathwayBrowser/#/R-HSA-9716542\">Signaling by Rho GTPases, Miro GTPases and RHOBTB3</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors.</strong> Our external authors are <a href=\"https://orcid.org/0000-0002-4167-404X\">Jelena Kusic-Tisma</a>, <a href=\"https://orcid.org/0000-0003-0693-9994\">Sisira Kadambat Nair</a>. Our external reviewers are <a href=\"https://orcid.org/0000-0002-7779-8036\">Kaja Blagotinšek Cokan</a>, <a href=\"http://daslabpombe.com\">Maitreyi E Das</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Jiang+P&amp;cauthor_id=33824185\">Peihua Jiang</a>, <a href=\"https://orcid.org/0000-0002-4403-7169\">Jean-Yves Masson</a>, <a href=\"https://orcid.org/0000-0003-0834-4249\">Larissa Milano</a>, <a href=\"https://orcid.org/0000-0002-6958-4759\">Gemma Montalban</a>, <a href=\"https://orcid.org/0000-0003-0693-9994\">Sisira Kadambat Nair</a>, and <a href=\"https://orcid.org/0000-0002-6891-3178\">Francesco Raimondi</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong> Reactome comprises 13,890 human reactions organized into 2,546 pathways involving 10,918 proteins and modified forms of proteins encoded by 10,720 different human genes, 13,804 complexes, 1,940 small molecules, and 507 drugs. These annotations are supported by 34,025 literature references. We have projected these reactions onto 77,335 orthologous proteins, creating 18,698 orthologous pathways in 15 non-human species. Version 78 has annotations for 4,603 protein variants (mutated proteins) and their post-translationally modified forms, derived from 352 proteins, which have been used to annotate disease-specific 1,544 reactions and 673 pathways.</p>\r\n<p style=\"text-align: justify;\"><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"https://reactome.org/PathwayBrowser/#/TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and <a href=\"https://reactome.org/AnalysisService/\">Analysis</a> Services. The <a href=\"https://reactome.org/PathwayBrowser/#/TOOL=AT\">ReactomeGSA</a> package provides tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p style=\"text-align: justify;\"><strong>Documentation and Training.</strong> Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/community/training\">here</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/what-is-reactome\">website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information:</strong> If you have a question to ask or would like to give us your feedback, please contact our helpdesk.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2021-10-11 03:10:15','0000-00-00 00:00:00','{}','{}','{}',1,19,'','',1,0,'{}',0,'en-GB','',''),(170,847,'Reactome Frequently Asked Questions','reactome-frequently-asked-questions','','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2021-10-21 03:39:00','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,7,'','',2,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(171,848,'FAQ','faq','','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2021-10-21 03:40:18','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',2,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(172,849,'We want to hear your Success Story!','we-want-to-hear-your-success-story','<p style=\"text-align: justify;\"><img src=\"images/news/SSotM_Transp_BG.png\" alt=\"SSotM Transp BG\" width=\"650\" height=\"205\" style=\"display: block; margin-left: auto; margin-right: auto;\" />Our user community is built on success stories! There are thousands of you around the world and many of you have created amazing experiments, tools and resources using Reactome. We\'re always looking for ways to share interesting discoveries made with Reactome, and have decided to showcase projects on our website to inspire others to create new stories. If you have a novel success story or use case you\'d like to share, we would love to hear from you. </p>\r\n<p style=\"text-align: justify;\">To share your use case or success story, please fill in the <a href=\"https://forms.gle/gzckDAxWz7MgAGKj9\">success story form</a> or email us at <a href=\"mailto:help@reactome.org\">help@reactome.org</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>What is required?</strong></p>\r\n<p style=\"text-align: justify;\">There is no specific experiment, project or resource that we are looking for. We are mostly interested in hearing from you if you are willing to share what you have learned or what has worked well.</p>\r\n<p style=\"text-align: justify;\">Some potential stories............</p>\r\n<ul style=\"text-align: justify;\">\r\n<li>describe how Reactome data or software was integrated into your data resource or software tool? </li>\r\n<li>tell of how using Reactome helped you or your group to make a valuable discovery about your experiment?</li>\r\n</ul>\r\n<p style=\"text-align: justify;\"><strong>Where will my story be shared?</strong></p>\r\n<p style=\"text-align: justify;\">Your story will be reviewed by members of the Reactome team and may be shared on reactome.org. This will essentially be a post written by you and one of our team members. Your story will also be shared on Twitter.</p>\r\n<p style=\"text-align: justify;\"><strong>What will be the process?</strong></p>\r\n<p style=\"text-align: justify;\">Filling out this <a href=\"https://forms.gle/gzckDAxWz7MgAGKj9\">form</a> will simply indicate your interest in being featured on Reactome. If your submission is selected, we\'ll then email you for more detailed information about your success story and experiences. We will publish a novel success story every month. Every year, one entry from the \"Success Story of the Month\" will be selected to receive one $100 gift card redeemable through Amazon or another online vendor. </p>\r\n<p style=\"text-align: justify;\"></p>\r\n<p></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2021-11-01 17:25:15','0000-00-00 00:00:00','{}','{}','{}',1,18,'','',1,0,'{}',0,'en-GB','',''),(173,851,'FAQ','faq','','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2021-11-01 18:24:40','0000-00-00 00:00:00','{}','{}','{}',1,3,'','',2,0,'{}',0,'en-GB','',''),(174,852,'Version 79 Released','version-79-released','<p style=\"text-align: justify;\"><img src=\"images/stats/V79.png\" alt=\"V79\" width=\"650\" height=\"358\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p style=\"text-align: justify;\"></p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Topics and Pathways. </strong>In V79 we have introduced the new topic Drug Absorption, Distribution, Metabolism and Excretion in which we cover both pharmacokinetics and pharmacodynamics of <a href=\"/PathwayBrowser/#/R-HSA-9749641\">Aspirin</a>, <a href=\"/PathwayBrowser/#/R-HSA-9748787\">Azathioprine</a>, and <a href=\"/PathwayBrowser/#/R-HSA-9753281\">Paracetamol</a>. Topics with new or revised pathways in this release include Cell Cycle (<a href=\"/PathwayBrowser/#/R-HSA-69231\">Cyclin D associated events in G1</a> and <a href=\"/PathwayBrowser/#/R-HSA-9754119\">Drug-mediated inhibition of CDK4/CDK6 activity</a>), DNA Replication (<a href=\"/PathwayBrowser/#/R-HSA-68867\">Assembly of the pre-replicative complex</a>), Immune System (<a href=\"/PathwayBrowser/#/R-HSA-909733\">Interferon alpha/beta signaling</a>), Metabolism (<a href=\"/PathwayBrowser/#/R-HSA-191273\">Cholesterol biosynthesis</a> and <a href=\"/PathwayBrowser/#/R-HSA-6798163\">Choline catabolism</a>), and Sensory Perception (<a href=\"/PathwayBrowser/#/R-HSA-9752946\">Expression and translocation of Olfactory Receptors</a>).</p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Illustrations.</strong> Illustrations with embedded navigation features have been added or revised for <a href=\"/PathwayBrowser/#/R-HSA-3560782\">Diseases associated with glycosaminoglycan metabolism</a>, <a href=\"/PathwayBrowser/#/R-HSA-5609975\">Diseases associated with glycosylation precursor biosynthesis</a>,<a href=\"/PathwayBrowser/#/R-HSA-3781860\">Diseases associated with N-glycosylation of proteins</a>, <a href=\"/PathwayBrowser/#/R-HSA-9673013\">Diseases of Telomere Maintenance</a>, <a href=\"/PathwayBrowser/#/R-HSA-69306\">DNA Replication</a>, <a href=\"/PathwayBrowser/#/R-HSA-9748784\">Drug Absorption, Distribution, Metabolism, and Excretion (ADME)</a>, <a href=\"/PathwayBrowser/#/R-HSA-9700649\">Drug resistance of ALK mutants</a>, <a href=\"/PathwayBrowser/#/R-HSA-9702506\">Drug resistance of FLT3 mutants</a>, <a href=\"/PathwayBrowser/#/R-HSA-9669937\">Drug resistance of KIT mutants</a>, <a href=\"/PathwayBrowser/#/R-HSA-9674415\">Drug resistance of PDGFR mutants</a>, <a href=\"/PathwayBrowser/#/R-HSA-1430728\">Metabolism</a>, <a href=\"/PathwayBrowser/#/R-HSA-4839748\">Signaling by AMER1 mutants</a>, <a href=\"/PathwayBrowser/#/R-HSA-4839744\">Signaling by APC mutants</a>, <a href=\"/PathwayBrowser/#/R-HSA-4839735\">Signaling by AXIN mutants</a>, <a href=\"/PathwayBrowser/#/R-HSA-4839743\">Signaling by CTNNB1 phospho-site mutants</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors.</strong> Our external authors are <a href=\"https://orcid.org/0000-0002-4167-404X\">Jelena Kusic-Tisma</a> and <a href=\"https://orcid.org/0000-0002-6501-2163\">Damjana Rozman</a>. Our external reviewers are <a href=\"https://research.unsw.edu.au/people/professor-andrew-john-brown\">Andrew J Brown</a>, <a href=\"https://orcid.org/0000-0002-5332-8312\">Rachel Huddart</a>, <a href=\"https://orcid.org/0000-0002-1216-2969\">Anne Morgat</a>, <a href=\"https://orcid.org/0000-0003-0693-9994\">Sisira Kadambat Nair</a>, <a href=\"https://orcid.org/0000-0002-9700-1995\">Alexander F Palazzo</a>, <a href=\"https://orcid.org/0000-0002-6891-3178\">Francesco Raimondi</a>, <a href=\"https://orcid.org/0000-0001-5520-8480\">Qingtang Shen</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong> Reactome comprises 13,960 human reactions organized into 2,553 pathways involving 11,270 proteins and modified forms of proteins encoded by 11,071 different human genes, 13,853 complexes, 1,987 small molecules, and 527 drugs. These annotations are supported by 34,252 literature references. We have projected these reactions onto 77,418 orthologous proteins, creating 19,597 orthologous pathways in 15 non-human species. Version 79 has annotations for 5,064 protein variants (mutated proteins) and their post-translationally modified forms, derived from 347 proteins, which have been used to annotate disease-specific 1,540 reactions and 674 pathways.</p>\r\n<p style=\"text-align: justify;\"><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"/PathwayBrowser/#/TOOL=AT\">analyze</a> your own dataset, to <a href=\"/download-data\">download</a>, and access programmatically through our <a href=\"/ContentService/\">Content</a> and <a href=\"/AnalysisService/\">Analysis</a> Services. The <a href=\"/PathwayBrowser/#/TOOL=AT\">ReactomeGSA</a> package provides tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p style=\"text-align: justify;\"><strong>Documentation and Training.</strong> Visit our online <a href=\"/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"/community/training\">here</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"/what-is-reactome\">website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information:</strong> If you have a question to ask or would like to give us your feedback, please contact our helpdesk.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2021-12-13 06:01:33','0000-00-00 00:00:00','{}','{}','{}',1,17,'','',1,0,'{}',0,'en-GB','',''),(175,855,'Version 80 Released','version-80-released','<p><img src=\"images/news/R-HSA-3229121.png\" alt=\"R HSA 3229121\" width=\"650\" height=\"365\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Topics and Pathways.</strong> In V80, Topics with new or revised pathways include Cellular responses to stimuli (<a href=\"/PathwayBrowser/#/R-HSA-9711123&amp;PATH=R-HSA-8953897,R-HSA-2262752\">Cellular response to chemical stress</a>, <a href=\"/PathwayBrowser/#/R-HSA-9707564&amp;PATH=R-HSA-8953897,R-HSA-2262752,R-HSA-9711123\">Cytoprotection by HMOX1</a>, <a href=\"/PathwayBrowser/#/R-HSA-9755511\">KEAP1-NFE2L2 pathway</a>), Disease (<a href=\"/PathwayBrowser/#/R-HSA-9701190\">Defective HDR through Homologous Recombination (HRR) Due to BRCA2 Loss of Function</a>, <a href=\"/PathwayBrowser/#/R-HSA-9735804\">Diseases of nucleotide metabolism</a>, and <a href=\"/PathwayBrowser/#/R-HSA-9705683\">SARS-CoV-2-host interactions</a>), DNA repair (<a href=\"/PathwayBrowser/#R-HSA-5693616\">Presynaptic phase of homologous DNA pairing and strand exchange</a>), Drug ADME (<a href=\"/PathwayBrowser/#R-HSA-9754706\">Atorvastatin ADME</a>), Metabolism (<a href=\"/PathwayBrowser/#/R-HSA-196741\">Cobalamin (Cbl, vitamin B12) transport and metabolism</a>), and Metabolism of Proteins (<a href=\"/PathwayBrowser/#/R-HSA-8951664\">Neddylation</a>).</p>\r\n<p style=\"text-align: justify;\"><strong>New and Updated Illustrations.</strong> <a href=\"/PathwayBrowser/#9711123\">Cellular response to chemical stress</a>, <a href=\"/PathwayBrowser/#/R-HSA-3296469\">Defects in cobalamin (B12) metabolism</a>, <a href=\"/PathwayBrowser/#/R-HSA-5668914\">Diseases of Metabolism</a>, <a href=\"/PathwayBrowser/#/R-HSA-5423599\">Diseases of Mismatch Repair (MMR)</a>, <a href=\"/PathwayBrowser/#/R-HSA-9735804\">Diseases of nucleotide metabolism</a>, <a href=\"/PathwayBrowser/#/R-HSA-9748784\">Drug Absorption, Distribution, Metabolism and Excretion (ADME)</a>, <a href=\"/PathwayBrowser/#/R-HSA-3229121\">Glycogen storage diseases</a>, <a href=\"/PathwayBrowser/#/R-HSA-9649913\">RAS GTPase cycle mutants</a>, and <a href=\"/PathwayBrowser/#/R-HSA-9660537\">Signaling by MRAS-complex mutants</a> have a new or revised Illustration with embedded navigation features.</p>\r\n<p style=\"text-align: justify;\"><strong>Thanks to our Contributors.</strong> <a href=\"https://www.bq.uam.es/acuadrado/\">Antonio Cuadrado</a>, <a href=\"https://orcid.org/0000-0002-7774-1953\">Wolf-Dietrich Heyer</a>, <a href=\"https://orcid.org/0000-0001-7476-6306\">David P Hill</a>, <a href=\"https://orcid.org/0000-0002-5332-8312\">Rachel Huddart</a>, <a href=\"https://orcid.org/0000-0001-5132-3190\">Hang Phuong Le</a>, <a href=\"https://orcid.org/0000-0001-5817-9144\">Jie Liu</a>, <a href=\"https://orcid.org/0000-0001-8076-7217\">Francesco Messina</a>, and <a href=\"https://orcid.org/0000-0001-9489-2246\">Julia Somers</a> are our external reviewers.</p>\r\n<p style=\"text-align: justify;\"><strong>Annotation Statistics.</strong> Reactome comprises 14,108 human reactions organized into 2,580 pathways involving 11,285 proteins and modified forms of proteins encoded by 11,084 different human genes, 13,869 complexes, 1,987 small molecules, and 532 drugs. These annotations are supported by 34,703 literature references. We have projected these reactions onto 77,392 orthologous proteins, creating 18,830 orthologous pathways in 15 non-human species. Version 80 has annotations for 5,422 protein variants (mutated proteins) and their post-translationally modified forms, derived from 352 proteins, which have been used to annotate disease-specific 1,601 reactions and 695 pathways.</p>\r\n<p style=\"text-align: justify;\"><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"/PathwayBrowser/#/TOOL=AT\">analyze</a> your own dataset, to <a href=\"/download-data\">download</a>, and access programmatically through our <a href=\"/ContentService/\">Content</a> and <a href=\"/AnalysisService/\">Analysis</a> Services. The <a href=\"/PathwayBrowser//PathwayBrowser/#/TOOL=AT\">ReactomeGSA</a> package provides tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p style=\"text-align: justify;\"><strong>Documentation and Training.</strong> Visit our online <a href=\"/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"/community/training\">here</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The European Bioinformatics Institute. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence will apply to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"/what-is-reactome\">website</a>.</p>\r\n<p style=\"text-align: justify;\"><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p style=\"text-align: justify;\"><strong>For more information:</strong> If you have a question to ask or would like to give us your feedback, please contact our helpdesk.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-04-05 17:59:24','0000-00-00 00:00:00','{}','{}','{}',1,16,'','',1,0,'{}',0,'en-GB','',''),(176,857,'Version 82 Released','version-82-released','<p></p>\r\n<p></p>\r\n<p></p>\r\n<p style=\"text-align: center;\"><a href=\"PathwayBrowser/#9694516\"><span style=\"font-size: 14pt;\">SARS-CoV-2 Infection</span></a></p>\r\n<p></p>\r\n<p></p>\r\n<p style=\"text-align: center;\"><img src=\"images/news/SARS-COV-2_Infection.png\" alt=\"SARS COV 2 Infection\" width=\"703\" height=\"401\" /></p>\r\n<p><strong>New and Updated Topics and Pathways.</strong> In V82, Topics with new or revised pathways include Developmental Biology (<a href=\"https://reactome.org/PathwayBrowser/#9758919\">Epithelial-Mesenchymal Transition during gastrulation</a> and <a href=\"https://reactome.org/PathwayBrowser/#9754189\">Germ layer formation at gastrulation</a>), Disease-SARS-CoV-1 Infection (<a href=\"https://reactome.org/PathwayBrowser/#9692914\">SARS-CoV-1-host interactions</a>),  Disease-SARS-CoV-2 Infection (<a href=\"https://reactome.org/PathwayBrowser/#9733458\">Induction of Cell-Cell Fusion</a>, <a href=\"https://reactome.org/PathwayBrowser/#9694631\">Maturation of nucleoprotein</a>, <a href=\"https://reactome.org/PathwayBrowser/#9694493\">Maturation of protein E</a>, <a href=\"https://reactome.org/PathwayBrowser/#9694594\">Maturation of protein M</a>, <a href=\"https://reactome.org/PathwayBrowser/#9694548\">Maturation of Spike protein</a> and  <a href=\"https://reactome.org/PathwayBrowser/#9727281\">Translation of Accessory Proteins</a>, Gene expression (<a href=\"https://reactome.org/PathwayBrowser/#9634815\">Transcriptional Regulation by NPAS4</a>), Metabolism (<a href=\"https://reactome.org/PathwayBrowser/#2024096\">HS-GAG degradation</a>, <a href=\"https://reactome.org/PathwayBrowser/#2022857\">Keratan sulfate degradation</a>, and <a href=\"https://reactome.org/PathwayBrowser/#174362\">Transport and synthesis of PAPS</a>), and Signal Transduction (<a href=\"https://reactome.org/PathwayBrowser/#5357905\">Regulation of TNFR1 signaling</a> and <a href=\"https://reactome.org/PathwayBrowser/#75893\">TNF signaling</a>). In continuing our collaborative drug annotation work with Caroline Thorn and <a href=\"https://www.pharmgkb.org/\">PharmGKB</a>,  we have added the pathways  <a href=\"https://reactome.org/PathwayBrowser/#9757110\">Prednisone ADME</a> and  <a href=\"https://reactome.org/PathwayBrowser/#9755088\">Ribavirin ADME</a> under the topic of Drug Absorption, Distribution, Metabolism, and Excretion (ADME) pathways. </p>\r\n<p><strong>New and Updated Illustrations.</strong> <a href=\"https://reactome.org/PathwayBrowser/#9748784\">Drug Absorption, Distribution, Metabolism and Excretion (ADME)</a>, <a href=\"https://reactome.org/PathwayBrowser/#9758941\">Gastrulation</a>, <a href=\"https://reactome.org/PathwayBrowser/#9694516\">SARS-CoV-2 infection</a>,  <a href=\"https://reactome.org/PathwayBrowser/#2990846\">SUMOylation</a>, and  <a href=\"https://reactome.org/PathwayBrowser/#3108232\">SUMO E3 ligases SUMOylate target proteins</a> have a new or revised Illustration with embedded navigation features. The updated SARS-CoV-2 illustration featured above highlights the recent partitioning of the pathway into early and late subpathways based on our growing understanding of the life cycle of this virus.</p>\r\n<p><strong>Thanks to our Contributors.</strong>    <a href=\"https://orcid.org/0000-0001-7476-6306\">David Hill</a>,  <a href=\"https://orcid.org/0000-0002-7002-1275\">Yingxi Lin</a>, <a href=\"https://orcid.org/0000-0002-6950-1634\">Rainer de Martin</a>, <a href=\"https://orcid.org/0000-0001-5789-5747\">Caroline Thorn</a>, and <a href=\"https://orcid.org/0000-0002-9362-4343\">Hailin Tu</a> are our external reviewers.</p>\r\n<p><strong>Annotation Statistics.</strong> Reactome comprises 14398  human reactions organized into 2601 pathways involving 11393  proteins and modified forms of proteins encoded by 11097 different human genes, 14084  complexes, 1998 small molecules, and 1035 drugs. These annotations are supported by 35965 literature references. We have projected these reactions onto 83288 orthologous proteins, creating  19385 orthologous pathways in 15 non-human species. Version 82 has annotations for 4977  protein variants (mutated proteins) and their post-translationally modified forms, derived from 352 proteins, which have contributed to the annotation of 1659 disease-specific reactions and 704 pathways.</p>\r\n<p><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and  Services. The <a href=\"https://reactome.org/userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">ReactomeGSA</a> package provide tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p><strong>Documentation and Training.</strong> Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/community/training\">here</a>..</p>\r\n<p><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/what-is-reactome\">website</a>.</p>\r\n<p><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information:</strong> If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <a href=\"mailto:help@reactome.org.\"></a><a href=\"mailto:help@reactome.org\"></a><a href=\"mailto:help@reactome.org\">help@reactome.org</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-09-16 16:28:48','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,15,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(177,858,'New Paper pulished in Database (Oxford)','new-paper-pulished-in-database-oxford','<p><img src=\"images/baac009f1.jpeg\" alt=\"baac009f1\" width=\"650\" height=\"282\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p>A new research article entitled “<a href=\"https://doi.org/10.1093/database/baac009\">Evaluating the predictive accuracy of curated biological pathways in a public knowledgebase</a>” has been published in the <a href=\"https://academic.oup.com/database\">Database (Oxford)</a> journal. The paper describes the utility of Reactome pathways for predicting functional consequences of genetic perturbations. Predictions of perturbation effects based on Reactome pathways were compared against published empirical observations. More publications from the <a href=\"https://reactome.org/about/team\">Reactome Team</a> can be found <a href=\"https://reactome.org/community/publications/\">here</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-04-15 17:22:02','0000-00-00 00:00:00','{}','{}','{}',1,14,'','',1,0,'{}',0,'en-GB','',''),(178,860,'Version 81 Released','version-81-released','<p><img src=\"images/news/Gastrulation.png\" alt=\"Gastrulation\" width=\"650\" height=\"401\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p></p>\r\n<p><strong>New and Updated Topics and Pathways.</strong> In V81, Topics with new or revised pathways include Development (<a href=\"PathwayBrowser/#/R-HSA-9754189\">Germ layer formation at gastrulation</a>), Immune system (<a href=\"PathwayBrowser/#/R-HSA-445989\">TAK1-dependent IKK and NF-kB activation</a>), and Signal Transduction (<a href=\"PathwayBrowser/#/R-HSA-2173793&amp;PATH=R-HSA-162582,R-HSA-9006936,R-HSA-170834,R-HSA-2173793\">Transcriptional activity of SMAD2/SMAD3:SMAD4 heterotrimer</a>). The SARS-CoV-2 life cycle has been updated to reflect our growing understanding of the life cycle. Viral events have partitioned into early and late subpathways. Early events begin with viral entry and carry through to initial gene expression; late events include the massive organelle reorganizations and viral release as host cells transition to viral shedding factories. SARS-CoV-2 annotation now includes 224 events; with this release 61 of those events have been updated with experimental support from the body of COVID-19 literature that has emerged since March 2020. Additionally, in collaboration with Rachel Huddart with <a href=\"https://www.pharmgkb.org/\">PharmGKB</a>, we have added 101 drug interactions associated with 63 pathways in this release bringing the total number of drugs represented in Reactome to 1022.</p>\r\n<p><strong>New and Updated Illustrations.</strong> <a href=\"PathwayBrowser/#/R-HSA-1266738\">Developmental Biology</a>, <a href=\"PathwayBrowser/#/R-HSA-9758941\">Gastrulation</a>, <a href=\"PathwayBrowser/#/R-HSA-9735786\">Nucleotide catabolism defects</a>, and <a href=\"PathwayBrowser/#/R-HSA-9734207\">Nucleotide salvage defects</a> have a new or revised Illustration with embedded navigation Features.</p>\r\n<p><strong>Thanks to our Contributors.</strong> <a href=\"https://sites.google.com/view/osvaldocontreras\">Osvaldo Contreras</a>, <a href=\"https://orcid.org/0000-0002-6950-1634\">Rainer de Martin</a>, <a href=\"https://orcid.org/0000-0002-5332-8312\">Rachel Huddart</a>, <a href=\"https://orcid.org/0000-0002-8155-4296\">Naz Salehin</a>, and <a href=\"https://orcid.org/0000-0001-6950-8388\">Patrick P L Tam</a> are our external reviewers.</p>\r\n<p><strong>Annotation Statistics.</strong> Reactome comprises 14,246 human reactions organized into 2,585 pathways involving 11,291 proteins and modified forms of proteins encoded by 11,088 different human genes, 13,984 complexes, 1,986 small molecules, and 1,022 drugs. These annotations are supported by 35,629 literature references. We have projected these reactions onto 83,266 orthologous proteins, creating 19,328 orthologous pathways in 15 non-human species. Version 81 has annotations for 5,422 protein variants (mutated proteins) and their post-translationally modified forms, derived from 352 proteins, which have been used to annotate disease-specific 1,609 reactions and 697 pathways.</p>\r\n<p><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"PathwayBrowser/\">Pathway Browser</a>, to <a href=\"PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"download-data/\">download</a>, and access programmatically through our <a href=\"ContentService/\">Content</a> and <a href=\"AnalysisService/\">Analysis</a> Services. The <a href=\"userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"PathwayBrowser/#TOOL=AT\">ReactomeGSA</a> package provides tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p><strong>Documentation and Training.</strong> Visit our online <a href=\"userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"community/training\">here</a>.</p>\r\n<p><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"what-is-reactome\">website</a>.</p>\r\n<p><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information:</strong> If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <span id=\"cloak9175d4b26c21cb85c34dc05af39a2575\"><a href=\"mailto:help@reactome.org\"></a><a href=\"mailto:help@reactome.org\">help@reactome.org</a></span>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-06-14 17:25:02','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,13,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(179,861,'Collaboration with PharmGKB','collaboration-with-pharmgkb','<p style=\"text-align: center;\"><img src=\"images/news/Drug_ADME.png\" alt=\"Drug ADME\" width=\"650\" height=\"365\" /></p>\r\n<p></p>\r\n<p>Reactome is pleased to announce a partnership with PharmGKB to curate pharmacokinetic / Drug ADME (absorption distribution metabolism excretion) pathways. Sharing knowledge and distributing curation effort will increase the efficiency and consistency with which both resources are able to integrate pathway data. The cross-references between co-curated pathways in Reactome and PharmGKB will support seamless navigation between the two resources enabling users to understand, visualize, and analyze their data in contexts that are best suited to their research interests. The collaborative pathways released in Reactome Version 82 are <a href=\"PathwayBrowser/#/R-HSA-9757110\">Prednisone ADME</a> (found <a href=\"https://www.pharmgkb.org/pathway/PA166273601\">here</a> at PharmGKB), and <a href=\"PathwayBrowser/#/R-HSA-9755088\">Ribavirin ADME</a> (found <a href=\"https://www.pharmgkb.org/pathway/PA166271961\">here</a> at PharmGKB).</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-09-28 15:02:07','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,12,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(180,864,'Post-infusion CAR TReg cells identify patients resistant to CD19-CAR therapy','reactome-spotlight','<p>Reactome pathway enrichment analysis helps to pinpoint expansion of regulatory T cells as a new biomarker of CAR T cell therapy resistance and toxicity in patients with B cell lymphoma. The study was published by <a href=\"https://www.nature.com/articles/s41591-022-01960-7\">Good et al. in Nature Medicine on September 12, 2022</a>.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-10-18 14:26:35','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,23,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(181,865,'Reactome Research Spotlight test','reactome-research-spotlight-test','<p>test content</p>','',-2,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-10-19 10:09:35','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(182,866,'reactome spotlight test','reactome-spotlight-test','<p>test</p>','',-2,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-10-19 12:41:37','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(183,867,'Reactome Research Spotlight','reactome-research-spotlight','<p>Introducing the Reactome Research Spotlight!</p>\r\n<p></p>\r\n<p>In October 2022, Reactome introduced a new feature, the Reactome Research Spotlight. This monthly feature will highlight a recent standout publication that makes use of Reactome data or analysis tools in its research.</p>\r\n<p>In our first spotlight, we feature the paper “<a href=\"https://www.nature.com/articles/s41591-022-01960-7\"><strong>Post-infusion CAR TReg cells identify patients resistant to CD19-CAR therapy</strong></a>” by Good et al, published in September 2022 in Nature Medicine. This paper identifies expansion of regulatory T cells as a biomarker of resistance and toxicity after CD19-CAR therapy in patients with large B cell lymphoma. Differential gene expression and Reactome pathway enrichment analysis identified TReg development as one of the top enriched pathways in the TReg CD4+ CD57-Helios+ cell populations associated with post-therapy progression and decreased neurotoxicity.<img src=\"images/news/Spotlight_news_image.png\" alt=\"Spotlight news image\" width=\"650\" height=\"386\" style=\"display: block; margin-left: auto; margin-right: auto;\" />Do you use Reactome data or tools in your research? Email us at <a href=\"mailto:help@reactome.org\">help@reactome.org</a> to have your work highlighted in an upcoming Spotlight feature.</p>\r\n<p>Follow us on Twitter! @reactome to stay up to date with new and updated pathways, feature updates and more.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-11-10 17:22:32','0000-00-00 00:00:00','{}','{}','{}',1,11,'','',1,0,'{}',0,'en-GB','',''),(184,870,'Version 83 Released','version-83-released','<p style=\"text-align: center;\"><span style=\"font-size: 14pt;\"><a href=\"PathwayBrowser/#/R-HSA-1280215&amp;PATH=R-HSA-168256\">Cytokine Signaling in Immune System</a></span></p>\r\n<p></p>\r\n<p style=\"text-align: center;\"><img src=\"images/news/R-HSA-1280215.png\" alt=\"R HSA 1280215\" width=\"650\" height=\"371\" /></p>\r\n<p></p>\r\n<p>New and Updated Topics and Pathways. In V83, Topics with new or revised pathways include Developmental Biology (<a href=\"PathwayBrowser/#/R-HSA-9796292\">Formation of axial mesoderm</a>, <a href=\"PathwayBrowser/#/R-HSA-9758920\">Formation of lateral plate mesoderm</a>, <a href=\"PathwayBrowser/#/R-HSA-9793380\">Formation of paraxial mesoderm</a>), Immune System (<a href=\"PathwayBrowser/#/R-HSA-9680350\">Signaling by CSF1 (M-CSF) in myeloid cells</a>), Metabolism (<a href=\"PathwayBrowser/#/R-HSA-70263\">Gluconeogenesis</a>), Metabolism of proteins (<a href=\"PathwayBrowser/#/R-HSA-264876\">Insulin processing</a>), Metabolism of RNA (<a href=\"PathwayBrowser/#/R-HSA-72163\">mRNA Splicing - Major Pathway</a>), Programmed Cell Death (<a href=\"PathwayBrowser/#/R-HSA-5675482\">Regulation of necroptotic cell death</a>), Signal Transduction (<a href=\"PathwayBrowser/#/R-HSA-5357905\">Regulation of TNFR1 signaling</a>, <a href=\"PathwayBrowser/#/R-HSA-5357956\">TNFR1-induced NFkappaB signaling pathway</a>, and <a href=\"PathwayBrowser/#/R-HSA-5357786\">TNFR1-induced proapoptotic signaling</a>), and Transport of small molecules (<a href=\"PathwayBrowser/#/R-HSA-435368\">Zinc efflux and compartmentalization by the SLC30 family</a>).  In continuing our collaborative drug annotation work  with Caroline Thorn and <a href=\"https://www.pharmgkb.org/\">PharmGKB</a>,  we have added the pathway <a href=\"PathwayBrowser/#/R-HSA-9793528\">Ciprofloxacin ADME</a> under the topic of Drug Absorption, Distribution, Metabolism, and Excretion (ADME) pathways. </p>\r\n<p>New and Updated Illustrations. <a href=\"PathwayBrowser/#/R-HSA-1280215\">Cytokine Signaling in Immune system</a>, <a href=\"PathwayBrowser/#/R-HSA-9675136\">Diseases of DNA Double-Strand Break Repair</a>, <a href=\"PathwayBrowser/#/R-HSA-9697154\">Disorders of Nervous System Development</a>, <a href=\"PathwayBrowser/#/R-HSA-9748784\">Drug ADME</a>, <a href=\"PathwayBrowser/#/R-HSA-9758941\">Gastrulation</a>, <a href=\"PathwayBrowser/#/R-HSA-388396\">GPCR downstream signalling</a>, <a href=\"PathwayBrowser/#/R-HSA-9005891\">Loss of function of MECP2 in Rett syndrome</a>, <a href=\"PathwayBrowser/#/R-HSA-9005895\">Pervasive developmental disorders</a>, <a href=\"PathwayBrowser/#/R-HSA-76002\">Platelet activation, signaling and aggregation</a>, and <a href=\"PathwayBrowser/#/R-HSA-2990846\">SUMOylation</a> have a new or revised Illustration with embedded navigation features. </p>\r\n<p>Thanks to our Contributors.   <a href=\"https://orcid.org/0000-0002-2910-2065\">E Richard Stanley</a> is our external author.  <a href=\"https://orcid.org/0000-0001-9607-8838\">Ravindrababu Chalamalasetty</a>, <a href=\"https://orcid.org/0000-0002-8959-7012\">Clément Charenton</a>, <a href=\"https://orcid.org/0000-0001-7476-6306\">David P Hill</a>, <a href=\"https://orcid.org/0000-0002-0749-2576\">Christian Mosimann</a>, <a href=\"https://orcid.org/0000-0003-0195-3949\">James M Murphy</a>, <a href=\"https://orcid.org/0000-0001-6077-6407\">Karin Prummel</a>, <a href=\"https://orcid.org/0000-0001-8752-6846\">Hiroshi Sasaki</a>, <a href=\"https://orcid.org/0000-0002-2910-2065\">E Richard Stanley</a>, <a href=\"https://orcid.org/0000-0002-9362-4343\">Hailin Tu</a>, and <a href=\"https://orcid.org/0000-0002-7452-4419\">Terry P Yamaguchi</a> are our external reviewers.</p>\r\n<p>Annotation Statistics. Reactome comprises 14,471 human reactions organized into 2,610 pathways involving 30,656 proteins and modified forms of proteins encoded by 11,442 different human genes, 14,153  complexes, 2,002 small molecules, and 1,113 drugs. These annotations are supported by 36,290 literature references. We have projected these reactions onto 83,933 orthologous proteins, creating 19,417 orthologous pathways in 14 non-human species. Version 83 has annotations for 4,977 protein variants (mutated proteins) and their post-translationally modified forms, derived from 361 proteins, which have contributed to the annotation of 1,659 disease-specific reactions and 704 pathways. </p>\r\n<p> Tools and Data. Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and <a href=\"https://reactome.org/AnalysisService/\">Analysis</a> Services. The <a href=\"https://reactome.org/userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"PathwayBrowser/#TOOL=AT\">ReactomeGSA</a> package provides tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p>Documentation and Training. Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/community/training\">here</a>.</p>\r\n<p>About the Reactome Project. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/what-is-reactome\">website</a>.</p>\r\n<p>Follow us on Twitter: <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p>For more information: If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <a href=\"mailto:help@reactome.org\">help@reactome.org</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-11-30 16:53:24','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,10,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(185,872,'Circadian transcriptional pathway atlas highlights a proteasome switch in intermittent fasting','circadian-transcriptional-pathway-atlas-highlights-a-proteasome-switch-in-intermittent-fasting','<p>Reactome pathway gene sets in the MSigDB facilitated identification of the liver proteasome transcriptional switch that acts as the fasting timer in intermittent fasting. The study was published by <a href=\"https://www.cell.com/cell-reports/fulltext/S2211-1247(22)01403-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124722014036%3Fshowall%3Dtrue\">Wei et al. in Cell Reports on October 25, 2022</a>. </p>','',-2,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-12-09 19:10:16','0000-00-00 00:00:00','{}','{}','{}',1,1,'','',1,0,'{}',0,'en-GB','',''),(186,873,'Circadian transcriptional pathway atlas highlights a proteasome switch in intermittent fasting','reactome-pathway-gene-sets-in-the-msigdb-facilitated-identification-of-the-liver-proteasome-transcriptional-switch-that-acts-as-the-fasting-timer-in-intermittent-fasting','<p>Reactome pathway gene sets in the MSigDB facilitated identification of the liver proteasome transcriptional switch that acts as the fasting timer in intermittent fasting in work published by <a href=\"https://www.cell.com/cell-reports/fulltext/S2211-1247(22)01403-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124722014036%3Fshowall%3Dtrue\">Wei et al. in Cell Reports on October 25, 2022</a>. The authors suggest that a 16-hour interval in intermittent fasting may be most beneficial for health.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-12-12 16:41:07','0000-00-00 00:00:00','{}','{}','{}',1,22,'','',1,0,'{}',0,'en-GB','',''),(187,874,'Reactome named as a Global Core Biodata Resource','reactome-named-as-a-global-core-biodata-resource','<p>On December 15, the Global Biodata Coalition (<a href=\"https://globalbiodata.org/\"></a><a href=\"https://globalbiodata.org/\">https://globalbiodata.org/</a>) of research funders recognized the Reactome Knowledgebase (<a href=\"http://www.reactome.org/\">www.reactome.org</a>) as one of just 37 resources worldwide whose long-term funding and sustainability are critical to life science and biomedical research. See the full announcement <a href=\"https://globalbiodata.org/global-biodata-coalition-announces-the-first-set-of-global-core-biodata-resources/\">here</a>.</p>\r\n<p style=\"text-align: left;\">Many thanks to the selection panel at GBC for the recognition, and congratulations to the other resources who were selected.</p>\r\n<p></p>\r\n<p></p>\r\n<p></p>\r\n<p></p>\r\n<p style=\"text-align: left;\"><img src=\"images/news/GCBR-Logo-RGB.png\" alt=\"GCBR Logo RGB\" width=\"650\" height=\"276\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2022-12-21 18:16:35','0000-00-00 00:00:00','{}','{}','{}',1,9,'','',1,0,'{}',0,'en-GB','',''),(188,876,'Gene set enrichment analysis (GSEA) identifies upregulated carbohydrate metabolism pathways in tumors with high tumor-specific total mRNA expression (TmS)','gene-set-enrichment-analysis-gsea-identifies-the-two-most-frequently-upregulated-carbohydrate-metabolism-pathways-in-tumors-with-high-tumor-specific-total-mrna-expression-tms','<p>Gene set enrichment analysis (GSEA) conducted on Reactome’s carbohydrate metabolism pathways identifies the <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-71336\">Pentose phosphate pathway</a> and the <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-70326\">Glucose metabolism </a>pathway as the two most frequently upregulated pathways in tumors with high tumor-specific total mRNA expression (TmS) across 15 tumor types; TmS is a novel tumor phenotype-predictive quantitative feature described by <a href=\"https://www.nature.com/articles/s41587-022-01342-x\">Cao et al. in the November 2022 issue of Nature Biotechnology</a>.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-01-10 19:47:21','0000-00-00 00:00:00','{}','{}','{}',1,21,'','',1,0,'{}',0,'en-GB','',''),(189,877,'Common targetable inflammatory pathways in brain transcriptome of autism spectrum disorders and Tourette syndrome','common-targetable-inflammatory-pathways-in-brain-transcriptome-of-autism-spectrum-disorders-and-tourette-syndrome','<p>Reactome overrepresentation analyses of differentially expressed genes common to both Autism Spectrum Disorder and Tourette Syndrome help identify common targetable inflammatory pathways as described by <a href=\"https://www.frontiersin.org/articles/10.3389/fnins.2022.999346/full\">Alshammeryet al. in the December 2022 issue of Frontiers in Neuroscience.</a></p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-02-14 13:23:23','0000-00-00 00:00:00','{}','{}','{}',1,20,'','',1,0,'{}',0,'en-GB','',''),(190,878,'Probable Treatment Targets for Diabetic Retinopathy Based on an Integrated Proteomic and Genomic Analysis','probable-treatment-targets-for-diabetic-retinopathy-based-on-an-integrated-proteomic-and-genomic-analysis','<p>Analysis of all constituents of entire Reactome pathways identified by the presence of upregulated or mutated genes helped <a href=\"https://tvst.arvojournals.org/article.aspx?articleid=2785339\">Valdivia et al. in the February, 2023 issue of Translational Vision Science &amp; Technology</a> to identify druggable targets and potential drugs for the treatment of diabetic retinopathy (DR). Drugs affecting MMP13 and LGALS3 in the regulation of myeloid cell differentiation by RUNX2 were notable candidates.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-03-13 20:29:19','0000-00-00 00:00:00','{}','{}','{}',1,19,'','',1,0,'{}',0,'en-GB','',''),(191,879,'Reactome is hiring!','reactome-is-hiring','<p><span style=\"font-size: 14pt;\"><strong>Outreach Coordinator</strong></span></p>\r\n<p>We are looking for an experienced and enthusiastic person to lead our outreach efforts. </p>\r\n<p></p>\r\n<p>As Reactome Outreach Coordinator, you will lead our community outreach activities, strengthening existing collaborations and developing initiatives to foster new ones. Working with the software and curatorial teams, you will coordinate Reactome communications, including conference presentations, workshops, scientific and lay papers, and social media. You will develop education and training materials for in person and online learning and will contribute to the documentation on the Reactome web site.</p>\r\n<p></p>\r\n<p><span style=\"font-size: 12pt;\"><strong>Your responsibilities will include:</strong></span></p>\r\n<ul>\r\n<li>Maintaining a calendar of outreach activities, including community events, workshops, appearances and other communication opportunities</li>\r\n<li>Planning and delivering training workshops on Reactome web interface, tools and application programming interfaces (APIs) and their use in large-scale data analysis. This responsibility will involve multiple public presentations per year as well as international travel</li>\r\n<li>Promotion of Reactome and communication of Reactome news through social media channels like Twitter, LinkedIn, and Facebook</li>\r\n<li>Creating and maintaining documentation and e-learning materials on our advanced tools and APIs, and their application in bioinformatic analysis</li>\r\n<li>Direct support via email and tracker of user queries on using our APIs and advanced tools</li>\r\n<li>Direct interaction with our development teams to fix bugs identified by users and find solutions to user problems</li>\r\n<li>Supporting usability testing of tools and APIs</li>\r\n</ul>\r\n<h2><span style=\"font-size: 12pt;\"><strong>Required Qualifications:</strong></span></h2>\r\n<p>We are seeking a self-reliant, resourceful team player comfortable with multitasking who is open to cooperation and collaboration both within our group and in the larger bioinformatics community.</p>\r\n<ul>\r\n<li>You have a Master’s or PhD degree or equivalent experience in life sciences, preferably in bioinformatics, biochemistry, molecular biology, cancer research or a related field</li>\r\n<li>In addition to your academic training, you have at least 2 years of relevant job experience related to bioinformatics, pathway databases, analysis of biological data sets or similar. Knowledge of cancer genomics is helpful but not required</li>\r\n<li>Significant experience in teaching, training, public outreach, online help and/or event planning events is essential</li>\r\n<li>You should be comfortable speaking persuasively to large audiences as well as conducting teaching and tutorials in small group settings, both online and in person</li>\r\n<li>You have excellent English communication skills, both written and verbal, to facilitate effective communications with other team members and to communicate with external collaborators and users</li>\r\n</ul>\r\n<p>Working hours can be negotiated to fit the needs of the applicant and the project. A part-time renewable contract or full-time permanent position can be considered, or a Professional Services Agreement for those candidates out of province.</p>\r\n<p>To apply, please see the full posting <a href=\"https://www.recruitingsite.com/csbsites/oicr/JobDescription.asp?JobNumber=915583\">here</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-03-20 17:26:04','0000-00-00 00:00:00','{}','{}','{}',1,8,'','',1,0,'{}',0,'en-GB','',''),(192,880,'Review Status of Reactome Events ','review-status-of-reactome-events','<style type=\"text/css\">\r\n\r\n\r\ntable.reactome.review-status td{\r\ntext-align: center;\r\nvertical-align: middle;\r\n}  \r\n\r\ntable.reactome.review-status th{\r\ntext-align: center;\r\nvertical-align: middle;\r\n}  \r\n\r\n\r\n\r\n</style>\r\n<p>Reactome events (pathways and reaction-like events) undergo internal review by a member of the Reactome editorial staff and external review by a domain expert. Events that undergo both internal and external review are assigned the highest review score of  5. When such an event is revised and the revisions result in <span style=\"color: #ff0000;\">*structural modification</span> to that event, the event’s review score is demoted to 3. The revised event must be internally reviewed again to attain a review score of 4 and externally reviewed to regain a score of 5. The table below summarizes the definitions of each review status.</p>\r\n<p><strong>Status-based release of Reactome content</strong></p>\r\n<table class=\"reactome review-status\">\r\n<thead>\r\n<tr><th scope=\"col\">\r\n<p><b>Review Score</b></p>\r\n</th><th scope=\"col\">\r\n<p><b>Definition</b></p>\r\n</th><th scope=\"col\">\r\n<p><b>Release status</b></p>\r\n</th></tr>\r\n</thead>\r\n<tbody>\r\n<tr>\r\n<td>1</td>\r\n<td>Restructured after internal review</td>\r\n<td>NOT RELEASED</td>\r\n</tr>\r\n<tr>\r\n<td>2</td>\r\n<td>Restructured after external review</td>\r\n<td>NOT RELEASED</td>\r\n</tr>\r\n<tr>\r\n<td>3</td>\r\n<td>Internally reviewed, awaiting external review</td>\r\n<td>RELEASED</td>\r\n</tr>\r\n<tr>\r\n<td>4</td>\r\n<td>Restructured after external review, then internally re-reviewed</td>\r\n<td>RELEASED</td>\r\n</tr>\r\n<tr>\r\n<td>5</td>\r\n<td>Externally reviewed</td>\r\n<td>RELEASED</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p></p>\r\n<p><strong>*</strong><strong>What is a </strong><span style=\"color: #ff0000;\"><strong>structural modification</strong></span><strong> to an event?</strong></p>\r\n<p>A structural update involves at least one of the following modifications after the initial internal and/or external review of the event.</p>\r\n<ul>\r\n<li>Addition, removal, or replacement of a CatalystActivity, Regulator, Input, Output of a reaction-like event</li>\r\n<li>Addition, removal, or replacement of an Event of a pathway</li>\r\n</ul>\r\n<p><strong><em>Any one or more of these changes to an event demotes its review status and requires additional review as described in the table above in order to restore its review status.</em></strong></p>\r\n<p><em>A Reactome event that has been internally reviewed and is still awaiting an external review after six months is publically released with a score of 3. Once reviewed, it is assigned the maximum score of 5.</em><em>If an event with a score of 3 is </em><span style=\"color: #ff0000;\"><em>*structurally modified</em></span><em>, its score is demoted to 1 and must be internally reviewed to regain its original score of 3.  If an event with a score of 4 is structurally modified, it is demoted to a score of 2 and must be internally reviewed to recover its score of 4 and externally reviewed to raise its score to 5 . Reactome does not include events with a score lower than 3 on the public website.</em></p>\r\n<p><strong>Where can I see the review status of a Reactome event?</strong></p>\r\n<p>The review status of an event can be seen in the upper right corner of the  Pathway Browser details panel:</p>\r\n<p><img src=\"images/userguide/review_status/ReviewStatus_1.png\" alt=\"ReviewStatus 1\" width=\"941\" height=\"522\" /></p>\r\n<p> and also on the content details page reached via the search:</p>\r\n<p><img src=\"images/userguide/review_status/ReviewStatus_2.png\" alt=\"ReviewStatus 2\" width=\"996\" height=\"569\" /></p>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-03-23 14:06:37','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(193,881,'Version 84 released','version-84-released','<p><img src=\"images/news/InfectiousDisease.png\" alt=\"\" width=\"650\" height=\"401\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p><strong>New and Updated Topics and Pathways. </strong>In V83, Topics with new or revised pathways include Developmental Biology (<a href=\"#9796292\">Formation of axial mesoderm</a>, <a href=\"#9758920\">Formation of lateral plate mesoderm</a>, <a href=\"#9793380\">Formation of paraxial mesoderm</a>), Immune System (<a href=\"#9680350\">Signaling by CSF1 (M-CSF) in myeloid cells</a>), Metabolism (<a href=\"#70263\">Gluconeogenesis</a>), Metabolism of proteins (<a href=\"#264876\">Insulin processing</a>), Metabolism of RNA (<a href=\"#72163\">mRNA Splicing - Major Pathway</a>), Programmed Cell Death (<a href=\"#5675482\">Regulation of necroptotic cell death</a>), Signal Transduction (<a href=\"#5357905\">Regulation of TNFR1 signaling</a>, <a href=\"#5357956\">TNFR1-induced NFkappaB signaling pathway</a>, and <a href=\"#5357786\">TNFR1-induced proapoptotic signaling</a>), and Transport of small molecules (<a href=\"#435368\">Zinc efflux and compartmentalization by the SLC30 family</a>).  In continuing our collaborative drug annotation work  with Caroline Thorn and <a href=\"https://www.pharmgkb.org/\">PharmGKB</a>,  we have added the pathway <a href=\"#R-HSA-9793528\">Ciprofloxacin ADME</a> under the topic of Drug Absorption, Distribution, Metabolism, and Excretion (ADME) pathways. </p>\r\n<p><strong>New and Updated Illustrations. </strong><a href=\"#1280215\">Cytokine Signaling in Immune system</a>, <a href=\"#9675136\">Diseases of DNA Double-Strand Break Repair</a>, <a href=\"#9697154\">Disorders of Nervous System Development</a>, <a href=\"#9748784\">Drug ADME</a>, <a href=\"#9758941\">Gastrulation</a>, <a href=\"#388396\">GPCR downstream signalling</a>, <a href=\"#9005891\">Loss of function of MECP2 in Rett syndrome</a>, <a href=\"#9005895\">Pervasive developmental disorders</a>, <a href=\"#76002\">Platelet activation, signaling and aggregation</a>, and <a href=\"#2990846\">SUMOylation</a> have a new or revised Illustration with embedded navigation features. </p>\r\n<p><strong>Thanks to our Contributors.</strong>   <a href=\"https://orcid.org/0000-0002-2910-2065\">E Richard Stanley</a> is our external author.  <a href=\"https://orcid.org/0000-0001-9607-8838\">Ravindrababu Chalamalasetty</a>, <a href=\"https://orcid.org/0000-0002-8959-7012\">Clément Charenton</a>, <a href=\"https://orcid.org/0000-0001-7476-6306\">David P Hill</a>, <a href=\"https://orcid.org/0000-0002-0749-2576\">Christian Mosimann</a>, <a href=\"https://orcid.org/0000-0003-0195-3949\">James M Murphy</a>, <a href=\"https://orcid.org/0000-0001-6077-6407\">Karin Prummel</a>, <a href=\"https://orcid.org/0000-0001-8752-6846\">Hiroshi Sasaki</a>, <a href=\"https://orcid.org/0000-0002-2910-2065\">E Richard Stanley</a>, <a href=\"https://orcid.org/0000-0002-9362-4343\">Hailin Tu</a>, and <a href=\"https://orcid.org/0000-0002-7452-4419\">Terry P Yamaguchi</a> are our external reviewers.</p>\r\n<p><strong>Annotation Statistics.</strong> Reactome comprises 14,471 human reactions organized into 2,610 pathways involving 30,656  proteins and modified forms of proteins encoded by 11,442 different human genes, 14,153 complexes, 2,002 small molecules, and 1,113 drugs. These annotations are supported by 36,290 literature references. We have projected these reactions onto 83,933 orthologous proteins, creating  19,417 orthologous pathways in 14 non-human species. Version 83 has annotations for 4,977  protein variants (mutated proteins) and their post-translationally modified forms, derived from 361 proteins, which have contributed to the annotation of 1,659 disease-specific reactions and 704 pathways. </p>\r\n<p><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"#TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and <a href=\"https://reactome.org/AnalysisService/\">Analysis</a> Services. The <a href=\"https://reactome.org/userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"#TOOL=AT\">ReactomeGSA</a> package provides tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p><strong>Documentation and Training.</strong> Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/community/training\">here</a>.</p>\r\n<p><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/what-is-reactome\">website</a>.</p>\r\n<p><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information:</strong> If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <a href=\"mailto:help@reactome.org\">help@reactome.org</a>.</p>','',-2,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-03-23 15:10:54','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(194,882,'V84 released','v84-released','<p style=\"text-align: center;\"><span style=\"font-size: 14pt;\"><a href=\"PathwayBrowser/#/R-HSA-5663205\">Infectious Disease</a></span></p>\r\n<p></p>\r\n<p><img src=\"images/news/infectious_events.png\" alt=\"\" width=\"684\" height=\"401\" style=\"display: block; margin-left: auto; margin-right: auto;\" /></p>\r\n<p></p>\r\n<p><strong>New and Updated Topics and Pathways. </strong>In V84, Topics with new or revised pathways include Developmental Biology (<a href=\"PathwayBrowser/#/R-HSA-9823730\">Formation of definitive endoderm</a>), Gene Expression (<a href=\"PathwayBrowser/#/R-HSA-9772755\">Formation of WDR5-containing histone-modifying complexes</a>), and Signal Transduction (<a href=\"PathwayBrowser/#/5357905\">Regulation of TNFR1 signaling</a>, <a href=\"PathwayBrowser/#/5357956\">TNFR1-induced NF-kappa-B signaling pathway</a>, and <a href=\"PathwayBrowser/#/R-HSA-5357786\">TNFR1-induced proapoptotic signaling</a>). The <a href=\"PathwayBrowser/#/R-HSA-5663205\">Infectious disease</a> pathway has been reorganized with grouping pathways for <a href=\"PathwayBrowser/#/R-HSA-9824439\">Bacterial Infection Pathways</a>, <a href=\"PathwayBrowser/#/R-HSA-9824446\">Viral Infection Pathways</a>, and <a href=\"PathwayBrowser/#/R-HSA-9824443\">Parasitic Infection Pathways</a>. </p>\r\n<p><strong>New and Updated Illustrations.   </strong>In parallel with the restructuring of the infectious disease pathway in V84<strong>, </strong>new or revised Illustrations with embedded navigation features have been created for <a href=\"PathwayBrowser/#/R-HSA-5663205\">Infectious disease</a>, <a href=\"PathwayBrowser/#/9824439\">Bacterial Infection Pathways</a>, <a href=\"PathwayBrowser/#/R-HSA-9824446\">Viral Infection Pathways</a>, and <a href=\"PathwayBrowser/#/R-HSA-9824443\">Parasitic Infection Pathways</a>. Additional disease pathways with new or updated Illustrations include <a href=\"PathwayBrowser/#/R-HSA-9687139\">Aberrant regulation of mitotic cell cycle due to RB1 defects</a>, <a href=\"PathwayBrowser/#/R-HSA-9668250\">Defective factor IX causes hemophilia B</a>, <a href=\"PathwayBrowser/#/R-HSA-9662001\">Defective factor VIII causes hemophilia A</a>, <a href=\"PathwayBrowser/#/R-HSA-9630747\">Diseases of Cellular Senescence</a>, and <a href=\"PathwayBrowser/#/R-HSA-9675126\">Diseases of Mitotic Cell Cycle</a>. Normal pathways with new or revised illustrations include <a href=\"PathwayBrowser/#/R-HSA-212165\">Epigenetic regulation of gene expression</a> and <a href=\"PathwayBrowser/#/R-HSA-9758941\">Gastrulation</a>.</p>\r\n<p><strong>Thanks to our Contributors.</strong>   <a href=\"https://orcid.org/0000-0003-3532-8586\">Toni Franjkić</a>, <a href=\"https://orcid.org/0000-0002-7442-5138\">Kai Ge</a>, <a href=\"https://orcid.org/0000-0001-7476-6306\">David P Hill</a>, <a href=\"https://orcid.org/0000-0002-5171-9950\">Ivana Munitic</a>, <a href=\"https://orcid.org/0000-0002-8155-4296\">Nazmus Salehin</a>, <a href=\"https://orcid.org/0000-0001-6950-8388\">Patrick Tam</a>, and <a href=\"https://orcid.org/0000-0002-3328-915X\">Hieu Van</a> are our external reviewers.</p>\r\n<p><strong>Annotation Statistics.</strong> Reactome comprises 14516 human reactions organized into 2615 pathways involving 30095 proteins and modified forms of proteins encoded by 11371 different human genes, 14194 complexes, 2002 small molecules, and  1113 drugs. These annotations are supported by 36444 literature references. We have projected these reactions onto 84090 orthologous proteins, creating 19435 orthologous pathways in 14 non-human species. Version 84 has annotations for 4983 protein variants (mutated proteins) and their post-translationally modified forms, derived from 353 proteins, which have contributed to the annotation of  1,659 disease-specific reactions and  704 pathways. </p>\r\n<p><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and <a href=\"https://reactome.org/AnalysisService/\">Analysis</a> Services. The <a href=\"https://reactome.org/userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"PathwayBrowser/#TOOL=AT\">ReactomeGSA</a> package provide tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p><strong>Documentation and Training.</strong> Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/community/training\">here</a>.</p>\r\n<p><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a> as well as a <a href=\"https://globalbiodata.org/scientific-activities/global-core-biodata-resources/\">Global Core Biodata Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/what-is-reactome\">website</a>.</p>\r\n<p><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information:</strong> If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <a href=\"mailto:help@reactome.org\">help@reactome.org</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-03-23 19:07:16','0000-00-00 00:00:00','{}','{}','{}',1,7,'','',1,0,'{}',0,'en-GB','',''),(195,893,'I searched for (...) and didn’t get any results.','no-results','<p>Your search may be correct as Reactome does not yet have complete coverage of human biology. But before concluding that there is no content relevant to your query, please check that your search is formatted appropriately:</p>\r\n<p>The search bar on Reactome’s homepage takes a variety of inputs including but not limited to: HGNC gene names, protein names, identifiers from Uniprot, ChEBI or other resources, and simple word or phrase searches (“glucose”; “signaling by ERBB2”; “Li Fraumeni syndrome”). </p>\r\n<p>Multiple search terms separated by a space may be entered in a single query; the results will be the total hits generated by each of the terms searched independently. Use of other punctuation (slashes, commas) may not yield full results and are better avoided.</p>\r\n<p>To search for an exact match, enclose your search term(s) in quotation marks.</p>\r\n<p><a href=\"https://solr.apache.org/guide/6_6/the-standard-query-parser.html#TheStandardQueryParser-BooleanOperatorsSupportedbytheStandardQueryParser\" target=\"_blank\" rel=\"noopener\">Boolean operators</a> (AND, OR, NOT) may also be used in formatting a search.</p>\r\n<p>Wild cards may be used to expand your search:</p>\r\n<ul>\r\n<li aria-level=\"1\">? represents one character e.g. A1?? Matches both A1CF and A1BG</li>\r\n<li aria-level=\"1\">* represents n characters e.g. *A1* matches A1CF, A1BG and A1A4E9; also VWA1, ATP1A1</li>\r\n</ul>\r\n<p>When entering a database identifier, more accurate results will be generated if the search term is formatted using the syntax database:id (for instance, uniprot:P60484). For complex database names like Guide to Pharmacology, replace spaces with dashes (Guide-to-Pharmacology:7382)</p>\r\n<p>Hits of a successful search indicate that the search term is included somewhere in the identified record. This may be identification of a physical entity or event that directly involves the search term, but the search term may also be used in an event summary or as part of an associated literature reference, for example.</p>','',1,37,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 09:40:10','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(196,894,'Is there a way to download high level images which can be zoomed in locally (after download)?','high-levek-images','<p>If you would like to download icons and Enhanced High Level Diagrams(EHLD), yes, we provide SVG format for both. Please visit our icon library <a href=\"icon-lib\">here</a>, you can download any EHLDs in pathway browser by clicking the download button at the top right corner to export diagram to different formats including SVG.</p>\r\n<p>However, if you would like to download a high level image of Reacfoam, unfortunately, there is no way to export it for now, basically, it takes a screenshot of the current window when you download a Reacfoam image.</p>','',1,38,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 09:47:01','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(197,895,'I used the “Analyse gene expression” tool with my mouse proteomics data set. I notice that the database is based on human pathways – does the software automatically convert my mouse gene names to the human orthologs? Or can’t I use the software for mouse?','convert-to-human','<p>Reactome’s manual curation covers human proteins, but we do support analysis of non-human data sets.</p>\r\n<p>There are two methods to analyze data in Reactome. </p>\r\n<p>The “Analyse gene list” tool, accessed after “Analysis” is selected from the home page,  allows users to upload human or non-human data sets for overrepresentation analysis. After uploading the data with this tool, the user is given the option of ‘projecting to human’ (which is selected by default). If this toggle is selected, non-human identifiers in the data set are converted to their human equivalents using orthology information from Panther. </p>\r\n<p>Data sets can also be analyzed with the Reactome Gene Set Analysis (Reactome GSA) tool, accessed through the “Analyze Gene Expression” button after “Analysis” is selected from the home page.  Reactome GSA performs quantitative pathway analyses, increasing the statistical power of the differential gene expression analysis. The analysis software automatically converts the mouse proteins from your proteomics set to their human orthologs.<em> </em></p>','',1,40,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 09:49:03','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(198,896,'I am having trouble installing a local Reactome Neo4J server.','install-neo4j','<p>Unfortunately, installation of a local Neo4J server can be complicated due to variations in individual setup (computer, operating system, version of Neo4J and so forth). </p>\r\n<p>Please follow this <a href=\"dev/graph-database\">guide</a> to install the graph database locally, there are different ways to install the graph database, and we also provide graph tar file and dump file to meet different requirements.</p>\r\n<p>As a long-term solution to this issue, we are working on providing the Neo4J databases in Docker containers. When these are ready for deployment, they will be added to the Reactome Download page.</p>\r\n<p>In the meantime, if you are getting an error message that the database is unavailable, please feel free to contact our developers through the help email (<a href=\"mailto:help@reactome.org\"></a><a href=\"mailto:help@reactome.org\">help@reactome.org</a>) and we will try to assist you.</p>','',1,43,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 09:51:29','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,5,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(199,897,'Does Reactome contain pathway information from non-human species?','non-human-species','<p>Reactome is centered on the molecular functions of human proteins. When possible, we annotate these functions with published evidence from work with human systems. When such evidence isn’t available, but the function is known to be well conserved across species and experimental evidence exists for a non-human species, we annotate the reaction for the protein in that species and manually infer the reaction involving the homologous human protein.</p>\r\n<p>We also computationally infer reactions for a small group of model organisms from our manually annotated human data. This group of organisms is centered on organisms of interest to the Alliance for Genome Resources. The process for these computationally inferred events is <a href=\"documentation/inferred-events\">here</a>.</p>\r\n<p>In a collaborative project, the Reactome data model has been adopted by <a href=\"https://plantreactome.gramene.org/index.php?option=com_content&amp;view=featured&amp;Itemid=101&amp;lang=en\">Plant Reactome</a> to capture pathway information for diverse species of plants, including crop plants. </p>','',1,37,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 14:47:17','0000-00-00 00:00:00','{}','{}','{}',1,5,'','',1,0,'{}',0,'en-GB','',''),(200,898,'Patient-derived cell-based pharmacogenomic assessment to unveil underlying resistance mechanisms and novel therapeutics for advanced lung cancer','patient-derived-cell-based-pharmacogenomic-assessment-to-unveil-underlying-resistance-mechanisms-and-novel-therapeutics-for-advanced-lung-cancer','<p>The Reactome database helped <a href=\"https://jeccr.biomedcentral.com/articles/10.1186/s13046-023-02606-3\">Yu et al. in the January 2023 issue of the Journal of Experimental &amp; Clinical Cancer Research</a> identify candidate drugs for treatment of four subtypes of lung cancer that were categorized by pharmaco-genomic analysis of patient-derived cells and correlated with drug sensitivity of the patient-derived cells and with Reactome pathways identified by gene set variation analysis.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 17:42:02','0000-00-00 00:00:00','{}','{}','{}',1,18,'','',1,0,'{}',0,'en-GB','',''),(201,899,'Explain the identifiers associated with entities and events.','identifiers','<p>Each Reactome instance of any type gets a unique internal database identifier (DB_ID) when it is created that persists unchanged for the life of that instance and is not reused if that instance is deleted.</p>\r\n<p></p>\r\n<p>The stable IDs visible on our website and in our downloads are generated only for instances that are physical entities (chemicals, genome-encoded entities, complexes and sets of these) and events (reactions and pathways). Each stable ID takes the following form:  R (Reactome) - three-letter code for the species of the instance- DB_ID.version (to indicate its version if the instance has been revised since its creation). So human entities and events have stable ids with R-HSA (for <span style=\"color: #ff0000;\">H</span>omo <span style=\"color: #ff0000;\">sa</span>piens)-########.##,  stable ids for <span style=\"color: #ff0000;\">C</span>aenorhabditis <span style=\"color: #ff0000;\">el</span>egans are R-CEL-########.##, etc. For simple chemicals, ALL is used as the species code (R-ALL-########.##). Like DB_IDs, stable identifiers persist for the lifetime of the instance, unchanged except for versioning, and are not reused if the instance is deleted.</p>\r\n<p></p>\r\n<p>For manually annotated events, the species code is that of the species in which the event occurs. Barring curation errors, that species code always corresponds to the species listed in the details panel of the web page for the entity or event.</p>\r\n<p></p>\r\n<p>For computationally inferred events, the species code is that of the model organism species and the DB identifier is the one assigned to the human event that is the basis of the inference. This is to reflect the fact that these inferences are intrinsically UNstable (because as the model organism genome build changes, the inferences based on sequence similarity to that model organism will also change).</p>','',1,37,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:21:18','0000-00-00 00:00:00','{}','{}','{}',1,4,'','',1,0,'{}',0,'en-GB','',''),(202,900,'Where can I find earlier versions of the Reactome database?','earlier-versions','<p>We are working at making earlier versions of the Reactome database publicly available. In the meantime, MySQL copies of earlier database releases are available through the general URL structure:</p>\r\n<p>https://download.reactome.org/XX/databases/gk_current.sql.gz</p>\r\n<p></p>\r\n<p>where XX is the version code for the release (ie 80 for Version 80, released 3/2022). See <a href=\"about/release-calendar\">here</a> for a list of Reactome version release dates. Note that version 60, 65 and 70 onward are available for download.</p>\r\n<p></p>\r\n<p>Note that we are currently transitioning from a MySQL to Neo4J database. The current release of the database is available in both formats <a href=\"download-data\">here</a>.</p>','',1,37,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:24:44','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(203,901,'How can I find all pathways reported for one organ? For example, I want to extract all reported pathways in (tissue X)  for (species Y).','pathways-per-organ','<p>Reactome does not provide organ-specific annotation, we aim to annotate a generic (human) cell, and users can then use expression data overlay for their organ (cell, tissue) of interest through our <a href=\"/PathwayBrowser/#TOOL=AT\">analysis tool</a> (select \"Microarray data\" for an example). </p>\r\n<p></p>\r\n<p>You can also use the \"Tissue Distribution\" tool to see how data from the <a href=\"https://www.proteinatlas.org/\">Human Protein Atlas</a> maps to Reactome pathway space.</p>\r\n<p></p>\r\n<p>Documentation is at <a href=\"https://reactome.org/userguide/analysis\"></a><a href=\"userguide/analysis\">https://reactome.org/userguide/analysis</a></p>\r\n<p></p>\r\n<p>Please keep in mind that Reactome curation focuses on human, and while you can switch to one of the other species supported by our inference protocol through the drop down in the upper left area of the pathway browser, this data is inferred from human by orthology.</p>','',1,37,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:28:00','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(204,902,'Is there a mapping file between KEGG and Reactome pathways?','kegg-to-reactome','<p>Reactome does not have a mapping file to KEGG. The best reference for such a comparison is <a href=\"https://compath.scai.fraunhofer.de/\">ComPath</a> (reference <a href=\"https://pubmed.ncbi.nlm.nih.gov/30564458/\">here</a>), however this resource may not be up to date.</p>\r\n<p></p>\r\n<p>As an alternate approach, users can try the method used to establish links from models in the BioModels database to Reactome. For each model, we took its gene set, ran a gene set enrichment analysis against Reactome using the Reactome API documented <a href=\"dev/analysis\">here</a>, and then added links to the Reactome pathways below a certain p value cutoff. </p>\r\n<p></p>\r\n<p>You could use that method to identify Reactome pathways that are related to a given KEGG pathway, and consider pathways with no match below your p value cutoff as not matched. The Reactome analysis interface is fast, each query should return in a few seconds, so this is perfectly feasible for all of KEGG. We\'d be happy to communicate further on this.</p>','',1,37,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:30:35','0000-00-00 00:00:00','{}','{}','{}',1,1,'','',1,0,'{}',0,'en-GB','',''),(205,903,'Is it possible to download the inferred pathways for mouse (or another species), similar to the GMT file for human pathways on the downloads page? ','inferred-pathways-download','<p>Unfortunately, this is not possible at this time.</p>','',1,37,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:32:36','0000-00-00 00:00:00','{}','{}','{}',1,0,'','',1,0,'{}',0,'en-GB','',''),(206,904,'DELETE:I used the “Analyse gene expression” tool with my mouse proteomics data set. I notice that the database is based on human pathways – does the software automatically convert my mouse gene names to the human orthologs? Or can’t I use the software for','non-human-analysis','<p>Reactome’s manual curation covers human proteins, but we do support analysis of non-human data sets.</p>\r\n<p></p>\r\n<p>There are two methods to analyze data in Reactome. </p>\r\n<p></p>\r\n<p>The “Analyse gene list” tool, accessed after “Analysis” is selected from the home page,  allows users to upload human or non-human data sets for overrepresentation analysis. After uploading the data with this tool, the user is given the option of ‘projecting to human’ (which is selected by default). If this toggle is selected, non-human identifiers in the data set are converted to their human equivalents using orthology information from Panther. </p>\r\n<p></p>\r\n<p>Data sets can also be analyzed with the Reactome Gene Set Analysis (Reactome GSA) tool, accessed through the “Analyze Gene Expression” button after “Analysis” is selected from the home page.  Reactome GSA performs quantitative pathway analyses, increasing the statistical power of the differential gene expression analysis. The analysis software automatically converts the mouse proteins from your proteomics set to their human orthologs.</p>','',-2,39,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:35:00','0000-00-00 00:00:00','{}','{}','{}',1,0,'','',1,0,'{}',0,'en-GB','',''),(207,905,'The analysis report includes lists of significantly up- and down-regulated single genes. Which statistical analysis does the software use for these values?','statistical-analysis','<p>As described in our <a href=\"userguide/analysis\">on-line documentation</a>, the analysis returns:</p>\r\n<ul>\r\n<li aria-level=\"1\">Entities p-value: the result of the statistical binomial test for over-representation for molecules of the results type selected.</li>\r\n<li aria-level=\"1\">Entities FDR: False discovery rate. Corrected over-representation probability.</li>\r\n</ul>\r\n<p>Note that if performing an analysis with non-human data, the statistics may be skewed due to changes in the sizes of gene families between human and the species associated with the submitted data.</p>','',1,40,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:41:58','0000-00-00 00:00:00','{}','{}','{}',1,4,'','',1,0,'{}',0,'en-GB','',''),(208,906,'Can I visualize gene expression data (color nodes) according to gene expression level with Reactome?','visualizing-gene-expression','<p>From the <a href=\"https://reactome.org/\">home page</a>, click the “Analysis Tools” button, and then select “Analyse gene list” (preselected on the analysis page).</p>\r\n<p>Submit your list of differentially expressed genes, or try one of the sample data sets.</p>\r\n<p>The leftmost column of the uploaded data set must have gene names or other gene/protein identifiers. The remainder of the data set consists of as many columns with numerical values as necessary. Columns may have headers, or not.</p>\r\n<p>Click “Continue” and then “Analyse”,  maintaining the options in the second window (‘Project to human’, ‘Include interactors’) at their default values.</p>\r\n<p>Reactome will perform a standard gene set enrichment analysis, based only on the gene list. In the results overview, pathways will be greyed out if they are not significantly enriched. If a pathway is enriched, its colour will be determined as follows:</p>\r\n<p>The top end of the colour map is the highest value of all submitted numerical values. The bottom end of the colour map is the lowest of all submitted numerical values, across all columns.</p>\r\n<p>The colour of a pathway is based on the average expression value for all genes which are in the submitted dataset and in the pathway. In the initial view, this average is based on the first column. If the data set has more than one column, users can cycle through the subsequent columns with the “Play” button at the bottom of the pathway window.</p>\r\n<p>In the pathways overview (either the \"Fireworks\" or the \"Reacfoam\" view, selectable in the top left area of the main window), the user can double click (long click in Reacfoam to select a pathway and zoom in to the detailed molecular map view. In this view, proteins are coloured according to the user-provided values, and again the user can cycle through multiple columns with the \"Play\" controls.</p>','',1,40,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:45:29','0000-00-00 00:00:00','{}','{}','{}',1,3,'','',1,0,'{}',0,'en-GB','',''),(209,907,'After submitting a list of genes for Reactome Analysis, how do I download the fully expanded \'Event Hierarchy\' list on the left side of the screen? ','expanded-event-hierarchy','<p>We provide an <a href=\"https://urldefense.com/v3/__https:/reactome.org/ContentService/*/events/getEventHierarchy__;Iw!!MXfaZl3l!f5prtwb7oWN5ivYvI2TbaVKQ1sEpdIqEKZk7ZiI_fbWN8xZgQOnaiH3XTJI8hNGxM5F0XF-XL61JiWfqWGysBHu7T0I$\">endpoint</a> to retrieve the full event hierarchy for a given species. For example, you can get human data with a token like the one below.  Please replace “your_analysis_token” in the URL below with your own token.</p>\r\n<p><em> </em></p>\r\n<p><a href=\"https://urldefense.com/v3/__https:/reactome.org/ContentService/data/eventsHierarchy/9606?pathwaysOnly=false&amp;token=your_analysis_token&amp;resource=TOTAL&amp;interactors=false__;!!MXfaZl3l!f5prtwb7oWN5ivYvI2TbaVKQ1sEpdIqEKZk7ZiI_fbWN8xZgQOnaiH3XTJI8hNGxM5F0XF-XL61JiWfqWGysRwYFfd8$\"><em></em></a><em>https://reactome.org/ContentService/data/eventsHierarchy/9606?pathwaysOnly=false&amp;token=</em><em>your_analysis_token</em><em>&amp;resource=TOTAL&amp;interactors=false</em></p>','',1,40,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:46:53','0000-00-00 00:00:00','{}','{}','{}',1,2,'','',1,0,'{}',0,'en-GB','',''),(210,908,'Is it possible to know how many of my uploaded genes are found within each specific hierarchical pathway category identified in the Reacfoam view? For instance, how many of my genes belong to pathways that would be included in the \"Immune\" category, etc. ','query-genes-per-pathway','<p>At the moment we don’t have a way to filter the results by top level pathway, or diagram-level pathway or other criteria. This feature is on our radar to implement. </p>\r\n<p>In the meantime, you can get a sense of this as follows:</p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li>Click on the ‘Analysis’ tab in the Details panel (below the pathway diagram window- the Analysis tab should be open by default after performing analysis). </li>\r\n<li>To the left of the Details panel, click on the ‘Download’ button.</li>\r\n<li>Click on the “Pathway analysis results”. This gives the analysis results for all pathways, and for higher-order pathways the columns \"Entities found\" and \"Entities total\" contain the aggregated results from the sub-pathways. </li>\r\n<li>Import the results into a spreadsheet. Although we don\'t have a column \"Pathway level\" or similar, which would allow you to filter for only top level pathways, if you sort the pathways by decreasing \"Entities found\", you get a good view of the top level, and thus typically (but not always) largest pathways.</li>\r\n</ul>','',1,40,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:49:56','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(211,909,'Is there a permanent link available for my analysis results?','permanent-analysis-results','<p>Unfortunately, there is no permanent link for analysis results. Analysis data is available through the token for 7 days after your last usage. Analysis results are deleted when Reactome releases new data, regardless of when you performed your analysis or last accessed your data.</p>','',1,40,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:51:45','0000-00-00 00:00:00','{}','{}','{}',1,0,'','',1,0,'{}',0,'en-GB','',''),(212,910,'I have a proteomics data set that I would like to use for quantitative pathway analysis. Is there training material available to help me? ','gsa-training-material','<p>To process quantitative proteomics data, the best, but also most complex analysis option is ReactomeGSA. A description of the tool, its use, and documentation are all part of <a href=\"https://pubmed.ncbi.nlm.nih.gov/32907876/\">this publication</a></p>\r\n<p>There is also a comprehensive training video <a href=\"https://www.ebi.ac.uk/training/events/multiomics-comparative-pathway-analysis-reactomegsa/\">here</a>.</p>\r\n<p>If after going through these resources you have additional questions, please reach out to the help desk at <a href=\"mailto:help@reactome.org\">help@reactome.org</a></p>','',1,41,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:53:16','0000-00-00 00:00:00','{}','{}','{}',1,2,'','',1,0,'{}',0,'en-GB','',''),(213,911,'Can I submit a list of differentially expressed genes, as log2 fold change values, coming from two different conditions to have a quantitative representation of pathways involvement? If yes, how can I prepare and submit my file?','differentially-expressed-genes','<p>For this, please use the Reactome Gene Set Analysis tool, accessed through the “Analyze Gene Expression” button after “Analysis” is selected from the home page. The detailed analysis approaches underlying this feature are described <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7710148/\">here</a><em>.</em></p>\r\n<p>The Reactome GSA tool is an enhancement of <a href=\"https://www.gsea-msigdb.org/gsea/index.jsp\">GSEA</a> and can take your original expression data directly, perform differential expression analysis and then highlight pathways according scores.</p>','',1,41,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:55:43','0000-00-00 00:00:00','{}','{}','{}',1,1,'','',1,0,'{}',0,'en-GB','',''),(214,912,'I submitted a list for expression analysis using the PADOG or CAMERA tools and the output Excel file with the statistics is missing a lot of significant genes. What is the explanation for this?','truncated-gsa-analysis-output','<p>Although there is no limit on the number of genes that can be returned, the analysis is restricted to those genes that are present in Reactome. Since Reactome is a manually curated resource, it does not cover all human genes.</p>','',1,41,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 22:57:32','0000-00-00 00:00:00','{}','{}','{}',1,0,'','',1,0,'{}',0,'en-GB','',''),(215,913,'Is there an API for Reactome? Can I do pathway analysis through the API?','pathway-analysis-api','<p>Pathway analysis can be performed through our API, documented <a href=\"AnalysisService/\">here</a>.</p>','',1,44,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 23:03:48','0000-00-00 00:00:00','{}','{}','{}',1,2,'','',1,0,'{}',0,'en-GB','',''),(216,914,'Should I use the gene symbol or the NCBI identifiers to make requests to the identifiers endpoint of the Analysis Service?','gene-symbol','<p>We suggest using the HGVS gene name or the Uniprot identifier in your analysis. Different results may be obtained using other identifiers (like NCBIs) if those identifiers also map to other resources, bringing in more pathways.</p>','',1,44,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 23:06:06','0000-00-00 00:00:00','{}','{}','{}',1,1,'','',1,0,'{}',0,'en-GB','',''),(217,915,'Can I do pathway analysis using R?','pathway-analysis-r','<p>Although we don\'t provide a specific R package suitable for visualizing expression values on Reactome pathways, we do provide a full BioConductor package for quantitative pathway analysis based on expression data.  Please see <a href=\"https://bioconductor.org/packages/release/bioc/vignettes/ReactomeGSA/inst/doc/using-reactomegsa.html\">here</a> for documentation, and <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7710148/\">here</a> for the relevant publication.</p>','',1,44,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 23:07:48','0000-00-00 00:00:00','{}','{}','{}',1,0,'','',1,0,'{}',0,'en-GB','',''),(218,916,'I need to do a network analysis for (species) from RNA seq data. Can you please guide me on how to generate an FI network? ','non-human-fiviz','<p>Unfortunately our ReactomeFIViz tool currently supports networks for human and mouse only. For bacterial species, you can try <a href=\"https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fstring-db.org%2F&amp;data=05%7C01%7CKaren.Rothfels%40oicr.on.ca%7C772e45301d1e4b0bd52408db26427356%7C9df949f8a6eb419d9caa1f8c83db674f%7C0%7C0%7C638145839076745935%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=H0R0c7HE2PZPXEM0e82%2FdnlkA6sNw%2Fk%2FLtV2fLjGgLg%3D&amp;reserved=0\">StringDB</a>, which provides software tools to build networks for your genes.</p>','',1,42,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 23:08:51','0000-00-00 00:00:00','{}','{}','{}',1,1,'','',1,0,'{}',0,'en-GB','',''),(219,917,'Can I submit a list of differentially expressed genes, as log2 fold change values, coming from two different conditions to have a quantitative representation of pathways involvement?','fiviz-differential-expression','<p>If you want to use your log fold change data directly, you may use the Reactome Cytoscape app, described <a href=\"userguide/reactome-fiviz\">here</a> (search for “Perform GSEA analysis”). This feature will sort your genes based on log fold change and then calculate a score for each pathway using GSEA’s ranked gene list input.</p>','',1,42,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 23:18:28','0000-00-00 00:00:00','{}','{}','{}',1,0,'','',1,0,'{}',0,'en-GB','',''),(220,918,'DELETE:I am having trouble installing a local Reactome Neo4J server.','local-neo4j','<p>Unfortunately, installation of a local Neo4J server can be complicated due to variations in individual setup (computer, operating system, version of Neo4J and so forth). </p>\r\n<p></p>\r\n<p>Please follow this <a href=\"dev/graph-database\">guide</a> to install the graph database locally, there are different ways to install the graph database, and we also provide graph tar file and dump file to meet different requirements.</p>\r\n<p></p>\r\n<p>As a long-term solution to this issue, we are working on providing the Neo4J databases in Docker containers. When these are ready for deployment, they will be added to the Reactome Download page.</p>\r\n<p><br />In the meantime, if you are getting an error message that the database is unavailable, please feel free to contact our developers through the help email (<a href=\"mailto:help@reactome.org\"></a><a href=\"mailto:help@reactome.org\">help@reactome.org</a>) and we will try to assist you.</p>','',-2,43,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 23:20:08','0000-00-00 00:00:00','{}','{}','{}',1,4,'','',1,0,'{}',0,'en-GB','',''),(221,919,'I have a list of input genes. I want to retrieve all the pathways in which the genes are involved. How can I do this using a Neo4j Cypher query? ','cypher-gene-list-to-pathways','<style type=\"text/css\">.rcypher, .rtip, .rresult {\r\n  border: 1px solid #dddddd;\r\n  border-radius: 5px;\r\n  background: #f9f9f9;\r\n  margin-left: 15px;\r\n  margin-right: 15px;\r\n  overflow-x: scroll;\r\n  padding: 10px;\r\n}\r\n.rtip {\r\n  border: 1px solid pink;\r\n  background-color: #ffe6e6;\r\n}\r\n.rresult{\r\n  font-family: monospace;\r\n  font-size: x-small;\r\n  color: darkgray;\r\n        word-wrap: initial;jq\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n.rcypher code, .rtip code {\r\n        word-wrap: initial;\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n@media only screen and (max-width: 768px) {\r\n    .rcypher, .rtip, .rresult {\r\n  margin-left: 0px;\r\n        margin-right: 0px;\r\n    }\r\n    .rresult {\r\n        overflow-y: auto;\r\n        white-space: pre;\r\n    }\r\n}</style>\r\n<p>There is a <a href=\"dev/graph-database/extract-participating-molecules\">tutorial</a>  on using neo4j Cypher query here.</p>\r\n<p>To retrieve all human pathways for a single gene (for example: <a href=\"https://www.uniprot.org/uniprotkb/P36897/entry\">P36897</a>, the Uniprot identifier for TGFR1)</p>\r\n<p>{source}<span style=\"font-family: courier new, courier, monospace;\">&lt;pre class=\"rcypher\"&gt;&lt;code class=\"language-cypher\" data-lang=\"cypher\"&gt;<br />MATCH (n)-[:referenceDatabase]-&gt;(rd:ReferenceDatabase)<br />WHERE toLower(rd.displayName) = toLower(\"UniProt\") AND (n.identifier = \"P36897\" OR n.variantIdentifier =\"P36897\" OR \"P36897\" IN n.geneName OR \"P36897\" IN n.name)<br />WITH DISTINCT n <br />MATCH (pe:PhysicalEntity)-[:referenceEntity|referenceSequence|crossReference|referenceGene*]-&gt;(n)<br />WITH DISTINCT pe<br />MATCH (rle:ReactionLikeEvent)-[:input|output|catalystActivity|physicalEntity|entityFunctionalStatus|diseaseEntity|regulatedBy|regulator|hasComponent|hasMember|hasCandidate|repeatedUnit*]-&gt;(pe)<br />WITH DISTINCT rle <br />MATCH (:Species{taxId:\"9606\"})&lt;-[:species]-(p:Pathway)-[:hasEvent]-&gt;(rle)<br />RETURN DISTINCT p <br />ORDER BY p.stId<br />&lt;/code&gt;&lt;/pre&gt;</span>{/source}</p>\r\n<p>If you have a list of genes, we suggest using <a href=\"https://neo4j.com/docs/cypher-manual/current/clauses/unwind/\">UNWIND</a> neo4j cypher to flatten your list back to individual items and then execute the query.</p>','',1,43,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 23:27:26','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(222,920,'I have a list of Reactome pathway IDs. I want to find whether any of them are disease pathways and also retrieve associated disease names. Is it possible to retrieve the information using API or neo4j graph database query?','disease-to-pathways-api-or-neo4j','<style type=\"text/css\">.rcypher, .rtip, .rresult {\r\n  border: 1px solid #dddddd;\r\n  border-radius: 5px;\r\n  background: #f9f9f9;\r\n  margin-left: 15px;\r\n  margin-right: 15px;\r\n  overflow-x: scroll;\r\n  padding: 10px;\r\n}\r\n.rtip {\r\n  border: 1px solid pink;\r\n  background-color: #ffe6e6;\r\n}\r\n.rresult{\r\n  font-family: monospace;\r\n  font-size: x-small;\r\n  color: darkgray;\r\n        word-wrap: initial;jq\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n.rcypher code, .rtip code {\r\n        word-wrap: initial;\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n@media only screen and (max-width: 768px) {\r\n    .rcypher, .rtip, .rresult {\r\n  margin-left: 0px;\r\n        margin-right: 0px;\r\n    }\r\n    .rresult {\r\n        overflow-y: auto;\r\n        white-space: pre;\r\n    }\r\n}</style>\r\n<p>To do this, you can do a POST query with your pathway identifiers to the endpoint described <a href=\"#/query/findByIds\">here</a>: </p>\r\n<p>To know if a Pathway is a disease, look at  the “isInDisease” property. The disease name is contained in the displayName of the diseases in the “disease” array of the Pathway models.</p>\r\n<p>One query example would be </p>\r\n<p>{source}<span style=\"font-family: \'courier new\', courier, monospace;\">&lt;pre class=\"rcypher\"&gt;&lt;code class=\"language-cypher\" data-lang=\"cypher\"&gt;<br />curl -X <span style=\"color: #339966;\">\'POST\'</span> \\<br /><img src=\"/media/sourcerer/images/tab.png\" alt=\"\" />\'https://reactome.org/ContentService/data/query/ids\' \\<br /><img src=\"/media/sourcerer/images/tab.png\" alt=\"\" />-H <span style=\"color: #339966;\">\'accept: */*\'</span> \\<br /><img src=\"/media/sourcerer/images/tab.png\" alt=\"\" />-H <span style=\"color: #339966;\">\'Content-Type: text/plain\'</span> \\<br /><img src=\"/media/sourcerer/images/tab.png\" alt=\"\" />-d <span style=\"color: #339966;\">\'R-HSA-9679506,R-HSA-8876384\'</span> &gt; pathwaysDescription.json<br />&lt;/code&gt;&lt;/pre&gt;</span>{/source}</p>\r\n<p>To do this for multiple ids using cypher, you can use this query:</p>\r\n<p></p>\r\n<p>{source}<span style=\"font-family: \'courier new\', courier, monospace;\">&lt;pre class=\"rcypher\"&gt;&lt;code class=\"language-cypher\" data-lang=\"cypher\"&gt;<br /><span style=\"color: #339966;\">MATCH</span> (p:Pathway)<br /><span style=\"color: #339966;\">WHERE</span> p.stId in [<span style=\"color: #ff9900;\">\"R-HSA-9679506\",\"R-HSA-8876384\"</span>]<br /><span style=\"color: #339966;\">OPTIONAL MATCH</span> (p)-[:disease]-&gt;(d:Disease)<br /><span style=\"color: #339966;\">RETURN</span> p,d<br />&lt;/code&gt;&lt;/pre&gt;</span>{/source}</p>\r\n<p>Using the optional match allows you to get both pathways associated with a disease and those which are not. </p>\r\n<p>If you want to filter for only disease pathways, use the following:</p>\r\n<p>{source}<span style=\"font-family: \'courier new\', courier, monospace;\">&lt;pre class=\"rcypher\"&gt;&lt;code class=\"language-cypher\" data-lang=\"cypher\"&gt;<br /><span style=\"color: #339966;\">MATCH</span> (p:Pathway)-[:disease]-&gt;(d:Disease)<br /><span style=\"color: #339966;\">WHERE</span> p.stId in [<span style=\"color: #ff9900;\">\"R-HSA-9679506\",\"R-HSA-8876384\"</span>]<br /><span style=\"color: #339966;\">RETURN</span> p,d<br />&lt;/code&gt;&lt;/pre&gt;</span>{/source}</p>\r\n<p>You just need to provide your list of stId (reactome identifiers) in the “where” list.</p>\r\n<p>Finally, if you want to export a table instead of viewing it in the browser, use the following:</p>\r\n<p>{source}<span style=\"font-family: \'courier new\', courier, monospace;\">&lt;pre class=\"rcypher\"&gt;&lt;code class=\"language-cypher\" data-lang=\"cypher\"&gt;<br /><span style=\"color: #339966;\">MATCH</span> (p:Pathway)<br /><span style=\"color: #339966;\">WHERE</span> p.stId in [<span style=\"color: #ff9900;\">\"R-HSA-9679506\",\"R-HSA-8876384\"</span>]<br /><span style=\"color: #339966;\">OPTIONAL MATCH</span> (p)-[:disease]-&gt;(d:Disease)<br /><span style=\"color: #339966;\">WITH</span> p, collect(d) as diseases<br /><span style=\"color: #339966;\">RETURN</span> p.stId, p.displayName, p.isInDisease, [d in diseases | d.displayName] as diseaseNames, [d in diseases | d.databaseName + \":\" + d.identifier] as diseaseIdentifiers<br />&lt;/code&gt;&lt;/pre&gt;</span>{/source}</p>\r\n<p>Note that some pathways, like R-HSA-9608290 are associated with several diseases.</p>','',1,43,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 23:33:49','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(223,921,'How can I use Neo4J to identify all the genes for a given pathway?','neo4j-all-genes-for-a-pathway','<style type=\"text/css\">.rcypher, .rtip, .rresult {\r\n  border: 1px solid #dddddd;\r\n  border-radius: 5px;\r\n  background: #f9f9f9;\r\n  margin-left: 15px;\r\n  margin-right: 15px;\r\n  overflow-x: scroll;\r\n  padding: 10px;\r\n}\r\n.rtip {\r\n  border: 1px solid pink;\r\n  background-color: #ffe6e6;\r\n}\r\n.rresult{\r\n  font-family: monospace;\r\n  font-size: x-small;\r\n  color: darkgray;\r\n        word-wrap: initial;jq\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n.rcypher code, .rtip code {\r\n        word-wrap: initial;\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n@media only screen and (max-width: 768px) {\r\n    .rcypher, .rtip, .rresult {\r\n  margin-left: 0px;\r\n        margin-right: 0px;\r\n    }\r\n    .rresult {\r\n        overflow-y: auto;\r\n        white-space: pre;\r\n    }\r\n}</style>\r\n<p>The query for collecting all genes for a given pathway is</p>\r\n<p>{source}<span style=\"font-family: courier new, courier, monospace;\">&lt;pre class=\"rcypher\"&gt;&lt;code class=\"language-cypher\" data-lang=\"cypher\"&gt;<br />MATCH (n:DatabaseObject{stId:\"R-HSA-3371599\"})-[:hasEvent|input|output|catalystActivity|physicalEntity|entityFunctionalStatus|diseaseEntity|regulatedBy|regulator|hasComponent|hasMember|hasCandidate|repeatedUnit|referenceEntity*]-&gt;(m:ReferenceGeneProduct)<br />RETURN DISTINCT m<br />&lt;/code&gt;&lt;/pre&gt;</span>{/source}</p>\r\n<p>For guidance on formatting other queries, please take a look at our <a href=\"content/schema/DatabaseObject\">data schema</a>.  Our graph core projects also include most of the queries used in Reactome; you can find lots of examples <a href=\"https://github.com/reactome/graph-core/tree/master/src/main/java/org/reactome/server/graph/repository\">here</a>.</p>','',1,43,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 23:37:19','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(224,922,'Is it possible to directly connect my protein protein interaction network with the Reactome database or do I have to extract the required data and separately build the graph?','ppi-to-pathways-graph','<style type=\"text/css\">.rcypher, .rtip, .rresult {\r\n  border: 1px solid #dddddd;\r\n  border-radius: 5px;\r\n  background: #f9f9f9;\r\n  margin-left: 15px;\r\n  margin-right: 15px;\r\n  overflow-x: scroll;\r\n  padding: 10px;\r\n}\r\n.rtip {\r\n  border: 1px solid pink;\r\n  background-color: #ffe6e6;\r\n}\r\n.rresult{\r\n  font-family: monospace;\r\n  font-size: x-small;\r\n  color: darkgray;\r\n        word-wrap: initial;jq\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n.rcypher code, .rtip code {\r\n        word-wrap: initial;\r\n        white-space: pre;\r\n        word-break: break-all;\r\n}\r\n@media only screen and (max-width: 768px) {\r\n    .rcypher, .rtip, .rresult {\r\n  margin-left: 0px;\r\n        margin-right: 0px;\r\n    }\r\n    .rresult {\r\n        overflow-y: auto;\r\n        white-space: pre;\r\n    }\r\n</style>\r\n<p>Yes, you can directly query in our graph database, for example:</p>\r\n<p>{source}<span style=\"font-family: courier new, courier, monospace;\">&lt;pre class=\"rcypher\"&gt;&lt;code class=\"language-cypher\" data-lang=\"cypher\"&gt;<br />MATCH (ref1:ReferenceEntity)&lt;-[:referenceEntity]-(p1:EntityWithAccessionedSequence)&lt;-[:input|output|catalystActivity|physicalEntity|entityFunctionalStatus|diseaseEntity|regulatedBy|regulator|hasComponent|hasMember|hasCandidate|repeatedUnit*]-(reaction:ReactionLikeEvent)-[:input|output|catalystActivity|physicalEntity|entityFunctionalStatus|diseaseEntity|regulatedBy|regulator|hasComponent|hasMember|hasCandidate|repeatedUnit*]-&gt;(p2:EntityWithAccessionedSequence)-[:referenceEntity]-&gt;(ref2:ReferenceEntity)<br />WHERE ref1.databaseName = \'UniProt\' AND ref2.databaseName = \'UniProt\' AND ref1.identifier = \'P60484’ AND ref2.identifier = \'P42685\'<br />MATCH eventPath=(reaction)&lt;-[:hasEvent*]-(pathway)<br />UNWIND (nodes(eventPath)) as p<br />RETURN DISTINCT p.stId<br />&lt;/code&gt;&lt;/pre&gt;</span>{/source}</p>','',1,43,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-04-11 23:43:19','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(225,923,'V85 released','v85-released','<p><img src=\"images/news/R-HSA-9758941.png\" alt=\"R HSA 9758941\" width=\"2513\" height=\"1436\" /></p>\r\n<p><strong>New and Updated Topics and Pathways. </strong>In V85, Topics with new or revised pathways include Developmental Biology (<a href=\"PathwayBrowser/#/R-HSA-9733709\">Cardiogenesis</a>, <a href=\"PathwayBrowser/#/R-HSA-9761174\">Formation of intermediate mesoderm</a>, <a href=\"PathwayBrowser/#/R-HSA-9793380\">Formation of paraxial mesoderm</a>, <a href=\"PathwayBrowser/#/R-HSA-9824272\">Somitogenesis</a>, <a href=\"PathwayBrowser/#/R-HSA-9827857\">Specification of primordial germ cells</a> ), Cell-Cell communication (<a href=\"PathwayBrowser/#/R-HSA-9764260\">Regulation of Expression and Function of Type II Classical Cadherins</a>, <a href=\"PathwayBrowser/#/R-HSA-9759475\">Regulation of CDH11 Expression and Function</a>, and <a href=\"PathwayBrowser/#/R-HSA-9764302\">Regulation of CDH19 Expression and Function</a>), Cellular responses to stimuli (<a href=\"PathwayBrowser/#/R-HSA-8874177\">ATF6B (ATF6-beta) activates chaperones</a>), Immune System (<a href=\"PathwayBrowser/#/R-HSA-9732724\">IFNG signaling activates MAPKs</a> and <a href=\"PathwayBrowser/#/R-HSA-877300\">Interferon gamma signaling</a>), and Metabolism (<a href=\"PathwayBrowser/#/R-HSA-5652227\">Fructose biosynthesis</a> and <a href=\"PathwayBrowser/#/R-HSA-70350\">Fructose catabolism</a>).  </p>\r\n<p><strong>New and Updated Illustrations.  </strong>New or revised Illustrations with embedded navigation features have been created for  <a href=\"PathwayBrowser/#/R-HSA-1266738\">Developmental Biology</a>, <a href=\"PathwayBrowser/#/R-HSA-9758941\">Gastrulation</a>, and <a href=\"PathwayBrowser/#/R-HSA-1474165\">Reproduction</a>.</p>\r\n<p><strong>Thanks to our Contributors.</strong>   <a href=\"https://orcid.org/0000-0001-9607-8838\">Ravindra Chalamalasetty</a> and <a href=\"https://orcid.org/0000-0002-7452-4419\">Terry Yamaguchi</a> are our external authors.  <a href=\"https://orcid.org/0000-0002-1831-6244\">Julia Brasch</a>, <a href=\"https://orcid.org/0000-0001-9607-8838\">Ravindra Chalamalasetty</a>, <a href=\"https://orcid.org/0000-0001-7476-6306\">David Hill</a>, <a href=\"https://orcid.org/0000-0001-6950-8388\">Patrick Tam</a>,  and <a href=\"https://orcid.org/0000-0002-7452-4419\">Terry  Yamaguchi</a> are our external reviewers.</p>\r\n<p><strong>Annotation Statistics</strong><strong>.</strong> Reactome comprises  14,628 human reactions organized into 2,629 <strong> </strong> pathways involving 30,155  proteins and modified forms of proteins encoded by 11,396 different human genes,  14,277 complexes, 2,004 small molecules, and 1,114 drugs. These annotations are supported by 36,706 literature references. We have projected these reactions onto 84,167 orthologous proteins, creating 19,505 orthologous pathways in 14 non-human species. Version 85 has annotations for 4,919 protein variants (mutated proteins) and their post-translationally modified forms, derived from 354  proteins, which have contributed to the annotation of 1,659 disease-specific reactions and 707 pathways. </p>\r\n<p><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and <a href=\"https://reactome.org/AnalysisService/\">Analysis</a> Services. The <a href=\"https://reactome.org/userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"PathwayBrowser/#TOOL=AT\">ReactomeGSA</a> package provide tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p><strong>Documentation and Training.</strong> Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/community/training\">here</a>.</p>\r\n<p><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a> as well as a <a href=\"https://globalbiodata.org/scientific-activities/global-core-biodata-resources/\">Global Core Biodata Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/what-is-reactome\">website</a>.</p>\r\n<p><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information:</strong> If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <a href=\"mailto:help@reactome.org\">help@reactome.org</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-05-26 00:43:26','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,6,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(226,925,'Label-Free Mass Spectrometry Proteomics Reveals Different Pathways Modulated in THP‐1 Cells Infected with Therapeutic Failure and Drug Resistance Leishmania infantum Clinical Isolates ','label-free-mass-spectrometry-proteomics-reveals-different-pathways-modulated-in-thp-1-cells-infected-with-therapeutic-failure-and-drug-resistance-leishmania-infantum-clinical-isolates','<p><a href=\"https://pubs.acs.org/doi/pdf/10.1021/acsinfecdis.2c00457\">Tagliazucchi L et al. 2023 in the March 2023 issue of ACS Infectious Diseases </a>used the REACTOME overrepresentation and pathway topology analyses to identify <a href=\"PathwayBrowser/#/R-HSA-382551\">Transport of small molecules</a>, <a href=\"PathwayBrowser/#/2262752&amp;PATH=R-HSA-8953897\">Cellular response to stress</a> and other pathways associated with drug resistance and therapeutic failure (TF) during <em>Leishmania infantum </em>infection; they also discovered NDK3 and TFRC as potential targets for host-directed anti-Leishmania therapies to overcome drug-resistance.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-06-13 13:48:16','0000-00-00 00:00:00','{}','{}','{}',1,17,'','',1,0,'{}',0,'en-GB','',''),(227,926,'Severe COVID-19 in pregnancy has a distinct serum profile, including greater complement activation and dysregulation of serum lipids','severe-covid-19-in-pregnancy-has-a-distinct-serum-profile-including-greater-complement-activation-and-dysregulation-of-serum-lipids','<p>Pregnancies complicated by Coronavirus Disease 2019 (COVID-19) are at an increased risk of severe morbidity. In multi-omics analyses investigating the pathophysiology behind severe COVID-19 disease, <a href=\"https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0276766\">Altendahl et al, in the November 2022 issue of PLoS One</a> found precipitous changes in maternal serum in those with severe COVID-19 infection. Reactome pathway enrichment analysis revealed upregulated analytes in 4 pathways: <a href=\"PathwayBrowser/#/R-HSA-166658\">Complement cascade</a>, <a href=\"PathwayBrowser/#/R-HSA-983705 \">Signaling by the B Cell Receptor (BCR)</a>, <a href=\"PathwayBrowser/#/R-HSA-2454202\">Fc epsilon receptor (FCERI) signaling</a>, and <a href=\"PathwayBrowser/#/R-HSA-2029481\"> FCGR activation</a>.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-06-13 14:02:39','0000-00-00 00:00:00','{}','{}','{}',1,16,'','',1,0,'{}',0,'en-GB','',''),(228,927,'In Vitro Zika Virus Infection of Human Neural Progenitor Cells: Meta-Analysis of RNA-Seq Assays','in-vitro-zika-virus-infection-of-human-neural-progenitor-cells-meta-analysis-of-rna-seq-assays','<p>The Zika virus (ZIKV) is an emergent arthropod-borne virus (arbovirus) responsible for congenital Zika syndrome (CZS) and a range of other congenital malformations. With little known about the pathways involved in CZS, <a href=\"https://www.mdpi.com/2076-2607/8/2/270\">Gratton et al in the February 2020 issue of Microorganisms</a> conducted a meta-analysis of transcriptome studies to identify the genes and pathways altered during Zika infection. Reactome analysis identified interferon, pro-inflammatory, and chemokines signaling as well as apoptosis as key IFN signaling pathways in ZIKV-infected cells with three new candidate genes involved in hNPCs infection identified: APOL6, XAF1, and TNFRSF1.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-07-14 21:08:49','0000-00-00 00:00:00','{}','{}','{}',1,15,'','',1,0,'{}',0,'en-GB','',''),(229,928,'Genetic Networks of Alzheimer’s Disease, Aging, and Longevity in Humans','genetic-networks-of-alzheimer-s-disease-aging-and-longevity-in-humans','<p>Using Reactome analysis tools and FIVIz, <a href=\"https://www.mdpi.com/1422-0067/24/6/5178\">Balmorez et al. in the March 2023 issue of <em>Int. J. Mol. Sci.</em></a>, established a commonality between the genes and pathways associated with Alzheimer\'s disease (AD), Ageing (AR) and Longevity. The pathways shared between AD and AR are <a href=\"PathwayBrowser/#/R-HSA-69580\">p53-Dependent G1/S DNA damage checkpoint</a>, <a href=\"PathwayBrowser/#/R-HSA-9614085\">FOXO-mediated transcription</a>, and <a href=\"PathwayBrowser/#/R-HSA-2990846\">SUMOylation</a>; between AD and longevity are <a href=\"PathwayBrowser/#/R-HSA-1280215\">Cytokine Signaling in Immune system</a>, <a href=\"PathwayBrowser/#/R-HSA-174824\">Plasma lipoprotein assembly, remodeling, and clearance</a>, <a href=\"PathwayBrowser/#/R-HSA-6806667\">Metabolism of fat-soluble vitamins</a>, and <a href=\"PathwayBrowser/#/R-HSA-9024446\">NR1H2- and NR1H3-mediated signalling</a>; and between AR and Longevity are <a href=\"PathwayBrowser/#/R-HSA-168256\">Immune system</a> and <a href=\"PathwayBrowser/#/R-HSA-1280215\">Cytokine signaling</a>.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-08-11 17:49:31','0000-00-00 00:00:00','{}','{}','{}',1,14,'','',1,0,'{}',0,'en-GB','',''),(230,929,'V86 Released  ','version-86-released','<p><img src=\"images/R-HSA-9816359-2242x1261.png\" alt=\"R HSA 9816359 2242x1261\" width=\"2242\" height=\"1260\" /></p>\r\n<p style=\"text-align: center;\"><span style=\"font-size: 10pt;\">Maternal to zygotic transition</span></p>\r\n<p></p>\r\n<p><strong>New and Updated Topics and Pathways. </strong>In V86, Topics with new or revised pathways include Cellular responses to stimuli (<a href=\"PathwayBrowser/#/R-HSA-9755511\">KEAP1-NFE2L2 pathway</a>), Developmental biology (<a href=\"PathwayBrowser/#/R-HSA-9823739\">Formation of anterior neural plate</a>, <a href=\"PathwayBrowser/#9832991\">Formation of posterior neural plate</a>, and <a href=\"PathwayBrowser/#9816359\">Maternal to zygotic transition (MZT)</a>), Immune system (<a href=\"PathwayBrowser/#9833482\">PKR-mediated signaling</a>), Metabolism (<a href=\"PathwayBrowser/#199220\">Vitamin B5 (pantothenate) metabolism</a>),  Metabolism of RNA (<a href=\"PathwayBrowser/#9836573\">Mitochondrial RNA degradation</a>), and Metabolism of proteins (<a href=\"PathwayBrowser/#9629569\">Protein hydroxylation</a>).  </p>\r\n<p><strong>New and Updated Illustrations.  </strong>New or revised Illustrations with embedded navigation features have been created for <a href=\"PathwayBrowser/#1169410\">Antiviral mechanism by IFN-stimulated genes</a>, <a href=\"PathwayBrowser/#1266738\">Developmental Biology</a>, <a href=\"PathwayBrowser/#/R-HSA-9758941\">Gastrulation</a>, <a href=\"PathwayBrowser/#9816359\">Maternal to zygotic transition</a>, and <a href=\"PathwayBrowser/#8953854\">Metabolism of RNA</a>. </p>\r\n<p><strong>Thanks to our Contributors.</strong>   <a href=\"https://orcid.org/0000-0001-7476-6306\">David Hill</a>, <a href=\"https://www.researchgate.net/profile/Hisato-Kondoh/research\">Hisato Kondoh</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Patel+RC&amp;cauthor_id=35149730\">Rekha Patel</a>, and <a href=\"https://orcid.org/0000-0003-2126-3849\">Wei Xie</a> are our external reviewers.</p>\r\n<p><strong>Annotation Statistics</strong><strong>.</strong> Reactome comprises  <strong>14,803</strong>  human reactions organized into <strong>2,647</strong> <strong> </strong> pathways involving <strong>30,338</strong>  proteins and modified forms of proteins encoded by <strong>11,154 </strong>different human genes, <strong>14,441</strong> complexes, <strong>2,025</strong> small molecules, and <strong>1,119</strong> drugs. These annotations are supported by <strong>37,156 </strong> literature references. We have projected these reactions onto <strong>84,414</strong> orthologous proteins, creating <strong>19,561</strong> orthologous pathways in <strong>14</strong> non-human species. Version 86 has annotations for <strong>4,919</strong>  protein variants (mutated proteins) and their post-translationally modified forms, derived from <strong>354</strong>  proteins, which have contributed to the annotation of <strong> </strong><strong>1,667</strong> disease-specific reactions and  <strong>707</strong> pathways. </p>\r\n<p><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and <a href=\"https://reactome.org/AnalysisService/\">Analysis</a> Services. The <a href=\"https://reactome.org/userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">ReactomeGSA</a> package provide tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p><strong>Documentation and Training.</strong> Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/community/training\">here</a>.</p>\r\n<p><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a> as well as a <a href=\"https://globalbiodata.org/scientific-activities/global-core-biodata-resources/\">Global Core Biodata Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/what-is-reactome\">website</a>.</p>\r\n<p><strong>For more information:</strong> If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <a href=\"mailto:help@reactome.org\">help@reactome.org</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-09-07 04:15:08','0000-00-00 00:00:00','{}','{}','{}',1,5,'','',1,0,'{}',0,'en-GB','',''),(231,931,'V86','v86','<p>Version 86 Released  </p>\r\n<p><strong>New and Updated Topics and Pathways. </strong>In V86, Topics with new or revised pathways include Cellular responses to stimuli (<a href=\"PathwayBrowser/#9755511\">KEAP1-NFE2L2 pathway</a>), Developmental biology (<a href=\"PathwayBrowser/#9823739\">Formation of anterior neural plate</a> , <a href=\"PathwayBrowser/#9832991\">Formation of posterior neural plate</a>, and <a href=\"PathwayBrowser/#9816359\">Maternal to zygotic transition (MZT)</a>), Immune system (<a href=\"PathwayBrowser/#9833482\">PKR-mediated signaling</a>), Metabolism (<a href=\"PathwayBrowser/#199220\">Vitamin B5 (pantothenate) metabolism</a>),  Metabolism of RNA (<a href=\"PathwayBrowser/#9836573\">Mitochondrial RNA degradation</a>), and Metabolism of proteins (<a href=\"PathwayBrowser/#9629569\">Protein hydroxylation</a>).  </p>\r\n<p><strong>New and Updated Illustrations.  </strong>New or revised Illustrations with embedded navigation features have been created for <a href=\"PathwayBrowser/#1169410\">Antiviral mechanism by IFN-stimulated genes</a>, <a href=\"PathwayBrowser/#1266738\">Developmental Biology</a> , <a href=\"PathwayBrowser/#/R-HSA-9758941\">Gastrulation</a>,  <a href=\"PathwayBrowser/#9816359\">Maternal to zygotic transition</a>,  and  <a href=\"PathwayBrowser/#8953854\">Metabolism of RNA</a>. </p>\r\n<p><strong>Thanks to our Contributors.</strong>   <a href=\"https://orcid.org/0000-0001-7476-6306\">David Hill</a>, <a href=\"https://www.researchgate.net/profile/Hisato-Kondoh/research\">Hisato Kondoh</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Patel+RC&amp;cauthor_id=35149730\">Rekha Patel</a>, and <a href=\"https://orcid.org/0000-0003-2126-3849\">Wei Xie</a> are our external reviewers.</p>\r\n<p><strong>Annotation Statistics</strong><strong>.</strong> Reactome comprises  <strong>14,803</strong>  human reactions organized into <strong>2,647</strong> <strong> </strong> pathways involving <strong>30,338</strong>  proteins and modified forms of proteins encoded by <strong>11,154 </strong>different human genes, <strong>14,441</strong> complexes, <strong>2,025</strong> small molecules, and <strong>1,119</strong> drugs. These annotations are supported by <strong>37,156 </strong> literature references. We have projected these reactions onto <strong>84,414</strong> orthologous proteins, creating <strong>19,561</strong> orthologous pathways in <strong>14</strong> non-human species. Version 86 has annotations for <strong>4,919</strong>  protein variants (mutated proteins) and their post-translationally modified forms, derived from <strong>354</strong>  proteins, which have contributed to the annotation of <strong> </strong><strong>1,667</strong> disease-specific reactions and  <strong>707</strong> pathways. </p>\r\n<p><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and <a href=\"https://reactome.org/AnalysisService/\">Analysis</a> Services. The <a href=\"https://reactome.org/userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">ReactomeGSA</a> package provide tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p><strong>Documentation and Training.</strong> Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/community/training\">here</a>.</p>\r\n<p><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a> as well as a <a href=\"https://globalbiodata.org/scientific-activities/global-core-biodata-resources/\">Global Core Biodata Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/what-is-reactome\">website</a>.</p>\r\n<p><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information:</strong> If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <a href=\"mailto:help@reactome.org\">help@reactome.org</a>.</p>','',-2,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-09-07 04:27:49','0000-00-00 00:00:00','{}','{}','{}',1,0,'','',1,0,'{}',0,'en-GB','',''),(232,933,'Computational drug repositioning of clopidogrel as a novel therapeutic option for focal segmental glomerulosclerosis','computational-drug-repositioning-of-clopidogrel-as-a-novel-therapeutic-option-for-focal-segmental-glomerulosclerosis','<p><br />​​With current treatments, focal segmental glomerulosclerosis (FSGS), the largest cause of nephrotic syndrome, frequently progresses to end-stage kidney disease.<a href=\"https://www.sciencedirect.com/science/article/pii/S1931524423000579?via%3Dihub\"> Gebeshuber et al. (2023)</a> assembled 376 FSGS-associated proteins into a FSGS pathophysiology model, major components of which were Reactome pathways for<a href=\"https://reactome.org/content/detail/R-HSA-162582\"> signal transduction</a> and<a href=\"https://reactome.org/content/detail/R-HSA-109582\"> hemostasis</a>. The 39 proteins shared between FSGS model and a 102-protein model for the antiplatelet drug clopidogrel included 20 therapeutic targets of the drug. Tested in an FSGS mouse model, clopidogrel significantly attenuated disease severity, repositioning the drug as an attractive candidate for human clinical trials for FSGS.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-09-07 19:56:12','0000-00-00 00:00:00','{}','{}','{}',1,13,'','',1,0,'{}',0,'en-GB','',''),(233,934,'DNA methylation and 28-year cardiovascular disease risk in type 1 diabetes: the Epidemiology of Diabetes Complications (EDC) cohort study','dna-methylation-and-28-year-cardiovascular-disease-risk-in-type-1-diabetes-the-epidemiology-of-diabetes-complications-edc-cohort-study','<p>In the <a href=\"https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-023-01539-0\"> August 2, 2023 issue of Clinical Epigenetics</a>, Miller et al. performed an epigenome-wide association study using Reactome  Functional Interaction network analysis and determined that DNA methylation at loci involved in calcium channel activity and development was associated with long-term cardiovascular disease risk beyond known risk factors in type 1 diabetes, particularly in individuals with greater glycemic exposure.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-10-23 03:06:05','0000-00-00 00:00:00','{}','{}','{}',1,12,'','',1,0,'{}',0,'en-GB','',''),(234,935,'XMR: an explainable multimodal neural network for drug response prediction','xmr-an-explainable-multimodal-neural-network-for-drug-response-prediction','<p>November 1, 2023: In their paper titled “<a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10433751/pdf/fbinf-03-1164482.pdf\">XMR: an explainable multimodal neural network for drug response prediction</a>” published in Frontiers in Bioinformatics in August 2023, Wang et al. use five Reactome pathways: <a href=\"PathwayBrowser/#/R-HSA-1640170\">Cell Cycle</a>, “DNA repair”, “Disease”, “Signal transduction”, and “Metabolism”, as an architecture of a visible neural network that is part of a deep learning model for prediction of drug responses in triple negative breast cancer.</p>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-11-02 16:55:31','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(235,936,'XMR: an explainable multimodal neural network for drug response prediction','xmr-an-explainable-multimodal-neural-network-for-drug-response-prediction','<p>In their paper titled “<a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10433751/pdf/fbinf-03-1164482.pdf\">XMR: an explainable multimodal neural network for drug response prediction</a>” published in Frontiers in Bioinformatics in August 2023, Wang et al. use five Reactome pathways, <a href=\"PathwayBrowser/#/R-HSA-1640170\">Cell Cycle</a>, <a href=\"PathwayBrowser/#/R-HSA-73894\">DNA repair</a>, <a href=\"PathwayBrowser/#/R-HSA-1643685\">Disease</a>, <a href=\"PathwayBrowser/#/R-HSA-162582\">Signal transduction</a>, and <a href=\"PathwayBrowser/#/R-HSA-1430728\">Metabolism</a>, as an architecture of a visible neural network that is part of a deep learning model for prediction of drug responses in triple negative breast cancer.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-11-02 16:57:25','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,11,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(236,938,'New Insights into Clinical Management for Sickle Cell Disease: Uncovering the Significant Pathways Affected by the Involvement of Sickle Cell Disease ','new-insights-into-clinical-management-for-sickle-cell-disease-uncovering-the-significant-pathways-affected-by-the-involvement-of-sickle-cell-disease','<p>[December 1, 2023]  In the chapter entitled “<a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4969007/\">New Insights into Clinical Management for Sickle Cell Disease: Uncovering the Significant Pathways Affected by the Involvement of Sickle Cell Disease</a>”, published in Methods in Molecular Biology 2024, Chouhan et al. describe the use of Reactome FIviz Cytoscape plugin to analyze pathway enrichment and construct a functional interaction network for DisGNET-derived sickle cell disease-associated genes, identifying genes involved in “Glucuronidation”, “Aspirin ADME”, “Phase II-Conjugation of compounds”, “Interleukin-4 and interleukin-13 signaling”, “Interleukin-10 signaling”, “Signaling by interleukins”, “Biological oxidations”, and “Cytokine signaling in immune system”. </p>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-11-29 20:16:48','0000-00-00 00:00:00','{}','{}','{}',1,3,'','',1,0,'{}',0,'en-GB','',''),(237,939,'New Insights into Clinical Management for Sickle Cell Disease: Uncovering the Significant Pathways Affected by the Involvement of Sickle Cell Disease','new-insights-into-clinical-management-for-sickle-cell-disease-uncovering-the-significant-pathways-affected-by-the-involvement-of-sickle-cell-disease','<p>In the chapter entitled “<a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4969007/\">New Insights into Clinical Management for Sickle Cell Disease: Uncovering the Significant Pathways Affected by the Involvement of Sickle Cell Disease</a>”, published in Methods in Molecular Biology 2024, Chouhan et al. describe the use of Reactome FIviz Cytoscape plugin to analyze pathway enrichment and construct a functional interaction network for DisGNET-derived sickle cell disease-associated genes, identifying genes involved in “<a href=\"https://reactome.org/PathwayBrowser#/R-HSA-156588\">Glucuronidation</a>”, “<a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-9749641\">Aspirin ADME</a>”, “<a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-156580\">Phase II-Conjugation of compounds</a>”, “<a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-6785807\">Interleukin-4 and interleukin-13 signaling</a>”, “<a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-6783783\">Interleukin-10 signaling</a>”, “<a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-449147\">Signaling by interleukins</a>”, “<a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-211859\">Biological oxidations</a>”, and “<a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-1280215\">Cytokine signaling in immune system”</a>.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-11-29 20:19:05','0000-00-00 00:00:00','{}','{}','{}',1,10,'','',1,0,'{}',0,'en-GB','',''),(238,940,'V87 released','version-87-released','<p style=\"text-align: center;\"><img src=\"images/R-HSA-9820952.png\" alt=\"R HSA 9820952\" width=\"1396\" height=\"798\" /><strong>Respiratory Syncytial Virus Infection Pathway</strong></p>\r\n<p><strong>New and Updated Topics and Pathways. </strong>As we head into winter and medical experts warn, once again, of a potential tripledemic of flu, COVID-19, and Respiratory Syncytial Virus (RSV), Reactome completes its coverage of these three viruses with the release of the RSV infection pathway in V87. Additional topics with new or revised pathways in this release include Cellular responses to stimuli (<a href=\"PathwayBrowser/#/9840373\">Cellular response to mitochondrial stress</a>), Developmental Biology (<a href=\"PathwayBrowser/#9830369\">Kidney formation</a> and <a href=\"PathwayBrowser/#9834899\">Specification of the neural plate border</a>), Disease (<a href=\"PathwayBrowser/#9823587\">Defects of platelet adhesion to exposed collagen</a>, and <a href=\"PathwayBrowser/#9700206\">Signaling by ALK in cancer</a>), DNA Repair (<a href=\"PathwayBrowser/#5693607\">Processing of DNA double-strand break ends</a>), Hemostasis (<a href=\"PathwayBrowser/#75892\">Platelet Adhesion to exposed collagen</a>), Metabolism (<a href=\"PathwayBrowser/#428157\">Sphingolipid metabolism</a>), Metabolism of proteins (<a href=\"PathwayBrowser/#9837999\">Mitochondrial protein degradation</a>), Signal Transduction (<a href=\"PathwayBrowser/#201556\">Signaling by ALK</a> and <a href=\"PathwayBrowser/#177929\">Signaling by EGFR</a>), Transport of small molecules (<a href=\"PathwayBrowser/#9845576\">Glycosphingolipid transport</a>).</p>\r\n<p><strong>New and Updated Illustrations.  </strong>New or revised Illustrations with embedded navigation features have been created for <a href=\"PathwayBrowser/#2262752\">Cellular responses to stress</a>, <a href=\"PathwayBrowser/#1266738\">Developmental Biology</a>, <a href=\"PathwayBrowser/#9671793\">Diseases of hemostasis</a>, <a href=\"PathwayBrowser/#9758941\">Gastrulation</a>, <a href=\"PathwayBrowser/#9830369\">Kidney development</a>, <a href=\"PathwayBrowser/#392499\">Metabolism of proteins</a>, <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-382551\">Transport of small molecules</a>, <a href=\"PathwayBrowser/#9824446\">Viral Infection Pathways</a> and <a href=\"PathwayBrowser/#9820952\">Respiratory Syncytial Virus Infection Pathway</a>.</p>\r\n<p><strong>Thanks to our Contributors.</strong>   <a href=\"https://orcid.org/0009-0003-2990-1873\">Mazen Aljghami</a>, <a href=\"https://orcid.org/0000-0002-1329-0209\">Harrison Bergeron</a>, Rui Gao, <a href=\"https://orcid.org/0000-0003-3059-6311\">Lena Gunhaga</a>, <a href=\"https://orcid.org/0000-0002-9388-484X\">Xiaoyan Guo</a>, <a href=\"https://orcid.org/0000-0001-7476-6306\">David P Hill</a>, <a href=\"https://orcid.org/0000-0002-1861-3441\">Walid A Houry</a>, Anthony Mak, <a href=\"https://orcid.org/0000-0002-3405-2963\">Trevor Morey</a>, <a href=\"https://orcid.org/0000-0002-2627-9578\">Cedric Patthey</a>, and <a href=\"https://orcid.org/0000-0002-8439-4507\">Suzanne D Turner</a> are our external reviewers.</p>\r\n<p><strong>Annotation Statistics</strong><strong>.</strong> Reactome comprises  15,046  human reactions organized into 2,673 <strong>  </strong>pathways involving 30,451  proteins and modified forms of proteins encoded by 11,180 different human genes, 14,594 complexes, 2,120 small molecules, and 1,046 drugs. These annotations are supported by 37,933 literature references. We have projected these reactions onto 79,040  orthologous proteins, creating 19,398 orthologous pathways in 14 non-human species. Version 87 has annotations for 4,942 protein variants (mutated proteins) and their post-translationally modified forms, derived from  357 proteins, which have contributed to the annotation of <strong><em> </em></strong>1,796 disease-specific reactions and  723 pathways. </p>\r\n<p><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and <a href=\"https://reactome.org/AnalysisService/\">Analysis</a> Services. The <a href=\"https://reactome.org/userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"https://reactome.org/gsa\">ReactomeGSA</a> package provide tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p><strong>Documentation and Training.</strong> Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/community/training\">here</a>.</p>\r\n<p><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a> as well as a <a href=\"https://globalbiodata.org/scientific-activities/global-core-biodata-resources/\">Global Core Biodata Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/what-is-reactome\">website</a>.</p>\r\n<p><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information:</strong> If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <a href=\"mailto:help@reactome.org\">help@reactome.org</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2023-12-05 03:25:30','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,4,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(239,942,'Contact us','contact-us','<h4></h4>\r\n<h4>Helpdesk</h4>\r\n<p></p>\r\n<p><a href=\"mailto:help@reactome.org\"><img src=\"images/HelpDeskLogo.png\" alt=\"HelpDeskLogo.png\" width=\"334\" height=\"96\" /></a></p>\r\n<p>Contact us by emailing our helpdesk: <a href=\"mailto:help@reactome.org\">help@reactome.org</a></p>\r\n<p>Please visit our <a href=\"https://reactome.org/documentation/faq\">FAQ page</a> for additional information.</p>\r\n<p></p>\r\n<h4>Social Media</h4>\r\n<p></p>\r\n<p>Find us on the following social media platforms: <a href=\"https://www.linkedin.com/groups/2118372/\" target=\"_blank\" rel=\"noopener\">LinkedIn</a>, <a href=\"https://twitter.com/reactome\" target=\"_blank\" rel=\"noopener\">X</a>, <a href=\"https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmastodon.social%2F%40reactome&amp;data=05%7C01%7CNancy.Li%40oicr.on.ca%7Cf075ba465eef4eb4109908dbe175d28a%7C9df949f8a6eb419d9caa1f8c83db674f%7C0%7C0%7C638351668403588364%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=Nt91af9eQN9i4rBPg0JpPEtKYAGVbReJBraHgIYLu4U%3D&amp;reserved=0\" target=\"_blank\" rel=\"noopener\">Mastodon</a>, <a href=\"https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fbsky.app%2Fprofile%2Freactome.bsky.social&amp;data=05%7C01%7CNancy.Li%40oicr.on.ca%7Cf075ba465eef4eb4109908dbe175d28a%7C9df949f8a6eb419d9caa1f8c83db674f%7C0%7C0%7C638351668403588364%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=da32vGCpuM7gvwdGJ2wyma3xLyOP0CH2fVhQNlHWXSY%3D&amp;reserved=0\" target=\"_blank\" rel=\"noopener\">Bluesky</a>, <a href=\"https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.instagram.com%2Freactome%2F&amp;data=05%7C01%7CNancy.Li%40oicr.on.ca%7Cf075ba465eef4eb4109908dbe175d28a%7C9df949f8a6eb419d9caa1f8c83db674f%7C0%7C0%7C638351668403588364%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=m2xio7hVmybN22zC5rijv3L95siswjk4aR6grQtJanU%3D&amp;reserved=0\" target=\"_blank\" rel=\"noopener\">Instagram</a>, and <a href=\"https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.threads.net%2F%40reactome&amp;data=05%7C01%7CNancy.Li%40oicr.on.ca%7Cf075ba465eef4eb4109908dbe175d28a%7C9df949f8a6eb419d9caa1f8c83db674f%7C0%7C0%7C638351668403588364%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=jQvxPbQXexKbwYByv493NleEU7nok%2FMXkTrQiU5VCh8%3D&amp;reserved=0\" target=\"_blank\" rel=\"noopener\">Threads</a>, and subscribe to our <a href=\"https://www.youtube.com/@Reactome\" target=\"_blank\" rel=\"noopener\">Youtube</a> Channel!</p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-01-10 15:11:26','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(240,943,'Machine learning-based analysis of cancer cell-derived vesicular proteins revealed significant tumor-specificity and predictive potential of extracellular vesicles for cell invasion and proliferation – A meta-analysis','machine-learning-based-analysis-of-cancer-cell-derived-vesicular-proteins-revealed-significant-tumor-specificity-and-predictive-potential-of-extracellular-vesicles-for-cell-invasion-and-proliferation-a-meta-analysis','<p>In the November 2023 issue of Cell Communication and Signaling, <a href=\"https://biosignaling.biomedcentral.com/articles/10.1186/s12964-023-01344-5#Sec16\">Bukva et al. (2023)</a> analyzed the proteomes of tumor-produced extracellular vesicles and identified sets of proteins that could discriminate tumor types, invasiveness, and proliferative capacity. In this analysis, 172 most predictive proteins were identified and used to classify nine tumor types with 91.67% efficiency. Reactome Pathway enrichment analysis of these proteins showed that each tumor type had perturbations in a distinct set of pathways. The proteins could be organized and used to discriminate the invasiveness and proliferative capacity of the tumors. High expression of proteins positively associated with invasiveness and proliferation correlated with reduced patient survival times.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-01-12 17:00:24','0000-00-00 00:00:00','{}','{}','{}',1,9,'','',1,0,'{}',0,'en-GB','',''),(241,944,'Digital Preservation','digital-preservation','<h3>The Reactome consortium and host institutions OICR, EMBL-EBI, NYU and OHSU are committed to the long-term development and maintenance of the Reactome database.</h3>\r\n<p></p>\r\n<h4>Reactome data, code and tools are robustly safeguarded by many levels of backup:</h4>\r\n<p>The Reactome dataset is replicated across two independent servers hosted by AWS. We store our legacy data for each release in an AWS Simple Storage System (S3) bucket, allowing regeneration and remounting of the database or rollback to earlier versions if necessary. Work is underway to make this data available to the public via a Request Review system. The Extract Transform Load pipeline used to generate releases can regenerate parts of the production system as necessary. Reactome is Dockerizing all components of the architecture and offloading computational tasks to AWS services. Zabbix monitors all major services hosted on our servers. Multiple personnel across the host institutions have the expertise required to restore services and regenerate the Reactome website.</p>\r\n<p></p>\r\n<h4>Access to Reactome content:</h4>\r\n<p>In addition to the backup of Reactome content described above, Reactome data and resources are also available for direct download from our website “<a href=\"https://reactome.org/download-data\">Download</a>” page, from our GitHub repository (<a href=\"https://github.com/reactome/\">Reactome</a> and <a href=\"https://github.com/reactome-pwp/\">Reactome-PWP</a>), and indirectly from aggregators of Reactome data including NCBI, EMBL-EBI, MSigDB and WikiPathways.</p>\r\n<h4>Long-term preservation plan:</h4>\r\n<p>Reactome has a 20+ year history of stable funding from varied sources, providing stability and continuity. In recognition of its value to the bioinformatics community and its long-term stability, Reactome has been recognized as an ELIXIR Core Data Resource and as a Global Core Biodata Resource. </p>\r\n<p>In the unlikely event that Reactome were to lose funding or otherwise cease operating, existing content and tools would be preserved in the latest format on the public website if possible and would persist independently in other resources that make use of Reactome content such as WikiPathways and GO-CAM models. Reactome source code is preserved and available on GitHub. </p>\r\n<p>Reactome is developing a long-term data preservation plan that will see Reactome content and infrastructure housed with an archiving service such as Zenodo. In case of funding loss, archived content will include the full data schema and internal process documents such as the Reactome Curation Guide and the User’s Guide, ensuring future interpretability and reusability of stored digital content.</p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-01-31 15:15:56','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(242,945,'Discovering the anti-cancer phytochemical rutin against breast cancer through the methodical platform based on traditional medicinal knowledge','discovering-the-anti-cancer-phytochemical-rutin-against-breast-cancer-through-the-methodical-platform-based-on-traditional-medicinal-knowledge','<p>In the July 2023 issue of BMB Reports, <a href=\"https://www.bmbreports.org/journal/view.html?volume=56&amp;number=11&amp;spage=594\">Lee et al. (2023)</a> employed the Reactome pathway database and tools to predict the anti-cancer effects of rutin, a natural phytochemical identified as a lead chemotherapeutic against breast cancer by text mining Korean traditional medicinal compendia from 1596 CE and 1613 CE. Genes that may be associated with rutin\'s effects were analyzed for pathway enrichment and functional interactions by the Reactome Functional Interaction (FI) plugin app of Cytoscape. Focal adhesion and <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-109581&amp;PATH=R-HSA-5357801\">Apoptosis</a> were among the pathways predicted to be affected by rutin and these effects were confirmed by treatment of breast cancer cells with rutin in vitro and in xenografts in mice.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-02-09 18:21:53','0000-00-00 00:00:00','{}','{}','{}',1,8,'','',1,0,'{}',0,'en-GB','',''),(243,946,'Identification of potential biological processes and key genes in diabetes-related stroke through weighted gene co-expression network analysis','identification-of-potential-biological-processes-and-key-genes-in-diabetes-related-stroke-through-weighted-gene-co-expression-network-analysis','<p>[March 15, 2024]</p>\r\n<p>Using WGCNA, GO and KEGG data analysis tools,<a href=\"https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-023-01752-z\"> He Y et al. in the January 2024 issue of BMC Medical Genomics</a>, established a connection among the genes and pathways associated with type 2 diabetes (T2D) and ischemic stroke (IS) and identified GRN (granulin precursor) as the hub gene in T2D-related stroke. The functional enrichment analysis using Reactome analysis tool for GRN identified <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-6798695&amp;PATH=R-HSA-168256,R-HSA-168249\">Neutrophil degranulation</a>, <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-168898&amp;PATH=R-HSA-168256,R-HSA-168249\">Toll-like Receptor Cascades</a>, <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-168928&amp;PATH=R-HSA-168256,R-HSA-168249\">DDX58/IFIH1-mediated induction of interferon-alpha/ beta</a>, and<a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-168643&amp;PATH=R-HSA-168256,R-HSA-168249\"> NLR signaling pathways</a> as shared biological processes in T2D and IS.</p>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-03-13 07:21:59','0000-00-00 00:00:00','{}','{}','{}',1,2,'','',1,0,'{}',0,'en-GB','',''),(244,947,'Identification of potential biological processes and key genes in diabetes-related stroke through weighted gene co-expression network analysis','identification-of-potential-biological-processes-and-key-genes-in-diabetes-related-stroke-through-weighted-gene-co-expression-network-analysis','<p>Using WGCNA, GO and KEGG data analysis tools,<a href=\"https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-023-01752-z\"> He Y et al. in the January 2024 issue of BMC Medical Genomics</a>, established a connection among the genes and pathways associated with type 2 diabetes (T2D) and ischemic stroke (IS) and identified GRN (granulin precursor) as the hub gene in T2D-related stroke. The functional enrichment analysis using Reactome analysis tool for GRN identified <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-6798695&amp;PATH=R-HSA-168256,R-HSA-168249\">Neutrophil degranulation</a>, <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-168898&amp;PATH=R-HSA-168256,R-HSA-168249\">Toll-like Receptor Cascades</a>, <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-168928&amp;PATH=R-HSA-168256,R-HSA-168249\">DDX58/IFIH1-mediated induction of interferon-alpha/ beta</a>, and<a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-168643&amp;PATH=R-HSA-168256,R-HSA-168249\"> NLR signaling pathways</a> as shared biological processes in T2D and IS.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-03-13 07:23:12','0000-00-00 00:00:00','{}','{}','{}',1,7,'','',1,0,'{}',0,'en-GB','',''),(245,949,'V88 Released','v88-released','','',-2,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-03-20 04:29:18','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(246,950,'V88 Released','v88-released-2','<p><img src=\"images/R-HSA-9734767.png\" alt=\"R HSA 9734767\" width=\"701\" height=\"401\" /></p>\r\n<p><strong>Cytomics Reactome. </strong>The trillions of cells in the adult human body, specialized to fulfill diverse roles in tissues, organs, and organ systems, all originate from a single cell, a zygote formed at conception. From zygote to adulthood, cells divide and commit to different fates. The steps from a pluripotent stem cell to a specialized descendant constitute a cell lineage path. <a href=\"PathwayBrowser/#9734767\">Cytomics Reactome</a> extends the data structure used for molecular-level annotation of cell biology to annotate cell lineage paths, organizing them by organ systems, cross-referencing them to Gene Ontology biological processes, and dividing each into causally connected cell development steps. These reaction-like steps describe transitions between cell states during development or differentiation, characterized by positive and negative regulators and required input components (cell state biomarkers required for the action of regulators). Each cell state corresponds to a unique combination of cell type (Cell Ontology), anatomical location (UBERON), and protein and/or RNA markers, as described in the Cytomics section of the  <a href=\"https://reactome.org/userguide\">User Guide</a>. Recently developed tools that harvest omics data from single cells of multicellular organisms and track fates open the door to deciphering cell lineage paths at single-cell resolution, a critical requirement of regenerative medicine and cancer medicine. Cytomics Reactome is intended to support such technologies and their application to human biology as they emerge. <br /><br />The first Cytomics cell lineage path, <a href=\"PathwayBrowser/#9725554\">Differentiation of keratinocytes in interfollicular epidermis in mammalian skin</a>, describes the differentiation of keratinocytes from stem cells to corneocytes in the interfollicular epidermis, the skin surface layer in between the adnexa (hair follicles, sweat glands, and sebaceous glands). The path is described in four cell differentiation steps involving five distinct cellular states: keratinocyte stem cells of the epidermal basal layer, transit amplifying cells, spinous keratinocytes, granular keratinocytes, and corneocytes. Each differentiation step is regulated by a distinct combination of regulatory molecules present in the microenvironments of the differentiating cells.</p>\r\n<p><strong>New and Updated Topics and Pathways. </strong>Topics with new or revised pathways in this release include Cell Cycle (<a href=\"PathwayBrowser/#/R-HSA-2500257\">Resolution of Sister Chromatid Cohesion</a>), Developmental Biology (<a href=\"PathwayBrowser/#/R-HSA-9725554\">Differentiation of keratinocytes in intefrollicular skin epidermis</a>, <a href=\"PathwayBrowser/#/R-HSA-9730414\">MITF-M-regulation melanocyte development</a>, <a href=\"PathwayBrowser/#/R-HSA-9843743\">Transcriptional regulation of brown and beige adipocyte differentiation</a>), Disease (<a href=\"PathwayBrowser/#/R-HSA-9824856\">Defective regulation of TLR by endogenous ligand</a>, <a href=\"PathwayBrowser/#/R-HSA-9842640\">Signaling by LTK in cancer</a>), Immune System (<a href=\"PathwayBrowser/#/R-HSA-936964\">Activation of IRF3, IRF7 mediated by TBK1, IKKε (IKBKE)</a>, <a href=\"PathwayBrowser/#/R-HSA-5686938\">Regulation of TLR by endogenous ligand</a>, <a href=\"PathwayBrowser/#/R-HSA-9013973\">TICAM1-dependent activation of IRF3/IRF7</a> ), Metabolism (<a href=\"PathwayBrowser/#/R-HSA-399997\">Acetylcholine regulates insulin secretion</a>, <a href=\"PathwayBrowser/#/R-HSA-400042\">Adrenaline,noradrenaline inhibits insulin secretion</a>, <a href=\"PathwayBrowser/#/R-HSA-1428517\">Aerobic respiration and respiratory electron transport</a>, <a href=\"PathwayBrowser/#/R-HSA-163680\">AMPK inhibits chREBP transcriptional activation activity</a>, <a href=\"PathwayBrowser/#/R-HSA-70895\">Branched-chain amino acid catabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-163765\">ChREBP activates metabolic gene expression</a>, <a href=\"PathwayBrowser/#/R-HSA-71403\">Citric acid cycle (TCA cycle)</a>, <a href=\"PathwayBrowser/#/R-HSA-163359\">Glucagon signaling in metabolic regulation</a>, <a href=\"PathwayBrowser/#/R-HSA-381676\">Glucagon-like Peptide-1 (GLP1) regulates insulin secretion</a>, <a href=\"PathwayBrowser/#/R-HSA-189451\">Heme biosynthesis</a>, <a href=\"PathwayBrowser/#/R-HSA-9856872\">Malate-aspartate shuttle</a>, <a href=\"PathwayBrowser/#/R-HSA-9854311\">Maturation of TCA enzymes and regulation of TCA cycle</a>, <a href=\"PathwayBrowser/#/R-HSA-389542\">NADPH regeneration</a>, <a href=\"PathwayBrowser/#/R-HSA-1483196\">PI and PC transport between ER and Golgi membranes</a>, <a href=\"PathwayBrowser/#/R-HSA-163358\">PKA-mediated phosphorylation of key metabolic factors</a>, <a href=\"PathwayBrowser/#/R-HSA-70268\">Pyruvate metabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-204174\">Regulation of pyruvate dehydrogenase (PDH) complex</a>, <a href=\"PathwayBrowser/#/R-HSA-611105\">Respiratory electron transport</a>), Metabolism of proteins (<a href=\"PathwayBrowser/#/R-HSA-9857492\">Protein lipoylation</a>), Signal Transduction (<a href=\"PathwayBrowser/#/R-HSA-2179392\">EGFR Transactivation by Gastrin</a>, <a href=\"PathwayBrowser/#/R-HSA-9842663\">Signaling by LTK</a>), Transport of small molecules (<a href=\"PathwayBrowser/#917937\">Iron uptake and transport</a>).</p>\r\n<p><strong>New and Updated Illustrations.  </strong>New or revised Illustrations with embedded navigation features have been created for <a href=\"PathwayBrowser/#/R-HSA-9843745\">Adipogenesis</a>, <a href=\"PathwayBrowser/#/R-HSA-1266738\">Developmental Biology</a>, <a href=\"PathwayBrowser/#/R-HSA-9734767\">Developmental Cell Lineages</a>, <a href=\"PathwayBrowser/#/R-HSA-5602358\">Diseases associated with TLR signaling cascade</a>, <a href=\"PathwayBrowser/#/R-HSA-5663202\">Diseases of signal transduction by growth factor receptors and second messengers</a>, <a href=\"PathwayBrowser/#/R-HSA-1430728\">Metabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-9730414\">MITF-M-regulated melanocyte development</a>, <a href=\"PathwayBrowser/#/R-HSA-597592\">Post-translational protein modification</a>, and <a href=\"PathwayBrowser/#/R-HSA-9006934\">Signalling by Receptor Tyrosine Kinases</a>. A new static illustration is available for <a href=\"PathwayBrowser/#/R-HSA-9820960\">Respiratory syncytial virus (RSV) attachment and entry</a>.</p>\r\n<p><strong>Thanks to our Contributors.</strong>   <a href=\"https://orcid.org/0000-0001-9333-2589\">Tae-Hwa Chun</a>, <a href=\"https://orcid.org/0000-0003-3532-8586\">Toni Franjkić</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Ganier+C&amp;cauthor_id=35667795\">Clarisse Ganier</a>, <a href=\"https://orcid.org/0000-0003-2537-1134\">Heinz Arnheiter</a>, <a href=\"https://orcid.org/0000-0001-7476-6306\">David Hill</a>, <a href=\"https://orcid.org/0000-0002-7480-5085\">Xing Liu</a>, <a href=\"https://orcid.org/0000-0002-5171-9950\">Ivana Munitic</a>, and <a href=\"https://orcid.org/0000-0001-8982-5911\">Xuebiao Yao</a> are our external reviewers.</p>\r\n<p><strong>Annotation Statistics</strong><strong>.</strong> Reactome comprises 15,212 human reactions organized into 2,698<strong>  </strong>pathways involving 30585 proteins and modified forms of proteins encoded by 11226 different human genes,<strong>  </strong>14789 complexes, 2128 small molecules, and 1047 drugs. These annotations are supported by 38549  literature references. We have projected these reactions onto 79407 orthologous proteins, creating 19520 orthologous pathways in 14 non-human species. Version 88 has annotations for 4944 protein variants (mutated proteins) and their post-translationally modified forms, derived from  359 proteins, which have contributed to the annotation of 1802 disease-specific reactions and 725 pathways. </p>\r\n<p><strong>Tools and Data.</strong> Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and <a href=\"https://reactome.org/AnalysisService/\">Analysis</a> Services. The <a href=\"https://reactome.org/userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"https://reactome.org/gsa\">ReactomeGSA</a> package provide tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p><strong>Documentation and Training.</strong> Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/community/training\">here</a>.</p>\r\n<p><strong>About the Reactome Project.</strong> Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a> as well as a <a href=\"https://globalbiodata.org/scientific-activities/global-core-biodata-resources/\">Global Core Biodata Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art, and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/what-is-reactome\">website</a>.</p>\r\n<p><strong>Follow us on Twitter:</strong> <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p><strong>For more information:</strong> If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <a href=\"mailto:help@reactome.org\">help@reactome.org</a>.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-03-20 04:35:16','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,3,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'en-GB','',''),(247,951,'Cytomics','cytomics','<h4>Introduction</h4>\r\n<p>Human bodies are built of &gt;30 trillion cells specialized to fulfill diverse roles within our tissues, organs, and organ systems. All these cells originate from a single cell, a zygote formed at conception. From zygote to fetus, and throughout childhood, adolescence, and adulthood, cells divide and commit to different fates in order for the organism to develop, sustain and regenerate. The series of steps that lead from an undifferentiated progenitor cell, such as a stem cell, to one of a number of possible specialized descendants constitutes a cell lineage path. Recent technological advances have allowed researchers to collect high-throughput omics data from single cells of multicellular organisms and use it to track and manipulate cell fates <a href=\"https://paperpile.com/c/N1L8cm/Zlar+2YBt\">(Burgess 2018; Saelens et al. 2019)</a>. This opens the door to the possibility of deciphering cell lineage paths at single-cell resolution, a critical requirement for the advancement of regenerative medicine and cancer medicine.</p>\r\n<p>Researchers who conduct experiments that involve cell lineage tracing through the study of genomic, epigenomic, transcriptomic, and proteomic features of single cells currently expend significant effort extracting information from published sources to create short-lived, limited-scope local annotations of biological markers that are needed to interpret and analyze their high-throughput experimental data. This creates the urgent need for an open source, comprehensive, continuously maintained reference repository of cell lineage paths framed on the pre-existing knowledge of tissue morphogenesis, as well as a cell type-specific pathway resource, which can serve as a framework for interpretation and integration of the latest experimental findings. </p>\r\n<p>Cytomics Reactome aims to become an integrative systems biology resource of cell lineage paths and a toolset for the analysis of single-cell omics data. The Reactome web application and pathway visualization tools <a href=\"https://paperpile.com/c/N1L8cm/exFx\">(Croft et al. 2011)</a> have been extended to enable Cytomics Reactome visual display and search functions (Milacic et al. 2024) on the live website.</p>\r\n<h4>Database schema features to support Cytomics</h4>\r\n<p>The Reactome data model provides a robust framework for the annotation of biological entities and events <a href=\"https://paperpile.com/c/N1L8cm/vDrf\">(Joshi-Tope et al. 2005)</a>. The basic unit of the Reactome data model is a Reaction, an event that converts input to output physical entities. The participants in reactions are PhysicalEntities, which can be simple or complex molecules or molecular complexes. These PhysicalEntities can also act as reaction catalysts and regulators. Reactions are grouped into causal chains to form Pathways <a href=\"https://paperpile.com/c/N1L8cm/vDrf\">(Joshi-Tope et al. 2005)</a>.</p>\r\n<p>The Reactome data schema was expanded as described below to accommodate new classes needed for Cytomics Reactome annotations.</p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li aria-level=\"1\">Cell: A subclass of PhysicalEntity, represents a type of cell in a particular state of development/differentiation (Figure 1A).\r\n<ul style=\"list-style-type: disc;\">\r\n<li aria-level=\"1\"> A Cell instance has CellType, TissueLayer, Tissue, and Organ as single-valued attributes which are populated using terms from public open source databases, the Cell Ontology <a href=\"https://paperpile.com/c/N1L8cm/oNqo+9HsK\">(Sarntivijai et al. 2014; Osumi-Sutherland 2017)</a> and UBERON <a href=\"https://paperpile.com/c/N1L8cm/l9BZ\">(Haendel et al. 2014)</a>, and cross-reference these databases.</li>\r\n<li>Cell instances also have multi-valued ProteinMarker and RNAMarker attributes, which are specific to the cell type and/or the cell state or are differentially upregulated in the Cell. (Figure 1B). The markers (protein or RNA) are manually curated; each marker instance is associated with one or more literature references for evidence, including, if applicable, citations of CellMarker (Hu et al. 2022) and PanglaoDB databases (Franzén et al. 2019) (Figure 1B) </li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<p><img src=\"images/userguide/cytomics/Figure_1.png\" alt=\"Figure 1. Cell Physical Entity class A and new attributes B\" width=\"1918\" height=\"1182\" style=\"border: 1px solid #000000;\" title=\" Figure 1. Cell Physical Entity class (A)  and new attributes (B).\" /></p>\r\n<p></p>\r\n<p style=\"text-align: center;\"><strong>Figure 1</strong>. Cell Physical Entity class A and new attributes B</p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li aria-level=\"1\">Cell Development Step and Cell Lineage Path are two Event subclasses (Figure 2A), that describe developmental/differentiation relationships among Cells: \r\n<ul style=\"list-style-type: disc;\">\r\n<li aria-level=\"2\">A Cell Development Step is a ReactionLikeEvent that contains Cell instances as its inputs and outputs. The input attribute represents the cell of origin, and the output attribute represents the destination cell type. Additional Cell Development Step attributes include regulators (molecules promoting or inhibiting the step) and required input components (input cell proteins required for the action of regulators) (Figure 2B). </li>\r\n<li aria-level=\"2\">A Cell Lineage Path, organizationally similar to the preexisting Pathway subclass, is composed of other Cell Lineage Path instances or Cell Development Steps as subevents that are connected through shared input/output Cells (Figure 2C).</li>\r\n<li>Both Cell Development Step and Cell Lineage Path are associated with a relevant Gene Ontology (GO) biological process term <a href=\"https://paperpile.com/c/N1L8cm/vwF9\">(The Gene Ontology Consortium 2019)</a> and with a “tissue” term from UBERON <a href=\"https://paperpile.com/c/N1L8cm/l9BZ\">(Haendel et al. 2014)</a>.</li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<p><img src=\"images/userguide/cytomics/Figure_2.png\" alt=\"Figure 2\" width=\"2004\" height=\"1256\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 2.</strong> Cell Development Step and Cell Lineage Path schema (A) and attributes (B,C).</p>\r\n<p>The Cytomics content is searchable on the Reactome website. Running a search from the homepage search bar may list hits within various categories including pathway, reaction, complex, and cell. Clicking on an item in the search results list will redirect to a Details page, which displays more information about the specific record.</p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li>For example, a details page for Cell (Figure 3A) includes information on location in the pathway hierarchy, cell type, and histological description of the given cell with ontology terms, a list of protein and RNA markers, and corresponding literature references for each marker. A scroll bar appears when there are more than 5 markers in the same category. The page also lists event(s) (i.e., Cell Development Step), in which the Cell participates as an input/output.  \r\n<ul style=\"list-style-type: disc;\">\r\n<li>Clicking on marker or Cell Development Step, will redirect to the details page of the marker or event respectively. </li>\r\n<li>Clicking on the Cell in the expanded hierarchical view of the Developmental Biology pathway (Figure 3B) in the details page, will redirect to the Pathway Browser view highlighting the Cell Lineage Path in the hierarchy panel and showing the selected Cell in the pathway diagram.</li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<p><img src=\"images/userguide/cytomics/Figure_3A.png\" alt=\"Figure 3A. Details page for Cell \" width=\"1322\" height=\"938\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong> Figure 3</strong>. Details page for Cell (A)</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_3B.png\" alt=\"Figure 3B. expanded view of the Cell location in the Pathway Browser B . \" width=\"1656\" height=\"384\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 3</strong>. Expanded view of the Cell location in the Pathway Browser (B)</p>\r\n<ul style=\"list-style-type: disc;\">\r\n<li aria-level=\"1\">See the details pages for marker instance (Figure 4), Cell Development Step (Figure 5), Cell Lineage Path (Figure 6) below. \r\n<ul style=\"list-style-type: disc;\">\r\n<li aria-level=\"1\">The content of individual events can be exported from the event’s details pages (Figure 5, 6) as SBML, PDF, SVG, PNG, PPTX, SBGN files.</li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<p><img src=\"images/userguide/cytomics/Figure_4.png\" alt=\" Figure 4. Details page for Marker instance\" width=\"1672\" height=\"1524\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 4</strong>. Details page for Marker instance</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_5.png\" alt=\"Figure 5. Details page for Cell Development Step\" width=\"1190\" height=\"1764\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 5</strong>. Details page for Cell Development Step</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_6.png\" alt=\"Figure 6. Details page for Cell Lineage Path\" width=\"1338\" height=\"1740\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 6</strong>. Details page for Cell Lineage Path</p>\r\n<h4>Navigation of a Cell Lineage Path in the Pathway Browse</h4>\r\n<p>Cell Lineage Paths are available under the “Developmental Biology” pathway in the Reactome Pathway Browser, where they are grouped in the “Developmental Cell Lineages” subpathway (Figure 7). </p>\r\n<p><br />Although the pathway diagram depicts a selection of cell types that are planned for annotation, only one type has been annotated so far and this is indicated by a selectable blue box label in the diagram.</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_7.png\" alt=\"Figure 7. Pathway browser view of Developmental Cell Lineages subpathway\" width=\"1880\" height=\"1014\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 7.</strong> Pathway browser view of “Developmental Cell Lineages” subpathway</p>\r\n<p>Selecting an individual Cell Lineage Path in the pathway hierarchy will display all Cell Development Steps of the selected path in the pathway diagram (Figure 8). An individual Cell is depicted with a double-layered outer compartment representing the plasma membrane and the cytosol, along with an inner blue rectangle symbolizing the nucleus.</p>\r\n<p>The Description tab in the Details panel of a selected Cell Lineage Path provides additional information about the Cell Lineage Path including name, species, assigned GO Biological Process term (if applicable), and literature reference(s) (Figure 8).</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_8.png\" alt=\"Figure 8. Selecting a Cell Lineage Path from the pathway hierarchy\" width=\"2880\" height=\"1554\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 8.</strong> Selecting a Cell Lineage Path from the pathway hierarchy</p>\r\n<p>Clicking on an individual Cell Development Step either in the pathway hierarchy or in the pathway diagram will display relevant information for the selected event in the details panel below the diagram (Figure 9). The description tab of the details panel shows input and output, which are defined by Cell instances, regulators of the event (when applicable), GO biological process, and UBERON tissue terms. It may also display preceding event(s) connecting individual Cell Development Steps within the same Cell Lineage Path (Figure 9). Details of participating Cell instances including histological terms, specific protein and RNA markers, and corresponding marker references are accessible from the description tab of Cell Development Step upon clicking on the plus sign to the right of the input/output attributes.</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_9.png\" alt=\"Figure 9. Selecting a Cell Development Step\" width=\"2880\" height=\"1554\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 9.</strong> Selecting a Cell Development Step</p>\r\n<p>Alternatively, the Cell details can be displayed in the Details panel by selecting a “Cell” icon in the diagram of the Cell Lineage Path (Figure 10). Right-clicking or clicking the blue info icon when hovering over a “Cell” icon in the pathway diagram will open a popup list of protein and RNA markers for the selected Cell (Figure 11).</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_10.png\" alt=\"Figure 10. Selecting a Cell\" width=\"2880\" height=\"1554\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 10.</strong> Selecting a Cell</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_11.png\" alt=\"Figure 11. Displaying the list of Cell markers\" width=\"2880\" height=\"1554\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 11.</strong> Displaying the list of Cell markers</p>\r\n<p>The Molecules tab in the Details panel (Figure 12A) displays a downloadable list of all markers for a Cell and both the markers and regulators for a Cell Development Step and a Cell Lineage Path.</p>\r\n<p>The Structure tab (Figure 12B) in the details panel displays the Protein Data Bank structures of molecules listed in the Molecular tab and the Expression tab shows gene expression data from Expression Atlas.</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_12A.png\" alt=\"Figure 12A\" width=\"2880\" height=\"1482\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 12.</strong> Displaying Molecules Tab (A) </p>\r\n<p><img src=\"images/userguide/cytomics/Figure_12B.png\" alt=\"Figure 12B\" width=\"2880\" height=\"1562\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 12.</strong> Structures tab (B)</p>\r\n<p>The Analysis tab (Figure 13, 14) in the details panel displays data after running the dataset analysis. Refer to (Rothfels et al. 2023) for detailed instructions on how to use the Reactome Analysis Tool.</p>\r\n<p>The results of the analysis are also visualized in the pathway diagram panel, where the “Cell” icons are colored (Figure 13, 14). The colored area in the nucleus of the individual Cell corresponds to the coverage of the cell markers in the submitted data set.</p>\r\n<p>In Pathway Enrichment Analysis, olive green is applied to color the nucleus of the cells in the Cell Lineage Path that contain markers from the query list. The extent of coloration is proportional to the number of markers hit (Figure 13).</p>\r\n<p>The results of gene expression analysis, including Reactome Gene Set Analysis (Reactome GSA), appear as a single vertical bar of color when zoomed out representing the average expression of the markers from the query list (Figure 14a). Zooming in on the “Cell” icon reveals bars for individual cell markers with their expression values, as shown in Figure 14b.Expression values of cell markers from the query data set can be visualized in a popup information panel of protein and RNA markers for the selected Cell (Figure 14b).</p>\r\n<p>Expression values for the markers of a selected cell are displayed as blue horizontal lines within the expression scale (Figure 14C). The median cell marker expression among the cell\'s matching markers is shown in black. On the left, when a specific marker bar is hovered over, the currently hovered marker’s expression value is displayed in red, and the expression values of other markers in the same cell are shown in yellow.</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_13_samll.png\" alt=\"Figure 13 samll\" width=\"1092\" height=\"548\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 13.</strong> Overrepresentation analysis displayed within the nuclei of the cells in the Cell Lineage Path</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_14A.png\" alt=\"Figure 14A\" width=\"2678\" height=\"1384\" style=\"border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 14.</strong> Gene expression analysis results are displayed within the nuclei of the cells. The level with coloration reflects the average expression value of the cell markers (A)</p>\r\n<p><img src=\"images/userguide/cytomics/Figure_14B.png\" alt=\"Figure 14B\" width=\"1026\" height=\"620\" style=\"display: block; margin-left: auto; margin-right: auto; border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 14.</strong> A zoomed-in view of the expression bars of individual markers (B) </p>\r\n<p><img src=\"images/userguide/cytomics/Figure_14C.png\" alt=\"Figure 14C\" width=\"554\" height=\"498\" style=\"display: block; margin-left: auto; margin-right: auto; border: 1px solid #000000;\" /></p>\r\n<p style=\"text-align: center;\"><strong>Figure 14. T</strong>he expression scale (C)</p>\r\n<p></p>\r\n<ol>\r\n<li aria-level=\"1\">Burgess, Darren J. 2018. “Tracing Cell-Lineage Histories.” <em>Nature Reviews. Genetics</em>.</li>\r\n<li aria-level=\"1\">Croft, David, Gavin O’Kelly, Guanming Wu, Robin Haw, Marc Gillespie, Lisa Matthews, Michael Caudy, et al. 2011. “Reactome: A Database of Reactions, Pathways and Biological Processes.” <em>Nucleic Acids Research</em> 39 (Database issue): D691–97.</li>\r\n<li aria-level=\"1\">Franzén O, Gan LM, Björkegren JLM, \"PanglaoDB: a web server for exploration of mouse and human single cell RNA sequencing data\", Database (Oxford), 2019, 2019.</li>\r\n<li aria-level=\"1\">Haendel, Melissa A., James P. Balhoff, Frederic B. Bastian, David C. Blackburn, Judith A. Blake, Yvonne Bradford, Aurelie Comte, et al. 2014. “Unification of Multi-Species Vertebrate Anatomy Ontologies for Comparative Biology in Uberon.” <em>Journal of Biomedical Semantics</em> 5 (May): 21.</li>\r\n<li aria-level=\"1\">Hu C, Li T, Xu Y, Zhang X, Li F, Bai J, Chen J, Jiang W, Yang K, Ou Q, Li X, Wang P, Zhang Y, \"CellMarker 2.0: an updated database of manually curated cell markers in human/mouse and web tools based on scRNA seq data\", Nucleic Acids Res, 2022.</li>\r\n<li aria-level=\"1\">Joshi-Tope, G., M. Gillespie, I. Vastrik, P. D’Eustachio, E. Schmidt, B. de Bono, B. Jassal, et al. 2005. “Reactome: A Knowledgebase of Biological Pathways.” <em>Nucleic Acids Research</em> 33 (Database issue): D428–32.</li>\r\n<li aria-level=\"1\">Milacic M, Beavers D, Conley P, et al. The Reactome Pathway Knowledgebase 2024. <em>Nucleic Acids Res</em>. 2024;52(D1):D672-D678. </li>\r\n<li aria-level=\"1\">Osumi-Sutherland, David. 2017. “Cell Ontology in an Age of Data-Driven Cell Classification.” <em>BMC Bioinformatics</em> 18 (Suppl 17): 558.</li>\r\n<li aria-level=\"1\">Saelens, Wouter, Robrecht Cannoodt, Helena Todorov, and Yvan Saeys. 2019. “A Comparison of Single-Cell Trajectory Inference Methods.” <em>Nature Biotechnology</em> 37 (5): 547–54.</li>\r\n<li aria-level=\"1\">Rothfels Karen, Milacic Marija, Matthews Lisa et al. 2023. “Using the Reactome Database.” <em>Current protocols</em> vol. 3,4: e722.</li>\r\n<li aria-level=\"1\">Sarntivijai, Sirarat, Yu Lin, Zuoshuang Xiang, Terrence F. Meehan, Alexander D. Diehl, Uma D. Vempati, Stephan C. Schürer, et al. 2014. “CLO: The Cell Line Ontology.” <em>Journal of Biomedical Semantics</em> 5 (August): 37.</li>\r\n<li>The Gene Ontology Consortium. 2019. “The Gene Ontology Resource: 20 Years and Still GOing Strong.” <em>Nucleic Acids Research</em> 47 (D1): D330–38.</li>\r\n</ol>','',1,28,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-03-21 13:54:51','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(248,953,'Nickel-induced transcriptional memory in lung epithelial cells promotes interferon signaling upon nicotine exposure','nickel-induced-transcriptional-memory-in-lung-epithelial-cells-promotes-interferon-signaling-upon-nicotine-exposure','<p>In the December 2023 issue of<a href=\"https://clinicalepigeneticsjournal.biomedcentral.com/articles/10.1186/s13148-023-01539-0\"> </a>Toxicology and Applied Pharmacology, <a href=\"https://pubmed.ncbi.nlm.nih.gov/37951547/\">Zhang et al</a>. used the R package, ReactomePA <a href=\"https://pubs.rsc.org/en/content/articlelanding/2016/mb/c5mb00663e\">(Yu and He, 2016)</a>, to identify enriched pathways responding to nickel-induced transcriptional memory changes in response to a second respiratory toxicant, nicotine. Nicotine exposure upregulated a specific subset of genes in the cells previously exposed to nickel, identifying a robust activation of <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-913531\">Interferon (IFN) signaling</a>, a driver of inflammation associated with many chronic lung diseases.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-04-09 04:51:43','0000-00-00 00:00:00','{}','{}','{}',1,6,'','',1,0,'{}',0,'en-GB','',''),(249,954,'IBPGNET: lung adenocarcinoma recurrence prediction based on neural network interpretability','ibpgnet-lung-adenocarcinoma-recurrence-prediction-based-on-neural-network-interpretability','<p>In the May 2024 issue of Briefings in Bioinformatics, <a href=\"https://doi.org/10.1093/bib/bbae080\">Xu et al.</a> develop an Interpretable Biological Pathway Graph Neural Network (IBPGNET) framework based on Reactome pathway hierarchy to predict regulatory mechanisms that lead to lung adenocarcinoma recurrences. IBPGNET identified two genes of interest and performed in vitro knockdown models for drug sensitivity experimental validation. This study offers an approach for exploring molecular mechanisms underlying recurrence using Reactome’s hierarchical pathway structure.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-05-10 16:15:21','0000-00-00 00:00:00','{}','{}','{}',1,5,'','',1,0,'{}',0,'en-GB','',''),(250,955,'New Publication in Database (Oxford)! ','new-publication-in-database-oxford','<p>Introducing a new publication from the Reactome team in the Database (Oxford) journal by Orlic-Milacic et al, titled \"<a href=\"https://doi.org/10.1093/database/baae031\">Pathway-based, reaction-specific annotation of disease variants for elucidation of molecular phenotypes</a>\"! </p>\r\n<p>The study outlines Reactome\'s expansion to encompass annotations for <strong>disease variants</strong>, providing insights into aberrant reactions and pathways caused by germline and somatic mutations. Aligning with the <strong>American College of Medical Genetics and Genomics (ACMG)/Association for Molecular Pathology (AMP)</strong> standards, Reactome classifies variants as benign or pathogenic, facilitating integration with other variant databases like<strong> ClinGen</strong> and <strong>ClinVar.</strong> Rather than exhaustively cataloging variants, Reactome focuses on characterizing the impact of representative variants on pathway activity, with annotations cross-referencing external resources such as <strong>OMIM</strong> and <strong>COSMIC</strong>. Despite the availability of computational tools, manual curation remains essential due to limitations in predicting functional impact and the narrow focus on missense variants.</p>\r\n<p>The paper also introduces a protocol developed by the Reactome team for annotating variants in pathways, alongside an expanded dataset covering diverse classes of protein variants, including fusion proteins. These pathway-based annotations not only aid in identifying gaps in computational predictions but also enhance the interpretation and modeling of clinically relevant variants.</p>\r\n<p>For other publications by the Reactome Team, visit our <a href=\"https://reactome.org/community/publications\">publications</a> page.</p>\r\n<p><img src=\"images/news/DiseaseVarFig7.jpeg\" alt=\"\" /></p>\r\n<p>Shown here is Figure 7: High-level graphical summary of Reactome’s ERBB2 cancer variants content. (A) Heatmap representation of Reactome electronic textbook knowledge on the sensitivity of different ERBB2 cancer variants to ERBB2-targeted anti-cancer therapeutics. The heatmap was generated using the R package pheatmap with default settings. (B) Interactive textbook style diagram for ‘Signaling by ERBB2 in Cancer’ pathway.</p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-05-14 21:45:16','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,2,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(251,957,'V89 Released','v89-released','<p><img src=\"images/news/R-HSA-212165.png\" alt=\"R HSA 212165\" width=\"1396\" height=\"798\" /></p>\r\n<p>New and Updated Topics and Pathways. Topics with new or revised pathways in this release include Autophagy (<a href=\"PathwayBrowser/#/R-HSA-5205647\">Mitophagy</a>), Cell Cycle (<a href=\"PathwayBrowser/#/R-HSA-912446\">Meiotic recombination</a> and <a href=\"PathwayBrowser/#/R-HSA-2500257\">Resolution of sister chromatid cohesion</a>); Developmental Biology (<a href=\"PathwayBrowser/#/R-HSA-9730414\">MITF-M-regulation melanocyte development</a>); DNA Repair (<a href=\"PathwayBrowser/#/R-HSA-5693568\">Resolution of D-loop structures through Holliday Junction Intermediates</a>); Gene Expression (<a href=\"PathwayBrowser/#/R-HSA-9842860\">Regulation of endogenous retroelements</a>); Immune System (<a href=\"PathwayBrowser/#9860276\">SLC15A4:TASL-dependent IRF5 activation</a>); Metabolism (<a href=\"PathwayBrowser/#1428517\">Aerobic respiration and respiratory electron transport</a> including <a href=\"PathwayBrowser/#611105\">Respiratory electron transport</a>, <a href=\"PathwayBrowser/#6799198\">Complex I biogenesis</a>, <a href=\"PathwayBrowser/#9865881\">Complex III assembly</a>, and <a href=\"PathwayBrowser/#9864848\">Complex IV assembly</a>; <a href=\"PathwayBrowser/#166187\">Mitochondrial uncoupling</a>; <a href=\"PathwayBrowser/#75105\">Fatty acyl-CoA biosynthesis</a>; <a href=\"PathwayBrowser/#211976\">Endogenous sterols</a>; <a href=\"PathwayBrowser/#/R-HSA-75876\">Synthesis of very long-chain fatty acyl-CoAs</a>, <a href=\"PathwayBrowser/#2142789\">Ubiquinol biosynthesis</a>; <a href=\"PathwayBrowser/#/R-HSA-9858328\">OADH complex synthesizes glutaryl-CoA from 2-OA</a>; <a href=\"PathwayBrowser/#9859138\">BCKDH synthesizes BCAA-CoA from KIC, KMVA, KIV</a>, Metabolism of proteins (<a href=\"PathwayBrowser/#446205\">Synthesis of GDP-mannose</a>), and Signal Transduction (<a href=\"PathwayBrowser/#9839373\">TGFBR3 signaling</a>).</p>\r\n<p>New and Updated Illustrations.  New or revised illustrations with embedded navigation features have been created for <a href=\"PathwayBrowser/#212165\">Epigenetic regulation of gene expression</a>, <a href=\"PathwayBrowser/#1643713\">Signaling by EGFR in Cancer</a>, and <a href=\"PathwayBrowser/#9006936\">Signaling by TGFB family members</a>.</p>\r\n<p>Thanks to our Contributors. <a href=\"https://orcid.org/0000-0001-9499-7291\">Deborah Bourc\'his</a>, <a href=\"https://orcid.org/0000-0002-6429-287X\">Ivana de la Serna</a>, <a href=\"https://orcid.org/0000-0001-6377-0799\">Liane P Fernandes</a>, <a href=\"https://orcid.org/0000-0002-6921-1493\">Leonhard X Heinz</a>, <a href=\"https://orcid.org/0000-0001-7476-6306\">David P Hill</a>, <a href=\"https://orcid.org/0000-0003-0764-7483\">Christian Löw</a>, <a href=\"https://orcid.org/0009-0003-7061-4972\">Marco Marchi</a>, <a href=\"https://orcid.org/0000-0002-7350-3413\">Alexandre Orthwein</a>, <a href=\"https://orcid.org/0000-0003-2083-3002\">Fabien Pierrel</a>, <a href=\"https://pubmed.ncbi.nlm.nih.gov/?term=Tonello+F&amp;cauthor_id=35503380\">Fiorella Tonello</a>, <a href=\"https://orcid.org/0000-0003-3474-0456\">Noriko Toyama-Sorimachi</a>, and <a href=\"https://orcid.org/0000-0003-4534-394X\">Mark Williams</a> are our external reviewers.</p>\r\n<p>Annotation Statistics. Reactome comprises 15,326  human reactions organized into 2,711  pathways involving 30,733  proteins and modified forms of proteins encoded by 11,279 different human genes, 14,897 complexes, 2,127 small molecules, and 1,047 drugs. These annotations are supported by 38,895 literature references. We have projected these reactions onto 79,737  orthologous proteins, creating 19,657 orthologous pathways in 14 non-human species. Version 89 has annotations for 4,857 protein variants (mutated proteins) and their post-translationally modified forms, derived from 376 proteins, which have contributed to the annotation of 1,802 disease-specific reactions and  725 pathways. </p>\r\n<p> </p>\r\n<p>Tools and Data. Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and <a href=\"https://reactome.org/AnalysisService/\">Analysis</a> Services. The <a href=\"https://reactome.org/userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"https://reactome.org/gsa\">ReactomeGSA</a> package provide tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p>Documentation and Training. Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/community/training\">here</a>.</p>\r\n<p>About the Reactome Project. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a> as well as a <a href=\"https://globalbiodata.org/scientific-activities/global-core-biodata-resources/\">Global Core Biodata Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art, and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/\">website</a>.</p>\r\n<p>Follow us on Twitter: <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p>For more information: If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <a href=\"mailto:help@reactome.org.\">help@reactome.org.</a></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-06-10 02:59:54','0000-00-00 00:00:00','{}','{}','{}',1,1,'','',1,0,'{}',0,'en-GB','',''),(252,959,'Acquired resistance to immunotherapy and chemoradiation in MYC amplified head and neck cancer','acquired-resistance-to-immunotherapy-and-chemoradiation-in-myc-amplified-head-and-neck-cancer','<p>In the May 2024 issue of NPJ Precision Oncology, <a href=\"https://www.nature.com/articles/s41698-024-00606-w\">Cyberski et al.</a> used Reactome’s hierarchically arranged pathways with their in silico Pathway Activation Network Decomposition Analysis (iPANDA) algorithm to identify upregulation of networks associated with <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-1640170\">cell cycle </a>progression, <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-162582\">signal transduction</a>, and <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-1430728\">metabolism</a> and down-regulation of <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-168256\">immune cellular process </a>and <a href=\"https://reactome.org/PathwayBrowser/#/R-HSA-109581&amp;PATH=R-HSA-5357801\">apoptosis</a> in MYC-amplified cases of recurrent/metastatic head and neck squamous cell carcinoma.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-06-18 14:04:25','0000-00-00 00:00:00','{}','{}','{}',1,4,'','',1,0,'{}',0,'en-GB','',''),(253,960,'Drug target prediction through deep learning functional representation of gene signatures','drug-target-prediction-through-deep-learning-functional-representation-of-gene-signatures','<p>In their May 2024 Nature Communications paper, <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10904399/\">Chen et al.</a>. used data simulated based on information from Reactome pathways to validate their Functional Representation of Gene Signatures (FRoGS) algorithm, a deep learning-based approach that was designed to improve the accuracy of drug target predictions by addressing limitations of gene identity-based pathway analysis.</p>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-07-02 10:02:10','0000-00-00 00:00:00','{}','{}','{}',1,1,'','',1,0,'{}',0,'en-GB','',''),(254,961,'Drug target prediction through deep learning functional representation of gene signatures','drug-target-prediction-through-deep-learning-functional-representation-of-gene-signatures','<p>In their May 2024 Nature Communications paper, <a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10904399/\">Chen et al</a> used data simulated based on Reactome pathways to validate their Functional Representation of Gene Signatures (FRoGS) algorithm, a deep learning-based approach that was designed to improve the accuracy of drug target predictions by addressing limitations of gene identity-based pathway analysis.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-07-04 20:52:17','0000-00-00 00:00:00','{}','{}','{}',1,3,'','',1,0,'{}',0,'en-GB','',''),(255,962,'The landscape of cancer-rewired GPCR signaling axes','the-landscape-of-cancer-rewired-gpcr-signaling-axes','<p>In their May 2024 paper in Cell Genomics,<a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11099383/\"> Arora et al.</a> used a framework of Reactome signaling and metabolism pathways to integrate RHEA metabolic reactions and IUPhAR catalogs of G Protein-Coupled Receptors (GPCRs) and their ligands to define axes that combine<a href=\"PathwayBrowser/#/R-HSA-162582\"> signaling cascades</a> and ligand<a href=\"PathwayBrowser/#/R-HSA-1430728\"> metabolic processes</a>. Altered expression of the sets of proteins that make up these axes correlate with patient survival cataloged in The Cancer Genome Atlas (TCGA) for many tumor types and suggest novel druggable targets.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-07-23 17:34:24','0000-00-00 00:00:00','{}','{}','{}',1,2,'','',1,0,'{}',0,'en-GB','',''),(256,963,'Engineering Toxoplasma gondii secretion systems for intracellular delivery of multiple large therapeutic proteins to neurons','engineering-toxoplasma-gondii-secretion-systems-for-intracellular-delivery-of-multiple-large-therapeutic-proteins-to-neurons','<p>In the August 2024 issue of Nature Microbiology, <a href=\"https://pubmed.ncbi.nlm.nih.gov/39075233/\">Bracha et al</a>. use Reactome expression analysis to confirm that they successfully delivered multiple large (&gt;100 kDa) therapeutic proteins across the blood-brain barrier into target neurons in mice using engineered <em>Toxoplasma gondii</em> secretion systems.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-08-29 22:48:21','0000-00-00 00:00:00','{}','{}','{}',1,1,'','',1,0,'{}',0,'en-GB','',''),(257,965,'Community','community','<p> {loadposition community}</p>\r\n<p></p>\r\n<h1>Contribute to Reactome!</h1>\r\n<p></p>\r\n<p><a href=\"https://reactome.org/userguide/claim-your-work\"><img src=\"images/Benefits.png\" alt=\"\" width=\"100%\" height=\"187\" /></a></p>\r\n<p><span style=\"background-color: inherit; color: inherit; font-family: inherit; font-size: 10pt; caret-color: auto;\">In 4 simple steps, you can become a part of Reactome\'s network of <a href=\"https://reactome.org/content/contributors\">over 900 contributors</a>! Reactome would not exist without contributions from the scientific community.</span></p>\r\n<p><img src=\"images/CollaborateWithUs.png\" alt=\"\" width=\"100%\" height=\"696\" /></p>\r\n<h2></h2>\r\n<h2>1. Begin</h2>\r\n<p><span style=\"font-size: 14pt;\"><strong>We are currently seeking reviewers for these </strong><a href=\"https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Freactome.org%2Fcommunity%2Fcollaboration&amp;data=05%7C02%7Cn2li%40oicr.on.ca%7C8a40e1f1a10544721be208dcc91aca61%7C9df949f8a6eb419d9caa1f8c83db674f%7C0%7C0%7C638606364161407941%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&amp;sdata=YL7k6cxBwunAGp8aiz0MTXXx8rayG4ygLQTrF2kcfh0%3D&amp;reserved=0\" data-ogsc data-outlook-id=\"c8ae2f00-2dd3-4901-9eff-9e8111d48f1e\"><strong>pathways</strong></a><strong>!  </strong></span></p>\r\n<p><span style=\"font-size: 10pt;\">Please let us know if you would be interested in helping with one of them.</span></p>\r\n<p><span style=\"font-size: 10pt;\">We would also welcome reviews or revisions to Reactome pathways not on this list and contributions of pathways not yet represented in Reactome!</span></p>\r\n<h2><em>Questions?</em></h2>\r\n<p><span style=\"font-size: 10pt;\">Email our team at <a href=\"mailto:help@reactome.org\">help@reactome.org</a> with any questions!</span></p>\r\n<p>Here is an <a href=\"https://www.reactome.org/community/collaboration/faq-for-prospective-reviewers-and-authors\">FAQ</a> for reviewers and authors </p>\r\n<h2>2. Expert feedback</h2>\r\n<p><span style=\"font-size: 10pt; color: inherit;\">Conduct your peer-review by providing your comments and suggestions for improvement to our curation team. The peer-review process for a pathway is akin to conducting peer-review for academic journals. We will provide you with a text document, where you can provide your comments.</span></p>\r\n<p><span style=\"font-size: 10pt; color: inherit;\">Our database updates are released quarterly, we are happy to work with you to find the best timeline for your busy schedule. </span></p>\r\n<p><a href=\"https://www.reactome.org/community/collaboration/faq-for-prospective-reviewers-and-authors\"><span style=\"font-size: 10pt; color: inherit;\"><img src=\"images/Responsibilities.png\" alt=\"\" width=\"100%\" height=\"152\" /></span></a></p>\r\n<h2>3. Revisions</h2>\r\n<p><span style=\"font-size: 10pt;\">Provide your expert feedback to our curation team. Our curation team will take care of making your suggested changes.</span></p>\r\n<h2>4. Public Release</h2>\r\n<p><span style=\"font-size: 10pt;\"><span style=\"background-color: inherit; color: inherit; font-family: inherit; caret-color: auto;\">You will be credited with authorship or reviewership for all of your contributions. Each pathway is associated with a DOI and can be cited as a publication.</span></span></p>\r\n<p><span style=\"font-size: 10pt;\"><span style=\"background-color: inherit; color: inherit; font-family: inherit; caret-color: auto;\">You can quickly and easily claim your Reactome contributions in ORCID using our new ORCID claiming feature when you search for your name in Reactome as described <a href=\"https://reactome.org/userguide/claim-your-work\">here</a>.<br /></span></span></p>\r\n<p></p>','',1,2,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-08-30 15:10:36','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(258,966,'Funding','funding','<p>Reactome  core curation, software development and outreach is supported by the <a href=\"https://www.nih.gov/\" target=\"_blank\" rel=\"noopener\">National Institute of Health</a> (U24HG012198, 2022-2027), the <a href=\"https://www.embl.org/\" target=\"_blank\" rel=\"noopener\">European Molecular Biology Laboratory</a> and <a href=\"https://www.opentargets.org/\" target=\"_blank\" rel=\"noopener\">Open Targets</a> (OTAR-006, 2015-2024). Additional NIH funding (U24HG011851, 2021-2026) is used to support the development of GO-CAM models, aligning the Gene Ontology and the Reactome data resources. </p>\r\n<h3>Past funding:</h3>\r\n<p>Core Reactome activities have been supported by NIH grants U41HG003751 (2007-2022) and (2002-2006).</p>\r\n<p>Development of the Reactome IDG (Illuminating the Druggable Genome) portal for pathway-based analysis and visualization of understudied proteins) was supported by NIH grant U01CA239069 (2019-2023). Improvements to Reactome’s TRUST-worthiness and Continuous Integration/Continuous Deployment (CI/CD) were supported by NIH supplementary grants U24HG12198-02S1 and U24HG12198-02S2, respectively. </p>\r\n<p>Rapid curation of the COVID-19 pathway was supported by NIH supplementary grant U41HG003751-13S1.</p>\r\n<p>Development of tools to scale up biological pathway knowledge acquisition through text mining and crowdsourcing was supported by NIH supplementary grant U41HG003751-12S1.</p>\r\n<p>Development of our cell lineage pathways was supported by a University of Toronto <a href=\"https://mbd.utoronto.ca/\">Medicine by Design</a> Seed Fund (MbDNISF-2020-03).</p>\r\n<p>Development of the Reactome GSA tool was supported by a grant from the <a href=\"https://marie-sklodowska-curie-actions.ec.europa.eu/\">European Union Marie Sklodowska-Curie Actions</a> fund (2019).</p>\r\n<p>Development of a Reactome portal for NURSA (Nuclear Receptor Signaling Atlas) was supported by NIH grant U24DK097748 (2017-2018).</p>\r\n<p>Curation of neural stem cell signaling pathways was supported by a University of <a href=\"https://mbd.utoronto.ca/\">Toronto Medicine by Design</a> grant (2016-2019).</p>','',1,25,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-09-03 09:01:36','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(259,967,'Curator Guide','curator-guide','<p>The current Curator Guide with associated appendices (last updated V90) is available for download <a href=\"https://download.reactome.org/documentation/CuratorGuideAndAppendices_V90.pdf\">here</a>.</p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-09-03 09:29:17','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,1,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(260,968,'Release Documentation','release-documentation','<p>The current Release SOP with associated appendices (last updated V90) is available for download <a href=\"https://download.reactome.org/documentation/ReleaseSOPandAppendices_V90.pdf\">here</a>. </p>','',1,14,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-09-03 09:49:33','0000-00-00 00:00:00','{\"image_intro\":\"\",\"float_intro\":\"\",\"image_intro_alt\":\"\",\"image_intro_caption\":\"\",\"image_fulltext\":\"\",\"float_fulltext\":\"\",\"image_fulltext_alt\":\"\",\"image_fulltext_caption\":\"\"}','{\"urla\":false,\"urlatext\":\"\",\"targeta\":\"\",\"urlb\":false,\"urlbtext\":\"\",\"targetb\":\"\",\"urlc\":false,\"urlctext\":\"\",\"targetc\":\"\"}','{\"article_layout\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_tags\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_vote\":\"\",\"show_hits\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"alternative_readmore\":\"\",\"article_page_title\":\"\",\"show_publishing_options\":\"\",\"show_article_options\":\"\",\"show_urls_images_backend\":\"\",\"show_urls_images_frontend\":\"\"}',1,0,'','',1,0,'{\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}',0,'*','',''),(261,969,'V90 Released','v90-news','<p><img src=\"images/news/R-HSA-2262752.png\" alt=\"R HSA 2262752\" width=\"1396\" height=\"798\" /></p>\r\n<p style=\"text-align: center;\"><a href=\"PathwayBrowser/#/R-HSA-2262752\"><span style=\"font-size: 14pt;\">Cellular response to stress</span></a></p>\r\n<p></p>\r\n<p>New and Updated Topics and Pathways. Topics with new or revised pathways in this release include Cellular responses to stimuli (<a href=\"PathwayBrowser/#/R-HSA-9856530\">High laminar flow shear stress activates signaling by PIEZO1 and PECAM1:CDH5:KDR in endothelial cells</a>, <a href=\"PathwayBrowser/#/R-HSA-9860927\">Turbulent (oscillatory, disturbed) flow shear stress activates signaling by PIEZO1 and integrins in endothelial cells</a>, <a href=\"PathwayBrowser/#/R-HSA-9841251\">Mitochondrial unfolded protein response (mtUPR)</a>), Disease (<a href=\"PathwayBrowser/#/R-HSA-9865118\">Disease of branched-chain amino acid catabolism</a>), Gene Expression (<a href=\"PathwayBrowser/#/R-HSA-9818564\">Epigenetic regulation of gene expression by MLL3 and MLL4 complexes</a>, <a href=\"PathwayBrowser/#/R-HSA-9851695\">Epigenetic regulation of adipogenesis genes by MLL3 and MLL4</a>), Metabolism (<a href=\"PathwayBrowser/#/R-HSA-5423646\">Aflatoxin activation and detoxification</a>, <a href=\"PathwayBrowser/#/R-HSA-2142753\">Arachidonate metabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-9027604\">Biosynthesis of electrophilic ω-3 PUFA oxo-derivatives</a>, <a href=\"PathwayBrowser/#/R-HSA-9018678\">Biosynthesis of specialized pro-resolving mediators (SPMs)</a>, <a href=\"PathwayBrowser/#/R-HSA-200425\">Carnitine shuttle</a>, <a href=\"PathwayBrowser/#/R-HSA-9759218\">Cobalamin (Cbl) metabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-211976\">Endogenous sterols</a>, <a href=\"PathwayBrowser/#/R-HSA-173599\">Formation of the active cofactor, UDP-glucuronate</a>, <a href=\"PathwayBrowser/#/R-HSA-194002\">Glucocorticoid biosynthesis</a>, <a href=\"PathwayBrowser/#/R-HSA-202131\">Metabolism of nitric oxide: NOS3 activation and regulation</a>, <a href=\"PathwayBrowser/#/R-HSA-390918\">Peroxisomal lipid metabolism</a>), and Metabolism of proteins (<a href=\"PathwayBrowser/#/R-HSA-9907900\">Proteasome assembly</a>).</p>\r\n<p>New and Updated Illustrations.  New or revised Illustrations with embedded navigation features have been created for  <a href=\"PathwayBrowser/#/R-HSA-9855142\">Cellular responses to mechanical stimuli</a>, <a href=\"PathwayBrowser/#/R-HSA-8953897\">Cellular responses to stimuli</a>, <a href=\"PathwayBrowser/#/R-HSA-2262752\">Cellular responses to stress</a>, <a href=\"PathwayBrowser/#/R-HSA-5668914\">Diseases of Metabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-9865118\">Diseases of branched-chain amino acid catabolism</a>, <a href=\"PathwayBrowser/#/R-HSA-9865114\">Maple Syrup Urine disease</a>, <a href=\"PathwayBrowser/#/R-HSA-212165\">Epigenetic regulation of gene expression</a>, <a href=\"PathwayBrowser/#/R-HSA-597592\">Post-translational protein modification</a>, <a href=\"PathwayBrowser/#/R-HSA-9860931\">Response of endothelial cells to shear stress</a>, and <a href=\"PathwayBrowser/#/R-HSA-168898\">Toll-Like Receptors Cascades</a>.</p>\r\n<p>Thanks to our Contributors.  <a href=\"https://orcid.org/0000-0001-7476-6306\">David P Hill</a> and <a href=\"https://orcid.org/0000-0001-5541-6685\">Rui Xiao</a> are our external reviewers.</p>\r\n<p>Annotation Statistics. Reactome comprises 15,492  human reactions organized into 2,742  pathways involving 30,892  proteins and modified forms of proteins encoded by 11,289 different human genes, 15,299 complexes, 2,127 small molecules, and 1,057 drugs. These annotations are supported by 39,318 literature references. We have projected these reactions onto 79,857 orthologous proteins, creating 19,783 orthologous pathways in 14 non-human species. Version 90 has annotations for 4,943 protein variants (mutated proteins) and their post-translationally modified forms, derived from 391 proteins, which have contributed to the annotation of 1,808 disease-specific reactions and 746 pathways. </p>\r\n<p>Tools and Data. Our services and software tools are designed for biologists, bioinformaticians, and software developers. Pathway data is available to view in our <a href=\"https://reactome.org/PathwayBrowser/\">Pathway Browser</a>, to <a href=\"https://reactome.org/PathwayBrowser/#TOOL=AT\">analyze</a> your own dataset, to <a href=\"https://reactome.org/download-data\">download</a>, and access programmatically through our <a href=\"https://reactome.org/ContentService/\">Content</a> and <a href=\"https://reactome.org/AnalysisService/\">Analysis</a> Services. The <a href=\"https://reactome.org/userguide/reactome-fiviz\">ReactomeFIViz</a> app and <a href=\"https://reactome.org/gsa\">ReactomeGSA</a> package provide tools for multi-omics data analysis. The <a href=\"https://idg.reactome.org/\">idg.reactome.org</a> Web Portal provides a collection of web-based tools to help researchers place understudied proteins in a pathway context. </p>\r\n<p>Documentation and Training. Visit our online <a href=\"https://reactome.org/userguide\">User Guide</a> to access documentation supporting pathway analysis of experimental data. The <a href=\"https://reactome.org/dev\">Developer\'s Zone</a> provides detailed documentation regarding our software, tools, and web services. Training and learning materials can be found <a href=\"https://reactome.org/documentation\">here</a>.</p>\r\n<p>About the Reactome Project. Reactome is a collaboration between groups at the Ontario Institute for Cancer Research, Oregon Health and Science University, New York University Langone Medical Center, and The EMBL - European Bioinformatics Institute. Reactome is an <a href=\"https://elixir-europe.org/platforms/data/core-data-resources\">ELIXIR Core Data Resource</a> as well as a <a href=\"https://globalbiodata.org/scientific-activities/global-core-biodata-resources/\">Global Core Biodata Resource</a>. Reactome annotation files and interaction data derived from Reactome are distributed under a Creative Commons Public Domain (CC0 1.0 Universal) Licence. A Creative Commons Attribution 4.0 International (CC BY 4.0) Licence applies to all software and code, database data dumps, and Pathway Illustrations (Enhanced High-Level Diagrams), Icon Library, Art, and Branding Materials. A full description of the new and updated content is available on the Reactome <a href=\"https://reactome.org/what-is-reactome\">website</a>.</p>\r\n<p>Follow us on Twitter: <a href=\"https://twitter.com/reactome\">@reactome</a> to get frequent updates about new and updated pathways, feature updates, and more!</p>\r\n<p>For more information: If you have a question, want to provide feedback, or are interested in collaborating with us to annotate a topic, please contact us at <a href=\"mailto:help@reactome.org.\">help@reactome.org.</a></p>','',1,26,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-09-17 04:01:08','0000-00-00 00:00:00','{}','{}','{}',1,0,'','',1,0,'{}',0,'en-GB','',''),(262,971,'Chemical coverage of human biological pathways','chemical-coverage-of-human-biological-pathways','<p>In the feature article of the October 2024 issue of <a href=\"https://www.sciencedirect.com/journal/drug-discovery-today\">Drug Discovery Today</a> titled <a href=\"https://pubmed.ncbi.nlm.nih.gov/39179147/\">“Chemical coverage of human biological pathways</a>”, Kwak et al. describe the <a href=\"https://www.target2035.net/\">Target 2035</a> initiative, whose mission is to discover chemical tools for all human proteins by 2035. The authors use Reactome as the reference standard to determine the chemical coverage of human biological pathways and to outline the advantages of adopting the pathway-based rather than the proteome-based approach in guiding Target 2035 efforts.</p>','',1,36,'2017-01-01 09:00:00',156,'','2017-01-01 09:00:00',156,0,'1000-01-01 00:00:00','2024-10-03 15:25:44','0000-00-00 00:00:00','{}','{}','{}',1,0,'','',1,0,'{}',0,'en-GB','','');
/*!40000 ALTER TABLE `rlp_content` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_content_frontpage`
--

DROP TABLE IF EXISTS `rlp_content_frontpage`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_content_frontpage` (
  `content_id` int NOT NULL DEFAULT '0',
  `ordering` int NOT NULL DEFAULT '0',
  PRIMARY KEY (`content_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_content_frontpage`
--

LOCK TABLES `rlp_content_frontpage` WRITE;
/*!40000 ALTER TABLE `rlp_content_frontpage` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_content_frontpage` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_content_rating`
--

DROP TABLE IF EXISTS `rlp_content_rating`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_content_rating` (
  `content_id` int NOT NULL DEFAULT '0',
  `rating_sum` int unsigned NOT NULL DEFAULT '0',
  `rating_count` int unsigned NOT NULL DEFAULT '0',
  `lastip` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  PRIMARY KEY (`content_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_content_rating`
--

LOCK TABLES `rlp_content_rating` WRITE;
/*!40000 ALTER TABLE `rlp_content_rating` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_content_rating` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_content_types`
--

DROP TABLE IF EXISTS `rlp_content_types`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_content_types` (
  `type_id` int unsigned NOT NULL AUTO_INCREMENT,
  `type_title` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `type_alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `table` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `rules` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `field_mappings` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `router` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `content_history_options` varchar(5120) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL COMMENT 'JSON string for com_contenthistory options',
  PRIMARY KEY (`type_id`),
  KEY `idx_alias` (`type_alias`(100))
) ENGINE=InnoDB AUTO_INCREMENT=17 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_content_types`
--

LOCK TABLES `rlp_content_types` WRITE;
/*!40000 ALTER TABLE `rlp_content_types` DISABLE KEYS */;
INSERT INTO `rlp_content_types` VALUES (1,'Article','com_content.article','{\"special\":{\"dbtable\":\"#__content\",\"key\":\"id\",\"type\":\"Content\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"state\",\"core_alias\":\"alias\",\"core_created_time\":\"created\",\"core_modified_time\":\"modified\",\"core_body\":\"introtext\", \"core_hits\":\"hits\",\"core_publish_up\":\"publish_up\",\"core_publish_down\":\"publish_down\",\"core_access\":\"access\", \"core_params\":\"attribs\", \"core_featured\":\"featured\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"images\", \"core_urls\":\"urls\", \"core_version\":\"version\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"catid\", \"core_xreference\":\"xreference\", \"asset_id\":\"asset_id\", \"note\":\"note\"}, \"special\":{\"fulltext\":\"fulltext\"}}','ContentHelperRoute::getArticleRoute','{\"formFile\":\"administrator\\/components\\/com_content\\/models\\/forms\\/article.xml\", \"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\"],\"ignoreChanges\":[\"modified_by\", \"modified\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"ordering\"],\"convertToInt\":[\"publish_up\", \"publish_down\", \"featured\", \"ordering\"],\"displayLookup\":[{\"sourceColumn\":\"catid\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"created_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"} ]}'),(2,'Contact','com_contact.contact','{\"special\":{\"dbtable\":\"#__contact_details\",\"key\":\"id\",\"type\":\"Contact\",\"prefix\":\"ContactTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"name\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created\",\"core_modified_time\":\"modified\",\"core_body\":\"address\", \"core_hits\":\"hits\",\"core_publish_up\":\"publish_up\",\"core_publish_down\":\"publish_down\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"featured\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"image\", \"core_urls\":\"webpage\", \"core_version\":\"version\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"catid\", \"core_xreference\":\"xreference\", \"asset_id\":\"null\"}, \"special\":{\"con_position\":\"con_position\",\"suburb\":\"suburb\",\"state\":\"state\",\"country\":\"country\",\"postcode\":\"postcode\",\"telephone\":\"telephone\",\"fax\":\"fax\",\"misc\":\"misc\",\"email_to\":\"email_to\",\"default_con\":\"default_con\",\"user_id\":\"user_id\",\"mobile\":\"mobile\",\"sortname1\":\"sortname1\",\"sortname2\":\"sortname2\",\"sortname3\":\"sortname3\"}}','ContactHelperRoute::getContactRoute','{\"formFile\":\"administrator\\/components\\/com_contact\\/models\\/forms\\/contact.xml\",\"hideFields\":[\"default_con\",\"checked_out\",\"checked_out_time\",\"version\",\"xreference\"],\"ignoreChanges\":[\"modified_by\", \"modified\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"ordering\"],\"convertToInt\":[\"publish_up\", \"publish_down\", \"featured\", \"ordering\"], \"displayLookup\":[ {\"sourceColumn\":\"created_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"catid\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"} ] }'),(3,'Newsfeed','com_newsfeeds.newsfeed','{\"special\":{\"dbtable\":\"#__newsfeeds\",\"key\":\"id\",\"type\":\"Newsfeed\",\"prefix\":\"NewsfeedsTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"name\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created\",\"core_modified_time\":\"modified\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"publish_up\",\"core_publish_down\":\"publish_down\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"featured\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"images\", \"core_urls\":\"link\", \"core_version\":\"version\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"catid\", \"core_xreference\":\"xreference\", \"asset_id\":\"null\"}, \"special\":{\"numarticles\":\"numarticles\",\"cache_time\":\"cache_time\",\"rtl\":\"rtl\"}}','NewsfeedsHelperRoute::getNewsfeedRoute','{\"formFile\":\"administrator\\/components\\/com_newsfeeds\\/models\\/forms\\/newsfeed.xml\",\"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\"],\"ignoreChanges\":[\"modified_by\", \"modified\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"ordering\"],\"convertToInt\":[\"publish_up\", \"publish_down\", \"featured\", \"ordering\"],\"displayLookup\":[{\"sourceColumn\":\"catid\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"created_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"} ]}'),(4,'User','com_users.user','{\"special\":{\"dbtable\":\"#__users\",\"key\":\"id\",\"type\":\"User\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"name\",\"core_state\":\"null\",\"core_alias\":\"username\",\"core_created_time\":\"registerdate\",\"core_modified_time\":\"lastvisitDate\",\"core_body\":\"null\", \"core_hits\":\"null\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"access\":\"null\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"null\", \"core_language\":\"null\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"null\", \"core_ordering\":\"null\", \"core_metakey\":\"null\", \"core_metadesc\":\"null\", \"core_catid\":\"null\", \"core_xreference\":\"null\", \"asset_id\":\"null\"}, \"special\":{}}','UsersHelperRoute::getUserRoute',''),(5,'Article Category','com_content.category','{\"special\":{\"dbtable\":\"#__categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\",\"extension\":\"extension\",\"note\":\"note\"}}','ContentHelperRoute::getCategoryRoute','{\"formFile\":\"administrator\\/components\\/com_categories\\/models\\/forms\\/category.xml\", \"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\",\"lft\",\"rgt\",\"level\",\"path\",\"extension\"], \"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"],\"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"parent_id\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}]}'),(6,'Contact Category','com_contact.category','{\"special\":{\"dbtable\":\"#__categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\",\"extension\":\"extension\",\"note\":\"note\"}}','ContactHelperRoute::getCategoryRoute','{\"formFile\":\"administrator\\/components\\/com_categories\\/models\\/forms\\/category.xml\", \"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\",\"lft\",\"rgt\",\"level\",\"path\",\"extension\"], \"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"],\"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"parent_id\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}]}'),(7,'Newsfeeds Category','com_newsfeeds.category','{\"special\":{\"dbtable\":\"#__categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\",\"extension\":\"extension\",\"note\":\"note\"}}','NewsfeedsHelperRoute::getCategoryRoute','{\"formFile\":\"administrator\\/components\\/com_categories\\/models\\/forms\\/category.xml\", \"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\",\"lft\",\"rgt\",\"level\",\"path\",\"extension\"], \"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"],\"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"parent_id\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}]}'),(8,'Tag','com_tags.tag','{\"special\":{\"dbtable\":\"#__tags\",\"key\":\"tag_id\",\"type\":\"Tag\",\"prefix\":\"TagsTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"featured\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"images\", \"core_urls\":\"urls\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"null\", \"core_xreference\":\"null\", \"asset_id\":\"null\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\"}}','TagsHelperRoute::getTagRoute','{\"formFile\":\"administrator\\/components\\/com_tags\\/models\\/forms\\/tag.xml\", \"hideFields\":[\"checked_out\",\"checked_out_time\",\"version\", \"lft\", \"rgt\", \"level\", \"path\", \"urls\", \"publish_up\", \"publish_down\"],\"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"],\"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}, {\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}, {\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}]}'),(9,'Banner','com_banners.banner','{\"special\":{\"dbtable\":\"#__banners\",\"key\":\"id\",\"type\":\"Banner\",\"prefix\":\"BannersTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"name\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created\",\"core_modified_time\":\"modified\",\"core_body\":\"description\", \"core_hits\":\"null\",\"core_publish_up\":\"publish_up\",\"core_publish_down\":\"publish_down\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"images\", \"core_urls\":\"link\", \"core_version\":\"version\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"catid\", \"core_xreference\":\"null\", \"asset_id\":\"null\"}, \"special\":{\"imptotal\":\"imptotal\", \"impmade\":\"impmade\", \"clicks\":\"clicks\", \"clickurl\":\"clickurl\", \"custombannercode\":\"custombannercode\", \"cid\":\"cid\", \"purchase_type\":\"purchase_type\", \"track_impressions\":\"track_impressions\", \"track_clicks\":\"track_clicks\"}}','','{\"formFile\":\"administrator\\/components\\/com_banners\\/models\\/forms\\/banner.xml\", \"hideFields\":[\"checked_out\",\"checked_out_time\",\"version\", \"reset\"],\"ignoreChanges\":[\"modified_by\", \"modified\", \"checked_out\", \"checked_out_time\", \"version\", \"imptotal\", \"impmade\", \"reset\"], \"convertToInt\":[\"publish_up\", \"publish_down\", \"ordering\"], \"displayLookup\":[{\"sourceColumn\":\"catid\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}, {\"sourceColumn\":\"cid\",\"targetTable\":\"#__banner_clients\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}, {\"sourceColumn\":\"created_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"modified_by\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"} ]}'),(10,'Banners Category','com_banners.category','{\"special\":{\"dbtable\":\"#__categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\": {\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\",\"extension\":\"extension\",\"note\":\"note\"}}','','{\"formFile\":\"administrator\\/components\\/com_categories\\/models\\/forms\\/category.xml\", \"hideFields\":[\"asset_id\",\"checked_out\",\"checked_out_time\",\"version\",\"lft\",\"rgt\",\"level\",\"path\",\"extension\"], \"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"], \"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"parent_id\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}]}'),(11,'Banner Client','com_banners.client','{\"special\":{\"dbtable\":\"#__banner_clients\",\"key\":\"id\",\"type\":\"Client\",\"prefix\":\"BannersTable\"}}','','','','{\"formFile\":\"administrator\\/components\\/com_banners\\/models\\/forms\\/client.xml\", \"hideFields\":[\"checked_out\",\"checked_out_time\"], \"ignoreChanges\":[\"checked_out\", \"checked_out_time\"], \"convertToInt\":[], \"displayLookup\":[]}'),(12,'User Notes','com_users.note','{\"special\":{\"dbtable\":\"#__user_notes\",\"key\":\"id\",\"type\":\"Note\",\"prefix\":\"UsersTable\"}}','','','','{\"formFile\":\"administrator\\/components\\/com_users\\/models\\/forms\\/note.xml\", \"hideFields\":[\"checked_out\",\"checked_out_time\", \"publish_up\", \"publish_down\"],\"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\"], \"convertToInt\":[\"publish_up\", \"publish_down\"],\"displayLookup\":[{\"sourceColumn\":\"catid\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}, {\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}, {\"sourceColumn\":\"user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}, {\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}]}'),(13,'User Notes Category','com_users.category','{\"special\":{\"dbtable\":\"#__categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Corecontent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"hits\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"metadata\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"version\", \"core_ordering\":\"null\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"path\",\"extension\":\"extension\",\"note\":\"note\"}}','','{\"formFile\":\"administrator\\/components\\/com_categories\\/models\\/forms\\/category.xml\", \"hideFields\":[\"checked_out\",\"checked_out_time\",\"version\",\"lft\",\"rgt\",\"level\",\"path\",\"extension\"], \"ignoreChanges\":[\"modified_user_id\", \"modified_time\", \"checked_out\", \"checked_out_time\", \"version\", \"hits\", \"path\"], \"convertToInt\":[\"publish_up\", \"publish_down\"], \"displayLookup\":[{\"sourceColumn\":\"created_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"}, {\"sourceColumn\":\"access\",\"targetTable\":\"#__viewlevels\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"},{\"sourceColumn\":\"modified_user_id\",\"targetTable\":\"#__users\",\"targetColumn\":\"id\",\"displayColumn\":\"name\"},{\"sourceColumn\":\"parent_id\",\"targetTable\":\"#__categories\",\"targetColumn\":\"id\",\"displayColumn\":\"title\"}]}'),(14,'Hikashop Product','com_hikashop.product','{\"special\":{\"dbtable\":\"#__hikashop_product\",\"key\":\"product_id\",\"type\":\"Hikashop Product\",\"prefix\":\"JTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"CoreContent\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":[{\"core_content_item_id\":\"product_id\",\"core_title\":\"product_name\",\"core_state\":\"product_published\",\"core_alias\":\"product_alias\",\"core_created_time\":\"product_created\",\"core_modified_time\":\"product_modified\",\"core_body\":\"product_description\",\"core_hits\":\"product_hit\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"null\",\"core_params\":\"null\",\"core_featured\":\"null\",\"core_metadata\":\"null\",\"core_language\":\"null\",\"core_images\":\"null\",\"core_urls\":\"null\",\"core_version\":\"null\",\"core_ordering\":\"null\",\"core_metakey\":\"product_keywords\",\"core_metadesc\":\"product_meta_description\",\"core_catid\":\"null\",\"core_xreference\":\"null\",\"asset_id\":\"null\"}],\"special\":[[]]}','hikashopTagRouteHelper::getProductRoute',NULL),(15,'jDownloads Download','com_jdownloads.download','{\"special\":{\"dbtable\":\"#__jdownloads_files\",\"key\":\"file_id\",\"type\":\"Download\",\"prefix\":\"JdownloadsTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Download\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"file_id\",\"core_title\":\"file_title\",\"core_state\":\"published\",\"core_alias\":\"file_alias\",\"core_created_time\":\"date_added\",\"core_modified_time\":\"modified_date\",\"core_body\":\"description\", \"core_hits\":\"views\",\"core_publish_up\":\"publish_from\",\"core_publish_down\":\"publish_to\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"featured\", \"core_metadata\":\"null\", \"core_language\":\"language\", \"core_images\":\"images\", \"core_urls\":\"null\", \"core_version\":\"null\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"cat_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"description_long\":\"description_long\"}}','JdownloadsHelperRoute::getDownloadRoute',NULL),(16,'jDownloads Category','com_jdownloads.category','{\"special\":{\"dbtable\":\"#__jdownloads_categories\",\"key\":\"id\",\"type\":\"Category\",\"prefix\":\"JdownloadsTable\",\"config\":\"array()\"},\"common\":{\"dbtable\":\"#__ucm_content\",\"key\":\"ucm_id\",\"type\":\"Category\",\"prefix\":\"JTable\",\"config\":\"array()\"}}','','{\"common\":{\"core_content_item_id\":\"id\",\"core_title\":\"title\",\"core_state\":\"published\",\"core_alias\":\"alias\",\"core_created_time\":\"created_time\",\"core_modified_time\":\"modified_time\",\"core_body\":\"description\", \"core_hits\":\"views\",\"core_publish_up\":\"null\",\"core_publish_down\":\"null\",\"core_access\":\"access\", \"core_params\":\"params\", \"core_featured\":\"null\", \"core_metadata\":\"null\", \"core_language\":\"language\", \"core_images\":\"null\", \"core_urls\":\"null\", \"core_version\":\"null\", \"core_ordering\":\"ordering\", \"core_metakey\":\"metakey\", \"core_metadesc\":\"metadesc\", \"core_catid\":\"parent_id\", \"core_xreference\":\"null\", \"asset_id\":\"asset_id\"}, \"special\":{\"parent_id\":\"parent_id\",\"lft\":\"lft\",\"rgt\":\"rgt\",\"level\":\"level\",\"path\":\"null\",\"extension\":\"null\",\"note\":\"null\"}}','JdownloadsHelperRoute::getCategoryRoute',NULL);
/*!40000 ALTER TABLE `rlp_content_types` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_contentitem_tag_map`
--

DROP TABLE IF EXISTS `rlp_contentitem_tag_map`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_contentitem_tag_map` (
  `type_alias` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `core_content_id` int unsigned NOT NULL COMMENT 'PK from the core content table',
  `content_item_id` int NOT NULL COMMENT 'PK from the content type table',
  `tag_id` int unsigned NOT NULL COMMENT 'PK from the tag table',
  `tag_date` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'Date of most recent save for this tag-item',
  `type_id` mediumint NOT NULL COMMENT 'PK from the content_type table',
  UNIQUE KEY `uc_ItemnameTagid` (`type_id`,`content_item_id`,`tag_id`),
  KEY `idx_tag_type` (`tag_id`,`type_id`),
  KEY `idx_date_id` (`tag_date`,`tag_id`),
  KEY `idx_core_content_id` (`core_content_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Maps items from content tables to tags';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_contentitem_tag_map`
--

LOCK TABLES `rlp_contentitem_tag_map` WRITE;
/*!40000 ALTER TABLE `rlp_contentitem_tag_map` DISABLE KEYS */;
INSERT INTO `rlp_contentitem_tag_map` VALUES ('com_content.article',1,54,5,'0000-00-00 00:00:00',1),('com_content.article',1,54,6,'0000-00-00 00:00:00',1),('com_content.article',29,71,7,'0000-00-00 00:00:00',1),('com_content.article',6,72,7,'0000-00-00 00:00:00',1),('com_content.article',5,74,7,'0000-00-00 00:00:00',1),('com_content.article',4,77,7,'0000-00-00 00:00:00',1),('com_content.article',3,78,7,'0000-00-00 00:00:00',1),('com_content.article',2,90,7,'0000-00-00 00:00:00',1),('com_content.article',8,98,7,'0000-00-00 00:00:00',1),('com_content.article',9,102,7,'0000-00-00 00:00:00',1),('com_content.article',10,103,7,'0000-00-00 00:00:00',1),('com_content.article',11,106,7,'0000-00-00 00:00:00',1),('com_content.article',12,116,7,'0000-00-00 00:00:00',1),('com_content.article',13,125,7,'0000-00-00 00:00:00',1),('com_content.article',14,127,5,'0000-00-00 00:00:00',1),('com_content.article',15,137,7,'0000-00-00 00:00:00',1),('com_content.article',30,142,7,'0000-00-00 00:00:00',1),('com_content.article',18,144,8,'0000-00-00 00:00:00',1),('com_content.article',17,145,7,'0000-00-00 00:00:00',1),('com_content.article',19,146,8,'0000-00-00 00:00:00',1),('com_content.article',20,147,7,'0000-00-00 00:00:00',1),('com_content.article',21,150,8,'0000-00-00 00:00:00',1),('com_content.article',22,151,9,'0000-00-00 00:00:00',1),('com_content.article',22,151,10,'0000-00-00 00:00:00',1),('com_content.article',22,151,11,'0000-00-00 00:00:00',1),('com_content.article',23,155,7,'0000-00-00 00:00:00',1),('com_content.article',25,161,7,'0000-00-00 00:00:00',1),('com_content.article',26,162,8,'0000-00-00 00:00:00',1),('com_content.article',27,163,7,'0000-00-00 00:00:00',1),('com_content.article',27,163,13,'0000-00-00 00:00:00',1),('com_content.article',28,164,7,'0000-00-00 00:00:00',1),('com_content.article',28,164,14,'0000-00-00 00:00:00',1),('com_content.article',31,165,9,'0000-00-00 00:00:00',1),('com_content.article',31,165,15,'0000-00-00 00:00:00',1),('com_content.article',32,166,7,'0000-00-00 00:00:00',1),('com_content.article',32,166,16,'0000-00-00 00:00:00',1),('com_content.article',33,167,5,'0000-00-00 00:00:00',1),('com_content.article',33,167,17,'0000-00-00 00:00:00',1),('com_content.article',34,169,7,'0000-00-00 00:00:00',1),('com_content.article',35,172,18,'0000-00-00 00:00:00',1),('com_content.article',35,172,19,'0000-00-00 00:00:00',1),('com_content.article',36,174,7,'0000-00-00 00:00:00',1),('com_content.article',36,174,20,'0000-00-00 00:00:00',1),('com_content.article',37,175,7,'0000-00-00 00:00:00',1),('com_content.article',37,175,21,'0000-00-00 00:00:00',1),('com_content.article',40,176,7,'0000-00-00 00:00:00',1),('com_content.article',38,177,8,'0000-00-00 00:00:00',1),('com_content.article',38,177,9,'0000-00-00 00:00:00',1),('com_content.article',39,178,7,'0000-00-00 00:00:00',1),('com_content.article',41,184,7,'0000-00-00 00:00:00',1),('com_content.article',42,194,7,'0000-00-00 00:00:00',1),('com_content.article',43,225,7,'0000-00-00 00:00:00',1),('com_content.article',44,230,7,'0000-00-00 00:00:00',1),('com_content.article',45,231,7,'0000-00-00 00:00:00',1),('com_content.article',46,238,7,'0000-00-00 00:00:00',1),('com_content.article',47,246,7,'0000-00-00 00:00:00',1),('com_content.article',48,251,7,'0000-00-00 00:00:00',1),('com_content.article',49,261,7,'0000-00-00 00:00:00',1);
/*!40000 ALTER TABLE `rlp_contentitem_tag_map` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_core_log_searches`
--

DROP TABLE IF EXISTS `rlp_core_log_searches`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_core_log_searches` (
  `search_term` varchar(128) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `hits` int unsigned NOT NULL DEFAULT '0'
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_core_log_searches`
--

LOCK TABLES `rlp_core_log_searches` WRITE;
/*!40000 ALTER TABLE `rlp_core_log_searches` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_core_log_searches` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_easyjoomlabackup`
--

DROP TABLE IF EXISTS `rlp_easyjoomlabackup`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_easyjoomlabackup` (
  `id` int NOT NULL AUTO_INCREMENT,
  `date` datetime NOT NULL,
  `comment` tinytext NOT NULL,
  `type` varchar(32) NOT NULL,
  `size` varchar(12) NOT NULL,
  `duration` varchar(8) NOT NULL,
  `name` tinytext NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `id` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_easyjoomlabackup`
--

LOCK TABLES `rlp_easyjoomlabackup` WRITE;
/*!40000 ALTER TABLE `rlp_easyjoomlabackup` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_easyjoomlabackup` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_extensions`
--

DROP TABLE IF EXISTS `rlp_extensions`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_extensions` (
  `extension_id` int NOT NULL AUTO_INCREMENT,
  `package_id` int NOT NULL DEFAULT '0' COMMENT 'Parent package ID for extensions installed as a package.',
  `name` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `type` varchar(20) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `element` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `folder` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `client_id` tinyint NOT NULL,
  `enabled` tinyint NOT NULL DEFAULT '0',
  `access` int unsigned NOT NULL DEFAULT '1',
  `protected` tinyint NOT NULL DEFAULT '0',
  `manifest_cache` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `params` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `custom_data` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `system_data` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `checked_out` int unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `ordering` int DEFAULT '0',
  `state` int DEFAULT '0',
  PRIMARY KEY (`extension_id`),
  KEY `element_clientid` (`element`,`client_id`),
  KEY `element_folder_clientid` (`element`,`folder`,`client_id`),
  KEY `extension` (`type`,`element`,`folder`,`client_id`)
) ENGINE=InnoDB AUTO_INCREMENT=10467 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_extensions`
--

LOCK TABLES `rlp_extensions` WRITE;
/*!40000 ALTER TABLE `rlp_extensions` DISABLE KEYS */;
INSERT INTO `rlp_extensions` VALUES (1,0,'com_mailto','component','com_mailto','',0,1,1,1,'{\"name\":\"com_mailto\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_MAILTO_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mailto\"}','','','',0,'0000-00-00 00:00:00',0,0),(2,0,'com_wrapper','component','com_wrapper','',0,1,1,1,'{\"name\":\"com_wrapper\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2007 Open Source Matters, Inc.\\n\\t\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_WRAPPER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"wrapper\"}','','','',0,'0000-00-00 00:00:00',0,0),(3,0,'com_admin','component','com_admin','',1,1,1,1,'{\"name\":\"com_admin\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_ADMIN_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(4,0,'com_banners','component','com_banners','',1,1,1,0,'{\"name\":\"com_banners\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_BANNERS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"banners\"}','{\"purchase_type\":\"3\",\"track_impressions\":\"0\",\"track_clicks\":\"0\",\"metakey_prefix\":\"\",\"save_history\":\"1\",\"history_limit\":10}','','',0,'0000-00-00 00:00:00',0,0),(5,0,'com_cache','component','com_cache','',1,1,1,1,'{\"name\":\"com_cache\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CACHE_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(6,0,'com_categories','component','com_categories','',1,1,1,1,'{\"name\":\"com_categories\",\"type\":\"component\",\"creationDate\":\"December 2007\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2007 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CATEGORIES_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(7,0,'com_checkin','component','com_checkin','',1,1,1,1,'{\"name\":\"com_checkin\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CHECKIN_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(8,0,'com_contact','component','com_contact','',1,1,1,0,'{\"name\":\"com_contact\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CONTACT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contact\"}','{\"show_contact_category\":\"hide\",\"save_history\":\"1\",\"history_limit\":10,\"show_contact_list\":\"0\",\"presentation_style\":\"sliders\",\"show_name\":\"1\",\"show_position\":\"1\",\"show_email\":\"0\",\"show_street_address\":\"1\",\"show_suburb\":\"1\",\"show_state\":\"1\",\"show_postcode\":\"1\",\"show_country\":\"1\",\"show_telephone\":\"1\",\"show_mobile\":\"1\",\"show_fax\":\"1\",\"show_webpage\":\"1\",\"show_misc\":\"1\",\"show_image\":\"1\",\"image\":\"\",\"allow_vcard\":\"0\",\"show_articles\":\"0\",\"show_profile\":\"0\",\"show_links\":\"0\",\"linka_name\":\"\",\"linkb_name\":\"\",\"linkc_name\":\"\",\"linkd_name\":\"\",\"linke_name\":\"\",\"contact_icons\":\"0\",\"icon_address\":\"\",\"icon_email\":\"\",\"icon_telephone\":\"\",\"icon_mobile\":\"\",\"icon_fax\":\"\",\"icon_misc\":\"\",\"show_headings\":\"1\",\"show_position_headings\":\"1\",\"show_email_headings\":\"0\",\"show_telephone_headings\":\"1\",\"show_mobile_headings\":\"0\",\"show_fax_headings\":\"0\",\"allow_vcard_headings\":\"0\",\"show_suburb_headings\":\"1\",\"show_state_headings\":\"1\",\"show_country_headings\":\"1\",\"show_email_form\":\"1\",\"show_email_copy\":\"1\",\"banned_email\":\"\",\"banned_subject\":\"\",\"banned_text\":\"\",\"validate_session\":\"1\",\"custom_reply\":\"0\",\"redirect\":\"\",\"show_category_crumb\":\"0\",\"metakey\":\"\",\"metadesc\":\"\",\"robots\":\"\",\"author\":\"\",\"rights\":\"\",\"xreference\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(9,0,'com_cpanel','component','com_cpanel','',1,1,1,1,'{\"name\":\"com_cpanel\",\"type\":\"component\",\"creationDate\":\"Jun 2007\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2007 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CPANEL_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(10,0,'com_installer','component','com_installer','',1,1,1,1,'{\"name\":\"com_installer\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_INSTALLER_XML_DESCRIPTION\",\"group\":\"\"}','{\"show_jed_info\":\"0\",\"cachetimeout\":\"6\",\"minimum_stability\":\"4\"}','','',0,'0000-00-00 00:00:00',0,0),(11,0,'com_languages','component','com_languages','',1,1,1,1,'{\"name\":\"com_languages\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_LANGUAGES_XML_DESCRIPTION\",\"group\":\"\"}','{\"administrator\":\"en-GB\",\"site\":\"en-GB\"}','','',0,'0000-00-00 00:00:00',0,0),(12,0,'com_login','component','com_login','',1,1,1,1,'{\"name\":\"com_login\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_LOGIN_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(13,0,'com_media','component','com_media','',1,1,0,1,'{\"name\":\"com_media\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_MEDIA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"media\"}','{\"upload_extensions\":\"bmp,csv,doc,gif,ico,jpg,jpeg,odg,odp,ods,odt,pdf,png,ppt,swf,txt,xcf,xls,BMP,CSV,DOC,GIF,ICO,JPG,JPEG,ODG,ODP,ODS,ODT,PDF,PNG,PPT,SWF,TXT,XCF,XLS\",\"upload_maxsize\":\"10\",\"file_path\":\"images\",\"image_path\":\"images\",\"restrict_uploads\":\"1\",\"allowed_media_usergroup\":\"3\",\"check_mime\":\"1\",\"image_extensions\":\"bmp,gif,jpg,png\",\"ignore_extensions\":\"\",\"upload_mime\":\"image\\/jpeg,image\\/gif,image\\/png,image\\/bmp,application\\/x-shockwave-flash,application\\/msword,application\\/excel,application\\/pdf,application\\/powerpoint,text\\/plain,application\\/x-zip\",\"upload_mime_illegal\":\"text\\/html\"}','','',0,'0000-00-00 00:00:00',0,0),(14,0,'com_menus','component','com_menus','',1,1,1,1,'{\"name\":\"com_menus\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_MENUS_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(15,0,'com_messages','component','com_messages','',1,1,1,1,'{\"name\":\"com_messages\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_MESSAGES_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(16,0,'com_modules','component','com_modules','',1,1,1,1,'{\"name\":\"com_modules\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_MODULES_XML_DESCRIPTION\",\"group\":\"\"}','{\"redirect_edit\":\"site\"}','','',0,'0000-00-00 00:00:00',0,0),(17,0,'com_newsfeeds','component','com_newsfeeds','',1,1,1,0,'{\"name\":\"com_newsfeeds\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_NEWSFEEDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"newsfeeds\"}','{\"newsfeed_layout\":\"_:default\",\"save_history\":\"1\",\"history_limit\":5,\"show_feed_image\":\"1\",\"show_feed_description\":\"1\",\"show_item_description\":\"1\",\"feed_character_count\":\"0\",\"feed_display_order\":\"des\",\"float_first\":\"right\",\"float_second\":\"right\",\"show_tags\":\"1\",\"category_layout\":\"_:default\",\"show_category_title\":\"1\",\"show_description\":\"1\",\"show_description_image\":\"1\",\"maxLevel\":\"-1\",\"show_empty_categories\":\"0\",\"show_subcat_desc\":\"1\",\"show_cat_items\":\"1\",\"show_cat_tags\":\"1\",\"show_base_description\":\"1\",\"maxLevelcat\":\"-1\",\"show_empty_categories_cat\":\"0\",\"show_subcat_desc_cat\":\"1\",\"show_cat_items_cat\":\"1\",\"filter_field\":\"1\",\"show_pagination_limit\":\"1\",\"show_headings\":\"1\",\"show_articles\":\"0\",\"show_link\":\"1\",\"show_pagination\":\"1\",\"show_pagination_results\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(18,0,'com_plugins','component','com_plugins','',1,1,1,1,'{\"name\":\"com_plugins\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_PLUGINS_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(19,0,'com_search','component','com_search','',1,1,1,0,'{\"name\":\"com_search\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_SEARCH_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"search\"}','{\"enabled\":\"0\",\"search_phrases\":\"0\",\"search_areas\":\"0\",\"show_date\":\"0\",\"opensearch_name\":\"\",\"opensearch_description\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(20,0,'com_templates','component','com_templates','',1,1,1,1,'{\"name\":\"com_templates\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_TEMPLATES_XML_DESCRIPTION\",\"group\":\"\"}','{\"template_positions_display\":\"0\",\"upload_limit\":\"10\",\"image_formats\":\"gif,bmp,jpg,jpeg,png\",\"source_formats\":\"txt,less,ini,xml,js,php,css\",\"font_formats\":\"woff,ttf,otf\",\"compressed_formats\":\"zip\"}','','',0,'0000-00-00 00:00:00',0,0),(22,0,'com_content','component','com_content','',1,1,0,1,'{\"name\":\"com_content\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CONTENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"content\"}','{\"article_layout\":\"_:default\",\"show_title\":\"1\",\"link_titles\":\"1\",\"show_intro\":\"1\",\"info_block_position\":\"0\",\"info_block_show_title\":\"1\",\"show_category\":\"0\",\"link_category\":\"0\",\"show_parent_category\":\"0\",\"link_parent_category\":\"0\",\"show_associations\":\"0\",\"flags\":\"1\",\"show_author\":\"0\",\"link_author\":\"1\",\"show_create_date\":\"0\",\"show_modify_date\":\"0\",\"show_publish_date\":\"0\",\"show_item_navigation\":\"0\",\"show_vote\":\"0\",\"show_readmore\":\"0\",\"show_readmore_title\":\"0\",\"readmore_limit\":\"100\",\"show_tags\":\"0\",\"show_icons\":\"0\",\"show_print_icon\":\"0\",\"show_email_icon\":\"0\",\"show_hits\":\"0\",\"show_noauth\":\"0\",\"urls_position\":\"0\",\"captcha\":\"0\",\"show_publishing_options\":\"1\",\"show_article_options\":\"1\",\"save_history\":\"1\",\"history_limit\":10,\"show_urls_images_frontend\":\"0\",\"show_urls_images_backend\":\"1\",\"targeta\":0,\"targetb\":0,\"targetc\":0,\"float_intro\":\"none\",\"float_fulltext\":\"none\",\"category_layout\":\"_:blog\",\"show_category_heading_title_text\":\"0\",\"show_category_title\":\"0\",\"show_description\":\"0\",\"show_description_image\":\"0\",\"maxLevel\":\"0\",\"show_empty_categories\":\"0\",\"show_no_articles\":\"0\",\"show_subcat_desc\":\"0\",\"show_cat_num_articles\":\"0\",\"show_cat_tags\":\"0\",\"show_base_description\":\"0\",\"maxLevelcat\":\"-1\",\"show_empty_categories_cat\":\"0\",\"show_subcat_desc_cat\":\"0\",\"show_cat_num_articles_cat\":\"0\",\"num_leading_articles\":\"0\",\"num_intro_articles\":\"1\",\"num_columns\":\"1\",\"num_links\":\"4\",\"multi_column_order\":\"0\",\"show_subcategory_content\":\"0\",\"show_pagination_limit\":\"1\",\"filter_field\":\"hide\",\"show_headings\":\"1\",\"list_show_date\":\"0\",\"date_format\":\"\",\"list_show_hits\":\"0\",\"list_show_author\":\"0\",\"list_show_votes\":\"0\",\"list_show_ratings\":\"0\",\"orderby_pri\":\"none\",\"orderby_sec\":\"order\",\"order_date\":\"published\",\"show_pagination\":\"2\",\"show_pagination_results\":\"0\",\"show_featured\":\"hide\",\"show_feed_link\":\"1\",\"feed_summary\":\"0\",\"feed_show_readmore\":\"0\",\"sef_advanced\":0,\"sef_ids\":0,\"custom_fields_enable\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(23,0,'com_config','component','com_config','',1,1,0,1,'{\"name\":\"com_config\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_CONFIG_XML_DESCRIPTION\",\"group\":\"\"}','{\"filters\":{\"1\":{\"filter_type\":\"NH\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"9\":{\"filter_type\":\"BL\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"6\":{\"filter_type\":\"BL\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"7\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"2\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"3\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"4\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"11\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"5\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"10\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"},\"8\":{\"filter_type\":\"NONE\",\"filter_tags\":\"\",\"filter_attributes\":\"\"}}}','','',0,'0000-00-00 00:00:00',0,0),(24,0,'com_redirect','component','com_redirect','',1,1,0,1,'{\"name\":\"com_redirect\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_REDIRECT_XML_DESCRIPTION\",\"group\":\"\"}','{\"mode\":\"1\",\"separator\":\"|\",\"defaultImportState\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(25,0,'com_users','component','com_users','',1,1,0,1,'{\"name\":\"com_users\",\"type\":\"component\",\"creationDate\":\"April 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_USERS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"users\"}','{\"allowUserRegistration\":\"0\",\"new_usertype\":\"2\",\"guest_usergroup\":\"9\",\"sendpassword\":\"1\",\"useractivation\":\"1\",\"mail_to_admin\":\"0\",\"captcha\":\"\",\"frontend_userparams\":\"1\",\"site_language\":\"0\",\"change_login_name\":\"0\",\"reset_count\":\"10\",\"reset_time\":\"1\",\"minimum_length\":\"8\",\"minimum_integers\":\"1\",\"minimum_symbols\":\"1\",\"minimum_uppercase\":\"1\",\"save_history\":\"1\",\"history_limit\":5,\"mailSubjectPrefix\":\"\",\"mailBodySuffix\":\"\",\"debugUsers\":\"1\",\"debugGroups\":\"1\",\"custom_fields_enable\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(27,0,'com_finder','component','com_finder','',1,1,0,0,'{\"name\":\"com_finder\",\"type\":\"component\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"COM_FINDER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"finder\"}','{\"enabled\":\"0\",\"show_description\":\"1\",\"description_length\":255,\"allow_empty_query\":\"0\",\"show_url\":\"0\",\"show_autosuggest\":\"1\",\"show_suggested_query\":\"1\",\"show_explained_query\":\"0\",\"show_advanced\":\"0\",\"show_advanced_tips\":\"1\",\"expand_advanced\":\"0\",\"show_date_filters\":\"1\",\"sort_order\":\"relevance\",\"sort_direction\":\"desc\",\"highlight_terms\":\"1\",\"opensearch_name\":\"\",\"opensearch_description\":\"\",\"batch_size\":\"300\",\"memory_table_limit\":20000,\"title_multiplier\":\"1.7\",\"text_multiplier\":\"0.7\",\"meta_multiplier\":\"1.2\",\"path_multiplier\":\"2.0\",\"misc_multiplier\":\"0.3\",\"stem\":\"1\",\"stemmer\":\"snowball\",\"enable_logging\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(28,0,'com_joomlaupdate','component','com_joomlaupdate','',1,1,0,1,'{\"name\":\"com_joomlaupdate\",\"type\":\"component\",\"creationDate\":\"February 2012\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2012 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.10.1\",\"description\":\"COM_JOOMLAUPDATE_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(29,0,'com_tags','component','com_tags','',1,1,1,1,'{\"name\":\"com_tags\",\"type\":\"component\",\"creationDate\":\"December 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2013 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.1.0\",\"description\":\"COM_TAGS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"tags\"}','{\"tag_layout\":\"_:default\",\"save_history\":\"1\",\"history_limit\":5,\"show_tag_title\":\"0\",\"tag_list_show_tag_image\":\"0\",\"tag_list_show_tag_description\":\"0\",\"tag_list_image\":\"\",\"show_tag_num_items\":\"0\",\"tag_list_orderby\":\"title\",\"tag_list_orderby_direction\":\"ASC\",\"show_headings\":\"0\",\"tag_list_show_date\":\"0\",\"tag_list_show_item_image\":\"0\",\"tag_list_show_item_description\":\"0\",\"tag_list_item_maximum_characters\":0,\"return_any_or_all\":\"1\",\"include_children\":\"0\",\"maximum\":200,\"tag_list_language_filter\":\"all\",\"tags_layout\":\"_:default\",\"all_tags_orderby\":\"title\",\"all_tags_orderby_direction\":\"ASC\",\"all_tags_show_tag_image\":\"0\",\"all_tags_show_tag_descripion\":\"0\",\"all_tags_tag_maximum_characters\":20,\"all_tags_show_tag_hits\":\"0\",\"filter_field\":\"1\",\"show_pagination_limit\":\"1\",\"show_pagination\":\"2\",\"show_pagination_results\":\"1\",\"tag_field_ajax_mode\":\"1\",\"show_feed_link\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(30,0,'com_contenthistory','component','com_contenthistory','',1,1,1,0,'{\"name\":\"com_contenthistory\",\"type\":\"component\",\"creationDate\":\"May 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2013 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.0\",\"description\":\"COM_CONTENTHISTORY_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contenthistory\"}','','','',0,'0000-00-00 00:00:00',0,0),(31,0,'com_ajax','component','com_ajax','',1,1,1,1,'{\"name\":\"com_ajax\",\"type\":\"component\",\"creationDate\":\"August 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2013 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.0\",\"description\":\"COM_AJAX_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"ajax\"}','','','',0,'0000-00-00 00:00:00',0,0),(32,0,'com_postinstall','component','com_postinstall','',1,1,1,1,'{\"name\":\"com_postinstall\",\"type\":\"component\",\"creationDate\":\"September 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2013 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.0\",\"description\":\"COM_POSTINSTALL_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(33,0,'com_fields','component','com_fields','',1,1,1,0,'{\"name\":\"com_fields\",\"type\":\"component\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"COM_FIELDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"fields\"}','','','',0,'0000-00-00 00:00:00',0,0),(34,0,'com_associations','component','com_associations','',1,1,1,0,'{\"name\":\"com_associations\",\"type\":\"component\",\"creationDate\":\"January 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2017 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"COM_ASSOCIATIONS_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(35,0,'com_privacy','component','com_privacy','',1,1,1,1,'{\"name\":\"com_privacy\",\"type\":\"component\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"COM_PRIVACY_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"privacy\"}','','','',0,'0000-00-00 00:00:00',0,0),(36,0,'com_actionlogs','component','com_actionlogs','',1,1,1,1,'{\"name\":\"com_actionlogs\",\"type\":\"component\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"COM_ACTIONLOGS_XML_DESCRIPTION\",\"group\":\"\"}','{\"ip_logging\":0,\"csv_delimiter\":\",\",\"loggable_extensions\":[\"com_banners\",\"com_cache\",\"com_categories\",\"com_checkin\",\"com_config\",\"com_contact\",\"com_content\",\"com_installer\",\"com_media\",\"com_menus\",\"com_messages\",\"com_modules\",\"com_newsfeeds\",\"com_plugins\",\"com_redirect\",\"com_tags\",\"com_templates\",\"com_users\"]}','','',0,'0000-00-00 00:00:00',0,0),(102,0,'LIB_PHPUTF8','library','phputf8','',0,1,1,1,'{\"name\":\"LIB_PHPUTF8\",\"type\":\"library\",\"creationDate\":\"2006\",\"author\":\"Harry Fuecks\",\"copyright\":\"Copyright various authors\",\"authorEmail\":\"hfuecks@gmail.com\",\"authorUrl\":\"http:\\/\\/sourceforge.net\\/projects\\/phputf8\",\"version\":\"0.5\",\"description\":\"LIB_PHPUTF8_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"phputf8\"}','','','',0,'0000-00-00 00:00:00',0,0),(103,0,'LIB_JOOMLA','library','joomla','',0,1,1,1,'{\"name\":\"LIB_JOOMLA\",\"type\":\"library\",\"creationDate\":\"2008\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2008 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"https:\\/\\/www.joomla.org\",\"version\":\"13.1\",\"description\":\"LIB_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','{\"mediaversion\":\"a5dda60b046125a413aacce660618d49\"}','','',0,'0000-00-00 00:00:00',0,0),(104,0,'LIB_IDNA','library','idna_convert','',0,1,1,1,'{\"name\":\"LIB_IDNA\",\"type\":\"library\",\"creationDate\":\"2004\",\"author\":\"phlyLabs\",\"copyright\":\"2004-2011 phlyLabs Berlin, http:\\/\\/phlylabs.de\",\"authorEmail\":\"phlymail@phlylabs.de\",\"authorUrl\":\"http:\\/\\/phlylabs.de\",\"version\":\"0.8.0\",\"description\":\"LIB_IDNA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"idna_convert\"}','','','',0,'0000-00-00 00:00:00',0,0),(105,0,'FOF','library','fof','',0,1,1,1,'{\"name\":\"FOF\",\"type\":\"library\",\"creationDate\":\"2015-04-22 13:15:32\",\"author\":\"Nicholas K. Dionysopoulos \\/ Akeeba Ltd\",\"copyright\":\"(C)2011-2015 Nicholas K. Dionysopoulos\",\"authorEmail\":\"nicholas@akeebabackup.com\",\"authorUrl\":\"https:\\/\\/www.akeebabackup.com\",\"version\":\"2.4.3\",\"description\":\"LIB_FOF_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"fof\"}','','','',0,'0000-00-00 00:00:00',0,0),(106,0,'LIB_PHPASS','library','phpass','',0,1,1,1,'{\"name\":\"LIB_PHPASS\",\"type\":\"library\",\"creationDate\":\"2004-2006\",\"author\":\"Solar Designer\",\"copyright\":\"\",\"authorEmail\":\"solar@openwall.com\",\"authorUrl\":\"http:\\/\\/www.openwall.com\\/phpass\\/\",\"version\":\"0.3\",\"description\":\"LIB_PHPASS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"phpass\"}','','','',0,'0000-00-00 00:00:00',0,0),(200,0,'mod_articles_archive','module','mod_articles_archive','',0,1,1,0,'{\"name\":\"mod_articles_archive\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_ARTICLES_ARCHIVE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_archive\"}','','','',0,'0000-00-00 00:00:00',0,0),(201,0,'mod_articles_latest','module','mod_articles_latest','',0,1,1,0,'{\"name\":\"mod_articles_latest\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_LATEST_NEWS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_latest\"}','','','',0,'0000-00-00 00:00:00',0,0),(202,0,'mod_articles_popular','module','mod_articles_popular','',0,1,1,0,'{\"name\":\"mod_articles_popular\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_POPULAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_popular\"}','','','',0,'0000-00-00 00:00:00',0,0),(203,0,'mod_banners','module','mod_banners','',0,1,1,0,'{\"name\":\"mod_banners\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_BANNERS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_banners\"}','','','',0,'0000-00-00 00:00:00',0,0),(204,0,'mod_breadcrumbs','module','mod_breadcrumbs','',0,1,1,1,'{\"name\":\"mod_breadcrumbs\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_BREADCRUMBS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_breadcrumbs\"}','','','',0,'0000-00-00 00:00:00',0,0),(205,0,'mod_custom','module','mod_custom','',0,1,1,1,'{\"name\":\"mod_custom\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_CUSTOM_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_custom\"}','','','',0,'0000-00-00 00:00:00',0,0),(206,0,'mod_feed','module','mod_feed','',0,1,1,0,'{\"name\":\"mod_feed\",\"type\":\"module\",\"creationDate\":\"July 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_FEED_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_feed\"}','','','',0,'0000-00-00 00:00:00',0,0),(207,0,'mod_footer','module','mod_footer','',0,1,1,0,'{\"name\":\"mod_footer\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_FOOTER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_footer\"}','','','',0,'0000-00-00 00:00:00',0,0),(208,0,'mod_login','module','mod_login','',0,1,1,1,'{\"name\":\"mod_login\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_LOGIN_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_login\"}','','','',0,'0000-00-00 00:00:00',0,0),(209,0,'mod_menu','module','mod_menu','',0,1,1,1,'{\"name\":\"mod_menu\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_MENU_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_menu\"}','','','',0,'0000-00-00 00:00:00',0,0),(210,0,'mod_articles_news','module','mod_articles_news','',0,1,1,0,'{\"name\":\"mod_articles_news\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_ARTICLES_NEWS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_news\"}','','','',0,'0000-00-00 00:00:00',0,0),(211,0,'mod_random_image','module','mod_random_image','',0,1,1,0,'{\"name\":\"mod_random_image\",\"type\":\"module\",\"creationDate\":\"July 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_RANDOM_IMAGE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_random_image\"}','','','',0,'0000-00-00 00:00:00',0,0),(212,0,'mod_related_items','module','mod_related_items','',0,1,1,0,'{\"name\":\"mod_related_items\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_RELATED_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_related_items\"}','','','',0,'0000-00-00 00:00:00',0,0),(213,0,'mod_search','module','mod_search','',0,1,1,0,'{\"name\":\"mod_search\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_SEARCH_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_search\"}','','','',0,'0000-00-00 00:00:00',0,0),(214,0,'mod_stats','module','mod_stats','',0,1,1,0,'{\"name\":\"mod_stats\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_STATS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_stats\"}','','','',0,'0000-00-00 00:00:00',0,0),(215,0,'mod_syndicate','module','mod_syndicate','',0,1,1,1,'{\"name\":\"mod_syndicate\",\"type\":\"module\",\"creationDate\":\"May 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_SYNDICATE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_syndicate\"}','','','',0,'0000-00-00 00:00:00',0,0),(216,0,'mod_users_latest','module','mod_users_latest','',0,1,1,0,'{\"name\":\"mod_users_latest\",\"type\":\"module\",\"creationDate\":\"December 2009\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2009 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_USERS_LATEST_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_users_latest\"}','','','',0,'0000-00-00 00:00:00',0,0),(218,0,'mod_whosonline','module','mod_whosonline','',0,1,1,0,'{\"name\":\"mod_whosonline\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_WHOSONLINE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_whosonline\"}','','','',0,'0000-00-00 00:00:00',0,0),(219,0,'mod_wrapper','module','mod_wrapper','',0,1,1,0,'{\"name\":\"mod_wrapper\",\"type\":\"module\",\"creationDate\":\"October 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_WRAPPER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_wrapper\"}','','','',0,'0000-00-00 00:00:00',0,0),(220,0,'mod_articles_category','module','mod_articles_category','',0,1,1,0,'{\"name\":\"mod_articles_category\",\"type\":\"module\",\"creationDate\":\"February 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2010 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_ARTICLES_CATEGORY_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_category\"}','','','',0,'0000-00-00 00:00:00',0,0),(221,0,'mod_articles_categories','module','mod_articles_categories','',0,1,1,0,'{\"name\":\"mod_articles_categories\",\"type\":\"module\",\"creationDate\":\"February 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2010 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_ARTICLES_CATEGORIES_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_articles_categories\"}','','','',0,'0000-00-00 00:00:00',0,0),(222,0,'mod_languages','module','mod_languages','',0,1,1,1,'{\"name\":\"mod_languages\",\"type\":\"module\",\"creationDate\":\"February 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2010 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.5.0\",\"description\":\"MOD_LANGUAGES_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_languages\"}','','','',0,'0000-00-00 00:00:00',0,0),(223,0,'mod_finder','module','mod_finder','',0,1,0,0,'{\"name\":\"mod_finder\",\"type\":\"module\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_FINDER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_finder\"}','','','',0,'0000-00-00 00:00:00',0,0),(300,0,'mod_custom','module','mod_custom','',1,1,1,1,'{\"name\":\"mod_custom\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_CUSTOM_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_custom\"}','','','',0,'0000-00-00 00:00:00',0,0),(301,0,'mod_feed','module','mod_feed','',1,1,1,0,'{\"name\":\"mod_feed\",\"type\":\"module\",\"creationDate\":\"July 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_FEED_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_feed\"}','','','',0,'0000-00-00 00:00:00',0,0),(302,0,'mod_latest','module','mod_latest','',1,1,1,0,'{\"name\":\"mod_latest\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_LATEST_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_latest\"}','','','',0,'0000-00-00 00:00:00',0,0),(303,0,'mod_logged','module','mod_logged','',1,1,1,0,'{\"name\":\"mod_logged\",\"type\":\"module\",\"creationDate\":\"January 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_LOGGED_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_logged\"}','','','',0,'0000-00-00 00:00:00',0,0),(304,0,'mod_login','module','mod_login','',1,1,1,1,'{\"name\":\"mod_login\",\"type\":\"module\",\"creationDate\":\"March 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_LOGIN_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_login\"}','','','',0,'0000-00-00 00:00:00',0,0),(305,0,'mod_menu','module','mod_menu','',1,1,1,0,'{\"name\":\"mod_menu\",\"type\":\"module\",\"creationDate\":\"March 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_MENU_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_menu\"}','','','',0,'0000-00-00 00:00:00',0,0),(307,0,'mod_popular','module','mod_popular','',1,1,1,0,'{\"name\":\"mod_popular\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_POPULAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_popular\"}','','','',0,'0000-00-00 00:00:00',0,0),(308,0,'mod_quickicon','module','mod_quickicon','',1,1,1,1,'{\"name\":\"mod_quickicon\",\"type\":\"module\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_QUICKICON_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_quickicon\"}','','','',0,'0000-00-00 00:00:00',0,0),(309,0,'mod_status','module','mod_status','',1,1,1,0,'{\"name\":\"mod_status\",\"type\":\"module\",\"creationDate\":\"February 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_STATUS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_status\"}','','','',0,'0000-00-00 00:00:00',0,0),(310,0,'mod_submenu','module','mod_submenu','',1,1,1,0,'{\"name\":\"mod_submenu\",\"type\":\"module\",\"creationDate\":\"February 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_SUBMENU_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_submenu\"}','','','',0,'0000-00-00 00:00:00',0,0),(311,0,'mod_title','module','mod_title','',1,1,1,0,'{\"name\":\"mod_title\",\"type\":\"module\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_TITLE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_title\"}','','','',0,'0000-00-00 00:00:00',0,0),(312,0,'mod_toolbar','module','mod_toolbar','',1,1,1,1,'{\"name\":\"mod_toolbar\",\"type\":\"module\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_TOOLBAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_toolbar\"}','','','',0,'0000-00-00 00:00:00',0,0),(313,0,'mod_multilangstatus','module','mod_multilangstatus','',1,1,1,0,'{\"name\":\"mod_multilangstatus\",\"type\":\"module\",\"creationDate\":\"September 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_MULTILANGSTATUS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_multilangstatus\"}','{\"cache\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(314,0,'mod_version','module','mod_version','',1,1,1,0,'{\"name\":\"mod_version\",\"type\":\"module\",\"creationDate\":\"January 2012\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2012 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_VERSION_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_version\"}','{\"format\":\"short\",\"product\":\"1\",\"cache\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(315,0,'mod_stats_admin','module','mod_stats_admin','',1,1,1,0,'{\"name\":\"mod_stats_admin\",\"type\":\"module\",\"creationDate\":\"July 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"MOD_STATS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_stats_admin\"}','{\"serverinfo\":\"0\",\"siteinfo\":\"0\",\"counter\":\"0\",\"increase\":\"0\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\"}','','',0,'0000-00-00 00:00:00',0,0),(316,0,'mod_tags_popular','module','mod_tags_popular','',0,1,1,0,'{\"name\":\"mod_tags_popular\",\"type\":\"module\",\"creationDate\":\"January 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2013 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.1.0\",\"description\":\"MOD_TAGS_POPULAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_tags_popular\"}','{\"maximum\":\"5\",\"timeframe\":\"alltime\",\"owncache\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(317,0,'mod_tags_similar','module','mod_tags_similar','',0,1,1,0,'{\"name\":\"mod_tags_similar\",\"type\":\"module\",\"creationDate\":\"January 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2013 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.1.0\",\"description\":\"MOD_TAGS_SIMILAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_tags_similar\"}','{\"maximum\":\"5\",\"matchtype\":\"any\",\"owncache\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(318,0,'mod_sampledata','module','mod_sampledata','',1,1,1,0,'{\"name\":\"mod_sampledata\",\"type\":\"module\",\"creationDate\":\"July 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2017 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.8.0\",\"description\":\"MOD_SAMPLEDATA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_sampledata\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(319,0,'mod_latestactions','module','mod_latestactions','',1,1,1,0,'{\"name\":\"mod_latestactions\",\"type\":\"module\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"MOD_LATESTACTIONS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_latestactions\"}','{}','','',0,'1970-01-01 00:00:00',0,0),(320,0,'mod_privacy_dashboard','module','mod_privacy_dashboard','',1,1,1,0,'{\"name\":\"mod_privacy_dashboard\",\"type\":\"module\",\"creationDate\":\"June 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"MOD_PRIVACY_DASHBOARD_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_privacy_dashboard\"}','{}','','',0,'1970-01-01 00:00:00',0,0),(400,0,'plg_authentication_gmail','plugin','gmail','authentication',0,0,1,0,'{\"name\":\"plg_authentication_gmail\",\"type\":\"plugin\",\"creationDate\":\"February 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_GMAIL_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"gmail\"}','{\"applysuffix\":\"0\",\"suffix\":\"\",\"verifypeer\":\"1\",\"user_blacklist\":\"\"}','','',0,'0000-00-00 00:00:00',1,0),(401,0,'plg_authentication_joomla','plugin','joomla','authentication',0,1,1,1,'{\"name\":\"plg_authentication_joomla\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_AUTH_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','','','',0,'0000-00-00 00:00:00',0,0),(402,0,'plg_authentication_ldap','plugin','ldap','authentication',0,0,1,0,'{\"name\":\"plg_authentication_ldap\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_LDAP_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"ldap\"}','{\"host\":\"\",\"port\":\"389\",\"use_ldapV3\":\"0\",\"negotiate_tls\":\"0\",\"no_referrals\":\"0\",\"auth_method\":\"bind\",\"base_dn\":\"\",\"search_string\":\"\",\"users_dn\":\"\",\"username\":\"admin\",\"password\":\"bobby7\",\"ldap_fullname\":\"fullName\",\"ldap_email\":\"mail\",\"ldap_uid\":\"uid\"}','','',0,'0000-00-00 00:00:00',3,0),(403,0,'plg_content_contact','plugin','contact','content',0,1,1,0,'{\"name\":\"plg_content_contact\",\"type\":\"plugin\",\"creationDate\":\"January 2014\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2014 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.2\",\"description\":\"PLG_CONTENT_CONTACT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contact\"}','','','',0,'0000-00-00 00:00:00',1,0),(404,0,'plg_content_emailcloak','plugin','emailcloak','content',0,1,1,0,'{\"name\":\"plg_content_emailcloak\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CONTENT_EMAILCLOAK_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"emailcloak\"}','{\"mode\":\"1\"}','','',0,'0000-00-00 00:00:00',1,0),(406,0,'plg_content_loadmodule','plugin','loadmodule','content',0,1,1,0,'{\"name\":\"plg_content_loadmodule\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_LOADMODULE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"loadmodule\"}','{\"style\":\"xhtml\"}','','',0,'0000-00-00 00:00:00',0,0),(407,0,'plg_content_pagebreak','plugin','pagebreak','content',0,1,1,0,'{\"name\":\"plg_content_pagebreak\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CONTENT_PAGEBREAK_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"pagebreak\"}','{\"title\":\"1\",\"article_index\":\"1\",\"article_index_text\":\"\",\"multipage_toc\":\"1\",\"showall\":\"1\",\"style\":\"sliders\"}','','',0,'0000-00-00 00:00:00',4,0),(408,0,'plg_content_pagenavigation','plugin','pagenavigation','content',0,1,1,0,'{\"name\":\"plg_content_pagenavigation\",\"type\":\"plugin\",\"creationDate\":\"January 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_PAGENAVIGATION_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"pagenavigation\"}','{\"position\":\"1\"}','','',0,'0000-00-00 00:00:00',5,0),(409,0,'plg_content_vote','plugin','vote','content',0,1,1,0,'{\"name\":\"plg_content_vote\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_VOTE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"vote\"}','','','',0,'0000-00-00 00:00:00',6,0),(410,0,'plg_editors_codemirror','plugin','codemirror','editors',0,1,1,1,'{\"name\":\"plg_editors_codemirror\",\"type\":\"plugin\",\"creationDate\":\"28 March 2011\",\"author\":\"Marijn Haverbeke\",\"copyright\":\"Copyright (C) 2014 - 2021 by Marijn Haverbeke <marijnh@gmail.com> and others\",\"authorEmail\":\"marijnh@gmail.com\",\"authorUrl\":\"https:\\/\\/codemirror.net\\/\",\"version\":\"5.60.0\",\"description\":\"PLG_CODEMIRROR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"codemirror\"}','{\"lineNumbers\":\"1\",\"lineWrapping\":\"1\",\"matchTags\":\"1\",\"matchBrackets\":\"1\",\"marker-gutter\":\"1\",\"autoCloseTags\":\"1\",\"autoCloseBrackets\":\"1\",\"autoFocus\":\"1\",\"theme\":\"default\",\"tabmode\":\"indent\"}','','',0,'0000-00-00 00:00:00',1,0),(411,0,'plg_editors_none','plugin','none','editors',0,1,1,1,'{\"name\":\"plg_editors_none\",\"type\":\"plugin\",\"creationDate\":\"September 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_NONE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"none\"}','','','',0,'0000-00-00 00:00:00',2,0),(412,0,'plg_editors_tinymce','plugin','tinymce','editors',0,1,1,0,'{\"name\":\"plg_editors_tinymce\",\"type\":\"plugin\",\"creationDate\":\"2005-2020\",\"author\":\"Tiny Technologies, Inc\",\"copyright\":\"Tiny Technologies, Inc\",\"authorEmail\":\"N\\/A\",\"authorUrl\":\"https:\\/\\/www.tiny.cloud\",\"version\":\"4.5.12\",\"description\":\"PLG_TINY_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"tinymce\"}','{\"configuration\":{\"toolbars\":{\"2\":{\"toolbar1\":[\"bold\",\"underline\",\"strikethrough\",\"|\",\"undo\",\"redo\",\"|\",\"bullist\",\"numlist\",\"|\",\"pastetext\"]},\"1\":{\"menu\":[\"edit\",\"insert\",\"view\",\"format\",\"table\",\"tools\"],\"toolbar1\":[\"bold\",\"italic\",\"underline\",\"strikethrough\",\"|\",\"alignleft\",\"aligncenter\",\"alignright\",\"alignjustify\",\"|\",\"formatselect\",\"|\",\"bullist\",\"numlist\",\"|\",\"outdent\",\"indent\",\"|\",\"undo\",\"redo\",\"|\",\"link\",\"unlink\",\"anchor\",\"code\",\"|\",\"hr\",\"table\",\"|\",\"subscript\",\"superscript\",\"|\",\"charmap\",\"pastetext\",\"preview\"]},\"0\":{\"menu\":[\"edit\",\"insert\",\"view\",\"format\",\"table\",\"tools\"],\"toolbar1\":[\"bold\",\"italic\",\"underline\",\"strikethrough\",\"|\",\"alignleft\",\"aligncenter\",\"alignright\",\"alignjustify\",\"|\",\"styleselect\",\"|\",\"formatselect\",\"fontselect\",\"fontsizeselect\",\"|\",\"searchreplace\",\"|\",\"bullist\",\"numlist\",\"|\",\"outdent\",\"indent\",\"|\",\"undo\",\"redo\",\"|\",\"link\",\"unlink\",\"anchor\",\"image\",\"|\",\"code\",\"|\",\"forecolor\",\"backcolor\",\"|\",\"fullscreen\",\"|\",\"table\",\"|\",\"subscript\",\"superscript\",\"|\",\"charmap\",\"emoticons\",\"media\",\"hr\",\"ltr\",\"rtl\",\"|\",\"cut\",\"copy\",\"paste\",\"pastetext\",\"|\",\"visualchars\",\"visualblocks\",\"nonbreaking\",\"blockquote\",\"template\",\"|\",\"print\",\"preview\",\"codesample\",\"insertdatetime\",\"removeformat\"]}},\"setoptions\":{\"2\":{\"access\":[\"1\"],\"skin\":\"0\",\"skin_admin\":\"0\",\"mobile\":\"0\",\"drag_drop\":\"1\",\"path\":\"\",\"entity_encoding\":\"raw\",\"lang_mode\":\"1\",\"text_direction\":\"ltr\",\"content_css\":\"1\",\"content_css_custom\":\"\",\"relative_urls\":\"1\",\"newlines\":\"0\",\"use_config_textfilters\":\"0\",\"invalid_elements\":\"script,applet,iframe\",\"valid_elements\":\"\",\"extended_elements\":\"\",\"resizing\":\"1\",\"resize_horizontal\":\"1\",\"element_path\":\"1\",\"wordcount\":\"1\",\"image_advtab\":\"1\",\"advlist\":\"1\",\"contextmenu\":\"1\",\"custom_plugin\":\"\",\"custom_button\":\"\"},\"1\":{\"access\":[\"6\",\"2\",\"3\"],\"skin\":\"0\",\"skin_admin\":\"0\",\"mobile\":\"0\",\"drag_drop\":\"1\",\"path\":\"\",\"entity_encoding\":\"raw\",\"lang_mode\":\"1\",\"text_direction\":\"ltr\",\"content_css\":\"1\",\"content_css_custom\":\"\",\"relative_urls\":\"1\",\"newlines\":\"0\",\"use_config_textfilters\":\"0\",\"invalid_elements\":\"applet\",\"valid_elements\":\"\",\"extended_elements\":\"\",\"resizing\":\"1\",\"resize_horizontal\":\"1\",\"element_path\":\"1\",\"wordcount\":\"1\",\"image_advtab\":\"1\",\"advlist\":\"1\",\"contextmenu\":\"1\",\"custom_plugin\":\"\",\"custom_button\":\"\"},\"0\":{\"access\":[\"7\",\"4\",\"8\"],\"skin\":\"0\",\"skin_admin\":\"0\",\"mobile\":\"0\",\"drag_drop\":\"1\",\"path\":\"\",\"entity_encoding\":\"raw\",\"lang_mode\":\"1\",\"text_direction\":\"ltr\",\"content_css\":\"1\",\"content_css_custom\":\"\",\"relative_urls\":\"1\",\"newlines\":\"0\",\"use_config_textfilters\":\"0\",\"invalid_elements\":\"applet\",\"valid_elements\":\"\",\"extended_elements\":\"\",\"resizing\":\"1\",\"resize_horizontal\":\"1\",\"element_path\":\"1\",\"wordcount\":\"1\",\"image_advtab\":\"1\",\"advlist\":\"1\",\"contextmenu\":\"1\",\"custom_plugin\":\"\",\"custom_button\":\"\"}}},\"sets_amount\":3,\"html_height\":\"550\",\"html_width\":\"750\"}','','',0,'0000-00-00 00:00:00',3,0),(413,0,'plg_editors-xtd_article','plugin','article','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_article\",\"type\":\"plugin\",\"creationDate\":\"October 2009\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2009 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_ARTICLE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"article\"}','','','',0,'0000-00-00 00:00:00',1,0),(414,0,'plg_editors-xtd_image','plugin','image','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_image\",\"type\":\"plugin\",\"creationDate\":\"August 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_IMAGE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"image\"}','','','',0,'0000-00-00 00:00:00',2,0),(415,0,'plg_editors-xtd_pagebreak','plugin','pagebreak','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_pagebreak\",\"type\":\"plugin\",\"creationDate\":\"August 2004\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_EDITORSXTD_PAGEBREAK_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"pagebreak\"}','','','',0,'0000-00-00 00:00:00',3,0),(416,0,'plg_editors-xtd_readmore','plugin','readmore','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_readmore\",\"type\":\"plugin\",\"creationDate\":\"March 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_READMORE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"readmore\"}','','','',0,'0000-00-00 00:00:00',4,0),(417,0,'plg_search_categories','plugin','categories','search',0,1,1,0,'{\"name\":\"plg_search_categories\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEARCH_CATEGORIES_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"categories\"}','{\"search_limit\":\"50\",\"search_content\":\"1\",\"search_archived\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(418,0,'plg_search_contacts','plugin','contacts','search',0,1,1,0,'{\"name\":\"plg_search_contacts\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEARCH_CONTACTS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contacts\"}','{\"search_limit\":\"50\",\"search_content\":\"1\",\"search_archived\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(419,0,'plg_search_content','plugin','content','search',0,1,1,0,'{\"name\":\"plg_search_content\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEARCH_CONTENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"content\"}','{\"search_limit\":\"50\",\"search_content\":\"1\",\"search_archived\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(420,0,'plg_search_newsfeeds','plugin','newsfeeds','search',0,1,1,0,'{\"name\":\"plg_search_newsfeeds\",\"type\":\"plugin\",\"creationDate\":\"November 2005\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEARCH_NEWSFEEDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"newsfeeds\"}','{\"search_limit\":\"50\",\"search_content\":\"1\",\"search_archived\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(422,0,'plg_system_languagefilter','plugin','languagefilter','system',0,1,1,1,'{\"name\":\"plg_system_languagefilter\",\"type\":\"plugin\",\"creationDate\":\"July 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2010 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SYSTEM_LANGUAGEFILTER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"languagefilter\"}','{\"detect_browser\":\"0\",\"automatic_change\":\"1\",\"item_associations\":\"1\",\"remove_default_prefix\":\"1\",\"lang_cookie\":\"0\",\"alternate_meta\":\"0\"}','','',0,'0000-00-00 00:00:00',16,0),(423,0,'plg_system_p3p','plugin','p3p','system',0,0,1,0,'{\"name\":\"plg_system_p3p\",\"type\":\"plugin\",\"creationDate\":\"September 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2010 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_P3P_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"p3p\"}','{\"headers\":\"NOI ADM DEV PSAi COM NAV OUR OTRo STP IND DEM\"}','','',0,'0000-00-00 00:00:00',17,0),(424,0,'plg_system_cache','plugin','cache','system',0,0,1,1,'{\"name\":\"plg_system_cache\",\"type\":\"plugin\",\"creationDate\":\"February 2007\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2007 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CACHE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"cache\"}','{\"browsercache\":\"0\",\"cachetime\":\"15\"}','','',0,'0000-00-00 00:00:00',25,0),(425,0,'plg_system_debug','plugin','debug','system',0,1,1,0,'{\"name\":\"plg_system_debug\",\"type\":\"plugin\",\"creationDate\":\"December 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_DEBUG_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"debug\"}','{\"profile\":\"1\",\"queries\":\"1\",\"memory\":\"1\",\"language_files\":\"1\",\"language_strings\":\"1\",\"strip-first\":\"1\",\"strip-prefix\":\"\",\"strip-suffix\":\"\"}','','',0,'0000-00-00 00:00:00',19,0),(426,0,'plg_system_log','plugin','log','system',0,1,1,1,'{\"name\":\"plg_system_log\",\"type\":\"plugin\",\"creationDate\":\"April 2007\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2007 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_LOG_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"log\"}','','','',0,'0000-00-00 00:00:00',20,0),(427,0,'plg_system_redirect','plugin','redirect','system',0,0,1,1,'{\"name\":\"plg_system_redirect\",\"type\":\"plugin\",\"creationDate\":\"April 2009\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2009 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SYSTEM_REDIRECT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"redirect\"}','{\"collect_urls\":1,\"includeUrl\":1,\"exclude_urls\":[]}','','',0,'0000-00-00 00:00:00',21,0),(428,0,'plg_system_remember','plugin','remember','system',0,1,1,1,'{\"name\":\"plg_system_remember\",\"type\":\"plugin\",\"creationDate\":\"April 2007\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2007 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_REMEMBER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"remember\"}','','','',0,'0000-00-00 00:00:00',22,0),(429,0,'plg_system_sef','plugin','sef','system',0,1,1,0,'{\"name\":\"plg_system_sef\",\"type\":\"plugin\",\"creationDate\":\"December 2007\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2007 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEF_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"sef\"}','','','',0,'0000-00-00 00:00:00',24,0),(430,0,'plg_system_logout','plugin','logout','system',0,1,1,1,'{\"name\":\"plg_system_logout\",\"type\":\"plugin\",\"creationDate\":\"April 2009\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2009 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SYSTEM_LOGOUT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"logout\"}','','','',0,'0000-00-00 00:00:00',18,0),(431,0,'plg_user_contactcreator','plugin','contactcreator','user',0,0,1,0,'{\"name\":\"plg_user_contactcreator\",\"type\":\"plugin\",\"creationDate\":\"August 2009\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2009 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CONTACTCREATOR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contactcreator\"}','{\"autowebpage\":\"\",\"category\":\"34\",\"autopublish\":\"0\"}','','',0,'0000-00-00 00:00:00',1,0),(432,0,'plg_user_joomla','plugin','joomla','user',0,1,1,0,'{\"name\":\"plg_user_joomla\",\"type\":\"plugin\",\"creationDate\":\"December 2006\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_USER_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','{\"autoregister\":\"1\",\"mail_to_user\":\"1\",\"forceLogout\":\"1\"}','','',0,'0000-00-00 00:00:00',2,0),(433,0,'plg_user_profile','plugin','profile','user',0,0,1,0,'{\"name\":\"plg_user_profile\",\"type\":\"plugin\",\"creationDate\":\"January 2008\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2008 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_USER_PROFILE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"profile\"}','{\"register-require_address1\":\"1\",\"register-require_address2\":\"1\",\"register-require_city\":\"1\",\"register-require_region\":\"1\",\"register-require_country\":\"1\",\"register-require_postal_code\":\"1\",\"register-require_phone\":\"1\",\"register-require_website\":\"1\",\"register-require_favoritebook\":\"1\",\"register-require_aboutme\":\"1\",\"register-require_tos\":\"2\",\"register_tos_article\":\"\",\"register-require_dob\":\"1\",\"profile-require_address1\":\"1\",\"profile-require_address2\":\"1\",\"profile-require_city\":\"1\",\"profile-require_region\":\"1\",\"profile-require_country\":\"1\",\"profile-require_postal_code\":\"1\",\"profile-require_phone\":\"1\",\"profile-require_website\":\"1\",\"profile-require_favoritebook\":\"1\",\"profile-require_aboutme\":\"1\",\"profile-require_dob\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(434,0,'plg_extension_joomla','plugin','joomla','extension',0,1,1,1,'{\"name\":\"plg_extension_joomla\",\"type\":\"plugin\",\"creationDate\":\"May 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2010 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_EXTENSION_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','','','',0,'0000-00-00 00:00:00',1,0),(435,0,'plg_content_joomla','plugin','joomla','content',0,1,1,0,'{\"name\":\"plg_content_joomla\",\"type\":\"plugin\",\"creationDate\":\"November 2010\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2010 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CONTENT_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','','','',0,'0000-00-00 00:00:00',0,0),(436,0,'plg_system_languagecode','plugin','languagecode','system',0,0,1,0,'{\"name\":\"plg_system_languagecode\",\"type\":\"plugin\",\"creationDate\":\"November 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SYSTEM_LANGUAGECODE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"languagecode\"}','','','',0,'0000-00-00 00:00:00',26,0),(437,0,'plg_quickicon_joomlaupdate','plugin','joomlaupdate','quickicon',0,1,1,1,'{\"name\":\"plg_quickicon_joomlaupdate\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_QUICKICON_JOOMLAUPDATE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomlaupdate\"}','','','',0,'0000-00-00 00:00:00',0,0),(438,0,'plg_quickicon_extensionupdate','plugin','extensionupdate','quickicon',0,1,1,1,'{\"name\":\"plg_quickicon_extensionupdate\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_QUICKICON_EXTENSIONUPDATE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"extensionupdate\"}','','','',0,'0000-00-00 00:00:00',0,0),(439,0,'plg_captcha_recaptcha','plugin','recaptcha','captcha',0,0,1,0,'{\"name\":\"plg_captcha_recaptcha\",\"type\":\"plugin\",\"creationDate\":\"December 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.4.0\",\"description\":\"PLG_CAPTCHA_RECAPTCHA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"recaptcha\"}','{\"public_key\":\"\",\"private_key\":\"\",\"theme\":\"clean\"}','','',0,'0000-00-00 00:00:00',0,0),(440,0,'plg_system_highlight','plugin','highlight','system',0,1,1,0,'{\"name\":\"plg_system_highlight\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SYSTEM_HIGHLIGHT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"highlight\"}','','','',0,'0000-00-00 00:00:00',23,0),(441,0,'plg_content_finder','plugin','finder','content',0,1,1,0,'{\"name\":\"plg_content_finder\",\"type\":\"plugin\",\"creationDate\":\"December 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_CONTENT_FINDER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"finder\"}','','','',0,'0000-00-00 00:00:00',0,0),(442,0,'plg_finder_categories','plugin','categories','finder',0,0,1,0,'{\"name\":\"plg_finder_categories\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_FINDER_CATEGORIES_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"categories\"}','','','',0,'0000-00-00 00:00:00',1,0),(443,0,'plg_finder_contacts','plugin','contacts','finder',0,0,1,0,'{\"name\":\"plg_finder_contacts\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_FINDER_CONTACTS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contacts\"}','','','',0,'0000-00-00 00:00:00',2,0),(444,0,'plg_finder_content','plugin','content','finder',0,0,1,0,'{\"name\":\"plg_finder_content\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_FINDER_CONTENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"content\"}','','','',0,'0000-00-00 00:00:00',3,0),(445,0,'plg_finder_newsfeeds','plugin','newsfeeds','finder',0,0,1,0,'{\"name\":\"plg_finder_newsfeeds\",\"type\":\"plugin\",\"creationDate\":\"August 2011\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2011 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_FINDER_NEWSFEEDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"newsfeeds\"}','','','',0,'0000-00-00 00:00:00',4,0),(447,0,'plg_finder_tags','plugin','tags','finder',0,0,1,0,'{\"name\":\"plg_finder_tags\",\"type\":\"plugin\",\"creationDate\":\"February 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2013 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_FINDER_TAGS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"tags\"}','','','',0,'0000-00-00 00:00:00',0,0),(448,0,'plg_twofactorauth_totp','plugin','totp','twofactorauth',0,0,1,0,'{\"name\":\"plg_twofactorauth_totp\",\"type\":\"plugin\",\"creationDate\":\"August 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2013 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.0\",\"description\":\"PLG_TWOFACTORAUTH_TOTP_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"totp\"}','','','',0,'0000-00-00 00:00:00',0,0),(449,0,'plg_authentication_cookie','plugin','cookie','authentication',0,1,1,0,'{\"name\":\"plg_authentication_cookie\",\"type\":\"plugin\",\"creationDate\":\"July 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2013 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_AUTH_COOKIE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"cookie\"}','','','',0,'0000-00-00 00:00:00',0,0),(450,0,'plg_twofactorauth_yubikey','plugin','yubikey','twofactorauth',0,0,1,0,'{\"name\":\"plg_twofactorauth_yubikey\",\"type\":\"plugin\",\"creationDate\":\"September 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2013 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.2.0\",\"description\":\"PLG_TWOFACTORAUTH_YUBIKEY_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"yubikey\"}','','','',0,'0000-00-00 00:00:00',0,0),(451,0,'plg_search_tags','plugin','tags','search',0,1,1,0,'{\"name\":\"plg_search_tags\",\"type\":\"plugin\",\"creationDate\":\"March 2014\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2014 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.0.0\",\"description\":\"PLG_SEARCH_TAGS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"tags\"}','{\"search_limit\":\"50\",\"show_tagged_items\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(455,0,'plg_installer_packageinstaller','plugin','packageinstaller','installer',0,1,1,1,'{\"name\":\"plg_installer_packageinstaller\",\"type\":\"plugin\",\"creationDate\":\"May 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.6.0\",\"description\":\"PLG_INSTALLER_PACKAGEINSTALLER_PLUGIN_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"packageinstaller\"}','','','',0,'0000-00-00 00:00:00',1,0),(456,0,'PLG_INSTALLER_FOLDERINSTALLER','plugin','folderinstaller','installer',0,1,1,1,'{\"name\":\"PLG_INSTALLER_FOLDERINSTALLER\",\"type\":\"plugin\",\"creationDate\":\"May 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.6.0\",\"description\":\"PLG_INSTALLER_FOLDERINSTALLER_PLUGIN_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"folderinstaller\"}','','','',0,'0000-00-00 00:00:00',2,0),(457,0,'PLG_INSTALLER_URLINSTALLER','plugin','urlinstaller','installer',0,1,1,1,'{\"name\":\"PLG_INSTALLER_URLINSTALLER\",\"type\":\"plugin\",\"creationDate\":\"May 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.6.0\",\"description\":\"PLG_INSTALLER_URLINSTALLER_PLUGIN_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"urlinstaller\"}','','','',0,'0000-00-00 00:00:00',3,0),(458,0,'plg_quickicon_phpversioncheck','plugin','phpversioncheck','quickicon',0,1,1,1,'{\"name\":\"plg_quickicon_phpversioncheck\",\"type\":\"plugin\",\"creationDate\":\"August 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_QUICKICON_PHPVERSIONCHECK_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"phpversioncheck\"}','','','',0,'0000-00-00 00:00:00',0,0),(459,0,'plg_editors-xtd_menu','plugin','menu','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_menu\",\"type\":\"plugin\",\"creationDate\":\"August 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_EDITORS-XTD_MENU_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"menu\"}','','','',0,'0000-00-00 00:00:00',0,0),(460,0,'plg_editors-xtd_contact','plugin','contact','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_contact\",\"type\":\"plugin\",\"creationDate\":\"October 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_EDITORS-XTD_CONTACT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contact\"}','','','',0,'0000-00-00 00:00:00',0,0),(461,0,'plg_system_fields','plugin','fields','system',0,1,1,0,'{\"name\":\"plg_system_fields\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_SYSTEM_FIELDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"fields\"}','','','',0,'0000-00-00 00:00:00',15,0),(462,0,'plg_fields_calendar','plugin','calendar','fields',0,1,1,0,'{\"name\":\"plg_fields_calendar\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_CALENDAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"calendar\"}','','','',0,'0000-00-00 00:00:00',0,0),(463,0,'plg_fields_checkboxes','plugin','checkboxes','fields',0,1,1,0,'{\"name\":\"plg_fields_checkboxes\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_CHECKBOXES_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"checkboxes\"}','','','',0,'0000-00-00 00:00:00',0,0),(464,0,'plg_fields_color','plugin','color','fields',0,1,1,0,'{\"name\":\"plg_fields_color\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_COLOR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"color\"}','','','',0,'0000-00-00 00:00:00',0,0),(465,0,'plg_fields_editor','plugin','editor','fields',0,1,1,0,'{\"name\":\"plg_fields_editor\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_EDITOR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"editor\"}','','','',0,'0000-00-00 00:00:00',0,0),(466,0,'plg_fields_imagelist','plugin','imagelist','fields',0,1,1,0,'{\"name\":\"plg_fields_imagelist\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_IMAGELIST_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"imagelist\"}','','','',0,'0000-00-00 00:00:00',0,0),(467,0,'plg_fields_integer','plugin','integer','fields',0,1,1,0,'{\"name\":\"plg_fields_integer\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_INTEGER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"integer\"}','{\"multiple\":\"0\",\"first\":\"1\",\"last\":\"100\",\"step\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(468,0,'plg_fields_list','plugin','list','fields',0,1,1,0,'{\"name\":\"plg_fields_list\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_LIST_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"list\"}','','','',0,'0000-00-00 00:00:00',0,0),(469,0,'plg_fields_media','plugin','media','fields',0,1,1,0,'{\"name\":\"plg_fields_media\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_MEDIA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"media\"}','','','',0,'0000-00-00 00:00:00',0,0),(470,0,'plg_fields_radio','plugin','radio','fields',0,1,1,0,'{\"name\":\"plg_fields_radio\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_RADIO_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"radio\"}','','','',0,'0000-00-00 00:00:00',0,0),(471,0,'plg_fields_sql','plugin','sql','fields',0,1,1,0,'{\"name\":\"plg_fields_sql\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_SQL_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"sql\"}','','','',0,'0000-00-00 00:00:00',0,0),(472,0,'plg_fields_text','plugin','text','fields',0,1,1,0,'{\"name\":\"plg_fields_text\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_TEXT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"text\"}','','','',0,'0000-00-00 00:00:00',0,0),(473,0,'plg_fields_textarea','plugin','textarea','fields',0,1,1,0,'{\"name\":\"plg_fields_textarea\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_TEXTAREA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"textarea\"}','','','',0,'0000-00-00 00:00:00',0,0),(474,0,'plg_fields_url','plugin','url','fields',0,1,1,0,'{\"name\":\"plg_fields_url\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_URL_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"url\"}','','','',0,'0000-00-00 00:00:00',0,0),(475,0,'plg_fields_user','plugin','user','fields',0,1,1,0,'{\"name\":\"plg_fields_user\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_USER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"user\"}','','','',0,'0000-00-00 00:00:00',0,0),(476,0,'plg_fields_usergrouplist','plugin','usergrouplist','fields',0,1,1,0,'{\"name\":\"plg_fields_usergrouplist\",\"type\":\"plugin\",\"creationDate\":\"March 2016\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2016 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_FIELDS_USERGROUPLIST_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"usergrouplist\"}','','','',0,'0000-00-00 00:00:00',0,0),(477,0,'plg_content_fields','plugin','fields','content',0,1,1,0,'{\"name\":\"plg_content_fields\",\"type\":\"plugin\",\"creationDate\":\"February 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2017 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_CONTENT_FIELDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"fields\"}','','','',0,'0000-00-00 00:00:00',0,0),(478,0,'plg_editors-xtd_fields','plugin','fields','editors-xtd',0,1,1,0,'{\"name\":\"plg_editors-xtd_fields\",\"type\":\"plugin\",\"creationDate\":\"February 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2017 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.7.0\",\"description\":\"PLG_EDITORS-XTD_FIELDS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"fields\"}','','','',0,'0000-00-00 00:00:00',0,0),(479,0,'plg_sampledata_blog','plugin','blog','sampledata',0,0,1,0,'{\"name\":\"plg_sampledata_blog\",\"type\":\"plugin\",\"creationDate\":\"July 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2017 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.8.0\",\"description\":\"PLG_SAMPLEDATA_BLOG_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"blog\"}','','','',0,'0000-00-00 00:00:00',0,0),(480,0,'plg_system_sessiongc','plugin','sessiongc','system',0,1,1,0,'{\"name\":\"plg_system_sessiongc\",\"type\":\"plugin\",\"creationDate\":\"February 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.8.6\",\"description\":\"PLG_SYSTEM_SESSIONGC_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"sessiongc\"}','','','',0,'0000-00-00 00:00:00',0,0),(481,0,'plg_fields_repeatable','plugin','repeatable','fields',0,1,1,0,'{\"name\":\"plg_fields_repeatable\",\"type\":\"plugin\",\"creationDate\":\"April 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_FIELDS_REPEATABLE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"repeatable\"}','','','',0,'0000-00-00 00:00:00',0,0),(482,0,'plg_content_confirmconsent','plugin','confirmconsent','content',0,0,1,0,'{\"name\":\"plg_content_confirmconsent\",\"type\":\"plugin\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_CONTENT_CONFIRMCONSENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"confirmconsent\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(483,0,'PLG_SYSTEM_ACTIONLOGS','plugin','actionlogs','system',0,0,1,0,'{\"name\":\"PLG_SYSTEM_ACTIONLOGS\",\"type\":\"plugin\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_SYSTEM_ACTIONLOGS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"actionlogs\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(484,0,'PLG_ACTIONLOG_JOOMLA','plugin','joomla','actionlog',0,1,1,0,'{\"name\":\"PLG_ACTIONLOG_JOOMLA\",\"type\":\"plugin\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_ACTIONLOG_JOOMLA_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"joomla\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(485,0,'plg_system_privacyconsent','plugin','privacyconsent','system',0,0,1,0,'{\"name\":\"plg_system_privacyconsent\",\"type\":\"plugin\",\"creationDate\":\"April 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_SYSTEM_PRIVACYCONSENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"privacyconsent\"}','{\"privacy_note\":\"\",\"privacy_article\":\"\",\"messageOnRedirect\":\"\",\"enabled\":0,\"cachetimeout\":30,\"consentexpiration\":360,\"remind\":30,\"lastrun\":0}','','',0,'0000-00-00 00:00:00',0,0),(486,0,'plg_system_logrotation','plugin','logrotation','system',0,1,1,0,'{\"name\":\"plg_system_logrotation\",\"type\":\"plugin\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_SYSTEM_LOGROTATION_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"logrotation\"}','{\"lastrun\":1727111061}','','',0,'0000-00-00 00:00:00',0,0),(487,0,'plg_privacy_user','plugin','user','privacy',0,1,1,0,'{\"name\":\"plg_privacy_user\",\"type\":\"plugin\",\"creationDate\":\"May 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_USER_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"user\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(488,0,'plg_quickicon_privacycheck','plugin','privacycheck','quickicon',0,1,1,0,'{\"name\":\"plg_quickicon_privacycheck\",\"type\":\"plugin\",\"creationDate\":\"June 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_QUICKICON_PRIVACYCHECK_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"privacycheck\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(489,0,'plg_user_terms','plugin','terms','user',0,0,1,0,'{\"name\":\"plg_user_terms\",\"type\":\"plugin\",\"creationDate\":\"June 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_USER_TERMS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"terms\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(490,0,'plg_privacy_contact','plugin','contact','privacy',0,1,1,0,'{\"name\":\"plg_privacy_contact\",\"type\":\"plugin\",\"creationDate\":\"July 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_CONTACT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"contact\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(491,0,'plg_privacy_content','plugin','content','privacy',0,1,1,0,'{\"name\":\"plg_privacy_content\",\"type\":\"plugin\",\"creationDate\":\"July 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_CONTENT_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"content\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(492,0,'plg_privacy_message','plugin','message','privacy',0,1,1,0,'{\"name\":\"plg_privacy_message\",\"type\":\"plugin\",\"creationDate\":\"July 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_MESSAGE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"message\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(493,0,'plg_privacy_actionlogs','plugin','actionlogs','privacy',0,1,1,0,'{\"name\":\"plg_privacy_actionlogs\",\"type\":\"plugin\",\"creationDate\":\"July 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_ACTIONLOGS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"actionlogs\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(494,0,'plg_captcha_recaptcha_invisible','plugin','recaptcha_invisible','captcha',0,0,1,0,'{\"name\":\"plg_captcha_recaptcha_invisible\",\"type\":\"plugin\",\"creationDate\":\"November 2017\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2017 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.8\",\"description\":\"PLG_CAPTCHA_RECAPTCHA_INVISIBLE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"recaptcha_invisible\"}','{\"public_key\":\"\",\"private_key\":\"\",\"theme\":\"clean\"}','','',0,'0000-00-00 00:00:00',0,0),(495,0,'plg_privacy_consents','plugin','consents','privacy',0,1,1,0,'{\"name\":\"plg_privacy_consents\",\"type\":\"plugin\",\"creationDate\":\"July 2018\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2018 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.9.0\",\"description\":\"PLG_PRIVACY_CONSENTS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"consents\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(503,0,'beez3','template','beez3','',0,1,1,0,'{\"name\":\"beez3\",\"type\":\"template\",\"creationDate\":\"25 November 2009\",\"author\":\"Angie Radtke\",\"copyright\":\"(C) 2010 Open Source Matters, Inc.\",\"authorEmail\":\"a.radtke@derauftritt.de\",\"authorUrl\":\"http:\\/\\/www.der-auftritt.de\",\"version\":\"3.1.0\",\"description\":\"TPL_BEEZ3_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"templateDetails\"}','{\"wrapperSmall\":\"53\",\"wrapperLarge\":\"72\",\"sitetitle\":\"\",\"sitedescription\":\"\",\"navposition\":\"center\",\"templatecolor\":\"nature\"}','','',0,'0000-00-00 00:00:00',0,0),(504,0,'hathor','template','hathor','',1,1,1,0,'{\"name\":\"hathor\",\"type\":\"template\",\"creationDate\":\"May 2010\",\"author\":\"Andrea Tarr\",\"copyright\":\"(C) 2010 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"\",\"version\":\"3.0.0\",\"description\":\"TPL_HATHOR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"templateDetails\"}','{\"showSiteName\":\"0\",\"colourChoice\":\"0\",\"boldText\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(506,0,'protostar','template','protostar','',0,1,1,0,'{\"name\":\"protostar\",\"type\":\"template\",\"creationDate\":\"4\\/30\\/2012\",\"author\":\"Kyle Ledbetter\",\"copyright\":\"(C) 2012 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"\",\"version\":\"1.0\",\"description\":\"TPL_PROTOSTAR_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"templateDetails\"}','{\"templateColor\":\"\",\"logoFile\":\"\",\"googleFont\":\"1\",\"googleFontName\":\"Open+Sans\",\"fluidContainer\":\"0\"}','','',0,'0000-00-00 00:00:00',0,0),(507,0,'isis','template','isis','',1,1,1,0,'{\"name\":\"isis\",\"type\":\"template\",\"creationDate\":\"3\\/30\\/2012\",\"author\":\"Kyle Ledbetter\",\"copyright\":\"(C) 2012 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"\",\"version\":\"1.0\",\"description\":\"TPL_ISIS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"templateDetails\"}','{\"templateColor\":\"\",\"logoFile\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(600,802,'English (en-GB)','language','en-GB','',0,1,1,1,'{\"name\":\"English (en-GB)\",\"type\":\"language\",\"creationDate\":\"July 2023\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2006 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.10.12\",\"description\":\"en-GB site language\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(601,802,'English (en-GB)','language','en-GB','',1,1,1,1,'{\"name\":\"English (en-GB)\",\"type\":\"language\",\"creationDate\":\"July 2023\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2005 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.10.12\",\"description\":\"en-GB administrator language\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(700,0,'files_joomla','file','joomla','',0,1,1,1,'{\"name\":\"files_joomla\",\"type\":\"file\",\"creationDate\":\"July 2023\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2019 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.10.12\",\"description\":\"FILES_JOOMLA_XML_DESCRIPTION\",\"group\":\"\"}','','','',0,'0000-00-00 00:00:00',0,0),(802,0,'English (en-GB) Language Pack','package','pkg_en-GB','',0,1,1,1,'{\"name\":\"English (en-GB) Language Pack\",\"type\":\"package\",\"creationDate\":\"July 2023\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2019 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.10.12.1\",\"description\":\"en-GB language pack\",\"group\":\"\",\"filename\":\"pkg_en-GB\"}','','','',0,'0000-00-00 00:00:00',0,0),(10180,0,'FavGlyph','module','mod_favglyph','',0,1,0,0,'{\"name\":\"FavGlyph\",\"type\":\"module\",\"creationDate\":\"2014\",\"author\":\"FavThemes\",\"copyright\":\"Copyright (C) FavThemes. All rights reserved.\",\"authorEmail\":\"\",\"authorUrl\":\"http:\\/\\/www.favthemes.com\",\"version\":\"2.2\",\"description\":\"\\n  \\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favglyph\\\" target=\\\"_blank\\\"><strong>FavGlyph<\\/strong><\\/a> is a free responsive Joomla! module that uses any icon from Font Awesome and assigns it a short description and a title.\\n    <br\\/>\\n    For more info, please see the <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favglyph\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">demo and documentation<\\/a> for the FavGlyph module.\\n    <br\\/><br\\/>\\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favglyph\\\" target=\\\"_blank\\\">\\n      <img src=\\\"..\\/modules\\/mod_favglyph\\/theme\\/img\\/favglyph.jpg\\\" alt=\\\"FavGlyph Responsive Module\\\">\\n    <\\/a>\\n\\n    <br\\/><br\\/>\\n    Copyright &#169; 2012-2018 <a href=\\\"http:\\/\\/www.favthemes.com\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">FavThemes<\\/a>.\\n    <br\\/><br\\/>\\n\\n    <link rel=\\\"stylesheet\\\" href=\\\"..\\/modules\\/mod_favglyph\\/theme\\/css\\/admin.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/use.fontawesome.com\\/releases\\/v5.1.0\\/css\\/all.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/fonts.googleapis.com\\/css?family=Open+Sans:400\\\" type=\\\"text\\/css\\\" \\/>\\n    <script src=\\\"..\\/modules\\/mod_favglyph\\/theme\\/js\\/jscolor\\/jscolor.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n    <script src=\\\"..\\/modules\\/mod_favglyph\\/theme\\/js\\/admin\\/admin.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n\\n  \\n  \",\"group\":\"\",\"filename\":\"mod_favglyph\"}','{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect1\",\"icon_width\":\"77%\",\"icon_border_radius\":\"50%\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_size\":\"21px\",\"title_margin\":\"28px 0px 14px\",\"description_description_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_layout1\":\"center\",\"icon_name1\":\"fa-globe\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"CCCCCC\",\"icon_link1\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target1\":\"blank\",\"icon_font_size1\":\"6em\",\"title_text1\":\"Title Text\",\"title_color1\":\"444444\",\"description_layout1\":\"0\",\"description_text1\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_layout2\":\"center\",\"icon_name2\":\"fa-html5\",\"icon_color2\":\"ff5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"CCCCCC\",\"icon_link2\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target2\":\"blank\",\"icon_font_size2\":\"6em\",\"title_text2\":\"Title Text\",\"title_color2\":\"444444\",\"description_layout2\":\"0\",\"description_text2\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_layout3\":\"center\",\"icon_name3\":\"fa-recycle\",\"icon_color3\":\"8bc34a\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"CCCCCC\",\"icon_link3\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target3\":\"blank\",\"icon_font_size3\":\"6em\",\"title_text3\":\"Title Text\",\"title_color3\":\"444444\",\"description_layout3\":\"0\",\"description_text3\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_layout4\":\"center\",\"icon_name4\":\"fa-lightbulb-o\",\"icon_color4\":\"ffc107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"CCCCCC\",\"icon_link4\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target4\":\"blank\",\"icon_font_size4\":\"6em\",\"title_text4\":\"Title Text\",\"title_color4\":\"444444\",\"description_layout4\":\"0\",\"description_text4\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_layout5\":\"center\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673ab7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"description_layout5\":\"0\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_layout6\":\"center\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00bcd4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"description_layout6\":\"0\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\"}','','',0,'0000-00-00 00:00:00',0,0),(10182,0,'FavPromote','module','mod_favpromote','',0,1,0,0,'{\"name\":\"FavPromote\",\"type\":\"module\",\"creationDate\":\"2013\",\"author\":\"FavThemes\",\"copyright\":\"Copyright (C) FavThemes. All rights reserved.\",\"authorEmail\":\"\",\"authorUrl\":\"http:\\/\\/www.favthemes.com\",\"version\":\"2.2\",\"description\":\"\\n  \\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\"><strong>FavPromote<\\/strong><\\/a> is a free responsive Joomla! module that lets you upload any image and assign it a short description and a title with a Font Awesome icon.\\n    <br\\/>\\n    For more info, please see the <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">demo and documentation<\\/a> for the FavPromote module.\\n    <br\\/><br\\/>\\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\">\\n      <img src=\\\"..\\/modules\\/mod_favpromote\\/theme\\/img\\/favpromote.jpg\\\" alt=\\\"FavPromote Responsive Module\\\">\\n    <\\/a>\\n\\n    <br\\/><br\\/>\\n    Copyright &#169; 2012-2018 <a href=\\\"http:\\/\\/www.favthemes.com\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">FavThemes<\\/a>.\\n    <br\\/><br\\/>\\n\\n    <link rel=\\\"stylesheet\\\" href=\\\"..\\/modules\\/mod_favpromote\\/theme\\/css\\/admin.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/use.fontawesome.com\\/releases\\/v5.1.0\\/css\\/all.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/fonts.googleapis.com\\/css?family=Open+Sans:400\\\" type=\\\"text\\/css\\\" \\/>\\n    <script src=\\\"..\\/modules\\/mod_favpromote\\/theme\\/js\\/jscolor\\/jscolor.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n    <script src=\\\"..\\/modules\\/mod_favpromote\\/theme\\/js\\/admin\\/admin.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n\\n  \\n\\t\",\"group\":\"\",\"filename\":\"mod_favpromote\"}','{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect1\",\"title_google_font\":\"Open Sans\",\"title_padding\":\"10px 20px\",\"title_font_size\":\"18px\",\"title_line_height\":\"1.4em\",\"title_text_align\":\"left\",\"title_icon_font_size\":\"18px\",\"title_icon_vertical_align\":\"baseline\",\"description_text_font_size\":\"14px\",\"description_text_line_height\":\"1.5em\",\"description_text_align\":\"justify\",\"show_column1\":\"1\",\"column_border_color1\":\"DDDDDD\",\"column_border_radius1\":\"4px\",\"show_image_link1\":\"1\",\"image_link1\":\"http:\\/\\/www.favthemes.com\",\"image_target1\":\"blank\",\"image_alt1\":\"\",\"title_text1\":\"Title Text\",\"title_color1\":\"FFFFFF\",\"title_bg_color1\":\"D63E52\",\"show_title_link1\":\"1\",\"title_link1\":\"http:\\/\\/www.favthemes.com\",\"title_target1\":\"blank\",\"title_icon1\":\"fa-coffee\",\"description_text1\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color1\":\"444444\",\"show_column2\":\"1\",\"column_border_color2\":\"DDDDDD\",\"column_border_radius2\":\"4px\",\"show_image_link2\":\"1\",\"image_link2\":\"http:\\/\\/www.favthemes.com\",\"image_target2\":\"blank\",\"image_alt2\":\"\",\"title_text2\":\"Title Text\",\"title_color2\":\"FFFFFF\",\"title_bg_color2\":\"D63E52\",\"show_title_link2\":\"1\",\"title_link2\":\"http:\\/\\/www.favthemes.com\",\"title_target2\":\"blank\",\"title_icon2\":\"fa-puzzle-piece\",\"description_text2\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color2\":\"444444\",\"show_column3\":\"1\",\"column_border_color3\":\"DDDDDD\",\"column_border_radius3\":\"4px\",\"show_image_link3\":\"1\",\"image_link3\":\"http:\\/\\/www.favthemes.com\",\"image_target3\":\"blank\",\"image_alt3\":\"\",\"title_text3\":\"Title Text\",\"title_color3\":\"FFFFFF\",\"title_bg_color3\":\"D63E52\",\"show_title_link3\":\"1\",\"title_link3\":\"http:\\/\\/www.favthemes.com\",\"title_target3\":\"blank\",\"title_icon3\":\"fa-bookmark\",\"description_text3\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color3\":\"444444\",\"show_column4\":\"1\",\"column_border_color4\":\"DDDDDD\",\"column_border_radius4\":\"4px\",\"show_image_link4\":\"1\",\"image_link4\":\"http:\\/\\/www.favthemes.com\",\"image_target4\":\"blank\",\"image_alt4\":\"\",\"title_text4\":\"Title Text\",\"title_color4\":\"FFFFFF\",\"title_bg_color4\":\"D63E52\",\"show_title_link4\":\"1\",\"title_link4\":\"http:\\/\\/www.favthemes.com\",\"title_target4\":\"blank\",\"title_icon4\":\"fa-magic\",\"description_text4\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color4\":\"444444\",\"show_column5\":\"0\",\"column_border_color5\":\"DDDDDD\",\"column_border_radius5\":\"4px\",\"show_image_link5\":\"1\",\"image_link5\":\"http:\\/\\/www.favthemes.com\",\"image_target5\":\"blank\",\"image_alt5\":\"\",\"title_text5\":\"Title Text\",\"title_color5\":\"FFFFFF\",\"title_bg_color5\":\"D63E52\",\"show_title_link5\":\"1\",\"title_link5\":\"http:\\/\\/www.favthemes.com\",\"title_target5\":\"blank\",\"title_icon5\":\"fa-quote-right\",\"description_text5\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color5\":\"444444\",\"show_column6\":\"0\",\"column_border_color6\":\"DDDDDD\",\"column_border_radius6\":\"4px\",\"show_image_link6\":\"1\",\"image_link6\":\"http:\\/\\/www.favthemes.com\",\"image_target6\":\"blank\",\"image_alt6\":\"\",\"title_text6\":\"Title Text\",\"title_color6\":\"FFFFFF\",\"title_bg_color6\":\"D63E52\",\"show_title_link6\":\"1\",\"title_link6\":\"http:\\/\\/www.favthemes.com\",\"title_target6\":\"blank\",\"title_icon6\":\"fa-recycle\",\"description_text6\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color6\":\"444444\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\"}','','',0,'0000-00-00 00:00:00',0,0),(10184,0,'FavSocial','module','mod_favsocial','',0,1,0,0,'{\"name\":\"FavSocial\",\"type\":\"module\",\"creationDate\":\"2013\",\"author\":\"FavThemes\",\"copyright\":\"Copyright (C) FavThemes. All rights reserved.\",\"authorEmail\":\"\",\"authorUrl\":\"http:\\/\\/www.favthemes.com\",\"version\":\"1.6\",\"description\":\"\\n  \\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favsocial\\\" target=\\\"_blank\\\"><strong>FavSocial<\\/strong><\\/a> is a free responsive Joomla! module that lets you choose up to 10 social media icons from Font Awesome and link each one to your social media accounts.\\n    <br\\/>\\n    For more info, please see the <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favsocial\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">demo and documentation<\\/a> for the FavSocial module.\\n    <br\\/><br\\/>\\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favsocial\\\" target=\\\"_blank\\\">\\n      <img src=\\\"..\\/modules\\/mod_favsocial\\/theme\\/img\\/favsocial.jpg\\\" alt=\\\"FavSocial Responsive Module\\\">\\n    <\\/a>\\n\\n    <br\\/><br\\/>\\n    Copyright &#169; 2012-2016 <a href=\\\"http:\\/\\/www.favthemes.com\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">FavThemes<\\/a>.\\n    <br\\/><br\\/>\\n\\n    <link rel=\\\"stylesheet\\\" href=\\\"..\\/modules\\/mod_favsocial\\/theme\\/css\\/admin.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/maxcdn.bootstrapcdn.com\\/font-awesome\\/4.7.0\\/css\\/font-awesome.min.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/fonts.googleapis.com\\/css?family=Open+Sans:400\\\" type=\\\"text\\/css\\\" \\/>\\n    <script src=\\\"..\\/modules\\/mod_favsocial\\/theme\\/js\\/jscolor\\/jscolor.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n\\n  \\n  \",\"group\":\"\",\"filename\":\"mod_favsocial\"}','{\"module_align\":\"favsocial-left\",\"show_icon1\":\"1\",\"icon_name1\":\"fa-twitter\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"00ACED\",\"icon_font_size1\":\"21px\",\"icon_link1\":\"https:\\/\\/twitter.com\\/\",\"icon_link_target1\":\"blank\",\"icon_padding1\":\"0.5em\",\"icon_border_radius1\":\"50%\",\"show_icon2\":\"1\",\"icon_name2\":\"fa-facebook\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"3B5998\",\"icon_font_size2\":\"21px\",\"icon_link2\":\"https:\\/\\/www.facebook.com\\/\",\"icon_link_target2\":\"blank\",\"icon_padding2\":\"0.5em\",\"icon_border_radius2\":\"50%\",\"show_icon3\":\"1\",\"icon_name3\":\"fa-dribbble\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"EA4C89\",\"icon_font_size3\":\"21px\",\"icon_link3\":\"https:\\/\\/dribbble.com\\/\",\"icon_link_target3\":\"blank\",\"icon_padding3\":\"0.5em\",\"icon_border_radius3\":\"50%\",\"show_icon4\":\"0\",\"icon_name4\":\"\",\"icon_color4\":\"\",\"icon_bg_color4\":\"\",\"icon_font_size4\":\"\",\"icon_link4\":\"\",\"icon_link_target4\":\"blank\",\"icon_padding4\":\"\",\"icon_border_radius4\":\"\",\"show_icon5\":\"0\",\"icon_name5\":\"\",\"icon_color5\":\"\",\"icon_bg_color5\":\"\",\"icon_font_size5\":\"\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_padding5\":\"\",\"icon_border_radius5\":\"\",\"show_icon6\":\"0\",\"icon_name6\":\"\",\"icon_color6\":\"\",\"icon_bg_color6\":\"\",\"icon_font_size6\":\"\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_padding6\":\"\",\"icon_border_radius6\":\"\",\"show_icon7\":\"0\",\"icon_name7\":\"\",\"icon_color7\":\"\",\"icon_bg_color7\":\"\",\"icon_font_size7\":\"\",\"icon_link7\":\"\",\"icon_link_target7\":\"blank\",\"icon_padding7\":\"\",\"icon_border_radius7\":\"\",\"show_icon8\":\"0\",\"icon_name8\":\"\",\"icon_color8\":\"\",\"icon_bg_color8\":\"\",\"icon_font_size8\":\"\",\"icon_link8\":\"\",\"icon_link_target8\":\"blank\",\"icon_padding8\":\"\",\"icon_border_radius8\":\"\",\"show_icon9\":\"0\",\"icon_name9\":\"\",\"icon_color9\":\"\",\"icon_bg_color9\":\"\",\"icon_font_size9\":\"\",\"icon_link9\":\"\",\"icon_link_target9\":\"blank\",\"icon_padding9\":\"\",\"icon_border_radius9\":\"\",\"show_icon10\":\"0\",\"icon_name10\":\"\",\"icon_color10\":\"\",\"icon_bg_color10\":\"\",\"icon_font_size10\":\"\",\"icon_link10\":\"\",\"icon_link_target10\":\"blank\",\"icon_padding10\":\"\",\"icon_border_radius10\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\"}','','',0,'0000-00-00 00:00:00',0,0),(10186,0,'Favourite','template','favourite','',0,1,1,0,'{\"name\":\"Favourite\",\"type\":\"template\",\"creationDate\":\"September 2012\",\"author\":\"FavThemes\",\"copyright\":\"Copyright (C) 2012-2016 FavThemes. All rights reserved.\",\"authorEmail\":\"hello@favthemes.com\",\"authorUrl\":\"http:\\/\\/www.favthemes.com\",\"version\":\"3.4\",\"description\":\"\\n\\n  <p style=\\\"max-width: 900px; margin-bottom: 21px; text-align: justify;\\\"><strong>Favourite<\\/strong> is a free responsive Joomla! template with over 200 settings for an easy and fast customization to suit your clients\\u2019 needs as quickly as possible and is built using Bootstrap, HTML5, CSS3, Google Fonts and Font Awesome. Design amazing responsive websites with this smart template that adapts and resizes itself to desktop and mobile devices, making your website looking great for everyone!\\n\\n  <\\/br><\\/br>\\n\\n  This free template can be used on unlimited personal or commercial websites, just don\'t resell it! Once you downloaded the template, you have free and unlimited access to all future Joomla! versions of the product.<\\/p>\\n\\n<a href=\\\"http:\\/\\/demo.favthemes.com\\/favourite\\/\\\" class=\\\"btn btn-success\\\" target=\\\"_blank\\\">Demo<\\/a>\\n\\n<a href=\\\"http:\\/\\/www.favthemes.com\\/docs.html\\\" class=\\\"btn btn-info\\\" target=\\\"_blank\\\">Documentation<\\/a>\\n\\n<\\/br><\\/br>\\n\\n<a href=\\\"http:\\/\\/demo.favthemes.com\\/favourite\\/\\\" target=\\\"_blank\\\">\\n  <img class=\\\"fav-admin-img\\\" style=\\\"padding: 4px; border: 1px solid #DDD; border-radius: 4px;\\\" src=\\\"..\\/templates\\/favourite\\/template_preview.png\\\" alt=\\\"Favourite Responsive Template\\\">\\n<\\/a>\\n\\n<\\/br><\\/br>\\nCopyright &#169; 2012-2016 <a href=\\\"http:\\/\\/www.favthemes.com\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">FavThemes<\\/a>.\\n<\\/br><\\/br>\\n\\n<script src=\\\"..\\/templates\\/favourite\\/admin\\/jscolor\\/jscolor.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n<link rel=\\\"stylesheet\\\" href=\\\"..\\/templates\\/favourite\\/admin\\/admin.css\\\" type=\\\"text\\/css\\\" \\/>\\n\\n\",\"group\":\"\",\"filename\":\"templateDetails\"}','{\"template_styles\":\"style1\",\"max_width\":\"\",\"show_copyright\":\"1\",\"copyright_text\":\"FavThemes\",\"copyright_text_link\":\"www.favthemes.com\",\"header_style\":\"fav-dark\",\"header_position\":\"fav-fixed\",\"header_fixed_height\":\"\",\"header_fixed_height_tablet\":\"\",\"nav_style\":\"navstyle-1\",\"nav_style_icon\":\"navstyle-icon-horizontal\",\"nav_style_color\":\"\",\"nav_google_font\":\"\",\"nav_google_font_weight\":\"400\",\"nav_google_font_style\":\"normal\",\"nav_text_transform\":\"uppercase\",\"nav_font_size\":\"\",\"nav_link_padding\":\"\",\"nav_icons_color\":\"\",\"nav_icons_font_size\":\"\",\"product_img_padding\":\"0\",\"link_color\":\"\",\"link_color_hover\":\"\",\"titles_google_font\":\"\",\"titles_google_font_weight\":\"400\",\"titles_google_font_style\":\"normal\",\"titles_text_align\":\"left\",\"titles_text_transform\":\"none\",\"article_title_color\":\"\",\"article_title_link_color\":\"\",\"article_title_link_hover_color\":\"\",\"article_title_font_size\":\"\",\"article_title_line_height\":\"\",\"module_title_color\":\"\",\"module_title_font_size\":\"\",\"module_title_line_height\":\"\",\"module_title_icon_color\":\"\",\"module_title_icon_font_size\":\"\",\"module_title_icon_padding\":\"\",\"module_title_icon_border_color\":\"\",\"module_title_icon_border_radius\":\"\",\"variation_style_color\":\"\",\"variation_style_icon_font_size\":\"\",\"variation_style_icon_padding\":\"\",\"button_color\":\"\",\"button_bg_color\":\"\",\"button_hover_color\":\"\",\"button_bg_hover_color\":\"\",\"button_google_font\":\"\",\"button_google_font_weight\":\"400\",\"button_google_font_style\":\"normal\",\"button_text_transform\":\"\",\"vertical_menus_style_color\":\"\",\"vertical_menus_text_transform\":\"none\",\"horizontal_menus_style_color\":\"\",\"horizontal_menus_google_font\":\"\",\"horizontal_menus_google_font_weight\":\"400\",\"horizontal_menus_google_font_style\":\"normal\",\"horizontal_menus_text_transform\":\"none\",\"list_style_color\":\"\",\"show_back_to_top\":\"1\",\"error_page_article_id\":\"3\",\"offline_page_style\":\"offline-dark\",\"offline_page_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"responsive_k2_image\":\"100%!important\",\"body_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"body_bg_image_overlay\":\"fav-transparent\",\"body_bg_color\":\"\",\"body_color\":\"\",\"body_title_color\":\"\",\"body_link_color\":\"\",\"body_link_hover_color\":\"\",\"show_advert_button\":\"1\",\"advert_bg_color\":\"\",\"advert_color\":\"\",\"advert_title_color\":\"\",\"advert_link_color\":\"\",\"advert_link_hover_color\":\"\",\"topbar_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"topbar_bg_image_overlay\":\"fav-transparent\",\"topbar_bg_color\":\"\",\"topbar_color\":\"\",\"topbar_title_color\":\"\",\"topbar_link_color\":\"\",\"topbar_link_hover_color\":\"\",\"slide_width\":\"\",\"slide_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"slide_bg_image_overlay\":\"fav-transparent\",\"slide_bg_color\":\"\",\"slide_color\":\"\",\"slide_title_color\":\"\",\"slide_link_color\":\"\",\"slide_link_hover_color\":\"\",\"intro_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"intro_bg_image_overlay\":\"fav-overlay\",\"intro_bg_color\":\"\",\"intro_color\":\"\",\"intro_title_color\":\"\",\"intro_link_color\":\"\",\"intro_link_hover_color\":\"\",\"lead_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"lead_bg_image_overlay\":\"fav-transparent\",\"lead_bg_color\":\"\",\"lead_color\":\"\",\"lead_title_color\":\"\",\"lead_link_color\":\"\",\"lead_link_hover_color\":\"\",\"promo_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"promo_bg_image_overlay\":\"fav-transparent\",\"promo_bg_color\":\"\",\"promo_color\":\"\",\"promo_title_color\":\"\",\"promo_link_color\":\"\",\"promo_link_hover_color\":\"\",\"prime_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"prime_bg_image_overlay\":\"fav-overlay\",\"prime_bg_color\":\"\",\"prime_color\":\"\",\"prime_title_color\":\"\",\"prime_link_color\":\"\",\"prime_link_hover_color\":\"\",\"showcase_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"showcase_bg_image_overlay\":\"fav-transparent\",\"showcase_bg_color\":\"\",\"showcase_color\":\"\",\"showcase_title_color\":\"\",\"showcase_link_color\":\"\",\"showcase_link_hover_color\":\"\",\"feature_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"feature_bg_image_overlay\":\"fav-transparent\",\"feature_bg_color\":\"\",\"feature_color\":\"\",\"feature_title_color\":\"\",\"feature_link_color\":\"\",\"feature_link_hover_color\":\"\",\"focus_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"focus_bg_image_overlay\":\"fav-transparent\",\"focus_bg_color\":\"\",\"focus_color\":\"\",\"focus_title_color\":\"\",\"focus_link_color\":\"\",\"focus_link_hover_color\":\"\",\"portfolio_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"portfolio_bg_image_overlay\":\"fav-transparent\",\"portfolio_bg_color\":\"\",\"portfolio_color\":\"\",\"portfolio_title_color\":\"\",\"portfolio_link_color\":\"\",\"portfolio_link_hover_color\":\"\",\"screen_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"screen_bg_image_overlay\":\"fav-overlay\",\"screen_bg_color\":\"\",\"screen_color\":\"\",\"screen_title_color\":\"\",\"screen_link_color\":\"\",\"screen_link_hover_color\":\"\",\"top_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"top_bg_image_overlay\":\"fav-transparent\",\"top_bg_color\":\"\",\"top_color\":\"\",\"top_title_color\":\"\",\"top_link_color\":\"\",\"top_link_hover_color\":\"\",\"maintop_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"maintop_bg_image_overlay\":\"fav-transparent\",\"maintop_bg_color\":\"\",\"maintop_color\":\"\",\"maintop_title_color\":\"\",\"maintop_link_color\":\"\",\"maintop_link_hover_color\":\"\",\"mainbottom_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"mainbottom_bg_image_overlay\":\"fav-transparent\",\"mainbottom_bg_color\":\"\",\"mainbottom_color\":\"\",\"mainbottom_title_color\":\"\",\"mainbottom_link_color\":\"\",\"mainbottom_link_hover_color\":\"\",\"bottom_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"bottom_bg_image_overlay\":\"fav-transparent\",\"bottom_bg_color\":\"\",\"bottom_color\":\"\",\"bottom_title_color\":\"\",\"bottom_link_color\":\"\",\"bottom_link_hover_color\":\"\",\"note_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"note_bg_image_overlay\":\"fav-transparent\",\"note_bg_color\":\"\",\"note_color\":\"\",\"note_title_color\":\"\",\"note_link_color\":\"\",\"note_link_hover_color\":\"\",\"base_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"base_bg_image_overlay\":\"fav-overlay\",\"base_bg_color\":\"\",\"base_color\":\"\",\"base_title_color\":\"\",\"base_link_color\":\"\",\"base_link_hover_color\":\"\",\"block_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"block_bg_image_overlay\":\"fav-transparent\",\"block_bg_color\":\"\",\"block_color\":\"\",\"block_title_color\":\"\",\"block_link_color\":\"\",\"block_link_hover_color\":\"\",\"user_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"user_bg_image_overlay\":\"fav-transparent\",\"user_bg_color\":\"\",\"user_color\":\"\",\"user_title_color\":\"\",\"user_link_color\":\"\",\"user_link_hover_color\":\"\",\"footer_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"footer_bg_image_overlay\":\"fav-transparent\",\"footer_bg_color\":\"\",\"footer_color\":\"\",\"footer_title_color\":\"\",\"footer_link_color\":\"\",\"footer_link_hover_color\":\"\",\"show_default_logo\":\"1\",\"default_logo\":\"logo.png\",\"default_logo_img_alt\":\"Favourite template\",\"default_logo_padding\":\"\",\"default_logo_margin\":\"\",\"show_media_logo\":\"0\",\"media_logo_img_alt\":\"Favourite template\",\"media_logo_padding\":\"\",\"media_logo_margin\":\"\",\"show_text_logo\":\"0\",\"text_logo\":\"Favourite\",\"text_logo_color\":\"\",\"text_logo_font_size\":\"\",\"text_logo_google_font\":\"\",\"text_logo_google_font_weight\":\"400\",\"text_logo_google_font_style\":\"normal\",\"text_logo_line_height\":\"\",\"text_logo_padding\":\"\",\"text_logo_margin\":\"\",\"show_slogan\":\"0\",\"slogan\":\"slogan text here\",\"slogan_color\":\"\",\"slogan_font_size\":\"\",\"slogan_line_height\":\"\",\"slogan_padding\":\"\",\"slogan_margin\":\"\",\"show_retina_logo\":\"0\",\"retina_logo_height\":\"52px\",\"retina_logo_width\":\"188px\",\"retina_logo_img_alt\":\"Favourite template\",\"retina_logo_padding\":\"0px\",\"retina_logo_margin\":\"0px\",\"mobile_nav_color\":\"navbar\",\"show_mobile_submenu\":\"none\",\"show_mobile_menu_text\":\"1\",\"mobile_menu_text\":\"Menu\",\"mobile_show_images\":\"inline-base\",\"paragraph_mobile_font_size\":\"\",\"article_mobile_title_font_size\":\"\",\"module_mobile_title_font_size\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(10187,0,'plg_editors-xtd_module','plugin','module','editors-xtd',0,0,1,0,'{\"name\":\"plg_editors-xtd_module\",\"type\":\"plugin\",\"creationDate\":\"October 2015\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2015 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.5.0\",\"description\":\"PLG_MODULE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"module\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10188,0,'plg_system_stats','plugin','stats','system',0,0,1,0,'{\"name\":\"plg_system_stats\",\"type\":\"plugin\",\"creationDate\":\"November 2013\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2013 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.5.0\",\"description\":\"PLG_SYSTEM_STATS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"stats\"}','{\"interval\":\"12\",\"mode\":\"1\",\"lastrun\":\"\",\"debug\":\"0\"}','','',0,'0000-00-00 00:00:00',4,0),(10189,0,'plg_system_updatenotification','plugin','updatenotification','system',0,0,1,0,'{\"name\":\"plg_system_updatenotification\",\"type\":\"plugin\",\"creationDate\":\"May 2015\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2015 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.5.0\",\"description\":\"PLG_SYSTEM_UPDATENOTIFICATION_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"updatenotification\"}','{\"email\":\"\",\"language_override\":\"\",\"lastrun\":\"0\"}','','',0,'0000-00-00 00:00:00',8,0),(10190,0,'PRO Twitter Widget','module','mod_pro_twitter_widget','',0,1,0,0,'{\"name\":\"PRO Twitter Widget\",\"type\":\"module\",\"creationDate\":\"January 2016\",\"author\":\"Joe Morrison\",\"copyright\":\"Copyright (C) 2016. All rights reserved by highschooldiplomaonlinefast.com.\",\"authorEmail\":\"joemorrisonyt@yahoo.com\",\"authorUrl\":\"http:\\/\\/www.highschooldiplomaonlinefast.com\\/extensions\",\"version\":\"1.0\",\"description\":\"Thanks for installing PRO Twitter Widget.\",\"group\":\"\",\"filename\":\"mod_pro_twitter_widget\"}','{\"twitter_id\":\"470475991895138304\",\"twitter_name\":\"BarackObama\",\"width\":\"300\",\"height\":\"150px\",\"scrollbar\":\"true\",\"color_scheme\":\"0\",\"backgroundColor\":\"#f5784f\",\"header\":\"true\",\"footer\":\"true\",\"border\":\"true\",\"tranparent\":\"false\",\"backgroundColors\":\"#e6e1e2\"}','','',0,'0000-00-00 00:00:00',0,0),(10193,10200,'COM_JCE','component','com_jce','',1,1,0,0,'{\"name\":\"COM_JCE\",\"type\":\"component\",\"creationDate\":\"24-09-2024\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2024 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"www.joomlacontenteditor.net\",\"version\":\"2.9.81\",\"description\":\"COM_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"jce\"}','{\"help_url\":\"http:\\/\\/www.joomlacontenteditor.net\",\"feed\":\"0\",\"feed_limit\":\"2\",\"browser_width\":\"780\",\"browser_height\":\"560\",\"replace_media_manager\":\"1\",\"secureparams\":\"0\",\"updates_key\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(10194,10200,'plg_editors_jce','plugin','jce','editors',0,1,1,0,'{\"name\":\"plg_editors_jce\",\"type\":\"plugin\",\"creationDate\":\"24-09-2024\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2024 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"http:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.9.81\",\"description\":\"WF_EDITOR_PLUGIN_DESC\",\"group\":\"\",\"filename\":\"jce\"}','{\"verify_html\":\"1\",\"schema\":\"mixed\",\"entity_encoding\":\"raw\",\"keep_nbsp\":\"0\",\"pad_empty_tags\":\"0\",\"cleanup_pluginmode\":\"0\",\"forced_root_block\":\"p\",\"content_style_reset\":\"0\",\"content_css\":\"1\",\"content_css_custom\":\"\",\"body_class\":\"\",\"compress_javascript\":\"0\",\"compress_css\":\"0\",\"compress_gzip\":\"0\",\"custom_config\":\"\",\"callback_file\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(10196,10200,'plg_extension_jce','plugin','jce','extension',0,1,1,0,'{\"name\":\"plg_extension_jce\",\"type\":\"plugin\",\"creationDate\":\"24-09-2024\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2024 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"http:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.9.81\",\"description\":\"PLG_EXTENSION_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"jce\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10197,10200,'plg_installer_jce','plugin','jce','installer',0,1,1,0,'{\"name\":\"plg_installer_jce\",\"type\":\"plugin\",\"creationDate\":\"24-09-2024\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2024 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"http:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.9.81\",\"description\":\"PLG_INSTALLER_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"jce\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10198,10200,'plg_quickicon_jce','plugin','jce','quickicon',0,1,1,0,'{\"name\":\"plg_quickicon_jce\",\"type\":\"plugin\",\"creationDate\":\"24-09-2024\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2024 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"http:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.9.81\",\"description\":\"PLG_QUICKICON_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"jce\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10199,10200,'plg_system_jce','plugin','jce','system',0,1,1,0,'{\"name\":\"plg_system_jce\",\"type\":\"plugin\",\"creationDate\":\"24-09-2024\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2024 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"http:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.9.81\",\"description\":\"PLG_SYSTEM_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"jce\"}','{}','','',0,'0000-00-00 00:00:00',11,0),(10200,0,'PKG_JCE','package','pkg_jce','',0,1,1,0,'{\"name\":\"PKG_JCE\",\"type\":\"package\",\"creationDate\":\"24-09-2024\",\"author\":\"Ryan Demmer\",\"copyright\":\"\",\"authorEmail\":\"\",\"authorUrl\":\"\",\"version\":\"2.9.81\",\"description\":\"PKG_JCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"pkg_jce\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10208,0,'COM_EASYJOOMLABACKUP','component','com_easyjoomlabackup','',1,1,0,0,'{\"name\":\"COM_EASYJOOMLABACKUP\",\"type\":\"component\",\"creationDate\":\"2021-09-09\",\"author\":\"Viktor Vogel\",\"copyright\":\"Copyright 2021 Viktor Vogel. All rights reserved.\",\"authorEmail\":\"admin@kubik-rubik.de\",\"authorUrl\":\"https:\\/\\/kubik-rubik.de\\/\",\"version\":\"3.4.1.0-FREE\",\"description\":\"COM_EASYJOOMLABACKUP_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"easyjoomlabackup\"}','{\"exclude_files\":\".DS_Store\\r\\n.gitignore\",\"exclude_folders\":\".git\\r\\n.idea\\r\\nehld-icons\\r\\ndownload\\r\\nfigures\\r\\nimg-tmp\\r\\ncgi-tmp\",\"add_drop_statement\":\"1\",\"add_db_tables\":\"\",\"max_number_backups\":8,\"prefix_archive\":\"\",\"add_suffix_archive\":\"1\",\"donation_code\":\"\"}','','',0,'0000-00-00 00:00:00',0,0),(10209,0,'PLG_SYSTEM_EASYJOOMLABACKUPCRONJOB','plugin','easyjoomlabackupcronjob','system',0,1,1,0,'{\"name\":\"PLG_SYSTEM_EASYJOOMLABACKUPCRONJOB\",\"type\":\"plugin\",\"creationDate\":\"2021-09-09\",\"author\":\"Viktor Vogel\",\"copyright\":\"Copyright 2021 Viktor Vogel. All rights reserved.\",\"authorEmail\":\"\",\"authorUrl\":\"\",\"version\":\"3.4.1.0-FREE\",\"description\":\"PLG_SYSTEM_EASYJOOMLABACKUPCRONJOB_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"easyjoomlabackupcronjob\"}','{\"token\":\"\",\"type\":\"1\",\"donation_code\":\"\"}','','',0,'0000-00-00 00:00:00',14,0),(10220,10226,'plg_installer_jeventsinstaller','plugin','jeventsinstaller','installer',0,1,1,0,'{\"name\":\"plg_installer_jeventsinstaller\",\"type\":\"plugin\",\"creationDate\":\"April 2017\",\"author\":\"GWE Systems Ltd\",\"copyright\":\"(C) 2016 GWE Systems Ltd. All rights reserved.\",\"authorEmail\":\"via website\",\"authorUrl\":\"www.gwesystems.com\",\"version\":\"3.4.36\",\"description\":\"PLG_INSTALLER_JEVENTSINSTALLER\",\"group\":\"\",\"filename\":\"jeventsinstaller\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10225,10441,'PLG_SYSTEM_REGULARLABS','plugin','regularlabs','system',0,1,1,0,'{\"name\":\"PLG_SYSTEM_REGULARLABS\",\"type\":\"plugin\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"23.9.3039\",\"description\":\"PLG_SYSTEM_REGULARLABS_DESC\",\"group\":\"\",\"filename\":\"regularlabs\"}','{\"combine_admin_menu\":\"0\",\"show_help_menu\":\"1\",\"max_list_count\":\"2500\"}','','',0,'0000-00-00 00:00:00',7,0),(10226,0,'PLG_EDITORS-XTD_TABS','plugin','tabs','editors-xtd',0,1,1,0,'{\"name\":\"PLG_EDITORS-XTD_TABS\",\"type\":\"plugin\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"8.4.0\",\"description\":\"PLG_EDITORS-XTD_TABS_DESC\",\"group\":\"\",\"filename\":\"tabs\"}','[]','','',0,'0000-00-00 00:00:00',0,0),(10227,0,'PLG_SYSTEM_TABS','plugin','tabs','system',0,1,1,0,'{\"name\":\"PLG_SYSTEM_TABS\",\"type\":\"plugin\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"8.4.0\",\"description\":\"PLG_SYSTEM_TABS_DESC\",\"group\":\"\",\"filename\":\"tabs\"}','{\"load_stylesheet\":\"1\",\"mainclass\":\"\",\"alignment\":\"\",\"color_inactive_handles\":\"0\",\"outline_handles\":\"1\",\"outline_content\":\"1\",\"tag_open\":\"tab\",\"tag_close\":\"tabs\",\"tag_delimiter\":\"space\",\"tag_characters\":\"{.}\",\"title_tag\":\"h2\",\"use_hash\":\"0\",\"reload_iframes\":\"0\",\"init_timeout\":\"0\",\"enable_admin\":\"0\",\"place_comments\":\"1\",\"media_versioning\":\"1\",\"load_bootstrap_framework\":\"1\",\"load_jquery\":\"0\",\"button_text\":\"Tabs\",\"enable_frontend\":\"1\",\"button_use_simple_button\":\"0\",\"button_max_count\":\"20\",\"button_use_custom_code\":\"0\",\"button_custom_code\":\"{tab Tab Title 1}<br \\/>[:SELECTION:]<br \\/>{tab Tab Title 2}<br \\/>Tab text...<br \\/>{\\/tabs}\"}','','',0,'0000-00-00 00:00:00',3,0),(10230,0,'PLG_EDITORS-XTD_SLIDERS','plugin','sliders','editors-xtd',0,1,1,0,'{\"name\":\"PLG_EDITORS-XTD_SLIDERS\",\"type\":\"plugin\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"8.4.0\",\"description\":\"PLG_EDITORS-XTD_SLIDERS_DESC\",\"group\":\"\",\"filename\":\"sliders\"}','[]','','',0,'0000-00-00 00:00:00',0,0),(10231,0,'PLG_SYSTEM_SLIDERS','plugin','sliders','system',0,1,1,0,'{\"name\":\"PLG_SYSTEM_SLIDERS\",\"type\":\"plugin\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"8.4.0\",\"description\":\"PLG_SYSTEM_SLIDERS_DESC\",\"group\":\"\",\"filename\":\"sliders\"}','{\"load_stylesheet\":\"1\",\"mainclass\":\"\",\"state_first\":\"open\",\"tag_open\":\"slider\",\"tag_close\":\"sliders\",\"tag_delimiter\":\"space\",\"tag_characters\":\"{.}\",\"title_tag\":\"h2\",\"use_hash\":\"0\",\"reload_iframes\":\"0\",\"init_timeout\":\"0\",\"enable_admin\":\"0\",\"place_comments\":\"1\",\"media_versioning\":\"1\",\"load_bootstrap_framework\":\"1\",\"load_jquery\":\"0\",\"button_text\":\"Sliders\",\"enable_frontend\":\"1\",\"button_use_simple_button\":\"0\",\"button_max_count\":\"20\",\"button_use_custom_code\":\"0\",\"button_custom_code\":\"{slider Slider Title 1}<br \\/>[:SELECTION:]<br \\/>{slider Slider Title 2}<br \\/>Slider text...<br \\/>{\\/sliders}\"}','','',0,'0000-00-00 00:00:00',6,0),(10232,0,'plg_system_directalias','plugin','directalias','system',0,1,1,0,'{\"name\":\"plg_system_directalias\",\"type\":\"plugin\",\"creationDate\":\"09\\/10\\/2020\",\"author\":\"AlterBrains\",\"copyright\":\"(C) 2013-2020 AlterBrains. All rights reserved.\",\"authorEmail\":\"alterbrains@gmail.com\",\"authorUrl\":\"http:\\/\\/alterbrains.com\",\"version\":\"2.1.0\",\"description\":\"PLG_SYSTEM_DIRECTALIAS_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"directalias\"}','{\"shorten_all\":\"0\"}','','',0,'0000-00-00 00:00:00',13,0),(10234,0,'Footer Menu','module','mod_footermenu','',0,1,0,0,'{\"name\":\"Footer Menu\",\"type\":\"module\",\"creationDate\":\"January 2015\",\"author\":\"KMO Design\",\"copyright\":\"Copyright (C) 2013-2014 KMO Design Pty Ltd. All rights reserved.\",\"authorEmail\":\"support@kmo.com.au\",\"authorUrl\":\"www.kmo.com.au\",\"version\":\"1.0.1\",\"description\":\"MOD_FOOTERMENU_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_footermenu\"}','{\"startLevel\":\"1\",\"endLevel\":\"0\",\"showAllChildren\":\"0\",\"use_default_stylesheet\":\"0\",\"columns\":\"0\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"itemid\"}','','',0,'0000-00-00 00:00:00',0,0),(10235,0,'System - JU Sticky Panel','plugin','justickypanel','system',0,0,1,0,'{\"name\":\"System - JU Sticky Panel\",\"type\":\"plugin\",\"creationDate\":\"Mar 2014\",\"author\":\"JoomUltra\",\"copyright\":\"Copyright (C) 2010 - 2014 JoomUltra. All rights reserved.\",\"authorEmail\":\"admin@joomultra.com\",\"authorUrl\":\"www.joomultra.com\",\"version\":\"1.2\",\"description\":\"\\n\\t\\t\\n\\t\\t<div style=\'font-weight: normal\'>\\n\\t\\t<h2 style=\'color: #0A71A5;\'>JU Sticky Panel Plugin<\\/strong><\\/h2>\\n\\t\\t<span style=\'color: #ff0000;\'><strong>Features:<\\/strong><\\/span>\\n\\t\\t<br\\/><br\\/>\\n\\t\\tStick any element in your site, the sticked elements will be keep always visible when you scroll the page down.\\n\\t\\t<br\\/><br\\/>\\n\\t\\t<span style=\'color: #ff0000;\'><strong>Usage:<\\/strong><\\/span>\\n\\t\\t<br\\/><br\\/>\\n\\t\\tYou can use <a href=\\\"http:\\/\\/getfirebug.com\\/\\\" title=\\\"Download Firebug\\\" target=\\\"_blank\\\">Firebug<\\/a> on Firefox to inspect the element you want to stick, then put the class or id or name,...<i>(See <a href=\\\"http:\\/\\/api.jquery.com\\/category\\/selectors\\/\\\" title=\\\"Read more about jQuery Selectors\\\" target=\\\"_blank\\\">jQuery selectors<\\/a>)<\\/i> of elements into the selectors config.\\n\\t\\tYou can put as many selectors as you want, seperate by commas, all elements with selectors you put in selectors config will be sticked by JU Sticky Panel.\\n\\t\\t<br\\/><br\\/>\\n\\t\\tHere is an example how to stick a module:<br\\/>\\n\\t\\t<ul>\\n\\t\\t\\t<li>1. In the module config, set <b>Module class suffix:<\\/b> \\\"<b> sticky<\\/b>\\\" <i>(without quote, and remember we have a white space preceded)<\\/i><\\/li>\\n\\t\\t\\t<li>2. In the JU Sticky Panel Plugin set <b>Selectors:<\\/b> <b>.sticky<\\/b><\\/li>\\n\\t\\t\\t<li>3. That\'s all, now check the frontend, scroll down your site, you will see the module with class suffix \\\"<b> sticky<\\/b>\\\" is ticked, and when it has been sticking it will be add new class in <b>After detach css class<\\/b>, you can use this class to format sticked element if you want. And now you can adjust other config in the plugin to fit your needs. Remember that JU Sticky Panel support multi selectors, that means you can stick many elements in your site.<\\/li>\\n\\t\\t<\\/ul>\\n\\t\\t<br\\/><br\\/>\\n\\t\\t<strong><span style=\'color: #ff0000;\'>Requirements:<\\/span><\\/strong>\\n\\t\\t<br\\/><br \\/>\\n\\t\\tJoomla! 2.5.x, Joomla! 3.x<br\\/>\\n\\t\\tThis plugin require jQuery javascript library to work. If your site gets jQuery conflict issue, you should install <b>Plugin <a href=\\\"http:\\/\\/extensions.joomla.org\\/extensions\\/core-enhancements\\/performance\\/jquery-scripts\\/18327\\\" target=\\\"_blank\\\" title=\\\"Download jQuery Easy\\\">jQuery Easy<\\/a><\\/b> <i>(Remember activate this plugin after installing it)<\\/i>.<br\\/><br\\/>\\n\\t\\tIf you use JU Sticky Panel, please post a review for <a href=\\\"http:\\/\\/extensions.joomla.org\\/extensions\\/style-a-design\\/modules-panel\\/25232\\\" target=\\\"_blank\\\" title=\\\"Post a review for JU Sticky Panel\\\">JU Sticky Panel on JED<\\/a>\\n\\t\\t<p>Copyright 2010 - 2014 <a href=\\\"http:\\/\\/www.joomultra.com\\/\\\" title=\\\"Visit JoomUltra.com!\\\" target=\\\"_blank\\\">JoomUltra.com<\\/a>.<\\/p>\\n\\t\\t<\\/div>\\n\\t\\t\\n\\t\",\"group\":\"\",\"filename\":\"justickypanel\"}','{\"loadjquery\":\"2\",\"loadjqueryfrom\":\"1\",\"selectors\":\".sticky\",\"topPadding\":\"0\",\"afterDetachCSSClass\":\"detached\",\"savePanelSpace\":\"true\",\"parentSelector\":\"\",\"onDetached\":\"null\",\"onReAttached\":\"null\",\"enablepaths\":\"\",\"disablepaths\":\"\"}','','',0,'0000-00-00 00:00:00',9,0),(10238,10445,'PLG_EDITORS-XTD_SOURCERER','plugin','sourcerer','editors-xtd',0,1,1,0,'{\"name\":\"PLG_EDITORS-XTD_SOURCERER\",\"type\":\"plugin\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"9.8.0\",\"description\":\"PLG_EDITORS-XTD_SOURCERER_DESC\",\"group\":\"\",\"filename\":\"sourcerer\"}','[]','','',0,'0000-00-00 00:00:00',0,0),(10239,10445,'PLG_SYSTEM_SOURCERER','plugin','sourcerer','system',0,1,1,0,'{\"name\":\"PLG_SYSTEM_SOURCERER\",\"type\":\"plugin\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"9.8.0\",\"description\":\"PLG_SYSTEM_SOURCERER_DESC\",\"group\":\"\",\"filename\":\"sourcerer\"}','{\"syntax_word\":\"source\",\"tag_characters\":\"{.}\",\"trim\":\"0\",\"enable_in_head\":\"0\",\"remove_from_search\":\"0\",\"include_path\":\"\\/\",\"place_comments\":\"1\",\"enable_css\":\"1\",\"enable_js\":\"1\",\"enable_php\":\"1\",\"forbidden_php\":\"dl, escapeshellarg, escapeshellcmd, exec, passthru, popen, proc_close, proc_open, shell_exec, symlink, system\",\"forbidden_tags\":\"\",\"@notice_articles_security_level\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_articles_enable_css\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_articles_enable_js\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_articles_enable_php\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_components_enable_css\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_components_enable_js\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_components_enable_php\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@wizard\":\"0\",\"@noticeother_enable_css\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_other_enable_js\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"@notice_other_enable_php\":\"RL_ONLY_AVAILABLE_IN_PRO\",\"button_text\":\"Code\",\"enable_frontend\":\"1\",\"addsourcetags\":\"1\",\"@notice_use_example_code\":\"RL_ONLY_AVAILABLE_IN_PRO\"}','','',0,'0000-00-00 00:00:00',5,0),(10254,10442,'PLG_SYSTEM_MODALS','plugin','modals','system',0,1,1,0,'{\"name\":\"PLG_SYSTEM_MODALS\",\"type\":\"plugin\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"12.6.1\",\"description\":\"PLG_SYSTEM_MODALS_DESC\",\"group\":\"\",\"filename\":\"modals\"}','{\"load_stylesheet\":\"1\",\"style\":\"bootstrap\",\"autoconvert_classnames\":\"1\",\"classnames\":\"modal\",\"opacity\":\"0.8\",\"width\":\"\",\"height\":\"\",\"externalwidth\":\"95%\",\"externalheight\":\"95%\",\"initialwidth\":\"600\",\"initialheight\":\"450\",\"maxwidth\":\"95%\",\"maxheight\":\"95%\",\"tag\":\"modal\",\"tag_characters\":\"{.}\",\"tmpl\":\"modal\",\"iframe\":\"0\",\"load_head\":\"0\",\"mediafiles\":\"jpg,png,gif,eps,pdf\",\"iframefiles\":\"pdf\",\"exclude_urls\":\"\",\"auto_titles\":\"1\",\"title_case\":\"titlecase\",\"lowercase_words\":\"a,the,to,at,in,with,and,but,or\",\"enable_admin\":\"0\",\"place_comments\":\"1\",\"add_redirect\":\"1\",\"media_versioning\":\"1\",\"load_jquery\":\"1\"}','','',0,'0000-00-00 00:00:00',0,0),(10259,0,'FavPromote4col','module','mod_favpromote4col','',0,1,0,0,'{\"name\":\"FavPromote4col\",\"type\":\"module\",\"creationDate\":\"2013\",\"author\":\"FavThemes\",\"copyright\":\"Copyright (C) FavThemes. All rights reserved.\",\"authorEmail\":\"\",\"authorUrl\":\"http:\\/\\/www.favthemes.com\",\"version\":\"2.1\",\"description\":\"\\n  \\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\"><strong>FavPromote<\\/strong><\\/a> is a free responsive Joomla! module that lets you upload any image and assign it a short description and a title with a Font Awesome icon.\\n    <br\\/>\\n    It works exactly in the same way as the pure FavPromote, however it is fixed in 4 cols or less. It means, if only one column is present it will occupy 3 empty spaces (12\\/4columns).\\n    For more info, please see the <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">demo and documentation<\\/a> for the FavPromote module.\\n    <br\\/><br\\/>\\n\\n    <a href=\\\"http:\\/\\/extensions.favthemes.com\\/favpromote\\\" target=\\\"_blank\\\">\\n      <img src=\\\"..\\/modules\\/mod_favpromote4col\\/theme\\/img\\/favpromote4col.jpg\\\" alt=\\\"FavPromote Responsive Module\\\">\\n    <\\/a>\\n\\n    <br\\/><br\\/>\\n    Copyright &#169; 2012-2017 <a href=\\\"http:\\/\\/www.favthemes.com\\\" target=\\\"_blank\\\" style=\\\"font-weight: bold;\\\">FavThemes<\\/a>.\\n    <br\\/><br\\/>\\n\\n\\n    <link rel=\\\"stylesheet\\\" href=\\\"..\\/modules\\/mod_favpromote4col\\/theme\\/css\\/admin.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/maxcdn.bootstrapcdn.com\\/font-awesome\\/4.7.0\\/css\\/font-awesome.min.css\\\" type=\\\"text\\/css\\\" \\/>\\n    <link rel=\\\"stylesheet\\\" href=\\\"\\/\\/fonts.googleapis.com\\/css?family=Open+Sans:400\\\" type=\\\"text\\/css\\\" \\/>\\n    <script src=\\\"..\\/modules\\/mod_favpromote4col\\/theme\\/js\\/jscolor\\/jscolor.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n    <script src=\\\"..\\/modules\\/mod_favpromote4col\\/theme\\/js\\/admin\\/admin.js\\\" type=\\\"text\\/javascript\\\"><\\/script>\\n\\n  \\n\\t\",\"group\":\"\",\"filename\":\"mod_favpromote4col\"}','{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect1\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_padding\":\"10px 20px\",\"title_font_size\":\"18px\",\"title_line_height\":\"1.4em\",\"title_text_align\":\"left\",\"title_icon_font_size\":\"18px\",\"title_icon_vertical_align\":\"baseline\",\"description_text_font_size\":\"14px\",\"description_text_line_height\":\"1.5em\",\"description_text_align\":\"justify\",\"show_column1\":\"1\",\"column_border_color1\":\"DDDDDD\",\"column_border_radius1\":\"4px\",\"show_image_link1\":\"1\",\"image_link1\":\"\",\"image_target1\":\"blank\",\"image_alt1\":\"\",\"title_text1\":\"Title Text\",\"title_color1\":\"FFFFFF\",\"title_bg_color1\":\"D63E52\",\"show_title_link1\":\"1\",\"title_link1\":\"\",\"title_target1\":\"blank\",\"title_icon1\":\"fa-coffee\",\"description_text1\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color1\":\"444444\",\"show_column2\":\"1\",\"column_border_color2\":\"DDDDDD\",\"column_border_radius2\":\"4px\",\"show_image_link2\":\"1\",\"image_link2\":\"\",\"image_target2\":\"blank\",\"image_alt2\":\"\",\"title_text2\":\"Title Text\",\"title_color2\":\"FFFFFF\",\"title_bg_color2\":\"D63E52\",\"show_title_link2\":\"1\",\"title_link2\":\"\",\"title_target2\":\"blank\",\"title_icon2\":\"fa-puzzle-piece\",\"description_text2\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color2\":\"444444\",\"show_column3\":\"1\",\"column_border_color3\":\"DDDDDD\",\"column_border_radius3\":\"4px\",\"show_image_link3\":\"1\",\"image_link3\":\"\",\"image_target3\":\"blank\",\"image_alt3\":\"\",\"title_text3\":\"Title Text\",\"title_color3\":\"FFFFFF\",\"title_bg_color3\":\"D63E52\",\"show_title_link3\":\"1\",\"title_link3\":\"\",\"title_target3\":\"blank\",\"title_icon3\":\"fa-bookmark\",\"description_text3\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color3\":\"444444\",\"show_column4\":\"1\",\"column_border_color4\":\"DDDDDD\",\"column_border_radius4\":\"4px\",\"show_image_link4\":\"1\",\"image_link4\":\"\",\"image_target4\":\"blank\",\"image_alt4\":\"\",\"title_text4\":\"Title Text\",\"title_color4\":\"FFFFFF\",\"title_bg_color4\":\"D63E52\",\"show_title_link4\":\"1\",\"title_link4\":\"\",\"title_target4\":\"blank\",\"title_icon4\":\"fa-magic\",\"description_text4\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color4\":\"444444\",\"show_column5\":\"0\",\"column_border_color5\":\"DDDDDD\",\"column_border_radius5\":\"4px\",\"show_image_link5\":\"1\",\"image_link5\":\"\",\"image_target5\":\"blank\",\"image_alt5\":\"\",\"title_text5\":\"Title Text\",\"title_color5\":\"FFFFFF\",\"title_bg_color5\":\"D63E52\",\"show_title_link5\":\"1\",\"title_link5\":\"\",\"title_target5\":\"blank\",\"title_icon5\":\"fa-quote-right\",\"description_text5\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color5\":\"444444\",\"show_column6\":\"0\",\"column_border_color6\":\"DDDDDD\",\"column_border_radius6\":\"4px\",\"show_image_link6\":\"1\",\"image_link6\":\"\",\"image_target6\":\"blank\",\"image_alt6\":\"\",\"title_text6\":\"Title Text\",\"title_color6\":\"FFFFFF\",\"title_bg_color6\":\"D63E52\",\"show_title_link6\":\"1\",\"title_link6\":\"\",\"title_target6\":\"blank\",\"title_icon6\":\"fa-recycle\",\"description_text6\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color6\":\"444444\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\"}','','',0,'0000-00-00 00:00:00',0,0),(10269,10442,'PLG_EDITORS-XTD_MODALS','plugin','modals','editors-xtd',0,1,1,0,'{\"name\":\"PLG_EDITORS-XTD_MODALS\",\"type\":\"plugin\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"12.6.1\",\"description\":\"PLG_EDITORS-XTD_MODALS_DESC\",\"group\":\"\",\"filename\":\"modals\"}','[]','','',0,'0000-00-00 00:00:00',0,0),(10302,0,'mod_jsocialfeed','module','mod_jsocialfeed','',0,0,1,0,'{\"name\":\"mod_jsocialfeed\",\"type\":\"module\",\"creationDate\":\"03\\/06\\/2014\",\"author\":\"Garda Informatica\",\"copyright\":\"Garda Informatica\",\"authorEmail\":\"info@gardainformatica.it\",\"authorUrl\":\"http:\\/\\/www.gardainformatica.it\\/\",\"version\":\"1.5.4\",\"description\":\"MOD_JSOCIALFEED_DESCRIPTION\",\"group\":\"\",\"filename\":\"mod_jsocialfeed\"}','{}','','',0,'0000-00-00 00:00:00',0,-1),(10303,0,'jsocialfeed','library','jsocialfeed','',0,0,1,0,'{\"name\":\"JSocialFeed\",\"type\":\"library\",\"creationDate\":\"03\\/06\\/2014\",\"author\":\"Garda Informatica\",\"copyright\":\"Garda Informatica\",\"authorEmail\":\"info@gardainformatica.it\",\"authorUrl\":\"http:\\/\\/www.gardainformatica.it\\/\",\"version\":\"1.5.4\",\"description\":\"\",\"group\":\"\",\"filename\":\"jsocialfeed\"}','{}','','',0,'0000-00-00 00:00:00',0,-1),(10304,0,'com_jsocialfeed','component','com_jsocialfeed','',1,0,1,0,'{\"name\":\"COM_JSOCIALFEED\",\"type\":\"component\",\"creationDate\":\"03\\/06\\/2014\",\"author\":\"Garda Informatica\",\"copyright\":\"Garda Informatica\",\"authorEmail\":\"info@gardainformatica.it\",\"authorUrl\":\"http:\\/\\/www.gardainformatica.it\\/\",\"version\":\"1.5.4\",\"description\":\"COM_JSOCIALFEED_DESCRIPTION\",\"group\":\"\",\"filename\":\"jsocialfeed\"}','{}','','',0,'0000-00-00 00:00:00',0,-1),(10307,10200,'plg_fields_mediajce','plugin','mediajce','fields',0,1,1,0,'{\"name\":\"plg_fields_mediajce\",\"type\":\"plugin\",\"creationDate\":\"24-09-2024\",\"author\":\"Ryan Demmer\",\"copyright\":\"Copyright (C) 2006 - 2024 Ryan Demmer. All rights reserved\",\"authorEmail\":\"info@joomlacontenteditor.net\",\"authorUrl\":\"https:\\/\\/www.joomlacontenteditor.net\",\"version\":\"2.9.81\",\"description\":\"PLG_FIELDS_MEDIAJCE_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"mediajce\"}','[]','','',0,'0000-00-00 00:00:00',0,0),(10417,0,'plg_quickicon_eos310','plugin','eos310','quickicon',0,1,1,0,'{\"name\":\"plg_quickicon_eos310\",\"type\":\"plugin\",\"creationDate\":\"June 2021\",\"author\":\"Joomla! Project\",\"copyright\":\"(C) 2021 Open Source Matters, Inc.\",\"authorEmail\":\"admin@joomla.org\",\"authorUrl\":\"www.joomla.org\",\"version\":\"3.10.0\",\"description\":\"PLG_QUICKICON_EOS310_XML_DESCRIPTION\",\"group\":\"\",\"filename\":\"eos310\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10441,0,'PKG_REGULARLABS','package','pkg_regularlabs','',0,1,1,0,'{\"name\":\"PKG_REGULARLABS\",\"type\":\"package\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"23.9.3039\",\"description\":\"\",\"group\":\"\",\"filename\":\"pkg_regularlabs\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10442,0,'PKG_MODALS','package','pkg_modals','',0,1,1,0,'{\"name\":\"PKG_MODALS\",\"type\":\"package\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"12.6.1\",\"description\":\"\",\"group\":\"\",\"filename\":\"pkg_modals\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10444,0,'PKG_SLIDERS','package','pkg_sliders','',0,1,1,0,'{\"name\":\"PKG_SLIDERS\",\"type\":\"package\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"8.4.0\",\"description\":\"\",\"group\":\"\",\"filename\":\"pkg_sliders\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10445,0,'PKG_SOURCERER','package','pkg_sourcerer','',0,1,1,0,'{\"name\":\"PKG_SOURCERER\",\"type\":\"package\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"9.8.0\",\"description\":\"\",\"group\":\"\",\"filename\":\"pkg_sourcerer\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10447,0,'PKG_TABS','package','pkg_tabs','',0,1,1,0,'{\"name\":\"PKG_TABS\",\"type\":\"package\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"8.4.0\",\"description\":\"\",\"group\":\"\",\"filename\":\"pkg_tabs\"}','{}','','',0,'0000-00-00 00:00:00',0,0),(10465,10441,'Regular Labs Library','library','regularlabs','',0,1,1,0,'{\"name\":\"Regular Labs Library\",\"type\":\"library\",\"creationDate\":\"September 2023\",\"author\":\"Regular Labs (Peter van Westen)\",\"copyright\":\"Copyright \\u00a9 2023 Regular Labs - All Rights Reserved\",\"authorEmail\":\"info@regularlabs.com\",\"authorUrl\":\"https:\\/\\/regularlabs.com\",\"version\":\"23.9.3039\",\"description\":\"Regular Labs Library - used by Regular Labs extensions\",\"group\":\"\",\"filename\":\"regularlabs\"}','{}','','',0,'0000-00-00 00:00:00',0,0);
/*!40000 ALTER TABLE `rlp_extensions` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_fields`
--

DROP TABLE IF EXISTS `rlp_fields`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_fields` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `asset_id` int unsigned NOT NULL DEFAULT '0',
  `context` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `group_id` int unsigned NOT NULL DEFAULT '0',
  `title` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `name` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `label` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `default_value` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `type` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT 'text',
  `note` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `description` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `state` tinyint(1) NOT NULL DEFAULT '0',
  `required` tinyint(1) NOT NULL DEFAULT '0',
  `checked_out` int NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `ordering` int NOT NULL DEFAULT '0',
  `params` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `fieldparams` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `language` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `created_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_user_id` int unsigned NOT NULL DEFAULT '0',
  `modified_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int unsigned NOT NULL DEFAULT '0',
  `access` int NOT NULL DEFAULT '1',
  PRIMARY KEY (`id`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_state` (`state`),
  KEY `idx_created_user_id` (`created_user_id`),
  KEY `idx_access` (`access`),
  KEY `idx_language` (`language`),
  KEY `idx_context` (`context`(191))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_fields`
--

LOCK TABLES `rlp_fields` WRITE;
/*!40000 ALTER TABLE `rlp_fields` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_fields` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_fields_categories`
--

DROP TABLE IF EXISTS `rlp_fields_categories`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_fields_categories` (
  `field_id` int NOT NULL DEFAULT '0',
  `category_id` int NOT NULL DEFAULT '0',
  PRIMARY KEY (`field_id`,`category_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_fields_categories`
--

LOCK TABLES `rlp_fields_categories` WRITE;
/*!40000 ALTER TABLE `rlp_fields_categories` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_fields_categories` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_fields_groups`
--

DROP TABLE IF EXISTS `rlp_fields_groups`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_fields_groups` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `asset_id` int unsigned NOT NULL DEFAULT '0',
  `context` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `title` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `note` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `description` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `state` tinyint(1) NOT NULL DEFAULT '0',
  `checked_out` int NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `ordering` int NOT NULL DEFAULT '0',
  `params` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `language` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by` int unsigned NOT NULL DEFAULT '0',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int unsigned NOT NULL DEFAULT '0',
  `access` int NOT NULL DEFAULT '1',
  PRIMARY KEY (`id`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_state` (`state`),
  KEY `idx_created_by` (`created_by`),
  KEY `idx_access` (`access`),
  KEY `idx_language` (`language`),
  KEY `idx_context` (`context`(191))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_fields_groups`
--

LOCK TABLES `rlp_fields_groups` WRITE;
/*!40000 ALTER TABLE `rlp_fields_groups` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_fields_groups` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_fields_values`
--

DROP TABLE IF EXISTS `rlp_fields_values`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_fields_values` (
  `field_id` int unsigned NOT NULL,
  `item_id` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'Allow references to items which have strings as ids, eg. none db systems.',
  `value` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  KEY `idx_field_id` (`field_id`),
  KEY `idx_item_id` (`item_id`(191))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_fields_values`
--

LOCK TABLES `rlp_fields_values` WRITE;
/*!40000 ALTER TABLE `rlp_fields_values` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_fields_values` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_filters`
--

DROP TABLE IF EXISTS `rlp_finder_filters`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_filters` (
  `filter_id` int unsigned NOT NULL AUTO_INCREMENT,
  `title` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `alias` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `state` tinyint(1) NOT NULL DEFAULT '1',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by` int unsigned NOT NULL,
  `created_by_alias` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int unsigned NOT NULL DEFAULT '0',
  `checked_out` int unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `map_count` int unsigned NOT NULL DEFAULT '0',
  `data` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `params` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci,
  PRIMARY KEY (`filter_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_filters`
--

LOCK TABLES `rlp_finder_filters` WRITE;
/*!40000 ALTER TABLE `rlp_finder_filters` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_filters` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links`
--

DROP TABLE IF EXISTS `rlp_finder_links`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links` (
  `link_id` int unsigned NOT NULL AUTO_INCREMENT,
  `url` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `route` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `title` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci DEFAULT NULL,
  `description` text CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci,
  `indexdate` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `md5sum` varchar(32) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci DEFAULT NULL,
  `published` tinyint(1) NOT NULL DEFAULT '1',
  `state` int DEFAULT '1',
  `access` int DEFAULT '0',
  `language` varchar(8) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `publish_start_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_end_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `start_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `end_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `list_price` double unsigned NOT NULL DEFAULT '0',
  `sale_price` double unsigned NOT NULL DEFAULT '0',
  `type_id` int NOT NULL,
  `object` mediumblob NOT NULL,
  PRIMARY KEY (`link_id`),
  KEY `idx_type` (`type_id`),
  KEY `idx_md5` (`md5sum`),
  KEY `idx_url` (`url`(75)),
  KEY `idx_published_list` (`published`,`state`,`access`,`publish_start_date`,`publish_end_date`,`list_price`),
  KEY `idx_published_sale` (`published`,`state`,`access`,`publish_start_date`,`publish_end_date`,`sale_price`),
  KEY `idx_title` (`title`(100))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links`
--

LOCK TABLES `rlp_finder_links` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms0`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms0`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_terms0` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms0`
--

LOCK TABLES `rlp_finder_links_terms0` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms0` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms0` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms1`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms1`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_terms1` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms1`
--

LOCK TABLES `rlp_finder_links_terms1` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms1` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms1` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms2`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms2`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_terms2` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms2`
--

LOCK TABLES `rlp_finder_links_terms2` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms2` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms2` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms3`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms3`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_terms3` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms3`
--

LOCK TABLES `rlp_finder_links_terms3` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms3` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms3` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms4`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms4`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_terms4` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms4`
--

LOCK TABLES `rlp_finder_links_terms4` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms4` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms4` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms5`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms5`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_terms5` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms5`
--

LOCK TABLES `rlp_finder_links_terms5` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms5` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms5` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms6`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms6`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_terms6` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms6`
--

LOCK TABLES `rlp_finder_links_terms6` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms6` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms6` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms7`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms7`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_terms7` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms7`
--

LOCK TABLES `rlp_finder_links_terms7` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms7` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms7` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms8`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms8`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_terms8` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms8`
--

LOCK TABLES `rlp_finder_links_terms8` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms8` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms8` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_terms9`
--

DROP TABLE IF EXISTS `rlp_finder_links_terms9`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_terms9` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_terms9`
--

LOCK TABLES `rlp_finder_links_terms9` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_terms9` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_terms9` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termsa`
--

DROP TABLE IF EXISTS `rlp_finder_links_termsa`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_termsa` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termsa`
--

LOCK TABLES `rlp_finder_links_termsa` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termsa` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termsa` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termsb`
--

DROP TABLE IF EXISTS `rlp_finder_links_termsb`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_termsb` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termsb`
--

LOCK TABLES `rlp_finder_links_termsb` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termsb` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termsb` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termsc`
--

DROP TABLE IF EXISTS `rlp_finder_links_termsc`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_termsc` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termsc`
--

LOCK TABLES `rlp_finder_links_termsc` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termsc` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termsc` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termsd`
--

DROP TABLE IF EXISTS `rlp_finder_links_termsd`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_termsd` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termsd`
--

LOCK TABLES `rlp_finder_links_termsd` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termsd` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termsd` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termse`
--

DROP TABLE IF EXISTS `rlp_finder_links_termse`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_termse` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termse`
--

LOCK TABLES `rlp_finder_links_termse` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termse` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termse` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_links_termsf`
--

DROP TABLE IF EXISTS `rlp_finder_links_termsf`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_links_termsf` (
  `link_id` int unsigned NOT NULL,
  `term_id` int unsigned NOT NULL,
  `weight` float unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`term_id`),
  KEY `idx_term_weight` (`term_id`,`weight`),
  KEY `idx_link_term_weight` (`link_id`,`term_id`,`weight`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_links_termsf`
--

LOCK TABLES `rlp_finder_links_termsf` WRITE;
/*!40000 ALTER TABLE `rlp_finder_links_termsf` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_links_termsf` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_taxonomy`
--

DROP TABLE IF EXISTS `rlp_finder_taxonomy`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_taxonomy` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `parent_id` int unsigned NOT NULL DEFAULT '0',
  `title` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `state` tinyint unsigned NOT NULL DEFAULT '1',
  `access` tinyint unsigned NOT NULL DEFAULT '0',
  `ordering` tinyint unsigned NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`),
  KEY `parent_id` (`parent_id`),
  KEY `state` (`state`),
  KEY `ordering` (`ordering`),
  KEY `access` (`access`),
  KEY `idx_parent_published` (`parent_id`,`state`,`access`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_taxonomy`
--

LOCK TABLES `rlp_finder_taxonomy` WRITE;
/*!40000 ALTER TABLE `rlp_finder_taxonomy` DISABLE KEYS */;
INSERT INTO `rlp_finder_taxonomy` VALUES (1,1,'Type',1,1,0);
/*!40000 ALTER TABLE `rlp_finder_taxonomy` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_taxonomy_map`
--

DROP TABLE IF EXISTS `rlp_finder_taxonomy_map`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_taxonomy_map` (
  `link_id` int unsigned NOT NULL,
  `node_id` int unsigned NOT NULL,
  PRIMARY KEY (`link_id`,`node_id`),
  KEY `link_id` (`link_id`),
  KEY `node_id` (`node_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_taxonomy_map`
--

LOCK TABLES `rlp_finder_taxonomy_map` WRITE;
/*!40000 ALTER TABLE `rlp_finder_taxonomy_map` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_taxonomy_map` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_terms`
--

DROP TABLE IF EXISTS `rlp_finder_terms`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_terms` (
  `term_id` int unsigned NOT NULL AUTO_INCREMENT,
  `term` varchar(75) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `stem` varchar(75) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `common` tinyint unsigned NOT NULL DEFAULT '0',
  `phrase` tinyint unsigned NOT NULL DEFAULT '0',
  `weight` float unsigned NOT NULL DEFAULT '0',
  `soundex` varchar(75) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `links` int NOT NULL DEFAULT '0',
  `language` char(3) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL DEFAULT '',
  PRIMARY KEY (`term_id`),
  UNIQUE KEY `idx_term` (`term`),
  KEY `idx_term_phrase` (`term`,`phrase`),
  KEY `idx_stem_phrase` (`stem`,`phrase`),
  KEY `idx_soundex_phrase` (`soundex`,`phrase`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_terms`
--

LOCK TABLES `rlp_finder_terms` WRITE;
/*!40000 ALTER TABLE `rlp_finder_terms` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_terms` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_terms_common`
--

DROP TABLE IF EXISTS `rlp_finder_terms_common`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_terms_common` (
  `term` varchar(75) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `language` varchar(3) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  KEY `idx_word_lang` (`term`,`language`),
  KEY `idx_lang` (`language`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_terms_common`
--

LOCK TABLES `rlp_finder_terms_common` WRITE;
/*!40000 ALTER TABLE `rlp_finder_terms_common` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_terms_common` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_tokens`
--

DROP TABLE IF EXISTS `rlp_finder_tokens`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_tokens` (
  `term` varchar(75) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `stem` varchar(75) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `common` tinyint unsigned NOT NULL DEFAULT '0',
  `phrase` tinyint unsigned NOT NULL DEFAULT '0',
  `weight` float unsigned NOT NULL DEFAULT '1',
  `context` tinyint unsigned NOT NULL DEFAULT '2',
  `language` char(3) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL DEFAULT '',
  KEY `idx_word` (`term`),
  KEY `idx_context` (`context`)
) ENGINE=MEMORY DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_tokens`
--

LOCK TABLES `rlp_finder_tokens` WRITE;
/*!40000 ALTER TABLE `rlp_finder_tokens` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_tokens` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_tokens_aggregate`
--

DROP TABLE IF EXISTS `rlp_finder_tokens_aggregate`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_tokens_aggregate` (
  `term_id` int unsigned NOT NULL,
  `map_suffix` char(1) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `term` varchar(75) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `stem` varchar(75) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `common` tinyint unsigned NOT NULL DEFAULT '0',
  `phrase` tinyint unsigned NOT NULL DEFAULT '0',
  `term_weight` float unsigned NOT NULL,
  `context` tinyint unsigned NOT NULL DEFAULT '2',
  `context_weight` float unsigned NOT NULL,
  `total_weight` float unsigned NOT NULL,
  `language` char(3) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL DEFAULT '',
  KEY `token` (`term`),
  KEY `keyword_id` (`term_id`)
) ENGINE=MEMORY DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_tokens_aggregate`
--

LOCK TABLES `rlp_finder_tokens_aggregate` WRITE;
/*!40000 ALTER TABLE `rlp_finder_tokens_aggregate` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_finder_tokens_aggregate` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_finder_types`
--

DROP TABLE IF EXISTS `rlp_finder_types`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_finder_types` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `title` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  `mime` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `title` (`title`)
) ENGINE=InnoDB AUTO_INCREMENT=12 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_finder_types`
--

LOCK TABLES `rlp_finder_types` WRITE;
/*!40000 ALTER TABLE `rlp_finder_types` DISABLE KEYS */;
INSERT INTO `rlp_finder_types` VALUES (6,'Tag',''),(7,'Category',''),(8,'Contact',''),(9,'Article',''),(10,'News Feed',''),(11,'Event','');
/*!40000 ALTER TABLE `rlp_finder_types` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_languages`
--

DROP TABLE IF EXISTS `rlp_languages`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_languages` (
  `lang_id` int unsigned NOT NULL AUTO_INCREMENT,
  `asset_id` int unsigned NOT NULL DEFAULT '0',
  `lang_code` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
  `title` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `title_native` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `sef` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `image` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `description` varchar(512) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `metakey` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadesc` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `sitename` varchar(1024) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `published` int NOT NULL DEFAULT '0',
  `access` int unsigned NOT NULL DEFAULT '0',
  `ordering` int NOT NULL DEFAULT '0',
  PRIMARY KEY (`lang_id`),
  UNIQUE KEY `idx_sef` (`sef`),
  UNIQUE KEY `idx_langcode` (`lang_code`),
  KEY `idx_access` (`access`),
  KEY `idx_ordering` (`ordering`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_languages`
--

LOCK TABLES `rlp_languages` WRITE;
/*!40000 ALTER TABLE `rlp_languages` DISABLE KEYS */;
INSERT INTO `rlp_languages` VALUES (1,0,'en-GB','English (UK)','English (UK)','en','en','','','','',1,1,-1);
/*!40000 ALTER TABLE `rlp_languages` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_menu`
--

DROP TABLE IF EXISTS `rlp_menu`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_menu` (
  `id` int NOT NULL AUTO_INCREMENT,
  `menutype` varchar(24) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The type of menu this item belongs to. FK to #__menu_types.menutype',
  `title` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The display title of the menu item.',
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL COMMENT 'The SEF alias of the menu item.',
  `note` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `path` varchar(1024) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The computed path of the menu item based on the alias field.',
  `link` varchar(1024) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The actually link the menu item refers to.',
  `type` varchar(16) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The type of link: Component, URL, Alias, Separator',
  `published` tinyint NOT NULL DEFAULT '0' COMMENT 'The published state of the menu link.',
  `parent_id` int unsigned NOT NULL DEFAULT '1' COMMENT 'The parent menu item in the menu tree.',
  `level` int unsigned NOT NULL DEFAULT '0' COMMENT 'The relative level in the tree.',
  `component_id` int unsigned NOT NULL DEFAULT '0' COMMENT 'FK to #__extensions.id',
  `checked_out` int unsigned NOT NULL DEFAULT '0' COMMENT 'FK to #__users.id',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00' COMMENT 'The time the menu item was checked out.',
  `browserNav` tinyint NOT NULL DEFAULT '0' COMMENT 'The click behaviour of the link.',
  `access` int unsigned NOT NULL DEFAULT '0' COMMENT 'The access level required to view the menu item.',
  `img` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The image of the menu item.',
  `template_style_id` int unsigned NOT NULL DEFAULT '0',
  `params` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'JSON encoded data for the menu item.',
  `lft` int NOT NULL DEFAULT '0' COMMENT 'Nested set lft.',
  `rgt` int NOT NULL DEFAULT '0' COMMENT 'Nested set rgt.',
  `home` tinyint unsigned NOT NULL DEFAULT '0' COMMENT 'Indicates if this menu item is the home or default page.',
  `language` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `client_id` tinyint NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`),
  UNIQUE KEY `idx_client_id_parent_id_alias_language` (`client_id`,`parent_id`,`alias`(100),`language`),
  KEY `idx_componentid` (`component_id`,`menutype`,`published`,`access`),
  KEY `idx_menutype` (`menutype`),
  KEY `idx_left_right` (`lft`,`rgt`),
  KEY `idx_language` (`language`),
  KEY `idx_alias` (`alias`(100)),
  KEY `idx_path` (`path`(100))
) ENGINE=InnoDB AUTO_INCREMENT=815 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_menu`
--

LOCK TABLES `rlp_menu` WRITE;
/*!40000 ALTER TABLE `rlp_menu` DISABLE KEYS */;
INSERT INTO `rlp_menu` VALUES (1,'','Menu_Item_Root','root','','','','',1,0,0,0,0,'0000-00-00 00:00:00',0,0,'',0,'',0,305,0,'*',0),(2,'main','com_banners','Banners','','Banners','index.php?option=com_banners','component',1,1,1,4,0,'0000-00-00 00:00:00',0,0,'class:banners',0,'',1,10,0,'*',1),(3,'main','com_banners','Banners','','Banners/Banners','index.php?option=com_banners','component',1,2,2,4,0,'0000-00-00 00:00:00',0,0,'class:banners',0,'',2,3,0,'*',1),(4,'main','com_banners_categories','Categories','','Banners/Categories','index.php?option=com_categories&extension=com_banners','component',1,2,2,6,0,'0000-00-00 00:00:00',0,0,'class:banners-cat',0,'',4,5,0,'*',1),(5,'main','com_banners_clients','Clients','','Banners/Clients','index.php?option=com_banners&view=clients','component',1,2,2,4,0,'0000-00-00 00:00:00',0,0,'class:banners-clients',0,'',6,7,0,'*',1),(6,'main','com_banners_tracks','Tracks','','Banners/Tracks','index.php?option=com_banners&view=tracks','component',1,2,2,4,0,'0000-00-00 00:00:00',0,0,'class:banners-tracks',0,'',8,9,0,'*',1),(7,'main','com_contact','Contacts','','Contacts','index.php?option=com_contact','component',1,1,1,8,0,'0000-00-00 00:00:00',0,0,'class:contact',0,'',247,252,0,'*',1),(8,'main','com_contact','Contacts','','Contacts/Contacts','index.php?option=com_contact','component',1,7,2,8,0,'0000-00-00 00:00:00',0,0,'class:contact',0,'',248,249,0,'*',1),(9,'main','com_contact_categories','Categories','','Contacts/Categories','index.php?option=com_categories&extension=com_contact','component',1,7,2,6,0,'0000-00-00 00:00:00',0,0,'class:contact-cat',0,'',250,251,0,'*',1),(10,'main','com_messages','Messaging','','Messaging','index.php?option=com_messages','component',1,1,1,15,0,'0000-00-00 00:00:00',0,0,'class:messages',0,'',253,258,0,'*',1),(11,'main','com_messages_add','New Private Message','','Messaging/New Private Message','index.php?option=com_messages&task=message.add','component',1,10,2,15,0,'0000-00-00 00:00:00',0,0,'class:messages-add',0,'',254,255,0,'*',1),(12,'main','com_messages_read','Read Private Message','','Messaging/Read Private Message','index.php?option=com_messages','component',1,10,2,15,0,'0000-00-00 00:00:00',0,0,'class:messages-read',0,'',256,257,0,'*',1),(13,'main','com_newsfeeds','News Feeds','','News Feeds','index.php?option=com_newsfeeds','component',1,1,1,17,0,'0000-00-00 00:00:00',0,0,'class:newsfeeds',0,'',259,264,0,'*',1),(14,'main','com_newsfeeds_feeds','Feeds','','News Feeds/Feeds','index.php?option=com_newsfeeds','component',1,13,2,17,0,'0000-00-00 00:00:00',0,0,'class:newsfeeds',0,'',260,261,0,'*',1),(15,'main','com_newsfeeds_categories','Categories','','News Feeds/Categories','index.php?option=com_categories&extension=com_newsfeeds','component',1,13,2,6,0,'0000-00-00 00:00:00',0,0,'class:newsfeeds-cat',0,'',262,263,0,'*',1),(16,'main','com_redirect','Redirect','','Redirect','index.php?option=com_redirect','component',1,1,1,24,0,'0000-00-00 00:00:00',0,0,'class:redirect',0,'',265,266,0,'*',1),(17,'main','com_search','Basic Search','','Basic Search','index.php?option=com_search','component',1,1,1,19,0,'0000-00-00 00:00:00',0,0,'class:search',0,'',267,268,0,'*',1),(18,'main','com_finder','Smart Search','','Smart Search','index.php?option=com_finder','component',1,1,1,27,0,'0000-00-00 00:00:00',0,0,'class:finder',0,'',269,270,0,'*',1),(20,'main','com_tags','Tags','','Tags','index.php?option=com_tags','component',1,1,1,29,0,'0000-00-00 00:00:00',0,1,'class:tags',0,'',271,272,0,'',1),(21,'main','com_postinstall','Post-installation messages','','Post-installation messages','index.php?option=com_postinstall','component',1,1,1,32,0,'0000-00-00 00:00:00',0,1,'class:postinstall',0,'',273,274,0,'*',1),(101,'mainmenu','Home','homepage','','homepage','index.php?option=com_content&view=featured','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"0\",\"info_block_show_title\":\"\",\"show_category\":\"0\",\"link_category\":\"0\",\"show_parent_category\":\"0\",\"link_parent_category\":\"0\",\"show_associations\":\"\",\"show_author\":\"0\",\"link_author\":\"0\",\"show_create_date\":\"0\",\"show_modify_date\":\"0\",\"show_publish_date\":\"0\",\"show_item_navigation\":\"0\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"0\",\"show_print_icon\":\"0\",\"show_email_icon\":\"0\",\"show_hits\":\"0\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-home\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',207,208,1,'*',0),(102,'usermenu','Your Profile','your-profile','','your-profile','index.php?option=com_users&view=profile&layout=edit','component',1,1,1,25,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"0\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',233,234,0,'*',0),(103,'usermenu','Site Administrator','2013-11-16-23-26-41','','2013-11-16-23-26-41','administrator','url',1,1,1,0,0,'0000-00-00 00:00:00',0,6,'',7,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1}',239,240,0,'*',0),(104,'usermenu','Submit an Article','submit-an-article','','submit-an-article','index.php?option=com_content&view=form&layout=edit','component',1,1,1,22,0,'0000-00-00 00:00:00',0,3,' ',0,'{\"enable_category\":\"0\",\"catid\":\"2\",\"redirect_menuitem\":\"\",\"custom_cancel_redirect\":\"0\",\"cancel_redirect_menuitem\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"0\",\"page_heading\":\"\",\"pageclass_sfx\":\" submit-an-article-show-help\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',235,236,0,'*',0),(106,'usermenu','Template Settings','template-settings','','template-settings','index.php?option=com_config&view=templates&controller=config.display.templates','component',-2,1,1,23,0,'0000-00-00 00:00:00',0,6,'',7,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"page_title\":\"\",\"show_page_heading\":0,\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',241,242,0,'*',0),(107,'usermenu','Site Settings','site-settings','','site-settings','index.php?option=com_config&view=config&controller=config.display.config','component',-2,1,1,23,0,'0000-00-00 00:00:00',0,6,'',7,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"page_title\":\"\",\"show_page_heading\":0,\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',243,244,0,'*',0),(152,'mainmenu','Docs','documentation','','documentation','index.php?option=com_content&view=article&id=134','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-graduation-cap\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',75,170,0,'*',0),(229,'main','com_joomlaupdate','com-joomlaupdate','','com-joomlaupdate','index.php?option=com_joomlaupdate','component',1,1,1,28,0,'0000-00-00 00:00:00',0,1,'class:joomlaupdate',0,'{}',275,276,0,'',1),(234,'mainmenu','About','about','','about','','heading',1,1,1,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-info-circle\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',25,60,0,'*',0),(235,'mainmenu','What is Reactome ?','what-is-reactome','','about/what-is-reactome','index.php?option=com_content&view=article&id=11','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',26,27,0,'*',0),(236,'mainmenu','News','news','','about/news','index.php?option=com_content&view=category&layout=blog&id=26','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"layout_type\":\"blog\",\"show_category_heading_title_text\":\"\",\"show_category_title\":\"\",\"show_description\":\"\",\"show_description_image\":\"\",\"maxLevel\":\"\",\"show_empty_categories\":\"\",\"show_no_articles\":\"\",\"show_subcat_desc\":\"\",\"show_cat_num_articles\":\"\",\"show_cat_tags\":\"\",\"page_subheading\":\"\",\"num_leading_articles\":\"9999\",\"num_intro_articles\":\"1\",\"num_columns\":\"2\",\"num_links\":\"\",\"multi_column_order\":\"1\",\"show_subcategory_content\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"rdate\",\"order_date\":\"\",\"show_pagination\":\"0\",\"show_pagination_results\":\"\",\"show_featured\":\"\",\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"0\",\"absent_alias\":\"\"}',28,29,0,'*',0),(237,'mainmenu','Team','team','','about/team','index.php?option=com_content&view=article&id=13','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',30,31,0,'*',0),(238,'mainmenu','Tools','tools','','tools','','heading',1,1,1,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-cogs\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',171,202,0,'*',0),(239,'mainmenu','Pathway Browser','pathway-browser','','tools/pathway-browser','/PathwayBrowser','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1}',172,173,0,'*',0),(240,'mainmenu','Analyse gene list','analyse-gene-list','','tools/analyse-gene-list','/PathwayBrowser/#TOOL=AT','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',174,175,0,'*',0),(241,'mainmenu','Community','community','','community','index.php?option=com_content&view=article&id=257','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-users\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',209,232,0,'*',0),(242,'mainmenu','Download','download-data','','download-data','index.php?option=com_content&view=article&id=18','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-download\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',289,292,0,'*',0),(244,'main','com_associations','multilingual-associations','','multilingual-associations','index.php?option=com_associations','component',1,1,1,34,0,'0000-00-00 00:00:00',0,1,'class:associations',0,'{}',277,278,0,'*',1),(245,'mainmenu','Species Comparison','species-comparison','','tools/species-comparison','/PathwayBrowser/#TOOL=AT','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1}',178,179,0,'*',0),(246,'mainmenu','ReactomeFIViz','reactome-fiviz','','tools/reactome-fiviz','index.php?option=com_content&view=article&id=44','component',1,238,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',186,187,0,'*',0),(247,'mainmenu','Advanced Data Search','advanced-search','','tools/advanced-search','/content/advanced','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',196,197,0,'*',0),(248,'mainmenu','Analysis Service','analysis-service-tool','','tools/analysis-service-tool','/AnalysisService','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',182,183,0,'*',0),(249,'mainmenu','Content Service','content-service','','tools/content-service','/ContentService','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1}',184,185,0,'*',0),(251,'mainmenu','Scientific Advisory Board','sab','','about/sab','index.php?option=com_content&view=article&id=14','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',32,33,0,'*',0),(252,'mainmenu','Other Reactomes','other-reactomes','','about/other-reactomes','index.php?option=com_content&view=article&id=15','component',0,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',36,37,0,'*',0),(253,'mainmenu','License Agreement','license','','about/license','index.php?option=com_content&view=article&id=16','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',46,47,0,'*',0),(254,'mainmenu','Disclaimer','disclaimer','','about/disclaimer','index.php?option=com_content&view=article&id=17','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',50,51,0,'*',0),(259,'mainmenu','Outreach','outreach','','community/outreach','index.php?option=com_content&view=article&id=22','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',216,217,0,'*',0),(260,'mainmenu','Events','events','','community/events','index.php?option=com_content&view=article&id=23','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',218,219,0,'*',0),(261,'mainmenu','Training','training','','community/training','index.php?option=com_content&view=article&id=25','component',0,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',220,221,0,'*',0),(262,'mainmenu','Publications','publications','','community/publications','index.php?option=com_content&view=article&id=24','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',222,223,0,'*',0),(263,'mainmenu','Partners','partners','','community/partners','index.php?option=com_content&view=article&id=26','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',224,225,0,'*',0),(264,'mainmenu','Papers Citing Us','citation-list','','community/citation-list','index.php?option=com_content&view=article&id=69','component',0,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',228,229,0,'*',0),(265,'mainmenu','Resources Guide','resources','','community/resources','index.php?option=com_content&view=article&id=27','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',230,231,0,'*',0),(266,'mainmenu','Icon Library','icon-lib','','community/icon-lib','index.php?option=com_content&view=featured','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"Community library of icons that are used in the Reactome Enhanced High Level Diagrams (EHLD). We would like to develop the EHLD library further as a community resource. If you use library elements, and design similar elements that are still missing, we would be very happy to incorporate them into the library, naming you as the author\",\"menu-anchor_css\":\"icons-library-mi\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',214,215,0,'*',0),(267,'mainmenu','User','user','','documentation/user','user/guide','url',0,152,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',164,165,0,'*',0),(268,'mainmenu','Developer\'s Zone','dev','','documentation/dev','index.php?option=com_content&view=article&id=64','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',106,137,0,'*',0),(269,'mainmenu','Data Model','data-model','','documentation/data-model','index.php?option=com_content&view=article&id=53','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',144,147,0,'*',0),(270,'mainmenu','Computationally inferred events','inferred-events','','documentation/inferred-events','index.php?option=com_content&view=article&id=68','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',152,153,0,'*',0),(271,'mainmenu','Linking to Us','linking-to-us','','documentation/linking-to-us','index.php?option=com_content&view=article&id=28','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',156,159,0,'*',0),(272,'mainmenu','Citing us','cite','','documentation/cite','index.php?option=com_content&view=article&id=29','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',160,161,0,'*',0),(277,'usermenu','Logout','logout','','logout','index.php?option=com_users&view=login&layout=logout&task=user.menulogout','component',1,1,1,25,0,'0000-00-00 00:00:00',0,2,' ',7,'{\"logout\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0}',245,246,0,'*',0),(279,'mainmenu','Account','reactome','','reactome','','heading',1,1,1,0,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-users\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',11,24,0,'*',0),(280,'mainmenu','Logout','userlogout','','reactome/userlogout','index.php?Itemid=','alias',1,279,2,0,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"aliasoptions\":\"277\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1}',22,23,0,'*',0),(281,'mainmenu','Publish an Article','publish-an-article','','reactome/publish-an-article','index.php?Itemid=','alias',1,279,2,0,0,'0000-00-00 00:00:00',0,2,' ',4,'{\"aliasoptions\":\"104\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',16,17,0,'*',0),(282,'mainmenu','Your Profile','user-profile','','reactome/user-profile','index.php?Itemid=','alias',1,279,2,0,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"aliasoptions\":\"102\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1}',12,13,0,'*',0),(284,'mainmenu','Staff','staff','This menu entry is visible only in ReactomeRelease. SyncTool unpublished when syncing to production','about/staff','index.php?option=com_users&view=login','component',1,234,2,25,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"loginredirectchoice\":\"1\",\"login_redirect_url\":\"\",\"login_redirect_menuitem\":\"101\",\"logindescription_show\":\"1\",\"login_description\":\"Note: Login is only available for Reactome Staff Members.\",\"login_image\":\"\",\"logoutredirectchoice\":\"1\",\"logout_redirect_url\":\"\",\"logout_redirect_menuitem\":\"101\",\"logoutdescription_show\":\"1\",\"logout_description\":\"\",\"logout_image\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"Reactome Staff Member Login\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"noindex, nofollow\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',54,55,0,'*',0),(288,'hidden-menu','Template Search Page','template-datacontent','','template-datacontent','index.php?option=com_content&view=featured','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"0\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"0\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"Reactome\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"index, follow\",\"secure\":-1,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',279,280,0,'*',0),(290,'mainmenu','Graph Database','graph-database','','documentation/dev/graph-database','index.php?option=com_content&view=article&id=45','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"0\",\"absent_alias\":\"\"}',107,112,0,'*',0),(291,'mainmenu','Teleconferences','ateleconferences','','reactome/ateleconferences','index.php?Itemid=','alias',0,279,2,0,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"aliasoptions\":\"398\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"1\",\"absent_alias\":\"0\"}',18,19,0,'*',0),(292,'mainmenu','Analysis Service ','analysis','','documentation/dev/analysis','index.php?option=com_content&view=article&id=47','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',113,114,0,'*',0),(293,'mainmenu','Content Service','content-service','','documentation/dev/content-service','index.php?option=com_content&view=article&id=48','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',115,118,0,'*',0),(294,'mainmenu','Pathways Overview','pathways-overview','','documentation/dev/pathways-overview','index.php?option=com_content&view=article&id=49','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',119,124,0,'*',0),(295,'mainmenu','Pathway Diagrams','diagram','','documentation/dev/diagram','index.php?option=com_content&view=article&id=50','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',125,132,0,'*',0),(296,'mainmenu','RESTFul API','restful-api','','documentation/dev/restful-api','index.php?option=com_content&view=article&id=51','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',133,134,0,'*',0),(297,'mainmenu','Simplified DB','simplified-db','','documentation/dev/simplified-db','index.php?option=com_content&view=article&id=52','component',1,268,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',135,136,0,'*',0),(299,'mainmenu','Tutorial: Extracting participating molecules using the Graph Database','extract-participating-molecules','','documentation/dev/graph-database/extract-participating-molecules','index.php?option=com_content&view=article&id=54','component',1,290,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"0\",\"absent_alias\":\"\"}',108,109,0,'*',0),(301,'mainmenu','Site Search','site-search','Search for contents in the WebSite. It does not support data content search, like the one provided by SolR','tools/site-search','index.php?option=com_search&view=search&searchword=pathway','component',1,238,2,19,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"search_phrases\":\"\",\"search_areas\":\"\",\"show_date\":\"\",\"searchphrase\":\"0\",\"ordering\":\"alpha\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"1\",\"page_heading\":\"Use this tool to search the Reactome static web pages (Documentation, news, etc).\",\"pageclass_sfx\":\" search-text\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',200,201,0,'*',0),(302,'mainmenu','Icon Info','icon-info','','documentation/icon-info','index.php?option=com_content&view=article&id=55','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',138,143,0,'*',0),(303,'mainmenu','Userguide','userguide','user guide','documentation/userguide','index.php?option=com_content&view=article&id=39','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"0\",\"info_block_position\":\"\",\"info_block_show_title\":\"0\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',76,105,0,'*',0),(304,'mainmenu','Fireworks JS','js','','documentation/dev/pathways-overview/js','index.php?option=com_content&view=article&id=56','component',1,294,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',120,121,0,'*',0),(305,'mainmenu','Fireworks GWT','gwt','','documentation/dev/pathways-overview/gwt','index.php?option=com_content&view=article&id=57','component',1,294,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',122,123,0,'*',0),(306,'mainmenu','Diagram JS','js','','documentation/dev/diagram/js','index.php?option=com_content&view=article&id=58','component',1,295,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',126,127,0,'*',0),(307,'mainmenu','Diagram GWT','gwt','','documentation/dev/diagram/gwt','index.php?option=com_content&view=article&id=59','component',1,295,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',128,129,0,'*',0),(355,'mainmenu','Author/Reviewer Search','author-reviewer-search','','tools/author-reviewer-search','cgi-bin/author_contributions/','url',0,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',198,199,0,'*',0),(356,'mainmenu','Content','content','','content','','heading',1,1,1,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"fa-pencil-square-o\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',61,74,0,'*',0),(357,'mainmenu','Data Schema','data-schema','','content/data-schema','/content/schema','url',1,356,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',66,67,0,'*',0),(358,'mainmenu','Table of Contents','table-of-contents','','content/table-of-contents','/content/toc','url',1,356,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',62,63,0,'*',0),(359,'mainmenu','DOIs','dois','','content/dois','/content/doi','url',1,356,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',64,65,0,'*',0),(360,'mainmenu','Statistics','statistics','','about/statistics','index.php?option=com_content&view=article&id=66','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',42,43,0,'*',0),(361,'mainmenu','ORCID Integration Project','orcid','','content/orcid','index.php?option=com_content&view=article&id=67','component',1,356,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"1\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',70,71,0,'*',0),(362,'mainmenu','Editorial Calendar','editorial-calendar','','about/editorial-calendar','index.php?option=com_wrapper&view=wrapper','component',1,234,2,2,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"url\":\"https:\\/\\/docs.google.com\\/spreadsheets\\/d\\/e\\/2PACX-1vTebSE76KQx1_I36_lwNnklYyDniGdWV0WqXOHjGDiJbeqwGv2W5F91EM0APfZjLXr_TVTHaAiLp8dt\\/pubhtml?widget=true#gid=5&headers=false\",\"scrolling\":\"auto\",\"width\":\"100%\",\"height\":\"500\",\"height_auto\":\"0\",\"add_scheme\":\"1\",\"frameborder\":\"1\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',38,39,0,'*',0),(371,'hidden-menu','Template Swagger Page','template-swagger','','template-swagger','index.php?option=com_content&view=featured','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"Reactome\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"index, follow\",\"secure\":-1,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',281,282,0,'*',0),(377,'mainmenu','External Identifiers','identifiers','','documentation/linking-to-us/identifiers','index.php?option=com_content&view=article&id=70','component',1,271,3,22,169,'2020-04-30 18:40:09',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',157,158,0,'*',0),(378,'mainmenu','Pathway Browser','pathway-browser','','documentation/userguide/pathway-browser','index.php?option=com_content&view=article&id=81','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',77,78,0,'*',0),(379,'mainmenu','Details Panel','details-panel','','documentation/userguide/details-panel','index.php?option=com_content&view=article&id=82','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',81,82,0,'*',0),(380,'mainmenu','How do I search ?','searching','','documentation/userguide/searching','index.php?option=com_content&view=article&id=83','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',79,80,0,'*',0),(381,'mainmenu','Analysis Tools','analysis','','documentation/userguide/analysis','index.php?option=com_content&view=article&id=84','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',83,94,0,'*',0),(382,'mainmenu','Diseases','diseases','','documentation/userguide/diseases','index.php?option=com_content&view=article&id=85','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',95,96,0,'*',0),(383,'hidden-menu','Template CGI','template-cgi','','template-cgi','index.php?option=com_content&view=featured','component',1,1,1,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"Reactome\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"noindex, nofollow\",\"secure\":-1,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',283,284,0,'*',0),(384,'mainmenu','Reactome Curator Tool 3','reactome-curator-tool','','download-data/reactome-curator-tool','index.php?option=com_content&view=article&id=63','component',1,242,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',290,291,0,'*',0),(397,'mainmenu','Our Logo','logo','','about/logo','index.php?option=com_content&view=article&id=91','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"0\",\"absent_alias\":\"\"}',44,45,0,'*',0),(398,'usermenu','Teleconferences','teleconferences','','teleconferences','index.php?option=com_content&view=article&id=46','component',0,1,1,22,0,'0000-00-00 00:00:00',0,3,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',237,238,0,'*',0),(405,'hidden-menu','Synchronization Tool','p2p','','p2p','index.php?option=com_content&view=featured','component',0,1,1,22,0,'0000-00-00 00:00:00',0,7,' ',24,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"page_subheading\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"noindex, nofollow\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',285,286,0,'*',0),(406,'mainmenu','Synchronization Tool','sync-tool','','reactome/sync-tool','index.php?option=com_content&view=featured','component',1,279,2,22,0,'0000-00-00 00:00:00',0,7,' ',0,'{\"featured_categories\":[\"\"],\"layout_type\":\"blog\",\"num_leading_articles\":\"\",\"num_intro_articles\":\"\",\"num_columns\":\"\",\"num_links\":\"\",\"multi_column_order\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"page_subheading\":\"\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',20,21,0,'*',0),(407,'mainmenu','Diagram exporter','diagram-exporter','','documentation/dev/content-service/diagram-exporter','index.php?option=com_content&view=article&id=96','component',1,293,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',116,117,0,'*',0),(412,'mainmenu','Release','release','Hidden Menu for the Release Articles','tag/release','index.php?option=com_tags&view=tag&id[0]=7&types[0]=1','component',1,413,2,29,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_tag_title\":\"\",\"tag_list_show_tag_image\":\"\",\"tag_list_show_tag_description\":\"\",\"tag_list_image\":\"\",\"tag_list_description\":\"\",\"tag_list_orderby\":\"c.core_created_time\",\"tag_list_orderby_direction\":\"DESC\",\"tag_list_show_item_image\":\"\",\"tag_list_show_item_description\":\"\",\"tag_list_item_maximum_characters\":\"\",\"filter_field\":\"0\",\"show_pagination_limit\":\"0\",\"show_pagination\":\"0\",\"show_pagination_results\":\"0\",\"return_any_or_all\":\"\",\"include_children\":\"\",\"show_feed_link\":\"0\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"Release\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',204,205,0,'*',0),(413,'mainmenu','Tag','tag','','tag','','heading',1,1,1,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"direct_alias\":\"1\",\"absent_alias\":\"\"}',203,206,0,'*',0),(422,'usermenu','Help','staff-help','','staff-help','index.php?option=com_content&view=article&id=99','component',1,1,1,22,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"noindex, nofollow\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',287,288,0,'*',0),(423,'mainmenu','Help','help','This is an Alias to a menu item in the Users Menu','reactome/help','index.php?Itemid=','alias',1,279,2,0,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"aliasoptions\":\"422\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',14,15,0,'*',0),(424,'mainmenu','Pathway Diagrams Specifications','pathway-diagram-specs','','documentation/dev/diagram/pathway-diagram-specs','index.php?option=com_content&view=article&id=100','component',1,295,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',130,131,0,'*',0),(471,'mainmenu','EHLD Specs & Guidelines','ehld-specs-guideline','','documentation/icon-info/ehld-specs-guideline','index.php?option=com_content&view=article&id=129','component',1,302,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',139,140,0,'*',0),(472,'mainmenu','Icon Library Guidelines','icons-guidelines','','documentation/icon-info/icons-guidelines','index.php?option=com_content&view=article&id=130','component',1,302,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',141,142,0,'*',0),(477,'mainmenu','User 2','user2','','documentation/user2','','heading',-2,152,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',166,169,0,'*',0),(478,'mainmenu','Guide2','guide2','','documentation/user2/guide2','index.php?Itemid=','alias',-2,477,3,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"aliasoptions\":\"303\",\"alias_redirect\":0,\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',167,168,0,'*',0),(479,'mainmenu','ReactomeFIViz','reactome-fiviz','','documentation/userguide/reactome-fiviz','index.php?option=com_content&view=article&id=44','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',101,102,0,'*',0),(488,'mainmenu','Claim your work','claim-your-work','','documentation/userguide/claim-your-work','index.php?option=com_content&view=article&id=139','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',103,104,0,'*',0),(493,'mainmenu','Contribute Pathway Knowledge','collaboration','','community/collaboration','index.php?option=com_content&view=article&id=140','component',1,241,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',210,213,0,'*',0),(499,'mainmenu','DisGeNET','disgenet-app','','tools/overlays-h/disgenet-app','','heading',0,557,3,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"disgenet\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',189,194,0,'*',0),(504,'mainmenu','FAQ for prospective reviewers and authors','faq-for-prospective-reviewers-and-authors','','community/collaboration/faq-for-prospective-reviewers-and-authors','index.php?option=com_content&view=article&id=143','component',1,493,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"0\",\"absent_alias\":\"\"}',211,212,0,'*',0),(530,'mainmenu','Privacy Statement','privacy-statement','','about/privacy-statement','index.php?option=com_content&view=article&id=148','component',-2,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',58,59,0,'*',0),(539,'mainmenu','User Guide TOC','user-guide-toc','','documentation/user-guide-toc','index.php?option=com_content&view=article&id=152','component',1,152,2,22,0,'0000-00-00 00:00:00',0,2,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"1\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',162,163,0,'*',0),(540,'mainmenu','Data Model Glossary','data-model-glossary','','documentation/data-model/data-model-glossary','index.php?option=com_content&view=article&id=153','component',0,269,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',145,146,0,'*',0),(557,'mainmenu','Overlays','overlays-h','','tools/overlays-h','','heading',0,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"overlays\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',188,195,0,'*',0),(558,'mainmenu','API','disgenet-api','disgenet-api','tools/overlays-h/disgenet-app/disgenet-api','/overlays/disgenet/api','url',0,499,4,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',192,193,0,'*',0),(559,'mainmenu','Web','disgenet-web','disgenet-web','tools/overlays-h/disgenet-app/disgenet-web','/overlays/disgenet','url',0,499,4,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',190,191,0,'*',0),(560,'mainmenu','Analysis Gene Expression','gsa','','documentation/userguide/analysis/gsa','index.php?option=com_content&view=article&id=156','component',1,381,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',86,87,0,'*',0),(561,'mainmenu','Analyse gene expression','analyse-gene-expression','','tools/analyse-gene-expression','gsa/home','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',176,177,0,'*',0),(572,'mainmenu','COVID-19 Disease Pathways','covid-19','','content/covid-19','index.php?option=com_content&view=article&id=158','component',1,356,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"1\",\"absent_alias\":\"0\"}',72,73,0,'*',0),(578,'mainmenu','Analysis Gene Expression','analysis-gene-expression','','documentation/userguide/analysis/analysis-gene-expression','index.php?option=com_content&view=article&id=156','component',-2,381,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',92,93,0,'*',0),(579,'mainmenu','Analysis Data','analysis-data','','documentation/userguide/analysis/analysis-data','userguide/analysis#analysis','url',1,381,4,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',84,85,0,'*',0),(580,'mainmenu','Species Comparison','species-comparison','','documentation/userguide/analysis/species-comparison','userguide/analysis#species','url',1,381,4,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',88,89,0,'*',0),(581,'mainmenu','Tissue Distribution','tissue-distribution','','documentation/userguide/analysis/tissue-distribution','userguide/analysis#tissue','url',1,381,4,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',90,91,0,'*',0),(582,'mainmenu','Tissue Distribution','tissue-distribution','','tools/tissue-distribution','PathwayBrowser/#TOOL=AT','url',1,238,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',180,181,0,'*',0),(588,'mainmenu','Privacy Notice','privacy-notice','','about/privacy-notice','index.php?option=com_content&view=article&id=148','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',48,49,0,'*',0),(629,'mainmenu','Neo4j Desktop','neo4j-desktop','','documentation/dev/graph-database/neo4j-desktop','index.php?option=com_content&view=article&id=168','component',1,290,4,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":0,\"menu_show\":0,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',110,111,0,'*',0),(657,'main','COM_EASYJOOMLABACKUP','com-easyjoomlabackup','','com-easyjoomlabackup','index.php?option=com_easyjoomlabackup','component',1,1,1,10208,0,'0000-00-00 00:00:00',0,1,'components/com_easyjoomlabackup/images/ejb-menu.png',0,'{}',293,294,0,'',1),(693,'mainmenu','Reactome Research Spotlight','reactome-research-spotlight','Reactome Research Spotlight','content/reactome-research-spotlight','index.php?option=com_content&view=category&layout=blog&id=36','component',1,356,2,22,0,'0000-00-00 00:00:00',0,1,' ',24,'{\"layout_type\":\"blog\",\"show_category_heading_title_text\":\"\",\"show_category_title\":\"\",\"show_description\":\"\",\"show_description_image\":\"\",\"maxLevel\":\"\",\"show_empty_categories\":\"\",\"show_no_articles\":\"\",\"show_subcat_desc\":\"\",\"show_cat_num_articles\":\"\",\"show_cat_tags\":\"\",\"page_subheading\":\"\",\"num_leading_articles\":\"9999\",\"num_intro_articles\":\"1\",\"num_columns\":\"2\",\"num_links\":\"\",\"multi_column_order\":\"\",\"show_subcategory_content\":\"\",\"orderby_pri\":\"\",\"orderby_sec\":\"rdate\",\"order_date\":\"\",\"show_pagination\":\"0\",\"show_pagination_results\":\"0\",\"show_featured\":\"\",\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"1\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',68,69,0,'en-GB',0),(715,'mainmenu','Review Status of Reactome Events ','review-status','','documentation/userguide/review-status','index.php?option=com_content&view=article&id=192','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"0\",\"absent_alias\":\"0\"}',99,100,0,'*',0),(721,'mainmenu','FAQ','faq','','documentation/faq','index.php?option=com_content&view=category&layout=blog&id=35','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"layout_type\":\"blog\",\"show_category_heading_title_text\":\"\",\"show_category_title\":\"\",\"show_description\":\"\",\"show_description_image\":\"\",\"maxLevel\":\"\",\"show_empty_categories\":\"\",\"show_no_articles\":\"\",\"show_subcat_desc\":\"\",\"show_cat_num_articles\":\"1\",\"show_cat_tags\":\"\",\"page_subheading\":\"\",\"num_leading_articles\":\"0\",\"num_intro_articles\":\"0\",\"num_columns\":\"\",\"num_links\":\"9999\",\"multi_column_order\":\"\",\"show_subcategory_content\":\"-1\",\"orderby_pri\":\"none\",\"orderby_sec\":\"rorder\",\"order_date\":\"\",\"show_pagination\":\"\",\"show_pagination_results\":\"\",\"show_featured\":\"\",\"article_layout\":\"_:default\",\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_readmore\":\"\",\"show_readmore_title\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"show_feed_link\":\"\",\"feed_summary\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',154,155,0,'*',0),(722,'mainmenu','Release Calendar','release-calendar','','about/release-calendar','index.php?option=com_wrapper&view=wrapper','component',1,234,2,2,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"url\":\"https:\\/\\/docs.google.com\\/spreadsheets\\/d\\/e\\/2PACX-1vQR7_-1pf24gjQh_sR-gvTg_mPXJ_zjHc-N3jUoqb9M7f9i1I5NugQkAd-ve7LYgVYSsgMFMqGRcCCE\\/pubhtml\",\"scrolling\":\"auto\",\"width\":\"100%\",\"height\":\"500\",\"height_auto\":\"0\",\"add_scheme\":\"1\",\"frameborder\":\"1\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',40,41,0,'*',0),(758,'mainmenu','Contact us','contact-us','','about/contact-us','index.php?option=com_content&view=article&id=239','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',56,57,0,'*',0),(769,'mainmenu','Digital Preservation','digital-preservation','','about/digital-preservation','index.php?option=com_content&view=article&id=241','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"0\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',52,53,0,'*',0),(770,'mainmenu','Contributors','contributors','','community/contributors','/content/contributors','url',1,241,2,0,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu-anchor_rel\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"direct_alias\":\"\",\"absent_alias\":\"\"}',226,227,0,'*',0),(781,'mainmenu','Cytomics','cytomics','','documentation/userguide/cytomics','index.php?option=com_content&view=article&id=247','component',1,303,3,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',97,98,0,'*',0),(807,'mainmenu','Funding','funding','','about/funding','index.php?option=com_content&view=article&id=258','component',1,234,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',34,35,0,'*',0),(808,'mainmenu','Curator Guide','curator-guide','','documentation/curator-guide','index.php?option=com_content&view=article&id=259','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',148,149,0,'*',0),(809,'mainmenu','Release Documentation','release-documentation','','documentation/release-documentation','index.php?option=com_content&view=article&id=260','component',1,152,2,22,0,'0000-00-00 00:00:00',0,1,' ',0,'{\"show_title\":\"\",\"link_titles\":\"\",\"show_intro\":\"\",\"info_block_position\":\"\",\"info_block_show_title\":\"\",\"show_category\":\"\",\"link_category\":\"\",\"show_parent_category\":\"\",\"link_parent_category\":\"\",\"show_associations\":\"\",\"show_author\":\"\",\"link_author\":\"\",\"show_create_date\":\"\",\"show_modify_date\":\"\",\"show_publish_date\":\"\",\"show_item_navigation\":\"\",\"show_vote\":\"\",\"show_icons\":\"\",\"show_print_icon\":\"\",\"show_email_icon\":\"\",\"show_hits\":\"\",\"show_tags\":\"\",\"show_noauth\":\"\",\"urls_position\":\"\",\"menu-anchor_title\":\"\",\"menu-anchor_css\":\"\",\"menu_image\":\"\",\"menu_image_css\":\"\",\"menu_text\":1,\"menu_show\":1,\"page_title\":\"\",\"show_page_heading\":\"\",\"page_heading\":\"\",\"pageclass_sfx\":\"\",\"menu-meta_description\":\"\",\"menu-meta_keywords\":\"\",\"robots\":\"\",\"secure\":0,\"direct_alias\":\"\",\"absent_alias\":\"\"}',150,151,0,'*',0),(810,'main','COM_JCE','com-jce','','com-jce','index.php?option=com_jce&view=cpanel','component',1,1,1,10193,0,'0000-00-00 00:00:00',0,1,'class:component',0,'{}',295,304,0,'',1),(811,'main','COM_JCE_MENU_CPANEL','com-jce-menu-cpanel','','com-jce/com-jce-menu-cpanel','index.php?option=com_jce&view=cpanel','component',1,810,2,10193,0,'0000-00-00 00:00:00',0,1,'class:component',0,'{}',296,297,0,'',1),(812,'main','COM_JCE_MENU_CONFIG','com-jce-menu-config','','com-jce/com-jce-menu-config','index.php?option=com_jce&view=config','component',1,810,2,10193,0,'0000-00-00 00:00:00',0,1,'class:component',0,'{}',298,299,0,'',1),(813,'main','COM_JCE_MENU_PROFILES','com-jce-menu-profiles','','com-jce/com-jce-menu-profiles','index.php?option=com_jce&view=profiles','component',1,810,2,10193,0,'0000-00-00 00:00:00',0,1,'class:component',0,'{}',300,301,0,'',1),(814,'main','COM_JCE_MENU_FILEBROWSER','com-jce-menu-filebrowser','','com-jce/com-jce-menu-filebrowser','index.php?option=com_jce&view=browser','component',1,810,2,10193,0,'0000-00-00 00:00:00',0,1,'class:component',0,'{}',302,303,0,'',1);
/*!40000 ALTER TABLE `rlp_menu` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_menu_types`
--

DROP TABLE IF EXISTS `rlp_menu_types`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_menu_types` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `asset_id` int unsigned NOT NULL DEFAULT '0',
  `menutype` varchar(24) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `title` varchar(48) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `description` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `client_id` int NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`),
  UNIQUE KEY `idx_menutype` (`menutype`)
) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_menu_types`
--

LOCK TABLES `rlp_menu_types` WRITE;
/*!40000 ALTER TABLE `rlp_menu_types` DISABLE KEYS */;
INSERT INTO `rlp_menu_types` VALUES (1,552,'mainmenu','Main Menu','Landing Page menu',0),(2,568,'usermenu','User Menu','A Menu for logged-in Users',0),(4,599,'hidden-menu','Hidden Menu','Menu to hold hidden options',0);
/*!40000 ALTER TABLE `rlp_menu_types` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_messages`
--

DROP TABLE IF EXISTS `rlp_messages`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_messages` (
  `message_id` int unsigned NOT NULL AUTO_INCREMENT,
  `user_id_from` int unsigned NOT NULL DEFAULT '0',
  `user_id_to` int unsigned NOT NULL DEFAULT '0',
  `folder_id` tinyint unsigned NOT NULL DEFAULT '0',
  `date_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `state` tinyint(1) NOT NULL DEFAULT '0',
  `priority` tinyint unsigned NOT NULL DEFAULT '0',
  `subject` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `message` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`message_id`),
  KEY `useridto_state` (`user_id_to`,`state`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_messages`
--

LOCK TABLES `rlp_messages` WRITE;
/*!40000 ALTER TABLE `rlp_messages` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_messages` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_messages_cfg`
--

DROP TABLE IF EXISTS `rlp_messages_cfg`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_messages_cfg` (
  `user_id` int unsigned NOT NULL DEFAULT '0',
  `cfg_name` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `cfg_value` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  UNIQUE KEY `idx_user_var_name` (`user_id`,`cfg_name`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_messages_cfg`
--

LOCK TABLES `rlp_messages_cfg` WRITE;
/*!40000 ALTER TABLE `rlp_messages_cfg` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_messages_cfg` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_modules`
--

DROP TABLE IF EXISTS `rlp_modules`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_modules` (
  `id` int NOT NULL AUTO_INCREMENT,
  `asset_id` int unsigned NOT NULL DEFAULT '0' COMMENT 'FK to the #__assets table.',
  `title` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `note` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `content` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `ordering` int NOT NULL DEFAULT '0',
  `position` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `checked_out` int unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `published` tinyint(1) NOT NULL DEFAULT '0',
  `module` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `access` int unsigned NOT NULL DEFAULT '0',
  `showtitle` tinyint unsigned NOT NULL DEFAULT '1',
  `params` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `client_id` tinyint NOT NULL DEFAULT '0',
  `language` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`id`),
  KEY `published` (`published`,`access`),
  KEY `newsfeeds` (`module`,`published`),
  KEY `idx_language` (`language`)
) ENGINE=InnoDB AUTO_INCREMENT=603 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_modules`
--

LOCK TABLES `rlp_modules` WRITE;
/*!40000 ALTER TABLE `rlp_modules` DISABLE KEYS */;
INSERT INTO `rlp_modules` VALUES (1,39,'Main Menu','','',1,'nav',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_menu',1,0,'{\"menutype\":\"mainmenu\",\"base\":\"\",\"startLevel\":\"1\",\"endLevel\":\"0\",\"showAllChildren\":\"1\",\"tag_id\":\"\",\"class_sfx\":\"\",\"window_open\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"itemid\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(2,40,'Login','','',1,'login',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_login',1,1,'',1,'*'),(3,41,'Popular Articles','','',3,'cpanel',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_popular',3,1,'{\"count\":\"5\",\"catid\":\"\",\"user_id\":\"0\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\",\"automatic_title\":\"1\"}',1,'*'),(4,42,'Recently Added Articles','','',4,'cpanel',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_latest',3,1,'{\"count\":\"5\",\"ordering\":\"c_dsc\",\"catid\":\"\",\"user_id\":\"0\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\",\"automatic_title\":\"1\"}',1,'*'),(8,43,'Toolbar','','',1,'toolbar',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_toolbar',3,1,'',1,'*'),(9,44,'Quick Icons','','',1,'icon',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_quickicon',3,1,'',1,'*'),(10,45,'Logged-in Users','','',2,'cpanel',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_logged',3,1,'{\"count\":\"5\",\"name\":\"1\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\",\"automatic_title\":\"1\"}',1,'*'),(12,46,'Admin Menu','','',1,'menu',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_menu',3,1,'{\"layout\":\"\",\"moduleclass_sfx\":\"\",\"shownew\":\"1\",\"showhelp\":\"1\",\"cache\":\"0\"}',1,'*'),(13,47,'Admin Submenu','','',1,'submenu',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_submenu',3,1,'',1,'*'),(14,48,'User Status','','',2,'status',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_status',3,1,'',1,'*'),(15,49,'Title','','',1,'title',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_title',3,1,'',1,'*'),(17,51,'Breadcrumb','','',1,'breadcrumbs',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_breadcrumbs',1,0,'{\"showHere\":0,\"showHome\":1,\"homeText\":\"Home\",\"showLast\":1,\"separator\":\">\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" hidden-phone\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"itemid\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(79,52,'Multilanguage status','','',1,'status',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',0,'mod_multilangstatus',3,1,'{\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\"}',1,'*'),(86,53,'Joomla Version','','',1,'footer',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_version',3,1,'{\"format\":\"short\",\"product\":\"1\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\"}',1,'*'),(88,55,'Site Information','','',3,'cpanel',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_stats_admin',3,1,'{\"serverinfo\":\"1\",\"siteinfo\":\"1\",\"counter\":\"0\",\"increase\":\"0\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',1,'*'),(89,56,'Release News','','',0,'postinstall',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_feed',1,1,'{\"rssurl\":\"http:\\/\\/www.joomla.org\\/announcements\\/release-news.feed\",\"rssrtl\":\"0\",\"rsstitle\":\"1\",\"rssdesc\":\"1\",\"rssimage\":\"1\",\"rssitems\":\"3\",\"rssitemdesc\":\"1\",\"word_count\":\"0\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',1,'*'),(475,483,'FavGlyph - Home Page','Glyphs for PathwayBrowser, Analyze Data, ReactomeFIViz and Documentation','',1,'lead1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"uppercase\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-sitemap\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"C8C8C8\",\"icon_link1\":\"\\/PathwayBrowser\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Pathway Browser\",\"title_color1\":\"444444\",\"title_link1\":\"\\/PathwayBrowser\",\"title_link_target1\":\"self\",\"description_text1\":\"Visualize and interact with Reactome biological pathways\",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-bar-chart\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"\\/PathwayBrowser\\/#TOOL=AT\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Analysis Tools\",\"title_color2\":\"444444\",\"title_link2\":\"\\/PathwayBrowser\\/#TOOL=AT\",\"title_link_target2\":\"self\",\"description_text2\":\"Merges pathway identifier mapping, <br\\/> over-representation, and expression analysis\",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-share-alt-square\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"index.php?option=com_content&view=article&id=44\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"ReactomeFIViz\",\"title_color3\":\"444444\",\"title_link3\":\"index.php?option=com_content&view=article&id=44\",\"title_link_target3\":\"self\",\"description_text3\":\"Designed to find pathways and network patterns related to cancer and other types of diseases\",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-file-text-o\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"documentation\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Documentation\",\"title_color4\":\"444444\",\"title_link4\":\"documentation\",\"title_link_target4\":\"self\",\"description_text4\":\"Information to browse the database and use its principal tools for data analysis\",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(479,487,'FavSocial - Reactome Social Medias','Module to create social media icons. Facebook, Twitter and YouTube','',1,'copyright1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favsocial',1,0,'{\"module_align\":\"favsocial-center\",\"show_icon1\":\"1\",\"icon_name1\":\"fa-twitter\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"AAAAAA\",\"icon_font_size1\":\"18px\",\"icon_link1\":\"https:\\/\\/twitter.com\\/intent\\/user?screen_name=reactome\",\"icon_link_target1\":\"blank\",\"icon_padding1\":\"0.2em\",\"icon_border_radius1\":\"4px\",\"show_icon2\":\"1\",\"icon_name2\":\"fa-facebook\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"AAAAAA\",\"icon_font_size2\":\"18px\",\"icon_link2\":\"https:\\/\\/www.facebook.com\\/reactome\",\"icon_link_target2\":\"blank\",\"icon_padding2\":\"0.2em\",\"icon_border_radius2\":\"4px\",\"show_icon3\":\"1\",\"icon_name3\":\"fa-youtube\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"AAAAAA\",\"icon_font_size3\":\"18px\",\"icon_link3\":\"https:\\/\\/www.youtube.com\\/@Reactome\",\"icon_link_target3\":\"blank\",\"icon_padding3\":\"0.2em\",\"icon_border_radius3\":\"4px\",\"show_icon4\":\"0\",\"icon_name4\":\"\",\"icon_color4\":\"\",\"icon_bg_color4\":\"\",\"icon_font_size4\":\"\",\"icon_link4\":\"\",\"icon_link_target4\":\"blank\",\"icon_padding4\":\"\",\"icon_border_radius4\":\"\",\"show_icon5\":\"0\",\"icon_name5\":\"\",\"icon_color5\":\"\",\"icon_bg_color5\":\"\",\"icon_font_size5\":\"\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_padding5\":\"\",\"icon_border_radius5\":\"\",\"show_icon6\":\"0\",\"icon_name6\":\"\",\"icon_color6\":\"\",\"icon_bg_color6\":\"\",\"icon_font_size6\":\"\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_padding6\":\"\",\"icon_border_radius6\":\"\",\"show_icon7\":\"0\",\"icon_name7\":\"\",\"icon_color7\":\"\",\"icon_bg_color7\":\"\",\"icon_font_size7\":\"\",\"icon_link7\":\"\",\"icon_link_target7\":\"blank\",\"icon_padding7\":\"\",\"icon_border_radius7\":\"\",\"show_icon8\":\"0\",\"icon_name8\":\"\",\"icon_color8\":\"\",\"icon_bg_color8\":\"\",\"icon_font_size8\":\"\",\"icon_link8\":\"\",\"icon_link_target8\":\"blank\",\"icon_padding8\":\"\",\"icon_border_radius8\":\"\",\"show_icon9\":\"0\",\"icon_name9\":\"\",\"icon_color9\":\"\",\"icon_bg_color9\":\"\",\"icon_font_size9\":\"\",\"icon_link9\":\"\",\"icon_link_target9\":\"blank\",\"icon_padding9\":\"\",\"icon_border_radius9\":\"\",\"show_icon10\":\"0\",\"icon_name10\":\"\",\"icon_color10\":\"\",\"icon_bg_color10\":\"\",\"icon_font_size10\":\"\",\"icon_link10\":\"\",\"icon_link_target10\":\"blank\",\"icon_padding10\":\"\",\"icon_border_radius10\":\"\",\"moduleclass_sfx\":\"pull-center favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(484,505,'Homepage black banner - alternative','Homepage black banner - alternative','<div id=\"fav-download\">\r\n<p id=\"fav-download-message\"><span style=\"color: #ffffff;\">USE REACTOME GRAPH DATABASE IN YOUR PROJECT</span></p>\r\n<p id=\"fav-download-btn\"><a href=\"dev/graph-database\" class=\"favth-btn\">LEARN MORE</a></p>\r\n</div>',1,'promo1',0,'0000-00-00 00:00:00','2020-09-25 14:30:15','0000-00-00 00:00:00',0,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(496,518,'Tweets','','',1,'prime2',0,'0000-00-00 00:00:00','2017-04-19 13:41:08','0000-00-00 00:00:00',0,'mod_pro_twitter_widget',1,1,'{\"twitter_id\":\"344458797739294721\",\"twitter_name\":\"reactome\",\"width\":\"600\",\"height\":\"456\",\"scrollbar\":\"false\",\"color_scheme\":\"1\",\"backgroundColor\":\"#58c3e5\",\"header\":\"false\",\"footer\":\"true\",\"border\":\"false\",\"tranparent\":\"true\",\"backgroundColors\":\"#292f33\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-clear favth-shadow tweet-widget\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-twitter\",\"style\":\"0\"}',0,'*'),(499,521,'Find Reactions, Proteins and Pathways','SolR Search input text. Always visible','<h3 class=\"visible-desktop\">Find Reactions, Proteins and Pathways</h3>\r\n<div class=\"nav-collapse-search search\"><form action=\"/content/query\" method=\"get\" id=\"search_form\" class=\"form-inline\"><label for=\"local-searchbox\" class=\"element-invisible\">Search ...</label> <input maxlength=\"200\" name=\"q\" size=\"10\" type=\"search\" id=\"local-searchbox\" class=\"inputbox search-query\" autofocus=\"\" placeholder=\"e.g. O95631, NTN1, signaling by EGFR, glucose\" /> <input name=\"species\" type=\"hidden\" value=\"Homo sapiens\" /> <input name=\"species\" type=\"hidden\" value=\"Entries without species\" /> <input name=\"cluster\" type=\"hidden\" value=\"true\" /> <button class=\"button btn btn-primary btn-info\">Go!</button></form></div>',1,'slide',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" text-center main-search\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-search text-center\",\"style\":\"0\"}',0,'*'),(500,522,'Notice','Warning this is Reactome Dev / Release...','<div class=\"hidden-phone\">\r\n<p class=\"rlp-noticemsg\" style=\"margin-top: 5px;\"><i class=\"fa fa-exclamation-triangle fa-1\"></i><span style=\"font-size: 8pt;\">THIS SITE IS USED FOR SOFTWARE DEVELOPMENT AND TESTING&nbsp;</span><br /> <span style=\"font-size: 8pt;\">IT IS NOT STABLE, IS LINKED TO AN INCOMPLETE DATA SET, AND IS NOT MONITORED FOR PERFORMANCE. WE STRONGLY RECOMMEND THE USE OF OUR&nbsp;<a href=\"https://reactome.org/\">PUBLIC SITE</a></span></p>\r\n</div>\r\n<div class=\"hidden-tablet hidden-desktop\">\r\n<p class=\"rlp-noticemsg\"><i class=\"fa fa-exclamation-triangle fa-1\"></i><span style=\"font-size: 8pt;\">THIS SITE IS USED FOR SOFTWARE DEVELOPMENT AND TESTING&nbsp;</span><span style=\"font-size: 8pt;\">WE STRONGLY RECOMMEND THE USE OF OUR&nbsp;<a href=\"https://reactome.org/\">PUBLIC SITE</a></span></p>\r\n</div>',1,'notice',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(505,539,'Why Reactome','Short description for Landing Page. Be careful when adding text here. It may misalign the Twitter Widget. To fix, adjust the .twitter-timeline height in custom.css','<p style=\"text-align: left;\">Reactome is a free, open-source, curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology and education. </p>\r\n<p class=\"text-center\" style=\"font-size: 12px;\"></p>\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 pull-center favth-hidden-sm favth-hidden-xs\" style=\"background-color: ghostwhite; margin: 0; padding: 10px; border-radius: 8px; border: 2px solid #58C3E5;\">\r\n<div class=\"favth-col-lg-3 favth-col-md-3 favth-col-sm-1 text-center\"><a href=\"http://www.ebi.ac.uk/\" target=\"_blank\" rel=\"noopener noreferrer\"><img src=\"images/sponsor/embl_ebi_logo.png\" alt=\"European Bioinformatics Institute (EMBL-EBI)\" style=\"height: 34px; margin-top: 7px;\" /></a></div>\r\n<div class=\"favth-col-lg-3 favth-col-md-3 favth-col-sm-1 text-right\" style=\"padding-right: 0;\"><span style=\"font-size: 10pt;\"><a href=\"https://med.nyu.edu/\" target=\"_blank\" rel=\"noopener noreferrer\"><img src=\"images/sponsor/nyul_health_logo.png\" alt=\"NYU Langone Health\" style=\"height: 39px; padding-right: 0;\" /></a></span></div>\r\n<div class=\"favth-col-lg-3 favth-col-md-3 favth-col-sm-1 text-center\"><span style=\"font-size: 10pt;\"><a href=\"http://www.ohsu.edu/\" target=\"_blank\" rel=\"noopener noreferrer\"><img src=\"images/sponsor/ohsu_logo.png\" alt=\"Oregon Health &amp; Science University\" style=\"height: 39px;\" /></a></span></div>\r\n<div class=\"favth-col-lg-3 favth-col-md-3 favth-col-sm-1 text-center\"><span style=\"font-size: 10pt;\"><a href=\"https://oicr.on.ca/\" target=\"_blank\" rel=\"noopener noreferrer\"><img src=\"images/sponsor/oicr_logo.png\" alt=\"Ontario Institute for Cancer Research\" style=\"height: 39px;\" /></a></span></div>\r\n</div>\r\n<p style=\"text-align: center; line-height: 13px; clear: both; padding-top: 10px;\"><span style=\"font-size: 8pt;\">The development of Reactome is supported by <a href=\"index.php?Itemid=807\">grants</a> from the US National Institutes of Health (U24 HG012198) and the European Molecular Biology Laboratory.</span></p>',1,'prime1',0,'0000-00-00 00:00:00','2017-10-03 13:44:12','0000-00-00 00:00:00',1,'mod_custom',1,1,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-clear favth-shadow\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-search\",\"style\":\"0\"}',0,'*'),(508,544,'Latest News','','',1,'prime2',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_articles_category',1,1,'{\"mode\":\"normal\",\"show_on_article_page\":0,\"count\":8,\"show_front\":\"hide\",\"category_filtering_type\":1,\"catid\":[26],\"show_child_category_articles\":0,\"levels\":1,\"author_filtering_type\":1,\"author_alias_filtering_type\":1,\"excluded_articles\":\"\",\"date_filtering\":\"off\",\"date_field\":\"a.created\",\"start_date_range\":\"\",\"end_date_range\":\"\",\"relative_date\":30,\"article_ordering\":\"publish_up\",\"article_ordering_direction\":\"DESC\",\"article_grouping\":\"none\",\"date_grouping_field\":\"created\",\"month_year_format\":\"F Y\",\"article_grouping_direction\":\"ksort\",\"link_titles\":1,\"show_date\":0,\"show_date_field\":\"created\",\"show_date_format\":\"Y-m-d H:i:s\",\"show_category\":0,\"show_hits\":0,\"show_author\":0,\"show_tags\":0,\"show_introtext\":0,\"introtext_limit\":100,\"show_readmore\":0,\"show_readmore_title\":0,\"readmore_limit\":15,\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-clear favth-shadow latest-news\",\"owncache\":1,\"cache_time\":900,\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-newspaper-o\",\"style\":\"0\"}',0,'*'),(518,571,'Maintop Blank Space','Blank Space between nav and main-content','',1,'maintop3',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(519,572,'Find Reactions, Proteins and Pathways - Others','SolR Search input text. Short version without description. To be used in all other pages.','<div class=\"search\"><form action=\"/content/query\" method=\"get\" class=\"clean-form form-inline\"><label for=\"local-searchbox\" class=\"element-invisible\">Search ...</label> <input maxlength=\"200\" name=\"q\" size=\"10\" type=\"search\" id=\"local-searchbox\" class=\"inputbox search-query alt-searchbox\" autofocus=\"\" placeholder=\"e.g. O95631, NTN1, signaling by EGFR, glucose\" /> <input name=\"species\" type=\"hidden\" value=\"Homo sapiens\" /> <input name=\"species\" type=\"hidden\" value=\"Entries without species\" /> <input name=\"cluster\" type=\"hidden\" value=\"true\" /> <button class=\"button btn btn-primary btn-info\">Go!</button></form></div>',1,'slide',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" text-center alt-search\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-search text-center\",\"style\":\"0\"}',0,'*'),(524,591,'Articles Sidebar Menu','','',1,'sidebar1',0,'0000-00-00 00:00:00','2017-04-26 13:39:34','0000-00-00 00:00:00',1,'mod_menu',1,0,'{\"menutype\":\"mainmenu\",\"base\":\"\",\"startLevel\":2,\"endLevel\":2,\"showAllChildren\":1,\"tag_id\":\"\",\"class_sfx\":\"favth-side\",\"window_open\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"itemid\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(525,592,'Icon Library','','',1,'icon-lib',0,'0000-00-00 00:00:00','2017-05-17 11:12:38','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(526,595,'[TEMPLATE] data-content / swagger placeholder','Div placeholder for template-page. This is hidden from the normal website','<!-- template-placeholder -->',1,'lead1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":0,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(527,596,'[TEMPLATE] search placeholder','Search Form Placeholder, To be used as template in the DataContent Project. This is hidden from the normal website','<!--SearchForm-->\r\n<p>&nbsp;</p>\r\n<!--/SearchForm-->',1,'slide',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" text-center alt-search\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-search text-center\",\"style\":\"0\"}',0,'*'),(533,603,'Footer','','',1,'footer1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_menu',1,0,'{\"menutype\":\"mainmenu\",\"base\":101,\"startLevel\":1,\"endLevel\":2,\"showAllChildren\":1,\"tag_id\":\"\",\"class_sfx\":\" footer-menu\",\"window_open\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-visible-md favth-visible-lg favth-visible-sm\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"itemid\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(540,615,'Developers Zone - Row 1','','',1,'dev-guide-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"28px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"40%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-bar-chart\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"CCCCCC\",\"icon_link1\":\"index.php?option=com_content&view=article&id=47\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4em\",\"title_text1\":\"Analysis Service\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=47\",\"title_link_target1\":\"self\",\"description_text1\":\"Use the Analysis Service to analyse your data against Reactome\\u2019s content\",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"40%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-code\",\"icon_color2\":\"FF5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"CCCCCC\",\"icon_link2\":\"index.php?option=com_content&view=article&id=48\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4em\",\"title_text2\":\"Content Service\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=48\",\"title_link_target2\":\"self\",\"description_text2\":\"Use the Content Service to access all our knowledgebase content from your client\",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"40%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-database\",\"icon_color3\":\"8BC34A\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"CCCCCC\",\"icon_link3\":\"index.php?option=com_content&view=article&id=45\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4em\",\"title_text3\":\"Graph Database\",\"title_color3\":\"444444\",\"title_link3\":\"index.php?option=com_content&view=article&id=45\",\"title_link_target3\":\"self\",\"description_text3\":\"Access to the Reactome knowledgebase content as an interconnected graph database\",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-lightbulb-o\",\"icon_color4\":\"FFC107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"CCCCCC\",\"icon_link4\":\"\",\"icon_link_target4\":\"blank\",\"icon_font_size4\":\"4em\",\"title_text4\":\"<<<< READ DESCRIPTION >>>>\",\"title_color4\":\"444444\",\"title_link4\":\"\",\"title_link_target4\":\"blank\",\"description_text4\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4em\",\"title_text5\":\"<<< READ DESCRIPTION >>>\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"4em\",\"title_text6\":\"<<< READ DESCRIPTION >>>\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"-sfx favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(544,627,'Documentation','','',1,'userguide-cp-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favpromote',1,1,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_padding\":\"10px 20px\",\"title_font_size\":\"18px\",\"title_line_height\":\"1.4em\",\"title_text_align\":\"left\",\"title_icon_font_size\":\"18px\",\"title_icon_vertical_align\":\"baseline\",\"description_text_font_size\":\"14px\",\"description_text_line_height\":\"1.5em\",\"description_text_align\":\"justify\",\"show_column1\":\"1\",\"column_border_color1\":\"DDDDDD\",\"column_border_radius1\":\"4px\",\"upload_image1\":\"images\\/docs\\/for_users_110x110.png\",\"image_link1\":\"index.php?option=com_content&view=article&id=39\",\"image_target1\":\"self\",\"image_alt1\":\"\",\"title_text1\":\"For Users\",\"title_color1\":\"FFFFFF\",\"title_bg_color1\":\"58C3E5\",\"title_link1\":\"index.php?option=com_content&view=article&id=39\",\"title_link_target1\":\"self\",\"title_icon1\":\"fa fa-users\",\"description_text1\":\"Take the most out of our tools and data analysis. All you need is our User Guide.\",\"description_text_color1\":\"444444\",\"show_column2\":\"1\",\"column_border_color2\":\"DDDDDD\",\"column_border_radius2\":\"4px\",\"upload_image2\":\"images\\/docs\\/for_developers_110x110.png\",\"image_link2\":\"dev\",\"image_target2\":\"self\",\"image_alt2\":\"\",\"title_text2\":\"For Developers\",\"title_color2\":\"FFFFFF\",\"title_bg_color2\":\"58C3E5\",\"title_link2\":\"dev\",\"title_link_target2\":\"self\",\"title_icon2\":\"fa fa-code\",\"description_text2\":\"Are you interested in programatically querying our data or integrating our Widgets ?\",\"description_text_color2\":\"444444\",\"show_column3\":\"1\",\"column_border_color3\":\"DDDDDD\",\"column_border_radius3\":\"4px\",\"upload_image3\":\"images\\/docs\\/citing_us_110x110.png\",\"image_link3\":\"cite\",\"image_target3\":\"self\",\"image_alt3\":\"\",\"title_text3\":\"Citing us\",\"title_color3\":\"FFFFFF\",\"title_bg_color3\":\"58C3E5\",\"title_link3\":\"cite\",\"title_link_target3\":\"self\",\"title_icon3\":\"fa fa-pencil\",\"description_text3\":\"Have our data been useful in your research or experiment ? Please, remember to cite us!\",\"description_text_color3\":\"444444\",\"show_column4\":\"0\",\"column_border_color4\":\"DDDDDD\",\"column_border_radius4\":\"4px\",\"upload_image4\":\"\",\"image_link4\":\"\",\"image_target4\":\"blank\",\"image_alt4\":\"\",\"title_text4\":\"Title Text\",\"title_color4\":\"FFFFFF\",\"title_bg_color4\":\"D63E52\",\"title_link4\":\"\",\"title_link_target4\":\"blank\",\"title_icon4\":\"fa-magic\",\"description_text4\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color4\":\"444444\",\"show_column5\":\"0\",\"column_border_color5\":\"DDDDDD\",\"column_border_radius5\":\"4px\",\"upload_image5\":\"\",\"image_link5\":\"\",\"image_target5\":\"blank\",\"image_alt5\":\"\",\"title_text5\":\"Title Text\",\"title_color5\":\"FFFFFF\",\"title_bg_color5\":\"D63E52\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"title_icon5\":\"fa-quote-right\",\"description_text5\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color5\":\"444444\",\"show_column6\":\"0\",\"column_border_color6\":\"DDDDDD\",\"column_border_radius6\":\"4px\",\"upload_image6\":\"\",\"image_link6\":\"\",\"image_target6\":\"blank\",\"image_alt6\":\"\",\"title_text6\":\"Title Text\",\"title_color6\":\"FFFFFF\",\"title_bg_color6\":\"D63E52\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"title_icon6\":\"fa-recycle\",\"description_text6\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"System-none\"}',0,'*'),(551,647,'Partners','','',1,'block1',184,'2024-10-03 18:18:32','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favpromote4col',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_padding\":\"10px 20px\",\"title_font_size\":\"15px\",\"title_line_height\":\"1.4em\",\"title_text_align\":\"left\",\"title_icon_font_size\":\"15px\",\"title_icon_vertical_align\":\"baseline\",\"description_text_font_size\":\"14px\",\"description_text_line_height\":\"1.5em\",\"description_text_align\":\"justify\",\"order1\":\"1\",\"show_column1\":\"1\",\"column_border_color1\":\"DDDDDD\",\"column_border_radius1\":\"4px\",\"upload_image1\":\"images\\/partners\\/openTargets_logo.png\",\"image_link1\":\"http:\\/\\/www.targetvalidation.org\\/target\\/ENSG00000157764\",\"image_target1\":\"blank\",\"image_alt1\":\"\",\"title_text1\":\"Open Targets\",\"title_color1\":\"FFFFFF\",\"title_bg_color1\":\"58C3E5\",\"title_link1\":\"http:\\/\\/www.targetvalidation.org\\/target\\/ENSG00000157764\",\"title_link_target1\":\"blank\",\"title_icon1\":\"fa-empty\",\"description_text1\":\"For biomedical researchers who need to identify a biological target for a new therapy, Open Targets is a public-private initiative to generate evidence on the validity of therapeutic targets based on genome-scale experiments and analysis.\",\"widget_text1\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color1\":\"444444\",\"order2\":\"2\",\"show_column2\":\"1\",\"column_border_color2\":\"DDDDDD\",\"column_border_radius2\":\"4px\",\"upload_image2\":\"images\\/partners\\/The_Human_Protein_Atlas.png\",\"image_link2\":\"http:\\/\\/www.proteinatlas.org\\/ENSG00000132155-RAF1\\/cell\",\"image_target2\":\"blank\",\"image_alt2\":\"\",\"title_text2\":\"The Human Protein Atlas\",\"title_color2\":\"FFFFFF\",\"title_bg_color2\":\"58C3E5\",\"title_link2\":\"http:\\/\\/www.proteinatlas.org\\/ENSG00000132155-RAF1\\/cell\",\"title_link_target2\":\"blank\",\"title_icon2\":\"fa-empty\",\"description_text2\":\"Contains information for a large majority of all human protein-coding genes regarding the expression and localization of the corresponding proteins based on both RNA and protein data.\",\"widget_text2\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color2\":\"444444\",\"order3\":\"5\",\"show_column3\":\"1\",\"column_border_color3\":\"DDDDDD\",\"column_border_radius3\":\"4px\",\"upload_image3\":\"images\\/partners\\/complexportal.png\",\"image_link3\":\"https:\\/\\/www.ebi.ac.uk\\/complexportal\\/complex\\/EBI-9008420\",\"image_target3\":\"blank\",\"image_alt3\":\"\",\"title_text3\":\"Complex Portal\",\"title_color3\":\"FFFFFF\",\"title_bg_color3\":\"58C3E5\",\"title_link3\":\"https:\\/\\/www.ebi.ac.uk\\/complexportal\\/complex\\/EBI-9008420\",\"title_link_target3\":\"blank\",\"title_icon3\":\"fa-empty\",\"description_text3\":\"The Complex Portal is a manually curated, encyclopaedic resource of macromolecular complexes from a number of key model organisms. The majority of complexes are made up of proteins but may also include nucleic acids or small molecules.\",\"widget_text3\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Content Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color3\":\"444444\",\"order4\":\"4\",\"show_column4\":\"1\",\"column_border_color4\":\"DDDDDD\",\"column_border_radius4\":\"4px\",\"upload_image4\":\"images\\/partners\\/blueprint_logo.png\",\"image_link4\":\"http:\\/\\/dcc.blueprint-epigenome.eu\\/#\\/experiments\\/ERX641310\",\"image_target4\":\"blank\",\"image_alt4\":\"\",\"title_text4\":\"BLUEPRINT\",\"title_color4\":\"FFFFFF\",\"title_bg_color4\":\"58C3E5\",\"title_link4\":\"http:\\/\\/dcc.blueprint-epigenome.eu\\/#\\/experiments\\/ERX641310\",\"title_link_target4\":\"blank\",\"title_icon4\":\"fa-empty\",\"description_text4\":\"The BLUEPRINT consortium has been formed with the aim to further the understanding of how genes are activated or repressed in both healthy and diseased human cells.\\r\\n\",\"widget_text4\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color4\":\"444444\",\"order5\":\"6\",\"show_column5\":\"1\",\"column_border_color5\":\"DDDDDD\",\"column_border_radius5\":\"4px\",\"upload_image5\":\"images\\/partners\\/ChEBI_logo.png\",\"image_link5\":\"http:\\/\\/www.ebi.ac.uk\\/chebi\\/pathway.do?chebiId=CHEBI:11230\",\"image_target5\":\"blank\",\"image_alt5\":\"\",\"title_text5\":\"ChEBI\",\"title_color5\":\"FFFFFF\",\"title_bg_color5\":\"58C3E5\",\"title_link5\":\"http:\\/\\/www.ebi.ac.uk\\/chebi\\/pathway.do?chebiId=CHEBI:11230\",\"title_link_target5\":\"blank\",\"title_icon5\":\"fa-empty\",\"description_text5\":\"Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on \\u2018small\\u2019 chemical compounds. The term \\u2018molecular entity\\u2019 refers to any constitutionally or isotopically distinct atom, molecule, ion, ion pair, radical, radical ion, complex, conformer, etc., identifiable as a separately distinguishable entity.\",\"widget_text5\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color5\":\"444444\",\"order6\":\"8\",\"show_column6\":\"1\",\"column_border_color6\":\"DDDDDD\",\"column_border_radius6\":\"4px\",\"upload_image6\":\"images\\/partners\\/pride_logo.png\",\"image_link6\":\"http:\\/\\/www.ebi.ac.uk\\/pride\\/archive\\/projects\\/PXD000764\",\"image_target6\":\"blank\",\"image_alt6\":\"\",\"title_text6\":\"PRIDE\",\"title_color6\":\"FFFFFF\",\"title_bg_color6\":\"58C3E5\",\"title_link6\":\"http:\\/\\/www.ebi.ac.uk\\/pride\\/archive\\/projects\\/PXD000764\",\"title_link_target6\":\"blank\",\"title_icon6\":\"fa-empty\",\"description_text6\":\"PRoteomics IDEntifications (PRIDE) database is a centralized, standards compliant, public data repository for proteomics data, including protein and peptide identifications, post-translational modifications and supporting spectral evidence.\",\"widget_text6\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color6\":\"444444\",\"order7\":\"9\",\"show_column7\":\"1\",\"column_border_color7\":\"DDDDDD\",\"column_border_radius7\":\"4px\",\"upload_image7\":\"images\\/partners\\/logo.png\",\"image_link7\":\"http:\\/\\/iomics.ugent.be\\/tabloidproteome\",\"image_target7\":\"blank\",\"image_alt7\":\"\",\"title_text7\":\"Tabloid Proteome\",\"title_color7\":\"FFFFFF\",\"title_bg_color7\":\"2F9EC2\",\"title_link7\":\"http:\\/\\/iomics.ugent.be\\/tabloidproteome\",\"title_link_target7\":\"blank\",\"title_icon7\":\"fa-empty\",\"description_text7\":\"Tabloid Proteome is a database of protein association network generated using publicly available mass spectrometry based experiments in PRIDE. These associations represent a broad range of biological associations between pairs of proteins that goes well beyond mere binary protein interactions.\",\"widget_text7\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color7\":\"444444\",\"order8\":\"10\",\"show_column8\":\"1\",\"column_border_color8\":\"DDDDDD\",\"column_border_radius8\":\"4px\",\"upload_image8\":\"images\\/partners\\/DCIC_500x375_cropped.png\",\"image_link8\":\"http:\\/\\/www.ilincs.org\\/ilincs\\/signature\\/GDS_5917\",\"image_target8\":\"blank\",\"image_alt8\":\"\",\"title_text8\":\"iLINCS\",\"title_color8\":\"FFFFFF\",\"title_bg_color8\":\"2F9EC2\",\"title_link8\":\"http:\\/\\/www.ilincs.org\\/ilincs\\/signature\\/GDS_5917\",\"title_link_target8\":\"blank\",\"title_icon8\":\"fa-empty\",\"description_text8\":\"Integrative LINCS is an integrative web platform for analysis of LINCS data and signatures.The BD2K-LINCS Data Coordination and Integration Center is part of the Big Data to Knowledge (BD2K) NIH initiative.\",\"widget_text8\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color8\":\"444444\",\"order9\":\"7\",\"show_column9\":\"0\",\"column_border_color9\":\"DDDDDD\",\"column_border_radius9\":\"4px\",\"upload_image9\":\"images\\/partners\\/EPD.png\",\"image_link9\":\"https:\\/\\/peptracker.com\\/epd\\/analytics\\/?section_id=20202\",\"image_target9\":\"blank\",\"image_alt9\":\"\",\"title_text9\":\"PepTracker: EPD\",\"title_color9\":\"FFFFFF\",\"title_bg_color9\":\"2F9EC2\",\"title_link9\":\"https:\\/\\/peptracker.com\\/epd\\/analytics\\/?section_id=20202\",\"title_link_target9\":\"blank\",\"title_icon9\":\"fa-empty\",\"description_text9\":\"The Encyclopedia of Proteome Dynamics is a polyglot persistent database and web-application that provides open access to integrated proteomics data from published studies on human cells and model organisms featuring graphical navigation with interactive visualizations that facilitate powerful data exploration in an intuitive manner.\",\"widget_text9\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color9\":\"444444\",\"order10\":\"11\",\"show_column10\":\"0\",\"column_border_color10\":\"DDDDDD\",\"column_border_radius10\":\"4px\",\"upload_image10\":\"images\\/partners\\/ipa_logo.png\",\"image_link10\":\"http:\\/\\/www.integratedproteomics.com\\/\",\"image_target10\":\"blank\",\"image_alt10\":\"\",\"title_text10\":\"IPA\",\"title_color10\":\"FFFFFF\",\"title_bg_color10\":\"2F9EC2\",\"title_link10\":\"http:\\/\\/www.integratedproteomics.com\\/\",\"title_link_target10\":\"blank\",\"title_icon10\":\"fa-empty\",\"description_text10\":\"Integrated Proteomics Applications provides a comprehensive proteomic data analysis solution, the Integrated Proteomics Pipeline (IP2), which allows researchers to identify, quantify, and functionally analyze all of the proteins in a biological sample on one, easy to use interface.\",\"widget_text10\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color10\":\"444444\",\"order11\":\"19\",\"show_column11\":\"1\",\"column_border_color11\":\"DDDDDD\",\"column_border_radius11\":\"4px\",\"upload_image11\":\"images\\/partners\\/alliance_logo.png\",\"image_link11\":\"https:\\/\\/www.alliancegenome.org\\/gene\\/HGNC:9588#pathways\",\"image_target11\":\"blank\",\"image_alt11\":\"\",\"title_text11\":\"Alliance of Genome Resources\",\"title_color11\":\"FFFFFF\",\"title_bg_color11\":\"2F9EC2\",\"title_link11\":\"https:\\/\\/www.alliancegenome.org\\/gene\\/HGNC:9588#pathways\",\"title_link_target11\":\"blank\",\"title_icon11\":\"fa-empty\",\"description_text11\":\"A consortium of 7 model organism databases (MODs) and the Gene Ontology (GO) Consortium whose goal is to provide an integrated view of their data to all biologists, clinicians and other interested parties.\\r\\n\",\"widget_text11\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color11\":\"444444\",\"order12\":\"18\",\"show_column12\":\"1\",\"column_border_color12\":\"DDDDDD\",\"column_border_radius12\":\"4px\",\"upload_image12\":\"images\\/partners\\/SimpliFiByProtifi.png\",\"image_link12\":\"https:\\/\\/simplifi.protifi.com\\/#\\/p\\/97fb34a0-c963-11eb-83a8-05b21c48984a\",\"image_target12\":\"blank\",\"image_alt12\":\"\",\"title_text12\":\"SimpliFi\",\"title_color12\":\"FFFFFF\",\"title_bg_color12\":\"2F9EC2\",\"title_link12\":\"https:\\/\\/simplifi.protifi.com\\/#\\/p\\/97fb34a0-c963-11eb-83a8-05b21c48984a\",\"title_link_target12\":\"blank\",\"title_icon12\":\"fa-empty\",\"description_text12\":\"SimpliFi is an online, browser-accessible platform that translates omics data into biological understanding. Users of all skill levels can explore and visualize their data to understand a biological state and form hypotheses. SimpliFi accepts and integrates data into clean, interactive displays of enriched differentially expressed pathways, GO enrichments, cellular markers, protein-protein interaction networks and molecular-level classifications.\",\"widget_text12\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color12\":\"444444\",\"order13\":\"3\",\"show_column13\":\"1\",\"column_border_color13\":\"DDDDDD\",\"column_border_radius13\":\"4px\",\"upload_image13\":\"images\\/partners\\/ensembl.png\",\"image_link13\":\"http:\\/\\/www.ensembl.org\\/Homo_sapiens\\/Gene\\/Pathway?db=core;g=ENSG00000139618;r=13:32315474-32400266\",\"image_target13\":\"blank\",\"image_alt13\":\"\",\"title_text13\":\"Ensembl\",\"title_color13\":\"FFFFFF\",\"title_bg_color13\":\"2F9EC2\",\"title_link13\":\"http:\\/\\/www.ensembl.org\\/Homo_sapiens\\/Gene\\/Pathway?db=core;g=ENSG00000139618;r=13:32315474-32400266\",\"title_link_target13\":\"blank\",\"title_icon13\":\"fa-empty\",\"description_text13\":\"Ensembl is a genome browser for vertebrate genomes that supports research in comparative genomics, evolution, sequence variation and transcriptional regulation. Ensembl annotate genes, computes multiple alignments, predicts regulatory function and collects disease data. \",\"widget_text13\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color13\":\"444444\",\"order14\":\"14\",\"show_column14\":\"1\",\"column_border_color14\":\"DDDDDD\",\"column_border_radius14\":\"4px\",\"upload_image14\":\"images\\/partners\\/targetpathogen.png\",\"image_link14\":\"http:\\/\\/target.sbg.qb.fcen.uba.ar\\/patho\\/protein\\/56425909be737e6c7a9fd658#patho_host_row\",\"image_target14\":\"blank\",\"image_alt14\":\"\",\"title_text14\":\"Target-Pathogen\",\"title_color14\":\"FFFFFF\",\"title_bg_color14\":\"2F9EC2\",\"title_link14\":\"http:\\/\\/target.sbg.qb.fcen.uba.ar\\/patho\\/protein\\/56425909be737e6c7a9fd658#patho_host_row\",\"title_link_target14\":\"blank\",\"title_icon14\":\"fa-empty\",\"description_text14\":\"Target-Pathogen database is a bioinformatic approach to prioritize and identify candidates drug targets for pathogens. Designed and developed as an online resource to allow genome wide data consolidation from diverse sources focusing on structural druggability, essentiality and metabolic role of proteins. \\r\\n\",\"widget_text14\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color14\":\"444444\",\"order15\":\"15\",\"show_column15\":\"1\",\"column_border_color15\":\"DDDDDD\",\"column_border_radius15\":\"4px\",\"upload_image15\":\"images\\/partners\\/BioModel_logo.png\",\"image_link15\":\"https:\\/\\/www.ebi.ac.uk\\/biomodels\\/BIOMD0000000300\",\"image_target15\":\"blank\",\"image_alt15\":\"\",\"title_text15\":\"BioModels\",\"title_color15\":\"FFFFFF\",\"title_bg_color15\":\"2F9EC2\",\"title_link15\":\"https:\\/\\/www.ebi.ac.uk\\/biomodels\\/BIOMD0000000300\",\"title_link_target15\":\"blank\",\"title_icon15\":\"fa-empty\",\"description_text15\":\"BioModels is a repository of mathematical models of biological and biomedical systems. It hosts a vast selection of existing literature-based physiologically and pharmaceutically relevant mechanistic models in standard formats. Our mission is to provide the systems modelling community with reproducible, high-quality, freely-accessible models published in the scientific literature. \",\"widget_text15\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color15\":\"444444\",\"order16\":\"16\",\"show_column16\":\"1\",\"column_border_color16\":\"DDDDDD\",\"column_border_radius16\":\"4px\",\"upload_image16\":\"images\\/partners\\/pharos.png\",\"image_link16\":\"https:\\/\\/pharos.nih.gov\\/targets\\/O94886#pathways\",\"image_target16\":\"blank\",\"image_alt16\":\"\",\"title_text16\":\"PHAROS\",\"title_color16\":\"FFFFFF\",\"title_bg_color16\":\"2F9EC2\",\"title_link16\":\"https:\\/\\/pharos.nih.gov\\/targets\\/O94886#pathways\",\"title_link_target16\":\"blank\",\"title_icon16\":\"fa-empty\",\"description_text16\":\"Pharos is the user interface to the Knowledge Management Center (KMC) for the Illuminating the Druggable Genome (IDG) program.\",\"widget_text16\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color16\":\"444444\",\"order17\":\"17\",\"show_column17\":\"1\",\"column_border_color17\":\"DDDDDD\",\"column_border_radius17\":\"4px\",\"upload_image17\":\"images\\/partners\\/UniProt.png\",\"image_link17\":\"https:\\/\\/disease.uniprot.org\\/disease\\/DI-03832\\/protein\\/O00116\\/pathway\",\"image_target17\":\"blank\",\"image_alt17\":\"\",\"title_text17\":\"UniProt \",\"title_color17\":\"FFFFFF\",\"title_bg_color17\":\"2F9EC2\",\"title_link17\":\"http:\\/\\/disease.uniprot.org\\/disease\\/Alzheimer%20disease\\/protein\\/O00116\\/pathway\",\"title_link_target17\":\"blank\",\"title_icon17\":\"fa-empty\",\"description_text17\":\"UniProt has developed the Alzheimer\\u2019s disease portal to help researchers explore and access current AD genomic-based data from the UniProtKB database.\",\"widget_text17\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color17\":\"444444\",\"order18\":\"18\",\"show_column18\":\"1\",\"column_border_color18\":\"DDDDDD\",\"column_border_radius18\":\"4px\",\"upload_image18\":\"images\\/MD-logo-circle-alt-colour.png\",\"image_link18\":\"https:\\/\\/massdynamics.com\\/\",\"image_target18\":\"blank\",\"image_alt18\":\"\",\"title_text18\":\"Mass Dynamics\",\"title_color18\":\"FFFFFF\",\"title_bg_color18\":\"2F9EC2\",\"title_link18\":\"https:\\/\\/massdynamics.com\\/\",\"title_link_target18\":\"blank\",\"title_icon18\":\"fa-empty\",\"description_text18\":\"Mass dynamics is a commercial software product that enables proteomics workflow creation, allowing users to include custom analysis modules, such as Reactome Overrepresentation Analysis (ORA) and visualisation tools. \",\"widget_text18\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color18\":\"444444\",\"order19\":\"19\",\"show_column19\":\"1\",\"column_border_color19\":\"DDDDDD\",\"column_border_radius19\":\"4px\",\"upload_image19\":\"images\\/BioBox_-_Light_mode.png\",\"image_link19\":\"https:\\/\\/biobox.io\\/\",\"image_target19\":\"blank\",\"image_alt19\":\"\",\"title_text19\":\"BioBox\",\"title_color19\":\"FFFFFF\",\"title_bg_color19\":\"2F9EC2\",\"title_link19\":\"\",\"title_link_target19\":\"blank\",\"title_icon19\":\"fa-empty\",\"description_text19\":\"BioBox is a bio-data intelligence platform for drug discovery that helps research teams understand and interpret disease biology. Transform multi-omic data into a semantically enhanced data graph that can be interactively mined and analyzed to guide decision making in target prioritization, indication selection, and MOA analysis tasks.\",\"widget_text19\":\"<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color19\":\"444444\",\"order20\":\"20\",\"show_column20\":\"1\",\"column_border_color20\":\"DDDDDD\",\"column_border_radius20\":\"4px\",\"upload_image20\":\"images\\/partners\\/Metabolon-FullC-Logo-Sm2x.png\",\"image_link20\":\"\",\"image_target20\":\"blank\",\"image_alt20\":\"\",\"title_text20\":\"Metabolon\",\"title_color20\":\"FFFFFF\",\"title_bg_color20\":\"2F9EC2\",\"title_link20\":\"https:\\/\\/www.metaboloninc.com\\/?_gl=1*1t2z8z5*_up*MQ..&gclid=CjwKCAjwgfm3BhBeEiwAFfxrG58Pi9_EZsxjBNvohRyhhSPk92NOF6gn7jlemMhXt9eObGNogRYDrRoCbbQQAvD_BwE\",\"title_link_target20\":\"blank\",\"title_icon20\":\"fa-empty\",\"description_text20\":\"Metabolon is the global leader in metabolomics, with a mission to deliver biochemical data and insights that expand and accelerate the impact of life sciences research and complement other \'omics\' technologies.\",\"widget_text20\":\"<i class=\\\"fa fa-check-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Analysis Service\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Widgets\\r\\n<br\\/>\\r\\n<i class=\\\"fa fa-square-o\\\" aria-hidden=\\\"true\\\"><\\/i>Graph Database\",\"description_text_color20\":\"444444\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(555,651,'Do you need help ?','','',1,'screen1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,1,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"50%\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"300\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"20px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-heart\",\"icon_color1\":\"006FFF\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"C8C8C8\",\"icon_link1\":\"index.php?option=com_content&view=article&id=39\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4em\",\"title_text1\":\"Use Reactome!\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=39\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-code\",\"icon_color2\":\"0099FF\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"C8C8C8\",\"icon_link2\":\"dev\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4em\",\"title_text2\":\"For Developers\",\"title_color2\":\"444444\",\"title_link2\":\"dev\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-pencil\",\"icon_color3\":\"0099FF\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"C8C8C8\",\"icon_link3\":\"index.php?option=com_content&view=article&id=29\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4em\",\"title_text3\":\"Citing us\",\"title_color3\":\"444444\",\"title_link3\":\"index.php?option=com_content&view=article&id=29\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-envelope\",\"icon_color4\":\"0099FF\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"DDDDDD\",\"icon_link4\":\"mailto:help@reactome.org\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4em\",\"title_text4\":\"Contact Us\",\"title_color4\":\"444444\",\"title_link4\":\"mailto:help@reactome.org\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"DDDDDD\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4em\",\"title_text5\":\"Title Text\",\"title_color5\":\"111111\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"DDDDDD\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"4em\",\"title_text6\":\"Title Text\",\"title_color6\":\"111111\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favcustom\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h4\",\"header_class\":\"fa-question-circle\",\"style\":\"0\"}',0,'*'),(556,652,'API and Data access','','',1,'top1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,1,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"50%\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"300\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"20px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-laptop\",\"icon_color1\":\"006FFF\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"C8C8C8\",\"icon_link1\":\"\\/ContentService\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4em\",\"title_text1\":\"Content Service\",\"title_color1\":\"444444\",\"title_link1\":\"\\/ContentService\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-line-chart\",\"icon_color2\":\"0099FF\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"C8C8C8\",\"icon_link2\":\"\\/AnalysisService\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4em\",\"title_text2\":\"Analysis Service\",\"title_color2\":\"444444\",\"title_link2\":\"\\/AnalysisService\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-archive\",\"icon_color3\":\"0099FF\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"C8C8C8\",\"icon_link3\":\"icon-lib\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4em\",\"title_text3\":\"Icon Library\",\"title_color3\":\"444444\",\"title_link3\":\"icon-lib\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-database\",\"icon_color4\":\"0099FF\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"C8C8C8\",\"icon_link4\":\"index.php?option=com_content&view=article&id=45\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4em\",\"title_text4\":\"Graph Database\",\"title_color4\":\"444444\",\"title_link4\":\"index.php?option=com_content&view=article&id=45\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"DDDDDD\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4em\",\"title_text5\":\"Title Text\",\"title_color5\":\"111111\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"DDDDDD\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"4em\",\"title_text6\":\"Title Text\",\"title_color6\":\"111111\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favcustom\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa-bookmark\",\"style\":\"0\"}',0,'*'),(559,655,'Pathway Diagram - Dev Guide, 4 buttons','4 buttons in Pathway Diagram Dev Guide Page','',1,'dev-guide-pd-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(560,656,'Pathway Overview - Dev Guide, 4 buttons','4 buttons in Pathway Overview Dev Guide Page','',1,'dev-guide-po-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(561,657,'Pathway Diagram - Dev Guide - Get Started, 2 buttons ','Get Started 2 buttons in Pathway Diagram Dev Guide Page','',1,'dev-guide-pd-2',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"25%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"rglyph-gs-diagram-js\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"index.php?option=com_content&view=article&id=58\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Diagram JS\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=58\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"25%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"rglyph-gs-diagram-gwt\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"index.php?option=com_content&view=article&id=59\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Diagram GWT\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=59\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"0\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle fav2buttons\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(563,659,'Pathway Diagram - Dev Guide - API, 2 buttons','API 2 buttons in Pathway Diagram Dev Guide Page','',1,'dev-guide-pd-3',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"25%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"rglyph-diagram-js\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"index.php?option=com_content&view=article&id=58#API\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Diagram JS API\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=58#API\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"25%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"rglyph-diagram-gwt\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"index.php?option=com_content&view=article&id=59#API\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Diagram GWT API\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=59#API\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"0\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle fav2buttons\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(564,660,'Pathway Overview - Dev Guide - Get Started, 2 buttons','Get Started 2 buttons in Pathway Overview Dev Guide Page','',1,'dev-guide-po-2',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"25%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"rglyph-gs-fireworks-js\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"index.php?option=com_content&view=article&id=56\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Fireworks JS\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=56\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"25%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"rglyph-gs-fireworks-gwt\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"index.php?option=com_content&view=article&id=57\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Fireworks GWT\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=57\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"0\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle fav2buttons\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(565,661,'Pathway Overview - Dev Guide - API, 2 buttons','API 2 buttons in Pathway Overview Dev Guide Page','',1,'dev-guide-po-3',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"25%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"rglyph-fireworks-js\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"index.php?option=com_content&view=article&id=56#API\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Fireworks JS API\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=56#API\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"25%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"rglyph-fireworks-gwt\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"index.php?option=com_content&view=article&id=57#API\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Fireworks GWT API\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=57#API\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"0\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle fav2buttons\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(566,662,'Pathway Overview GWT - Dev Guide, 4 buttons','4 buttons in Pathway Overview GWT Dev Guide Page','',1,'dev-guide-po-gwt-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(567,663,'Pathway Diagram GWT - Dev Guide, 4 buttons','4 buttons in Pathway Diagram GWT Dev Guide Page','',1,'dev-guide-pd-gwt-1',0,'0000-00-00 00:00:00','2017-06-29 10:09:22','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(568,664,'Content Service - Dev Guide, 4 buttons','4 buttons in Content Service Dev Guide Page','',1,'dev-guide-cs-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"\\/ContentService\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"\\/ContentService\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(569,665,'Analysis Service - Dev Guide, 4 buttons','4 buttons in Analysis Service Dev Guide Page','',1,'dev-guide-as-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"\\/AnalysisService\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"\\/AnalysisService\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"6em\",\"title_text5\":\"Title Text\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(570,666,'Graph Database - Dev Guide, 4 buttons','4 buttons in Graph Database Dev Guide Page','',1,'dev-guide-gd-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"30px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"24px\",\"title_margin\":\"24px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"50%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-question-circle\",\"icon_color1\":\"FFFFFF\",\"icon_bg_color1\":\"0099FF\",\"icon_border_color1\":\"0099FF\",\"icon_link1\":\"#MoreInfo\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"More Info\",\"title_color1\":\"444444\",\"title_link1\":\"#MoreInfo\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"50%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-rocket\",\"icon_color2\":\"FFFFFF\",\"icon_bg_color2\":\"0099FF\",\"icon_border_color2\":\"0099FF\",\"icon_link2\":\"#GetStarted\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4.8em\",\"title_text2\":\"Get Started\",\"title_color2\":\"444444\",\"title_link2\":\"#GetStarted\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"50%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-code\",\"icon_color3\":\"FFFFFF\",\"icon_bg_color3\":\"0099FF\",\"icon_border_color3\":\"0099FF\",\"icon_link3\":\"#API\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4.8em\",\"title_text3\":\"API\",\"title_color3\":\"444444\",\"title_link3\":\"#API\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"50%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-book\",\"icon_color4\":\"FFFFFF\",\"icon_bg_color4\":\"0099FF\",\"icon_border_color4\":\"0099FF\",\"icon_link4\":\"#Resources\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"4.8em\",\"title_text4\":\"Resources\",\"title_color4\":\"444444\",\"title_link4\":\"#Resources\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"50%\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-download\",\"icon_color5\":\"FFFFFF\",\"icon_bg_color5\":\"0099FF\",\"icon_border_color5\":\"0099FF\",\"icon_link5\":\"\\/download\\/current\\/reactome.graphdb.tgz\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4.8em\",\"title_text5\":\"Download\",\"title_color5\":\"444444\",\"title_link5\":\"\\/download\\/current\\/reactome.graphdb.tgz\",\"title_link_target5\":\"blank\",\"description_text5\":\" \",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"http:\\/\\/www.favthemes.com\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"6em\",\"title_text6\":\"Title Text\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Lorem ipsum dolor sit amet, dolore magna aliqua.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(571,667,'Developers Zone - Row 2','','',1,'dev-guide-2',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"28px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"40%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-sitemap\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"CCCCCC\",\"icon_link1\":\"index.php?option=com_content&view=article&id=49\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4.8em\",\"title_text1\":\"Pathways Overview\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=49\",\"title_link_target1\":\"self\",\"description_text1\":\"Use this widget to include our pathways overview in your web application\",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"40%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-television\",\"icon_color2\":\"FF5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"CCCCCC\",\"icon_link2\":\"index.php?option=com_content&view=article&id=50\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4em\",\"title_text2\":\"Pathway Diagrams\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=50\",\"title_link_target2\":\"self\",\"description_text2\":\"Use this widget to include our pathway diagrams in your web application\",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"40%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-handshake-o\",\"icon_color3\":\"8BC34A\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"CCCCCC\",\"icon_link3\":\"index.php?option=com_content&view=article&id=26\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4em\",\"title_text3\":\"Reactome Partners\",\"title_color3\":\"444444\",\"title_link3\":\"index.php?option=com_content&view=article&id=26\",\"title_link_target3\":\"self\",\"description_text3\":\"Check out who is currently using Reactome web services and widgets\",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-lightbulb-o\",\"icon_color4\":\"FFC107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"CCCCCC\",\"icon_link4\":\"\",\"icon_link_target4\":\"blank\",\"icon_font_size4\":\"4em\",\"title_text4\":\"<<<< READ DESCRIPTION >>>>\",\"title_color4\":\"444444\",\"title_link4\":\"\",\"title_link_target4\":\"blank\",\"description_text4\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4em\",\"title_text5\":\"<<< READ DESCRIPTION >>>\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"4em\",\"title_text6\":\"<<< READ DESCRIPTION >>>\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"-sfx last favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(574,674,'Version 86 released on September 13th, 2023','Release Statistics (stats) for Landing Page','',1,'portfolio1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',0,'mod_favglyph',1,1,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"1px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"0px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"600\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"28px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"25%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-male\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"FFFFFF\",\"icon_link1\":\"\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"2.5em\",\"title_text1\":\"2,647\",\"title_color1\":\"444444\",\"title_link1\":\"\",\"title_link_target1\":\"self\",\"description_text1\":\"Human Pathways\",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"25%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"rglyph-reaction\",\"icon_color2\":\"FF5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"FFFFFF\",\"icon_link2\":\"\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"2.5em\",\"title_text2\":\"14,803\",\"title_color2\":\"444444\",\"title_link2\":\"\",\"title_link_target2\":\"self\",\"description_text2\":\"Reactions\",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"25%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"rglyph-protein\",\"icon_color3\":\"8BC34A\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"FFFFFF\",\"icon_link3\":\"\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"2.5em\",\"title_text3\":\"11,148\",\"title_color3\":\"444444\",\"title_link3\":\"\",\"title_link_target3\":\"self\",\"description_text3\":\"Proteins\",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"25%\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"rglyph-small-molecule\",\"icon_color4\":\"FFC107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"FFFFFF\",\"icon_link4\":\"\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"2.5em\",\"title_text4\":\"2,025\",\"title_color4\":\"444444\",\"title_link4\":\"\",\"title_link_target4\":\"self\",\"description_text4\":\"Small Molecules\",\"description_text_color4\":\"444444\",\"show_icon5\":\"1\",\"icon_width5\":\"25%\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa fa-medkit\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"FFFFFF\",\"icon_link5\":\"\",\"icon_link_target5\":\"self\",\"icon_font_size5\":\"2.5em\",\"title_text5\":\"1,119\",\"title_color5\":\"111111\",\"title_link5\":\"\",\"title_link_target5\":\"self\",\"description_text5\":\"Drugs\",\"description_text_color5\":\"444444\",\"show_icon6\":\"1\",\"icon_width6\":\"25%\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa fa-book\",\"icon_color6\":\"673AB7\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"DDDDDD\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"2.5em\",\"title_text6\":\"37,156\",\"title_color6\":\"111111\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"Literature References\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\" favstats\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa-calendar\",\"style\":\"0\"}',0,'*'),(579,681,'Footer (For Small Devices)','','',1,'footer1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',0,'mod_menu',1,0,'{\"menutype\":\"mainmenu\",\"base\":\"101\",\"startLevel\":\"1\",\"endLevel\":\"1\",\"showAllChildren\":\"0\",\"tag_id\":\"\",\"class_sfx\":\" footer-menu\",\"window_open\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-visible- favth-visible-xs\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"itemid\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(580,700,'User Guide Glyphs Row 1','','',1,'user-guide-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"50%\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"600\",\"title_font_style\":\"normal\",\"title_font_size\":\"16px\",\"title_margin\":\"28px 0px 14px\",\"title_text_transform\":\"uppercase\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-sitemap\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"DDDDDD\",\"icon_link1\":\"userguide\\/pathway-browser\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"3em\",\"title_text1\":\"Pathway Browser\",\"title_color1\":\"2196F3\",\"title_link1\":\"user\\/guide\\/pathway-browser\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-laptop\",\"icon_color2\":\"FF5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"DDDDDD\",\"icon_link2\":\"userguide\\/details-panel\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"3em\",\"title_text2\":\"Details Panel\",\"title_color2\":\"FF5252\",\"title_link2\":\"user\\/guide\\/details-panel\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"1\",\"icon_width3\":\"\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-search\",\"icon_color3\":\"8BC34A\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"DDDDDD\",\"icon_link3\":\"userguide\\/searching\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"3em\",\"title_text3\":\"How do I search?\",\"title_color3\":\"8BC34A\",\"title_link3\":\"user\\/guide\\/searching\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-area-chart\",\"icon_color4\":\"FFC107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"DDDDDD\",\"icon_link4\":\"userguide\\/analysis\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"3em\",\"title_text4\":\"Analysis Tools\",\"title_color4\":\"FFC107\",\"title_link4\":\"user\\/guide\\/analysis\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa fa-stethoscope\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"DDDDDD\",\"icon_link5\":\"userguide\\/diseases\",\"icon_link_target5\":\"self\",\"icon_font_size5\":\"3em\",\"title_text5\":\"Diseases\",\"title_color5\":\"673AB7\",\"title_link5\":\"user\\/guide\\/diseases\",\"title_link_target5\":\"self\",\"description_text5\":\" \",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa fa-share-alt-square\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"DDDDDD\",\"icon_link6\":\"tools\\/reactome-fiviz\",\"icon_link_target6\":\"self\",\"icon_font_size6\":\"3em\",\"title_text6\":\"ReactomeFIVIz\",\"title_color6\":\"00BCD4\",\"title_link6\":\"user\\/reactome-fiviz\",\"title_link_target6\":\"self\",\"description_text6\":\" \",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(581,712,'(*** WORK ON THE TAB CODE *** ) BANNER','Banner - Temporary Warnings. IMPORTANT: It is duplicated to address responsive design. Less text is shown on the small devices.','',1,'topbar1',0,'0000-00-00 00:00:00','1000-01-01 00:00:00','1000-01-01 00:00:00',0,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(582,713,'Why Reactome','CHINA :: Short description for Landing Page. Be careful when adding text here. It may misalign the Twitter Widget. To fix, adjust the .twitter-timeline height in custom.css','<p style=\"text-align: justify;\">Reactome is a free, open-source, curated and peer reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology and education.</p>\r\n<div class=\"favth-col-lg-12 favth-col-md-12 favth-col-sm-12 favth-col-xs-12 pull-center  favth-hidden-sm favth-hidden-xs\" style=\"background-color: ghostwhite; margin: 18px 0; padding: 10px; border-radius: 8px; border: 2px solid #58C3E5;\">\r\n<div class=\"favth-col-lg-2-4 favth-col-md-2 favth-col-sm-1 text-center\" style=\"padding: 0 4px;\"><a href=\"http://www.ebi.ac.uk/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px;\" src=\"images/sponsor/embl_ebi_logo.png\" alt=\"European Bioinformatics Institute (EMBL-EBI)\" /></a></div>\r\n<div class=\"favth-col-lg-2-4 favth-col-md-2 favth-col-sm-1 text-center\"><a href=\"http://www.ncpsb.org/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px;\" src=\"images/sponsor/ncpsb.png\" alt=\"National Center for Protein Sciences - Beijing\" /></a></div>\r\n<div class=\"favth-col-lg-2-4 favth-col-md-2 favth-col-sm-1 text-center\" style=\"padding-right: 0;\"><a href=\"https://med.nyu.edu/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px; padding-right: 0;\" src=\"images/sponsor/nyul_health_logo.png\" alt=\"NYU Langone Health\" /></a></div>\r\n<div class=\"favth-col-lg-2-4 favth-col-md-2 favth-col-sm-1 text-center\"><a href=\"http://www.ohsu.edu/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px;\" src=\"images/sponsor/ohsu_logo.png\" alt=\"Oregon Health &amp; Science University\" /></a></div>\r\n<div class=\"favth-col-lg-2-4 favth-col-md-2 favth-col-sm-1 text-center\"><a href=\"https://oicr.on.ca/\" target=\"_blank\" rel=\"noopener noreferrer\"><img style=\"height: 39px;\" src=\"images/sponsor/oicr_logo.png\" alt=\"Ontario Institute for Cancer Research\" /></a></div>\r\n</div>\r\n<p style=\"text-align: center; line-height: 13px; clear: both; padding-top: 10px;\"><span style=\"font-size: 8pt;\">The development of Reactome is supported by grants from the US National Institutes of Health (P41 HG003751 and 1U54GM114833-01), Ontario Research Fund, and the European Molecular Biology Laboratory.</span></p>',1,'prime1',0,'0000-00-00 00:00:00','2018-06-22 09:58:12','0000-00-00 00:00:00',0,'mod_custom',1,1,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-clear favth-shadow\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa fa-search\",\"style\":\"0\"}',0,'*'),(583,715,'[TEMPLATE] CGI placeholder','Div placeholder for template-page. This is hidden from the normal website. CGI pages','<!-- template-placeholder -->\r\n<p>&nbsp;</p>',1,'lead1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"0\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(584,720,'Synchronization Tool','Synchronization Tool: Joomla Content only','',1,'sync-tool',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',7,1,'{\"prepare_content\":\"0\",\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(585,738,'Elixir Banner','Includes and customises Elixir logo to fit nicely in the Reactome Page.','<style type=\"text/css\">.elixir-ribbon {\r\n    padding-bottom: 0px !important;\r\n    padding-top: 16px !important;\r\n}.elixir-ribbon .row {\r\nmargin: 0 !important; \r\n}.elixir-ribbon .elixir-logo-kite {\r\ntop: -12px !important;\r\n}</style>\r\n<script type=\"text/javascript\">jQuery(document).ready(function() {jQuery(\'#elixir-ribbon a\').each(function(){\r\n  jQuery(this).attr(\'target\', \'_blank\');\r\n });});</script>\r\n<div id=\"elixir-banner\" data-color=\"none\" data-name=\"Reactome\" data-description=\"Reactome is an Elixir Core Data Resource\" data-use-basic-styles=\"false\" data-use-cdr-logo=\"true\" data-more-information-link=\"https://elixir-europe.org/platforms/data/core-data-resources\"></div>\r\n<script src=\"https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/js/elixirBanner.js\" defer=\"defer\" type=\"text/javascript\"></script>',1,'user1',0,'0000-00-00 00:00:00','2023-01-10 08:57:48','0000-00-00 00:00:00',0,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(586,772,'EHLD / Icon Library (Specs & Guidelines)','','',1,'icon-lib-pos-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"12px\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_font_size\":\"21px\",\"title_margin\":\"28px 0px 14px\",\"title_text_transform\":\"none\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"1\",\"icon_width1\":\"30%\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-magic\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"CCCCCC\",\"icon_link1\":\"index.php?option=com_content&view=article&id=129\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"4em\",\"title_text1\":\"EHLD Specs & Guidelines\",\"title_color1\":\"444444\",\"title_link1\":\"index.php?option=com_content&view=article&id=129\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"1\",\"icon_width2\":\"30%\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-archive\",\"icon_color2\":\"FF5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"CCCCCC\",\"icon_link2\":\"index.php?option=com_content&view=article&id=130\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"4em\",\"title_text2\":\"Icon Library guidelines\",\"title_color2\":\"444444\",\"title_link2\":\"index.php?option=com_content&view=article&id=130\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"0\",\"icon_width3\":\"40%\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-database\",\"icon_color3\":\"8BC34A\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"CCCCCC\",\"icon_link3\":\"index.php?option=com_content&view=article&id=45\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"4em\",\"title_text3\":\"Graph Database\",\"title_color3\":\"444444\",\"title_link3\":\"index.php?option=com_content&view=article&id=45\",\"title_link_target3\":\"self\",\"description_text3\":\"Access to the Reactome knowledgebase content as an interconnected graph database\",\"description_text_color3\":\"444444\",\"show_icon4\":\"0\",\"icon_width4\":\"\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa-lightbulb-o\",\"icon_color4\":\"FFC107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"CCCCCC\",\"icon_link4\":\"\",\"icon_link_target4\":\"blank\",\"icon_font_size4\":\"4em\",\"title_text4\":\"<<<< READ DESCRIPTION >>>>\",\"title_color4\":\"444444\",\"title_link4\":\"\",\"title_link_target4\":\"blank\",\"description_text4\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color4\":\"444444\",\"show_icon5\":\"0\",\"icon_width5\":\"\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa-tablet\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"CCCCCC\",\"icon_link5\":\"\",\"icon_link_target5\":\"blank\",\"icon_font_size5\":\"4em\",\"title_text5\":\"<<< READ DESCRIPTION >>>\",\"title_color5\":\"444444\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"description_text5\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color5\":\"444444\",\"show_icon6\":\"0\",\"icon_width6\":\"\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa-puzzle-piece\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"CCCCCC\",\"icon_link6\":\"\",\"icon_link_target6\":\"blank\",\"icon_font_size6\":\"4em\",\"title_text6\":\"<<< READ DESCRIPTION >>>\",\"title_color6\":\"444444\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"description_text6\":\"<<< DON\'T CREATE ICON 4, 5, 6, DUPLICATE THE EXISTING INSTEAD >>>\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"-sfx favstyle\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(587,796,'Cite us','','',1,'cite-us',0,'0000-00-00 00:00:00','2020-05-04 16:36:00','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(590,809,'USERGGUIDE MODULE CATEGORY','',NULL,1,'slide',0,'0000-00-00 00:00:00','2020-04-17 15:22:45','0000-00-00 00:00:00',-2,'mod_articles_category',1,0,'{\"mode\":\"normal\",\"show_on_article_page\":1,\"count\":0,\"show_front\":\"show\",\"category_filtering_type\":1,\"catid\":[28],\"show_child_category_articles\":0,\"levels\":1,\"author_filtering_type\":1,\"author_alias_filtering_type\":1,\"excluded_articles\":\"\",\"date_filtering\":\"off\",\"date_field\":\"a.created\",\"start_date_range\":\"\",\"end_date_range\":\"\",\"relative_date\":30,\"article_ordering\":\"a.title\",\"article_ordering_direction\":\"ASC\",\"article_grouping\":\"none\",\"date_grouping_field\":\"created\",\"month_year_format\":\"F Y\",\"article_grouping_direction\":\"ksort\",\"link_titles\":1,\"show_date\":0,\"show_date_field\":\"created\",\"show_date_format\":\"Y-m-d H:i:s\",\"show_category\":0,\"show_hits\":0,\"show_author\":0,\"show_tags\":0,\"show_introtext\":0,\"introtext_limit\":100,\"show_readmore\":0,\"show_readmore_title\":1,\"readmore_limit\":15,\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"owncache\":1,\"cache_time\":900,\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(591,811,'User Guide TOC','','<div id=\"tocbot\" class=\"js-toc desktop-toc\"></div>',1,'sidebar1-toc',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',2,1,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(592,825,'Use Reactome Graph Database in your Project','Download Reactome Graph Database','<div id=\"fav-download\">\r\n<p id=\"fav-download-message\"><span style=\"color: #ffffff;\">USE REACTOME GRAPH DATABASE IN YOUR PROJECT</span></p>\r\n<p id=\"fav-download-btn\"><a href=\"dev/graph-database\" class=\"favth-btn\">LEARN MORE</a></p>\r\n</div>',1,'promo1',0,'0000-00-00 00:00:00','2021-02-11 15:09:43','0000-00-00 00:00:00',0,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(593,830,'Banner','','<div id=\"fav-download\">\r\n<p id=\"fav-download-message\"><span style=\"color: #ffffff;\">We want to hear about your Success Story using Reactome</span></p>\r\n<p id=\"fav-download-btn\"><a href=\"index.php?option=com_content&amp;view=article&amp;id=172:we-want-to-hear-your-success-story&amp;catid=26:news&amp;lang=en-GB&amp;Itemid=236\" class=\"favth-btn\">LEARN MORE</a></p>\r\n</div>',1,'promo1',0,'0000-00-00 00:00:00','2021-04-01 04:00:00','0000-00-00 00:00:00',0,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(594,831,'New training material','','',1,'new-training-1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favpromote',1,1,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_padding\":\"10px 20px\",\"title_font_size\":\"18px\",\"title_line_height\":\"1.4em\",\"title_text_align\":\"left\",\"title_icon_font_size\":\"18px\",\"title_icon_vertical_align\":\"baseline\",\"description_text_font_size\":\"14px\",\"description_text_line_height\":\"1.5em\",\"description_text_align\":\"justify\",\"show_column1\":\"1\",\"column_border_color1\":\"DDDDDD\",\"column_border_radius1\":\"4px\",\"upload_image1\":\"images\\/training\\/training_1.png\",\"image_link1\":\"https:\\/\\/www.ebi.ac.uk\\/training\\/online\\/course\\/reactome-quick-tour\",\"image_target1\":\"blank\",\"image_alt1\":\"\",\"title_text1\":\"Quick tour\",\"title_color1\":\"FFFFFF\",\"title_bg_color1\":\"58C3E5\",\"title_link1\":\"https:\\/\\/www.ebi.ac.uk\\/training\\/online\\/course\\/reactome-quick-tour\",\"title_link_target1\":\"blank\",\"title_icon1\":\"fa fa-window-maximize\",\"description_text1\":\"This quick tour provides a brief introduction to Reactome knowledgebase and its various features.\",\"description_text_color1\":\"444444\",\"show_column2\":\"1\",\"column_border_color2\":\"DDDDDD\",\"column_border_radius2\":\"4px\",\"upload_image2\":\"images\\/training\\/training_2.png\",\"image_link2\":\"https:\\/\\/www.ebi.ac.uk\\/training\\/online\\/course\\/reactome-exploring-biological-pathways\",\"image_target2\":\"blank\",\"image_alt2\":\"\",\"title_text2\":\"Exploring Pathways\",\"title_color2\":\"FFFFFF\",\"title_bg_color2\":\"58C3E5\",\"title_link2\":\"https:\\/\\/www.ebi.ac.uk\\/training\\/online\\/course\\/reactome-exploring-biological-pathways\",\"title_link_target2\":\"blank\",\"title_icon2\":\"fa fa-sitemap\",\"description_text2\":\"This course provides a more detailed view of the Reactome pathway database web interface and the database content.\",\"description_text_color2\":\"444444\",\"show_column3\":\"1\",\"column_border_color3\":\"DDDDDD\",\"column_border_radius3\":\"4px\",\"upload_image3\":\"images\\/training\\/training_3.png\",\"image_link3\":\"https:\\/\\/www.ebi.ac.uk\\/training\\/online\\/course\\/reactome-tools-analysis-biological-pathways\",\"image_target3\":\"blank\",\"image_alt3\":\"\",\"title_text3\":\"Analysis Tools\",\"title_color3\":\"FFFFFF\",\"title_bg_color3\":\"58C3E5\",\"title_link3\":\"https:\\/\\/www.ebi.ac.uk\\/training\\/online\\/course\\/reactome-tools-analysis-biological-pathways\",\"title_link_target3\":\"blank\",\"title_icon3\":\"fa fa-flask\",\"description_text3\":\"This course introduces the analysis tools available via the Reactome pathway database web interface.\",\"description_text_color3\":\"444444\",\"show_column4\":\"0\",\"column_border_color4\":\"DDDDDD\",\"column_border_radius4\":\"4px\",\"upload_image4\":\"\",\"image_link4\":\"\",\"image_target4\":\"blank\",\"image_alt4\":\"\",\"title_text4\":\"Title Text\",\"title_color4\":\"FFFFFF\",\"title_bg_color4\":\"D63E52\",\"title_link4\":\"\",\"title_link_target4\":\"blank\",\"title_icon4\":\"fa-magic\",\"description_text4\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color4\":\"444444\",\"show_column5\":\"0\",\"column_border_color5\":\"DDDDDD\",\"column_border_radius5\":\"4px\",\"upload_image5\":\"\",\"image_link5\":\"\",\"image_target5\":\"blank\",\"image_alt5\":\"\",\"title_text5\":\"Title Text\",\"title_color5\":\"FFFFFF\",\"title_bg_color5\":\"D63E52\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"title_icon5\":\"fa-quote-right\",\"description_text5\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color5\":\"444444\",\"show_column6\":\"0\",\"column_border_color6\":\"DDDDDD\",\"column_border_radius6\":\"4px\",\"upload_image6\":\"\",\"image_link6\":\"\",\"image_target6\":\"blank\",\"image_alt6\":\"\",\"title_text6\":\"Title Text\",\"title_color6\":\"FFFFFF\",\"title_bg_color6\":\"D63E52\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"title_icon6\":\"fa-recycle\",\"description_text6\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"System-none\"}',0,'*'),(595,832,'Please Take The EBI Impact Survey','Please Take The EBI Impact Survey','<div id=\"fav-download\" style=\"display: flex;\">\r\n<p id=\"fav-download-message\" style=\"font-size: 14px !important;\"><span style=\"color: #ffffff;\">We need your help! EMBL-EBI, one of the four Reactome consortium partners, is running a large scale impact survey. If you’ve ever found our data helpful, please take our impact survey (15 min). Your replies will help keep the data flowing to the scientific community. </span></p>\r\n<p id=\"fav-download-btn\"><a href=\"https://www.surveymonkey.co.uk/r/EMBL-EBI_Impact_DR\" target=\"_blank\" rel=\"noopener\" class=\"favth-btn\">Survey</a></p>\r\n</div>',1,'promo1',0,'0000-00-00 00:00:00','2021-03-01 05:00:00','2021-04-01 04:00:00',1,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(596,863,'Reactome Research Spotlight','','<style type=\"text/css\">\r\n\r\n@media only screen and (min-device-width : 360px) and (max-device-width : 640px){\r\n  #fav-download{\r\n    display: flex;\r\n    flex-direction: column;\r\n    }\r\n}\r\n</style>\r\n<div id=\"fav-download\" style=\"display: flex;\">\r\n<p id=\"fav-download-message\" style=\"font-size: 14px !important;\"><span style=\"color: #ffffff;\"> [October 2, 2024] In the feature article of the October 2024 issue of <a href=\"https://www.sciencedirect.com/journal/drug-discovery-today\" onmouseover=\"this.style.color=\'#ffffff\'\" onmouseout=\"this.style.color=\'#2f9ec2\'\"> Drug Discovery Today </a> titled <a href=\"https://pubmed.ncbi.nlm.nih.gov/39179147/\" onmouseover=\"this.style.color=\'#ffffff\'\" onmouseout=\"this.style.color=\'#2f9ec2\'\"> Chemical coverage of human biological pathways </a> Kwak et al. describe the <a href=\"https://www.target2035.net/\" onmouseover=\"this.style.color=\'#ffffff\'\" onmouseout=\"this.style.color=\'#2f9ec2\'\"> Target 2035 </a> initiative, whose mission is to discover chemical tools for all human proteins by 2035. The authors use Reactome as the reference standard to determine the chemical coverage of human biological pathways and to outline the advantages of adopting the pathway-based rather than the proteome-based approach in guiding Target 2035 efforts. </span></p>\r\n<p id=\"fav-download-btn\"><a href=\"index.php?option=com_content&amp;view=category&amp;id=36\" class=\"favth-btn\" style=\"color: #ffffff;\">ARCHIVE</a></p>\r\n</div>',1,'promo1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,1,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\" favth-clear \",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h2\",\"header_class\":\"fa fa-eye\",\"style\":\"0\"}',0,'*'),(597,875,'Elixir and GCBR Banner','Includes and customises Elixir and GCBR logo to fit nicely in the Reactome Page.','<style type=\"text/css\">\r\n.elixir-ribbon {\r\n    padding-bottom: 0px !important;\r\n    padding-top: 16px !important;\r\n}.elixir-ribbon .row {\r\nmargin: 0 !important; \r\n}.elixir-ribbon .elixir-logo-kite {\r\ntop: -7px !important;\r\n}\r\n\r\n\r\n.row {\r\n  display: flex;\r\n  flex-direction: row;\r\n  flex-wrap: wrap;\r\n  width: 100%;\r\n}\r\n\r\n.column {\r\n  display: flex;\r\n  flex-direction: column;\r\n  flex-basis: 100%;\r\n  flex: 1;\r\n}\r\n\r\n#GBCR-container {\r\n  display: flex;\r\n  flex-direction: row;\r\n}\r\n\r\n#GBCR-logo {\r\n height: 60px;\r\n position: relative;\r\n top: -7px;\r\n margin-right: 10px;\r\n}\r\n\r\n@media only screen and (min-device-width : 320px) and (max-device-width : 480px) {\r\n .row {\r\n  display: flex;\r\n  flex-direction: column;\r\n  flex-wrap: wrap;\r\n  width: 100%;\r\n  }\r\n\r\n@media only screen and (min-device-width : 360px) and (max-device-width : 640px){\r\n #GBCR-logo {\r\n   margin-top: 7px;\r\n  }\r\n\r\n\r\n</style>\r\n<script type=\"text/javascript\">\r\n\r\njQuery(document).ready(function() {\r\n\r\njQuery(\'#elixir-ribbon a\').each(function(){\r\n  jQuery(this).attr(\'target\', \'_blank\');\r\n });\r\n\r\n\r\njQuery(\'#elixir-ribbon .column\').after(`\r\n<div class=\"column\">\r\n   <a href=\"https://globalbiodata.org/scientific-activities/global-core-biodata-resources/\" target=\"_blank\" id=\"GBCR-container\">\r\n   <img src=\"/images/others//GCBR-Logo-RGB_WcircleBG.png\" id=\"GBCR-logo\">\r\n   <div>\r\n      <h5>\r\n         <span class=\"elixir-banner-name\">Reactome</span> is a Global Core Biodata Resource\r\n      </h5>\r\n      <div id=\"elixir-banner-info\">\r\n         <small>\r\n         <span class=\"readmore\">Learn more ›</span>\r\n         </small>\r\n      </div>\r\n   </div>\r\n   </a>\r\n</div>\r\n`);\r\n  \r\n});\r\n\r\n</script>\r\n<div id=\"elixir-banner\" data-color=\"none\" data-name=\"Reactome\" data-description=\"Reactome is an Elixir Core Data Resource\" data-use-basic-styles=\"false\" data-use-cdr-logo=\"true\" data-more-information-link=\"https://elixir-europe.org/platforms/data/core-data-resources\"></div>\r\n<script src=\"https://ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.3/js/elixirBanner.js\" defer=\"defer\" type=\"text/javascript\"></script>',1,'user1',0,'0000-00-00 00:00:00','2023-01-11 15:47:43','0000-00-00 00:00:00',1,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(598,892,'FAQ Sidebar Menu','',NULL,1,'sidebar1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_articles_categories',1,0,'{\"parent\":35,\"show_description\":0,\"numitems\":0,\"show_children\":1,\"count\":0,\"maxlevel\":0,\"layout\":\"_:default\",\"item_heading\":\"5\",\"moduleclass_sfx\":\" favth-side-faq\",\"owncache\":1,\"cache_time\":900,\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(599,937,'Version 90 released on October 2, 2024','Release Statistics (stats) custom module for Landing Page','<div class=\"moduletable favstats\">\r\n<h3></h3>\r\n<p>{source}<span style=\"font-family: courier new, courier, monospace;\">&lt;?php</span></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">    $host =</span><code class=\"hljs language-php\"><span class=\"hljs-variable\">$_SERVER</span>[<span class=\"hljs-string\">\'SERVER_NAME\'</span>]</code><span style=\"font-family: courier new, courier, monospace;\">;</span></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">    IF ($host=== \"localhost\"|| $host=== \"127.0.1\") {</span></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\"><span style=\"font-family: courier new, courier, monospace;\">     $host = \"dev.reactome.org\";</span></span><span style=\"font-family: courier new, courier, monospace;\"><br /><img src=\"/Website/media/sourcerer/images/tab.png\" alt=\"\" />}</span></p>\r\n<p><span class=\"hljs-variable\">      $context</span> = <span class=\"hljs-title function_ invoke__\">stream_context_create</span>([ <span class=\"hljs-string\">\'http\'</span> =&gt; [ <span class=\"hljs-string\">\'header\'</span> =&gt; <span class=\"hljs-string\">\'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3\'</span>, ], ]);</p>\r\n<p>       $versionUrl = \"https://\" .$host . \"/ContentService/data/database/version\";</p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">   $version = file_get_contents(<span style=\"font-family: courier new, courier, monospace;\">$versionUrl</span>); </span></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">   if(<span style=\"font-family: courier new, courier, monospace;\">$version ===false</span>){</span></p>\r\n<p>      echo \"Error fetching Reactome version number\";</p>\r\n<p>      exit;</p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">}</span></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\"><br />$url = \"https://download.reactome.org/\".<span style=\"font-family: courier new, courier, monospace;\">$version.\"</span>/stats/summary_stats.json\";<br /><br />//read JSON file from URL in PHP<br />$readJSONFile = file_get_contents($url);</span></p>\r\n<p><span class=\"hljs-keyword\">if</span> (<span class=\"hljs-variable\">$readJSONFile</span> === <span class=\"hljs-literal\">false</span>) {</p>\r\n<p><span class=\"hljs-keyword\">echo</span> <span class=\"hljs-string\">\"Error fetching summary stats JSON file\"</span>;</p>\r\n<p><span class=\"hljs-keyword\">exit</span>; }</p>\r\n<p></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">//convert JSON to array in PHP<br />$summary = json_decode($readJSONFile, TRUE);</span></p>\r\n<p><span style=\"font-family: courier new, courier, monospace;\">?&gt;</span>{/source}</p>\r\n<div id=\"favglyph-17271\" class=\"row-fluid\">\r\n<div id=\"favglyph-box1\" class=\"favth-col-lg-2 favth-col-md-4 favth-col-sm-6 favth-col-xs-6 favglyph1 layout-effect\">\r\n<div class=\"favglyph-icon-top favth-clearfix\">\r\n<div id=\"favglyph-icon\" class=\"favglyph-icon\" style=\"background-color: #ffffff; width: 25%; border: 0px solid #FFFFFF; -webkit-border-radius: 1px; -moz-border-radius: 1px; border-radius: 1px;\"><i class=\"fa fa-male\" style=\"color: #2196f3; font-size: 2.5em;\"> </i></div>\r\n</div>\r\n<div class=\"favglyph-description-top favth-clearfix\">\r\n<h2 id=\"favglyph-description-title\" style=\"font-family: Open Sans, sans-serif; font-weight: 600; font-style: normal; font-size: 21px; margin: 28px 0px 14px; text-transform: none;\">{source}<?php\r\necho number_format($summary[10][\'value\'])\r\n?>{/source}</h2>\r\n<p id=\"favglyph-description-text\" style=\"font-size: 14px; color: #444444;\">Human Pathways</p>\r\n</div>\r\n</div>\r\n<div id=\"favglyph-box2\" class=\"favth-col-lg-2 favth-col-md-4 favth-col-sm-6 favth-col-xs-6 favglyph2 layout-effect\">\r\n<div class=\"favglyph-icon-top favth-clearfix\">\r\n<div id=\"favglyph-icon\" class=\"favglyph-icon\" style=\"background-color: #ffffff; width: 25%; border: 0px solid #FFFFFF; -webkit-border-radius: 1px; -moz-border-radius: 1px; border-radius: 1px;\"><i class=\"rglyph-reaction\" style=\"color: #ff5252; font-size: 2.5em;\"> </i></div>\r\n</div>\r\n<div class=\"favglyph-description-top favth-clearfix\">\r\n<h2 id=\"favglyph-description-title\" style=\"font-family: Open Sans, sans-serif; font-weight: 600; font-style: normal; font-size: 21px; margin: 28px 0px 14px; text-transform: none;\">{source}<?php\r\necho number_format($summary[2][\'value\'])\r\n?>{/source}</h2>\r\n<p id=\"favglyph-description-text\" style=\"font-size: 14px; color: #444444;\">Reactions</p>\r\n</div>\r\n</div>\r\n<div id=\"favglyph-box3\" class=\"favth-col-lg-2 favth-col-md-4 favth-col-sm-6 favth-col-xs-6 favglyph3 layout-effect\">\r\n<div class=\"favglyph-icon-top favth-clearfix\">\r\n<div id=\"favglyph-icon\" class=\"favglyph-icon\" style=\"background-color: #ffffff; width: 25%; border: 0px solid #FFFFFF; -webkit-border-radius: 1px; -moz-border-radius: 1px; border-radius: 1px;\"><i class=\"rglyph-protein\" style=\"color: #8bc34a; font-size: 2.5em;\"> </i></div>\r\n</div>\r\n<div class=\"favglyph-description-top favth-clearfix\">\r\n<h2 id=\"favglyph-description-title\" style=\"font-family: Open Sans, sans-serif; font-weight: 600; font-style: normal; font-size: 21px; margin: 28px 0px 14px; text-transform: none;\">{source}<?php\r\necho number_format($summary[0][\'value\'])\r\n?>{/source}</h2>\r\n<p id=\"favglyph-description-text\" style=\"font-size: 14px; color: #444444;\">Proteins</p>\r\n</div>\r\n</div>\r\n<div id=\"favglyph-box4\" class=\"favth-col-lg-2 favth-col-md-4 favth-col-sm-6 favth-col-xs-6 favglyph4 layout-effect\">\r\n<div class=\"favglyph-icon-top favth-clearfix\">\r\n<div id=\"favglyph-icon\" class=\"favglyph-icon\" style=\"background-color: #ffffff; width: 25%; border: 0px solid #FFFFFF; -webkit-border-radius: 1px; -moz-border-radius: 1px; border-radius: 1px;\"><i class=\"rglyph-small-molecule\" style=\"color: #ffc107; font-size: 2.5em;\"> </i></div>\r\n</div>\r\n<div class=\"favglyph-description-top favth-clearfix\">\r\n<h2 id=\"favglyph-description-title\" style=\"font-family: Open Sans, sans-serif; font-weight: 600; font-style: normal; font-size: 21px; margin: 28px 0px 14px; text-transform: none;\">{source}<?php\r\necho number_format($summary[4][\'value\'])\r\n?>{/source}</h2>\r\n<p id=\"favglyph-description-text\" style=\"font-size: 14px; color: #444444;\">Small Molecules</p>\r\n</div>\r\n</div>\r\n<div id=\"favglyph-box5\" class=\"favth-col-lg-2 favth-col-md-4 favth-col-sm-6 favth-col-xs-6 favglyph5 layout-effect\">\r\n<div class=\"favglyph-icon-top favth-clearfix\">\r\n<div id=\"favglyph-icon\" class=\"favglyph-icon\" style=\"background-color: #ffffff; width: 25%; border: 0px solid #FFFFFF; -webkit-border-radius: 1px; -moz-border-radius: 1px; border-radius: 1px;\"><i class=\"fa fa-medkit\" style=\"color: #673ab7; font-size: 2.5em;\"> </i></div>\r\n</div>\r\n<div class=\"favglyph-description-top favth-clearfix\">\r\n<h2 id=\"favglyph-description-title\" style=\"font-family: Open Sans, sans-serif; font-weight: 600; font-style: normal; font-size: 21px; margin: 28px 0px 14px; text-transform: none;\">{source}<?php\r\necho number_format($summary[8][\'value\'] + $summary[9][\'value\'])\r\n?>{/source}</h2>\r\n<p id=\"favglyph-description-text\" style=\"font-size: 14px; color: #444444;\">Drugs</p>\r\n</div>\r\n</div>\r\n<div id=\"favglyph-box6\" class=\"favth-col-lg-2 favth-col-md-4 favth-col-sm-6 favth-col-xs-6 favglyph6 layout-effect\">\r\n<div class=\"favglyph-icon-top favth-clearfix\">\r\n<div id=\"favglyph-icon\" class=\"favglyph-icon\" style=\"background-color: #ffffff; width: 25%; border: 0px solid #DDDDDD; -webkit-border-radius: 1px; -moz-border-radius: 1px; border-radius: 1px;\"><i class=\"fa fa-book\" style=\"color: #673ab7; font-size: 2.5em;\"> </i></div>\r\n</div>\r\n<div class=\"favglyph-description-top favth-clearfix\">\r\n<h2 id=\"favglyph-description-title\" style=\"font-family: Open Sans, sans-serif; font-weight: 600; font-style: normal; font-size: 21px; margin: 28px 0px 14px; text-transform: none;\">{source}<?php\r\necho number_format($summary[3][\'value\'])\r\n?>{/source}</h2>\r\n<p id=\"favglyph-description-text\" style=\"font-size: 14px; color: #444444;\">Literature References</p>\r\n<p></p>\r\n<p></p>\r\n</div>\r\n</div>\r\n</div>\r\n</div>',1,'portfolio1',172,'2024-10-03 14:46:53','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_custom',1,1,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"favstats\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"fa-calendar\",\"style\":\"0\"}',0,'*'),(600,948,'User Guide Glyphs Row 2','','',1,'user-guide-1',0,'0000-00-00 00:00:00','2024-03-19 15:19:35','0000-00-00 00:00:00',1,'mod_favglyph',1,0,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"icon_border_radius\":\"50%\",\"icon_border_type\":\"solid\",\"icon_border_width\":\"1px\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"600\",\"title_font_style\":\"normal\",\"title_font_size\":\"16px\",\"title_margin\":\"28px 0px 14px\",\"title_text_transform\":\"uppercase\",\"description_text_font_size\":\"14px\",\"show_icon1\":\"0\",\"icon_width1\":\"\",\"description_width1\":\"\",\"icon_layout1\":\"top\",\"icon_name1\":\"fa fa-sitemap\",\"icon_color1\":\"2196F3\",\"icon_bg_color1\":\"FFFFFF\",\"icon_border_color1\":\"DDDDDD\",\"icon_link1\":\"userguide\\/pathway-browser\",\"icon_link_target1\":\"self\",\"icon_font_size1\":\"3em\",\"title_text1\":\"Pathway Browser\",\"title_color1\":\"2196F3\",\"title_link1\":\"user\\/guide\\/pathway-browser\",\"title_link_target1\":\"self\",\"description_text1\":\" \",\"description_text_color1\":\"444444\",\"show_icon2\":\"0\",\"icon_width2\":\"\",\"description_width2\":\"\",\"icon_layout2\":\"top\",\"icon_name2\":\"fa fa-laptop\",\"icon_color2\":\"FF5252\",\"icon_bg_color2\":\"FFFFFF\",\"icon_border_color2\":\"DDDDDD\",\"icon_link2\":\"userguide\\/details-panel\",\"icon_link_target2\":\"self\",\"icon_font_size2\":\"3em\",\"title_text2\":\"Details Panel\",\"title_color2\":\"FF5252\",\"title_link2\":\"user\\/guide\\/details-panel\",\"title_link_target2\":\"self\",\"description_text2\":\" \",\"description_text_color2\":\"444444\",\"show_icon3\":\"0\",\"icon_width3\":\"\",\"description_width3\":\"\",\"icon_layout3\":\"top\",\"icon_name3\":\"fa fa-search\",\"icon_color3\":\"8BC34A\",\"icon_bg_color3\":\"FFFFFF\",\"icon_border_color3\":\"DDDDDD\",\"icon_link3\":\"userguide\\/searching\",\"icon_link_target3\":\"self\",\"icon_font_size3\":\"3em\",\"title_text3\":\"How do I search?\",\"title_color3\":\"8BC34A\",\"title_link3\":\"user\\/guide\\/searching\",\"title_link_target3\":\"self\",\"description_text3\":\" \",\"description_text_color3\":\"444444\",\"show_icon4\":\"1\",\"icon_width4\":\"111px\",\"description_width4\":\"\",\"icon_layout4\":\"top\",\"icon_name4\":\"fa fa-circle-o-notch\",\"icon_color4\":\"FFC107\",\"icon_bg_color4\":\"FFFFFF\",\"icon_border_color4\":\"DDDDDD\",\"icon_link4\":\"user\\/guide\\/cytomics\",\"icon_link_target4\":\"self\",\"icon_font_size4\":\"3em\",\"title_text4\":\"Cytomics\",\"title_color4\":\"FFC107\",\"title_link4\":\"user\\/guide\\/cytomics\",\"title_link_target4\":\"self\",\"description_text4\":\" \",\"description_text_color4\":\"444444\",\"show_icon5\":\"1\",\"icon_width5\":\"111px\",\"description_width5\":\"\",\"icon_layout5\":\"top\",\"icon_name5\":\"fa fa-stethoscope\",\"icon_color5\":\"673AB7\",\"icon_bg_color5\":\"FFFFFF\",\"icon_border_color5\":\"DDDDDD\",\"icon_link5\":\"userguide\\/diseases\",\"icon_link_target5\":\"self\",\"icon_font_size5\":\"3em\",\"title_text5\":\"Diseases\",\"title_color5\":\"673AB7\",\"title_link5\":\"user\\/guide\\/diseases\",\"title_link_target5\":\"self\",\"description_text5\":\" \",\"description_text_color5\":\"444444\",\"show_icon6\":\"1\",\"icon_width6\":\"111px\",\"description_width6\":\"\",\"icon_layout6\":\"top\",\"icon_name6\":\"fa fa-share-alt-square\",\"icon_color6\":\"00BCD4\",\"icon_bg_color6\":\"FFFFFF\",\"icon_border_color6\":\"DDDDDD\",\"icon_link6\":\"tools\\/reactome-fiviz\",\"icon_link_target6\":\"self\",\"icon_font_size6\":\"3em\",\"title_text6\":\"ReactomeFIVIz\",\"title_color6\":\"00BCD4\",\"title_link6\":\"user\\/reactome-fiviz\",\"title_link_target6\":\"self\",\"description_text6\":\" \",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(601,956,'EMBL-EBI User Survey 2024','Please Take The EBI Impact Survey','<div id=\"fav-download\" style=\"display: flex;\">\r\n<p id=\"fav-download-message\" style=\"font-size: 14px !important;\"><span style=\"color: #ffffff;\">Do data resources managed by EMBL-EBI and collaborators make a difference to your work? Please take 10 minutes to fill in the annual user survey, and help make the case for why sustaining open data resources is critical for life sciences research.</span></p>\r\n<p id=\"fav-download-btn\"><a href=\"https://www.surveymonkey.com/r/HJKYKTT?channel=[webpage]\" target=\"_blank\" rel=\"noopener\" class=\"favth-btn\">Survey</a></p>\r\n</div>',1,'intro1',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',0,'mod_custom',1,0,'{\"prepare_content\":0,\"backgroundimage\":\"\",\"layout\":\"_:default\",\"moduleclass_sfx\":\"\",\"cache\":1,\"cache_time\":900,\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"0\"}',0,'*'),(602,964,'Community','',NULL,1,'community',0,'0000-00-00 00:00:00','0000-00-00 00:00:00','0000-00-00 00:00:00',1,'mod_favpromote',1,1,'{\"jquery_load\":\"1\",\"layout_effect\":\"layout-effect-none\",\"title_google_font\":\"Open Sans\",\"title_font_weight\":\"400\",\"title_font_style\":\"normal\",\"title_padding\":\"10px 20px\",\"title_font_size\":\"18px\",\"title_line_height\":\"1.4em\",\"title_text_align\":\"left\",\"title_icon_font_size\":\"18px\",\"title_icon_vertical_align\":\"baseline\",\"description_text_font_size\":\"14px\",\"description_text_line_height\":\"1.5em\",\"description_text_align\":\"justify\",\"show_column1\":\"1\",\"column_border_color1\":\"DDDDDD\",\"column_border_radius1\":\"4px\",\"upload_image1\":\"images\\/docs\\/for_users_110x110.png\",\"image_link1\":\"community\\/collaboration\",\"image_target1\":\"self\",\"image_alt1\":\"\",\"title_text1\":\"Contribute Pathway Knowledge\",\"title_color1\":\"FFFFFF\",\"title_bg_color1\":\"58C3E5\",\"title_link1\":\"community\\/collaboration\",\"title_link_target1\":\"self\",\"title_icon1\":\"fa fa-users\",\"description_text1\":\"Contribute your pathway knowledge!\",\"description_text_color1\":\"444444\",\"show_column2\":\"1\",\"column_border_color2\":\"DDDDDD\",\"column_border_radius2\":\"4px\",\"upload_image2\":\"images\\/training\\/training_3.png\",\"image_link2\":\"community\\/events\",\"image_target2\":\"self\",\"image_alt2\":\"\",\"title_text2\":\"Attend Community Events\",\"title_color2\":\"FFFFFF\",\"title_bg_color2\":\"58C3E5\",\"title_link2\":\"community\\/events\",\"title_link_target2\":\"self\",\"title_icon2\":\"fa fa-code\",\"description_text2\":\"Attend our community events!\",\"description_text_color2\":\"444444\",\"show_column3\":\"1\",\"column_border_color3\":\"DDDDDD\",\"column_border_radius3\":\"4px\",\"upload_image3\":\"images\\/docs\\/citing_us_110x110.png\",\"image_link3\":\"community\\/publications\",\"image_target3\":\"self\",\"image_alt3\":\"\",\"title_text3\":\"Learn more about Reactome\",\"title_color3\":\"FFFFFF\",\"title_bg_color3\":\"58C3E5\",\"title_link3\":\"community\\/publications\",\"title_link_target3\":\"self\",\"title_icon3\":\"fa fa-pencil\",\"description_text3\":\"Keep up with Reactome\'s recent publications!\",\"description_text_color3\":\"444444\",\"show_column4\":\"0\",\"column_border_color4\":\"DDDDDD\",\"column_border_radius4\":\"4px\",\"upload_image4\":\"\",\"image_link4\":\"\",\"image_target4\":\"blank\",\"image_alt4\":\"\",\"title_text4\":\"Title Text\",\"title_color4\":\"FFFFFF\",\"title_bg_color4\":\"D63E52\",\"title_link4\":\"\",\"title_link_target4\":\"blank\",\"title_icon4\":\"fa-magic\",\"description_text4\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color4\":\"444444\",\"show_column5\":\"0\",\"column_border_color5\":\"DDDDDD\",\"column_border_radius5\":\"4px\",\"upload_image5\":\"\",\"image_link5\":\"\",\"image_target5\":\"blank\",\"image_alt5\":\"\",\"title_text5\":\"Title Text\",\"title_color5\":\"FFFFFF\",\"title_bg_color5\":\"D63E52\",\"title_link5\":\"\",\"title_link_target5\":\"blank\",\"title_icon5\":\"fa-quote-right\",\"description_text5\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color5\":\"444444\",\"show_column6\":\"0\",\"column_border_color6\":\"DDDDDD\",\"column_border_radius6\":\"4px\",\"upload_image6\":\"\",\"image_link6\":\"\",\"image_target6\":\"blank\",\"image_alt6\":\"\",\"title_text6\":\"Title Text\",\"title_color6\":\"FFFFFF\",\"title_bg_color6\":\"D63E52\",\"title_link6\":\"\",\"title_link_target6\":\"blank\",\"title_icon6\":\"fa-recycle\",\"description_text6\":\"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor.\",\"description_text_color6\":\"444444\",\"moduleclass_sfx\":\"\",\"cache\":\"1\",\"cache_time\":\"900\",\"cachemode\":\"static\",\"module_tag\":\"div\",\"bootstrap_size\":\"0\",\"header_tag\":\"h3\",\"header_class\":\"\",\"style\":\"System-none\"}',0,'*');
/*!40000 ALTER TABLE `rlp_modules` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_modules_menu`
--

DROP TABLE IF EXISTS `rlp_modules_menu`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_modules_menu` (
  `moduleid` int NOT NULL DEFAULT '0',
  `menuid` int NOT NULL DEFAULT '0',
  PRIMARY KEY (`moduleid`,`menuid`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_modules_menu`
--

LOCK TABLES `rlp_modules_menu` WRITE;
/*!40000 ALTER TABLE `rlp_modules_menu` DISABLE KEYS */;
INSERT INTO `rlp_modules_menu` VALUES (0,0),(1,0),(2,0),(3,0),(4,0),(6,0),(7,0),(8,0),(9,0),(10,0),(12,0),(13,0),(14,0),(15,0),(17,235),(17,236),(17,237),(17,251),(17,252),(17,253),(17,254),(17,259),(17,260),(17,261),(17,262),(17,263),(17,264),(17,265),(17,269),(17,270),(17,271),(17,272),(17,290),(17,292),(17,293),(17,294),(17,295),(17,296),(17,297),(17,299),(17,301),(17,302),(17,303),(17,304),(17,305),(17,306),(17,307),(17,360),(17,361),(17,362),(17,377),(17,378),(17,379),(17,380),(17,381),(17,382),(17,384),(17,407),(17,424),(17,471),(17,472),(79,0),(86,0),(88,0),(89,0),(475,101),(479,-104),(479,-102),(484,101),(496,101),(499,101),(500,0),(505,101),(508,101),(518,-288),(518,-266),(518,-246),(518,-101),(519,-288),(519,-284),(519,-266),(519,-101),(523,267),(524,-721),(524,-539),(524,-422),(524,-406),(524,-398),(524,-397),(524,-371),(524,-361),(524,-288),(524,-284),(524,-277),(524,-266),(524,-263),(524,-246),(524,-242),(524,-241),(524,-152),(524,-104),(524,-102),(524,-101),(525,266),(526,288),(526,371),(527,288),(533,0),(540,268),(544,152),(551,263),(555,101),(556,101),(559,295),(560,294),(561,295),(563,295),(564,294),(565,294),(566,305),(567,307),(568,293),(569,292),(570,290),(571,268),(573,266),(574,101),(579,0),(580,303),(581,0),(582,101),(583,383),(584,406),(585,101),(586,302),(587,101),(587,152),(587,235),(587,236),(587,237),(587,242),(587,246),(587,251),(587,252),(587,253),(587,254),(587,259),(587,260),(587,261),(587,262),(587,263),(587,264),(587,265),(587,266),(587,284),(587,288),(587,290),(587,299),(587,301),(587,303),(587,360),(587,361),(587,362),(587,371),(587,378),(587,379),(587,380),(587,381),(587,382),(587,383),(587,397),(587,479),(587,488),(587,493),(587,504),(587,530),(587,560),(587,561),(588,152),(589,303),(590,303),(591,539),(592,101),(593,101),(594,261),(594,303),(595,101),(596,101),(597,101),(598,721),(599,101),(600,303),(601,101),(602,241);
/*!40000 ALTER TABLE `rlp_modules_menu` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_newsfeeds`
--

DROP TABLE IF EXISTS `rlp_newsfeeds`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_newsfeeds` (
  `catid` int NOT NULL DEFAULT '0',
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `name` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `link` varchar(2048) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `published` tinyint(1) NOT NULL DEFAULT '0',
  `numarticles` int unsigned NOT NULL DEFAULT '1',
  `cache_time` int unsigned NOT NULL DEFAULT '3600',
  `checked_out` int unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `ordering` int NOT NULL DEFAULT '0',
  `rtl` tinyint NOT NULL DEFAULT '0',
  `access` int unsigned NOT NULL DEFAULT '0',
  `language` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `params` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by` int unsigned NOT NULL DEFAULT '0',
  `created_by_alias` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `modified` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_by` int unsigned NOT NULL DEFAULT '0',
  `metakey` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadesc` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadata` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `xreference` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `description` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `version` int unsigned NOT NULL DEFAULT '1',
  `hits` int unsigned NOT NULL DEFAULT '0',
  `images` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`id`),
  KEY `idx_access` (`access`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_state` (`published`),
  KEY `idx_catid` (`catid`),
  KEY `idx_createdby` (`created_by`),
  KEY `idx_language` (`language`),
  KEY `idx_xreference` (`xreference`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_newsfeeds`
--

LOCK TABLES `rlp_newsfeeds` WRITE;
/*!40000 ALTER TABLE `rlp_newsfeeds` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_newsfeeds` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_overrider`
--

DROP TABLE IF EXISTS `rlp_overrider`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_overrider` (
  `id` int NOT NULL AUTO_INCREMENT COMMENT 'Primary Key',
  `constant` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `string` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `file` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_overrider`
--

LOCK TABLES `rlp_overrider` WRITE;
/*!40000 ALTER TABLE `rlp_overrider` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_overrider` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_postinstall_messages`
--

DROP TABLE IF EXISTS `rlp_postinstall_messages`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_postinstall_messages` (
  `postinstall_message_id` bigint unsigned NOT NULL AUTO_INCREMENT,
  `extension_id` bigint NOT NULL DEFAULT '700' COMMENT 'FK to #__extensions',
  `title_key` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'Lang key for the title',
  `description_key` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'Lang key for description',
  `action_key` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `language_extension` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT 'com_postinstall' COMMENT 'Extension holding lang keys',
  `language_client_id` tinyint NOT NULL DEFAULT '1',
  `type` varchar(10) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT 'link' COMMENT 'Message type - message, link, action',
  `action_file` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '' COMMENT 'RAD URI to the PHP file containing action method',
  `action` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '' COMMENT 'Action method name or URL',
  `condition_file` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL COMMENT 'RAD URI to file holding display condition method',
  `condition_method` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL COMMENT 'Display condition method, must return boolean',
  `version_introduced` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '3.2.0' COMMENT 'Version when this message was introduced',
  `enabled` tinyint NOT NULL DEFAULT '1',
  PRIMARY KEY (`postinstall_message_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_postinstall_messages`
--

LOCK TABLES `rlp_postinstall_messages` WRITE;
/*!40000 ALTER TABLE `rlp_postinstall_messages` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_postinstall_messages` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_privacy_consents`
--

DROP TABLE IF EXISTS `rlp_privacy_consents`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_privacy_consents` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `user_id` int unsigned NOT NULL DEFAULT '0',
  `state` int NOT NULL DEFAULT '1',
  `created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `subject` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `body` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `remind` tinyint NOT NULL DEFAULT '0',
  `token` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  PRIMARY KEY (`id`),
  KEY `idx_user_id` (`user_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_privacy_consents`
--

LOCK TABLES `rlp_privacy_consents` WRITE;
/*!40000 ALTER TABLE `rlp_privacy_consents` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_privacy_consents` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_privacy_requests`
--

DROP TABLE IF EXISTS `rlp_privacy_requests`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_privacy_requests` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `email` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `requested_at` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `status` tinyint NOT NULL DEFAULT '0',
  `request_type` varchar(25) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `confirm_token` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `confirm_token_created_at` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_privacy_requests`
--

LOCK TABLES `rlp_privacy_requests` WRITE;
/*!40000 ALTER TABLE `rlp_privacy_requests` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_privacy_requests` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_redirect_links`
--

DROP TABLE IF EXISTS `rlp_redirect_links`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_redirect_links` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `old_url` varchar(2048) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `new_url` varchar(2048) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
  `referer` varchar(2048) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `comment` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `hits` int unsigned NOT NULL DEFAULT '0',
  `published` tinyint NOT NULL,
  `created_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `header` smallint NOT NULL DEFAULT '301',
  PRIMARY KEY (`id`),
  KEY `idx_link_modifed` (`modified_date`),
  KEY `idx_old_url` (`old_url`(100))
) ENGINE=InnoDB AUTO_INCREMENT=37 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_redirect_links`
--

LOCK TABLES `rlp_redirect_links` WRITE;
/*!40000 ALTER TABLE `rlp_redirect_links` DISABLE KEYS */;
INSERT INTO `rlp_redirect_links` VALUES (1,'/tools/site-map','/tools/site-search','','',0,-2,'2020-04-30 13:08:58','2020-04-30 13:08:58',302),(2,'/user/guide','/userguide','','',0,-2,'2020-04-30 13:09:54','2020-04-30 13:09:54',302),(3,'/tutorials/pwp','https://youtu.be/-skixrvI4nU','','',0,-2,'2020-04-30 13:11:12','2020-04-30 13:11:12',302),(4,'/cgi-bin/classbrowser','/content/schema','','',0,-2,'2020-04-30 13:12:06','2020-04-30 13:12:06',302),(5,'/documentation/index.php/Glossary_Data_Model','/documentation/data-model','','',0,-2,'2020-04-30 13:12:50','2020-04-30 13:12:50',302),(6,'https://release.reactome.org/favicon.ico',NULL,'https://release.reactome.org/jenkins/login?from=%2Fjenkins%2Fjob%2FReleases%2F','',3,-2,'2020-04-30 13:40:54','0000-00-00 00:00:00',301),(7,'https://169.254.169.254/latest/dynamic/instance-identity/document',NULL,'','',3,-2,'2020-04-30 14:00:29','0000-00-00 00:00:00',301),(8,'https://[::ffff:a9fe:a9fe]/latest/dynamic/instance-identity/document',NULL,'','',3,-2,'2020-04-30 14:00:32','0000-00-00 00:00:00',301),(9,'https://release.reactome.org/gui',NULL,'','',1,-2,'2020-04-30 14:43:22','0000-00-00 00:00:00',301),(10,'https://100.26.130.52:443/.env',NULL,'','',1,-2,'2020-04-30 16:06:10','0000-00-00 00:00:00',301),(11,'https://100.26.130.52:443/v2/_catalog',NULL,'','',1,-2,'2020-04-30 16:06:11','0000-00-00 00:00:00',301),(12,'https://100.26.130.52:443/api/v1/pods',NULL,'','',1,-2,'2020-04-30 16:06:11','0000-00-00 00:00:00',301),(13,'https://100.26.130.52/home.asp',NULL,'','',5,-2,'2020-04-30 16:53:37','0000-00-00 00:00:00',301),(14,'https://100.26.130.52/login.cgi?uri=',NULL,'','',5,-2,'2020-04-30 16:53:37','0000-00-00 00:00:00',301),(15,'https://100.26.130.52/vpn/index.html',NULL,'','',5,-2,'2020-04-30 16:53:38','0000-00-00 00:00:00',301),(16,'https://100.26.130.52/cgi-bin/luci',NULL,'','',5,-2,'2020-04-30 16:53:39','0000-00-00 00:00:00',301),(17,'https://100.26.130.52/dana-na/auth/url_default/welcome.cgi',NULL,'','',5,-2,'2020-04-30 16:53:39','0000-00-00 00:00:00',301),(18,'https://100.26.130.52/remote/login?lang=en',NULL,'','',5,-2,'2020-04-30 16:53:40','0000-00-00 00:00:00',301),(19,'https://100.26.130.52/index.asp',NULL,'','',5,-2,'2020-04-30 16:53:41','0000-00-00 00:00:00',301),(20,'https://100.26.130.52/htmlv/welcomemain.htm',NULL,'','',5,-2,'2020-04-30 16:53:41','0000-00-00 00:00:00',301),(21,'https://release.reactome.org/admin',NULL,'','',1,-2,'2020-04-30 17:30:24','0000-00-00 00:00:00',301),(22,'https://release.reactome.org/website/media/sourcerer/images/tab.png',NULL,'https://release.reactome.org/administrator/index.php?option=com_content&view=article&layout=edit&id=152','',31,-2,'2020-04-30 17:31:40','0000-00-00 00:00:00',301),(23,'https://release.reactome.org/userguide-toc',NULL,'','',1,-2,'2020-04-30 17:46:10','0000-00-00 00:00:00',301),(24,'https://100.26.130.52/x',NULL,'','',1,-2,'2020-04-30 18:43:56','0000-00-00 00:00:00',301),(25,'https://100.26.130.52/s',NULL,'','',1,-2,'2020-04-30 20:19:18','0000-00-00 00:00:00',301),(26,'https://100.26.130.52:443/api/jsonws/invoke',NULL,'','',1,-2,'2020-05-01 01:01:50','0000-00-00 00:00:00',301),(27,'https://100.26.130.52:443/vendor/phpunit/phpunit/src/util/php/eval-stdin.php',NULL,'','',2,-2,'2020-05-01 01:11:41','0000-00-00 00:00:00',301),(28,'https://100.26.130.52/cgi-bin/kerbynet?action=startsessionsubmit&user=\'&id&\'&pw=',NULL,'https://100.26.130.52:443/cgi-bin/kerbynet?Action=StartSessionSubmit&User=%27%26id%26%27&PW=','',1,-2,'2020-05-01 08:08:17','0000-00-00 00:00:00',301),(29,'https://100.26.130.52/cgi-bin/kerbynet?action=x509view&section=noauthreq&user=&x509type=\'&id&\'',NULL,'https://100.26.130.52:443/cgi-bin/kerbynet?Action=x509view&Section=NoAuthREQ&User=&x509type=%27%26id%26%27','',1,-2,'2020-05-01 08:08:18','0000-00-00 00:00:00',301),(30,'https://100.26.130.52/cgi-bin/kerbynet?section=noauthreq&action=x509list&type=\"&id&\"',NULL,'https://100.26.130.52:443/cgi-bin/kerbynet?Section=NoAuthREQ&Action=x509List&type=%22%26id%26%22','',1,-2,'2020-05-01 08:08:18','0000-00-00 00:00:00',301),(31,'https://100.26.130.52/cgi-bin/kerbynet?section=noauthreq&action=x509list&type=*\";id;\"',NULL,'https://100.26.130.52:443/cgi-bin/kerbynet?Section=NoAuthREQ&Action=x509List&type=*%22;id;%22','',1,-2,'2020-05-01 08:08:18','0000-00-00 00:00:00',301),(32,'https://100.26.130.52/cgi-bin/kerbynet?stk=insert-stk-token-here&action=render&object=acctentries&filter=&id&',NULL,'https://100.26.130.52:443/cgi-bin/kerbynet?STk=INSERT-STk-TOKEN-HERE&Action=Render&Object=acctEntries&Filter=%26id%26','',1,-2,'2020-05-01 08:08:19','0000-00-00 00:00:00',301),(33,'https://100.26.130.52/cgi-bin/kerbynet?stk=insert-stk-token-here&section=router&action=checkipping&ip=&id&',NULL,'https://100.26.130.52:443/cgi-bin/kerbynet?STk=INSERT-STk-TOKEN-HERE&Section=Router&Action=CheckIPPING&IP=%26id%26','',1,-2,'2020-05-01 08:08:20','0000-00-00 00:00:00',301),(34,'https://100.26.130.52/cgi-bin/kerbynet?stk=insert-stk-token-here&section=network&action=showinterface&interface=&id&',NULL,'https://100.26.130.52:443/cgi-bin/kerbynet?STk=INSERT-STk-TOKEN-HERE&Section=Network&Action=ShowInterface&Interface=%26id%26','',1,-2,'2020-05-01 08:08:20','0000-00-00 00:00:00',301),(35,'https://100.26.130.52/ilpx',NULL,'','',1,-2,'2020-05-01 08:22:16','0000-00-00 00:00:00',301),(36,'https://100.26.130.52/owa/auth/logon.aspx?url=https://1/ecp/',NULL,'','',1,-2,'2020-05-01 10:41:23','0000-00-00 00:00:00',301);
/*!40000 ALTER TABLE `rlp_redirect_links` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_schemas`
--

DROP TABLE IF EXISTS `rlp_schemas`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_schemas` (
  `extension_id` int NOT NULL,
  `version_id` varchar(20) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`extension_id`,`version_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_schemas`
--

LOCK TABLES `rlp_schemas` WRITE;
/*!40000 ALTER TABLE `rlp_schemas` DISABLE KEYS */;
INSERT INTO `rlp_schemas` VALUES (700,'3.10.7-2022-03-18'),(10208,'');
/*!40000 ALTER TABLE `rlp_schemas` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_session`
--

DROP TABLE IF EXISTS `rlp_session`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_session` (
  `session_id` varbinary(192) NOT NULL,
  `client_id` tinyint unsigned DEFAULT NULL,
  `guest` tinyint unsigned DEFAULT '1',
  `time` int NOT NULL DEFAULT '0',
  `data` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `userid` int DEFAULT '0',
  `username` varchar(150) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '',
  PRIMARY KEY (`session_id`),
  KEY `userid` (`userid`),
  KEY `time` (`time`),
  KEY `client_id_guest` (`client_id`,`guest`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_session`
--

LOCK TABLES `rlp_session` WRITE;
/*!40000 ALTER TABLE `rlp_session` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_session` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_tags`
--

DROP TABLE IF EXISTS `rlp_tags`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_tags` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `parent_id` int unsigned NOT NULL DEFAULT '0',
  `lft` int NOT NULL DEFAULT '0',
  `rgt` int NOT NULL DEFAULT '0',
  `level` int unsigned NOT NULL DEFAULT '0',
  `path` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `title` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `note` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `description` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `published` tinyint(1) NOT NULL DEFAULT '0',
  `checked_out` int unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `access` int unsigned NOT NULL DEFAULT '0',
  `params` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `metadesc` varchar(1024) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The meta description for the page.',
  `metakey` varchar(1024) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'The meta keywords for the page.',
  `metadata` varchar(2048) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'JSON encoded metadata properties.',
  `created_user_id` int unsigned NOT NULL DEFAULT '0',
  `created_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_by_alias` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `modified_user_id` int unsigned NOT NULL DEFAULT '0',
  `modified_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `images` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `urls` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `hits` int unsigned NOT NULL DEFAULT '0',
  `language` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `version` int unsigned NOT NULL DEFAULT '1',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  PRIMARY KEY (`id`),
  KEY `tag_idx` (`published`,`access`),
  KEY `idx_access` (`access`),
  KEY `idx_checkout` (`checked_out`),
  KEY `idx_left_right` (`lft`,`rgt`),
  KEY `idx_language` (`language`),
  KEY `idx_path` (`path`(100)),
  KEY `idx_alias` (`alias`(100))
) ENGINE=InnoDB AUTO_INCREMENT=22 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_tags`
--

LOCK TABLES `rlp_tags` WRITE;
/*!40000 ALTER TABLE `rlp_tags` DISABLE KEYS */;
INSERT INTO `rlp_tags` VALUES (1,0,0,35,0,'','ROOT','root','','',1,0,'1000-01-01 00:00:00',1,'{}','','','',156,'2011-01-01 00:00:01','',0,'0000-00-00 00:00:00','','',0,'*',1,'0000-00-00 00:00:00','0000-00-00 00:00:00'),(5,1,1,2,1,'tutorial','tutorial','tutorial','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',156,'2017-06-05 09:55:40','',0,'2017-06-05 09:55:40','{}','{}',0,'*',1,'2017-06-05 09:55:40','2017-06-05 09:55:40'),(6,1,3,4,1,'graph','graph','graph','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',156,'2017-06-05 09:55:40','',0,'2017-06-05 09:55:40','{}','{}',0,'*',1,'2017-06-05 09:55:40','2017-06-05 09:55:40'),(7,1,5,6,1,'release','release','release','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',156,'2017-09-28 21:26:13','',0,'2017-09-28 21:26:13','{}','{}',0,'*',1,'2017-09-28 21:26:13','2017-09-28 21:26:13'),(8,1,7,8,1,'paper','paper','paper','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2019-12-05 16:19:06','',0,'2019-12-05 16:19:06','{}','{}',0,'*',1,'2019-12-05 16:19:06','2019-12-05 16:19:06'),(9,1,9,10,1,'tools','tools ','tools','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2020-04-06 17:59:35','',0,'2020-04-06 17:59:35','{}','{}',0,'*',1,'2020-04-06 17:59:35','2020-04-06 17:59:35'),(10,1,11,12,1,'python','python','python','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2020-04-06 17:59:35','',0,'2020-04-06 17:59:35','{}','{}',0,'*',1,'2020-04-06 17:59:35','2020-04-06 17:59:35'),(11,1,13,14,1,'developer','developer','developer','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2020-04-06 17:59:35','',0,'2020-04-06 17:59:35','{}','{}',0,'*',1,'2020-04-06 17:59:35','2020-04-06 17:59:35'),(12,1,15,16,1,'curation','curation','curation','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2020-09-18 20:52:57','',0,'2020-09-18 20:52:57','{}','{}',0,'*',1,'2020-09-18 20:52:57','2020-09-18 20:52:57'),(13,1,17,18,1,'v75','V75','v75','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2020-12-02 18:58:30','',0,'2020-12-02 18:58:30','{}','{}',0,'*',1,'2020-12-02 18:58:30','2020-12-02 18:58:30'),(14,1,19,20,1,'v76','V76','v76','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2021-03-22 03:18:41','',0,'2021-03-22 03:18:41','{}','{}',0,'*',1,'2021-03-22 03:18:41','2021-03-22 03:18:41'),(15,1,21,22,1,'idg','idg','idg','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2021-06-07 17:47:37','',0,'2021-06-07 17:47:37','{}','{}',0,'*',1,'2021-06-07 17:47:37','2021-06-07 17:47:37'),(16,1,23,24,1,'v77','V77','v77','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2021-06-09 19:53:12','',0,'2021-06-09 19:53:12','{}','{}',0,'*',1,'2021-06-09 19:53:12','2021-06-09 19:53:12'),(17,1,25,26,1,'reactomegsa','reactomegsa','reactomegsa','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2021-06-21 16:32:18','',0,'2021-06-21 16:32:18','{}','{}',0,'*',1,'2021-06-21 16:32:18','2021-06-21 16:32:18'),(18,1,27,28,1,'successstory','successstory','successstory','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2021-11-01 17:28:19','',0,'2021-11-01 17:28:19','{}','{}',0,'*',1,'2021-11-01 17:28:19','2021-11-01 17:28:19'),(19,1,29,30,1,'usecase','usecase','usecase','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2021-11-01 17:28:19','',0,'2021-11-01 17:28:19','{}','{}',0,'*',1,'2021-11-01 17:28:19','2021-11-01 17:28:19'),(20,1,31,32,1,'v79','v79','v79','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2021-12-13 06:03:05','',0,'2021-12-13 06:03:05','{}','{}',0,'*',1,'2021-12-13 06:03:05','2021-12-13 06:03:05'),(21,1,33,34,1,'v80','v80','v80','','',1,0,'1000-01-01 00:00:00',1,'{}','','','{}',169,'2022-04-05 18:01:49','',0,'2022-04-05 18:01:49','{}','{}',0,'*',1,'2022-04-05 18:01:49','2022-04-05 18:01:49');
/*!40000 ALTER TABLE `rlp_tags` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_template_styles`
--

DROP TABLE IF EXISTS `rlp_template_styles`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_template_styles` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `template` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `client_id` tinyint unsigned NOT NULL DEFAULT '0',
  `home` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '0',
  `title` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `params` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `inheritable` tinyint NOT NULL DEFAULT '0',
  `parent` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '',
  PRIMARY KEY (`id`),
  KEY `idx_template` (`template`),
  KEY `idx_client_id` (`client_id`),
  KEY `idx_client_id_home` (`client_id`,`home`)
) ENGINE=InnoDB AUTO_INCREMENT=25 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_template_styles`
--

LOCK TABLES `rlp_template_styles` WRITE;
/*!40000 ALTER TABLE `rlp_template_styles` DISABLE KEYS */;
INSERT INTO `rlp_template_styles` VALUES (4,'beez3',0,'0','Beez3 - Default','{\"wrapperSmall\":\"53\",\"wrapperLarge\":\"72\",\"logo\":\"images\\/joomla_black.gif\",\"sitetitle\":\"Joomla!\",\"sitedescription\":\"Open Source Content Management\",\"navposition\":\"left\",\"templatecolor\":\"personal\",\"html5\":\"0\"}',0,''),(5,'hathor',1,'0','Hathor - Default','{\"showSiteName\":\"0\",\"colourChoice\":\"\",\"boldText\":\"0\"}',0,''),(7,'protostar',0,'0','protostar - Default','{\"templateColor\":\"#0088cc\",\"templateBackgroundColor\":\"#f7f2f2\",\"logoFile\":\"\",\"sitetitle\":\"\",\"sitedescription\":\"\",\"googleFont\":\"1\",\"googleFontName\":\"Open+Sans\",\"fluidContainer\":\"0\"}',0,''),(8,'isis',1,'1','isis - Default','{\"templateColor\":\"\",\"logoFile\":\"\"}',0,''),(24,'favourite',0,'1','Favourite - Default','{\"template_styles\":\"style2\",\"custom_style\":\"\",\"max_width\":\"1190px\",\"show_copyright\":\"1\",\"copyright_text\":\"Reactome\",\"copyright_text_link\":\"reactome.org\",\"nav_link_padding\":\"\",\"nav_font_size\":\"\",\"nav_text_transform\":\"uppercase\",\"nav_icons_color\":\"\",\"nav_icons_font_size\":\"\",\"nav_google_font\":\"Open Sans\",\"nav_google_font_weight\":\"400\",\"nav_google_font_style\":\"normal\",\"titles_font_size\":\"\",\"titles_line_height\":\"\",\"titles_text_align\":\"left\",\"titles_text_transform\":\"uppercase\",\"module_title_icon_font_size\":\"\",\"module_title_icon_padding\":\"\",\"module_title_icon_border_radius\":\"\",\"titles_google_font\":\"Open Sans\",\"titles_google_font_weight\":\"300\",\"titles_google_font_style\":\"normal\",\"error_page_article_id\":\"43\",\"offline_page_style\":\"offline-light\",\"offline_page_bg_image\":\"\",\"offline_page_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"show_back_to_top\":\"1\",\"back_to_top_style_color\":\"\",\"back_to_top_text\":\"Back to Top\",\"body_bg_image\":\"\",\"body_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"body_bg_image_overlay\":\"fav-transparent\",\"body_bg_color\":\"\",\"body_color\":\"\",\"body_title_color\":\"\",\"body_link_color\":\"\",\"body_link_hover_color\":\"\",\"notice_bg_style\":\"fav-module-block-color\",\"notice_bg_color\":\"FCF8E3\",\"notice_color\":\"8A6D3B\",\"notice_title_color\":\"\",\"notice_link_color\":\"\",\"notice_link_hover_color\":\"\",\"topbar_bg_style\":\"fav-module-block-color\",\"topbar_bg_image\":\"\",\"topbar_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"topbar_bg_image_overlay\":\"fav-transparent\",\"topbar_bg_color\":\"F2DEDE\",\"topbar_color\":\"A94442\",\"topbar_title_color\":\"\",\"topbar_link_color\":\"\",\"topbar_link_hover_color\":\"\",\"slide_width\":\"\",\"slide_bg_style\":\"fav-module-block-light\",\"slide_bg_image\":\"\",\"slide_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"slide_bg_image_overlay\":\"fav-transparent\",\"slide_bg_color\":\"F0F0F0\",\"slide_color\":\"\",\"slide_title_color\":\"\",\"slide_link_color\":\"\",\"slide_link_hover_color\":\"\",\"intro_bg_style\":\"fav-module-block-light\",\"intro_bg_image\":\"\",\"intro_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"intro_bg_image_overlay\":\"fav-overlay\",\"intro_bg_color\":\"\",\"intro_color\":\"\",\"intro_title_color\":\"\",\"intro_link_color\":\"\",\"intro_link_hover_color\":\"\",\"breadcrumbs_bg_style\":\"fav-module-block-light\",\"breadcrumbs_bg_image\":\"\",\"breadcrumbs_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"breadcrumbs_bg_image_overlay\":\"fav-transparent\",\"breadcrumbs_bg_color\":\"\",\"breadcrumbs_color\":\"\",\"breadcrumbs_title_color\":\"\",\"breadcrumbs_link_color\":\"\",\"breadcrumbs_link_hover_color\":\"\",\"lead_bg_style\":\"fav-module-block-light\",\"lead_bg_image\":\"\",\"lead_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"lead_bg_image_overlay\":\"fav-transparent\",\"lead_bg_color\":\"\",\"lead_color\":\"\",\"lead_title_color\":\"\",\"lead_link_color\":\"\",\"lead_link_hover_color\":\"\",\"promo_bg_style\":\"fav-module-block-light\",\"promo_bg_image\":\"\",\"promo_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"promo_bg_image_overlay\":\"fav-transparent\",\"promo_bg_color\":\"\",\"promo_color\":\"\",\"promo_title_color\":\"\",\"promo_link_color\":\"\",\"promo_link_hover_color\":\"\",\"prime_bg_style\":\"fav-module-block-light\",\"prime_bg_image\":\"\",\"prime_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"prime_bg_image_overlay\":\"fav-overlay\",\"prime_bg_color\":\"FFFFFF\",\"prime_color\":\"\",\"prime_title_color\":\"\",\"prime_link_color\":\"\",\"prime_link_hover_color\":\"\",\"showcase_bg_style\":\"fav-module-block-light\",\"showcase_bg_image\":\"\",\"showcase_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"showcase_bg_image_overlay\":\"fav-transparent\",\"showcase_bg_color\":\"111111\",\"showcase_color\":\"\",\"showcase_title_color\":\"\",\"showcase_link_color\":\"\",\"showcase_link_hover_color\":\"\",\"feature_bg_style\":\"fav-module-block-light\",\"feature_bg_image\":\"\",\"feature_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"feature_bg_image_overlay\":\"fav-transparent\",\"feature_bg_color\":\"\",\"feature_color\":\"\",\"feature_title_color\":\"\",\"feature_link_color\":\"\",\"feature_link_hover_color\":\"\",\"focus_bg_style\":\"fav-module-block-color\",\"focus_bg_image\":\"\",\"focus_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"focus_bg_image_overlay\":\"fav-transparent\",\"focus_bg_color\":\"\",\"focus_color\":\"\",\"focus_title_color\":\"\",\"focus_link_color\":\"\",\"focus_link_hover_color\":\"\",\"portfolio_bg_style\":\"fav-module-block-light\",\"portfolio_bg_image\":\"\",\"portfolio_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"portfolio_bg_image_overlay\":\"fav-transparent\",\"portfolio_bg_color\":\"\",\"portfolio_color\":\"\",\"portfolio_title_color\":\"\",\"portfolio_link_color\":\"\",\"portfolio_link_hover_color\":\"\",\"screen_bg_style\":\"fav-module-block-color\",\"screen_bg_image\":\"\",\"screen_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"screen_bg_image_overlay\":\"fav-transparent\",\"screen_bg_color\":\"F0F0F0\",\"screen_color\":\"\",\"screen_title_color\":\"444444\",\"screen_link_color\":\"\",\"screen_link_hover_color\":\"\",\"top_bg_style\":\"fav-module-block-light\",\"top_bg_image\":\"\",\"top_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"top_bg_image_overlay\":\"fav-transparent\",\"top_bg_color\":\"\",\"top_color\":\"\",\"top_title_color\":\"444444\",\"top_link_color\":\"\",\"top_link_hover_color\":\"\",\"maintop_bg_style\":\"fav-module-block-light\",\"maintop_bg_image\":\"\",\"maintop_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"maintop_bg_image_overlay\":\"fav-transparent\",\"maintop_bg_color\":\"\",\"maintop_color\":\"\",\"maintop_title_color\":\"\",\"maintop_link_color\":\"\",\"maintop_link_hover_color\":\"\",\"mainbottom_bg_style\":\"fav-module-block-light\",\"mainbottom_bg_image\":\"\",\"mainbottom_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"mainbottom_bg_image_overlay\":\"fav-transparent\",\"mainbottom_bg_color\":\"\",\"mainbottom_color\":\"\",\"mainbottom_title_color\":\"\",\"mainbottom_link_color\":\"\",\"mainbottom_link_hover_color\":\"\",\"bottom_bg_style\":\"fav-module-block-light\",\"bottom_bg_image\":\"\",\"bottom_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"bottom_bg_image_overlay\":\"fav-transparent\",\"bottom_bg_color\":\"\",\"bottom_color\":\"\",\"bottom_title_color\":\"\",\"bottom_link_color\":\"\",\"bottom_link_hover_color\":\"\",\"note_bg_style\":\"fav-module-block-dark\",\"note_bg_image\":\"\",\"note_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"note_bg_image_overlay\":\"fav-transparent\",\"note_bg_color\":\"\",\"note_color\":\"\",\"note_title_color\":\"\",\"note_link_color\":\"\",\"note_link_hover_color\":\"\",\"base_bg_style\":\"fav-module-block-light\",\"base_bg_image\":\"\",\"base_bg_image_style\":\"no-repeat; background-attachment: fixed; -webkit-background-size: cover; -moz-background-size: cover; -o-background-size: cover; background-size: cover;\",\"base_bg_image_overlay\":\"fav-overlay\",\"base_bg_color\":\"\",\"base_color\":\"\",\"base_title_color\":\"\",\"base_link_color\":\"\",\"base_link_hover_color\":\"\",\"block_bg_style\":\"fav-module-block-light\",\"block_bg_image\":\"\",\"block_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"block_bg_image_overlay\":\"fav-transparent\",\"block_bg_color\":\"\",\"block_color\":\"\",\"block_title_color\":\"\",\"block_link_color\":\"\",\"block_link_hover_color\":\"\",\"user_bg_style\":\"fav-module-block-color\",\"user_bg_image\":\"\",\"user_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"user_bg_image_overlay\":\"fav-transparent\",\"user_bg_color\":\"F0F0F0\",\"user_color\":\"\",\"user_title_color\":\"\",\"user_link_color\":\"\",\"user_link_hover_color\":\"\",\"footer_bg_style\":\"fav-module-block-color\",\"footer_bg_image\":\"\",\"footer_bg_image_style\":\"repeat; background-attachment: initial; -webkit-background-size: auto; -moz-background-size: auto; -o-background-size: auto; background-size: auto;\",\"footer_bg_image_overlay\":\"fav-transparent\",\"footer_bg_color\":\"58C3E5\",\"footer_color\":\"FFFFFF\",\"footer_title_color\":\"\",\"footer_link_color\":\"FFFFFF\",\"footer_link_hover_color\":\"\",\"show_default_logo\":\"1\",\"default_logo\":\"logo.png\",\"default_logo_img_alt\":\"Reactome Pathway Database\",\"default_logo_padding\":\"\",\"default_logo_margin\":\"\",\"show_media_logo\":\"0\",\"media_logo\":\"\",\"media_logo_img_alt\":\"Reactome Pathway Database\",\"media_logo_padding\":\"\",\"media_logo_margin\":\"\",\"show_text_logo\":\"0\",\"text_logo\":\"REACTOME\",\"text_logo_color\":\"58C3E5\",\"text_logo_font_size\":\"42px\",\"text_logo_google_font\":\"Source Sans Pro\",\"text_logo_google_font_weight\":\"700\",\"text_logo_google_font_style\":\"normal\",\"text_logo_line_height\":\"\",\"text_logo_padding\":\"\",\"text_logo_margin\":\"6px 0 0 0\",\"show_slogan\":\"0\",\"slogan\":\"A curated Pathway Database\",\"slogan_color\":\"\",\"slogan_font_size\":\"\",\"slogan_line_height\":\"14px\",\"slogan_padding\":\"\",\"slogan_margin\":\"\",\"show_retina_logo\":\"0\",\"retina_logo\":\"\",\"retina_logo_height\":\"52px\",\"retina_logo_width\":\"188px\",\"retina_logo_img_alt\":\"Reactome Pathway Database\",\"retina_logo_padding\":\"0px\",\"retina_logo_margin\":\"0px\",\"mobile_nav_color\":\"favth-navbar-default\",\"show_mobile_menu_text\":\"0\",\"mobile_menu_text\":\"MENU\",\"mobile_paragraph_font_size\":\"12px\",\"mobile_paragraph_line_height\":\"\",\"mobile_title_font_size\":\"\",\"analytics_code\":\"\"}',0,'');
/*!40000 ALTER TABLE `rlp_template_styles` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_ucm_base`
--

DROP TABLE IF EXISTS `rlp_ucm_base`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_ucm_base` (
  `ucm_id` int unsigned NOT NULL,
  `ucm_item_id` int NOT NULL,
  `ucm_type_id` int NOT NULL,
  `ucm_language_id` int NOT NULL,
  PRIMARY KEY (`ucm_id`),
  KEY `idx_ucm_item_id` (`ucm_item_id`),
  KEY `idx_ucm_type_id` (`ucm_type_id`),
  KEY `idx_ucm_language_id` (`ucm_language_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_ucm_base`
--

LOCK TABLES `rlp_ucm_base` WRITE;
/*!40000 ALTER TABLE `rlp_ucm_base` DISABLE KEYS */;
INSERT INTO `rlp_ucm_base` VALUES (1,54,1,0),(2,90,1,0),(3,78,1,0),(4,77,1,0),(5,74,1,0),(6,72,1,0),(8,98,1,1),(9,102,1,1),(10,103,1,1),(11,106,1,1),(12,116,1,1),(13,125,1,1),(14,127,1,1),(15,137,1,1),(17,145,1,1),(18,144,1,1),(19,146,1,1),(20,147,1,1),(21,150,1,1),(22,151,1,1),(23,155,1,1),(25,161,1,1),(26,162,1,1),(27,163,1,1),(28,164,1,1),(29,71,1,0),(30,142,1,1),(31,165,1,1),(32,166,1,1),(33,167,1,1),(34,169,1,1),(35,172,1,1),(36,174,1,1),(37,175,1,1),(38,177,1,1),(39,178,1,1),(40,176,1,1),(41,184,1,1),(42,194,1,1),(43,225,1,1),(44,230,1,1),(45,231,1,1),(46,238,1,1),(47,246,1,1),(48,251,1,1),(49,261,1,1);
/*!40000 ALTER TABLE `rlp_ucm_base` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_ucm_content`
--

DROP TABLE IF EXISTS `rlp_ucm_content`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_ucm_content` (
  `core_content_id` int unsigned NOT NULL AUTO_INCREMENT,
  `core_type_alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'FK to the content types table',
  `core_title` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `core_alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL DEFAULT '',
  `core_body` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `core_state` tinyint(1) NOT NULL DEFAULT '0',
  `core_checked_out_time` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '0000-00-00 00:00:00',
  `core_checked_out_user_id` int unsigned NOT NULL DEFAULT '0',
  `core_access` int unsigned NOT NULL DEFAULT '0',
  `core_params` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `core_featured` tinyint unsigned NOT NULL DEFAULT '0',
  `core_metadata` varchar(2048) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'JSON encoded metadata properties.',
  `core_created_user_id` int unsigned NOT NULL DEFAULT '0',
  `core_created_by_alias` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `core_created_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `core_modified_user_id` int unsigned NOT NULL DEFAULT '0' COMMENT 'Most recent user that modified',
  `core_modified_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `core_language` char(7) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `core_publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `core_publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `core_content_item_id` int unsigned NOT NULL DEFAULT '0' COMMENT 'ID from the individual type table',
  `asset_id` int unsigned NOT NULL DEFAULT '0' COMMENT 'FK to the #__assets table.',
  `core_images` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `core_urls` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `core_hits` int unsigned NOT NULL DEFAULT '0',
  `core_version` int unsigned NOT NULL DEFAULT '1',
  `core_ordering` int NOT NULL DEFAULT '0',
  `core_metakey` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `core_metadesc` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
  `core_catid` int unsigned NOT NULL DEFAULT '0',
  `core_xreference` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'A reference to enable linkages to external data sets.',
  `core_type_id` int unsigned NOT NULL DEFAULT '0',
  PRIMARY KEY (`core_content_id`),
  KEY `tag_idx` (`core_state`,`core_access`),
  KEY `idx_access` (`core_access`),
  KEY `idx_language` (`core_language`),
  KEY `idx_modified_time` (`core_modified_time`),
  KEY `idx_created_time` (`core_created_time`),
  KEY `idx_core_modified_user_id` (`core_modified_user_id`),
  KEY `idx_core_checked_out_user_id` (`core_checked_out_user_id`),
  KEY `idx_core_created_user_id` (`core_created_user_id`),
  KEY `idx_core_type_id` (`core_type_id`),
  KEY `idx_alias` (`core_alias`(100)),
  KEY `idx_title` (`core_title`(100)),
  KEY `idx_content_type` (`core_type_alias`(100))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Contains core content data in name spaced fields';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_ucm_content`
--

LOCK TABLES `rlp_ucm_content` WRITE;
/*!40000 ALTER TABLE `rlp_ucm_content` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_ucm_content` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_ucm_history`
--

DROP TABLE IF EXISTS `rlp_ucm_history`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_ucm_history` (
  `version_id` int unsigned NOT NULL AUTO_INCREMENT,
  `ucm_item_id` int unsigned NOT NULL,
  `ucm_type_id` int unsigned NOT NULL,
  `version_note` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'Optional version name',
  `save_date` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `editor_user_id` int unsigned NOT NULL DEFAULT '0',
  `character_count` int unsigned NOT NULL DEFAULT '0' COMMENT 'Number of characters in this version.',
  `sha1_hash` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'SHA1 hash of the version_data column.',
  `version_data` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'json-encoded string of version data',
  `keep_forever` tinyint NOT NULL DEFAULT '0' COMMENT '0=auto delete; 1=keep',
  PRIMARY KEY (`version_id`),
  KEY `idx_ucm_item_id` (`ucm_type_id`,`ucm_item_id`),
  KEY `idx_save_date` (`save_date`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_ucm_history`
--

LOCK TABLES `rlp_ucm_history` WRITE;
/*!40000 ALTER TABLE `rlp_ucm_history` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_ucm_history` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_update_sites`
--

DROP TABLE IF EXISTS `rlp_update_sites`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_update_sites` (
  `update_site_id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '',
  `type` varchar(20) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '',
  `location` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `enabled` int DEFAULT '0',
  `last_check_timestamp` bigint DEFAULT '0',
  `extra_query` varchar(1000) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '',
  PRIMARY KEY (`update_site_id`)
) ENGINE=InnoDB AUTO_INCREMENT=74 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Update Sites';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_update_sites`
--

LOCK TABLES `rlp_update_sites` WRITE;
/*!40000 ALTER TABLE `rlp_update_sites` DISABLE KEYS */;
INSERT INTO `rlp_update_sites` VALUES (1,'Joomla! Core','collection','https://update.joomla.org/core/list.xml',1,NULL,''),(3,'Accredited Joomla! Translations','collection','https://update.joomla.org/language/translationlist_3.xml',1,NULL,''),(4,'Joomla! Update Component Update Site','extension','https://update.joomla.org/core/extensions/com_joomlaupdate.xml',1,NULL,''),(9,'COM_EASYJOOMLABACKUP','extension','https://check.kubik-rubik.de/updates/com_easyjoomlabackup.xml',1,NULL,''),(12,'Regular Labs - Tabs','extension','https://download.regularlabs.com/updates.xml?e=tabs&type=.xml',1,NULL,''),(13,'Regular Labs - Sliders','extension','https://download.regularlabs.com/updates.xml?e=sliders&type=.xml',1,NULL,''),(14,'AlterBrains Updates','collection','http://www.alterbrains.com/update.xml',1,NULL,''),(16,'','collection','http://www.joomultra.com/service/tracking/list.xml',0,NULL,''),(17,'Regular Labs - Sourcerer','extension','https://download.regularlabs.com/updates.xml?e=sourcerer&type=.xml',1,NULL,''),(23,'Alledia Framework','extension','https://deploy.ostraining.com/client/update/free/stable/lib_allediaframework',1,NULL,''),(24,'Regular Labs - Modals','extension','https://download.regularlabs.com/updates.xml?e=modals&type=.xml',1,NULL,''),(25,'FavPromote Updates','extension','http://updates.favthemes.com/extensions/favpromote.xml',1,NULL,''),(27,'FavGlyph Updates','extension','http://updates.favthemes.com/extensions/favglyph.xml',1,NULL,''),(72,'JCE Editor Package','extension','https://cdn.joomlacontenteditor.net/updates/xml/editor/pkg_jce.xml',1,NULL,''),(73,'Regular Labs Library','extension','https://download.regularlabs.com/updates.xml?e=library&type=.xml',1,NULL,'');
/*!40000 ALTER TABLE `rlp_update_sites` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_update_sites_extensions`
--

DROP TABLE IF EXISTS `rlp_update_sites_extensions`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_update_sites_extensions` (
  `update_site_id` int NOT NULL DEFAULT '0',
  `extension_id` int NOT NULL DEFAULT '0',
  PRIMARY KEY (`update_site_id`,`extension_id`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Links extensions to update sites';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_update_sites_extensions`
--

LOCK TABLES `rlp_update_sites_extensions` WRITE;
/*!40000 ALTER TABLE `rlp_update_sites_extensions` DISABLE KEYS */;
INSERT INTO `rlp_update_sites_extensions` VALUES (1,700),(3,802),(4,28),(9,10208),(12,10226),(12,10227),(12,10447),(13,10230),(13,10231),(13,10444),(14,10232),(16,10235),(17,10238),(17,10239),(17,10445),(23,10250),(24,10254),(24,10269),(24,10442),(25,10182),(27,10180),(72,10200),(73,10225),(73,10411),(73,10413),(73,10419),(73,10420),(73,10425),(73,10426),(73,10428),(73,10429),(73,10431),(73,10432),(73,10434),(73,10435),(73,10437),(73,10441);
/*!40000 ALTER TABLE `rlp_update_sites_extensions` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_updates`
--

DROP TABLE IF EXISTS `rlp_updates`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_updates` (
  `update_id` int NOT NULL AUTO_INCREMENT,
  `update_site_id` int DEFAULT '0',
  `extension_id` int DEFAULT '0',
  `name` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '',
  `description` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `element` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '',
  `type` varchar(20) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '',
  `folder` varchar(20) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '',
  `client_id` tinyint DEFAULT '0',
  `version` varchar(32) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '',
  `data` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `detailsurl` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `infourl` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `extra_query` varchar(1000) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT '',
  PRIMARY KEY (`update_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Available Updates';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_updates`
--

LOCK TABLES `rlp_updates` WRITE;
/*!40000 ALTER TABLE `rlp_updates` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_updates` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_user_keys`
--

DROP TABLE IF EXISTS `rlp_user_keys`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_user_keys` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `user_id` varchar(150) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `token` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `series` varchar(191) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `invalid` tinyint NOT NULL,
  `time` varchar(200) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `uastring` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `series` (`series`),
  KEY `user_id` (`user_id`)
) ENGINE=InnoDB AUTO_INCREMENT=23 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_user_keys`
--

LOCK TABLES `rlp_user_keys` WRITE;
/*!40000 ALTER TABLE `rlp_user_keys` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_user_keys` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_user_notes`
--

DROP TABLE IF EXISTS `rlp_user_notes`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_user_notes` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `user_id` int unsigned NOT NULL DEFAULT '0',
  `catid` int unsigned NOT NULL DEFAULT '0',
  `subject` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `body` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `state` tinyint NOT NULL DEFAULT '0',
  `checked_out` int unsigned NOT NULL DEFAULT '0',
  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `created_user_id` int unsigned NOT NULL DEFAULT '0',
  `created_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `modified_user_id` int unsigned NOT NULL,
  `modified_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `review_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_up` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `publish_down` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  PRIMARY KEY (`id`),
  KEY `idx_user_id` (`user_id`),
  KEY `idx_category_id` (`catid`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_user_notes`
--

LOCK TABLES `rlp_user_notes` WRITE;
/*!40000 ALTER TABLE `rlp_user_notes` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_user_notes` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_user_profiles`
--

DROP TABLE IF EXISTS `rlp_user_profiles`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_user_profiles` (
  `user_id` int NOT NULL,
  `profile_key` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `profile_value` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `ordering` int NOT NULL DEFAULT '0',
  UNIQUE KEY `idx_user_id_profile_key` (`user_id`,`profile_key`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Simple user profile storage table';
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_user_profiles`
--

LOCK TABLES `rlp_user_profiles` WRITE;
/*!40000 ALTER TABLE `rlp_user_profiles` DISABLE KEYS */;
/*!40000 ALTER TABLE `rlp_user_profiles` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_user_usergroup_map`
--

DROP TABLE IF EXISTS `rlp_user_usergroup_map`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_user_usergroup_map` (
  `user_id` int unsigned NOT NULL DEFAULT '0' COMMENT 'Foreign Key to #__users.id',
  `group_id` int unsigned NOT NULL DEFAULT '0' COMMENT 'Foreign Key to #__usergroups.id',
  PRIMARY KEY (`user_id`,`group_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_user_usergroup_map`
--

LOCK TABLES `rlp_user_usergroup_map` WRITE;
/*!40000 ALTER TABLE `rlp_user_usergroup_map` DISABLE KEYS */;
INSERT INTO `rlp_user_usergroup_map` VALUES (156,8),(162,2),(163,2),(165,2),(166,2),(167,2),(168,2),(170,2),(171,2),(172,2),(173,2),(174,2),(175,2),(176,2),(177,2),(178,2),(179,2),(180,2),(182,2),(183,2),(184,2);
/*!40000 ALTER TABLE `rlp_user_usergroup_map` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_usergroups`
--

DROP TABLE IF EXISTS `rlp_usergroups`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_usergroups` (
  `id` int unsigned NOT NULL AUTO_INCREMENT COMMENT 'Primary Key',
  `parent_id` int unsigned NOT NULL DEFAULT '0' COMMENT 'Adjacency List Reference Id',
  `lft` int NOT NULL DEFAULT '0' COMMENT 'Nested set lft.',
  `rgt` int NOT NULL DEFAULT '0' COMMENT 'Nested set rgt.',
  `title` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  PRIMARY KEY (`id`),
  UNIQUE KEY `idx_usergroup_parent_title_lookup` (`parent_id`,`title`),
  KEY `idx_usergroup_title_lookup` (`title`),
  KEY `idx_usergroup_adjacency_lookup` (`parent_id`),
  KEY `idx_usergroup_nested_set_lookup` (`lft`,`rgt`) USING BTREE
) ENGINE=InnoDB AUTO_INCREMENT=12 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_usergroups`
--

LOCK TABLES `rlp_usergroups` WRITE;
/*!40000 ALTER TABLE `rlp_usergroups` DISABLE KEYS */;
INSERT INTO `rlp_usergroups` VALUES (1,0,1,22,'Public'),(2,1,8,19,'Registered'),(3,2,9,18,'Author'),(4,3,10,17,'Editor'),(5,4,13,14,'Publisher'),(6,1,4,7,'Manager'),(7,6,5,6,'Administrator'),(8,1,20,21,'Super Users'),(9,1,2,3,'Guest'),(10,4,15,16,'SyncTool'),(11,4,11,12,'FrontEnd Editor');
/*!40000 ALTER TABLE `rlp_usergroups` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_users`
--

DROP TABLE IF EXISTS `rlp_users`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_users` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `username` varchar(150) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `email` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `password` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `block` tinyint NOT NULL DEFAULT '0',
  `sendEmail` tinyint DEFAULT '0',
  `registerDate` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `lastvisitDate` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
  `activation` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `params` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
  `lastResetTime` datetime NOT NULL DEFAULT '0000-00-00 00:00:00' COMMENT 'Date of last password reset',
  `resetCount` int NOT NULL DEFAULT '0' COMMENT 'Count of password resets since lastResetTime',
  `otpKey` varchar(1000) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'Two factor authentication encrypted keys',
  `otep` varchar(1000) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'One time emergency passwords',
  `requireReset` tinyint NOT NULL DEFAULT '0' COMMENT 'Require user to reset password on next login',
  `authProvider` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'Name of used authentication plugin',
  PRIMARY KEY (`id`),
  UNIQUE KEY `idx_username` (`username`),
  KEY `idx_block` (`block`),
  KEY `email` (`email`),
  KEY `idx_name` (`name`(100))
) ENGINE=InnoDB AUTO_INCREMENT=185 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_users`
--

LOCK TABLES `rlp_users` WRITE;
/*!40000 ALTER TABLE `rlp_users` DISABLE KEYS */;
INSERT INTO `rlp_users` VALUES (156,'Joomla Admin','admin','change-me@nodomain.com','$2y$10$suDIc1tF2MApag9nthYrOez4G9mMDT.ushhi6CeEspDp5aKQjLnlC',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','0','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(158,'Name 158','unknown158','invalid158@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,1,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(159,'Name 159','unknown159','invalid159@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(161,'Name 161','unknown161','invalid161@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(162,'Name 162','unknown162','invalid162@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(163,'Name 163','unknown163','invalid163@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(165,'Name 165','unknown165','invalid165@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(166,'Name 166','unknown166','invalid166@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(167,'Name 167','unknown167','invalid167@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(168,'Name 168','unknown168','invalid168@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(169,'Name 169','unknown169','invalid169@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(170,'Name 170','unknown170','invalid170@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(171,'Name 171','unknown171','invalid171@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(172,'Name 172','unknown172','invalid172@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(173,'Name 173','unknown173','invalid173@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(174,'Name 174','unknown174','invalid174@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(175,'Name 175','unknown175','invalid175@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(176,'Name 176','unknown176','invalid176@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(177,'Name 177','unknown177','invalid177@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(178,'Name 178','unknown178','invalid178@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(179,'Name 179','unknown179','invalid179@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(180,'Name 180','unknown180','invalid180@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(181,'Name 181','unknown181','invalid181@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,1,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(182,'Name 182','unknown182','invalid182@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',1,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(183,'Name 183','unknown183','invalid183@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,''),(184,'Name 184','unknown184','invalid184@nodomain.com','$2y$10$V.KRp85mMReYkTQlk45.o.fcIzBS/FAorz8cEsIJTSM6yLjM6jhnS',0,0,'2017-01-01 09:00:00','2017-01-01 10:00:00','','{\"admin_style\":\"\",\"admin_language\":\"\",\"language\":\"\",\"editor\":\"\",\"helpsite\":\"\",\"timezone\":\"\"}','2017-01-01 00:00:00',0,'','',0,'');
/*!40000 ALTER TABLE `rlp_users` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_utf8_conversion`
--

DROP TABLE IF EXISTS `rlp_utf8_conversion`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_utf8_conversion` (
  `converted` tinyint NOT NULL DEFAULT '0'
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_utf8_conversion`
--

LOCK TABLES `rlp_utf8_conversion` WRITE;
/*!40000 ALTER TABLE `rlp_utf8_conversion` DISABLE KEYS */;
INSERT INTO `rlp_utf8_conversion` VALUES (5);
/*!40000 ALTER TABLE `rlp_utf8_conversion` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_viewlevels`
--

DROP TABLE IF EXISTS `rlp_viewlevels`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_viewlevels` (
  `id` int unsigned NOT NULL AUTO_INCREMENT COMMENT 'Primary Key',
  `title` varchar(100) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
  `ordering` int NOT NULL DEFAULT '0',
  `rules` varchar(5120) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'JSON encoded access control.',
  PRIMARY KEY (`id`),
  UNIQUE KEY `idx_assetgroup_title_lookup` (`title`)
) ENGINE=InnoDB AUTO_INCREMENT=8 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_viewlevels`
--

LOCK TABLES `rlp_viewlevels` WRITE;
/*!40000 ALTER TABLE `rlp_viewlevels` DISABLE KEYS */;
INSERT INTO `rlp_viewlevels` VALUES (1,'Public',1,'[1]'),(2,'Registered',3,'[6,2,8]'),(3,'Special',4,'[6,3,8]'),(5,'Guest',2,'[9]'),(6,'Super Users',5,'[8]'),(7,'Synchronization Tool',6,'[10]');
/*!40000 ALTER TABLE `rlp_viewlevels` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `rlp_wf_profiles`
--

DROP TABLE IF EXISTS `rlp_wf_profiles`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `rlp_wf_profiles` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(255) NOT NULL,
  `description` text NOT NULL,
  `users` text NOT NULL,
  `types` text NOT NULL,
  `components` text NOT NULL,
  `area` tinyint NOT NULL,
  `device` varchar(255) NOT NULL,
  `rows` text NOT NULL,
  `plugins` text NOT NULL,
  `published` tinyint NOT NULL,
  `ordering` int NOT NULL,
  `checked_out` int unsigned DEFAULT NULL,
  `checked_out_time` datetime DEFAULT NULL,
  `params` text NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8mb3;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `rlp_wf_profiles`
--

LOCK TABLES `rlp_wf_profiles` WRITE;
/*!40000 ALTER TABLE `rlp_wf_profiles` DISABLE KEYS */;
INSERT INTO `rlp_wf_profiles` VALUES (1,'Default','Default Profile for all users','','6,7,2,3,4,5,8','',0,'phone,tablet,desktop','help,newdocument,undo,redo,spacer,bold,italic,underline,strikethrough,justifyfull,justifycenter,justifyleft,justifyright,spacer,blockquote,formatselect,styleselect,removeformat,table,cleanup;searchreplace,fontselect,fontsizeselect,fontcolor,spacer,clipboard,indent,outdent,lists,sub,sup,textcase,directionality,charmap,style,hr,fullscreen,xhtmlxtras;visualaid,visualchars,visualblocks,nonbreaking,anchor,unlink,link,imgmanager,spellchecker,article,emotions,layer,autosave','formatselect,styleselect,table,cleanup,searchreplace,fontselect,fontsizeselect,fontcolor,clipboard,lists,textcase,directionality,charmap,style,hr,fullscreen,xhtmlxtras,visualchars,visualblocks,nonbreaking,anchor,link,imgmanager,spellchecker,article,emotions,layer,autosave,browser,contextmenu,inlinepopups,media,preview,source',1,1,0,'0000-00-00 00:00:00','{\"fontselect\":{\"fonts\":\"{\\\"Andale Mono\\\":\\\"andale mono,times\\\",\\\"Arial\\\":\\\"arial,helvetica,sans-serif\\\",\\\"Arial Black\\\":\\\"arial black,avant garde\\\",\\\"Book Antiqua\\\":\\\"book antiqua,palatino\\\",\\\"Comic Sans MS\\\":\\\"comic sans ms,sans-serif\\\",\\\"Courier New\\\":\\\"courier new,courier\\\",\\\"Georgia\\\":\\\"georgia,palatino\\\",\\\"Helvetica\\\":\\\"helvetica\\\",\\\"Impact\\\":\\\"impact,chicago\\\",\\\"Symbol\\\":\\\"symbol\\\",\\\"Tahoma\\\":\\\"tahoma,arial,helvetica,sans-serif\\\",\\\"Terminal\\\":\\\"terminal,monaco\\\",\\\"Times New Roman\\\":\\\"times new roman,times\\\",\\\"Trebuchet MS\\\":\\\"trebuchet ms,geneva\\\",\\\"Verdana\\\":\\\"verdana,geneva\\\",\\\"Webdings\\\":\\\"webdings\\\",\\\"Wingdings\\\":\\\"wingdings,zapf dingbats\\\"}\"},\"editor\":{\"verify_html\":\"\",\"schema\":\"\",\"allow_css\":\"1\",\"allow_javascript\":\"1\",\"allow_php\":\"1\",\"forced_root_block\":\"\",\"profile_content_css\":\"0\",\"profile_content_css_custom\":\"\\/media\\/font-awesome-4.7.0\\/css\\/font-awesome.css\",\"toolbar_theme\":\"default\"},\"media\":{\"iframes\":\"1\"},\"browser\":{\"max_size\":\"1024\"},\"imgmanager\":{\"extensions\":\"jpeg,jpg,png,apng,gif,webp\"}}'),(2,'Front End','Sample Front-end Profile','','3,4,5','',1,'desktop,tablet,phone','help,newdocument,undo,redo,spacer,bold,italic,underline,strikethrough,justifyfull,justifycenter,justifyleft,justifyright,spacer,formatselect,styleselect;clipboard,searchreplace,indent,outdent,lists,cleanup,charmap,removeformat,hr,sub,sup,textcase,nonbreaking,visualchars,visualblocks;fullscreen,print,visualaid,style,xhtmlxtras,anchor,unlink,link,imgmanager,spellchecker,article','charmap,contextmenu,inlinepopups,help,clipboard,searchreplace,fullscreen,preview,print,style,textcase,nonbreaking,visualchars,visualblocks,xhtmlxtras,imgmanager,anchor,link,spellchecker,article,lists,formatselect,styleselect,hr',0,2,0,'0000-00-00 00:00:00','{\"editor\":{\"toolbar_theme\":\"default\"}}'),(3,'Blogger','Simple Blogging Profile','','3,4,5,6,8,7','',0,'desktop,tablet,phone','bold,italic,strikethrough,lists,blockquote,spacer,justifyleft,justifycenter,justifyright,spacer,link,unlink,imgmanager,article,spellchecker,fullscreen,kitchensink;formatselect,styleselect,underline,justifyfull,clipboard,removeformat,charmap,indent,outdent,undo,redo,help','link,imgmanager,article,spellchecker,fullscreen,kitchensink,clipboard,contextmenu,inlinepopups,lists,formatselect,styleselect,textpattern',0,3,0,'0000-00-00 00:00:00','{\"editor\":{\"toggle\":\"0\",\"toolbar_theme\":\"default\"}}'),(4,'Mobile','Sample Mobile Profile','','3,4,5,6,8,7','',0,'tablet,phone','undo,redo,spacer,bold,italic,underline,formatselect,spacer,justifyleft,justifycenter,justifyfull,justifyright,spacer,fullscreen,kitchensink;styleselect,lists,spellchecker,article,link,unlink','fullscreen,kitchensink,spellchecker,article,link,inlinepopups,lists,formatselect,styleselect,textpattern',0,4,0,'0000-00-00 00:00:00','{\"editor\":{\"toolbar_theme\":\"mobile\",\"resizing\":\"0\",\"resize_horizontal\":\"0\",\"resizing_use_cookie\":\"0\",\"toggle\":\"0\",\"links\":{\"popups\":{\"default\":\"\",\"jcemediabox\":{\"enable\":\"0\"},\"window\":{\"enable\":\"0\"}}}}}'),(5,'Markdown','Sample Markdown Profile','','6,7,3,4,5,8','',0,'desktop,tablet,phone','fullscreen,justifyleft,justifycenter,justifyfull,justifyright,link,unlink,imgmanager,styleselect','fullscreen,link,imgmanager,styleselect,inlinepopups,media,textpattern',0,5,0,'0000-00-00 00:00:00','{\"editor\":{\"toolbar_theme\":\"mobile\"}}');
/*!40000 ALTER TABLE `rlp_wf_profiles` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed
